---
layout: post
title: ADGRE1
date: 2025-01-17 16:55 CST
description: ADGRE1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2015) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2015  | ADGRE1 | ENSG00000174837 | 19p13.3-p13.2 |



The gene is involved in the [adaptive immune response](https://amigo.geneontology.org/amigo/term/GO:0002250) and [cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0007155). It enables [G protein-coupled receptor activity](https://amigo.geneontology.org/amigo/term/GO:0004930) and [calcium ion binding](https://amigo.geneontology.org/amigo/term/GO:0005509). The gene is located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886) and the [external side of plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009897). Additionally, it is involved in the [cell surface receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007166), [G protein-coupled receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007186), and [adenylate cyclase-activating G protein-coupled receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007189).


The gene length is 40,535 base pairs (60.59% of all genes), the mature length is 1,018 base pairs (23.18% of all genes), and the primary transcript length is 40,481 base pairs (70.25% of all genes).


The gene ADGRE1 (NCBI ID: 2015) has been mentioned in [7 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ADGRE1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1995 and the middle 50% of publications occurring between 1998 and 2014. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning ADGRE1, ranked by their relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows:

- [EMR1, the human homolog of F4/80, is an eosinophil-specific receptor.](https://pubmed.ncbi.nlm.nih.gov/17823986) (2007) (relative citation ratio: 2.06)
- [EMR1, an unusual member in the family of hormone receptors with seven transmembrane segments.](https://pubmed.ncbi.nlm.nih.gov/7601460) (1995) (relative citation ratio: 1.96)
- [The eosinophil surface receptor epidermal growth factor-like module containing mucin-like hormone receptor 1 (EMR1): a novel therapeutic target for eosinophilic disorders.](https://pubmed.ncbi.nlm.nih.gov/24530099) (2014) (relative citation ratio: 1.66)
- [The myeloid cell biomarker EMR1 is ectopically expressed in colon cancer.](https://pubmed.ncbi.nlm.nih.gov/34486997) (2021) (relative citation ratio: 0.43)
- [Comparative evaluation of eosinophils in normal mucosa, dysplastic mucosa and oral squamous cell carcinoma with hematoxylin-eosin, Congo red, and EMR1 immunohistochemical staining techniques.](https://pubmed.ncbi.nlm.nih.gov/26120409) (2015) (relative citation ratio: 0.41)

Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ADGRE1](https://www.proteinatlas.org/ENSG00000174837-ADGRE1) is a G-protein coupled receptor with evidence at the protein level. Its RNA is detected in some tissues, and it is expressed in various immune-related cell types, including eosinophils, macrophages, microglia, lymphoma cells, and monocytes. ADGRE1 is involved in immune response processes and is found in lymphoid tissue. In cancer prognostics, ADGRE1 is unprognostic for several cancer types, including Bladder Urothelial Carcinoma, Colon Adenocarcinoma, Glioblastoma Multiforme, Head and Neck Squamous Cell Carcinoma, Kidney Renal Clear Cell Carcinoma, Lung Adenocarcinoma, Lung Squamous Cell Carcinoma, and Rectum Adenocarcinoma.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [STAT3](https://www.ncbi.nlm.nih.gov/gene/6774), [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861), and [SPI1](https://www.ncbi.nlm.nih.gov/gene/6688), each with 3 experiments. Additionally, [SMC1A](https://www.ncbi.nlm.nih.gov/gene/8243) and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) were found to be regulating in 2 experiments each.



The GWAS data indicates associations with a range of conditions, including cardiovascular system disease, cognitive disorder, disease of mental health, substance-related disorder, substance dependence, alcohol dependence, congestive heart failure, heart disease, gastrointestinal system disease, periodontitis, periodontal disease, mouth disease, tooth disease, drug dependence, and cocaine dependence.



The gene is expressed in multiple tissue samples, including lung, blood, and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in various cell types and tissues such as Cardiac Myocytes, B lymphoblasts, CD33+ Myeloid cells, CD56+ NK cells, CD14+ Monocytes, Whole Blood, and Uterus Corpus.


The input data includes various targets related to biological processes and diseases. These targets encompass Class B/2 (Secretin family receptors), Defective ACTH causing Obesity and Pro-opiomelanocortin deficiency (POMCD), GPCR ligand binding, Metabolic disorders of biological oxidation enzymes, Disease, Signaling by GPCR, and Signal Transduction.



The analyzed protein sequence has a GRAVY value of 0.024 (85.09th percentile), indicating a slightly hydrophobic nature. The charge at pH 7.0 is -4.42 (35.18th percentile), and the median structural flexibility is 0.993 (15.78th percentile). The protein has a propensity for helix, sheet, and turn structures of 25.40% (5.96th percentile), 37.58% (76.35th percentile), and 30.25% (65.56th percentile), respectively. The instability index is 48.09 (50.52nd percentile), with an isoelectric point of 6.42 (40.83rd percentile). The protein is 886 amino acids long (83.93rd percentile) and has a molecular weight of 97682.34 Da (83.62nd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |