---
layout: post
title: CDKN2A
date: 2025-01-17 16:55 CST
description: CDKN2A description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1029) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1029  | CDKN2A | ENSG00000147889 | 9p21.3 |



The gene is involved in the [regulation of cyclin-dependent protein serine/threonine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0000079) and the [negative regulation of cell-matrix adhesion](https://amigo.geneontology.org/amigo/term/GO:0001953). It enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and acts as a [cyclin-dependent protein serine/threonine kinase inhibitor](https://amigo.geneontology.org/amigo/term/GO:0004861). Additionally, it enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is active in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), also located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). The gene is involved in the [cell cycle](https://amigo.geneontology.org/amigo/term/GO:0007049), [Ras protein signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007265), and the [negative regulation of cell population proliferation](https://amigo.geneontology.org/amigo/term/GO:0008285). It also enables [protein kinase binding](https://amigo.geneontology.org/amigo/term/GO:0019901) and is involved in the [negative regulation of cell growth](https://amigo.geneontology.org/amigo/term/GO:0030308) and the [negative regulation of NF-kappaB transcription factor activity](https://amigo.geneontology.org/amigo/term/GO:0032088). Furthermore, it is part of the [senescence-associated heterochromatin focus](https://amigo.geneontology.org/amigo/term/GO:0035985) and involved in the [negative regulation of phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0042326) and the [negative regulation of cyclin-dependent protein serine/threonine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0045736). The gene enables [NF-kappaB binding](https://amigo.geneontology.org/amigo/term/GO:0051059) and is involved in the [regulation of cell cycle](https://amigo.geneontology.org/amigo/term/GO:0051726), [cellular senescence](https://amigo.geneontology.org/amigo/term/GO:0090398), and [replicative senescence](https://amigo.geneontology.org/amigo/term/GO:0090399). Additionally, it is involved in the [regulation of G1/S transition of mitotic cell cycle](https://amigo.geneontology.org/amigo/term/GO:2000045) and the [positive regulation of macrophage apoptotic process](https://amigo.geneontology.org/amigo/term/GO:2000111).


The gene length is 20,315 base pairs (42.42% of all genes), the mature length is 831 base pairs, and the primary transcript length is 20,172.5 base pairs.


The gene CDKN2A (NCBI ID: 1029) has been mentioned in [3,806 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CDKN2A%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning this gene is 1971, and the middle 50% of publications occurred between 2003 and 2016.


The top five publications mentioning CDKN2A, ranked by their scientific influence, include "[Oncogenic ras provokes premature cell senescence associated with accumulation of p53 and p16INK4a.](https://pubmed.ncbi.nlm.nih.gov/9054499)" (1997) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 73.26, "[Naturally occurring p16(Ink4a)-positive cells shorten healthy lifespan.](https://pubmed.ncbi.nlm.nih.gov/26840489)" (2016) with an RCR of 62.1, "[Clearance of p16Ink4a-positive senescent cells delays ageing-associated disorders.](https://pubmed.ncbi.nlm.nih.gov/22048312)" (2011) with an RCR of 59.78, "[5' CpG island methylation is associated with transcriptional silencing of the tumour suppressor p16/CDKN2/MTS1 in human cancers.](https://pubmed.ncbi.nlm.nih.gov/7585152)" (1995) with an RCR of 38.48, and "[Role of the INK4a locus in tumor suppression and cell mortality.](https://pubmed.ncbi.nlm.nih.gov/8620534)" (1996) with an RCR of 24.82. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CDKN2A](https://www.proteinatlas.org/ENSG00000147889-CDKN2A) is a DNA-binding protein involved in various cancer-related processes, including Li-Fraumeni syndrome, and functions as a tumor suppressor. It is detected in many tissues and is primarily localized in the nucleoplasm. CDKN2A is associated with cell proliferation in blood, cilium organization in tissues, immune response in the brain, membrane components in kidney cancer cell lines, and mixed functions in single cells.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313), [MAX](https://www.ncbi.nlm.nih.gov/gene/4149), [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), [ZBTB7A](https://www.ncbi.nlm.nih.gov/gene/51341), and [POU5F1](https://www.ncbi.nlm.nih.gov/gene/5460), each shown to be regulating in 5 experiments.



The input data highlights a range of disease conditions associated with various infectious agents and cellular processes. These include infectious diseases such as viral infections and human immunodeficiency virus (HIV) infections, as well as proliferative disorders like cancer, leukemia, and hematologic cancers. Cardiovascular conditions such as coronary artery disease, artery disease, and vascular disease are also noted. Additionally, the data encompasses nervous system diseases, eye diseases including retinal degeneration and glaucoma, and specific types of leukemia such as acute lymphocytic leukemia and lymphoblastic leukemia.


The CDKN2A gene has been associated with several diseases and traits through genome-wide association studies. Notably, it has been linked to [Acute lymphoblastic leukemia in childhood (B cell precursor)](https://pubmed.ncbi.nlm.nih.gov/29632299), which maps closely to the MeSH term [Leukemia](https://meshb.nlm.nih.gov/record/ui?ui=D007938) using the BioWordVec method. Additionally, CDKN2A is associated with [Glycated hemoglobin levels](https://pubmed.ncbi.nlm.nih.gov/33462484), corresponding to the MeSH term [Hemoglobins](https://meshb.nlm.nih.gov/record/ui?ui=D006454). Furthermore, it has been implicated in the [Rate of cognitive decline in Alzheimer's disease](https://pubmed.ncbi.nlm.nih.gov/32573913), which aligns with the MeSH term [Disease](https://meshb.nlm.nih.gov/record/ui?ui=D004194). These associations highlight the diverse roles of CDKN2A in various biological processes and disease states.


The gene is expressed in various tissues, including skin, pituitary, and adrenal gland, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Lymphoma (Burkitt's, Raji), Leukemia (promyelocytic, HL-60), Colorectal adenocarcinoma, Superior Cervical Ganglion, and Uterus Corpus.


The proteins are involved in various pathways related to cellular senescence, including cellular responses to stress, oxidative stress-induced senescence, and the senescence-associated secretory phenotype (SASP). Additionally, they participate in cell cycle regulation, specifically in G1 phase processes such as cyclin D-associated events and mitotic G1-G1/S phases, as well as broader cell cycle and mitotic cell cycle pathways.


The mouse ortholog gene 12578 and its human ortholog gene 1029 are associated with several phenotypic abnormalities, including abnormal eye morphology, abnormal lens morphology, and cataract. These phenotypes are observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.332 (54.11th percentile), indicating a hydrophilic nature. It exhibits a charge of 8.44 at pH 7.0 (78.04th percentile) and a median structural flexibility of 0.9997 (40th percentile). The protein's secondary structure is predicted to be 33.13% helix (60.38th percentile), 23.43% sheet (5.56th percentile), and 31.35% turn (72.93rd percentile). The instability index is 48.53 (52.09th percentile), with an isoelectric point of 8.76 (72.74th percentile). The protein has a length of 144 amino acids (7.32nd percentile) and a molecular weight of 15217.77 Da (6.48th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |