---
layout: post
title: IQGAP1
date: 2025-01-17 16:55 CST
description: IQGAP1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8826) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8826  | IQGAP1 | ENSG00000140575 | 15q26.1 |



The gene is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [microtubule](https://amigo.geneontology.org/amigo/term/GO:0005874), [actin filament](https://amigo.geneontology.org/amigo/term/GO:0005884), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925), and [cell cortex](https://amigo.geneontology.org/amigo/term/GO:0005938). It is involved in the [regulation of cytokine production](https://amigo.geneontology.org/amigo/term/GO:0001817), [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), [epidermal growth factor receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007173), [regulation of mitotic cell cycle](https://amigo.geneontology.org/amigo/term/GO:0007346), and [fibroblast growth factor receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0008543). The gene enables [MAP-kinase scaffold activity](https://amigo.geneontology.org/amigo/term/GO:0005078), [GTPase inhibitor activity](https://amigo.geneontology.org/amigo/term/GO:0005095), [GTPase activator activity](https://amigo.geneontology.org/amigo/term/GO:0005096), [calcium ion binding](https://amigo.geneontology.org/amigo/term/GO:0005509), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [calmodulin binding](https://amigo.geneontology.org/amigo/term/GO:0005516), and [phosphatidylinositol-3,4,5-trisphosphate binding](https://amigo.geneontology.org/amigo/term/GO:0005547). Additionally, it colocalizes with the [actin cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0015629) and [microtubule cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0015630), and is active in the nucleus and cell cortex.


The gene length is 10,374 base pairs (26.54% of all genes), the mature length is 1,219 base pairs, and the primary transcript length is 10,359 base pairs.


The gene IQGAP1 (NCBI ID: 8826) has been mentioned in [265 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22IQGAP1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning this gene is 1996, and the middle 50% of publications occurred between 2009 and 2018.


The top five publications mentioning IQGAP1, ranked by their scientific influence, include "[Rac1 and Cdc42 capture microtubules through IQGAP1 and CLIP-170.](https://pubmed.ncbi.nlm.nih.gov/12110184)" (2002) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 7.68, "[IQGAP1, a calmodulin-binding protein with a rasGAP-related domain, is a potential effector for cdc42Hs.](https://pubmed.ncbi.nlm.nih.gov/8670801)" (1996) with an RCR of 5.95, "[IQGAP1: a key regulator of adhesion and migration.](https://pubmed.ncbi.nlm.nih.gov/15890984)" (2005) with an RCR of 5.34, "[IQGAP1 and its binding proteins control diverse biological functions.](https://pubmed.ncbi.nlm.nih.gov/22182509)" (2012) with an RCR of 4.52, and "[The interaction of IQGAP1 with the exocyst complex is required for tumor cell invasion downstream of Cdc42 and RhoA.](https://pubmed.ncbi.nlm.nih.gov/18541705)" (2008) with an RCR of 4.48. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[IQGAP1](https://www.proteinatlas.org/ENSG00000140575-IQGAP1) is a calmodulin-binding protein with evidence at the protein level. It is detected in all tissues and is primarily localized to the plasma membrane and cell junctions. IQGAP1 is highly expressed in monocytes and neutrophils, particularly during degranulation, and is also found in epithelial extracellular exosomes. In the brain, it is involved in signal transduction within white matter. Additionally, IQGAP1 is expressed in various cell lines, where it participates in nuclear processes, and is notably present in neutrophils during degranulation.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599) with 6 experiments, [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025) with 5 experiments, [PGR](https://www.ncbi.nlm.nih.gov/gene/5241) with 5 experiments, [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664) with 5 experiments, and [EP300](https://www.ncbi.nlm.nih.gov/gene/2033) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.


The top transcription factors (TFs) identified in terms of the number of Perturb-seq experiments are [23028](https://www.ncbi.nlm.nih.gov/gene/23028), which was shown to be regulating genes in 3 experiments, and [2623](https://www.ncbi.nlm.nih.gov/gene/2623), which was shown to be regulating genes in 2 experiments. These findings are based on data from GTRD's database of TF binding sites from uniform CHIP-seq data and Perturb-seq data from a study that examined how changes in TFs affect gene expression.


The input data indicates a wide range of disease conditions associated with various biological systems. These include several types of cancer, such as leukemia, lymphoma, and immune system cancer, as well as respiratory diseases like asthma, bronchial disease, and obstructive lung disease. Metabolic disorders are also prominent, with conditions like diabetes mellitus, particularly type 2 diabetes mellitus, and diseases related to carbohydrate and glucose metabolism. Additionally, the data encompasses cardiovascular system diseases and nervous system diseases, including neurodegenerative conditions like amyotrophic lateral sclerosis and motor neuron disease. Other notable conditions include hematopoietic system diseases, immune system diseases, and various metabolic disorders such as lipid storage disease and lysosomal storage disease.


IQGAP1 has been associated with several traits and diseases through genome-wide association studies. Notably, it has been linked to [Basophil count](https://pubmed.ncbi.nlm.nih.gov/32888493), which maps closely to the MeSH term [Basophils](https://meshb.nlm.nih.gov/record/ui?ui=D001491) using the BioWordVec method. Additionally, IQGAP1 is associated with [Basophil percentage of granulocytes](https://pubmed.ncbi.nlm.nih.gov/27863252) and [Basophil percentage of white cells](https://pubmed.ncbi.nlm.nih.gov/27863252), both of which also map to [Basophils](https://meshb.nlm.nih.gov/record/ui?ui=D001491). Other associations include [Eczema](https://pubmed.ncbi.nlm.nih.gov/30595370), which directly matches the MeSH term [Eczema](https://meshb.nlm.nih.gov/record/ui?ui=D004485), and [Male-pattern baldness](https://pubmed.ncbi.nlm.nih.gov/28196072), which maps to the MeSH term [Alopecia](https://meshb.nlm.nih.gov/record/ui?ui=D000505). Furthermore, IQGAP1 is linked to [Red cell distribution width](https://pubmed.ncbi.nlm.nih.gov/30595370), mapping to [Erythrocyte Indices](https://meshb.nlm.nih.gov/record/ui?ui=D004909), and both [Testosterone levels](https://pubmed.ncbi.nlm.nih.gov/35192695) and [Total testosterone levels](https://pubmed.ncbi.nlm.nih.gov/32042192), which map to [Testosterone](https://meshb.nlm.nih.gov/record/ui?ui=D013739). Lastly, it is associated with [White blood cell count (basophil)](https://pubmed.ncbi.nlm.nih.gov/27863252), mapping to [Blood](https://meshb.nlm.nih.gov/record/ui?ui=D001769).


The gene is expressed in various immune cell types, including CD33+ myeloid cells, BDCA4+ dendritic cells, CD56+ NK cells, CD8+ T cells, and CD19+ B cells (neg. sel.), as well as CD14+ monocytes and whole blood. Additionally, expression is observed in bronchial epithelial cells.


The proteins are involved in various pathways, including metabolism, nephrin interactions, glucagon-like peptide-1 (GLP1) regulation of insulin secretion, integration of energy metabolism, regulation of insulin secretion, and cell-cell communication.


The mouse ortholog gene 29875 and its human ortholog gene 8826 are associated with the phenotype 'short tibia', which is observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.495 (35.44th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -15.52 (14.8th percentile). The median structural flexibility is 1.004 (59.02nd percentile), with a significant portion of the sequence predicted to be in helical (37.9%, 88.35th percentile) and sheet (34.94%, 58.63rd percentile) conformations. The instability index is 46.2 (43.69th percentile), and the isoelectric point is 6.08 (32.64th percentile). The protein has a length of 1657 amino acids (96th percentile) and a molecular weight of 189249.7 Da (96.26th percentile). For detailed sequence analysis methods, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |