---
layout: post
title: YWHAE
date: 2025-01-17 16:55 CST
description: YWHAE description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7531) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7531  | YWHAE | ENSG00000108953 | 17p13.3 |



The gene is involved in various biological processes, including the [MAPK cascade](https://amigo.geneontology.org/amigo/term/GO:0000165), [neuron migration](https://amigo.geneontology.org/amigo/term/GO:0001764), and [cytoplasmic pattern recognition receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0002753). It plays a role in the [regulation of heart rate by hormone](https://amigo.geneontology.org/amigo/term/GO:0003064) and is crucial for [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165) and [regulation of mitotic cell cycle](https://amigo.geneontology.org/amigo/term/GO:0007346). The gene enables functions such as [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [calcium channel regulator activity](https://amigo.geneontology.org/amigo/term/GO:0005246), and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925), and colocalizes with the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). Additionally, it is involved in [protein targeting](https://amigo.geneontology.org/amigo/term/GO:0006605) and various developmental processes such as [substantia nigra development](https://amigo.geneontology.org/amigo/term/GO:0021762), [hippocampus development](https://amigo.geneontology.org/amigo/term/GO:0021766), and [cerebral cortex development](https://amigo.geneontology.org/amigo/term/GO:0021987). The gene also enables activities like [potassium channel regulator activity](https://amigo.geneontology.org/amigo/term/GO:0015459), [enzyme binding](https://amigo.geneontology.org/amigo/term/GO:0019899), and [protein phosphatase binding](https://amigo.geneontology.org/amigo/term/GO:0019903). It is active in the [mitochondrial membrane](https://amigo.geneontology.org/amigo/term/GO:0031966) and located in the [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020) and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). The gene is involved in processes such as [protein localization to nucleus](https://amigo.geneontology.org/amigo/term/GO:0034504), [cellular response to heat](https://amigo.geneontology.org/amigo/term/GO:0034605), and [intracellular signal transduction](https://amigo.geneontology.org/amigo/term/GO:0035556). It enables [signaling adaptor activity](https://amigo.geneontology.org/amigo/term/GO:0035591) and is located in the [melanosome](https://amigo.geneontology.org/amigo/term/GO:0042470). The gene is also involved in the [positive regulation of protein export from nucleus](https://amigo.geneontology.org/amigo/term/GO:0046827) and enables [protein heterodimerization activity](https://amigo.geneontology.org/amigo/term/GO:0046982). It plays a role in the [regulation of cytosolic calcium ion concentration](https://amigo.geneontology.org/amigo/term/GO:0051480) and [regulation of membrane repolarization](https://amigo.geneontology.org/amigo/term/GO:0060306). Furthermore, it is involved in [membrane repolarization during cardiac muscle cell action potential](https://amigo.geneontology.org/amigo/term/GO:0086013) and [regulation of heart rate by cardiac conduction](https://amigo.geneontology.org/amigo/term/GO:0086091). The gene enables [scaffold protein binding](https://amigo.geneontology.org/amigo/term/GO:0097110) and [protein sequestering activity](https://amigo.geneontology.org/amigo/term/GO:0140311). It is involved in the [regulation of potassium ion transmembrane transporter activity](https://amigo.geneontology.org/amigo/term/GO:1901016), [negative regulation of calcium ion transmembrane transporter activity](https://amigo.geneontology.org/amigo/term/GO:1901020), [negative regulation of peptidyl-serine dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:1902309), and [negative regulation of calcium ion export across plasma membrane](https://amigo.geneontology.org/amigo/term/GO:1905913).


The gene length is 49,167 base pairs (65.82 percentile of all genes), the mature length is 9,621 base pairs (98.84 percentile of all genes), and the primary transcript length is 49,167 base pairs (74.82 percentile of all genes).


The gene YWHAE (NCBI ID: 7531) has been mentioned in [132 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22YWHAE%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1996 and the middle 50% of publications occurring between 2011 and 2019. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning YWHAE, ranked by their relative citation ratio (RCR), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications (set at 1.0), are as follows:

1. [The clinicopathologic features of YWHAE-FAM22 endometrial stromal sarcomas: a histologically high-grade and clinically aggressive tumor.](https://pubmed.ncbi.nlm.nih.gov/22456610) (2012) (relative citation ratio: 6.58)
2. [Recurrent BCOR Internal Tandem Duplication and YWHAE-NUTM2B Fusions in Soft Tissue Undifferentiated Round Cell Sarcoma of Infancy: Overlapping Genetic Features With Clear Cell Sarcoma of Kidney.](https://pubmed.ncbi.nlm.nih.gov/26945340) (2016) (relative citation ratio: 5.66)
3. [14-3-3epsilon is important for neuronal migration by binding to NUDEL: a molecular explanation for Miller-Dieker syndrome.](https://pubmed.ncbi.nlm.nih.gov/12796778) (2003) (relative citation ratio: 5.43)
4. [The mitochondrial targeting chaperone 14-3-3Îµ regulates a RIG-I translocon that mediates membrane association and innate antiviral immunity.](https://pubmed.ncbi.nlm.nih.gov/22607805) (2012) (relative citation ratio: 4.08)
5. [DISC1 regulates the transport of the NUDEL/LIS1/14-3-3epsilon complex through kinesin-1.](https://pubmed.ncbi.nlm.nih.gov/17202468) (2007) (relative citation ratio: 3.81)

The citation count is sourced from [iCite](https://icite.od.nih.gov).


[YWHAE](https://www.proteinatlas.org/ENSG00000108953-YWHAE) is a gene involved in cancer-related processes, with evidence supporting its role at the protein level. It is detected in all RNA tissue distributions and is primarily localized in the cytosol. The gene is part of Cluster 10 for blood expression, which is associated with non-specific membrane trafficking, and Cluster 54 for tissue expression, linked to basic cellular processes. In the brain, it is part of Cluster 28, which is associated with oligodendrocytes and mixed functions. Additionally, YWHAE is part of Cluster 42 for cell line expression and Cluster 15 for single cell expression, both of which have non-specific and mixed functions.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SP1](https://www.ncbi.nlm.nih.gov/gene/6667) and [KDM1A](https://www.ncbi.nlm.nih.gov/gene/23028), each with 6 experiments. Additionally, [CHD8](https://www.ncbi.nlm.nih.gov/gene/57680), [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), and [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664) were each found to be regulating in 5 experiments.



The GWAS data indicates associations with various conditions, including cognitive disorder, disease of mental health, mood disorder, and mental depression.



The gene is expressed in various brain regions, including the subthalamic nucleus, parietal lobe, occipital lobe, cerebellum, and cerebellum peduncles.


The input data highlights various cellular processes and pathways, including cellular responses to stress, cell cycle regulation (particularly the mitotic phase), apoptosis, and programmed cell death. Additionally, it covers centrosome maturation, mitotic centrosome protein recruitment, and the regulation of PLK1 activity during the G2/M transition. The data also encompasses organelle biogenesis, signaling pathways such as those mediated by NGF and p75 NTR, and heat shock response regulation via HSF1. Furthermore, it includes the intrinsic pathway for apoptosis, Hippo signaling, and membrane trafficking processes like GLUT4 translocation to the plasma membrane.



The analyzed protein sequence has a GRAVY value of -0.54 (31.03rd percentile), indicating a hydrophilic nature. The charge at pH 7 is -19.19 (11.69th percentile), and the median structural flexibility is 1.003 (57.21st percentile). The protein has a high affinity for helix structures (0.427, 96.77th percentile), with lower affinities for sheet (0.322, 40.4th percentile) and turn (0.243, 16.98th percentile) structures. The instability index is 41.39 (26.66th percentile), and the isoelectric point is 4.63 (2.38th percentile). The protein is 255 amino acids long (22.37th percentile) with a molecular weight of 29173.58 Da (23.34th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |