---
layout: post
title: FTH1
date: 2024-12-08 23:57 CST
description: FTH1 description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2495) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2495  | FTH1 | ENSG00000167996 | 11q12.3 |



The gene length is 1,959 base pairs (5.17% of all genes). The mature length and primary transcript length are also 1,959 base pairs.


The gene FTH1 (NCBI ID: 2495) has been the subject of [280 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22FTH1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication dating back to 1946. The middle 50% of these publications were published between 1991 and 2018.


The top five publications mentioning FTH1, ranked by their scientific influence, include "[Inhibiting Ferroptosis through Disrupting the NCOA4-FTH1 Interaction: A New Mechanism of Action.](https://pubmed.ncbi.nlm.nih.gov/34235259)" (2021) (relative citation ratio: 15.09), "[Ferritin heavy chain upregulation by NF-kappaB inhibits TNFalpha-induced apoptosis by suppressing reactive oxygen species.](https://pubmed.ncbi.nlm.nih.gov/15537542)" (2004) (relative citation ratio: 10.24), "[Curcumenol triggered ferroptosis in lung cancer cells via lncRNA H19/miR-19b-3p/FTH1 axis.](https://pubmed.ncbi.nlm.nih.gov/35224289)" (2021) (relative citation ratio: 7.0), "[Electron density map of apoferritin at 2.8-A resolution.](https://pubmed.ncbi.nlm.nih.gov/563983)" (1978) (relative citation ratio: 6.24), and "[The formation of ferritin from apoferritin. Kinetics and mechanism of iron uptake.](https://pubmed.ncbi.nlm.nih.gov/5075227)" (1972) (relative citation ratio: 6.09). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [AR](https://www.ncbi.nlm.nih.gov/gene/367) with 6 experiments, [JUN](https://www.ncbi.nlm.nih.gov/gene/3725) with 5 experiments, [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599) with 5 experiments, [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930) with 5 experiments, and [TP53](https://www.ncbi.nlm.nih.gov/gene/7157) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.


The top transcription factors (TFs) identified from the GTRD's database of TF binding sites, filtered for Perturb-seq data, include [TCF7L2](https://www.ncbi.nlm.nih.gov/gene/4149) with 4 experiments, [ZNF384](https://www.ncbi.nlm.nih.gov/gene/6872) with 3 experiments, [ZNF274](https://www.ncbi.nlm.nih.gov/gene/9968) with 3 experiments, [ZBTB33](https://www.ncbi.nlm.nih.gov/gene/4602) with 2 experiments, and [ZNF263](https://www.ncbi.nlm.nih.gov/gene/8861) with 2 experiments. These TFs were found to regulate gene expression in multiple Perturb-seq experiments, highlighting their significant roles in transcriptional regulation.





The mouse ortholog gene 14319 and its human ortholog gene 2495 are associated with several phenotypes. Both sexes exhibit abnormal eye morphology and preweaning lethality with complete penetrance. Additionally, the gene is linked to embryonic lethality prior to organogenesis and prior to the tooth bud stage, although these phenotypes do not affect both sexes equally.


The protein sequence analyzed has a GRAVY value of -0.773 (13.08th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -10.62 (21.58th percentile), and the median structural flexibility is 1.005 (63.03rd percentile). The protein has a high affinity for helix structures (37.70%, 87.75th percentile), with lower affinities for sheet (31.15%, 34.81st percentile) and turn (26.78%, 36.12th percentile) structures. The instability index is 43.63 (34.38th percentile), and the isoelectric point is 5.31 (13.58th percentile). The protein is 183 amino acids long (11.98th percentile) with a molecular weight of 21225.40 Da (12.95th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |