---
layout: post
title: VPS26A
date: 2025-01-17 16:55 CST
description: VPS26A description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9559) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9559  | VPS26A | ENSG00000122958 | 10q22.1 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is located in the [lysosome](https://amigo.geneontology.org/amigo/term/GO:0005764), [endosome](https://amigo.geneontology.org/amigo/term/GO:0005768), [early endosome](https://amigo.geneontology.org/amigo/term/GO:0005769), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0010008), [vesicle](https://amigo.geneontology.org/amigo/term/GO:0031982), and [tubular endosome](https://amigo.geneontology.org/amigo/term/GO:0097422). It is involved in [intracellular protein transport](https://amigo.geneontology.org/amigo/term/GO:0006886), [regulation of macroautophagy](https://amigo.geneontology.org/amigo/term/GO:0016241), [endocytic recycling](https://amigo.geneontology.org/amigo/term/GO:0032456), and [retrograde transport, endosome to Golgi](https://amigo.geneontology.org/amigo/term/GO:0042147). Additionally, it is part of the [retromer complex](https://amigo.geneontology.org/amigo/term/GO:0030904) and the [retromer, cargo-selective complex](https://amigo.geneontology.org/amigo/term/GO:0030906).


The gene length is 1,956 base pairs (5.16% of all genes), the mature length is 501 base pairs, and the primary transcript length is 1,567 base pairs.


The gene VPS26A (NCBI ID: 9559) has been mentioned in [8 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22VPS26A%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning this gene was in 2001, and the middle 50% of publications occurred between 2010 and 2018.


The top 5 publications mentioning VPS26A, ranked by their relative citation ratio (RCR), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows. The most influential publication is "[Phospholipase PLA2G6, a Parkinsonism-Associated Gene, Affects Vps26 and Vps35, Retromer Function, and Ceramide Levels, Similar to Î±-Synuclein Gain.](https://pubmed.ncbi.nlm.nih.gov/29909971)" (2018) with a relative citation ratio of 5.53. This is followed by "[High glucose-mediated VPS26a down-regulation dysregulates neuronal amyloid precursor protein processing and tau phosphorylation.](https://pubmed.ncbi.nlm.nih.gov/35297035)" (2022) with a relative citation ratio of 2.08. Other notable publications include "[Vps26A and Vps26B subunits define distinct retromer complexes.](https://pubmed.ncbi.nlm.nih.gov/21920005)" (2011) with a relative citation ratio of 1.74, and "[Rabankyrin-5 interacts with EHD1 and Vps26 to regulate endocytic trafficking and retromer function.](https://pubmed.ncbi.nlm.nih.gov/22284051)" (2012) with a relative citation ratio of 1.04. Additionally, "[Novel crosstalk between Vps26a and Nox4 signaling during neurogenesis.](https://pubmed.ncbi.nlm.nih.gov/30464227)" (2019) has a relative citation ratio of 0.4. For more information on RCR, visit [here](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[VPS26A](https://www.proteinatlas.org/ENSG00000122958-VPS26A) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in endosomes and lysosomes. The gene is expressed in various clusters, including Cluster 42 for non-specific vesicular transport in blood, Cluster 4 for basic cellular processes in testis, Cluster 55 for non-specific transcription in the brain, Cluster 52 for non-specific mitochondria in cell lines, and Cluster 27 for syncytiotrophoblasts related to pregnancy in single-cell expression. The gene is unprognostic for multiple cancer types, including Bladder Urothelial Carcinoma, Breast Invasive Carcinoma, and others.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [YY1](https://www.ncbi.nlm.nih.gov/gene/7528) with 7 experiments, [KMT2B](https://www.ncbi.nlm.nih.gov/gene/9757) with 6 experiments, [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170) with 5 experiments, [TCF12](https://www.ncbi.nlm.nih.gov/gene/6938) with 5 experiments, and [RXRA](https://www.ncbi.nlm.nih.gov/gene/6256) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The input data indicates associations with various metabolic disorders, including diseases of metabolism, acquired metabolic diseases, carbohydrate metabolism diseases, glucose metabolism diseases, diabetes mellitus, and specifically type 2 diabetes mellitus.



The gene is expressed in adipose tissue, nerve, and skin according to the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles indicate expression in CD34+ cells, CD56+ NK cells, CD105+ endothelial cells, and various testis tissues including interstitial, seminiferous tubule, and germ cells.


The input data highlights several pathways involving WNT signaling, including 'Signaling by WNT in cancer', 'Signaling by Wnt', 'WNT ligand biogenesis and trafficking', and 'WNT ligand secretion is abrogated by the PORCN inhibitor LGK974'. Additionally, the data mentions broader categories such as 'Disease' and 'Signal Transduction'.



The analyzed protein sequence has a GRAVY value of -0.426 (43.09th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -3.75 (37.27th percentile). The median structural flexibility is 1.004 (60.8th percentile), with a predicted secondary structure composition of 33.94% helix (67.36th percentile), 39.14% sheet (83.56th percentile), and 22.94% turn (9.67th percentile). The instability index is 46.12 (43.37th percentile), and the isoelectric point is 6.13 (34.01st percentile). The protein is 327 amino acids long (34.19th percentile) with a molecular weight of 38169.45 Da (36.72nd percentile). For more details on the sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |