---
layout: post
title: RAB10
date: 2025-01-17 16:55 CST
description: RAB10 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10890) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10890  | RAB10 | ENSG00000084733 | 2p23.3 |



The gene is located in the [Golgi membrane](https://amigo.geneontology.org/amigo/term/GO:0000139), [endosome](https://amigo.geneontology.org/amigo/term/GO:0005768), [endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0005789), [Golgi apparatus](https://amigo.geneontology.org/amigo/term/GO:0005794), [trans-Golgi network](https://amigo.geneontology.org/amigo/term/GO:0005802), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0005856), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [adherens junction](https://amigo.geneontology.org/amigo/term/GO:0005912), [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925), [cilium](https://amigo.geneontology.org/amigo/term/GO:0005929), [endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0010008), [cytoplasmic vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030659), [secretory granule membrane](https://amigo.geneontology.org/amigo/term/GO:0030667), [phagocytic vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030670), [insulin-responsive compartment](https://amigo.geneontology.org/amigo/term/GO:0032593), [recycling endosome](https://amigo.geneontology.org/amigo/term/GO:0055037), [recycling endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0055038), [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062), [exocytic vesicle](https://amigo.geneontology.org/amigo/term/GO:0070382), and [endoplasmic reticulum tubular network](https://amigo.geneontology.org/amigo/term/GO:0071782). It is part of the [exocyst](https://amigo.geneontology.org/amigo/term/GO:0000145) and is active in the [Golgi apparatus](https://amigo.geneontology.org/amigo/term/GO:0005794), [insulin-responsive compartment](https://amigo.geneontology.org/amigo/term/GO:0032593), and [recycling endosome](https://amigo.geneontology.org/amigo/term/GO:0055037). The gene enables [G protein activity](https://amigo.geneontology.org/amigo/term/GO:0003925), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [GTP binding](https://amigo.geneontology.org/amigo/term/GO:0005525), [GDP binding](https://amigo.geneontology.org/amigo/term/GO:0019003), [myosin V binding](https://amigo.geneontology.org/amigo/term/GO:0031489), and [cadherin binding involved in cell-cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0098641). It is involved in various processes including [Golgi to plasma membrane transport](https://amigo.geneontology.org/amigo/term/GO:0006893), [axonogenesis](https://amigo.geneontology.org/amigo/term/GO:0007409), [vesicle-mediated transport](https://amigo.geneontology.org/amigo/term/GO:0016192), [endosomal transport](https://amigo.geneontology.org/amigo/term/GO:0016197), [antigen processing and presentation](https://amigo.geneontology.org/amigo/term/GO:0019882), [polarized epithelial cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0030859), [cellular response to insulin stimulus](https://amigo.geneontology.org/amigo/term/GO:0032869), [Golgi to plasma membrane protein transport](https://amigo.geneontology.org/amigo/term/GO:0043001), [regulated exocytosis](https://amigo.geneontology.org/amigo/term/GO:0045055), [establishment of neuroblast polarity](https://amigo.geneontology.org/amigo/term/GO:0045200), [endoplasmic reticulum tubular network organization](https://amigo.geneontology.org/amigo/term/GO:0071786), [protein localization to plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0072659), [establishment of protein localization to membrane](https://amigo.geneontology.org/amigo/term/GO:0090150), [establishment of protein localization to endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0097051), [cell-cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0098609), and [protein localization to basolateral plasma membrane](https://amigo.geneontology.org/amigo/term/GO:1903361).


The gene length is 23,109 base pairs (45.62% of all genes), the mature length is 1,404 base pairs, and the primary transcript length is 12,752 base pairs (39.74% of all genes).


The gene RAB10 (NCBI ID: 10890) has been mentioned in [81 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RAB10%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning RAB10 is 1998, and the middle 50% of publications occurred between 2014 and 2020.


The top 5 publications mentioning RAB10, ranked by their scientific influence, include studies that explore the gene's role in various biological processes. The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Notable publications include [<i>LRRK2</i> mutations impair depolarization-induced mitophagy through inhibition of mitochondrial accumulation of RAB10.](https://pubmed.ncbi.nlm.nih.gov/30945962) (2020) (relative citation ratio: 8.05), [Rab10, a target of the AS160 Rab GAP, is required for insulin-stimulated translocation of GLUT4 to the adipocyte plasma membrane.](https://pubmed.ncbi.nlm.nih.gov/17403373) (2007) (relative citation ratio: 5.6), [Interrogating Parkinson's disease LRRK2 kinase pathway activity by assessing Rab10 phosphorylation in human neutrophils.](https://pubmed.ncbi.nlm.nih.gov/29127255) (2018) (relative citation ratio: 4.93), [Phos-tag analysis of Rab10 phosphorylation by LRRK2: a powerful assay for assessing kinase function and inhibitors.](https://pubmed.ncbi.nlm.nih.gov/27474410) (2016) (relative citation ratio: 4.48), and [Pathogenic LRRK2 regulates ciliation probability upstream of tau tubulin kinase 2 via Rab10 and RILPL1 proteins.](https://pubmed.ncbi.nlm.nih.gov/33653948) (2021) (relative citation ratio: 4.34). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RAB10](https://www.proteinatlas.org/ENSG00000084733-RAB10) is a gene encoding a protein with hydrolase molecular function. It is involved in neurodegeneration and has evidence at the protein level. The gene is detected in all tissues and is associated with various expression clusters, including Cluster 40 in monocytes related to innate immune response, Cluster 19 in non-specific tissues involved in basic cellular processes, Cluster 27 in the brain with mixed function, Cluster 7 in liver cancer cell lines related to metabolism, and Cluster 59 in suprabasal keratinocytes involved in cornification.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ZBTB7A](https://www.ncbi.nlm.nih.gov/gene/51341) and [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313), each with 5 experiments. Additionally, [STAT1](https://www.ncbi.nlm.nih.gov/gene/6772), [E2F6](https://www.ncbi.nlm.nih.gov/gene/1876), and [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227) were each found to be regulating in 4 experiments.



The GWAS data indicates associations with a wide range of disease conditions, including various types of cancer such as leukemia, lymphoma, and organ system cancer. Additionally, the data highlights infectious diseases, particularly viral infections and smallpox. Respiratory conditions such as asthma, bronchial disease, and obstructive lung disease are also implicated. Furthermore, the data includes hematologic and immune system disorders, as well as cardiovascular and mental health conditions, encompassing cognitive disorders, mood disorders, and psychotic disorders.



The gene is expressed in various tissues, including the esophagus, vagina, and salivary gland, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the prefrontal cortex, CD33+ myeloid cells, CD14+ monocytes, fetal brain, spinal cord, colon, and tongue.


The input data highlights two key pathways: membrane trafficking and the translocation of GLUT4 to the plasma membrane. These processes are integral to cellular function and glucose homeostasis, respectively.


The mouse ortholog gene 19325 and its human ortholog 10890 are associated with several phenotypic abnormalities. Both sexes exhibit decreased mean corpuscular hemoglobin, decreased mean corpuscular volume, iris synechia, and complete penetrance preweaning lethality. Additionally, there is evidence of embryonic lethality prior to organogenesis, although this is not observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.33 (54.4th percentile), indicating a hydrophilic nature. It exhibits a charge of 2.75 at pH 7.0 (60.35th percentile) and a median structural flexibility of 1.003 (57.4th percentile). The protein's secondary structure is predicted to be 32% helix (50.31st percentile), 38% sheet (78.49th percentile), and 27.5% turn (42.7th percentile). The instability index is 29.38 (3.52nd percentile), with an isoelectric point of 8.58 (68.38th percentile). The protein has a length of 200 amino acids (14.5th percentile) and a molecular weight of 22540.67 Da (14.87th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |