---
layout: post
title: DYNC1I1
date: 2025-01-17 16:55 CST
description: DYNC1I1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1780) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1780  | DYNC1I1 | ENSG00000158560 | 7q21.3 |



The gene is located in the [kinetochore](https://amigo.geneontology.org/amigo/term/GO:0000776), [spindle pole](https://amigo.geneontology.org/amigo/term/GO:0000922), [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [microtubule](https://amigo.geneontology.org/amigo/term/GO:0005874), [vesicle](https://amigo.geneontology.org/amigo/term/GO:0031982), [cytoplasmic ribonucleoprotein granule](https://amigo.geneontology.org/amigo/term/GO:0036464), [perinuclear region of cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0048471), and [recycling endosome](https://amigo.geneontology.org/amigo/term/GO:0055037). It is part of the [cytoplasmic dynein complex](https://amigo.geneontology.org/amigo/term/GO:0005868). The gene enables [cytoskeletal motor activity](https://amigo.geneontology.org/amigo/term/GO:0003774), [microtubule motor activity](https://amigo.geneontology.org/amigo/term/GO:0003777), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [microtubule binding](https://amigo.geneontology.org/amigo/term/GO:0008017), [spectrin binding](https://amigo.geneontology.org/amigo/term/GO:0030507), [dynein light chain binding](https://amigo.geneontology.org/amigo/term/GO:0045503), and [dynein heavy chain binding](https://amigo.geneontology.org/amigo/term/GO:0045504). It is involved in [transport along microtubule](https://amigo.geneontology.org/amigo/term/GO:0010970) and [vesicle transport along microtubule](https://amigo.geneontology.org/amigo/term/GO:0047496).


The gene length is 1,730 base pairs (4.59% of all genes), the mature length is 794 base pairs, and the primary transcript length is 1,649 base pairs (5.06% of all genes).


The gene DYNC1I1 (NCBI ID: 1780) has been mentioned in [11 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22DYNC1I1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 2014 and the middle 50% of publications occurring between 2015 and 2021. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning DYNC1I1, ranked by their scientific influence, include studies that explore the gene's role in various cancers and developmental conditions. The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Notable publications include "[CircCCNB1 silencing acting as a miR-106b-5p sponge inhibited GPM6A expression to promote HCC progression by enhancing DYNC1I1 expression and activating the AKT/ERK signaling pathway.](https://pubmed.ncbi.nlm.nih.gov/35002514)" (2022) (relative citation ratio: 3.01), "[DYNC1I1 Promotes the Proliferation and Migration of Gastric Cancer by Up-Regulating IL-6 Expression.](https://pubmed.ncbi.nlm.nih.gov/31249807)" (2019) (relative citation ratio: 1.26), and "[Deletions of exons with regulatory activity at the DYNC1I1 locus are associated with split-hand/split-foot malformation: array CGH screening of 134 unrelated families.](https://pubmed.ncbi.nlm.nih.gov/25231166)" (2014) (relative citation ratio: 0.79). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[DYNC1I1](https://www.proteinatlas.org/ENSG00000158560-DYNC1I1) is a motor protein with evidence at the protein level. It is detected in many tissues and is particularly expressed in neuronal cells, as indicated by its tissue expression cluster (Cluster 28: Neuronal - Transcription) and brain expression cluster (Cluster 1: Neurons - Mixed function). In blood, it is associated with neutrophils and protein binding (Cluster 35). Additionally, it is found in ciliated cells with ion channel activity (Cluster 10) and is involved in neuronal signaling in single cells (Cluster 34).


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) and [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), each with 4 experiments. Additionally, [ELF1](https://www.ncbi.nlm.nih.gov/gene/1997), [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025), and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) were each found to be regulating in 3 experiments.



The input data highlights a wide range of disease conditions associated with various infectious agents, metabolic disorders, and systemic diseases. These include viral infectious diseases such as human immunodeficiency virus (HIV) and smallpox, as well as respiratory conditions like lower respiratory tract disease and lung disease. Metabolic disorders are also prominent, with a focus on diabetes mellitus, particularly type 2 diabetes, and obesity. Cardiovascular and nervous system diseases are well-represented, including hypertension, heart disease, and neurodegenerative conditions like Alzheimer's disease and tauopathy. Additionally, the data encompasses musculoskeletal system diseases such as arthritis and osteoarthritis, along with immune system and hematopoietic system diseases.



The gene is expressed in various tissues, including the bladder, brain, and pituitary, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the pineal gland (night), amygdala, whole brain, fetal brain, hypothalamus, subthalamic nucleus, and occipital lobe.


The proteins are involved in several key immune-related pathways, including the adaptive immune system, the broader immune system, and MHC class II antigen presentation.



The analyzed protein sequence has a GRAVY value of -0.607 (24.68th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -35.59 (4.71st percentile), and the median structural flexibility is 1.004 (61.36th percentile). The protein has a length of 645 amino acids (71.85th percentile) and a molecular weight of 72954.51 Da (72.35th percentile). The secondary structure affinities are 30.7% for helix (38.06th percentile), 33.18% for sheet (46.5th percentile), and 29.3% for turn (58.37th percentile). The instability index is 51.44 (61.85th percentile), and the isoelectric point is 4.997 (7.72nd percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |