---
layout: post
title: TBCA
date: 2025-01-17 16:55 CST
description: TBCA description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6902) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6902  | TBCA | ENSG00000171530 | 5q14.1 |



The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is located in the [nucleolus](https://amigo.geneontology.org/amigo/term/GO:0005730), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), and [microtubule](https://amigo.geneontology.org/amigo/term/GO:0005874). It is involved in [protein folding](https://amigo.geneontology.org/amigo/term/GO:0006457), [tubulin complex assembly](https://amigo.geneontology.org/amigo/term/GO:0007021), and the [post-chaperonin tubulin folding pathway](https://amigo.geneontology.org/amigo/term/GO:0007023). Additionally, it is active in and located in the [microtubule cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0015630), enabling [tubulin binding](https://amigo.geneontology.org/amigo/term/GO:0015631), [beta-tubulin binding](https://amigo.geneontology.org/amigo/term/GO:0048487), and [protein-folding chaperone binding](https://amigo.geneontology.org/amigo/term/GO:0051087).


The gene length is 7,001 base pairs (18.96% of all genes), the mature length is 747 base pairs, and the primary transcript length is 1,386.5 base pairs.


The gene TBCA, with NCBI ID 6902, has been mentioned in [23 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22TBCA%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning TBCA is 1987, and the middle 50% of publications occurred between 1998 and 2014.


The top 5 publications mentioning TBCA, ranked by their relative citation ratio (RCR), include "[Colchicine Blocks Tubulin Heterodimer Recycling by Tubulin Cofactors TBCA, TBCB, and TBCE.](https://pubmed.ncbi.nlm.nih.gov/33968934)" (2021) (RCR: 1.3), "[Poly(l-lysine)-graft-folic acid-coupled poly(2-methyl-2-oxazoline) (PLL-g-PMOXA-c-FA): a bioactive copolymer for specific targeting to folate receptor-positive cancer cells.](https://pubmed.ncbi.nlm.nih.gov/25581478)" (2015) (RCR: 1.26), "[CfaE tip mutations in enterotoxigenic Escherichia coli CFA/I fimbriae define critical human intestinal binding sites.](https://pubmed.ncbi.nlm.nih.gov/19207729)" (2009) (RCR: 1.18), "[Tubulin cofactor A functions as a novel positive regulator of ccRCC progression, invasion and metastasis.](https://pubmed.ncbi.nlm.nih.gov/23740643)" (2013) (RCR: 1.08), and "[Cofactor A is a molecular chaperone required for beta-tubulin folding: functional and structural characterization.](https://pubmed.ncbi.nlm.nih.gov/8756698)" (1996) (RCR: 0.97). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[TBCA](https://www.proteinatlas.org/ENSG00000171530-TBCA) is a gene that functions as a chaperone, with evidence supporting its role at the protein level. It is detected in all RNA tissue distributions and is primarily localized in the nucleoli and microtubules. The gene is expressed in various clusters, including Cluster 13 in blood, which is non-specific to mitochondria, Cluster 69 in tissue, which is specific to the liver with oxidoreductase activity, Cluster 56 in the brain, which is specific to the hypothalamus with neuropeptide signaling, Cluster 22 in cell lines, which is non-specific to mRNA splicing and the cell cycle, and Cluster 43 in single cells, which is specific to hepatocytes with metabolism.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [HDAC2](https://www.ncbi.nlm.nih.gov/gene/3066) with 6 experiments, [TP53](https://www.ncbi.nlm.nih.gov/gene/7157) and [AR](https://www.ncbi.nlm.nih.gov/gene/367) each with 5 experiments, and [PPARG](https://www.ncbi.nlm.nih.gov/gene/5468) and [MYC](https://www.ncbi.nlm.nih.gov/gene/4609) each with 4 experiments.



The GWAS data indicates associations with a wide range of disease conditions, including various types of cancer such as leukemia, lymphoma, and immune system cancers. Additionally, respiratory diseases like asthma, bronchial disease, and obstructive lung disease are implicated. Metabolic disorders, particularly diabetes mellitus and type 2 diabetes mellitus, are also linked. Furthermore, the data highlights associations with cardiovascular, kidney, urinary, nervous, and sensory system diseases, including specific conditions like retinal disease.



The gene is expressed in various tissues, including the ovary and adrenal gland, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in pineal gland (both day and night), Burkitt's lymphoma (Raji and Daudi cell lines), promyelocytic leukemia (HL-60 cell line), CD105+ endothelial cells, and lymphoblastic leukemia (MOLT-4 cell line).


The input data highlights several key pathways involved in protein processing, including metabolism of proteins, protein folding, and the post-chaperonin tubulin folding pathway.



The analyzed protein sequence has a GRAVY value of -0.944 (5.78th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -3.47 (37.99th percentile), and the median structural flexibility is 1.02 (96.6th percentile). The protein has a high affinity for helix structures (52.78%, 99.84th percentile), with lower affinities for sheet (26.85%, 13.91st percentile) and turn (13.89%, 0.15th percentile) structures. The instability index is 57.33 (77.77th percentile), and the isoelectric point is 5.25 (12.51st percentile). The protein is 108 amino acids long (3.45th percentile) with a molecular weight of 12854.69 Da (4.22nd percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |