---
layout: post
title: IFNAR1
date: 2025-01-17 16:55 CST
description: IFNAR1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3454) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3454  | IFNAR1 | ENSG00000142166 | 21q22.11 |



The gene enables [type I interferon receptor activity](https://amigo.geneontology.org/amigo/term/GO:0004905) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is involved in [receptor signaling pathway via JAK-STAT](https://amigo.geneontology.org/amigo/term/GO:0007259), [response to virus](https://amigo.geneontology.org/amigo/term/GO:0009615), [response to lipopolysaccharide](https://amigo.geneontology.org/amigo/term/GO:0032496), [cellular response to interferon-alpha](https://amigo.geneontology.org/amigo/term/GO:0035457), [cellular response to interferon-beta](https://amigo.geneontology.org/amigo/term/GO:0035458), [regulation of receptor signaling pathway via JAK-STAT](https://amigo.geneontology.org/amigo/term/GO:0046425), [type I interferon-mediated signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0060337), [cellular response to virus](https://amigo.geneontology.org/amigo/term/GO:0098586), and [positive regulation of cellular respiration](https://amigo.geneontology.org/amigo/term/GO:1901857). It is located in the [lysosome](https://amigo.geneontology.org/amigo/term/GO:0005764), [late endosome](https://amigo.geneontology.org/amigo/term/GO:0005770), and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). Additionally, it enables [JAK pathway signal transduction adaptor activity](https://amigo.geneontology.org/amigo/term/GO:0008269), [cytokine binding](https://amigo.geneontology.org/amigo/term/GO:0019955), and [type I interferon binding](https://amigo.geneontology.org/amigo/term/GO:0019962).


The gene length is 46,613 base pairs (64.54% of all genes), the mature length is 2,223 base pairs, and the primary transcript length is 24,902 base pairs.


The gene IFNAR1 (NCBI ID: 3454) has been mentioned in [155 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22IFNAR1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1982, and the middle 50% of publications range from 2007 to 2020.


The top five publications mentioning IFNAR1, ranked by their scientific influence, include "[A Case of Autosomal Recessive Interferon Alpha/Beta Receptor Alpha Chain (IFNAR1) Deficiency with Severe COVID-19.](https://pubmed.ncbi.nlm.nih.gov/34713375)" (2022) (relative citation ratio: 5.32), "[Inherited IFNAR1 deficiency in otherwise healthy patients with adverse reaction to measles and yellow fever live vaccines.](https://pubmed.ncbi.nlm.nih.gov/31270247)" (2019) (relative citation ratio: 5.03), "[Lack of Neuronal IFN-Î²-IFNAR Causes Lewy Body- and Parkinson's Disease-like Dementia.](https://pubmed.ncbi.nlm.nih.gov/26451483)" (2015) (relative citation ratio: 4.52), "[STAT3 as an adapter to couple phosphatidylinositol 3-kinase to the IFNAR1 chain of the type I interferon receptor.](https://pubmed.ncbi.nlm.nih.gov/9162009)" (1997) (relative citation ratio: 3.92), and "[Nucleoside-Modified mRNA Vaccines Protect IFNAR<sup>-/-</sup> Mice against Crimean-Congo Hemorrhagic Fever Virus Infection.](https://pubmed.ncbi.nlm.nih.gov/34817199)" (2022) (relative citation ratio: 3.72). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[IFNAR1](https://www.proteinatlas.org/ENSG00000142166-IFNAR1) is a receptor involved in cancer-related genes and is an FDA-approved drug target. Evidence for its function is available at the protein level. The gene is detected in all RNA tissue distributions and is associated with various expression clusters, including Cluster 2 in blood (Dendritic cells - Unknown function), Cluster 40 in tissue (Non-specific - Transcription), Cluster 27 in the brain (Non-specific - Mixed function), Cluster 51 in cell lines (Myeloma - Immunoglobulins), and Cluster 77 in single cells (Plasma cells - Protein processing).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [EP300](https://www.ncbi.nlm.nih.gov/gene/2033) and [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227), each with 6 experiments. Additionally, [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025), [TFAP2C](https://www.ncbi.nlm.nih.gov/gene/7022), and [TAL1](https://www.ncbi.nlm.nih.gov/gene/6886) were each found to be regulating in 5 experiments.



The GWAS data indicates associations with several disease conditions, including artery disease, vascular disease, cardiovascular system disease, hypertension, and broader categories such as disease and disease of anatomical entity.


Genome-wide association studies have identified several traits related to IFNAR1. Notably, a study ([reference](https://pubmed.ncbi.nlm.nih.gov/34648354)) found associations with [Interferon alpha/beta receptor 1 levels](https://pubmed.ncbi.nlm.nih.gov/34648354), which was mapped to the MeSH term [Interferons](https://meshb.nlm.nih.gov/record/ui?ui=D007372) using the BioWordVec method. Another study ([reference](https://pubmed.ncbi.nlm.nih.gov/29875488)) also reported on [Interferon alpha/beta receptor 1 levels (IFNAR1.9183.7.3)](https://pubmed.ncbi.nlm.nih.gov/29875488), similarly mapped to [Interferons](https://meshb.nlm.nih.gov/record/ui?ui=D007372). Additionally, a study ([reference](https://pubmed.ncbi.nlm.nih.gov/35078996)) investigated [Serum levels of protein IFNAR1](https://pubmed.ncbi.nlm.nih.gov/35078996), which was mapped to the MeSH term [Proteins](https://meshb.nlm.nih.gov/record/ui?ui=D011506). These findings highlight the importance of IFNAR1 in various biological contexts.


The gene is ubiquitously expressed in multiple tissue samples, including liver, adipose tissue, and blood, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the prefrontal cortex, cardiac myocytes, Burkitt's lymphoma (Raji), heart, liver, 721 B lymphoblasts, CD33+ myeloid cells, and thyroid.


The proteins are involved in various immune-related pathways, including cytokine signaling in the immune system, interferon signaling, interferon alpha/beta signaling, regulation of IFNA signaling, and the broader immune system.


The mouse ortholog gene 15975 and its human ortholog gene 3454 exhibit several phenotypic associations. Both sexes of mice with alterations in this gene show a decreased number of KLRG1-positive NK cells and monocytes, as well as increased circulating levels of amylase and creatinine.


The analyzed protein sequence has a GRAVY value of -0.277 (60.58th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -9.74 (22.95th percentile), and the median structural flexibility is 1.001 (45.04th percentile). The protein has a propensity for helix, turn, and sheet structures of 29.08% (24.16th percentile), 28.01% (47.37th percentile), and 40.57% (87.83rd percentile), respectively. The instability index is 46.23 (43.81st percentile), with an isoelectric point of 5.58 (20.22nd percentile). The protein length is 557 amino acids (64.73rd percentile), and the molecular weight is 63524.55 Da (65.91st percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |