---
layout: post
title: KPNA5
date: 2025-01-17 16:55 CST
description: KPNA5 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3841) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3841  | KPNA5 | ENSG00000196911 | 6q22.1 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [nuclear localization sequence binding](https://amigo.geneontology.org/amigo/term/GO:0008139), and is involved in [protein import into nucleus](https://amigo.geneontology.org/amigo/term/GO:0006606) and [NLS-bearing protein import into nucleus](https://amigo.geneontology.org/amigo/term/GO:0006607). It is active in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), and is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654) and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). Additionally, it is part of the [NLS-dependent protein nuclear import complex](https://amigo.geneontology.org/amigo/term/GO:0042564) and enables [nuclear import signal receptor activity](https://amigo.geneontology.org/amigo/term/GO:0061608).


The gene length is 14,650 base pairs (34.54% of all genes), the mature length is 1,318 base pairs, and the primary transcript length is 8,022 base pairs.


There are no publications that mention this gene.



[KPNA5](https://www.proteinatlas.org/ENSG00000196911-KPNA5) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily located in the cytosol and is expressed in various clusters, including Cluster 32 in basophils related to transcription, Cluster 16 in non-specific tissues related to transcription, Cluster 37 in neurons with mixed functions, Cluster 10 in ciliated cells with ion channel activity, and Cluster 70 in spermatocytes and spermatids involved in spermatid development.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [CDK7](https://www.ncbi.nlm.nih.gov/gene/1022) with 5 experiments, [MAX](https://www.ncbi.nlm.nih.gov/gene/4149) with 4 experiments, [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 4 experiments, [TAL1](https://www.ncbi.nlm.nih.gov/gene/6886) with 4 experiments, and [AR](https://www.ncbi.nlm.nih.gov/gene/367) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with a range of conditions, including coronary artery disease, artery disease, vascular disease, and cardiovascular system disease. Additionally, the data highlights links to arteriosclerosis, arteriosclerotic cardiovascular disease, and atherosclerosis. Furthermore, there are associations with bone disease, connective tissue disease, and various musculoskeletal system diseases, such as bone remodeling disease, bone resorption disease, muscle tissue disease, myopathy, and muscular disease.



The gene is expressed in various tissues and cell types, including CD8+ T cells, chronic myelogenous leukemia K-562 cells, the globus pallidus, and multiple testis-related tissues such as interstitial cells, Leydig cells, seminiferous tubules, and germ cells. Additionally, the gene is expressed in the ovary.


The proteins are involved in several key pathways related to immune response, including the antiviral mechanism by IFN-stimulated genes, ISG15 antiviral mechanism, cytokine signaling in the immune system, interferon signaling, and the broader immune system.



The analyzed protein sequence has a GRAVY value of -0.265 (62.1% percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -16.992 (13.53% percentile), and the median structural flexibility is 1.001 (47.91% percentile). The protein has a helix affinity of 0.323 (53.1% percentile), a sheet affinity of 0.354 (62.27% percentile), and a turn affinity of 0.276 (44.08% percentile). The instability index is 52.869 (66.29% percentile), with an isoelectric point of 5.026 (8.29% percentile). The protein length is 539 amino acids (62.98% percentile), and the molecular weight is 60665.63 Da (63.42% percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |