---
layout: post
title: GOSR2
date: 2025-01-17 16:55 CST
description: GOSR2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9570) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9570  | GOSR2 | ENSG00000108433 | 17q21.32 |



The gene is located in the [Golgi membrane](https://amigo.geneontology.org/amigo/term/GO:0000139), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0005789), [Golgi apparatus](https://amigo.geneontology.org/amigo/term/GO:0005794), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [ER to Golgi transport vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0012507), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), and [endoplasmic reticulum-Golgi intermediate compartment membrane](https://amigo.geneontology.org/amigo/term/GO:0033116). It enables [SNARE binding](https://amigo.geneontology.org/amigo/term/GO:0000149), [SNAP receptor activity](https://amigo.geneontology.org/amigo/term/GO:0005484), and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is active in the [endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0005789), [Golgi apparatus](https://amigo.geneontology.org/amigo/term/GO:0005794), [ER to Golgi transport vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0012507), and [late endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0031902). It is involved in [intra-Golgi vesicle-mediated transport](https://amigo.geneontology.org/amigo/term/GO:0006891), [vesicle fusion](https://amigo.geneontology.org/amigo/term/GO:0006906), and [protein transport](https://amigo.geneontology.org/amigo/term/GO:0015031). Additionally, it is part of the [SNARE complex](https://amigo.geneontology.org/amigo/term/GO:0031201).


The gene length is 39,616 base pairs (60.11% of all genes), the mature length is 651 base pairs (7.38% of all genes), and the primary transcript length is 3,380 base pairs (12.48% of all genes).


The gene GOSR2 (NCBI ID: 9570) has been mentioned in [16 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22GOSR2%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1999 and the middle 50% of publications occurring between 2012 and 2017.


The top 5 publications mentioning GOSR2 include "[A mutation in the Golgi Qb-SNARE gene GOSR2 causes progressive myoclonus epilepsy with early ataxia.](https://pubmed.ncbi.nlm.nih.gov/21549339)" (2011) (relative citation ratio: 2.05), "[TRAPPC11 and GOSR2 mutations associate with hypoglycosylation of Î±-dystroglycan and muscular dystrophy.](https://pubmed.ncbi.nlm.nih.gov/29855340)" (2018) (relative citation ratio: 1.58), "[Ramsay Hunt syndrome: clinical characterization of progressive myoclonus ataxia caused by GOSR2 mutation.](https://pubmed.ncbi.nlm.nih.gov/24458321)" (2014) (relative citation ratio: 1.4), "[North Sea' progressive myoclonus epilepsy: phenotype of subjects with GOSR2 mutation.](https://pubmed.ncbi.nlm.nih.gov/23449775)" (2013) (relative citation ratio: 1.31), and "[Targeting of Arf-1 to the early Golgi by membrin, an ER-Golgi SNARE.](https://pubmed.ncbi.nlm.nih.gov/15781476)" (2005) (relative citation ratio: 1.18). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[GOSR2](https://www.proteinatlas.org/ENSG00000108433-GOSR2) is implicated in several diseases, including congenital muscular dystrophy, epilepsy, and neurodegeneration. Evidence for its involvement is supported at the protein level. The gene is detected in all RNA tissue distributions and is localized in the nucleoplasm and Golgi apparatus. Its expression is associated with various clusters, including Cluster 5 for non-specific cell proliferation in blood, Cluster 24 for vesicular transport in the parathyroid gland, Cluster 15 for non-specific transcription in the brain, Cluster 27 in breast cancer cell lines, and Cluster 11 in late spermatids.


The top transcription factors (TFs) identified from the GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170), [PGR](https://www.ncbi.nlm.nih.gov/gene/5241), [KDM1A](https://www.ncbi.nlm.nih.gov/gene/23028), [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908), and [GATA1](https://www.ncbi.nlm.nih.gov/gene/2623), each shown to be regulating in 5 experiments.



The input data indicates associations with various disease conditions, including cancer, particularly leukemia and its subtypes such as acute leukemia, acute lymphocytic leukemia, and lymphoblastic leukemia. Additionally, there are associations with immune system cancer, organ system cancer, and hematologic cancer. Cardiovascular system diseases are also implicated, including artery disease, vascular disease, hypertension, and heart conduction disease. Furthermore, the data encompasses broader categories such as disease of cellular proliferation, disease of anatomical entity, and general disease conditions.


The gene GOSR2 has been associated with [Hemoglobin levels](https://pubmed.ncbi.nlm.nih.gov/32327693), which maps closely to the MeSH term [Hemoglobins](https://meshb.nlm.nih.gov/record/ui?ui=D006454) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method.


The gene is expressed in the pituitary and spleen according to BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles indicate expression in cardiac myocytes, heart, liver, superior cervical ganglion, trigeminal ganglion, skeletal muscle, and uterus corpus.


The input data highlights several key pathways involved in protein regulation and stress response. These include the metabolism of proteins, the activation of chaperones by IRE1alpha, and the activation of chaperone genes by XBP1(S). Additionally, the unfolded protein response (UPR) pathway is noted, which is crucial for managing cellular stress caused by misfolded proteins.


The mouse ortholog gene 56494 and its human ortholog 9570 are associated with phenotypes including abnormal gait and preweaning lethality with complete penetrance, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.5495 (29.97th percentile), indicating a hydrophilic nature. It exhibits a charge of 1.216 at pH 7.0 (54.76th percentile) and a median structural flexibility of 1.001 (47.49th percentile). The protein's secondary structure is predicted to be 31.6% helix (46.74th percentile), 35.85% sheet (65.24th percentile), and 22.64% turn (8.56th percentile). The instability index is 57.57 (78.3rd percentile), suggesting potential instability. The isoelectric point is 7.86 (57.49th percentile), with a molecular weight of 24775.13 Da (17.69th percentile) and a length of 212 amino acids (16.3rd percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |