---
layout: post
title: PPP1CA
date: 2025-01-17 16:55 CST
description: PPP1CA description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5499) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5499  | PPP1CA | ENSG00000172531 | 11q13.2 |



The gene colocalizes with the [chromosome, telomeric region](https://amigo.geneontology.org/amigo/term/GO:0000781) and is located in various cellular components, including the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [nucleolus](https://amigo.geneontology.org/amigo/term/GO:0005730), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [adherens junction](https://amigo.geneontology.org/amigo/term/GO:0005912), [glycogen granule](https://amigo.geneontology.org/amigo/term/GO:0042587), [dendritic spine](https://amigo.geneontology.org/amigo/term/GO:0043197), [perikaryon](https://amigo.geneontology.org/amigo/term/GO:0043204), [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062), [presynapse](https://amigo.geneontology.org/amigo/term/GO:0098793), and [glutamatergic synapse](https://amigo.geneontology.org/amigo/term/GO:0098978). It enables several molecular functions, such as [phosphoprotein phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0004721), [protein serine/threonine phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0004722), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [protein phosphatase 1 binding](https://amigo.geneontology.org/amigo/term/GO:0008157), [phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0016791), [myosin phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0017018), [ribonucleoprotein complex binding](https://amigo.geneontology.org/amigo/term/GO:0043021), [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872), and [cadherin binding involved in cell-cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0098641). Additionally, it is involved in numerous biological processes, including [glycogen metabolic process](https://amigo.geneontology.org/amigo/term/GO:0005977), [regulation of glycogen biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0005979), [regulation of glycogen catabolic process](https://amigo.geneontology.org/amigo/term/GO:0005981), [regulation of translational initiation](https://amigo.geneontology.org/amigo/term/GO:0006446), [protein dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0006470), [dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0016311), [cell cycle](https://amigo.geneontology.org/amigo/term/GO:0007049), [response to lead ion](https://amigo.geneontology.org/amigo/term/GO:0010288), [lung development](https://amigo.geneontology.org/amigo/term/GO:0030324), [circadian regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0032922), [regulation of circadian rhythm](https://amigo.geneontology.org/amigo/term/GO:0042752), [entrainment of circadian clock by photoperiod](https://amigo.geneontology.org/amigo/term/GO:0043153), [branching morphogenesis of an epithelial tube](https://amigo.geneontology.org/amigo/term/GO:0048754), [cell division](https://amigo.geneontology.org/amigo/term/GO:0051301), [regulation of canonical Wnt signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0060828), [cell-cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0098609), and [positive regulation of extrinsic apoptotic signaling pathway in absence of ligand](https://amigo.geneontology.org/amigo/term/GO:2001241). Furthermore, it is part of the [PTW/PP1 phosphatase complex](https://amigo.geneontology.org/amigo/term/GO:0072357).


The gene length is 32,658 base pairs (54.94% of all genes), the mature length is 3,521 base pairs, and the primary transcript length is 32,553 base pairs.


The gene PPP1CA (NCBI ID: 5499) has been mentioned in [49 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PPP1CA%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1994. The middle 50% of these publications were published between 2010 and 2019.


The top 5 publications mentioning PPP1CA, ranked by their relative citation ratio (RCR), include studies that explore various biological processes and disease mechanisms. The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. The most influential publication is ['Processing of the SARS-CoV pp1a/ab nsp7-10 region.'](https://pubmed.ncbi.nlm.nih.gov/32083638) (2020) (relative citation ratio: 5.61), followed by ['The Smad4-MYO18A-PP1A complex regulates β-catenin phosphorylation and pemigatinib resistance by inhibiting PAK1 in cholangiocarcinoma.'](https://pubmed.ncbi.nlm.nih.gov/34799729) (2022) (relative citation ratio: 4.91). Another notable study is ['SRPK1/2 and PP1α exert opposite functions by modulating SRSF1-guided MKNK2 alternative splicing in colon adenocarcinoma.'](https://pubmed.ncbi.nlm.nih.gov/33602301) (2021) (relative citation ratio: 4.84). Additionally, ['USP11 promotes growth and metastasis of colorectal cancer via PPP1CA-mediated activation of ERK/MAPK signaling pathway.'](https://pubmed.ncbi.nlm.nih.gov/31521612) (2019) (relative citation ratio: 3.08) and ['Canonical Wnt signalling activates TAZ through PP1A during osteogenic differentiation.'](https://pubmed.ncbi.nlm.nih.gov/24510127) (2014) (relative citation ratio: 2.68) are also highly cited. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PPP1CA](https://www.proteinatlas.org/ENSG00000172531-PPP1CA) is a gene encoding a protein with hydrolase and protein phosphatase molecular functions. Evidence for this gene is available at the protein level. Its RNA is detected in all tissues, with notable expression in squamous epithelium, where it is involved in innate immune response. The protein is localized in various subcellular compartments, including the nucleoplasm, plasma membrane, cytosol, flagellar centriole, mid piece, and end piece. The gene's expression in blood is non-specific with an unknown function, while in tissues, it is associated with squamous epithelium and innate immune response. In the brain, its expression is non-specific with mixed functions. Similarly, its expression in cell lines and single cells is non-specific with unknown and mixed functions, respectively.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861) and [SP1](https://www.ncbi.nlm.nih.gov/gene/6667), each with 6 experiments. Additionally, [MAX](https://www.ncbi.nlm.nih.gov/gene/4149), [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599), and [PGR](https://www.ncbi.nlm.nih.gov/gene/5241) were each found to be regulating in 5 experiments.





The gene is expressed in various tissues, including blood and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles show expression in the heart, CD56+ NK cells, colorectal adenocarcinoma, and thymus.


The input data highlights various pathways involving lipid metabolism, including lipid digestion, mobilization, and transport, as well as the broader metabolism of lipids and lipoproteins. Additionally, it covers several aspects of TGF-beta receptor signaling, such as its downregulation, activation of SMADs, and various loss-of-function mutations in SMAD2/3, SMAD4, TGFBR1, and TGFBR2 in cancer. The data also includes pathways related to disease, signaling by GPCR, signal transduction, opioid signaling, DARPP-32 events, and hormone-sensitive lipase (HSL)-mediated triacylglycerol hydrolysis.



The analyzed protein sequence has a GRAVY value of -0.301 (57.84th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -4.06 (36.43rd percentile), and the median structural flexibility is 0.994 (19.2nd percentile). The protein has a propensity for helix, sheet, and turn structures of 31.21% (43.17th percentile), 35.15% (60.29th percentile), and 30.30% (65.97th percentile), respectively. The instability index is 44.05 (35.88th percentile), with an isoelectric point of 5.94 (29.02nd percentile). The sequence length is 330 amino acids (34.73rd percentile), and the molecular weight is 37511.66 Da (35.82nd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |