---
layout: post
title: GFRA2
date: 2025-01-17 16:55 CST
description: GFRA2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2675) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2675  | GFRA2 | ENSG00000168546 | 8p21.3 |



The gene is located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886) and the [extrinsic component of membrane](https://amigo.geneontology.org/amigo/term/GO:0019898), and is part of the [receptor complex](https://amigo.geneontology.org/amigo/term/GO:0043235). It is involved in the [transmembrane receptor protein tyrosine kinase signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007169), [nervous system development](https://amigo.geneontology.org/amigo/term/GO:0007399), and the [glial cell-derived neurotrophic factor receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0035860). The gene is active on the [external side of plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009897) and enables [glial cell-derived neurotrophic factor receptor activity](https://amigo.geneontology.org/amigo/term/GO:0016167).


The gene length is 50,130 base pairs (66.38% of all genes), the mature length is 784 base pairs (13.13% of all genes), and the primary transcript length is 34,253 base pairs (66.17% of all genes).


The gene GFRA2 (NCBI ID: 2675) has been mentioned in [27 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22GFRA2%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1997 and the middle 50% of publications occurring between 2000 and 2014. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning GFRA2, ranked by their scientific influence, include "[TrnR2, a novel receptor that mediates neurturin and GDNF signaling through Ret.](https://pubmed.ncbi.nlm.nih.gov/9182803)" (1997) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 6.12, "[GFRalpha-2 and GFRalpha-3 are two new receptors for ligands of the GDNF family.](https://pubmed.ncbi.nlm.nih.gov/9407096)" (1997) with an RCR of 4.13, "[A genome-wide association study suggests an association of Chr8p21.3 (GFRA2) with diabetic neuropathic pain.](https://pubmed.ncbi.nlm.nih.gov/24974787)" (2015) with an RCR of 2.27, "[GFRÎ±2 prompts cell growth and chemoresistance through down-regulating tumor suppressor gene PTEN via Mir-17-5p in pancreatic cancer.](https://pubmed.ncbi.nlm.nih.gov/27400681)" (2016) with an RCR of 1.93, and "[Cloning, mRNA distribution and chromosomal localisation of the gene for glial cell line-derived neurotrophic factor receptor beta, a homologue to GDNFR-alpha.](https://pubmed.ncbi.nlm.nih.gov/9259272)" (1997) with an RCR of 1.53. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[GFRA2](https://www.proteinatlas.org/ENSG00000168546-GFRA2) is a receptor with evidence at the protein level. It is detected in many tissues and is primarily located in vesicles and the plasma membrane. GFRA2 is part of Cluster 25 in blood expression, indicating non-specific and unknown function, and Cluster 44 in tissue expression, which is associated with testis and spermatogenesis. In the brain, it belongs to Cluster 53, linked to neurons with mixed functions. Additionally, GFRA2 is found in Cluster 17 in cell line expression, related to neuroblastoma and neuronal signaling, and Cluster 36 in single-cell expression, associated with late spermatids. The gene is unprognostic in several cancer types, including Breast Invasive Carcinoma, Colon Adenocarcinoma, Glioblastoma Multiforme, Ovary Serous Cystadenocarcinoma, Rectum Adenocarcinoma, Testicular Germ Cell Tumor, and Thyroid Carcinoma.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [EZH2](https://www.ncbi.nlm.nih.gov/gene/2146) with 5 experiments, [MYC](https://www.ncbi.nlm.nih.gov/gene/4609) with 4 experiments, [EP300](https://www.ncbi.nlm.nih.gov/gene/2033) with 4 experiments, [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) with 4 experiments, and [AR](https://www.ncbi.nlm.nih.gov/gene/367) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with a wide range of disease conditions, including leukocyte disease, leukopenia, hematopoietic system disease, immune system disease, disease of metabolism, acquired metabolic disease, artery disease, vascular disease, cardiovascular system disease, kidney disease, urinary system disease, cognitive disorder, disease of mental health, arteriosclerosis, arteriosclerotic cardiovascular disease, atherosclerosis, agranulocytosis, neutropenia, mineral metabolism disease, calcium metabolism disease, hepatobiliary disease, liver disease, and gastrointestinal system disease.



The gene is expressed in the thyroid and spleen according to the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles indicate that the gene is ubiquitously expressed in multiple brain regions, including the Prefrontal Cortex, Whole Brain, Cingulate Cortex, Temporal Lobe, Medulla Oblongata, Pons, Cerebellum, and Globus Pallidus.


The proteins are involved in several key pathways, including axon guidance, developmental biology, NCAM signaling for neurite outgrowth, and NCAM1 interactions.



The analyzed protein sequence has a GRAVY value of -0.355 (51.58th percentile), indicating a hydrophilic nature. It exhibits a charge of 3.79 at pH 7.0 (63.87th percentile) and a median structural flexibility of 1.001 (46.42nd percentile). The protein's secondary structure is predicted to be 29.31% helix (25.89th percentile), 30.60% sheet (31.49th percentile), and 30.82% turn (69.38th percentile). The instability index is 58.13 (79.41st percentile), with an isoelectric point of 8.00 (58.8th percentile). The protein has a length of 464 amino acids (53.63rd percentile) and a molecular weight of 51543.17 Da (53.39th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |