---
layout: post
title: EEF2
date: 2025-01-17 16:55 CST
description: EEF2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1938) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1938  | EEF2 | ENSG00000167658 | 19p13.3 |



The gene is involved in [hematopoietic progenitor cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0002244) and [translational elongation](https://amigo.geneontology.org/amigo/term/GO:0006414), and it plays a role in the [positive regulation of translation](https://amigo.geneontology.org/amigo/term/GO:0045727). It enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [translation elongation factor activity](https://amigo.geneontology.org/amigo/term/GO:0003746), [GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0003924), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [GTP binding](https://amigo.geneontology.org/amigo/term/GO:0005525), [protein kinase binding](https://amigo.geneontology.org/amigo/term/GO:0019901), [cadherin binding](https://amigo.geneontology.org/amigo/term/GO:0045296), and [ribosome binding](https://amigo.geneontology.org/amigo/term/GO:0043022). The gene is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [ribosome](https://amigo.geneontology.org/amigo/term/GO:0005840), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [aggresome](https://amigo.geneontology.org/amigo/term/GO:0016235), [secretory granule lumen](https://amigo.geneontology.org/amigo/term/GO:0034774), [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062), [ficolin-1-rich granule lumen](https://amigo.geneontology.org/amigo/term/GO:1904813), and [synapse](https://amigo.geneontology.org/amigo/term/GO:0045202). Additionally, it is part of the [ribonucleoprotein complex](https://amigo.geneontology.org/amigo/term/GO:1990904).


The gene length is 403,101 base pairs (97.45th percentile), the mature length is 3,427 base pairs, and the primary transcript length is 402,693 base pairs (98.84th percentile).


The gene EEF2 (NCBI ID: 1938) has been mentioned in [192 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22EEF2%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1972 and the middle 50% of publications occurring between 1988 and 2013. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning EEF2, ranked by their scientific influence, include studies that explore the phosphorylation and ADP-ribosylation of elongation factor 2, as well as its role in protein synthesis and depression models. The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. The most influential publications are: [Activation of AMP-activated protein kinase leads to the phosphorylation of elongation factor 2 and an inhibition of protein synthesis.](https://pubmed.ncbi.nlm.nih.gov/12194824) (2002) (RCR: 7.01), [Mechanism of action of Pseudomonas aeruginosa exotoxin Aiadenosine diphosphate-ribosylation of mammalian elongation factor 2 in vitro and in vivo.](https://pubmed.ncbi.nlm.nih.gov/188760) (1977) (RCR: 5.88), [Identification of the major Mr 100,000 substrate for calmodulin-dependent protein kinase III in mammalian cells as elongation factor-2.](https://pubmed.ncbi.nlm.nih.gov/3693353) (1987) (RCR: 5.78), [Fluoxetine regulates eEF2 activity (phosphorylation) via HDAC1 inhibitory mechanism in an LPS-induced mouse model of depression.](https://pubmed.ncbi.nlm.nih.gov/33526073) (2021) (RCR: 5.31), and [ADP-ribosylation of elongation factor 2 by diphtheria toxin. NMR spectra and proposed structures of ribosyl-diphthamide and its hydrolysis products.](https://pubmed.ncbi.nlm.nih.gov/7430147) (1980) (RCR: 5.17). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[EEF2](https://www.proteinatlas.org/ENSG00000167658-EEF2) is a gene with molecular functions as an elongation factor and hydrolase. It is involved in various diseases, including cancer-related genes, neurodegeneration, and spinocerebellar ataxia, and is a target for FDA-approved drugs. Evidence for EEF2 is available at the protein level. The gene is detected in all RNA tissue distributions and is localized in the plasma membrane and cytosol. In terms of expression clusters, EEF2 is associated with non-specific translation in blood, ribosome in tissue, endothelial cells in the brain, and ribosome in cell lines and single cells.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 5 experiments, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 5 experiments, [SPI1](https://www.ncbi.nlm.nih.gov/gene/6688) with 5 experiments, [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113) with 4 experiments, and [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.





The gene is expressed in various tissues, including the pancreas and ovary, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Lymphoma (Burkitt's, Raji), Thyroid, Leukemia (Chronic Myelogenous, K-562), Cerebellum, and Prostate.


The proteins are involved in various pathways, including post-translational protein modification, metabolism of proteins, gamma carboxylation, hypusine formation and arylsulfatase activation, synthesis of diphthamide-EEF2, gene expression, eukaryotic translation elongation, peptide chain elongation, translation, uptake and function of diphtheria toxin, disease, and uptake and actions of bacterial toxins.



The protein sequence analyzed has a GRAVY value of -0.209, indicating a hydrophilic nature (68.15th percentile). It exhibits a charge of -4.13 at pH 7 (36.13th percentile) and a median structural flexibility of 1.000 (41.8th percentile). The protein's secondary structure is predicted to be 33.8% helix (66.28th percentile), 35.0% sheet (58.9th percentile), and 27.4% turn (41.69th percentile). The instability index is 35.58 (11.33rd percentile), with an isoelectric point of 6.41 (40.51st percentile). The protein has a length of 858 amino acids (82.95th percentile) and a molecular weight of 95337.08 Da (82.75th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |