---
layout: post
title: PNRC1
date: 2024-12-08 23:57 CST
description: PNRC1 description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10957) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10957  | PNRC1 | ENSG00000146278 | 6q15 |



The gene length is 3,729 base pairs (10.36% of all genes), the mature length is 1,417 base pairs (35.47% of all genes), and the primary transcript length is 2,767 base pairs (9.76% of all genes).


The gene PNRC1, with NCBI ID 10957, has been featured in [7 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PNRC1%22) (count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1981, and the middle 50% of publications span from 1982 to 2014.


The top 5 publications mentioning PNRC1, ranked by their scientific influence, include "[Cage-Like B41 (+) and B42 (2+) : New Chiral Members of the Borospherene Family.](https://pubmed.ncbi.nlm.nih.gov/26014224)" (2015) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 2.94, "[A combined NMR/DFT study on the ion pair structure of [(PR1(2)R2)Au(Î·2-3-hexyne)]BF4 complexes.](https://pubmed.ncbi.nlm.nih.gov/23160563)" (2013) with an RCR of 0.65, "[miR-199a Targeting PNRC1 to Promote Keratinocyte Proliferation and Invasion in Cholesteatoma.](https://pubmed.ncbi.nlm.nih.gov/34825000)" (2021) with an RCR of 0.38, "[Improved separation and quantification of the "middle molecule" b4-2 in uremia.](https://pubmed.ncbi.nlm.nih.gov/6831703)" (1983) with an RCR of 0.35, and "[Characterization of sub-peak b4.2, middle molecule.](https://pubmed.ncbi.nlm.nih.gov/7295091)" (1981) with an RCR of 0.25. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), [CDK8](https://www.ncbi.nlm.nih.gov/gene/1024), [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), and [MYC](https://www.ncbi.nlm.nih.gov/gene/4609), each shown to be regulating in 5 experiments.


The top transcription factors (TFs) identified from the GTRD's database of TF binding sites, filtered for Perturb-seq data, include [CTCF](https://www.ncbi.nlm.nih.gov/gene/6829) and [BRCA1](https://www.ncbi.nlm.nih.gov/gene/6872), each shown to be regulating genes in 4 experiments. Additionally, [ZNF274](https://www.ncbi.nlm.nih.gov/gene/8243) and [HNF4A](https://www.ncbi.nlm.nih.gov/gene/9126) were also found to regulate genes in 4 experiments. [ZBTB33](https://www.ncbi.nlm.nih.gov/gene/23028) and [GABPA](https://www.ncbi.nlm.nih.gov/gene/4849) were each involved in 3 experiments, while [NRF1](https://www.ncbi.nlm.nih.gov/gene/2623), [ZNF143](https://www.ncbi.nlm.nih.gov/gene/123169), and [RAD21](https://www.ncbi.nlm.nih.gov/gene/9968) were each involved in 2 experiments.








The analyzed protein sequence has a GRAVY value of -0.622 (23.38th percentile), indicating a hydrophilic nature. It exhibits a charge of 27.29 at pH 7.0 (96.54th percentile) and a median structural flexibility of 1.011 (84.81st percentile). The protein's secondary structure is predicted to be 33.94% helix (67.37th percentile), 26.61% sheet (12.97th percentile), and 34.86% turn (88.86th percentile). The instability index is 56.05 (74.83rd percentile), with an isoelectric point of 10.51 (98.34th percentile). The protein has a length of 327 amino acids (34.19th percentile) and a molecular weight of 35224.12 Da (31.82nd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |