---
layout: post
title: PLPP1
date: 2025-01-17 16:55 CST
description: PLPP1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8611) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8611  | PLPP1 | ENSG00000067113 | 5q11.2 |



The gene enables [diacylglycerol diphosphate phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0000810) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [caveola](https://amigo.geneontology.org/amigo/term/GO:0005901), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [apical plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0016324), [membrane raft](https://amigo.geneontology.org/amigo/term/GO:0045121), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). The gene is involved in various processes including [phospholipid metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006644), [sphingolipid metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006665), [sphingosine metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006670), [ceramide metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006672), [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), [protein kinase C-activating G protein-coupled receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007205), [negative regulation of cell population proliferation](https://amigo.geneontology.org/amigo/term/GO:0008285), [regulation of lipid metabolic process](https://amigo.geneontology.org/amigo/term/GO:0019216), [intracellular steroid hormone receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0030518), [androgen receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0030521), and [phospholipid dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0046839). Additionally, the gene enables [phosphatidate phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0008195), [sphingosine-1-phosphate phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0042392), [lipid phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0042577), and [ceramide-1-phosphate phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0106235).


The gene length is 64,813 base pairs (72.83rd percentile), the mature length is 7,713 base pairs (97.4th percentile), and the primary transcript length is 64,770 base pairs (80.98th percentile).


The gene PLPP1 (NCBI ID: 8611) has been mentioned in [21 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PLPP1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1964 and the middle 50% of publications ranging from 1971 to 2007. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning PLPP1, ranked by their scientific influence as measured by the [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541), include: ["GROWTH CHARACTERISTICS OF THE BLUE-GREEN ALGAL VIRUS LPP-1."](https://pubmed.ncbi.nlm.nih.gov/14208517) (1964) (relative citation ratio: 4.77), ["Lipid phosphate phosphohydrolase type 1 (LPP1) degrades extracellular lysophosphatidic acid in vivo."](https://pubmed.ncbi.nlm.nih.gov/19215222) (2009) (relative citation ratio: 2.2), ["The LPP1 and DPP1 gene products account for most of the isoprenoid phosphate phosphatase activities in Saccharomyces cerevisiae."](https://pubmed.ncbi.nlm.nih.gov/10329682) (1999) (relative citation ratio: 2.19), ["Lipid phosphate phosphatase-1 regulates lysophosphatidate-induced fibroblast migration by controlling phospholipase D2-dependent phosphatidate generation."](https://pubmed.ncbi.nlm.nih.gov/17057224) (2006) (relative citation ratio: 1.06), and ["Intact pap2 downstream of mcr-1 appears to be required for colistin resistance."](https://pubmed.ncbi.nlm.nih.gov/32139115) (2020) (relative citation ratio: 1.05). The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PLPP1](https://www.proteinatlas.org/ENSG00000067113-PLPP1) is a hydrolase with evidence at the protein level. It is detected in many tissues and is primarily located in the plasma membrane. The gene is expressed in various clusters, including Cluster 3 for non-specific nuclear processes in blood, Cluster 41 for smooth muscle tissue and ECM organization, Cluster 30 for white matter and myelination in the brain, Cluster 44 for connective tissue cells and ECM organization in cell lines, and Cluster 41 for adipocytes and endothelial cells with mixed function in single cells.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2B](https://www.ncbi.nlm.nih.gov/gene/9757), [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599), [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046), and [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908), each with 5 experiments demonstrating their regulatory roles.



The GWAS data indicates associations with various mental health conditions, including cognitive disorder, bipolar disorder, mood disorder, psychotic disorder, schizophrenia, and schizoaffective disorder.



The gene is expressed in multiple tissue samples, including the prostate, spinal cord, adipocytes, adrenal gland, and uterus.


The input data highlights several key pathways involved in lipid metabolism, including sphingolipid metabolism, sphingolipid de novo biosynthesis, and the broader categories of metabolism and metabolism of lipids and lipoproteins.



The analyzed protein sequence has a GRAVY value of 0.0898 (87.61% percentile), indicating a relatively hydrophobic nature. The charge at pH 7 is 2.32 (58.68% percentile), and the median structural flexibility is 0.987 (8.73% percentile). The protein has a propensity for helix, turn, and sheet structures of 26.76% (10.65% percentile), 27.11% (38.97% percentile), and 43.66% (93.09% percentile), respectively. The instability index is 30.87 (4.79% percentile), suggesting a stable protein. The isoelectric point is 8.10 (59.84% percentile), with a length of 284 amino acids (26.21% percentile) and a molecular weight of 32155.69 Da (26.89% percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |