---
layout: post
title: OFD1
date: 2025-01-17 16:55 CST
description: OFD1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8481) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8481  | OFD1 | ENSG00000046651 | Xp22.2  |



The gene is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [centrosome](https://amigo.geneontology.org/amigo/term/GO:0005813), [centriole](https://amigo.geneontology.org/amigo/term/GO:0005814), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [cilium](https://amigo.geneontology.org/amigo/term/GO:0005929), [microtubule cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0015630), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [centriolar satellite](https://amigo.geneontology.org/amigo/term/GO:0034451), and [ciliary basal body](https://amigo.geneontology.org/amigo/term/GO:0036064). It is active in the [centrosome](https://amigo.geneontology.org/amigo/term/GO:0005813), [motile cilium](https://amigo.geneontology.org/amigo/term/GO:0031514), and [ciliary basal body](https://amigo.geneontology.org/amigo/term/GO:0036064). The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802), [alpha-tubulin binding](https://amigo.geneontology.org/amigo/term/GO:0043014), [gamma-tubulin binding](https://amigo.geneontology.org/amigo/term/GO:0043015), and [molecular adaptor activity](https://amigo.geneontology.org/amigo/term/GO:0060090). It is involved in [cilium assembly](https://amigo.geneontology.org/amigo/term/GO:0060271) and [epithelial cilium movement involved in determination of left/right asymmetry](https://amigo.geneontology.org/amigo/term/GO:0060287). The gene is not involved in [mitotic cell cycle](https://amigo.geneontology.org/amigo/term/GO:0000278), [centriole replication](https://amigo.geneontology.org/amigo/term/GO:0007099), or [mitotic spindle assembly](https://amigo.geneontology.org/amigo/term/GO:0090307).


The gene length is 70,919.0 base pairs (74.82% of all genes), the mature length is 1,382.5 base pairs, and the primary transcript length is 18,817.5 base pairs.


The gene OFD1 (NCBI ID: 8481) has been mentioned in [59 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22OFD1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1998 and the middle 50% of publications occurring between 2011 and 2020. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning OFD1, ranked by their scientific influence, include "[Autophagy promotes primary ciliogenesis by removing OFD1 from centriolar satellites.](https://pubmed.ncbi.nlm.nih.gov/24089205)" (2013) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 8.15, "[Ofd1, a human disease gene, regulates the length and distal structure of centrioles.](https://pubmed.ncbi.nlm.nih.gov/20230748)" (2010) with an RCR of 4.23, "[OFD1 is mutated in X-linked Joubert syndrome and interacts with LCA5-encoded lebercilin.](https://pubmed.ncbi.nlm.nih.gov/19800048)" (2009) with an RCR of 3.3, "[OFD1: One gene, several disorders.](https://pubmed.ncbi.nlm.nih.gov/35112477)" (2022) with an RCR of 3.19, and "[OFD1 is a centrosomal/basal body protein expressed during mesenchymal-epithelial transition in human nephrogenesis.](https://pubmed.ncbi.nlm.nih.gov/15466260)" (2004) with an RCR of 2.22. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[OFD1](https://www.proteinatlas.org/ENSG00000046651-OFD1) is a gene implicated in various diseases, including Ciliopathy, Joubert syndrome, and Retinitis pigmentosa. Evidence for its involvement is supported at the protein level. The gene's RNA is detected in all tissues, with its protein localized to microtubules, the primary cilium, and the end piece. OFD1 is expressed in Cluster 8 of blood cells, specifically in plasmacytoid dendritic cells, and in Cluster 18 of tissues, associated with ribosomal function. In the brain, it is found in Cluster 44, indicating a mixed function. Additionally, OFD1 is expressed in Cluster 36 of cell lines, related to mRNA processing, and in Cluster 37 of single cells, associated with glandular and luminal cells.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [TFAP2C](https://www.ncbi.nlm.nih.gov/gene/7022) and [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613), each with 6 experiments. Additionally, [YY1](https://www.ncbi.nlm.nih.gov/gene/7528), [ARID2](https://www.ncbi.nlm.nih.gov/gene/196528), and [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113) were each found to be regulating in 5 experiments.





The gene is expressed in various immune cell types, including CD34+ cells, BDCA4+ dendritic cells, CD56+ NK cells, CD4+ T cells, CD8+ T cells, and CD19+ B cells (neg. sel.), as well as in the lymph node. Additionally, it is expressed in the thyroid.


The input data highlights various biological pathways, including the "Hedgehog 'off' state" and "Signaling by Hedgehog," which are crucial for developmental processes. Additionally, it covers multiple aspects of the cell cycle, such as "Cell Cycle," "Cell Cycle, Mitotic," "G2/M Transition," and "Mitotic G2-G2/M phases." The data also includes pathways related to centrosome dynamics, such as "Centrosome maturation," "Recruitment of mitotic centrosome proteins and complexes," and "Loss of Nlp from mitotic centrosomes." Furthermore, it touches on organelle biogenesis, signal transduction, and the assembly of the primary cilium, indicating a broad range of cellular processes.



The analyzed protein sequence has a GRAVY value of -0.808 (11.32 percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -17.36 (13.16 percentile). The median structural flexibility is 1.013 (89.0 percentile), with a high affinity for helix structures (42.69%, 96.71 percentile). The instability index is 60.11 (83.24 percentile), suggesting potential instability. The isoelectric point is 5.82 (25.84 percentile), and the protein has a molecular weight of 116669.77 Da (88.72 percentile). The sequence length is 1012 amino acids (87.82 percentile). Secondary structure affinities include 29.15% for sheets (23.98 percentile) and 23.91% for turns (14.51 percentile). For detailed sequence analysis methods, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |