---
layout: post
title: DPYSL3
date: 2025-01-17 16:55 CST
description: DPYSL3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1809) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1809  | DPYSL3 | ENSG00000113657 | 5q32  |



The gene is located in the [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [lamellipodium](https://amigo.geneontology.org/amigo/term/GO:0030027), [growth cone](https://amigo.geneontology.org/amigo/term/GO:0030426), [cell body](https://amigo.geneontology.org/amigo/term/GO:0044297), and [synapse](https://amigo.geneontology.org/amigo/term/GO:0045202). It is part of [filamentous actin](https://amigo.geneontology.org/amigo/term/GO:0031941) and colocalizes with [exocytic vesicle](https://amigo.geneontology.org/amigo/term/GO:0070382). The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [hydrolase activity, acting on carbon-nitrogen (but not peptide) bonds, in cyclic amides](https://amigo.geneontology.org/amigo/term/GO:0016812), [SH3 domain binding](https://amigo.geneontology.org/amigo/term/GO:0017124), [filamin binding](https://amigo.geneontology.org/amigo/term/GO:0031005), [chondroitin sulfate binding](https://amigo.geneontology.org/amigo/term/GO:0035374), and [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802). It is involved in [positive regulation of neuron projection development](https://amigo.geneontology.org/amigo/term/GO:0010976), [negative regulation of neuron projection development](https://amigo.geneontology.org/amigo/term/GO:0010977), [negative regulation of cell migration](https://amigo.geneontology.org/amigo/term/GO:0030336), [neuron development](https://amigo.geneontology.org/amigo/term/GO:0048666), [response to axon injury](https://amigo.geneontology.org/amigo/term/GO:0048678), [actin filament bundle assembly](https://amigo.geneontology.org/amigo/term/GO:0051017), [positive regulation of filopodium assembly](https://amigo.geneontology.org/amigo/term/GO:0051491), [actin crosslink formation](https://amigo.geneontology.org/amigo/term/GO:0051764), and [cellular response to cytokine stimulus](https://amigo.geneontology.org/amigo/term/GO:0071345).


The gene length is 257,031 base pairs (94.79% of all genes), the mature length is 2,653.5 base pairs (66.87% of all genes), and the primary transcript length is 204,589 base pairs (96.01% of all genes).


The gene DPYSL3 (NCBI ID: 1809) has been mentioned in [46 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22DPYSL3%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1997 and the middle 50% of publications occurring between 2007 and 2018. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning DPYSL3, ranked by their scientific influence, include "[Expression patterns of immature neuronal markers PSA-NCAM, CRMP-4 and NeuroD in the hippocampus of young adult and aged rodents.](https://pubmed.ncbi.nlm.nih.gov/12391592)" (2002) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 3.2, "[CRMP4 and CRMP2 Interact to Coordinate Cytoskeleton Dynamics, Regulating Growth Cone Development and Axon Elongation.](https://pubmed.ncbi.nlm.nih.gov/26064693)" (2015) with an RCR of 2.02, "[Identification of CRMP4 as a convergent regulator of axon outgrowth inhibition.](https://pubmed.ncbi.nlm.nih.gov/17301178)" (2007) with an RCR of 1.92, "[The Ulip family phosphoproteins--common and specific properties.](https://pubmed.ncbi.nlm.nih.gov/9652388)" (1998) with an RCR of 1.91, and "[DPYSL3 modulates mitosis, migration, and epithelial-to-mesenchymal transition in claudin-low breast cancer.](https://pubmed.ncbi.nlm.nih.gov/30498031)" (2018) with an RCR of 1.56. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[DPYSL3](https://www.proteinatlas.org/ENSG00000113657-DPYSL3) is a gene with evidence at the protein level, detected in many RNA tissue distributions. It is primarily localized in the cytosol and is associated with neuronal transcription in tissue expression clusters (Cluster 64). In the brain, it is linked to astrocytes with mixed functions (Cluster 39). Additionally, DPYSL3 is expressed in non-specific cell proliferation clusters (Cluster 32) in cell lines and is found in smooth muscle cells involved in muscle contraction in single-cell expression clusters (Cluster 32).


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613), [MYC](https://www.ncbi.nlm.nih.gov/gene/4609), and [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), each with 5 experiments. Additionally, [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313) was found to be regulating in 4 experiments.



The GWAS data indicates associations with a range of disease conditions, including lower respiratory tract disease, lung disease, respiratory system disease, kidney disease, urinary system disease, nervous system disease, central nervous system disease, neurodegenerative disease, cognitive disorder, disease of mental health, substance-related disorder, substance dependence, alcohol dependence, synucleinopathy, Parkinson's disease, developmental disorder of mental health, specific developmental disorder, sensory system disease, communication disorder, language disorder, speech disorder, learning disability, reading disorder, and auditory system disease.



The gene is expressed in various tissues, including blood vessels, pineal gland (both day and night), cardiac myocytes, retina, fetal brain, spinal cord, and smooth muscle.


The proteins are involved in several pathways related to axon guidance and developmental biology, including CRMPs in Sema3A signaling and semaphorin interactions.



The analyzed protein sequence has a GRAVY value of -0.264 (62.25th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -6.37 (29.89th percentile). The median structural flexibility is 1.002 (49.59th percentile), with a predicted secondary structure composition of 28.42% helix (19.69th percentile), 34.21% sheet (53.48th percentile), and 32.11% turn (77.3rd percentile). The instability index is 35.31 (10.78th percentile), and the isoelectric point is 6.04 (31.62nd percentile). The protein has a length of 570 amino acids (65.86th percentile) and a molecular weight of 61962.59 Da (64.59th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |