---
layout: post
title: NBR1
date: 2025-01-17 16:55 CST
description: NBR1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4077) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4077  | NBR1 | ENSG00000188554 | 17q21.31 |



The gene is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [mitochondrial intermembrane space](https://amigo.geneontology.org/amigo/term/GO:0005758), [lysosome](https://amigo.geneontology.org/amigo/term/GO:0005764), [late endosome](https://amigo.geneontology.org/amigo/term/GO:0005770), [peroxisomal membrane](https://amigo.geneontology.org/amigo/term/GO:0005778), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [nuclear body](https://amigo.geneontology.org/amigo/term/GO:0016604), [M band](https://amigo.geneontology.org/amigo/term/GO:0031430), and [intracellular membrane-bounded organelle](https://amigo.geneontology.org/amigo/term/GO:0043231). It colocalizes with the [mitochondrion](https://amigo.geneontology.org/amigo/term/GO:0005739) and [autophagosome](https://amigo.geneontology.org/amigo/term/GO:0005776). The gene is involved in [macroautophagy](https://amigo.geneontology.org/amigo/term/GO:0016236), [regulation of bone mineralization](https://amigo.geneontology.org/amigo/term/GO:0030500), [regulation of stress-activated MAPK cascade](https://amigo.geneontology.org/amigo/term/GO:0032872), and [negative regulation of osteoblast differentiation](https://amigo.geneontology.org/amigo/term/GO:0045668). It enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [zinc ion binding](https://amigo.geneontology.org/amigo/term/GO:0008270), [ubiquitin binding](https://amigo.geneontology.org/amigo/term/GO:0043130), and [mitogen-activated protein kinase binding](https://amigo.geneontology.org/amigo/term/GO:0051019). Additionally, it is part of the [receptor complex](https://amigo.geneontology.org/amigo/term/GO:0043235) and is active in the [phagophore assembly site](https://amigo.geneontology.org/amigo/term/GO:0000407).


The gene length is 9,912.0 base pairs (25.58% of all genes), the mature length is 788.5 base pairs, and the primary transcript length is 9,845.0 base pairs.


The gene NBR1 (NCBI ID: 4077) has been mentioned in [44 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22NBR1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 2001 and the middle 50% of publications occurring between 2011 and 2019. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning NBR1, ranked by their scientific influence, include "[NBR1 and p62 as cargo receptors for selective autophagy of ubiquitinated targets.](https://pubmed.ncbi.nlm.nih.gov/19502794)" (2009) (relative citation ratio: 7.33), "[NBR1 acts as an autophagy receptor for peroxisomes.](https://pubmed.ncbi.nlm.nih.gov/23239026)" (2013) (relative citation ratio: 7.2), "[Selective autophagy limits cauliflower mosaic virus infection by NBR1-mediated targeting of viral capsid protein and particles.](https://pubmed.ncbi.nlm.nih.gov/28223514)" (2017) (relative citation ratio: 7.2), "[NBR1-mediated selective autophagy targets insoluble ubiquitinated protein aggregates in plant stress responses.](https://pubmed.ncbi.nlm.nih.gov/23341779)" (2013) (relative citation ratio: 7.16), and "[Selective autophagy regulates heat stress memory in Arabidopsis by NBR1-mediated targeting of HSP90.1 and ROF1.](https://pubmed.ncbi.nlm.nih.gov/32967551)" (2021) (relative citation ratio: 6.17). The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[NBR1](https://www.proteinatlas.org/ENSG00000188554-NBR1) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in the nucleoplasm, nuclear bodies, and vesicles. The gene is part of Cluster 42 in blood expression, indicating involvement in non-specific vesicular transport, and Cluster 78 in tissue expression, suggesting a role in testis transcription. In the brain, it is associated with Cluster 13, linked to oligodendrocytes with mixed functions. Additionally, NBR1 is expressed in breast cancer cell lines (Cluster 27) and late spermatids (Cluster 36), although its function in these contexts is unknown.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [TP53](https://www.ncbi.nlm.nih.gov/gene/7157), [SIN3A](https://www.ncbi.nlm.nih.gov/gene/25942), [ELF1](https://www.ncbi.nlm.nih.gov/gene/1997), [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113), and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), each shown to be regulating in 5 experiments.



The input data indicates associations with various disease conditions, including nervous system diseases, cognitive disorders, diseases of mental health, mood disorders, mental depression, and sensory system diseases.



The gene is expressed in muscle and bladder tissues according to the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles indicate expression in pineal gland (both day and night), testis, and various testicular substructures including interstitial tissue, Leydig cells, seminiferous tubules, and germ cells.



The mouse ortholog gene 17966 and its human ortholog gene 4077 are associated with impaired glucose tolerance, a phenotype observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.497 (35.13th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -50.74 (2.36th percentile), and the median structural flexibility is 1.007 (71.26th percentile). The protein has a high instability index of 58.99 (81.16th percentile) and an isoelectric point of 5.03 (8.35th percentile). The sequence length is 966 amino acids (86.69th percentile) with a molecular weight of 107411.98 Da (86.63rd percentile). The secondary structure affinities are 34.16% for helix (69.07th percentile), 31.37% for sheet (35.94th percentile), and 29.81% for turn (62.44th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |