---
layout: post
title: CA2
date: 2025-01-17 16:55 CST
description: CA2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/760) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 760  | CA2 | ENSG00000104267 | 8q21.2 |



The gene is involved in various biological processes, including [morphogenesis of an epithelium](https://amigo.geneontology.org/amigo/term/GO:0002009), [one-carbon metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006730), [carbon dioxide transport](https://amigo.geneontology.org/amigo/term/GO:0015670), and [positive regulation of synaptic transmission, GABAergic](https://amigo.geneontology.org/amigo/term/GO:0032230). It enables several molecular functions such as [arylesterase activity](https://amigo.geneontology.org/amigo/term/GO:0004064), [carbonate dehydratase activity](https://amigo.geneontology.org/amigo/term/GO:0004089), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [zinc ion binding](https://amigo.geneontology.org/amigo/term/GO:0008270). The gene is located in and active in multiple cellular components, including the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [apical part of cell](https://amigo.geneontology.org/amigo/term/GO:0045177), and [myelin sheath](https://amigo.geneontology.org/amigo/term/GO:0043209). Additionally, it is involved in processes like [secretion](https://amigo.geneontology.org/amigo/term/GO:0046903) and [regulation of intracellular pH](https://amigo.geneontology.org/amigo/term/GO:0051453), and is found in the [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062).


The gene length is 76,742 base pairs (76.51 percentile of all genes), the mature length is 6,823 base pairs, and the primary transcript length is 76,742 base pairs.


The gene CA2 (NCBI ID: 760) has been mentioned in [2,558 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CA2%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning CA2 is 1952, and the middle 50% of publications occurred between 1999 and 2018.


The top five publications mentioning CA2, ranked by their scientific influence, include "[Randomised double-blind comparison of chimeric monoclonal antibody to tumour necrosis factor alpha (cA2) versus placebo in rheumatoid arthritis.](https://pubmed.ncbi.nlm.nih.gov/7934491)" (1994) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 36.58, "[STIM1, an essential and conserved component of store-operated Ca2+ channel function.](https://pubmed.ncbi.nlm.nih.gov/15866891)" (2005) with an RCR of 29.44, "[Differential activation of transcription factors induced by Ca2+ response amplitude and duration.](https://pubmed.ncbi.nlm.nih.gov/9126747)" (1997) with an RCR of 28.01, "[Annexins: linking Ca2+ signalling to membrane dynamics.](https://pubmed.ncbi.nlm.nih.gov/15928709)" (2005) with an RCR of 22.78, and "[Guard cell signal transduction network: advances in understanding abscisic acid, CO2, and Ca2+ signaling.](https://pubmed.ncbi.nlm.nih.gov/20192751)" (2010) with an RCR of 22.04. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CA2](https://www.proteinatlas.org/ENSG00000104267-CA2) is a lyase with evidence at the protein level, implicated in osteopetrosis and identified as an FDA-approved drug target. The gene is detected in many tissues and is notably expressed in monocytes involved in innate immune response, as well as in the choroid plexus where it plays a role in transmembrane transport. Additionally, CA2 is found in enterocytes, contributing to absorption processes. The gene is also expressed in various cancer cell lines, particularly in kidney cancer membrane components.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SP1](https://www.ncbi.nlm.nih.gov/gene/6667) and [ZBTB7A](https://www.ncbi.nlm.nih.gov/gene/51341), each with 6 experiments. Additionally, [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227), [MYC](https://www.ncbi.nlm.nih.gov/gene/4609), and [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930) were each found to be regulating in 5 experiments.




The genome-wide association study has identified a significant association between genetic variants and [lung function (forced vital capacity)](https://pubmed.ncbi.nlm.nih.gov/36914875). This trait was mapped to the closest existing MeSH term, [Lung](https://meshb.nlm.nih.gov/record/ui?ui=D008168), using the method called [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0).


The gene is expressed in various tissues, including the colon, stomach, and kidney, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles show expression in CD105+ endothelial cells, CD71+ early erythroid cells, hypothalamus, trachea, colon, and bone marrow.


The input data highlights several key processes involving erythrocytes and metabolism. Erythrocytes play a crucial role in gas exchange, taking up carbon dioxide and releasing oxygen, as well as the reverse process of taking up oxygen and releasing carbon dioxide. Additionally, the data mentions the O2/CO2 exchange in erythrocytes and the reversible hydration of carbon dioxide, all of which are integral to metabolic functions.



The analyzed protein sequence has a GRAVY value of -0.579 (27.18th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -0.438 (48.48th percentile), and the median structural flexibility is 1.008 (74.2nd percentile). The protein has a propensity for helix, sheet, and turn structures of 30.0% (32.08th percentile), 35.0% (59.19th percentile), and 33.1% (82.27th percentile), respectively. The instability index is 21.68 (0.78th percentile), with an isoelectric point of 6.87 (48.78th percentile). The protein is 260 amino acids long (23.02nd percentile) and has a molecular weight of 29245.67 Da (23.43rd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |