---
layout: post
title: EIF4G3
date: 2025-01-17 16:55 CST
description: EIF4G3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8672) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8672  | EIF4G3 | ENSG00000075151 | 1p36.12 |



The gene enables [RNA cap binding](https://amigo.geneontology.org/amigo/term/GO:0000339), [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [mRNA binding](https://amigo.geneontology.org/amigo/term/GO:0003729), and [translation initiation factor activity](https://amigo.geneontology.org/amigo/term/GO:0003743). It is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829) and is involved in [translational initiation](https://amigo.geneontology.org/amigo/term/GO:0006413), [regulation of translational initiation](https://amigo.geneontology.org/amigo/term/GO:0006446), and [spermatid development](https://amigo.geneontology.org/amigo/term/GO:0007286). Additionally, it enables [translation factor activity, RNA binding](https://amigo.geneontology.org/amigo/term/GO:0008135) and is part of the [eukaryotic translation initiation factor 4F complex](https://amigo.geneontology.org/amigo/term/GO:0016281). The gene is active in [intracellular non-membrane-bounded organelles](https://amigo.geneontology.org/amigo/term/GO:0043232) and is involved in the [positive regulation of translation](https://amigo.geneontology.org/amigo/term/GO:0045727).


The gene length is 15,107 base pairs (35.25th percentile), the mature length is 2,757 base pairs (68.68th percentile), and the primary transcript length is 15,054 base pairs (44.26th percentile).


The gene EIF4G3 (NCBI ID: 8672) has been mentioned in [18 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22EIF4G3%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727). The earliest year of publication is 1998, and the middle 50% of publications occurred between 2002 and 2013.


The top 5 publications mentioning EIF4G3, ranked by their relative citation ratio (RCR), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows. The most influential publication is "[Proteolysis of human eukaryotic translation initiation factor eIF4GII, but not eIF4GI, coincides with the shutoff of host protein synthesis after poliovirus infection.](https://pubmed.ncbi.nlm.nih.gov/9736694)" (1998) with an RCR of 5.01. Another notable study is "[Circular RNA EIF4G3 suppresses gastric cancer progression through inhibition of β-catenin by promoting δ-catenin ubiquitin degradation and upregulating SIK1.](https://pubmed.ncbi.nlm.nih.gov/35780119)" (2022), which has an RCR of 4.0. Additionally, "[Eukaryotic initiation factor 4GII (eIF4GII), but not eIF4GI, cleavage correlates with inhibition of host cell protein synthesis after human rhinovirus infection.](https://pubmed.ncbi.nlm.nih.gov/10074204)" (1999) has an RCR of 1.9. The publication "[IRES interaction with translation initiation factors: functional characterization of novel RNA contacts with eIF3, eIF4B, and eIF4GII.](https://pubmed.ncbi.nlm.nih.gov/11565745)" (2001) has an RCR of 1.73, and "[Down-regulation of eIF4GII by miR-520c-3p represses diffuse large B cell lymphoma development.](https://pubmed.ncbi.nlm.nih.gov/24497838)" (2014) has an RCR of 1.08. The citation count is sourced from [iCite](https://icite.od.nih.gov).


[EIF4G3](https://www.proteinatlas.org/ENSG00000075151-EIF4G3) is an RNA-binding initiation factor primarily located in the cytosol. It is detected in all tissues and is particularly expressed in the testis, where it plays a role in spermatogenesis. Additionally, EIF4G3 is found in the choroid plexus of the brain and is associated with dendritic cells in the blood, although its function in these contexts is not well understood. The gene is also expressed in various cell lines involved in mRNA processing. Evidence for EIF4G3's function is supported at the protein level.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 7 experiments, [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) with 6 experiments, [STAT3](https://www.ncbi.nlm.nih.gov/gene/6774) with 5 experiments, [SUMO2](https://www.ncbi.nlm.nih.gov/gene/6613) with 5 experiments, and [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The input data indicates associations with a variety of disease conditions, including several types of cancer such as leukemia, lymphoma, and immune system cancer. Additionally, respiratory conditions like asthma, bronchial disease, and obstructive lung disease are noted. Cardiovascular issues, including coronary artery disease and vascular disease, are also present. Furthermore, the data encompasses broader categories such as hematopoietic system disease, immune system disease, and general disease of anatomical entities.


EIF4G3 has been associated with several traits and diseases through genome-wide association studies. Notably, it has been linked to [Cigarettes smoked per day](https://pubmed.ncbi.nlm.nih.gov/36477530), which was mapped to the MeSH term [Tobacco Products](https://meshb.nlm.nih.gov/record/ui?ui=D062789) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method. Additionally, EIF4G3 is associated with [Menarche (age at onset)](https://pubmed.ncbi.nlm.nih.gov/30595370), mapped to the MeSH term [Menarche](https://meshb.nlm.nih.gov/record/ui?ui=D008572), and [Subjective well-being](https://pubmed.ncbi.nlm.nih.gov/27089181), mapped to the MeSH term [Diagnostic Self Evaluation](https://meshb.nlm.nih.gov/record/ui?ui=D059026), both using the embedding_match method.


The gene is expressed in various tissues, including the thyroid and pituitary gland. Additionally, it is expressed in the pineal gland during both day and night, the prefrontal cortex, thalamus, and testis (both interstitial and germ cells). Notably, the gene is also expressed in specific cancer cell lines, such as Burkitt's lymphoma (Raji) and promyelocytic leukemia (HL-60).


The proteins are involved in several key pathways related to immune response, including the antiviral mechanism by IFN-stimulated genes, ISG15 antiviral mechanism, cytokine signaling in the immune system, interferon signaling, and the broader immune system.



The analyzed protein sequence has a GRAVY value of -0.613 (24.14th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -40.57 (3.76th percentile), and the median structural flexibility is 1.014 (89.64th percentile). The protein has a high instability index of 58.75 (80.63rd percentile) and an isoelectric point of 5.27 (12.75th percentile). The sequence length is 1585 amino acids (95.6th percentile), with a molecular weight of 176650.18 Da (95.68th percentile). The secondary structure affinities are 33.82% for helix (66.39th percentile), 30.73% for sheet (32.23rd percentile), and 30.79% for turn (69.19th percentile). For more details on the sequence analysis, refer to the [BioPython ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |