---
layout: post
title: DGKZ
date: 2025-01-17 16:55 CST
description: DGKZ description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8525) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8525  | DGKZ | ENSG00000149091 | 11p11.2 |



The gene enables [ATP-dependent diacylglycerol kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004143), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524). It is active in and located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), and also found in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). Additionally, it is located in the [nuclear speck](https://amigo.geneontology.org/amigo/term/GO:0016607) and [lamellipodium](https://amigo.geneontology.org/amigo/term/GO:0030027). The gene is involved in various processes including the [phosphatidic acid biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006654), [protein kinase C-activating G protein-coupled receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007205), [cell migration](https://amigo.geneontology.org/amigo/term/GO:0016477), [platelet activation](https://amigo.geneontology.org/amigo/term/GO:0030168), [intracellular signal transduction](https://amigo.geneontology.org/amigo/term/GO:0035556), [diacylglycerol metabolic process](https://amigo.geneontology.org/amigo/term/GO:0046339), [glycerolipid metabolic process](https://amigo.geneontology.org/amigo/term/GO:0046486), [lipid phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0046834), [negative regulation of T cell receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0050860), and [regulation of synaptic transmission, glutamatergic](https://amigo.geneontology.org/amigo/term/GO:0051966). Furthermore, the gene enables [kinase activity](https://amigo.geneontology.org/amigo/term/GO:0016301), [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872), and [alkylglycerol kinase activity](https://amigo.geneontology.org/amigo/term/GO:0047649).


The gene length is 47,835 base pairs (65.11% of all genes), the mature length is 1,920 base pairs, and the primary transcript length is 42,657 base pairs (71.47% of all genes).


The gene DGKZ, with NCBI ID 8525, has been mentioned in [51 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22DGKZ%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727). The earliest year of publication is 1996, and the middle 50% of publications occurred between 2006 and 2018. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning DGKZ, ranked by their scientific influence, include "[Protein kinase C regulates the nuclear localization of diacylglycerol kinase-zeta.](https://pubmed.ncbi.nlm.nih.gov/9716136)" (1998) (relative citation ratio: 4.49), "[The role of diacylglycerol kinase ζ and phosphatidic acid in the mechanical activation of mammalian target of rapamycin (mTOR) signaling and skeletal muscle hypertrophy.](https://pubmed.ncbi.nlm.nih.gov/24302719)" (2014) (relative citation ratio: 3.89), "[Molecular cloning and characterization of a novel human diacylglycerol kinase zeta.](https://pubmed.ncbi.nlm.nih.gov/8626588)" (1996) (relative citation ratio: 3.17), "[A DGKζ-FoxO-ubiquitin proteolytic axis controls fiber size during skeletal muscle remodeling.](https://pubmed.ncbi.nlm.nih.gov/29764991)" (2018) (relative citation ratio: 1.77), and "[Association of diacylglycerol kinase zeta with protein kinase C alpha: spatial regulation of diacylglycerol signaling.](https://pubmed.ncbi.nlm.nih.gov/12629049)" (2003) (relative citation ratio: 1.66). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[DGKZ](https://www.proteinatlas.org/ENSG00000149091-DGKZ) is a kinase and transferase with evidence at the protein level. It is detected in all tissues and is primarily localized to nuclear speckles. The gene is highly expressed in monocytes and neutrophils, particularly during degranulation, and is involved in neuronal signaling in the brain. Additionally, DGKZ is expressed in myeloid leukemia cell lines, NK-cells, and neurons, where it plays a role in innate and adaptive immune responses, as well as mixed neuronal functions.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [MYC](https://www.ncbi.nlm.nih.gov/gene/4609), [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), and [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), each with 5 experiments. Additionally, [TRIM24](https://www.ncbi.nlm.nih.gov/gene/8805) and [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113) were found to be regulating in 4 experiments each.



The GWAS data indicates associations with various disease conditions, including immune system disease, B cell deficiency, primary immunodeficiency disease, dysgammaglobulinemia, selective immunoglobulin deficiency disease, and selective IgA deficiency disease.



The gene is expressed in various brain regions, including the prefrontal cortex, amygdala, whole brain, cingulate cortex, temporal lobe, and globus pallidus. Additionally, it is expressed in immune cells such as CD56+ NK cells and CD4+ T cells, as well as in the pituitary and spleen.


The proteins are involved in various signaling pathways, including G alpha (q) signaling events, the gastrin-CREB signaling pathway via PKC and MAPK, GPCR downstream signaling, and signaling by GPCR. Additionally, they participate in signal transduction, the effects of PIP2 hydrolysis, hemostasis, and platelet activation, signaling, and aggregation.



The analyzed protein sequence has a GRAVY value of -0.595 (25.81 percentile), indicating a hydrophilic nature. It exhibits a charge of 9.46 at pH 7.0 (80.31 percentile) and a median structural flexibility of 1.005 (62.99 percentile). The protein's secondary structure is predicted to be 30.28% helix (34.5 percentile), 29.53% sheet (25.98 percentile), and 29.09% turn (56.74 percentile). The instability index is 56.95 (76.93 percentile), with an isoelectric point of 8.34 (63.6 percentile). The protein has a length of 928 amino acids (85.46 percentile) and a molecular weight of 103979.76 Da (85.66 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |