---
layout: post
title: CD3G
date: 2025-01-17 16:55 CST
description: CD3G description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/917) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 917  | CD3G | ENSG00000160654 | 11q23.3 |



The gene is involved in various biological processes, including [adaptive immune response](https://amigo.geneontology.org/amigo/term/GO:0002250), [establishment or maintenance of cell polarity](https://amigo.geneontology.org/amigo/term/GO:0007163), [cell surface receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007166), [protein transport](https://amigo.geneontology.org/amigo/term/GO:0015031), [T cell activation](https://amigo.geneontology.org/amigo/term/GO:0042110), [positive thymic T cell selection](https://amigo.geneontology.org/amigo/term/GO:0045059), [gamma-delta T cell activation](https://amigo.geneontology.org/amigo/term/GO:0046629), [alpha-beta T cell activation](https://amigo.geneontology.org/amigo/term/GO:0046631), [T cell receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0050852), [protein-containing complex assembly](https://amigo.geneontology.org/amigo/term/GO:0065003), and [regulation of lymphocyte apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0070228). It enables [transmembrane signaling receptor activity](https://amigo.geneontology.org/amigo/term/GO:0004888), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [signaling receptor complex adaptor activity](https://amigo.geneontology.org/amigo/term/GO:0030159), [T cell receptor binding](https://amigo.geneontology.org/amigo/term/GO:0042608), and [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802). The gene is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), and [clathrin-coated endocytic vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030669), and is active on the [external side of plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009897). Additionally, it is part of the [alpha-beta T cell receptor complex](https://amigo.geneontology.org/amigo/term/GO:0042105) and [gamma-delta T cell receptor complex](https://amigo.geneontology.org/amigo/term/GO:0042106).


The gene length is 345,913 base pairs (96.65% of all genes), the mature length is 589 base pairs (4.93% of all genes), and the primary transcript length is 12,487 base pairs (39.23% of all genes).


The gene CD3G (NCBI ID: 917) has been mentioned in [56 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CD3G%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1987 and the middle 50% of publications occurring between 1995 and 2010. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning CD3G, ranked by their scientific influence, include "[CD3 gamma contains a phosphoserine-dependent di-leucine motif involved in down-regulation of the T cell receptor.](https://pubmed.ncbi.nlm.nih.gov/8187769)" (1994) (relative citation ratio: 3.31), "[Expression of cytoplasmic CD3 epsilon proteins in activated human adult natural killer (NK) cells and CD3 gamma, delta, epsilon complexes in fetal NK cells. Implications for the relationship of NK and T lymphocytes.](https://pubmed.ncbi.nlm.nih.gov/1387664)" (1992) (relative citation ratio: 2.8), "[Regulation and function of the CD3gamma DxxxLL motif: a binding site for adaptor protein-1 and adaptor protein-2 in vitro.](https://pubmed.ncbi.nlm.nih.gov/9230070)" (1997) (relative citation ratio: 2.37), "[The protein interactions of the immunoglobulin receptor family tyrosine-based activation motifs present in the T cell receptor zeta subunits and the CD3 gamma, delta and epsilon chains.](https://pubmed.ncbi.nlm.nih.gov/8647168)" (1996) (relative citation ratio: 1.32), and "[Close linkage of the mouse and human CD3 gamma- and delta-chain genes suggests that their transcription is controlled by common regulatory elements.](https://pubmed.ncbi.nlm.nih.gov/2827170)" (1987) (relative citation ratio: 1.18). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CD3G](https://www.proteinatlas.org/ENSG00000160654-CD3G) is a receptor with evidence at the protein level, playing a crucial role as an FDA-approved drug target. It is detected in many tissues and is primarily located in the plasma membrane and cytosol. CD3G is highly expressed in T-cells and is associated with adaptive immune responses in the thymus. Its expression is also noted in leukemia cell lines and is involved in the immune system and transcription processes in NK-cells and T-cells.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SPI1](https://www.ncbi.nlm.nih.gov/gene/6688) with 3 experiments, [STAT5B](https://www.ncbi.nlm.nih.gov/gene/6777) with 2 experiments, [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664) with 2 experiments, [GATA2](https://www.ncbi.nlm.nih.gov/gene/2624) with 2 experiments, and [CDK7](https://www.ncbi.nlm.nih.gov/gene/1022) with 2 experiments.





The gene is expressed in various tissues and cell types, including blood, spleen, and small intestine, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in CD56+ NK cells, CD4+ T cells, CD8+ T cells, whole blood, thymus, leukemia lymphoblastic cells (MOLT-4), and lymph nodes.


The input data highlights various pathways involved in the immune system, including both the adaptive and innate immune systems. Specific pathways mentioned include costimulation by the CD28 family, FCGR activation, and the role of phospholipids in phagocytosis. Additionally, the data covers Fcgamma receptor (FCGR) dependent phagocytosis, regulation of actin dynamics for phagocytic cup formation, and immunoregulatory interactions between lymphoid and non-lymphoid cells. TCR signaling and its downstream processes, such as the generation of second messenger molecules, PD-1 signaling, phosphorylation of CD3 and TCR zeta chains, and translocation of ZAP-70 to the immunological synapse, are also noted.



The analyzed protein sequence has a GRAVY value of -0.348 (52.4th percentile), indicating a hydrophilic nature. It exhibits a charge of 2.65 at pH 7.0 (59.92nd percentile) and a median structural flexibility of 1.002 (52.86th percentile). The protein's secondary structure is predicted to be 32.42% helix (54.34th percentile), 35.16% sheet (60.39th percentile), and 28.02% turn (47.48th percentile). The instability index is 40.22 (22.91st percentile), with an isoelectric point of 8.60 (68.83rd percentile). The protein has a length of 182 amino acids (11.85th percentile) and a molecular weight of 20469.24 Da (11.94th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |