---
layout: post
title: PGM5
date: 2025-01-17 16:55 CST
description: PGM5 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5239) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5239  | PGM5 | ENSG00000154330 | 9q21.11 |



The gene enables [magnesium ion binding](https://amigo.geneontology.org/amigo/term/GO:0000287) and [structural molecule activity](https://amigo.geneontology.org/amigo/term/GO:0005198). It is located in various cellular components, including [stress fiber](https://amigo.geneontology.org/amigo/term/GO:0001725), [adherens junction](https://amigo.geneontology.org/amigo/term/GO:0005912), [spot adherens junction](https://amigo.geneontology.org/amigo/term/GO:0005914), [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925), [cytoplasmic side of plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009898), [intercalated disc](https://amigo.geneontology.org/amigo/term/GO:0014704), [Z disc](https://amigo.geneontology.org/amigo/term/GO:0030018), [sarcolemma](https://amigo.geneontology.org/amigo/term/GO:0042383), and [costamere](https://amigo.geneontology.org/amigo/term/GO:0043034). It is active in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [cell-substrate junction](https://amigo.geneontology.org/amigo/term/GO:0030055), and [sarcolemma](https://amigo.geneontology.org/amigo/term/GO:0042383). Additionally, it is involved in several biological processes, including [carbohydrate metabolic process](https://amigo.geneontology.org/amigo/term/GO:0005975), [cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0007155), [striated muscle tissue development](https://amigo.geneontology.org/amigo/term/GO:0014706), and [myofibril assembly](https://amigo.geneontology.org/amigo/term/GO:0030239). Furthermore, it is part of the [dystrophin-associated glycoprotein complex](https://amigo.geneontology.org/amigo/term/GO:0016010).


The gene length is 41,764 base pairs (61.44% of all genes), the mature length is 3,984.5 base pairs, and the primary transcript length is 41,065.5 base pairs.


The gene PGM5 (NCBI ID: 5239) has been mentioned in [9 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PGM5%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1995 and the middle 50% of publications occurring between 2014 and 2021. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning PGM5, ranked by their relative citation ratio (RCR), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows:

1. [Aciculin interacts with filamin C and Xin and is essential for myofibril assembly, remodeling and maintenance.](https://pubmed.ncbi.nlm.nih.gov/24963132) (2014) (relative citation ratio: 1.64)
2. [miR-1224-3p Promotes Breast Cancer Cell Proliferation and Migration through PGM5-Mediated Aerobic Glycolysis.](https://pubmed.ncbi.nlm.nih.gov/33986801) (2021) (relative citation ratio: 1.35)
3. [The Downregulation of lncRNA <i>pgm5-as1</i> Inhibits the Proliferation and Metastasis Via Increasing miR-484 Expression in Colorectal Cancer.](https://pubmed.ncbi.nlm.nih.gov/32354224) (2021) (relative citation ratio: 1.02)
4. [Circulating lncRNA UCA1 and lncRNA PGM5-AS1 act as potential diagnostic biomarkers for early-stage colorectal cancer.](https://pubmed.ncbi.nlm.nih.gov/34212174) (2021) (relative citation ratio: 1.02)
5. [PGM5 is a promising biomarker and may predict the prognosis of colorectal cancer patients.](https://pubmed.ncbi.nlm.nih.gov/31582909) (2019) (relative citation ratio: 0.85)

For more information on the [Relative Citation Ratio (RCR)], please refer to [this article](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541). The citation count is sourced from [iCite](https://icite.od.nih.gov).


[PGM5](https://www.proteinatlas.org/ENSG00000154330-PGM5) is a gene with evidence at the protein level, detected in many RNA tissue distributions. It is primarily located in the cytosol and is associated with various expression clusters, including Cluster 35 in neutrophils for blood expression, Cluster 41 in smooth muscle tissue for tissue expression, Cluster 25 in non-specific brain regions for brain expression, Cluster 10 in ciliated cells for cell line expression, and Cluster 54 in lymphatic endothelial cells for single cell expression. The gene's prognostic significance in various cancers is unprognostic, including Bladder Urothelial Carcinoma, Breast Invasive Carcinoma, and Kidney Renal Clear Cell Carcinoma.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SUZ12](https://www.ncbi.nlm.nih.gov/gene/23512) with 6 experiments, followed by [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861), [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), [KLF9](https://www.ncbi.nlm.nih.gov/gene/687), and [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), each with 4 experiments.



The input data indicates associations with various disease conditions, including neurodegenerative diseases such as Alzheimer's disease and tauopathy. Additionally, it encompasses cognitive disorders, dementia, and diseases of mental health, all of which are subcategories of central nervous system diseases and broader nervous system diseases.



The gene is expressed in various tissues, including bladder, cardiac myocytes, heart, prostate, adrenal cortex, and superior cervical ganglion.




The analyzed protein sequence has a GRAVY value of -0.099 (78.11th percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is -0.66 (47.82nd percentile), and the median structural flexibility is 0.998 (32.89th percentile). The protein has an instability index of 41.76 (27.9th percentile) and an isoelectric point of 6.81 (47.92nd percentile). The sequence length is 567 amino acids (65.59th percentile), with a molecular weight of 62224.48 Da (64.87th percentile). The secondary structure affinities are 26.46% for helix (9.42nd percentile), 36.16% for sheet (67.36th percentile), and 31.75% for turn (75.27th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |