---
layout: post
title: PDE4A
date: 2025-01-17 16:55 CST
description: PDE4A description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5141) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5141  | PDE4A | ENSG00000065989 | 19p13.2 |



The gene enables [3',5'-cyclic-AMP phosphodiesterase activity](https://amigo.geneontology.org/amigo/term/GO:0004115) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is involved in various processes including the [cAMP catabolic process](https://amigo.geneontology.org/amigo/term/GO:0006198), [G protein-coupled receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007186), [sensory perception of smell](https://amigo.geneontology.org/amigo/term/GO:0007608), and [regulation of protein kinase A signaling](https://amigo.geneontology.org/amigo/term/GO:0010738). It is active in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [ruffle membrane](https://amigo.geneontology.org/amigo/term/GO:0032587), and [perinuclear region of cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0048471). Additionally, it enables [cAMP binding](https://amigo.geneontology.org/amigo/term/GO:0030552), [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872), and [3',5'-cyclic-GMP phosphodiesterase activity](https://amigo.geneontology.org/amigo/term/GO:0047555). The gene is also involved in [cAMP-mediated signaling](https://amigo.geneontology.org/amigo/term/GO:0019933) and the [cellular response to xenobiotic stimulus](https://amigo.geneontology.org/amigo/term/GO:0071466), as well as the [regulation of adenylate cyclase-activating G protein-coupled receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0106070).


The gene length is 15,267 base pairs (35.44% of all genes), the mature length is 1,904 base pairs, and the primary transcript length is 15,265 base pairs (44.59% of all genes).


The gene PDE4A (NCBI ID: 5141) has been mentioned in [480 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PDE4A%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning this gene is 1995, and the middle 50% of publications occurred between 2004 and 2017.


The top five publications mentioning PDE4A, ranked by their scientific influence, include "[PDE4 cAMP phosphodiesterases: modular enzymes that orchestrate signalling cross-talk, desensitization and compartmentalization.](https://pubmed.ncbi.nlm.nih.gov/12444918)" (2003) (relative citation ratio: 11.89), "[Keynote review: phosphodiesterase-4 as a therapeutic target.](https://pubmed.ncbi.nlm.nih.gov/16257373)" (2005) (relative citation ratio: 11.44), "[Apremilast, an oral phosphodiesterase 4 inhibitor, in patients with psoriatic arthritis and current skin involvement: a phase III, randomised, controlled trial (PALACE 3).](https://pubmed.ncbi.nlm.nih.gov/26792812)" (2016) (relative citation ratio: 8.52), "[Apremilast is a selective PDE4 inhibitor with regulatory effects on innate immunity.](https://pubmed.ncbi.nlm.nih.gov/24882690)" (2014) (relative citation ratio: 8.12), and "[Cyclic AMP-specific PDE4 phosphodiesterases as critical components of cyclic AMP signaling.](https://pubmed.ncbi.nlm.nih.gov/12493749)" (2003) (relative citation ratio: 7.79). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PDE4A](https://www.proteinatlas.org/ENSG00000065989-PDE4A) is a hydrolase with evidence at the protein level, and it is an FDA-approved drug target. The gene is detected in all tissues, with notable expression in the nucleoplasm and plasma membrane. It is associated with Cluster 25 in blood expression, Cluster 79 in tissue expression, Cluster 52 in brain expression, Cluster 47 in cell line expression, and Cluster 29 in single-cell expression, indicating roles in synaptic function, hemostasis, and innate immune response.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861), [MAX](https://www.ncbi.nlm.nih.gov/gene/4149), [BRD9](https://www.ncbi.nlm.nih.gov/gene/65980), [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), and [MYC](https://www.ncbi.nlm.nih.gov/gene/4609), each shown to be regulating in 5 experiments.



The input data indicates associations with various disease conditions, including metabolic disorders such as diabetes mellitus, type 1 diabetes mellitus, and fatty liver disease. Additionally, there are links to cardiovascular conditions like coronary artery disease, artery disease, vascular disease, cardiovascular system disease, and myocardial infarction. The data also highlights inherited metabolic disorders, including lipid storage disease and lysosomal storage disease.


PDE4A has been associated with [Inflammatory skin disease](https://pubmed.ncbi.nlm.nih.gov/25574825), which maps closely to the MeSH term [Inflammation](https://meshb.nlm.nih.gov/record/ui?ui=D007249) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method. Additionally, it has been linked to [Intercellular adhesion molecule 5 levels](https://pubmed.ncbi.nlm.nih.gov/28240269), which is closely related to the MeSH term [Integrins](https://meshb.nlm.nih.gov/record/ui?ui=D016023) through the same embedding match approach.


The gene is expressed in adipose tissue and brain according to the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles indicate expression in pineal gland (night), prefrontal cortex, chronic myelogenous leukemia (K-562), subthalamic nucleus, temporal lobe, and globus pallidus.


The proteins are involved in various signaling pathways, including g alpha (s) signaling events, gpcr downstream signaling, signaling by gpcr, signal transduction, opioid signaling, and darpp-32 events.



The protein sequence analyzed has a GRAVY value of -0.5111738148984186 (33.85th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -40.03249934509982 (3.84th percentile), and the median structural flexibility is 1.0058690476190475 (67.44th percentile). The protein has a high instability index of 63.04334085778808 (87.89th percentile) and an isoelectric point of 5.091091346740721 (9.47th percentile). The sequence length is 886 amino acids (83.93rd percentile) with a molecular weight of 98141.98570000057 Da (83.74th percentile). The secondary structure affinities are 33.41% for helix (62.98th percentile), 30.25% for sheet (29.68th percentile), and 30.81% for turn (69.33rd percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |