---
layout: post
title: SRI
date: 2025-01-17 16:55 CST
description: SRI description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6717) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6717  | SRI | ENSG00000075142 | 7q21.12 |



The gene is involved in various biological processes, including [action potential](https://amigo.geneontology.org/amigo/term/GO:0001508), [calcium ion transport](https://amigo.geneontology.org/amigo/term/GO:0006816), and [regulation of striated muscle contraction](https://amigo.geneontology.org/amigo/term/GO:0006942). It enables functions such as [protease binding](https://amigo.geneontology.org/amigo/term/GO:0002020), [signaling receptor binding](https://amigo.geneontology.org/amigo/term/GO:0005102), and [calcium ion binding](https://amigo.geneontology.org/amigo/term/GO:0005509). The gene is located in multiple cellular components, including the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), and [endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0005789). Additionally, it colocalizes with the [chromaffin granule membrane](https://amigo.geneontology.org/amigo/term/GO:0042584) and is involved in processes like [heart development](https://amigo.geneontology.org/amigo/term/GO:0007507) and [regulation of heart contraction](https://amigo.geneontology.org/amigo/term/GO:0008016).


The gene length is 14,645 base pairs (34.51% of all genes), the mature length is 793 base pairs (13.55% of all genes), and the primary transcript length is 6,764.5 base pairs (24.27% of all genes).


The gene SRI (NCBI ID: 6717) has been mentioned in [3,847 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SRI%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication mentioning SRI is 1972, and the middle 50% of publications occurred between 2004 and 2019.


The top five publications mentioning SRI, ranked by their relative citation ratio (RCR), include studies on periodontal disease, neonatal encephalopathy, schizophrenia, and pesticide regulations. The RCR values, which measure the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications (set at 1.0), are as follows: ["Natural history of periodontal disease in man. Rapid, moderate and no loss of attachment in Sri Lankan laborers 14 to 46 years of age."](https://pubmed.ncbi.nlm.nih.gov/3487557) (1986) (RCR: 25.62), ["Hypothermia for moderate or severe neonatal encephalopathy in low-income and middle-income countries (HELIX): a randomised controlled trial in India, Sri Lanka, and Bangladesh."](https://pubmed.ncbi.nlm.nih.gov/34358491) (2021) (RCR: 20.58), ["Is outcome for schizophrenia better in nonindustrial societies? The case of Sri Lanka."](https://pubmed.ncbi.nlm.nih.gov/438782) (1979) (RCR: 8.23), ["The impact of pesticide regulations on suicide in Sri Lanka."](https://pubmed.ncbi.nlm.nih.gov/17726039) (2007) (RCR: 7.51), and ["Case fatality of agricultural pesticides after self-poisoning in Sri Lanka: a prospective cohort study."](https://pubmed.ncbi.nlm.nih.gov/33901424) (2021) (RCR: 7.44). The citation count is sourced from [iCite](https://icite.od.nih.gov).


[SRI](https://www.proteinatlas.org/ENSG00000075142-SRI) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in the nucleoplasm and cytosol. The gene is associated with various expression clusters, including Cluster 42 for non-specific vesicular transport in blood, Cluster 45 for intestinal digestion in tissues, Cluster 21 for non-specific metabolism in the brain, Cluster 39 for non-specific transcription in cell lines, and Cluster 5 for enterocyte absorption in single cells.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD9](https://www.ncbi.nlm.nih.gov/gene/65980) with 6 experiments, followed by [HNF4A](https://www.ncbi.nlm.nih.gov/gene/3172), [CREB1](https://www.ncbi.nlm.nih.gov/gene/1385), [GATAD2A](https://www.ncbi.nlm.nih.gov/gene/54815), and [JUN](https://www.ncbi.nlm.nih.gov/gene/3725), each with 5 experiments.



The input data indicates associations with a range of disease conditions, including metabolic disorders such as obesity and overnutrition, cardiovascular diseases like coronary artery disease, artery disease, and vascular disease, as well as mental health conditions such as cognitive disorder, bipolar disorder, and mood disorder. Additionally, the data encompasses broader categories such as disease of anatomical entity and general disease conditions.



The gene is expressed in various tissues, including the brain and colon, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in CD105+ endothelial cells, the amygdala, whole brain, hypothalamus, spinal cord, occipital lobe, and colon, suggesting a ubiquitous expression pattern across multiple tissue samples.


The input data highlights several key pathways involving proteins, including transmembrane transport of small molecules, ion channel transport, and stimuli-sensing channels.



The analyzed protein sequence has a GRAVY value of -0.321 (55.56th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -3.36 (38.38th percentile), and the median structural flexibility is 1.000 (43.97th percentile). The protein has a propensity for helix, sheet, and turn structures of 23.74% (3.22nd percentile), 33.33% (47.43rd percentile), and 33.84% (85.38th percentile), respectively. The instability index is 41.81 (28.08th percentile), with an isoelectric point of 5.32 (13.89th percentile). The protein length is 198 amino acids (14.18th percentile), and the molecular weight is 21676.15 Da (13.66th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |