---
layout: post
title: ATOX1
date: 2025-01-17 16:55 CST
description: ATOX1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/475) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 475  | ATOX1 | ENSG00000177556 | 5q33.1 |



The gene enables [copper ion binding](https://amigo.geneontology.org/amigo/term/GO:0005507) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is active in and located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). It is involved in [copper ion transport](https://amigo.geneontology.org/amigo/term/GO:0006825), [intracellular copper ion homeostasis](https://amigo.geneontology.org/amigo/term/GO:0006878), and [response to oxidative stress](https://amigo.geneontology.org/amigo/term/GO:0006979). Additionally, the gene enables [metallochaperone activity](https://amigo.geneontology.org/amigo/term/GO:0016530), [copper chaperone activity](https://amigo.geneontology.org/amigo/term/GO:0016531), [copper-dependent protein binding](https://amigo.geneontology.org/amigo/term/GO:0032767), and [cuprous ion binding](https://amigo.geneontology.org/amigo/term/GO:1903136).


The gene length is 120,971 base pairs (85.32% of all genes), the mature length is 1,847 base pairs, and the primary transcript length is 44,213 base pairs.


The gene ATOX1 (NCBI ID: 475) has been mentioned in [90 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ATOX1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning ATOX1 is 1997, and the middle 50% of publications occurred between 2006 and 2016.


The top 5 publications mentioning ATOX1, ranked by their scientific influence, include "[Identification and functional expression of HAH1, a novel human gene involved in copper homeostasis.](https://pubmed.ncbi.nlm.nih.gov/9083055)" (1997) (relative citation ratio: 7.03), "[Single-cell tracking demonstrates copper chaperone Atox1 to be required for breast cancer cell migration.](https://pubmed.ncbi.nlm.nih.gov/31932435)" (2020) (relative citation ratio: 5.3), "[Crystal structure of the Atx1 metallochaperone protein at 1.02 A resolution.](https://pubmed.ncbi.nlm.nih.gov/10404590)" (1999) (relative citation ratio: 4.47), "[The Atx1-Ccc2 complex is a metal-mediated protein-protein interaction.](https://pubmed.ncbi.nlm.nih.gov/16732294)" (2006) (relative citation ratio: 4.07), and "[Energetics of copper trafficking between the Atx1 metallochaperone and the intracellular copper transporter, Ccc2.](https://pubmed.ncbi.nlm.nih.gov/10764731)" (2000) (relative citation ratio: 3.61). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ATOX1](https://www.proteinatlas.org/ENSG00000177556-ATOX1) is a gene that functions as a chaperone, with evidence supporting its role at the protein level. Its RNA is detected in all tissues, and the protein is localized to the nucleoplasm and plasma membrane. The gene is expressed in various clusters, including Cluster 29 in blood, Cluster 60 in tissues (associated with transmembrane transport in the choroid plexus), Cluster 27 in the brain, Cluster 9 in skin cancer cell lines (related to melanin biosynthesis), and Cluster 56 in single cells (associated with amino acid metabolism in proximal tubular cells).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908), [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), and [HDAC1](https://www.ncbi.nlm.nih.gov/gene/3065), each with 5 experiments. Additionally, [SSRP1](https://www.ncbi.nlm.nih.gov/gene/6749) and [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170) were found to be regulating in 4 experiments each.





The gene is ubiquitously expressed in multiple tissue samples, including the adrenal gland, spleen, and kidney, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in cardiac myocytes, heart, liver, B lymphoblasts, smooth muscle, bronchial epithelial cells, colorectal adenocarcinoma, and kidney.


The input data highlights two key biological processes: cellular responses to stress and the detoxification of reactive oxygen species. These processes are crucial for maintaining cellular homeostasis and protecting cells from oxidative damage.



The protein sequence analyzed has a GRAVY value of -0.15147058823529425 (73.84th percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is -0.34122728274359915 (48.88th percentile), and the median structural flexibility is 1.0017142857142858 (49.66th percentile). The protein has a high affinity for helix structures (39.71%, 92.88th percentile) and sheet structures (35.29%, 61.29th percentile), with a lower affinity for turn structures (27.94%, 46.7th percentile). The instability index is 35.56323529411764 (11.3th percentile), and the isoelectric point is 6.711116218566894 (46.46th percentile). The protein has a length of 68 amino acids (0.55th percentile) and a molecular weight of 7401.5596 Da (0.48th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |