---
layout: post
title: PTPN6
date: 2025-01-17 16:55 CST
description: PTPN6 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5777) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5777  | PTPN6 | ENSG00000111679 | 12p13.31 |



The gene is involved in various biological processes, including the [MAPK cascade](https://amigo.geneontology.org/amigo/term/GO:0000165), [mitotic cell cycle](https://amigo.geneontology.org/amigo/term/GO:0000278), and [hematopoietic progenitor cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0002244). It enables functions such as [phosphotyrosine residue binding](https://amigo.geneontology.org/amigo/term/GO:0001784), [protein tyrosine phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0004725), and [non-membrane spanning protein tyrosine phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0004726). The gene is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [nucleolus](https://amigo.geneontology.org/amigo/term/GO:0005730), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [cell-cell junction](https://amigo.geneontology.org/amigo/term/GO:0005911), and [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020). Additionally, it is involved in processes like [protein dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0006470), [G protein-coupled receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007186), and [cytokine-mediated signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0019221). The gene also plays a role in [cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0030154), [platelet formation](https://amigo.geneontology.org/amigo/term/GO:0030220), and [T cell costimulation](https://amigo.geneontology.org/amigo/term/GO:0031295). Furthermore, it is part of the [protein-containing complex](https://amigo.geneontology.org/amigo/term/GO:0032991) and is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737).


The gene length is 6,412.0 base pairs (17.65% of all genes), the mature length is 1,513.0 base pairs, and the primary transcript length is 6,199.5 base pairs.


The gene PTPN6 (NCBI ID: 5777) has been mentioned in [340 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PTPN6%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1973 and the middle 50% of publications occurring between 2001 and 2016. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning PTPN6, ranked by their scientific influence, include "[Specific recruitment of SH-PTP1 to the erythropoietin receptor causes inactivation of JAK2 and termination of proliferative signals.](https://pubmed.ncbi.nlm.nih.gov/7889566)" (1995) (relative citation ratio: 16.01), "[Recruitment of tyrosine phosphatase HCP by the killer cell inhibitor receptor.](https://pubmed.ncbi.nlm.nih.gov/8574854)" (1996) (relative citation ratio: 10.0), "[Deletion of SHIP or SHP-1 reveals two distinct pathways for inhibitory signaling.](https://pubmed.ncbi.nlm.nih.gov/9244303)" (1997) (relative citation ratio: 6.29), "[Direct regulation of ZAP-70 by SHP-1 in T cell antigen receptor signaling.](https://pubmed.ncbi.nlm.nih.gov/8638162)" (1996) (relative citation ratio: 5.47), and "[Capillarisin inhibits constitutive and inducible STAT3 activation through induction of SHP-1 and SHP-2 tyrosine phosphatases.](https://pubmed.ncbi.nlm.nih.gov/24333736)" (2014) (relative citation ratio: 4.9). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PTPN6](https://www.proteinatlas.org/ENSG00000111679-PTPN6) is a gene encoding a protein with hydrolase and protein phosphatase molecular functions. The protein is detected at the protein level and is localized in the nucleoplasm and nucleoli. The gene is expressed in many tissues, with notable expression in lymphoid tissue, macrophages, and microglia, suggesting a role in immune response. It is also expressed in lymphoma cell lines and B-cells, further indicating its involvement in immune function. The gene's expression in various cancers is unprognostic, implying that its expression levels do not significantly correlate with patient outcomes in the studied cancer types.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 6 experiments, [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613) with 5 experiments, [AR](https://www.ncbi.nlm.nih.gov/gene/367) with 5 experiments, [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 5 experiments, and [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664) with 5 experiments.



The GWAS data indicates associations with various conditions, including cognitive disorder, bipolar disorder, and mood disorder. These findings highlight the potential genetic underpinnings of mental health diseases.



The gene is ubiquitously expressed in multiple tissue samples, including blood, spleen, and small intestine, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in various immune cell types such as CD33+ Myeloid cells, CD56+ NK cells, CD4+ T cells, CD8+ T cells, CD19+ B cells (neg. sel.), CD14+ Monocytes, and Whole Blood.


The input data highlights various pathways involved in immune system regulation and cellular communication. Key pathways include cytokine signaling in the immune system, interferon signaling (including interferon gamma, alpha/beta, and their regulation), and signaling by interleukins (such as interleukin-2, -3, -5, and GM-CSF). Additionally, the data covers adaptive immune system processes, costimulation by the CD28 family, and interactions involving PECAM1 and the SIRP family. Other notable pathways include PD-1 signaling, signal transduction, platelet homeostasis, and cell-cell communication, as well as cell surface interactions at the vascular wall and hemostasis.



The analyzed protein sequence has a GRAVY value of -0.687 (18.45th percentile), indicating a hydrophilic nature. It exhibits a charge of 2.093 at pH 7.0 (57.93rd percentile) and a median structural flexibility of 1.005 (62.13th percentile). The protein's secondary structure is predicted to be 29.58% helix (28.23rd percentile), 33.61% sheet (49.29th percentile), and 28.40% turn (51.01st percentile). The instability index is 44.88 (38.8th percentile), with an isoelectric point of 7.65 (55.73rd percentile). The protein has a length of 595 amino acids (68.2nd percentile) and a molecular weight of 67560.32 Da (68.87th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |