---
layout: post
title: NOP56
date: 2025-01-17 16:55 CST
description: NOP56 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10528) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10528  | NOP56 | ENSG00000101361 | 20p13 |



The gene is located in the [fibrillar center](https://amigo.geneontology.org/amigo/term/GO:0001650), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [nucleolus](https://amigo.geneontology.org/amigo/term/GO:0005730), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), and [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020). It is part of the [sno(s)RNA-containing ribonucleoprotein complex](https://amigo.geneontology.org/amigo/term/GO:0005732), [box C/D RNP complex](https://amigo.geneontology.org/amigo/term/GO:0031428), [small-subunit processome](https://amigo.geneontology.org/amigo/term/GO:0032040), and [pre-snoRNP complex](https://amigo.geneontology.org/amigo/term/GO:0070761). The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [snoRNA binding](https://amigo.geneontology.org/amigo/term/GO:0030515), [cadherin binding](https://amigo.geneontology.org/amigo/term/GO:0045296), and [histone methyltransferase binding](https://amigo.geneontology.org/amigo/term/GO:1990226). Additionally, it is involved in [rRNA processing](https://amigo.geneontology.org/amigo/term/GO:0006364) and [ribosomal small subunit biogenesis](https://amigo.geneontology.org/amigo/term/GO:0042274).


The gene length is 18,966 base pairs (40.81 percentile of all genes), the mature length is 700 base pairs, and the primary transcript length is 10,333 base pairs.


The gene NOP56 (NCBI ID: 10528) has been mentioned in [16 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22NOP56%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning this gene was in 1997, and the middle 50% of publications occurred between 2012 and 2021.


The top 5 publications mentioning NOP56, ranked by their relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows:

- [Nucleolar KKE/D repeat proteins Nop56p and Nop58p interact with Nop1p and are required for ribosome biogenesis.](https://pubmed.ncbi.nlm.nih.gov/9372940) (1997) (relative citation ratio: 3.72)
- [Association between promoter methylation and gene expression of <i>CGB3</i> and <i>NOP56</i> in HPV-infected cervical cancer cells.](https://pubmed.ncbi.nlm.nih.gov/34820124) (2022) (relative citation ratio: 1.76)
- [Evidence that the AAA+ proteins TIP48 and TIP49 bridge interactions between 15.5K and the related NOP56 and NOP58 proteins during box C/D snoRNP biogenesis.](https://pubmed.ncbi.nlm.nih.gov/19620283) (2009) (relative citation ratio: 1.07)
- [Identification of the Largest SCA36 Pedigree in Asia: with Multimodel Neuroimaging Evaluation for the First Time.](https://pubmed.ncbi.nlm.nih.gov/34264505) (2022) (relative citation ratio: 0.98)
- [Frequency of SCA8, SCA10, SCA12, SCA36, FXTAS and C9orf72 repeat expansions in SCA patients negative for the most common SCA subtypes.](https://pubmed.ncbi.nlm.nih.gov/29316893) (2018) (relative citation ratio: 0.85)

Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[NOP56](https://www.proteinatlas.org/ENSG00000101361-NOP56) is a ribonucleoprotein involved in neurodegeneration and spinocerebellar ataxia, with evidence at the protein level. It is detected in all RNA tissues and is primarily located in the nucleoli fibrillar center. NOP56 is part of various expression clusters, including Cluster 12 for blood (non-specific - mitochondrial translation), Cluster 16 for tissue (non-specific - transcription), Cluster 14 for brain (neurons - synaptic function), Cluster 62 for cell line (non-specific - plasma proteins), and Cluster 15 for single cell (non-specific - mixed function).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908), [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), and [GATA3](https://www.ncbi.nlm.nih.gov/gene/2625), each with 6 experiments. Additionally, [ZBTB7A](https://www.ncbi.nlm.nih.gov/gene/51341) and [CHD1](https://www.ncbi.nlm.nih.gov/gene/1105) were found to be regulating in 5 experiments each.


The transcription factor [9968](https://www.ncbi.nlm.nih.gov/gene/9968) was found to be regulating genes in 2 Perturb-seq experiments, according to data from GTRD's database of transcription factor binding sites and the study published in [PubMed](https://pubmed.ncbi.nlm.nih.gov/35688146/).




The gene is expressed in various tissues and cell types, including the spleen, chronic myelogenous leukemia (K-562), promyelocytic leukemia (HL-60), Burkitt's lymphoma (Daudi), colorectal adenocarcinoma, skeletal muscle, and lymphoblastic leukemia (MOLT-4). This indicates a ubiquitous expression pattern across multiple tissue samples.


The input data highlights several key processes involved in protein metabolism and folding. These include the broader category of metabolism of proteins, as well as more specific pathways such as the association of TriC/CCT with target proteins during biosynthesis, chaperonin-mediated protein folding, and general protein folding.



The analyzed protein sequence has a GRAVY value of -0.471 (38.13th percentile), indicating a hydrophilic nature. It exhibits a charge of 17.77 at pH 7.0 (91.81st percentile) and a median structural flexibility of 1.009 (77.3rd percentile). The protein's secondary structure is predicted to be predominantly helical, with 41.75% of amino acids in helix form (95.76th percentile), 30.30% in sheet form (29.94th percentile), and 26.26% in turn form (31.49th percentile). The instability index is 47.16 (47.2nd percentile), and the isoelectric point is 9.24 (85.67th percentile). The protein has a length of 594 amino acids (68.11th percentile) and a molecular weight of 66049.11 Da (67.9th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |