---
layout: post
title: SSTR1
date: 2025-01-17 16:55 CST
description: SSTR1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6751) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6751  | SSTR1 | ENSG00000139874 | 14q21.1 |



The gene enables [somatostatin receptor activity](https://amigo.geneontology.org/amigo/term/GO:0004994) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). It is involved in various signaling pathways, including the [G protein-coupled receptor signaling pathway, coupled to cyclic nucleotide second messenger](https://amigo.geneontology.org/amigo/term/GO:0007187), [glutamate receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007215), [neuropeptide signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007218), and [somatostatin signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0038170). Additionally, it plays a role in [spermatogenesis](https://amigo.geneontology.org/amigo/term/GO:0007283), [negative regulation of cell population proliferation](https://amigo.geneontology.org/amigo/term/GO:0008285), [cerebellum development](https://amigo.geneontology.org/amigo/term/GO:0021549), [forebrain development](https://amigo.geneontology.org/amigo/term/GO:0030900), [response to starvation](https://amigo.geneontology.org/amigo/term/GO:0042594), [cellular response to estradiol stimulus](https://amigo.geneontology.org/amigo/term/GO:0071392), and [cellular response to leukemia inhibitory factor](https://amigo.geneontology.org/amigo/term/GO:1990830). The gene is active in the [neuron projection](https://amigo.geneontology.org/amigo/term/GO:0043005) and enables [neuropeptide binding](https://amigo.geneontology.org/amigo/term/GO:0042923).


The gene length is 15,600 base pairs (35.98th percentile), the mature length is 349 base pairs, and the primary transcript length is 14,134 base pairs.


The gene SSTR1 (NCBI ID: 6751) has been mentioned in [30 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SSTR1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1994 and the middle 50% of publications ranging from 1997 to 2014. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning SSTR1, ranked by their scientific influence, include "[Expression and localization of somatostatin receptor SSTR1, SSTR2, and SSTR3 messenger RNAs in primary human tumors using in situ hybridization.](https://pubmed.ncbi.nlm.nih.gov/8012966)" (1994) (relative citation ratio: 9.03), "[Stimulation of tyrosine phosphatase and inhibition of cell proliferation by somatostatin analogues: mediation by human somatostatin receptor subtypes SSTR1 and SSTR2.](https://pubmed.ncbi.nlm.nih.gov/7907795)" (1994) (relative citation ratio: 8.64), "[Expression of somatostatin receptors (SSTR1-SSTR5) in meningiomas and its clinicopathological significance.](https://pubmed.ncbi.nlm.nih.gov/26722517)" (2015) (relative citation ratio: 2.28), "[Development of a selective agonist at the somatostatin receptor subtype sstr1.](https://pubmed.ncbi.nlm.nih.gov/8786539)" (1996) (relative citation ratio: 2.27), and "[Expression of Somatostatin Receptor (SSTR) Subtypes (SSTR-1, 2A, 3, 4 and 5) in Neuroendocrine Tumors Using Real-time RT-PCR Method and Immunohistochemistry.](https://pubmed.ncbi.nlm.nih.gov/22829710)" (2012) (relative citation ratio: 2.2). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[SSTR1](https://www.proteinatlas.org/ENSG00000139874-SSTR1) is a G-protein coupled receptor involved in transduction, with evidence at the protein level. It is an FDA-approved drug target and is detected in many tissues. The gene is primarily located in vesicles and is highly expressed in the small intestine, involved in absorption processes. In the brain, it is found in neurons with mixed functions, while in cell lines, it is associated with connective tissue cells involved in extracellular matrix organization. Single-cell expression analysis shows its presence in enteroendocrine cells, contributing to hormone signaling.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [MAX](https://www.ncbi.nlm.nih.gov/gene/4149), [ELF3](https://www.ncbi.nlm.nih.gov/gene/1999), [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), and [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599), each shown to be regulating in 2 experiments.





The gene is expressed in various tissues, including the brain, lung, and stomach, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the Prefrontal Cortex, Cardiac Myocytes, Heart, Liver, Skeletal Muscle, and Atrioventricular Node, as well as in Lymphoma Burkitts (Raji and Daudi).


The input data includes various pathways and processes involving proteins, such as class a/1 (rhodopsin-like receptors), defective acth causing obesity and pro-opiomelanocortin deficiency (pomcd), gpcr ligand binding, metabolic disorders of biological oxidation enzymes, peptide ligand-binding receptors, g alpha (i) signalling events, disease, gpcr downstream signaling, signaling by gpcr, and signal transduction.



The analyzed protein sequence has a GRAVY value of 0.488 (94.95th percentile), indicating a high level of hydrophobicity. The protein exhibits a charge of 5.58 at pH 7.0 (69.8th percentile) and a median structural flexibility of 0.984 (6.91th percentile). The secondary structure affinities are notable, with 29.92% of amino acids tending to be in a helix (31.31st percentile), 42.20% in a sheet (91.11th percentile), and 27.88% in a turn (46.12th percentile). The instability index is 52.30 (64.54th percentile), and the isoelectric point is 8.68 (70.63rd percentile). The protein has a length of 391 amino acids (44.06th percentile) and a molecular weight of 42685.74 Da (43.06th percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |