---
layout: post
title: DNASE1L1
date: 2025-01-17 16:55 CST
description: DNASE1L1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1774) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1774  | DNASE1L1 | ENSG00000013563 | Xq28  |



The gene enables [DNA binding](https://amigo.geneontology.org/amigo/term/GO:0003677), [deoxyribonuclease I activity](https://amigo.geneontology.org/amigo/term/GO:0004530), [DNA nuclease activity](https://amigo.geneontology.org/amigo/term/GO:0004536), and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [endoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0005783), and [specific granule lumen](https://amigo.geneontology.org/amigo/term/GO:0035580), and is active in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634). Additionally, it is involved in the [DNA metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006259) and [DNA catabolic process](https://amigo.geneontology.org/amigo/term/GO:0006308).


The gene length is 34,195 base pairs (56.3% of all genes), the mature length is 853 base pairs (16.24% of all genes), and the primary transcript length is 11,631 base pairs (37.27% of all genes).


The gene DNASE1L1 (NCBI ID: 1774) has been mentioned in [14 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22DNASE1L1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1999 and the middle 50% of publications occurring between 2006 and 2014. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning DNASE1L1 include "[Ehrlichia chaffeensis uses its surface protein EtpE to bind GPI-anchored protein DNase X and trigger entry into mammalian cells.](https://pubmed.ncbi.nlm.nih.gov/24098122)" (2013) (relative citation ratio: 1.24), "[<i>Ehrlichia chaffeensis</i> and Its Invasin EtpE Block Reactive Oxygen Species Generation by Macrophages in a DNase X-Dependent Manner.](https://pubmed.ncbi.nlm.nih.gov/29162709)" (2017) (relative citation ratio: 0.87), "[Functional characterization of DNase X, a novel endonuclease expressed in muscle cells.](https://pubmed.ncbi.nlm.nih.gov/10858283)" (2000) (relative citation ratio: 0.82), "[Observation of two new Ξ(b)(-) baryon resonances.](https://pubmed.ncbi.nlm.nih.gov/25723210)" (2015) (relative citation ratio: 0.8), and "[Precision measurement of the mass and lifetime of the Ξb⁻ baryon.](https://pubmed.ncbi.nlm.nih.gov/25541768)" (2014) (relative citation ratio: 0.69). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[DNASE1L1](https://www.proteinatlas.org/ENSG00000013563-DNASE1L1) is a gene with molecular functions including endonuclease, hydrolase, and nuclease activities. Evidence for its function is available at the protein level. The gene's RNA is detected in all tissues, with notable expression clusters in skeletal muscle for tissue expression, choroid plexus for brain expression, and adipocytes and endothelial cells for single-cell expression. Additionally, it is part of the non-specific membrane trafficking cluster in blood expression and the mRNA processing cluster in cell line expression.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [AR](https://www.ncbi.nlm.nih.gov/gene/367) with 5 experiments, [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930) with 4 experiments, [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) with 4 experiments, [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 4 experiments, and [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.





The gene is expressed in various tissues, including muscle, thyroid, and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Cardiac Myocytes, CD33+ Myeloid cells, BDCA4+ Dendritic Cells, CD56+ NK Cells, CD14+ Monocytes, Smooth Muscle, Bronchial Epithelial Cells, and Bone Marrow.




The protein sequence analyzed has a GRAVY value of 0.046, indicating a slightly hydrophobic nature (85.94th percentile). It exhibits a charge of -9.64 at pH 7 (23.06th percentile) and a median structural flexibility of 0.992 (14.85th percentile). The protein's secondary structure is predicted to be 33.77% helix (66.07th percentile), 41.72% sheet (90.25th percentile), and 24.83% turn (20.36th percentile). The instability index is 48.69 (52.64th percentile), with an isoelectric point of 5.45 (16.98th percentile). The protein has a length of 302 amino acids (28.85th percentile) and a molecular weight of 33892.41 Da (29.08th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |