---
layout: post
title: TGFBR1
date: 2025-01-17 16:55 CST
description: TGFBR1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7046) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7046  | TGFBR1 | ENSG00000106799 | 9q22.33 |



The gene is involved in various biological processes, including [skeletal system development](https://amigo.geneontology.org/amigo/term/GO:0001501), [in utero embryonic development](https://amigo.geneontology.org/amigo/term/GO:0001701), [kidney development](https://amigo.geneontology.org/amigo/term/GO:0001822), and [blastocyst development](https://amigo.geneontology.org/amigo/term/GO:0001824). It plays a role in [epithelial to mesenchymal transition](https://amigo.geneontology.org/amigo/term/GO:0001837) and regulates [endothelial cell proliferation](https://amigo.geneontology.org/amigo/term/GO:0001935), both positively and negatively. The gene enables [protein kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004672), specifically [protein serine/threonine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004674), and contributes to [transforming growth factor beta receptor activity](https://amigo.geneontology.org/amigo/term/GO:0005024). It is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [endosome](https://amigo.geneontology.org/amigo/term/GO:0005768), and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), and is active in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). Additionally, the gene is involved in processes such as [regulation of DNA-templated transcription](https://amigo.geneontology.org/amigo/term/GO:0006355), [protein phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0006468), and [apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0006915). It also plays a role in [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165) and the [transforming growth factor beta receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007179). The gene is part of the [receptor complex](https://amigo.geneontology.org/amigo/term/GO:0043235) and the [activin receptor complex](https://amigo.geneontology.org/amigo/term/GO:0048179), enabling [activin binding](https://amigo.geneontology.org/amigo/term/GO:0048185) and [SMAD binding](https://amigo.geneontology.org/amigo/term/GO:0046332). It contributes to various developmental processes, including [nervous system development](https://amigo.geneontology.org/amigo/term/GO:0007399), [heart development](https://amigo.geneontology.org/amigo/term/GO:0007507), and [male gonad development](https://amigo.geneontology.org/amigo/term/GO:0008584). The gene is also involved in the regulation of cell migration, both positively and negatively, and in the regulation of cell growth and proliferation.


The gene length is 99,960 base pairs (81.74% of all genes), the mature length is 6,164 base pairs (94.58% of all genes), and the primary transcript length is 99,932 base pairs (88.23% of all genes).


The gene TGFBR1 (NCBI ID: 7046) has been mentioned in [412 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22TGFBR1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning this gene is 1993, and the middle 50% of publications occurred between 2008 and 2019.


The top 5 publications mentioning TGFBR1, ranked by their scientific influence, include "[A syndrome of altered cardiovascular, craniofacial, neurocognitive and skeletal development caused by mutations in TGFBR1 or TGFBR2.](https://pubmed.ncbi.nlm.nih.gov/15731757)" (2005) (relative citation ratio: 25.73), "[SB-431542 is a potent and specific inhibitor of transforming growth factor-beta superfamily type I activin receptor-like kinase (ALK) receptors ALK4, ALK5, and ALK7.](https://pubmed.ncbi.nlm.nih.gov/12065756)" (2002) (relative citation ratio: 24.52), "[M6A RNA methylation-mediated RMRP stability renders proliferation and progression of non-small cell lung cancer through regulating TGFBR1/SMAD2/SMAD3 pathway.](https://pubmed.ncbi.nlm.nih.gov/34628486)" (2023) (relative citation ratio: 20.35), "[Cloning of a TGF beta type I receptor that forms a heteromeric complex with the TGF beta type II receptor.](https://pubmed.ncbi.nlm.nih.gov/8242743)" (1993) (relative citation ratio: 15.31), and "[Activin receptor-like kinase (ALK)1 is an antagonistic mediator of lateral TGFbeta/ALK5 signaling.](https://pubmed.ncbi.nlm.nih.gov/14580334)" (2003) (relative citation ratio: 10.51). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[TGFBR1](https://www.proteinatlas.org/ENSG00000106799-TGFBR1) is a kinase and receptor with serine/threonine-protein kinase and transferase molecular functions. It is involved in diseases such as aortic aneurysm and craniosynostosis, with evidence at the protein level. The gene is detected in all RNA tissue distributions and is primarily located in the plasma membrane and primary cilium. It is expressed in various clusters, including Cluster 42 for non-specific vesicular transport in blood, Cluster 58 for placenta-related processes in tissues, Cluster 20 for immune response in the brain, Cluster 33 for muscle contraction in cell lines, and Cluster 28 for antigen presentation in single cells.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908) with 6 experiments, [SMAD2](https://www.ncbi.nlm.nih.gov/gene/4087) with 5 experiments, [EP300](https://www.ncbi.nlm.nih.gov/gene/2033) with 5 experiments, [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227) with 5 experiments, and [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with a range of disease conditions, including various types of cancer such as gastrointestinal system cancer, organ system cancer, and biliary tract cancer, specifically gallbladder cancer. Additionally, the data highlights metabolic disorders, including disease of metabolism, acquired metabolic disease, and nutrition disease, with a particular focus on overnutrition and obesity. Kidney and urinary system diseases are also implicated, along with nervous system diseases and several eye-related conditions, notably retinal disease, retinal degeneration, and age-related macular degeneration.



The gene is expressed in various tissues, including the blood vessel and stomach. Additionally, it is expressed in multiple tissue samples such as the prefrontal cortex, cardiac myocytes, heart, liver, CD56+ NK cells, and lymphoma (Burkitt's, including Raji and Daudi cell lines).


The input data highlights various pathways and processes involving TGF-beta receptor signaling, including its downregulation and activation of SMADs. Notably, several pathways are associated with cancer, such as the loss of function of SMAD2/3, SMAD4, TGFBR1, and TGFBR2, as well as mutations in TGFBR2 kinase domain and frameshift mutants due to microsatellite instability (MSI). Additionally, the data includes pathways related to TGF-beta receptor complex signaling, SMAD mutations, and the role of TGF-beta signaling in epithelial to mesenchymal transition (EMT). Broad categories of disease and signal transduction are also mentioned.



The analyzed protein sequence has a GRAVY value of -0.055 (81.04th percentile), indicating a hydrophilic nature. It exhibits a charge of 1.78 at pH 7.0 (56.69th percentile) and a median structural flexibility of 0.993 (16.06th percentile). The protein's secondary structure is predicted to be 32.01% helix (50.4th percentile), 36.78% sheet (71.63rd percentile), and 24.85% turn (20.52nd percentile). The instability index is 43.48 (33.83rd percentile), with an isoelectric point of 7.51 (54.07th percentile). The protein has a length of 503 amino acids (58.61st percentile) and a molecular weight of 55959.16 Da (58.49th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |