---
layout: post
title: GNA13
date: 2025-01-17 16:55 CST
description: GNA13 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10672) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10672  | GNA13 | ENSG00000120063 | 17q24.1 |



The gene is involved in various biological processes, including [branching involved in blood vessel morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0001569), [in utero embryonic development](https://amigo.geneontology.org/amigo/term/GO:0001701), and [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165). It enables molecular functions such as [GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0003924), [guanyl-nucleotide exchange factor activity](https://amigo.geneontology.org/amigo/term/GO:0005085), and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is located in multiple cellular components, including the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), and [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925). Additionally, it is part of the [heterotrimeric G-protein complex](https://amigo.geneontology.org/amigo/term/GO:0005834) and is active in the [brush border membrane](https://amigo.geneontology.org/amigo/term/GO:0031526). The gene also plays roles in the regulation of various processes, such as [positive regulation of cytosolic calcium ion concentration](https://amigo.geneontology.org/amigo/term/GO:0007204), [Rho protein signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007266), and [regulation of blood pressure](https://amigo.geneontology.org/amigo/term/GO:0008217).


The gene length is 10,452 base pairs (26.63% of all genes), the mature length is 3,239 base pairs, and the primary transcript length is 10,086 base pairs (33.5% of all genes).


GNA13 (NCBI Gene ID: 10672) has been mentioned in [143 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22GNA13%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1994 and the middle 50% of publications occurring between 2004 and 2017. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning GNA13, ranked by their scientific influence, include "[p115 RhoGEF, a GTPase activating protein for Galpha12 and Galpha13.](https://pubmed.ncbi.nlm.nih.gov/9641915)" (1998) (relative citation ratio: 13.18), "[Amphetamines signal through intracellular TAAR1 receptors coupled to Gα<sub>13</sub> and Gα<sub>S</sub> in discrete subcellular domains.](https://pubmed.ncbi.nlm.nih.gov/31399635)" (2021) (relative citation ratio: 6.53), "[Activation of G12/G13 results in shape change and Rho/Rho-kinase-mediated myosin light chain phosphorylation in mouse platelets.](https://pubmed.ncbi.nlm.nih.gov/10037795)" (1999) (relative citation ratio: 5.68), "[Loss of signalling via Gα13 in germinal centre B-cell-derived lymphoma.](https://pubmed.ncbi.nlm.nih.gov/25274307)" (2014) (relative citation ratio: 5.67), and "[The Structural Basis of Oncogenic Mutations G12, G13 and Q61 in Small GTPase K-Ras4B.](https://pubmed.ncbi.nlm.nih.gov/26902995)" (2016) (relative citation ratio: 4.32). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[GNA13](https://www.proteinatlas.org/ENSG00000120063-GNA13) is a transducer gene involved in cancer-related processes. Evidence for its function is available at the protein level. The gene is detected in all RNA tissue distributions and is primarily located in the cytosol. It is expressed in neutrophils with unknown function (Cluster 28) and is associated with cell proliferation in bone marrow (Cluster 77). In the brain, it is involved in non-specific transcription (Cluster 55), while in cell lines, it is linked to the humoral immune response in lymphoma (Cluster 46). Additionally, GNA13 is involved in antigen presentation in dendritic cells (Cluster 28).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885) and [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), each with 5 experiments. Additionally, [KLF5](https://www.ncbi.nlm.nih.gov/gene/688), [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), and [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908) were each found to be regulating in 4 experiments.



The input data indicates associations with a range of disease conditions, including asthma, bronchial disease, obstructive lung disease, lower respiratory tract disease, lung disease, respiratory system disease, artery disease, vascular disease, cardiovascular system disease, nervous system disease, central nervous system disease, brain disease, cerebral arterial disease, intracranial arterial disease, brain ischemia, ischemia, and cerebrovascular disease.



The gene is expressed in various tissues, including muscle, lung, and blood, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in pineal gland during the day, cardiac myocytes, Burkitt's lymphoma (Raji), heart, B lymphoblasts (721), BDCA4+ dendritic cells, CD8+ T cells, and whole blood.


The proteins are involved in various signaling pathways, including G alpha (12/13) signaling events, signal amplification, thromboxane signaling through TP receptor, and thrombin signaling through proteinase activated receptors (PARs). Additionally, they participate in GPCR downstream signaling, signaling by GPCR, and broader signal transduction processes. These proteins also play roles in hemostasis and platelet activation, signaling, and aggregation.


The mouse ortholog gene 14674 and its human ortholog 10672 are associated with several phenotypic abnormalities. Both sexes exhibit abnormal embryo size and abnormal visceral yolk sac morphology. Additionally, there is embryonic lethality prior to the tooth bud stage, although this is not observed in both sexes. Complete penetrance preweaning lethality is also noted in both sexes.


The analyzed protein sequence has a GRAVY value of -0.426 (43.18th percentile), indicating a hydrophilic nature. It exhibits a charge of 2.207 at pH 7.0 (58.35th percentile) and a median structural flexibility of 0.998 (33.13th percentile). The protein's secondary structure is predicted to be 30.5% helix (36.37th percentile), 37.14% sheet (73.87th percentile), and 25.46% turn (24.89th percentile). The instability index is 52.97 (66.53rd percentile), with an isoelectric point of 8.12 (60.09th percentile). The protein is 377 amino acids long (42.23rd percentile) and has a molecular weight of 44049.06 Da (44.63rd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |