---
layout: post
title: TARS1
date: 2025-01-17 16:55 CST
description: TARS1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6897) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6897  | TARS1 | ENSG00000113407 | 5p13.3 |



The gene enables [tRNA binding](https://amigo.geneontology.org/amigo/term/GO:0000049) and [threonine-tRNA ligase activity](https://amigo.geneontology.org/amigo/term/GO:0004829), and is involved in [threonyl-tRNA aminoacylation](https://amigo.geneontology.org/amigo/term/GO:0006435). It also enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524), [zinc ion binding](https://amigo.geneontology.org/amigo/term/GO:0008270), and [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802). The gene is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829) and the [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062).


The gene length is 4,595 base pairs (12.88% of all genes), the mature length is 573 base pairs (3.87% of all genes), and the primary transcript length is 1,815 base pairs (5.7% of all genes).


The gene TARS1 (NCBI ID: 6897) has been mentioned in [38 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22TARS1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1982 and the middle 50% of publications ranging from 1993 to 2018. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning TARS1, ranked by their scientific influence, include "[Anti-threonyl-tRNA synthetase, a second myositis-related autoantibody.](https://pubmed.ncbi.nlm.nih.gov/6206177)" (1984) (relative citation ratio: 5.55), "[Anti-PL-7 (anti-threonyl-tRNA synthetase) antisynthetase syndrome: clinical manifestations in a series of patients from a European multicenter study (EUMYONET) and review of the literature.](https://pubmed.ncbi.nlm.nih.gov/22732951)" (2012) (relative citation ratio: 3.31), "[Transfer RNA-mediated editing in threonyl-tRNA synthetase. The class II solution to the double discrimination problem.](https://pubmed.ncbi.nlm.nih.gov/11136973)" (2000) (relative citation ratio: 2.78), "[Zinc ion mediated amino acid discrimination by threonyl-tRNA synthetase.](https://pubmed.ncbi.nlm.nih.gov/10881191)" (2000) (relative citation ratio: 2.49), and "[Mitochondrial Threonyl-tRNA Synthetase TARS2 Is Required for Threonine-Sensitive mTORC1 Activation.](https://pubmed.ncbi.nlm.nih.gov/33340489)" (2021) (relative citation ratio: 2.13). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[TARS1](https://www.proteinatlas.org/ENSG00000113407-TARS1) is a gene with molecular functions including aminoacyl-tRNA synthetase, ligase, RNA-binding, and tRNA-binding. It is involved in disease variants and has evidence at the protein level. The gene is detected in all RNA tissue distributions and is localized in actin filaments and the cytosol. TARS1 is part of Cluster 5 for blood expression, associated with non-specific cell proliferation, and Cluster 68 for tissue expression, linked to non-specific protein processing. In the brain, it is part of Cluster 26, associated with oligodendrocytes and mixed function. Additionally, TARS1 is found in Cluster 29 for cell line expression and Cluster 59 for single cell expression, related to suprabasal keratinocytes and cornification.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2B](https://www.ncbi.nlm.nih.gov/gene/9757) with 6 experiments, followed by [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113), [YY1](https://www.ncbi.nlm.nih.gov/gene/7528), [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313), and [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), each with 5 experiments.



The GWAS data indicates associations with several disease conditions, including artery disease, vascular disease, cardiovascular system disease, hypertension, and broader categories such as disease and disease of anatomical entity.



The gene is expressed in multiple tissue samples, including liver, skin, and colon. Additionally, it is expressed in various cell types such as cardiac myocytes, B lymphoblasts, chronic myelogenous leukemia cells (K-562), Burkitt's lymphoma cells (Daudi), CD105+ endothelial cells, smooth muscle cells, bronchial epithelial cells, and lymphoblastic leukemia cells (MOLT-4).


The input data indicates involvement in several biological processes, including gene expression, cytosolic tRNA aminoacylation, and tRNA aminoacylation.



The analyzed protein sequence has a GRAVY value of -0.579 (27.16th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -6.85 (28.86th percentile), and the median structural flexibility is 1.002 (52.08th percentile). The protein has a length of 723 amino acids (76.69th percentile) and a molecular weight of 83434.13 Da (78.07th percentile). The isoelectric point is 6.23 (36.46th percentile), and the instability index is 41.13 (25.78th percentile). The secondary structure affinities are 34.16% for helix (69.09th percentile), 33.61% for sheet (49.26th percentile), and 26.00% for turn (29.11th percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |