---
layout: post
title: CALB1
date: 2025-01-17 16:55 CST
description: CALB1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/793) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 793  | CALB1 | ENSG00000104327 | 8q21.3 |



The gene enables [vitamin D binding](https://amigo.geneontology.org/amigo/term/GO:0005499), [calcium ion binding](https://amigo.geneontology.org/amigo/term/GO:0005509), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [zinc ion binding](https://amigo.geneontology.org/amigo/term/GO:0008270). It is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [axon](https://amigo.geneontology.org/amigo/term/GO:0030424), [dendrite](https://amigo.geneontology.org/amigo/term/GO:0030425), [stereocilium](https://amigo.geneontology.org/amigo/term/GO:0032420), [cuticular plate](https://amigo.geneontology.org/amigo/term/GO:0032437), [neuronal cell body](https://amigo.geneontology.org/amigo/term/GO:0043025), [terminal bouton](https://amigo.geneontology.org/amigo/term/GO:0043195), [dendritic spine](https://amigo.geneontology.org/amigo/term/GO:0043197), [synapse](https://amigo.geneontology.org/amigo/term/GO:0045202), [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062), [hippocampal mossy fiber to CA3 synapse](https://amigo.geneontology.org/amigo/term/GO:0098686), [glutamatergic synapse](https://amigo.geneontology.org/amigo/term/GO:0098978), [GABA-ergic synapse](https://amigo.geneontology.org/amigo/term/GO:0098982), [presynaptic cytosol](https://amigo.geneontology.org/amigo/term/GO:0099523), and [postsynaptic cytosol](https://amigo.geneontology.org/amigo/term/GO:0099524). The gene is involved in various processes including [short-term memory](https://amigo.geneontology.org/amigo/term/GO:0007614), [long-term memory](https://amigo.geneontology.org/amigo/term/GO:0007616), [locomotory behavior](https://amigo.geneontology.org/amigo/term/GO:0007626), [retina layer formation](https://amigo.geneontology.org/amigo/term/GO:0010842), [metanephric part of ureteric bud development](https://amigo.geneontology.org/amigo/term/GO:0035502), [cellular response to organic substance](https://amigo.geneontology.org/amigo/term/GO:0071310), [metanephric collecting duct development](https://amigo.geneontology.org/amigo/term/GO:0072205), [metanephric distal convoluted tubule development](https://amigo.geneontology.org/amigo/term/GO:0072221), [metanephric connecting tubule development](https://amigo.geneontology.org/amigo/term/GO:0072286), [cochlea development](https://amigo.geneontology.org/amigo/term/GO:0090102), [regulation of presynaptic cytosolic calcium ion concentration](https://amigo.geneontology.org/amigo/term/GO:0099509), [regulation of postsynaptic cytosolic calcium ion concentration](https://amigo.geneontology.org/amigo/term/GO:0099566), and [regulation of long-term synaptic potentiation](https://amigo.geneontology.org/amigo/term/GO:1900271).


The gene length is 60,815 base pairs (71.34% of all genes), the mature length is 1,923 base pairs (50.81% of all genes), and the primary transcript length is 13,298.5 base pairs (40.89% of all genes).


The gene CALB1 (NCBI ID: 793) has been mentioned in [183 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CALB1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1982 and the middle 50% of publications occurring between 1995 and 2012. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning CALB1, ranked by their scientific influence, include "[Cellular distribution of the calcium-binding proteins parvalbumin, calbindin, and calretinin in the neocortex of mammals: phylogenetic and developmental patterns.](https://pubmed.ncbi.nlm.nih.gov/10223310)" (1999) with a relative citation ratio of 7.28, "[Synapses of double bouquet cells in monkey cerebral cortex visualized by calbindin immunoreactivity.](https://pubmed.ncbi.nlm.nih.gov/2611658)" (1989) with a relative citation ratio of 6.25, "[Proliferative and degenerative changes in striatal spiny neurons in Huntington's disease: a combined study using the section-Golgi method and calbindin D28k immunocytochemistry.](https://pubmed.ncbi.nlm.nih.gov/1836019)" (1991) with a relative citation ratio of 5.92, "[Differential localization of NADPH-diaphorase and calbindin-D28k within the cholinergic neurons of the basal forebrain, striatum and brainstem in the rat, monkey, baboon and human.](https://pubmed.ncbi.nlm.nih.gov/8336832)" (1993) with a relative citation ratio of 5.25, and "[Role of facilitated diffusion of calcium by calbindin in intestinal calcium absorption.](https://pubmed.ncbi.nlm.nih.gov/1539638)" (1992) with a relative citation ratio of 4.88. The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CALB1](https://www.proteinatlas.org/ENSG00000104327-CALB1) is a gene with evidence at the protein level. Its RNA is detected in many tissues, and the protein is localized to vesicles. The gene is expressed in various clusters, including Cluster 7 in blood, associated with non-specific innate immune response, Cluster 52 in tissues, linked to kidney transmembrane transport, Cluster 10 in the brain, involved in nucleic acid binding, Cluster 15 in kidney cancer cell lines, related to membrane components, and Cluster 66 in single cells, associated with photoreceptor cells and visual perception.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [AR](https://www.ncbi.nlm.nih.gov/gene/367), [REST](https://www.ncbi.nlm.nih.gov/gene/5978), [SMAD2](https://www.ncbi.nlm.nih.gov/gene/4087), [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), and [HDAC2](https://www.ncbi.nlm.nih.gov/gene/3066), each shown to be regulating in 4 experiments.





The gene is expressed in various tissues, including the brain, pituitary, and kidney, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Leukemia (chronic myelogenous K-562), Caudate nucleus, Hypothalamus, Thalamus, Cerebellum, Cerebellum Peduncles, Pancreas, and Kidney.



The mouse ortholog gene 12307 and its human ortholog 793 are associated with various morphological abnormalities. Both sexes exhibit abnormal heart, spleen, thymus, vertebrae, and vertebral arch morphologies, as well as enlarged heart, spleen, and thymus, kyphosis, and small spleen. Additionally, abnormal testis and uterus morphologies, small testis, and hydrometra are observed, although these phenotypes are not present in both sexes.


The protein sequence analyzed has a GRAVY value of -0.498 (34.99th percentile), indicating a hydrophilic nature. It exhibits a charge of -17.14 at pH 7.0 (13.42nd percentile) and a median structural flexibility of 1.008 (76.3rd percentile). The protein's secondary structure is predicted to be 42.53% helix (96.58th percentile), 36.02% sheet (66.35th percentile), and 25.67% turn (26.5th percentile). The instability index is 35.67 (11.48th percentile), with an isoelectric point of 4.70 (3.12nd percentile). The protein has a length of 261 amino acids (23.19th percentile) and a molecular weight of 30024.76 Da (24.36th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |