---
layout: post
title: MT1X
date: 2025-01-17 16:55 CST
description: MT1X description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4501) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4501  | MT1X | ENSG00000187193 | 16q13  |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [zinc ion binding](https://amigo.geneontology.org/amigo/term/GO:0008270), and is active in and located in both the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737). It is involved in various processes including [intracellular zinc ion homeostasis](https://amigo.geneontology.org/amigo/term/GO:0006882), [response to metal ion](https://amigo.geneontology.org/amigo/term/GO:0010038), [detoxification of copper ion](https://amigo.geneontology.org/amigo/term/GO:0010273), [cellular response to erythropoietin](https://amigo.geneontology.org/amigo/term/GO:0036018), [negative regulation of growth](https://amigo.geneontology.org/amigo/term/GO:0045926), [cellular response to cadmium ion](https://amigo.geneontology.org/amigo/term/GO:0071276), [cellular response to copper ion](https://amigo.geneontology.org/amigo/term/GO:0071280), and [cellular response to zinc ion](https://amigo.geneontology.org/amigo/term/GO:0071294). Additionally, it enables [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872).


The gene length is 36,271 base pairs (57.82% of all genes), the mature length is 4,332 base pairs (86.46% of all genes), and the primary transcript length is 19,105 base pairs (50.38% of all genes).


The gene MT1X (NCBI ID: 4501) has been mentioned in [7 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22MT1X%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 2000, and the middle 50% of publications occurred between 2006 and 2017.


The top 5 publications mentioning MT1X, ranked by their relative citation ratio (RCR), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows: [Metallothionein isoform 1 and 2 gene expression in the human prostate: downregulation of MT-1X in advanced prostate cancer.](https://pubmed.ncbi.nlm.nih.gov/10754528) (2000) (relative citation ratio: 1.54), [Metallothionein isoform 1 and 2 gene expression in the human bladder: evidence for upregulation of MT-1X mRNA in bladder cancer.](https://pubmed.ncbi.nlm.nih.gov/11270423) (2001) (relative citation ratio: 0.93), [Microarray-assisted pathway analysis identifies MT1X & NFÎºB as mediators of TCRP1-associated resistance to cisplatin in oral squamous cell carcinoma.](https://pubmed.ncbi.nlm.nih.gov/23251525) (2012) (relative citation ratio: 0.89), [T([20]) repeat in the 3'-untranslated region of the MT1X gene: a marker with high sensitivity and specificity to detect microsatellite instability in colorectal cancer.](https://pubmed.ncbi.nlm.nih.gov/22108904) (2012) (relative citation ratio: 0.54), and [Identification of Potential Biomarkers From Hepatocellular Carcinoma With MT1 Deletion.](https://pubmed.ncbi.nlm.nih.gov/34257549) (2021) (relative citation ratio: 0.32). The citation count is sourced from [iCite](https://icite.od.nih.gov).


[MT1X](https://www.proteinatlas.org/ENSG00000187193-MT1X) is a gene with evidence at the protein level. Its RNA is detected in all tissues, and it is expressed in various cell types, including T-cells, astrocytes, leukemia cells, proximal tubular cells, and others. The gene is involved in basic cellular processes, signal transduction, and amino acid metabolism. However, it is unprognostic for multiple cancer types, including bladder urothelial carcinoma, breast invasive carcinoma, and others.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [TAF1](https://www.ncbi.nlm.nih.gov/gene/6872) with 6 experiments, [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664) with 5 experiments, [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613) with 5 experiments, [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930) with 5 experiments, and [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.


The transcription factor [RPS14](https://www.ncbi.nlm.nih.gov/gene/5929) was found to be regulating genes in 2 Perturb-seq experiments.




The gene is expressed in multiple tissue samples, including muscle and liver, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in liver, bronchial epithelial cells, kidney, fetal liver, and skeletal muscle.




The protein sequence analyzed has a GRAVY value of 0.159 (89.52nd percentile), indicating a relatively hydrophobic nature. It exhibits a charge of 4.3 at pH 7.0 (65.67th percentile) and a median structural flexibility of 0.991 (12.71st percentile). The protein's secondary structure is predicted to be 24.6% helix (4.35th percentile), 8.2% sheet (0.28th percentile), and 32.8% turn (80.86th percentile). The instability index is 76.2 (97.31st percentile), suggesting high instability. The isoelectric point is 8.38 (64.25th percentile), with a molecular weight of 6068.28 Da (0.14th percentile). The sequence length is 61 amino acids (0.29th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |