---
layout: post
title: EXOSC8
date: 2025-01-17 16:55 CST
description: EXOSC8 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/11340) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 11340  | EXOSC8 | ENSG00000120699 | 13q13.3 |



The gene is part of the [nuclear exosome (RNase complex)](https://amigo.geneontology.org/amigo/term/GO:0000176), [cytoplasmic exosome (RNase complex)](https://amigo.geneontology.org/amigo/term/GO:0000177), and [exosome (RNase complex)](https://amigo.geneontology.org/amigo/term/GO:0000178). It is located in the [fibrillar center](https://amigo.geneontology.org/amigo/term/GO:0001650), [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [chromosome](https://amigo.geneontology.org/amigo/term/GO:0005694), [nucleolus](https://amigo.geneontology.org/amigo/term/GO:0005730), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [mRNA 3'-UTR AU-rich region binding](https://amigo.geneontology.org/amigo/term/GO:0035925), and [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802). It is involved in various processes including [exonucleolytic trimming to generate mature 3'-end of 5.8S rRNA](https://amigo.geneontology.org/amigo/term/GO:0000467), [RNA processing](https://amigo.geneontology.org/amigo/term/GO:0006396), [RNA catabolic process](https://amigo.geneontology.org/amigo/term/GO:0006401), [nuclear-transcribed mRNA catabolic process, exonucleolytic, 3'-5'](https://amigo.geneontology.org/amigo/term/GO:0034427), [U1 snRNA 3'-end processing](https://amigo.geneontology.org/amigo/term/GO:0034473), [U4 snRNA 3'-end processing](https://amigo.geneontology.org/amigo/term/GO:0034475), [U5 snRNA 3'-end processing](https://amigo.geneontology.org/amigo/term/GO:0034476), [exonucleolytic catabolism of deadenylated mRNA](https://amigo.geneontology.org/amigo/term/GO:0043928), [nuclear mRNA surveillance](https://amigo.geneontology.org/amigo/term/GO:0071028), [nuclear polyadenylation-dependent rRNA catabolic process](https://amigo.geneontology.org/amigo/term/GO:0071035), [nuclear polyadenylation-dependent tRNA catabolic process](https://amigo.geneontology.org/amigo/term/GO:0071038), and [nuclear polyadenylation-dependent mRNA catabolic process](https://amigo.geneontology.org/amigo/term/GO:0071042). Additionally, it is part of the [nucleolar exosome (RNase complex)](https://amigo.geneontology.org/amigo/term/GO:0101019).


The gene length is 89,395 base pairs (79.68% of all genes), the mature length is 2,052 base pairs, and the primary transcript length is 83,731.5 base pairs.


The gene EXOSC8 (NCBI ID: 11340) has been mentioned in [3 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22EXOSC8%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1999 and the middle 50% of publications occurring between 2000 and 2008. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top publications mentioning EXOSC8 include "[A protein subunit of human RNase P, Rpp14, and its interacting partner, OIP2, have 3'--&gt;5' exoribonuclease activity.](https://pubmed.ncbi.nlm.nih.gov/11929972)" (2002) (relative citation ratio: 0.28), "[Predominance of genotype P[9]G3 in rotavirus gastroenteritis in Polish children.](https://pubmed.ncbi.nlm.nih.gov/26170851)" (2015) (relative citation ratio: 0.25), and "[Isolation of bovine serum albumin fragment P-9 and P-9-mediated fusion of small unilamellar vesicles.](https://pubmed.ncbi.nlm.nih.gov/10746170)" (1999) (relative citation ratio: 0.22). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[EXOSC8](https://www.proteinatlas.org/ENSG00000120699-EXOSC8) is an RNA-binding protein with evidence at the protein level. It is involved in neurodegeneration and is detected in all tissues. The protein is localized in the nucleoplasm, nucleoli fibrillar center, mitotic chromosome, vesicles, and cytosol. EXOSC8 is expressed in various clusters, including Cluster 3 for nuclear processes in blood, Cluster 4 for basic cellular processes in testis, Cluster 28 for mixed function in oligodendrocytes in the brain, Cluster 19 for unknown function in cell lines, and Cluster 68 for muscle contraction in cardiomyocytes in single cells.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [NEUROD1](https://www.ncbi.nlm.nih.gov/gene/4760), [TBP](https://www.ncbi.nlm.nih.gov/gene/6908), and [PPARG](https://www.ncbi.nlm.nih.gov/gene/5468), each with 5 experiments. Additionally, [SSRP1](https://www.ncbi.nlm.nih.gov/gene/6749) and [TAF1](https://www.ncbi.nlm.nih.gov/gene/6872) were found to be regulating in 4 experiments each.





The gene is expressed in various immune cell types, including B lymphoblasts, CD34+ cells, CD4+ T cells, and CD19+ B cells, as well as in endothelial cells. Additionally, it is expressed in specific cancer cell lines such as promyelocytic leukemia (HL-60) and Burkitt's lymphoma (Daudi). Notably, the gene is also expressed in the ovary.


The input data highlights several key pathways involved in protein metabolism and gene expression. These include the regulation of mRNA stability by proteins that bind AU-rich elements, ATF4 activation of genes, and PERK regulation of gene expression. Additionally, the data mentions specific mechanisms such as BRF1 and TTP destabilizing mRNA, mRNA decay by 3' to 5' exoribonuclease, KSRP destabilizing mRNA, deadenylation-dependent mRNA decay, and the unfolded protein response (UPR).


The mouse ortholog gene 69639 and its human ortholog 11340 are associated with several phenotypic outcomes. Notably, the mouse ortholog is linked to embryonic lethality prior to organogenesis and embryonic lethality prior to the tooth bud stage, both specific to one sex. Additionally, it is associated with preweaning lethality with complete penetrance, which affects both sexes.


The analyzed protein sequence has a GRAVY value of -0.079 (79.55th percentile), indicating a hydrophilic nature. The charge at pH 7 is -8.20 (25.96th percentile), and the median structural flexibility is 1.000 (41.87th percentile). The protein has a propensity for helix and sheet structures, with affinities of 0.351 (75.64th percentile) and 0.370 (72.7th percentile), respectively, and a turn affinity of 0.275 (43.09th percentile). The instability index is 40.75 (24.35th percentile), with an isoelectric point of 5.17 (11.01th percentile). The protein is 276 amino acids long (25.24th percentile) and has a molecular weight of 30039.25 Da (24.38th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |