---
layout: post
title: KPNA4
date: 2025-01-17 16:55 CST
description: KPNA4 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3840) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3840  | KPNA4 | ENSG00000186432 | 3q25.33 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [nuclear localization sequence binding](https://amigo.geneontology.org/amigo/term/GO:0008139), and is involved in [protein import into nucleus](https://amigo.geneontology.org/amigo/term/GO:0006606) and [NLS-bearing protein import into nucleus](https://amigo.geneontology.org/amigo/term/GO:0006607). It is active in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), and is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and [nuclear membrane](https://amigo.geneontology.org/amigo/term/GO:0031965). Additionally, it is part of the [nuclear pore](https://amigo.geneontology.org/amigo/term/GO:0005643) and the [NLS-dependent protein nuclear import complex](https://amigo.geneontology.org/amigo/term/GO:0042564). The gene acts upstream of positive effect on [gene expression](https://amigo.geneontology.org/amigo/term/GO:0010467) and [dopamine secretion](https://amigo.geneontology.org/amigo/term/GO:0014046), and enables [nuclear import signal receptor activity](https://amigo.geneontology.org/amigo/term/GO:0061608).


The gene length is 9,132.0 base pairs (23.82% of all genes), the mature length is 4,609.5 base pairs, and the primary transcript length is 8,882.5 base pairs.


KPNA4 (Gene ID: 3840) has been mentioned in [18 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22KPNA4%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest mention in 2000. The middle 50% of publications occurred between 2017 and 2021. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning KPNA4 include "[Circ_ZFR contributes to the paclitaxel resistance and progression of non-small cell lung cancer by upregulating KPNA4 through sponging miR-195-5p.](https://pubmed.ncbi.nlm.nih.gov/33407505)" (2021) (relative citation ratio: 3.56), "[STAT3-induced upregulation of circCCDC66 facilitates the progression of non-small cell lung cancer by targeting miR-33a-5p/KPNA4 axis.](https://pubmed.ncbi.nlm.nih.gov/32151944)" (2020) (relative citation ratio: 2.22), "[Upregulation of miR-181s reverses mesenchymal transition by targeting KPNA4 in glioblastoma.](https://pubmed.ncbi.nlm.nih.gov/26283154)" (2015) (relative citation ratio: 2.18), "[MiR-3619-5p hampers proliferation and cisplatin resistance in cutaneous squamous-cell carcinoma via KPNA4.](https://pubmed.ncbi.nlm.nih.gov/30967266)" (2019) (relative citation ratio: 2.03), and "[Berberine Attenuates MPP<sup>+</sup>-Induced Neuronal Injury by Regulating LINC00943/miR-142-5p/KPNA4/NF-ÎºB Pathway in SK-N-SH Cells.](https://pubmed.ncbi.nlm.nih.gov/34427876)" (2021) (relative citation ratio: 1.86). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[KPNA4](https://www.proteinatlas.org/ENSG00000186432-KPNA4) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in the nucleoplasm and nuclear membrane. The gene is expressed in various clusters, including Cluster 35 in neutrophils related to protein binding, Cluster 2 in skeletal muscle associated with striated muscle contraction, Cluster 55 in non-specific brain tissue related to transcription, Cluster 63 in non-specific cell lines involved in protein processing, and Cluster 29 in monocytes related to the innate immune response.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), [HDAC1](https://www.ncbi.nlm.nih.gov/gene/3065), [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169), [PGR](https://www.ncbi.nlm.nih.gov/gene/5241), and [JUND](https://www.ncbi.nlm.nih.gov/gene/3727). Each of these TFs, except for JUND, was shown to be regulating in 5 experiments, while JUND was found in 4 experiments.



The GWAS data indicates associations with various conditions, including cognitive disorder, disease of mental health, anxiety disorder, and panic disorder.



The gene is expressed in various tissues, including heart and muscle. Additionally, it is expressed in several immune cell types such as B lymphoblasts, dendritic cells, NK cells, and B cells, as well as in whole blood. The gene is also found in colorectal adenocarcinoma and tongue tissue.


The proteins are involved in several key pathways related to immune response, including the antiviral mechanism by IFN-stimulated genes, ISG15 antiviral mechanism, cytokine signaling in the immune system, interferon signaling, and the broader immune system.


The mouse ortholog gene 16649 and its human ortholog gene 3840 are associated with abnormal vocalization, a phenotype observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.164 (72.74th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -28.26 (6.63rd percentile). The median structural flexibility is 0.9998 (40.45th percentile), with a propensity for helix, sheet, and turn structures of 29.56% (28.03rd percentile), 36.85% (72.08th percentile), and 29.37% (59.0th percentile), respectively. The instability index is 40.001 (22.21st percentile), and the isoelectric point is 4.798 (4.49th percentile). The protein has a length of 521 amino acids (60.82nd percentile) and a molecular weight of 57886.06 Da (60.59th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |