---
layout: post
title: HMBS
date: 2025-01-17 16:55 CST
description: HMBS description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3145) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3145  | HMBS | ENSG00000256269 | 11q23.3 |



The gene enables [hydroxymethylbilane synthase activity](https://amigo.geneontology.org/amigo/term/GO:0004418) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737) and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). The gene is involved in the [protoporphyrinogen IX biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006782), [heme biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006783), [heme A biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006784), [heme B biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006785), and [heme O biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0048034).


The gene length is 79,956.0 base pairs (77.39% of all genes), the mature length is 1,290.5 base pairs, and the primary transcript length is 61,464.0 base pairs.


The gene HMBS (NCBI ID: 3145) has been mentioned in [267 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22HMBS%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1961, and the middle 50% of publications occurred between 1987 and 2006.


The top five publications mentioning HMBS, ranked by their scientific influence, include "[Decreased red cell uroporphyrinogen I synthetase activity in intermittent acute porphyria.](https://pubmed.ncbi.nlm.nih.gov/5056653)" (1972) (relative citation ratio: 7.29), "[Acute Intermittent Porphyria: Predicted Pathogenicity of HMBS Variants Indicates Extremely Low Penetrance of the Autosomal Dominant Disease.](https://pubmed.ncbi.nlm.nih.gov/27539938)" (2016) (relative citation ratio: 5.64), "[Tissue-specific expression of porphobilinogen deaminase. Two isoenzymes from a single gene.](https://pubmed.ncbi.nlm.nih.gov/3816774)" (1987) (relative citation ratio: 5.24), "[Two tissue-specific factors bind the erythroid promoter of the human porphobilinogen deaminase gene.](https://pubmed.ncbi.nlm.nih.gov/2911469)" (1989) (relative citation ratio: 5.19), and "[Cis- and trans-acting elements involved in the regulation of the erythroid promoter of the human porphobilinogen deaminase gene.](https://pubmed.ncbi.nlm.nih.gov/2771941)" (1989) (relative citation ratio: 4.87). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[HMBS](https://www.proteinatlas.org/ENSG00000256269-HMBS) is a transferase with evidence at the protein level, implicated in disease variants. It is detected in all RNA tissues and localizes to lipid droplets. The gene is expressed in various clusters, including Cluster 13 for non-specific mitochondrial function in blood, Cluster 6 for bone marrow and innate immune response in tissues, Cluster 27 for mixed function in the brain, Cluster 47 for leukemia and hemostasis in cell lines, and Cluster 1 for erythroid cells and oxygen transport in single cells.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169), [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885), and [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476). Each of these TFs, except BRD4, was shown to be regulating in 5 experiments, while BRD4 was involved in 4 experiments.


Based on the data from GTRD's database of transcription factor (TF) binding sites and filtered for Perturb-seq data, the top regulating TFs identified include [863](https://www.ncbi.nlm.nih.gov/gene/863), which was shown to regulate genes in 1 experiment, [4149](https://www.ncbi.nlm.nih.gov/gene/4149) in 3 experiments, and [57634](https://www.ncbi.nlm.nih.gov/gene/57634) in 2 experiments.



The HMBS gene has been associated with several traits and diseases through genome-wide association studies. Notably, it has been linked to [Albumin levels](https://pubmed.ncbi.nlm.nih.gov/34321204), with the closest MeSH match being [Albumins](https://meshb.nlm.nih.gov/record/ui?ui=D000418) identified using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method. Additionally, HMBS has been associated with [Waist-hip ratio](https://pubmed.ncbi.nlm.nih.gov/30239722), which directly matches the MeSH term [Waist-Hip Ratio](https://meshb.nlm.nih.gov/record/ui?ui=D049629).


The gene is expressed in various tissues, including the adrenal gland, blood, and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in chronic myelogenous leukemia (K-562), promyelocytic leukemia (HL-60), CD105+ endothelial cells, CD71+ early erythroid cells, bone marrow, fetal liver, and fetal lung.


The proteins are involved in several key pathways, including heme biosynthesis, metabolism of porphyrins, and general metabolism.


The mouse ortholog gene 15288 and its human ortholog gene 3145 are associated with phenotypes including embryonic lethality prior to organogenesis, which is not observed in both sexes, and preweaning lethality with complete penetrance, which is observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.205 (68.67th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -1.342 (45.25th percentile), and the median structural flexibility is 0.999 (35.19th percentile). The protein has a notable affinity for helix (0.346, 72.37th percentile) and sheet (0.343, 54.48th percentile) structures, with a lower affinity for turns (0.271, 39.31st percentile). The instability index is 41.723 (27.75th percentile), and the isoelectric point is 6.678 (45.87th percentile). The protein sequence is 361 amino acids long (39.87th percentile) with a molecular weight of 39329.6 Da (38.52nd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |