---
layout: post
title: PSMD5
date: 2025-01-17 16:55 CST
description: PSMD5 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5711) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5711  | PSMD5 | ENSG00000095261 | 9q33.2  |



The gene is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654) and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and is part of the [proteasome complex](https://amigo.geneontology.org/amigo/term/GO:0000502), specifically the [proteasome regulatory particle, base subcomplex](https://amigo.geneontology.org/amigo/term/GO:0008540) and the [proteasome accessory complex](https://amigo.geneontology.org/amigo/term/GO:0022624). It enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is involved in [proteasome regulatory particle assembly](https://amigo.geneontology.org/amigo/term/GO:0070682).


The gene length is 153,162.0 base pairs (89.21% of all genes), the mature length is 5,430.5 base pairs, and the primary transcript length is 153,029.5 base pairs.


PSMD5 has been mentioned in [3 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PSMD5%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 2000, and the middle 50% of publications range from 2004 to 2013.


The top publications mentioning PSMD5 include "[PSMD5 Inactivation Promotes 26S Proteasome Assembly during Colorectal Tumor Progression.](https://pubmed.ncbi.nlm.nih.gov/29716915)" (2018) (relative citation ratio: 1.08), "[Mapping subunit contacts in the regulatory complex of the 26 S proteasome. S2 and S5b form a tetramer with ATPase subunits S4 and S7.](https://pubmed.ncbi.nlm.nih.gov/10625621)" (2000) (relative citation ratio: 0.73), and "[Hsm3/S5b joins the ranks of 26S proteasome assembly chaperones.](https://pubmed.ncbi.nlm.nih.gov/19250902)" (2009) (relative citation ratio: 0.12). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PSMD5](https://www.proteinatlas.org/ENSG00000095261-PSMD5) is a gene with a molecular function as a chaperone. Evidence for its function is available at the protein level. The gene is detected in all RNA tissue distributions and is localized in various subcellular locations including the nucleoplasm, centrosome, basal body, cytosol, connecting piece, and mid piece. PSMD5 is part of several non-specific expression clusters related to cell proliferation and transcription across different tissue types, including blood, brain, and cell lines.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD9](https://www.ncbi.nlm.nih.gov/gene/65980) and [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599), each with 6 experiments. Additionally, [PGR](https://www.ncbi.nlm.nih.gov/gene/5241), [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613), and [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) were each found to be regulating in 5 experiments.





The gene is expressed in various tissues, including skin and adrenal gland, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in cardiac myocytes, chronic myelogenous leukemia K-562 cells, CD105+ endothelial cells, smooth muscle, adrenal gland, skeletal muscle, ciliary ganglion, and dorsal root ganglion.


The input data highlights various biological processes and pathways, including metabolism, gene expression, and HIV infection. Several mutations and their effects are noted, such as AMER1 and APC truncation mutants destabilizing the destruction complex, AXIN mutants enhancing WNT signaling, and misspliced GSK3beta mutants stabilizing beta-catenin. Additionally, the data covers the regulation of the cell cycle, including APC/C-mediated degradation of cell cycle proteins and the regulation of mitotic cell cycle phases. Other processes mentioned include the degradation of various proteins, regulation of apoptosis, DNA damage response, and signaling pathways like WNT and Hedgehog. The data also touches on immune system processes, including antigen processing and presentation, as well as host interactions of HIV factors.



The protein sequence analyzed has a GRAVY value of 0.0036 (84.14th percentile), indicating a slightly hydrophilic nature. It exhibits a charge of -12.68 at pH 7.0 (18.4th percentile) and a median structural flexibility of 0.995 (21.16th percentile). The protein's secondary structure is predicted to be 35.52% helix (78.01st percentile), 39.29% sheet (83.99th percentile), and 25.20% turn (22.97th percentile). The instability index is 38.13 (17.16th percentile), with an isoelectric point of 5.35 (14.76th percentile). The protein has a length of 504 amino acids (58.74th percentile) and a molecular weight of 56195.04 Da (58.74th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |