---
layout: post
title: MADD
date: 2025-01-17 16:55 CST
description: MADD description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8567) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8567  | MADD | ENSG00000110514 | 11p11.2 |



The gene enables [guanyl-nucleotide exchange factor activity](https://amigo.geneontology.org/amigo/term/GO:0005085) and [death receptor binding](https://amigo.geneontology.org/amigo/term/GO:0005123), and is involved in [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737) and is active in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). Additionally, it is located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886) and the [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), as well as the [axon](https://amigo.geneontology.org/amigo/term/GO:0030424). The gene is involved in various processes, including the [cell surface receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007166), [regulation of Rab protein signal transduction](https://amigo.geneontology.org/amigo/term/GO:0032483), [regulation of apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0042981), [positive regulation of MAPK cascade](https://amigo.geneontology.org/amigo/term/GO:0043410), [regulation of cell cycle](https://amigo.geneontology.org/amigo/term/GO:0051726), [execution phase of apoptosis](https://amigo.geneontology.org/amigo/term/GO:0097194), [regulation of extrinsic apoptotic signaling pathway via death domain receptors](https://amigo.geneontology.org/amigo/term/GO:1902041), and [regulation of extrinsic apoptotic signaling pathway](https://amigo.geneontology.org/amigo/term/GO:2001236). Furthermore, it enables [protein kinase activator activity](https://amigo.geneontology.org/amigo/term/GO:0030295).


The gene length is 84,174 base pairs (78.45th percentile), the mature length is 6,842 base pairs (96.14th percentile), and the primary transcript length is 43,924 base pairs (72.23rd percentile).


The gene MADD (NCBI ID: 8567) has been mentioned in [57 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22MADD%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1988, and the middle 50% of publications occurred between 2008 and 2019.


The top 5 publications mentioning MADD, ranked by their scientific influence, include "[The product of C9orf72, a gene strongly implicated in neurodegeneration, is structurally related to DENN Rab-GEFs.](https://pubmed.ncbi.nlm.nih.gov/23329412)" (2013) (relative citation ratio: 7.35), "[Family-wide characterization of the DENN domain Rab GDP-GTP exchange factors.](https://pubmed.ncbi.nlm.nih.gov/20937701)" (2010) (relative citation ratio: 5.02), "[Discovery of Novel DENN Proteins: Implications for the Evolution of Eukaryotic Intracellular Membrane Structures and Human Disease.](https://pubmed.ncbi.nlm.nih.gov/23248642)" (2012) (relative citation ratio: 4.91), "[DENN domain proteins: regulators of Rab GTPases.](https://pubmed.ncbi.nlm.nih.gov/21330364)" (2011) (relative citation ratio: 4.24), and "[Clinical, pathological and genetic features and follow-up of 110 patients with late-onset MADD: a single-center retrospective study.](https://pubmed.ncbi.nlm.nih.gov/34718578)" (2022) (relative citation ratio: 3.83). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[MADD](https://www.proteinatlas.org/ENSG00000110514-MADD) is a guanine-nucleotide releasing factor implicated in intellectual disability, with evidence supporting its function at the protein level. The gene is detected in all RNA tissue distributions and is primarily localized in the plasma membrane and cytosol. It is notably expressed in Cluster 32 of basophils and Cluster 28 of neuronal tissues, with non-specific expression patterns observed in the brain and cell lines. Additionally, MADD shows specific expression in photoreceptor cells related to visual perception.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [PGR](https://www.ncbi.nlm.nih.gov/gene/5241), [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169), [TCF12](https://www.ncbi.nlm.nih.gov/gene/6938), and [CEBPA](https://www.ncbi.nlm.nih.gov/gene/1050), each with 6 experiments, followed by [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113) with 5 experiments.



The input data from GWAS indicates associations with a wide range of disease conditions, including various types of cancer such as leukemia, lymphoma, and immune system cancer. Additionally, respiratory diseases like asthma, bronchial disease, and obstructive lung disease are highlighted. Metabolic disorders, particularly diabetes mellitus and type 2 diabetes mellitus, are also noted. Cardiovascular conditions, including coronary artery disease and vascular disease, are present in the data. Furthermore, the data includes neurodegenerative diseases such as Alzheimer's disease, tauopathy, and dementia, as well as cognitive disorders and diseases of mental health.


Genome-wide association studies have identified several traits and diseases associated with specific genetic variations. Notably, [Blood glucose levels](https://pubmed.ncbi.nlm.nih.gov/33462484) have been mapped to the closest MeSH term [Glucose](https://meshb.nlm.nih.gov/record/ui?ui=D005947) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method. Additionally, [Chronotype](https://pubmed.ncbi.nlm.nih.gov/30696823) has been directly matched to the MeSH term [Chronotype](https://meshb.nlm.nih.gov/record/ui?ui=D000094407). Furthermore, [Fasting blood proinsulin levels](https://pubmed.ncbi.nlm.nih.gov/23263489) have been associated with the MeSH term [Fasting](https://meshb.nlm.nih.gov/record/ui?ui=D005215) through embedding matching, and [Glycated hemoglobin levels](https://pubmed.ncbi.nlm.nih.gov/33462484) have been linked to the MeSH term [Hemoglobins](https://meshb.nlm.nih.gov/record/ui?ui=D006454) using the same method.


The gene is expressed in various brain regions, including the prefrontal cortex, whole brain, and cerebellum peduncles, as well as in the pituitary gland. Additionally, it is expressed in specific immune cell types such as CD56+ NK cells and CD4+ T cells. Notably, the gene also shows expression in the pineal gland during both day and night cycles.




The analyzed protein sequence has a GRAVY value of -0.4493 (40.49th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -27.45 (6.98th percentile), and the median structural flexibility is 1.0064 (69.39th percentile). The protein has a length of 1647 amino acids (95.95th percentile) and a molecular weight of 183301.11 Da (96.01st percentile). The secondary structure affinities are 30.78% for helix (38.84th percentile), 32.79% for sheet (44.07th percentile), and 32.85% for turn (81.18th percentile). The instability index is 55.86 (74.35th percentile), and the isoelectric point is 5.72 (23.45th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |