---
layout: post
title: PITPNM1
date: 2025-01-17 16:55 CST
description: PITPNM1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9600) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9600  | PITPNM1 | ENSG00000110697 | 11q13.2 |



The gene enables [calcium ion binding](https://amigo.geneontology.org/amigo/term/GO:0005509) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737). It is located in the [endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0005789), [lipid droplet](https://amigo.geneontology.org/amigo/term/GO:0005811), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [midbody](https://amigo.geneontology.org/amigo/term/GO:0030496), [cleavage furrow](https://amigo.geneontology.org/amigo/term/GO:0032154), [Golgi cisterna membrane](https://amigo.geneontology.org/amigo/term/GO:0032580), [intracellular membrane-bounded organelle](https://amigo.geneontology.org/amigo/term/GO:0043231), and [cell body](https://amigo.geneontology.org/amigo/term/GO:0044297). The gene is involved in various processes including [lipid metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006629), [phosphatidylinositol biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006661), [brain development](https://amigo.geneontology.org/amigo/term/GO:0007420), [phototransduction](https://amigo.geneontology.org/amigo/term/GO:0007602), [protein transport](https://amigo.geneontology.org/amigo/term/GO:0015031), [phospholipid transport](https://amigo.geneontology.org/amigo/term/GO:0015914), and [intermembrane lipid transfer](https://amigo.geneontology.org/amigo/term/GO:0120009). Additionally, it enables [phosphatidylcholine transporter activity](https://amigo.geneontology.org/amigo/term/GO:0008525), [phosphatidylinositol transfer activity](https://amigo.geneontology.org/amigo/term/GO:0008526), [receptor tyrosine kinase binding](https://amigo.geneontology.org/amigo/term/GO:0030971), [phosphatidylcholine binding](https://amigo.geneontology.org/amigo/term/GO:0031210), [phosphatidylinositol binding](https://amigo.geneontology.org/amigo/term/GO:0035091), and [phosphatidic acid binding](https://amigo.geneontology.org/amigo/term/GO:0070300).


The gene length is 103,272 base pairs (82.41% of all genes), the mature length is 2,059 base pairs, and the primary transcript length is 86,989 base pairs.


The gene PITPNM1 (NCBI ID: 9600) has been mentioned in [26 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PITPNM1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1977 and the middle 50% of publications occurring between 2002 and 2015. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning PITPNM1, ranked by their scientific influence, include "[Phosphatidylinositol-Phosphatidic Acid Exchange by Nir2 at ER-PM Contact Sites Maintains Phosphoinositide Signaling Competence.](https://pubmed.ncbi.nlm.nih.gov/26028218)" (2015) (relative citation ratio: 5.08), "[Phosphatidylinositol 4,5-Bisphosphate Homeostasis Regulated by Nir2 and Nir3 Proteins at Endoplasmic Reticulum-Plasma Membrane Junctions.](https://pubmed.ncbi.nlm.nih.gov/25887399)" (2015) (relative citation ratio: 2.89), "[The phosphatidylinositol-transfer protein Nir2 binds phosphatidic acid and positively regulates phosphoinositide signalling.](https://pubmed.ncbi.nlm.nih.gov/23897088)" (2013) (relative citation ratio: 2.51), "[Maintenance of the diacylglycerol level in the Golgi apparatus by the Nir2 protein is critical for Golgi secretory function.](https://pubmed.ncbi.nlm.nih.gov/15723057)" (2005) (relative citation ratio: 2.42), and "[Identification of a novel family of targets of PYK2 related to Drosophila retinal degeneration B (rdgB) protein.](https://pubmed.ncbi.nlm.nih.gov/10022914)" (1999) (relative citation ratio: 2.09). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PITPNM1](https://www.proteinatlas.org/ENSG00000110697-PITPNM1) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in vesicles and the cytosol. The gene is expressed in monocytes and neutrophils, playing a role in the innate immune response, and is also associated with neuronal and synaptic functions. Additionally, it is found in sub-cortical regions of the brain and in enterocytes, where it is involved in absorption.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 6 experiments, [MYC](https://www.ncbi.nlm.nih.gov/gene/4609) with 5 experiments, [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) with 5 experiments, [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113) with 5 experiments, and [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.




The gene PITPNM1 has been associated with [Leukocyte telomere length](https://pubmed.ncbi.nlm.nih.gov/37216007), which maps closely to the MeSH term [Telomere](https://meshb.nlm.nih.gov/record/ui?ui=D016615) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method.


The gene is expressed in the pituitary and spleen according to BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles indicate expression in the Caudate nucleus, Thalamus, Cingulate Cortex, Subthalamic Nucleus, Temporal Lobe, Cerebellum Peduncles, and Skeletal Muscle.



The mouse ortholog gene 18739 and its human ortholog gene 9600 are associated with the phenotype of decreased circulating calcium level, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.315 (56.39th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -25.76 (7.73rd percentile), and the median structural flexibility is 1.002 (51.58th percentile). The protein has a propensity for helix, sheet, and turn structures of 30.71% (38.12th percentile), 31.11% (34.53rd percentile), and 32.15% (77.58th percentile), respectively. The instability index is 52.06 (63.8th percentile), with an isoelectric point of 5.64 (21.47th percentile). The sequence length is 1244 amino acids (92.45th percentile), and the molecular weight is 134846.04 Da (92.02nd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |