---
layout: post
title: TIMP1
date: 2025-01-17 16:55 CST
description: TIMP1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7076) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7076  | TIMP1 | ENSG00000102265 | Xp11.3  |



The gene enables [protease binding](https://amigo.geneontology.org/amigo/term/GO:0002020) and is involved in [connective tissue replacement involved in inflammatory response wound healing](https://amigo.geneontology.org/amigo/term/GO:0002248). It enables [cytokine activity](https://amigo.geneontology.org/amigo/term/GO:0005125) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576) and [basement membrane](https://amigo.geneontology.org/amigo/term/GO:0005604). It is active in the [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615) and located in the [endoplasmic reticulum lumen](https://amigo.geneontology.org/amigo/term/GO:0005788). The gene is involved in [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165) and enables [growth factor activity](https://amigo.geneontology.org/amigo/term/GO:0008083) and [metalloendopeptidase inhibitor activity](https://amigo.geneontology.org/amigo/term/GO:0008191). It also enables [zinc ion binding](https://amigo.geneontology.org/amigo/term/GO:0008270) and is involved in the [positive regulation of cell population proliferation](https://amigo.geneontology.org/amigo/term/GO:0008284) and [response to hormone](https://amigo.geneontology.org/amigo/term/GO:0009725). Additionally, it is involved in the [negative regulation of endopeptidase activity](https://amigo.geneontology.org/amigo/term/GO:0010951) and enables [peptidase inhibitor activity](https://amigo.geneontology.org/amigo/term/GO:0030414). The gene is active in the [extracellular matrix](https://amigo.geneontology.org/amigo/term/GO:0031012) and located in the [platelet alpha granule lumen](https://amigo.geneontology.org/amigo/term/GO:0031093). It is involved in the [response to cytokine](https://amigo.geneontology.org/amigo/term/GO:0034097), [negative regulation of apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0043066), and [negative regulation of catalytic activity](https://amigo.geneontology.org/amigo/term/GO:0043086). Furthermore, it is involved in the [response to peptide hormone](https://amigo.geneontology.org/amigo/term/GO:0043434), [negative regulation of membrane protein ectodomain proteolysis](https://amigo.geneontology.org/amigo/term/GO:0051045), and [cartilage development](https://amigo.geneontology.org/amigo/term/GO:0051216). The gene is located in the [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062) and is involved in the [cellular response to UV-A](https://amigo.geneontology.org/amigo/term/GO:0071492), [negative regulation of trophoblast cell migration](https://amigo.geneontology.org/amigo/term/GO:1901164), [negative regulation of metallopeptidase activity](https://amigo.geneontology.org/amigo/term/GO:1905049), and [regulation of integrin-mediated signaling pathway](https://amigo.geneontology.org/amigo/term/GO:2001044).


The gene length is 280,910 base pairs (95.37% of all genes), the mature length is 3,955.5 base pairs, and the primary transcript length is 279,324 base pairs (97.56% of all genes).


The gene TIMP1 (NCBI ID: 7076) has been mentioned in [1,459 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22TIMP1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1970, and the middle 50% of publications occurred between 2001 and 2015.


The top 5 publications mentioning TIMP1, ranked by their scientific influence, include "[Tissue inhibitor of metalloproteinase (TIMP-2). A new member of the metalloproteinase inhibitor family.](https://pubmed.ncbi.nlm.nih.gov/2793861)" (1989) (relative citation ratio: 21.49), "[Mechanism of inhibition of the human matrix metalloproteinase stromelysin-1 by TIMP-1.](https://pubmed.ncbi.nlm.nih.gov/9288970)" (1997) (relative citation ratio: 11.29), "[Discoordinate expression of stromelysin, collagenase, and tissue inhibitor of metalloproteinases-1 in rheumatoid human synovial fibroblasts. Synergistic effects of interleukin-1 and tumor necrosis factor-alpha on stromelysin expression.](https://pubmed.ncbi.nlm.nih.gov/1698773)" (1990) (relative citation ratio: 11.04), "[Inhibition by human recombinant tissue inhibitor of metalloproteinases of human amnion invasion and lung colonization by murine B16-F10 melanoma cells.](https://pubmed.ncbi.nlm.nih.gov/3416307)" (1988) (relative citation ratio: 9.31), and "[Human neutrophils uniquely release TIMP-free MMP-9 to provide a potent catalytic stimulator of angiogenesis.](https://pubmed.ncbi.nlm.nih.gov/18077379)" (2007) (relative citation ratio: 8.66). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[TIMP1](https://www.proteinatlas.org/ENSG00000102265-TIMP1) is a gene with multiple molecular functions, including growth factor, metalloenzyme inhibitor, metalloprotease inhibitor, and protease inhibitor. It is involved in cancer-related genes and has evidence at the protein level. TIMP1 is detected in all tissues and is localized in the Golgi apparatus and vesicles. It is expressed in monocytes and neutrophils, playing a role in the innate immune response, and is also involved in the extracellular matrix (ECM) organization in connective tissue. Additionally, TIMP1 is expressed in the vasculature of the brain and in connective tissue cells.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 5 experiments, [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599) with 4 experiments, [TP53](https://www.ncbi.nlm.nih.gov/gene/7157) with 4 experiments, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 4 experiments, and [HDAC2](https://www.ncbi.nlm.nih.gov/gene/3066) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent detection in multiple experiments.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [TBP](https://www.ncbi.nlm.nih.gov/gene/2118) and [BRCA1](https://www.ncbi.nlm.nih.gov/gene/6830), each shown to be regulating genes in one experiment. Additionally, [KLF12](https://www.ncbi.nlm.nih.gov/gene/8861) was found to regulate genes in two experiments, while [TCF7L2](https://www.ncbi.nlm.nih.gov/gene/9968) was the most prominent, regulating genes in three experiments.




The gene is expressed in various tissues, including blood vessel, prostate, and lung, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in cardiac myocytes, adipocytes, smooth muscle, uterus corpus, and ovary.


The input data highlights several key biological processes, including the activation of matrix metalloproteinases, degradation of the extracellular matrix, and extracellular matrix organization. Additionally, it covers hemostasis, platelet activation, signaling and aggregation, platelet degranulation, and the response to elevated platelet cytosolic Ca2+.


The mouse ortholog gene 21857, which corresponds to the human ortholog gene 7076, is associated with abnormal femur morphology and abnormal humerus morphology. These phenotypes are not observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.0198 (83.05th percentile), indicating a hydrophilic nature. It exhibits a charge of 3.92 at pH 7.0 (64.43rd percentile) and a median structural flexibility of 0.99 (11.27th percentile). The protein's secondary structure is predicted to be 29.47% helix (27.26th percentile), 38.16% sheet (79.4th percentile), and 24.15% turn (15.97th percentile). The instability index is 70.54 (94.93rd percentile), suggesting high instability. The isoelectric point is 8.46 (65.76th percentile), with a molecular weight of 23170.63 Da (15.54th percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |