---
layout: post
title: ICAM1
date: 2025-01-17 16:55 CST
description: ICAM1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3383) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3383  | ICAM1 | ENSG00000090339 | 19p13.2 |



The gene enables [virus receptor activity](https://amigo.geneontology.org/amigo/term/GO:0001618) and [transmembrane signaling receptor activity](https://amigo.geneontology.org/amigo/term/GO:0004888), and is involved in various processes such as [regulation of leukocyte mediated cytotoxicity](https://amigo.geneontology.org/amigo/term/GO:0001910), [T cell activation via T cell receptor contact with antigen bound to MHC molecule on antigen presenting cell](https://amigo.geneontology.org/amigo/term/GO:0002291), and [T cell antigen processing and presentation](https://amigo.geneontology.org/amigo/term/GO:0002457). It is located in the [immunological synapse](https://amigo.geneontology.org/amigo/term/GO:0001772), [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925), [external side of plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009897), [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [membrane raft](https://amigo.geneontology.org/amigo/term/GO:0045121), [collagen-containing extracellular matrix](https://amigo.geneontology.org/amigo/term/GO:0062023), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). Additionally, it is involved in processes like [cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0007155), [heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules](https://amigo.geneontology.org/amigo/term/GO:0007157), [leukocyte cell-cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0007159), and [cell adhesion mediated by integrin](https://amigo.geneontology.org/amigo/term/GO:0033627). The gene also plays roles in [positive regulation of cellular extravasation](https://amigo.geneontology.org/amigo/term/GO:0002693), [leukocyte migration](https://amigo.geneontology.org/amigo/term/GO:0050900), [T cell extravasation](https://amigo.geneontology.org/amigo/term/GO:0072683), [establishment of endothelial barrier](https://amigo.geneontology.org/amigo/term/GO:0061028), [membrane to membrane docking](https://amigo.geneontology.org/amigo/term/GO:0022614), [adhesion of symbiont to host](https://amigo.geneontology.org/amigo/term/GO:0044406), [symbiont entry into host cell](https://amigo.geneontology.org/amigo/term/GO:0046718), and [receptor-mediated virion attachment to host cell](https://amigo.geneontology.org/amigo/term/GO:0046813). Furthermore, it is involved in the [positive regulation of ERK1 and ERK2 cascade](https://amigo.geneontology.org/amigo/term/GO:0070374), [cellular response to glucose stimulus](https://amigo.geneontology.org/amigo/term/GO:0071333), [regulation of ruffle assembly](https://amigo.geneontology.org/amigo/term/GO:1900027), [negative regulation of extrinsic apoptotic signaling pathway via death domain receptors](https://amigo.geneontology.org/amigo/term/GO:1902042), [cellular response to amyloid-beta](https://amigo.geneontology.org/amigo/term/GO:1904646), [cellular response to leukemia inhibitory factor](https://amigo.geneontology.org/amigo/term/GO:1990830), and [negative regulation of endothelial cell apoptotic process](https://amigo.geneontology.org/amigo/term/GO:2000352).


The gene has a length of 53,840 base pairs (68.43% of all genes), a mature length of 672 base pairs (8.14% of all genes), and a primary transcript length of 48,850 base pairs (74.68% of all genes).


The gene ICAM1 (NCBI ID: 3383) has been mentioned in [3,467 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ICAM1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1986, and the middle 50% of publications occurred between 1996 and 2011.


The top five publications mentioning ICAM1, ranked by their scientific influence, include [Induction by IL 1 and interferon-gamma: tissue distribution, biochemistry, and function of a natural adherence molecule (ICAM-1).](https://pubmed.ncbi.nlm.nih.gov/3086451) (1986) (relative citation ratio: 64.19), [A human intercellular adhesion molecule (ICAM-1) distinct from LFA-1.](https://pubmed.ncbi.nlm.nih.gov/3525675) (1986) (relative citation ratio: 32.34), [Circulating adhesion molecules VCAM-1, ICAM-1, and E-selectin in carotid atherosclerosis and incident coronary heart disease cases: the Atherosclerosis Risk In Communities (ARIC) study.](https://pubmed.ncbi.nlm.nih.gov/9416885) (1997) (relative citation ratio: 28.98), [Cooperative interactions of LFA-1 and Mac-1 with intercellular adhesion molecule-1 in facilitating adherence and transendothelial migration of human neutrophils in vitro.](https://pubmed.ncbi.nlm.nih.gov/2566624) (1989) (relative citation ratio: 28.22), and [Plasma concentration of soluble intercellular adhesion molecule 1 and risks of future myocardial infarction in apparently healthy men.](https://pubmed.ncbi.nlm.nih.gov/9439492) (1998) (relative citation ratio: 26.69). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ICAM1](https://www.proteinatlas.org/ENSG00000090339-ICAM1) functions as a host cell receptor for virus entry and is a receptor involved in cancer-related processes. It is an FDA-approved drug target with evidence at the protein level. ICAM1 is detected in all tissues and is primarily localized in vesicles, the plasma membrane, and the cytosol. It is highly expressed in monocytes and neutrophils, particularly in the context of degranulation, and is associated with immune response in lymphoid tissue. Additionally, ICAM1 is expressed in the vasculature of the brain and in squamous epithelial cells involved in keratinization. It is also found in adipocytes and endothelial cells with mixed functions.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [HDAC2](https://www.ncbi.nlm.nih.gov/gene/3066) with 6 experiments, [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885) and [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025) each with 5 experiments, and [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113) and [ZBTB7A](https://www.ncbi.nlm.nih.gov/gene/51341) each with 4 experiments.



The GWAS data indicates associations with a range of conditions, including cancer, diseases of cellular proliferation and metabolism, coronary artery disease, artery disease, vascular disease, and cardiovascular system disease. Additionally, the data highlights links to inherited metabolic disorders, lipid storage diseases, lysosomal storage diseases, fatty liver disease, and myocardial infarction.



The gene is expressed in various tissues, including lung and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Leukemia promyelocytic-HL-60 cells, smooth muscle, pancreatic islet, superior cervical ganglion, trigeminal ganglion, ciliary ganglion, and fetal lung.


The proteins are involved in various immune-related pathways, including cytokine signaling in the immune system, interferon signaling, interferon gamma signaling, and the adaptive immune system. Additionally, they participate in immunoregulatory interactions between lymphoid and non-lymphoid cells. Beyond immune functions, these proteins also play roles in extracellular matrix organization and integrin cell surface interactions.



The analyzed protein sequence has a GRAVY value of -0.271 (61.26th percentile), indicating a hydrophilic nature. It exhibits a charge of 3.87 at pH 7.0 (64.25th percentile) and a median structural flexibility of 1.004 (58.51st percentile). The protein's secondary structure is predicted to be 30.1% helix (32.61st percentile), 37.2% sheet (74.37th percentile), and 29.3% turn (58.56th percentile). Additionally, it has an instability index of 36.47 (13.01st percentile), an isoelectric point of 8.31 (63.01st percentile), a length of 532 amino acids (62.24th percentile), and a molecular weight of 57824.55 Da (60.51st percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |