---
layout: post
title: RRBP1
date: 2025-01-17 16:55 CST
description: RRBP1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6238) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6238  | RRBP1 | ENSG00000125844 | 20p12.1 |



The gene is involved in [osteoblast differentiation](https://amigo.geneontology.org/amigo/term/GO:0001649) and [translation](https://amigo.geneontology.org/amigo/term/GO:0006412), and it enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and [signaling receptor activity](https://amigo.geneontology.org/amigo/term/GO:0038023). It is located in the [endoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0005783), [endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0005789), [ribosome](https://amigo.geneontology.org/amigo/term/GO:0005840), and [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), and it is involved in [protein transport](https://amigo.geneontology.org/amigo/term/GO:0015031).


The gene length is 168,572 base pairs (90.52% of all genes), the mature length is 6,431 base pairs, and the primary transcript length is 66,410 base pairs (81.39% of all genes).


The gene RRBP1 (NCBI ID: 6238) has been mentioned in [92 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RRBP1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1968 and the middle 50% of publications occurring between 2003 and 2018. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning RRBP1, ranked by their scientific influence, include "[Data Resource Profile: Hospital Episode Statistics Admitted Patient Care (HES APC).](https://pubmed.ncbi.nlm.nih.gov/28338941)" (2017) (relative citation ratio: 17.49), "[HES and HERP families: multiple effectors of the Notch signaling pathway.](https://pubmed.ncbi.nlm.nih.gov/12548545)" (2003) (relative citation ratio: 16.78), "[The Hes gene family: repressors and oscillators that orchestrate embryogenesis.](https://pubmed.ncbi.nlm.nih.gov/17329370)" (2007) (relative citation ratio: 9.22), "[Hes genes regulate size, shape and histogenesis of the nervous system by control of the timing of neural stem cell differentiation.](https://pubmed.ncbi.nlm.nih.gov/15496443)" (2004) (relative citation ratio: 7.48), and "[Hes binding to STAT3 mediates crosstalk between Notch and JAK-STAT signalling.](https://pubmed.ncbi.nlm.nih.gov/15156153)" (2004) (relative citation ratio: 5.68). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RRBP1](https://www.proteinatlas.org/ENSG00000125844-RRBP1) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in the endoplasmic reticulum. The gene is expressed in various clusters, including Cluster 29 in blood with an unknown function, Cluster 15 in tissue related to pancreas proteolysis, Cluster 30 in brain associated with white matter myelination, Cluster 63 in cell lines involved in protein processing, and Cluster 77 in single cells related to plasma cells and protein processing.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [PGR](https://www.ncbi.nlm.nih.gov/gene/5241), [SP1](https://www.ncbi.nlm.nih.gov/gene/6667), and [MAX](https://www.ncbi.nlm.nih.gov/gene/4149), each with 6 experiments. Additionally, [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) and [EP300](https://www.ncbi.nlm.nih.gov/gene/2033) were found to be regulating in 5 experiments each.



The GWAS data indicates associations with various disease conditions, including metabolic disorders such as diabetes mellitus and type 2 diabetes mellitus, as well as cardiovascular system diseases. Additionally, the data highlights links to kidney diseases, including chronic kidney failure and end-stage renal failure. Furthermore, associations with bone and musculoskeletal system diseases, particularly those related to bone remodeling and resorption, are also noted.


RRBP1 has been associated with several traits and diseases through genome-wide association studies. Notably, it has been linked to [Apolipoprotein A1 levels](https://pubmed.ncbi.nlm.nih.gov/32203549) and [Apolipoprotein B levels](https://pubmed.ncbi.nlm.nih.gov/33462484), both of which map to the MeSH term [Apolipoproteins](https://meshb.nlm.nih.gov/record/ui?ui=D001053) using the BioWordVec method. Additionally, RRBP1 is associated with [Coronary artery disease](https://pubmed.ncbi.nlm.nih.gov/35915156), which directly matches the MeSH term [Coronary Artery Disease](https://meshb.nlm.nih.gov/record/ui?ui=D003324). It is also linked to [HDL cholesterol](https://pubmed.ncbi.nlm.nih.gov/34594039), which maps to the MeSH term [Cholesterol, HDL](https://meshb.nlm.nih.gov/record/ui?ui=D008076) through a synonym direct match. Furthermore, RRBP1 is associated with [HDL cholesterol levels](https://pubmed.ncbi.nlm.nih.gov/32203549) and [High density lipoprotein cholesterol levels](https://pubmed.ncbi.nlm.nih.gov/33462484), which map to the MeSH terms [Cholesterol](https://meshb.nlm.nih.gov/record/ui?ui=D002784) and [Lipoproteins](https://meshb.nlm.nih.gov/record/ui?ui=D008074), respectively, using the BioWordVec method.


The gene is expressed in multiple tissue samples, including the pancreas, liver, and thyroid, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Lymphoma (Burkitt's), thyroid, BDCA4+ dendritic cells, smooth muscle, placenta, and pancreas.




The protein sequence analyzed has a GRAVY value of -1.0695 (3.13rd percentile), indicating a hydrophilic nature. It exhibits a charge of 13.18 at pH 7.0 (86.91st percentile) and a median structural flexibility of 1.0292 (99.23rd percentile). The protein's secondary structure is predicted to be predominantly helical, with 46.95% of amino acids in helix form (99.11th percentile), 20.43% in sheet form (2.39th percentile), and 23.90% in turn form (14.45th percentile). The instability index is 37.12 (14.49th percentile), and the isoelectric point is 8.69 (70.86th percentile). The protein has a length of 1410 amino acids (94.29th percentile) and a molecular weight of 152454.37 Da (94.02nd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |