---
layout: post
title: ITPKA
date: 2025-01-17 16:55 CST
description: ITPKA description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3706) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3706  | ITPKA | ENSG00000137825 | 15q15.1 |



The gene enables [calmodulin-dependent protein kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004683), [calmodulin binding](https://amigo.geneontology.org/amigo/term/GO:0005516), [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524), [inositol-1,4,5-trisphosphate 3-kinase activity](https://amigo.geneontology.org/amigo/term/GO:0008440), and [small GTPase binding](https://amigo.geneontology.org/amigo/term/GO:0031267). It is active in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), and is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0005856), [dendritic spine](https://amigo.geneontology.org/amigo/term/GO:0043197), [postsynaptic actin cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0098871), and [glutamatergic synapse](https://amigo.geneontology.org/amigo/term/GO:0098978). The gene is involved in various processes including [inositol metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006020), [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), [phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0016310), [actin cytoskeleton organization](https://amigo.geneontology.org/amigo/term/GO:0030036), [inositol phosphate biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0032958), [phosphatidylinositol phosphate biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0046854), [regulation of synaptic plasticity](https://amigo.geneontology.org/amigo/term/GO:0048167), [response to calcium ion](https://amigo.geneontology.org/amigo/term/GO:0051592), [positive regulation of dendritic spine morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0061003), [cellular response to calcium ion](https://amigo.geneontology.org/amigo/term/GO:0071277), [dendritic spine maintenance](https://amigo.geneontology.org/amigo/term/GO:0097062), and [modification of postsynaptic actin cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0098885).


The gene length is 536,646 base pairs (98.57% of all genes), the mature length is 996 base pairs (22.38% of all genes), and the primary transcript length is 191,690 base pairs (95.48% of all genes).


The gene ITPKA (NCBI ID: 3706) has been mentioned in [15 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ITPKA%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 2001 and the middle 50% of publications occurring between 2011 and 2017. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning ITPKA, ranked by their relative citation ratio (RCR), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows. The most influential publication is "[Inositol 1,4,5-trisphosphate 3-kinase A associates with F-actin and dendritic spines via its N terminus.](https://pubmed.ncbi.nlm.nih.gov/11468283)" (2001) with a relative citation ratio of 1.85. This is followed by "[TFAP2A Induced ITPKA Serves as an Oncogene and Interacts with DBN1 in Lung Adenocarcinoma.](https://pubmed.ncbi.nlm.nih.gov/32015686)" (2020) with a relative citation ratio of 1.32. The publication "[ITPKA Gene Body Methylation Regulates Gene Expression and Serves as an Early Diagnostic Marker in Lung and Other Cancers.](https://pubmed.ncbi.nlm.nih.gov/27234602)" (2016) has a relative citation ratio of 1.07. Additionally, "[Inositol 1,4,5-trisphosphate 3-kinase-A is a new cell motility-promoting protein that increases the metastatic potential of tumor cells by two functional activities.](https://pubmed.ncbi.nlm.nih.gov/20022963)" (2010) has a relative citation ratio of 0.85, and "[Functional role of inositol-1,4,5-trisphosphate-3-kinase-A for motility of malignant transformed cells.](https://pubmed.ncbi.nlm.nih.gov/21792881)" (2011) has a relative citation ratio of 0.58. For more information on RCR, please refer to [this article](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541). The citation count is sourced from [iCite](https://icite.od.nih.gov).


[ITPKA](https://www.proteinatlas.org/ENSG00000137825-ITPKA) is a gene with molecular functions including calmodulin-binding, kinase, and transferase activities. Evidence for its function is available at the protein level. The gene is detected in many tissues and is primarily localized in vesicles and the plasma membrane. It is notably expressed in the brain, specifically in neuronal signaling clusters, and in neurons with mixed functions. Additionally, ITPKA is expressed in late spermatids, playing a role in spermatogenesis. The gene is also found in the RS4;11 cell line, although its function in this context is unknown.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SP1](https://www.ncbi.nlm.nih.gov/gene/6667), [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861), [KLF5](https://www.ncbi.nlm.nih.gov/gene/688), [AR](https://www.ncbi.nlm.nih.gov/gene/367), and [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664). Each of these TFs, except CTCF, was shown to be regulating genes in 5 experiments, while CTCF was involved in 4 experiments.



The GWAS data indicates associations with various disease conditions, including immune system disease, general disease, disease of anatomical entity, and hypersensitivity reaction disease.



The gene is ubiquitously expressed in multiple tissue samples, including the brain, colon, and stomach, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in various brain regions such as the Prefrontal Cortex, Whole brain, Caudate nucleus, Cingulate Cortex, Temporal Lobe, Occipital Lobe, and Pons.


The input data highlights key pathways involved in metabolism, specifically focusing on inositol phosphate metabolism. Notably, it includes the synthesis of IP3 and IP4 in the cytosol.



The analyzed protein sequence has a GRAVY value of -0.543 (30.75th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of 1.384 (55.21st percentile). The median structural flexibility is 1.006 (67.88th percentile), with a predicted secondary structure composition of 32.54% helix (55.42nd percentile), 29.50% sheet (25.82nd percentile), and 30.15% turn (64.93rd percentile). The instability index is 59.35 (81.79th percentile), suggesting a relatively stable protein. The isoelectric point is 7.59 (55.06th percentile), with a molecular weight of 51008.25 Da (52.82nd percentile). The sequence length is 461 amino acids (53.25th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |