---
layout: post
title: CCKBR
date: 2025-01-17 16:55 CST
description: CCKBR description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/887) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 887  | CCKBR | ENSG00000110148 | 11p15.4 |



The gene is involved in [gastric acid secretion](https://amigo.geneontology.org/amigo/term/GO:0001696) and [cell surface receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007166), including the [phospholipase C-activating G protein-coupled receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007200). It enables [cholecystokinin receptor activity](https://amigo.geneontology.org/amigo/term/GO:0004951) and [gastrin receptor activity](https://amigo.geneontology.org/amigo/term/GO:0015054), and is involved in the [cholecystokinin signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0038188). The gene is located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886) and [intracellular membrane-bounded organelle](https://amigo.geneontology.org/amigo/term/GO:0043231). Additionally, it plays a role in [positive regulation of cytosolic calcium ion concentration](https://amigo.geneontology.org/amigo/term/GO:0007204), [positive regulation of cell population proliferation](https://amigo.geneontology.org/amigo/term/GO:0008284), [pH reduction](https://amigo.geneontology.org/amigo/term/GO:0045851), [digestive tract development](https://amigo.geneontology.org/amigo/term/GO:0048565), and [gland development](https://amigo.geneontology.org/amigo/term/GO:0048732). The gene also enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [peptide hormone binding](https://amigo.geneontology.org/amigo/term/GO:0017046), [type B gastrin/cholecystokinin receptor binding](https://amigo.geneontology.org/amigo/term/GO:0031741), and [1-phosphatidylinositol-3-kinase regulator activity](https://amigo.geneontology.org/amigo/term/GO:0046935).


The gene length is 379,977 base pairs (97.14% of all genes), the mature length is 3,033 base pairs (72.94% of all genes), and the primary transcript length is 179,933 base pairs (95.11% of all genes).


The gene CCKBR (NCBI ID: 887) has been mentioned in [321 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CCKBR%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning this gene is 1976, and the middle 50% of publications occurred between 1995 and 2011.


The top 5 publications mentioning CCKBR, ranked by their scientific influence, include "[Cholecystokinin(CCK)-A and CCK-B/gastrin receptors in human tumors.](https://pubmed.ncbi.nlm.nih.gov/9102227)" (1997) (relative citation ratio: 8.57), "[The human brain cholecystokinin-B/gastrin receptor. Cloning and characterization.](https://pubmed.ncbi.nlm.nih.gov/7681836)" (1993) (relative citation ratio: 6.65), "[A single amino acid of the cholecystokinin-B/gastrin receptor determines specificity for non-peptide antagonists.](https://pubmed.ncbi.nlm.nih.gov/8455720)" (1993) (relative citation ratio: 6.52), "[Antiproliferative effects of gastrin receptor antagonists and antibodies to gastrin on human colon carcinoma cell lines.](https://pubmed.ncbi.nlm.nih.gov/3191491)" (1988) (relative citation ratio: 4.95), and "[Radiolabeled peptides for targeting cholecystokinin-B/gastrin receptor-expressing tumors.](https://pubmed.ncbi.nlm.nih.gov/10452322)" (1999) (relative citation ratio: 4.29). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CCKBR](https://www.proteinatlas.org/ENSG00000110148-CCKBR) is a G-protein coupled receptor involved in transduction, with evidence at the protein level. It is implicated in cancer-related genes and is an FDA-approved drug target. The gene is detected in some tissues and is localized in vesicles and the plasma membrane. Its expression is notable in endocrine and neural tissues, neurons, testis cancer cell lines, and is involved in neuronal signaling in single cells. While it is unprognostic in several cancer types, including Breast Invasive Carcinoma, Ovary Serous Cystadenocarcinoma, Pancreatic Adenocarcinoma, and Uterine Corpus Endometrial Carcinoma, its role in these contexts remains an area of interest.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 5 experiments, [EP300](https://www.ncbi.nlm.nih.gov/gene/2033) with 4 experiments, [EZH2](https://www.ncbi.nlm.nih.gov/gene/2146) with 4 experiments, [AR](https://www.ncbi.nlm.nih.gov/gene/367) with 4 experiments, and [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025) with 3 experiments.





The gene is expressed in multiple tissue samples, including the pancreas, brain, and stomach. Additionally, it shows expression in various regions of the brain such as the whole brain, cingulate cortex, subthalamic nucleus, temporal lobe, medulla oblongata, parietal lobe, pons, and cerebellum.


The input data includes various pathways and processes involving proteins, such as class a/1 (rhodopsin-like receptors), defective acth causing obesity and pro-opiomelanocortin deficiency (pomcd), gpcr ligand binding, metabolic disorders of biological oxidation enzymes, peptide ligand-binding receptors, disease, g alpha (q) signalling events, gastrin-creb signalling pathway via pkc and mapk, gpcr downstream signaling, signaling by gpcr, and signal transduction.



The analyzed protein sequence has a GRAVY value of 0.364 (93.15th percentile), indicating a high level of hydrophobicity. The protein exhibits a charge of 23.9 at pH 7.0 (95.36th percentile) and an isoelectric point of 10.03 (96.69th percentile). The median structural flexibility is 0.985 (7.58th percentile), with a length of 447 amino acids (51.55th percentile) and a molecular weight of 48418.57 Da (49.96th percentile). The secondary structure affinities are 29.08% for helix (24.15th percentile), 40.27% for sheet (87.05th percentile), and 28.86% for turn (54.82th percentile). The instability index is 42.97 (32.08th percentile). These values were calculated using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |