---
layout: post
title: PRKCH
date: 2025-01-17 16:55 CST
description: PRKCH description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5583) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5583  | PRKCH | ENSG00000027075 | 14q23.1 |



The gene enables [protein kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004672), [protein serine/threonine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004674), [diacylglycerol-dependent serine/threonine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004697), [diacylglycerol-dependent, calcium-independent serine/threonine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004699), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524), [enzyme binding](https://amigo.geneontology.org/amigo/term/GO:0019899), [small GTPase binding](https://amigo.geneontology.org/amigo/term/GO:0031267), [protein serine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0106310), and [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872). It is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [cell-cell junction](https://amigo.geneontology.org/amigo/term/GO:0005911), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). The gene is involved in [protein phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0006468), [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), [positive regulation of macrophage derived foam cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0010744), [peptidyl-serine phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0018105), [cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0030154), [negative regulation of glial cell apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0034351), [intracellular signal transduction](https://amigo.geneontology.org/amigo/term/GO:0035556), [positive regulation of keratinocyte differentiation](https://amigo.geneontology.org/amigo/term/GO:0045618), [positive regulation of B cell receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0050861), [positive regulation of NF-kappaB transcription factor activity](https://amigo.geneontology.org/amigo/term/GO:0051092), [positive regulation of glial cell proliferation](https://amigo.geneontology.org/amigo/term/GO:0060252), [protein kinase C signaling](https://amigo.geneontology.org/amigo/term/GO:0070528), [positive regulation of protein localization to plasma membrane](https://amigo.geneontology.org/amigo/term/GO:1903078), and [regulation of bicellular tight junction assembly](https://amigo.geneontology.org/amigo/term/GO:2000810).


The gene length is 17686.0 base pairs (39.0% of all genes), the mature length is 1494.5 base pairs, and the primary transcript length is 13400.0 base pairs.


The gene PRKCH (NCBI ID: 5583) has been mentioned in [40 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PRKCH%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1991 and the middle 50% of publications occurring between 2006 and 2015. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning PRKCH, ranked by their scientific influence as measured by the [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541), include: "[Isolation and characterization of PKC-L, a new member of the protein kinase C-related gene family specifically expressed in lung, skin, and heart.](https://pubmed.ncbi.nlm.nih.gov/1986216)" (1991) (relative citation ratio: 4.95), "[A nonsynonymous SNP in PRKCH (protein kinase C eta) increases the risk of cerebral infarction.](https://pubmed.ncbi.nlm.nih.gov/17206144)" (2007) (relative citation ratio: 3.26), "[The protein kinase C-related PKC-L(eta) gene product is localized in the cell nucleus.](https://pubmed.ncbi.nlm.nih.gov/1545811)" (1992) (relative citation ratio: 1.8), "[PKC-eta mediates glioblastoma cell proliferation through the Akt and mTOR signaling pathways.](https://pubmed.ncbi.nlm.nih.gov/15489897)" (2004) (relative citation ratio: 1.54), and "[Differentiation-associated localization of nPKC eta, a Ca(++)-independent protein kinase C, in normal human skin and skin diseases.](https://pubmed.ncbi.nlm.nih.gov/8245514)" (1993) (relative citation ratio: 1.43). The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PRKCH](https://www.proteinatlas.org/ENSG00000027075-PRKCH) is a gene encoding a protein with kinase, serine/threonine-protein kinase, and transferase molecular functions. It is involved in cancer-related genes and has evidence at the protein level. The gene is detected in all RNA tissue distributions and is primarily located in the plasma membrane and cytosol. PRKCH is expressed in various clusters, including T-cells with unknown function in blood, lymphoid tissue related to immune response, sub-cortical regions with mixed function in the brain, leukemia T-cell receptor in cell lines, and NK-cells involved in adaptive immune response in single cells.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [EP300](https://www.ncbi.nlm.nih.gov/gene/2033) with 6 experiments, [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) with 5 experiments, [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885) with 5 experiments, [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 5 experiments, and [AR](https://www.ncbi.nlm.nih.gov/gene/367) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The input data from GWAS indicates associations with a wide range of disease conditions, including various types of cancer such as lung cancer and respiratory system cancer, as well as obstructive lung disease and lower respiratory tract disease. Additionally, the data highlights links to metabolic disorders like obesity and acquired metabolic disease, along with cardiovascular conditions such as coronary artery disease and vascular disease. Mental health disorders, including bipolar disorder, schizophrenia, and anxiety disorder, are also implicated. Furthermore, the data points to associations with musculoskeletal system diseases like rheumatoid arthritis, sensory system diseases such as retinal degeneration, and specific developmental disorders like attention deficit hyperactivity disorder. Other notable conditions include chronic obstructive pulmonary disease, myocardial infarction, and familial atrial fibrillation.



The gene is expressed in various immune cell types, including CD56+ NK cells, CD4+ T cells, and CD8+ T cells, as well as in whole blood. Additionally, expression is observed in the lung, spleen, placenta, salivary gland, thymus, and lymph node.


The input data highlights several signaling pathways and processes, including G alpha (z) and G alpha (q) signaling events, the gastrin-CREB signaling pathway via PKC and MAPK, GPCR downstream signaling, and signaling by GPCR. Additionally, it covers signal transduction, the effects of PIP2 hydrolysis, hemostasis, and platelet activation, signaling, and aggregation.


The mouse ortholog gene 18755 and its human ortholog gene 5583 are associated with the phenotype of decreased grip strength, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.286 (59.56th percentile), indicating a hydrophilic nature. It exhibits a charge of 4.72 at pH 7.0 (67.09th percentile) and a median structural flexibility of 0.996 (24.27th percentile). The protein's secondary structure is predicted to be 28.6% helix (20.56th percentile), 36.7% sheet (71.43rd percentile), and 25.6% turn (26.11th percentile). The instability index is 40.8 (24.62nd percentile), with an isoelectric point of 7.94 (58.22nd percentile). The protein has a length of 683 amino acids (74.26th percentile) and a molecular weight of 77827.6 Da (75.1st percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |