---
layout: post
title: NUDT4
date: 2025-01-17 16:55 CST
description: NUDT4 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/11163) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 11163  | NUDT4 | ENSG00000173598 | 12q22 |



The gene enables [endopolyphosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0000298), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [diphosphoinositol-polyphosphate diphosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0008486), [snoRNA binding](https://amigo.geneontology.org/amigo/term/GO:0030515), [bis(5'-adenosyl)-hexaphosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0034431), [bis(5'-adenosyl)-pentaphosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0034432), [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872), [inositol-5-diphosphate-1,2,3,4,6-pentakisphosphate diphosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0052845), and [inositol-3,5-bisdiphosphate-2,3,4,6-tetrakisphosphate 5-diphosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0052848). It is active in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), and is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). The gene is involved in [calcium-mediated signaling](https://amigo.geneontology.org/amigo/term/GO:0019722), [cyclic-nucleotide-mediated signaling](https://amigo.geneontology.org/amigo/term/GO:0019935), [intracellular signal transduction](https://amigo.geneontology.org/amigo/term/GO:0035556), [diphosphoinositol polyphosphate metabolic process](https://amigo.geneontology.org/amigo/term/GO:0071543), [diadenosine pentaphosphate catabolic process](https://amigo.geneontology.org/amigo/term/GO:1901907), [diadenosine hexaphosphate catabolic process](https://amigo.geneontology.org/amigo/term/GO:1901909), and [adenosine 5'-(hexahydrogen pentaphosphate) catabolic process](https://amigo.geneontology.org/amigo/term/GO:1901911).


The gene length is 13,165 base pairs (31.91 percentile of all genes), the mature length is 2,188 base pairs, and the primary transcript length is 13,116 base pairs.


The gene NUDT4 (NCBI ID: 11163) has been mentioned in [6 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22NUDT4%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 2001, and the middle 50% of publications occurred between 2001 and 2004.


The top 5 publications mentioning NUDT4, ranked by their relative citation ratio (RCR), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows. The most influential publication is "[Reactivity of Ar'GeGeAr' (Ar' = C6H3-2,6-Dipp2, Dipp = C6H3-2,6-iPr2) toward alkynes: isolation of a stable digermacyclobutadiene.](https://pubmed.ncbi.nlm.nih.gov/15099077)" (2004) with an RCR of 1.67. This is followed by "[Stable, Monomeric Imides of Aluminum and Gallium: Synthesis and Characterization of [{HC(MeCDippN)<sub>2</sub> }MN-2,6-Trip<sub>2</sub> C<sub>6</sub> H<sub>3</sub> ] (M=Al or Ga; Dipp=2,6-iPr<sub>2</sub> C<sub>6</sub> H<sub>3</sub> ; Trip=2,4,6-iPr<sub>3</sub> C<sub>6</sub> H<sub>2</sub> ).](https://pubmed.ncbi.nlm.nih.gov/29712210)" (2001) with an RCR of 1.61. The third most influential publication is "[The "diindene" ArInInAr (Ar = C(6)H(3)-2,6-Dipp(2), Dipp = C(6)H(3)-2,6-Pr(i)(2)). Dimeric versus monomeric in(I) aryls: para-substituent effects in terphenyl ligands.](https://pubmed.ncbi.nlm.nih.gov/12121089)" (2002) with an RCR of 1.3. The RCR values are sourced from [iCite](https://icite.od.nih.gov).


[NUDT4](https://www.proteinatlas.org/ENSG00000173598-NUDT4) is a gene with hydrolase and RNA-binding molecular functions, supported by evidence at the protein level. It is detected in all tissues and primarily localized in the cytosol. NUDT4 is expressed in various clusters, including Cluster 22 in eosinophils related to transcription, Cluster 56 in non-specific tissues with unknown function, Cluster 53 in neurons with mixed function, Cluster 47 in leukemia cell lines related to hemostasis, and Cluster 48 in smooth muscle cells involved in signal transduction.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613), [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), [PGR](https://www.ncbi.nlm.nih.gov/gene/5241), [RARA](https://www.ncbi.nlm.nih.gov/gene/5914), and [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861), each shown to be regulating in 4 experiments.





The gene is expressed in various tissues, including the heart and ovary, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in leukemia (chronic myelogenous K-562), CD71+ early erythroid cells, prostate, colon, bone marrow, fetal liver, superior cervical ganglion, and trigeminal ganglion.


The input data highlights several key pathways involved in cellular metabolism. These include inositol phosphate metabolism and the synthesis of pyrophosphates in the cytosol.



The analyzed protein sequence exhibits a GRAVY value of -0.574 (27.68th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -2.234 (42.3rd percentile), and the median structural flexibility is 1.006 (68.09th percentile). The protein has a helix affinity of 0.338 (66.22nd percentile), a sheet affinity of 0.327 (43.51st percentile), and a turn affinity of 0.277 (44.62nd percentile). The instability index is 51.02 (60.5th percentile), with an isoelectric point of 5.994 (30.41st percentile). The sequence length is 180.5 amino acids (11.69th percentile), and the molecular weight is 20369.85 Da (11.82nd percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |