---
layout: post
title: HLA-DPB1
date: 2025-01-17 16:55 CST
description: HLA-DPB1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3115) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3115  | HLA-DPB1 | ENSG00000223865 | 6p21.32 |



The gene is located in the [Golgi membrane](https://amigo.geneontology.org/amigo/term/GO:0000139), [lysosomal membrane](https://amigo.geneontology.org/amigo/term/GO:0005765), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986), [ER to Golgi transport vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0012507), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [transport vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030658), [endocytic vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030666), [clathrin-coated endocytic vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030669), [trans-Golgi network membrane](https://amigo.geneontology.org/amigo/term/GO:0032729), and the [lumenal side of endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0098553). It is involved in the [adaptive immune response](https://amigo.geneontology.org/amigo/term/GO:0002250), [peptide antigen assembly with MHC class II protein complex](https://amigo.geneontology.org/amigo/term/GO:0002503), [antigen processing and presentation of exogenous peptide antigen via MHC class II](https://amigo.geneontology.org/amigo/term/GO:0019886), [positive regulation of type II interferon production](https://amigo.geneontology.org/amigo/term/GO:0032729), [positive regulation of T cell proliferation](https://amigo.geneontology.org/amigo/term/GO:0042102), [positive regulation of immune response](https://amigo.geneontology.org/amigo/term/GO:0050778), [T cell receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0050852), and [positive regulation of T cell activation](https://amigo.geneontology.org/amigo/term/GO:0050870). The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [MHC class II protein complex binding](https://amigo.geneontology.org/amigo/term/GO:0023026), and [peptide antigen binding](https://amigo.geneontology.org/amigo/term/GO:0042605). It is active in the [lysosomal membrane](https://amigo.geneontology.org/amigo/term/GO:0005765) and [late endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0031902), and is part of the [MHC class II protein complex](https://amigo.geneontology.org/amigo/term/GO:0042613).


The gene length is 5,247 base pairs (14.7% of all genes), the mature length is 3,353 base pairs, and the primary transcript length is 5,163 base pairs (19.19% of all genes).


The gene HLA-DPB1 (NCBI ID: 3115) has been mentioned in [606 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22HLA-DPB1%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1985, and the middle 50% of publications range from 1995 to 2016.


The top 5 publications mentioning HLA-DPB1, ranked by their scientific influence, include "[The IPD and IMGT/HLA database: allele variant databases.](https://pubmed.ncbi.nlm.nih.gov/25414341)" (2015) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 51.24, "[A genome-wide association study identifies variants in the HLA-DP locus associated with chronic hepatitis B in Asians.](https://pubmed.ncbi.nlm.nih.gov/19349983)" (2009) with an RCR of 9.25, "[Effect of T-cell-epitope matching at HLA-DPB1 in recipients of unrelated-donor haemopoietic-cell transplantation: a retrospective study.](https://pubmed.ncbi.nlm.nih.gov/22340965)" (2012) with an RCR of 7.78, "[High HLA-DP Expression and Graft-versus-Host Disease.](https://pubmed.ncbi.nlm.nih.gov/26267621)" (2015) with an RCR of 7.07, and "[Nonpermissive HLA-DPB1 mismatch increases mortality after myeloablative unrelated allogeneic hematopoietic cell transplantation.](https://pubmed.ncbi.nlm.nih.gov/25161269)" (2014) with an RCR of 6.54. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[HLA-DPB1](https://www.proteinatlas.org/ENSG00000223865-HLA-DPB1) is a gene with evidence at the protein level, detected in many RNA tissues. It is primarily located in the nucleoplasm and cell junctions. The gene is expressed in dendritic cells with unknown function (Cluster 2) and is involved in immune response in the spleen (Cluster 47). In the brain, it is associated with non-specific transcription (Cluster 25), and in lymphoma cell lines, it is involved in immune response (Cluster 55). Single-cell expression analysis places it in dendritic cells, contributing to antigen presentation (Cluster 28). The gene is unprognostic for various cancers, including Bladder Urothelial Carcinoma, Breast Invasive Carcinoma, and Cervical Squamous Cell Carcinoma, among others.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 5 experiments, [EP300](https://www.ncbi.nlm.nih.gov/gene/2033) with 4 experiments, [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908) with 4 experiments, [SPI1](https://www.ncbi.nlm.nih.gov/gene/6688) with 4 experiments, and [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with a wide range of disease conditions, including nephrosis, nephrotic syndrome, various types of cancer such as reproductive organ cancer, gastrointestinal system cancer, and hematologic cancer. Additionally, the data highlights infectious diseases, particularly viral infectious diseases, and respiratory conditions like asthma, bronchial disease, and obstructive lung disease. Metabolic disorders, including diabetes mellitus and obesity, are also implicated, along with cardiovascular diseases such as coronary artery disease and vascular disease. Mental health disorders, arthritis, and various autoimmune diseases, including rheumatoid arthritis and systemic scleroderma, are also noted. Furthermore, the data includes liver diseases like liver cirrhosis and primary biliary cirrhosis, as well as specific conditions such as myocardial infarction, primary immunodeficiency disease, and hepatocellular carcinoma.


The HLA-DPB1 gene has been associated with several diseases and traits through genome-wide association studies. Notably, it has been linked to [Antineutrophil cytoplasmic antibody-associated vasculitis](https://pubmed.ncbi.nlm.nih.gov/28029757), which maps closely to the MeSH term [Antibodies, Antineutrophil Cytoplasmic](https://meshb.nlm.nih.gov/record/ui?ui=D019268) using the BioWordVec method. Additionally, HLA-DPB1 is associated with [Aspirin exacerbated respiratory disease in asthmatics](https://pubmed.ncbi.nlm.nih.gov/23180272), mapped to the MeSH term [Asthma](https://meshb.nlm.nih.gov/record/ui?ui=D001249). Other associations include [Fibrinogen](https://pubmed.ncbi.nlm.nih.gov/20031577), mapped directly to the MeSH term [Fibrinogen](https://meshb.nlm.nih.gov/record/ui?ui=D005340), and [Monocyte count](https://pubmed.ncbi.nlm.nih.gov/27863252), mapped to the MeSH term [Monocytes](https://meshb.nlm.nih.gov/record/ui?ui=D009000). Furthermore, HLA-DPB1 is linked to [Response to hepatitis B vaccine](https://pubmed.ncbi.nlm.nih.gov/31243853), mapped to the MeSH term [Hepatitis](https://meshb.nlm.nih.gov/record/ui?ui=D006505), and [Seropositivity for staphylococcus aureus peptide (twist_30077)](https://pubmed.ncbi.nlm.nih.gov/37164013), mapped to the MeSH term [Staphylococcus](https://meshb.nlm.nih.gov/record/ui?ui=D013210). Lastly, associations include [Serum glutamic-oxaloacetic transaminase levels](https://pubmed.ncbi.nlm.nih.gov/36329257), mapped to the MeSH term [Transaminases](https://meshb.nlm.nih.gov/record/ui?ui=D000637), and [Severe aplastic anemia](https://pubmed.ncbi.nlm.nih.gov/32004448), mapped to the MeSH term [Anemia](https://meshb.nlm.nih.gov/record/ui?ui=D000740).


The gene is expressed in multiple tissue samples, including lung, blood, and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Lymphoma Burkitt's (Raji), BDCA4+ Dendritic Cells, CD19+ B Cells (neg. sel.), Lung, Tonsil, and Lymph node.


The input data highlights various pathways involved in immune system function, including cytokine signaling in the immune system, interferon signaling, and interferon gamma signaling. Additionally, it covers adaptive immune system processes, costimulation by the CD28 family, and downstream TCR signaling. Other notable pathways include TCR signaling, generation of second messenger molecules, PD-1 signaling, phosphorylation of CD3 and TCR zeta chains, translocation of ZAP-70 to the immunological synapse, and MHC class II antigen presentation.



The analyzed protein sequence has a GRAVY value of -0.2895 (59.2% percentile), indicating a hydrophilic nature. It exhibits a charge of 4.01 at pH 7.0 (64.78% percentile) and a median structural flexibility of 0.997 (27.73% percentile). The protein's secondary structure is predicted to be 27.52% helix (14.47% percentile), 37.21% sheet (74.32% percentile), and 25.19% turn (22.94% percentile). The instability index is 48.66 (52.54% percentile), with an isoelectric point of 8.83 (74.53% percentile). The protein has a length of 258 amino acids (22.74% percentile) and a molecular weight of 29159.01 Da (23.31% percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |