---
layout: post
title: FZD2
date: 2025-01-17 16:55 CST
description: FZD2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2535) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2535  | FZD2 | ENSG00000180340 | 17q21.31 |



The gene is involved in various biological processes, including [membranous septum morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0003149), [muscular septum morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0003150), and [outflow tract morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0003151). It enables [G protein-coupled receptor activity](https://amigo.geneontology.org/amigo/term/GO:0004930) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737) and is active in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), where it is also located. Additionally, it is found in the [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925) and the [clathrin-coated endocytic vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030669). The gene is involved in the [G protein-coupled receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007186), [sensory perception of smell](https://amigo.geneontology.org/amigo/term/GO:0007608), and various Wnt signaling pathways, including the [canonical Wnt signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0060070) and the [non-canonical Wnt signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0035567). It enables [Wnt-protein binding](https://amigo.geneontology.org/amigo/term/GO:0017147) and [Wnt receptor activity](https://amigo.geneontology.org/amigo/term/GO:0042813), and is involved in processes such as [neuron differentiation](https://amigo.geneontology.org/amigo/term/GO:0030182), [endothelial cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0045446), and [hard palate development](https://amigo.geneontology.org/amigo/term/GO:0060022). The gene also plays a role in [inner ear receptor cell development](https://amigo.geneontology.org/amigo/term/GO:0060119), [cochlea morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0090103), and the [planar cell polarity pathway involved in neural tube closure](https://amigo.geneontology.org/amigo/term/GO:0090179).


The gene length is 360,235 base pairs (96.82% of all genes), the mature length is 2,689 base pairs, and the primary transcript length is 334,886 base pairs.


The gene FZD2 (NCBI ID: 2535) has been mentioned in [28 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22FZD2%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning FZD2 is 1998, and the middle 50% of publications occurred between 2013 and 2018.


The top 5 publications mentioning FZD2, ranked by their scientific influence, include "[A noncanonical Frizzled2 pathway regulates epithelial-mesenchymal transition and metastasis.](https://pubmed.ncbi.nlm.nih.gov/25417160)" (2014) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 7.56, "[Wnt5a regulates distinct signalling pathways by binding to Frizzled2.](https://pubmed.ncbi.nlm.nih.gov/19910923)" (2010) with an RCR of 5.31, "[Molecular cloning, differential expression, and chromosomal localization of human frizzled-1, frizzled-2, and frizzled-7.](https://pubmed.ncbi.nlm.nih.gov/9813155)" (1998) with an RCR of 3.4, "[Frizzled 1 and frizzled 2 genes function in palate, ventricular septum and neural tube closure: general implications for tissue fusion processes.](https://pubmed.ncbi.nlm.nih.gov/20940229)" (2010) with an RCR of 2.58, and "[Frizzled 2-induced epithelial-mesenchymal transition correlates with vasculogenic mimicry, stemness, and Hippo signaling in hepatocellular carcinoma.](https://pubmed.ncbi.nlm.nih.gov/30677195)" (2019) with an RCR of 1.49. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[FZD2](https://www.proteinatlas.org/ENSG00000180340-FZD2) is a developmental protein that functions as a G-protein coupled receptor and transducer. It is involved in cancer-related processes and has disease-associated variants. Evidence for its function is available at the protein level. FZD2 is detected in many tissues and is localized to various subcellular regions, including the nucleoplasm, nuclear bodies, cell junctions, actin filaments, and basal body. It is expressed in dendritic cells with unknown function, ciliated cells involved in cilium organization, choroid plexus with mixed function, neuronal cell lines involved in signal transduction, and connective tissue cells involved in ECM organization.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [JUN](https://www.ncbi.nlm.nih.gov/gene/3725) with 6 experiments, [E2F8](https://www.ncbi.nlm.nih.gov/gene/79733) with 4 experiments, [KMT2B](https://www.ncbi.nlm.nih.gov/gene/9757) with 4 experiments, [KDM1A](https://www.ncbi.nlm.nih.gov/gene/23028) with 4 experiments, and [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent detection in multiple experiments.



The GWAS data indicates associations with various conditions, including cognitive disorders, diseases of mental health, mood disorders, and sleep disorders.



The gene is expressed in nerve and skin tissues according to the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles indicate expression in CD33+ Myeloid cells, BDCA4+ Dendritic Cells, CD14+ Monocytes, Smooth Muscle, Colorectal adenocarcinoma, Lung, and Uterus.


The input data highlights various pathways and processes involving proteins, including asymmetric localization of PCP proteins, the PCP/CE pathway, beta-catenin independent WNT signaling, and enhanced WNT signaling and proliferation due to RNF mutants. Additionally, it covers TCF dependent signaling in response to WNT, the stabilization of AXIN by XAV939 through tankyrase inhibition, and enhanced beta-catenin-dependent signaling by misspliced LRP5 mutants. The data also includes signaling by WNT in cancer, general WNT signaling, disassembly of the destruction complex and recruitment of AXIN to the membrane, and the Ca2+ pathway. Furthermore, it mentions Class B/2 (Secretin family receptors), defective ACTH causing obesity and pro-opiomelanocortin deficiency (POMCD), GPCR ligand binding, metabolic disorders of biological oxidation enzymes, and general disease processes. Other notable pathways include signaling by GPCR and signal transduction, as well as WNT5A-dependent internalization of FZD2, FZD5, and ROR2.


The mouse ortholog gene 57265, which corresponds to the human ortholog gene 2535, is associated with several phenotypic outcomes. These include embryonic lethality prior to organogenesis, embryonic lethality prior to the tooth bud stage, and preweaning lethality with complete penetrance. Notably, the preweaning lethality phenotype is observed in both sexes.


The analyzed protein sequence has a GRAVY value of 0.141 (89.07th percentile), indicating a relatively hydrophobic nature. The protein exhibits a charge of 7.82 at pH 7.0 (76.43rd percentile) and an isoelectric point of 8.47 (66.09th percentile). Structurally, the protein shows a median flexibility of 0.985 (7.54th percentile), with secondary structure affinities of 28.32% for helix (19.03rd percentile), 40.71% for sheet (88.17th percentile), and 24.78% for turn (19.96th percentile). The instability index is 36.86 (13.86th percentile), suggesting moderate stability. The protein has a length of 565 amino acids (65.44th percentile) and a molecular weight of 63553.44 Da (65.98th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |