---
layout: post
title: RABEP1
date: 2025-01-17 16:55 CST
description: RABEP1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9135) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9135  | RABEP1 | ENSG00000029725 | 17p13.2 |



The gene enables [GTPase activator activity](https://amigo.geneontology.org/amigo/term/GO:0005096) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is active in and located in the [endosome](https://amigo.geneontology.org/amigo/term/GO:0005768). It is also located in the [early endosome](https://amigo.geneontology.org/amigo/term/GO:0005769), [endocytic vesicle](https://amigo.geneontology.org/amigo/term/GO:0030139), [early endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0031901), [recycling endosome](https://amigo.geneontology.org/amigo/term/GO:0055037), and [intracellular membrane-bounded organelle](https://amigo.geneontology.org/amigo/term/GO:0043231). The gene is involved in various processes including [Golgi to plasma membrane transport](https://amigo.geneontology.org/amigo/term/GO:0006893), [endocytosis](https://amigo.geneontology.org/amigo/term/GO:0006897), [apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0006915), [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), [protein transport](https://amigo.geneontology.org/amigo/term/GO:0015031), [vesicle-mediated transport](https://amigo.geneontology.org/amigo/term/GO:0016192), and [membrane fusion](https://amigo.geneontology.org/amigo/term/GO:0061025). Additionally, it enables [growth factor activity](https://amigo.geneontology.org/amigo/term/GO:0008083), [protein domain specific binding](https://amigo.geneontology.org/amigo/term/GO:0019904), and [protein homodimerization activity](https://amigo.geneontology.org/amigo/term/GO:0042803). The gene is also part of a [protein-containing complex](https://amigo.geneontology.org/amigo/term/GO:0032991) and involved in [protein localization to ciliary membrane](https://amigo.geneontology.org/amigo/term/GO:1903441).


The gene length is 222,932.0 base pairs (93.58% of all genes), the mature length is 2,043.5 base pairs, and the primary transcript length is 133,334.5 base pairs.


The gene RABEP1 (NCBI ID: 9135) has been mentioned in [36 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RABEP1%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication mentioning this gene is 1997, and the middle 50% of publications occurred between 2001 and 2011.


The top 5 publications mentioning RABEP1, ranked by their scientific influence, include "[A novel Rab5 GDP/GTP exchange factor complexed to Rabaptin-5 links nucleotide exchange to effector recruitment and function.](https://pubmed.ncbi.nlm.nih.gov/9323142)" (1997) (relative citation ratio: 8.02), "[Distinct Rab-binding domains mediate the interaction of Rabaptin-5 with GTP-bound Rab4 and Rab5.](https://pubmed.ncbi.nlm.nih.gov/9524117)" (1998) (relative citation ratio: 2.95), "[Ciliary membrane proteins traffic through the Golgi via a Rabep1/GGA1/Arl3-dependent mechanism.](https://pubmed.ncbi.nlm.nih.gov/25405894)" (2014) (relative citation ratio: 2.55), "[Functional synergy between Rab5 effector Rabaptin-5 and exchange factor Rabex-5 when physically associated in a complex.](https://pubmed.ncbi.nlm.nih.gov/11452015)" (2001) (relative citation ratio: 2.36), and "[Divalent interaction of the GGAs with the Rabaptin-5-Rabex-5 complex.](https://pubmed.ncbi.nlm.nih.gov/12505986)" (2003) (relative citation ratio: 1.96). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RABEP1](https://www.proteinatlas.org/ENSG00000029725-RABEP1) is a cancer-related gene with evidence at the protein level. Its RNA is detected in all tissues, and the protein is localized in vesicles and the cytosol. The gene is expressed in various clusters, including Cluster 14 for blood, Cluster 3 for tissue, Cluster 13 for brain (associated with oligodendrocytes), Cluster 42 for cell lines, and Cluster 15 for single cells, all with non-specific or mixed functions.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [KDM2B](https://www.ncbi.nlm.nih.gov/gene/84678), [SP1](https://www.ncbi.nlm.nih.gov/gene/6667), and [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885), each with 5 experiments. Additionally, [GATA2](https://www.ncbi.nlm.nih.gov/gene/2624) and [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) were found to be regulating in 4 experiments each.



The input data indicates associations with a wide range of disease conditions, including various types of cancer such as leukemia, lymphoma, and immune system cancer. Additionally, respiratory conditions like asthma, bronchial disease, and obstructive lung disease are noted. The data also encompasses hematopoietic and immune system diseases, as well as cardiovascular system disease. Furthermore, kidney disease, urinary system disease, cognitive disorder, and disease of mental health are included in the dataset.



The gene is expressed in various tissues, including muscle, breast, and brain, according to the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles indicate expression in Cardiac Myocytes, Heart, Liver, 721_B_lymphoblasts, Superior Cervical Ganglion, and Skeletal Muscle.




The analyzed protein sequence has a GRAVY value of -0.848 (9.51 percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -47.86 (2.65 percentile), and the median structural flexibility is 1.015 (92.08 percentile). The protein has a high affinity for helix structures (44.66%, 98.09 percentile), with lower affinities for sheet (28.54%, 20.85 percentile) and turn (20.53%, 3.13 percentile) structures. The instability index is 56.27 (75.37 percentile), and the isoelectric point is 4.95 (6.89 percentile). The protein is 862 amino acids long (83.14 percentile) with a molecular weight of 99289.29 Da (84.07 percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |