---
layout: post
title: HNRNPH2
date: 2025-01-17 16:55 CST
description: HNRNPH2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3188) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3188  | HNRNPH2 | ENSG00000126945 | Xq22.1 |



The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [postsynaptic density](https://amigo.geneontology.org/amigo/term/GO:0014069), and [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020). It is involved in the [regulation of RNA splicing](https://amigo.geneontology.org/amigo/term/GO:0043484) and is part of the [ribonucleoprotein complex](https://amigo.geneontology.org/amigo/term/GO:1990904).


The gene length is 15,103 base pairs (35.24% of all genes), the mature length is 1,461.5 base pairs, and the primary transcript length is 7,443 base pairs.


The gene HNRNPH2 (NCBI ID: 3188) has been mentioned in [32 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22HNRNPH2%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1999 and the middle 50% of publications occurring between 2007 and 2020. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning HNRNPH2, ranked by their scientific influence, include "[hnRNP H is a component of a splicing enhancer complex that activates a c-src alternative exon in neuronal cells.](https://pubmed.ncbi.nlm.nih.gov/9858532)" (1999) (relative citation ratio: 3.6), "[Essential role for the interaction between hnRNP H/F and a G quadruplex in maintaining p53 pre-mRNA 3'-end processing and function during DNA damage.](https://pubmed.ncbi.nlm.nih.gov/21289067)" (2011) (relative citation ratio: 2.88), "[Splicing factor hnRNPH drives an oncogenic splicing switch in gliomas.](https://pubmed.ncbi.nlm.nih.gov/21915099)" (2011) (relative citation ratio: 2.75), "[Interaction of muscleblind, CUG-BP1 and hnRNP H proteins in DM1-associated aberrant IR splicing.](https://pubmed.ncbi.nlm.nih.gov/16946708)" (2006) (relative citation ratio: 2.29), and "[SR proteins and hnRNP H regulate the splicing of the HIV-1 tev-specific exon 6D.](https://pubmed.ncbi.nlm.nih.gov/11847131)" (2002) (relative citation ratio: 1.8). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[HNRNPH2](https://www.proteinatlas.org/ENSG00000126945-HNRNPH2) is an RNA-binding ribonucleoprotein primarily located in the nucleoplasm. It is implicated in intellectual disability, with evidence supporting its role at the protein level. The gene is detected in all RNA tissue distributions and is associated with various expression clusters, including Cluster 28 in neutrophils, Cluster 35 in non-specific mitochondria, Cluster 21 in brain metabolism, Cluster 36 in cell line mRNA processing, and Cluster 27 in syncytiotrophoblasts related to pregnancy.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2B](https://www.ncbi.nlm.nih.gov/gene/9757), [TAL1](https://www.ncbi.nlm.nih.gov/gene/6886), and [RELA](https://www.ncbi.nlm.nih.gov/gene/5970), each with 5 experiments. Additionally, [E2F1](https://www.ncbi.nlm.nih.gov/gene/1869) and [E2F8](https://www.ncbi.nlm.nih.gov/gene/79733) were found to be regulating in 4 experiments each.



The GWAS data indicates associations with various disease conditions, including cancer, diseases of cellular proliferation, organ system cancer, lung cancer, respiratory system cancer, and general disease.



The gene is expressed in various tissues, including the pineal gland (both day and night), prefrontal cortex, whole blood, amygdala, colon, and small intestine.


The proteins are involved in several key processes, including gene expression, processing of capped intron-containing pre-mRNA, and mRNA splicing, with a particular focus on the major pathway of mRNA splicing.


The mouse ortholog gene 56258, which corresponds to the human ortholog gene 3188, is associated with several phenotypic abnormalities. These include abnormal coat/hair morphology, abnormal coat/hair pigmentation, and abnormal vocalization, all of which are observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.6002 (25.38th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -7.43 (27.48th percentile), and the median structural flexibility is 1.007 (70.52nd percentile). The protein has a length of 449 amino acids (51.75th percentile) and a molecular weight of 49263.03 Da (50.83rd percentile). The secondary structure affinities are 25.84% for helix (7.1st percentile), 28.51% for sheet (20.72nd percentile), and 36.53% for turn (92.8th percentile). The instability index is 32.99 (7.1st percentile), and the isoelectric point is 5.89 (27.7th percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |