---
layout: post
title: SRSF6
date: 2025-01-17 16:55 CST
description: SRSF6 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6431) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6431  | SRSF6 | ENSG00000124193 | 20q13.11 |



The gene is involved in [alternative mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0000380) and its regulation, acting upstream of or within [regulation of alternative mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0000381). It also plays a role in [mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0000398) and [mRNA splice site recognition](https://amigo.geneontology.org/amigo/term/GO:0006376). The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654) and [nuclear speck](https://amigo.geneontology.org/amigo/term/GO:0016607), where it is active. Additionally, the gene is involved in the [response to insulin](https://amigo.geneontology.org/amigo/term/GO:0032868), [regulation of keratinocyte proliferation](https://amigo.geneontology.org/amigo/term/GO:0010837), and [regulation of wound healing](https://amigo.geneontology.org/amigo/term/GO:0061041). It also plays a role in the [negative regulation of keratinocyte differentiation](https://amigo.geneontology.org/amigo/term/GO:0045617), [negative regulation of mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0048025), and [negative regulation of type B pancreatic cell apoptotic process](https://amigo.geneontology.org/amigo/term/GO:2000675). The gene acts upstream of or within [positive regulation of epithelial cell proliferation involved in lung morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0060501) and enables [pre-mRNA binding](https://amigo.geneontology.org/amigo/term/GO:0036002).


The gene length is 9,371 base pairs (24.37th percentile), the mature length is 4,295 base pairs (86.18th percentile), and the primary transcript length is 9,293 base pairs (31.4th percentile).


SRSF6 (Gene ID: 6431) has been mentioned in [36 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SRSF6%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest mention in 1986. The middle 50% of these publications were published between 2003 and 2021.


The top 5 publications mentioning SRSF6, ranked by their scientific influence, include studies that explore the gene's role in various cancers and diseases. The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Notable publications include "[LncRNA CRNDE attenuates chemoresistance in gastric cancer via SRSF6-regulated alternative splicing of PICALM.](https://pubmed.ncbi.nlm.nih.gov/33397371)" (2021) (relative citation ratio: 9.26), "[Long non-coding RNA LINC01133 inhibits epithelial-mesenchymal transition and metastasis in colorectal cancer by interacting with SRSF6.](https://pubmed.ncbi.nlm.nih.gov/27443606)" (2016) (relative citation ratio: 5.46), "[SRSF6-regulated alternative splicing that promotes tumour progression offers a therapy target for colorectal cancer.](https://pubmed.ncbi.nlm.nih.gov/29114070)" (2019) (relative citation ratio: 5.01), "[miR-193a-5p promotes pancreatic cancer cell metastasis through SRSF6-mediated alternative splicing of OGDHL and ECM1.](https://pubmed.ncbi.nlm.nih.gov/32064152)" (2020) (relative citation ratio: 2.96), and "[DRAK2 aggravates nonalcoholic fatty liver disease progression through SRSF6-associated RNA alternative splicing.](https://pubmed.ncbi.nlm.nih.gov/34614409)" (2021) (relative citation ratio: 2.92). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[SRSF6](https://www.proteinatlas.org/ENSG00000124193-SRSF6) is a repressor and RNA-binding protein with evidence at the protein level. It is detected in all RNA tissue distributions and is primarily located in nuclear speckles. The gene is part of Cluster 33 in blood expression, which is associated with non-specific ATP binding, and Cluster 6 in tissue expression, which is linked to the bone marrow and innate immune response. In brain expression, it belongs to Cluster 50, associated with non-specific nucleic acid binding. Additionally, SRSF6 is part of Cluster 62 in cell line expression, which is related to plasma proteins, and Cluster 15 in single-cell expression, which has mixed functions.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2B](https://www.ncbi.nlm.nih.gov/gene/9757) and [SP1](https://www.ncbi.nlm.nih.gov/gene/6667), each with 5 experiments. Additionally, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), [TAF1](https://www.ncbi.nlm.nih.gov/gene/6872), and [E2F8](https://www.ncbi.nlm.nih.gov/gene/79733) were each found to be regulating in 4 experiments.





The gene is expressed in various tissues, including the ovary, thyroid, and prostate. Additionally, it is expressed in specific cell types such as CD33+ myeloid cells, BDCA4+ dendritic cells, and CD105+ endothelial cells. Expression is also noted in the pituitary gland, leukemia lymphoblastic cells (MOLT-4), and lymph nodes.


The input data highlights various pathways involved in gene expression and mRNA processing. These include transport of mature transcripts to the cytoplasm, RNA polymerase II transcription, cleavage of growing transcripts in the termination region, and post-elongation processing of transcripts. Additionally, the data covers mRNA 3'-end processing, mRNA splicing through both major and minor pathways, and the processing of capped intron-containing pre-mRNA.


The mouse ortholog gene 67996, which corresponds to the human ortholog gene 6431, is associated with several phenotypic abnormalities. These include abnormal embryo size, abnormal vocalization, and preweaning lethality with incomplete penetrance. Notably, these phenotypes are observed in both sexes.


The analyzed protein sequence has a GRAVY value of -1.55 (0.49 percentile), indicating a hydrophilic nature. It exhibits a charge of 55.95 at pH 7.0 (99.57 percentile) and a median structural flexibility of 1.024 (98.43 percentile). The protein's secondary structure is predicted to be 20.35% helix (1.29 percentile), 18.60% sheet (1.43 percentile), and 41.57% turn (98.17 percentile). The instability index is 103.80 (99.65 percentile), suggesting high instability. The isoelectric point is 11.42 (99.55 percentile), with a molecular weight of 39586.33 Da (38.9 percentile) and a length of 344 amino acids (36.98 percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |