---
layout: post
title: OLFM1
date: 2025-01-17 16:55 CST
description: OLFM1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10439) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10439  | OLFM1 | ENSG00000130558 | 9q34.3 |



The gene is involved in various biological processes, including [atrioventricular valve formation](https://amigo.geneontology.org/amigo/term/GO:0003190), [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), [nervous system development](https://amigo.geneontology.org/amigo/term/GO:0007399), [positive regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0010628), [negative regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0010629), [positive regulation of epithelial to mesenchymal transition](https://amigo.geneontology.org/amigo/term/GO:0010718), [neuronal signal transduction](https://amigo.geneontology.org/amigo/term/GO:0023041), [regulation of axon extension](https://amigo.geneontology.org/amigo/term/GO:0030516), [positive regulation of apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0043065), and [cardiac epithelial to mesenchymal transition](https://amigo.geneontology.org/amigo/term/GO:0060317). It enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is active in the [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615). The gene is located in the [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615), [endoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0005783), [axon](https://amigo.geneontology.org/amigo/term/GO:0030424), [neuronal cell body](https://amigo.geneontology.org/amigo/term/GO:0043025), [perikaryon](https://amigo.geneontology.org/amigo/term/GO:0043204), [axonal growth cone](https://amigo.geneontology.org/amigo/term/GO:0044295), and [synapse](https://amigo.geneontology.org/amigo/term/GO:0045202).


The gene length is 7,803 base pairs (20.8% of all genes), the mature length is 959.5 base pairs, and the primary transcript length is 5,399.5 base pairs.


The gene OLFM1 (NCBI ID: 10439) has been mentioned in [8 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22OLFM1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 2000 and the middle 50% of publications occurring between 2015 and 2016.


The top 5 publications mentioning OLFM1, ranked by their relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows: [Noelin-1 is a secreted glycoprotein involved in generation of the neural crest.](https://pubmed.ncbi.nlm.nih.gov/10783240) (2000) (relative citation ratio: 1.84), [Noelin1 Affects Lateral Mobility of Synaptic AMPA Receptors.](https://pubmed.ncbi.nlm.nih.gov/30067977) (2018) (relative citation ratio: 1.01), [Olfactomedin 1 negatively regulates NF-ÎºB signalling and suppresses the growth and metastasis of colorectal cancer cells.](https://pubmed.ncbi.nlm.nih.gov/27555280) (2016) (relative citation ratio: 0.97), [MicroRNA-212 Regulates the Expression of Olfactomedin 1 and C-Terminal Binding Protein 1 in Human Endometrial Epithelial Cells to Enhance Spheroid Attachment In Vitro.](https://pubmed.ncbi.nlm.nih.gov/26377223) (2015) (relative citation ratio: 0.9), and [Olfactomedin-1 Has a V-shaped Disulfide-linked Tetrameric Structure.](https://pubmed.ncbi.nlm.nih.gov/25903135) (2015) (relative citation ratio: 0.59). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[OLFM1](https://www.proteinatlas.org/ENSG00000130558-OLFM1) is a developmental protein with evidence at the protein level. It is detected in many tissues and is localized to the nucleoplasm, nucleoli, and cytosol. OLFM1 is expressed in monocytes and neutrophils, particularly in the context of degranulation (Cluster 38). In tissue expression, it is associated with neuronal signaling in the brain (Cluster 61). Additionally, OLFM1 is found in neurons with mixed functions in the brain (Cluster 53) and is linked to neuronal signaling in bone cancer cell lines (Cluster 12) and single-cell neurons (Cluster 34).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) with 6 experiments, [MYC](https://www.ncbi.nlm.nih.gov/gene/4609) with 5 experiments, [MBL2](https://www.ncbi.nlm.nih.gov/gene/4153) with 5 experiments, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 5 experiments, and [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with a range of disease conditions, including lower respiratory tract disease, lung disease, respiratory system disease, disease of metabolism, acquired metabolic disease, nutrition disease, overnutrition, obesity, kidney disease, urinary system disease, iron metabolism disease, and mineral metabolism disease.



The gene is ubiquitously expressed in multiple tissue samples, including nerve, brain, and pituitary, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in various brain regions such as the Amygdala, Whole brain, Cingulate Cortex, Subthalamic Nucleus, Temporal Lobe, Medulla Oblongata, and Occipital Lobe.




The protein sequence analyzed has a GRAVY value of -0.461 (39.22nd percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -2.46 (41.4th percentile). The median structural flexibility is 0.999 (35.89th percentile), with a secondary structure composition of 29.28% helix (25.62nd percentile), 37.32% sheet (74.88th percentile), and 28.25% turn (49.53rd percentile). The instability index is 52.25 (64.4th percentile), and the isoelectric point is 6.48 (42.08th percentile). The protein has a length of 485 amino acids (56.47th percentile) and a molecular weight of 55342.01 Da (57.72nd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |