---
layout: post
title: PDLIM5
date: 2025-01-17 16:55 CST
description: PDLIM5 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10611) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10611  | PDLIM5 | ENSG00000163110 | 4q22.3 |



The gene is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [adherens junction](https://amigo.geneontology.org/amigo/term/GO:0005912), [postsynaptic density](https://amigo.geneontology.org/amigo/term/GO:0014069), [actin cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0015629), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [Z disc](https://amigo.geneontology.org/amigo/term/GO:0030018), [cell projection](https://amigo.geneontology.org/amigo/term/GO:0042995), and [presynapse](https://amigo.geneontology.org/amigo/term/GO:0098793). It is active in the [stress fiber](https://amigo.geneontology.org/amigo/term/GO:0001725) and [adherens junction](https://amigo.geneontology.org/amigo/term/GO:0005912). The gene enables [actin binding](https://amigo.geneontology.org/amigo/term/GO:0003779), [protein kinase C binding](https://amigo.geneontology.org/amigo/term/GO:0005080), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [actinin binding](https://amigo.geneontology.org/amigo/term/GO:0042805), [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872), [muscle alpha-actinin binding](https://amigo.geneontology.org/amigo/term/GO:0051371), and [cadherin binding involved in cell-cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0098641). It is involved in [heart development](https://amigo.geneontology.org/amigo/term/GO:0007507), [actin cytoskeleton organization](https://amigo.geneontology.org/amigo/term/GO:0030036), [regulation of synapse assembly](https://amigo.geneontology.org/amigo/term/GO:0051963), [regulation of dendritic spine morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0061001), [cell growth involved in cardiac muscle cell development](https://amigo.geneontology.org/amigo/term/GO:0061049), [muscle structure development](https://amigo.geneontology.org/amigo/term/GO:0061061), and [cell-cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0098609). Additionally, it is part of [filamentous actin](https://amigo.geneontology.org/amigo/term/GO:0031941).


The gene length is 106,830 base pairs (83.14% of all genes), the mature length is 5,163 base pairs, and the primary transcript length is 106,815 base pairs (89.37% of all genes).


The gene PDLIM5 (NCBI ID: 10611) has been mentioned in [332 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PDLIM5%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning this gene is 1960, and the middle 50% of publications occurred between 2003 and 2017.


The top 5 publications mentioning PDLIM5, ranked by their scientific influence, include "[Requirement for LIM homeobox gene Isl1 in motor neuron generation reveals a motor neuron-dependent step in interneuron differentiation.](https://pubmed.ncbi.nlm.nih.gov/8565076)" (1996) (relative citation ratio: 12.25), "[The LIM domain: from the cytoskeleton to the nucleus.](https://pubmed.ncbi.nlm.nih.gov/15520811)" (2004) (relative citation ratio: 10.13), "[LIM domains: multiple roles as adapters and functional modifiers in protein interactions.](https://pubmed.ncbi.nlm.nih.gov/9594664)" (1998) (relative citation ratio: 8.65), "[Aurora-A and an interacting activator, the LIM protein Ajuba, are required for mitotic commitment in human cells.](https://pubmed.ncbi.nlm.nih.gov/13678582)" (2003) (relative citation ratio: 8.43), and "[The LIM domain is a modular protein-binding interface.](https://pubmed.ncbi.nlm.nih.gov/7954790)" (1994) (relative citation ratio: 7.47). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PDLIM5](https://www.proteinatlas.org/ENSG00000163110-PDLIM5) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in the nucleoplasm, plasma membrane, and focal adhesion sites. The gene is highly expressed in basophils (Cluster 32), heart muscle (Cluster 12), immune cells in the brain (Cluster 8), connective tissue cells (Cluster 64), and cardiomyocytes (Cluster 68), where it is involved in transcription, cardiac muscle contraction, immune response, ECM organization, and muscle contraction, respectively.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 5 experiments, followed by [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169), [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885), and [MED12](https://www.ncbi.nlm.nih.gov/gene/9968), each with 4 experiments.



The input data from GWAS indicates associations with a wide range of disease conditions, including various types of cancer such as male reproductive organ cancer, prostate cancer, gastrointestinal system cancer, and pharynx cancer, including nasopharynx carcinoma. Additionally, the data highlights metabolic disorders like type 2 diabetes mellitus, obesity, and diseases related to carbohydrate and glucose metabolism. Cardiovascular conditions such as hypertension, artery disease, and vascular disease are also noted. Furthermore, the data includes nervous system diseases, specifically neurodegenerative diseases like amyotrophic lateral sclerosis, cognitive disorders, and substance-related disorders.


PDLIM5 has been associated with several diseases and traits through genome-wide association studies. Notably, it has been linked to [asparaginase-induced acute pancreatitis in acute lymphoblastic leukemia (onset time)](https://pubmed.ncbi.nlm.nih.gov/27114598), which maps closely to [Leukemia](https://meshb.nlm.nih.gov/record/ui?ui=D007938) using the BioWordVec method. Additionally, PDLIM5 is associated with [Coronary artery disease](https://pubmed.ncbi.nlm.nih.gov/29212778), directly matching the MeSH term [Coronary Artery Disease](https://meshb.nlm.nih.gov/record/ui?ui=D003324). Furthermore, it has been found to influence [Pulse pressure](https://pubmed.ncbi.nlm.nih.gov/33230300), which directly maps to the MeSH term [Blood Pressure](https://meshb.nlm.nih.gov/record/ui?ui=D001794).


The gene is ubiquitously expressed in multiple tissue samples, including heart, muscle, and blood vessel, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Cardiac Myocytes, Heart, Smooth Muscle, Prostate, Placenta, Skeletal Muscle, Tongue, and Uterus.




The analyzed protein sequence has a GRAVY value of -0.403 (45.8th percentile), indicating a hydrophilic nature. It exhibits a charge of 8.93 at pH 7.0 (79.23rd percentile) and a median structural flexibility of 1.007 (71.96th percentile). The protein's secondary structure is predicted to be 27.68% helix (15.26th percentile), 29.53% sheet (26.01st percentile), and 33.22% turn (82.98th percentile). The instability index is 48.93 (53.61st percentile), with an isoelectric point of 8.55 (67.61st percentile). The protein has a length of 596 amino acids (68.29th percentile) and a molecular weight of 63944.41 Da (66.26th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |