---
layout: post
title: RAB5A
date: 2025-01-17 16:55 CST
description: RAB5A description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5868) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5868  | RAB5A | ENSG00000144566 | 3p24.3 |



The gene is located in various cellular components, including the [ruffle](https://amigo.geneontology.org/amigo/term/GO:0001726), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [endosome](https://amigo.geneontology.org/amigo/term/GO:0005768), [early endosome](https://amigo.geneontology.org/amigo/term/GO:0005769), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [synaptic vesicle](https://amigo.geneontology.org/amigo/term/GO:0008021), [endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0010008), [actin cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0015629), [axon](https://amigo.geneontology.org/amigo/term/GO:0030424), [dendrite](https://amigo.geneontology.org/amigo/term/GO:0030425), [clathrin-coated endocytic vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030669), [phagocytic vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030670), [early endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0031901), [early phagosome](https://amigo.geneontology.org/amigo/term/GO:0032009), [somatodendritic compartment](https://amigo.geneontology.org/amigo/term/GO:0036477), [melanosome](https://amigo.geneontology.org/amigo/term/GO:0042470), [neuronal cell body](https://amigo.geneontology.org/amigo/term/GO:0043025), [terminal bouton](https://amigo.geneontology.org/amigo/term/GO:0043195), [intracellular membrane-bounded organelle](https://amigo.geneontology.org/amigo/term/GO:0043231), [axon terminus](https://amigo.geneontology.org/amigo/term/GO:0043679), [membrane raft](https://amigo.geneontology.org/amigo/term/GO:0045121), [phagocytic vesicle](https://amigo.geneontology.org/amigo/term/GO:0045335), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). It enables [GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0003924), [G protein activity](https://amigo.geneontology.org/amigo/term/GO:0003925), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [GTP binding](https://amigo.geneontology.org/amigo/term/GO:0005525), and [GDP binding](https://amigo.geneontology.org/amigo/term/GO:0019003). The gene is involved in various biological processes, including [intracellular protein transport](https://amigo.geneontology.org/amigo/term/GO:0006886), [endocytosis](https://amigo.geneontology.org/amigo/term/GO:0006897), [phagocytosis](https://amigo.geneontology.org/amigo/term/GO:0006909), [receptor internalization](https://amigo.geneontology.org/amigo/term/GO:0031623), [synaptic vesicle recycling](https://amigo.geneontology.org/amigo/term/GO:0036465), [modulation by host of viral process](https://amigo.geneontology.org/amigo/term/GO:0044788), [early endosome to late endosome transport](https://amigo.geneontology.org/amigo/term/GO:0045022), [positive regulation of exocytosis](https://amigo.geneontology.org/amigo/term/GO:0045921), [regulation of long-term neuronal synaptic plasticity](https://amigo.geneontology.org/amigo/term/GO:0048169), [regulation of endosome size](https://amigo.geneontology.org/amigo/term/GO:0051036), [regulation of filopodium assembly](https://amigo.geneontology.org/amigo/term/GO:0051489), [canonical Wnt signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0060070), [amyloid-beta clearance by transcytosis](https://amigo.geneontology.org/amigo/term/GO:0150093), [regulation of synaptic vesicle exocytosis](https://amigo.geneontology.org/amigo/term/GO:2000300), and [regulation of autophagosome assembly](https://amigo.geneontology.org/amigo/term/GO:2000785). Additionally, the gene is active in the [endomembrane system](https://amigo.geneontology.org/amigo/term/GO:0012505), [axon](https://amigo.geneontology.org/amigo/term/GO:0030424), [dendrite](https://amigo.geneontology.org/amigo/term/GO:0030425), and [postsynaptic early endosome](https://amigo.geneontology.org/amigo/term/GO:0098842).


The gene length is 15,802 base pairs (36.24% of all genes), the mature length is 3,763 base pairs, and the primary transcript length is 14,935 base pairs.


The gene RAB5A, with NCBI ID 5868, has been mentioned in [377 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RAB5A%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication mentioning this gene is 1991, and the middle 50% of publications occurred between 2004 and 2017.


The top five publications mentioning RAB5A, ranked by their scientific influence, include "[rab5 controls early endosome fusion in vitro.](https://pubmed.ncbi.nlm.nih.gov/1900457)" (1991) (relative citation ratio: 18.05), "[EEA1 links PI(3)K function to Rab5 regulation of endosome fusion.](https://pubmed.ncbi.nlm.nih.gov/9697774)" (1998) (relative citation ratio: 15.61), "[The Rab5 effector EEA1 is a core component of endosome docking.](https://pubmed.ncbi.nlm.nih.gov/10050856)" (1999) (relative citation ratio: 11.23), "[Phosphatidylinositol-3-OH kinases are Rab5 effectors.](https://pubmed.ncbi.nlm.nih.gov/10559924)" (1999) (relative citation ratio: 8.43), and "[A novel Rab5 GDP/GTP exchange factor complexed to Rabaptin-5 links nucleotide exchange to effector recruitment and function.](https://pubmed.ncbi.nlm.nih.gov/9323142)" (1997) (relative citation ratio: 8.02). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RAB5A](https://www.proteinatlas.org/ENSG00000144566-RAB5A) is a gene that functions as a hydrolase, with evidence supporting its activity at the protein level. Its RNA is detected in all tissues, and the protein is localized in the nucleoplasm and vesicles. RAB5A is notably expressed in monocytes and neutrophils, playing a role in the innate immune response. Its expression is also observed in various tissues and cell lines, indicating involvement in mixed functions and basic cellular processes.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [POU5F1](https://www.ncbi.nlm.nih.gov/gene/5460) and [PGR](https://www.ncbi.nlm.nih.gov/gene/5241), each with 6 experiments. Additionally, [STAT5B](https://www.ncbi.nlm.nih.gov/gene/6777), [E2F1](https://www.ncbi.nlm.nih.gov/gene/1869), and [BRD3](https://www.ncbi.nlm.nih.gov/gene/8019) were each found to be regulating in 5 experiments.



The GWAS data indicates associations with various disease conditions, including metabolic disorders such as diabetes mellitus and type 2 diabetes mellitus, as well as musculoskeletal system diseases like arthritis, bone inflammation disease, and osteoarthritis. Additionally, the data highlights connections to broader categories such as disease of metabolism, acquired metabolic disease, carbohydrate metabolism disease, glucose metabolism disease, disease, disease of anatomical entity, bone disease, and connective tissue disease.



The gene is expressed in various tissues, including the amygdala, spinal cord, occipital lobe, placenta, and colon. Additionally, it is expressed in the skin.


The proteins are involved in factors involved in megakaryocyte development and platelet production, as well as hemostasis.


The mouse ortholog gene 271457, which corresponds to the human ortholog gene 5868, is associated with abnormal sleep behavior. This phenotype was observed in a sex-specific manner.


The protein sequence analyzed has a GRAVY value of -0.428 (42.94th percentile), indicating a hydrophilic nature. It exhibits a charge of 1.664 at pH 7.0 (56.18th percentile) and a median structural flexibility of 1.008 (75.96th percentile). The protein's secondary structure is predicted to be 32.56% helix (55.57th percentile), 31.63% sheet (37.41st percentile), and 28.84% turn (54.65th percentile). The instability index is 28.93 (3.24th percentile), with an isoelectric point of 8.32 (63.18th percentile). The protein has a length of 215 amino acids (16.75th percentile) and a molecular weight of 23658.41 Da (16.24th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |