---
layout: post
title: MCM6
date: 2024-12-08 23:57 CST
description: MCM6 description
tags: [cooccuring-genes]
categories: longest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4175) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4175  | MCM6 | ENSG00000076003 | 2q21.3 |



The gene length is 791,805 base pairs (99.35th percentile of all genes), the mature length is 16,574 base pairs, and the primary transcript length is 791,805 base pairs (99.77th percentile of all genes).


The gene MCM6, with NCBI ID 4175, has been the subject of [39 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22MCM6%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ). The earliest year of publication is 1998, and the middle 50% of publications occurred between 2010 and 2019.


The top five publications mentioning MCM6, ranked by their scientific influence, are as follows: ["MCM family in HCC: MCM6 indicates adverse tumor features and poor outcomes and promotes S/G2 cell cycle progression."](https://pubmed.ncbi.nlm.nih.gov/29463213) (2018) (relative citation ratio: 3.83), ["MCM2, MCM4, and MCM6 in Breast Cancer: Clinical Utility in Diagnosis and Prognosis."](https://pubmed.ncbi.nlm.nih.gov/31476594) (2019) (relative citation ratio: 3.22), ["The DNA replication regulator MCM6: An emerging cancer biomarker and target."](https://pubmed.ncbi.nlm.nih.gov/33609557) (2021) (relative citation ratio: 2.72), ["Cell growth-regulated expression of mammalian MCM5 and MCM6 genes mediated by the transcription factor E2F."](https://pubmed.ncbi.nlm.nih.gov/10327050) (1999) (relative citation ratio: 1.87), and ["Methyl(R217)HuR and MCM6 are inversely correlated and are prognostic markers in non small cell lung carcinoma."](https://pubmed.ncbi.nlm.nih.gov/26013954) (2015) (relative citation ratio: 1.7). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 5 experiments, [PGR](https://www.ncbi.nlm.nih.gov/gene/5241) with 3 experiments, [SIN3A](https://www.ncbi.nlm.nih.gov/gene/25942) with 3 experiments, [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) with 3 experiments, and [RELA](https://www.ncbi.nlm.nih.gov/gene/5970) with 3 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.











The analyzed protein sequence has a GRAVY value of -0.377 (48.92 percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -27.79 (6.83 percentile), and the median structural flexibility is 1.003 (53.69 percentile). The protein has a predicted secondary structure with 32.76% helix (57.41 percentile), 35.20% sheet (60.64 percentile), and 25.70% turn (26.75 percentile) affinities. The instability index is 46.20 (43.69 percentile), and the isoelectric point is 5.29 (13.26 percentile). The protein is 821 amino acids long (81.52 percentile) with a molecular weight of 92888.18 Da (81.96 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |