---
layout: post
title: SUMO3
date: 2025-01-17 16:55 CST
description: SUMO3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6612) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6612  | SUMO3 | ENSG00000184900 | 21q22.3 |



The gene is located in the [kinetochore](https://amigo.geneontology.org/amigo/term/GO:0000776), [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), and [PML body](https://amigo.geneontology.org/amigo/term/GO:0016605). It enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [protein tag activity](https://amigo.geneontology.org/amigo/term/GO:0031386), and [ubiquitin-like protein ligase binding](https://amigo.geneontology.org/amigo/term/GO:0044389). Additionally, it is involved in [protein sumoylation](https://amigo.geneontology.org/amigo/term/GO:0016925), [negative regulation of DNA binding](https://amigo.geneontology.org/amigo/term/GO:0043392), and [regulation of protein localization to nucleus](https://amigo.geneontology.org/amigo/term/GO:1900180).


The gene length is 33,260 base pairs (55.45th percentile), the mature length is 6,911 base pairs (96.31st percentile), and the primary transcript length is 33,260 base pairs (65.34th percentile).


SUMO3 has been mentioned in [32 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SUMO3%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest mention in 2001 and the middle 50% of publications occurring between 2008 and 2017. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning SUMO3, ranked by their scientific influence, include "[Polymeric chains of SUMO-2 and SUMO-3 are conjugated to protein substrates by SAE1/SAE2 and Ubc9.](https://pubmed.ncbi.nlm.nih.gov/11451954)" (2001) (relative citation ratio: 10.64), "[Characterization of a family of nucleolar SUMO-specific proteases with preference for SUMO-2 or SUMO-3.](https://pubmed.ncbi.nlm.nih.gov/16608850)" (2006) (relative citation ratio: 3.46), "[A synergy control motif within the attenuator domain of CCAAT/enhancer-binding protein alpha inhibits transcriptional synergy through its PIASy-enhanced modification by SUMO-1 or SUMO-3.](https://pubmed.ncbi.nlm.nih.gov/12511558)" (2003) (relative citation ratio: 2.05), "[SENP3-mediated de-conjugation of SUMO2/3 from promyelocytic leukemia is correlated with accelerated cell proliferation under mild oxidative stress.](https://pubmed.ncbi.nlm.nih.gov/20181954)" (2010) (relative citation ratio: 1.9), and "[The poly-SUMO2/3 protease SENP6 enables assembly of the constitutive centromere-associated network by group deSUMOylation.](https://pubmed.ncbi.nlm.nih.gov/31485003)" (2019) (relative citation ratio: 1.87). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[SUMO3](https://www.proteinatlas.org/ENSG00000184900-SUMO3) is a gene with evidence at the protein level. Its RNA is detected in all tissues, with notable expression in the nucleoplasm, nuclear bodies, calyx, connecting piece, and flagellar centriole. The gene is part of Cluster 8 in blood expression, which is associated with plasmacytoid dendritic cells but has an unknown function. In tissue expression, it belongs to Cluster 5, which is linked to brain and testis with a role in signal transduction. Additionally, SUMO3 is part of Cluster 14 in brain expression, associated with neurons and synaptic function, and Cluster 4 in cell line expression, which is non-specific but related to transcription. In single-cell expression, it is part of Cluster 14, associated with Langerhans cells and the innate immune response.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [ZBTB7A](https://www.ncbi.nlm.nih.gov/gene/51341) with 6 experiments, [BRD9](https://www.ncbi.nlm.nih.gov/gene/65980) with 5 experiments, [NFYA](https://www.ncbi.nlm.nih.gov/gene/4800) with 5 experiments, [PGR](https://www.ncbi.nlm.nih.gov/gene/5241) with 5 experiments, and [STAT5B](https://www.ncbi.nlm.nih.gov/gene/6777) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with various metabolic and nutritional conditions, including diseases of metabolism, acquired metabolic diseases, nutrition diseases, overnutrition, and obesity.



The gene is expressed in various tissues and cell types, including brain, B lymphoblasts, CD34+ cells, BDCA4+ dendritic cells, CD105+ endothelial cells, and colorectal adenocarcinoma.


The input data highlights several key processes involving proteins, including post-translational protein modification and metabolism of proteins. Specifically, it focuses on the processing and activation of SUMO, a process known as SUMOylation. This involves the proteolytic processing of SUMO, its conjugation to E1 (UBA2:SAE1), and subsequent transfer from E1 to E2 (UBE2I, UBC9).



The analyzed protein sequence has a GRAVY value of -0.8281553398058255 (10.38th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -4.221114799973902 (35.75th percentile), and the median structural flexibility is 1.0137261904761903 (89.53rd percentile). The protein has a helix affinity of 0.3106796116504854 (41.75th percentile), a sheet affinity of 0.2815533980582524 (19.08th percentile), and a turn affinity of 0.2912621359223301 (56.97th percentile). The instability index is 44.36601941747572 (37.06th percentile), with an isoelectric point of 5.3206071853637695 (13.97th percentile). The protein has a length of 103 amino acids (3.02nd percentile) and a molecular weight of 11636.91369999999 Da (3.17th percentile). These values were calculated using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |