---
layout: post
title: ARF3
date: 2025-01-17 16:55 CST
description: ARF3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/377) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 377  | ARF3 | ENSG00000134287 | 12q13.12 |



The gene is located in the [Golgi membrane](https://amigo.geneontology.org/amigo/term/GO:0000139) and the [perinuclear region of cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0048471), and is also found in the [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). It is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737) and the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). The gene enables [GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0003924), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [GTP binding](https://amigo.geneontology.org/amigo/term/GO:0005525). It is involved in [intracellular protein transport](https://amigo.geneontology.org/amigo/term/GO:0006886), [retrograde vesicle-mediated transport, Golgi to endoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0006890), and [vesicle-mediated transport](https://amigo.geneontology.org/amigo/term/GO:0016192).


The gene length is 3,116 base pairs (8.63% of all genes), the mature length is 1,458 base pairs (36.7% of all genes), and the primary transcript length is 3,055 base pairs (11.06% of all genes).


ARF3 (Gene ID: 377) has been mentioned in [18 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ARF3%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning ARF3 was in 1991, and the middle 50% of publications occurred between 2002 and 2016.


The top 5 publications mentioning ARF3, ranked by their relative citation ratio (RCR), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows: [The THO Complex Non-Cell-Autonomously Represses Female Germline Specification through the TAS3-ARF3 Module.](https://pubmed.ncbi.nlm.nih.gov/28552357) (2017) (relative citation ratio: 2.04), [ARF1 and ARF3 are required for the integrity of recycling endosomes and the recycling pathway.](https://pubmed.ncbi.nlm.nih.gov/22971977) (2012) (relative citation ratio: 1.15), [De novo ARF3 variants cause neurodevelopmental disorder with brain abnormality.](https://pubmed.ncbi.nlm.nih.gov/34346499) (2021) (relative citation ratio: 1.05), [BIG1 controls macrophage pro-inflammatory responses through ARF3-mediated PI(4,5)P2 synthesis.](https://pubmed.ncbi.nlm.nih.gov/32415087) (2020) (relative citation ratio: 0.98), and [Arf3 is activated uniquely at the trans-Golgi network by brefeldin A-inhibited guanine nucleotide exchange factors.](https://pubmed.ncbi.nlm.nih.gov/20357002) (2010) (relative citation ratio: 0.93). The citation count is sourced from [iCite](https://icite.od.nih.gov).


[ARF3](https://www.proteinatlas.org/ENSG00000134287-ARF3) is a gene with evidence at the protein level. Its RNA is detected in all tissues, and it is notably expressed in the brain, where it is involved in neuronal signaling. In blood, ARF3 is associated with the innate immune response. Additionally, it is expressed in enterocytes, where it plays a role in absorption. The gene is part of various expression clusters, including those related to transcription in cell lines and mixed function in brain neurons.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [PPARG](https://www.ncbi.nlm.nih.gov/gene/5468) with 6 experiments, [MYC](https://www.ncbi.nlm.nih.gov/gene/4609) with 5 experiments, [TP53](https://www.ncbi.nlm.nih.gov/gene/7157) with 5 experiments, [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 5 experiments, and [SIN3A](https://www.ncbi.nlm.nih.gov/gene/25942) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.





The gene is ubiquitously expressed in multiple tissue samples, including the brain and adrenal gland. Specifically, it is detected in various regions of the brain such as the Prefrontal Cortex, Amygdala, Whole brain, Caudate nucleus, Cingulate Cortex, Subthalamic Nucleus, Temporal Lobe, and Cerebellum.


The input data highlights various pathways involved in metabolism, specifically focusing on lipid and lipoprotein metabolism. Within this broader category, phospholipid metabolism is emphasized, particularly the metabolism of phosphatidylinositol (PI). Notably, the synthesis of phosphatidylinositol phosphates (PIPs) at the Golgi membrane is a key process mentioned.


The mouse ortholog gene 11842 and its human ortholog gene 377 are associated with abnormal retina morphology, a phenotype observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.246 (64.26th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -0.220 (49.51st percentile). The median structural flexibility is 0.997 (28.41st percentile), with a notable affinity for helix (36.5%, 82.98th percentile) and sheet (39.8%, 85.61st percentile) structures. The instability index is 28.28 (2.82nd percentile), and the isoelectric point is 6.84 (48.49th percentile). The protein has a length of 181 amino acids (11.75th percentile) and a molecular weight of 20600.52 Da (12.12th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |