---
layout: post
title: CLTB
date: 2025-01-17 16:55 CST
description: CLTB description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1212) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1212  | CLTB | ENSG00000175416 | 5q35.2 |



The gene enables [structural molecule activity](https://amigo.geneontology.org/amigo/term/GO:0005198) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), including [clathrin heavy chain binding](https://amigo.geneontology.org/amigo/term/GO:0032050) and [peptide binding](https://amigo.geneontology.org/amigo/term/GO:0042277). It is located in the [trans-Golgi network](https://amigo.geneontology.org/amigo/term/GO:0005802), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [intracellular membrane-bounded organelle](https://amigo.geneontology.org/amigo/term/GO:0043231), [clathrin-coated endocytic vesicle](https://amigo.geneontology.org/amigo/term/GO:0045334), [ciliary membrane](https://amigo.geneontology.org/amigo/term/GO:0060170), and [presynaptic endocytic zone membrane](https://amigo.geneontology.org/amigo/term/GO:0098835). It is also part of the [clathrin coat](https://amigo.geneontology.org/amigo/term/GO:0030118), [clathrin vesicle coat](https://amigo.geneontology.org/amigo/term/GO:0030125), [clathrin coat of trans-Golgi network vesicle](https://amigo.geneontology.org/amigo/term/GO:0030130), and [clathrin coat of coated pit](https://amigo.geneontology.org/amigo/term/GO:0030132). The gene is active in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [synaptic vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030672), and [postsynaptic endocytic zone cytoplasmic component](https://amigo.geneontology.org/amigo/term/GO:0099631). Additionally, it is involved in [intracellular protein transport](https://amigo.geneontology.org/amigo/term/GO:0006886) and [clathrin-dependent endocytosis](https://amigo.geneontology.org/amigo/term/GO:0072583).


The gene length is 83,072 base pairs (78.19% of all genes), the mature length is 7,559.5 base pairs, and the primary transcript length is 83,062 base pairs.


The gene CLTB (NCBI ID: 1212) has been mentioned in [9 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CLTB%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning CLTB was in 1988, and the middle 50% of publications occurred between 1994 and 2006.


The top 5 publications mentioning CLTB, ranked by their scientific influence as measured by the [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541), include "[Licochalcone B Arrests Cell Cycle Progression and Induces Apoptosis in Human Breast Cancer MCF-7 Cells.](https://pubmed.ncbi.nlm.nih.gov/27719653)" (2016) (RCR: 1.25), "[Predominance of clathrin light chain LCb correlates with the presence of a regulated secretory pathway.](https://pubmed.ncbi.nlm.nih.gov/2211818)" (1990) (RCR: 1.11), "[A concise synthesis of tetrahydroxy-LCB, alpha-galactosyl ceramide, and 1,4-dideoxy-1,4-imino-L-ribitol via D-allosamines as key building blocks.](https://pubmed.ncbi.nlm.nih.gov/16438541)" (2006) (RCR: 0.85), "[Identification of the phosphorylation sites of clathrin light chain LCb.](https://pubmed.ncbi.nlm.nih.gov/3128543)" (1988) (RCR: 0.83), and "[The LCB Data Warehouse.](https://pubmed.ncbi.nlm.nih.gov/16455749)" (2006) (RCR: 0.64). The RCR values measure the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CLTB](https://www.proteinatlas.org/ENSG00000175416-CLTB) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in vesicles, the plasma membrane, and the cytosol. The gene is expressed in eosinophils with unknown function (Cluster 45) and is highly expressed in the testis (Cluster 21). In the brain, CLTB is associated with neurons and synapses, suggesting a role in synaptic function (Cluster 52). Additionally, it is expressed in squamous epithelial cells, contributing to keratinization (Cluster 53). The gene's expression in single cells is non-specific with mixed function (Cluster 45).


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025) with 6 experiments, [NFYA](https://www.ncbi.nlm.nih.gov/gene/4800) with 6 experiments, [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113) with 6 experiments, [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599) with 5 experiments, and [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with several disease conditions, including artery disease, vascular disease, cardiovascular system disease, hypertension, and broader categories such as disease and disease of anatomical entity.



The gene is expressed in various tissues, including the esophagus, vagina, and skin, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the prefrontal cortex, heart, whole brain, temporal lobe, bronchial epithelial cells, and placenta.


The proteins are involved in various pathways, including the PCP/CE pathway, beta-catenin independent WNT signaling, signaling by Wnt, axon guidance, and developmental biology. Additionally, they participate in EPH-Ephrin signaling, EPH-ephrin mediated repulsion of cells, and signal transduction. Notably, they are implicated in WNT5A-dependent internalization of FZD2, FZD5, and ROR2, as well as WNT5A-dependent internalization of FZD4. Furthermore, these proteins play roles in membrane trafficking, formation of annular gap junctions, gap junction degradation, and gap junction trafficking and regulation.



The analyzed protein sequence has a GRAVY value of -0.91 (7th percentile), indicating a hydrophilic nature. It exhibits a charge of -17.35 at pH 7.0 (13th percentile) and a median structural flexibility of 1.02 (98th percentile). The protein's secondary structure is predicted to be 38.43% helix (90th percentile), 22.27% sheet (4th percentile), and 30.13% turn (65th percentile). The instability index is 57.86 (79th percentile), with an isoelectric point of 4.57 (2nd percentile). The protein has a length of 229 amino acids (19th percentile) and a molecular weight of 25190.2 Da (18th percentile). These calculations were performed using BioPython's sequence analysis tools, which can be found [here](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |