---
layout: post
title: DUT
date: 2025-01-17 16:55 CST
description: DUT description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1854) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1854  | DUT | ENSG00000128951 | 15q21.1 |



The gene enables [magnesium ion binding](https://amigo.geneontology.org/amigo/term/GO:0000287) and [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), and is involved in [liver development](https://amigo.geneontology.org/amigo/term/GO:0001889). It also enables [dUTP diphosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0004170) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is active in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654) and [mitochondrion](https://amigo.geneontology.org/amigo/term/GO:0005739). Additionally, it is involved in the [nucleobase-containing compound metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006139), [dUMP biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006226), [dTMP biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006231), and [DNA replication](https://amigo.geneontology.org/amigo/term/GO:0006260). The gene also enables [signaling receptor inhibitor activity](https://amigo.geneontology.org/amigo/term/GO:0030547) and [pyrimidine deoxyribonucleotide binding](https://amigo.geneontology.org/amigo/term/GO:0032556), and is involved in the [response to organic cyclic compound](https://amigo.geneontology.org/amigo/term/GO:0014070). Furthermore, it enables [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802) and [peroxisome proliferator activated receptor binding](https://amigo.geneontology.org/amigo/term/GO:0042975), and is involved in the [regulation of protein-containing complex assembly](https://amigo.geneontology.org/amigo/term/GO:0043254) and [dUTP catabolic process](https://amigo.geneontology.org/amigo/term/GO:0046081). Lastly, the gene is located in the [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062).


The gene length is 17,271 base pairs (38.38% of all genes), the mature length is 2,630.5 base pairs (66.43% of all genes), and the primary transcript length is 15,563.5 base pairs (45.04% of all genes).


The gene DUT (NCBI ID: 1854) has been mentioned in [26 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22DUT%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning DUT was in 1970, and the middle 50% of publications occurred between 1999 and 2018.


The top 5 publications mentioning DUT, ranked by their relative citation ratio (RCR), include "[A novel in-vitro method for assessing contact lens surface dewetting: Non-invasive keratograph dry-up time (NIK-DUT).](https://pubmed.ncbi.nlm.nih.gov/28847466)" (2017) (relative citation ratio: 1.52), "[CKS2 modulates cell-cycle progression of tongue squamous cell carcinoma cells partly via modulating the cellular distribution of DUTPase.](https://pubmed.ncbi.nlm.nih.gov/33107644)" (2021) (relative citation ratio: 1.47), "[Kaposi's sarcoma-associated herpesvirus ORF54/dUTPase downregulates a ligand for the NK activating receptor NKp44.](https://pubmed.ncbi.nlm.nih.gov/22674989)" (2012) (relative citation ratio: 1.18), "[High-Pressure in Situ <sup>129</sup>Xe NMR Spectroscopy: Insights into Switching Mechanisms of Flexible Metal-Organic Frameworks Isoreticular to DUT-49.](https://pubmed.ncbi.nlm.nih.gov/35601358)" (2019) (relative citation ratio: 0.98), and "[A randomized, phase 2 study of deoxyuridine triphosphatase inhibitor, TAS-114, in combination with S-1 versus S-1 alone in patients with advanced non-small-cell lung cancer.](https://pubmed.ncbi.nlm.nih.gov/32246224)" (2020) (relative citation ratio: 0.78). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[DUT](https://www.proteinatlas.org/ENSG00000128951-DUT) is a hydrolase implicated in diabetes mellitus, with evidence at the protein level. It is detected in all tissues and primarily localized in the nucleoplasm. The gene is associated with various expression clusters, including Cluster 5 for blood expression related to cell proliferation, Cluster 40 for tissue expression related to transcription, Cluster 28 for brain expression in oligodendrocytes with mixed function, Cluster 58 for cell line expression related to nuclear processes, and Cluster 52 for single cell expression related to cell proliferation.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861), [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), and [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025), each with 6 experiments. Additionally, [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) and [NIPBL](https://www.ncbi.nlm.nih.gov/gene/25836) were found to be regulating in 5 experiments each.



The GWAS data indicates associations with various disease conditions, including general diseases, diseases of anatomical entities, integumentary system diseases, and skin diseases.



The gene is expressed in various tissues and cell types, including pituitary, 721_B_lymphoblasts, CD34+ cells, Leukemia promyelocytic HL-60 cells, CD105+ endothelial cells, CD71+ early erythroid cells, and Leukemia lymphoblastic MOLT-4 cells.


The input data highlights several pathways involved in metabolism, specifically focusing on nucleotide metabolism. Key processes include pyrimidine biosynthesis and pyrimidine metabolism.



The analyzed protein sequence has a GRAVY value of -0.304 (57.5th percentile), indicating a hydrophilic nature. It exhibits a charge of 8.83 at pH 7.0 (78.91st percentile) and a median structural flexibility of 1.004 (61.24th percentile). The protein's secondary structure is predicted to be 31.75% helix (48.15th percentile), 27.38% sheet (15.92nd percentile), and 32.94% turn (81.64th percentile). The instability index is 34.52 (9.24th percentile), with an isoelectric point of 9.46 (90.26th percentile). The protein has a length of 252 amino acids (21.94th percentile) and a molecular weight of 26562.94 Da (20.11th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |