---
layout: post
title: DDX42
date: 2025-01-17 16:55 CST
description: DDX42 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/11325) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 11325  | DDX42 | ENSG00000198231 | 17q23.3 |



The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and [RNA helicase activity](https://amigo.geneontology.org/amigo/term/GO:0003724), and is involved in [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524). It is active in and located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), and is also found in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [Cajal body](https://amigo.geneontology.org/amigo/term/GO:0015030), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), and [nuclear speck](https://amigo.geneontology.org/amigo/term/GO:0016607). The gene is involved in [protein localization](https://amigo.geneontology.org/amigo/term/GO:0008104) and the [regulation of apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0042981). Additionally, it is part of the [U2-type prespliceosome](https://amigo.geneontology.org/amigo/term/GO:0071004) and is involved in its assembly. The gene also enables [ATP hydrolysis activity](https://amigo.geneontology.org/amigo/term/GO:0016887).


The gene length is 88,303 base pairs (79.46% of all genes), the mature length is 2,220 base pairs, and the primary transcript length is 11,837 base pairs.


DDX42 has been mentioned in [4 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22DDX42%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 2008, and the middle 50% of publications range from 2008 to 2019.


The top publications mentioning DDX42 include "[Cancer-associated mutations in SF3B1 disrupt the interaction between SF3B1 and DDX42.](https://pubmed.ncbi.nlm.nih.gov/35652295)" (2022) (relative citation ratio: 1.11), "[Interferon antagonist function of Japanese encephalitis virus NS4A and its interaction with DEAD-box RNA helicase DDX42.](https://pubmed.ncbi.nlm.nih.gov/18588927)" (2008) (relative citation ratio: 0.98), "[The DEAD box protein Ddx42p modulates the function of ASPP2, a stimulator of apoptosis.](https://pubmed.ncbi.nlm.nih.gov/19377511)" (2009) (relative citation ratio: 0.4), and "[The ATP-dependent RNA helicase, DDX42 interacts with paxillin and regulates apoptosis and polarization of Ba/F3 cells.](https://pubmed.ncbi.nlm.nih.gov/30834153)" (2019) (relative citation ratio: 0.22). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[DDX42](https://www.proteinatlas.org/ENSG00000198231-DDX42) is a gene with molecular functions including helicase, hydrolase, and RNA-binding. Evidence for its function is available at the protein level. The gene is detected in all RNA tissue distributions and is primarily localized to nuclear speckles. In terms of expression, DDX42 is found in Cluster 45 of blood expression, which is associated with eosinophils but has an unknown function. It is also part of Cluster 19 in tissue expression, indicating involvement in basic cellular processes. In the brain, it belongs to Cluster 44, suggesting a mixed function. Additionally, DDX42 is expressed in Cluster 27 of cell line expression, which is related to breast cancer but with an unknown function. In single-cell expression, it is part of Cluster 39, again indicating involvement in basic cellular processes.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [KLF4](https://www.ncbi.nlm.nih.gov/gene/9314), [TAL1](https://www.ncbi.nlm.nih.gov/gene/6886), [POU5F1](https://www.ncbi.nlm.nih.gov/gene/5460), [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885), and [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), each shown to be regulating in 5 experiments.



The GWAS data indicates associations with several disease conditions, including artery disease, vascular disease, cardiovascular system disease, hypertension, and broader categories such as disease and disease of anatomical entity.



The gene is expressed in various tissues, including the ovary, CD34+ cells, appendix, superior cervical ganglion, trigeminal ganglion, skeletal muscle, and atrioventricular node.



The mouse ortholog gene 72047 and its human ortholog gene 11325 are associated with the phenotype of preweaning lethality with complete penetrance, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.829 (10.33rd percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -4.758 (34.4th percentile). The median structural flexibility is 1.014 (89.31st percentile), with a predicted secondary structure composition of 29.64% helix (28.84th percentile), 25.91% sheet (11.02nd percentile), and 36.14% turn (92.07th percentile). The instability index is 44.359 (37.04th percentile), and the isoelectric point is 6.535 (43.35th percentile). The protein is 938 amino acids long (85.81st percentile) with a molecular weight of 102973.94 Da (85.33rd percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |