---
layout: post
title: PDE4B
date: 2025-01-17 16:55 CST
description: PDE4B description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5142) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5142  | PDE4B | ENSG00000184588 | 1p31.3 |



The gene is part of the [gamma-tubulin complex](https://amigo.geneontology.org/amigo/term/GO:0000930) and the [voltage-gated calcium channel complex](https://amigo.geneontology.org/amigo/term/GO:0005891). It is located in the [centrosome](https://amigo.geneontology.org/amigo/term/GO:0005813), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [synaptic vesicle](https://amigo.geneontology.org/amigo/term/GO:0008021), [postsynaptic density](https://amigo.geneontology.org/amigo/term/GO:0014069), [Z disc](https://amigo.geneontology.org/amigo/term/GO:0030018), [dendritic spine](https://amigo.geneontology.org/amigo/term/GO:0043197), and [excitatory synapse](https://amigo.geneontology.org/amigo/term/GO:0060076). It is active in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and [perinuclear region of cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0048471). The gene enables [3',5'-cyclic-AMP phosphodiesterase activity](https://amigo.geneontology.org/amigo/term/GO:0004115), [calcium channel regulator activity](https://amigo.geneontology.org/amigo/term/GO:0005246), [cAMP binding](https://amigo.geneontology.org/amigo/term/GO:0030552), [gamma-tubulin binding](https://amigo.geneontology.org/amigo/term/GO:0043015), [transmembrane transporter binding](https://amigo.geneontology.org/amigo/term/GO:0044325), [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872), and [3',5'-cyclic-GMP phosphodiesterase activity](https://amigo.geneontology.org/amigo/term/GO:0047555). It is involved in various processes including [neutrophil homeostasis](https://amigo.geneontology.org/amigo/term/GO:0001780), [cAMP catabolic process](https://amigo.geneontology.org/amigo/term/GO:0006198), [cAMP-mediated signaling](https://amigo.geneontology.org/amigo/term/GO:0019933), [neutrophil chemotaxis](https://amigo.geneontology.org/amigo/term/GO:0030593), [positive regulation of type II interferon production](https://amigo.geneontology.org/amigo/term/GO:0032729), [positive regulation of interleukin-2 production](https://amigo.geneontology.org/amigo/term/GO:0032743), [T cell receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0050852), [leukocyte migration](https://amigo.geneontology.org/amigo/term/GO:0050900), [cellular response to lipopolysaccharide](https://amigo.geneontology.org/amigo/term/GO:0071222), [cellular response to xenobiotic stimulus](https://amigo.geneontology.org/amigo/term/GO:0071466), [cellular response to epinephrine stimulus](https://amigo.geneontology.org/amigo/term/GO:0071872), [negative regulation of adenylate cyclase-activating adrenergic receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0071878), [regulation of cardiac muscle cell contraction](https://amigo.geneontology.org/amigo/term/GO:0086004), [negative regulation of relaxation of cardiac muscle](https://amigo.geneontology.org/amigo/term/GO:1901898), and [regulation of calcium ion transmembrane transport via high voltage-gated calcium channel](https://amigo.geneontology.org/amigo/term/GO:1902514).


The gene length is 38,045 base pairs (59.07% of all genes), the mature length is 4,414 base pairs, and the primary transcript length is 36,887.5 base pairs.


The gene PDE4B, with NCBI ID 5142, has been mentioned in [93 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PDE4B%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727). The earliest year of publication is 1997, and the middle 50% of publications occurred between 2011 and 2019.


The top 5 publications mentioning PDE4B, ranked by their scientific influence, include "[Trial of a Preferential Phosphodiesterase 4B Inhibitor for Idiopathic Pulmonary Fibrosis.](https://pubmed.ncbi.nlm.nih.gov/35569036)" (2022) (relative citation ratio: 12.81), "[DISC1 and PDE4B are interacting genetic factors in schizophrenia that regulate cAMP signaling.](https://pubmed.ncbi.nlm.nih.gov/16293762)" (2005) (relative citation ratio: 10.41), "[BI 1015550 is a PDE4B Inhibitor and a Clinical Drug Candidate for the Oral Treatment of Idiopathic Pulmonary Fibrosis.](https://pubmed.ncbi.nlm.nih.gov/35517783)" (2022) (relative citation ratio: 7.13), "[Cardiac Overexpression of PDE4B Blunts Î²-Adrenergic Response and Maladaptive Remodeling in Heart Failure.](https://pubmed.ncbi.nlm.nih.gov/32264695)" (2020) (relative citation ratio: 2.9), and "[Suppression of human inflammatory cell function by subtype-selective PDE4 inhibitors correlates with inhibition of PDE4A and PDE4B.](https://pubmed.ncbi.nlm.nih.gov/10602317)" (1999) (relative citation ratio: 2.87). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PDE4B](https://www.proteinatlas.org/ENSG00000184588-PDE4B) is a hydrolase with evidence at the protein level, making it an FDA-approved drug target. Its RNA is detected in all tissues, with notable expression in the brain and testis, where it is involved in signal transduction. Subcellularly, PDE4B is located in the Golgi apparatus, primary cilium, and cytosol. In the brain, it is specifically expressed in oligodendrocytes. The gene is also expressed in various cell lines and single cells, although its function in these contexts is less clear.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), [RELA](https://www.ncbi.nlm.nih.gov/gene/5970), [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613), and [AR](https://www.ncbi.nlm.nih.gov/gene/367), each with 5 experiments, followed by [CEBPB](https://www.ncbi.nlm.nih.gov/gene/1051) with 4 experiments.



The input data from GWAS indicates associations with a wide range of disease conditions, including various types of cancer such as leukemia, hematologic cancer, immune system cancer, and organ system cancer. Additionally, the data highlights respiratory system diseases, including lower respiratory tract disease and lung disease. Metabolic disorders are also prominent, with specific mentions of diabetes mellitus, particularly type 2 diabetes mellitus, and diseases related to carbohydrate and glucose metabolism. Cardiovascular system diseases, including artery disease, vascular disease, and hypertension, are also noted. Neurological conditions are well-represented, with mentions of neurodegenerative diseases, cognitive disorders, and specific mental health conditions such as bipolar disorder, schizophrenia, and substance-related disorders. Other notable conditions include eye diseases, sensory system diseases, sleep disorders, and connective tissue diseases, with specific mentions of scleroderma, systemic lupus erythematosus, and cerebrovascular disease.



The gene is expressed in various immune cell types, including BDCA4+ dendritic cells, CD4+ T cells, and CD19+ B cells (negatively selected). Additionally, it is found in whole blood, spinal cord, occipital lobe, and skeletal muscle. The gene is also expressed in the brain.


The proteins are involved in various signaling pathways, including g alpha (s) signaling events, gpcr downstream signaling, signaling by gpcr, signal transduction, opioid signaling, and darpp-32 events.



The protein sequence analyzed has a GRAVY value of -0.5561 (29.2% percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -33.45 (5.21% percentile). The median structural flexibility is 1.0046 (62.08% percentile), with a propensity for helix, sheet, and turn structures of 30.84% (39.44% percentile), 33.29% (47.22% percentile), and 31.52% (74.03% percentile), respectively. The instability index is 54.93 (71.98% percentile), and the isoelectric point is 5.09 (9.36% percentile). The protein has a length of 736 amino acids (77.43% percentile) and a molecular weight of 83342.55 Da (78.02% percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |