---
layout: post
title: NEU1
date: 2025-01-17 16:55 CST
description: NEU1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4758) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4758  | NEU1 | ENSG00000204386 | 6p21.33 |



The gene enables [exo-alpha-sialidase activity](https://amigo.geneontology.org/amigo/term/GO:0004308) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is involved in the [ganglioside catabolic process](https://amigo.geneontology.org/amigo/term/GO:0006689) and [oligosaccharide catabolic process](https://amigo.geneontology.org/amigo/term/GO:0009313). It is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [lysosome](https://amigo.geneontology.org/amigo/term/GO:0005764), [lysosomal membrane](https://amigo.geneontology.org/amigo/term/GO:0005765), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [cell junction](https://amigo.geneontology.org/amigo/term/GO:0030054), [specific granule lumen](https://amigo.geneontology.org/amigo/term/GO:0035580), [lysosomal lumen](https://amigo.geneontology.org/amigo/term/GO:0043202), [intracellular membrane-bounded organelle](https://amigo.geneontology.org/amigo/term/GO:0043231), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). Additionally, it is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [lysosome](https://amigo.geneontology.org/amigo/term/GO:0005764), and [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020). The gene also enables [alpha-sialidase activity](https://amigo.geneontology.org/amigo/term/GO:0016997), [exo-alpha-(2->3)-sialidase activity](https://amigo.geneontology.org/amigo/term/GO:0052794), [exo-alpha-(2->6)-sialidase activity](https://amigo.geneontology.org/amigo/term/GO:0052795), and [exo-alpha-(2->8)-sialidase activity](https://amigo.geneontology.org/amigo/term/GO:0052796).


The gene length is 14,685 base pairs (34.61% of all genes), the mature length is 3,291 base pairs (76.38% of all genes), and the primary transcript length is 14,632 base pairs (43.46% of all genes).


The gene NEU1 (NCBI ID: 4758) has been mentioned in [2,224 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22NEU1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning NEU1 is 1948, with the middle 50% of publications occurring between 1982 and 2014.


The top 5 publications mentioning NEU1, ranked by their scientific influence as measured by the [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541), include: [Fluorometric assay of neuraminidase with a sodium (4-methylumbelliferyl-alpha-D-N-acetylneuraminate) substrate.](https://pubmed.ncbi.nlm.nih.gov/464297) (1979) (RCR: 25.6), [Influenza neuraminidase inhibitors possessing a novel hydrophobic interaction in the enzyme active site: design, synthesis, and structural analysis of carbocyclic sialic acid analogues with potent anti-influenza activity.](https://pubmed.ncbi.nlm.nih.gov/16526129) (1997) (RCR: 18.44), [Effectiveness of neuraminidase inhibitors in reducing mortality in patients admitted to hospital with influenza A H1N1pdm09 virus infection: a meta-analysis of individual participant data.](https://pubmed.ncbi.nlm.nih.gov/24815805) (2014) (RCR: 17.21), [Efficacy and safety of the neuraminidase inhibitor zanamivir in the treatment of influenzavirus infections. GG167 Influenza Study Group.](https://pubmed.ncbi.nlm.nih.gov/9302301) (1997) (RCR: 14.67), and [Neuraminidase inhibitors for preventing and treating influenza in adults and children.](https://pubmed.ncbi.nlm.nih.gov/24718923) (2014) (RCR: 14.67). The RCR values measure the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[NEU1](https://www.proteinatlas.org/ENSG00000204386-NEU1) is a gene encoding a protein with glycosidase and hydrolase molecular functions. It is involved in disease variants and is an FDA-approved drug target. Evidence for this gene is available at the protein level. NEU1 is detected in many tissues and is localized in vesicles and cell junctions. Its expression clusters include non-specific DNA binding in blood, non-specific transcription in tissues and brain, immune response in myeloma cell lines, and non-specific transcription in single cells. While NEU1 is unprognostic for several cancer types, including breast invasive carcinoma, colon adenocarcinoma, and kidney renal clear cell carcinoma, it is notably unprognostic for kidney chromophobe with a p-value of 9.24e-3.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [PGR](https://www.ncbi.nlm.nih.gov/gene/5241), [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599), [GATA3](https://www.ncbi.nlm.nih.gov/gene/2625), [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170), and [FOS](https://www.ncbi.nlm.nih.gov/gene/2353), each shown to be regulating in 5 experiments.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [STAT1](https://www.ncbi.nlm.nih.gov/gene/6829), [GATA1](https://www.ncbi.nlm.nih.gov/gene/2623), and [STAT2](https://www.ncbi.nlm.nih.gov/gene/6872), each shown to be regulating genes in two experiments. Additionally, [STAT3](https://www.ncbi.nlm.nih.gov/gene/6749) and [IRF1](https://www.ncbi.nlm.nih.gov/gene/8243) were found to regulate genes in one experiment each.


The GWAS data indicates associations with various disease conditions, including immune system disease, general disease, disease of anatomical entity, hypersensitivity reaction disease, hypersensitivity reaction type II disease, and vitiligo.



The gene is expressed in various tissues and cell types, including the pituitary and adrenal gland, as indicated by BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles show expression in CD33+ Myeloid cells, thyroid, CD14+ Monocytes, leukemia (chronic myelogenous K-562), placenta, pancreatic islet, and kidney.


The input data highlights various diseases and pathways related to glycosylation, particularly asparagine N-linked glycosylation. Notably, defects in specific proteins such as ALG2, ALG3, ALG6, ALG8, ALG9, DPAGT1, MAN1B1, MGAT2, MOGS, MPDU1, ALG1, ALG12, ALG11, ALG14, and RFT1 are linked to distinct congenital disorders of glycosylation (CDGs) and other conditions like congenital myasthenic syndrome. Additionally, the data covers broader processes including the biosynthesis of N-glycan precursors, post-translational protein modification, sialic acid metabolism, and the metabolism of proteins, glycosphingolipids, and sphingolipids. These pathways are integral to overall metabolism and are implicated in various diseases.



The analyzed protein sequence has a GRAVY value of -0.165 (72.68th percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is -6.92 (28.67th percentile), and the median structural flexibility is 0.999 (35.87th percentile). The protein has a moderate affinity for helix (0.246, 4.33rd percentile) and sheet (0.366, 70.6th percentile) structures, with a turn affinity of 0.342 (86.85th percentile). The instability index is 33.93 (8.42nd percentile), and the isoelectric point is 5.59 (20.31st percentile). The protein is 415 amino acids long (47.26th percentile) with a molecular weight of 45466.81 Da (46.31st percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |