---
layout: post
title: MAP3K9
date: 2025-01-17 16:55 CST
description: MAP3K9 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4293) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4293  | MAP3K9 | ENSG00000006432 | 14q24.2 |



The gene is involved in the [MAPK cascade](https://amigo.geneontology.org/amigo/term/GO:0000165) and [protein phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0006468), playing a role in the [apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0006915) and its [positive regulation](https://amigo.geneontology.org/amigo/term/GO:0043065). It enables [protein serine/threonine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004674), [JUN kinase kinase kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004706), [MAP kinase kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004708), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524), [protein homodimerization activity](https://amigo.geneontology.org/amigo/term/GO:0042803), and [protein serine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0106310). Additionally, it is active in the [cellular component](https://amigo.geneontology.org/amigo/term/GO:0005575) and involved in [protein autophosphorylation](https://amigo.geneontology.org/amigo/term/GO:0046777).


The gene length is 47,512.0 base pairs (64.98% of all genes), the mature length is 4,168.5 base pairs, and the primary transcript length is 46,269.5 base pairs.


MAP3K9 (Gene ID: 4293) has been mentioned in [16 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22MAP3K9%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning this gene was in 2004, and the middle 50% of publications occurred between 2016 and 2021.


The top 5 publications mentioning MAP3K9, ranked by their scientific influence, include studies that highlight the gene's role in various cancers and diseases. The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Notable publications include "[Frequent somatic mutations in MAP3K5 and MAP3K9 in metastatic melanoma identified by exome sequencing.](https://pubmed.ncbi.nlm.nih.gov/22197930)" (2011) (relative citation ratio: 3.3), "[Promoter methylation-regulated miR-148a-3p inhibits lung adenocarcinoma (LUAD) progression by targeting MAP3K9.](https://pubmed.ncbi.nlm.nih.gov/35388129)" (2022) (relative citation ratio: 2.55), and "[Role of miR-15a in intervertebral disc degeneration through targeting MAP3K9.](https://pubmed.ncbi.nlm.nih.gov/28081468)" (2017) (relative citation ratio: 2.01). Additionally, "[MiRNA profile of osteosarcoma with CD117 and stro-1 expression: miR-1247 functions as an onco-miRNA by targeting MAP3K9.](https://pubmed.ncbi.nlm.nih.gov/25973030)" (2015) (relative citation ratio: 1.36) and "[Targeted genetic dependency screen facilitates identification of actionable mutations in FGFR4, MAP3K9, and PAK5 in lung cancer.](https://pubmed.ncbi.nlm.nih.gov/23836671)" (2013) (relative citation ratio: 1.22) are also significant. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[MAP3K9](https://www.proteinatlas.org/ENSG00000006432-MAP3K9) is a kinase with serine/threonine-protein kinase and transferase molecular functions. Evidence for this gene is available at the protein level. Its RNA is detected in many tissues, with notable expression in the retina and testis, specifically in cilium-related structures. In the brain, MAP3K9 is associated with neurons and synapses, playing a role in synaptic function. At the subcellular level, it is localized in the nucleoplasm and nucleoli. In single-cell expression analysis, MAP3K9 clusters with neurons involved in neuronal signaling. Additionally, it is expressed in squamous epithelial cells, contributing to keratinization processes.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861) with 6 experiments, [SMAD2](https://www.ncbi.nlm.nih.gov/gene/4087) with 5 experiments, [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908) with 5 experiments, [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113) with 5 experiments, and [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The input data from GWAS indicates associations with a range of disease conditions, including lower respiratory tract disease, lung disease, respiratory system disease, disease of metabolism, acquired metabolic disease, nutrition disease, overnutrition, obesity, cognitive disorder, disease of mental health, bipolar disorder, mood disorder, psychotic disorder, schizophrenia, gastrointestinal system disease, mouth disease, tooth disease, and tooth agenesis.



The gene is expressed in various brain regions, including the prefrontal cortex, amygdala, hypothalamus, thalamus, cingulate cortex, and cerebellum, as well as in the pituitary gland.



The mouse ortholog gene 338372, which corresponds to the human ortholog gene 4293, is associated with several phenotypic abnormalities. These include abnormal kidney morphology, abnormal skin morphology, and small kidney, which are observed in both sexes. Additionally, abnormal testis morphology and enlarged testis are noted, although these are not observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.582 (26.79th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -20.57 (10.74th percentile), and the median structural flexibility is 1.01 (80.96th percentile). The protein has a high instability index of 66.65 (91.96th percentile) and an isoelectric point of 5.63 (21.36th percentile). The sequence length is 1104 amino acids (90th percentile), with a molecular weight of 121893.10 Da (89.72nd percentile). The secondary structure affinities are 32.79% for helix (57.6th percentile), 29.44% for sheet (25.49th percentile), and 33.97% for turn (85.89th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |