---
layout: post
title: MAPRE2
date: 2025-01-17 16:55 CST
description: MAPRE2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10982) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10982  | MAPRE2 | ENSG00000166974 | 18q12.1-q12.2 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [microtubule binding](https://amigo.geneontology.org/amigo/term/GO:0008017), and is involved in [protein kinase binding](https://amigo.geneontology.org/amigo/term/GO:0019901) and [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802). It is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925), and [microtubule cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0015630). The gene is active in the [microtubule organizing center](https://amigo.geneontology.org/amigo/term/GO:0005815), [cytoplasmic microtubule](https://amigo.geneontology.org/amigo/term/GO:0005881), [microtubule plus-end](https://amigo.geneontology.org/amigo/term/GO:0035371), and [spindle midzone](https://amigo.geneontology.org/amigo/term/GO:0051233). It is involved in the regulation of [microtubule polymerization or depolymerization](https://amigo.geneontology.org/amigo/term/GO:0031110), [positive regulation of ARF protein signal transduction](https://amigo.geneontology.org/amigo/term/GO:0032014), [protein localization to microtubule](https://amigo.geneontology.org/amigo/term/GO:0035372), [positive regulation of GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0043547), [spindle assembly](https://amigo.geneontology.org/amigo/term/GO:0051225), [cell division](https://amigo.geneontology.org/amigo/term/GO:0051301), [positive regulation of keratinocyte migration](https://amigo.geneontology.org/amigo/term/GO:0051549), and [positive regulation of focal adhesion disassembly](https://amigo.geneontology.org/amigo/term/GO:0120183).


The gene length is 4,417 base pairs (12.42% of all genes), the mature length is 3,392 base pairs, and the primary transcript length is 3,576 base pairs (13.17% of all genes).


The gene MAPRE2 (NCBI ID: 10982) has been mentioned in [30 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22MAPRE2%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1965 and the middle 50% of publications occurring between 2002 and 2020. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning MAPRE2, ranked by their relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)), include: [Both Epstein-Barr virus (EBV)-encoded trans-acting factors, EB1 and EB2, are required to activate transcription from an EBV early promoter.](https://pubmed.ncbi.nlm.nih.gov/3028777) (1986) (relative citation ratio: 6.35), [The Epstein-Barr virus (EBV) early protein EB2 is a posttranscriptional activator expressed under the control of EBV transcription factors EB1 and R.](https://pubmed.ncbi.nlm.nih.gov/2555554) (1989) (relative citation ratio: 1.9), [Combining Continuous Smartphone Native Sensors Data Capture and Unsupervised Data Mining Techniques for Behavioral Changes Detection: A Case Series of the Evidence-Based Behavior (eB2) Study.](https://pubmed.ncbi.nlm.nih.gov/30530465) (2018) (relative citation ratio: 1.82), [Use of Ecological Momentary Assessment Through a Passive Smartphone-Based App (eB2) by Patients With Schizophrenia: Acceptability Study.](https://pubmed.ncbi.nlm.nih.gov/34309576) (2021) (relative citation ratio: 1.44), and [THE BEHAVIOUR AND MORPHOLOGY OF A SECOND TISSUE CULTURE STRAIN (EB2) OF LYMPHOBLASTS FROM BURKITT'S LYMPHOMA.](https://pubmed.ncbi.nlm.nih.gov/14284371) (1965) (relative citation ratio: 1.36). The relative citation ratio measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[MAPRE2](https://www.proteinatlas.org/ENSG00000166974-MAPRE2) is a gene with evidence at the protein level. Its RNA is detected in all tissues, and it is associated with various expression clusters, including Cluster 5 for non-specific cell proliferation in blood, Cluster 9 for oligodendrocytes and myelination in tissues, Cluster 28 for mixed function in the brain, Cluster 46 for lymphoma and humoral immune response in cell lines, and Cluster 78 for NK-cells, T-cells, and immune system and transcription in single cells.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) and [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885), each with 6 experiments. Additionally, [KMT2B](https://www.ncbi.nlm.nih.gov/gene/9757), [GATA2](https://www.ncbi.nlm.nih.gov/gene/2624), and [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) were each found to be regulating in 5 experiments.



The GWAS data indicates associations with a range of disease conditions, including various metabolic disorders such as diabetes mellitus and type 2 diabetes mellitus, as well as cardiovascular system disease. Additionally, the data highlights links to kidney disease, including chronic kidney failure and end stage renal failure. Neurological conditions are also implicated, with associations to neurodegenerative diseases like Parkinson's disease, cognitive disorders, and mood disorders such as mental depression and alcohol dependence. Furthermore, the data suggests connections to eye diseases, specifically myopia and refractive errors.



The gene is expressed in various tissues, including the heart, brain, and adrenal gland, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the pineal gland (both day and night), cingulate cortex, medulla oblongata, parietal lobe, occipital lobe, and cerebellum peduncles.




The analyzed protein sequence has a GRAVY value of -0.774 (13.02 percentile), indicating a hydrophilic nature. The charge at pH 7 is -12.51 (18.64 percentile), and the median structural flexibility is 1.012 (86.65 percentile). The protein has a predicted affinity for helix, sheet, and turn structures of 32.11% (51.35 percentile), 28.75% (21.96 percentile), and 30.58% (67.99 percentile), respectively. The instability index is 42.89 (31.73 percentile), with an isoelectric point of 5.36 (14.98 percentile). The protein is 327 amino acids long (34.19 percentile) and has a molecular weight of 37030.89 Da (35.17 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |