---
layout: post
title: RPSA
date: 2024-12-08 23:57 CST
description: RPSA description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3921) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3921  | RPSA | ENSG00000168028 | 3p22.1 |



The gene length is 6,074 base pairs (16.79% of all genes), the mature length is 1,571.5 base pairs, and the primary transcript length is 5,247 base pairs.


The gene RPSA, with NCBI ID 3921, has been the subject of [147 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RPSA%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1991 and the middle 50% of publications occurring between 2002 and 2016. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning RPSA, ranked by their scientific influence, include "[Drug resistance-associated marker Lrp for prediction of response to chemotherapy and prognoses in advanced ovarian carcinoma.](https://pubmed.ncbi.nlm.nih.gov/7563169)" (1995) (relative citation ratio: 6.11), "[Green tea polyphenol epigallocatechin-3-gallate signaling pathway through 67-kDa laminin receptor.](https://pubmed.ncbi.nlm.nih.gov/18079119)" (2008) (relative citation ratio: 4.17), "[Epigallocatechin-3-O-gallate up-regulates microRNA-let-7b expression by activating 67-kDa laminin receptor signaling in melanoma cells.](https://pubmed.ncbi.nlm.nih.gov/26754091)" (2016) (relative citation ratio: 3.83), "[TLR4 signaling inhibitory pathway induced by green tea polyphenol epigallocatechin-3-gallate through 67-kDa laminin receptor.](https://pubmed.ncbi.nlm.nih.gov/20511545)" (2010) (relative citation ratio: 3.79), and "[Expression of the 67-kD laminin receptor, galectin-1, and galectin-3 in advanced human uterine adenocarcinoma.](https://pubmed.ncbi.nlm.nih.gov/8912829)" (1996) (relative citation ratio: 3.65). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [GATA3](https://www.ncbi.nlm.nih.gov/gene/2625) and [PGR](https://www.ncbi.nlm.nih.gov/gene/5241), each with 5 experiments. Additionally, [OTX2](https://www.ncbi.nlm.nih.gov/gene/5015), [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), and [RELA](https://www.ncbi.nlm.nih.gov/gene/5970) were each found to be regulating in 4 experiments.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, filtered for Perturb-seq data, include [CEBPB](https://www.ncbi.nlm.nih.gov/gene/6829) and [STAT1](https://www.ncbi.nlm.nih.gov/gene/6872), each shown to be regulating genes in 2 experiments. Additionally, [ZNF384](https://www.ncbi.nlm.nih.gov/gene/58517), [CEBPD](https://www.ncbi.nlm.nih.gov/gene/6830), [MMP1](https://www.ncbi.nlm.nih.gov/gene/8861), [TCF3](https://www.ncbi.nlm.nih.gov/gene/9968), and [ZNF354C](https://www.ncbi.nlm.nih.gov/gene/8243) were each found to be regulating genes in 1 experiment.








The protein sequence analyzed has a GRAVY value of -0.309 (56.88th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -14.10 (16.53rd percentile). The median structural flexibility is 1.001 (45.83rd percentile). The protein's secondary structure is predicted to be 34.24% helix (69.68th percentile), 35.93% sheet (65.76th percentile), and 24.75% turn (19.74th percentile). The instability index is 35.97 (12.03rd percentile), and the isoelectric point is 4.79 (4.3rd percentile). The protein is 295 amino acids long (27.76th percentile) with a molecular weight of 32853.68 Da (27.69th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |