---
layout: post
title: BTNL3
date: 2025-01-17 16:55 CST
description: BTNL3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10917) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10917  | BTNL3 | ENSG00000168903 | 5q35.3 |



The gene is involved in the [regulation of cytokine production](https://amigo.geneontology.org/amigo/term/GO:0001817) and the [T cell receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0050852), which are crucial biological processes. It enables [molecular function](https://amigo.geneontology.org/amigo/term/GO:0003674) through [signaling receptor binding](https://amigo.geneontology.org/amigo/term/GO:0005102) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is active in the [cellular component](https://amigo.geneontology.org/amigo/term/GO:0005575), specifically on the [external side of the plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009897).


The gene length is 8,759 base pairs (23.01% of all genes), the mature length is 2,590 base pairs, and the primary transcript length is 8,561 base pairs (29.42% of all genes).


The gene BTNL3 (NCBI ID: 10917) has been mentioned in [1 publication](https://pubmed.ncbi.nlm.nih.gov/?term=%22BTNL3%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 2019, with the middle 50% of publications occurring from 2019 to 2019.


The top publication mentioning BTNL3 is "[Butyrophilin-like 3 Directly Binds a Human VÎ³4<sup>+</sup> T Cell Receptor Using a Modality Distinct from Clonally-Restricted Antigen](https://pubmed.ncbi.nlm.nih.gov/31628053)" (2019), with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 4.12. The RCR measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[BTNL3](https://www.proteinatlas.org/ENSG00000168903-BTNL3) is a gene with evidence at the protein level. Its RNA is detected in some tissues, and it is notably expressed in neutrophils involved in degranulation (Cluster 4). In terms of tissue expression, BTNL3 is associated with the small intestine, specifically in absorption processes (Cluster 25). Additionally, it is expressed in stomach and colon cancer cell lines related to digestion (Cluster 57) and in enterocytes involved in absorption (Cluster 5). The gene is unprognostic for several cancer types, including Breast Invasive Carcinoma, Colon Adenocarcinoma, Liver Hepatocellular Carcinoma, Pancreatic Adenocarcinoma, Rectum Adenocarcinoma, and Stomach Adenocarcinoma.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [RELA](https://www.ncbi.nlm.nih.gov/gene/5970) and [MAX](https://www.ncbi.nlm.nih.gov/gene/4149), each shown to be regulating in 2 experiments. Additionally, [AR](https://www.ncbi.nlm.nih.gov/gene/367) was also found to be regulating in 2 experiments. Furthermore, [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) and [BMI1](https://www.ncbi.nlm.nih.gov/gene/648) were each identified in 1 experiment.




The gene BTNL3 has been associated with [GIP levels in response to oral glucose tolerance test (120 minutes)](https://pubmed.ncbi.nlm.nih.gov/29093273), which maps closely to the MeSH term [Glucose](https://meshb.nlm.nih.gov/record/ui?ui=D005947) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method.


The gene is expressed in various tissues, including blood, colon, and small intestine, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the trigeminal ganglion, skeletal muscle, atrioventricular node, dorsal root ganglion, and tongue.




The analyzed protein sequence has a GRAVY value of -0.114 (77.07th percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is -8.40 (25.52nd percentile), and the median structural flexibility is 0.998 (33.61st percentile). The protein has a length of 466 amino acids (53.93rd percentile) and a molecular weight of 52250 Da (54.19th percentile). The secondary structure affinities are 26.82% for helix (10.88th percentile), 39.48% for sheet (84.66th percentile), and 28.54% for turn (52.09th percentile). The instability index is 33.92 (8.41st percentile), and the isoelectric point is 5.84 (26.28th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |