---
layout: post
title: GIT2
date: 2025-01-17 16:55 CST
description: GIT2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9815) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9815  | GIT2 | ENSG00000139436 | 12q24.11 |



The gene enables [GTPase activator activity](https://amigo.geneontology.org/amigo/term/GO:0005096) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925), and [presynapse](https://amigo.geneontology.org/amigo/term/GO:0098793). It is involved in [brain development](https://amigo.geneontology.org/amigo/term/GO:0007420), [regulation of G protein-coupled receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0008277), [regulation of ARF protein signal transduction](https://amigo.geneontology.org/amigo/term/GO:0032012), and [synaptic vesicle recycling](https://amigo.geneontology.org/amigo/term/GO:0036465). Additionally, it enables [small GTPase binding](https://amigo.geneontology.org/amigo/term/GO:0031267) and [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872), and is active in the [synapse](https://amigo.geneontology.org/amigo/term/GO:0045202).


The gene length is 8,436 base pairs (22.3% of all genes), the mature length is 745 base pairs, and the primary transcript length is 7,755 base pairs.


The gene GIT2 (NCBI ID: 9815) has been mentioned in [23 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22GIT2%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 2000 and the middle 50% of publications occurring between 2006 and 2018. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning GIT2, ranked by their scientific influence, include "[Reliability and validity of the University of California, Los Angeles Scleroderma Clinical Trial Consortium Gastrointestinal Tract Instrument.](https://pubmed.ncbi.nlm.nih.gov/19714600)" (2009) with a relative citation ratio of 5.11, "[Relationship Between Gastrointestinal Transit, Medsger Gastrointestinal Severity, and University of California-Los Angeles Scleroderma Clinical Trial Consortium Gastrointestinal Tract 2.0 Symptoms in Patients With Systemic Sclerosis.](https://pubmed.ncbi.nlm.nih.gov/33064934)" (2022) with a relative citation ratio of 3.73, "[Paxillin-dependent paxillin kinase linker and p21-activated kinase localization to focal adhesions involves a multistep activation pathway.](https://pubmed.ncbi.nlm.nih.gov/12006652)" (2002) with a relative citation ratio of 2.21, "[Mesenchymal stem cells promote endothelial progenitor cell migration, vascularization, and bone repair in tissue-engineered constructs via activating CXCR2-Src-PKL/Vav2-Rac1.](https://pubmed.ncbi.nlm.nih.gov/29229683)" (2018) with a relative citation ratio of 1.93, and "[The GIT family of ADP-ribosylation factor GTPase-activating proteins. Functional diversity of GIT2 through alternative splicing.](https://pubmed.ncbi.nlm.nih.gov/10896954)" (2000) with a relative citation ratio of 1.82. The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[GIT2](https://www.proteinatlas.org/ENSG00000139436-GIT2) is a gene involved in GTPase activation, with evidence supporting its function at the protein level. It is detected in all RNA tissue distributions and is primarily localized to microtubules. The gene is expressed in various clusters, including Cluster 7 for blood expression related to the innate immune response, Cluster 16 for tissue expression related to transcription, Cluster 37 for brain expression in neurons with mixed function, Cluster 46 for cell line expression in lymphoma related to the humoral immune response, and Cluster 29 for single cell expression in monocytes related to the innate immune response.


The top transcription factors (TFs) identified from the GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025), [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046), [MYOD1](https://www.ncbi.nlm.nih.gov/gene/4654), and [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599), each with 4 experiments demonstrating their regulatory roles.



The GWAS data indicates associations with a range of metabolic and cardiovascular conditions, including diabetes mellitus, particularly type 2 diabetes mellitus, and various metabolic disorders such as those related to carbohydrate and lipid metabolism. Additionally, the data highlights links to hypertension, kidney disease, and several mental health conditions, including cognitive disorders, mood disorders, and psychotic disorders. Furthermore, associations with overnutrition and cardiovascular system diseases are also noted.



The gene is expressed in various immune cell types, including CD33+ myeloid cells, BDCA4+ dendritic cells, CD56+ NK cells, CD4+ T cells, CD8+ T cells, CD19+ B cells (neg. sel.), and CD14+ monocytes. Additionally, it is expressed in blood and spleen tissues.




The analyzed protein sequence has a GRAVY value of -0.695 (17.93rd percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -1.48 (44.82nd percentile), and the median structural flexibility is 1.01 (80.82nd percentile). The protein has a predicted secondary structure with 31.1% helix (42.02nd percentile), 29.6% sheet (26.48th percentile), and 32.0% turn (76.82nd percentile) affinities. The instability index is 51.64 (62.44th percentile), and the isoelectric point is 6.78 (47.53rd percentile). The protein is 759 amino acids long (78.73rd percentile) with a molecular weight of 84541.77 Da (78.59th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |