---
layout: post
title: ADAP1
date: 2025-01-17 16:55 CST
description: ADAP1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/11033) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 11033  | ADAP1 | ENSG00000105963 | 7p22.3 |



The gene enables [GTPase activator activity](https://amigo.geneontology.org/amigo/term/GO:0005096) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is involved in [phosphatidylinositol-3,4,5-trisphosphate binding](https://amigo.geneontology.org/amigo/term/GO:0005547), [inositol 1,3,4,5 tetrakisphosphate binding](https://amigo.geneontology.org/amigo/term/GO:0043533), [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872), and [phosphatidylinositol bisphosphate binding](https://amigo.geneontology.org/amigo/term/GO:1902936). It is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), and is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), and [intracellular membrane-bounded organelle](https://amigo.geneontology.org/amigo/term/GO:0043231). Additionally, it is involved in the [cell surface receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007166) and the [regulation of GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0043087).


The gene length is 29,492 base pairs (52.23% of all genes), the mature length is 690 base pairs (8.91% of all genes), and the primary transcript length is 6,826.5 base pairs (24.45% of all genes).


ADAP1 has been mentioned in [22 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ADAP1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest mention in 1996 and the middle 50% of publications occurring between 2001 and 2016. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning ADAP1, ranked by their scientific influence, include "[Identification and cloning of centaurin-alpha. A novel phosphatidylinositol 3,4,5-trisphosphate-binding protein from rat brain.](https://pubmed.ncbi.nlm.nih.gov/8702546)" (1996) (relative citation ratio: 2.26), "[Phosphorylation-independent dual-site binding of the FHA domain of KIF13 mediates phosphoinositide transport via centaurin alpha1.](https://pubmed.ncbi.nlm.nih.gov/21057110)" (2010) (relative citation ratio: 0.92), "[Identification of centaurin-alpha1 as a potential in vivo phosphatidylinositol 3,4,5-trisphosphate-binding protein that is functionally homologous to the yeast ADP-ribosylation factor (ARF) GTPase-activating protein, Gcs1.](https://pubmed.ncbi.nlm.nih.gov/10333475)" (1999) (relative citation ratio: 0.86), "[Centaurin-alpha1 is an in vivo phosphatidylinositol 3,4,5-trisphosphate-dependent GTPase-activating protein for ARF6 that is involved in actin cytoskeleton organization.](https://pubmed.ncbi.nlm.nih.gov/14625293)" (2004) (relative citation ratio: 0.79), and "[Casein kinase I associates with members of the centaurin-alpha family of phosphatidylinositol 3,4,5-trisphosphate-binding proteins.](https://pubmed.ncbi.nlm.nih.gov/11278595)" (2001) (relative citation ratio: 0.73). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ADAP1](https://www.proteinatlas.org/ENSG00000105963-ADAP1) is a gene involved in GTPase activation, with evidence supporting its function at the protein level. It is detected in many tissues and is primarily localized in the plasma membrane and cytosol. ADAP1 is highly expressed in monocytes and neutrophils, playing a role in the innate immune response. Additionally, it is associated with oligodendrocytes involved in myelination and is found in white matter. In terms of cell line expression, ADAP1 is clustered with stomach and colon cancer cells, suggesting a role in digestion. It is also expressed in enterocytes, contributing to absorption processes.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885), [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), and [RELA](https://www.ncbi.nlm.nih.gov/gene/5970), each shown to be regulating in 5 experiments.





The gene is expressed in various tissues, including the brain, blood, and colon, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in CD33+ Myeloid cells, the whole brain, spinal cord, temporal lobe, medulla oblongata, and pons, suggesting a ubiquitous expression pattern across multiple tissue samples.



The mouse ortholog gene 231821, which corresponds to the human ortholog gene 11033, is associated with phenotypic changes in NK cell numbers. Specifically, it is linked to a decrease in memory-marker NK cell number in one sex and an overall increase in NK cell number in both sexes.


The analyzed protein sequence has a GRAVY value of -0.681 (18.91% percentile), indicating a hydrophilic nature. It exhibits a charge of 9.63 at pH 7.0 (80.59% percentile) and a median structural flexibility of 1.002 (50.35% percentile). The protein's secondary structure is predicted to be 31.82% helix (48.78% percentile), 31.82% sheet (38.49% percentile), and 25.40% turn (24.42% percentile). The instability index is 40.76 (24.4% percentile), with an isoelectric point of 9.10 (81.71% percentile). The protein has a length of 374 amino acids (41.74% percentile) and a molecular weight of 43394.75 Da (43.89% percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |