---
layout: post
title: SDC2
date: 2025-01-17 16:55 CST
description: SDC2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6383) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6383  | SDC2 | ENSG00000169439 | 8q22.1 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [PDZ domain binding](https://amigo.geneontology.org/amigo/term/GO:0030165), and is involved in [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802). It is located in the [endoplasmic reticulum lumen](https://amigo.geneontology.org/amigo/term/GO:0005788), [Golgi lumen](https://amigo.geneontology.org/amigo/term/GO:0005796), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [lysosomal lumen](https://amigo.geneontology.org/amigo/term/GO:0043202), and [collagen-containing extracellular matrix](https://amigo.geneontology.org/amigo/term/GO:0062023). The gene is active on the [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986) and plays roles in various biological processes including [cell migration](https://amigo.geneontology.org/amigo/term/GO:0016477), [dendrite morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0048813), and [regulation of dendrite morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0048814).


The gene length is 137,561 base pairs (87.59% of all genes), the mature length is 1,404 base pairs (35.16% of all genes), and the primary transcript length is 33,234.5 base pairs (65.31% of all genes).


The gene SDC2 (NCBI ID: 6383) has been mentioned in [143 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SDC2%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1989 and the middle 50% of publications occurring between 2005 and 2019. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning SDC2, ranked by their scientific influence, include "[Human CASK/LIN-2 binds syndecan-2 and protein 4.1 and localizes to the basolateral membrane of epithelial cells.](https://pubmed.ncbi.nlm.nih.gov/9660868)" (1998) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 5.33, "[Human basement membrane heparan sulfate proteoglycan core protein: a 467-kD protein containing multiple domains resembling elements of the low density lipoprotein receptor, laminin, neural cell adhesion molecules, and epidermal growth factor.](https://pubmed.ncbi.nlm.nih.gov/1730768)" (1992) with an RCR of 5.31, "[Low density lipoprotein receptor internalizes low density and very low density lipoproteins that are bound to heparan sulfate proteoglycans via lipoprotein lipase.](https://pubmed.ncbi.nlm.nih.gov/8387492)" (1993) with an RCR of 4.68, "[Genome-wide identification and validation of a novel methylation biomarker, SDC2, for blood-based detection of colorectal cancer.](https://pubmed.ncbi.nlm.nih.gov/23747112)" (2013) with an RCR of 3.59, and "[Colorectal adenoma and cancer detection based on altered methylation pattern of SFRP1, SFRP2, SDC2, and PRIMA1 in plasma samples.](https://pubmed.ncbi.nlm.nih.gov/28753106)" (2017) with an RCR of 3.31. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[SDC2](https://www.proteinatlas.org/ENSG00000169439-SDC2) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily located in the plasma membrane and is associated with various expression clusters, including Cluster 21 in neutrophils with unknown function, Cluster 1 in liver and kidney related to metabolism, Cluster 21 in the brain with non-specific metabolism, Cluster 64 in connective tissue cells involved in ECM organization, and Cluster 53 in fibroblasts also involved in ECM organization.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [E2F1](https://www.ncbi.nlm.nih.gov/gene/1869), [RELA](https://www.ncbi.nlm.nih.gov/gene/5970), [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), and [AR](https://www.ncbi.nlm.nih.gov/gene/367), each with 5 experiments. Additionally, [BRD9](https://www.ncbi.nlm.nih.gov/gene/65980) was found to be regulating in 4 experiments.



The input data from GWAS indicates associations with a range of disease conditions, including obstructive lung disease, lower respiratory tract disease, lung disease, respiratory system disease, disease of metabolism, acquired metabolic disease, nutrition disease, overnutrition, obesity, coronary artery disease, artery disease, vascular disease, cardiovascular system disease, hypertension, kidney disease, urinary system disease, cognitive disorder, disease of mental health, bipolar disorder, mood disorder, psychotic disorder, schizophrenia, inherited metabolic disorder, lipid metabolism disorder, chronic obstructive pulmonary disease, and cerebrovascular disease.



The gene is expressed in various tissues, including the ovary and thyroid, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the retina, thyroid, adipocyte, pituitary, fetal thyroid, ovary, and uterus.


The input data includes various pathways and diseases related to carbohydrate metabolism, glycosaminoglycan metabolism, and glycosylation. Notable conditions include Myoclonic epilepsy of Lafora, Glycogen storage diseases, and several types of Mucopolysaccharidoses (MPS), such as Sly syndrome, Maroteaux-Lamy syndrome, and various Sanfilippo syndromes. Defects in specific genes like B3GAT3, B4GALT1, CHST14, and EXT1/EXT2 are associated with distinct disorders, including JDSSDHD, B4GALT1-CDG, EDS types, and exostoses. Additionally, the data covers visual transduction, retinoid metabolism, axon guidance, and EPH-Ephrin signaling, highlighting a broad range of biological processes and diseases.


The mouse ortholog gene 15529 and its human ortholog gene 6383 are associated with phenotypes including abnormal coat/hair pigmentation and abnormal lens morphology, with these effects observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.407 (45.3% percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -11.29 (20.5% percentile), and the median structural flexibility is 1.016 (93.75% percentile). The protein has a high affinity for helix structures (38.81%, 90.97% percentile) and sheet structures (34.33%, 54.29% percentile), with a moderate affinity for turn structures (29.35%, 58.87% percentile). The instability index is 54.32 (70.23% percentile), suggesting potential instability. The isoelectric point is 4.75 (3.66% percentile), and the sequence length is 201 amino acids (14.68% percentile), with a molecular weight of 22159.64 Da (14.34% percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |