---
layout: post
title: PPP1R13L
date: 2024-12-08 23:57 CST
description: PPP1R13L description
tags: [cooccuring-genes]
categories: longest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10848) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10848  | PPP1R13L | ENSG00000104881 | 19q13.32 |



The gene length is 952,006 base pairs (99.58th percentile), the mature length is 3,557 base pairs (79.33rd percentile), and the primary transcript length is 951,097 base pairs (99.88th percentile).


The gene PPP1R13L, with NCBI ID 10848, has been featured in [114 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PPP1R13L%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest publication year is 1999, and the middle 50% of publications occurred between 2011 and 2018.


The top five publications mentioning PPP1R13L, ranked by their scientific influence, include "[iASPP oncoprotein is a key inhibitor of p53 conserved from worm to human.](https://pubmed.ncbi.nlm.nih.gov/12524540)" (2003) with a relative citation ratio of 4.65, "[iASPP Is an Antioxidative Factor and Drives Cancer Growth and Drug Resistance by Competing with Nrf2 for Keap1 Binding.](https://pubmed.ncbi.nlm.nih.gov/29033244)" (2017) with a relative citation ratio of 4.28, "[iASPP preferentially binds p53 proline-rich region and modulates apoptotic function of codon 72-polymorphic p53.](https://pubmed.ncbi.nlm.nih.gov/16964264)" (2006) with a relative citation ratio of 3.85, "[The lncRNA UCA1 interacts with miR-182 to modulate glioma proliferation and migration by targeting iASPP.](https://pubmed.ncbi.nlm.nih.gov/28137422)" (2017) with a relative citation ratio of 3.28, and "[Restoring p53 function in human melanoma cells by inhibiting MDM2 and cyclin B1/CDK1-phosphorylated nuclear iASPP.](https://pubmed.ncbi.nlm.nih.gov/23623661)" (2013) with a relative citation ratio of 2.88. The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[Repressor](https://www.proteinatlas.org/[Ensembl]-[Gene]) is a gene with evidence at the protein level, implicated in various diseases such as Cardiomyopathy, Disease variant, and Ectodermal dysplasia. It is detected in many tissues and primarily localized in the cytosol. The gene is notably expressed in squamous epithelium with keratinization (Cluster 27), astrocytes with mixed function (Cluster 19), and squamous epithelial cells with keratinization in both cell line and single cell expression clusters (Cluster 53 and Cluster 12, respectively).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908) with 6 experiments, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 5 experiments, [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) with 5 experiments, [MAX](https://www.ncbi.nlm.nih.gov/gene/4149) with 5 experiments, and [ZBTB7A](https://www.ncbi.nlm.nih.gov/gene/51341) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.











The analyzed protein sequence has a GRAVY value of -0.655 (20.76th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -3.38 (38.31st percentile), and the median structural flexibility is 1.013 (88.39th percentile). The protein has a high instability index of 72.87 (96.1st percentile) and an isoelectric point of 6.37 (39.73rd percentile). The sequence length is 828 amino acids (81.82nd percentile) with a molecular weight of 89090 Da (80.61st percentile). The secondary structure affinities are 29.35% for helix (26.19th percentile), 24.88% for sheet (8.41st percentile), and 39.37% for turn (96.89th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |