---
layout: post
title: FLT3LG
date: 2025-01-17 16:55 CST
description: FLT3LG description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2323) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2323  | FLT3LG | ENSG00000090554 | 19q13.33 |



The gene enables [signaling receptor binding](https://amigo.geneontology.org/amigo/term/GO:0005102), [cytokine activity](https://amigo.geneontology.org/amigo/term/GO:0005125), and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), and [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020). The gene is active in the [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615) and [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986). It is involved in [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), [positive regulation of cell population proliferation](https://amigo.geneontology.org/amigo/term/GO:0008284), and [embryonic hemopoiesis](https://amigo.geneontology.org/amigo/term/GO:0035162). Additionally, it enables [receptor tyrosine kinase binding](https://amigo.geneontology.org/amigo/term/GO:0030971).


The gene length is 75,936 base pairs (76.29 percentile of all genes), the mature length is 1,684 base pairs, and the primary transcript length is 34,090 base pairs.


The gene FLT3LG (NCBI ID: 2323) has been mentioned in [321 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22FLT3LG%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication mentioning this gene is 1957, and the middle 50% of publications occurred between 1999 and 2013.


The top 5 publications mentioning FLT3LG, ranked by their scientific influence as measured by the [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541), include: ["Dramatic increase in the numbers of functionally mature dendritic cells in Flt3 ligand-treated mice: multiple dendritic cell subpopulations identified."](https://pubmed.ncbi.nlm.nih.gov/8920882) (1996) (relative citation ratio: 17.4), ["c-kit ligand and Flt3 ligand: stem/progenitor cell factors with overlapping yet distinct activities."](https://pubmed.ncbi.nlm.nih.gov/9454740) (1998) (relative citation ratio: 9.22), ["Altered peptide ligand vaccination with Flt3 ligand expanded dendritic cells for tumor immunotherapy."](https://pubmed.ncbi.nlm.nih.gov/11427731) (2001) (relative citation ratio: 7.53), ["Sodium fluorescein-guided resection under the YELLOW 560 nm surgical microscope filter in malignant brain tumor surgery--a feasibility study."](https://pubmed.ncbi.nlm.nih.gov/23430234) (2013) (relative citation ratio: 6.6), and ["Automated Lung Sound Classification Using a Hybrid CNN-LSTM Network and Focal Loss Function."](https://pubmed.ncbi.nlm.nih.gov/35161977) (2022) (relative citation ratio: 6.1). The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[FLT3LG](https://www.proteinatlas.org/ENSG00000090554-FLT3LG) is a cytokine with evidence at the protein level. Its RNA is detected in all tissues, and it is expressed in various clusters, including Cluster 3 for non-specific nuclear processes in blood, Cluster 41 for smooth muscle tissue and ECM organization, Cluster 43 for mixed functions in the hindbrain, Cluster 59 for T-cell receptor in leukemia cell lines, and Cluster 7 for T-cell receptor in single cells.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [STAT5B](https://www.ncbi.nlm.nih.gov/gene/6777), [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), and [KDM2B](https://www.ncbi.nlm.nih.gov/gene/84678), each with 5 experiments. Additionally, [GATAD2B](https://www.ncbi.nlm.nih.gov/gene/57459) was found to be regulating in 4 experiments.




FLT3LG has been associated with several traits and diseases through genome-wide association studies. Notably, it has been linked to [mean corpuscular hemoglobin](https://pubmed.ncbi.nlm.nih.gov/34594039) and [mean corpuscular volume](https://pubmed.ncbi.nlm.nih.gov/34594039), both of which map to the MeSH term [Erythrocyte Indices](https://meshb.nlm.nih.gov/record/ui?ui=D004909) using BioWordVec. Additionally, FLT3LG is associated with [monocyte count](https://pubmed.ncbi.nlm.nih.gov/32888493) and [monocyte percentage of white cells](https://pubmed.ncbi.nlm.nih.gov/32888494), mapping to the MeSH term [Monocytes](https://meshb.nlm.nih.gov/record/ui?ui=D009000). Other associations include [mosaic loss of chromosome Y (Y chromosome dosage)](https://pubmed.ncbi.nlm.nih.gov/31624269), mapping to [Chromosomes](https://meshb.nlm.nih.gov/record/ui?ui=D002875), and various protein level traits such as [non-albumin protein levels](https://pubmed.ncbi.nlm.nih.gov/33462484), [serum albumin levels](https://pubmed.ncbi.nlm.nih.gov/33462484), and [serum total protein levels](https://pubmed.ncbi.nlm.nih.gov/33462484), mapping to [Proteins](https://meshb.nlm.nih.gov/record/ui?ui=D011506), [Albumins](https://meshb.nlm.nih.gov/record/ui?ui=D000418), and [Serum](https://meshb.nlm.nih.gov/record/ui?ui=D044967), respectively.


The gene is expressed in a variety of tissues and cell types, including Lymphoma Burkitt's (Raji), CD4+ T cells, CD8+ T cells, whole blood, skeletal muscle, ciliary ganglion, and uterus corpus.




The analyzed protein sequence has a GRAVY value of -0.025 (82.81 percentile), indicating a slightly hydrophilic nature. At pH 7.0, the protein carries a charge of 0.813 (53.13 percentile). The median structural flexibility is 0.994 (18.4 percentile), suggesting a relatively rigid structure. The protein has a high instability index of 69.37 (94.22 percentile) and an isoelectric point of 7.60 (55.23 percentile). The sequence length is 235 amino acids (19.53 percentile), with a molecular weight of 26415.49 Da (19.94 percentile). The secondary structure affinities are 33.19% for helix (60.89 percentile), 40.43% for sheet (87.51 percentile), and 26.81% for turn (36.32 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |