---
layout: post
title: CEACAM4
date: 2025-01-17 16:55 CST
description: CEACAM4 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1089) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1089  | CEACAM4 | ENSG00000105352 | 19q13.2 |



The gene is involved in the [regulation of immune system process](https://amigo.geneontology.org/amigo/term/GO:0002682) and [phagocytosis](https://amigo.geneontology.org/amigo/term/GO:0006909), and plays a role in [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165). It is active in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886) and on the [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986), and is located in the [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020). Additionally, it enables [protein tyrosine kinase binding](https://amigo.geneontology.org/amigo/term/GO:1990782).


The gene length is 30,161.0 base pairs (52.86% of all genes), the mature length is 2,079.0 base pairs, and the primary transcript length is 30,161.0 base pairs.


CEACAM4 (Gene NCBI ID: 1089) has been mentioned in [61 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CEACAM4%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1962, and the middle 50% of publications range from 1983 to 2007.


The top 5 publications mentioning CEACAM4, ranked by their scientific influence, include "[In vivo labelling of granulocytes with 99mTc anti-NCA monoclonal antibodies for imaging inflammation.](https://pubmed.ncbi.nlm.nih.gov/3181186)" (1988) (relative citation ratio: 4.14), "[High-Nickel NMA: A Cobalt-Free Alternative to NMC and NCA Cathodes for Lithium-Ion Batteries.](https://pubmed.ncbi.nlm.nih.gov/32627875)" (2020) (relative citation ratio: 3.87), "[A specific heterotypic cell adhesion activity between members of carcinoembryonic antigen family, W272 and NCA, is mediated by N-domains.](https://pubmed.ncbi.nlm.nih.gov/2022629)" (1991) (relative citation ratio: 2.39), "[CEA and NCA in benign and malignant breast tumors.](https://pubmed.ncbi.nlm.nih.gov/6388310)" (1984) (relative citation ratio: 1.97), and "[A putative cation channel, NCA-1, and a novel protein, UNC-80, transmit neuronal activity in C. elegans.](https://pubmed.ncbi.nlm.nih.gov/18336069)" (2008) (relative citation ratio: 1.9). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CEACAM4](https://www.proteinatlas.org/ENSG00000105352-CEACAM4) is a gene with evidence at the protein level. Its RNA is detected in some tissues, and it is notably expressed in neutrophils with an unknown function (Cluster 21). In terms of tissue expression, CEACAM4 is associated with lymphoid tissue and bone marrow, playing a role in the innate immune response (Cluster 81). Additionally, it is expressed in myeloid leukemia cell lines, also linked to the innate immune response (Cluster 14). At the single-cell level, CEACAM4 is involved in neutrophil degranulation (Cluster 74).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) and [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861), each with 3 experiments. Additionally, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169), and [SP1](https://www.ncbi.nlm.nih.gov/gene/6667) were each found to be regulating in 2 experiments.




The gene CEACAM4 has been associated with [Low density lipoprotein cholesterol levels](https://pubmed.ncbi.nlm.nih.gov/34887591), which maps closely to the MeSH term [Lipoproteins](https://meshb.nlm.nih.gov/record/ui?ui=D008074) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method.


The gene is expressed in multiple tissue samples, including lung, blood, and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Whole Blood, Temporal Lobe, Bone Marrow, Superior Cervical Ganglion, Trigeminal Ganglion, Skeletal Muscle, and Uterus Corpus.




The protein sequence analyzed has a GRAVY value of -0.0213 (82.97th percentile), indicating a slightly hydrophilic nature. At pH 7.0, the protein carries a charge of -3.81 (37.04th percentile). The median structural flexibility is 0.994 (19.32nd percentile), with a propensity for secondary structures as follows: 27.05% helix (12.01st percentile), 37.70% sheet (76.97th percentile), and 32.38% turn (78.65th percentile). The instability index is 45.82 (42.24th percentile), and the isoelectric point is 6.03 (31.37th percentile). The protein has a length of 244 amino acids (20.78th percentile) and a molecular weight of 25908.97 Da (19.25th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |