---
layout: post
title: INPP4A
date: 2025-01-17 16:55 CST
description: INPP4A description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3631) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3631  | INPP4A | ENSG00000040933 | 2q11.2  |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is involved in [phosphatidylinositol biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006661), [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), [phosphatidylinositol-3-phosphate biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0036092), and [inositol phosphate metabolic process](https://amigo.geneontology.org/amigo/term/GO:0043647). It is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [postsynaptic density](https://amigo.geneontology.org/amigo/term/GO:0014069), [early endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0031901), [nuclear membrane](https://amigo.geneontology.org/amigo/term/GO:0031965), and [recycling endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0055038). Additionally, it enables [phosphatidylinositol-3,4-bisphosphate 4-phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0016316), [inositol-1,3,4-trisphosphate 4-phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0017161), and [inositol-3,4-bisphosphate 4-phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0052828).


The gene length is 7,395 base pairs (19.97% of all genes), the mature length is 893 base pairs, and the primary transcript length is 7,286 base pairs.


The gene INPP4A (NCBI ID: 3631) has been mentioned in [11 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22INPP4A%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1999 and the middle 50% of publications occurring between 2016 and 2018. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning INPP4A include "[miR-1290 Contributes to Colorectal Cancer Cell Proliferation by Targeting INPP4B.](https://pubmed.ncbi.nlm.nih.gov/28915933)" (2018) (relative citation ratio: 1.43), "[MicroRNA-940 Targets INPP4A or GSK3β and Activates the Wnt/β-Catenin Pathway to Regulate the Malignant Behavior of Bladder Cancer Cells.](https://pubmed.ncbi.nlm.nih.gov/28337959)" (2018) (relative citation ratio: 1.21), "[Overexpression of miR-4443 promotes the resistance of non-small cell lung cancer cells to epirubicin by targeting INPP4A and regulating the activation of JAK2/STAT3 pathway.](https://pubmed.ncbi.nlm.nih.gov/30001772)" (2018) (relative citation ratio: 1.09), "[The hMeDIP-Seq identified\xa0<i>INPP4A</i> as a novel biomarker for eosinophilic chronic rhinosinusitis with nasal polyps.](https://pubmed.ncbi.nlm.nih.gov/35765979)" (2022) (relative citation ratio: 0.7), and "[Whole exome sequencing identified a novel nonsense INPP4A mutation in a family with intellectual disability.](https://pubmed.ncbi.nlm.nih.gov/31978615)" (2020) (relative citation ratio: 0.52). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[INPP4A](https://www.proteinatlas.org/ENSG00000040933-INPP4A) is a hydrolase with evidence at the protein level. It is detected in all RNA tissues and is primarily localized in the nucleoplasm and nuclear membrane. The gene is expressed in various clusters, including Cluster 10 for blood expression related to membrane trafficking, Cluster 22 for tissue expression involved in brain synaptic signal transduction, Cluster 24 for brain expression in neurons with mixed functions, Cluster 51 for cell line expression in myeloma related to immunoglobulins, and Cluster 28 for single cell expression in dendritic cells involved in antigen presentation.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [E2F1](https://www.ncbi.nlm.nih.gov/gene/1869) with 5 experiments, [SMAD4](https://www.ncbi.nlm.nih.gov/gene/4089) with 4 experiments, [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613) with 4 experiments, [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227) with 4 experiments, and [YY1](https://www.ncbi.nlm.nih.gov/gene/7528) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with various disease conditions, including metabolic disorders such as diabetes mellitus and type 2 diabetes mellitus, as well as kidney and urinary system diseases. Additionally, the data highlights links to cognitive and mental health disorders, including bipolar disorder, mood disorder, psychotic disorder, and schizophrenia.


The gene INPP4A has been associated with the trait [Height](https://pubmed.ncbi.nlm.nih.gov/36224396) through genome-wide association studies.


The gene is expressed in various tissues, including the brain, blood, and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the retina, 721_B_lymphoblasts, BDCA4+ Dentritic Cells, CD56+ NK Cells, CD4+ T cells, CD8+ T cells, Amygdala, and Fetal brain.


The input data highlights various pathways involved in metabolism, particularly focusing on lipid and lipoprotein metabolism. Key processes include phospholipid metabolism and inositol phosphate metabolism. Specifically, the synthesis of IP2, IP, and Ins in the cytosol, as well as PI metabolism, are noted. Additionally, the synthesis of PIPs at both the plasma membrane and the early endosome membrane are significant components of these metabolic pathways.



The analyzed protein sequence has a GRAVY value of -0.327, indicating a hydrophilic nature (54.84th percentile). At pH 7.0, the protein carries a charge of -4.35 (35.4th percentile). The median structural flexibility is 0.999 (36.92nd percentile), with a secondary structure composition of 31.83% helix (48.89th percentile), 35.01% sheet (59.22nd percentile), and 26.51% turn (33.76th percentile). The instability index is 50.93 (60.24th percentile), and the isoelectric point is 6.53 (43.35th percentile). The protein is 977 amino acids long (87.01st percentile) with a molecular weight of 109954.53 Da (87.23rd percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |