---
layout: post
title: RAP2A
date: 2025-01-17 16:55 CST
description: RAP2A description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5911) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5911  | RAP2A | ENSG00000125249 | 13q32.1 |



The gene enables [magnesium ion binding](https://amigo.geneontology.org/amigo/term/GO:0000287) and [GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0003924), and is involved in [positive regulation of protein phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0001934). It is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829) and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), and is active in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886) and [Schaffer collateral - CA1 synapse](https://amigo.geneontology.org/amigo/term/GO:0098685). The gene also enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [GTP binding](https://amigo.geneontology.org/amigo/term/GO:0005525), and [GDP binding](https://amigo.geneontology.org/amigo/term/GO:0019003). Additionally, it is involved in various processes such as [protein localization](https://amigo.geneontology.org/amigo/term/GO:0008104), [microvillus assembly](https://amigo.geneontology.org/amigo/term/GO:0030033), [actin cytoskeleton organization](https://amigo.geneontology.org/amigo/term/GO:0030036), [negative regulation of cell migration](https://amigo.geneontology.org/amigo/term/GO:0030336), [Rap protein signal transduction](https://amigo.geneontology.org/amigo/term/GO:0032486), [regulation of JNK cascade](https://amigo.geneontology.org/amigo/term/GO:0046328), [regulation of dendrite morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0048814), [regulation of synapse assembly](https://amigo.geneontology.org/amigo/term/GO:0051963), [cellular response to xenobiotic stimulus](https://amigo.geneontology.org/amigo/term/GO:0071466), and [protein localization to plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0072659). The gene is also located in the [midbody](https://amigo.geneontology.org/amigo/term/GO:0030496), [recycling endosome](https://amigo.geneontology.org/amigo/term/GO:0055037), [recycling endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0055038), and [synaptic membrane](https://amigo.geneontology.org/amigo/term/GO:0097060). Furthermore, it is involved in the [regulation of postsynaptic membrane neurotransmitter receptor levels](https://amigo.geneontology.org/amigo/term/GO:0099072).


The gene length is 1700.0 base pairs (4.46% of all genes), the mature length is 1085.0 base pairs, and the primary transcript length is 1700.0 base pairs.


RAP2A (Gene ID: 5911) has been mentioned in [75 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RAP2A%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest mention in 1988. The middle 50% of these publications were published between 1999 and 2017. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning RAP2A, ranked by their scientific influence, include "[Human cDNAs rap1 and rap2 homologous to the Drosophila gene Dras3 encode proteins closely related to ras in the 'effector' region.](https://pubmed.ncbi.nlm.nih.gov/3045729)" (1988) (relative citation ratio: 6.18), "[Regulation of Rap2A by the ubiquitin ligase Nedd4-1 controls neurite development.](https://pubmed.ncbi.nlm.nih.gov/20159449)" (2010) (relative citation ratio: 3.16), "[Rap2-JNK removes synaptic AMPA receptors during depotentiation.](https://pubmed.ncbi.nlm.nih.gov/15953419)" (2005) (relative citation ratio: 3.07), "[Association of Rap1a and Rap1b proteins with late endocytic/phagocytic compartments and Rap2a with the Golgi complex.](https://pubmed.ncbi.nlm.nih.gov/7962206)" (1994) (relative citation ratio: 2.74), and "[Rap2A links intestinal cell polarity to brush border formation.](https://pubmed.ncbi.nlm.nih.gov/22797597)" (2012) (relative citation ratio: 2.01). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RAP2A](https://www.proteinatlas.org/ENSG00000125249-RAP2A) is a gene encoding a protein with hydrolase molecular function, supported by evidence at the protein level. Its RNA is detected in all tissues, with notable expression in the brain, particularly in neuronal signaling clusters. The gene is also expressed in smooth muscle cells, where it is involved in muscle contraction. In terms of blood expression, RAP2A is found in non-specific mitochondria clusters. However, its expression is not prognostic for various cancers, including bladder urothelial carcinoma, breast invasive carcinoma, and others.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613) with 6 experiments, [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861) and [E2F1](https://www.ncbi.nlm.nih.gov/gene/1869) each with 5 experiments, and [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025) and [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930) each with 4 experiments.



The input data indicates associations with various metabolic and anatomical diseases, including diseases of metabolism, acquired metabolic diseases, carbohydrate metabolism diseases, glucose metabolism diseases, diabetes mellitus, and specifically type 2 diabetes mellitus. Additionally, the data highlights associations with kidney diseases and urinary system diseases.



The gene is ubiquitously expressed in multiple tissue samples, including nerve, blood vessel, and brain. Additionally, it shows significant expression in various regions of the brain such as the Prefrontal Cortex, Fetal brain, Subthalamic Nucleus, Temporal Lobe, Parietal Lobe, Occipital Lobe, and Globus Pallidus.




The analyzed protein sequence has a GRAVY value of -0.2803 (60.08th percentile), indicating a hydrophilic nature. The charge at pH 7 is -8.497 (25.26th percentile), and the median structural flexibility is 1.004 (58.73rd percentile). The protein has a propensity for helix, sheet, and turn structures of 30.6%, 35.52%, and 27.32% respectively, with corresponding percentiles of 37.19, 62.93, and 41.03. The instability index is 44.34 (36.92nd percentile), and the isoelectric point is 4.73 (3.42nd percentile). The protein length is 183 amino acids (11.98th percentile), with a molecular weight of 20615.23 Da (12.13th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |