---
layout: post
title: ITGA4
date: 2025-01-17 16:55 CST
description: ITGA4 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3676) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3676  | ITGA4 | ENSG00000115232 | 2q31.3 |



The gene enables [fibronectin binding](https://amigo.geneontology.org/amigo/term/GO:0001968) and [integrin binding](https://amigo.geneontology.org/amigo/term/GO:0005178), and is involved in various processes such as [immune response in gut-associated lymphoid tissue](https://amigo.geneontology.org/amigo/term/GO:0002387), [cell-matrix adhesion involved in ameboidal cell migration](https://amigo.geneontology.org/amigo/term/GO:0003366), and [integrin-mediated signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007229). It is located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925), and [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986), and is part of the [integrin complex](https://amigo.geneontology.org/amigo/term/GO:0008305). Additionally, it contributes to [C-X3-C chemokine binding](https://amigo.geneontology.org/amigo/term/GO:0019960) and enables [coreceptor activity](https://amigo.geneontology.org/amigo/term/GO:0015026) and [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872). The gene is also involved in various cellular processes such as [B cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0030183), [cell adhesion mediated by integrin](https://amigo.geneontology.org/amigo/term/GO:0033627), and [heterotypic cell-cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0034113). Furthermore, it plays a role in [endodermal cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0035987), [axonogenesis involved in innervation](https://amigo.geneontology.org/amigo/term/GO:0060385), and [cellular response to cytokine stimulus](https://amigo.geneontology.org/amigo/term/GO:0071345). The gene is also found in the [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062) and is active on the [external side of the plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009897).


The gene length is 3,794 base pairs (10.54% of all genes), the mature length is 1,227 base pairs (29.83% of all genes), and the primary transcript length is 3,794 base pairs (14.05% of all genes).


The gene ITGA4 (NCBI ID: 3676) has been mentioned in [167 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ITGA4%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1976, and the middle 50% of publications occurred between 2006 and 2017.


The top 5 publications mentioning ITGA4, ranked by their scientific influence, include "[A new I subregion (I-J) marked by a locus (Ia-4) controlling surface determinants on suppressor T lymphocytes.](https://pubmed.ncbi.nlm.nih.gov/1085338)" (1976) with a [relative citation ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 10.27, "[Adhesion through the LFA-1 (CD11a/CD18)-ICAM-1 (CD54) and the VLA-4 (CD49d)-VCAM-1 (CD106) pathways prevents apoptosis of germinal center B cells.](https://pubmed.ncbi.nlm.nih.gov/7511659)" (1994) with an RCR of 4.98, "[Relevance of CD49d protein expression as overall survival and progressive disease prognosticator in chronic lymphocytic leukemia.](https://pubmed.ncbi.nlm.nih.gov/17959854)" (2008) with an RCR of 4.24, "[Transmembrane-4 superfamily proteins CD81 (TAPA-1), CD82, CD63, and CD53 specifically associated with integrin alpha 4 beta 1 (CD49d/CD29).](https://pubmed.ncbi.nlm.nih.gov/8757325)" (1996) with an RCR of 3.84, and "[CD49d is the strongest flow cytometry-based predictor of overall survival in chronic lymphocytic leukemia.](https://pubmed.ncbi.nlm.nih.gov/24516016)" (2014) with an RCR of 3.72. The relative citation ratio measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ITGA4](https://www.proteinatlas.org/ENSG00000115232-ITGA4) is an integrin receptor with evidence at the protein level. It is involved in cancer-related genes and is an FDA-approved drug target. ITGA4 is detected in many tissues and is primarily located in the plasma membrane. Its expression is associated with lymphoid tissue and immune response, as well as NK-cells and T-cells in single-cell expression clusters. Additionally, ITGA4 is found in leukemia cell lines and is involved in hemostasis.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313) with 6 experiments, [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930) with 5 experiments, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 5 experiments, [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 5 experiments, and [MYC](https://www.ncbi.nlm.nih.gov/gene/4609) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The input data from GWAS indicates associations with a wide range of disease conditions, including various types of cancer such as male reproductive organ cancer, testicular cancer, and organ system cancer. Additionally, infectious diseases, particularly those caused by viral agents like human immunodeficiency virus, are also implicated. Cardiovascular diseases, including coronary artery disease, artery disease, and myocardial infarction, are notably present. Neurodegenerative diseases, such as Alzheimer's disease, Parkinson's disease, and other forms of dementia, are also linked. Furthermore, the data highlights associations with musculoskeletal system diseases, including arthritis and osteoarthritis, as well as immune system diseases and hematopoietic system diseases.


ITGA4 has been associated with several traits and diseases through genome-wide association studies. Notably, it has been linked to [Drinks per week](https://pubmed.ncbi.nlm.nih.gov/36477530), which was mapped to the MeSH term [Beverages](https://meshb.nlm.nih.gov/record/ui?ui=D001628) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method. Additionally, ITGA4 is associated with [High light scatter reticulocyte count](https://pubmed.ncbi.nlm.nih.gov/32888494) and [High light scatter reticulocyte percentage of red cells](https://pubmed.ncbi.nlm.nih.gov/32888494), mapped to the MeSH terms [Reticulocytes](https://meshb.nlm.nih.gov/record/ui?ui=D012156) and [Cells](https://meshb.nlm.nih.gov/record/ui?ui=D002477), respectively. Furthermore, ITGA4 is linked to the [Immature fraction of reticulocytes](https://pubmed.ncbi.nlm.nih.gov/27863252) and [Monocyte count](https://pubmed.ncbi.nlm.nih.gov/32888494), both mapped to [Reticulocytes](https://meshb.nlm.nih.gov/record/ui?ui=D012156) and [Monocytes](https://meshb.nlm.nih.gov/record/ui?ui=D009000), respectively. Lastly, ITGA4 is associated with [Monocyte percentage of white cells](https://pubmed.ncbi.nlm.nih.gov/32888494), also mapped to [Monocytes](https://meshb.nlm.nih.gov/record/ui?ui=D009000).


The gene is expressed in various immune cell types, including CD34+ cells, BDCA4+ dendritic cells, CD56+ NK cells, CD4+ T cells, CD8+ T cells, and CD71+ early erythroid cells. Additionally, expression is observed in leukemia lymphoblastic cells (MOLT-4) and in tissue samples from the blood and spleen.


The proteins are involved in several key pathways, including the adaptive immune system, immune system, immunoregulatory interactions between lymphoid and non-lymphoid cells, extracellular matrix organization, integrin cell surface interactions, cell surface interactions at the vascular wall, and hemostasis.



The protein sequence analyzed has a GRAVY value of -0.237 (65.37th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -12.33 (18.91st percentile). The median structural flexibility is 0.999 (35.77th percentile), with a secondary structure composition of 28.68% helix (21.35th percentile), 36.24% sheet (67.97th percentile), and 30.62% turn (68.23rd percentile). The instability index is 36.08 (12.24th percentile), and the isoelectric point is 6.05 (31.78th percentile). The protein has a length of 1032 amino acids (88.31st percentile) and a molecular weight of 114898.34 Da (88.33rd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |