---
layout: post
title: PTPN11
date: 2025-01-17 16:55 CST
description: PTPN11 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5781) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5781  | PTPN11 | ENSG00000179295 | 12q24.13 |



The gene is involved in [DNA damage checkpoint signaling](https://amigo.geneontology.org/amigo/term/GO:0000077) and enables [phosphotyrosine residue binding](https://amigo.geneontology.org/amigo/term/GO:0001784). It also enables [phosphoprotein phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0004721), [protein tyrosine phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0004725), and [non-membrane spanning protein tyrosine phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0004726). Additionally, it enables [insulin receptor binding](https://amigo.geneontology.org/amigo/term/GO:0005158) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), and is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737) and located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). It is involved in various processes such as [protein dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0006470), [triglyceride metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006641), [epidermal growth factor receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007173), [integrin-mediated signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007229), [axonogenesis](https://amigo.geneontology.org/amigo/term/GO:0007409), [brain development](https://amigo.geneontology.org/amigo/term/GO:0007420), and [heart development](https://amigo.geneontology.org/amigo/term/GO:0007507). Furthermore, it enables [protein kinase binding](https://amigo.geneontology.org/amigo/term/GO:0019901) and [signaling receptor complex adaptor activity](https://amigo.geneontology.org/amigo/term/GO:0030159), and is involved in [platelet formation](https://amigo.geneontology.org/amigo/term/GO:0030220) and [T cell costimulation](https://amigo.geneontology.org/amigo/term/GO:0031295). The gene is also part of a [protein-containing complex](https://amigo.geneontology.org/amigo/term/GO:0032991) and enables [cadherin binding](https://amigo.geneontology.org/amigo/term/GO:0045296) and [cell adhesion molecule binding](https://amigo.geneontology.org/amigo/term/GO:0050839). Additionally, it enables [molecular adaptor activity](https://amigo.geneontology.org/amigo/term/GO:0060090) and [protein tyrosine kinase binding](https://amigo.geneontology.org/amigo/term/GO:1990782).


The gene length is 113,998 base pairs (84.22% of all genes), the mature length is 1,201 base pairs (29.03% of all genes), and the primary transcript length is 52,252 base pairs (76.32% of all genes).


The gene PTPN11 (NCBI ID: 5781) has been mentioned in [1,134 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PTPN11%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning this gene is 1982, and the middle 50% of publications occurred between 2006 and 2019.


The top 5 publications mentioning PTPN11, ranked by their scientific influence, include "[Allosteric inhibition of SHP2 phosphatase inhibits cancers driven by receptor tyrosine kinases.](https://pubmed.ncbi.nlm.nih.gov/27362227)" (2016) (relative citation ratio: 18.26), "[SHP-1 and SHP-2 associate with immunoreceptor tyrosine-based switch motif of programmed death 1 upon primary human T cell stimulation, but only receptor ligation prevents T cell activation.](https://pubmed.ncbi.nlm.nih.gov/15240681)" (2004) (relative citation ratio: 14.4), "[Somatic mutations in PTPN11 in juvenile myelomonocytic leukemia, myelodysplastic syndromes and acute myeloid leukemia.](https://pubmed.ncbi.nlm.nih.gov/12717436)" (2003) (relative citation ratio: 13.18), "[Crystal structure of the tyrosine phosphatase SHP-2.](https://pubmed.ncbi.nlm.nih.gov/9491886)" (1998) (relative citation ratio: 12.49), and "[PTPN11 mutations in Noonan syndrome: molecular spectrum, genotype-phenotype correlation, and phenotypic heterogeneity.](https://pubmed.ncbi.nlm.nih.gov/11992261)" (2002) (relative citation ratio: 9.64). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PTPN11](https://www.proteinatlas.org/ENSG00000179295-PTPN11) is a gene encoding a protein with hydrolase and protein phosphatase molecular functions. It is implicated in various diseases, including cancer and deafness, with evidence supporting its role at the protein level. The gene is expressed in all tissues and is localized to the nucleoplasm, nucleoli rim, actin filaments, and cytosol. Its expression is associated with oligodendrocytes in tissue and brain clusters, and with smooth muscle cells in single-cell expression clusters. The gene's expression in blood is non-specific with an unknown function.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [MAX](https://www.ncbi.nlm.nih.gov/gene/4149) and [TP53](https://www.ncbi.nlm.nih.gov/gene/7157), each with 6 experiments. Additionally, [TAL1](https://www.ncbi.nlm.nih.gov/gene/6886), [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227), and [GATA1](https://www.ncbi.nlm.nih.gov/gene/2623) were each found to be regulating in 5 experiments.



The GWAS data indicates associations with a wide range of disease conditions, including hematopoietic system disease, immune system disease, and various metabolic disorders such as diabetes mellitus, particularly type 2 diabetes mellitus. Additionally, the data highlights associations with cardiovascular system diseases, including coronary artery disease, hypertension, and myocardial infarction. Other notable conditions include kidney disease, urinary system disease, and specific inherited metabolic disorders like lipid storage disease and lysosomal storage disease. Furthermore, the data points to associations with endocrine system diseases, specifically thyroid gland disease and hypothyroidism.



The gene is expressed in adipose tissue and brain according to BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles indicate expression in the Subthalamic Nucleus, Occipital Lobe, Pons, Appendix, and Superior Cervical Ganglion.


The input data includes a diverse set of proteins participating in various pathways from Reactome. These pathways encompass cytokine signaling in the immune system, interferon signaling (including interferon gamma and alpha/beta signaling), regulation of IFNA and IFNG signaling, and activation of IRF3/IRF7 mediated by TBK1/IKK epsilon. Additionally, the data covers FGF signaling regulation by Spry, ARMS-mediated activation, Frs2-mediated activation, and prolonged ERK activation events. Signaling pathways involving p38 via RIT and RIN, GRB2 and SHC1 events in EGFR and ERBB2 signaling, RAF/MAP kinase cascade, and SOS-mediated signaling are also included. The data highlights signaling by leptin, ERK activation, interleukin-6 signaling, Tie2 signaling, and FRS2-mediated cascade. Furthermore, the adaptive, innate, and general immune systems are represented, along with PI3K/AKT signaling in cancer, GAB1 signalosome, and PI-3K cascade. The data also includes signaling by various receptors such as SCF-KIT, ERBB4, DAP12, PDGF, FGFR, EGFR, NGF, FCERI, BCR, IGF1R, insulin receptor, VEGF, and TLRs. Additionally, the data covers costimulation by the CD28 family, interleukin signaling, activated TLR4 signaling, and various toll-like receptor cascades. Other pathways include TRAF6-mediated induction of NFkB and MAP kinases, CTLA4 inhibitory signaling, platelet sensitization by LDL, nuclear signaling by ERBB4, prolactin receptor signaling, axon guidance, developmental biology, NCAM signaling for neurite out-growth, netrin-1 signaling, PECAM1 interactions, SIRP family interactions, FCERI-mediated MAPK activation, PD-1 signaling, disease, gastrin-CREB signaling pathway via PKC and MAPK, signaling by GPCR, signal transduction, platelet homeostasis, cell-cell communication, cell surface interactions at the vascular wall, and hemostasis.



The analyzed protein sequence has a GRAVY value of -0.735 (15.41% percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -0.743 (47.57% percentile). The median structural flexibility is 1.007 (71.7% percentile), with a secondary structure composition of 29.34% helix (26.14% percentile), 33.90% sheet (51.21% percentile), and 27.32% turn (40.95% percentile). The instability index is 43.07 (32.45% percentile), and the isoelectric point is 6.87 (48.78% percentile). The protein is 593 amino acids long (68.0% percentile) with a molecular weight of 68010 Da (69.23% percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |