---
layout: post
title: HNRNPM
date: 2025-01-17 16:55 CST
description: HNRNPM description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4670) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4670  | HNRNPM | ENSG00000099783 | 19p13.2 |



The gene is involved in [alternative mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0000380) and [mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0000398). It enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [mRNA binding](https://amigo.geneontology.org/amigo/term/GO:0003729), and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), as well as [protein domain specific binding](https://amigo.geneontology.org/amigo/term/GO:0019904). The gene is active in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [nucleolus](https://amigo.geneontology.org/amigo/term/GO:0005730), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [nuclear matrix](https://amigo.geneontology.org/amigo/term/GO:0016363), [paraspeckles](https://amigo.geneontology.org/amigo/term/GO:0042382), [synapse](https://amigo.geneontology.org/amigo/term/GO:0045202), [collagen-containing extracellular matrix](https://amigo.geneontology.org/amigo/term/GO:0062023), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). Additionally, it is part of the [spliceosomal complex](https://amigo.geneontology.org/amigo/term/GO:0005681) and the [catalytic step 2 spliceosome](https://amigo.geneontology.org/amigo/term/GO:0071013).


The gene length is 668.0 base pairs (0.78% of all genes), with a mature length of 668.0 base pairs and a primary transcript length of 668.0 base pairs.


The gene HNRNPM (NCBI ID: 4670) has been mentioned in [39 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22HNRNPM%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1993 and the middle 50% of publications occurring between 2010 and 2019. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning HNRNPM, ranked by their relative citation ratio (RCR), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows:

1. [Cell type-restricted activity of hnRNPM promotes breast cancer metastasis via regulating alternative splicing.](https://pubmed.ncbi.nlm.nih.gov/24840202) (2014) (relative citation ratio: 4.81)
2. [*CircURI1* interacts with hnRNPM to inhibit metastasis by modulating alternative splicing in gastric cancer.](https://pubmed.ncbi.nlm.nih.gov/34385309) (2021) (relative citation ratio: 4.39)
3. [DAR, a new RhD variant involving exons 4, 5, and 7, often in linkage with ceAR, a new Rhce variant frequently found in African blacks.](https://pubmed.ncbi.nlm.nih.gov/10590079) (1999) (relative citation ratio: 2.71)
4. [Cancer-Associated MORC2-Mutant M276I Regulates an hnRNPM-Mediated CD44 Splicing Switch to Promote Invasion and Metastasis in Triple-Negative Breast Cancer.](https://pubmed.ncbi.nlm.nih.gov/30093560) (2018) (relative citation ratio: 2.52)
5. [Targeting HNRNPM Inhibits Cancer Stemness and Enhances Antitumor Immunity in Wnt-activated Hepatocellular Carcinoma.](https://pubmed.ncbi.nlm.nih.gov/35158098) (2022) (relative citation ratio: 2.02)

The citation count is sourced from [iCite](https://icite.od.nih.gov). For more information on the Relative Citation Ratio (RCR), please refer to [this article](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541).


[HNRNPM](https://www.proteinatlas.org/ENSG00000099783-HNRNPM) is a ribonucleoprotein with RNA-binding molecular function, supported by evidence at the protein level. It is detected in all RNA tissue distributions and is primarily located in the nucleoplasm. The gene is expressed in various clusters related to basic cellular processes, including mRNA splicing, cell cycle, and transcription, across different tissue types and cell lines. Notably, it is part of Cluster 12 for blood expression, which is non-specific and related to mitochondrial translation, and Cluster 40 for tissue expression, which is also non-specific and related to transcription.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227), [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908), [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), and [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613), each shown to be regulating in 5 experiments.



The input data indicates associations with various disease conditions, including cancer, particularly lung and respiratory system cancers, as well as cognitive and mental health disorders such as bipolar and mood disorders.



The gene is expressed in various tissues, including the ovary and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in multiple cell types such as 721_B_lymphoblasts, CD34+ cells, CD33+ myeloid cells, BDCA4+ dendritic cells, CD4+ T cells, CD8+ T cells, and CD105+ endothelial cells.


The proteins are involved in several key processes, including gene expression, processing of capped intron-containing pre-mRNA, and mRNA splicing, with a particular focus on the major pathway of mRNA splicing.


The mouse ortholog gene 76936 and its human ortholog gene 4670 are associated with the phenotype of preweaning lethality with complete penetrance, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.342 (53.15th percentile), indicating a hydrophilic nature. It exhibits a charge of 6.48 at pH 7.0 (72.56th percentile) and a median structural flexibility of 1.007 (69.98th percentile). The protein's secondary structure is predicted to be 36.6% helix (83.45th percentile), 22.7% sheet (4.53rd percentile), and 35.1% turn (89.47th percentile). The instability index is 39.7 (21.27th percentile), with an isoelectric point of 8.84 (75.01st percentile). The protein has a length of 730 amino acids (77.12th percentile) and a molecular weight of 77514.86 Da (74.96th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |