---
layout: post
title: CD58
date: 2025-01-17 16:55 CST
description: CD58 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/965) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 965  | CD58 | ENSG00000116815 | 1p13.1 |



The gene enables [signaling receptor binding](https://amigo.geneontology.org/amigo/term/GO:0005102) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [secretory granule membrane](https://amigo.geneontology.org/amigo/term/GO:0030667), [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062), and [ficolin-1-rich granule membrane](https://amigo.geneontology.org/amigo/term/GO:0101003). The gene is involved in the [positive regulation of interleukin-8 production](https://amigo.geneontology.org/amigo/term/GO:0032757), [heterotypic cell-cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0034113), [cellular response to type II interferon](https://amigo.geneontology.org/amigo/term/GO:0071346), [cellular response to tumor necrosis factor](https://amigo.geneontology.org/amigo/term/GO:0071356), and [cell-cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0098609).


The gene length is 17,048 base pairs (38.09% of all genes), the mature length is 2,526.5 base pairs (64.37% of all genes), and the primary transcript length is 15,901.5 base pairs (45.64% of all genes).


The gene CD58, with NCBI ID 965, has been mentioned in [161 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CD58%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication mentioning CD58 is 1982, and the middle 50% of publications occurred between 1991 and 1998.


The top 5 publications mentioning CD58, ranked by their scientific influence, include "[The lymphocyte function-associated LFA-1, CD2, and LFA-3 molecules: cell adhesion receptors of the immune system.](https://pubmed.ncbi.nlm.nih.gov/3109455)" (1987) (relative citation ratio: 28.96), "[Human memory T lymphocytes express increased levels of three cell adhesion molecules (LFA-3, CD2, and LFA-1) and three other molecules (UCHL1, CDw29, and Pgp-1) and have enhanced IFN-gamma production.](https://pubmed.ncbi.nlm.nih.gov/2894392)" (1988) (relative citation ratio: 23.07), "[Three distinct antigens associated with human T-lymphocyte-mediated cytolysis: LFA-1, LFA-2, and LFA-3.](https://pubmed.ncbi.nlm.nih.gov/6984191)" (1982) (relative citation ratio: 18.11), "[An LFA-3 cDNA encodes a phospholipid-linked membrane protein homologous to its receptor CD2.](https://pubmed.ncbi.nlm.nih.gov/3313052)" (1987) (relative citation ratio: 13.74), and "[Differential costimulatory effects of adhesion molecules B7, ICAM-1, LFA-3, and VCAM-1 on resting and antigen-primed CD4+ T lymphocytes.](https://pubmed.ncbi.nlm.nih.gov/1372018)" (1992) (relative citation ratio: 9.48). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CD58](https://www.proteinatlas.org/ENSG00000116815-CD58) is a gene with evidence at the protein level. Its RNA is detected in all tissues, and it is highly expressed in monocytes and neutrophils, playing a role in the innate immune response. In tissues, CD58 is predominantly found in lymphoid tissue and bone marrow, again associated with the innate immune response. In the brain, it is expressed in macrophages and microglia, contributing to the immune response. Additionally, CD58 is expressed in lymphoma cell lines and Langerhans cells, further highlighting its immune-related functions.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [MYC](https://www.ncbi.nlm.nih.gov/gene/4609), [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861), [TP53](https://www.ncbi.nlm.nih.gov/gene/7157), [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169), and [JUN](https://www.ncbi.nlm.nih.gov/gene/3725). Each of these TFs, except for JUN, was shown to be regulating in 5 experiments, while JUN was found in 4 experiments.



The input data indicates associations with various disease conditions, including metabolic disorders such as diseases of metabolism, acquired metabolic diseases, carbohydrate metabolism diseases, and glucose metabolism diseases, with a specific focus on diabetes mellitus. Additionally, the data highlights nervous system diseases, particularly central nervous system diseases, neurodegenerative diseases, demyelinating diseases, and specifically multiple sclerosis.



The gene is expressed in various tissues, including the salivary gland, lung, and blood. Additionally, it is expressed in multiple cell types such as B lymphoblasts, CD33+ myeloid cells, CD56+ NK cells, CD14+ monocytes, whole blood, CD105+ endothelial cells, CD71+ early erythroid cells, and colorectal adenocarcinoma.


The proteins are involved in cell surface interactions at the vascular wall and hemostasis.



The protein sequence analyzed has a GRAVY value of -0.05 (81.38th percentile), indicating a slightly hydrophilic nature. The charge at pH 7 is -2.06 (42.96th percentile), and the median structural flexibility is 0.994 (18.96th percentile). The protein has a propensity for secondary structures with 26% helix (7.68th percentile), 40% sheet (86.16th percentile), and 30.4% turn (66.73th percentile). The instability index is 38.8 (18.87th percentile), and the isoelectric point is 6.3 (38.03rd percentile). The sequence length is 250 amino acids (21.69th percentile), with a molecular weight of 28147.1 Da (22.06th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |