---
layout: post
title: TST
date: 2025-01-17 16:55 CST
description: TST description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7263) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7263  | TST | ENSG00000128311 | 22q12.3 |



The gene is involved in several biological processes, including the [sulfur amino acid catabolic process](https://amigo.geneontology.org/amigo/term/GO:0000098) and the [cyanate catabolic process](https://amigo.geneontology.org/amigo/term/GO:0009440). It enables [thiosulfate sulfurtransferase activity](https://amigo.geneontology.org/amigo/term/GO:0004792) and [3-mercaptopyruvate sulfurtransferase activity](https://amigo.geneontology.org/amigo/term/GO:0016784), and is capable of [5S rRNA binding](https://amigo.geneontology.org/amigo/term/GO:0008097). The gene is located in the [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615), [mitochondrion](https://amigo.geneontology.org/amigo/term/GO:0005739), and specifically within the [mitochondrial matrix](https://amigo.geneontology.org/amigo/term/GO:0005759). Additionally, it is involved in [epithelial cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0030855), [rRNA import into mitochondrion](https://amigo.geneontology.org/amigo/term/GO:0035928), and [rRNA transport](https://amigo.geneontology.org/amigo/term/GO:0051029).


The gene length is 10,686 base pairs (27.06% of all genes), the mature length is 3,843 base pairs, and the primary transcript length is 10,686 base pairs.


The gene TST (NCBI ID: 7263) has been mentioned in [47 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22TST%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1978, and the middle 50% of publications occurred between 2007 and 2017.


The top 5 publications mentioning TST, ranked by their scientific influence, include "[Comparison of TST and IGRA in Diagnosis of Latent Tuberculosis Infection in a High TB-Burden Setting.](https://pubmed.ncbi.nlm.nih.gov/28060926)" (2017) (relative citation ratio: 3.65), "[Purification of thiosulfate sulfurtransferase by selective immobilization on blue agarose.](https://pubmed.ncbi.nlm.nih.gov/655428)" (1978) (relative citation ratio: 3.2), "[Unraveling the role of thiosulfate sulfurtransferase in metabolic diseases.](https://pubmed.ncbi.nlm.nih.gov/32061776)" (2020) (relative citation ratio: 3.08), "[Intracranial Hemorrhage in the TST Trial.](https://pubmed.ncbi.nlm.nih.gov/34963300)" (2022) (relative citation ratio: 2.54), and "[Expression of the cloned toxic shock syndrome toxin 1 gene (tst) in vivo with a rabbit uterine model.](https://pubmed.ncbi.nlm.nih.gov/4044040)" (1985) (relative citation ratio: 2.43). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[TST](https://www.proteinatlas.org/ENSG00000128311-TST) is a gene with molecular functions in RNA-binding and transferase activity. Evidence for its function is available at the protein level. The gene is detected in all RNA tissue distributions and is primarily localized in the mitochondria. In terms of expression, TST is associated with Cluster 38 in blood, which involves monocytes and neutrophils in degranulation, and Cluster 83 in tissue, which is related to liver metabolism. Additionally, it is linked to Cluster 19 in the brain, involving astrocytes with mixed functions, Cluster 57 in cell lines, which includes stomach and colon cancer related to digestion, and Cluster 5 in single cells, involving enterocytes in absorption.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [E2F1](https://www.ncbi.nlm.nih.gov/gene/1869) and [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613), each with 5 experiments, followed by [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 5 experiments, [ZBTB7A](https://www.ncbi.nlm.nih.gov/gene/51341) with 4 experiments, and [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046) with 4 experiments.



The GWAS data indicates associations with various disease conditions, including hematopoietic system disease, immune system disease, and general disease of anatomical entities.


Genome-wide association studies have identified associations with specific metabolite levels. Notably, the trait [Homoarginine levels](https://pubmed.ncbi.nlm.nih.gov/36357675) was mapped to the closest MeSH term [Homoarginine](https://meshb.nlm.nih.gov/record/ui?ui=D006709) using the method [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0). Additionally, the trait [Lysine levels](https://pubmed.ncbi.nlm.nih.gov/33414548) was mapped to the closest MeSH term [Lysine](https://meshb.nlm.nih.gov/record/ui?ui=D008239) using the same method.


The gene is ubiquitously expressed in multiple tissue samples, including the liver, adrenal gland, colon, small intestine, kidney, and fetal liver. Additionally, it is expressed in CD105+ endothelial cells.


The input data highlights several key pathways involved in metabolism, including the degradation of cysteine and homocysteine, sulfide oxidation to sulfate, sulfur amino acid metabolism, and the broader metabolism of amino acids and derivatives.


The mouse gene with NCBI ID 22117, which is orthologous to the human gene with NCBI ID 7263, is associated with several phenotypic abnormalities in male mice. These include abnormal epididymis morphology, abnormal testis morphology, enlarged epididymis, and small testis.


The analyzed protein sequence has a GRAVY value of -0.388 (47.65th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -0.694 (47.72nd percentile), and the median structural flexibility is 1.001 (44.56th percentile). The protein has a propensity for helix, sheet, and turn structures of 31.65% (47.23rd percentile), 34.68% (56.6th percentile), and 28.28% (49.92nd percentile), respectively. The instability index is 49.27 (54.78th percentile), with an isoelectric point of 6.77 (47.42nd percentile). The protein is 297 amino acids long (28.05th percentile) and has a molecular weight of 33428.53 Da (28.47th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |