---
layout: post
title: SPRY2
date: 2025-01-17 16:55 CST
description: SPRY2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10253) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10253  | SPRY2 | ENSG00000136158 | 13q31.1 |



The gene is involved in various biological processes, including the [establishment of mitotic spindle orientation](https://amigo.geneontology.org/amigo/term/GO:0000132), [sensory perception of sound](https://amigo.geneontology.org/amigo/term/GO:0007605), and [negative regulation of cell population proliferation](https://amigo.geneontology.org/amigo/term/GO:0008285). It enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [protein kinase binding](https://amigo.geneontology.org/amigo/term/GO:0019901), and is active in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). The gene is located in multiple cellular components, including the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0005856), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [actin cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0015629), [microtubule cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0015630), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [ruffle membrane](https://amigo.geneontology.org/amigo/term/GO:0032587), and [microtubule end](https://amigo.geneontology.org/amigo/term/GO:1990752). Additionally, it plays roles in the [fibroblast growth factor receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0008543), [positive regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0010628), and [negative regulation of epithelial to mesenchymal transition](https://amigo.geneontology.org/amigo/term/GO:0010719), among other processes.


The gene length is 4,579 base pairs (12.84% of all genes), the mature length is 1,415 base pairs (35.44% of all genes), and the primary transcript length is 4,128 base pairs (15.42% of all genes).


The gene SPRY2, with NCBI ID 10253, has been mentioned in [101 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SPRY2%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication mentioning this gene is 2000, and the middle 50% of publications occurred between 2011 and 2018.


The top 5 publications mentioning SPRY2, ranked by their scientific influence, include "[MicroRNA-21 targets Sprouty2 and promotes cellular outgrowths.](https://pubmed.ncbi.nlm.nih.gov/18508928)" (2008) (relative citation ratio: 6.37), "[miR-21 modulates the ERK-MAPK signaling pathway by regulating SPRY2 expression during human mesenchymal stem cell differentiation.](https://pubmed.ncbi.nlm.nih.gov/23239100)" (2013) (relative citation ratio: 4.16), "[miR-27a regulates the growth, colony formation and migration of pancreatic cancer cells by targeting Sprouty2.](https://pubmed.ncbi.nlm.nih.gov/20638779)" (2010) (relative citation ratio: 3.91), "[c-MYC-regulated miR-23a/24-2/27a cluster promotes mammary carcinoma cell invasion and hepatic metastasis by targeting Sprouty2.](https://pubmed.ncbi.nlm.nih.gov/23649631)" (2013) (relative citation ratio: 3.71), and "[Downregulation of Spry2 by miR-21 triggers malignancy in human gliomas.](https://pubmed.ncbi.nlm.nih.gov/21278789)" (2011) (relative citation ratio: 3.29). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[SPRY2](https://www.proteinatlas.org/ENSG00000136158-SPRY2) is a developmental protein with evidence at the protein level, implicated as a disease variant. It is detected in all tissues and is localized to actin filaments and microtubules. SPRY2 is expressed in NK-cells with unknown function and is involved in non-specific transcription across various tissue types. In skin cancer cell lines, it is associated with extracellular matrix organization, and in single cells, it is expressed in fibroblasts involved in ECM organization.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD9](https://www.ncbi.nlm.nih.gov/gene/65980) with 5 experiments, [INTS13](https://www.ncbi.nlm.nih.gov/gene/55726) with 4 experiments, [MYC](https://www.ncbi.nlm.nih.gov/gene/4609) with 4 experiments, [SMC3](https://www.ncbi.nlm.nih.gov/gene/9126) with 4 experiments, and [TBP](https://www.ncbi.nlm.nih.gov/gene/6908) with 4 experiments.




The gene SPRY2 has been associated with [Height](https://pubmed.ncbi.nlm.nih.gov/36224396) through genome-wide association studies.


The gene is expressed in various tissues, including adipose tissue, breast, and nerve, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the whole brain, cerebellum, cerebellum peduncles, and pancreas.


The input data highlights several pathways involving EGFR and FGFR signaling. These include the downregulation of EGFR, regulation of FGF signaling by Spry, negative regulation of FGFR signaling, and various signaling pathways by FGFR and EGFR, particularly in disease and cancer contexts. Additionally, the data mentions general disease processes and signal transduction.



The analyzed protein sequence has a GRAVY value of -0.667 (19.96th percentile), indicating a hydrophilic nature. It exhibits a charge of 12.15 at pH 7.0 (85.37th percentile) and a median structural flexibility of 1.005 (62.53rd percentile). The protein's secondary structure is predicted to be 22.86% helix (2.33rd percentile), 25.08% sheet (8.89th percentile), and 35.56% turn (90.75th percentile). The instability index is 49.49 (55.56th percentile), with an isoelectric point of 8.78 (73.11th percentile). The protein is 315 amino acids long (31.86th percentile) and has a molecular weight of 34688.05 Da (30.42nd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |