---
layout: post
title: ATP5MC3
date: 2025-01-17 16:55 CST
description: ATP5MC3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/518) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 518  | ATP5MC3 | ENSG00000154518 | 2q31.1 |



The gene is part of the [mitochondrial proton-transporting ATP synthase complex, coupling factor F(o)](https://amigo.geneontology.org/amigo/term/GO:0000276) and the [mitochondrial proton-transporting ATP synthase complex](https://amigo.geneontology.org/amigo/term/GO:0005753). It is located in the [mitochondrial outer membrane](https://amigo.geneontology.org/amigo/term/GO:0005741) and the [mitochondrial inner membrane](https://amigo.geneontology.org/amigo/term/GO:0005743). The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [lipid binding](https://amigo.geneontology.org/amigo/term/GO:0008289), and it has [proton transmembrane transporter activity](https://amigo.geneontology.org/amigo/term/GO:0015078). It is involved in [proton motive force-driven ATP synthesis](https://amigo.geneontology.org/amigo/term/GO:0015986) and [proton transmembrane transport](https://amigo.geneontology.org/amigo/term/GO:1902600).


The gene length is 435,258 base pairs (97.79% of all genes), the mature length is 2,142 base pairs, and the primary transcript length is 238,718 base pairs (96.86% of all genes).


The gene ATP5MC3 (NCBI ID: 518) has been mentioned in [6 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ATP5MC3%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning this gene was in 1992, with the middle 50% of publications occurring between 1995 and 2010.


The top 5 publications mentioning ATP5MC3, ranked by their relative citation ratio (RCR), include studies that explore various aspects of mitochondrial ATP synthase and its subunits. The RCR values measure the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. The most influential publication is "[Interaction of mite allergens Der p3 and Der p9 with protease-activated receptor-2 expressed by lung epithelial cells.](https://pubmed.ncbi.nlm.nih.gov/11441110)" (2001) with a relative citation ratio of 5.37. Other notable publications include "[Rat mitochondrial ATP synthase ATP5G3: cloning and upregulation in pancreas after chronic ethanol feeding.](https://pubmed.ncbi.nlm.nih.gov/11459924)" (2001) with a relative citation ratio of 1.13, and "[Total syntheses of (-)-pyrimidoblamic acid and P-3A.](https://pubmed.ncbi.nlm.nih.gov/24410439)" (2014) with a relative citation ratio of 1.08. Additionally, "[An in vitro system for the editing of ATP synthase subunit 9 mRNA using wheat mitochondrial extracts.](https://pubmed.ncbi.nlm.nih.gov/1531271)" (1992) has a relative citation ratio of 1.01, and "[Sequence analysis and mapping of a novel human mitochondrial ATP synthase subunit 9 cDNA (ATP5G3).](https://pubmed.ncbi.nlm.nih.gov/7698763)" (1994) has a relative citation ratio of 0.87. The citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ATP5MC3](https://www.proteinatlas.org/ENSG00000154518-ATP5MC3) is a gene implicated in several diseases, including dystonia, hereditary spastic paraplegia, and neurodegeneration. Evidence for its involvement is supported at the protein level. The gene's RNA is detected in all tissues, and it is associated with various expression clusters, including Cluster 13 for blood expression (non-specific - mitochondria), Cluster 23 for tissue expression (non-specific - cellular respiration), Cluster 45 for brain expression (non-specific - mixed function), Cluster 26 for cell line expression (non-specific - mitochondria & RNA binding), and Cluster 39 for single cell expression (non-specific - basic cellular processes).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) and [SPI1](https://www.ncbi.nlm.nih.gov/gene/6688), each with 5 experiments. Following closely are [GATA2](https://www.ncbi.nlm.nih.gov/gene/2624) and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), each with 4 experiments. Additionally, [TP53](https://www.ncbi.nlm.nih.gov/gene/7157) was found to be regulating in 3 experiments.





The gene is expressed in various tissues, including the heart, colon, and kidney, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the heart, colon, kidney, small intestine, B lymphoblasts (721_B), and chronic myelogenous leukemia cells (K-562).


The input data highlights several key metabolic processes, including the formation of ATP by chemiosmotic coupling, respiratory electron transport, and the citric acid (TCA) cycle. These processes are integral to cellular metabolism and energy production.


The mouse ortholog gene 228033 and its human ortholog gene 518 are associated with the phenotype of decreased locomotor activity, observed in both sexes.


The analyzed protein sequence exhibits a GRAVY value of 0.573 (96.12th percentile), indicating a high level of hydrophobicity. The protein has a charge of 4.48 at pH 7.0 (66.23rd percentile) and a median structural flexibility of 0.995 (20.36th percentile). The secondary structure affinities are notable, with 32.4% of residues tending to be in a helix (54.15th percentile), 36.6% in a sheet (70.55th percentile), and 28.2% in a turn (48.85th percentile). The instability index is 32.82 (6.81st percentile), and the isoelectric point is 9.57 (92nd percentile). The protein has a length of 142 amino acids (7.15th percentile) and a molecular weight of 14692.91 Da (6.07th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |