---
layout: post
title: POLRMT
date: 2024-12-08 23:57 CST
description: POLRMT description
tags: [cooccuring-genes]
categories: shortest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5442) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5442  | POLRMT | ENSG00000099821 | 19p13.3 |



The gene length is 412.0 base pairs (0.3% of all genes), with a mature length of 412.0 base pairs and a primary transcript length of 412.0 base pairs.


The gene POLRMT, with NCBI ID 5442, has been featured in [15 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22POLRMT%22) (count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1997, and the middle 50% of publications occurred between 2013 and 2019.


The top five publications mentioning POLRMT, ranked by their scientific influence, include "[POLRMT regulates the switch between replication primer formation and gene expression of mammalian mtDNA.](https://pubmed.ncbi.nlm.nih.gov/27532055)" (2016) (relative citation ratio: 2.87), "[Identification of the gene encoding the human mitochondrial RNA polymerase (h-mtRPOL) by cyberscreening of the Expressed Sequence Tags database.](https://pubmed.ncbi.nlm.nih.gov/9097968)" (1997) (relative citation ratio: 2.83), "[POLRMT mutations impair mitochondrial transcription causing neurological disease.](https://pubmed.ncbi.nlm.nih.gov/33602924)" (2021) (relative citation ratio: 2.04), "[TFB2M and POLRMT are essential for mammalian mitochondrial DNA replication.](https://pubmed.ncbi.nlm.nih.gov/34744028)" (2022) (relative citation ratio: 1.5), and "[Structure of human mitochondrial RNA polymerase elongation complex.](https://pubmed.ncbi.nlm.nih.gov/24096365)" (2013) (relative citation ratio: 1.4). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313), [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613), and [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599), each shown to be regulating in 4 experiments.





The genome-wide association study identified a significant association between the POLRMT gene and [Mitochondrial DNA heteroplasmy (chrM:16183:A:ACC case-only heteroplasmy)](https://pubmed.ncbi.nlm.nih.gov/37587338), which was mapped to the closest MeSH term [DNA, Mitochondrial](https://meshb.nlm.nih.gov/record/ui?ui=D004272) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method.


The mouse ortholog gene 216151, which corresponds to the human ortholog gene 5442, is associated with several phenotypic abnormalities. These include abnormal brain and lymph node morphology, embryonic lethality prior to organogenesis, enlarged lymph nodes, hydrocephaly, no spontaneous movement, and preweaning lethality with incomplete penetrance. Most of these phenotypes are observed in both sexes, except for embryonic lethality prior to organogenesis.


The analyzed protein sequence has a GRAVY value of -0.332, indicating a hydrophilic nature (54.16th percentile). It exhibits a charge of 34.95 at pH 7.0 (98.03rd percentile) and a median structural flexibility of 0.998 (34.18th percentile). The protein's secondary structure is predicted to be 35.28% helix (76.48th percentile), 33.09% sheet (46.03rd percentile), and 23.74% turn (13.64th percentile). The instability index is 50.18 (57.91st percentile), with an isoelectric point of 9.19 (84.28th percentile). The protein has a length of 1230 amino acids (92.22nd percentile) and a molecular weight of 138618.79 Da (92.51st percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |