---
layout: post
title: KCNA3
date: 2025-01-17 16:55 CST
description: KCNA3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3738) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3738  | KCNA3 | ENSG00000177272 | 1p13.3 |



The gene enables [voltage-gated monoatomic ion channel activity](https://amigo.geneontology.org/amigo/term/GO:0005244) and [delayed rectifier potassium channel activity](https://amigo.geneontology.org/amigo/term/GO:0005251), and is involved in [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [presynaptic membrane](https://amigo.geneontology.org/amigo/term/GO:0042734), [calyx of Held](https://amigo.geneontology.org/amigo/term/GO:0044305), [membrane raft](https://amigo.geneontology.org/amigo/term/GO:0045121), [postsynaptic membrane](https://amigo.geneontology.org/amigo/term/GO:0045211), and [glutamatergic synapse](https://amigo.geneontology.org/amigo/term/GO:0098978). The gene is part of the [voltage-gated potassium channel complex](https://amigo.geneontology.org/amigo/term/GO:0008076) and is active in the [axon](https://amigo.geneontology.org/amigo/term/GO:0030424). It is involved in [potassium ion transport](https://amigo.geneontology.org/amigo/term/GO:0006813), [optic nerve development](https://amigo.geneontology.org/amigo/term/GO:0021554), [corpus callosum development](https://amigo.geneontology.org/amigo/term/GO:0022038), [protein homooligomerization](https://amigo.geneontology.org/amigo/term/GO:0051260), and [potassium ion transmembrane transport](https://amigo.geneontology.org/amigo/term/GO:0071805). Additionally, the gene enables [outward rectifier potassium channel activity](https://amigo.geneontology.org/amigo/term/GO:0015271).


The gene length is 109,310 base pairs (83.54% of all genes), the mature length is 6,102 base pairs (94.43% of all genes), and the primary transcript length is 109,310 base pairs (89.7% of all genes).


The gene KCNA3 (NCBI ID: 3738) has been mentioned in [340 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22KCNA3%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning this gene is 1989, and the middle 50% of publications occurred between 2007 and 2017.


The top 5 publications mentioning KCNA3, ranked by their scientific influence as measured by the [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541), include: ["Cytochrome P-450 hPCN3, a novel cytochrome P-450 IIIA gene product that is differentially expressed in adult human liver. cDNA and deduced amino acid sequence and distinct specificities of cDNA-expressed hPCN1 and hPCN3 for the metabolism of steroid hormones and cyclosporine."](https://pubmed.ncbi.nlm.nih.gov/2732228) (1989) (relative citation ratio: 19.14), ["Kv1.3 channels are a therapeutic target for T cell-mediated autoimmune diseases."](https://pubmed.ncbi.nlm.nih.gov/17088564) (2006) (relative citation ratio: 9.27), ["Kv1.3 modulates neuroinflammation and neurodegeneration in Parkinson's disease."](https://pubmed.ncbi.nlm.nih.gov/32597830) (2020) (relative citation ratio: 5.22), ["ShK-Dap22, a potent Kv1.3-specific immunosuppressive polypeptide."](https://pubmed.ncbi.nlm.nih.gov/9830012) (1998) (relative citation ratio: 5.04), and ["Differential Kv1.3, KCa3.1, and Kir2.1 expression in 'classically' and 'alternatively' activated microglia."](https://pubmed.ncbi.nlm.nih.gov/27696527) (2017) (relative citation ratio: 4.46). The relative citation ratio measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[KCNA3](https://www.proteinatlas.org/ENSG00000177272-KCNA3) is a gene encoding a protein with molecular functions as an ion channel, specifically a voltage-gated potassium channel. It is an FDA-approved drug target with evidence at the protein level. The gene's RNA is detected in some tissues, and it is expressed in various clusters, including T-cells with unknown function (Cluster 11), spleen for immune response (Cluster 47), neurons with mixed function (Cluster 1), myeloma cells producing immunoglobulins (Cluster 51), and T-cells related to T-cell receptor (Cluster 7).


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 5 experiments, [JARID2](https://www.ncbi.nlm.nih.gov/gene/3720) with 4 experiments, [SUZ12](https://www.ncbi.nlm.nih.gov/gene/23512) with 4 experiments, [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 4 experiments, and [EZH2](https://www.ncbi.nlm.nih.gov/gene/2146) with 3 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.





The gene is expressed in various tissues, including lung, blood, spleen, and small intestine, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Cardiac Myocytes, Heart, CD33+ Myeloid cells, Superior Cervical Ganglion, Ciliary Ganglion, Dorsal Root Ganglion, and Testis Interstitial cells.


The input data highlights the involvement of potassium channels, specifically voltage-gated potassium channels, in the neuronal system.



The analyzed protein sequence has a GRAVY value of -0.121, indicating a hydrophilic nature (76.52 percentile). At pH 7.0, the protein carries a charge of -10.55 (21.66 percentile). The median structural flexibility is 1.0005 (43.84 percentile), with a secondary structure composition of 27.13% helix (12.45 percentile), 36.17% sheet (67.48 percentile), and 30.96% turn (70.3 percentile). The instability index is 58.12 (79.4 percentile), and the isoelectric point is 5.61 (20.85 percentile). The protein has a length of 575 amino acids (66.29 percentile) and a molecular weight of 63841.00 Da (66.16 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |