---
layout: post
title: EEF1D
date: 2024-12-08 23:57 CST
description: EEF1D description
tags: [cooccuring-genes]
categories: longest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1936) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1936  | EEF1D | ENSG00000104529 | 8q24.3 |



The gene length is 1,506,192.0 base pairs (99.92% of all genes), the mature length is 3,664.0 base pairs (80.49% of all genes), and the primary transcript length is 689,568.0 base pairs (99.64% of all genes).


EEF1D has been studied in 21 publications, with the earliest publication in 1990 and the middle 50% of publications occurring between 2003 and 2018. For more information, see the [21 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22EEF1D%22) (note: the count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)).


The top five publications mentioning EEF1D, ranked by their relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)), include studies that explore the interaction of EEF1D with various viral proteins and its role in cellular processes. The most influential study, "[Interaction of herpes simplex virus 1 alpha regulatory protein ICP0 with elongation factor 1delta: ICP0 affects translational machinery.](https://pubmed.ncbi.nlm.nih.gov/8995621)" (1997), has an RCR of 2.98. Another notable publication, "[Conserved protein kinases encoded by herpesviruses and cellular protein kinase cdc2 target the same phosphorylation site in eukaryotic elongation factor 1delta.](https://pubmed.ncbi.nlm.nih.gov/12551973)" (2003), has an RCR of 2.32. Additionally, the study "[Elongation factor 1 beta of artemia: localization of functional sites and homology to elongation factor 1 delta.](https://pubmed.ncbi.nlm.nih.gov/2207149)" (1990) has an RCR of 2.23. More recent works include "[EEF1D modulates proliferation and epithelial-mesenchymal transition in oral squamous cell carcinoma.](https://pubmed.ncbi.nlm.nih.gov/26823560)" (2016) with an RCR of 1.32, and "[Eukaryotic Translation Elongation Factor 1 Delta Inhibits the Nuclear Import of the Nucleoprotein and PA-PB1 Heterodimer of Influenza A Virus.](https://pubmed.ncbi.nlm.nih.gov/33087462)" (2020) with an RCR of 1.27. The relative citation ratio measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SMC3](https://www.ncbi.nlm.nih.gov/gene/9126) and [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), each with 6 experiments. Additionally, [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), [BRCA1](https://www.ncbi.nlm.nih.gov/gene/672), and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) were each found to be regulating in 5 experiments.








The mouse ortholog gene 66656 and its human ortholog gene 1936 are associated with various abnormal morphologies, including those of the blood vessels, eyes, kidneys, placenta, skin, and vitreous body. Additionally, they are linked to conditions such as cataracts, edema, embryonic growth retardation, microphthalmia, and complete preweaning lethality. These phenotypes are observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.585 (26.57th percentile), indicating a hydrophilic nature. The charge at pH 7 is -15.00 (15.35th percentile), and the median structural flexibility is 1.013 (88.3rd percentile). The protein has a high affinity for helix structures (37.37%, 86.55th percentile), with lower affinities for sheet (28.47%, 20.55th percentile) and turn (28.11%, 48.36th percentile) structures. The instability index is 45.72 (41.84th percentile), and the isoelectric point is 4.90 (6.01st percentile). The protein is 281 amino acids long (25.84th percentile) with a molecular weight of 31121.46 Da (25.68th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |