---
layout: post
title: PFDN4
date: 2025-01-17 16:55 CST
description: PFDN4 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5203) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5203  | PFDN4 | ENSG00000101132 | 20q13.2  |



The gene enables [amyloid-beta binding](https://amigo.geneontology.org/amigo/term/GO:0001540) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is involved in [protein folding](https://amigo.geneontology.org/amigo/term/GO:0006457) and [chaperone-mediated protein folding](https://amigo.geneontology.org/amigo/term/GO:0061077). It is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [mitochondrion](https://amigo.geneontology.org/amigo/term/GO:0005739), and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). Additionally, it is part of the [prefoldin complex](https://amigo.geneontology.org/amigo/term/GO:0016272) and enables [unfolded protein binding](https://amigo.geneontology.org/amigo/term/GO:0051082) and [protein-folding chaperone binding](https://amigo.geneontology.org/amigo/term/GO:0051087). The gene is also involved in the [negative regulation of amyloid fibril formation](https://amigo.geneontology.org/amigo/term/GO:1905907).


The gene length is 13,897 base pairs (33.24% of all genes), the mature length is 3,008 base pairs, and the primary transcript length is 12,823 base pairs.


The gene PFDN4, with NCBI ID 5203, has been mentioned in [13 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PFDN4%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning this gene is 1985, and the middle 50% of publications occurred between 2006 and 2019.


The top 5 publications mentioning PFDN4 include "[Emerging and Novel Functions of Complement Protein C1q.](https://pubmed.ncbi.nlm.nih.gov/26175731)" (2015) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 4.39, "[miR-26a regulates extracellular vesicle secretion from prostate cancer cells via targeting SHC4, PFDN4, and CHORDC1.](https://pubmed.ncbi.nlm.nih.gov/32494663)" (2020) with an RCR of 2.23, "[Is the Complement Protein C1q a Pro- or Anti-tumorigenic Factor? Bioinformatics Analysis Involving Human Carcinomas.](https://pubmed.ncbi.nlm.nih.gov/31130944)" (2019) with an RCR of 1.75, "[Novel long noncoding RNAs upregulation may have synergistic effects on the CYP24A1 and PFDN4 biomarker role in human colorectal cancer.](https://pubmed.ncbi.nlm.nih.gov/32743796)" (2021) with an RCR of 0.93, and "[Expression of a secretory protein C1qTNF6, a C1qTNF family member, in hepatocellular carcinoma.](https://pubmed.ncbi.nlm.nih.gov/21508531)" (2011) with an RCR of 0.87. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PFDN4](https://www.proteinatlas.org/ENSG00000101132-PFDN4) is a gene that functions as a chaperone, with evidence supporting its role at the protein level. Its RNA is detected in all tissues, and it is expressed in various clusters, including Cluster 12 for blood expression, which is associated with mitochondrial translation, and Cluster 3 for tissue expression, which has an unknown function. In the brain, it is part of Cluster 1, which is associated with neurons and mixed functions. Additionally, it is found in Cluster 62 for cell line expression, related to plasma proteins, and Cluster 39 for single-cell expression, involved in basic cellular processes.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) with 6 experiments, [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227) with 5 experiments, [POU5F1](https://www.ncbi.nlm.nih.gov/gene/5460) with 5 experiments, [GATAD2B](https://www.ncbi.nlm.nih.gov/gene/57459) with 5 experiments, and [TRIM24](https://www.ncbi.nlm.nih.gov/gene/8805) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with various disease conditions, including cancer, particularly lung cancer and respiratory system cancer. Additionally, the data highlights associations with diseases of cellular proliferation, organ system cancer, kidney disease, and urinary system disease. These findings suggest a broad spectrum of genetic influences on both general and specific disease entities.



The gene is expressed in various tissues, including the brain, cardiac myocytes, Burkitt's lymphoma (Raji), promyelocytic leukemia (HL-60), and the superior cervical ganglion.


The input data highlights several pathways involved in protein processing, including metabolism of proteins, chaperonin-mediated protein folding, and protein folding. Additionally, it emphasizes the cooperation of prefoldin and TriC/CCT in actin and tubulin folding, as well as prefoldin-mediated transfer of substrates to CCT/TriC.



The analyzed protein sequence has a GRAVY value of -0.578 (27.22nd percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -14.38 (16.11th percentile), and the median structural flexibility is 1.012 (86.3rd percentile). The protein has a high affinity for helix structures (0.448, 98.17th percentile), with lower affinities for sheet (0.299, 27.46th percentile) and turn (0.216, 5.47th percentile) structures. The instability index is 61.6 (85.72nd percentile), suggesting high instability. The isoelectric point is 4.42 (1.21st percentile), and the protein is 134 amino acids long (6.26th percentile), with a molecular weight of 15314.06 Da (6.57th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |