---
layout: post
title: CA3
date: 2025-01-17 16:55 CST
description: CA3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/761) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 761  | CA3 | ENSG00000164879 | 8q21.2 |



The gene enables [carbonate dehydratase activity](https://amigo.geneontology.org/amigo/term/GO:0004089) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is involved in [one-carbon metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006730) and [response to bacterium](https://amigo.geneontology.org/amigo/term/GO:0009617). It is active in and located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737) and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). Additionally, it enables [zinc ion binding](https://amigo.geneontology.org/amigo/term/GO:0008270) and [nickel cation binding](https://amigo.geneontology.org/amigo/term/GO:0016151).


The gene length is 93,097 base pairs (80.54% of all genes), the mature length is 4,561 base pairs, and the primary transcript length is 48,316 base pairs (74.46% of all genes).


The gene CA3 (NCBI ID: 761) has been mentioned in [611 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CA3%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning CA3 is 1944, and the middle 50% of publications occurred between 1996 and 2014.


The top 5 publications mentioning CA3, ranked by their scientific influence, include "[Childhood maltreatment is associated with reduced volume in the hippocampal subfields CA3, dentate gyrus, and subiculum.](https://pubmed.ncbi.nlm.nih.gov/22331913)" (2012) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 14.28, "[Selective reinnervation of hippocampal area CA1 and the fascia dentata after destruction of CA3-CA4 afferents with kainic acid.](https://pubmed.ncbi.nlm.nih.gov/7350980)" (1980) with an RCR of 11.37, "[Pattern separation deficits associated with increased hippocampal CA3 and dentate gyrus activity in nondemented older adults.](https://pubmed.ncbi.nlm.nih.gov/20865732)" (2011) with an RCR of 11.15, "[Computational constraints suggest the need for two distinct input systems to the hippocampal CA3 network.](https://pubmed.ncbi.nlm.nih.gov/1308182)" (1992) with an RCR of 10.52, and "[High-resolution structural and functional MRI of hippocampal CA3 and dentate gyrus in patients with amnestic Mild Cognitive Impairment.](https://pubmed.ncbi.nlm.nih.gov/20338246)" (2010) with an RCR of 9.44. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CA3](https://www.proteinatlas.org/ENSG00000164879-CA3) is a lyase with evidence at the protein level, making it an FDA-approved drug target. Its RNA is detected in many tissues, with notable expression in skeletal muscle, where it is involved in striated muscle contraction. In the brain, CA3 is expressed in neurons with mixed functions. Additionally, it is expressed in sarcoma cell lines and skeletal myocytes, where it plays a role in muscle contraction.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SUZ12](https://www.ncbi.nlm.nih.gov/gene/23512), [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), each with 4 experiments, followed by [CEBPB](https://www.ncbi.nlm.nih.gov/gene/1051) with 3 experiments.





The gene is expressed in various tissues, including muscle and lung, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Thyroid, CD71+ Early Erythroid cells, Trigeminal Ganglion, Skeletal Muscle, Atrioventricular Node, Ciliary Ganglion, and Skin.


The proteins are involved in metabolism and the reversible hydration of carbon dioxide.



The analyzed protein sequence has a GRAVY value of -0.603 (25.07th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -0.469 (48.36th percentile). The median structural flexibility is 1.005 (62.69th percentile), with a secondary structure composition of 28.08% helix (17.46th percentile), 33.08% sheet (45.96th percentile), and 33.08% turn (82.27th percentile). The instability index is 39.78 (21.6th percentile), and the isoelectric point is 6.86 (48.71st percentile). The protein has a length of 260 amino acids (23.02nd percentile) and a molecular weight of 29557.06 Da (23.85th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |