---
layout: post
title: GOLGA4
date: 2025-01-17 16:55 CST
description: GOLGA4 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2803) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2803  | GOLGA4 | ENSG00000144674 | 3p22.2 |



The gene is located in the [Golgi membrane](https://amigo.geneontology.org/amigo/term/GO:0000139), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [Golgi apparatus](https://amigo.geneontology.org/amigo/term/GO:0005794), [trans-Golgi network](https://amigo.geneontology.org/amigo/term/GO:0005802), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). It enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [small GTPase binding](https://amigo.geneontology.org/amigo/term/GO:0031267), and [GTPase binding](https://amigo.geneontology.org/amigo/term/GO:0051020). The gene is involved in [vesicle-mediated transport](https://amigo.geneontology.org/amigo/term/GO:0016192), [Golgi to plasma membrane protein transport](https://amigo.geneontology.org/amigo/term/GO:0043001), [positive regulation of axon extension](https://amigo.geneontology.org/amigo/term/GO:0045773), and [Golgi vesicle transport](https://amigo.geneontology.org/amigo/term/GO:0048193).


The gene length is 1714.0 base pairs (4.52% of all genes). The mature length is 1714.0 base pairs, and the primary transcript length is 1714.0 base pairs.


The gene GOLGA4 (NCBI ID: 2803) has been mentioned in [17 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22GOLGA4%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1983 and the middle 50% of publications occurring between 2000 and 2013. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning GOLGA4, ranked by their scientific influence, include "[The P190, P210, and P230 forms of the BCR/ABL oncogene induce a similar chronic myeloid leukemia-like syndrome in mice but have different lymphoid leukemogenic activity.](https://pubmed.ncbi.nlm.nih.gov/10224280)" (1999) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 6.6, "[Immunolocalization of a novel, cytoskeleton-associated polypeptide of Mr 230,000 daltons (p230).](https://pubmed.ncbi.nlm.nih.gov/6339521)" (1983) with an RCR of 2.3, "[Interaction of Arl1-GTP with GRIP domains recruits autoantigens Golgin-97 and Golgin-245/p230 onto the Golgi.](https://pubmed.ncbi.nlm.nih.gov/12972563)" (2003) with an RCR of 1.95, "[Molecular characterization of Golgin-245, a novel Golgi complex protein containing a granin signature.](https://pubmed.ncbi.nlm.nih.gov/8537393)" (1995) with an RCR of 1.86, and "[Molecular characterization of trans-Golgi p230. A human peripheral membrane protein encoded by a gene on chromosome 6p12-22 contains extensive coiled-coil alpha-helical domains and a granin motif.](https://pubmed.ncbi.nlm.nih.gov/8626529)" (1996) with an RCR of 1.39. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[GOLGA4](https://www.proteinatlas.org/ENSG00000144674-GOLGA4) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in the nucleoplasm and Golgi apparatus. The gene is expressed in various clusters, including Cluster 22 in eosinophils related to transcription, Cluster 3 in non-specific tissues with unknown function, Cluster 50 in the brain associated with nucleic acid binding, Cluster 8 in cell lines involved in basic cellular processes, and Cluster 75 in serous glandular cells related to salivary secretion.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885) and [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), each with 5 experiments. Additionally, [SMC3](https://www.ncbi.nlm.nih.gov/gene/9126), [BRCA1](https://www.ncbi.nlm.nih.gov/gene/672), and [RNF2](https://www.ncbi.nlm.nih.gov/gene/6045) were each found to be regulating in 4 experiments.


The transcription factor [9968](https://www.ncbi.nlm.nih.gov/gene/9968) was found to be regulating genes in one Perturb-seq experiment, according to data from GTRD's database of transcription factor binding sites and the study published in [PubMed](https://pubmed.ncbi.nlm.nih.gov/35688146/).



The gene GOLGA4 has been associated with [Pulse pressure](https://pubmed.ncbi.nlm.nih.gov/33230300), which maps closely to the MeSH term [Blood Pressure](https://meshb.nlm.nih.gov/record/ui?ui=D001794) using the method [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0).


The gene is expressed in various tissues, including muscle, thyroid, and pituitary, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in pineal gland (both day and night), prostate, colon, small intestine, skin, and uterus.



The mouse ortholog gene 54214 and its human ortholog gene 2803 are associated with the phenotype of preweaning lethality with incomplete penetrance, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -1.0512556053811717 (3.44th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -82.64753700793517 (0.78th percentile), and the median structural flexibility is 1.0206666666666666 (97.22nd percentile). The protein has a high affinity for helix structures (49.15%, 99.61st percentile), with lower affinities for sheet (29.28%, 24.63rd percentile) and turn (17.17%, 0.62nd percentile) structures. The instability index is 55.864349775785136 (74.37th percentile), and the isoelectric point is 5.328166770935058 (14.15th percentile). The protein is 2230 amino acids long (98.19th percentile) with a molecular weight of 261137.11160000102 Da (98.39th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |