---
layout: post
title: KTN1
date: 2025-01-17 16:55 CST
description: KTN1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3895) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3895  | KTN1 | ENSG00000126777 | 14q22.3 |



The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), including specific interactions such as [kinesin binding](https://amigo.geneontology.org/amigo/term/GO:0019894) and [cadherin binding](https://amigo.geneontology.org/amigo/term/GO:0045296). It is located in various cellular components, including the [endoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0005783), [endoplasmic reticulum lumen](https://amigo.geneontology.org/amigo/term/GO:0005788), [endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0005789), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), and [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020). Additionally, it is involved in [microtubule-based movement](https://amigo.geneontology.org/amigo/term/GO:0007018) and [protein transport](https://amigo.geneontology.org/amigo/term/GO:0015031).


The gene length is 20,787 base pairs (43rd percentile of all genes), with a mature length of 1,470 base pairs and a primary transcript length of 17,311 base pairs.


The gene KTN1 (NCBI ID: 3895) has been mentioned in [31 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22KTN1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1995 and the middle 50% of publications occurring between 2002 and 2020. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning KTN1, ranked by their scientific influence, include studies that explore the gene's role in various cancers and its regulatory functions. The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Notable publications include "[MALAT1-KTN1-EGFR regulatory axis promotes the development of cutaneous squamous cell carcinoma.](https://pubmed.ncbi.nlm.nih.gov/30683916)" (2019) (relative citation ratio: 1.75), "[STAT1-induced upregulation of lncRNA KTN1-AS1 predicts poor prognosis and facilitates non-small cell lung cancer progression via miR-23b/DEPDC1 axis.](https://pubmed.ncbi.nlm.nih.gov/32396871)" (2020) (relative citation ratio: 1.7), "[Long noncoding RNA <i>KTN1-AS1</i> promotes head and neck squamous cell carcinoma cell epithelial-mesenchymal transition by targeting <i>miR-153-3p</i>.](https://pubmed.ncbi.nlm.nih.gov/32267161)" (2020) (relative citation ratio: 1.52), "[lncRNA KTN1‑AS1 promotes glioma cell proliferation and invasion by negatively regulating miR‑505‑3p.](https://pubmed.ncbi.nlm.nih.gov/33125151)" (2020) (relative citation ratio: 1.23), and "[Novel δ1-receptor agonist KNT-127 increases the release of dopamine and L-glutamate in the striatum, nucleus accumbens and median pre-frontal cortex.](https://pubmed.ncbi.nlm.nih.gov/22266218)" (2012) (relative citation ratio: 1.13). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[KTN1](https://www.proteinatlas.org/ENSG00000126777-KTN1) is a cancer-related gene with evidence at the protein level. It is detected in all RNA tissue distributions and is primarily localized in the endoplasmic reticulum. The gene is expressed in various clusters, including Cluster 30 for blood expression (non-specific - DNA binding), Cluster 14 for tissue expression (skeletal muscle - muscle contraction), Cluster 50 for brain expression (non-specific - nucleic acid binding), Cluster 30 for cell line expression (non-specific - gene expression regulation), and Cluster 12 for single cell expression (squamous epithelial cells - keratinization).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664) with 6 experiments, [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861) with 5 experiments, [RELA](https://www.ncbi.nlm.nih.gov/gene/5970) with 5 experiments, [YY1](https://www.ncbi.nlm.nih.gov/gene/7528) with 5 experiments, and [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113) with 5 experiments. These TFs are prominently involved in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.


The transcription factor [TCF7L2](https://www.ncbi.nlm.nih.gov/gene/6830) was found to be regulating genes in one Perturb-seq experiment, according to data from GTRD's database of transcription factor binding sites and the study published in [PubMed](https://pubmed.ncbi.nlm.nih.gov/35688146/).


The input data indicates associations with various disease conditions, including kidney disease, urinary system disease, nervous system disease, central nervous system disease, neurodegenerative disease, Alzheimer's disease, tauopathy, dementia, cognitive disorder, disease of mental health, reproductive system disease, prostate disease, prostatic hypertrophy, and male reproductive system disease.


The gene KTN1 has been associated with the trait [Height](https://pubmed.ncbi.nlm.nih.gov/30595370) through genome-wide association studies.


The gene is expressed in muscle and bladder tissues according to the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles indicate expression in the thyroid, pituitary, uterus, and various testis tissues, including interstitial, seminiferous tubule, and germ cells.




The analyzed protein sequence has a GRAVY value of -0.901 (7.34th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -33.94 (5.05th percentile), and the median structural flexibility is 1.021 (97.17th percentile). The protein has a high affinity for helix structures (47.16%, 99.2nd percentile), with lower affinities for sheet (29.85%, 27.43rd percentile) and turn (19.53%, 1.99th percentile) structures. The instability index is 49.97 (57.14th percentile), and the isoelectric point is 5.52 (18.75th percentile). The protein is 1357 amino acids long (93.84th percentile) with a molecular weight of 156273.38 Da (94.31st percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |