---
layout: post
title: FZD3
date: 2025-01-17 16:55 CST
description: FZD3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7976) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7976  | FZD3 | ENSG00000104290 | 8p21.1 |



The gene is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986), [apical plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0016324), [lateral plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0016328), [axon](https://amigo.geneontology.org/amigo/term/GO:0030424), [dendrite](https://amigo.geneontology.org/amigo/term/GO:0030425), [filopodium tip](https://amigo.geneontology.org/amigo/term/GO:0032433), and [neuronal cell body](https://amigo.geneontology.org/amigo/term/GO:0043025). It is active in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886) and enables [G protein-coupled receptor activity](https://amigo.geneontology.org/amigo/term/GO:0004930), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [Wnt-protein binding](https://amigo.geneontology.org/amigo/term/GO:0017147), [PDZ domain binding](https://amigo.geneontology.org/amigo/term/GO:0030165), and [Wnt receptor activity](https://amigo.geneontology.org/amigo/term/GO:0042813). The gene is involved in various biological processes including [establishment of planar polarity](https://amigo.geneontology.org/amigo/term/GO:0001736), [neural tube closure](https://amigo.geneontology.org/amigo/term/GO:0001843), [hair follicle development](https://amigo.geneontology.org/amigo/term/GO:0001942), [positive regulation of neuroblast proliferation](https://amigo.geneontology.org/amigo/term/GO:0002052), [G protein-coupled receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007186), [response to xenobiotic stimulus](https://amigo.geneontology.org/amigo/term/GO:0009410), [neuron differentiation](https://amigo.geneontology.org/amigo/term/GO:0030182), [cell proliferation in midbrain](https://amigo.geneontology.org/amigo/term/GO:0033278), [non-canonical Wnt signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0035567), [post-anal tail morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0036342), [dopaminergic neuron axon guidance](https://amigo.geneontology.org/amigo/term/GO:0036514), [serotonergic neuron axon guidance](https://amigo.geneontology.org/amigo/term/GO:0036515), [inner ear morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0042472), [negative regulation of mitotic cell cycle, embryonic](https://amigo.geneontology.org/amigo/term/GO:0045976), [response to electrical stimulus](https://amigo.geneontology.org/amigo/term/GO:0051602), [canonical Wnt signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0060070), [Wnt signaling pathway, planar cell polarity pathway](https://amigo.geneontology.org/amigo/term/GO:0060071), [sympathetic ganglion development](https://amigo.geneontology.org/amigo/term/GO:0061549), [commissural neuron axon guidance](https://amigo.geneontology.org/amigo/term/GO:0071679), [motor neuron migration](https://amigo.geneontology.org/amigo/term/GO:0097475), [negative regulation of execution phase of apoptosis](https://amigo.geneontology.org/amigo/term/GO:1900118), and [midbrain morphogenesis](https://amigo.geneontology.org/amigo/term/GO:1904693).


The gene length is 81,643.0 base pairs (77.77% of all genes), the mature length is 6,857.5 base pairs (96.18% of all genes), and the primary transcript length is 36,047.0 base pairs (67.44% of all genes).


The gene FZD3 (NCBI ID: 7976) has been mentioned in [26 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22FZD3%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1999 and the middle 50% of publications occurring between 2006 and 2018. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning FZD3, ranked by their scientific influence, include studies that explore the gene's role in various biological processes. The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Notable publications include [The role of Frizzled3 and Frizzled6 in neural tube closure and in the planar polarity of inner-ear sensory hair cells.](https://pubmed.ncbi.nlm.nih.gov/16495441) (2006) (relative citation ratio: 7.61), [Circular RNA circ-CBFB promotes proliferation and inhibits apoptosis in chronic lymphocytic leukemia through regulating miR-607/FZD3/Wnt/β-catenin pathway.](https://pubmed.ncbi.nlm.nih.gov/29902450) (2018) (relative citation ratio: 4.35), [A tRNA fragment, 5'-tiRNA<sup>Val</sup>, suppresses the Wnt/β-catenin signaling pathway by targeting FZD3 in breast cancer.](https://pubmed.ncbi.nlm.nih.gov/31078732) (2019) (relative citation ratio: 4.33), [Circular RNA CircPVT1 Inhibits 5-Fluorouracil Chemosensitivity by Regulating Ferroptosis Through MiR-30a-5p/FZD3 Axis in Esophageal Cancer Cells.](https://pubmed.ncbi.nlm.nih.gov/34966683) (2021) (relative citation ratio: 2.79), and [Molecular cloning and genomic structure of human frizzled-3 at chromosome 8p21.](https://pubmed.ncbi.nlm.nih.gov/10777673) (2000) (relative citation ratio: 2.44). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[FZD3](https://www.proteinatlas.org/ENSG00000104290-FZD3) is a developmental protein that functions as a G-protein coupled receptor and transducer. It is detected in many tissues and is localized to the nucleoplasm, nuclear membrane, and vesicles. FZD3 is expressed in various clusters, including Cluster 30 for non-specific DNA binding in blood, Cluster 39 for retina and testis cilium in tissues, Cluster 17 for choroid plexus cilium in the brain, Cluster 40 for non-specific unknown function in cell lines, and Cluster 6 for ovarian stromal cells transcription in single cells. Evidence for FZD3 is available at the protein level.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [RELA](https://www.ncbi.nlm.nih.gov/gene/5970) and [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169), each with 5 experiments. Additionally, [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), and [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) were each found to be regulating in 4 experiments.



The input data indicates associations with various disease conditions, including neurodegenerative diseases such as Alzheimer's disease and tauopathy. Additionally, it encompasses cognitive disorders, dementia, and diseases of mental health, all of which are subcategories of central nervous system diseases and broader nervous system diseases.



The gene is expressed in various tissues, including the ovary, brain, and pituitary, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the pineal gland (both day and night), prefrontal cortex, cingulate cortex, appendix, superior cervical ganglion, trigeminal ganglion, and skeletal muscle.


The input data includes various pathways and processes such as asymmetric localization of PCP proteins, the PCP/CE pathway, beta-catenin independent WNT signaling, signaling by Wnt, the Ca2+ pathway, and Class B/2 (Secretin family receptors). Additionally, it covers defective ACTH causing obesity and pro-opiomelanocortin deficiency (POMCD), GPCR ligand binding, metabolic disorders of biological oxidation enzymes, and general disease processes. Signaling by GPCR and signal transduction are also highlighted.


The mouse ortholog gene 14365 and its human ortholog gene 7976 are associated with the phenotype of preweaning lethality with incomplete penetrance, observed in both sexes.


The analyzed protein sequence has a GRAVY value of 0.049 (86.02 percentile), indicating a slightly hydrophobic nature. The charge at pH 7 is 4.085 (64.99 percentile), and the median structural flexibility is 0.988 (9.53 percentile). The protein has a propensity for helix, sheet, and turn structures of 27.63% (14.98 percentile), 40.09% (86.54 percentile), and 26.13% (30.33 percentile), respectively. The instability index is 45.72 (41.84 percentile), and the isoelectric point is 7.94 (58.2 percentile). The protein is 666 amino acids long (73.23 percentile) with a molecular weight of 76261.9 Da (74.31 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |