---
layout: post
title: ENSA
date: 2025-01-17 16:55 CST
description: ENSA description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2029) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2029  | ENSA | ENSG00000143420 | 1q21.3 |



The gene is involved in the [G2/M transition of mitotic cell cycle](https://amigo.geneontology.org/amigo/term/GO:0000086) and the [mitotic cell cycle](https://amigo.geneontology.org/amigo/term/GO:0000278). It enables [protein phosphatase inhibitor activity](https://amigo.geneontology.org/amigo/term/GO:0004864), [signaling receptor binding](https://amigo.geneontology.org/amigo/term/GO:0005102), and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654) and is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737). Additionally, it is involved in the [response to nutrient](https://amigo.geneontology.org/amigo/term/GO:0007584) and [cell division](https://amigo.geneontology.org/amigo/term/GO:0051301). The gene also enables [ion channel inhibitor activity](https://amigo.geneontology.org/amigo/term/GO:0008200), [phosphatase inhibitor activity](https://amigo.geneontology.org/amigo/term/GO:0019212), [potassium channel inhibitor activity](https://amigo.geneontology.org/amigo/term/GO:0019870), and [protein phosphatase regulator activity](https://amigo.geneontology.org/amigo/term/GO:0019888). Furthermore, it is involved in the [negative regulation of protein dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0035308) and acts upstream of or within the [regulation of insulin secretion](https://amigo.geneontology.org/amigo/term/GO:0050796). Lastly, it enables [protein phosphatase 2A binding](https://amigo.geneontology.org/amigo/term/GO:0051721).


The gene length is 797.0 base pairs (1.07% of all genes), with a mature length of 797.0 base pairs and a primary transcript length of 797.0 base pairs.


The gene ENSA (NCBI ID: 2029) has been mentioned in [15 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ENSA%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning ENSA was in 2004, and the middle 50% of publications occurred between 2013 and 2019.


The top 5 publications mentioning ENSA, ranked by their scientific influence, include "[The BEG (PP2A-B55/ENSA/Greatwall) pathway ensures cytokinesis follows chromosome separation.](https://pubmed.ncbi.nlm.nih.gov/24120663)" (2013) (relative citation ratio: 2.73), "[Copy number amplification of ENSA promotes the progression of triple-negative breast cancer via cholesterol biosynthesis.](https://pubmed.ncbi.nlm.nih.gov/35145111)" (2022) (relative citation ratio: 1.9), "[PP2A/B55 and Fcp1 regulate Greatwall and Ensa dephosphorylation during mitotic exit.](https://pubmed.ncbi.nlm.nih.gov/24391510)" (2014) (relative citation ratio: 1.28), "[ENSA and ARPP19 differentially control cell cycle progression and development.](https://pubmed.ncbi.nlm.nih.gov/30626720)" (2019) (relative citation ratio: 1.23), and "[Endosulfine-alpha inhibits membrane-induced α-synuclein aggregation and protects against α-synuclein neurotoxicity.](https://pubmed.ncbi.nlm.nih.gov/28069058)" (2017) (relative citation ratio: 1.01). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ENSA](https://www.proteinatlas.org/ENSG00000143420-ENSA) is a protein phosphatase inhibitor with evidence at the protein level. It is detected in all RNA tissues and is primarily localized in the nucleoplasm, nucleoli, microtubules, and cytosol. ENSA is expressed in various clusters, including Cluster 45 in eosinophils with unknown function, Cluster 19 in non-specific tissues involved in basic cellular processes, Cluster 53 in neurons with mixed function, Cluster 45 in cell lines involved in nuclear processes, and Cluster 14 in Langerhans cells involved in the innate immune response.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [RELA](https://www.ncbi.nlm.nih.gov/gene/5970), [KDM5B](https://www.ncbi.nlm.nih.gov/gene/10765), [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885), and [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227), each shown to be regulating in 5 experiments.





The gene is expressed in the esophagus and vagina according to the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles indicate expression in the pineal gland (night), whole brain, cingulate cortex, medulla oblongata, occipital lobe, cerebellum, and placenta.


The proteins are involved in various stages of the cell cycle, including mitotic prophase, the broader cell cycle, and specifically the mitotic phase. Additionally, they play a role in facilitating mitotic progression through the MASTL pathway.


The mouse ortholog gene 56205 and its human ortholog gene 2029 are associated with the phenotype of hyperactivity, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -1.2471 (1.56 percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -0.383 (48.71 percentile), and the median structural flexibility is 1.027 (98.98 percentile). The protein has a high affinity for helix structures (42.98%, 96.94 percentile) and turns (32.23%, 77.96 percentile), with a lower affinity for sheet structures (20.66%, 2.51 percentile). The instability index is 66.08 (91.43 percentile), suggesting the protein may be unstable. The isoelectric point is 6.62 (44.94 percentile), with a molecular weight of 13388.86 Da (4.72 percentile). The sequence length is 121 amino acids (4.74 percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |