---
layout: post
title: SLBP
date: 2025-01-17 16:55 CST
description: SLBP description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7884) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7884  | SLBP | ENSG00000163950 | 4p16.3 |



The gene is involved in [cap-dependent translational initiation](https://amigo.geneontology.org/amigo/term/GO:0002191) and [mRNA 3'-end processing by stem-loop binding and cleavage](https://amigo.geneontology.org/amigo/term/GO:0006398), as well as [mRNA transport](https://amigo.geneontology.org/amigo/term/GO:0051028). It enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [mRNA binding](https://amigo.geneontology.org/amigo/term/GO:0003729), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802), [histone pre-mRNA stem-loop binding](https://amigo.geneontology.org/amigo/term/GO:0071207), and [histone pre-mRNA DCP binding](https://amigo.geneontology.org/amigo/term/GO:0071208). The gene is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [nucleolus](https://amigo.geneontology.org/amigo/term/GO:0005730), and [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), specifically in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). It is part of the [histone mRNA stem-loop binding complex](https://amigo.geneontology.org/amigo/term/GO:0062073), [histone pre-mRNA 3'end processing complex](https://amigo.geneontology.org/amigo/term/GO:0071204), and [ribonucleoprotein complex](https://amigo.geneontology.org/amigo/term/GO:1990904).


The gene length is 103,685 base pairs (82.53% of all genes), the mature length is 7,850 base pairs (97.56% of all genes), and the primary transcript length is 86,202 base pairs (86.08% of all genes).


The gene SLBP (NCBI ID: 7884) has been mentioned in [42 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SLBP%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning SLBP was in 1995, and the middle 50% of publications occurred between 2004 and 2016.


The top 5 publications mentioning SLBP, ranked by their scientific influence, include "[Stem-loop binding protein, the protein that binds the 3' end of histone mRNA, is cell cycle regulated by both translational and posttranslational mechanisms.](https://pubmed.ncbi.nlm.nih.gov/10825184)" (2000) (relative citation ratio: 2.99), "[Structure of histone mRNA stem-loop, human stem-loop binding protein, and 3'hExo ternary complex.](https://pubmed.ncbi.nlm.nih.gov/23329046)" (2013) (relative citation ratio: 2.12), "[The gene for histone RNA hairpin binding protein is located on human chromosome 4 and encodes a novel type of RNA binding protein.](https://pubmed.ncbi.nlm.nih.gov/9049306)" (1997) (relative citation ratio: 2.03), "[Stem-loop binding protein facilitates 3'-end formation by stabilizing U7 snRNP binding to histone pre-mRNA.](https://pubmed.ncbi.nlm.nih.gov/10207079)" (1999) (relative citation ratio: 1.67), and "[Cyclin F-Mediated Degradation of SLBP Limits H2A.X Accumulation and Apoptosis upon Genotoxic Stress in G2.](https://pubmed.ncbi.nlm.nih.gov/27773672)" (2016) (relative citation ratio: 1.65). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[SLBP](https://www.proteinatlas.org/ENSG00000163950-SLBP) is a gene with a molecular function in RNA-binding as a ribonucleoprotein. It is detected in all tissues and is primarily localized in the nucleoplasm, nucleoli, and cytosol. SLBP is expressed in various clusters, including Cluster 38 in monocytes and neutrophils related to degranulation, Cluster 77 in bone marrow associated with cell proliferation, Cluster 14 in neurons involved in synaptic function, Cluster 22 in non-specific cell lines related to mRNA splicing and cell cycle, and Cluster 52 in non-specific single cells related to cell proliferation. Evidence for SLBP is available at the protein level.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), [KMT2B](https://www.ncbi.nlm.nih.gov/gene/9757), [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), and [TAL1](https://www.ncbi.nlm.nih.gov/gene/6886), each with 6 experiments, followed by [BRD9](https://www.ncbi.nlm.nih.gov/gene/65980) with 5 experiments.



The GWAS data indicates associations with various conditions, including cognitive disorder, bipolar disorder, and mood disorder. These findings highlight the potential genetic underpinnings of mental health diseases.



The gene is expressed in various tissues, including skin and pituitary, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in 721_B_lymphoblasts, CD105+_Endothelial cells, CD71+_EarlyErythroid cells, Leukemialymphoblastic (MOLT-4) cells, and Testis.


The input data highlights various pathways involved in gene expression and RNA processing. These include the transport of mature mRNAs derived from intronless transcripts, SLBP-dependent mature mRNA transport, and the transport of mature transcripts to the cytoplasm. Additionally, the data covers RNA polymerase II transcription, cleavage of growing transcripts in the termination region, and RNA polymerase II transcription termination. Post-elongation processing of transcripts, including intronless pre-mRNA and SLBP-dependent processing of replication-dependent histone pre-mRNAs, is also featured. The pathways extend to the processing of capped intronless pre-mRNA and capped intron-containing pre-mRNA, as well as general transcription processes.


The mouse ortholog gene 20492 and its human ortholog gene 7884 are associated with phenotypes including embryonic lethality prior to the tooth bud stage, which is not observed in both sexes, and preweaning lethality with complete penetrance, which is observed in both sexes.


The analyzed protein sequence has a GRAVY value of -1.24 (1.61 percentile), indicating a hydrophilic nature. The charge at pH 7.0 is 0.14 (50.91 percentile), and the median structural flexibility is 1.02 (93.93 percentile). The protein has a high instability index of 85.80 (98.96 percentile) and an isoelectric point of 7.06 (51.12 percentile). The sequence length is 270 amino acids (24.39 percentile) with a molecular weight of 31285.33 Da (25.86 percentile). The secondary structure affinities are 29.26% for helix (25.45 percentile), 22.22% for sheet (3.88 percentile), and 33.33% for turn (83.56 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |