---
layout: post
title: KRT10
date: 2025-01-17 16:55 CST
description: KRT10 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3858) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3858  | KRT10 | ENSG00000186395 | 17q21.2 |



The gene is located in the [cornified envelope](https://amigo.geneontology.org/amigo/term/GO:0001533), [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615), [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [intermediate filament](https://amigo.geneontology.org/amigo/term/GO:0005882), [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). It is active in the [cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0005856) and [keratin filament](https://amigo.geneontology.org/amigo/term/GO:0045095). The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [structural constituent of skin epidermis](https://amigo.geneontology.org/amigo/term/GO:0030280), and [protein heterodimerization activity](https://amigo.geneontology.org/amigo/term/GO:0046982). It is involved in [epidermis development](https://amigo.geneontology.org/amigo/term/GO:0008544), [peptide cross-linking](https://amigo.geneontology.org/amigo/term/GO:0018149), [keratinocyte differentiation](https://amigo.geneontology.org/amigo/term/GO:0030216), [epithelial cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0030855), [intermediate filament organization](https://amigo.geneontology.org/amigo/term/GO:0045109), [positive regulation of epidermis development](https://amigo.geneontology.org/amigo/term/GO:0045684), and [protein heterotetramerization](https://amigo.geneontology.org/amigo/term/GO:0051290).


The gene length is 116,065 base pairs (84.58% of all genes), the mature length is 4,192 base pairs (85.31% of all genes), and the primary transcript length is 63,574.5 base pairs (80.52% of all genes).


The gene KRT10 (NCBI ID: 3858) has been mentioned in [194 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22KRT10%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning KRT10 is 1978, and the middle 50% of publications occurred between 2001 and 2017.


The top 5 publications mentioning KRT10, ranked by their scientific influence, include "[Interpreting scores on the Kessler Psychological Distress Scale (K10).](https://pubmed.ncbi.nlm.nih.gov/11824981)" (2001) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 35.51, "[The performance of the Japanese version of the K6 and K10 in the World Mental Health Survey Japan.](https://pubmed.ncbi.nlm.nih.gov/18763695)" (2008) with an RCR of 30.68, "[The performance of the K6 and K10 screening scales for psychological distress in the Australian National Survey of Mental Health and Well-Being.](https://pubmed.ncbi.nlm.nih.gov/12622315)" (2003) with an RCR of 29.64, "[The psychometric properties of the 10-item Kessler Psychological Distress Scale (K10) in Canadian military personnel.](https://pubmed.ncbi.nlm.nih.gov/29698459)" (2018) with an RCR of 4.94, and "[The performance of the K10, K6 and GHQ-12 to screen for present state DSM-IV disorders among disability claimants.](https://pubmed.ncbi.nlm.nih.gov/23402478)" (2013) with an RCR of 4.09. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[KRT10](https://www.proteinatlas.org/ENSG00000186395-KRT10) is a gene with evidence at the protein level, known for its involvement in the disease variant Ichthyosis. It is detected in all RNA tissue distributions and is particularly expressed in skin tissue, where it plays a role in cornification. In single-cell expression clusters, KRT10 is associated with squamous epithelial cells involved in keratinization. Additionally, it is found in eosinophils within blood expression clusters, although its function in this context is unknown.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [REST](https://www.ncbi.nlm.nih.gov/gene/5978) and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), each with 5 experiments. Additionally, [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), [GATA6](https://www.ncbi.nlm.nih.gov/gene/2627), and [SMAD3](https://www.ncbi.nlm.nih.gov/gene/4088) were each found to be regulating in 4 experiments.





The gene is expressed in various tissues, including the cervix uteri, vagina, and skin, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in chronic myelogenous leukemia (K-562), prostate, lung, skin, tongue, and thymus.




The analyzed protein sequence has a GRAVY value of -0.624 (23.22nd percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -15.75 (14.66th percentile), and the median structural flexibility is 1.017 (94.25th percentile). The protein has an instability index of 58.87 (80.95th percentile) and an isoelectric point of 5.13 (10.27th percentile). The sequence length is 584 amino acids (67.17th percentile) with a molecular weight of 58826.39 Da (61.4th percentile). The secondary structure affinities are 24.32% for both helix and sheet (3.91st and 7.19th percentiles, respectively), and 48.80% for turns (99.53rd percentile). These calculations were performed using BioPython's sequence analysis tools, which can be referenced [here](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |