---
layout: post
title: BTG2
date: 2024-12-08 23:57 CST
description: BTG2 description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7832) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7832  | BTG2 | ENSG00000159388 | 1q32.1 |



The gene length is 50,172.0 base pairs (66.43% of all genes), the mature length is 1,211.5 base pairs, and the primary transcript length is 36,761.0 base pairs.


The gene BTG2 (NCBI ID: 7832) has been the subject of [139 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22BTG2%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink). The earliest year of publication is 1995, and the middle 50% of publications occurred between 2010 and 2020.


The top five publications mentioning BTG2, ranked by their scientific influence, include "[The mammalian immediate-early TIS21 protein and the leukemia-associated BTG1 protein interact with a protein-arginine N-methyltransferase.](https://pubmed.ncbi.nlm.nih.gov/8663146)" (1996) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 7.04, "[Identification of BTG2, an antiproliferative p53-dependent component of the DNA damage cellular response pathway.](https://pubmed.ncbi.nlm.nih.gov/8944033)" (1996) with an RCR of 6.53, "[Tumor suppressors BTG1 and BTG2: Beyond growth control.](https://pubmed.ncbi.nlm.nih.gov/30350856)" (2019) with an RCR of 6.23, "[TEAD4 modulated LncRNA MNX1-AS1 contributes to gastric cancer progression partly through suppressing BTG2 and activating BCL2.](https://pubmed.ncbi.nlm.nih.gov/31924214)" (2020) with an RCR of 5.48, and "[Androgen-regulated miR-32 targets BTG2 and is overexpressed in castration-resistant prostate cancer.](https://pubmed.ncbi.nlm.nih.gov/22266859)" (2012) with an RCR of 4.77. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861) with 6 experiments, [EP300](https://www.ncbi.nlm.nih.gov/gene/2033) with 5 experiments, [RARA](https://www.ncbi.nlm.nih.gov/gene/5914) with 5 experiments, [MAX](https://www.ncbi.nlm.nih.gov/gene/4149) with 5 experiments, and [BRD9](https://www.ncbi.nlm.nih.gov/gene/65980) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.


The top transcription factors (TFs) identified from the GTRD's database of TF binding sites, filtered for Perturb-seq data, include [6829](https://www.ncbi.nlm.nih.gov/gene/6829) and [9126](https://www.ncbi.nlm.nih.gov/gene/9126), each shown to be regulating genes in 3 experiments. Additionally, [8243](https://www.ncbi.nlm.nih.gov/gene/8243) was found to regulate genes in 2 experiments. Other notable TFs include [65123](https://www.ncbi.nlm.nih.gov/gene/65123), [8861](https://www.ncbi.nlm.nih.gov/gene/8861), and [9968](https://www.ncbi.nlm.nih.gov/gene/9968), each shown to regulate genes in 1 experiment.


The gene BTG2 has been associated with [Mean reticulocyte volume](https://pubmed.ncbi.nlm.nih.gov/34226706), which maps closely to the MeSH term [Reticulocytes](https://meshb.nlm.nih.gov/record/ui?ui=D012156) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method.


The mouse ortholog gene 12227 and its human ortholog 7832 are associated with several phenotypes. Notably, the mouse model exhibits an enlarged uterus, a phenotype observed only in females. Additionally, both sexes display increased fasting circulating glucose levels, increased mean corpuscular hemoglobin, and increased mean corpuscular volume. These findings highlight the potential relevance of these orthologs in understanding corresponding human conditions.


The analyzed protein sequence has a GRAVY value of -0.134 (75.37th percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is 1.992 (57.52nd percentile), and the median structural flexibility is 0.996 (23.17th percentile). The protein has a propensity for helix, sheet, and turn structures of 32.28% (53.07th percentile), 33.54% (48.82nd percentile), and 29.75% (61.95th percentile), respectively. The instability index is 49.19 (54.54th percentile), and the isoelectric point is 8.29 (62.62nd percentile). The protein is 158 amino acids long (9.01st percentile) with a molecular weight of 17415.88 Da (8.76th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |