---
layout: post
title: NPM1
date: 2024-12-08 23:57 CST
description: NPM1 description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4869) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4869  | NPM1 | ENSG00000181163 | 5q35.1 |



The gene length is 26,318 base pairs (49.04% of all genes), the mature length is 2,302 base pairs, and the primary transcript length is 26,256 base pairs.


The gene NPM1 (NCBI ID: 4869) has been featured in [1,072 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22NPM1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1951, and the middle 50% of publications occurred between 2009 and 2019.


The top 5 publications mentioning NPM1, ranked by their scientific influence, include "[Fusion of a kinase gene, ALK, to a nucleolar protein gene, NPM, in non-Hodgkin's lymphoma.](https://pubmed.ncbi.nlm.nih.gov/8122112)" (1994) (relative citation ratio: 37.14), "[Cytoplasmic nucleophosmin in acute myelogenous leukemia with a normal karyotype.](https://pubmed.ncbi.nlm.nih.gov/15659725)" (2005) (relative citation ratio: 25.21), "[IDH1 and IDH2 mutations are frequent genetic alterations in acute myeloid leukemia and confer adverse prognosis in cytogenetically normal acute myeloid leukemia with NPM1 mutation without FLT3 internal tandem duplication.](https://pubmed.ncbi.nlm.nih.gov/20567020)" (2010) (relative citation ratio: 13.61), "[Nucleophosmin and cancer.](https://pubmed.ncbi.nlm.nih.gov/16794633)" (2006) (relative citation ratio: 11.67), and "[The impact of FLT3 internal tandem duplication mutant level, number, size, and interaction with NPM1 mutations in a large cohort of young adult patients with acute myeloid leukemia.](https://pubmed.ncbi.nlm.nih.gov/17957027)" (2008) (relative citation ratio: 11.46). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) with 6 experiments, followed by [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313), [KMT2B](https://www.ncbi.nlm.nih.gov/gene/9757), [TAL1](https://www.ncbi.nlm.nih.gov/gene/6886), and [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), each with 5 experiments.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, filtered for Perturb-seq data, include [GATA1](https://www.ncbi.nlm.nih.gov/gene/6829) with 4 experiments, [FOXA1](https://www.ncbi.nlm.nih.gov/gene/23028) with 2 experiments, [GATA2](https://www.ncbi.nlm.nih.gov/gene/2623) with 3 experiments, [MYC](https://www.ncbi.nlm.nih.gov/gene/8243) with 2 experiments, and [BCL6](https://www.ncbi.nlm.nih.gov/gene/8861) with 2 experiments. Additionally, [BCL11A](https://www.ncbi.nlm.nih.gov/gene/9968) was found in 3 experiments, and [BCL11B](https://www.ncbi.nlm.nih.gov/gene/9126) was found in 1 experiment. These TFs were shown to regulate gene expression in multiple Perturb-seq experiments, highlighting their significant roles in transcriptional regulation.








The analyzed protein sequence has a GRAVY value of -0.9697 (5.14th percentile), indicating a hydrophilic nature. The charge at pH 7 is -24.01 (8.64th percentile), and the median structural flexibility is 1.026 (98.74th percentile). The protein has a high affinity for helix (37.41%, 86.72nd percentile) and turn (35.37%, 90.26th percentile) structures, with a lower affinity for sheet structures (23.81%, 6.14th percentile). The instability index is 51.30 (61.44th percentile), and the isoelectric point is 4.64 (2.44th percentile). The protein is 294 amino acids long (27.62nd percentile) with a molecular weight of 32574.66 Da (27.33rd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |