---
layout: post
title: EXOG
date: 2024-12-08 23:57 CST
description: EXOG description
tags: [cooccuring-genes]
categories: shortest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9941) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9941  | EXOG | ENSG00000157036 | 3p22.2 |



The gene length is 467.0 base pairs (0.37 percentile of all genes), with a mature length of 467.0 base pairs and a primary transcript length of 467.0 base pairs.


The gene EXOG, with NCBI ID 9941, has been featured in [19 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22EXOG%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1954, and the middle 50% of publications occurred between 2006 and 2018.


The top 5 publications mentioning EXOG, ranked by their scientific influence, include "[Apoptosis induced by persistent single-strand breaks in mitochondrial genome: critical role of EXOG (5'-EXO/endonuclease) in their repair.](https://pubmed.ncbi.nlm.nih.gov/21768646)" (2011) (relative citation ratio: 2.91), "[EXOG, a novel paralog of Endonuclease G in higher eukaryotes.](https://pubmed.ncbi.nlm.nih.gov/18187503)" (2008) (relative citation ratio: 1.21), "[Cytotoxicity of botanicals and isolated phytochemicals from Araliopsis soyauxii Engl. (Rutaceae) towards a panel of human cancer cells.](https://pubmed.ncbi.nlm.nih.gov/33166626)" (2021) (relative citation ratio: 1.17), and "[Soyauxinine, a New Indolopyridoquinazoline Alkaloid from the Stem Bark of <i>Araliopsis soyauxii</i> Engl. (Rutaceae).](https://pubmed.ncbi.nlm.nih.gov/35164367)" (2022) (relative citation ratio: 1.0). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025), [PGR](https://www.ncbi.nlm.nih.gov/gene/5241), and [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), each with 5 experiments. Additionally, [NFYA](https://www.ncbi.nlm.nih.gov/gene/4800) and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) were found to be regulating in 4 experiments each.











The analyzed protein sequence has a GRAVY value of -0.359 (51.21st percentile), indicating a hydrophilic nature. It exhibits a charge of 3.032 at pH 7 (61.43rd percentile) and a median structural flexibility of 1.003 (54.04th percentile). The protein's secondary structure is predicted to be 32.34% helix (53.59th percentile), 33.70% sheet (49.87th percentile), and 29.35% turn (58.79th percentile). The instability index is 46.55 (44.96th percentile), with an isoelectric point of 8.45 (65.6th percentile). The protein has a length of 368 amino acids (40.83rd percentile) and a molecular weight of 41084.24 Da (40.92nd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |