---
layout: post
title: AUH
date: 2025-01-17 16:55 CST
description: AUH description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/549) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 549  | AUH | ENSG00000148090 | 9q22.31 |



The gene enables [mRNA 3'-UTR binding](https://amigo.geneontology.org/amigo/term/GO:0003730) and [enoyl-CoA hydratase activity](https://amigo.geneontology.org/amigo/term/GO:0004300). It also enables [methylglutaconyl-CoA hydratase activity](https://amigo.geneontology.org/amigo/term/GO:0004490) and [itaconyl-CoA hydratase activity](https://amigo.geneontology.org/amigo/term/GO:0050011). The gene is active in and located in the [mitochondrion](https://amigo.geneontology.org/amigo/term/GO:0005739), specifically within the [mitochondrial matrix](https://amigo.geneontology.org/amigo/term/GO:0005759). Additionally, it is involved in the [leucine catabolic process](https://amigo.geneontology.org/amigo/term/GO:0006552) and [fatty acid beta-oxidation](https://amigo.geneontology.org/amigo/term/GO:0006635).


The gene length is 26,575 base pairs (49.39% of all genes), the mature length is 953 base pairs (20.61% of all genes), and the primary transcript length is 2,834 base pairs (10.09% of all genes).


The gene AUH (NCBI ID: 549) has been mentioned in [20 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22AUH%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning AUH was in 1964, and the middle 50% of publications occurred between 1996 and 2015.


The top 5 publications mentioning AUH, ranked by their scientific influence, include "[The gold-hydrogen bond, Au-H, and the hydrogen bond to gold, Au∙∙∙H-X.](https://pubmed.ncbi.nlm.nih.gov/23999756)" (2014) (relative citation ratio: 4.09), "[AUH, a gene encoding an AU-specific RNA binding protein with intrinsic enoyl-CoA hydratase activity.](https://pubmed.ncbi.nlm.nih.gov/7892223)" (1995) (relative citation ratio: 2.52), "[Spin trapping of Au-H intermediate in the alcohol oxidation by supported and unsupported gold catalysts.](https://pubmed.ncbi.nlm.nih.gov/19405535)" (2009) (relative citation ratio: 2.25), "[An element through the looking glass: exploring the Au-C, Au-H and Au-O energy landscape.](https://pubmed.ncbi.nlm.nih.gov/26584519)" (2015) (relative citation ratio: 1.74), and "[3-Methylglutaconyl-CoA hydratase, 3-methylcrotonyl-CoA carboxylase and 3-hydroxy-3-methylglutaryl-CoA lyase deficiencies: a coupled enzyme assay useful for their detection.](https://pubmed.ncbi.nlm.nih.gov/2480857)" (1989) (relative citation ratio: 0.75). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[AUH](https://www.proteinatlas.org/ENSG00000148090-AUH) is a gene with lyase and RNA-binding molecular functions, known to be involved in disease variants. Evidence for its function is available at the protein level. The gene is detected in all RNA tissue distributions and is localized in vesicles, centrosomes, and the cytosol. AUH is expressed in various clusters, including Cluster 36 in eosinophils related to protein ubiquitination, Cluster 55 in the kidney and intestine related to metabolism, Cluster 50 in the brain related to nucleic acid binding, Cluster 21 in lymphoma related to the adaptive immune response, and Cluster 8 in neurons related to neuronal signaling.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 5 experiments, [MAX](https://www.ncbi.nlm.nih.gov/gene/4149) with 4 experiments, [SPI1](https://www.ncbi.nlm.nih.gov/gene/6688) with 4 experiments, [AR](https://www.ncbi.nlm.nih.gov/gene/367) with 4 experiments, and [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq studies.



The GWAS data indicates associations with a wide range of disease conditions, including lower respiratory tract disease, lung disease, respiratory system disease, immune system disease, coronary artery disease, artery disease, vascular disease, cardiovascular system disease, nervous system disease, central nervous system disease, neurodegenerative disease, Alzheimer's disease, tauopathy, dementia, cognitive disorder, disease of mental health, bone disease, connective tissue disease, musculoskeletal system disease, bone remodeling disease, bone resorption disease, lupus erythematosus, systemic lupus erythematosus, hypersensitivity reaction disease, hypersensitivity reaction type II disease, and myocardial infarction.



The gene is expressed in the adrenal gland and kidney according to BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles indicate expression in the pineal gland (both day and night), thyroid, amygdala, hypothalamus, thalamus, prostate, and kidney.


The input data highlights several key pathways involved in metabolism, specifically focusing on branched-chain amino acid catabolism and the broader metabolism of amino acids and their derivatives.


The mouse ortholog gene 11992 and its human ortholog gene 549 are associated with abnormal sternum morphology, a phenotype observed in both sexes.


The analyzed protein sequence has a GRAVY value of 0.0697 (86.9th percentile), indicating a relatively hydrophobic nature. The charge at pH 7 is 11.76 (84.72nd percentile), and the median structural flexibility is 0.998 (31.71st percentile). The protein has a high affinity for helix structures (0.422, 96.19th percentile) and a moderate affinity for sheet structures (0.304, 30.34th percentile) and turn structures (0.274, 42.03rd percentile). The instability index is 40.38 (23.38th percentile), and the isoelectric point is 9.53 (91.5th percentile). The protein sequence is 339 amino acids long (36.22nd percentile) with a molecular weight of 35608.21 Da (32.81st percentile). These calculations were performed using BioPython's sequence analysis tools, which can be found [here](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |