---
layout: post
title: DEDD
date: 2025-01-17 16:55 CST
description: DEDD description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9191) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9191  | DEDD | ENSG00000158796 | 1q23.3  |



The gene enables [DNA binding](https://amigo.geneontology.org/amigo/term/GO:0003677) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is located in the [nucleolus](https://amigo.geneontology.org/amigo/term/GO:0005730) and the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737). The gene is involved in various biological processes, including [spermatogenesis](https://amigo.geneontology.org/amigo/term/GO:0007283), the [extrinsic apoptotic signaling pathway via death domain receptors](https://amigo.geneontology.org/amigo/term/GO:0008625), [negative regulation of protein catabolic process](https://amigo.geneontology.org/amigo/term/GO:0042177), [regulation of apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0042981), [decidualization](https://amigo.geneontology.org/amigo/term/GO:0046697), and [negative regulation of transcription of nucleolar large rRNA by RNA polymerase I](https://amigo.geneontology.org/amigo/term/GO:1901837).


The gene length is 28,327 base pairs (51.11% of all genes), the mature length is 2,076 base pairs (54.76% of all genes), and the primary transcript length is 28,312 base pairs (61.14% of all genes).


The gene DEDD, with NCBI ID 9191, has been mentioned in [16 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22DEDD%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1998, and the middle 50% of publications range from 2002 to 2018.


The top 5 publications mentioning DEDD include "[DEDD interacts with PI3KC3 to activate autophagy and attenuate epithelial-mesenchymal transition in human breast cancer.](https://pubmed.ncbi.nlm.nih.gov/22719072) (2012) (relative citation ratio: 3.76)," "[miR-24-3p regulates bladder cancer cell proliferation, migration, invasion and autophagy by targeting DEDD.](https://pubmed.ncbi.nlm.nih.gov/28000900) (2017) (relative citation ratio: 2.73)," "[MicroRNA-17 inhibition overcomes chemoresistance and suppresses epithelial-mesenchymal transition through a DEDD-dependent mechanism in gastric cancer.](https://pubmed.ncbi.nlm.nih.gov/29953965) (2018) (relative citation ratio: 1.43)," "[Long Noncoding RNA LINC00472 Inhibits Proliferation and Promotes Apoptosis of Lung Adenocarcinoma Cells via Regulating miR-24-3p/ DEDD.](https://pubmed.ncbi.nlm.nih.gov/30175664) (2018) (relative citation ratio: 1.4)," and "[DEDD, a novel tumor repressor, reverses epithelial-mesenchymal transition by activating selective autophagy.](https://pubmed.ncbi.nlm.nih.gov/22874565) (2012) (relative citation ratio: 1.27)." The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[DEDD](https://www.proteinatlas.org/ENSG00000158796-DEDD) is a gene with a molecular function in DNA-binding and acts as a repressor. Evidence for its function is available at the protein level. The gene is detected in all RNA tissue distributions and is localized in the nucleoplasm, nucleoli, and cytosol. It is part of various expression clusters, including Cluster 42 for blood (non-specific - vesicular transport), Cluster 19 for tissue (non-specific - basic cellular processes), Cluster 6 for brain (non-specific - immune response), Cluster 45 for cell line (non-specific - nuclear processes), and Cluster 44 for single cell (oocytes - unknown function).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 6 experiments, [TBP](https://www.ncbi.nlm.nih.gov/gene/6908) with 6 experiments, [MAX](https://www.ncbi.nlm.nih.gov/gene/4149) with 6 experiments, [TCF12](https://www.ncbi.nlm.nih.gov/gene/6938) with 5 experiments, and [KDM1A](https://www.ncbi.nlm.nih.gov/gene/23028) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with various conditions, including cognitive disorder, bipolar disorder, and mood disorder. These findings highlight the potential genetic underpinnings of mental health diseases.


The genome-wide association study has identified a link between certain genetic variants and [Alzheimer's disease or gastroesophageal reflux disease](https://pubmed.ncbi.nlm.nih.gov/35851147). The closest MeSH match for this trait is [Alzheimer Disease](https://meshb.nlm.nih.gov/record/ui?ui=D004194), determined using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method.


The gene is expressed in adipose tissue, as well as in various other tissues including cardiac myocytes, heart, cerebellum, placenta, superior cervical ganglion, trigeminal ganglion, and testis.




The analyzed protein sequence has a GRAVY value of -0.5503 (29.92% percentile), indicating a hydrophilic nature. It exhibits a charge of 8.35 at pH 7.0 (77.84% percentile) and a median structural flexibility of 0.9985 (34.5% percentile). The protein's secondary structure affinities are 29.87% for helix (30.84% percentile), 34.91% for sheet (58.35% percentile), and 25.79% for turn (27.4% percentile). The instability index is 44.91 (38.92% percentile), with an isoelectric point of 9.07 (81.22% percentile). The protein has a length of 318 amino acids (32.57% percentile) and a molecular weight of 36793.8 Da (34.77% percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |