---
layout: post
title: NEU1
date: 2024-12-08 23:57 CST
description: NEU1 description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4758) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4758  | NEU1 | ENSG00000204386 | 6p21.33 |



The gene length is 14,685 base pairs (34.61% of all genes), the mature length is 3,291 base pairs (76.38% of all genes), and the primary transcript length is 14,632 base pairs (43.46% of all genes).


The gene NEU1, with NCBI ID 4758, has been featured in [2,224 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22NEU1%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1948, and the middle 50% of publications span from 1982 to 2014.


The top 5 publications mentioning NEU1, ranked by their scientific influence, include "[Fluorometric assay of neuraminidase with a sodium (4-methylumbelliferyl-alpha-D-N-acetylneuraminate) substrate.](https://pubmed.ncbi.nlm.nih.gov/464297)" (1979) (relative citation ratio: 25.6), "[Influenza neuraminidase inhibitors possessing a novel hydrophobic interaction in the enzyme active site: design, synthesis, and structural analysis of carbocyclic sialic acid analogues with potent anti-influenza activity.](https://pubmed.ncbi.nlm.nih.gov/16526129)" (1997) (relative citation ratio: 18.44), "[Effectiveness of neuraminidase inhibitors in reducing mortality in patients admitted to hospital with influenza A H1N1pdm09 virus infection: a meta-analysis of individual participant data.](https://pubmed.ncbi.nlm.nih.gov/24815805)" (2014) (relative citation ratio: 17.21), "[Efficacy and safety of the neuraminidase inhibitor zanamivir in the treatment of influenzavirus infections. GG167 Influenza Study Group.](https://pubmed.ncbi.nlm.nih.gov/9302301)" (1997) (relative citation ratio: 14.67), and "[Neuraminidase inhibitors for preventing and treating influenza in adults and children.](https://pubmed.ncbi.nlm.nih.gov/24718923)" (2014) (relative citation ratio: 14.67). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


The gene [Glycosidase, Hydrolase](https://www.proteinatlas.org/[Ensembl]-[Gene]) is involved in disease variants and is an FDA-approved drug target. It functions as a glycosidase and hydrolase, with evidence at the protein level. The gene is detected in many tissues and is localized in vesicles and cell junctions. It is part of various expression clusters, including Cluster 20 for blood (non-specific - DNA binding), Cluster 66 for tissue (non-specific - transcription), Cluster 15 for brain (non-specific - transcription), Cluster 20 for cell line (myeloma - immune response), and Cluster 80 for single cell (non-specific - transcription). The gene is unprognostic for several cancer types, including breast invasive carcinoma, colon adenocarcinoma, glioblastoma multiforme, kidney chromophobe, kidney renal clear cell carcinoma, kidney renal papillary cell carcinoma, liver hepatocellular carcinoma, ovarian serous cystadenocarcinoma, pancreatic adenocarcinoma, rectum adenocarcinoma, and uterine corpus endometrial carcinoma.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [PGR](https://www.ncbi.nlm.nih.gov/gene/5241), [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599), [GATA3](https://www.ncbi.nlm.nih.gov/gene/2625), [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170), and [FOS](https://www.ncbi.nlm.nih.gov/gene/2353), each shown to be regulating in 5 experiments.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [STAT1](https://www.ncbi.nlm.nih.gov/gene/6829), [GATA1](https://www.ncbi.nlm.nih.gov/gene/2623), and [STAT2](https://www.ncbi.nlm.nih.gov/gene/6872), each shown to be regulating genes in two experiments. Additionally, [STAT3](https://www.ncbi.nlm.nih.gov/gene/6749) and [IRF1](https://www.ncbi.nlm.nih.gov/gene/8243) were found to regulate genes in one experiment each.








The analyzed protein sequence has a GRAVY value of -0.165 (72.68th percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is -6.92 (28.67th percentile), and the median structural flexibility is 0.999 (35.87th percentile). The protein has a moderate affinity for helix (0.246, 4.33rd percentile) and sheet (0.366, 70.6th percentile) structures, with a turn affinity of 0.342 (86.85th percentile). The instability index is 33.93 (8.42nd percentile), and the isoelectric point is 5.59 (20.31st percentile). The protein is 415 amino acids long (47.26th percentile) with a molecular weight of 45466.81 Da (46.31st percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |