---
layout: post
title: RHOH
date: 2025-01-17 16:55 CST
description: RHOH description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/399) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 399  | RHOH | ENSG00000168421 | 4p14 |



The gene is located in the [immunological synapse](https://amigo.geneontology.org/amigo/term/GO:0001772) and the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), and is active in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). It enables [GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0003924), [GTPase inhibitor activity](https://amigo.geneontology.org/amigo/term/GO:0005095), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [GTP binding](https://amigo.geneontology.org/amigo/term/GO:0005525), [kinase inhibitor activity](https://amigo.geneontology.org/amigo/term/GO:0019210), [protein kinase binding](https://amigo.geneontology.org/amigo/term/GO:0019901), and [small GTPase binding](https://amigo.geneontology.org/amigo/term/GO:0031267). The gene is involved in the [regulation of DNA-templated transcription](https://amigo.geneontology.org/amigo/term/GO:0006355), [actin filament organization](https://amigo.geneontology.org/amigo/term/GO:0007015), [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), [small GTPase-mediated signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007264), [T cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0030217), [negative regulation of canonical NF-kappaB signal transduction](https://amigo.geneontology.org/amigo/term/GO:0043124), [mast cell activation](https://amigo.geneontology.org/amigo/term/GO:0045576), and [positive regulation of T cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0045582).


The gene has a length of 52,297 base pairs (67.64% of all genes), a mature length of 843 base pairs (15.82% of all genes), and a primary transcript length of 45,375 base pairs (73.04% of all genes).


The gene RHOH (NCBI ID: 399) has been mentioned in [164 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RHOH%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1986 and the middle 50% of publications occurring between 2004 and 2013. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning RHOH, ranked by their scientific influence, include "[Single-component molecular metals with extended-TTF dithiolate ligands.](https://pubmed.ncbi.nlm.nih.gov/15535649)" (2004) (relative citation ratio: 3.82), "[Organic conductors: from charge density wave TTF-TCNQ to superconducting (TMTSF)2PF6.](https://pubmed.ncbi.nlm.nih.gov/15535660)" (2004) (relative citation ratio: 2.99), "[Magnetic TTF-based charge-transfer complexes.](https://pubmed.ncbi.nlm.nih.gov/15535656)" (2004) (relative citation ratio: 2.58), "[TTF, a gene encoding a novel small G protein, fuses to the lymphoma-associated LAZ3 gene by t(3;4) chromosomal translocation.](https://pubmed.ncbi.nlm.nih.gov/7784061)" (1995) (relative citation ratio: 1.95), and "[The hematopoiesis-specific GTP-binding protein RhoH is GTPase deficient and modulates activities of other Rho GTPases by an inhibitory function.](https://pubmed.ncbi.nlm.nih.gov/11809807)" (2002) (relative citation ratio: 1.75). The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RHOH](https://www.proteinatlas.org/ENSG00000168421-RHOH) is a cancer-related gene with evidence at the protein level. It is detected in many tissues and is primarily localized in the Golgi apparatus and vesicles. RHOH is associated with nuclear processes in blood, cytokine signaling in lymphoid tissue, mixed functions in astrocytes, T-cell receptor signaling in leukemia cell lines, and immune system and transcription processes in NK-cells and T-cells.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 5 experiments, [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664) with 4 experiments, [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025) with 4 experiments, [AR](https://www.ncbi.nlm.nih.gov/gene/367) with 4 experiments, and [CREBBP](https://www.ncbi.nlm.nih.gov/gene/1387) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.


The transcription factors [STAT1](https://www.ncbi.nlm.nih.gov/gene/2623) and [KLF6](https://www.ncbi.nlm.nih.gov/gene/8861) were each found to regulate gene expression in one Perturb-seq experiment, as indicated by the GTRD's database of transcription factor binding sites from uniform CHIP-seq data.




The gene is expressed in various tissues and cell types, including blood, spleen, and small intestine, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles show expression in CD4+ T cells, CD8+ T cells, CD19+ B cells (neg. sel.), Burkitt's lymphoma (Daudi), tonsil, thymus, and lymphoblastic leukemia (MOLT-4).


The proteins are involved in the Rho GTPase cycle, signaling by Rho GTPases, and signal transduction pathways.



The analyzed protein sequence has a GRAVY value of -0.159 (73.19th percentile), indicating a hydrophilic nature. It exhibits a charge of 6.63 at pH 7.0 (73.07th percentile) and a median structural flexibility of 0.999 (37.75th percentile). The protein's secondary structure is predicted to be 28.3% helix (18.68th percentile), 35.6% sheet (63.51st percentile), and 27.2% turn (40.07th percentile). The instability index is 41.24 (26.16th percentile), with an isoelectric point of 9.17 (83.65th percentile). The protein has a length of 191 amino acids (13.19th percentile) and a molecular weight of 21330.3 Da (13.08th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |