---
layout: post
title: PDGFRA
date: 2025-01-17 16:55 CST
description: PDGFRA description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5156) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5156  | PDGFRA | ENSG00000134853 | 4q12  |



The gene is involved in various biological processes, including [luteinization](https://amigo.geneontology.org/amigo/term/GO:0001553), [in utero embryonic development](https://amigo.geneontology.org/amigo/term/GO:0001701), and [cell activation](https://amigo.geneontology.org/amigo/term/GO:0001775). It plays a role in [hematopoietic progenitor cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0002244) and is involved in signaling pathways such as the [transmembrane receptor protein tyrosine kinase signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007169). The gene enables several molecular functions, including [protein kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004672), [transmembrane receptor protein tyrosine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004714), and [platelet-derived growth factor alpha-receptor activity](https://amigo.geneontology.org/amigo/term/GO:0005018). It is located in various cellular components, such as the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). Additionally, the gene is involved in the development of various tissues and organs, including the [lung](https://amigo.geneontology.org/amigo/term/GO:0030324), [adrenal gland](https://amigo.geneontology.org/amigo/term/GO:0030325), and [male genitalia](https://amigo.geneontology.org/amigo/term/GO:0030539). It also plays a role in processes such as [wound healing](https://amigo.geneontology.org/amigo/term/GO:0042060) and [odontogenesis of dentin-containing tooth](https://amigo.geneontology.org/amigo/term/GO:0042475).


The gene length is 26,013 base pairs (48.74th percentile), the mature length is 647 base pairs (7.24th percentile), and the primary transcript length is 19,947 base pairs (51.6th percentile).


The gene PDGFRA (NCBI ID: 5156) has been mentioned in [918 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PDGFRA%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning this gene is 1990, and the middle 50% of publications occurred between 2009 and 2018.


The top 5 publications mentioning PDGFRA, ranked by their scientific influence, include "[Integrated genomic analysis identifies clinically relevant subtypes of glioblastoma characterized by abnormalities in PDGFRA, IDH1, EGFR, and NF1.](https://pubmed.ncbi.nlm.nih.gov/20129251)" (2010) (relative citation ratio: 123.62), "[PDGFRA activating mutations in gastrointestinal stromal tumors.](https://pubmed.ncbi.nlm.nih.gov/12522257)" (2003) (relative citation ratio: 34.71), "[A tyrosine kinase created by fusion of the PDGFRA and FIP1L1 genes as a therapeutic target of imatinib in idiopathic hypereosinophilic syndrome.](https://pubmed.ncbi.nlm.nih.gov/12660384)" (2003) (relative citation ratio: 24.3), "[Defects in succinate dehydrogenase in gastrointestinal stromal tumors lacking KIT and PDGFRA mutations.](https://pubmed.ncbi.nlm.nih.gov/21173220)" (2011) (relative citation ratio: 11.25), and "[PDGFRÎ± and CD51 mark human nestin+ sphere-forming mesenchymal stem cells capable of hematopoietic progenitor cell expansion.](https://pubmed.ncbi.nlm.nih.gov/23776077)" (2013) (relative citation ratio: 10.35). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PDGFRA](https://www.proteinatlas.org/ENSG00000134853-PDGFRA) is a developmental protein with kinase, receptor, transferase, and tyrosine-protein kinase molecular functions. It is implicated in various cancer-related processes, serving as a proto-oncogene and a target for FDA-approved drugs. Evidence supports its role at the protein level. PDGFRA is detected in all tissues and is localized in the nucleoplasm, nuclear bodies, plasma membrane, cell junctions, and primary cilium. Its expression is notable in connective tissue, particularly in ECM organization, and in macrophages and microglia involved in immune response. In single-cell studies, PDGFRA is expressed in fibroblasts, also involved in ECM organization.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [EZH2](https://www.ncbi.nlm.nih.gov/gene/2146) with 5 experiments, [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861) with 4 experiments, [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 4 experiments, [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664) with 4 experiments, and [PGR](https://www.ncbi.nlm.nih.gov/gene/5241) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with a range of conditions, including coronary artery disease, artery disease, vascular disease, and broader cardiovascular system diseases. Additionally, the data highlights links to nervous system diseases, eye and adnexa diseases, and more specifically, eye diseases and corneal diseases. These findings suggest a multifaceted impact on various anatomical and sensory systems.


The gene PDGFRA has been associated with [Educational attainment](https://pubmed.ncbi.nlm.nih.gov/35361970), which maps closely to the MeSH term [Educational Status](https://meshb.nlm.nih.gov/record/ui?ui=D004522) using the method [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0).


The gene is expressed in various tissues, including the ovary and skin, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in cardiac myocytes, adipocytes, smooth muscle, appendix, ovary, fetal lung, and uterus.


The input data highlights various pathways involved in immune system function, including adaptive and innate immune systems. Additionally, it emphasizes multiple signaling pathways, particularly those involving PI3K/AKT, such as constitutive PI3K/AKT signaling in cancer, PI3K events in ERBB2 and ERBB4 signaling, and PIP3 activation of AKT signaling. Other notable pathways include signaling by various receptors like SCF-KIT, ERBB4, PDGF, FGFR, and EGFR, as well as downstream signal transduction events. The data also covers signaling related to NGF, Fc epsilon receptor (FCERI), and B Cell Receptor (BCR), along with general disease and signal transduction processes.



The analyzed protein sequence has a GRAVY value of -0.262 (62.45th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -45.27 (3.02nd percentile), and the median structural flexibility is 1.001 (45.16th percentile). The protein has a propensity for helix, sheet, and turn structures of 31.86% (49.18th percentile), 38.84% (82.43rd percentile), and 28.65% (53.06th percentile), respectively. The instability index is 43.49 (33.84th percentile), with an isoelectric point of 5.05 (8.76th percentile). The protein is 1089 amino acids long (89.65th percentile) and has a molecular weight of 122668.13 Da (89.87th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |