---
layout: post
title: HLA-E
date: 2025-01-17 16:55 CST
description: HLA-E description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3133) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3133  | HLA-E | ENSG00000204592 | 6p22.1 |



The gene is located in the [Golgi membrane](https://amigo.geneontology.org/amigo/term/GO:0000139), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986), [ER to Golgi transport vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0012507), [phagocytic vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030670), [early endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0031901), [recycling endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0055038), [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062), and the [lumenal side of endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0098553). It is active in the [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615) and the [external side of plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009897). The gene is involved in various biological processes, including the [positive regulation of antibody-dependent cellular cytotoxicity](https://amigo.geneontology.org/amigo/term/GO:0001815), [positive regulation of T cell mediated cytotoxicity](https://amigo.geneontology.org/amigo/term/GO:0001916), [adaptive immune response](https://amigo.geneontology.org/amigo/term/GO:0002250), [antigen processing and presentation of endogenous peptide antigen via MHC class Ib](https://amigo.geneontology.org/amigo/term/GO:0002476), [antigen processing and presentation of exogenous peptide antigen via MHC class Ib](https://amigo.geneontology.org/amigo/term/GO:0002477), [antigen processing and presentation of endogenous peptide antigen via MHC class I via ER pathway, TAP-independent](https://amigo.geneontology.org/amigo/term/GO:0002486), [natural killer cell tolerance induction](https://amigo.geneontology.org/amigo/term/GO:0002519), [positive regulation of immunoglobulin production](https://amigo.geneontology.org/amigo/term/GO:0002639), [regulation of natural killer cell mediated immunity](https://amigo.geneontology.org/amigo/term/GO:0002715), [positive regulation of natural killer cell mediated immunity](https://amigo.geneontology.org/amigo/term/GO:0002717), [positive regulation of natural killer cell cytokine production](https://amigo.geneontology.org/amigo/term/GO:0002729), [immune response](https://amigo.geneontology.org/amigo/term/GO:0006955), [antibacterial humoral response](https://amigo.geneontology.org/amigo/term/GO:0019731), [positive regulation of interleukin-13 production](https://amigo.geneontology.org/amigo/term/GO:0032736), [positive regulation of interleukin-4 production](https://amigo.geneontology.org/amigo/term/GO:0032753), [positive regulation of TRAIL production](https://amigo.geneontology.org/amigo/term/GO:0032759), [positive regulation of tumor necrosis factor production](https://amigo.geneontology.org/amigo/term/GO:0032760), [negative regulation of natural killer cell activation](https://amigo.geneontology.org/amigo/term/GO:0032815), [positive regulation of natural killer cell activation](https://amigo.geneontology.org/amigo/term/GO:0032816), [positive regulation of natural killer cell proliferation](https://amigo.geneontology.org/amigo/term/GO:0032819), [CD8-positive, alpha-beta T cell activation](https://amigo.geneontology.org/amigo/term/GO:0036037), [negative regulation of T cell proliferation](https://amigo.geneontology.org/amigo/term/GO:0042130), [protection from natural killer cell mediated cytotoxicity](https://amigo.geneontology.org/amigo/term/GO:0042270), [innate immune response](https://amigo.geneontology.org/amigo/term/GO:0045087), [negative regulation of natural killer cell mediated cytotoxicity](https://amigo.geneontology.org/amigo/term/GO:0045953), [positive regulation of natural killer cell mediated cytotoxicity](https://amigo.geneontology.org/amigo/term/GO:0045954), [defense response to Gram-positive bacterium](https://amigo.geneontology.org/amigo/term/GO:0050830), [positive regulation of CD8-positive, alpha-beta T cell proliferation](https://amigo.geneontology.org/amigo/term/GO:2000566), and [positive regulation of CD8-positive, alpha-beta T cell activation](https://amigo.geneontology.org/amigo/term/GO:2001187). The gene enables functions such as [signaling receptor binding](https://amigo.geneontology.org/amigo/term/GO:0005102), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [beta-2-microglobulin binding](https://amigo.geneontology.org/amigo/term/GO:0030881), [MHC class I protein binding](https://amigo.geneontology.org/amigo/term/GO:0042288), [peptide antigen binding](https://amigo.geneontology.org/amigo/term/GO:0042605), [T cell receptor binding](https://amigo.geneontology.org/amigo/term/GO:0042608), and [natural killer cell lectin-like receptor binding](https://amigo.geneontology.org/amigo/term/GO:0046703). It is also part of the [MHC class Ib protein complex](https://amigo.geneontology.org/amigo/term/GO:0032398) and the [MHC class I protein complex](https://amigo.geneontology.org/amigo/term/GO:0042612).


The gene length is 103,322 base pairs (82.44% of all genes), the mature length is 1,512 base pairs, and the primary transcript length is 70,020 base pairs.


HLA-E (Gene ID: 3133) has been mentioned in [323 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22HLA-E%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest publication mentioning HLA-E was in 1980, and the middle 50% of publications occurred between 2003 and 2017.


The top five publications mentioning HLA-E, ranked by their scientific influence, include "[HLA-E binds to natural killer cell receptors CD94/NKG2A, B and C.](https://pubmed.ncbi.nlm.nih.gov/9486650)" (1998) (relative citation ratio: 31.89), "[HLA-E is a major ligand for the natural killer inhibitory receptor CD94/NKG2A.](https://pubmed.ncbi.nlm.nih.gov/9560253)" (1998) (relative citation ratio: 14.49), "[HLA-E-expressing pluripotent stem cells escape allogeneic responses and lysis by NK cells.](https://pubmed.ncbi.nlm.nih.gov/28504668)" (2017) (relative citation ratio: 12.27), "[Recognition of human histocompatibility leukocyte antigen (HLA)-E complexed with HLA class I signal sequence-derived peptides by CD94/NKG2 confers protection from natural killer cell-mediated lysis.](https://pubmed.ncbi.nlm.nih.gov/9480992)" (1998) (relative citation ratio: 10.27), and "[Surface expression of HLA-E, an inhibitor of natural killer cells, enhanced by human cytomegalovirus gpUL40.](https://pubmed.ncbi.nlm.nih.gov/10669413)" (2000) (relative citation ratio: 8.18). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[HLA-E](https://www.proteinatlas.org/ENSG00000204592-HLA-E) is a gene with evidence at the protein level, detected in many RNA tissues. It is primarily localized in vesicles and the plasma membrane. The gene is expressed in neutrophils, associated with protein binding, and is involved in immune responses in various tissues, including the spleen, brain (macrophages and microglia), and lymphoma cell lines. Additionally, it is expressed in Langerhans cells, contributing to the innate immune response.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) with 5 experiments, [BRD9](https://www.ncbi.nlm.nih.gov/gene/65980) with 4 experiments, [KDM1A](https://www.ncbi.nlm.nih.gov/gene/23028) with 4 experiments, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 4 experiments, and [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.




HLA-E has been associated with several traits and diseases through genome-wide association studies. Notably, it has been linked to [Hemoglobin levels](https://pubmed.ncbi.nlm.nih.gov/32327693), which maps closely to the MeSH term [Hemoglobins](https://meshb.nlm.nih.gov/record/ui?ui=D006454) using the BioWordVec method. Additionally, HLA-E is associated with [Immunoglobulin M levels](https://pubmed.ncbi.nlm.nih.gov/34648354), corresponding to the MeSH term [Immunoglobulins](https://meshb.nlm.nih.gov/record/ui?ui=D007136), and [Plateletcrit](https://pubmed.ncbi.nlm.nih.gov/32888494), which aligns with the MeSH term [Blood Platelets](https://meshb.nlm.nih.gov/record/ui?ui=D001792), both identified through embedding matches.


The gene is ubiquitously expressed in multiple tissue samples, including lung, blood, and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in various immune cell types such as CD56+ NK cells, CD4+ T cells, CD8+ T cells, CD19+ B cells (neg. sel.), and CD14+ monocytes, as well as in whole blood, lung, and lymph node.


The input data highlights various pathways involved in the immune system, including antigen presentation and processing, cytokine signaling, interferon signaling, and interactions between lymphoid and non-lymphoid cells. Specific pathways mentioned include the folding, assembly, and peptide loading of class I MHC, interferon gamma and alpha/beta signaling, ER-phagosome pathway, and endosomal/vacuolar pathway. Additionally, the data covers adaptive and innate immune system processes, as well as DAP12 interactions and signaling.



The analyzed protein sequence has a GRAVY value of -0.536 (31.4% percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -13.16 (17.69% percentile), and the median structural flexibility is 1.001 (46.18% percentile). The protein has a predicted secondary structure with 29.89% helix (30.97% percentile), 34.92% sheet (58.43% percentile), and 27.93% turn (46.6% percentile) affinities. The instability index is 34.75 (9.7% percentile), and the isoelectric point is 5.52 (18.83% percentile). The protein is 358 amino acids long (39.3% percentile) with a molecular weight of 40057.07 Da (39.49% percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |