---
layout: post
title: HCCS
date: 2025-01-17 16:55 CST
description: HCCS description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3052) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3052  | HCCS | ENSG00000004961 | Xp22.2  |



The gene enables [holocytochrome-c synthase activity](https://amigo.geneontology.org/amigo/term/GO:0004408) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is active in the [mitochondrion](https://amigo.geneontology.org/amigo/term/GO:0005739). It is located in the [mitochondrial inner membrane](https://amigo.geneontology.org/amigo/term/GO:0005743), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020). The gene is involved in [animal organ morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0009887) and [cytochrome c-heme linkage](https://amigo.geneontology.org/amigo/term/GO:0018063). Additionally, it enables [heme binding](https://amigo.geneontology.org/amigo/term/GO:0020037) and [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872).


The gene length is 21,558 base pairs (43.89% of all genes), the mature length is 584 base pairs (4.67% of all genes), and the primary transcript length is 14,169 base pairs (42.53% of all genes).


The gene HCCS (NCBI ID: 3052) has been mentioned in [39 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22HCCS%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1977 and the middle 50% of publications ranging from 1988 to 2016. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning HCCS, ranked by their scientific influence, include "[Transcriptional regulation of miR-224 upregulated in human HCCs by NFκB inflammatory pathways.](https://pubmed.ncbi.nlm.nih.gov/22178270)" (2012) (relative citation ratio: 3.24), "[Deep convolutional neural network for preoperative prediction of microvascular invasion and clinical outcomes in patients with HCCs.](https://pubmed.ncbi.nlm.nih.gov/34347160)" (2022) (relative citation ratio: 3.06), "[Critical analysis of the major and ancillary imaging features of LI-RADS on 127 proven HCCs evaluated with functional and morphological MRI: Lights and shadows.](https://pubmed.ncbi.nlm.nih.gov/28881643)" (2017) (relative citation ratio: 2.76), "[Hyperactivation of IL-6/STAT3 pathway leaded to the poor prognosis of post-TACE HCCs by HIF-1α/SNAI1 axis-induced epithelial to mesenchymal transition.](https://pubmed.ncbi.nlm.nih.gov/31942180)" (2020) (relative citation ratio: 2.0), and "[TMC-1, the starless core sulfur factory: Discovery of NCS, HCCS, H<sub>2</sub>CCS, H<sub>2</sub>CCCS, and C<sub>4</sub>S and detection of C<sub>5</sub>S.](https://pubmed.ncbi.nlm.nih.gov/33850333)" (2021) (relative citation ratio: 1.81). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[HCCS](https://www.proteinatlas.org/ENSG00000004961-HCCS) is a gene encoding a protein with lyase molecular function, primarily located in the mitochondria. It is involved in the disease variant Microphthalmia, with evidence supporting its role at the protein level. The gene's RNA is detected in all tissues, and it is expressed in various clusters, including Cluster 10 for blood expression related to membrane trafficking, Cluster 35 for tissue expression related to mitochondria, Cluster 23 for brain expression in the choroid plexus, Cluster 36 for cell line expression related to mRNA processing, and Cluster 45 for single cell expression with mixed function.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169), [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885), and [YY1](https://www.ncbi.nlm.nih.gov/gene/7528), each with 4 experiments, followed by [ATF3](https://www.ncbi.nlm.nih.gov/gene/467) with 3 experiments.





The gene is expressed in various tissues and cell types, including heart, muscle, and colon, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles show expression in BDCA4+ dendritic cells, chronic myelogenous leukemia K-562 cells, CD105+ endothelial cells, CD71+ early erythroid cells, bronchial epithelial cells, colon, and small intestine.




The analyzed protein sequence has a GRAVY value of -0.734 (15.51% percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -2.09 (42.86% percentile), and the median structural flexibility is 1.007 (72.8% percentile). The protein has a propensity for helix, sheet, and turn structures of 31.0% (40.7% percentile), 28.0% (18.4% percentile), and 32.1% (77.2% percentile), respectively. The instability index is 60.8 (84.41% percentile), suggesting high instability. The isoelectric point is 6.25 (36.75% percentile), with a length of 268 amino acids (24.15% percentile) and a molecular weight of 30601.25 Da (25.09% percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |