---
layout: post
title: ITGA3
date: 2025-01-17 16:55 CST
description: ITGA3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3675) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3675  | ITGA3 | ENSG00000005884 | 17q21.33 |



The gene is located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925), [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986), [basolateral plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0016323), [filopodium membrane](https://amigo.geneontology.org/amigo/term/GO:0031527), [neuromuscular junction](https://amigo.geneontology.org/amigo/term/GO:0031594), [postsynaptic membrane](https://amigo.geneontology.org/amigo/term/GO:0045211), [perinuclear region of cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0048471), [presynaptic active zone membrane](https://amigo.geneontology.org/amigo/term/GO:0048787), [excitatory synapse](https://amigo.geneontology.org/amigo/term/GO:0060076), [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062), [cell periphery](https://amigo.geneontology.org/amigo/term/GO:0071944), [synaptic membrane](https://amigo.geneontology.org/amigo/term/GO:0097060), and [glutamatergic synapse](https://amigo.geneontology.org/amigo/term/GO:0098978). It is involved in various biological processes such as [neuron migration](https://amigo.geneontology.org/amigo/term/GO:0001764), [cell-matrix adhesion](https://amigo.geneontology.org/amigo/term/GO:0007160), [integrin-mediated signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007229), [Rho protein signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007266), [heart development](https://amigo.geneontology.org/amigo/term/GO:0007507), [memory](https://amigo.geneontology.org/amigo/term/GO:0007613), [response to xenobiotic stimulus](https://amigo.geneontology.org/amigo/term/GO:0009410), [positive regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0010628), [positive regulation of epithelial cell migration](https://amigo.geneontology.org/amigo/term/GO:0010634), [positive regulation of cell-substrate adhesion](https://amigo.geneontology.org/amigo/term/GO:0010811), [positive regulation of neuron projection development](https://amigo.geneontology.org/amigo/term/GO:0010976), [regulation of transforming growth factor beta receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0017015), [regulation of Wnt signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0030111), [lung development](https://amigo.geneontology.org/amigo/term/GO:0030324), [regulation of BMP signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0030510), [negative regulation of cell projection organization](https://amigo.geneontology.org/amigo/term/GO:0031345), [response to gonadotropin](https://amigo.geneontology.org/amigo/term/GO:0034698), [negative regulation of Rho protein signal transduction](https://amigo.geneontology.org/amigo/term/GO:0035024), [exploration behavior](https://amigo.geneontology.org/amigo/term/GO:0035640), [skin development](https://amigo.geneontology.org/amigo/term/GO:0043588), [mesodermal cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0048333), [leukocyte migration](https://amigo.geneontology.org/amigo/term/GO:0050900), [maternal process involved in female pregnancy](https://amigo.geneontology.org/amigo/term/GO:0060135), [nephron development](https://amigo.geneontology.org/amigo/term/GO:0072006), [dendritic spine maintenance](https://amigo.geneontology.org/amigo/term/GO:0097062), [renal filtration](https://amigo.geneontology.org/amigo/term/GO:0097205), [cell-cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0098609), [synaptic membrane adhesion](https://amigo.geneontology.org/amigo/term/GO:0099560), and [positive regulation of protein localization to plasma membrane](https://amigo.geneontology.org/amigo/term/GO:1903078). Additionally, it enables various molecular functions such as [fibronectin binding](https://amigo.geneontology.org/amigo/term/GO:0001968), [protease binding](https://amigo.geneontology.org/amigo/term/GO:0002020), [integrin binding](https://amigo.geneontology.org/amigo/term/GO:0005178), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [collagen binding](https://amigo.geneontology.org/amigo/term/GO:0005518), [protein domain specific binding](https://amigo.geneontology.org/amigo/term/GO:0019904), [laminin binding](https://amigo.geneontology.org/amigo/term/GO:0043236), [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872), and [protein heterodimerization activity](https://amigo.geneontology.org/amigo/term/GO:0046982). It is also part of the [integrin complex](https://amigo.geneontology.org/amigo/term/GO:0008305), [integrin alpha3-beta1 complex](https://amigo.geneontology.org/amigo/term/GO:0034667), and [receptor complex](https://amigo.geneontology.org/amigo/term/GO:0043235), and is active in the [external side of plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009897) and located in the [growth cone filopodium](https://amigo.geneontology.org/amigo/term/GO:1990812).


The gene length is 3,769 base pairs (10.48% of all genes), the mature length is 742.5 base pairs, and the primary transcript length is 3,766.5 base pairs.


The gene ITGA3 (NCBI ID: 3675) has been mentioned in [81 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ITGA3%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning this gene is 1995, and the middle 50% of publications occurred between 2012 and 2020.


The top 5 publications mentioning ITGA3, ranked by their scientific influence, include studies that explore the gene's role in various biological contexts. The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Notable publications include "[Membrane-anchored heparin-binding EGF-like growth factor (HB-EGF) and diphtheria toxin receptor-associated protein (DRAP27)/CD9 form a complex with integrin alpha 3 beta 1 at cell-cell contact sites.](https://pubmed.ncbi.nlm.nih.gov/7790364)" (1995) (relative citation ratio: 4.55), "[Integrin Î±3 mutations with kidney, lung, and skin disease.](https://pubmed.ncbi.nlm.nih.gov/22512483)" (2012) (relative citation ratio: 4.55), "[Tumor-suppressive microRNA-223 inhibits cancer cell migration and invasion by targeting ITGA3/ITGB1 signaling in prostate cancer.](https://pubmed.ncbi.nlm.nih.gov/26509963)" (2016) (relative citation ratio: 4.55), "[Regulation of ITGA3 by the anti-tumor miR-199 family inhibits cancer cell migration and invasion in head and neck cancer.](https://pubmed.ncbi.nlm.nih.gov/28612520)" (2017) (relative citation ratio: 4.49), and "[Exosomal ITGA3 interferes with non-cancerous prostate cell functions and is increased in urine exosomes of metastatic prostate cancer patients.](https://pubmed.ncbi.nlm.nih.gov/24371517)" (2013) (relative citation ratio: 3.74). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ITGA3](https://www.proteinatlas.org/ENSG00000005884-ITGA3) is an integrin receptor with evidence at the protein level. It is involved in cancer-related genes and is a disease variant associated with epidermolysis bullosa. ITGA3 is detected in many tissues and is primarily located in the plasma membrane. It is expressed in the thyroid gland with unknown function (Cluster 51), in neurons with mixed function (Cluster 1), in neuronal cell lines involved in signal transduction (Cluster 72), and in non-specific single cells with mixed function (Cluster 45).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SP1](https://www.ncbi.nlm.nih.gov/gene/6667) and [AR](https://www.ncbi.nlm.nih.gov/gene/367), each with 6 experiments. Additionally, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613), and [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) were each found to be regulating in 5 experiments.



The GWAS data indicates associations with various disease conditions, including mental health disorders, sleep disorders, and specifically narcolepsy.



The gene is ubiquitously expressed in multiple tissue samples, including blood vessel, thyroid, lung, and kidney, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Cardiac Myocytes, Thyroid, Bronchial Epithelial Cells, Colorectal Adenocarcinoma, Lung, Placenta, and Fetal Thyroid.


The proteins are involved in various pathways, including extracellular matrix organization, integrin cell surface interactions, laminin interactions, basigin interactions, cell surface interactions at the vascular wall, and hemostasis.


The mouse ortholog gene 16400 and its human ortholog gene 3675 are associated with several phenotypic abnormalities, including abnormal craniofacial morphology, abnormal placenta morphology and size, embryonic growth retardation, and complete penetrance preweaning lethality. These phenotypes are observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.2078 (68.35th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -6.55 (29.51st percentile), and the median structural flexibility is 0.996 (23.89th percentile). The protein has a length of 1051 amino acids (88.82nd percentile) and a molecular weight of 116610.90 Da (88.7th percentile). The secondary structure affinities are 30.07% for helix (32.55th percentile), 36.44% for sheet (69.3rd percentile), and 30.26% for turn (65.61st percentile). The instability index is 40.41 (23.45th percentile), and the isoelectric point is 6.32 (38.63rd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |