---
layout: post
title: NUP50
date: 2025-01-17 16:55 CST
description: NUP50 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10762) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10762  | NUP50 | ENSG00000093000 | 22q13.31 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is located in the [nuclear envelope](https://amigo.geneontology.org/amigo/term/GO:0005635), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), and [nuclear membrane](https://amigo.geneontology.org/amigo/term/GO:0031965). It is also part of the [nuclear pore](https://amigo.geneontology.org/amigo/term/GO:0005643). The gene is involved in several processes, including [protein import into the nucleus](https://amigo.geneontology.org/amigo/term/GO:0006606), [nucleocytoplasmic transport](https://amigo.geneontology.org/amigo/term/GO:0006913), and [mRNA transport](https://amigo.geneontology.org/amigo/term/GO:0051028).


The gene length is 53,910 base pairs (68.48% of all genes), the mature length is 3,530 base pairs, and the primary transcript length is 53,910 base pairs.


The gene NUP50 (NCBI ID: 10762) has been mentioned in [9 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22NUP50%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning this gene was in 2000, and the middle 50% of publications occurred between 2007 and 2021.


The top five publications mentioning NUP50, ranked by their relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)), include "[hsa-miR-191-5p inhibits replication of human immunodeficiency virus type 1 by downregulating the expression of NUP50.](https://pubmed.ncbi.nlm.nih.gov/33420627)" (2021) (relative citation ratio: 1.43), "[Characterization and targeted disruption of murine Nup50, a p27(Kip1)-interacting component of the nuclear pore complex.](https://pubmed.ncbi.nlm.nih.gov/10891500)" (2000) (relative citation ratio: 1.31), "[The Nup153-Nup50 protein interface and its role in nuclear import.](https://pubmed.ncbi.nlm.nih.gov/23007389)" (2012) (relative citation ratio: 0.96), "[Nucleoporin Nup50 stabilizes closed conformation of armadillo repeat 10 in importin Î±5.](https://pubmed.ncbi.nlm.nih.gov/22130666)" (2012) (relative citation ratio: 0.54), and "[Nucleoporin 50 mediates Kcna4 transcription to regulate cardiac electrical activity.](https://pubmed.ncbi.nlm.nih.gov/34409458)" (2021) (relative citation ratio: 0.47). The relative citation ratio measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[NUP50](https://www.proteinatlas.org/ENSG00000093000-NUP50) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in the nucleoplasm and nuclear membrane. The gene is expressed in various clusters, including Cluster 35 in neutrophils related to protein binding, Cluster 67 in lymphoid tissue and bone marrow associated with cell proliferation, Cluster 5 in the brain with mixed function, Cluster 66 in non-specific cell lines related to mitochondria, and Cluster 55 in single cells related to transcription.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) and [MYC](https://www.ncbi.nlm.nih.gov/gene/4609), each with 6 experiments. Additionally, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), and [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313) were each found to be regulating in 5 experiments.



The input data indicates associations with a variety of disease conditions, including several types of cancer such as leukemia, lymphoma, and immune system cancer. Additionally, respiratory conditions like asthma, bronchial disease, and obstructive lung disease are noted. Cardiovascular issues, including coronary artery disease and vascular disease, are also present. Furthermore, the data encompasses broader categories such as hematopoietic system disease, immune system disease, and general disease of anatomical entities.



The gene is expressed in various tissues, including skin, blood, and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in 721_B_lymphoblasts, WholeBlood, CD105+_Endothelial cells, CD71+_EarlyErythroid cells, SuperiorCervicalGanglion, and Leukemialymphoblastic (MOLT-4) cells.


The input data highlights various biological pathways and processes, including the metabolism of carbohydrates, glycogen storage diseases, and gene expression. It also covers transmembrane transport mechanisms, such as SLC-mediated transport and the transport of small molecules. Additionally, the data includes pathways related to influenza infection, its life cycle, and antiviral mechanisms involving IFN-stimulated genes and ISG15. Several pathways related to HIV infection and its life cycle are also noted, including interactions with host cellular proteins and the nuclear export of viral RNA. Furthermore, the data encompasses glucose and hexose transport, the regulation of glucokinase, and various stages of the cell cycle, including mitotic prophase and nuclear envelope breakdown. Immune system processes, such as cytokine signaling and interferon signaling, are also represented, along with regulatory RNA pathways and the processing of pre-mRNA.



The analyzed protein sequence has a GRAVY value of -0.617 (23.78th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -0.973 (46.87th percentile), and the median structural flexibility is 1.017 (94.68th percentile). The protein has a length of 468 amino acids (54.21st percentile) and a molecular weight of 50143.57 Da (51.84th percentile). The isoelectric point is 6.64 (45.3rd percentile), and the instability index is 43.06 (32.42nd percentile). The fractions of amino acids in helix, sheet, and turn structures are 0.342 (69.27th percentile), 0.288 (22.45th percentile), and 0.368 (93.19th percentile), respectively. For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |