---
layout: post
title: HTR1B
date: 2024-12-08 23:57 CST
description: HTR1B description
tags: [cooccuring-genes]
categories: longest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3351) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3351  | HTR1B | ENSG00000135312 | 6q14.1 |



The gene length is 927,612.0 base pairs (99.56% of all genes), the mature length is 6,230.5 base pairs, and the primary transcript length is 926,988.5 base pairs.


The gene HTR1B, with NCBI ID 3351, has been the subject of [219 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22HTR1B%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1986, and the middle 50% of publications occurred between 1998 and 2012.


The top five publications mentioning HTR1B, ranked by their scientific influence, include "[Serotonin receptors in the human brain. I. Characterization and autoradiographic localization of 5-HT1A recognition sites. Apparent absence of 5-HT1B recognition sites.](https://pubmed.ncbi.nlm.nih.gov/3719375)" (1986) (relative citation ratio: 13.47), "[Triptans (serotonin, 5-HT1B/1D agonists) in migraine: detailed results and methods of a meta-analysis of 53 trials.](https://pubmed.ncbi.nlm.nih.gov/12383060)" (2002) (relative citation ratio: 11.82), "[Localization of 5-HT1B, 5-HT1D alpha, 5-HT1E and 5-HT1F receptor messenger RNA in rodent and primate brain.](https://pubmed.ncbi.nlm.nih.gov/7984275)" (1994) (relative citation ratio: 9.01), "[Cryo-EM structure of the serotonin 5-HT<sub>1B</sub> receptor coupled to heterotrimeric G<sub>o</sub>.](https://pubmed.ncbi.nlm.nih.gov/29925951)" (2018) (relative citation ratio: 7.51), and "[Sumatriptan (GR 43175) interacts selectively with 5-HT1B and 5-HT1D binding sites.](https://pubmed.ncbi.nlm.nih.gov/2545459)" (1989) (relative citation ratio: 6.74). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


The gene [G-protein coupled receptor](https://www.proteinatlas.org/[Ensembl]-[Gene]) functions as a receptor and transducer, with evidence supporting its role at the protein level. It is an FDA-approved drug target and is detected in some tissues. Subcellularly, it is localized to the endoplasmic reticulum and plasma membrane. The gene is expressed in various clusters, including Cluster 34 (Non-specific - Mixed function) in tissues, Cluster 56 (Hypothalamus - Neuropeptide signaling) in the brain, Cluster 48 (Adipocytes & Endothelial cells - Angiogenesis) in cell lines, and Cluster 32 (Smooth muscle cells - Muscle contraction) in single cells.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SP1](https://www.ncbi.nlm.nih.gov/gene/6667), [RELA](https://www.ncbi.nlm.nih.gov/gene/5970), and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), each with 4 experiments. Additionally, [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664) and [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) were found to be regulating in 3 experiments each.








The mouse ortholog gene 15551 and its human ortholog gene 3351 are associated with the phenotype of preweaning lethality with incomplete penetrance, observed in both sexes.


The analyzed protein sequence has a GRAVY value of 0.342, indicating a relatively hydrophobic nature (92.84th percentile). It exhibits a charge of 8.85 at pH 7.0 (78.98th percentile) and an isoelectric point of 8.96 (78.14th percentile). The protein's structural flexibility is characterized by a median flexibility of 0.985 (7.83rd percentile). In terms of secondary structure, the protein has a helix affinity of 0.297 (29.74th percentile), a sheet affinity of 0.456 (95.14th percentile), and a turn affinity of 0.241 (15.63rd percentile). The instability index is 43.50 (33.92nd percentile), with a length of 390 amino acids (43.91st percentile) and a molecular weight of 43567.57 Da (44.07th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |