---
layout: post
title: SSNA1
date: 2025-01-17 16:55 CST
description: SSNA1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8636) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8636  | SSNA1 | ENSG00000176101 | 9q34.3 |



The gene is involved in [microtubule cytoskeleton organization](https://amigo.geneontology.org/amigo/term/GO:0000226) and [microtubule nucleation](https://amigo.geneontology.org/amigo/term/GO:0007020), playing a crucial role in the [cell cycle](https://amigo.geneontology.org/amigo/term/GO:0007049) and [cell division](https://amigo.geneontology.org/amigo/term/GO:0051301). It enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [microtubule binding](https://amigo.geneontology.org/amigo/term/GO:0008017), [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802), and [protein self-association](https://amigo.geneontology.org/amigo/term/GO:0043621). The gene is located in various cellular components, including the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [centrosome](https://amigo.geneontology.org/amigo/term/GO:0005813), [centriole](https://amigo.geneontology.org/amigo/term/GO:0005814), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [axoneme](https://amigo.geneontology.org/amigo/term/GO:0005930), [axon](https://amigo.geneontology.org/amigo/term/GO:0030424), [midbody](https://amigo.geneontology.org/amigo/term/GO:0030496), [motile cilium](https://amigo.geneontology.org/amigo/term/GO:0031514), [ciliary basal body](https://amigo.geneontology.org/amigo/term/GO:0036064), and [supramolecular fiber](https://amigo.geneontology.org/amigo/term/GO:0099512). Additionally, it is involved in [axonogenesis](https://amigo.geneontology.org/amigo/term/GO:0007409), [axon extension](https://amigo.geneontology.org/amigo/term/GO:0048675), [axon arborization](https://amigo.geneontology.org/amigo/term/GO:0140060), [receptor clustering](https://amigo.geneontology.org/amigo/term/GO:0043113), and [intraciliary transport](https://amigo.geneontology.org/amigo/term/GO:0042073).


The gene length is 96,094 base pairs (81.05% of all genes), the mature length is 1,202 base pairs (29.08% of all genes), and the primary transcript length is 16,862 base pairs (47.16% of all genes).


The gene SSNA1 (NCBI ID: 8636) has been mentioned in [23 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SSNA1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1962 and the middle 50% of publications occurring between 1994 and 2016. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning SSNA1 include "[Direct induction of microtubule branching by microtubule nucleation factor SSNA1.](https://pubmed.ncbi.nlm.nih.gov/30250060)" (2018) (relative citation ratio: 1.12), "[SSNA1 stabilizes dynamic microtubules and detects microtubule damage.](https://pubmed.ncbi.nlm.nih.gov/34970964)" (2021) (relative citation ratio: 1.08), "[Chlamydomonas DIP13 and human NA14: a new class of proteins associated with microtubule structures is involved in cell division.](https://pubmed.ncbi.nlm.nih.gov/12640030)" (2003) (relative citation ratio: 0.75), "[An evolutionarily conserved SSNA1/DIP13 homologue is a component of both basal and apical complexes of Toxoplasma gondii.](https://pubmed.ncbi.nlm.nih.gov/27324377)" (2016) (relative citation ratio: 0.7), and "[Structure-Cytotoxicity Relationships of Analogues of N<sup>14</sup>-Desacetoxytubulysin H.](https://pubmed.ncbi.nlm.nih.gov/27809515)" (2016) (relative citation ratio: 0.46). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[SSNA1](https://www.proteinatlas.org/ENSG00000176101-SSNA1) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in the nucleoplasm, vesicles, basal body, and cytosol. The gene is expressed in various clusters, including Cluster 29 for non-specific unknown function in blood, Cluster 46 for testis-related spermatogenesis in tissues, Cluster 13 for oligodendrocytes with mixed function in the brain, and Cluster 43 for basic cellular processes in cell lines. Additionally, it is involved in basic cellular processes in single-cell expression clusters.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [E2F8](https://www.ncbi.nlm.nih.gov/gene/79733), [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), and [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), each with 5 experiments. Additionally, [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599) and [HDAC1](https://www.ncbi.nlm.nih.gov/gene/3065) were found to be regulating in 4 experiments each.





The gene is expressed in various tissues, including the prostate and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Lymphoma (Burkitt's), Heart, CD71+ Early Erythroid cells, Colorectal Adenocarcinoma, and multiple regions of the Testis, including the interstitial tissue and seminiferous tubules.


The proteins are involved in various pathways related to the cell cycle, including mitotic cell cycle, centrosome maturation, and G2/M transition. Additionally, they participate in the anchoring of the basal body to the plasma membrane, recruitment of mitotic centrosome proteins and complexes, and the regulation of PLK1 activity at the G2/M transition. These proteins also play roles in the assembly of the primary cilium and organelle biogenesis and maintenance.



The analyzed protein sequence has a GRAVY value of -0.9403 (5.93rd percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -2.49 (41.28th percentile), and the median structural flexibility is 1.015 (91.77th percentile). The protein has a high affinity for helix structures (0.437, 97.54th percentile), with lower affinities for sheet (0.277, 17.42nd percentile) and turn (0.202, 2.66th percentile) structures. The instability index is 56.89 (76.81st percentile), and the isoelectric point is 5.37 (15.11th percentile). The protein is 119 amino acids long (4.59th percentile) with a molecular weight of 13596.06 Da (4.88th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |