---
layout: post
title: PRPF3
date: 2025-01-17 16:55 CST
description: PRPF3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9129) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9129  | PRPF3 | ENSG00000117360 | 1q21.2 |



The gene is involved in [spliceosomal tri-snRNP complex assembly](https://amigo.geneontology.org/amigo/term/GO:0000244), [RNA splicing, via transesterification reactions](https://amigo.geneontology.org/amigo/term/GO:0000375), and [mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0000398). It enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [Cajal body](https://amigo.geneontology.org/amigo/term/GO:0015030), and [nuclear speck](https://amigo.geneontology.org/amigo/term/GO:0016607). It is part of the [spliceosomal complex](https://amigo.geneontology.org/amigo/term/GO:0005681), [U4/U6 x U5 tri-snRNP complex](https://amigo.geneontology.org/amigo/term/GO:0046540), and [U2-type precatalytic spliceosome](https://amigo.geneontology.org/amigo/term/GO:0071005). Additionally, it is involved in [mRNA processing](https://amigo.geneontology.org/amigo/term/GO:0006397) and [RNA splicing](https://amigo.geneontology.org/amigo/term/GO:0008380), and enables [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802).


The gene length is 19,163 base pairs (40.99% of all genes), the mature length is 2,934 base pairs, and the primary transcript length is 19,120.5 base pairs (50.4% of all genes).


The gene PRPF3 (NCBI ID: 9129) has been mentioned in [40 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PRPF3%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1973 and the middle 50% of publications occurring between 1998 and 2014. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning PRPF3, ranked by their scientific influence as measured by the [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541), include: ["The primary structures of six human salivary acidic proline-rich proteins (PRP-1, PRP-2, PRP-3, PRP-4, PIF-s and PIF-f)."](https://pubmed.ncbi.nlm.nih.gov/3196309) (1988) (RCR: 4.56), ["Mutations in HPRP3, a third member of pre-mRNA splicing factor genes, implicated in autosomal dominant retinitis pigmentosa."](https://pubmed.ncbi.nlm.nih.gov/11773002) (2002) (RCR: 3.03), ["Application of RP-18 TLC Retention Data to the Prediction of the Transdermal Absorption of Drugs."](https://pubmed.ncbi.nlm.nih.gov/33673150) (2021) (RCR: 2.39), ["TMEM43 promotes pancreatic cancer progression by stabilizing PRPF3 and regulating RAP2B/ERK axis."](https://pubmed.ncbi.nlm.nih.gov/35260078) (2022) (RCR: 2.04), and ["RP-18 TLC and Computational Descriptors of Skin Permeability of Sunscreens."](https://pubmed.ncbi.nlm.nih.gov/35130549) (2022) (RCR: 1.97). The RCR values measure the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PRPF3](https://www.proteinatlas.org/ENSG00000117360-PRPF3) is a gene with evidence at the protein level, known for its involvement in Retinitis pigmentosa. It is detected in all RNA tissue distributions and is primarily located in the nucleoplasm. The gene is part of various expression clusters, including Cluster 33 for blood (non-specific - ATP binding), Cluster 40 for tissue (non-specific - transcription), Cluster 6 for brain (non-specific - immune response), Cluster 45 for cell line (non-specific - nuclear processes), and Cluster 55 for single cell (non-specific - transcription).


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 4 experiments, [TAF1](https://www.ncbi.nlm.nih.gov/gene/6872) with 4 experiments, [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) with 4 experiments, [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599) with 3 experiments, and [SPI1](https://www.ncbi.nlm.nih.gov/gene/6688) with 3 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with a range of conditions, including cardiovascular system disease, cognitive disorder, bipolar disorder, mood disorder, psychotic disorder, schizophrenia, endogenous depression, major depressive disorder, mental depression, developmental disorder of mental health, specific developmental disorder, attention deficit hyperactivity disorder, heart conduction disease, autism spectrum disorder, and pervasive developmental disorder.



The gene is expressed in various immune cell types, including CD34+, CD4+ T cells, CD8+ T cells, and CD19+ B cells (neg. sel.). Additionally, it is found in leukemia cells, specifically chronic myelogenous leukemia (K-562) and lymphoblastic leukemia (MOLT-4), as well as in lymph node tissue. Furthermore, the gene is expressed in the thyroid.



The mouse ortholog gene 70767 and its human ortholog 9129 are associated with several phenotypes, including embryonic lethality prior to organogenesis, embryonic lethality prior to tooth bud stage, and preweaning lethality with complete penetrance. Notably, the preweaning lethality phenotype is observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.807 (11.39th percentile), indicating a hydrophilic nature. It exhibits a charge of 24.92 at pH 7.0 (95.75th percentile) and a median structural flexibility of 1.015 (91.48th percentile). The protein's secondary structure is predicted to be 39.53% helix (92.53rd percentile), 29.28% sheet (24.64th percentile), and 24.60% turn (18.69th percentile). The instability index is 48.13 (50.63rd percentile), with an isoelectric point of 9.50 (91.07th percentile). The protein has a length of 683 amino acids (74.26th percentile) and a molecular weight of 77528.02 Da (74.99th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |