---
layout: post
title: MBNL1
date: 2025-01-17 16:55 CST
description: MBNL1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4154) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4154  | MBNL1 | ENSG00000152601 | 3q25.1-q25.2 |



The gene enables [regulatory region RNA binding](https://amigo.geneontology.org/amigo/term/GO:0001069) and [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), and is involved in [in utero embryonic development](https://amigo.geneontology.org/amigo/term/GO:0001701). It is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and [cytoplasmic stress granule](https://amigo.geneontology.org/amigo/term/GO:0010494). Additionally, it is involved in [mRNA processing](https://amigo.geneontology.org/amigo/term/GO:0006397), [nervous system development](https://amigo.geneontology.org/amigo/term/GO:0007399), [RNA splicing](https://amigo.geneontology.org/amigo/term/GO:0008380), [embryonic limb morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0030326), [regulation of RNA splicing](https://amigo.geneontology.org/amigo/term/GO:0043484), and [myoblast differentiation](https://amigo.geneontology.org/amigo/term/GO:0045445). The gene also enables [double-stranded RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003725), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872).


The gene length is 3951.0 base pairs (11.05% of all genes), the mature length is 745.0 base pairs, and the primary transcript length is 3122.0 base pairs.


The gene MBNL1 (NCBI ID: 4154) has been mentioned in [157 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22MBNL1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count), with the earliest publication in 1975. The middle 50% of publications occurred between 2012 and 2020.


The top five publications mentioning MBNL1, ranked by their scientific influence, include "[Three proteins, MBNL, MBLL and MBXL, co-localize in vivo with nuclear foci of expanded-repeat transcripts in DM1 and DM2 cells.](https://pubmed.ncbi.nlm.nih.gov/11929853)" (2002) (relative citation ratio: 5.78), "[Antagonistic regulation of mRNA expression and splicing by CELF and MBNL proteins.](https://pubmed.ncbi.nlm.nih.gov/25883322)" (2015) (relative citation ratio: 4.18), "[MBNL proteins and their target RNAs, interaction and splicing regulation.](https://pubmed.ncbi.nlm.nih.gov/25183524)" (2014) (relative citation ratio: 3.6), "[A simple ligand that selectively targets CUG trinucleotide repeats and inhibits MBNL protein binding.](https://pubmed.ncbi.nlm.nih.gov/19805260)" (2009) (relative citation ratio: 3.5), and "[CUGBP1 and MBNL1 preferentially bind to 3' UTRs and facilitate mRNA decay.](https://pubmed.ncbi.nlm.nih.gov/22355723)" (2012) (relative citation ratio: 3.25). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[MBNL1](https://www.proteinatlas.org/ENSG00000152601-MBNL1) is an RNA-binding protein involved in corneal dystrophy, with evidence supporting its role at the protein level. It is detected in all tissues and primarily localizes to the nucleoplasm and cytosol. MBNL1 is expressed in various clusters, including Cluster 35 in neutrophils, Cluster 19 in non-specific tissues involved in basic cellular processes, Cluster 13 in oligodendrocytes, Cluster 63 in non-specific cell lines involved in protein processing, and Cluster 55 in non-specific single cells involved in transcription.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025) with 7 experiments, [SUPT5H](https://www.ncbi.nlm.nih.gov/gene/6829) and [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) each with 6 experiments, and [FOXM1](https://www.ncbi.nlm.nih.gov/gene/2305) and [YY1](https://www.ncbi.nlm.nih.gov/gene/7528) each with 5 experiments.



The input data from GWAS indicates associations with a wide range of disease conditions, including various types of cancer such as leukemia, lymphoma, and immune system cancer. Additionally, respiratory diseases like asthma, bronchial disease, and obstructive lung disease are highlighted. Metabolic disorders, particularly diabetes mellitus and type 2 diabetes mellitus, are also prominent. Furthermore, the data includes cardiovascular system diseases and neurodegenerative conditions such as Alzheimer's disease and dementia. Other notable associations include obesity and diseases of the hematopoietic and immune systems.



The gene is expressed in various tissues, including muscle, adipose tissue, and blood vessel. Additionally, it is expressed in several immune cell types such as CD33+ myeloid cells, CD56+ NK cells, CD4+ T cells, CD8+ T cells, and CD19+ B cells.




The protein sequence analyzed has a GRAVY value of -0.190 (70.2% percentile), indicating a relatively hydrophilic nature. It exhibits a charge of 12.27 at pH 7.0 (85.56% percentile) and a median structural flexibility of 0.998 (31.34% percentile). The protein's secondary structure is predicted to be 34.3% helix (70.0% percentile), 29.9% sheet (27.77% percentile), and 25.8% turn (27.31% percentile). The instability index is 33.4 (7.62% percentile), suggesting a stable protein. The isoelectric point is 9.16 (83.43% percentile), with a length of 388 amino acids (43.63% percentile) and a molecular weight of 41816.8 Da (41.82% percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |