---
layout: post
title: UCP3
date: 2025-01-17 16:55 CST
description: UCP3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7352) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7352  | UCP3 | ENSG00000175564 | 11q13.4 |



The gene is located in the [mitochondrion](https://amigo.geneontology.org/amigo/term/GO:0005739) and specifically in the [mitochondrial inner membrane](https://amigo.geneontology.org/amigo/term/GO:0005743). It is involved in various processes including [response to superoxide](https://amigo.geneontology.org/amigo/term/GO:0000303), [response to hypoxia](https://amigo.geneontology.org/amigo/term/GO:0001666), [lipid metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006629), [fatty acid metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006631), [response to nutrient](https://amigo.geneontology.org/amigo/term/GO:0007584), [respiratory gaseous exchange by respiratory system](https://amigo.geneontology.org/amigo/term/GO:0007585), [response to cold](https://amigo.geneontology.org/amigo/term/GO:0009409), [response to activity](https://amigo.geneontology.org/amigo/term/GO:0014823), [response to insulin](https://amigo.geneontology.org/amigo/term/GO:0032868), [cellular response to hormone stimulus](https://amigo.geneontology.org/amigo/term/GO:0032870), [response to glucocorticoid](https://amigo.geneontology.org/amigo/term/GO:0051384), [lipid hydroperoxide transport](https://amigo.geneontology.org/amigo/term/GO:1901373), [response to fenofibrate](https://amigo.geneontology.org/amigo/term/GO:1901557), [proton transmembrane transport](https://amigo.geneontology.org/amigo/term/GO:1902600), [mitochondrial transmembrane transport](https://amigo.geneontology.org/amigo/term/GO:1990542), and [adaptive thermogenesis](https://amigo.geneontology.org/amigo/term/GO:1990845). The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [proton transmembrane transporter activity](https://amigo.geneontology.org/amigo/term/GO:0015078), and [oxidative phosphorylation uncoupler activity](https://amigo.geneontology.org/amigo/term/GO:0017077).


The gene length is 6,480 base pairs (17.79% of all genes), the mature length is 4,256 base pairs, and the primary transcript length is 6,451 base pairs.


The gene UCP3 (NCBI ID: 7352) has been mentioned in [209 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22UCP3%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning UCP3 is 1997, and the middle 50% of publications occurred between 2002 and 2012.


The top five publications mentioning UCP3, ranked by their scientific influence, include "[Uncoupling protein-3: a new member of the mitochondrial carrier family with tissue-specific expression.](https://pubmed.ncbi.nlm.nih.gov/9180264)" (1997) (relative citation ratio: 23.08), "[UCP3: an uncoupling protein homologue expressed preferentially and abundantly in skeletal muscle and brown adipose tissue.](https://pubmed.ncbi.nlm.nih.gov/9196039)" (1997) (relative citation ratio: 16.07), "[The uncoupling protein homologues: UCP1, UCP2, UCP3, StUCP and AtUCP.](https://pubmed.ncbi.nlm.nih.gov/10620491)" (2000) (relative citation ratio: 14.08), "[Physiological functions of the mitochondrial uncoupling proteins UCP2 and UCP3.](https://pubmed.ncbi.nlm.nih.gov/16098826)" (2005) (relative citation ratio: 13.5), and "[Mice overexpressing human uncoupling protein-3 in skeletal muscle are hyperphagic and lean.](https://pubmed.ncbi.nlm.nih.gov/10935638)" (2000) (relative citation ratio: 10.58). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[UCP3](https://www.proteinatlas.org/ENSG00000175564-UCP3) is implicated in diabetes mellitus, disease variants, and obesity, with evidence at the protein level. The gene is detected in many tissues and is primarily expressed in skeletal muscle, where it is involved in muscle contraction. Subcellularly, UCP3 is localized to the nucleoplasm, plasma membrane, and cytosol. In the brain, it is found in non-specific clusters with mixed functions, while in cell lines, it is associated with lymphoma and adaptive immune response. Single-cell expression analysis shows its presence in skeletal myocytes involved in muscle contraction.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [HNF4A](https://www.ncbi.nlm.nih.gov/gene/3172), [TBP](https://www.ncbi.nlm.nih.gov/gene/6908), [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), [ONECUT2](https://www.ncbi.nlm.nih.gov/gene/9480), and [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), each shown to be regulating in 2 experiments.





The gene is expressed in various tissues, including muscle and thyroid, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the heart, subthalamic nucleus, pons, adrenal cortex, skeletal muscle, ciliary ganglion, and uterus corpus.


The input data highlights several key metabolic processes and pathways, including metabolism, respiratory electron transport, ATP synthesis by chemiosmotic coupling, and heat production by uncoupling proteins. Additionally, it covers the citric acid (TCA) cycle and respiratory electron transport, mitochondrial uncoupling proteins, the proton buffering model, and the fatty acid cycling model.



The analyzed protein sequence has a GRAVY value of 0.0747 (87.11th percentile), indicating a relatively hydrophobic nature. The charge at pH 7.0 is 10.70 (82.87th percentile), and the median structural flexibility is 0.993 (15.44th percentile). The protein has a propensity for helix, sheet, and turn structures of 29.17% (24.82nd percentile), 39.42% (84.48th percentile), and 26.28% (31.68th percentile), respectively. The instability index is 39.31 (20.31st percentile), and the isoelectric point is 9.31 (87.27th percentile). The protein is 312 amino acids long (30.84th percentile) with a molecular weight of 34215.50 Da (29.53rd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |