---
layout: post
title: MAP1A
date: 2024-12-08 23:57 CST
description: MAP1A description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4130) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4130  | MAP1A | ENSG00000166963 | 15q15.3 |



The gene length is 117,609.0 base pairs (84.87% of all genes), the mature length is 2,991.0 base pairs, and the primary transcript length is 112,112.0 base pairs.


MAP1A, with NCBI Gene ID 4130, has been the subject of [26 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22MAP1A%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1989 and the middle 50% of publications occurring between 1996 and 2008. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning MAP1A, ranked by their scientific influence, include "[DISC1 (Disrupted-In-Schizophrenia 1) is a centrosome-associated protein that interacts with MAP1A, MIPT3, ATF4/5 and NUDEL: regulation and loss of interaction with mutation.](https://pubmed.ncbi.nlm.nih.gov/12812986)" (2003) (relative citation ratio: 5.68), "[Microtubule formation and neurite growth in cerebellar macroneurons which develop in vitro: evidence for the involvement of the microtubule-associated proteins, MAP-1a, HMW-MAP2 and Tau.](https://pubmed.ncbi.nlm.nih.gov/2509111)" (1989) (relative citation ratio: 2.87), "[Differential binding regulation of microtubule-associated proteins MAP1A, MAP1B, and MAP2 by tubulin polyglutamylation.](https://pubmed.ncbi.nlm.nih.gov/11278895)" (2001) (relative citation ratio: 2.82), "[Microtubule-associated protein MAP1A, MAP1B, and MAP2 proteolysis during soluble amyloid beta-peptide-induced neuronal apoptosis. Synergistic involvement of calpain and caspase-3.](https://pubmed.ncbi.nlm.nih.gov/16234245)" (2006) (relative citation ratio: 2.19), and "[Gene localization and developmental expression of light chain 3: a common subunit of microtubule-associated protein 1A(MAP1A) and MAP1B.](https://pubmed.ncbi.nlm.nih.gov/8833088)" (1996) (relative citation ratio: 0.95). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [PPARG](https://www.ncbi.nlm.nih.gov/gene/5468) and [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170), each with 6 experiments. Additionally, [ELF1](https://www.ncbi.nlm.nih.gov/gene/1997), [TAL1](https://www.ncbi.nlm.nih.gov/gene/6886), and [HDAC2](https://www.ncbi.nlm.nih.gov/gene/3066) were each found to be regulating in 5 experiments.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, filtered for Perturb-seq data, include [STAT3](https://www.ncbi.nlm.nih.gov/gene/5885) with 3 experiments, [IRF1](https://www.ncbi.nlm.nih.gov/gene/9126) with 4 experiments, [CEBPB](https://www.ncbi.nlm.nih.gov/gene/6829) with 2 experiments, [EGR1](https://www.ncbi.nlm.nih.gov/gene/8861) with 2 experiments, [FOXA1](https://www.ncbi.nlm.nih.gov/gene/2623) with 1 experiment, and [RORA](https://www.ncbi.nlm.nih.gov/gene/9968) with 1 experiment. These TFs were found to regulate gene expression in multiple Perturb-seq experiments, highlighting their significant roles in transcriptional regulation.


MAP1A has been associated with various traits and diseases through genome-wide association studies. Notably, it has been linked to [Apolipoprotein A1 levels](https://pubmed.ncbi.nlm.nih.gov/33462484) (closest MeSH match: [Apolipoproteins](https://meshb.nlm.nih.gov/record/ui?ui=D001053)), [Body mass index](https://pubmed.ncbi.nlm.nih.gov/36581621) (closest MeSH match: [Body Mass Index](https://meshb.nlm.nih.gov/record/ui?ui=D015992)), and [C-reactive protein levels](https://pubmed.ncbi.nlm.nih.gov/33462484) (closest MeSH match: [Proteins](https://meshb.nlm.nih.gov/record/ui?ui=D011506)). Additionally, associations include [Calcium levels](https://pubmed.ncbi.nlm.nih.gov/33462484) (closest MeSH match: [Calcium](https://meshb.nlm.nih.gov/record/ui?ui=D002118)), [Cholesterol levels in small VLDL](https://pubmed.ncbi.nlm.nih.gov/35213538) (closest MeSH match: [Cholesterol](https://meshb.nlm.nih.gov/record/ui?ui=D002784)), and [Gamma glutamyl transferase levels](https://pubmed.ncbi.nlm.nih.gov/33462484) (closest MeSH match: [Transferases](https://meshb.nlm.nih.gov/record/ui?ui=D014166)). Other notable associations include [Hemoglobin levels](https://pubmed.ncbi.nlm.nih.gov/32327693) (closest MeSH match: [Hemoglobins](https://meshb.nlm.nih.gov/record/ui?ui=D006454)), [High density lipoprotein cholesterol levels](https://pubmed.ncbi.nlm.nih.gov/29507422) (closest MeSH match: [Lipoproteins](https://meshb.nlm.nih.gov/record/ui?ui=D008074)), and [Insulin-like growth factor 1 levels](https://pubmed.ncbi.nlm.nih.gov/33462484) (closest MeSH match: [Growth](https://meshb.nlm.nih.gov/record/ui?ui=D006128)). The mapping of these traits to MeSH terms was facilitated using a method called [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0).





The analyzed protein sequence has a GRAVY value of -0.893 (7.68th percentile), indicating a hydrophilic nature. It exhibits a charge of -165.32 at pH 7.0 (0.2nd percentile) and a median structural flexibility of 1.022 (97.59th percentile). The protein's secondary structure is predicted to be 37.14% helix (85.8th percentile), 24.97% sheet (8.58th percentile), and 34.50% turn (87.76th percentile). The instability index is 65.98 (91.33rd percentile), with an isoelectric point of 4.85 (5.2nd percentile). The protein has a length of 2803 amino acids (99.01st percentile) and a molecular weight of 305481.44 Da (98.98th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |