---
layout: post
title: FUT5
date: 2024-12-08 23:57 CST
description: FUT5 description
tags: [cooccuring-genes]
categories: longest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2527) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2527  | FUT5 | ENSG00000130383 | 19p13.3 |



The gene length is 768,769.0 base pairs (99.31% of all genes), the mature length is 3,806.0 base pairs, and the primary transcript length is 759,579.0 base pairs (99.74% of all genes).


The gene FUT5, with NCBI ID 2527, has been the subject of [17 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22FUT5%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1992, and the middle 50% of publications occurred between 1995 and 2010.


The top 5 publications mentioning FUT5, ranked by their scientific influence, are as follows. The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. The most influential publication is "[miR-125a-3p/FUT5-FUT6 axis mediates colorectal cancer cell proliferation, migration, invasion and pathological angiogenesis via PI3K-Akt pathway.](https://pubmed.ncbi.nlm.nih.gov/28771224)" (2017) with a relative citation ratio of 3.87. Another significant study is "[The cloning and expression of a human alpha-1,3 fucosyltransferase capable of forming the E-selectin ligand.](https://pubmed.ncbi.nlm.nih.gov/1520296)" (1992), which has a relative citation ratio of 3.57. Additionally, "[Acceptor specificity of different length constructs of human recombinant alpha 1,3/4-fucosyltransferases. Replacement of the stem region and the transmembrane domain of fucosyltransferase V by protein A results in an enzyme with GDP-fucose hydrolyzing activity.](https://pubmed.ncbi.nlm.nih.gov/7721776)" (1995) has a relative citation ratio of 3.41. Furthermore, "[Molecular basis for plasma alpha(1,3)-fucosyltransferase gene deficiency (FUT6).](https://pubmed.ncbi.nlm.nih.gov/8175676)" (1994) and "[Structure-function analysis of human alpha 1--&gt;3fucosyltransferases. A GDP-fucose-protected, N-ethylmaleimide-sensitive site in FucT-III and FucT-V corresponds to Ser178 in FucT-IV.](https://pubmed.ncbi.nlm.nih.gov/7713918)" (1995) have relative citation ratios of 1.89 and 1.87, respectively. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), [RBM39](https://www.ncbi.nlm.nih.gov/gene/9584), and [SPI1](https://www.ncbi.nlm.nih.gov/gene/6688), each with 2 experiments. Additionally, [ZNF652](https://www.ncbi.nlm.nih.gov/gene/22834) was identified in 1 experiment.





The genome-wide association study identified a significant association between genetic variants and [Alpha-(1,3)-fucosyltransferase 5 levels (FUT5.4549.78.2)](https://pubmed.ncbi.nlm.nih.gov/29875488). This finding highlights the potential role of FUT5 in modulating fucosyltransferase activity, which is crucial for various biological processes.





The analyzed protein sequence has a GRAVY value of -0.333 (54.08th percentile), indicating a hydrophilic nature. It exhibits a charge of 3.413 at pH 7.0 (62.48th percentile) and a median structural flexibility of 0.996 (24.97th percentile). The protein's secondary structure is predicted to be 28.88% helix (22.75th percentile), 36.10% sheet (66.98th percentile), and 29.95% turn (63.54th percentile). The instability index is 52.68 (65.75th percentile), with an isoelectric point of 8.35 (63.71st percentile). The protein has a length of 374 amino acids (41.74th percentile) and a molecular weight of 42988.59 Da (43.46th percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |