---
layout: post
title: POLR2B
date: 2024-12-08 23:57 CST
description: POLR2B description
tags: [cooccuring-genes]
categories: shortest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5431) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5431  | POLR2B | ENSG00000047315 | 4q12 |



The gene length is 2,792 base pairs (7.6% of all genes), with a mature length of 512 base pairs and a primary transcript length of 512 base pairs.


The gene POLR2B, with NCBI ID 5431, has been the subject of [13 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22POLR2B%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727). The earliest year of publication is 2000, and the middle 50% of publications occurred between 2005 and 2011.


The top five publications mentioning POLR2B, ranked by their scientific influence, include "[Phylogenetic analyses of RPB1 and RPB2 support a middle Cretaceous origin for a clade comprising all agriculturally and medically important fusaria.](https://pubmed.ncbi.nlm.nih.gov/23357352)" (2013) (relative citation ratio: 11.62), "[Contribution of RPB2 to multilocus phylogenetic studies of the euascomycetes (Pezizomycotina, Fungi) with special emphasis on the lichen-forming Acarosporaceae and evolution of polyspory.](https://pubmed.ncbi.nlm.nih.gov/15288074)" (2004) (relative citation ratio: 6.41), "[Ribosomal and RPB2 DNA sequence analyses suggest that Sporidesmium and morphologically similar genera are polyphyletic.](https://pubmed.ncbi.nlm.nih.gov/16908125)" (2006) (relative citation ratio: 1.62), "[Evolutionary relationships of the cup-fungus genus Peziza and Pezizaceae inferred from multiple nuclear genes: RPB2, beta-tubulin, and LSU rDNA.](https://pubmed.ncbi.nlm.nih.gov/15904853)" (2005) (relative citation ratio: 1.5), and "[Phylogenetic utility of protein (RPB2, beta-tubulin) and ribosomal (LSU, SSU) gene sequences in the systematics of Sordariomycetes (Ascomycota, Fungi).](https://pubmed.ncbi.nlm.nih.gov/17072532)" (2007) (relative citation ratio: 1.31). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170) with 6 experiments, followed by [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908), [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), and [TP53](https://www.ncbi.nlm.nih.gov/gene/7157), each with 5 experiments.








The mouse ortholog gene 231329 and its human ortholog gene 5431 are associated with the phenotype of preweaning lethality with complete penetrance, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.371 (49.61 percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -6.10 (30.72 percentile). The median structural flexibility is 1.0002 (42.79 percentile), with a fraction of amino acids in helix, sheet, and turn structures being 0.310 (41.09 percentile), 0.352 (60.48 percentile), and 0.261 (29.81 percentile), respectively. The instability index is 40.49 (23.67 percentile), and the isoelectric point is 6.44 (41.3 percentile). The protein has a length of 1174 amino acids (91.23 percentile) and a molecular weight of 133895.10 Da (91.79 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |