---
layout: post
title: PLCG2
date: 2025-01-17 16:55 CST
description: PLCG2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5336) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5336  | PLCG2 | ENSG00000197943 | 16q23.3 |



The gene is involved in various biological processes, including [cell activation](https://amigo.geneontology.org/amigo/term/GO:0001775), [response to yeast](https://amigo.geneontology.org/amigo/term/GO:0001878), and [positive regulation of receptor internalization](https://amigo.geneontology.org/amigo/term/GO:0002092). It plays a role in signaling pathways such as the [stimulatory C-type lectin receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0002223) and the [toll-like receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0002224). Additionally, it is involved in immune responses, including [macrophage activation](https://amigo.geneontology.org/amigo/term/GO:0002281) and [follicular B cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0002316). The gene enables functions like [phosphotyrosine residue binding](https://amigo.geneontology.org/amigo/term/GO:0001784) and [phosphatidylinositol phospholipase C activity](https://amigo.geneontology.org/amigo/term/GO:0004435), and it is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). It is also active in the [ruffle membrane](https://amigo.geneontology.org/amigo/term/GO:0032587) and involved in processes like [phosphatidylinositol biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006661) and [phospholipid catabolic process](https://amigo.geneontology.org/amigo/term/GO:0009395). Furthermore, the gene acts upstream of or within [positive regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0010628) and is involved in various regulatory processes, including [positive regulation of epithelial cell migration](https://amigo.geneontology.org/amigo/term/GO:0010634), [programmed cell death](https://amigo.geneontology.org/amigo/term/GO:0012501), and [Wnt signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0016055).


The gene length is 39,485 base pairs (60.03% of all genes), the mature length is 4,269.5 base pairs (85.99% of all genes), and the primary transcript length is 36,551 base pairs (67.85% of all genes).


The gene PLCG2 (NCBI ID: 5336) has been mentioned in [118 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PLCG2%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1992 and the middle 50% of publications occurring between 2002 and 2019. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning PLCG2, ranked by their scientific influence, include "[Rare coding variants in PLCG2, ABI3, and TREM2 implicate microglial-mediated innate immunity in Alzheimer's disease.](https://pubmed.ncbi.nlm.nih.gov/28714976)" (2017) (relative citation ratio: 24.28), "[Cold urticaria, immunodeficiency, and autoimmunity related to PLCG2 deletions.](https://pubmed.ncbi.nlm.nih.gov/22236196)" (2012) (relative citation ratio: 7.25), "[Alzheimer's-associated PLCÎ³2 is a signaling node required for both TREM2 function and the inflammatory response in human microglia.](https://pubmed.ncbi.nlm.nih.gov/32514138)" (2020) (relative citation ratio: 7.25), "[PLCG2 is associated with the inflammatory response and is induced by amyloid plaques in Alzheimer's disease.](https://pubmed.ncbi.nlm.nih.gov/35180881)" (2022) (relative citation ratio: 5.19), and "[BLNK required for coupling Syk to PLC gamma 2 and Rac1-JNK in B cells.](https://pubmed.ncbi.nlm.nih.gov/10023776)" (1999) (relative citation ratio: 4.16). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PLCG2](https://www.proteinatlas.org/ENSG00000197943-PLCG2) is a gene with hydrolase and transducer molecular functions, implicated in cancer-related processes and disease variants. Evidence for its function is available at the protein level. The gene is detected in many tissues, with notable expression in lymphoid tissue related to immune response. Subcellularly, PLCG2 is found in the nucleoplasm, vesicles, and cytosol. Its expression is also associated with B-cell function in single-cell expression clusters.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) and [AR](https://www.ncbi.nlm.nih.gov/gene/367), each with 5 experiments. Additionally, [BRD9](https://www.ncbi.nlm.nih.gov/gene/65980), [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), and [HNF4A](https://www.ncbi.nlm.nih.gov/gene/3172) were each found to be regulating in 4 experiments.


The top transcription factors (TFs) identified in Perturb-seq experiments include [GATA1](https://www.ncbi.nlm.nih.gov/gene/1054) and [TAL1](https://www.ncbi.nlm.nih.gov/gene/9584), each shown to regulate gene expression in one experiment. These findings are derived from the GTRD database of TF binding sites from uniform ChIP-seq data, filtered for Perturb-seq data from a study that examined how changes in TFs affect gene expression.


The input data from GWAS indicates associations with a wide range of disease conditions, including various types of cancer such as breast cancer, thoracic cancer, and hematologic cancer. Additionally, the data highlights infectious diseases, both viral and bacterial, including hepatitis C and anthrax disease. Metabolic disorders, particularly type 2 diabetes mellitus and glucose metabolism disease, are also implicated. Cardiovascular conditions, such as coronary artery disease, cardiomyopathy, and myocardial infarction, are noted. Neurological and mental health disorders, including Parkinson's disease, schizophrenia, and bipolar disorder, are part of the dataset. Furthermore, the data includes gastrointestinal diseases like ulcerative colitis and inflammatory bowel disease, as well as musculoskeletal and genetic disorders.


PLCG2 has been associated with several diseases and traits through genome-wide association studies. Notably, it has been linked to [Alzheimer's disease](https://pubmed.ncbi.nlm.nih.gov/35379992) (closest MeSH match: [Alzheimer Disease](https://meshb.nlm.nih.gov/record/ui?ui=D000544)), [Alzheimer's disease (late onset)](https://pubmed.ncbi.nlm.nih.gov/28714976) (closest MeSH match: [Disease](https://meshb.nlm.nih.gov/record/ui?ui=D004194)), and [Inflammatory bowel disease](https://pubmed.ncbi.nlm.nih.gov/28067908) (closest MeSH match: [Inflammatory Bowel Diseases](https://meshb.nlm.nih.gov/record/ui?ui=D015212)). Additionally, PLCG2 has been associated with [Ulcerative colitis](https://pubmed.ncbi.nlm.nih.gov/28067908) (closest MeSH match: [Colitis, Ulcerative](https://meshb.nlm.nih.gov/record/ui?ui=D003093)), [CD25 on IgD- CD24- B cell](https://pubmed.ncbi.nlm.nih.gov/32929287) (closest MeSH match: [B-Lymphocytes](https://meshb.nlm.nih.gov/record/ui?ui=D001402)), and [Non-albumin protein levels](https://pubmed.ncbi.nlm.nih.gov/33462484) (closest MeSH match: [Proteins](https://meshb.nlm.nih.gov/record/ui?ui=D011506)). These associations were mapped using a method called [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0).


The gene is expressed in various tissues, including blood, spleen, and kidney, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Lymphoma burkitts (Raji), 721 B lymphoblasts, CD56+ NK cells, CD19+ B cells (neg. sel.), whole blood, leukemia promyelocytic-HL-60, Lymphoma burkitts (Daudi), and tonsil.


The input data highlights various biological pathways, including metabolism, adaptive and innate immune systems, and specific signaling cascades such as DAP12 interactions, Fc epsilon receptor (FCERI) signaling, and B Cell Receptor (BCR) signaling. Additionally, it covers toll-like receptor cascades, phagocytosis processes, hemostasis, and platelet activation. Inositol phosphate metabolism and PLC-mediated hydrolysis of PIP2 are also noted.


The mouse ortholog gene 234779 and its human ortholog gene 5336 are associated with phenotypic changes in both sexes, including a decrease in mature B cell number and an increase in T cell number.


The analyzed protein sequence has a GRAVY value of -0.613 (24.16th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -10.93 (21.06th percentile), and the median structural flexibility is 1.003 (56.93rd percentile). The protein has a propensity for helix, sheet, and turn structures of 31.86% (49.1st percentile), 35.73% (64.43rd percentile), and 26.32% (32.1st percentile), respectively. The instability index is 45.52 (41.12th percentile), with an isoelectric point of 6.21 (35.81st percentile). The sequence length is 1265 amino acids (92.74th percentile), and the molecular weight is approximately 147868.19 Da (93.55th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |