---
layout: post
title: DNPH1
date: 2025-01-17 16:55 CST
description: DNPH1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10591) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10591  | DNPH1 | ENSG00000112667 | 6p21.1 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802), and is active in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634). It is also located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). The gene is involved in the [purine nucleotide catabolic process](https://amigo.geneontology.org/amigo/term/GO:0006195), [deoxyribonucleoside monophosphate catabolic process](https://amigo.geneontology.org/amigo/term/GO:0009159), [positive regulation of cell growth](https://amigo.geneontology.org/amigo/term/GO:0030307), [epithelial cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0030855), and [nucleoside salvage](https://amigo.geneontology.org/amigo/term/GO:0043174). Additionally, it enables [protein homodimerization activity](https://amigo.geneontology.org/amigo/term/GO:0042803) and [deoxyribonucleoside 5'-monophosphate N-glycosidase activity](https://amigo.geneontology.org/amigo/term/GO:0070694).


The gene length is 27,781 base pairs (50.55% of all genes), the mature length is 1,819.5 base pairs, and the primary transcript length is 13,811.5 base pairs.


The gene DNPH1 (NCBI ID: 10591) has been mentioned in [9 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22DNPH1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 2008 and the middle 50% of publications occurring between 2011 and 2018. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning DNPH1, ranked by their scientific influence as measured by the [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541), include: ["Targeting the nucleotide salvage factor DNPH1 sensitizes <i>BRCA</i>-deficient cells to PARP inhibitors."](https://pubmed.ncbi.nlm.nih.gov/33833118) (2021) (relative citation ratio: 4.8), ["Rcl is a novel ETV1/ER81 target gene upregulated in breast tumors."](https://pubmed.ncbi.nlm.nih.gov/18726892) (2008) (relative citation ratio: 0.83), ["Baseball commentary 'thumb ligament injuries: RCL and UCL'."](https://pubmed.ncbi.nlm.nih.gov/22883883) (2012) (relative citation ratio: 0.42), ["Commentary on RCL/UCL injury in basketball."](https://pubmed.ncbi.nlm.nih.gov/22883884) (2012) (relative citation ratio: 0.41), and ["Structural characterization of the mammalian deoxynucleotide N-hydrolase Rcl and its stabilizing interactions with two inhibitors."](https://pubmed.ncbi.nlm.nih.gov/19822152) (2009) (relative citation ratio: 0.4). The RCR values measure the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[DNPH1](https://www.proteinatlas.org/ENSG00000112667-DNPH1) is a gene encoding a protein with glycosidase and hydrolase molecular functions. Evidence for this protein is available at the protein level. The gene's RNA is detected in all tissues, with the protein localized in the nucleoplasm and cytosol. DNPH1 is part of various expression clusters, including Cluster 33 for blood (non-specific - ATP binding), Cluster 43 for tissue (non-specific - mitochondrial translation), Cluster 36 for brain (choroid plexus - mixed function), Cluster 28 for cell lines (non-specific - nuclear processes), and Cluster 22 for single cells (proximal enterocytes - digestion).


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [TP63](https://www.ncbi.nlm.nih.gov/gene/8626), [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), [GATA2](https://www.ncbi.nlm.nih.gov/gene/2624), [ZEB1](https://www.ncbi.nlm.nih.gov/gene/6935), and [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), each shown to be regulating in 4 experiments.





The gene is expressed in the adrenal gland and kidney according to BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles indicate expression in various tissues and cell types, including Lymphoma (Burkitt's - Raji and Daudi), Liver, 721 B lymphoblasts, CD34+ cells, Leukemia (promyelocytic - HL-60), CD105+ Endothelial cells, and Kidney.




The protein sequence analyzed has a GRAVY value of -0.329 (54.51 percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -7.29 (27.77 percentile). The median structural flexibility is 1.001 (48.5 percentile), with a predicted secondary structure composition of 33.33% helix (62.24 percentile), 31.03% sheet (34.11 percentile), and 27.01% turn (38.14 percentile). The instability index is 42.26 (29.47 percentile), and the isoelectric point is 4.97 (7.18 percentile). The protein has a length of 174 amino acids (10.9 percentile) and a molecular weight of 19108.2 Da (10.52 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |