---
layout: post
title: TRIM23
date: 2025-01-17 16:55 CST
description: TRIM23 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/373) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 373  | TRIM23 | ENSG00000113595 | 5q12.3 |



The gene is located in the [Golgi membrane](https://amigo.geneontology.org/amigo/term/GO:0000139) and the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), and is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [lysosomal membrane](https://amigo.geneontology.org/amigo/term/GO:0005765), and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). It enables [GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0003924), [ubiquitin-protein transferase activity](https://amigo.geneontology.org/amigo/term/GO:0004842), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [GTP binding](https://amigo.geneontology.org/amigo/term/GO:0005525), [enzyme activator activity](https://amigo.geneontology.org/amigo/term/GO:0008047), [zinc ion binding](https://amigo.geneontology.org/amigo/term/GO:0008270), [GDP binding](https://amigo.geneontology.org/amigo/term/GO:0019003), and [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802). Additionally, it is involved in [intracellular protein transport](https://amigo.geneontology.org/amigo/term/GO:0006886), [positive regulation of autophagy](https://amigo.geneontology.org/amigo/term/GO:0010508), [vesicle-mediated transport](https://amigo.geneontology.org/amigo/term/GO:0016192), [protein ubiquitination](https://amigo.geneontology.org/amigo/term/GO:0016567), and [innate immune response](https://amigo.geneontology.org/amigo/term/GO:0045087).


The gene length is 129,727 base pairs (86.59% of all genes), the mature length is 7,998 base pairs (97.7% of all genes), and the primary transcript length is 129,435 base pairs (91.87% of all genes).


TRIM23 has been mentioned in [17 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22TRIM23%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1993, and the middle 50% of publications occurred between 2001 and 2020.


The top 5 publications mentioning TRIM23, ranked by their relative citation ratio (RCR), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows. The most influential publication is "[TRIM23 mediates virus-induced autophagy via activation of TBK1.](https://pubmed.ncbi.nlm.nih.gov/28871090)" (2017) with a relative citation ratio of 4.86. This is followed by "[SNHG17 promotes colorectal tumorigenesis and metastasis via regulating Trim23-PES1 axis and miR-339-5p-FOSL2-SNHG17 positive feedback loop.](https://pubmed.ncbi.nlm.nih.gov/34782005)" (2021) with a relative citation ratio of 3.0. Other notable publications include "[Elevated TRIM23 expression predicts cisplatin resistance in lung adenocarcinoma.](https://pubmed.ncbi.nlm.nih.gov/31677335)" (2020) with a relative citation ratio of 1.8, "[The E3 ubiquitin ligase TRIM23 regulates adipocyte differentiation via stabilization of the adipogenic activator PPARÎ³.](https://pubmed.ncbi.nlm.nih.gov/25905670)" (2015) with a relative citation ratio of 1.77, and "[Disassembly of the TRIM23-TBK1 Complex by the Us11 Protein of Herpes Simplex Virus 1 Impairs Autophagy.](https://pubmed.ncbi.nlm.nih.gov/31189704)" (2019) with a relative citation ratio of 1.46. For more information on the [RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541), please refer to the linked resource. The citation count is sourced from [iCite](https://icite.od.nih.gov).


[TRIM23](https://www.proteinatlas.org/ENSG00000113595-TRIM23) is a transferase involved in cancer-related genes, with evidence at the protein level. Its RNA is detected in all tissues, and it is expressed in eosinophils, where it is involved in protein ubiquitination. The gene is part of various expression clusters, including non-specific clusters with unknown or mixed functions in different tissues and cell lines, as well as a transcription-related cluster in single cells.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [TAL1](https://www.ncbi.nlm.nih.gov/gene/6886) and [TP53](https://www.ncbi.nlm.nih.gov/gene/7157), each with 6 experiments. Additionally, [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), [PPARG](https://www.ncbi.nlm.nih.gov/gene/5468), and [RARA](https://www.ncbi.nlm.nih.gov/gene/5914) were each found to be regulating in 5 experiments.



The GWAS data indicates associations with a range of conditions, including artery disease, vascular disease, cardiovascular system disease, hypertension, nervous system disease, cognitive disorder, disease of mental health, sensory system disease, and auditory system disease.



The gene is expressed in various neural tissues, including the nerve, brain, prefrontal cortex, amygdala, occipital lobe, superior cervical ganglion, and globus pallidus. Additionally, it is expressed in CD71+ early erythroid cells.




The analyzed protein sequence has a GRAVY value of -0.1524 (73.73rd percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -14.28 (16.26th percentile), and the median structural flexibility is 0.997 (27.56th percentile). The protein has a length of 574 amino acids (66.17th percentile) and a molecular weight of 64065.89 Da (66.36th percentile). The secondary structure affinities are 33.28% for helix (61.77th percentile), 36.06% for sheet (66.7th percentile), and 21.95% for turn (6.3rd percentile). The instability index is 47.23 (47.44th percentile), and the isoelectric point is 5.93 (28.77th percentile). These calculations were performed using BioPython's sequence analysis tools, which can be found [here](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |