---
layout: post
title: SRSF10
date: 2025-01-17 16:55 CST
description: SRSF10 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10772) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10772  | SRSF10 | ENSG00000188529 | 1p36.11 |



The gene is involved in various biological processes, including [spliceosomal tri-snRNP complex assembly](https://amigo.geneontology.org/amigo/term/GO:0000244), [RNA splicing, via transesterification reactions](https://amigo.geneontology.org/amigo/term/GO:0000375), and [mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0000398). It enables functions such as [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). Additionally, it is involved in the [regulation of DNA-templated transcription](https://amigo.geneontology.org/amigo/term/GO:0006355), [mRNA splice site recognition](https://amigo.geneontology.org/amigo/term/GO:0006376), and [cytosolic transport](https://amigo.geneontology.org/amigo/term/GO:0016482). The gene is active in and located in the [nuclear speck](https://amigo.geneontology.org/amigo/term/GO:0016607), and is also found in the [dendrite](https://amigo.geneontology.org/amigo/term/GO:0030425), [neuronal cell body](https://amigo.geneontology.org/amigo/term/GO:0043025), and [axon terminus](https://amigo.geneontology.org/amigo/term/GO:0043679). Furthermore, it is involved in the [regulation of mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0048024) and the [negative regulation of mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0048025). The gene enables [RS domain binding](https://amigo.geneontology.org/amigo/term/GO:0050733) and [unfolded protein binding](https://amigo.geneontology.org/amigo/term/GO:0051082).


The gene length is 192,106 base pairs (92.01% of all genes), the mature length is 2,605.5 base pairs, and the primary transcript length is 186,387 base pairs.


SRSF10 (Gene ID: 10772) has been mentioned in [24 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SRSF10%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest mention in 1996. The middle 50% of publications occurred between 2006 and 2021. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning SRSF10, ranked by their relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)), include studies that explore the gene's role in various cancer types and its regulatory functions. The most influential study, "[BCLAF1 and its splicing regulator SRSF10 regulate the tumorigenic potential of colon cancer cells.](https://pubmed.ncbi.nlm.nih.gov/25091051)" (2014), has an RCR of 3.08. Another notable publication, "[SRSF10 inhibits biogenesis of circ-ATXN1 to regulate glioma angiogenesis via miR-526b-3p/MMP2 pathway.](https://pubmed.ncbi.nlm.nih.gov/32600379)" (2020), has an RCR of 3.06. Additionally, the study "[SRSF10-mediated IL1RAP alternative splicing regulates cervical cancer oncogenesis via mIL1RAP-NF-ÎºB-CD47 axis.](https://pubmed.ncbi.nlm.nih.gov/29429992)" (2018) has an RCR of 3.05. The publication "[The aberrant upregulation of exon 10-inclusive SREK1 through SRSF10 acts as an oncogenic driver in human hepatocellular carcinoma.](https://pubmed.ncbi.nlm.nih.gov/35296659)" (2022) has an RCR of 2.59. Lastly, the study "[The SR protein SRp38 represses splicing in M phase cells.](https://pubmed.ncbi.nlm.nih.gov/12419250)" (2002) has an RCR of 2.44. The relative citation ratio measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[SRSF10](https://www.proteinatlas.org/ENSG00000188529-SRSF10) is an RNA-binding protein with evidence at the protein level. It is detected in all tissues and primarily localized in the nucleoplasm. The gene is involved in various non-specific cellular processes, including vesicular transport, transcription, mRNA splicing, and cell cycle, as indicated by its expression clusters in different tissues, brain regions, cell lines, and single cells. Notably, SRSF10 does not show significant prognostic value in multiple cancer types, including bladder urothelial carcinoma, breast invasive carcinoma, and others, based on TCGA data.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2B](https://www.ncbi.nlm.nih.gov/gene/9757), [TAL1](https://www.ncbi.nlm.nih.gov/gene/6886), [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908), and [PPARG](https://www.ncbi.nlm.nih.gov/gene/5468), each with 5 experiments demonstrating their regulatory roles.



The GWAS data indicates associations with various metabolic and nutritional conditions, including diseases of metabolism, acquired metabolic diseases, nutrition diseases, overnutrition, and obesity.



The gene is expressed in various tissues, including the ovary and thyroid, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in multiple cell types such as 721_B_lymphoblasts, CD34+ cells, CD4+ T cells, CD8+ T cells, Leukemia promyelocytic-HL-60 cells, CD105+ endothelial cells, and CD71+ early erythroid cells.




The analyzed protein sequence has a GRAVY value of -1.762 (0.18 percentile), indicating a hydrophilic nature. It exhibits a charge of 44.03 at pH 7 (98.95 percentile) and a median structural flexibility of 1.02 (97.04 percentile). The protein's secondary structure is predicted to be 18.7% helix (0.96 percentile), 21% sheet (2.72 percentile), and 37.4% turn (94.42 percentile). The instability index is 100.46 (99.6 percentile), with an isoelectric point of 11.26 (99.34 percentile). The protein has a length of 262 amino acids (23.36 percentile) and a molecular weight of 31300.16 Da (25.89 percentile). These calculations were performed using BioPython's sequence analysis tools, as detailed [here](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |