---
layout: post
title: PRELID3A
date: 2024-12-08 23:57 CST
description: PRELID3A description
tags: [cooccuring-genes]
categories: longest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10650) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10650  | PRELID3A | ENSG00000141391 | 18p11.21 |



The gene length is 437,432 base pairs (97.82% of all genes), the mature length is 3,701.5 base pairs, and the primary transcript length is 435,095 base pairs.





The top 5 publications mentioning PRELID3A, ranked by their scientific influence as measured by the [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541), are as follows:

1. [Title of the first publication](https://pubmed.ncbi.nlm.nih.gov/[PubMed ID]) (Year) ...... (Relative Citation Ratio)
2. [Title of the second publication](https://pubmed.ncbi.nlm.nih.gov/[PubMed ID]) (Year) ...... (Relative Citation Ratio)
3. [Title of the third publication](https://pubmed.ncbi.nlm.nih.gov/[PubMed ID]) (Year) ...... (Relative Citation Ratio)
4. [Title of the fourth publication](https://pubmed.ncbi.nlm.nih.gov/[PubMed ID]) (Year) ...... (Relative Citation Ratio)
5. [Title of the fifth publication](https://pubmed.ncbi.nlm.nih.gov/[PubMed ID]) (Year) ...... (Relative Citation Ratio)

The Relative Citation Ratio (RCR) values measure the scientific influence of each paper by field- and time-adjusting the citations it has received, and benchmarking to the median for NIH publications, the value of which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) and [KDM1A](https://www.ncbi.nlm.nih.gov/gene/23028), each with 5 experiments. Additionally, [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), [HDAC2](https://www.ncbi.nlm.nih.gov/gene/3066), and [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) were each found to be regulating in 4 experiments.











The analyzed protein sequence has a GRAVY value of -0.106 (77.65th percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is 0.043 (50.58th percentile), and the median structural flexibility is 0.995 (20.46th percentile). The protein has a propensity for helix, sheet, and turn structures of 30.81% (39.12th percentile), 40.12% (86.63rd percentile), and 27.91% (46.37th percentile), respectively. The instability index is 28.87 (3.2nd percentile), and the isoelectric point is 7.02 (50.68th percentile). The protein is 172 amino acids long (10.67th percentile) with a molecular weight of 19246.95 Da (10.69th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |