---
layout: post
title: GABARAPL2
date: 2024-12-08 23:57 CST
description: GABARAPL2 description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/11345) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 11345  | GABARAPL2 | ENSG00000034713 | 16q23.1 |



The gene length is 67,050 base pairs (73.68% of all genes), the mature length is 1,812 base pairs (47.56% of all genes), and the primary transcript length is 58,245 base pairs (78.67% of all genes).


The gene GABARAPL2 (NCBI ID: 11345) has been featured in [65 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22GABARAPL2%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 2000, and the middle 50% of publications occurred between 2011 and 2020.


The top five publications mentioning GABARAPL2, ranked by their scientific influence, include "[Atg8 family LC3/GABARAP proteins are crucial for autophagosome-lysosome fusion but not autophagosome formation during PINK1/Parkin mitophagy and starvation.](https://pubmed.ncbi.nlm.nih.gov/27864321)" (2016) (relative citation ratio: 14.95), "[LC3 and GATE-16/GABARAP subfamilies are both essential yet act differently in autophagosome biogenesis.](https://pubmed.ncbi.nlm.nih.gov/20418806)" (2010) (relative citation ratio: 11.04), "[ATG8-Binding UIM Proteins Define a New Class of Autophagy Adaptors and Receptors.](https://pubmed.ncbi.nlm.nih.gov/30955882)" (2019) (relative citation ratio: 9.62), "[Role of the mammalian ATG8/LC3 family in autophagy: differential and compensatory roles in the spatiotemporal regulation of autophagy.](https://pubmed.ncbi.nlm.nih.gov/27418283)" (2016) (relative citation ratio: 8.66), and "[TEX264 Is an Endoplasmic Reticulum-Resident ATG8-Interacting Protein Critical for ER Remodeling during Nutrient Stress.](https://pubmed.ncbi.nlm.nih.gov/31006537)" (2019) (relative citation ratio: 7.84). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), [KLF4](https://www.ncbi.nlm.nih.gov/gene/9314), and [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908), each with 6 experiments. Additionally, [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) and [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227) were found to be regulating in 5 experiments each.


The top transcription factors (TFs) identified from the GTRD's database of TF binding sites, filtered for Perturb-seq data, include [6829](https://www.ncbi.nlm.nih.gov/gene/6829) and [9968](https://www.ncbi.nlm.nih.gov/gene/9968). These TFs were shown to be regulating genes in 1 and 2 Perturb-seq experiments, respectively.





The mouse ortholog gene 93739 and its human ortholog gene 11345 are associated with the phenotype of preweaning lethality with complete penetrance, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.316 (56.2 percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of 0.682 (52.64 percentile). The median structural flexibility is 1.002 (49.58 percentile), with a secondary structure composition of 31.6% helix (46.96 percentile), 38.5% sheet (80.82 percentile), and 23.9% turn (14.61 percentile). The instability index is 32.98 (7.08 percentile), and the isoelectric point is 7.81 (57.17 percentile). The protein has a length of 117 amino acids (4.34 percentile) and a molecular weight of 13666.67 Da (4.94 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |