---
layout: post
title: PIP4K2B
date: 2025-01-17 16:55 CST
description: PIP4K2B description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8396) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8396  | PIP4K2B | ENSG00000276293 | 17q12 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524), and [GTP binding](https://amigo.geneontology.org/amigo/term/GO:0005525). It is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [autophagosome](https://amigo.geneontology.org/amigo/term/GO:0005776), [endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0005789), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), where it is also active. The gene is involved in the [cell surface receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007166), [regulation of autophagy](https://amigo.geneontology.org/amigo/term/GO:0010506), [phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0016310), [negative regulation of insulin receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0046627), [phosphatidylinositol phosphate biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0046854), [autophagosome-lysosome fusion](https://amigo.geneontology.org/amigo/term/GO:0061909), [1-phosphatidyl-1D-myo-inositol 4,5-bisphosphate biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:1902635), and [positive regulation of autophagosome assembly](https://amigo.geneontology.org/amigo/term/GO:2000786). Additionally, it enables [1-phosphatidylinositol-4-phosphate 5-kinase activity](https://amigo.geneontology.org/amigo/term/GO:0016308), [1-phosphatidylinositol-5-phosphate 4-kinase activity](https://amigo.geneontology.org/amigo/term/GO:0016309), and [protein homodimerization activity](https://amigo.geneontology.org/amigo/term/GO:0042803).


The gene has a length of 31,606 base pairs (54.1% of all genes), a mature length of 2,037 base pairs, and a primary transcript length of 31,606 base pairs.


The gene PIP4K2B (NCBI ID: 8396) has been mentioned in [6 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PIP4K2B%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning this gene was in 1996, and the middle 50% of publications occurred between 2012 and 2018.


The top 5 publications mentioning PIP4K2B, ranked by their scientific influence as measured by the [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541), include: "[Long non-coding RNA PXN-AS1 suppresses pancreatic cancer progression by acting as a competing endogenous RNA of miR-3064 to upregulate PIP4K2B expression.](https://pubmed.ncbi.nlm.nih.gov/31488171)" (2019) (relative citation ratio: 1.86), "[The Lipid Kinase PI5P4KÎ² Is an Intracellular GTP Sensor for Metabolism and Tumorigenesis.](https://pubmed.ncbi.nlm.nih.gov/26774281)" (2016) (relative citation ratio: 1.63), "[Aggregation-dependent, integrin-mediated increases in cytoskeletally associated PtdInsP2 (4,5) levels in human platelets are controlled by translocation of PtdIns 4-P 5-kinase C to the cytoskeleton.](https://pubmed.ncbi.nlm.nih.gov/8978678)" (1996) (relative citation ratio: 1.05), and "[Phosphatidylinositol 5-phosphate 4-kinase type II beta is required for vitamin D receptor-dependent E-cadherin expression in SW480 cells.](https://pubmed.ncbi.nlm.nih.gov/21514270)" (2011) (relative citation ratio: 0.61). The citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PIP4K2B](https://www.proteinatlas.org/ENSG00000276293-PIP4K2B) is a kinase and transferase with evidence at the protein level, implicated in cancer-related genes. It is detected in all tissues and primarily localized in the nucleoplasm. The gene is expressed in T-cells of unknown function, and its tissue expression cluster suggests a role in non-specific transcription. In the brain, it is associated with neurons and synaptic function, while in cell lines, it is found in breast cancer cells of unknown function. Single-cell expression data links it to smooth muscle cells involved in muscle contraction.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025), [KLF5](https://www.ncbi.nlm.nih.gov/gene/688), [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227), [CHD1](https://www.ncbi.nlm.nih.gov/gene/1105), and [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), each shown to be regulating in 5 experiments.



The input data indicates associations with various disease conditions, including mental health disorders. Specifically, the data highlights attention deficit hyperactivity disorder, which is a specific developmental disorder within the broader category of developmental disorders of mental health.



The gene is expressed in various tissues, including the ovary and brain, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the fetal brain, subthalamic nucleus, cerebellum, superior cervical ganglion, and globus pallidus.


The input data highlights various pathways involved in metabolism, specifically focusing on lipid and lipoprotein metabolism. Within this broader category, phospholipid metabolism is emphasized, particularly the metabolism of phosphatidylinositol (PI). Notably, the synthesis of phosphatidylinositol phosphates (PIPs) at the plasma membrane is a key process identified.



The analyzed protein sequence has a GRAVY value of -0.503 (34.64th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -0.368 (48.77th percentile). The median structural flexibility is 1.004 (60.5th percentile), with a secondary structure composition of 32.69% helix (56.79th percentile), 34.62% sheet (56.19th percentile), and 28.37% turn (50.64th percentile). The instability index is 40.34 (23.25th percentile), and the isoelectric point is 6.90 (49.24th percentile). The protein has a length of 416 amino acids (47.4th percentile) and a molecular weight of 47377.42 Da (48.8th percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |