---
layout: post
title: ENTPD3
date: 2025-01-17 16:55 CST
description: ENTPD3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/956) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 956  | ENTPD3 | ENSG00000168032 | 3p22.1 |



The gene enables [apyrase activity](https://amigo.geneontology.org/amigo/term/GO:0004050), [GDP phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0004382), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524). It is active in and located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). Additionally, it is involved in the [nucleoside diphosphate catabolic process](https://amigo.geneontology.org/amigo/term/GO:0009134) and the [nucleoside triphosphate catabolic process](https://amigo.geneontology.org/amigo/term/GO:0009143). The gene also enables [nucleoside diphosphate phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0017110), [ribonucleoside triphosphate phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0017111), and [UDP phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0045134).


The gene length is 201,426.0 base pairs (92.51% of all genes), the mature length is 5,710.5 base pairs, and the primary transcript length is 200,851.5 base pairs.


The gene ENTPD3 (NCBI ID: 956) has been mentioned in [13 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ENTPD3%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1998 and the middle 50% of publications occurring between 2005 and 2014. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning ENTPD3, ranked by their scientific influence, include "[Ectonucleoside Triphosphate Diphosphohydrolase-3 Antibody Targets Adult Human Pancreatic β Cells for In Vitro and In Vivo Analysis.](https://pubmed.ncbi.nlm.nih.gov/30449685)" (2019) with a relative citation ratio of 2.67, "[The CD39-like gene family: identification of three new human members (CD39L2, CD39L3, and CD39L4), their murine homologues, and a member of the gene family from Drosophila melanogaster.](https://pubmed.ncbi.nlm.nih.gov/9676430)" (1998) with a relative citation ratio of 2.38, "[SNP-mediated lncRNA-ENTPD3-AS1 upregulation suppresses renal cell carcinoma via miR-155/HIF-1α signaling.](https://pubmed.ncbi.nlm.nih.gov/34218253)" (2021) with a relative citation ratio of 1.97, "[Ectonucleotidases in the digestive system: focus on NTPDase3 localization.](https://pubmed.ncbi.nlm.nih.gov/21233276)" (2011) with a relative citation ratio of 1.65, and "[Cloning and characterization of the ecto-nucleotidase NTPDase3 from rat brain: Predicted secondary structure and relation to other members of the E-NTPDase family and actin.](https://pubmed.ncbi.nlm.nih.gov/18404510)" (2005) with a relative citation ratio of 1.17. The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ENTPD3](https://www.proteinatlas.org/ENSG00000168032-ENTPD3) is a gene encoding a protein with hydrolase molecular function, supported by evidence at the protein level. Its RNA is detected in many tissues, with notable expression in the salivary gland, cerebral cortex, squamous epithelial cells, and neurons. The gene is associated with cell proliferation in blood and various specific functions in other tissues. While ENTPD3's role in disease is not specified, its expression patterns suggest involvement in salivary secretion, mixed brain functions, keratinization, and neuronal signaling.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [EZH2](https://www.ncbi.nlm.nih.gov/gene/2146), [JUN](https://www.ncbi.nlm.nih.gov/gene/3725), and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), each with 4 experiments. Additionally, [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) and [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227) were found to be regulating in 3 experiments each.



The input data highlights a range of disease conditions associated with GWAS, including various forms of cancer such as leukemia, lymphoma, and immune system cancer. Additionally, respiratory conditions like asthma, bronchial disease, and obstructive lung disease are noted. The data also encompasses hematopoietic and immune system diseases, including anemia and its subtypes like congenital hemolytic anemia and hemoglobinopathy. Furthermore, cardiovascular and urinary system diseases, as well as general disease conditions, are mentioned.



The gene is expressed in multiple tissue samples, including the pancreas, bladder, salivary gland, and brain, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the caudate nucleus, hypothalamus, medulla oblongata, bronchial epithelial cells, trachea, pancreatic islet, and superior cervical ganglion.




The protein sequence analyzed has a GRAVY value of -0.083 (79.28th percentile), indicating a hydrophilic nature. It exhibits a charge of -8.18 at pH 7.0 (26.01st percentile) and a median structural flexibility of 0.994 (17.52nd percentile). The protein's secondary structure is predicted to be 27.6% helix (14.87th percentile), 40.1% sheet (86.5th percentile), and 28.5% turn (52.13th percentile). The instability index is 33.54 (7.87th percentile), with an isoelectric point of 5.98 (29.89th percentile). The protein has a length of 529 amino acids (61.7th percentile) and a molecular weight of 59104.51 Da (61.65th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |