---
layout: post
title: MTMR4
date: 2025-01-17 16:55 CST
description: MTMR4 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9110) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9110  | MTMR4 | ENSG00000108389 | 17q22 |



The gene enables [phosphatidylinositol-3-phosphate phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0004438), [protein serine/threonine phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0004722), [protein tyrosine phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0004725), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [protein phosphatase binding](https://amigo.geneontology.org/amigo/term/GO:0019903). It is located in the [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615), [endosome](https://amigo.geneontology.org/amigo/term/GO:0005768), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and [early endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0031901), and is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737) and [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020). The gene is involved in [protein dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0006470), [phosphatidylinositol biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006661), [regulation of autophagy](https://amigo.geneontology.org/amigo/term/GO:0010506), [negative regulation of transforming growth factor beta receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0030512), [phosphatidylinositol dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0046856), and [regulation of phosphatidylinositol dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0060304). Additionally, it enables [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872) and [phosphatidylinositol-3,5-bisphosphate 3-phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0052629).


The gene length is 7,413 base pairs (20.02% of all genes), the mature length is 3,298 base pairs, and the primary transcript length is 6,840 base pairs.


The gene MTMR4 (NCBI ID: 9110) has been mentioned in [8 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22MTMR4%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 2009 and the middle 50% of publications occurring between 2010 and 2018. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning MTMR4, ranked by their scientific influence as measured by the [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541), include "[MTMR4 SNVs modulate ion channel degradation and clinical severity in congenital long QT syndrome: insights in the mechanism of action of protective modifier genes.](https://pubmed.ncbi.nlm.nih.gov/32173736)" (2021) (relative citation ratio: 2.65), "[The myotubularin phosphatase MTMR4 regulates sorting from early endosomes.](https://pubmed.ncbi.nlm.nih.gov/20736309)" (2010) (relative citation ratio: 0.86), "[MTMR4 attenuates transforming growth factor beta (TGFbeta) signaling by dephosphorylating R-Smads in endosomes.](https://pubmed.ncbi.nlm.nih.gov/20061380)" (2010) (relative citation ratio: 0.74), "[PtdIns3P phosphatases MTMR3 and MTMR4 negatively regulate innate immune responses to DNA through modulating STING trafficking.](https://pubmed.ncbi.nlm.nih.gov/30944173)" (2019) (relative citation ratio: 0.52), and "[The inositol phosphatase MTMR4 is a novel target of the ubiquitin ligase Nedd4.](https://pubmed.ncbi.nlm.nih.gov/19125695)" (2009) (relative citation ratio: 0.39). The RCR values measure the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[MTMR4](https://www.proteinatlas.org/ENSG00000108389-MTMR4) is a gene encoding a protein with hydrolase and protein phosphatase molecular functions. Evidence for this gene is available at the protein level. Its RNA is detected in all tissues, with notable expression in the hindbrain and breast cancer cell lines. The gene is part of various expression clusters, including Cluster 14 for blood, Cluster 16 for general tissue, Cluster 49 for brain, Cluster 27 for cell lines, and Cluster 55 for single cells, indicating a role in transcription and mixed functions.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2B](https://www.ncbi.nlm.nih.gov/gene/9757) with 6 experiments, [KLF5](https://www.ncbi.nlm.nih.gov/gene/688) with 5 experiments, [KLF4](https://www.ncbi.nlm.nih.gov/gene/9314) with 5 experiments, [EP300](https://www.ncbi.nlm.nih.gov/gene/2033) with 5 experiments, and [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with a range of disease conditions, including various forms of cancer such as lung cancer and respiratory system cancer. Additionally, the data highlights mental health disorders, including bipolar disorder, mood disorder, and substance-related disorders like substance dependence and cocaine dependence. Furthermore, associations with gastrointestinal system diseases, specifically dental caries and related tooth and mouth diseases, are also noted.


MTMR4 has been associated with several traits and diseases through genome-wide association studies. Notably, it has been linked to [cognitive aspects of educational attainment](https://pubmed.ncbi.nlm.nih.gov/33414549), which maps closely to the MeSH term [Education](https://meshb.nlm.nih.gov/record/ui?ui=D004493) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method. Additionally, MTMR4 is associated with [dental caries](https://pubmed.ncbi.nlm.nih.gov/21940522), directly matching the MeSH term [Dental Caries](https://meshb.nlm.nih.gov/record/ui?ui=D003731). Furthermore, it has been implicated in [general cognitive ability](https://pubmed.ncbi.nlm.nih.gov/29844566), which maps to the MeSH term [Cognition](https://meshb.nlm.nih.gov/record/ui?ui=D003071) through embedding. Lastly, MTMR4 is linked to [vitamin D levels](https://pubmed.ncbi.nlm.nih.gov/25208829), mapping to the MeSH term [Vitamins](https://meshb.nlm.nih.gov/record/ui?ui=D014815) using embedding.


The gene is expressed in various tissues, including liver and brain, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in 721_B_lymphoblasts, fetal brain, cerebellum, placenta, leukemia lymphoblastic (MOLT-4), and testis germ cells.


The input data highlights various pathways involved in metabolism, particularly the metabolism of lipids and lipoproteins, as well as phospholipid metabolism and PI metabolism. It also emphasizes the synthesis of PIPs at both early and late endosome membranes. Additionally, the data focuses on TGF-beta receptor signaling, including its downregulation and activation of SMADs, along with multiple loss-of-function mutations in SMAD2/3, SMAD4, TGFBR1, and TGFBR2 in cancer. Specific mutations such as TGFBR2 kinase domain mutants, TGFBR2 MSI frameshift mutants, and various SMAD domain mutants in cancer are also noted. The data further includes pathways related to disease and signal transduction.


The mouse ortholog gene 170749 and its human ortholog gene 9110 are associated with the phenotype of preweaning lethality with incomplete penetrance, observed in both sexes.


The protein sequence analyzed has a GRAVY value of -0.479 (37.2% percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -29.41 (6.32% percentile), and the median structural flexibility is 1.003 (57.45% percentile). The protein has a helix affinity of 0.283 (18.77% percentile), a sheet affinity of 0.304 (30.31% percentile), and a turn affinity of 0.327 (80.49% percentile). The instability index is 55.05 (72.3% percentile), and the isoelectric point is 5.68 (22.29% percentile). The sequence length is 1195 amino acids (91.63% percentile), with a molecular weight of 133351.94 Da (91.68% percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |