---
layout: post
title: KLC1
date: 2025-01-17 16:55 CST
description: KLC1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3831) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3831  | KLC1 | ENSG00000126214 | 14q32.33 |



The gene enables [cytoskeletal motor activity](https://amigo.geneontology.org/amigo/term/GO:0003774) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737). It is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829) and is part of the [kinesin complex](https://amigo.geneontology.org/amigo/term/GO:0005871), also found in the [microtubule](https://amigo.geneontology.org/amigo/term/GO:0005874). The gene is involved in [microtubule-based movement](https://amigo.geneontology.org/amigo/term/GO:0007018) and [cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0007155). Additionally, it is located in the [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [growth cone](https://amigo.geneontology.org/amigo/term/GO:0030426), and [cytoplasmic vesicle](https://amigo.geneontology.org/amigo/term/GO:0031410). The gene enables [kinesin binding](https://amigo.geneontology.org/amigo/term/GO:0019894) and is involved in [stress granule disassembly](https://amigo.geneontology.org/amigo/term/GO:0035617).


The gene length is 14,294 base pairs (33.93% of all genes), the mature length is 1,550.5 base pairs, and the primary transcript length is 13,746 base pairs.


The gene KLC1 (NCBI ID: 3831) has been mentioned in [29 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22KLC1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 2002 and the middle 50% of publications occurring between 2007 and 2017. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning KLC1, ranked by their scientific influence, include "[KLC1-ALK: a novel fusion in lung cancer identified using a formalin-fixed paraffin-embedded tissue only.](https://pubmed.ncbi.nlm.nih.gov/22347464)" (2012) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 4.82, "[Binding and transport of SFPQ-RNA granules by KIF5A/KLC1 motors promotes axon survival.](https://pubmed.ncbi.nlm.nih.gov/33284322)" (2021) with an RCR of 2.74, "[A scaffold protein JIP-1b enhances amyloid precursor protein phosphorylation by JNK and its association with kinesin light chain 1.](https://pubmed.ncbi.nlm.nih.gov/12665528)" (2003) with an RCR of 2.19, "[Amyloid beta protein precursor (AbetaPP), but not AbetaPP-like protein 2, is bridged to the kinesin light chain by the scaffold protein JNK-interacting protein 1.](https://pubmed.ncbi.nlm.nih.gov/12893827)" (2003) with an RCR of 1.4, and "[Cargo selection by specific kinesin light chain 1 isoforms.](https://pubmed.ncbi.nlm.nih.gov/17093494)" (2006) with an RCR of 1.26. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[KLC1](https://www.proteinatlas.org/ENSG00000126214-KLC1) is a motor protein with evidence at the protein level. It is detected in all tissues and is primarily localized in the nucleoplasm, plasma membrane, cytosol, acrosome, and equatorial segment. KLC1 is part of Cluster 25 for blood expression, indicating non-specific and unknown function, and Cluster 65 for tissue expression, suggesting a role in cerebellum and nervous system development. In the brain, it belongs to Cluster 24, associated with neurons and mixed function. Additionally, KLC1 is part of Cluster 30 for cell line expression, indicating gene expression regulation, and Cluster 71 for single cell expression, linked to horizontal and bipolar cells involved in neuronal signaling.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [TP53](https://www.ncbi.nlm.nih.gov/gene/7157) and [ZBTB7A](https://www.ncbi.nlm.nih.gov/gene/51341), each with 6 experiments. Additionally, [RELA](https://www.ncbi.nlm.nih.gov/gene/5970), [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), and [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) were each found to be regulating in 5 experiments.



The GWAS data indicates associations with various disease conditions, including immune system disease, lymphatic system disease, lymph node disease, lymphadenitis, and Kawasaki disease.



The gene is expressed in various neural tissues, including the nerve, amygdala, whole brain, fetal brain, and hypothalamus.


The proteins are involved in various pathways, including the adaptive immune system, immune system, factors involved in megakaryocyte development and platelet production, kinesins, MHC class II antigen presentation, and hemostasis.



The analyzed protein sequence has a GRAVY value of -0.8804537521814989 (8.13% percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -9.325549198710405 (23.57% percentile), and the median structural flexibility is 1.0129107142857143 (88.03% percentile). The protein has a high affinity for helix structures (40.66%, 94.38% percentile), with lower affinities for sheet (28.45%, 20.43% percentile) and turn (25.65%, 26.31% percentile) structures. The instability index is 47.44415357766146 (48.12% percentile), and the isoelectric point is 5.865579414367674 (27.06% percentile). The protein is 573 amino acids long (66.09% percentile) with a molecular weight of 65308.764599999995 Da (67.28% percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |