---
layout: post
title: RGS4
date: 2025-01-17 16:55 CST
description: RGS4 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5999) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5999  | RGS4 | ENSG00000117152 | 1q23.3 |



The gene enables [G-protein alpha-subunit binding](https://amigo.geneontology.org/amigo/term/GO:0001965), [GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0003924), [GTPase activator activity](https://amigo.geneontology.org/amigo/term/GO:0005096), and [calmodulin binding](https://amigo.geneontology.org/amigo/term/GO:0005516). It is involved in the [response to amphetamine](https://amigo.geneontology.org/amigo/term/GO:0001975), [G protein-coupled receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007186), [regulation of G protein-coupled receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0008277), and [positive regulation of heart rate](https://amigo.geneontology.org/amigo/term/GO:0010460). Additionally, it is involved in the [response to cocaine](https://amigo.geneontology.org/amigo/term/GO:0042220), [response to ethanol](https://amigo.geneontology.org/amigo/term/GO:0045471), [negative regulation of G protein-coupled receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0045744), and [regulation of calcium ion transport](https://amigo.geneontology.org/amigo/term/GO:0051924). The gene is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), and is part of a [protein-containing complex](https://amigo.geneontology.org/amigo/term/GO:0032991).


The gene length is 46,978 base pairs (64.69% of all genes), the mature length is 3,038.5 base pairs (73.02% of all genes), and the primary transcript length is 23,646 base pairs (56.46% of all genes).


The gene RGS4 (NCBI ID: 5999) has been mentioned in [144 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RGS4%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1996 and the middle 50% of publications occurring between 2005 and 2014. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning RGS4, ranked by their scientific influence, include "[GAIP and RGS4 are GTPase-activating proteins for the Gi subfamily of G protein alpha subunits.](https://pubmed.ncbi.nlm.nih.gov/8756726)" (1996) (relative citation ratio: 13.22), "[Structure of RGS4 bound to AlF4--activated G(i alpha1): stabilization of the transition state for GTP hydrolysis.](https://pubmed.ncbi.nlm.nih.gov/9108480)" (1997) (relative citation ratio: 12.59), "[RGS4 and GAIP are GTPase-activating proteins for Gq alpha and block activation of phospholipase C beta by gamma-thio-GTP-Gq alpha.](https://pubmed.ncbi.nlm.nih.gov/9012799)" (1997) (relative citation ratio: 6.33), "[The GTPase-activating protein RGS4 stabilizes the transition state for nucleotide hydrolysis.](https://pubmed.ncbi.nlm.nih.gov/8910288)" (1996) (relative citation ratio: 5.83), and "[Association and linkage analyses of RGS4 polymorphisms in schizophrenia.](https://pubmed.ncbi.nlm.nih.gov/12023979)" (2002) (relative citation ratio: 5.49). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RGS4](https://www.proteinatlas.org/ENSG00000117152-RGS4) is a signal transduction inhibitor implicated in schizophrenia, with evidence supporting its role at the protein level. The gene is detected in many tissues and is primarily localized in the cytosol. RGS4 is notably expressed in the brain, specifically in neurons involved in mixed functions, and is also found in connective tissue cells associated with extracellular matrix organization. Its expression is particularly prominent in Cluster 61, which is related to brain neuronal signaling.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [AR](https://www.ncbi.nlm.nih.gov/gene/367) and [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), each with 4 experiments. Additionally, [H2AZ1](https://www.ncbi.nlm.nih.gov/gene/3015), [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025), and [SMARCA2](https://www.ncbi.nlm.nih.gov/gene/6595) were each found to be regulating in 3 experiments.





The gene is expressed in various tissues, including blood vessel, brain, and pituitary, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in several brain regions, such as the Cingulate Cortex, Subthalamic Nucleus, Temporal Lobe, Medulla Oblongata, Parietal Lobe, Occipital Lobe, and Globus Pallidus.


The proteins are involved in various signaling pathways, including G alpha (z) signaling events, G alpha (i) signaling events, GPCR downstream signaling, signaling by GPCR, and signal transduction.


The mouse ortholog gene 19736 and its human ortholog gene 5999 are associated with several phenotypic effects observed in both sexes. These include a decrease in lymphocyte cell number, an increase in eosinophil cell number, and an increase in mean corpuscular volume.


The analyzed protein sequence has a GRAVY value of -0.577 (27.43rd percentile), indicating a hydrophilic nature. It exhibits a charge of 5.77 at pH 7.0 (70.49th percentile) and a median structural flexibility of 1.012 (85.99th percentile). The protein's secondary structure is predicted to be predominantly helical (40%, 93.39th percentile), with lower proportions in sheet (27.32%, 15.63rd percentile) and turn (26.83%, 36.48th percentile) conformations. The instability index is 59.20 (81.52nd percentile), suggesting potential instability. The isoelectric point is 8.69 (70.84th percentile), with a molecular weight of 23255.45 Da (15.65th percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |