---
layout: post
title: RPS2
date: 2024-12-08 23:57 CST
description: RPS2 description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6187) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6187  | RPS2 | ENSG00000140988 | 16p13.3 |



The gene length is 37,221 base pairs (58.58% of all genes), the mature length is 1,351 base pairs, and the primary transcript length is 23,589 base pairs.


The gene RPS2, with NCBI ID 6187, has been featured in [34 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RPS2%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727). The earliest year of publication is 1985, and the middle 50% of publications occurred between 1997 and 2013. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning RPS2, ranked by their scientific influence, include "[A [Mn18Dy] SMM resulting from the targeted replacement of the central MnII in the S = 83/2 [Mn19]-aggregate with DyIII.](https://pubmed.ncbi.nlm.nih.gov/19283285)" (2009) (relative citation ratio: 3.74), "[Ribosomal protein S2 is a substrate for mammalian PRMT3 (protein arginine methyltransferase 3).](https://pubmed.ncbi.nlm.nih.gov/15473865)" (2005) (relative citation ratio: 2.35), "[SL2B aptamer and folic acid dual-targeting DNA nanostructures for synergic biological effect with chemotherapy to combat colorectal cancer.](https://pubmed.ncbi.nlm.nih.gov/28435250)" (2017) (relative citation ratio: 1.96), "[Evolution of plastid gene rps2 in a lineage of hemiparasitic and holoparasitic plants: many losses of photosynthesis and complex patterns of rate variation.](https://pubmed.ncbi.nlm.nih.gov/9207097)" (1997) (relative citation ratio: 1.91), and "[A duplicated pair of Arabidopsis RING-finger E3 ligases contribute to the RPM1- and RPS2-mediated hypersensitive response.](https://pubmed.ncbi.nlm.nih.gov/16212605)" (2005) (relative citation ratio: 1.84). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


The gene [RPL10](https://www.proteinatlas.org/ENSG00000110225-RPL10) functions as a ribonucleoprotein and ribosomal protein, with evidence supporting its role at the protein level. It is implicated in cancer-related genes and is detected in all tissues. The gene is localized in the endoplasmic reticulum and cytosol. Its expression clusters in various tissues, including the brain, blood, and cell lines, are associated with non-specific ribosomal processes.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [AFF4](https://www.ncbi.nlm.nih.gov/gene/27125), [SMARCB1](https://www.ncbi.nlm.nih.gov/gene/6598), and [SIN3A](https://www.ncbi.nlm.nih.gov/gene/25942), each with 6 experiments. Additionally, [HDAC2](https://www.ncbi.nlm.nih.gov/gene/3066) and [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313) were found to be regulating in 5 experiments each.


The top transcription factors (TFs) identified from the GTRD's database of TF binding sites, filtered for Perturb-seq data, include [CTCF](https://www.ncbi.nlm.nih.gov/gene/6829) and [BRCA1](https://www.ncbi.nlm.nih.gov/gene/6872), each shown to be regulating genes in 4 experiments. Additionally, [FOXA1](https://www.ncbi.nlm.nih.gov/gene/2623), [RAD21](https://www.ncbi.nlm.nih.gov/gene/7936), and [ZNF143](https://www.ncbi.nlm.nih.gov/gene/8861) were found to regulate genes in 3 experiments each. Other notable TFs include [HNF4A](https://www.ncbi.nlm.nih.gov/gene/2118), [ZNF274](https://www.ncbi.nlm.nih.gov/gene/7469), and [RAD50](https://www.ncbi.nlm.nih.gov/gene/51755), each shown to regulate genes in 2 experiments. Furthermore, [ZNF263](https://www.ncbi.nlm.nih.gov/gene/8243), [ZNF608](https://www.ncbi.nlm.nih.gov/gene/26009), and [RAD51](https://www.ncbi.nlm.nih.gov/gene/9968) were found to regulate genes in 1 experiment each.








The analyzed protein sequence has a GRAVY value of -0.233 (65.81 percentile), indicating a hydrophilic nature. It exhibits a charge of 25.81 at pH 7.0 (96.08 percentile) and a median structural flexibility of 1.002 (51.19 percentile). The protein's secondary structure is predicted to be 29.01% helix (23.62 percentile), 33.79% sheet (50.48 percentile), and 30.38% turn (66.53 percentile). The instability index is 19.44 (0.54 percentile), with an isoelectric point of 10.25 (97.58 percentile). The protein has a length of 293 amino acids (27.49 percentile) and a molecular weight of 31324.06 Da (25.94 percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |