---
layout: post
title: RABGGTA
date: 2024-12-08 23:57 CST
description: RABGGTA description
tags: [cooccuring-genes]
categories: shortest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5875) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5875  | RABGGTA | ENSG00000100949 | 14q12 |



The gene length is 494.0 base pairs (0.45 percentile of all genes), with a mature length of 494.0 base pairs and a primary transcript length of 494.0 base pairs.





The top 5 publications mentioning RABGGTA, ranked by their scientific influence as measured by the [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541), are as follows:

1. [Title of the first publication](https://pubmed.ncbi.nlm.nih.gov/[PubMed ID]) (Year) ...... (Relative Citation Ratio)
2. [Title of the second publication](https://pubmed.ncbi.nlm.nih.gov/[PubMed ID]) (Year) ...... (Relative Citation Ratio)
3. [Title of the third publication](https://pubmed.ncbi.nlm.nih.gov/[PubMed ID]) (Year) ...... (Relative Citation Ratio)
4. [Title of the fourth publication](https://pubmed.ncbi.nlm.nih.gov/[PubMed ID]) (Year) ...... (Relative Citation Ratio)
5. [Title of the fifth publication](https://pubmed.ncbi.nlm.nih.gov/[PubMed ID]) (Year) ...... (Relative Citation Ratio)

The Relative Citation Ratio (RCR) values measure the scientific influence of each paper by field- and time-adjusting the citations it has received, and benchmarking to the median for NIH publications, the value of which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [KLF4](https://www.ncbi.nlm.nih.gov/gene/9314), [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170), [TBP](https://www.ncbi.nlm.nih.gov/gene/6908), [SP1](https://www.ncbi.nlm.nih.gov/gene/6667), and [HIF1A](https://www.ncbi.nlm.nih.gov/gene/3091). Each of these TFs, except HIF1A, was shown to be regulating in 5 experiments, while HIF1A was found in 4 experiments.











The analyzed protein sequence has a GRAVY value of -0.249 (63.97th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -16.24 (14.25th percentile). The median structural flexibility is 0.996 (23.78th percentile), with a significant portion of amino acids predicted to be in helix (38.27%, 89.51st percentile) and sheet (37.74%, 77.15th percentile) structures. The instability index is 58.28 (79.7th percentile), suggesting potential instability. The isoelectric point is 5.45 (17.12th percentile), and the molecular weight is 65071.03 Da (67.12th percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |