---
layout: post
title: PABPC4
date: 2025-01-17 16:55 CST
description: PABPC4 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8761) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8761  | PABPC4 | ENSG00000090621 | 1p34.3 |



The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [mRNA 3'-UTR binding](https://amigo.geneontology.org/amigo/term/GO:0003730), and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is active in and located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), as well as located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737) and active in and located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). The gene is involved in [RNA processing](https://amigo.geneontology.org/amigo/term/GO:0006396), [RNA catabolic process](https://amigo.geneontology.org/amigo/term/GO:0006401), [translation](https://amigo.geneontology.org/amigo/term/GO:0006412), and [blood coagulation](https://amigo.geneontology.org/amigo/term/GO:0007596). Additionally, it enables [poly(A) binding](https://amigo.geneontology.org/amigo/term/GO:0008143) and [poly(U) RNA binding](https://amigo.geneontology.org/amigo/term/GO:0008266), but does not enable [poly(C) RNA binding](https://amigo.geneontology.org/amigo/term/GO:0017130). The gene is active in and located in the [cytoplasmic stress granule](https://amigo.geneontology.org/amigo/term/GO:0010494) and is part of the [ribonucleoprotein complex](https://amigo.geneontology.org/amigo/term/GO:1990904). It is also involved in the [regulation of mRNA stability](https://amigo.geneontology.org/amigo/term/GO:0043488) and [myeloid cell development](https://amigo.geneontology.org/amigo/term/GO:0061515).


The gene length is 13,270 base pairs (32.07% of all genes), the mature length is 5,055 base pairs (90.62% of all genes), and the primary transcript length is 13,270 base pairs (40.84% of all genes).


The gene PABPC4, with NCBI ID 8761, has been mentioned in [11 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PABPC4%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727). The earliest year of publication is 1997, and the middle 50% of publications occurred between 2005 and 2011.


The top 5 publications mentioning PABPC4, ranked by their scientific influence, include "[PABPC4 Broadly Inhibits Coronavirus Replication by Degrading Nucleocapsid Protein through Selective Autophagy.](https://pubmed.ncbi.nlm.nih.gov/34612687)" (2021) (relative citation ratio: 2.45), "[Identification of App1 as a regulator of phagocytosis and virulence of Cryptococcus neoformans.](https://pubmed.ncbi.nlm.nih.gov/14523045)" (2003) (relative citation ratio: 2.36), "[Nuclear relocalisation of cytoplasmic poly(A)-binding proteins PABP1 and PABP4 in response to UV irradiation reveals mRNA-dependent export of metazoan PABPs.](https://pubmed.ncbi.nlm.nih.gov/21940797)" (2011) (relative citation ratio: 1.39), "[App1: an antiphagocytic protein that binds to complement receptors 3 and 2.](https://pubmed.ncbi.nlm.nih.gov/19109138)" (2009) (relative citation ratio: 1.1), and "[Interaction of anti-proliferative protein Tob with poly(A)-binding protein and inducible poly(A)-binding protein: implication of Tob in translational control.](https://pubmed.ncbi.nlm.nih.gov/15676026)" (2005) (relative citation ratio: 1.01). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PABPC4](https://www.proteinatlas.org/ENSG00000090621-PABPC4) is an RNA-binding protein with evidence at the protein level. It is detected in all RNA tissue distributions and is primarily located in the cytosol. The gene is expressed in various clusters, including Cluster 13 for blood expression, Cluster 19 for tissue expression, Cluster 12 for brain expression, Cluster 37 for cell line expression, and Cluster 58 for single cell expression, which is associated with macrophages and the innate immune response.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046), [BRD9](https://www.ncbi.nlm.nih.gov/gene/65980), [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170), [AR](https://www.ncbi.nlm.nih.gov/gene/367), and [ARID2](https://www.ncbi.nlm.nih.gov/gene/196528), each shown to be regulating in 5 experiments.



The GWAS data indicates associations with various disease conditions, including disease of metabolism, coronary artery disease, artery disease, vascular disease, cardiovascular system disease, inherited metabolic disorder, lipid storage disease, lysosomal storage disease, cholesterol ester storage disease, and cholesterol embolism.



The gene is expressed in various tissues, including muscle, pancreas, and ovary, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Lymphoma Burkitts (Raji), 721 B lymphoblasts, CD34+ cells, Lymphoma Burkitts (Daudi), and CD105+ endothelial cells.




The analyzed protein sequence has a GRAVY value of -0.443 (41.24th percentile), indicating a hydrophilic nature. It exhibits a charge of 16.48 at pH 7.0 (90.52nd percentile) and a median structural flexibility of 1.006 (68.2nd percentile). The protein's secondary structure is predicted to be 35.87% helix (79.97th percentile), 28.26% sheet (19.63rd percentile), and 28.57% turn (52.36th percentile). The instability index is 42.02 (28.71st percentile), with an isoelectric point of 9.31 (87.15th percentile). The protein has a length of 644 amino acids (71.78th percentile) and a molecular weight of 70782.05 Da (71.05th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |