---
layout: post
title: MOCS2
date: 2025-01-17 16:55 CST
description: MOCS2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4338) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4338  | MOCS2 | ENSG00000164172 | 5q11.2 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654) and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). It is involved in the [Mo-molybdopterin cofactor biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006777) and the [molybdopterin cofactor biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0032324). Additionally, it is part of the [molybdopterin synthase complex](https://amigo.geneontology.org/amigo/term/GO:0019008) and contributes to [molybdopterin synthase activity](https://amigo.geneontology.org/amigo/term/GO:0030366). The gene is also located in the [nuclear speck](https://amigo.geneontology.org/amigo/term/GO:0016607).


The gene length is 84,705 base pairs (78.56% of all genes), the mature length is 1,716 base pairs, and the primary transcript length is 15,453 base pairs (44.84% of all genes).


The gene MOCS2 (NCBI ID: 4338) has been mentioned in [8 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22MOCS2%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning this gene was in 2003, and the middle 50% of publications occurred between 2009 and 2022.


The top 5 publications mentioning MOCS2, ranked by their scientific influence, include "[Molybdenum cofactor deficiency: Mutations in GPHN, MOCS1, and MOCS2.](https://pubmed.ncbi.nlm.nih.gov/21031595)" (2011) (relative citation ratio: 2.58), "[Mutations in the molybdenum cofactor biosynthetic genes MOCS1, MOCS2, and GEPH.](https://pubmed.ncbi.nlm.nih.gov/12754701)" (2003) (relative citation ratio: 2.01), "[The Clinical and Molecular Characteristics of Molybdenum Cofactor Deficiency Due to MOCS2 Mutations.](https://pubmed.ncbi.nlm.nih.gov/31201073)" (2019) (relative citation ratio: 0.85), "[MOCS2 links nucleotide metabolism to nucleoli function.](https://pubmed.ncbi.nlm.nih.gov/34698840)" (2022) (relative citation ratio: 0.47), and "[<i>moc-6</i>/MOCS2A is necessary for molybdenum cofactor synthesis in <i>C. elegans</i>.](https://pubmed.ncbi.nlm.nih.gov/35224462)" (2022) (relative citation ratio: 0.47). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[MOCS2](https://www.proteinatlas.org/ENSG00000164172-MOCS2) is a transferase with evidence at the protein level, implicated in disease variants and FDA-approved drug targets. It is detected in all tissues and is localized to the nucleoplasm, nuclear speckles, and cytosol. MOCS2 is expressed in T-cells with unknown function and is involved in cellular respiration in non-specific tissues, synaptic function in neurons, mRNA splicing, and cell cycle in non-specific cell lines, as well as basic cellular processes in single cells.


The top transcription factors (TFs) identified from the GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [MYOD1](https://www.ncbi.nlm.nih.gov/gene/4654), [GATA2](https://www.ncbi.nlm.nih.gov/gene/2624), [INTS11](https://www.ncbi.nlm.nih.gov/gene/54973), [MYC](https://www.ncbi.nlm.nih.gov/gene/4609), and [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), each shown to be regulating in 4 experiments.





The gene is expressed in various tissues, including the brain, adrenal gland, and kidney, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the prefrontal cortex, thyroid, amygdala, colon, small intestine, adrenal gland, and kidney.


The input data highlights various pathways and defects related to metabolism, particularly focusing on vitamins and cofactors. Defects in genes such as AMN, BTD, CD320, CUBN, GIF, HLCS, LMBRD1, MMAA, MMAB, MMACHC, MMADHC, MTR, MTRR, MUT, and TCN2 are associated with conditions like hereditary megaloblastic anemia, biotidinase deficiency, methylmalonic aciduria, intrinsic factor deficiency, and multiple carboxylase deficiency. Additionally, the data includes defects in biotin and cobalamin metabolism, as well as molybdenum cofactor biosynthesis, all of which are linked to various diseases.



The analyzed protein sequence has a GRAVY value of 0.0194 (84.86th percentile), indicating a slightly hydrophobic nature. The charge at pH 7.0 is -5.32 (32.8th percentile), and the median structural flexibility is 1.001 (46.02nd percentile). The protein has a propensity for helix and sheet structures, with affinities of 0.343 (69.9th percentile) and 0.380 (78.54th percentile), respectively, and a turn affinity of 0.255 (24.85th percentile). The instability index is 49.75 (56.4th percentile), and the isoelectric point is 5.01 (7.93rd percentile). The protein is 138 amino acids long (6.71st percentile) with a molecular weight of 15349.46 Da (6.63rd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |