---
layout: post
title: MAP4
date: 2025-01-17 16:55 CST
description: MAP4 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4134) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4134  | MAP4 | ENSG00000047849 | 3p21.31 |



The gene is involved in [microtubule cytoskeleton organization](https://amigo.geneontology.org/amigo/term/GO:0000226) and [neuron projection development](https://amigo.geneontology.org/amigo/term/GO:0031175). It enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [structural molecule activity](https://amigo.geneontology.org/amigo/term/GO:0005198), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [microtubule binding](https://amigo.geneontology.org/amigo/term/GO:0008017). The gene is located in the [microtubule organizing center](https://amigo.geneontology.org/amigo/term/GO:0005815), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [axoneme](https://amigo.geneontology.org/amigo/term/GO:0005930), [microtubule cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0015630), [axon](https://amigo.geneontology.org/amigo/term/GO:0030424), and [mitotic spindle](https://amigo.geneontology.org/amigo/term/GO:0072686). It is active in the [microtubule](https://amigo.geneontology.org/amigo/term/GO:0005874) and [neuron projection](https://amigo.geneontology.org/amigo/term/GO:0043005), and is part of the [microtubule associated complex](https://amigo.geneontology.org/amigo/term/GO:0005875). The gene acts upstream of or within [mitotic spindle organization](https://amigo.geneontology.org/amigo/term/GO:0007052), [microtubule sliding](https://amigo.geneontology.org/amigo/term/GO:0051012), [establishment of spindle orientation](https://amigo.geneontology.org/amigo/term/GO:0051294), and [cell division](https://amigo.geneontology.org/amigo/term/GO:0051301). Additionally, it is involved in [microtubule polymerization](https://amigo.geneontology.org/amigo/term/GO:0046785), [cilium disassembly](https://amigo.geneontology.org/amigo/term/GO:0061523), and [negative regulation of non-motile cilium assembly](https://amigo.geneontology.org/amigo/term/GO:1902856). The gene also enables [microtubule stabilizing activity](https://amigo.geneontology.org/amigo/term/GO:0140778).


The gene length is 3,529 base pairs (9.81% of all genes), the mature length is 731.5 base pairs, and the primary transcript length is 3,223.5 base pairs.


The gene MAP4 (NCBI ID: 4134) has been mentioned in [67 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22MAP4%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1985 and the middle 50% of publications occurring between 1998 and 2015. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning MAP4, ranked by their scientific influence, include "[A GFP-MAP4 reporter gene for visualizing cortical microtubule rearrangements in living epidermal cells](https://pubmed.ncbi.nlm.nih.gov/9811799)" (1998) (relative citation ratio: 5.68), "[A MAP4 kinase related to Ste20 is a nutrient-sensitive regulator of mTOR signalling.](https://pubmed.ncbi.nlm.nih.gov/17253963)" (2007) (relative citation ratio: 3.77), "[Analysis of MAP 4 function in living cells using green fluorescent protein (GFP) chimeras.](https://pubmed.ncbi.nlm.nih.gov/7622564)" (1995) (relative citation ratio: 3.13), "[Phosphorylation of microtubule-associated proteins MAP2 and MAP4 by the protein kinase p110mark. Phosphorylation sites and regulation of microtubule dynamics.](https://pubmed.ncbi.nlm.nih.gov/8631898)" (1996) (relative citation ratio: 3.02), and "[Mammalian septins regulate microtubule stability through interaction with the microtubule-binding protein MAP4.](https://pubmed.ncbi.nlm.nih.gov/16093351)" (2005) (relative citation ratio: 2.71). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[MAP4](https://www.proteinatlas.org/ENSG00000047849-MAP4) is an FDA-approved drug target with evidence at the protein level. It is detected in all tissues and is primarily localized to the plasma membrane, microtubules, primary cilium, basal body, cytosol, acrosome, and equatorial segment. MAP4 is part of Cluster 44 in blood expression, which includes plasma membrane proteins in plasmacytoid dendritic cells. In tissue expression, it belongs to Cluster 54, involved in basic cellular processes. In the brain, it is part of Cluster 19, associated with astrocytes and mixed functions. Additionally, MAP4 is found in Cluster 8 of cell line expression and Cluster 68 of single-cell expression, which are related to basic cellular processes and muscle contraction in cardiomyocytes, respectively.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2B](https://www.ncbi.nlm.nih.gov/gene/9757) and [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313), each with 6 experiments. Additionally, [NFYA](https://www.ncbi.nlm.nih.gov/gene/4800), [HDAC1](https://www.ncbi.nlm.nih.gov/gene/3065), and [RELA](https://www.ncbi.nlm.nih.gov/gene/5970) were each found to be regulating in 5 experiments.



The GWAS data indicates associations with a range of conditions, including artery disease, vascular disease, cardiovascular system disease, hypertension, nervous system disease, central nervous system disease, neurodegenerative disease, disease of mental health, substance-related disorder, substance dependence, alcohol dependence, synucleinopathy, and Parkinson's disease.


The gene MAP4 has been associated with several traits and diseases through genome-wide association studies. Notably, it has been linked to [Height](https://pubmed.ncbi.nlm.nih.gov/36224396) and [Mean arterial pressure](https://pubmed.ncbi.nlm.nih.gov/27618448), which maps to the MeSH term [Arterial Pressure](https://meshb.nlm.nih.gov/record/ui?ui=D062186) using the method [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0).


The gene is expressed in various tissues, including heart, muscle, nerve, and brain, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in pineal gland (both day and night), retina, spinal cord, thalamus, skeletal muscle, and olfactory bulb.




The analyzed protein sequence has a GRAVY value of -0.541 (30.98th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -22.49 (9.57th percentile), and the median structural flexibility is 1.019 (96.03rd percentile). The protein has a high instability index of 57.14 (77.29th percentile) and an isoelectric point of 5.32 (13.84th percentile). The sequence length is 1152 amino acids (90.9th percentile) with a molecular weight of 121003.59 Da (89.57th percentile). The secondary structure affinities are 38.19% for helix (89.32nd percentile), 27.69% for sheet (17.22nd percentile), and 34.20% for turn (86.81st percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |