---
layout: post
title: EPS8
date: 2024-12-08 23:57 CST
description: EPS8 description
tags: [cooccuring-genes]
categories: shortest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2059) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2059  | EPS8 | ENSG00000151491 | 12p12.3 |



The gene length is 721.0 base pairs (0.89 percentile of all genes), the mature length is 528.0 base pairs, and the primary transcript length is 721.0 base pairs.


EPS8, with NCBI Gene ID 2059, has been the subject of [90 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22EPS8%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink). The earliest year of publication is 1993, and the middle 50% of publications occurred between 2007 and 2018.


The top five publications mentioning EPS8, ranked by their scientific influence, include "[EPS8 and E3B1 transduce signals from Ras to Rac.](https://pubmed.ncbi.nlm.nih.gov/10499589)" (1999) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 4.48, "[The Eps8 protein coordinates EGF receptor signalling through Rac and trafficking through Rab5.](https://pubmed.ncbi.nlm.nih.gov/11099046)" (2000) with an RCR of 3.53, "[Regulation of cell shape by Cdc42 is mediated by the synergic actin-bundling activity of the Eps8-IRSp53 complex.](https://pubmed.ncbi.nlm.nih.gov/17115031)" (2006) with an RCR of 3.43, "[Eps8, a substrate for the epidermal growth factor receptor kinase, enhances EGF-dependent mitogenic signals.](https://pubmed.ncbi.nlm.nih.gov/8404850)" (1993) with an RCR of 2.95, and "[Eps8 controls actin-based motility by capping the barbed ends of actin filaments.](https://pubmed.ncbi.nlm.nih.gov/15558031)" (2004) with an RCR of 2.68. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


The gene [Actin-binding](https://www.proteinatlas.org/[Ensembl]-[Gene]) is involved in deafness and non-syndromic deafness, with evidence at the protein level. It is detected in many tissues and is primarily located in the plasma membrane. The gene is expressed in monocytes and neutrophils, playing a role in the innate immune response. It is also involved in the organization of the extracellular matrix (ECM) in connective tissue and has mixed functions in the hindbrain. Additionally, it is expressed in smooth muscle cells, where it participates in signal transduction.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [TP53](https://www.ncbi.nlm.nih.gov/gene/7157), [HDAC2](https://www.ncbi.nlm.nih.gov/gene/3066), [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), [SP1](https://www.ncbi.nlm.nih.gov/gene/6667), and [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), each shown to be regulating in 5 experiments.











The analyzed protein sequence has a GRAVY value of -0.661 (20.37th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of 0.483 (51.78th percentile). The median structural flexibility is 1.009 (78.03rd percentile), with a secondary structure composition of 29.6% helix (28.07th percentile), 29.7% sheet (26.7th percentile), and 33.6% turn (84.38th percentile). The instability index is 54.3 (70.16th percentile), and the isoelectric point is 7.1 (51.43rd percentile). The protein is 822 amino acids long (81.57th percentile) with a molecular weight of 91880.4 Da (81.53rd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |