---
layout: post
title: MAPK4
date: 2025-01-17 16:55 CST
description: MAPK4 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5596) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5596  | MAPK4 | ENSG00000141639 | 18q21.1-q21.2 |



The gene is involved in the [MAPK cascade](https://amigo.geneontology.org/amigo/term/GO:0000165) and [protein phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0006468), playing a crucial role in [intracellular signal transduction](https://amigo.geneontology.org/amigo/term/GO:0035556) and the [cell cycle](https://amigo.geneontology.org/amigo/term/GO:0007049). It enables [protein serine/threonine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004674), [MAP kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004707), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524), [protein kinase binding](https://amigo.geneontology.org/amigo/term/GO:0019901), [protein homodimerization activity](https://amigo.geneontology.org/amigo/term/GO:0042803), [protein heterodimerization activity](https://amigo.geneontology.org/amigo/term/GO:0046982), and [protein serine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0106310). The gene is active and located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829).


The gene length is 20,221.0 base pairs (42.31% of all genes), the mature length is 1,546.0 base pairs, and the primary transcript length is 19,708.0 base pairs.


MAPK4 (Gene ID: 5596) has been mentioned in [19 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22MAPK4%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning MAPK4 was in 1996, and the middle 50% of publications occurred between 2016 and 2020.


The top 5 publications mentioning MAPK4, ranked by their scientific influence, are as follows. The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. The most influential publication is "[CircRNA circ_0000190 inhibits the progression of multiple myeloma through modulating miR-767-5p/MAPK4 pathway.](https://pubmed.ncbi.nlm.nih.gov/30728056)" (2019) (relative citation ratio: 5.11), followed by "[MAPK4 overexpression promotes tumor progression via noncanonical activation of AKT/mTOR signaling.](https://pubmed.ncbi.nlm.nih.gov/30688659)" (2019) (relative citation ratio: 2.7). Other notable publications include "[MAPK4 promotes prostate cancer by concerted activation of androgen receptor and AKT.](https://pubmed.ncbi.nlm.nih.gov/33586682)" (2021) (relative citation ratio: 2.51), "[MiR-127 attenuates adipogenesis by targeting MAPK4 and HOXC6 in porcine adipocytes.](https://pubmed.ncbi.nlm.nih.gov/31206669)" (2019) (relative citation ratio: 1.77), and "[Linear B-cell epitope mapping of MAPK3 and MAPK4 from Leishmania braziliensis: implications for the serodiagnosis of human and canine leishmaniasis.](https://pubmed.ncbi.nlm.nih.gov/25359475)" (2015) (relative citation ratio: 1.75). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[MAPK4](https://www.proteinatlas.org/ENSG00000141639-MAPK4) is a kinase with serine/threonine-protein kinase and transferase molecular functions. Evidence for this gene is available at the protein level. Its RNA is detected in many tissues, with notable expression in the brain, particularly in neuronal signaling clusters, and in astrocytes with mixed functions. Subcellularly, MAPK4 is localized to the nucleoplasm, Golgi apparatus, midbody, and cytosol. In terms of tissue expression, it is part of Cluster 61, which is associated with brain neuronal signaling. Additionally, it is expressed in liver cancer cell lines involved in metabolism and in neurons related to neuronal signaling at the single-cell level.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [ZBTB7A](https://www.ncbi.nlm.nih.gov/gene/51341), [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), each with 5 experiments. Additionally, [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613) was found to be regulating in 4 experiments.



The input data from GWAS indicates associations with a wide range of disease conditions, including various types of cancer such as male reproductive organ cancer, prostate cancer, lung cancer, and uterine cancer. Additionally, the data highlights links to hematopoietic and immune system diseases, including leukopenia, neutropenia, and agranulocytosis. Metabolic disorders, particularly type 2 diabetes mellitus and glucose metabolism disease, are also implicated. Furthermore, the data suggests associations with cardiovascular system diseases, including hypertension and vascular disease, as well as nervous system diseases such as cognitive disorders and prion disease.



The gene is expressed in various tissues, including the heart, brain, and adrenal gland, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the bone marrow, trigeminal ganglion, skeletal muscle, ciliary ganglion, uterus corpus, ovary, and testis interstitial.




The analyzed protein sequence has a GRAVY value of -0.418 (43.99th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -21.98 (9.88th percentile), and the median structural flexibility is 1.003 (55.71st percentile). The protein has a propensity for helical structures (34.41%, 70.88th percentile), with a lower tendency for sheet (32.03%, 39.65th percentile) and turn (30.66%, 68.51st percentile) conformations. The instability index is 49.20 (54.56th percentile), and the isoelectric point is 5.25 (12.52nd percentile). The sequence length is 587 amino acids (67.46th percentile), with a molecular weight of 65920.98 Da (67.8th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |