---
layout: post
title: CRKL
date: 2025-01-17 16:55 CST
description: CRKL description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1399) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1399  | CRKL | ENSG00000099942 | 22q11.21 |



The gene is involved in various biological processes, including [regulation of cell growth](https://amigo.geneontology.org/amigo/term/GO:0001558), [blood vessel development](https://amigo.geneontology.org/amigo/term/GO:0001568), and [urogenital system development](https://amigo.geneontology.org/amigo/term/GO:0001655). It plays a role in [neuron migration](https://amigo.geneontology.org/amigo/term/GO:0001764) and [B cell apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0001783), and enables functions such as [phosphotyrosine residue binding](https://amigo.geneontology.org/amigo/term/GO:0001784) and [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723). The gene is also involved in the regulation of protein phosphorylation, both [negative](https://amigo.geneontology.org/amigo/term/GO:0001933) and [positive](https://amigo.geneontology.org/amigo/term/GO:0001934), and participates in [outflow tract morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0003151). It is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654) and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and is part of a [protein-containing complex](https://amigo.geneontology.org/amigo/term/GO:0032991). Additionally, the gene is involved in various signaling pathways, including the [JNK cascade](https://amigo.geneontology.org/amigo/term/GO:0007254), [Ras protein signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007265), and [fibroblast growth factor receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0008543), among others. It also plays a role in developmental processes such as [spermatogenesis](https://amigo.geneontology.org/amigo/term/GO:0007283), [male gonad development](https://amigo.geneontology.org/amigo/term/GO:0008584), and [anterior/posterior pattern specification](https://amigo.geneontology.org/amigo/term/GO:0009952). The gene is involved in the [negative regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0010629) and various aspects of neural development, including [dendrite development](https://amigo.geneontology.org/amigo/term/GO:0016358), [hippocampus development](https://amigo.geneontology.org/amigo/term/GO:0021766), and [cerebral cortex development](https://amigo.geneontology.org/amigo/term/GO:0021987). Furthermore, it enables functions like [receptor tyrosine kinase binding](https://amigo.geneontology.org/amigo/term/GO:0030971) and [signaling adaptor activity](https://amigo.geneontology.org/amigo/term/GO:0035591), and is involved in processes such as [cell migration](https://amigo.geneontology.org/amigo/term/GO:0016477), [cell chemotaxis](https://amigo.geneontology.org/amigo/term/GO:0060326), and various cellular responses to stimuli.


The gene length is 18,191.0 base pairs (39.77% of all genes), the mature length is 629.5 base pairs, and the primary transcript length is 11,371.5 base pairs.


The gene CRKL (NCBI ID: 1399) has been mentioned in [145 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CRKL%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1993 and the middle 50% of publications occurring between 2001 and 2016. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning CRKL, ranked by their scientific influence, include "[The proto-oncogene product p120CBL and the adaptor proteins CRKL and c-CRK link c-ABL, p190BCR/ABL and p210BCR/ABL to the phosphatidylinositol-3' kinase pathway.](https://pubmed.ncbi.nlm.nih.gov/8632906)" (1996) (relative citation ratio: 4.54), "[Crk and CrkL adaptor proteins: networks for physiological and pathological signaling.](https://pubmed.ncbi.nlm.nih.gov/19426560)" (2009) (relative citation ratio: 4.15), "[Crkl is the major tyrosine-phosphorylated protein in neutrophils from patients with chronic myelogenous leukemia.](https://pubmed.ncbi.nlm.nih.gov/8083188)" (1994) (relative citation ratio: 3.91), "[Mice lacking the homologue of the human 22q11.2 gene CRKL phenocopy neurocristopathies of DiGeorge syndrome.](https://pubmed.ncbi.nlm.nih.gov/11242111)" (2001) (relative citation ratio: 3.78), and "[Isolation and chromosomal localization of CRKL, a human crk-like gene.](https://pubmed.ncbi.nlm.nih.gov/8361759)" (1993) (relative citation ratio: 3.46). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CRKL](https://www.proteinatlas.org/ENSG00000099942-CRKL) is a gene with evidence at the protein level. Its RNA is detected in all tissues, and the protein is localized in the nucleoplasm and cytosol. In terms of expression clusters, CRKL is found in Cluster 33 for blood expression, which is non-specific and related to ATP binding. For tissue expression, it is in Cluster 56 with non-specific and unknown function. In the brain, it is part of Cluster 19, associated with astrocytes and mixed function. In cell lines, it belongs to Cluster 66, which is non-specific and related to mitochondria. Additionally, in single-cell expression, it is part of Cluster 11, associated with late spermatids and unknown function.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [RELA](https://www.ncbi.nlm.nih.gov/gene/5970) with 6 experiments, [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170) with 5 experiments, [STAT3](https://www.ncbi.nlm.nih.gov/gene/6774) with 5 experiments, [ATF3](https://www.ncbi.nlm.nih.gov/gene/467) with 5 experiments, and [TCF12](https://www.ncbi.nlm.nih.gov/gene/6938) with 5 experiments.





The gene is expressed in various tissues, including muscle, skin, and brain. Additionally, it is expressed in several cell types such as 721_B_lymphoblasts, CD56+ NK cells, CD4+ T cells, leukemia chronic myelogenous K-562 cells, and CD71+ early erythroid cells.


The proteins are involved in various signaling pathways, including cytokine signaling in the immune system, Frs2-mediated activation, prolonged ERK activation events, and signaling to ERKs. Additionally, they participate in immune system processes, downstream signal transduction, and signaling by PDGF. The proteins also play roles in NGF signaling via TRKA from the plasma membrane, signaling by NGF, interleukin-3, 5, and GM-CSF signaling, regulation of signaling by CBL, signaling by interleukins, and general signal transduction.



The analyzed protein sequence has a GRAVY value of -0.582 (26.83rd percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -2.772 (40.43rd percentile), and the median structural flexibility is 1.006 (67.29th percentile). The protein has a propensity for helix, sheet, and turn structures of 26.07% (7.95th percentile), 32.01% (39.56th percentile), and 35.31% (90.1st percentile), respectively. The instability index is 58.66 (80.39th percentile), suggesting a relatively stable protein. The isoelectric point is 6.26 (37.06th percentile), with a molecular weight of 33776.57 Da (28.9th percentile) and a length of 303 amino acids (28.98th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |