---
layout: post
title: RPLP0
date: 2024-12-08 23:57 CST
description: RPLP0 description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6175) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6175  | RPLP0 | ENSG00000089157 | 12q24.23 |



The gene length is 82,153 base pairs (77.93rd percentile), the mature length is 4,223 base pairs (85.61st percentile), and the primary transcript length is 53,389 base pairs (76.92nd percentile).


The gene RPLP0, with NCBI ID 6175, has been featured in [5 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RPLP0%22) (count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1995, and the middle 50% of publications occurred between 2011 and 2019.


The top 5 publications mentioning RPLP0, ranked by their relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)), include studies such as "[Expression of the ribosomal proteins Rplp0, Rplp1, and Rplp2 in gynecologic tumors.](https://pubmed.ncbi.nlm.nih.gov/21040949)" (2011) (relative citation ratio: 1.52), "[Dysregulation of apoptotic signaling pathways by interaction of RPLP0 and cathepsin X/Z in gastric cancer.](https://pubmed.ncbi.nlm.nih.gov/25433997)" (2015) (relative citation ratio: 1.12), and "[Ricin A chain can be chemically cross-linked to the mammalian ribosomal proteins L9 and L10e.](https://pubmed.ncbi.nlm.nih.gov/7759553)" (1995) (relative citation ratio: 1.11). Additionally, "[The Ribosomal Protein RPLP0 Mediates PLAAT4-induced Cell Cycle Arrest and Cell Apoptosis.](https://pubmed.ncbi.nlm.nih.gov/31131438)" (2019) (relative citation ratio: 0.88) and "[miR-4731-5p Enhances Apoptosis and Alleviates Epithelial-Mesenchymal Transition through Targeting RPLP0 in Non-Small-Cell Lung Cancer.](https://pubmed.ncbi.nlm.nih.gov/35342398)" (2022) (relative citation ratio: 0.81) are also notable. The relative citation ratio measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [JUN](https://www.ncbi.nlm.nih.gov/gene/3725) with 6 experiments, [FOXM1](https://www.ncbi.nlm.nih.gov/gene/2305) with 5 experiments, [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599) with 5 experiments, [KDM1A](https://www.ncbi.nlm.nih.gov/gene/23028) with 5 experiments, and [SUPT5H](https://www.ncbi.nlm.nih.gov/gene/6829) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent detection in multiple experiments.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [CTCF](https://www.ncbi.nlm.nih.gov/gene/6829) (5 experiments), [STAT3](https://www.ncbi.nlm.nih.gov/gene/6872) (4 experiments), [GATA2](https://www.ncbi.nlm.nih.gov/gene/2623) (2 experiments), [MYC](https://www.ncbi.nlm.nih.gov/gene/4849) (2 experiments), and [IRF1](https://www.ncbi.nlm.nih.gov/gene/3659) (2 experiments). These TFs were found to regulate gene expression in multiple Perturb-seq experiments, highlighting their significant roles in transcriptional regulation.








The analyzed protein sequence has a GRAVY value of 0.040 (85.74th percentile), indicating a slightly hydrophobic nature. The charge at pH 7.0 is -4.13 (36.13th percentile), and the median structural flexibility is 0.999 (36.27th percentile). The protein has a propensity for helix and sheet structures, with affinities of 0.366 (83.52nd percentile) and 0.363 (68.2nd percentile), respectively, and a turn affinity of 0.268 (36.38th percentile). The instability index is 36.49 (13.07th percentile), and the isoelectric point is 5.72 (23.34th percentile). The protein is 317 amino acids long (32.33rd percentile) with a molecular weight of 34273.1 Da (29.62nd percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |