---
layout: post
title: FLOT2
date: 2025-01-17 16:55 CST
description: FLOT2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2319) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2319  | FLOT2 | ENSG00000132589 | 17q11.2 |



The gene is located in various cellular components, including the [uropod](https://amigo.geneontology.org/amigo/term/GO:0001931), [acrosomal membrane](https://amigo.geneontology.org/amigo/term/GO:0002080), [endosome](https://amigo.geneontology.org/amigo/term/GO:0005768), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [caveola](https://amigo.geneontology.org/amigo/term/GO:0005901), [adherens junction](https://amigo.geneontology.org/amigo/term/GO:0005912), [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [basolateral plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0016323), [lamellipodium](https://amigo.geneontology.org/amigo/term/GO:0030027), [endocytic vesicle](https://amigo.geneontology.org/amigo/term/GO:0030139), [cytoplasmic vesicle](https://amigo.geneontology.org/amigo/term/GO:0031410), [vesicle](https://amigo.geneontology.org/amigo/term/GO:0031982), [dendrite cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0032839), [intracellular membrane-bounded organelle](https://amigo.geneontology.org/amigo/term/GO:0043231), [cell-cell contact zone](https://amigo.geneontology.org/amigo/term/GO:0044291), [perinuclear region of cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0048471), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). It is also part of the [flotillin complex](https://amigo.geneontology.org/amigo/term/GO:0016600) and colocalizes with the [cortical actin cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0030864). The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is involved in various biological processes, including [cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0007155), [epidermis development](https://amigo.geneontology.org/amigo/term/GO:0008544), [negative regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0010629), [positive regulation of canonical NF-kappaB signal transduction](https://amigo.geneontology.org/amigo/term/GO:0043123), [protein localization to plasma membrane raft](https://amigo.geneontology.org/amigo/term/GO:0044860), [regulation of myoblast differentiation](https://amigo.geneontology.org/amigo/term/GO:0045661), [protein stabilization](https://amigo.geneontology.org/amigo/term/GO:0050821), [protein localization to plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0072659), [anterograde dendritic transport](https://amigo.geneontology.org/amigo/term/GO:0098937), [regulation of postsynaptic membrane neurotransmitter receptor levels](https://amigo.geneontology.org/amigo/term/GO:0099072), [negative regulation of amyloid precursor protein catabolic process](https://amigo.geneontology.org/amigo/term/GO:1902992), and [positive regulation of establishment of T cell polarity](https://amigo.geneontology.org/amigo/term/GO:1903905). Notably, the gene is not located in the [apical plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0016324) and is not involved in the [regulation of heterotypic cell-cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0034114) or the [regulation of toll-like receptor 3 signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0034139).


The gene length is 60,394.0 base pairs (71.16% of all genes), the mature length is 1,997.5 base pairs, and the primary transcript length is 33,623.0 base pairs.


The gene FLOT2 (NCBI ID: 2319) has been mentioned in [67 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22FLOT2%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1994 and the middle 50% of publications occurring between 2013 and 2019. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning FLOT2, ranked by their relative citation ratio (RCR), include "[Flotillin and epidermal surface antigen define a new family of caveolae-associated integral membrane proteins.](https://pubmed.ncbi.nlm.nih.gov/9153235)" (1997) (relative citation ratio: 9.55), "[Stomatin, flotillin-1, and flotillin-2 are major integral proteins of erythrocyte lipid rafts.](https://pubmed.ncbi.nlm.nih.gov/11159550)" (2001) (relative citation ratio: 5.36), "[MicroRNA-351 eases insulin resistance and liver gluconeogenesis via the PI3K/AKT pathway by inhibiting FLOT2 in mice of gestational diabetes mellitus.](https://pubmed.ncbi.nlm.nih.gov/31287224)" (2019) (relative citation ratio: 3.8), "[Membrane and raft association of reggie-1/flotillin-2: role of myristoylation, palmitoylation and oligomerization and induction of filopodia by overexpression.](https://pubmed.ncbi.nlm.nih.gov/14599293)" (2004) (relative citation ratio: 3.64), and "[N6-methyladenosine Demethylase FTO Induces the Dysfunctions of Ovarian Granulosa Cells by Upregulating Flotillin 2.](https://pubmed.ncbi.nlm.nih.gov/34254281)" (2022) (relative citation ratio: 2.42). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[FLOT2](https://www.proteinatlas.org/ENSG00000132589-FLOT2) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in vesicles and the plasma membrane. The gene is expressed in neutrophils with unknown function (Cluster 21) and is associated with immune response in lymphoid tissue (Cluster 70). In the brain, it is found in neurons with mixed function (Cluster 24), and in cell lines, it is linked to the inflammatory response in lymphoma (Cluster 71). Single-cell expression analysis places it in neutrophils involved in degranulation (Cluster 74).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [EP300](https://www.ncbi.nlm.nih.gov/gene/2033) and [GATA2](https://www.ncbi.nlm.nih.gov/gene/2624), each with 5 experiments. Additionally, [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), [TBP](https://www.ncbi.nlm.nih.gov/gene/6908), and [TP53](https://www.ncbi.nlm.nih.gov/gene/7157) were each found to be regulating in 4 experiments.




The gene FLOT2 has been associated with several traits related to cognitive and mathematical abilities. Specifically, it has been linked to [cognitive performance (MTAG)](https://pubmed.ncbi.nlm.nih.gov/30038396), which maps closely to the MeSH term [Cognition](https://meshb.nlm.nih.gov/record/ui?ui=D003071) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method. Additionally, FLOT2 is associated with [self-reported math ability (MTAG)](https://pubmed.ncbi.nlm.nih.gov/30038396), which aligns with the MeSH term [Mathematics](https://meshb.nlm.nih.gov/record/ui?ui=D008433) through embedding match.


The gene is expressed in various tissues, including the bladder, blood, and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the heart, CD56+ NK cells, CD19+ B cells (neg. sel.), whole blood, CD71+ early erythroid cells, and bone marrow.



The mouse ortholog gene 14252 and its human ortholog gene 2319 are associated with abnormal tail movements, a phenotype observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.148 (74.14th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -12.11 (19.24th percentile), and the median structural flexibility is 1.003 (53.66th percentile). The protein has a high affinity for helix structures (0.423, 96.3rd percentile) and a moderate affinity for sheet structures (0.343, 54.46th percentile), with a lower affinity for turn structures (0.180, 0.87th percentile). The instability index is 32.35 (6.23rd percentile), and the isoelectric point is 5.19 (11.27th percentile). The protein is 428 amino acids long (49.26th percentile) with a molecular weight of 47063.62 Da (48.4th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |