---
layout: post
title: PITPNA
date: 2025-01-17 16:55 CST
description: PITPNA description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5306) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5306  | PITPNA | ENSG00000174238 | 17p13.3 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), specifically in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829) and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). It is involved in [lipid metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006629), [axonogenesis](https://amigo.geneontology.org/amigo/term/GO:0007409), [visual perception](https://amigo.geneontology.org/amigo/term/GO:0007601), [phospholipid transport](https://amigo.geneontology.org/amigo/term/GO:0015914), and [intermembrane lipid transfer](https://amigo.geneontology.org/amigo/term/GO:0120009). Additionally, it enables [lipid binding](https://amigo.geneontology.org/amigo/term/GO:0008289), [phosphatidylcholine transporter activity](https://amigo.geneontology.org/amigo/term/GO:0008525), [phosphatidylinositol transfer activity](https://amigo.geneontology.org/amigo/term/GO:0008526), [phosphatidylcholine binding](https://amigo.geneontology.org/amigo/term/GO:0031210), [phosphatidylinositol binding](https://amigo.geneontology.org/amigo/term/GO:0035091), [phosphatidylcholine transfer activity](https://amigo.geneontology.org/amigo/term/GO:0120019), and [phosphatidylglycerol binding](https://amigo.geneontology.org/amigo/term/GO:1901611).


The gene length is 97,941.0 base pairs (81.37% of all genes), the mature length is 4,807.5 base pairs, and the primary transcript length is 95,284.5 base pairs.


The gene PITPNA (NCBI ID: 5306) has been mentioned in [11 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PITPNA%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1994 and the middle 50% of publications occurring between 2000 and 2004. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning PITPNA, ranked by their relative citation ratio (RCR), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows. The most influential publication is "[Structure-function analysis of human [corrected] phosphatidylinositol transfer protein alpha bound to phosphatidylinositol.](https://pubmed.ncbi.nlm.nih.gov/14962392)" (2004) with a relative citation ratio of 1.38. Other notable publications include "[Regulation of PI3K signalling by the phosphatidylinositol transfer protein PITPalpha during axonal extension in hippocampal neurons.](https://pubmed.ncbi.nlm.nih.gov/18285448)" (2008) with a relative citation ratio of 0.75, "[Overexpression of phosphatidylinositol transfer protein alpha in NIH3T3 cells activates a phospholipase A.](https://pubmed.ncbi.nlm.nih.gov/10585408)" (1999) with a relative citation ratio of 0.7, "[The first 5 amino acids of the carboxyl terminus of phosphatidylinositol transfer protein (PITP) alpha play a critical role in inositol lipid signaling. Transfer activity of PITP is essential but not sufficient for restoration of phospholipase C signaling.](https://pubmed.ncbi.nlm.nih.gov/9169461)" (1997) with a relative citation ratio of 0.61, and "[Acyl chain-based molecular selectivity for HL60 cellular phosphatidylinositol and of phosphatidylcholine by phosphatidylinositol transfer protein alpha.](https://pubmed.ncbi.nlm.nih.gov/15522822)" (2004) with a relative citation ratio of 0.41. For more information on RCR, please refer to [this article](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PITPNA](https://www.proteinatlas.org/ENSG00000174238-PITPNA) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in vesicles and the cytosol. The gene is expressed in various clusters, including Cluster 35 in neutrophils related to protein binding, Cluster 13 in skin associated with cornification, Cluster 52 in neurons and synapses related to synaptic function, Cluster 42 in non-specific cell lines with unknown function, and Cluster 15 in non-specific single cells with mixed function.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 6 experiments, [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599) with 5 experiments, [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613) with 5 experiments, [KMT2B](https://www.ncbi.nlm.nih.gov/gene/9757) with 5 experiments, and [EP300](https://www.ncbi.nlm.nih.gov/gene/2033) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The input data indicates associations with various disease conditions, including general disease, disease of anatomical entity, nervous system disease, neuropathy, peripheral neuropathy, and sensory peripheral neuropathy.



The gene is expressed in various tissues, including the retina, CD105+ endothelial cells, whole brain, hypothalamus, thalamus, and cerebellum.


The proteins are involved in several pathways related to developmental biology, including axon guidance, netrin-1 signaling, and the role of second messengers in netrin-1 signaling.



The analyzed protein sequence has a GRAVY value of -0.732 (15.63rd percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -3.785 (37.15th percentile), and the median structural flexibility is 1.006 (67.39th percentile). The protein has a propensity for helical structures (35.93%, 80.33rd percentile), sheet structures (35.56%, 63.22nd percentile), and turn structures (22.59%, 8.37th percentile). The instability index is 40.193 (22.8th percentile), with an isoelectric point of 6.115 (33.57th percentile). The protein is 270 amino acids long (24.39th percentile) and has a molecular weight of 31805.95 Da (26.49th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |