---
layout: post
title: AAAS
date: 2024-12-08 23:57 CST
description: AAAS description
tags: [cooccuring-genes]
categories: longest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8086) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8086  | AAAS | ENSG00000094914 | 12q13.13 |



The gene length is 878,010 base pairs (99.45% of all genes), the mature length is 4,948.5 base pairs, and the primary transcript length is 573,840 base pairs (99.46% of all genes).


The gene AAAS, with NCBI ID 8086, has been the subject of [94 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22AAAS%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1948, and the middle 50% of publications occurred between 1986 and 2010.


The top five publications mentioning the AAAS gene, ranked by their scientific influence, include "[Triple A syndrome is caused by mutations in AAAS, a new WD-repeat protein gene.](https://pubmed.ncbi.nlm.nih.gov/11159947)" (2001) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 3.43, "[The nuclear pore complex protein ALADIN is mislocalized in triple A syndrome.](https://pubmed.ncbi.nlm.nih.gov/12730363)" (2003) with an RCR of 2.16, "[Cellular localization of 17 natural mutant variants of ALADIN protein in triple A syndrome - shedding light on an unexpected splice mutation.](https://pubmed.ncbi.nlm.nih.gov/16609705)" (2006) with an RCR of 1.2, "[Spectrum of mutations of the AAAS gene in Allgrove syndrome: lack of mutations in six kindreds with isolated resistance to corticotropin.](https://pubmed.ncbi.nlm.nih.gov/11701718)" (2001) with an RCR of 1.14, and "[The nucleoporin ALADIN regulates Aurora A localization to ensure robust mitotic spindle formation.](https://pubmed.ncbi.nlm.nih.gov/26246606)" (2015) with an RCR of 1.13. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [RELA](https://www.ncbi.nlm.nih.gov/gene/5970), [REST](https://www.ncbi.nlm.nih.gov/gene/5978), [E2F6](https://www.ncbi.nlm.nih.gov/gene/1876), [PGR](https://www.ncbi.nlm.nih.gov/gene/5241), and [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227), each shown to be regulating in 5 experiments.











The analyzed protein sequence has a GRAVY value of -0.084 (79.18th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of 0.995 (53.96th percentile). The median structural flexibility is 0.997 (26.57th percentile), with a secondary structure composition of 26.2% helix (8.38th percentile), 36.8% sheet (71.81st percentile), and 33.5% turn (84.14th percentile). The instability index is 43.73 (34.71st percentile), and the isoelectric point is 7.26 (52.82nd percentile). The protein is 546 amino acids long (63.7th percentile) with a molecular weight of 59573.43 Da (62.11th percentile). For detailed sequence analysis methods, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |