---
layout: post
title: GSTA4
date: 2025-01-17 16:55 CST
description: GSTA4 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2941) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2941  | GSTA4 | ENSG00000170899 | 6p12.2 |



The gene enables [glutathione transferase activity](https://amigo.geneontology.org/amigo/term/GO:0004364) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), including [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802) and [protein homodimerization activity](https://amigo.geneontology.org/amigo/term/GO:0042803). It is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829) and is involved in the [glutathione metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006749) and [xenobiotic metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006805).


The gene length is 23,000 base pairs (45.46% of all genes), the mature length is 1,834.5 base pairs, and the primary transcript length is 19,085 base pairs.


The gene GSTA4 (NCBI ID: 2941) has been mentioned in [22 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22GSTA4%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning this gene is 1998, and the middle 50% of publications occurred between 2005 and 2012.


The top five publications mentioning GSTA4, ranked by their scientific influence, include "[Human glutathione transferase A4-4: an alpha class enzyme with high catalytic efficiency in the conjugation of 4-hydroxynonenal and other genotoxic products of lipid peroxidation.](https://pubmed.ncbi.nlm.nih.gov/9461507)" (1998) (relative citation ratio: 7.41), "[Human glutathione transferase A4-4 crystal structures and mutagenesis reveal the basis of high catalytic efficiency with toxic lipid peroxidation products.](https://pubmed.ncbi.nlm.nih.gov/10329152)" (1999) (relative citation ratio: 3.69), "[GSTA4 mediates reduction of cisplatin ototoxicity in female mice.](https://pubmed.ncbi.nlm.nih.gov/31515474)" (2019) (relative citation ratio: 2.28), "[Gsta4 controls apoptosis of differentiating adult oligodendrocytes during homeostasis and remyelination via the mitochondria-associated Fas-Casp8-Bid-axis.](https://pubmed.ncbi.nlm.nih.gov/32792491)" (2020) (relative citation ratio: 2.0), and "[Genotoxicity of 4-hydroxy-2-nonenal in human colon tumor cells is associated with cellular levels of glutathione and the modulation of glutathione S-transferase A4 expression by butyrate.](https://pubmed.ncbi.nlm.nih.gov/15829614)" (2005) (relative citation ratio: 1.16). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[GSTA4](https://www.proteinatlas.org/ENSG00000170899-GSTA4) is a transferase with evidence at the protein level. Its RNA is detected in all tissues, and it is notably expressed in monocytes and neutrophils, playing a role in the innate immune response. The gene is also involved in endocytosis, transcription, keratinization, and mitochondrial processes in various cell types and tissues.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908), [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046), [CEBPB](https://www.ncbi.nlm.nih.gov/gene/1051), [SPI1](https://www.ncbi.nlm.nih.gov/gene/6688), and [PAX5](https://www.ncbi.nlm.nih.gov/gene/5079), each shown to be regulating in 5 experiments.



The input data indicates associations with a range of disease conditions, including respiratory disorders such as asthma, bronchial disease, obstructive lung disease, lower respiratory tract disease, lung disease, and respiratory system disease. Additionally, metabolic conditions are highlighted, including disease of metabolism, acquired metabolic disease, carbohydrate metabolism disease, glucose metabolism disease, diabetes mellitus, type 2 diabetes mellitus, nutrition disease, overnutrition, and obesity. Furthermore, the data encompasses nervous system diseases, specifically central nervous system disease, neurodegenerative disease, Alzheimer's disease, tauopathy, dementia, cognitive disorder, disease of mental health, brain disease, and epilepsy syndrome.



The gene is expressed in various tissues, including the pituitary and adrenal gland, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the fetal brain, pituitary, adrenal cortex, and adrenal gland.


The input data highlights various pathways and disorders related to biological oxidations and metabolism. Key pathways include biological oxidations, phase II conjugation, glutathione conjugation, and metabolism. Notable disorders linked to defective proteins are Hypermethioninemia with S-adenosylhomocysteine hydrolase deficiency (HMAHCHD), hyperbilirubinemia caused by defective UGT1A1 and UGT1A4, Hemolytic anemia due to gamma-glutamylcysteine synthetase deficiency (HAGGSD), Glutathionuria (GLUTH), Glutathione synthetase deficiency (GSS deficiency), Methionine adenosyltransferase deficiency (MATD), 5-oxoprolinase deficiency (OPLAHD), Schneckenbecken dysplasia (SCHBCKD), and Thiopurine S-methyltransferase deficiency (TPMT deficiency). Additionally, the data mentions metabolic disorders of biological oxidation enzymes and general disease categories.



The analyzed protein sequence has a GRAVY value of -0.165 (72.67th percentile), indicating a hydrophilic nature. It exhibits a charge of 1.057 at pH 7.0 (54.19th percentile) and a median structural flexibility of 0.996 (24.89th percentile). The protein's secondary structure is predicted to be 36.94% helix (84.92nd percentile), 40.54% sheet (87.77th percentile), and 22.97% turn (9.83rd percentile). The instability index is 51.08 (60.68th percentile), with an isoelectric point of 7.89 (57.86th percentile). The protein has a length of 222 amino acids (17.8th percentile) and a molecular weight of 25703.78 Da (18.97th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |