---
layout: post
title: CTH
date: 2025-01-17 16:55 CST
description: CTH description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1491) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1491  | CTH | ENSG00000116761 | 1p31.1 |



The gene enables [cystathionine gamma-lyase activity](https://amigo.geneontology.org/amigo/term/GO:0004123), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [calmodulin binding](https://amigo.geneontology.org/amigo/term/GO:0005516). It is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737) and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and is also found in the [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). The gene is involved in various processes, including [cysteine metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006534), [lipid metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006629), [cysteine biosynthetic process via cystathionine](https://amigo.geneontology.org/amigo/term/GO:0019343), [transsulfuration](https://amigo.geneontology.org/amigo/term/GO:0019346), [endoplasmic reticulum unfolded protein response](https://amigo.geneontology.org/amigo/term/GO:0030968), [positive regulation of canonical NF-kappaB signal transduction](https://amigo.geneontology.org/amigo/term/GO:0043123), [protein sulfhydration](https://amigo.geneontology.org/amigo/term/GO:0044524), [hydrogen sulfide biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0070814), [positive regulation of aortic smooth muscle cell differentiation](https://amigo.geneontology.org/amigo/term/GO:1904831), [cellular response to leukemia inhibitory factor](https://amigo.geneontology.org/amigo/term/GO:1990830), and [negative regulation of apoptotic signaling pathway](https://amigo.geneontology.org/amigo/term/GO:2001234). Additionally, the gene enables [pyridoxal phosphate binding](https://amigo.geneontology.org/amigo/term/GO:0030170), [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802), [L-cystine L-cysteine-lyase (deaminating)](https://amigo.geneontology.org/amigo/term/GO:0044540), [homocysteine desulfhydrase activity](https://amigo.geneontology.org/amigo/term/GO:0047982), [L-cysteine desulfhydrase activity](https://amigo.geneontology.org/amigo/term/GO:0080146), and [selenocystathionine gamma-lyase activity](https://amigo.geneontology.org/amigo/term/GO:0098606).


The gene length is 74,638 base pairs (75.89th percentile), the mature length is 8,085.5 base pairs (97.78th percentile), and the primary transcript length is 73,477 base pairs (83.3rd percentile).


The gene CTH (NCBI ID: 1491) has been mentioned in [112 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CTH%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning CTH was in 1952, and the middle 50% of publications occurred between 2011 and 2019.


The top 5 publications mentioning CTH, ranked by their scientific influence, include "[Relative contributions of cystathionine beta-synthase and gamma-cystathionase to H2S biogenesis via alternative trans-sulfuration reactions.](https://pubmed.ncbi.nlm.nih.gov/19531479)" (2009) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 11.89, "[Persulfidation-based Modification of Cysteine Desulfhydrase and the NADPH Oxidase RBOHD Controls Guard Cell Abscisic Acid Signaling.](https://pubmed.ncbi.nlm.nih.gov/32024687)" (2020) with an RCR of 11.67, "[H2S biogenesis by human cystathionine gamma-lyase leads to the novel sulfur metabolites lanthionine and homolanthionine and is responsive to the grade of hyperhomocysteinemia.](https://pubmed.ncbi.nlm.nih.gov/19261609)" (2009) with an RCR of 9.37, "[Dysregulation of hydrogen sulfide producing enzyme cystathionine γ-lyase contributes to maternal hypertension and placental abnormalities in preeclampsia.](https://pubmed.ncbi.nlm.nih.gov/23704251)" (2013) with an RCR of 7.9, and "[Erastin sensitizes glioblastoma cells to temozolomide by restraining xCT and cystathionine-γ-lyase function.](https://pubmed.ncbi.nlm.nih.gov/25585997)" (2015) with an RCR of 5.99. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CTH](https://www.proteinatlas.org/ENSG00000116761-CTH) is a gene with molecular functions including calmodulin-binding and lyase activity. It is involved in disease variants and has evidence at the protein level. The gene is detected in all RNA tissue distributions and is particularly expressed in the liver, where it plays a role in metabolism. In the brain, it is found in neurons with mixed functions. Additionally, CTH is expressed in hepatocytes, contributing to metabolic processes. The gene is part of blood expression cluster 43, which is non-specific and related to transcription and translation.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 6 experiments, followed by [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613), [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), and [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313), each with 5 experiments.



The gene has been associated with various disease conditions through Genome-Wide Association Studies (GWAS), including breast cancer, thoracic cancer, and other forms of cancer. Additionally, it has been linked to diseases of cellular proliferation and broader organ system cancers, highlighting its potential role in multiple disease pathways.


Genome-wide association studies have linked CTH to several traits and diseases. Notably, CTH has been associated with [Cystathionine levels](https://pubmed.ncbi.nlm.nih.gov/36357675), which was mapped to the MeSH term [Cystathionine](https://meshb.nlm.nih.gov/record/ui?ui=D003540) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method. Additionally, CTH has been linked to [Metabolite levels](https://pubmed.ncbi.nlm.nih.gov/23823483) and [Metabolite levels (cystathionine; cmh_cystathionine)](https://pubmed.ncbi.nlm.nih.gov/37253714), both of which were mapped to the MeSH term [Metabolome](https://meshb.nlm.nih.gov/record/ui?ui=D055442) using embedding match.


The gene is expressed in multiple tissue samples, including the liver and ovary, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the pineal gland (night), liver, B lymphoblasts (721_B), chronic myelogenous leukemia cells (K-562), adrenal cortex, adrenal gland, and ovary.


The input data highlights several key pathways involved in metabolism, including cysteine formation from homocysteine, degradation of cysteine and homocysteine, sulfur amino acid metabolism, and the broader metabolism of amino acids and derivatives.


The mouse ortholog gene 107869 and its human ortholog gene 1491 are associated with abnormal lens morphology and abnormal retina morphology, with these phenotypes observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.082 (79.37th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -5.98 (31.07th percentile), and the median structural flexibility is 0.994 (17.67th percentile). The protein has a propensity for helix, sheet, and turn structures of 31.85% (49.06th percentile), 36.30% (68.32nd percentile), and 27.90% (46.32nd percentile), respectively. The instability index is 32.86 (6.83rd percentile), with an isoelectric point of 6.21 (35.93rd percentile). The protein length is 405 amino acids (45.97th percentile), and the molecular weight is 44507.50 Da (45.14th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |