---
layout: post
title: SF3A1
date: 2025-01-17 16:55 CST
description: SF3A1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10291) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10291  | SF3A1 | ENSG00000099995 | 22q12.2 |



The gene is involved in [mRNA 3'-splice site recognition](https://amigo.geneontology.org/amigo/term/GO:0000389) and [mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0000398). It enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), and is part of the [spliceosomal complex](https://amigo.geneontology.org/amigo/term/GO:0005681), [U2-type spliceosomal complex](https://amigo.geneontology.org/amigo/term/GO:0005684), [U2 snRNP](https://amigo.geneontology.org/amigo/term/GO:0005686), [U2-type prespliceosome](https://amigo.geneontology.org/amigo/term/GO:0071004), [U2-type precatalytic spliceosome](https://amigo.geneontology.org/amigo/term/GO:0071005), and [catalytic step 2 spliceosome](https://amigo.geneontology.org/amigo/term/GO:0071013). Additionally, it is involved in [mRNA processing](https://amigo.geneontology.org/amigo/term/GO:0006397), [mRNA cis splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0045292), and [U2-type prespliceosome assembly](https://amigo.geneontology.org/amigo/term/GO:1903241). The gene is also located in the [nuclear speck](https://amigo.geneontology.org/amigo/term/GO:0016607).


The gene length is 13,103 base pairs (31.79% of all genes), the mature length is 1,798 base pairs (47.14% of all genes), and the primary transcript length is 12,852 base pairs (39.98% of all genes).


The gene SF3A1, with NCBI ID 10291, has been mentioned in [7 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SF3A1%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication mentioning this gene is 1994, and the middle 50% of publications occurred between 2007 and 2015.


The top 5 publications mentioning SF3A1, ranked by their relative citation ratio (RCR), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows. The most influential publication is "[Stem-loop 4 of U1 snRNA is essential for splicing and interacts with the U2 snRNP-specific SF3A1 protein during spliceosome assembly.](https://pubmed.ncbi.nlm.nih.gov/25403181)" (2014) (relative citation ratio: 0.96). Other notable publications include "[Domains in human splicing factors SF3a60 and SF3a66 required for binding to SF3a120, assembly of the 17S U2 snRNP, and prespliceosome formation.](https://pubmed.ncbi.nlm.nih.gov/11533230)" (2001) (relative citation ratio: 0.62), "[Specific protein-protein interactions between the essential mammalian spliceosome-associated proteins SAP 61 and SAP 114.](https://pubmed.ncbi.nlm.nih.gov/8022796)" (1994) (relative citation ratio: 0.61), "[SF3A1 and pancreatic cancer: new evidence for the association of the spliceosome and cancer.](https://pubmed.ncbi.nlm.nih.gov/26498691)" (2015) (relative citation ratio: 0.4), and "[Subclonal evolution involving SF3B1 mutations in chronic lymphocytic leukemia.](https://pubmed.ncbi.nlm.nih.gov/23337928)" (2013) (relative citation ratio: 0.3). The citation count is sourced from [iCite](https://icite.od.nih.gov).


[SF3A1](https://www.proteinatlas.org/ENSG00000099995-SF3A1) is a gene with evidence at the protein level. Its RNA is detected in all tissues, and it is primarily localized in the nucleoplasm and nuclear speckles. The gene is part of various expression clusters, including Cluster 43 for blood, Cluster 4 for tissue (specific to testis), Cluster 16 for brain, Cluster 66 for cell lines, and Cluster 49 for single cells, all of which are associated with basic cellular processes such as transcription and translation.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) with 5 experiments, [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599) with 4 experiments, [STAT5B](https://www.ncbi.nlm.nih.gov/gene/6777) with 4 experiments, [PGR](https://www.ncbi.nlm.nih.gov/gene/5241) with 4 experiments, and [GATAD2A](https://www.ncbi.nlm.nih.gov/gene/54815) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent detection in CHIP-seq studies.



The GWAS data indicates associations with various metabolic and cardiovascular conditions, including diseases of metabolism, acquired metabolic diseases, carbohydrate metabolism diseases, glucose metabolism diseases, diabetes mellitus, and specifically type 2 diabetes mellitus. Additionally, associations were found with coronary artery disease, artery disease, vascular disease, and broader cardiovascular system diseases.



The gene is expressed in the ovary, as well as in CD19+ B cells (negatively selected), CD71+ early erythroid cells, testis, and testis interstitial tissue.


The proteins are involved in several key processes, including gene expression, processing of capped intron-containing pre-mRNA, and mRNA splicing, with a particular focus on the major pathway of mRNA splicing.



The analyzed protein sequence has a GRAVY value of -0.739 (15.08th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -25.35 (7.88th percentile), and the median structural flexibility is 1.014 (89.76th percentile). The protein has a high instability index of 64.32 (89.44th percentile) and an isoelectric point of 5.15 (10.63rd percentile). The sequence length is 793 amino acids (80.38th percentile) with a molecular weight of 88885.10 Da (80.51st percentile). The secondary structure tendencies are 32.28% for helix (53.1st percentile), 28.37% for sheet (20.06th percentile), and 31.40% for turn (73.2nd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |