---
layout: post
title: CD27
date: 2025-01-17 16:55 CST
description: CD27 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/939) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 939  | CD27 | ENSG00000139193 | 12p13.31 |



The gene enables [transmembrane signaling receptor activity](https://amigo.geneontology.org/amigo/term/GO:0004888) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576) and the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), specifically on the [external side of the plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009897). The gene is involved in the [cell surface receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007166) and plays a role in the [immunoglobulin mediated immune response](https://amigo.geneontology.org/amigo/term/GO:0016064). Additionally, it enables [cysteine-type endopeptidase inhibitor activity involved in the apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0043027) and is involved in the [negative regulation of the apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0043066). The gene also participates in the [response to ethanol](https://amigo.geneontology.org/amigo/term/GO:0045471), the [positive regulation of B cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0045579), the [positive regulation of T cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0045582), the [positive regulation of the JNK cascade](https://amigo.geneontology.org/amigo/term/GO:0046330), the [negative regulation of T cell apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0070233), the [extrinsic apoptotic signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0097191), and the [positive regulation of non-canonical NF-kappaB signal transduction](https://amigo.geneontology.org/amigo/term/GO:1901224).


The gene has a length of 16,802 base pairs (37.76% of all genes), a mature length of 2,129 base pairs (56.04% of all genes), and a primary transcript length of 7,017 base pairs (24.97% of all genes).


The gene CD27, with NCBI ID 939, has been mentioned in [115 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CD27%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication mentioning CD27 is 1991, and the middle 50% of publications occurred between 2008 and 2018.


The top five publications mentioning CD27, ranked by their scientific influence, include "[Human B1 cells in umbilical cord and adult peripheral blood express the novel phenotype CD20+ CD27+ CD43+ CD70-](https://pubmed.ncbi.nlm.nih.gov/21220451)" (2011) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 11.08, "[CD27 is a thymic determinant of the balance between interferon-gamma- and interleukin 17-producing gammadelta T cell subsets](https://pubmed.ncbi.nlm.nih.gov/19270712)" (2009) with an RCR of 8.79, "[The CD70-CD27 axis in oncology: the new kids on the block](https://pubmed.ncbi.nlm.nih.gov/34991665)" (2022) with an RCR of 7.14, "[New emerging targets in cancer immunotherapy: CD27 (TNFRSF7)](https://pubmed.ncbi.nlm.nih.gov/32152062)" (2020) with an RCR of 5.31, and "[Coexpression of CD25 and CD27 identifies FoxP3+ regulatory T cells in inflamed synovia](https://pubmed.ncbi.nlm.nih.gov/15939793)" (2005) with an RCR of 5.1. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CD27](https://www.proteinatlas.org/ENSG00000139193-CD27) is a receptor involved in cancer-related genes and disease variants, with evidence at the protein level. It is detected in many tissues and is primarily located in vesicles. CD27 is part of Cluster 43 in blood expression, which is non-specific and related to transcription and translation. In tissue expression, it belongs to Cluster 36, associated with lymphoid tissue and cytokine signaling. Additionally, CD27 is found in Cluster 11 in brain expression, related to white matter and signal transduction, and in Cluster 46 in cell line expression, linked to lymphoma and humoral immune response. In single-cell expression, it is part of Cluster 77, associated with plasma cells and protein processing.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 4 experiments, [MYC](https://www.ncbi.nlm.nih.gov/gene/4609) and [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) each with 3 experiments, and [STAT5B](https://www.ncbi.nlm.nih.gov/gene/6777) and [TCF3](https://www.ncbi.nlm.nih.gov/gene/6929) each with 2 experiments.





The gene is expressed in various tissues, including blood, spleen, and small intestine, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Lymphoma Burkitts (Raji), CD4+ T cells, CD8+ T cells, Leukemia promyelocytic-HL-60, Lymphoma Burkitts (Daudi), tonsil, and lymph node.



The mouse ortholog gene 21940 and its human ortholog gene 939 are associated with several phenotypic abnormalities. Both sexes exhibit abnormal heart morphology, including an enlarged heart, and abnormal spleen morphology. Additionally, males specifically show abnormalities in seminal vesicle and testis morphology, including small seminal vesicles and small testes.


The analyzed protein sequence has a GRAVY value of -0.482 (36.9% percentile), indicating a hydrophilic nature. It exhibits a charge of 2.47 at pH 7.0 (59.21% percentile) and a median structural flexibility of 0.995 (21.86% percentile). The protein's secondary structure is predicted to be 26.92% helix (11.38% percentile), 27.31% sheet (15.59% percentile), and 29.23% turn (57.86% percentile). The instability index is 68.25 (93.36% percentile), suggesting high instability. The isoelectric point is 7.80 (57.13% percentile), with a molecular weight of 29137.05 Da (23.28% percentile). The sequence length is 260 amino acids (23.02% percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |