---
layout: post
title: ETF1
date: 2025-01-17 16:55 CST
description: ETF1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2107) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2107  | ETF1 | ENSG00000120705 | 5q31.2 |



The gene is involved in the [nuclear-transcribed mRNA catabolic process, nonsense-mediated decay](https://amigo.geneontology.org/amigo/term/GO:0000184) and [cytoplasmic translational termination](https://amigo.geneontology.org/amigo/term/GO:0002184). It enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [translation release factor activity](https://amigo.geneontology.org/amigo/term/GO:0003747), [aminoacyl-tRNA hydrolase activity](https://amigo.geneontology.org/amigo/term/GO:0004045), and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is not located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) but is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737) and is active in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). It acts upstream of or within [translational termination](https://amigo.geneontology.org/amigo/term/GO:0006415) and is involved in the [regulation of translational termination](https://amigo.geneontology.org/amigo/term/GO:0006449). Additionally, it enables [translation termination factor activity](https://amigo.geneontology.org/amigo/term/GO:0008079), [translation release factor activity, codon specific](https://amigo.geneontology.org/amigo/term/GO:0016149), and [sequence-specific mRNA binding](https://amigo.geneontology.org/amigo/term/GO:1990825). The gene is part of the [translation release factor complex](https://amigo.geneontology.org/amigo/term/GO:0018444) and is active in the [cytosolic ribosome](https://amigo.geneontology.org/amigo/term/GO:0022626). It also enables [ribosome binding](https://amigo.geneontology.org/amigo/term/GO:0043022).


The gene length is 16,207 base pairs (36.84% of all genes), the mature length is 1,375 base pairs (34.28% of all genes), and the primary transcript length is 9,081 base pairs (30.79% of all genes).


ETF1 (Gene NCBI ID: 2107) has been mentioned in [73 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ETF1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1995, and the middle 50% of publications occurred between 2002 and 2012.


The top 5 publications mentioning ETF1, ranked by their scientific influence, include "[The products of the SUP45 (eRF1) and SUP35 genes interact to mediate translation termination in Saccharomyces cerevisiae.](https://pubmed.ncbi.nlm.nih.gov/7556078)" (1995) (relative citation ratio: 7.55), "[Binding of a novel SMG-1-Upf1-eRF1-eRF3 complex (SURF) to the exon junction complex triggers Upf1 phosphorylation and nonsense-mediated mRNA decay.](https://pubmed.ncbi.nlm.nih.gov/16452507)" (2006) (relative citation ratio: 7.55), "[The crystal structure of human eukaryotic release factor eRF1--mechanism of stop codon recognition and peptidyl-tRNA hydrolysis.](https://pubmed.ncbi.nlm.nih.gov/10676813)" (2000) (relative citation ratio: 6.05), "[Mutations in the highly conserved GGQ motif of class 1 polypeptide release factors abolish ability of human eRF1 to trigger peptidyl-tRNA hydrolysis.](https://pubmed.ncbi.nlm.nih.gov/10445876)" (1999) (relative citation ratio: 4.75), and "[A small molecule that induces translational readthrough of CFTR nonsense mutations by eRF1 depletion.](https://pubmed.ncbi.nlm.nih.gov/34272367)" (2021) (relative citation ratio: 4.65). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ETF1](https://www.proteinatlas.org/ENSG00000120705-ETF1) is a gene with evidence at the protein level. Its RNA is detected in all tissues, and it is notably expressed in monocytes and neutrophils, where it is involved in degranulation. In the brain, ETF1 is associated with metabolism, while in cell lines, it is linked to mRNA splicing and the cell cycle. Single-cell expression analysis indicates its role in the innate immune response within monocytes.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [RELA](https://www.ncbi.nlm.nih.gov/gene/5970) with 6 experiments, [CREB1](https://www.ncbi.nlm.nih.gov/gene/1385) with 5 experiments, [MAX](https://www.ncbi.nlm.nih.gov/gene/4149) with 5 experiments, [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170) with 5 experiments, and [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.





The gene is expressed in various tissues, including the esophagus, skin, and lung. Additionally, it is expressed in several cell types and tissues such as B lymphoblasts, CD33+ myeloid cells, chronic myelogenous leukemia K-562 cells, promyelocytic leukemia HL-60 cells, CD105+ endothelial cells, smooth muscle, and bronchial epithelial cells.


The input data highlights several key biological processes, including metabolism of proteins, gene expression, and various aspects of translation and nonsense-mediated decay (NMD). Specifically, it covers eukaryotic translation termination, NMD independent of the exon junction complex (EJC), NMD enhanced by the EJC, and general NMD processes.



The analyzed protein sequence has a GRAVY value of -0.3114 (56.72 percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -8.85 (24.64 percentile), and the median structural flexibility is 1.0041 (60.25 percentile). The protein has a predicted affinity for helix, sheet, and turn structures of 33.87% (66.77 percentile), 37.30% (74.76 percentile), and 27.69% (44.48 percentile), respectively. The instability index is 27.82 (2.56 percentile), with an isoelectric point of 5.51 (18.45 percentile). The protein is 437 amino acids long (50.34 percentile) and has a molecular weight of 49030.30 Da (50.57 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |