---
layout: post
title: SUCLA2
date: 2025-01-17 16:55 CST
description: SUCLA2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8803) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8803  | SUCLA2 | ENSG00000136143 | 13q14.2 |



The gene enables [magnesium ion binding](https://amigo.geneontology.org/amigo/term/GO:0000287) and [succinate-CoA ligase (ADP-forming) activity](https://amigo.geneontology.org/amigo/term/GO:0004775). It is involved in [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524). The gene is located in the [mitochondrion](https://amigo.geneontology.org/amigo/term/GO:0005739) and the [mitochondrial matrix](https://amigo.geneontology.org/amigo/term/GO:0005759), and is active in the [mitochondrion](https://amigo.geneontology.org/amigo/term/GO:0005739). It is also part of the [succinate-CoA ligase complex (ADP-forming)](https://amigo.geneontology.org/amigo/term/GO:0009361) and the [succinate-CoA ligase complex](https://amigo.geneontology.org/amigo/term/GO:0042709). The gene is involved in the [tricarboxylic acid cycle](https://amigo.geneontology.org/amigo/term/GO:0006099), [succinyl-CoA metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006104), [succinate metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006105), [succinyl-CoA pathway](https://amigo.geneontology.org/amigo/term/GO:0006781), and [succinyl-CoA catabolic process](https://amigo.geneontology.org/amigo/term/GO:1901289). Additionally, the gene is located in the [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062).


The gene length is 11,255 base pairs (28.23% of all genes), the mature length is 1,573 base pairs (40.35% of all genes), and the primary transcript length is 8,142 base pairs (28.31% of all genes).


The gene SUCLA2 (NCBI ID: 8803) has been mentioned in [49 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SUCLA2%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1978 and the middle 50% of publications occurring between 2010 and 2019. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning SUCLA2, ranked by their scientific influence, include "[SUCLA2-coupled regulation of GLS succinylation and activity counteracts oxidative stress in tumor cells.](https://pubmed.ncbi.nlm.nih.gov/33991485)" (2021) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 5.23, "[Interplay among folding, sequence, and lipophilicity in the antibacterial and hemolytic activities of alpha/beta-peptides.](https://pubmed.ncbi.nlm.nih.gov/17212422)" (2007) with an RCR of 4.78, "[Estrogen Receptor Beta (ERÎ²): A Ligand Activated Tumor Suppressor.](https://pubmed.ncbi.nlm.nih.gov/33194742)" (2020) with an RCR of 3.94, "[Mitochondrial encephalomyopathy with elevated methylmalonic acid is caused by SUCLA2 mutations.](https://pubmed.ncbi.nlm.nih.gov/17287286)" (2007) with an RCR of 3.33, and "[SUCLA2 mutations are associated with mild methylmalonic aciduria, Leigh-like encephalomyopathy, dystonia and deafness.](https://pubmed.ncbi.nlm.nih.gov/17301081)" (2007) with an RCR of 3.29. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[SUCLA2](https://www.proteinatlas.org/ENSG00000136143-SUCLA2) is a gene encoding a protein with ligase molecular function, primarily located in the mitochondria. It is involved in primary mitochondrial disease, with evidence supporting its role at the protein level. The gene is detected in all tissues, with notable expression in skeletal muscle related to muscle contraction. Additionally, it is expressed in late spermatids, playing a role in spermatogenesis. The gene is part of the non-specific mitochondrial translation cluster in blood expression and shows a mixed function in brain expression.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [HDAC2](https://www.ncbi.nlm.nih.gov/gene/3066) and [RELA](https://www.ncbi.nlm.nih.gov/gene/5970), each with 5 experiments. Additionally, [H2AZ1](https://www.ncbi.nlm.nih.gov/gene/3015), [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599), and [BRD9](https://www.ncbi.nlm.nih.gov/gene/65980) were each found to be regulating in 4 experiments.





The gene is expressed in various tissues, including the heart, muscle, and brain. Additionally, it is expressed in specific regions of the brain such as the amygdala, hypothalamus, and occipital lobe, as well as in the thyroid and pineal gland during both day and night.


The input data highlights several key pathways involved in cellular metabolism, including the citric acid cycle (TCA cycle), pyruvate metabolism, and respiratory electron transport. These pathways are integral to energy production and the regulation of metabolic processes within the cell.



The protein sequence analyzed has a GRAVY value of -0.046 (81.58th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of 0.112 (50.81st percentile). The median structural flexibility is 1.003 (53.18th percentile), with a predicted secondary structure composition of 36.72% helix (84.04th percentile), 33.69% sheet (49.85th percentile), and 26.13% turn (30.42nd percentile). The instability index is 41.13 (25.74th percentile), and the isoelectric point is 7.05 (50.97th percentile). The protein has a length of 463 amino acids (53.51st percentile) and a molecular weight of 50316.7 Da (52.05th percentile). For more details on the sequence analysis, refer to the [BioPython ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |