---
layout: post
title: UQCRB
date: 2025-01-17 16:55 CST
description: UQCRB description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7381) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7381  | UQCRB | ENSG00000156467 | 8q22.1 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is located in the [mitochondrial inner membrane](https://amigo.geneontology.org/amigo/term/GO:0005743) and the [mitochondrial respirasome](https://amigo.geneontology.org/amigo/term/GO:0005746). It is also part of the [mitochondrial respiratory chain complex III](https://amigo.geneontology.org/amigo/term/GO:0005750). The gene is involved in several processes, including [oxidative phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0006119), [mitochondrial electron transport, ubiquinol to cytochrome c](https://amigo.geneontology.org/amigo/term/GO:0006122), [aerobic respiration](https://amigo.geneontology.org/amigo/term/GO:0009060), and [cellular respiration](https://amigo.geneontology.org/amigo/term/GO:0045333).


The gene length is 27,761 base pairs (50.54% of all genes), the mature length is 776.5 base pairs, and the primary transcript length is 5,549 base pairs.


The gene UQCRB (NCBI ID: 7381) has been mentioned in [15 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22UQCRB%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1996 and the middle 50% of publications occurring between 2012 and 2017. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning UQCRB, ranked by their scientific influence, include studies that highlight the gene's role in various biological processes. The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Notable publications include "[Terpestacin inhibits tumor angiogenesis by targeting UQCRB of mitochondrial complex III and suppressing hypoxia-induced reactive oxygen species production and cellular oxygen sensing.](https://pubmed.ncbi.nlm.nih.gov/20145250)" (2010) (relative citation ratio: 2.33), "[A deletion in the human QP-C gene causes a complex III deficiency resulting in hypoglycaemia and lactic acidosis.](https://pubmed.ncbi.nlm.nih.gov/12709789)" (2003) (relative citation ratio: 1.99), "[MiR-4435 is an UQCRB-related circulating miRNA in human colorectal cancer.](https://pubmed.ncbi.nlm.nih.gov/32071343)" (2020) (relative citation ratio: 0.93), "[Downregulation of mitochondrial UQCRB inhibits cancer stem cell-like properties in glioblastoma.](https://pubmed.ncbi.nlm.nih.gov/29115404)" (2018) (relative citation ratio: 0.91), and "[A mutation in the mitochondrial protein UQCRB promotes angiogenesis through the generation of mitochondrial reactive oxygen species.](https://pubmed.ncbi.nlm.nih.gov/25446085)" (2014) (relative citation ratio: 0.8). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[UQCRB](https://www.proteinatlas.org/ENSG00000156467-UQCRB) is a gene implicated in primary mitochondrial disease, with evidence supporting its role at the protein level. The gene's RNA is detected in all tissues, and it is notably expressed in T-cells within the blood, where it is associated with the T-cell receptor. In various tissues, UQCRB is involved in cellular respiration and metabolism. Additionally, it is expressed in cardiomyocytes, where it plays a role in muscle contraction.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 5 experiments, [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) with 4 experiments, [SUPT5H](https://www.ncbi.nlm.nih.gov/gene/6829) with 4 experiments, [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 4 experiments, and [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.


The top transcription factors (TFs) identified in Perturb-seq experiments, based on data from GTRD's database of TF binding sites, include [2118](https://www.ncbi.nlm.nih.gov/gene/2118), [8243](https://www.ncbi.nlm.nih.gov/gene/8243), and [9968](https://www.ncbi.nlm.nih.gov/gene/9968), each shown to be regulating genes in one experiment.


The input data indicates associations with various disease conditions, including several types of cancer such as leukemia, lymphoma, and immune system cancer. Additionally, respiratory conditions like asthma, bronchial disease, and obstructive lung disease are noted. Other significant associations include hematopoietic system disease, immune system disease, and cardiovascular system disease.



The gene is expressed in various tissues, including the heart, muscle, and pituitary, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the pineal gland (both day and night), Burkitt's lymphoma (Raji), promyelocytic leukemia (HL-60), pituitary, and skeletal muscle.


The input data highlights several key biological processes, including metabolism, respiratory electron transport, ATP synthesis by chemiosmotic coupling, heat production by uncoupling proteins, and the citric acid (TCA) cycle. These processes are integral to cellular energy production and metabolic regulation.


The mouse ortholog gene 67530 and its human ortholog gene 7381 are associated with the phenotype of preweaning lethality with complete penetrance, observed in both sexes.


The protein sequence analyzed has a GRAVY value of -1.0495 (3.47th percentile), indicating a hydrophilic nature. It exhibits a charge of 1.615 at pH 7 (56.04th percentile) and a median structural flexibility of 1.007 (71.87th percentile). The protein's secondary structure is predicted to be predominantly helical (44.14%, 97.79th percentile), with a significant portion in sheet conformation (32.43%, 42.09th percentile) and a smaller fraction in turns (20.72%, 3.42nd percentile). The instability index is 35.96 (12.02nd percentile), and the isoelectric point is 8.73 (71.88th percentile). The protein has a length of 111 amino acids (3.69th percentile) and a molecular weight of 13530.28 Da (4.84th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |