---
layout: post
title: EEF1A1
date: 2025-01-17 16:55 CST
description: EEF1A1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1915) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1915  | EEF1A1 | ENSG00000156508 | 6q13  |



The gene enables [tRNA binding](https://amigo.geneontology.org/amigo/term/GO:0000049), [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [translation elongation factor activity](https://amigo.geneontology.org/amigo/term/GO:0003746), [GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0003924), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [GTP binding](https://amigo.geneontology.org/amigo/term/GO:0005525), [kinase binding](https://amigo.geneontology.org/amigo/term/GO:0019900), [protein kinase binding](https://amigo.geneontology.org/amigo/term/GO:0019901), and [molecular adaptor activity](https://amigo.geneontology.org/amigo/term/GO:0060090). It is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615), [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleolus](https://amigo.geneontology.org/amigo/term/GO:0005730), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [cortical actin cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0030864), [ruffle membrane](https://amigo.geneontology.org/amigo/term/GO:0032587), [secretory granule lumen](https://amigo.geneontology.org/amigo/term/GO:0034774), [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062), and [cytoplasmic side of lysosomal membrane](https://amigo.geneontology.org/amigo/term/GO:0098574). It is active in the [ribosome](https://amigo.geneontology.org/amigo/term/GO:0005840) and [cytosolic ribosome](https://amigo.geneontology.org/amigo/term/GO:0022626), and is part of the [eukaryotic translation elongation factor 1 complex](https://amigo.geneontology.org/amigo/term/GO:0005853). The gene is involved in [translation](https://amigo.geneontology.org/amigo/term/GO:0006412), [translational elongation](https://amigo.geneontology.org/amigo/term/GO:0006414), [positive regulation by host of viral genome replication](https://amigo.geneontology.org/amigo/term/GO:0044829), [cellular response to epidermal growth factor stimulus](https://amigo.geneontology.org/amigo/term/GO:0071364), [regulation of D-erythro-sphingosine kinase activity](https://amigo.geneontology.org/amigo/term/GO:1900022), and [regulation of chaperone-mediated autophagy](https://amigo.geneontology.org/amigo/term/GO:1904714).


The gene length is 39,698 base pairs (60.14% of all genes), the mature length is 1,660 base pairs, and the primary transcript length is 15,783 base pairs (45.43% of all genes).


The gene EEF1A1 (NCBI ID: 1915) has been mentioned in [310 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22EEF1A1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1973, and the middle 50% of publications range from 1987 to 2014.


The top 5 publications mentioning EEF1A1, ranked by their scientific influence, include "[Plitidepsin has potent preclinical efficacy against SARS-CoV-2 by targeting the host protein eEF1A.](https://pubmed.ncbi.nlm.nih.gov/33495306)" (2021) with a relative citation ratio of 19.85, "[The crystal structure of the ribosome bound to EF-Tu and aminoacyl-tRNA.](https://pubmed.ncbi.nlm.nih.gov/19833920)" (2009) with a relative citation ratio of 8.24, "[The many roles of the eukaryotic elongation factor 1 complex.](https://pubmed.ncbi.nlm.nih.gov/22555874)" (2012) with a relative citation ratio of 5.37, "[The eEF1A Proteins: At the Crossroads of Oncogenesis, Apoptosis, and Viral Infections.](https://pubmed.ncbi.nlm.nih.gov/25905039)" (2015) with a relative citation ratio of 4.69, and "[A novel 3'tRNA-derived fragment tRF-Val promotes proliferation and inhibits apoptosis by targeting EEF1A1 in gastric cancer.](https://pubmed.ncbi.nlm.nih.gov/35585048)" (2022) with a relative citation ratio of 4.65. The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[EEF1A1](https://www.proteinatlas.org/ENSG00000156508-EEF1A1) is a gene with molecular functions as an elongation factor and hydrolase. It is involved in cancer-related genes and has evidence at the protein level. The gene is detected in all RNA tissue distributions and is primarily located in the cytosol. It is part of various non-specific expression clusters, including Cluster 33 for blood expression related to ATP binding, and Cluster 18 for tissue expression related to the ribosome. Similar non-specific ribosome-related clusters are observed in brain, cell line, and single-cell expression.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), [AHR](https://www.ncbi.nlm.nih.gov/gene/196), [RELA](https://www.ncbi.nlm.nih.gov/gene/5970), [DPF2](https://www.ncbi.nlm.nih.gov/gene/5977), and [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885), each shown to be regulating in 5 experiments.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [FOXA1](https://www.ncbi.nlm.nih.gov/gene/6421) and [ITGA4](https://www.ncbi.nlm.nih.gov/gene/3276), each shown to be regulating genes in 2 experiments. Additionally, [ZNF787](https://www.ncbi.nlm.nih.gov/gene/58517), [GATA2](https://www.ncbi.nlm.nih.gov/gene/1054), [RAD51](https://www.ncbi.nlm.nih.gov/gene/6830), and [TMEM132A](https://www.ncbi.nlm.nih.gov/gene/30827) were each found to be regulating genes in 1 experiment.


The GWAS data indicates associations with several disease conditions, including artery disease, vascular disease, cardiovascular system disease, hypertension, and broader categories such as disease and disease of anatomical entity.



The gene is expressed in various tissues, including the ovary and cervix uteri. Additionally, it is expressed in the pineal gland during both day and night, CD4+ T cells, chronic myelogenous leukemia (K-562), promyelocytic leukemia (HL-60), and fetal thyroid.


The input data highlights several key biological processes, including metabolism of proteins, gene expression, and various stages of translation such as eukaryotic translation elongation and peptide chain elongation. Additionally, it covers cellular responses to stress, specifically focusing on HSF1 activation and the cellular response to heat stress.



The analyzed protein sequence has a GRAVY value of -0.257 (63.06th percentile), indicating a hydrophilic nature. It exhibits a charge of 11.36 at pH 7.0 (84.09th percentile) and a median structural flexibility of 1.005 (65.67th percentile). The protein's secondary structure is predicted to be 32.68% helix (56.73rd percentile), 34.63% sheet (56.33rd percentile), and 29.00% turn (55.93rd percentile). With an instability index of 27.03 (2.2nd percentile), the protein is considered stable. The isoelectric point is 9.10 (81.96th percentile), and the molecular weight is 50140.27 Da (51.82nd percentile). The sequence length is 462 amino acids (53.38th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |