---
layout: post
title: RPN1
date: 2025-01-17 16:55 CST
description: RPN1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6184) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6184  | RPN1 | ENSG00000163902 | 3q21.3 |



The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and contributes to [dolichyl-diphosphooligosaccharide-protein glycotransferase activity](https://amigo.geneontology.org/amigo/term/GO:0004579). It also enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is located in the [endoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0005783), [endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0005789), [rough endoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0005791), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), and [melanosome](https://amigo.geneontology.org/amigo/term/GO:0042470). It is involved in [protein N-linked glycosylation](https://amigo.geneontology.org/amigo/term/GO:0006487) and [protein N-linked glycosylation via asparagine](https://amigo.geneontology.org/amigo/term/GO:0018279), and is part of the [oligosaccharyltransferase complex](https://amigo.geneontology.org/amigo/term/GO:0008250).


The gene length is 17,999 base pairs (39.51% of all genes), the mature length is 2,816 base pairs, and the primary transcript length is 17,675 base pairs.


The gene RPN1 (NCBI ID: 6184) has been mentioned in [37 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RPN1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1992 and the middle 50% of publications occurring between 2007 and 2016. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning RPN1, ranked by their scientific influence, include "[Arabidopsis OST1 protein kinase mediates the regulation of stomatal aperture by abscisic acid and acts upstream of reactive oxygen species production.](https://pubmed.ncbi.nlm.nih.gov/12468729)" (2002) (relative citation ratio: 15.65), "[Protein phosphatases 2C regulate the activation of the Snf1-related kinase OST1 by abscisic acid in Arabidopsis.](https://pubmed.ncbi.nlm.nih.gov/19855047)" (2009) (relative citation ratio: 8.91), "[Ozone-triggered rapid stomatal response involves the production of reactive oxygen species, and is controlled by SLAC1 and OST1.](https://pubmed.ncbi.nlm.nih.gov/20128877)" (2010) (relative citation ratio: 5.28), "[OST1-mediated BTF3L phosphorylation positively regulates CBFs during plant cold responses.](https://pubmed.ncbi.nlm.nih.gov/29507081)" (2018) (relative citation ratio: 5.19), and "[Central functions of bicarbonate in S-type anion channel activation and OST1 protein kinase in CO2 signal transduction in guard cell.](https://pubmed.ncbi.nlm.nih.gov/21423149)" (2011) (relative citation ratio: 4.04). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RPN1](https://www.proteinatlas.org/ENSG00000163902-RPN1) is a cancer-related gene with evidence at the protein level. It is detected in all tissues and is primarily localized in the endoplasmic reticulum and cytosol. RPN1 is associated with various expression clusters, including Cluster 38 in blood (Monocytes & Neutrophils - Degranulation), Cluster 53 in tissues (Non-specific - Glycosylation), Cluster 21 in the brain (Non-specific - Metabolism), Cluster 63 in cell lines (Non-specific - Protein processing), and Cluster 61 in single cells (Plasma cells - Immunoglobulins & histones).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861) with 5 experiments, [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313) with 4 experiments, [EGR1](https://www.ncbi.nlm.nih.gov/gene/1958) with 4 experiments, [PPARG](https://www.ncbi.nlm.nih.gov/gene/5468) with 4 experiments, and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.





The gene is expressed in multiple tissue samples, including the thyroid and lung according to BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles indicate expression in the liver, BDCA4+ dendritic cells, prostate, lung, and pancreas.


The input data highlights various diseases and processes related to glycosylation and protein metabolism. Specifically, it includes diseases of glycosylation, such as asparagine N-linked glycosylation, and several conditions caused by defects in genes like ALG2, ALG3, ALG6, ALG8, ALG9, DPAGT1, MAN1B1, MGAT2, MOGS, MPDU1, ALG1, ALG12, ALG11, ALG14, and RFT1. These defects lead to various forms of congenital disorders of glycosylation (CDG) and other conditions like congenital myasthenic syndrome. Additionally, the data covers broader biological processes including post-translational protein modification, metabolism of proteins, gene expression, SRP-dependent cotranslational protein targeting to membrane, and translation.



The analyzed protein sequence has a GRAVY value of -0.2082 (68.29th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -9.85 (22.76th percentile). The median structural flexibility is 0.9994 (38.64th percentile), with a propensity for helix, sheet, and turn structures of 33.28% (61.8th percentile), 41.19% (89.1th percentile), and 25.21% (23.03rd percentile), respectively. The instability index is 46.54 (44.91st percentile), and the isoelectric point is 5.96 (29.45th percentile). The protein is 607 amino acids long (69.12th percentile) with a molecular weight of 68568.54 Da (69.67th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |