---
layout: post
title: HSPD1
date: 2024-12-08 23:57 CST
description: HSPD1 description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3329) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3329  | HSPD1 | ENSG00000144381 | 2q33.1 |



The gene length is 12,823 base pairs (31.26% of all genes), the mature length is 2,838 base pairs, and the primary transcript length is 10,324 base pairs.


The gene HSPD1 (NCBI ID: 3329) has been the subject of [1,146 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22HSPD1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1988, and the middle 50% of publications occurred between 1999 and 2014.


The top five publications mentioning HSPD1, ranked by their scientific influence, include "[The Hsp70 and Hsp60 chaperone machines.](https://pubmed.ncbi.nlm.nih.gov/9476895)" (1998) (relative citation ratio: 45.84), "[Cutting edge: heat shock protein 60 is a putative endogenous ligand of the toll-like receptor-4 complex.](https://pubmed.ncbi.nlm.nih.gov/10623794)" (2000) (relative citation ratio: 24.21), "[Chaperonin-mediated protein folding at the surface of groEL through a 'molten globule'-like intermediate.](https://pubmed.ncbi.nlm.nih.gov/1676490)" (1991) (relative citation ratio: 18.15), "[Protein folding in mitochondria requires complex formation with hsp60 and ATP hydrolysis.](https://pubmed.ncbi.nlm.nih.gov/2528694)" (1989) (relative citation ratio: 11.44), and "[Association of serum antibodies to heat-shock protein 65 with carotid atherosclerosis.](https://pubmed.ncbi.nlm.nih.gov/8093914)" (1993) (relative citation ratio: 10.39). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885) with 6 experiments, [BMAL1](https://www.ncbi.nlm.nih.gov/gene/406) with 5 experiments, [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930) with 5 experiments, [MAF](https://www.ncbi.nlm.nih.gov/gene/4094) with 5 experiments, and [SP1](https://www.ncbi.nlm.nih.gov/gene/6667) with 5 experiments.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [CTCF](https://www.ncbi.nlm.nih.gov/gene/6829) and [ZNF143](https://www.ncbi.nlm.nih.gov/gene/8243), each shown to regulate genes in four experiments. Additionally, [RAD21](https://www.ncbi.nlm.nih.gov/gene/6872) and [SMC3](https://www.ncbi.nlm.nih.gov/gene/8861) were found to regulate genes in three experiments each. [SMC1A](https://www.ncbi.nlm.nih.gov/gene/6830) was identified in one experiment, while [NRF1](https://www.ncbi.nlm.nih.gov/gene/2623), [RAD50](https://www.ncbi.nlm.nih.gov/gene/9968), and [HNF4A](https://www.ncbi.nlm.nih.gov/gene/9126) were each found in two experiments.





The mouse ortholog gene 15510 and its human ortholog gene 3329 are associated with several phenotypes. Both sexes exhibit abnormal coat/hair pigmentation and abnormal vocalization. Additionally, there is embryonic lethality prior to organogenesis, though this is not observed in both sexes. Furthermore, both sexes display preweaning lethality with complete penetrance.


The analyzed protein sequence has a GRAVY value of -0.076 (79.67th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -5.17 (33.33rd percentile). The median structural flexibility is 1.004 (60.25th percentile), with a predicted secondary structure composition of 38.22% helix (89.38th percentile), 34.90% sheet (58.33rd percentile), and 27.40% turn (41.79th percentile). The instability index is 30.19 (4.16th percentile), and the isoelectric point is 5.70 (22.9th percentile). The protein is 573 amino acids long (66.09th percentile) with a molecular weight of 61053.93 Da (63.76th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |