---
layout: post
title: MYL12A
date: 2024-12-08 23:57 CST
description: MYL12A description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10627) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10627  | MYL12A | ENSG00000101608 | 18p11.31 |



The gene length is 55,663 base pairs (69.22% of all genes), the mature length is 910 base pairs (18.62% of all genes), and the primary transcript length is 53,209.5 base pairs (76.81% of all genes).


The gene MYL12A, with NCBI ID 10627, has been mentioned in [1 publication](https://pubmed.ncbi.nlm.nih.gov/?term=%22MYL12A%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 2001, and the middle 50% of publications occurred from 2001 to 2001.


The gene MYL12A has been mentioned in several publications, with the top cited work being "[Steady-state kinetics of MgATP splitting by native myosin RLC-free subfragment 1.](https://pubmed.ncbi.nlm.nih.gov/11325532)" (2001) (relative citation ratio: 0.0). The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified in terms of the number of CHIP-seq experiments include [PGR](https://www.ncbi.nlm.nih.gov/gene/5241), [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), [RELA](https://www.ncbi.nlm.nih.gov/gene/5970), [IKZF1](https://www.ncbi.nlm.nih.gov/gene/10320), and [E2F8](https://www.ncbi.nlm.nih.gov/gene/79733), each shown to be regulating in 6 experiments.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [CTCF](https://www.ncbi.nlm.nih.gov/gene/6829) with 3 experiments, [CUX1](https://www.ncbi.nlm.nih.gov/gene/6830) with 1 experiment, [ZNF143](https://www.ncbi.nlm.nih.gov/gene/8861) with 2 experiments, and [ZNF263](https://www.ncbi.nlm.nih.gov/gene/9968) with 2 experiments. These TFs were found to regulate gene expression in multiple Perturb-seq experiments, highlighting their significant roles in transcriptional regulation.





The mouse ortholog gene 67938 and its human ortholog 10627 are associated with several phenotypic abnormalities observed in both sexes, including decreased circulating fructosamine level, decreased circulating serum albumin level, decreased hematocrit, decreased hemoglobin content, decreased mean corpuscular hemoglobin, and decreased mean corpuscular volume.


The analyzed protein sequence exhibits a GRAVY value of -0.803 (11.57th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -11.79 (19.78th percentile), and the median structural flexibility is 1.012 (86.1st percentile). The protein has a helix affinity of 0.362 (81.53rd percentile), a sheet affinity of 0.315 (36.59th percentile), and a turn affinity of 0.286 (52.55th percentile). The instability index is 31.49 (5.32nd percentile), with an isoelectric point of 4.69 (3rd percentile). The sequence length is 171.5 amino acids (10.58th percentile), and the molecular weight is 19786.43 Da (11.23rd percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |