---
layout: post
title: PPFIA1
date: 2025-01-17 16:55 CST
description: PPFIA1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8500) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8500  | PPFIA1 | ENSG00000131626 | 11q13.3 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925), [axon](https://amigo.geneontology.org/amigo/term/GO:0030424), [presynaptic active zone](https://amigo.geneontology.org/amigo/term/GO:0048786), [spine apparatus](https://amigo.geneontology.org/amigo/term/GO:0097444), and [Schaffer collateral - CA1 synapse](https://amigo.geneontology.org/amigo/term/GO:0098685). It is part of a [protein-containing complex](https://amigo.geneontology.org/amigo/term/GO:0032991) and is involved in [cell-matrix adhesion](https://amigo.geneontology.org/amigo/term/GO:0007160), [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), [synapse organization](https://amigo.geneontology.org/amigo/term/GO:0050808), and [regulation of postsynaptic membrane neurotransmitter receptor levels](https://amigo.geneontology.org/amigo/term/GO:0099072). Additionally, it acts upstream of or within [negative regulation of stress fiber assembly](https://amigo.geneontology.org/amigo/term/GO:0051497) and [negative regulation of protein localization to plasma membrane](https://amigo.geneontology.org/amigo/term/GO:1903077).


The gene length is 27,701 base pairs (50.48% of all genes), the mature length is 2,833 base pairs (69.93% of all genes), and the primary transcript length is 22,772 base pairs (55.17% of all genes).


The gene PPFIA1 (NCBI ID: 8500) has been mentioned in [26 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PPFIA1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1995 and the middle 50% of publications occurring between 2010 and 2017. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning PPFIA1, ranked by their scientific influence, include "[The LAR/PTP delta/PTP sigma subfamily of transmembrane protein-tyrosine-phosphatases: multiple human LAR, PTP delta, and PTP sigma isoforms are expressed in a tissue-specific manner and associate with the LAR-interacting protein LIP.1.](https://pubmed.ncbi.nlm.nih.gov/8524829)" (1995) (relative citation ratio: 3.23), "[PPFIA1 drives active α5β1 integrin recycling and controls fibronectin fibrillogenesis and vascular morphogenesis.](https://pubmed.ncbi.nlm.nih.gov/27876801)" (2016) (relative citation ratio: 2.17), "[Inhibitor of growth 4 suppresses cell spreading and cell migration by interacting with a novel binding partner, liprin alpha1.](https://pubmed.ncbi.nlm.nih.gov/17363573)" (2007) (relative citation ratio: 2.09), "[Genome wide association identifies PPFIA1 as a candidate gene for acute lung injury risk following major trauma.](https://pubmed.ncbi.nlm.nih.gov/22295056)" (2012) (relative citation ratio: 1.71), and "[Liprin-α1, ERC1 and LL5 define polarized and dynamic structures that are implicated in cell migration.](https://pubmed.ncbi.nlm.nih.gov/24982445)" (2014) (relative citation ratio: 1.53). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PPFIA1](https://www.proteinatlas.org/ENSG00000131626-PPFIA1) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized to focal adhesion sites and the cytosol. The gene is expressed in various clusters, including Cluster 20 for blood expression, Cluster 19 for tissue expression, Cluster 15 for brain expression, Cluster 24 for cell line expression, and Cluster 21 for single cell expression, which is associated with early spermatids and spermatogenesis.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [RELA](https://www.ncbi.nlm.nih.gov/gene/5970), [TAL1](https://www.ncbi.nlm.nih.gov/gene/6886), [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025), and [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613). Each of these TFs, except MYCN, was shown to be regulating in 6 experiments, while MYCN was found in 5 experiments.



The GWAS data indicates associations with several disease conditions, including lower respiratory tract disease, lung disease, respiratory system disease, kidney disease, and urinary system disease.



The gene is expressed in adipose tissue and pituitary, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in CD34+ cells, CD33+ myeloid cells, colorectal adenocarcinoma, and various testis tissues, including interstitial, Leydig cells, seminiferous tubules, and germ cells.


The input data highlights several pathways within the neuronal system, including transmission across chemical synapses and the release cycles of various neurotransmitters such as acetylcholine, dopamine, serotonin, norepinephrine, and glutamate. These pathways collectively contribute to the neurotransmitter release cycle, a critical process in neural communication.


The mouse ortholog gene 233977 and its human ortholog 8500 are associated with several phenotypic abnormalities, including abnormal coat/hair pigmentation, abnormal embryo size, abnormal vocalization, and preweaning lethality with complete penetrance. These phenotypes are observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.779 (12.72nd percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -21.19 (10.35th percentile), and the median structural flexibility is 1.012 (86.09th percentile). The protein has a high affinity for helix structures (38.94%, 91.31st percentile), with lower affinities for sheet (27.62%, 16.94th percentile) and turn (27.70%, 44.65th percentile) structures. The instability index is 57.54 (78.22nd percentile), and the isoelectric point is 5.91 (28.3rd percentile). The protein is 1202 amino acids long (91.73rd percentile) with a molecular weight of 135777.21 Da (92.14th percentile). For more details on the sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |