---
layout: post
title: GMFB
date: 2025-01-17 16:55 CST
description: GMFB description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2764) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2764  | GMFB | ENSG00000197045 | 14q22.2 |



The gene enables [actin binding](https://amigo.geneontology.org/amigo/term/GO:0003779) and [protein kinase inhibitor activity](https://amigo.geneontology.org/amigo/term/GO:0004860), and is involved in [protein phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0006468), [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), and [nervous system development](https://amigo.geneontology.org/amigo/term/GO:0007399). Additionally, it enables [enzyme activator activity](https://amigo.geneontology.org/amigo/term/GO:0008047) and [growth factor activity](https://amigo.geneontology.org/amigo/term/GO:0008083), and is involved in [negative regulation of Arp2/3 complex-mediated actin nucleation](https://amigo.geneontology.org/amigo/term/GO:0034316) and [actin filament debranching](https://amigo.geneontology.org/amigo/term/GO:0071846). Furthermore, it enables [Arp2/3 complex binding](https://amigo.geneontology.org/amigo/term/GO:0071933).


The gene length is 6,839 base pairs (18.63% of all genes), the mature length is 1,608 base pairs (41.36% of all genes), and the primary transcript length is 3,984 base pairs (14.87% of all genes).


The gene GMFB (NCBI ID: 2764) has been mentioned in [49 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22GMFB%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1965 and the middle 50% of publications occurring between 1991 and 2017. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning GMFB, ranked by their scientific influence, include "[Glia maturation factor-Î² induces ferroptosis by impairing chaperone-mediated autophagic degradation of ACSL4 in early diabetic retinopathy.](https://pubmed.ncbi.nlm.nih.gov/35325805)" (2022) (relative citation ratio: 8.09), "[Co-Localization of Glia Maturation Factor with NLRP3 Inflammasome and Autophagosome Markers in Human Alzheimer's Disease Brain.](https://pubmed.ncbi.nlm.nih.gov/28984607)" (2017) (relative citation ratio: 2.83), "[Molecular cloning and expression of biologically active human glia maturation factor-beta.](https://pubmed.ncbi.nlm.nih.gov/1712830)" (1991) (relative citation ratio: 1.93), "[GMF is a cofilin homolog that binds Arp2/3 complex to stimulate filament debranching and inhibit actin nucleation.](https://pubmed.ncbi.nlm.nih.gov/20362448)" (2010) (relative citation ratio: 1.77), and "[GMF severs actin-Arp2/3 complex branch junctions by a cofilin-like mechanism.](https://pubmed.ncbi.nlm.nih.gov/23727094)" (2013) (relative citation ratio: 1.46). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[GMFB](https://www.proteinatlas.org/ENSG00000197045-GMFB) is a growth factor with evidence at the protein level. It is detected in all RNA tissues and is primarily localized in the nucleoplasm, nuclear bodies, and cytosol. GMFB is expressed in monocytes and neutrophils, contributing to the innate immune response, and is also found in oligodendrocytes, where it plays a role in myelination. Additionally, GMFB is part of the ribosome in the brain and is involved in gene expression regulation in various cell lines.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), [TP53](https://www.ncbi.nlm.nih.gov/gene/7157), [TP63](https://www.ncbi.nlm.nih.gov/gene/8626), [MYOD1](https://www.ncbi.nlm.nih.gov/gene/4654), and [NCOR1](https://www.ncbi.nlm.nih.gov/gene/9611), each shown to be regulating in 5 experiments.





The gene is expressed in various neural tissues, including the nerve, brain, amygdala, occipital lobe, and olfactory bulb. Additionally, it is found in bronchial epithelial cells, colon, and small intestine, indicating a broad range of tissue-specific expressions.




The analyzed protein sequence has a GRAVY value of -0.52887323943662 (32.01 percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -5.314086460385521 (32.82 percentile), and the median structural flexibility is 1.0133333333333334 (88.83 percentile). The protein has a high affinity for helix (37.32%, 86.43 percentile) and sheet (35.92%, 65.64 percentile) structures, with a lower affinity for turns (23.24%, 11.05 percentile). The instability index is 47.597183098591536 (48.7 percentile), and the isoelectric point is 5.192776298522948 (11.39 percentile). The protein sequence is 142 amino acids long (7.15 percentile) with a molecular weight of 16712.9782 Da (8.01 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |