---
layout: post
title: ARPP19
date: 2025-01-17 16:55 CST
description: ARPP19 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10776) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10776  | ARPP19 | ENSG00000128989 | 15q21.2 |



The gene is involved in the [G2/M transition of mitotic cell cycle](https://amigo.geneontology.org/amigo/term/GO:0000086) and the [mitotic cell cycle](https://amigo.geneontology.org/amigo/term/GO:0000278). It enables [protein phosphatase inhibitor activity](https://amigo.geneontology.org/amigo/term/GO:0004864), [signaling receptor binding](https://amigo.geneontology.org/amigo/term/GO:0005102), and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654) and is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737). Additionally, it enables [potassium channel regulator activity](https://amigo.geneontology.org/amigo/term/GO:0015459), [phosphatase inhibitor activity](https://amigo.geneontology.org/amigo/term/GO:0019212), and [protein phosphatase regulator activity](https://amigo.geneontology.org/amigo/term/GO:0019888). The gene is involved in the [negative regulation of protein dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0035308), [positive regulation of gluconeogenesis](https://amigo.geneontology.org/amigo/term/GO:0045722), and [positive regulation of glucose import](https://amigo.geneontology.org/amigo/term/GO:0046326). It also plays a role in [cell division](https://amigo.geneontology.org/amigo/term/GO:0051301) and enables [protein phosphatase 2A binding](https://amigo.geneontology.org/amigo/term/GO:0051721).


The gene length is 13,765 base pairs (32.99% of all genes), the mature length is 574 base pairs (3.93% of all genes), and the primary transcript length is 6,380 base pairs (23.26% of all genes).


ARPP19 has been mentioned in [20 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ARPP19%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1994, and the middle 50% of publications occurred between 2014 and 2020.


The top 5 publications mentioning ARPP19, ranked by their relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)), include "[MicroRNA-320a sensitizes tamoxifen-resistant breast cancer cells to tamoxifen by targeting ARPP-19 and ERRÎ³.](https://pubmed.ncbi.nlm.nih.gov/25736597)" (2015) (relative citation ratio: 2.6), "[LncRNA IGFL2-AS1 functions as a ceRNA in regulating ARPP19 through competitive binding to miR-802 in gastric cancer.](https://pubmed.ncbi.nlm.nih.gov/31943339)" (2020) (relative citation ratio: 1.7), "[ENSA and ARPP19 differentially control cell cycle progression and development.](https://pubmed.ncbi.nlm.nih.gov/30626720)" (2019) (relative citation ratio: 1.23), "[MiR-26a inhibits thyroid cancer cell proliferation by targeting ARPP19.](https://pubmed.ncbi.nlm.nih.gov/30034940)" (2018) (relative citation ratio: 1.21), and "[The Cell Cycle Checkpoint System MAST(L)-ENSA/ARPP19-PP2A is Targeted by cAMP/PKA and cGMP/PKG in Anucleate Human Platelets.](https://pubmed.ncbi.nlm.nih.gov/32085646)" (2020) (relative citation ratio: 1.16). The relative citation ratio measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ARPP19](https://www.proteinatlas.org/ENSG00000128989-ARPP19) is a protein phosphatase inhibitor with evidence at the protein level. It is detected in all tissues and is primarily localized in the nucleoli, vesicles, basal body, and cytosol. ARPP19 is associated with Cluster 45 in blood expression, which is related to eosinophils with unknown function. In tissue expression, it belongs to Cluster 33, indicating a role in neuronal signaling in the brain. Additionally, ARPP19 is part of Cluster 33 in brain expression, suggesting mixed functions in neurons, and Cluster 58 in cell line expression, which is involved in nuclear processes. In single-cell expression, it is found in Cluster 63, associated with late spermatids and spermatogenesis.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [TP53](https://www.ncbi.nlm.nih.gov/gene/7157), [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), [TAL1](https://www.ncbi.nlm.nih.gov/gene/6886), [KLF4](https://www.ncbi.nlm.nih.gov/gene/9314), and [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313), each shown to be regulating in 5 experiments.



The GWAS data indicates associations with several disease conditions, including artery disease, vascular disease, cardiovascular system disease, hypertension, and broader categories such as disease and disease of anatomical entity.



The gene is expressed in various tissues, including muscle and brain. According to the GTEx Tissue Gene Expression Profiles, it is also expressed in specific regions of the brain such as the Caudate nucleus, Subthalamic nucleus, Temporal lobe, Medulla oblongata, Parietal lobe, Occipital lobe, and Pons.


The proteins are involved in various stages of the cell cycle, including mitotic prophase, the broader cell cycle, and specifically the mitotic phase. Additionally, they play a role in facilitating mitotic progression through the MASTL pathway.



The analyzed protein sequence has a GRAVY value of -1.083 (2.91 percentile), indicating a hydrophilic nature. It exhibits a charge of 2.692 at pH 7.0 (60.16 percentile) and a median structural flexibility of 1.026 (98.73 percentile). The protein's secondary structure is predicted to be predominantly helical (45.54%, 98.57 percentile), with lesser proportions in sheet (20.54%, 2.44 percentile) and turn (29.46%, 59.76 percentile) conformations. The instability index is 59.12 (81.4 percentile), and the isoelectric point is 9.07 (81.18 percentile). The protein has a length of 112 amino acids (3.76 percentile) and a molecular weight of 12322.83 Da (3.72 percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |