---
layout: post
title: ABCG2
date: 2025-01-17 16:55 CST
description: ABCG2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9429) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9429  | ABCG2 | ENSG00000118777 | 4q22.1 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524), and is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654) and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). It is involved in [lipid transport](https://amigo.geneontology.org/amigo/term/GO:0006869) and [organic anion transport](https://amigo.geneontology.org/amigo/term/GO:0015711), and enables various transporter activities, including [organic anion transmembrane transporter activity](https://amigo.geneontology.org/amigo/term/GO:0008514), [ABC-type xenobiotic transporter activity](https://amigo.geneontology.org/amigo/term/GO:0008559), [urate transmembrane transporter activity](https://amigo.geneontology.org/amigo/term/GO:0015143), [biotin transmembrane transporter activity](https://amigo.geneontology.org/amigo/term/GO:0015225), [efflux transmembrane transporter activity](https://amigo.geneontology.org/amigo/term/GO:0015562), and [riboflavin transmembrane transporter activity](https://amigo.geneontology.org/amigo/term/GO:0032217). Additionally, it is involved in processes such as [urate transport](https://amigo.geneontology.org/amigo/term/GO:0015747), [biotin transport](https://amigo.geneontology.org/amigo/term/GO:0015878), [riboflavin transport](https://amigo.geneontology.org/amigo/term/GO:0032218), [transmembrane transport](https://amigo.geneontology.org/amigo/term/GO:0055085), [transepithelial transport](https://amigo.geneontology.org/amigo/term/GO:0070633), [renal urate salt excretion](https://amigo.geneontology.org/amigo/term/GO:0097744), [export across plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0140115), [transport across blood-brain barrier](https://amigo.geneontology.org/amigo/term/GO:0150104), [cellular detoxification](https://amigo.geneontology.org/amigo/term/GO:1990748), and [xenobiotic transport across blood-brain barrier](https://amigo.geneontology.org/amigo/term/GO:1990962). The gene is also located in the [apical plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0016324), [brush border membrane](https://amigo.geneontology.org/amigo/term/GO:0031526), [mitochondrial membrane](https://amigo.geneontology.org/amigo/term/GO:0031966), [membrane raft](https://amigo.geneontology.org/amigo/term/GO:0045121), and the [external side of apical plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0098591). Furthermore, it enables [ATP hydrolysis activity](https://amigo.geneontology.org/amigo/term/GO:0016887), [ATPase-coupled transmembrane transporter activity](https://amigo.geneontology.org/amigo/term/GO:0042626), [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802), [protein homodimerization activity](https://amigo.geneontology.org/amigo/term/GO:0042803), and [xenobiotic transmembrane transporter activity](https://amigo.geneontology.org/amigo/term/GO:0042910).


The gene length is 31,350 base pairs (53.89% of all genes), the mature length is 3,215 base pairs, and the primary transcript length is 31,299 base pairs.


The gene ABCG2, with NCBI ID 9429, has been mentioned in [1,500 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ABCG2%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication mentioning this gene is 1976, and the middle 50% of publications occurred between 2009 and 2018.


The top 5 publications mentioning ABCG2, ranked by their scientific influence, include "[The Clinical Pharmacogenetics Implementation Consortium Guideline for SLCO1B1, ABCG2, and CYP2C9 genotypes and Statin-Associated Musculoskeletal Symptoms.](https://pubmed.ncbi.nlm.nih.gov/35152405)" (2022) (relative citation ratio: 25.71), "[Subcellular localization and distribution of the breast cancer resistance protein transporter in normal human tissues.](https://pubmed.ncbi.nlm.nih.gov/11309308)" (2001) (relative citation ratio: 21.83), "[A gene encoding a liver-specific ABC transporter is mutated in progressive familial intrahepatic cholestasis.](https://pubmed.ncbi.nlm.nih.gov/9806540)" (1998) (relative citation ratio: 17.2), "[A human placenta-specific ATP-binding cassette gene (ABCP) on chromosome 4q22 that is involved in multidrug resistance.](https://pubmed.ncbi.nlm.nih.gov/9850061)" (1998) (relative citation ratio: 16.03), and "[PTEN/PI3K/Akt pathway regulates the side population phenotype and ABCG2 activity in glioma tumor stem-like cells.](https://pubmed.ncbi.nlm.nih.gov/19265662)" (2009) (relative citation ratio: 14.34). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ABCG2](https://www.proteinatlas.org/ENSG00000118777-ABCG2) is a translocase involved in cancer-related genes, with evidence at the protein level. It is detected in many tissues and is localized in the nucleoplasm and plasma membrane. ABCG2 is expressed in NK-cells of unknown function, small intestine absorption cells, endothelial cells of the vasculature, squamous epithelial cells involved in keratinization, and basal prostatic cells involved in lipid metabolism.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313) with 6 experiments, [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) with 5 experiments, [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 5 experiments, [EZH2](https://www.ncbi.nlm.nih.gov/gene/2146) with 5 experiments, and [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with a wide range of disease conditions, including infectious diseases such as viral infectious disease and human immunodeficiency virus infectious disease. Additionally, the data highlights associations with hematopoietic system disease, immune system disease, and various cardiovascular conditions such as coronary artery disease, artery disease, vascular disease, and cardiovascular system disease. Furthermore, the data includes associations with kidney disease, urinary system disease, arteriosclerosis, atherosclerosis, and several musculoskeletal conditions such as bone inflammation disease, arthritis, bone disease, connective tissue disease, and gout. The data also points to associations with gastrointestinal system disease, myocardial infarction, and dental conditions including dental caries, teeth hard tissue disease, mouth disease, and tooth disease.


ABCG2 has been associated with various traits and diseases through genome-wide association studies. Notably, it has been linked to [3-bromo-5-chloro-2,6-dihydroxybenzoic acid levels](https://pubmed.ncbi.nlm.nih.gov/36635386), which maps to the MeSH term [Acids](https://meshb.nlm.nih.gov/record/ui?ui=D000143) using the BioWordVec method. Additionally, ABCG2 is associated with [4-hydroxychlorothalonil levels](https://pubmed.ncbi.nlm.nih.gov/36357675), mapped to the MeSH term [Methoxychlor](https://meshb.nlm.nih.gov/record/ui?ui=D008731), and [Caffeine consumption from coffee or tea](https://pubmed.ncbi.nlm.nih.gov/33287642), mapped to the MeSH term [Coffee](https://meshb.nlm.nih.gov/record/ui?ui=D003069). A significant association is also seen with [Gout](https://pubmed.ncbi.nlm.nih.gov/21983786), directly mapped to the MeSH term [Gout](https://meshb.nlm.nih.gov/record/ui?ui=D006073). Other associations include [Gout vs asymptomatic hyperuricemia](https://pubmed.ncbi.nlm.nih.gov/33832965), mapped to [Hyperuricemia](https://meshb.nlm.nih.gov/record/ui?ui=D033461), and [Hyperuricemia](https://pubmed.ncbi.nlm.nih.gov/29124443), directly mapped to the MeSH term [Hyperuricemia](https://meshb.nlm.nih.gov/record/ui?ui=D033461). Furthermore, ABCG2 is linked to [Plasma sulfate of piperine metabolite C18H21NO3 (1)* levels in chronic kidney disease](https://pubmed.ncbi.nlm.nih.gov/37277652), mapped to the MeSH term [Plasma](https://meshb.nlm.nih.gov/record/ui?ui=D010949), and [Serum urate levels in chronic kidney disease](https://pubmed.ncbi.nlm.nih.gov/30181573), mapped to the MeSH term [Kidney](https://meshb.nlm.nih.gov/record/ui?ui=D007668). These associations highlight the diverse roles of ABCG2 in various biological processes and diseases.


The gene is expressed in various tissues, including the prefrontal cortex, CD71+ early erythroid cells, hypothalamus, occipital lobe, placenta, small intestine, and uterus.


The input data highlights several key biological processes, including metabolism, iron uptake and transport, abacavir transmembrane transport, abacavir transport and metabolism, and the transmembrane transport of small molecules. These pathways are integral to cellular function and homeostasis, encompassing both general metabolic activities and specialized transport mechanisms.


The mouse ortholog gene 26357 and its human ortholog gene 9429 are associated with the phenotypes of 'fused cornea and lens' and 'increased circulating HDL cholesterol level', both of which are observed in both sexes.


The protein sequence analyzed has a GRAVY value of 0.195 (90.38th percentile), indicating a relatively hydrophobic nature. It exhibits a charge of 11.3 at pH 7.0 (83.97th percentile) and a median structural flexibility of 0.992 (14.27th percentile). The protein's secondary structure is predicted to be 32.5% helix (55.24th percentile), 41.5% sheet (89.81st percentile), and 27.9% turn (46.67th percentile). The instability index is 29.3 (3.44th percentile), suggesting a stable protein. The isoelectric point is 8.91 (76.73rd percentile), with a length of 655 amino acids (72.48th percentile) and a molecular weight of 72313.3 Da (72.03rd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |