---
layout: post
title: RBM7
date: 2025-01-17 16:55 CST
description: RBM7 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10179) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10179  | RBM7 | ENSG00000076053 | 11q23.2 |



The gene is involved in the [regulation of alternative mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0000381) and the [snRNA catabolic process](https://amigo.geneontology.org/amigo/term/GO:0016069), and plays a role in the [meiotic cell cycle](https://amigo.geneontology.org/amigo/term/GO:0051321). It enables functions such as [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [single-stranded RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003727), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [snRNA binding](https://amigo.geneontology.org/amigo/term/GO:0017069), [14-3-3 protein binding](https://amigo.geneontology.org/amigo/term/GO:0071889), and [pre-mRNA intronic binding](https://amigo.geneontology.org/amigo/term/GO:0097157). The gene is active and located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), but not in the [nucleolus](https://amigo.geneontology.org/amigo/term/GO:0005730).


The gene length is 66,393 base pairs (73.43rd percentile), the mature length is 3,465 base pairs (78.43rd percentile), and the primary transcript length is 66,263 base pairs (81.36th percentile).


The gene RBM7 (NCBI ID: 10179) has been mentioned in [10 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RBM7%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning this gene was in 2015, with the middle 50% of publications occurring between 2016 and 2019.


The top 5 publications mentioning RBM7, ranked by their scientific influence, include "[P-TEFb Activation by RBM7 Shapes a Pro-survival Transcriptional Response to Genotoxic Stress.](https://pubmed.ncbi.nlm.nih.gov/30824372)" (2019) (relative citation ratio: 2.26), "[Dysregulated Expression of the Nuclear Exosome Targeting Complex Component Rbm7 in Nonhematopoietic Cells Licenses the Development of Fibrosis.](https://pubmed.ncbi.nlm.nih.gov/32187520)" (2020) (relative citation ratio: 1.61), "[RBM7 subunit of the NEXT complex binds U-rich sequences and targets 3'-end extended forms of snRNAs.](https://pubmed.ncbi.nlm.nih.gov/25852104)" (2015) (relative citation ratio: 1.16), "[Oncogenic action of the exosome cofactor RBM7 by stabilization of CDK1 mRNA in breast cancer.](https://pubmed.ncbi.nlm.nih.gov/33145401)" (2020) (relative citation ratio: 1.16), and "[Altered RNA metabolism due to a homozygous RBM7 mutation in a patient with spinal motor neuropathy.](https://pubmed.ncbi.nlm.nih.gov/27193168)" (2016) (relative citation ratio: 1.03). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RBM7](https://www.proteinatlas.org/ENSG00000076053-RBM7) is an RNA-binding protein with evidence at the protein level, implicated in disease variants. It is detected in all tissues and primarily localized in the nucleoplasm. RBM7 is expressed in neutrophils with unknown function and is associated with cell proliferation in bone marrow. In the brain, it is involved in non-specific transcription, while in single cells, it is linked to the innate immune response in monocytes.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313) with 6 experiments, [PGR](https://www.ncbi.nlm.nih.gov/gene/5241) with 5 experiments, [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) with 5 experiments, [AFF4](https://www.ncbi.nlm.nih.gov/gene/27125) with 5 experiments, and [TP53](https://www.ncbi.nlm.nih.gov/gene/7157) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with several disease conditions, including coronary artery disease, artery disease, vascular disease, cardiovascular system disease, hypertension, and broader categories such as disease and disease of anatomical entity.



The gene is expressed in various tissues, including nerve, skin, and lung, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles show expression in Whole Blood, Smooth Muscle, Bronchial Epithelial Cells, Fetal Liver, Appendix, and Trigeminal Ganglion.




The protein sequence analyzed has a GRAVY value of -1.052 (3.42 percentile), indicating a hydrophilic nature. It exhibits a charge of 8.74 at pH 7.0 (78.69 percentile) and a median structural flexibility of 1.01 (81.06 percentile). The protein's secondary structure is predicted to be 21.05% helix (1.5 percentile), 25.94% sheet (11.15 percentile), and 37.59% turn (94.72 percentile). The instability index is 54.94 (72.0 percentile), with an isoelectric point of 9.56 (91.95 percentile). The protein has a length of 266 amino acids (23.9 percentile) and a molecular weight of 30503.16 Da (24.97 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |