---
layout: post
title: SRSF1
date: 2025-01-17 16:55 CST
description: SRSF1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6426) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6426  | SRSF1 | ENSG00000136450 | 17q22 |



The gene is involved in [alternative mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0000380), [mRNA 5'-splice site recognition](https://amigo.geneontology.org/amigo/term/GO:0000395), and [mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0000398). It enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [mRNA binding](https://amigo.geneontology.org/amigo/term/GO:0003729), and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), and [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737). Additionally, it is involved in [mRNA splice site recognition](https://amigo.geneontology.org/amigo/term/GO:0006376), [mRNA processing](https://amigo.geneontology.org/amigo/term/GO:0006397), and [positive regulation of RNA splicing](https://amigo.geneontology.org/amigo/term/GO:0033120). The gene is active in and located in the [nuclear speck](https://amigo.geneontology.org/amigo/term/GO:0016607), and it is part of the [exon-exon junction complex](https://amigo.geneontology.org/amigo/term/GO:0035145) and the [catalytic step 2 spliceosome](https://amigo.geneontology.org/amigo/term/GO:0071013). Furthermore, it enables [protein kinase B binding](https://amigo.geneontology.org/amigo/term/GO:0043422) and [DNA topoisomerase binding](https://amigo.geneontology.org/amigo/term/GO:0044547). The gene is also involved in the [regulation of RNA splicing](https://amigo.geneontology.org/amigo/term/GO:0043484), [oligodendrocyte differentiation](https://amigo.geneontology.org/amigo/term/GO:0048709), [mRNA transport](https://amigo.geneontology.org/amigo/term/GO:0051028), and [liver regeneration](https://amigo.geneontology.org/amigo/term/GO:0097421).


The gene length is 122,819.0 base pairs (85.64% of all genes), the mature length is 5,631.0 base pairs (92.91% of all genes), and the primary transcript length is 63,411.5 base pairs (80.43% of all genes).


The gene SRSF1 (NCBI ID: 6426) has been mentioned in [206 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SRSF1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1981, and the middle 50% of publications occurred between 2005 and 2018.


The top five publications mentioning SRSF1, ranked by their scientific influence, include "[SF1 and SF2 helicases: family matters.](https://pubmed.ncbi.nlm.nih.gov/20456941)" (2010) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 13.63, "[The gene encoding the splicing factor SF2/ASF is a proto-oncogene.](https://pubmed.ncbi.nlm.nih.gov/17310252)" (2007) with an RCR of 13.01, "[Inactivation of the SR protein splicing factor ASF/SF2 results in genomic instability.](https://pubmed.ncbi.nlm.nih.gov/16096057)" (2005) with an RCR of 8.82, "[Long Noncoding RNA MALAT1 Promotes Hepatocellular Carcinoma Development by SRSF1 Upregulation and mTOR Activation.](https://pubmed.ncbi.nlm.nih.gov/27993818)" (2017) with an RCR of 8.76, and "[The splicing factor SRSF1 regulates apoptosis and proliferation to promote mammary epithelial cell transformation.](https://pubmed.ncbi.nlm.nih.gov/22245967)" (2012) with an RCR of 7.49. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[SRSF1](https://www.proteinatlas.org/ENSG00000136450-SRSF1) is an RNA-binding protein with evidence at the protein level. It is involved in intellectual disability and is detected in all tissues. The protein is localized in the nucleoplasm and cytosol. SRSF1 is part of various non-specific expression clusters related to mitochondria, transcription, mRNA splicing, cell cycle, and basic cellular processes across different tissue types and cell lines.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [IRF1](https://www.ncbi.nlm.nih.gov/gene/3659) and [TP53](https://www.ncbi.nlm.nih.gov/gene/7157), each with 6 experiments. Additionally, [TAL1](https://www.ncbi.nlm.nih.gov/gene/6886), [INTS11](https://www.ncbi.nlm.nih.gov/gene/54973), and [IRF4](https://www.ncbi.nlm.nih.gov/gene/3662) were each found to be regulating in 5 experiments.





The gene is expressed in various tissues, including the ovary and uterus, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in 721_B_lymphoblasts, CD34+ cells, CD4+ T cells, CD8+ T cells, and CD105+ endothelial cells.


The input data highlights various pathways involved in gene expression and mRNA processing. These include transport of mature transcripts to the cytoplasm, RNA polymerase II transcription, cleavage of growing transcripts in the termination region, and post-elongation processing of transcripts. Additionally, the data covers mRNA 3'-end processing, mRNA splicing through both major and minor pathways, and the processing of capped intron-containing pre-mRNA.



The analyzed protein sequence has a GRAVY value of -1.152 (2.17th percentile), indicating a hydrophilic nature. It exhibits a charge of 19.77 at pH 7.0 (93.34th percentile) and a median structural flexibility of 1.016 (92.9th percentile). The protein's secondary structure affinities are 15.32% for helix (0.71st percentile), 24.19% for sheet (6.9th percentile), and 43.15% for turn (98.77th percentile). Additionally, the protein has an instability index of 79.73 (98.21st percentile), an isoelectric point of 10.37 (98.02nd percentile), a length of 248 amino acids (21.41st percentile), and a molecular weight of 27744.24 Da (21.53rd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |