---
layout: post
title: LIMK2
date: 2025-01-17 16:55 CST
description: LIMK2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3985) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3985  | LIMK2 | ENSG00000182541 | 22q12.2 |



The gene is involved in [positive regulation of protein phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0001934) and [protein phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0006468). It enables [protein serine/threonine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004674), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524), and [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872). The gene is active in and located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), and is also located in the [cis-Golgi network](https://amigo.geneontology.org/amigo/term/GO:0005801), [centrosome](https://amigo.geneontology.org/amigo/term/GO:0005813), [perinuclear region of cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0048471), and [mitotic spindle](https://amigo.geneontology.org/amigo/term/GO:0072686). Additionally, the gene is involved in various processes such as [spermatogenesis](https://amigo.geneontology.org/amigo/term/GO:0007283), [actin cytoskeleton organization](https://amigo.geneontology.org/amigo/term/GO:0030036), [astral microtubule organization](https://amigo.geneontology.org/amigo/term/GO:0030953), [establishment of vesicle localization](https://amigo.geneontology.org/amigo/term/GO:0051650), [head development](https://amigo.geneontology.org/amigo/term/GO:0060322), [cornea development in camera-type eye](https://amigo.geneontology.org/amigo/term/GO:0061303), [positive regulation of protein localization to nucleus](https://amigo.geneontology.org/amigo/term/GO:1900182), and [negative regulation of cilium assembly](https://amigo.geneontology.org/amigo/term/GO:1902018).


The gene length is 10,058 base pairs (25.93% of all genes), the mature length is 1,070 base pairs, and the primary transcript length is 9,856 base pairs.


LIMK2 (Gene ID: 3985) has been mentioned in [43 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22LIMK2%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1995 and the middle 50% of publications occurring between 2008 and 2019. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning LIMK2, ranked by their scientific influence, include "[Osteosarcoma cell intrinsic PD-L2 signals promote invasion and metastasis via the RhoA-ROCK-LIMK2 and autophagy pathways.](https://pubmed.ncbi.nlm.nih.gov/30886151)" (2019) (relative citation ratio: 2.48), "[Phosphorylation-dependent regulation of SPOP by LIMK2 promotes castration-resistant prostate cancer.](https://pubmed.ncbi.nlm.nih.gov/33311589)" (2021) (relative citation ratio: 1.73), "[LIMK2 promotes the metastatic progression of triple-negative breast cancer by activating SRPK1.](https://pubmed.ncbi.nlm.nih.gov/32859889)" (2020) (relative citation ratio: 1.5), "[LIMK-1 and LIMK-2, two members of a LIM motif-containing protein kinase family.](https://pubmed.ncbi.nlm.nih.gov/7651734)" (1995) (relative citation ratio: 1.41), and "[p53-mediated transcriptional regulation and activation of the actin cytoskeleton regulatory RhoC to LIMK2 signaling pathway promotes cell survival.](https://pubmed.ncbi.nlm.nih.gov/21079653)" (2011) (relative citation ratio: 1.41). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[LIMK2](https://www.proteinatlas.org/ENSG00000182541-LIMK2) is a serine/threonine-protein kinase with transferase activity, functioning as a kinase. Evidence supports its existence at the protein level. The gene is detected in all RNA tissue distributions and is localized in the nucleoplasm and cytosol. LIMK2 is expressed in various clusters, including Cluster 21 in neutrophils with unknown function, Cluster 27 in squamous epithelium related to keratinization, Cluster 44 in the brain with mixed function, Cluster 71 in lymphoma associated with inflammatory response, and Cluster 74 in neutrophils involved in degranulation.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [AR](https://www.ncbi.nlm.nih.gov/gene/367) with 6 experiments, [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664) with 5 experiments, [GATA2](https://www.ncbi.nlm.nih.gov/gene/2624) with 4 experiments, [IKZF1](https://www.ncbi.nlm.nih.gov/gene/10320) with 4 experiments, and [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with a wide range of disease conditions, including various types of cancer such as leukemia, lymphoma, and immune system cancers. Additionally, respiratory diseases like asthma, bronchial disease, and obstructive lung disease are implicated. Metabolic disorders, particularly diabetes mellitus and type 2 diabetes mellitus, are also linked. Furthermore, the data highlights associations with cardiovascular system diseases, kidney diseases including chronic kidney failure, and nervous system diseases such as neuropathy and peripheral neuropathy.



The gene is expressed in a variety of tissues, including thyroid, whole blood, spinal cord, bronchial epithelial cells, prostate, placenta, and colon. Additionally, it is expressed in blood according to the BioGPS Human Cell Type and Tissue Gene Expression Profiles.


The proteins are involved in various pathways related to axon guidance and developmental biology. Notably, they participate in EPH-Ephrin signaling, including EPHB-mediated forward signaling. Additionally, these proteins play roles in Sema4D-mediated semaphorin signaling, inducing cell migration and growth-cone collapse, as well as in broader semaphorin interactions.


The mouse ortholog gene 16886 and its human ortholog gene 3985 are associated with phenotypes such as increased startle reflex and preweaning lethality with incomplete penetrance, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.312, indicating a hydrophilic nature (56.63rd percentile). At pH 7.0, the protein carries a charge of -0.971 (46.88th percentile). The median structural flexibility is 0.999 (36.82nd percentile), with a secondary structure composition of 31.5% helix (45.78th percentile), 34.95% sheet (58.81st percentile), and 27.59% turn (43.53rd percentile). The instability index is 52.59 (65.45th percentile), and the isoelectric point is 6.83 (48.3rd percentile). The protein is 638 amino acids long (71.33rd percentile) with a molecular weight of 72231.24 Da (72nd percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |