---
layout: post
title: HSPA1B
date: 2025-01-17 16:55 CST
description: HSPA1B description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3304) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3304  | HSPA1B | ENSG00000204388 | 6p21.33 |



The gene enables [virus receptor activity](https://amigo.geneontology.org/amigo/term/GO:0001618), [G protein-coupled receptor binding](https://amigo.geneontology.org/amigo/term/GO:0001664), [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [signaling receptor binding](https://amigo.geneontology.org/amigo/term/GO:0005102), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524). It is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [mitochondrion](https://amigo.geneontology.org/amigo/term/GO:0005739), [endoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0005783), [centrosome](https://amigo.geneontology.org/amigo/term/GO:0005813), [centriole](https://amigo.geneontology.org/amigo/term/GO:0005814), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925), [inclusion body](https://amigo.geneontology.org/amigo/term/GO:0016234), [aggresome](https://amigo.geneontology.org/amigo/term/GO:0016235), [nuclear speck](https://amigo.geneontology.org/amigo/term/GO:0016607), [vesicle](https://amigo.geneontology.org/amigo/term/GO:0031982), [perinuclear region of cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0048471), [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062), [blood microparticle](https://amigo.geneontology.org/amigo/term/GO:0072562), and [ficolin-1-rich granule lumen](https://amigo.geneontology.org/amigo/term/GO:1904813). It is involved in processes such as [mRNA catabolic process](https://amigo.geneontology.org/amigo/term/GO:0006402), [negative regulation of cell population proliferation](https://amigo.geneontology.org/amigo/term/GO:0008285), [positive regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0010628), [negative regulation of cell growth](https://amigo.geneontology.org/amigo/term/GO:0030308), [regulation of protein ubiquitination](https://amigo.geneontology.org/amigo/term/GO:0031396), [positive regulation of proteasomal ubiquitin-dependent protein catabolic process](https://amigo.geneontology.org/amigo/term/GO:0032436), [positive regulation of interleukin-8 production](https://amigo.geneontology.org/amigo/term/GO:0032757), [cellular response to oxidative stress](https://amigo.geneontology.org/amigo/term/GO:0034599), [cellular response to heat](https://amigo.geneontology.org/amigo/term/GO:0034605), [protein refolding](https://amigo.geneontology.org/amigo/term/GO:0042026), [negative regulation of apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0043066), [positive regulation of erythrocyte differentiation](https://amigo.geneontology.org/amigo/term/GO:0045648), [protein stabilization](https://amigo.geneontology.org/amigo/term/GO:0050821), [chaperone cofactor-dependent protein refolding](https://amigo.geneontology.org/amigo/term/GO:0051085), [positive regulation of NF-kappaB transcription factor activity](https://amigo.geneontology.org/amigo/term/GO:0051092), [cellular response to steroid hormone stimulus](https://amigo.geneontology.org/amigo/term/GO:0071383), [positive regulation of microtubule nucleation](https://amigo.geneontology.org/amigo/term/GO:0090063), [negative regulation of inclusion body assembly](https://amigo.geneontology.org/amigo/term/GO:0090084), [regulation of mitotic spindle assembly](https://amigo.geneontology.org/amigo/term/GO:1901673), and [positive regulation of tumor necrosis factor-mediated signaling pathway](https://amigo.geneontology.org/amigo/term/GO:1903265). Additionally, it is part of the [protein-containing complex](https://amigo.geneontology.org/amigo/term/GO:0032991) and [ribonucleoprotein complex](https://amigo.geneontology.org/amigo/term/GO:1990904).


The gene length is 9,374 base pairs (24.38% of all genes), the mature length is 889.5 base pairs, and the primary transcript length is 7,167.5 base pairs.


The gene HSPA1B (NCBI ID: 3304) has been mentioned in [44 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22HSPA1B%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1994 and the middle 50% of publications occurring between 2005 and 2015. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning HSPA1B, ranked by their scientific influence, include studies that explore the gene's role in various diseases and biological processes. The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Notable publications include "[Upregulation of HSPA1A/HSPA1B/HSPA7 and Downregulation of HSPA9 Were Related to Poor Survival in Colon Cancer.](https://pubmed.ncbi.nlm.nih.gov/34765552)" (2021) (relative citation ratio: 2.09), "[The major histocompatibility complex region marked by HSP70-1 and HSP70-2 variants is associated with clozapine-induced agranulocytosis in two different ethnic groups.](https://pubmed.ncbi.nlm.nih.gov/7579351)" (1995) (relative citation ratio: 1.67), and "[Heat shock protein 72 (HSPA1B) gene polymorphism and Toll-like receptor (TLR) 4 mutation are associated with increased risk of urinary tract infection in children.](https://pubmed.ncbi.nlm.nih.gov/17314700)" (2007) (relative citation ratio: 1.58). Additionally, "[Cytotoxic effect of nanosilver particles on testicular tissue: Evidence for biochemical stress and Hsp70-2 protein expression.](https://pubmed.ncbi.nlm.nih.gov/26363132)" (2015) (relative citation ratio: 1.55) and "[A hsp70-2 mutation recognized by CTL on a human renal cell carcinoma.](https://pubmed.ncbi.nlm.nih.gov/9973436)" (1999) (relative citation ratio: 1.34) are also significant. The citation count is sourced from [iCite](https://icite.od.nih.gov).


[HSPA1B](https://www.proteinatlas.org/ENSG00000204388-HSPA1B) is a gene with molecular functions including chaperone, host cell receptor for virus entry, and receptor. Evidence for its function is available at the protein level. The gene's RNA is detected in many tissues, and its protein is localized in various subcellular locations such as vesicles, cytosol, perinuclear theca, calyx, flagellar centriole, and annulus. In terms of expression, HSPA1B is part of Cluster 66, indicating non-specific transcription across tissues. In the brain, it is associated with Cluster 10, suggesting a role in nucleic acid binding in the cerebellum. Additionally, it is part of Cluster 51 in cell lines, indicating a role in myeloma and immunoglobulins. In single-cell expression, it is part of Cluster 80, again indicating non-specific transcription.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599) with 6 experiments, [ZNF143](https://www.ncbi.nlm.nih.gov/gene/7702) with 5 experiments, [KDM1A](https://www.ncbi.nlm.nih.gov/gene/23028) with 5 experiments, [STAT3](https://www.ncbi.nlm.nih.gov/gene/6774) with 5 experiments, and [KLF5](https://www.ncbi.nlm.nih.gov/gene/688) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent detection in multiple experiments.



The input data from GWAS indicates associations with a wide range of disease conditions, including various types of cancer such as hematologic cancer, immune system cancer, lymphoma, and Hodgkin's lymphoma. Additionally, the data highlights diseases of cellular proliferation, immune system diseases, and metabolic disorders, specifically diabetes mellitus and type 1 diabetes mellitus. Furthermore, cognitive disorders, diseases of mental health, connective tissue diseases, and musculoskeletal system diseases are also implicated. Notably, the data includes associations with systemic lupus erythematosus and hypersensitivity reaction diseases, particularly type II hypersensitivity reactions.



The gene is expressed in multiple tissue samples, including the brain and lung.


The input data highlights several pathways involved in cellular stress responses, including cellular responses to stress, the attenuation phase, HSF1-dependent transactivation, HSF1 activation, cellular response to heat stress, and the regulation of HSF1-mediated heat shock response.



The analyzed protein sequence has a GRAVY value of -0.383 (48.11th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -10.82 (21.26th percentile). The median structural flexibility is 1.007 (72.44th percentile), with a predicted secondary structure composition of 33.23% helix (61.34th percentile), 34.32% sheet (54.23rd percentile), and 29.33% turn (58.6th percentile). The instability index is 32.17 (5.97th percentile), and the isoelectric point is 5.48 (17.75th percentile). The protein is 641 amino acids long (71.57th percentile) with a molecular weight of 70051.37 Da (70.56th percentile). For more details on the sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |