---
layout: post
title: ERCC6
date: 2025-01-17 16:55 CST
description: ERCC6 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2074) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2074  | ERCC6 | ENSG00000225830 | 10q11.23 |



The gene is involved in various biological processes, including [single strand break repair](https://amigo.geneontology.org/amigo/term/GO:0000012), [DNA damage checkpoint signaling](https://amigo.geneontology.org/amigo/term/GO:0000077), and [response to superoxide](https://amigo.geneontology.org/amigo/term/GO:0000303). It enables functions such as [DNA binding](https://amigo.geneontology.org/amigo/term/GO:0003677) and [chromatin binding](https://amigo.geneontology.org/amigo/term/GO:0003682), but does not enable [DNA helicase activity](https://amigo.geneontology.org/amigo/term/GO:0003678). Additionally, it is involved in [transcription-coupled nucleotide-excision repair](https://amigo.geneontology.org/amigo/term/GO:0006283), [base-excision repair](https://amigo.geneontology.org/amigo/term/GO:0006284), and [pyrimidine dimer repair](https://amigo.geneontology.org/amigo/term/GO:0006290). The gene is active in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654) and [nucleolus](https://amigo.geneontology.org/amigo/term/GO:0005730). It is also involved in [response to oxidative stress](https://amigo.geneontology.org/amigo/term/GO:0006979), [JNK cascade](https://amigo.geneontology.org/amigo/term/GO:0007254), and [intrinsic apoptotic signaling pathway in response to DNA damage](https://amigo.geneontology.org/amigo/term/GO:0008630). Furthermore, it enables [ATP-dependent activity, acting on DNA](https://amigo.geneontology.org/amigo/term/GO:0008094) and [ATP hydrolysis activity](https://amigo.geneontology.org/amigo/term/GO:0016887). The gene is part of the [transcription elongation factor complex](https://amigo.geneontology.org/amigo/term/GO:0008023) and the [B-WICH complex](https://amigo.geneontology.org/amigo/term/GO:0110016), and it is located at the [site of DNA damage](https://amigo.geneontology.org/amigo/term/GO:0090734). It plays roles in various regulatory processes, including [positive regulation of DNA repair](https://amigo.geneontology.org/amigo/term/GO:0045739), [positive regulation of transcription by RNA polymerase I](https://amigo.geneontology.org/amigo/term/GO:0045943), [positive regulation of transcription by RNA polymerase II](https://amigo.geneontology.org/amigo/term/GO:0045944), and [positive regulation of transcription by RNA polymerase III](https://amigo.geneontology.org/amigo/term/GO:0045945). Additionally, it is involved in [double-strand break repair via classical nonhomologous end joining](https://amigo.geneontology.org/amigo/term/GO:0097680) and [positive regulation of double-strand break repair via homologous recombination](https://amigo.geneontology.org/amigo/term/GO:1905168), as well as [negative regulation of double-strand break repair via nonhomologous end joining](https://amigo.geneontology.org/amigo/term/GO:2001033).


The gene length is 89,798 base pairs (79.8% of all genes), the mature length is 612 base pairs (5.99% of all genes), and the primary transcript length is 16,404 base pairs (46.41% of all genes).


The gene ERCC6 (NCBI ID: 2074) has been mentioned in [132 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ERCC6%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1957 and the middle 50% of publications occurring between 2007 and 2018.


The top 5 publications mentioning ERCC6, ranked by their scientific influence, include "[ERCC6, a member of a subfamily of putative helicases, is involved in Cockayne's syndrome and preferential repair of active genes.](https://pubmed.ncbi.nlm.nih.gov/1339317)" (1992) (relative citation ratio: 11.7), "[The cooperative action of CSB, CSA, and UVSSA target TFIIH to DNA damage-stalled RNA polymerase II.](https://pubmed.ncbi.nlm.nih.gov/32355176)" (2020) (relative citation ratio: 5.28), "[Transcription-coupled repair of 8-oxoguanine: requirement for XPG, TFIIH, and CSB and implications for Cockayne syndrome.](https://pubmed.ncbi.nlm.nih.gov/10786832)" (2000) (relative citation ratio: 4.9), "[NACOB presentation CSB New Investigator Award. Balance recovery from medio-lateral perturbations of the upper body during standing. North American Congress on Biomechanics.](https://pubmed.ncbi.nlm.nih.gov/10541064)" (1999) (relative citation ratio: 4.49), and "[Mutations in UVSSA cause UV-sensitive syndrome and destabilize ERCC6 in transcription-coupled DNA repair.](https://pubmed.ncbi.nlm.nih.gov/22466612)" (2012) (relative citation ratio: 3.71). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ERCC6](https://www.proteinatlas.org/ENSG00000225830-ERCC6) is a gene with molecular functions including DNA-binding, helicase, and hydrolase activities. It is implicated in various diseases such as age-related macular degeneration, cancer-related genes, cataract, Cockayne syndrome, deafness, dwarfism, intellectual disability, premature ovarian failure, and xeroderma pigmentosum. Evidence for this gene is available at the protein level. Its RNA is detected in all tissues, with subcellular localization in the nucleoplasm and nuclear bodies. ERCC6 is part of blood expression cluster 7, involved in the innate immune response, and tissue expression cluster 16, related to transcription. In the brain, it is part of cluster 42, associated with the choroid plexus and mitochondria. Additionally, it is found in cell line expression cluster 64, related to connective tissue cells and ECM organization, and single cell expression cluster 55, involved in transcription.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [AR](https://www.ncbi.nlm.nih.gov/gene/367), [BCL6](https://www.ncbi.nlm.nih.gov/gene/604), and [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), each with 5 experiments. Additionally, [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113) and [HDAC1](https://www.ncbi.nlm.nih.gov/gene/3065) were found to be regulating in 4 experiments each.



The input data indicates associations with various disease conditions, including nervous system diseases, central nervous system diseases, cognitive disorders, diseases of mental health, bipolar disorder, mood disorders, and brain diseases.


The gene ERCC6 has been associated with [Protein quantitative trait loci (liver)](https://pubmed.ncbi.nlm.nih.gov/32778093), which has been mapped to the closest existing MeSH term [Genes](https://meshb.nlm.nih.gov/record/ui?ui=D005796) using a method called [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0).


The gene is expressed in multiple tissue samples, including skin, thyroid, and pituitary.


The input data highlights various biological processes involving gene expression and DNA repair. Key pathways include dual incision reaction in TC-NER, formation of the transcription-coupled NER (TC-NER) repair complex, and nucleotide excision repair. Additionally, the data emphasizes transcription processes, such as RNA polymerase I promoter clearance, RNA polymerase I transcription, and RNA polymerase I transcription initiation. These pathways collectively contribute to the regulation of gene expression and the maintenance of genomic integrity.



The analyzed protein sequence has a GRAVY value of -0.712 (16.9th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -0.530 (48.12th percentile). The median structural flexibility is 1.009 (79.08th percentile), with a secondary structure composition of 32.72% helix (56.99th percentile), 30.20% sheet (29.38th percentile), and 29.28% turn (58.22th percentile). The instability index is 53.97 (69.21st percentile), and the isoelectric point is 7.22 (52.48th percentile). The protein has a length of 1277 amino acids (92.88th percentile) and a molecular weight of 143949.73 Da (93.14th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |