---
layout: post
title: PICALM
date: 2025-01-17 16:55 CST
description: PICALM description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8301) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8301  | PICALM | ENSG00000073921 | 11q14.2 |



The gene enables [SNARE binding](https://amigo.geneontology.org/amigo/term/GO:0000149), [amyloid-beta binding](https://amigo.geneontology.org/amigo/term/GO:0001540), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [1-phosphatidylinositol binding](https://amigo.geneontology.org/amigo/term/GO:0005545), and [phosphatidylinositol-4,5-bisphosphate binding](https://amigo.geneontology.org/amigo/term/GO:0005546). It is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [early endosome](https://amigo.geneontology.org/amigo/term/GO:0005769), [Golgi apparatus](https://amigo.geneontology.org/amigo/term/GO:0005794), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [clathrin-coated pit](https://amigo.geneontology.org/amigo/term/GO:0005905), [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [vesicle](https://amigo.geneontology.org/amigo/term/GO:0031982), [presynaptic membrane](https://amigo.geneontology.org/amigo/term/GO:0042734), [neuronal cell body](https://amigo.geneontology.org/amigo/term/GO:0043025), [intracellular membrane-bounded organelle](https://amigo.geneontology.org/amigo/term/GO:0043231), [postsynaptic membrane](https://amigo.geneontology.org/amigo/term/GO:0045211), [clathrin-coated endocytic vesicle](https://amigo.geneontology.org/amigo/term/GO:0045334), [endosome to plasma membrane transport vesicle](https://amigo.geneontology.org/amigo/term/GO:0070381), and [neurofibrillary tangle](https://amigo.geneontology.org/amigo/term/GO:0097418). It is active in the [synaptic vesicle](https://amigo.geneontology.org/amigo/term/GO:0008021), [clathrin-coated vesicle](https://amigo.geneontology.org/amigo/term/GO:0030136), and [extrinsic component of presynaptic endocytic zone membrane](https://amigo.geneontology.org/amigo/term/GO:0098894). The gene is involved in various processes such as [intracellular iron ion homeostasis](https://amigo.geneontology.org/amigo/term/GO:0006879), [endocytosis](https://amigo.geneontology.org/amigo/term/GO:0006897), [receptor-mediated endocytosis](https://amigo.geneontology.org/amigo/term/GO:0006898), [vesicle budding from membrane](https://amigo.geneontology.org/amigo/term/GO:0006900), [axonogenesis](https://amigo.geneontology.org/amigo/term/GO:0007409), [learning or memory](https://amigo.geneontology.org/amigo/term/GO:0007611), [negative regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0010629), [synaptic vesicle budding from presynaptic endocytic zone membrane](https://amigo.geneontology.org/amigo/term/GO:0016185), [synaptic vesicle maturation](https://amigo.geneontology.org/amigo/term/GO:0016188), [vesicle-mediated transport](https://amigo.geneontology.org/amigo/term/GO:0016192), [endosomal transport](https://amigo.geneontology.org/amigo/term/GO:0016197), [hemopoiesis](https://amigo.geneontology.org/amigo/term/GO:0030097), [regulation of endocytosis](https://amigo.geneontology.org/amigo/term/GO:0030100), [receptor internalization](https://amigo.geneontology.org/amigo/term/GO:0031623), [regulation of protein localization](https://amigo.geneontology.org/amigo/term/GO:0032880), [vesicle cargo loading](https://amigo.geneontology.org/amigo/term/GO:0035459), [positive regulation of GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0043547), [positive regulation of DNA-templated transcription](https://amigo.geneontology.org/amigo/term/GO:0045893), [negative regulation of receptor-mediated endocytosis](https://amigo.geneontology.org/amigo/term/GO:0048261), [clathrin coat assembly](https://amigo.geneontology.org/amigo/term/GO:0048268), [dendrite morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0048813), [multicellular organismal-level iron ion homeostasis](https://amigo.geneontology.org/amigo/term/GO:0060586), [protein-containing complex assembly](https://amigo.geneontology.org/amigo/term/GO:0065003), [clathrin-dependent endocytosis](https://amigo.geneontology.org/amigo/term/GO:0072583), [regulation of vesicle size](https://amigo.geneontology.org/amigo/term/GO:0097494), [membrane bending](https://amigo.geneontology.org/amigo/term/GO:0097753), [amyloid-beta clearance by transcytosis](https://amigo.geneontology.org/amigo/term/GO:0150093), [positive regulation of amyloid-beta formation](https://amigo.geneontology.org/amigo/term/GO:1902004), [regulation of aspartic-type endopeptidase activity involved in amyloid precursor protein catabolic process](https://amigo.geneontology.org/amigo/term/GO:1902959), [positive regulation of aspartic-type endopeptidase activity involved in amyloid precursor protein catabolic process](https://amigo.geneontology.org/amigo/term/GO:1902961), [negative regulation of metalloendopeptidase activity involved in amyloid precursor protein catabolic process](https://amigo.geneontology.org/amigo/term/GO:1902963), [negative regulation of protein localization to plasma membrane](https://amigo.geneontology.org/amigo/term/GO:1903077), and [negative regulation of protein localization to cell surface](https://amigo.geneontology.org/amigo/term/GO:2000009). Additionally, the gene enables [clathrin binding](https://amigo.geneontology.org/amigo/term/GO:0030276), [small GTPase binding](https://amigo.geneontology.org/amigo/term/GO:0031267), [clathrin heavy chain binding](https://amigo.geneontology.org/amigo/term/GO:0032050), [cadherin binding](https://amigo.geneontology.org/amigo/term/GO:0045296), and [tau protein binding](https://amigo.geneontology.org/amigo/term/GO:0048156). It is part of the [clathrin coat of coated pit](https://amigo.geneontology.org/amigo/term/GO:0030132) and colocalizes with the [perinuclear region of cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0048471).


The gene length is 8,377 base pairs (22.13% of all genes), the mature length is 1,416 base pairs (35.45% of all genes), and the primary transcript length is 7,900 base pairs (27.56% of all genes).


The gene PICALM (NCBI ID: 8301) has been mentioned in [127 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PICALM%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1996 and the middle 50% of publications occurring between 2011 and 2018. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning PICALM, ranked by their scientific influence, include "[Genome-wide association study identifies variants at CLU and PICALM associated with Alzheimer's disease.](https://pubmed.ncbi.nlm.nih.gov/19734902)" (2009) (relative citation ratio: 51.71), "[Central role for PICALM in amyloid-Î² blood-brain barrier transcytosis and clearance.](https://pubmed.ncbi.nlm.nih.gov/26005850)" (2015) (relative citation ratio: 10.65), "[Meta-analysis confirms CR1, CLU, and PICALM as alzheimer disease risk loci and reveals interactions with APOE genotypes.](https://pubmed.ncbi.nlm.nih.gov/20697030)" (2010) (relative citation ratio: 8.36), "[PICALM modulates autophagy activity and tau accumulation.](https://pubmed.ncbi.nlm.nih.gov/25241929)" (2014) (relative citation ratio: 5.62), and "[CALM regulates clathrin-coated vesicle size and maturation by directly sensing and driving membrane curvature.](https://pubmed.ncbi.nlm.nih.gov/25898166)" (2015) (relative citation ratio: 4.94). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PICALM](https://www.proteinatlas.org/ENSG00000073921-PICALM) is a proto-oncogene involved in cancer-related processes, with evidence supporting its function at the protein level. The gene is detected in all RNA tissue distributions and is primarily localized in vesicles. Its expression is notable in various clusters, including Cluster 7 for blood (non-specific - innate immune response), Cluster 11 for tissue (liver - plasma proteins), Cluster 11 for brain (white matter - signal transduction), Cluster 47 for cell lines (leukemia - hemostasis), and Cluster 51 for single cells (oligodendrocytes - unknown function).


The top transcription factors (TFs) identified from the GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [KDM1A](https://www.ncbi.nlm.nih.gov/gene/23028), [NEUROD1](https://www.ncbi.nlm.nih.gov/gene/4760), [HDAC1](https://www.ncbi.nlm.nih.gov/gene/3065), [TAL1](https://www.ncbi.nlm.nih.gov/gene/6886), and [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613), each with 5 experiments demonstrating their regulatory roles.



The input data from GWAS indicates associations with a wide range of disease conditions, including metabolic disorders such as diabetes mellitus, type 2 diabetes mellitus, and obesity. Additionally, there are links to cardiovascular diseases, including artery disease, hypertension, and cardiovascular system disease. Neurological conditions are also prominent, with associations to Alzheimer's disease, dementia, and various mental health disorders such as anxiety disorder, obsessive-compulsive disorder, and attention deficit hyperactivity disorder. Furthermore, the data highlights connections to kidney disease, urinary system disease, and cerebrovascular disease.



The gene is expressed in adipose tissue and blood vessels according to the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles indicate expression in CD33+ Myeloid cells, CD14+ Monocytes, Whole Blood, CD105+ Endothelial cells, CD71+ Early Erythroid cells, Placenta, and Fetal liver.


The proteins are involved in various pathways related to vesicle dynamics and trafficking, including clathrin-derived vesicle budding, trans-Golgi network vesicle budding, Golgi-associated vesicle biogenesis, and membrane trafficking.



The protein sequence analyzed has a GRAVY value of -0.228, indicating a hydrophilic nature (66.34th percentile). It exhibits a charge of 1.57 at pH 7.0 (55.81st percentile) and a median structural flexibility of 1.003 (54.6th percentile). The protein's secondary structure is predicted to be 30.7% helix (37.86th percentile), 36.0% sheet (66.57th percentile), and 33.0% turn (81.82nd percentile). The instability index is 28.95 (3.27th percentile), with an isoelectric point of 7.70 (56.29th percentile). The protein has a length of 652 amino acids (72.27th percentile) and a molecular weight of 70753.69 Da (71.04th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |