---
layout: post
title: PSMB1
date: 2024-12-08 23:57 CST
description: PSMB1 description
tags: [cooccuring-genes]
categories: longest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5689) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5689  | PSMB1 | ENSG00000008018 | 6q27  |



The gene length is 581,045 base pairs (98.8% of all genes), the mature length is 5,850 base pairs, and the primary transcript length is 581,045 base pairs.


The gene PSMB1, with NCBI ID 5689, has been featured in [7 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PSMB1%22) (count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest publication year is 2004, and the middle 50% of publications occurred between 2007 and 2014.


The top five publications mentioning PSMB1, ranked by their scientific influence, are as follows: "[Biallelic variants in PSMB1 encoding the proteasome subunit β6 cause impairment of proteasome function, microcephaly, intellectual disability, developmental delay and short stature.](https://pubmed.ncbi.nlm.nih.gov/32129449)" (2020) (relative citation ratio: 1.43), "[Inhibition on Proteasome β1 Subunit Might Contribute to the Anti-Cancer Effects of Fangchinoline in Human Prostate Cancer Cells.](https://pubmed.ncbi.nlm.nih.gov/26512898)" (2015) (relative citation ratio: 0.89), "[BCL-3 degradation involves its polyubiquitination through a FBW7-independent pathway and its binding to the proteasome subunit PSMB1.](https://pubmed.ncbi.nlm.nih.gov/20558726)" (2010) (relative citation ratio: 0.43), "[Interaction of plasminogen activator inhibitor-2 and proteasome subunit, beta type 1.](https://pubmed.ncbi.nlm.nih.gov/14732874)" (2004) (relative citation ratio: 0.41), and "[Role of nuclear localization of PSMB1 in transcriptional activation.](https://pubmed.ncbi.nlm.nih.gov/23924720)" (2013) (relative citation ratio: 0.15). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [PPARG](https://www.ncbi.nlm.nih.gov/gene/5468), [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), and [CEBPA](https://www.ncbi.nlm.nih.gov/gene/1050), each with 4 experiments. Additionally, [HDAC2](https://www.ncbi.nlm.nih.gov/gene/3066) and [ME3](https://www.ncbi.nlm.nih.gov/gene/10873) were found to be regulating in 3 experiments each.











The analyzed protein sequence has a GRAVY value of -0.1174 (76.77th percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is 1.92 (57.24th percentile), and the median structural flexibility is 0.996 (24.33rd percentile). The protein has a propensity for helix, sheet, and turn structures of 31.12% (42.28th percentile), 35.68% (64.15th percentile), and 29.88% (62.9th percentile), respectively. The instability index is 33.54 (7.85th percentile), suggesting a stable protein. The isoelectric point is 8.27 (62.35th percentile), with a molecular weight of 26489.08 Da (20.02nd percentile) and a length of 241 amino acids (20.37th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |