---
layout: post
title: RASGRP2
date: 2025-01-17 16:55 CST
description: RASGRP2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10235) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10235  | RASGRP2 | ENSG00000068831 | 11q13.1 |



The gene is involved in the [regulation of cell growth](https://amigo.geneontology.org/amigo/term/GO:0001558) and [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), including [Ras protein signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007265). It enables [guanyl-nucleotide exchange factor activity](https://amigo.geneontology.org/amigo/term/GO:0005085), [calcium ion binding](https://amigo.geneontology.org/amigo/term/GO:0005509), [lipid binding](https://amigo.geneontology.org/amigo/term/GO:0008289), and [diacylglycerol binding](https://amigo.geneontology.org/amigo/term/GO:0019992). The gene is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [ruffle membrane](https://amigo.geneontology.org/amigo/term/GO:0032587), [neuron projection](https://amigo.geneontology.org/amigo/term/GO:0043005), and [synapse](https://amigo.geneontology.org/amigo/term/GO:0045202). Additionally, it is involved in the [positive regulation of GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0043547) and the [cellular response to calcium ion](https://amigo.geneontology.org/amigo/term/GO:0071277).


The gene length is 21,143.0 base pairs (43.39% of all genes), the mature length is 674.0 base pairs, and the primary transcript length is 8,686.0 base pairs (29.81% of all genes).


The gene RASGRP2 (NCBI ID: 10235) has been mentioned in [22 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RASGRP2%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 2004, and the middle 50% of publications occurred between 2013 and 2020.


The top 5 publications mentioning RASGRP2, ranked by their scientific influence, include "[RasGRP2 Structure, Function and Genetic Variants in Platelet Pathophysiology.](https://pubmed.ncbi.nlm.nih.gov/32041177)" (2020) (relative citation ratio: 1.43), "[The inhibition of Bax activation-induced apoptosis by RasGRP2 via R-Ras-PI3K-Akt signaling pathway in the endothelial cells.](https://pubmed.ncbi.nlm.nih.gov/31723205)" (2019) (relative citation ratio: 1.36), "[Expanded repertoire of <i>RASGRP2</i> variants responsible for platelet dysfunction and severe bleeding.](https://pubmed.ncbi.nlm.nih.gov/28637664)" (2017) (relative citation ratio: 1.22), "[Phosphorylation of CalDAG-GEFI by protein kinase A prevents Rap1b activation.](https://pubmed.ncbi.nlm.nih.gov/23611601)" (2013) (relative citation ratio: 1.14), and "[Calcium-RasGRP2-Rap1 signaling mediates CD38-induced migration of chronic lymphocytic leukemia cells.](https://pubmed.ncbi.nlm.nih.gov/29970392)" (2018) (relative citation ratio: 1.11). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RASGRP2](https://www.proteinatlas.org/ENSG00000068831-RASGRP2) is a guanine-nucleotide releasing factor with evidence at the protein level. It is involved in disease variants and is detected in all RNA tissue distributions. The protein is primarily located in the plasma membrane. RASGRP2 is expressed in lymphoid tissue, associated with immune response, and is also found in neurons with mixed functions. Additionally, it is expressed in leukemia cell lines, involved in signal transduction, and is present in platelets, playing a role in hemostasis.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [AR](https://www.ncbi.nlm.nih.gov/gene/367) with 6 experiments, [HDAC2](https://www.ncbi.nlm.nih.gov/gene/3066) with 5 experiments, [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227) with 5 experiments, [EP300](https://www.ncbi.nlm.nih.gov/gene/2033) with 5 experiments, and [SUZ12](https://www.ncbi.nlm.nih.gov/gene/23512) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with various disease conditions, including bone inflammation disease, arthritis, bone disease, connective tissue disease, musculoskeletal system disease, and gout.



The gene is expressed in various tissues, including blood, spleen, and small intestine, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in CD34+ cells, BDCA4+ dendritic cells, CD56+ NK cells, CD4+ T cells, CD8+ T cells, CD19+ B cells (neg. sel.), and whole blood.


The input data highlights various pathways involved in immune system function, signal transduction, and hemostasis. Key pathways include the adaptive and innate immune systems, Fc epsilon receptor (FCERI) signaling, and FCERI-mediated NF-kB activation. Additionally, G protein-coupled receptor (GPCR) downstream signaling, Rap1 signaling, and integrin alphaIIb beta3 signaling are noted. Hemostasis-related processes, such as platelet aggregation and activation, are also significant.



The analyzed protein sequence has a GRAVY value of -0.324 (55.22 percentile), indicating a hydrophilic nature. It exhibits a charge of 3.55 at pH 7.0 (62.92 percentile) and a median structural flexibility of 0.999 (37.47 percentile). The protein's secondary structure is predicted to be 31.03% helix (41.34 percentile), 34.81% sheet (57.62 percentile), and 27.59% turn (43.53 percentile). The instability index is 62.11 (86.57 percentile), with an isoelectric point of 7.89 (57.88 percentile). The protein has a length of 609 amino acids (69.24 percentile) and a molecular weight of 69247.66 Da (70.11 percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |