---
layout: post
title: GNE
date: 2025-01-17 16:55 CST
description: GNE description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10020) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10020  | GNE | ENSG00000159921 | 9p13.3 |



The gene enables [hydrolase activity, hydrolyzing O-glycosyl compounds](https://amigo.geneontology.org/amigo/term/GO:0004553), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524). It is active in and located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). The gene is involved in the [N-acetylglucosamine biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006045) and the [UDP-N-acetylglucosamine metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006047). Additionally, it enables [UDP-N-acetylglucosamine 2-epimerase activity](https://amigo.geneontology.org/amigo/term/GO:0008761) and [N-acylmannosamine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0009384). The gene is also involved in the [N-acetylneuraminate biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0046380) and [carbohydrate phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0046835), and it enables [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872).


The gene length is 25,650 base pairs (48.38th percentile of all genes), the mature length is 1,352 base pairs, and the primary transcript length is 14,939 base pairs.


The gene GNE (NCBI ID: 10020) has been mentioned in [167 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22GNE%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning GNE is 1981, and the middle 50% of publications occurred between 2011 and 2020.


The top 5 publications mentioning GNE, ranked by their scientific influence, include "[Brain tyrosinase overexpression implicates age-dependent neuromelanin production in Parkinson's disease pathogenesis.](https://pubmed.ncbi.nlm.nih.gov/30846695)" (2019) (relative citation ratio: 11.99), "[Neuromelanin in Parkinson's Disease: Tyrosine Hydroxylase and Tyrosinase.](https://pubmed.ncbi.nlm.nih.gov/35456994)" (2022) (relative citation ratio: 9.11), "[The UDP-N-acetylglucosamine 2-epimerase/N-acetylmannosamine kinase gene is mutated in recessive hereditary inclusion body myopathy.](https://pubmed.ncbi.nlm.nih.gov/11528398)" (2001) (relative citation ratio: 8.03), "[Longitudinal Changes in Neuromelanin MRI Signal in Parkinson's Disease: A Progression Marker.](https://pubmed.ncbi.nlm.nih.gov/33751655)" (2021) (relative citation ratio: 5.92), and "[Neuromelanin activates microglia and induces degeneration of dopaminergic neurons: implications for progression of Parkinson's disease.](https://pubmed.ncbi.nlm.nih.gov/19957214)" (2011) (relative citation ratio: 5.06). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[GNE](https://www.proteinatlas.org/ENSG00000159921-GNE) is a multifunctional enzyme with allosteric, hydrolase, kinase, and transferase activities. It is implicated in disease variants and has evidence at the protein level. GNE is detected in all RNA tissue distributions and is primarily located in the cytosol. It is part of Cluster 33 in blood expression, which is non-specific and involved in ATP binding. In tissue expression, GNE is part of Cluster 83, associated with liver metabolism. Additionally, it is found in Cluster 26 in brain expression, related to oligodendrocytes with mixed functions, and in Cluster 53 in cell line expression, associated with squamous epithelial cells and keratinization. In single-cell expression, GNE is part of Cluster 16, linked to intestinal goblet cells and mucin production.


The top transcription factors (TFs) identified from the GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD9](https://www.ncbi.nlm.nih.gov/gene/65980), [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170), [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885), [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113), and [TP53](https://www.ncbi.nlm.nih.gov/gene/7157), each shown to be regulating in 5 experiments.



The GWAS data indicates associations with several disease conditions, including lower respiratory tract disease, lung disease, respiratory system disease, and general disease categories. These findings highlight the potential genetic underpinnings of various respiratory and broader disease conditions.



The gene is expressed in multiple tissue samples, including the liver, salivary gland, and colon, as indicated by both BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles show expression in the liver, prostate, placenta, salivary gland, trachea, and colon.


The input data highlights various diseases and processes related to glycosylation, particularly asparagine N-linked glycosylation. Notably, defects in specific genes such as ALG2, ALG3, ALG6, ALG8, ALG9, DPAGT1, MAN1B1, MGAT2, MOGS, MPDU1, ALG1, ALG12, ALG11, ALG14, and RFT1 are associated with distinct congenital disorders of glycosylation (CDGs) and other conditions like congenital myasthenic syndrome. Additionally, the data covers broader categories including diseases associated with N-glycosylation of proteins, post-translational protein modification, biosynthesis of the N-glycan precursor, synthesis of substrates in N-glycan biosynthesis, sialic acid metabolism, and general protein metabolism.


The mouse ortholog gene 50798, which corresponds to the human ortholog gene 10020, is associated with several phenotypic abnormalities. These include abnormal heart, neural tube, and placenta morphologies, as well as edema, embryonic growth retardation, microphthalmia, and persistence of the hyaloid vascular system, all of which affect both sexes. Additionally, the gene is linked to prenatal lethality prior to heart atrial septation, specifically in males, and preweaning lethality with complete penetrance in both sexes.


The analyzed protein sequence has a GRAVY value of 0.0234 (85.05th percentile), indicating a slightly hydrophobic nature. The charge at pH 7 is -7.95 (26.55th percentile), and the median structural flexibility is 0.994 (18.07th percentile). The protein has a propensity for helix, sheet, and turn structures of 32.13% (51.59th percentile), 37.12% (73.81st percentile), and 27.29% (40.65th percentile), respectively. The instability index is 32.63 (6.57th percentile), and the isoelectric point is 6.32 (38.51st percentile). The protein is 722 amino acids long (76.64th percentile) with a molecular weight of 79273.77 Da (75.83rd percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |