---
layout: post
title: RBM3
date: 2025-01-17 16:55 CST
description: RBM3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5935) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5935  | RBM3 | ENSG00000102317 | Xp11.23  |



The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), and [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737). It is part of the [spliceosomal complex](https://amigo.geneontology.org/amigo/term/GO:0005681) and the [large ribosomal subunit](https://amigo.geneontology.org/amigo/term/GO:0015934), and is also found in the [dendrite](https://amigo.geneontology.org/amigo/term/GO:0030425). The gene is involved in [RNA processing](https://amigo.geneontology.org/amigo/term/GO:0006396), [regulation of translation](https://amigo.geneontology.org/amigo/term/GO:0006417), [positive regulation of translation](https://amigo.geneontology.org/amigo/term/GO:0045727), and [positive regulation of mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0048026). Additionally, it enables [ribosomal large subunit binding](https://amigo.geneontology.org/amigo/term/GO:0043023).


The gene length is 37,308 base pairs (58.62% of all genes), the mature length is 573 base pairs (3.87% of all genes), and the primary transcript length is 10,367 base pairs (34.26% of all genes).


The gene RBM3 (NCBI ID: 5935) has been mentioned in [75 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RBM3%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning this gene was in 1995, and the middle 50% of publications occurred between 2014 and 2020.


The top 5 publications mentioning RBM3, ranked by their scientific influence, include "[The RNA-binding protein RBM3 promotes cell proliferation in hepatocellular carcinoma by regulating circular RNA SCD-circRNA 2 production.](https://pubmed.ncbi.nlm.nih.gov/31235426)" (2019) (relative citation ratio: 6.57), "[RBM3 mediates structural plasticity and protective effects of cooling in neurodegeneration.](https://pubmed.ncbi.nlm.nih.gov/25607368)" (2015) (relative citation ratio: 5.31), "[Cold-inducible proteins CIRP and RBM3, a unique couple with activities far beyond the cold.](https://pubmed.ncbi.nlm.nih.gov/27147467)" (2016) (relative citation ratio: 5.22), "[Oxygen-regulated expression of the RNA-binding proteins RBM3 and CIRP by a HIF-1-independent mechanism.](https://pubmed.ncbi.nlm.nih.gov/15075239)" (2004) (relative citation ratio: 3.83), and "[RBM3 promotes neurogenesis in a niche-dependent manner via IMP2-IGF2 signaling pathway after hypoxic-ischemic brain injury.](https://pubmed.ncbi.nlm.nih.gov/31484925)" (2019) (relative citation ratio: 2.56). The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RBM3](https://www.proteinatlas.org/ENSG00000102317-RBM3) is an RNA-binding protein with evidence at the protein level. It is detected in all tissues and primarily localized in the nucleoplasm. RBM3 is associated with various expression clusters, including Cluster 13 for blood expression, Cluster 18 for tissue expression, Cluster 30 for brain expression, Cluster 36 for cell line expression, and Cluster 45 for single cell expression. Notably, RBM3 is involved in white matter myelination in the brain.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), and [JUN](https://www.ncbi.nlm.nih.gov/gene/3725), each with 5 experiments. Additionally, [PPARG](https://www.ncbi.nlm.nih.gov/gene/5468) and [SIN3A](https://www.ncbi.nlm.nih.gov/gene/25942) were found to be regulating in 4 experiments each.





The gene is expressed in various tissues, including adipose tissue, breast, and skin, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in BDCA4+ dendritic cells, CD19+ B cells (neg. sel.), colon, and small intestine.



The mouse ortholog gene 19652 and its human ortholog gene 5935 are associated with abnormal liver morphology, a phenotype observed in both sexes.


The analyzed protein sequence has a GRAVY value of -1.089 (2.83rd percentile), indicating a hydrophilic nature. It exhibits a charge of 1.869 at pH 7 (57.08th percentile) and a median structural flexibility of 1.011 (83.53rd percentile). The protein's secondary structure is predicted to be 15.92% helix (0.74th percentile), 24.84% sheet (8.33rd percentile), and 44.59% turn (99.08th percentile). The instability index is 30.35 (4.3rd percentile), with an isoelectric point of 8.857 (75.35th percentile). The protein has a length of 157 amino acids (8.9th percentile) and a molecular weight of 17170.16 Da (8.49th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |