---
layout: post
title: KIF5B
date: 2025-01-17 16:55 CST
description: KIF5B description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3799) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3799  | KIF5B | ENSG00000170759 | 10p11.22 |



The gene enables [microtubule motor activity](https://amigo.geneontology.org/amigo/term/GO:0003777) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is involved in [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524). It is located in the [mitochondrion](https://amigo.geneontology.org/amigo/term/GO:0005739) and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and is part of the [kinesin complex](https://amigo.geneontology.org/amigo/term/GO:0005871). The gene is active in the [microtubule](https://amigo.geneontology.org/amigo/term/GO:0005874) and involved in various processes such as [microtubule-based movement](https://amigo.geneontology.org/amigo/term/GO:0007018), [cytoplasm organization](https://amigo.geneontology.org/amigo/term/GO:0007028), and [axon guidance](https://amigo.geneontology.org/amigo/term/GO:0007411). Additionally, it enables [microtubule binding](https://amigo.geneontology.org/amigo/term/GO:0008017) and [plus-end-directed microtubule motor activity](https://amigo.geneontology.org/amigo/term/GO:0008574), and is located in the [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020) and [vesicle](https://amigo.geneontology.org/amigo/term/GO:0031982). The gene is also involved in processes like [positive regulation of synaptic transmission, GABAergic](https://amigo.geneontology.org/amigo/term/GO:0032230), [lysosome localization](https://amigo.geneontology.org/amigo/term/GO:0032418), and [stress granule disassembly](https://amigo.geneontology.org/amigo/term/GO:0035617). Furthermore, it is located in the [dendrite cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0032839), [centriolar satellite](https://amigo.geneontology.org/amigo/term/GO:0034451), and [ciliary rootlet](https://amigo.geneontology.org/amigo/term/GO:0035253), and is involved in [natural killer cell mediated cytotoxicity](https://amigo.geneontology.org/amigo/term/GO:0042267) and [regulation of membrane potential](https://amigo.geneontology.org/amigo/term/GO:0042391). The gene enables [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802), [protein-containing complex binding](https://amigo.geneontology.org/amigo/term/GO:0044877), and [cadherin binding](https://amigo.geneontology.org/amigo/term/GO:0045296), and is located in the [phagocytic vesicle](https://amigo.geneontology.org/amigo/term/GO:0045335) and [perinuclear region of cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0048471). It is involved in [vesicle transport along microtubule](https://amigo.geneontology.org/amigo/term/GO:0047496), [mitochondrion transport along microtubule](https://amigo.geneontology.org/amigo/term/GO:0047497), and [synaptic vesicle transport](https://amigo.geneontology.org/amigo/term/GO:0048489). Additionally, the gene is involved in [centrosome localization](https://amigo.geneontology.org/amigo/term/GO:0051642), [cellular response to type II interferon](https://amigo.geneontology.org/amigo/term/GO:0071346), and [plus-end-directed vesicle transport along microtubule](https://amigo.geneontology.org/amigo/term/GO:0072383). It is also involved in [anterograde dendritic transport of neurotransmitter receptor complex](https://amigo.geneontology.org/amigo/term/GO:0098971), [anterograde axonal protein transport](https://amigo.geneontology.org/amigo/term/GO:0099641), and [mitocytosis](https://amigo.geneontology.org/amigo/term/GO:0160040). The gene is located in the [cytolytic granule membrane](https://amigo.geneontology.org/amigo/term/GO:0101004) and [axon cytoplasm](https://amigo.geneontology.org/amigo/term/GO:1904115), and is involved in [positive regulation of protein localization to plasma membrane](https://amigo.geneontology.org/amigo/term/GO:1903078), [anterograde neuronal dense core vesicle transport](https://amigo.geneontology.org/amigo/term/GO:1990048), and [retrograde neuronal dense core vesicle transport](https://amigo.geneontology.org/amigo/term/GO:1990049).


The gene length is 51,737 base pairs (67.39% of all genes), the mature length is 728 base pairs (10.44% of all genes), and the primary transcript length is 1,208 base pairs (3.25% of all genes).


KIF5B (Gene ID: 3799) has been mentioned in [106 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22KIF5B%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1992 and the middle 50% of publications occurring between 2010 and 2020. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning KIF5B, ranked by their scientific influence, include "[KIF5B-RET fusions in lung adenocarcinoma.](https://pubmed.ncbi.nlm.nih.gov/22327624)" (2012) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 15.78, "[KIF5B-ALK, a novel fusion oncokinase identified by an immunohistochemistry-based diagnostic system for ALK-positive lung cancer.](https://pubmed.ncbi.nlm.nih.gov/19383809)" (2009) with an RCR of 11.94, "[Axonal transport of mitochondria requires milton to recruit kinesin heavy chain and is light chain independent.](https://pubmed.ncbi.nlm.nih.gov/16717129)" (2006) with an RCR of 8.79, "[A transforming KIF5B and RET gene fusion in lung adenocarcinoma revealed from whole-genome and transcriptome sequencing.](https://pubmed.ncbi.nlm.nih.gov/22194472)" (2012) with an RCR of 8.77, and "[A kinesin heavy chain (KIF5A) mutation in hereditary spastic paraplegia (SPG10).](https://pubmed.ncbi.nlm.nih.gov/12355402)" (2002) with an RCR of 6.47. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[KIF5B](https://www.proteinatlas.org/ENSG00000170759-KIF5B) is a motor protein involved in cancer-related genes, with evidence at the protein level. It is detected in all tissues and is localized in the centriolar satellite, cytosol, and end piece. KIF5B is part of Cluster 10 for blood expression, which is non-specific and related to membrane trafficking. In tissue expression, it belongs to Cluster 9, associated with oligodendrocytes and myelination. For brain expression, it is in Cluster 13, linked to oligodendrocytes with mixed function. In cell line expression, it is part of Cluster 52, which is non-specific and related to mitochondria. Additionally, KIF5B is found in Cluster 32 for single cell expression, associated with smooth muscle cells and muscle contraction.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SPI1](https://www.ncbi.nlm.nih.gov/gene/6688) with 5 experiments, [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 3 experiments, [MYOD1](https://www.ncbi.nlm.nih.gov/gene/4654) with 3 experiments, [SMC3](https://www.ncbi.nlm.nih.gov/gene/9126) with 3 experiments, and [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113) with 3 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with various disease conditions, including diseases of metabolism, acquired metabolic diseases, nutrition diseases, overnutrition, obesity, cognitive disorders, diseases of mental health, bipolar disorder, and mood disorders.



The gene is expressed in various tissues, including blood vessel and brain, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the trachea, pituitary, uterus, and multiple regions of the testis, including interstitial, Leydig cell, seminiferous tubule, and germ cell areas.


The proteins are involved in various pathways, including metabolism of proteins, adaptive immune system, immune system, factors involved in megakaryocyte development and platelet production, kinesins, MHC class II antigen presentation, insulin processing, peptide hormone metabolism, and hemostasis.



The analyzed protein sequence has a GRAVY value of -0.8098 (11.24th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -9.64 (23.05th percentile), and the median structural flexibility is 1.013 (87.9th percentile). The protein has a high affinity for helix structures (39.67%, 92.79th percentile), with lower affinities for sheet (29.91%, 27.86th percentile) and turn (23.36%, 11.7th percentile) structures. The instability index is 40.95 (25.13th percentile), and the isoelectric point is 6.12 (33.66th percentile). The protein is 963 amino acids long (86.64th percentile) with a molecular weight of 109683.61 Da (87.11th percentile). For more details on the sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |