---
layout: post
title: HLA-DOA
date: 2025-01-17 16:55 CST
description: HLA-DOA description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3111) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3111  | HLA-DOA | ENSG00000204252 | 6p21.32 |



The gene is involved in various biological processes, including [adaptive immune response](https://amigo.geneontology.org/amigo/term/GO:0002250), [peptide antigen assembly with MHC class II protein complex](https://amigo.geneontology.org/amigo/term/GO:0002503), and [negative regulation of antigen processing and presentation of peptide antigen via MHC class II](https://amigo.geneontology.org/amigo/term/GO:0002587). It is also involved in [antigen processing and presentation of exogenous peptide antigen via MHC class II](https://amigo.geneontology.org/amigo/term/GO:0019886), [regulation of T cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0045580), [positive regulation of immune response](https://amigo.geneontology.org/amigo/term/GO:0050778), and [positive regulation of T cell activation](https://amigo.geneontology.org/amigo/term/GO:0050870). The gene enables functions such as [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [MHC class II protein complex binding](https://amigo.geneontology.org/amigo/term/GO:0023026), [MHC class II receptor activity](https://amigo.geneontology.org/amigo/term/GO:0032395), and [peptide antigen binding](https://amigo.geneontology.org/amigo/term/GO:0042605). It is part of the [MHC class II protein complex](https://amigo.geneontology.org/amigo/term/GO:0042613) and is active in the [lysosomal membrane](https://amigo.geneontology.org/amigo/term/GO:0005765) and [late endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0031902). Additionally, the gene is located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886).


The gene length is 16,974 base pairs (38.0% of all genes), the mature length is 1,865 base pairs, and the primary transcript length is 12,850 base pairs.


The gene HLA-DOA, with NCBI ID 3111, has been mentioned in [75 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22HLA-DOA%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest publication mentioning this gene was in 1965, and the middle 50% of publications occurred between 1986 and 2002.


The top 5 publications mentioning HLA-DOA, ranked by their scientific influence, include "[Cutaneous lymphocyte antigen is a specialized form of PSGL-1 expressed on skin-homing T cells.](https://pubmed.ncbi.nlm.nih.gov/9353122)" (1997) (relative citation ratio: 7.88), "[T-lymphocyte-antigen interactions in transplant rejection.](https://pubmed.ncbi.nlm.nih.gov/2405272)" (1990) (relative citation ratio: 6.17), "[IL-31 is associated with cutaneous lymphocyte antigen-positive skin homing T cells in patients with atopic dermatitis.](https://pubmed.ncbi.nlm.nih.gov/16461143)" (2006) (relative citation ratio: 5.66), "[Similarities in amino acid sequences of Aplysia ADP-ribosyl cyclase and human lymphocyte antigen CD38.](https://pubmed.ncbi.nlm.nih.gov/1471258)" (1992) (relative citation ratio: 4.62), and "[Human lymphocyte antigen expression in hydatidiform mole: androgenesis following fertilization by a haploid sperm.](https://pubmed.ncbi.nlm.nih.gov/92196)" (1979) (relative citation ratio: 4.5). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[HLA-DOA](https://www.proteinatlas.org/ENSG00000204252-HLA-DOA) is a gene with evidence at the protein level. Its RNA is detected in many tissues, and it is notably expressed in B-cells within the blood, lymphoid tissue involved in immune response, the hindbrain, lymphoma cell lines, and dendritic cells involved in antigen presentation. The gene's expression has been found to be unprognostic in various cancer types, including Bladder Urothelial Carcinoma, Breast Invasive Carcinoma, Cervical Squamous Cell Carcinoma, and Colon Adenocarcinoma, among others.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), [EZH2](https://www.ncbi.nlm.nih.gov/gene/2146), and [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046), each shown to be regulating in 3 experiments. Additionally, [CEBPB](https://www.ncbi.nlm.nih.gov/gene/1051) and [GFI1](https://www.ncbi.nlm.nih.gov/gene/2672) were found to be regulating in 2 experiments each.



The GWAS data indicates associations with a wide range of disease conditions, including infectious diseases such as viral infectious diseases and human immunodeficiency virus infectious diseases. Additionally, the data highlights associations with hematopoietic and immune system diseases, as well as various cardiovascular conditions such as coronary artery disease, artery disease, and vascular disease. Neurological disorders, including neurodegenerative diseases like Parkinson's disease and amyotrophic lateral sclerosis, are also implicated. Furthermore, the data shows links to musculoskeletal system diseases, including arthritis and rheumatoid arthritis, and gastrointestinal system diseases such as periodontitis and periodontal disease.



The gene is expressed in various tissues and cell types, including blood and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in cardiac myocytes, Burkitt's lymphoma (Raji), heart, liver, CD19+ B cells (neg. sel.), bone marrow, and skeletal muscle.


The proteins are involved in several key immune-related pathways, including the adaptive immune system, the broader immune system, and MHC class II antigen presentation.



The analyzed protein sequence has a GRAVY value of 0.133 (88.88th percentile), indicating a relatively hydrophobic nature. The protein exhibits a charge of -6.795 at pH 7.0 (28.98th percentile) and an isoelectric point of 5.57 (19.97th percentile). The median structural flexibility is 0.991 (13.23rd percentile), with an instability index of 49.24 (54.66th percentile). The protein's secondary structure is predicted to be predominantly sheet (40.4%, 87.43rd percentile), with notable fractions in helix (28.8%, 22.1st percentile) and turn (28.4%, 50.96th percentile) conformations. The protein has a length of 250 amino acids (21.69th percentile) and a molecular weight of 27598.46 Da (21.34th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |