---
layout: post
title: EIF4H
date: 2025-01-17 16:55 CST
description: EIF4H description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7458) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7458  | EIF4H | ENSG00000106682 | 7q11.23 |



The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and [translation initiation factor activity](https://amigo.geneontology.org/amigo/term/GO:0003743), and is involved in [translational initiation](https://amigo.geneontology.org/amigo/term/GO:0006413) and its [regulation](https://amigo.geneontology.org/amigo/term/GO:0006446). It is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829) and the [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), and is part of the [eukaryotic translation initiation factor 4F complex](https://amigo.geneontology.org/amigo/term/GO:0016281). Additionally, it plays a role in [sexual reproduction](https://amigo.geneontology.org/amigo/term/GO:0019953) and [developmental growth](https://amigo.geneontology.org/amigo/term/GO:0048589), with functions in [cadherin binding](https://amigo.geneontology.org/amigo/term/GO:0045296) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is also found in the [perinuclear region of cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0048471).


The gene length is 27,950 base pairs (50.69% of all genes), the mature length is 1,124 base pairs (26.66% of all genes), and the primary transcript length is 18,154 base pairs (49.13% of all genes).


EIF4H (Gene ID: 7458) has been mentioned in [11 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22EIF4H%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning EIF4H was in 2001, and the middle 50% of publications occurred between 2006 and 2016.


The top 5 publications mentioning EIF4H, ranked by their scientific influence, include studies that explore the modulation of helicase activity by eIF4A and its interaction with eIF4B and eIF4H, as well as the role of EIF4H in mRNA decay during herpes simplex virus infections. These publications are evaluated using the [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, set at 1.0. The most influential study, "[Modulation of the helicase activity of eIF4A by eIF4B, eIF4H, and eIF4F.](https://pubmed.ncbi.nlm.nih.gov/11418588)" (2001), has an RCR of 4.1. Other notable works include "[mRNA decay during herpes simplex virus (HSV) infections: protein-protein interactions involving the HSV virion host shutoff protein and translation factors eIF4H and eIF4A.](https://pubmed.ncbi.nlm.nih.gov/16014927)" (2005) with an RCR of 1.79, and "[eIF4B and eIF4H mediate GR production from expanded G4C2 in a Drosophila model for C9orf72-associated ALS.](https://pubmed.ncbi.nlm.nih.gov/31023341)" (2019) with an RCR of 1.39. Additionally, "[Interactions between eIF4AI and its accessory factors eIF4B and eIF4H.](https://pubmed.ncbi.nlm.nih.gov/18719248)" (2008) has an RCR of 1.36, and "[Herpes simplex virus virion host shutoff protein is stimulated by translation initiation factors eIF4B and eIF4H.](https://pubmed.ncbi.nlm.nih.gov/15078951)" (2004) has an RCR of 1.05. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[EIF4H](https://www.proteinatlas.org/ENSG00000106682-EIF4H) is an RNA-binding initiation factor involved in various cellular processes. It is implicated in cancer-related genes and Williams-Beuren syndrome, with evidence supporting its function at the protein level. EIF4H is detected in all tissues and is localized in the nucleoplasm, nuclear bodies, and cytosol. Its expression is associated with basic cellular processes across different tissue types, including vesicular transport in blood, transcription in cell lines, and spermatogenesis in single cells.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [TAF1](https://www.ncbi.nlm.nih.gov/gene/6872) and [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313), each with 6 experiments. Additionally, [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113), [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), and [NELFE](https://www.ncbi.nlm.nih.gov/gene/7936) were each found to be regulating in 5 experiments.





The gene is expressed in the esophagus and skin, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles show expression in Burkitt's lymphoma (Raji), chronic myelogenous leukemia (K-562), colorectal adenocarcinoma, and placenta.


The input data highlights various pathways involved in protein metabolism and gene expression. Key processes include '3'-UTR-mediated translational regulation, L13a-mediated translational silencing of Ceruloplasmin expression, and GTP hydrolysis and joining of the 60S ribosomal subunit. Additionally, the data covers cap-dependent translation initiation, eukaryotic translation initiation, and general translation processes. It also details the activation of mRNA upon binding of the cap-binding complex and eIFs, subsequent binding to 43S, translation initiation complex formation, and ribosomal scanning and start codon recognition.


The mouse ortholog gene 22384 and its human ortholog 7458 are associated with several phenotypic abnormalities, including abnormal embryo size, abnormal retina inner nuclear layer morphology, decreased total retina thickness, embryonic growth retardation, hemorrhage, preweaning lethality with complete penetrance, and thin ventricular wall. These phenotypes are observed in both sexes.


The analyzed protein sequence has a GRAVY value of -1.0088709677419354 (4.23rd percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -0.36312113448452266 (48.79th percentile), and the median structural flexibility is 1.0171309523809522 (94.42nd percentile). The protein has a length of 248 amino acids (21.41st percentile) and a molecular weight of 27384.732599999967 Da (21st percentile). The secondary structure affinities are 0.22983870967741937 for helix (2.45th percentile), 0.25 for sheet (8.71st percentile), and 0.4153225806451613 for turn (98.15th percentile). The instability index is 39.682661290322585 (21.23rd percentile), and the isoelectric point is 6.6716699600219735 (45.78th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |