---
layout: post
title: SRSF11
date: 2025-01-17 16:55 CST
description: SRSF11 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9295) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9295  | SRSF11 | ENSG00000116754 | 1p31.1 |



The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), and is active in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654). Additionally, it is involved in [mRNA processing](https://amigo.geneontology.org/amigo/term/GO:0006397) and [RNA splicing](https://amigo.geneontology.org/amigo/term/GO:0008380), and is also located in the [nuclear speck](https://amigo.geneontology.org/amigo/term/GO:0016607).


The gene length is 70,671.0 base pairs (74.75th percentile), the mature length is 2,901.5 base pairs, and the primary transcript length is 70,277.0 base pairs.


SRSF11 has been mentioned in [3 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SRSF11%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest publication mentioning SRSF11 was in 1998, and the middle 50% of publications occurred between 2002 and 2013.


The top publications mentioning SRSF11 include "[The RNA-splicing factor PSF/p54 controls DNA-topoisomerase I activity by a direct interaction.](https://pubmed.ncbi.nlm.nih.gov/9756848)" (1998) (relative citation ratio: 1.24), "[SRp54 (SFRS11), a regulator for tau exon 10 alternative splicing identified by an expression cloning strategy.](https://pubmed.ncbi.nlm.nih.gov/16943417)" (2006) (relative citation ratio: 0.89), and "[SFRS11 Loss Leads to Aging-Associated Cognitive Decline by Modulating LRP8 and ApoE.](https://pubmed.ncbi.nlm.nih.gov/32553159)" (2020) (relative citation ratio: 0.07). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[SRSF11](https://www.proteinatlas.org/ENSG00000116754-SRSF11) is an RNA-binding protein with evidence at the protein level. It is detected in all RNA tissue distributions and is primarily located in nuclear speckles. The gene is part of various expression clusters, including Cluster 5 for blood expression related to non-specific cell proliferation, Cluster 66 for tissue expression related to non-specific transcription, Cluster 4 for brain expression related to non-specific mRNA splicing and cell cycle, Cluster 37 for cell line expression related to non-specific mRNA processing, and Cluster 55 for single cell expression related to non-specific transcription.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [HDAC2](https://www.ncbi.nlm.nih.gov/gene/3066), [MYOD1](https://www.ncbi.nlm.nih.gov/gene/4654), [RELA](https://www.ncbi.nlm.nih.gov/gene/5970), and [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), each with 5 experiments. Additionally, [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313) was found to be regulating in 4 experiments.



The GWAS data indicates associations with various mental health conditions, including cognitive disorder, mood disorder, endogenous depression, major depressive disorder, and mental depression.



The gene is expressed in various tissues, including nerve, ovary, and thyroid, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles show expression in the Prefrontal Cortex, CD34+ cells, CD4+ T cells, CD8+ T cells, and the Uterus.


The input data highlights various pathways involved in gene expression and RNA processing. These include RNA polymerase II transcription, post-elongation processing of the transcript, mRNA 3'-end processing, and mRNA splicing. Additionally, the data covers the transport of mature transcripts to the cytoplasm and the cleavage of growing transcripts in the termination region. These pathways collectively contribute to the regulation and maturation of mRNA, ensuring proper gene expression.


The mouse ortholog gene 69207 and its human ortholog gene 9295 are associated with phenotypes including embryonic lethality prior to organogenesis, which is not observed in both sexes, and preweaning lethality with complete penetrance, which is observed in both sexes.


The analyzed protein sequence has a GRAVY value of -1.2394628099173546 (1.64 percentile), indicating a hydrophilic nature. It exhibits a charge of 48.4337109009833 at pH 7.0 (99.19 percentile) and a median structural flexibility of 1.0226190476190478 (97.96 percentile). The protein's secondary structure affinities are 32.23% for helix (52.48 percentile), 20.66% for sheet (2.51 percentile), and 36.57% for turn (92.88 percentile). The instability index is 97.74214876033083 (99.52 percentile), with an isoelectric point of 10.522223472595215 (98.37 percentile). The protein has a length of 484 amino acids (56.37 percentile) and a molecular weight of 53541.54200000015 Da (55.65 percentile). For more details on sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |