---
layout: post
title: SKAP2
date: 2025-01-17 16:55 CST
description: SKAP2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8935) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8935  | SKAP2 | ENSG00000005020 | 7p15.2 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654). It is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737) and located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). Additionally, it is active in and located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). The gene is involved in [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), [negative regulation of cell population proliferation](https://amigo.geneontology.org/amigo/term/GO:0008285), and [B cell activation](https://amigo.geneontology.org/amigo/term/GO:0042113).


The gene length is 30,779 base pairs (53.42% of all genes), the mature length is 1,120 base pairs, and the primary transcript length is 28,107 base pairs.


The gene SKAP2, with NCBI ID 8935, has been mentioned in [51 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SKAP2%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727). The earliest year of publication mentioning SKAP2 is 1932, and the middle 50% of publications occurred between 1998 and 2018. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning SKAP2, ranked by their scientific influence, include "[ICU severity of illness scores: APACHE, SAPS and MPM.](https://pubmed.ncbi.nlm.nih.gov/25137401)" (2014) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 6.03, "[Comparison of Charlson comorbidity index with SAPS and APACHE scores for prediction of mortality following intensive care.](https://pubmed.ncbi.nlm.nih.gov/21750629)" (2011) with an RCR of 4.99, "[Converting positive and negative symptom scores between PANSS and SAPS/SANS.](https://pubmed.ncbi.nlm.nih.gov/24332632)" (2014) with an RCR of 3.92, "[An Assessment of Five (PANSS, SAPS, SANS, NSA-16, CGI-SCH) commonly used Symptoms Rating Scales in Schizophrenia and Comparison to Newer Scales (CAINS, BNSS).](https://pubmed.ncbi.nlm.nih.gov/29430333)" (2017) with an RCR of 3.59, and "[A study of the interrelationship between and comparative interrater reliability of the SAPS, SANS and PANSS.](https://pubmed.ncbi.nlm.nih.gov/9147498)" (1996) with an RCR of 3.23. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[SKAP2](https://www.proteinatlas.org/ENSG00000005020-SKAP2) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in the nucleoplasm and cytosol. SKAP2 is expressed in dendritic cells with unknown function (Cluster 2) and is associated with immune response in lymphoid tissue (Cluster 70). Additionally, it is involved in mixed functions in the brain (Cluster 22), keratinization in squamous epithelial cells (Cluster 53), and degranulation in neutrophils (Cluster 74).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) with 6 experiments, [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 5 experiments, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 5 experiments, [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664) with 5 experiments, and [STAT1](https://www.ncbi.nlm.nih.gov/gene/6772) with 5 experiments.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [2623](https://www.ncbi.nlm.nih.gov/gene/2623) and [9968](https://www.ncbi.nlm.nih.gov/gene/9968). These TFs were found to regulate gene expression in 4 and 1 Perturb-seq experiments, respectively.


The input data from GWAS indicates associations with a range of disease conditions, including lower respiratory tract disease, lung disease, respiratory system disease, disease of metabolism, acquired metabolic disease, carbohydrate metabolism disease, glucose metabolism disease, diabetes mellitus, type 2 diabetes mellitus, kidney disease, urinary system disease, nervous system disease, central nervous system disease, neurodegenerative disease, type 1 diabetes mellitus, demyelinating disease, multiple sclerosis, inflammatory bowel disease, intestinal disease, Crohn's disease, and gastrointestinal system disease.


The gene SKAP2 has been associated with [Body mass index](https://pubmed.ncbi.nlm.nih.gov/36581621), which maps closely to the MeSH term [Body Mass Index](https://meshb.nlm.nih.gov/record/ui?ui=D015992) using the method [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0).


The gene is expressed in various blood-related cell types and tissues, including B lymphoblasts, CD34+ cells, CD33+ myeloid cells, BDCA4+ dendritic cells, CD56+ NK cells, CD19+ B cells, CD14+ monocytes, and whole blood. This indicates a ubiquitous expression pattern in multiple tissue samples related to the hematopoietic system.


The input data highlights two key pathways involving protein interactions: signal regulatory protein (SIRP) family interactions and cell-cell communication. These pathways are crucial for various biological processes, including cell signaling and intercellular communication.



The analyzed protein sequence has a GRAVY value of -0.809 (11.27th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -29.1 (6.41st percentile), and the median structural flexibility is 1.013 (87.75th percentile). The protein has a high instability index of 54.6 (71.04th percentile) and an isoelectric point of 4.54 (1.79th percentile). The sequence length is 359 amino acids (39.48th percentile) with a molecular weight of 41216.3 Da (41.08th percentile). The secondary structure affinities are 33.4% for helix (63.09th percentile), 30.6% for sheet (31.7th percentile), and 32.6% for turn (79.82nd percentile). These calculations were performed using BioPython's sequence analysis tools, which can be referenced [here](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |