---
layout: post
title: GABARAP
date: 2024-12-08 23:57 CST
description: GABARAP description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/11337) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 11337  | GABARAP | ENSG00000170296 | 17p13.1 |



The gene length is 3,967 base pairs (11.1% of all genes), the mature length is 2,922 base pairs, and the primary transcript length is 3,967 base pairs.


GABARAP has been studied in 284 publications, with the earliest publication in 1972 and the middle 50% of publications occurring between 1986 and 2015. For the most up-to-date publication count, please refer to the [284 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22GABARAP%22) (note that the count may differ since it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)).


The top five publications mentioning GABARAP, ranked by their scientific influence, are as follows: ["(-)Baclofen decreases neurotransmitter release in the mammalian CNS by an action at a novel GABA receptor."](https://pubmed.ncbi.nlm.nih.gov/6243177) (1980) (relative citation ratio: 32.79), ["Differential regulation of AMPA receptor and GABA receptor trafficking by tumor necrosis factor-alpha."](https://pubmed.ncbi.nlm.nih.gov/15788779) (2005) (relative citation ratio: 16.13), ["Atg8 family LC3/GABARAP proteins are crucial for autophagosome-lysosome fusion but not autophagosome formation during PINK1/Parkin mitophagy and starvation."](https://pubmed.ncbi.nlm.nih.gov/27864321) (2016) (relative citation ratio: 14.95), ["GABA receptor mechanisms in the central nervous system."](https://pubmed.ncbi.nlm.nih.gov/1847747) (1991) (relative citation ratio: 14.29), and ["Cloning of the gamma-aminobutyric acid (GABA) rho 1 cDNA: a GABA receptor subunit highly expressed in the retina."](https://pubmed.ncbi.nlm.nih.gov/1849271) (1991) (relative citation ratio: 12.11). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [RELA](https://www.ncbi.nlm.nih.gov/gene/5970), [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), [SUPT5H](https://www.ncbi.nlm.nih.gov/gene/6829), and [CREB1](https://www.ncbi.nlm.nih.gov/gene/1385), each with 5 experiments. Additionally, [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046) was found to be regulating in 4 experiments.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [6829](https://www.ncbi.nlm.nih.gov/gene/6829) with 5 experiments, [9968](https://www.ncbi.nlm.nih.gov/gene/9968) with 4 experiments, [2623](https://www.ncbi.nlm.nih.gov/gene/2623) and [8243](https://www.ncbi.nlm.nih.gov/gene/8243) each with 3 experiments, and [9126](https://www.ncbi.nlm.nih.gov/gene/9126) with 3 experiments. These TFs were found to regulate gene expression in multiple Perturb-seq experiments, highlighting their significant roles in transcriptional regulation.








The analyzed protein sequence has a GRAVY value of -0.534 (31.59th percentile), indicating a hydrophilic nature. It exhibits a charge of 1.87 at pH 7.0 (57.07th percentile) and a median structural flexibility of 1.005 (62.48th percentile). The protein's secondary structure is predicted to be 34.19% helix (69.27th percentile), 38.46% sheet (80.82nd percentile), and 22.22% turn (7.21st percentile). The instability index is 46.09 (43.25th percentile), with an isoelectric point of 8.73 (71.95th percentile). The protein has a length of 117 amino acids (4.34th percentile) and a molecular weight of 13917.86 Da (5.21st percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |