---
layout: post
title: PTPN3
date: 2025-01-17 16:55 CST
description: PTPN3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5774) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5774  | PTPN3 | ENSG00000070159 | 9q31.3 |



The gene is involved in the [MAPK cascade](https://amigo.geneontology.org/amigo/term/GO:0000165) and [protein dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0006470). It enables [phosphotyrosine residue binding](https://amigo.geneontology.org/amigo/term/GO:0001784), [protein tyrosine phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0004725), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [cytoskeletal protein binding](https://amigo.geneontology.org/amigo/term/GO:0008092), [ATPase binding](https://amigo.geneontology.org/amigo/term/GO:0051117), and [sodium channel regulator activity](https://amigo.geneontology.org/amigo/term/GO:0017080). It is active in and located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0005856), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), and the [cytoplasmic side of plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009898). Additionally, it is involved in the negative regulation of the [epidermal growth factor receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0042059), [mitotic cell cycle](https://amigo.geneontology.org/amigo/term/GO:0045930), and [membrane protein ectodomain proteolysis](https://amigo.geneontology.org/amigo/term/GO:0051045). The gene also plays a role in [liver regeneration](https://amigo.geneontology.org/amigo/term/GO:0097421) and the regulation of [membrane depolarization during action potential](https://amigo.geneontology.org/amigo/term/GO:0098902) and [sodium ion transmembrane transporter activity](https://amigo.geneontology.org/amigo/term/GO:2000649).


The gene length is 7,564 base pairs (20.28% of all genes), the mature length is 1,097 base pairs (25.83% of all genes), and the primary transcript length is 6,579 base pairs (23.82% of all genes).


The gene PTPN3 (NCBI ID: 5774) has been mentioned in [39 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PTPN3%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1993 and the middle 50% of publications occurring between 2006 and 2019. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning PTPN3, ranked by their scientific influence, include "[Activating mutations in PTPN3 promote cholangiocarcinoma cell proliferation and migration and are associated with tumor recurrence in patients.](https://pubmed.ncbi.nlm.nih.gov/24503127)" (2014) with a relative citation ratio of 3.26, "[Discovery of ALK-PTPN3 gene fusion from human non-small cell lung carcinoma cell line using next generation RNA sequencing.](https://pubmed.ncbi.nlm.nih.gov/22334442)" (2012) with a relative citation ratio of 1.68, "[Identification of the cell cycle regulator VCP (p97/CDC48) as a substrate of the band 4.1-related protein-tyrosine phosphatase PTPH1.](https://pubmed.ncbi.nlm.nih.gov/10364224)" (1999) with a relative citation ratio of 1.46, "[Protein tyrosine phosphatase PTPN3 inhibits lung cancer cell proliferation and migration by promoting EGFR endocytic degradation.](https://pubmed.ncbi.nlm.nih.gov/25263444)" (2015) with a relative citation ratio of 1.43, and "[Serine phosphorylation-dependent association of the band 4.1-related protein-tyrosine phosphatase PTPH1 with 14-3-3beta protein.](https://pubmed.ncbi.nlm.nih.gov/9341175)" (1997) with a relative citation ratio of 1.41. The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PTPN3](https://www.proteinatlas.org/ENSG00000070159-PTPN3) is a gene encoding a protein with hydrolase and protein phosphatase molecular functions. Evidence for this protein is established at the protein level. The RNA expression of PTPN3 is detected in many tissues and is associated with basic cellular processes in tissue expression cluster 19. In the brain, it is part of cluster 38, which includes sub-cortical regions with mixed functions. In cell lines, PTPN3 is found in cluster 15, associated with kidney cancer membrane components. Single-cell expression analysis places PTPN3 in cluster 37, which includes glandular and luminal cells with unknown functions.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227), [AR](https://www.ncbi.nlm.nih.gov/gene/367), and [EZH2](https://www.ncbi.nlm.nih.gov/gene/2146), each shown to be regulating in 4 experiments.



The GWAS data indicates associations with a range of conditions, including artery disease, vascular disease, cardiovascular system disease, hypertension, kidney disease, urinary system disease, cognitive disorder, disease of mental health, bipolar disorder, and mood disorder.


The gene PTPN3 has been associated with [Gamma glutamyl transpeptidase](https://pubmed.ncbi.nlm.nih.gov/34594039), which maps closely to the MeSH term [gamma-Glutamyltransferase](https://meshb.nlm.nih.gov/record/ui?ui=D005723) through a direct synonym match using [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0).


The gene is expressed in various tissues, including muscle, liver, and kidney, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the thalamus, bronchial epithelial cells, colorectal adenocarcinoma, placenta, pancreas, pancreatic islet, and leukemia lymphoblastic (MOLT-4) cells.




The analyzed protein sequence has a GRAVY value of -0.486 (36.47th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -4.602 (34.74th percentile). The median structural flexibility is 1.002 (48.85th percentile), with a predicted secondary structure composition of 27.49% helix (14.29th percentile), 34.94% sheet (58.59th percentile), and 29.46% turn (59.75th percentile). The instability index is 47.88 (49.79th percentile), and the isoelectric point is 6.58 (44.22nd percentile). The protein is 913 amino acids long (84.89th percentile) with a molecular weight of 103988.67 Da (85.67th percentile). For more details on the sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |