---
layout: post
title: EDAR
date: 2025-01-17 16:55 CST
description: EDAR description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10913) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10913  | EDAR | ENSG00000135960 | 2q13  |



The gene is involved in various biological processes, including [hair follicle development](https://amigo.geneontology.org/amigo/term/GO:0001942), [apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0006915), [epidermis development](https://amigo.geneontology.org/amigo/term/GO:0008544), and [odontogenesis of dentin-containing tooth](https://amigo.geneontology.org/amigo/term/GO:0042475). It enables [transmembrane signaling receptor activity](https://amigo.geneontology.org/amigo/term/GO:0004888) and [signaling receptor activity](https://amigo.geneontology.org/amigo/term/GO:0038023), and is involved in [positive regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0010628), [cytokine-mediated signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0019221), and [cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0030154). Additionally, it plays a role in [positive regulation of canonical NF-kappaB signal transduction](https://amigo.geneontology.org/amigo/term/GO:0043123), [positive regulation of JNK cascade](https://amigo.geneontology.org/amigo/term/GO:0046330), and [positive regulation of non-canonical NF-kappaB signal transduction](https://amigo.geneontology.org/amigo/term/GO:1901224). The gene is located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886) and the [apical part of the cell](https://amigo.geneontology.org/amigo/term/GO:0045177), and is also found in the [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020). Furthermore, it is involved in [pigmentation](https://amigo.geneontology.org/amigo/term/GO:0043473) and [salivary gland cavitation](https://amigo.geneontology.org/amigo/term/GO:0060662).


The gene length is 57,112.0 base pairs (69.83% of all genes), the mature length is 699.0 base pairs (9.26% of all genes), and the primary transcript length is 26,852.0 base pairs (59.7% of all genes).


The gene EDAR (NCBI ID: 10913) has been mentioned in [73 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22EDAR%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1972 and the middle 50% of publications occurring between 2008 and 2018. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning EDAR, ranked by their scientific influence, include "[Only four genes (EDA1, EDAR, EDARADD, and WNT10A) account for 90% of hypohidrotic/anhidrotic ectodermal dysplasia cases.](https://pubmed.ncbi.nlm.nih.gov/20979233)" (2011) (relative citation ratio: 6.0), "[Reciprocal requirements for EDA/EDAR/NF-kappaB and Wnt/beta-catenin signaling pathways in hair follicle induction.](https://pubmed.ncbi.nlm.nih.gov/19619491)" (2009) (relative citation ratio: 5.35), "[A genome-wide association scan implicates DCHS2, RUNX2, GLI3, PAX1 and EDAR in human facial variation.](https://pubmed.ncbi.nlm.nih.gov/27193062)" (2016) (relative citation ratio: 4.36), "[Characterization of an antigenic site that contains a dominant, type-specific neutralization determinant on the envelope protein domain III (ED3) of dengue 2 virus.](https://pubmed.ncbi.nlm.nih.gov/17719070)" (2007) (relative citation ratio: 3.57), and "[Zebrafish eda and edar mutants reveal conserved and ancestral roles of ectodysplasin signaling in vertebrates.](https://pubmed.ncbi.nlm.nih.gov/18833299)" (2008) (relative citation ratio: 3.25). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[EDAR](https://www.proteinatlas.org/ENSG00000135960-EDAR) is a developmental protein that functions as a receptor. It is implicated in ectodermal dysplasia, with evidence supporting its role at the protein level. EDAR is detected in some RNA tissue distributions and is expressed in epithelium and extracellular exosomes. It is also found in squamous epithelial cells, where it is involved in keratinization, and in T-cells, specifically related to the T-cell receptor. EDAR is part of the non-specific transcription and translation blood expression cluster.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [TP63](https://www.ncbi.nlm.nih.gov/gene/8626) and [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), each shown to be regulating in 3 experiments. Additionally, [SUZ12](https://www.ncbi.nlm.nih.gov/gene/23512), [CREB1](https://www.ncbi.nlm.nih.gov/gene/1385), and [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930) were each found to be regulating in 2 experiments.



The input data from GWAS indicates associations with a wide range of disease conditions, including various types of cancer such as colorectal cancer, intestinal cancer, large intestine cancer, and gastrointestinal system cancer. Additionally, the data highlights immune system diseases, metabolic disorders, and cardiovascular system diseases, including heart disease. Furthermore, there are associations with musculoskeletal system diseases, such as arthritis and rheumatoid arthritis, as well as bone diseases and connective tissue diseases. The data also includes primary immunodeficiency diseases, such as B cell deficiency and dysgammaglobulinemia, along with selective immunoglobulin deficiency diseases. Lastly, there are associations with mouth and tooth diseases.



The gene is expressed in various tissues, including the bladder, colorectal adenocarcinoma, pancreas, superior cervical ganglion, trigeminal ganglion, atrioventricular node, skin, and dorsal root ganglion.



The mouse gene with NCBI ID 13608 and its human ortholog with NCBI ID 10913 are associated with the phenotype of preweaning lethality with incomplete penetrance, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.275 (60.83rd percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -16.84 (13.7th percentile), and the median structural flexibility is 1.004 (61.1th percentile). The protein has a high affinity for helix (33.71%, 65.48th percentile) and turn (31.03%, 70.73rd percentile) structures, with a moderate affinity for sheet structures (30.58%, 31.36th percentile). The instability index is 51.51 (62.01st percentile), and the isoelectric point is 5.08 (9.15th percentile). The protein is 448 amino acids long (51.65th percentile) with a molecular weight of 48581.68 Da (50.13th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |