---
layout: post
title: SOCS1
date: 2024-12-08 23:57 CST
description: SOCS1 description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8651) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8651  | SOCS1 | ENSG00000185338 | 16p13.13 |



The gene length is 25,281.0 base pairs (47.97% of all genes), the mature length is 2,914.0 base pairs, and the primary transcript length is 25,281.0 base pairs.


The gene SOCS1, with NCBI ID 8651, has been featured in [825 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SOCS1%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1946, and the middle 50% of publications occurred between 2006 and 2018.


The top 5 publications mentioning SOCS1, ranked by their scientific influence, include "[MicroRNA-155 functions as an OncomiR in breast cancer by targeting the suppressor of cytokine signaling 1 gene.](https://pubmed.ncbi.nlm.nih.gov/20354188)" (2010) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 13.27, "[SOCS-1, a negative regulator of the JAK/STAT pathway, is silenced by methylation in human hepatocellular carcinoma and shows growth-suppression activity.](https://pubmed.ncbi.nlm.nih.gov/11326271)" (2001) with an RCR of 10.59, "[A small jab - a big effect: nonspecific immunomodulation by vaccines.](https://pubmed.ncbi.nlm.nih.gov/23680130)" (2013) with an RCR of 9.98, "[Suppressor of cytokine signaling 1 negatively regulates Toll-like receptor signaling by mediating Mal degradation.](https://pubmed.ncbi.nlm.nih.gov/16415872)" (2006) with an RCR of 8.14, and "[Inducible microRNA-155 feedback promotes type I IFN signaling in antiviral innate immunity by targeting suppressor of cytokine signaling 1.](https://pubmed.ncbi.nlm.nih.gov/20937844)" (2010) with an RCR of 8.0. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [MYC](https://www.ncbi.nlm.nih.gov/gene/4609), [SMC3](https://www.ncbi.nlm.nih.gov/gene/9126), [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908), [KLF5](https://www.ncbi.nlm.nih.gov/gene/688), and [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), each shown to be regulating in 5 experiments.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, filtered for Perturb-seq data, include [CTCF](https://www.ncbi.nlm.nih.gov/gene/6829) with 4 experiments, [ZNF274](https://www.ncbi.nlm.nih.gov/gene/8243) with 2 experiments, and [GATA2](https://www.ncbi.nlm.nih.gov/gene/2623), [KLF6](https://www.ncbi.nlm.nih.gov/gene/8861), and [RORA](https://www.ncbi.nlm.nih.gov/gene/9968) each with 1 experiment. These TFs were found to regulate gene expression in multiple Perturb-seq experiments, highlighting their significant roles in transcriptional regulation.





The mouse ortholog gene 12703 and its human ortholog gene 8651 are associated with the phenotype of preweaning lethality with complete penetrance, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.357 (51.35th percentile), indicating a hydrophilic nature. It exhibits a charge of 13.18 at pH 7.0 (86.9th percentile) and a median structural flexibility of 0.995 (20.83rd percentile). The protein's secondary structure is predicted to be 27.01% helix (11.82nd percentile), 28.91% sheet (22.75th percentile), and 29.38% turn (59.14th percentile). The instability index is 69.57 (94.35th percentile), suggesting high instability. The isoelectric point is 10.98 (99th percentile), with a molecular weight of 23550.62 Da (16.12th percentile) and a length of 211 amino acids (16.15th percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |