---
layout: post
title: IL7R
date: 2025-01-17 16:55 CST
description: IL7R description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3575) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3575  | IL7R | ENSG00000168685 | 5p13.2 |



The gene is involved in various biological processes, including [regulation of DNA recombination](https://amigo.geneontology.org/amigo/term/GO:0000018), [cell morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0000902), [B cell homeostasis](https://amigo.geneontology.org/amigo/term/GO:0001782), [T cell mediated cytotoxicity](https://amigo.geneontology.org/amigo/term/GO:0001913), and [negative regulation of T cell mediated cytotoxicity](https://amigo.geneontology.org/amigo/term/GO:0001915). It enables functions such as [antigen binding](https://amigo.geneontology.org/amigo/term/GO:0003823), [cytokine receptor activity](https://amigo.geneontology.org/amigo/term/GO:0004896), and [interleukin-7 receptor activity](https://amigo.geneontology.org/amigo/term/GO:0004917). The gene is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), and is active on the [external side of plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009897). Additionally, it is involved in processes like [immune response](https://amigo.geneontology.org/amigo/term/GO:0006955), [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), [cell surface receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007166), [positive regulation of cell population proliferation](https://amigo.geneontology.org/amigo/term/GO:0008284), and [regulation of cell size](https://amigo.geneontology.org/amigo/term/GO:0008361). The gene also plays roles in [gene expression](https://amigo.geneontology.org/amigo/term/GO:0010467), [positive regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0010628), [cytokine-mediated signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0019221), [hemopoiesis](https://amigo.geneontology.org/amigo/term/GO:0030097), [T cell differentiation in thymus](https://amigo.geneontology.org/amigo/term/GO:0033077), [positive regulation of T cell differentiation in thymus](https://amigo.geneontology.org/amigo/term/GO:0033089), [interleukin-7-mediated signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0038111), [B cell proliferation](https://amigo.geneontology.org/amigo/term/GO:0042100), [T cell homeostasis](https://amigo.geneontology.org/amigo/term/GO:0043029), [positive regulation of receptor signaling pathway via JAK-STAT](https://amigo.geneontology.org/amigo/term/GO:0046427), [lymph node development](https://amigo.geneontology.org/amigo/term/GO:0048535), [defense response to Gram-positive bacterium](https://amigo.geneontology.org/amigo/term/GO:0050830), [negative regulation of T cell apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0070233), and [positive regulation of receptor signaling pathway via STAT](https://amigo.geneontology.org/amigo/term/GO:1904894).


The gene length is 8,038 base pairs (21.41% of all genes), the mature length is 656 base pairs, and the primary transcript length is 4,064 base pairs.


The gene IL7R, with NCBI ID 3575, has been mentioned in [440 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22IL7R%22) (note: the count in the hyperlink may be different since the count is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication mentioning IL7R is 1991, and the middle 50% of publications occurred between 2009 and 2017.


The top 5 publications mentioning IL7R, ranked by their scientific influence, include "[CD127 expression inversely correlates with FoxP3 and suppressive function of human CD4+ T reg cells.](https://pubmed.ncbi.nlm.nih.gov/16818678)" (2006) (relative citation ratio: 40.88), "[Interleukin-2 receptor gamma chain: a functional component of the interleukin-7 receptor.](https://pubmed.ncbi.nlm.nih.gov/8266077)" (1993) (relative citation ratio: 14.61), "[Interleukin-12 and -23 Control Plasticity of CD127(+) Group 1 and Group 3 Innate Lymphoid Cells in the Intestinal Lamina Propria.](https://pubmed.ncbi.nlm.nih.gov/26187413)" (2015) (relative citation ratio: 13.5), "[First-in-man clinical results of the treatment of patients with graft versus host disease with human ex vivo expanded CD4+CD25+CD127- T regulatory cells.](https://pubmed.ncbi.nlm.nih.gov/19559653)" (2009) (relative citation ratio: 11.04), and "[The heterogeneity of human CD127(+) innate lymphoid cells revealed by single-cell RNA sequencing.](https://pubmed.ncbi.nlm.nih.gov/26878113)" (2016) (relative citation ratio: 10.39). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[IL7R](https://www.proteinatlas.org/ENSG00000168685-IL7R) is a receptor involved in cancer-related genes and is a disease variant associated with SCID. Evidence for its function is available at the protein level. IL7R is detected in many tissues and is primarily located in the plasma membrane. It is expressed in T-cells with unknown function and is particularly enriched in lymphoid tissue, where it plays a role in cytokine signaling. Additionally, IL7R is expressed in leukemia cell lines and is involved in signal transduction. In single-cell expression analysis, it is found in NK-cells and T-cells, contributing to the immune system and transcription.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), [RELA](https://www.ncbi.nlm.nih.gov/gene/5970), [JUNB](https://www.ncbi.nlm.nih.gov/gene/3726), [STAT3](https://www.ncbi.nlm.nih.gov/gene/6774), and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), each shown to be regulating in 4 experiments.



The input data indicates associations with a wide range of disease conditions, including various types of cancer such as male reproductive organ cancer, testicular cancer, head and neck cancer, colorectal cancer, intestinal cancer, large intestine cancer, gastrointestinal system cancer, lung cancer, and respiratory system cancer. Additionally, the data highlights immune system diseases, metabolic disorders like diabetes mellitus (both type 1 and type 2), obesity, and cardiovascular system diseases. Neurological conditions such as neurodegenerative diseases and multiple sclerosis are also noted. Furthermore, the data includes hepatobiliary diseases like liver cirrhosis and primary biliary cirrhosis, as well as gastrointestinal conditions such as colitis, ulcerative colitis, and inflammatory bowel disease.


The IL7R gene has been associated with several diseases and traits through genome-wide association studies. Notably, it has been linked to [Platelet-to-lymphocyte ratio](https://pubmed.ncbi.nlm.nih.gov/34469753) and [Ulcerative colitis](https://pubmed.ncbi.nlm.nih.gov/21297633), which maps to the MeSH term [Colitis, Ulcerative](https://meshb.nlm.nih.gov/record/ui?ui=D003093) using the BioWordVec method.


The gene is expressed in various tissues, including lung, spleen, and small intestine, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in CD56+ NK cells, CD4+ T cells, CD8+ T cells, whole blood, tonsil, thymus, and lymph node, suggesting a ubiquitous expression pattern across multiple tissue samples.


The proteins are involved in several key pathways, including cytokine signaling in the immune system, broader immune system processes, interleukin-7 signaling, and signaling by interleukins.



The analyzed protein sequence has a GRAVY value of -0.219, indicating a hydrophilic nature (67.26th percentile). It exhibits a charge of -12.77 at pH 7.0 (18.25th percentile) and a median structural flexibility of 1.002 (51.96th percentile). The protein's secondary structure is predicted to be 29.41% helix (26.73rd percentile), 38.56% sheet (81.25th percentile), and 30.50% turn (67.45th percentile). The instability index is 50.88 (60.08th percentile), with an isoelectric point of 5.27 (12.88th percentile). The protein has a length of 459 amino acids (53.03rd percentile) and a molecular weight of 51578.15 Da (53.46th percentile). These calculations were performed using BioPython's sequence analysis tools, which can be found [here](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |