---
layout: post
title: RPL12
date: 2024-12-08 23:57 CST
description: RPL12 description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6136) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6136  | RPL12 | ENSG00000197958 | 9q33.3 |



The gene length is 49,498 base pairs (66th percentile of all genes), with a mature length of 3,045 base pairs and a primary transcript length of 48,326 base pairs.


The gene RPL12 has an NCBI Gene ID of 6136. It has been featured in [38 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RPL12%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1956, and the middle 50% of publications occurred between 1966 and 2000.


The top 5 publications mentioning RPL12, ranked by their scientific influence, include "[The ribosomal protein L8 is a complex L7/L12 and L10.](https://pubmed.ncbi.nlm.nih.gov/773698)" (1976) with a relative citation ratio of 4.77, "[Phosphorylation of the Ribosomal Protein RPL12/uL11 Affects Translation during Mitosis.](https://pubmed.ncbi.nlm.nih.gov/30220558)" (2018) with a relative citation ratio of 2.29, "[Sequence alignment and evolutionary comparison of the L10 equivalent and L12 equivalent ribosomal proteins from archaebacteria, eubacteria, and eucaryotes.](https://pubmed.ncbi.nlm.nih.gov/2515294)" (1989) with a relative citation ratio of 2.07, "[Flexibility, conformational diversity and two dimerization modes in complexes of ribosomal protein L12.](https://pubmed.ncbi.nlm.nih.gov/10637222)" (2000) with a relative citation ratio of 1.51, and "[Mitochondrial ribosomal protein L12 selectively associates with human mitochondrial RNA polymerase to activate transcription.](https://pubmed.ncbi.nlm.nih.gov/22003127)" (2011) with a relative citation ratio of 1.5. The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [HDAC2](https://www.ncbi.nlm.nih.gov/gene/3066), [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), and [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), each with 5 experiments. Additionally, [PPARG](https://www.ncbi.nlm.nih.gov/gene/5468) and [ZBTB7A](https://www.ncbi.nlm.nih.gov/gene/51341) were found to be regulating in 4 experiments each.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [CTCF](https://www.ncbi.nlm.nih.gov/gene/6829) with 3 experiments, [ZNF274](https://www.ncbi.nlm.nih.gov/gene/7936) and [ZNF143](https://www.ncbi.nlm.nih.gov/gene/6908) each with 2 experiments, and [POU5F1](https://www.ncbi.nlm.nih.gov/gene/2623), [ARID3A](https://www.ncbi.nlm.nih.gov/gene/2475), [FOXA1](https://www.ncbi.nlm.nih.gov/gene/2958), [KLF16](https://www.ncbi.nlm.nih.gov/gene/8861), [ZNF783](https://www.ncbi.nlm.nih.gov/gene/9968), and [ZNF561](https://www.ncbi.nlm.nih.gov/gene/7469) each with 1 or 2 experiments. These TFs were found to regulate gene expression in multiple Perturb-seq experiments.





The mouse ortholog gene 269261, which corresponds to the human ortholog gene 6136, is associated with various abnormal morphologies, including those of the eye, kidney, liver, lymph node, skin, and spleen, as well as enlarged heart, kidney, lymph nodes, and spleen. These phenotypes are observed in both sexes. Additionally, the gene is linked to embryonic lethality prior to organogenesis, preweaning lethality with complete penetrance, and abnormal seminal vesicle morphology, specifically a small seminal vesicle, which are not observed in both sexes.


The protein sequence analyzed has a GRAVY value of -0.347 (52.6% percentile), indicating a hydrophilic nature. It exhibits a charge of 6.84 at pH 7.0 (73.82% percentile) and a median structural flexibility of 1.004 (61.22% percentile). The protein's secondary structure is predicted to be 30.3% helix (34.68% percentile), 30.9% sheet (33.39% percentile), and 31.5% turn (73.97% percentile). The instability index is 46.97 (46.51% percentile), with an isoelectric point of 9.48 (90.7% percentile). The protein has a length of 165 amino acids (9.9% percentile) and a molecular weight of 17818.38 Da (9.25% percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |