---
layout: post
title: MAP3K2
date: 2025-01-17 16:55 CST
description: MAP3K2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10746) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10746  | MAP3K2 | ENSG00000169967 | 2q14.3 |



The gene is involved in the [MAPK cascade](https://amigo.geneontology.org/amigo/term/GO:0000165) and [phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0016310), playing a crucial role in [intracellular signal transduction](https://amigo.geneontology.org/amigo/term/GO:0035556) and the [cellular response to mechanical stimulus](https://amigo.geneontology.org/amigo/term/GO:0071260). It enables [protein kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004672), [protein serine/threonine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004674), [MAP kinase kinase kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004709), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524), [protein kinase binding](https://amigo.geneontology.org/amigo/term/GO:0019901), [protein serine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0106310), and [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872). The gene is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654) and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737).


The gene length is 43,406 base pairs (62.49% of all genes), the mature length is 1,730 base pairs (45.15% of all genes), and the primary transcript length is 26,121 base pairs (58.93% of all genes).


The gene MAP3K2 (NCBI ID: 10746) has been mentioned in [62 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22MAP3K2%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1999, and the middle 50% of publications occurred between 2009 and 2020.


The top 5 publications mentioning MAP3K2, ranked by their scientific influence, include "[SMYD3 links lysine methylation of MAP3K2 to Ras-driven cancer.](https://pubmed.ncbi.nlm.nih.gov/24847881)" (2014) (relative citation ratio: 7.54), "[Imaging mass spectrometry of a specific fragment of mitogen-activated protein kinase/extracellular signal-regulated kinase kinase kinase 2 discriminates cancer from uninvolved prostate tissue.](https://pubmed.ncbi.nlm.nih.gov/19690195)" (2009) (relative citation ratio: 4.02), "[LncRNA FOXD3-AS1 promotes proliferation, invasion and migration of cutaneous malignant melanoma via regulating miR-325/MAP3K2.](https://pubmed.ncbi.nlm.nih.gov/31541886)" (2019) (relative citation ratio: 2.75), "[CircRNA circBACH1 facilitates hepatitis B virus replication and hepatoma development by regulating the miR-200a-3p/MAP3K2 axis.](https://pubmed.ncbi.nlm.nih.gov/35352818)" (2022) (relative citation ratio: 2.53), and "[The malectin-like receptor-like kinase LETUM1 modulates NLR protein SUMM2 activation via MEKK2 scaffolding.](https://pubmed.ncbi.nlm.nih.gov/32839517)" (2020) (relative citation ratio: 2.52). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[MAP3K2](https://www.proteinatlas.org/ENSG00000169967-MAP3K2) is a kinase with serine/threonine-protein kinase and transferase molecular functions. Evidence for this protein is available at the protein level. Its RNA is detected in all tissues, and it is primarily localized in the nucleoplasm and cytosol. In terms of expression clusters, MAP3K2 is associated with Cluster 35 in blood, specifically in neutrophils involved in protein binding. In tissues, it is part of Cluster 40 with non-specific transcription functions. Similarly, in the brain and cell lines, it is associated with Clusters 55 and 26, respectively, both with non-specific transcription and mitochondria & RNA binding functions. In single cells, it is part of Cluster 28, found in dendritic cells involved in antigen presentation.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SMAD3](https://www.ncbi.nlm.nih.gov/gene/4088) and [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), each with 5 experiments. Additionally, [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885), and [MAX](https://www.ncbi.nlm.nih.gov/gene/4149) were each found to be regulating in 4 experiments.



The GWAS data indicates associations with various disease conditions, including hematopoietic system disease, immune system disease, disease of metabolism, acquired metabolic disease, carbohydrate metabolism disease, glucose metabolism disease, diabetes mellitus, type 2 diabetes mellitus, disease of anatomical entity, blood coagulation disease, inherited blood coagulation disease, and protein C deficiency.



The gene is expressed in various tissues and cell types, including the thyroid and blood, as indicated by BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles show expression in BDCA4+ dendritic cells, CD56+ NK cells, whole blood, pancreatic islet, superior cervical ganglion, and skeletal muscle.




The analyzed protein sequence has a GRAVY value of -0.621 (23.43rd percentile), indicating a hydrophilic nature. It exhibits a charge of 3.1 at pH 7.0 (61.62nd percentile) and a median structural flexibility of 1.008 (74.56th percentile). The protein's secondary structure is predicted to be 28.6% helix (20.81st percentile), 32.3% sheet (41.34th percentile), and 33.4% turn (83.87th percentile). The instability index is 51.8 (63.07th percentile), with an isoelectric point of 8.04 (59.28th percentile). The protein has a length of 619 amino acids (69.89th percentile) and a molecular weight of 69740.05 Da (70.35th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |