---
layout: post
title: ATP2B2
date: 2025-01-17 16:55 CST
description: ATP2B2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/491) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 491  | ATP2B2 | ENSG00000157087 | 3p25.3 |



The gene is involved in [neural retina development](https://amigo.geneontology.org/amigo/term/GO:0003407) and [neuron differentiation](https://amigo.geneontology.org/amigo/term/GO:0030182). It enables [P-type calcium transporter activity](https://amigo.geneontology.org/amigo/term/GO:0005388), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [calmodulin binding](https://amigo.geneontology.org/amigo/term/GO:0005516), [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524), [ATP hydrolysis activity](https://amigo.geneontology.org/amigo/term/GO:0016887), [PDZ domain binding](https://amigo.geneontology.org/amigo/term/GO:0030165), [glutamate receptor binding](https://amigo.geneontology.org/amigo/term/GO:0035254), and [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872). It is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [basolateral plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0016323), [apical plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0016324), [dendritic spine membrane](https://amigo.geneontology.org/amigo/term/GO:0032591), [neuronal cell body membrane](https://amigo.geneontology.org/amigo/term/GO:0032809), [presynaptic active zone membrane](https://amigo.geneontology.org/amigo/term/GO:0048787), [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062), and [parallel fiber to Purkinje cell synapse](https://amigo.geneontology.org/amigo/term/GO:0098688). It is active in the [intracellular membrane-bounded organelle](https://amigo.geneontology.org/amigo/term/GO:0043231), [postsynaptic density membrane](https://amigo.geneontology.org/amigo/term/GO:0098839), [glutamatergic synapse](https://amigo.geneontology.org/amigo/term/GO:0098978), and [GABA-ergic synapse](https://amigo.geneontology.org/amigo/term/GO:0098982). Additionally, it is involved in [calcium ion transport](https://amigo.geneontology.org/amigo/term/GO:0006816), [sensory perception of sound](https://amigo.geneontology.org/amigo/term/GO:0007605), [monoatomic ion transmembrane transport](https://amigo.geneontology.org/amigo/term/GO:0034220), [calcium ion transmembrane transport](https://amigo.geneontology.org/amigo/term/GO:0070588), [regulation of cytosolic calcium ion concentration](https://amigo.geneontology.org/amigo/term/GO:0051480), [regulation of postsynaptic cytosolic calcium ion concentration](https://amigo.geneontology.org/amigo/term/GO:0099566), and [regulation of cardiac conduction](https://amigo.geneontology.org/amigo/term/GO:1903779). It also enables [P-type calcium transporter activity involved in regulation of postsynaptic cytosolic calcium ion concentration](https://amigo.geneontology.org/amigo/term/GO:1905059).


The gene length is 12,327 base pairs (30.23% of all genes), the mature length is 591 base pairs (5.09% of all genes), and the primary transcript length is 3,076 base pairs (11.18% of all genes).


The gene ATP2B2 (NCBI ID: 491) has been mentioned in [132 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ATP2B2%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning this gene is 1980, and the middle 50% of publications occurred between 2002 and 2014.


The top 5 publications mentioning ATP2B2, ranked by their scientific influence, include "[Plasma membrane Ca2+-ATPase extrudes Ca2+ from hair cell stereocilia.](https://pubmed.ncbi.nlm.nih.gov/9425003)" (1998) with a [relative citation ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 3.89, "[The plasma membrane calcium pump: functional domains, regulation of the activity, and tissue specificity of isoform expression.](https://pubmed.ncbi.nlm.nih.gov/8195792)" (1994) with an RCR of 2.95, "[Metabolic regulation of the PMCA: Role in cell death and survival.](https://pubmed.ncbi.nlm.nih.gov/28625348)" (2018) with an RCR of 2.81, "[Modification of human hearing loss by plasma-membrane calcium pump PMCA2.](https://pubmed.ncbi.nlm.nih.gov/15829536)" (2005) with an RCR of 2.75, and "[Structure of the human plasma membrane Ca<sup>2+</sup>-ATPase 1 in complex with its obligatory subunit neuroplastin.](https://pubmed.ncbi.nlm.nih.gov/30190470)" (2018) with an RCR of 2.58. The relative citation ratio measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ATP2B2](https://www.proteinatlas.org/ENSG00000157087-ATP2B2) is a gene with molecular functions in calmodulin-binding and translocase activity. It is involved in diseases such as deafness, non-syndromic deafness, and is a target for FDA-approved drugs. Evidence for this gene is available at the protein level. Its RNA is detected in many tissues, with notable expression in the brain, specifically in neurons and synapses, where it plays a role in synaptic function. Additionally, it is expressed in liver cancer cell lines, where it is involved in metabolism. The gene is also associated with various cancer types, although its prognostic value is unprognostic in several cases, including breast invasive carcinoma, glioblastoma multiforme, and kidney renal clear cell carcinoma.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [SUZ12](https://www.ncbi.nlm.nih.gov/gene/23512) with 4 experiments, [ZBTB7A](https://www.ncbi.nlm.nih.gov/gene/51341) with 3 experiments, [EZH2](https://www.ncbi.nlm.nih.gov/gene/2146) with 3 experiments, [TFAP2C](https://www.ncbi.nlm.nih.gov/gene/7022) with 3 experiments, and [MAX](https://www.ncbi.nlm.nih.gov/gene/4149) with 3 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The input data from GWAS indicates associations with a wide range of disease conditions, including various types of cancer such as lung cancer and respiratory system cancer, metabolic disorders like type 2 diabetes mellitus and carbohydrate metabolism disease, cardiovascular diseases including coronary artery disease and hypertension, and neurodegenerative diseases such as Alzheimer's disease and dementia. Additionally, the data highlights associations with mental health conditions like schizophrenia and substance dependence, including cocaine dependence.



The gene is expressed in multiple tissue samples, including the liver, brain, and pituitary, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in various brain regions such as the whole brain, cingulate cortex, subthalamic nucleus, occipital lobe, cerebellum, cerebellum peduncles, and globus pallidus.


The proteins are involved in various pathways, including transmembrane transport of small molecules, ion channel transport, platelet homeostasis, ion transport by P-type ATPases, platelet calcium homeostasis, reduction of cytosolic Ca++ levels, and hemostasis.



The analyzed protein sequence has a GRAVY value of -0.165 (72.67th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -17.91 (12.69th percentile), and the median structural flexibility is 1.003 (55.04th percentile). The protein has a propensity for helix, sheet, and turn structures of 32.58% (55.84th percentile), 36.12% (67.15th percentile), and 28.16% (48.73rd percentile), respectively. The instability index is 43.01 (32.24th percentile), with an isoelectric point of 5.66 (21.97th percentile). The sequence length is 1243 amino acids (92.43rd percentile), and the molecular weight is approximately 136874.76 Da (92.27th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |