---
layout: post
title: HNRNPH3
date: 2025-01-17 16:55 CST
description: HNRNPH3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3189) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3189  | HNRNPH3 | ENSG00000096746 | 10q21.3 |



The gene is involved in [mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0000398) and [RNA processing](https://amigo.geneontology.org/amigo/term/GO:0006396), including [RNA splicing](https://amigo.geneontology.org/amigo/term/GO:0008380) and its [regulation](https://amigo.geneontology.org/amigo/term/GO:0043484). It also plays a role in [epithelial cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0030855). The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), specifically in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654). It is part of the [spliceosomal complex](https://amigo.geneontology.org/amigo/term/GO:0005681) and the larger [ribonucleoprotein complex](https://amigo.geneontology.org/amigo/term/GO:1990904).


The gene length is 25,939 base pairs (48.66% of all genes), the mature length is 1,643 base pairs, and the primary transcript length is 24,874 base pairs.


The gene HNRNPH3 (NCBI ID: 3189) has been mentioned in [7 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22HNRNPH3%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning this gene was in 1987, and the middle 50% of publications occurred between 1998 and 2015.


The top 5 publications mentioning HNRNPH3, ranked by their relative citation ratio (RCR), include "[The repertoire of genetic alterations in salivary duct carcinoma including a novel HNRNPH3-ALK rearrangement.](https://pubmed.ncbi.nlm.nih.gov/30946933)" (2019) (relative citation ratio: 2.11), "[Effect of monoclonal antibodies anti-2H9, anti-IRac, and anti-HeFi-1 on the surface antigens of Reed-Sternberg cells.](https://pubmed.ncbi.nlm.nih.gov/3479635)" (1987) (relative citation ratio: 0.72), "[Cloning of human 2H9 heterogeneous nuclear ribonucleoproteins. Relation with splicing and early heat shock-induced splicing arrest.](https://pubmed.ncbi.nlm.nih.gov/8999868)" (1997) (relative citation ratio: 0.63), "[Identification of Hnrph3 as an autoantigen for acute anterior uveitis.](https://pubmed.ncbi.nlm.nih.gov/20943442)" (2011) (relative citation ratio: 0.57), and "[Long Noncoding RNA RP11-115N4.1 Promotes Inflammatory Responses by Interacting With HNRNPH3 and Enhancing the Transcription of HSP70 in Unexplained Recurrent Spontaneous Abortion.](https://pubmed.ncbi.nlm.nih.gov/34484222)" (2021) (relative citation ratio: 0.53). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[HNRNPH3](https://www.proteinatlas.org/ENSG00000096746-HNRNPH3) is an RNA-binding ribonucleoprotein with evidence at the protein level. It is detected in all tissues and primarily localized in the nucleoplasm. The gene is expressed in various clusters, including Cluster 33 for blood expression, Cluster 9 for tissue expression, Cluster 13 for brain expression, Cluster 52 for cell line expression, and Cluster 80 for single cell expression. Notably, it is associated with oligodendrocytes and myelination in tissue and brain expression clusters.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD9](https://www.ncbi.nlm.nih.gov/gene/65980) with 6 experiments, [KLF4](https://www.ncbi.nlm.nih.gov/gene/9314) with 6 experiments, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 6 experiments, [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) with 5 experiments, and [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.





The gene is expressed in various tissues, including the ovary and thyroid. Additionally, it is expressed in several immune cell types such as CD34+, CD4+ T cells, CD8+ T cells, and CD19+ B cells (negatively selected), as well as in CD105+ endothelial cells and CD71+ early erythroid cells.




The analyzed protein sequence has a GRAVY value of -0.734 (15.47th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -2.602 (40.85th percentile), and the median structural flexibility is 1.008 (74.86th percentile). The protein has a propensity for turns (45.38%, 99.2nd percentile), with lower affinities for helices (19.36%, 1.08th percentile) and sheets (25.14%, 9.04th percentile). The instability index is 34.904 (9.97th percentile), and the isoelectric point is 6.367 (39.62nd percentile). The protein is 346 amino acids long (37.32nd percentile) with a molecular weight of 36926.1 Da (34.97th percentile). For detailed sequence analysis methods, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |