---
layout: post
title: SRP14
date: 2024-12-08 23:57 CST
description: SRP14 description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6727) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6727  | SRP14 | ENSG00000140319 | 15q15.1 |



The gene length is 22,942 base pairs (45.38% of all genes), the mature length is 471 base pairs (1.31% of all genes), and the primary transcript length is 7,815 base pairs (27.35% of all genes).


The gene SRP14, with NCBI ID 6727, has been featured in [5 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SRP14%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1999, and the middle 50% of publications occurred between 2009 and 2021.


The top 5 publications mentioning SRP14, ranked by their relative citation ratio (RCR), include "[Host Cells Actively Resist Porcine Reproductive and Respiratory Syndrome Virus Infection via the IRF8-MicroRNA-10a-SRP14 Regulatory Pathway.](https://pubmed.ncbi.nlm.nih.gov/35293774)" (2022) (relative citation ratio: 2.19), "[Identification and validation of signal recognition particle 14 as a prognostic biomarker predicting overall survival in patients with acute myeloid leukemia.](https://pubmed.ncbi.nlm.nih.gov/33985510)" (2021) (relative citation ratio: 0.47), "[The RNA-Binding Proteins SRP14 and HMGB3 Control HIV-1 Tat mRNA Processing and Translation During HIV-1 Latency.](https://pubmed.ncbi.nlm.nih.gov/34194479)" (2021) (relative citation ratio: 0.41), "[The Alu domain homolog of the yeast signal recognition particle consists of an Srp14p homodimer and a yeast-specific RNA structure.](https://pubmed.ncbi.nlm.nih.gov/10573124)" (1999) (relative citation ratio: 0.31), and "[Structure of SRP14 from the Schizosaccharomyces pombe signal recognition particle.](https://pubmed.ncbi.nlm.nih.gov/19390147)" (2009) (relative citation ratio: 0.18). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) and [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), each with 6 experiments. Additionally, [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170), [ZBTB48](https://www.ncbi.nlm.nih.gov/gene/3104), and [HDAC1](https://www.ncbi.nlm.nih.gov/gene/3065) were each found to be regulating in 5 experiments.


The transcription factor [BRCA1](https://www.ncbi.nlm.nih.gov/gene/6829) was found to be regulating genes in 3 Perturb-seq experiments, according to data from GTRD's database of transcription factor binding sites and the study published in [PubMed](https://pubmed.ncbi.nlm.nih.gov/35688146/).


The SRP14 gene has been associated with [Reticulocyte count](https://pubmed.ncbi.nlm.nih.gov/32888494), which maps closely to the MeSH term [Reticulocyte Count](https://meshb.nlm.nih.gov/record/ui?ui=D017701) through a direct match. Additionally, SRP14 is linked to [Reticulocyte fraction of red cells](https://pubmed.ncbi.nlm.nih.gov/32888494), which corresponds to the MeSH term [Reticulocytes](https://meshb.nlm.nih.gov/record/ui?ui=D012156) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method.





The protein sequence analyzed has a GRAVY value of -0.445 (41.06th percentile), indicating a hydrophilic nature. It exhibits a charge of 14.48 at pH 7.0 (88.43rd percentile) and a median structural flexibility of 1.007 (70.31st percentile). The protein's secondary structure is predicted to be predominantly helical, with 47.06% of amino acids in helix form (99.13th percentile), 31.62% in sheet form (37.31st percentile), and 20.59% in turn form (3.21st percentile). The instability index is 31.74 (5.58th percentile), and the isoelectric point is 10.05 (96.72nd percentile). The protein has a length of 136 amino acids (6.48th percentile) and a molecular weight of 14569.76 Da (5.91st percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |