---
layout: post
title: ACTN4
date: 2025-01-17 16:55 CST
description: ACTN4 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/81) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 81  | ACTN4 | ENSG00000130402 | 19q13.2 |



The gene is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615), [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925), [actin cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0015629), [Z disc](https://amigo.geneontology.org/amigo/term/GO:0030018), [cell junction](https://amigo.geneontology.org/amigo/term/GO:0030054), [cortical actin cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0030864), [platelet alpha granule lumen](https://amigo.geneontology.org/amigo/term/GO:0031093), [pseudopodium](https://amigo.geneontology.org/amigo/term/GO:0031143), [perinuclear region of cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0048471), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). It is involved in [protein transport](https://amigo.geneontology.org/amigo/term/GO:0015031), [actin cytoskeleton organization](https://amigo.geneontology.org/amigo/term/GO:0030036), [vesicle transport along actin filament](https://amigo.geneontology.org/amigo/term/GO:0030050), [positive regulation of cell migration](https://amigo.geneontology.org/amigo/term/GO:0030335), [positive regulation of sodium:proton antiporter activity](https://amigo.geneontology.org/amigo/term/GO:0032417), [tumor necrosis factor-mediated signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0033209), [peroxisome proliferator activated receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0035357), [regulation of apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0042981), [positive regulation of transcription by RNA polymerase II](https://amigo.geneontology.org/amigo/term/GO:0045944), [retinoic acid receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0048384), [muscle cell development](https://amigo.geneontology.org/amigo/term/GO:0055001), [negative regulation of substrate adhesion-dependent cell spreading](https://amigo.geneontology.org/amigo/term/GO:1900025), and [positive regulation of non-canonical NF-kappaB signal transduction](https://amigo.geneontology.org/amigo/term/GO:1901224). The gene enables [RNA polymerase II transcription regulatory region sequence-specific DNA binding](https://amigo.geneontology.org/amigo/term/GO:0000977), [nucleoside binding](https://amigo.geneontology.org/amigo/term/GO:0001882), [transcription coactivator activity](https://amigo.geneontology.org/amigo/term/GO:0003713), [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [actin binding](https://amigo.geneontology.org/amigo/term/GO:0003779), [integrin binding](https://amigo.geneontology.org/amigo/term/GO:0005178), [calcium ion binding](https://amigo.geneontology.org/amigo/term/GO:0005509), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [nuclear receptor binding](https://amigo.geneontology.org/amigo/term/GO:0016922), [nuclear receptor coactivator activity](https://amigo.geneontology.org/amigo/term/GO:0030374), [chromatin DNA binding](https://amigo.geneontology.org/amigo/term/GO:0031490), [protein homodimerization activity](https://amigo.geneontology.org/amigo/term/GO:0042803), [nuclear retinoic acid receptor binding](https://amigo.geneontology.org/amigo/term/GO:0042974), [transmembrane transporter binding](https://amigo.geneontology.org/amigo/term/GO:0044325), and [actin filament binding](https://amigo.geneontology.org/amigo/term/GO:0051015). Additionally, it is part of a [protein-containing complex](https://amigo.geneontology.org/amigo/term/GO:0032991) and a [ribonucleoprotein complex](https://amigo.geneontology.org/amigo/term/GO:1990904).


The gene length is 139,198 base pairs (87.79% of all genes), the mature length is 4,337 base pairs, and the primary transcript length is 127,602 base pairs.


ACTN4 has been mentioned in [343 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ACTN4%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication mentioning ACTN4 is 1986, and the middle 50% of publications occurred between 2009 and 2019.


The top 5 publications mentioning ACTN4, ranked by their scientific influence, include "[Circular RNA ACTN4 promotes intrahepatic cholangiocarcinoma progression by recruiting YBX1 to initiate FZD7 transcription.](https://pubmed.ncbi.nlm.nih.gov/34509526)" (2022) (relative citation ratio: 18.56), "[APOL1 genetic variants in focal segmental glomerulosclerosis and HIV-associated nephropathy.](https://pubmed.ncbi.nlm.nih.gov/21997394)" (2011) (relative citation ratio: 16.94), "[Focal Segmental Glomerulosclerosis.](https://pubmed.ncbi.nlm.nih.gov/28242845)" (2017) (relative citation ratio: 14.75), "[Differentiating Primary, Genetic, and Secondary FSGS in Adults: A Clinicopathologic Approach.](https://pubmed.ncbi.nlm.nih.gov/29321142)" (2018) (relative citation ratio: 9.24), and "[Collapsing focal segmental glomerulosclerosis following treatment with high-dose pamidronate.](https://pubmed.ncbi.nlm.nih.gov/11373339)" (2001) (relative citation ratio: 7.65). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ACTN4](https://www.proteinatlas.org/ENSG00000130402-ACTN4) is an actin-binding protein with evidence at the protein level. It is involved in disease variants and is detected in all RNA tissue distributions. The protein is localized to actin filaments, focal adhesion sites, primary cilium, primary cilium transition zone, and cytosol. ACTN4 is expressed in various clusters, including Cluster 7 for blood (non-specific - innate immune response), Cluster 75 for tissue (epithelium - extracellular exosomes), Cluster 11 for brain (white matter - signal transduction), Cluster 4 for cell line (non-specific - transcription), and Cluster 32 for single cell (smooth muscle cells - muscle contraction).


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 6 experiments, [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170) with 5 experiments, [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) with 5 experiments, [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885) with 5 experiments, and [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The input data from GWAS indicates associations with a wide range of disease conditions, including various types of cancer such as lung cancer, leukemia, lymphoma, and immune system cancer. Additionally, respiratory system diseases like asthma, bronchial disease, and obstructive lung disease are highlighted. The data also encompasses hematopoietic and immune system diseases, as well as cardiovascular system diseases. Furthermore, nervous system diseases, including neurodegenerative conditions like Parkinson's disease, are noted. Other significant conditions include kidney disease, urinary system disease, and specific lung cancers such as lung adenocarcinoma and non-small cell lung carcinoma. Heart conduction disease and migraine, particularly migraine with aura, are also part of the reported associations.



The gene is expressed in various tissues, including blood vessel and uterus, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in colorectal adenocarcinoma, adrenal cortex, skin, and uterus.


The proteins are involved in various pathways, including nephrin interactions, cell-cell communication, hemostasis, platelet activation, signaling and aggregation, platelet degranulation, and the response to elevated platelet cytosolic Ca2+.


The mouse ortholog gene 60595 and its human ortholog gene 81.0 are associated with the phenotype of preweaning lethality with incomplete penetrance, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.637 (22.19th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -38.64 (4.13th percentile), and the median structural flexibility is 1.005 (62.83rd percentile). The protein has a high affinity for helix structures (36.22%, 81.85th percentile), with moderate affinities for sheet (32.16%, 40.45th percentile) and turn (24.81%, 20.25th percentile) structures. The instability index is 40.90 (24.95th percentile), and the isoelectric point is 5.27 (12.83rd percentile). The protein is 911 amino acids long (84.8th percentile) with a molecular weight of 104852.80 Da (85.92nd percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |