---
layout: post
title: RGS20
date: 2025-01-17 16:55 CST
description: RGS20 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8601) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8601  | RGS20 | ENSG00000147509 | 8q11.23 |



The gene enables [GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0003924) and [GTPase activator activity](https://amigo.geneontology.org/amigo/term/GO:0005096), and is involved in [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [trans-Golgi network](https://amigo.geneontology.org/amigo/term/GO:0005802), and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). Additionally, it plays a role in the [G protein-coupled receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007186), [regulation of G protein-coupled receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0008277), and [negative regulation of signal transduction](https://amigo.geneontology.org/amigo/term/GO:0009968).


The gene length is 855,635 base pairs (99.44% of all genes), the mature length is 1,583 base pairs (40.65% of all genes), and the primary transcript length is 78,012 base pairs (84.24% of all genes).


RGS20 has been mentioned in [13 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RGS20%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1997, and the middle 50% of publications range from 2002 to 2014.


The top 5 publications mentioning RGS20, ranked by their relative citation ratio (RCR), include studies that explore the gene's role in various biological processes. The RCR values measure the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. The most influential publication is "[lncRNA NEAT1 promotes cell proliferation and invasion by regulating miR‑365/RGS20 in oral squamous cell carcinoma.](https://pubmed.ncbi.nlm.nih.gov/29484420)" (2018) (relative citation ratio: 2.78). Other notable studies include "[Inhibition of brain Gz GAP and other RGS proteins by palmitoylation of G protein alpha subunits.](https://pubmed.ncbi.nlm.nih.gov/9353196)" (1997) (relative citation ratio: 2.49), "[RGSZ1, a Gz-selective RGS protein in brain. Structure, membrane association, regulation by Galphaz phosphorylation, and relationship to a Gz gtpase-activating protein subfamily.](https://pubmed.ncbi.nlm.nih.gov/9748280)" (1998) (relative citation ratio: 2.34), "[RGSZ1, a Gz-selective regulator of G protein signaling whose action is sensitive to the phosphorylation state of Gzalpha.](https://pubmed.ncbi.nlm.nih.gov/9748279)" (1998) (relative citation ratio: 2.09), and "[Molecular organization and dynamics of the melatonin MT₁ receptor/RGS20/G(i) protein complex reveal asymmetry of receptor dimers for RGS and G(i) coupling.](https://pubmed.ncbi.nlm.nih.gov/20859254)" (2010) (relative citation ratio: 1.47). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RGS20](https://www.proteinatlas.org/ENSG00000147509-RGS20) is a signal transduction inhibitor with evidence at the protein level. Its RNA is detected in some tissues, and it is notably expressed in brain tissue, specifically in neurons involved in mixed functions. In cell lines, RGS20 is found in neuronal clusters related to signal transduction, and in single cells, it is detected in astrocytes with unknown function. The gene's expression in various cancer types, such as Bladder Urothelial Carcinoma, Breast Invasive Carcinoma, Cervical Squamous Cell Carcinoma, Glioblastoma Multiforme, Head and Neck Squamous Cell Carcinoma, Lung Squamous Cell Carcinoma, and Skin Cutaneous Melanoma, is unprognostic.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 6 experiments, [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) with 5 experiments, [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664) with 5 experiments, [EZH2](https://www.ncbi.nlm.nih.gov/gene/2146) with 5 experiments, and [JUN](https://www.ncbi.nlm.nih.gov/gene/3725) with 4 experiments.



The input data indicates associations with various disease conditions, including several types of cancer such as leukemia, hematologic cancer, immune system cancer, and organ system cancer. Specifically, it highlights acute leukemia, acute lymphocytic leukemia, and lymphoblastic leukemia. Additionally, the data includes respiratory system diseases, such as lower respiratory tract disease and lung disease. Metabolic disorders are also represented, with a focus on diabetes mellitus, particularly type 2 diabetes mellitus, and related conditions like glucose metabolism disease and carbohydrate metabolism disease. Furthermore, the data encompasses cardiovascular system disease and general disease conditions affecting anatomical entities.



The gene is expressed in various tissues, including skin and brain, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in multiple brain regions such as the pineal gland, prefrontal cortex, amygdala, whole brain, caudate nucleus, and occipital lobe, as well as in bronchial epithelial cells.


The proteins are involved in various signaling pathways, including G alpha (z) signaling events, G alpha (i) signaling events, GPCR downstream signaling, signaling by GPCR, and signal transduction.



The analyzed protein sequence has a GRAVY value of -0.522 (32.68th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -1.824 (43.75th percentile), and the median structural flexibility is 1.007 (71.33rd percentile). The protein has a high instability index of 81.14 (98.46th percentile) and an isoelectric point of 6.48 (42.18th percentile). The sequence length is 388 amino acids (43.63rd percentile) with a molecular weight of 43691.36 Da (44.2nd percentile). The secondary structure affinities are 34.02% for helix (68th percentile), 27.06% for sheet (14.68th percentile), and 30.93% for turn (70.14th percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |