---
layout: post
title: SDCBP
date: 2024-12-08 23:57 CST
description: SDCBP description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6386) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6386  | SDCBP | ENSG00000137575 | 8q12.1 |



The gene length is 127,849 base pairs (86.32% of all genes), the mature length is 3,831 base pairs (82.23% of all genes), and the primary transcript length is 78,863 base pairs (84.47% of all genes).


The gene SDCBP (NCBI ID: 6386) has been the subject of [114 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SDCBP%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1978, and the middle 50% of publications occurred between 2008 and 2019.


The top five publications mentioning SDCBP, ranked by their scientific influence, include "[Quantitative proteomics identifies the core proteome of exosomes with syntenin-1 as the highest abundant protein and a putative universal biomarker.](https://pubmed.ncbi.nlm.nih.gov/34108659)" (2021) (relative citation ratio: 17.65), "[Syntenin-ALIX exosome biogenesis and budding into multivesicular bodies are controlled by ARF6 and PLD2.](https://pubmed.ncbi.nlm.nih.gov/24637612)" (2014) (relative citation ratio: 12.07), "[Heparanase activates the syndecan-syntenin-ALIX exosome pathway.](https://pubmed.ncbi.nlm.nih.gov/25732677)" (2015) (relative citation ratio: 8.57), "[Syntenin, a PDZ protein that binds syndecan cytoplasmic domains.](https://pubmed.ncbi.nlm.nih.gov/9391086)" (1997) (relative citation ratio: 6.05), and "[Syntenin and syndecan in the biogenesis of exosomes.](https://pubmed.ncbi.nlm.nih.gov/26032692)" (2015) (relative citation ratio: 5.01). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) and [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), each with 5 experiments. Additionally, [NFE2](https://www.ncbi.nlm.nih.gov/gene/4778), [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), and [TAL1](https://www.ncbi.nlm.nih.gov/gene/6886) were each found to be regulating in 4 experiments.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [TCF7L2](https://www.ncbi.nlm.nih.gov/gene/6829) and [SREBF1](https://www.ncbi.nlm.nih.gov/gene/6426), each shown to be regulating genes in 2 experiments. Additionally, [ZNF263](https://www.ncbi.nlm.nih.gov/gene/9968) was found to be regulating genes in 1 experiment.








The analyzed protein sequence has a GRAVY value of -0.055 (81.07th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is 0.118 (50.83rd percentile), and the median structural flexibility is 1.002 (49.66th percentile). The protein has a propensity for helix, sheet, and turn structures of 29.53% (27.8th percentile), 35.23% (60.9th percentile), and 30.87% (69.73rd percentile), respectively. The instability index is 43.96 (35.53rd percentile), and the isoelectric point is 7.06 (51.05th percentile). The protein sequence is 298 amino acids long (28.2nd percentile) with a molecular weight of 32444.01 Da (27.19th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |