---
layout: post
title: ARHGEF6
date: 2025-01-17 16:55 CST
description: ARHGEF6 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9459) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9459  | ARHGEF6 | ENSG00000129675 | Xq26.3  |



The gene enables [guanyl-nucleotide exchange factor activity](https://amigo.geneontology.org/amigo/term/GO:0005085) and [GTPase activator activity](https://amigo.geneontology.org/amigo/term/GO:0005096), and is involved in [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737) and [lamellipodium](https://amigo.geneontology.org/amigo/term/GO:0030027), and is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829) and [cell-cell junction](https://amigo.geneontology.org/amigo/term/GO:0005911). Additionally, the gene plays a role in the [apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0006915), [JNK cascade](https://amigo.geneontology.org/amigo/term/GO:0007254), and [lamellipodium assembly](https://amigo.geneontology.org/amigo/term/GO:0030032).


The gene length is 11,529 base pairs (28.78% of all genes), the mature length is 942 base pairs (20.1% of all genes), and the primary transcript length is 2,296 base pairs (7.71% of all genes).


ARHGEF6 has been mentioned in [22 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ARHGEF6%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning ARHGEF6 was in 1998, and the middle 50% of publications occurred between 2003 and 2014.


The top 5 publications mentioning ARHGEF6, ranked by their scientific influence, include studies that explore the gene's role in various cellular processes. The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Notable publications include "[Directional sensing requires G beta gamma-mediated PAK1 and PIX alpha-dependent activation of Cdc42.](https://pubmed.ncbi.nlm.nih.gov/12887923)" (2003) (relative citation ratio: 4.97), "[Mutations in ARHGEF6, encoding a guanine nucleotide exchange factor for Rho GTPases, in patients with X-linked mental retardation.](https://pubmed.ncbi.nlm.nih.gov/11017088)" (2000) (relative citation ratio: 4.57), "[The Cool-2/alpha-Pix protein mediates a Cdc42-Rac signaling cascade.](https://pubmed.ncbi.nlm.nih.gov/15649357)" (2005) (relative citation ratio: 1.8), "[Integrin-linked kinase activity regulates Rac- and Cdc42-mediated actin cytoskeleton reorganization via alpha-PIX.](https://pubmed.ncbi.nlm.nih.gov/15897874)" (2005) (relative citation ratio: 1.67), and "[AlphaPIX and betaPIX and their role in focal adhesion formation.](https://pubmed.ncbi.nlm.nih.gov/16337026)" (2006) (relative citation ratio: 1.28). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ARHGEF6](https://www.proteinatlas.org/ENSG00000129675-ARHGEF6) is a guanine-nucleotide releasing factor involved in cancer-related genes. Evidence for its function is available at the protein level. The gene is detected in all tissues and is primarily localized in the plasma membrane and cytosol. It is associated with vesicular transport in blood, basic cellular processes in tissues, mixed functions in astrocytes in the brain, innate immune response in HMC-1 cell lines, and unknown functions in granulocytes in single-cell expression clusters.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861) and [CREB1](https://www.ncbi.nlm.nih.gov/gene/1385), each with 6 experiments. Additionally, [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169), [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046), and [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025) were each found to be regulating in 5 experiments.



The input data indicates associations with various metabolic disorders, including diseases of metabolism, acquired metabolic diseases, carbohydrate metabolism diseases, glucose metabolism diseases, diabetes mellitus, and specifically type 2 diabetes mellitus.



The gene is expressed in various tissues, including adipose tissue, breast, and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in multiple immune cell types, such as CD34+ cells, CD33+ myeloid cells, BDCA4+ dendritic cells, CD56+ NK cells, CD4+ T cells, CD8+ T cells, and CD14+ monocytes.


The input data highlights various pathways involving signaling and cellular processes. These include signaling by NGF, cell death signaling via NRAGE, NRIF, and NADE, NRAGE signals death through JNK, p75 NTR receptor-mediated signaling, G alpha (12/13) signaling events, Rho GTPase cycle, signaling by Rho GTPases, GPCR downstream signaling, signaling by GPCR, signal transduction, cell junction organization, cell-cell communication, cell-extracellular matrix interactions, and regulation of cytoskeletal remodeling and cell spreading by IPP complex components.


The mouse ortholog gene 73341 and its human ortholog gene 9459 are associated with phenotypes including enlarged lymph nodes and increased circulating aspartate transaminase levels, observed in both sexes.


The protein sequence analyzed has a GRAVY value of -0.439 (41.77th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -10.5 (21.72nd percentile). The median structural flexibility is 1.007 (69.9th percentile), with a predicted secondary structure composition of 34.5% helix (71.8th percentile), 34.4% sheet (54.92nd percentile), and 27.6% turn (43.45th percentile). The instability index is 54.8 (71.68th percentile), and the isoelectric point is 5.79 (25.09th percentile). The protein has a length of 776 amino acids (79.49th percentile) and a molecular weight of 87497.89 Da (79.86th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |