---
layout: post
title: MSN
date: 2025-01-17 16:55 CST
description: MSN description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4478) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4478  | MSN | ENSG00000147065 | Xq12  |



The gene is involved in various biological processes, including [immunological synapse formation](https://amigo.geneontology.org/amigo/term/GO:0001771), [cytoskeleton organization](https://amigo.geneontology.org/amigo/term/GO:0007010), [leukocyte cell-cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0007159), and [regulation of cell shape](https://amigo.geneontology.org/amigo/term/GO:0008360). It enables molecular functions such as [double-stranded RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003725), [actin binding](https://amigo.geneontology.org/amigo/term/GO:0003779), and [signaling receptor binding](https://amigo.geneontology.org/amigo/term/GO:0005102). The gene is located in multiple cellular components, including the [uropod](https://amigo.geneontology.org/amigo/term/GO:0001931), [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615), [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0005856), and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), where it is also active. Additionally, it is involved in processes like [T cell proliferation](https://amigo.geneontology.org/amigo/term/GO:0042098), [leukocyte migration](https://amigo.geneontology.org/amigo/term/GO:0050900), and [positive regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0010628).


The gene length is 51,968 base pairs (67.5% of all genes), the mature length is 1,112 base pairs (26.3% of all genes), and the primary transcript length is 42,744 base pairs (71.55% of all genes).


The gene MSN (NCBI ID: 4478) has been mentioned in [401 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22MSN%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1978, and the middle 50% of publications occurred between 2004 and 2017.


The top 5 publications mentioning MSN, ranked by their scientific influence, include "[Hierarchical scheme for LC-MSn identification of chlorogenic acids.](https://pubmed.ncbi.nlm.nih.gov/12720369)" (2003) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 23.82, "[MSN anti-cancer nanomedicines: chemotherapy enhancement, overcoming of drug resistance, and metastasis inhibition.](https://pubmed.ncbi.nlm.nih.gov/24142549)" (2014) with an RCR of 10.23, "[Identification of EBP50: A PDZ-containing phosphoprotein that associates with members of the ezrin-radixin-moesin family.](https://pubmed.ncbi.nlm.nih.gov/9314537)" (1997) with an RCR of 9.4, "[Loss of XIST in Breast Cancer Activates MSN-c-Met and Reprograms Microglia via Exosomal miRNA to Promote Brain Metastasis.](https://pubmed.ncbi.nlm.nih.gov/30026327)" (2018) with an RCR of 8.82, and "[Dynamic interaction of VCAM-1 and ICAM-1 with moesin and ezrin in a novel endothelial docking structure for adherent leukocytes.](https://pubmed.ncbi.nlm.nih.gov/12082081)" (2002) with an RCR of 7.86. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[MSN](https://www.proteinatlas.org/ENSG00000147065-MSN) is a cancer-related gene with evidence at the protein level. It is detected in all tissues and primarily localized to the plasma membrane. MSN is expressed in monocytes and neutrophils, playing a role in degranulation, and is also involved in immune responses within lymphoid tissue. Additionally, it is associated with angiogenesis in adipocytes and endothelial cells.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313) with 4 experiments, [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) with 3 experiments, [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930) with 3 experiments, [YY1](https://www.ncbi.nlm.nih.gov/gene/7528) with 3 experiments, and [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170) with 3 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent detection in multiple experiments.



The input data indicates a focus on various disease conditions, particularly those affecting the nervous system and sensory systems. Specifically, the data highlights eye and adnexa diseases, with a notable emphasis on retinal diseases such as retinal degeneration and age-related macular degeneration. Additionally, the data includes degeneration of the macula and posterior pole, as well as general macular degeneration.



The gene is expressed in various tissues, including blood vessels, lung, and blood, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in CD33+ myeloid cells, CD56+ NK cells, and whole blood.


The proteins are involved in several key pathways, including axon guidance, developmental biology, L1CAM interactions, and the recycling pathway of L1.


The mouse ortholog gene 17698 and its human ortholog gene 4478 exhibit several phenotypic abnormalities in both sexes, including decreased erythrocyte cell number, decreased hematocrit, decreased hemoglobin content, decreased leukocyte cell number, impaired pupillary reflex, increased heart weight, increased mean corpuscular hemoglobin, increased mean corpuscular volume, and thrombocytopenia.


The analyzed protein sequence has a GRAVY value of -0.975 (4.97th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -5.55 (32.19th percentile). The median structural flexibility is 1.015 (92.02nd percentile), suggesting high flexibility. The protein has a strong tendency to form helices (45.41%, 98.52nd percentile) and sheets (28.77%, 22.08th percentile), with a lower tendency for turns (18.20%, 1.01st percentile). The instability index is 52.60 (65.49th percentile), and the isoelectric point is 6.08 (32.45th percentile). The protein is 577 amino acids long (66.48th percentile) with a molecular weight of 67819.25 Da (69.1st percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |