---
layout: post
title: RPL17
date: 2025-01-17 16:55 CST
description: RPL17 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6139) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6139  | RPL17 | ENSG00000265681 | 18q21.1 |



The gene is involved in [cytoplasmic translation](https://amigo.geneontology.org/amigo/term/GO:0002181) and [translation](https://amigo.geneontology.org/amigo/term/GO:0006412). It enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), serves as a [structural constituent of ribosome](https://amigo.geneontology.org/amigo/term/GO:0003735), and facilitates [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). Additionally, it is part of the [cytosolic large ribosomal subunit](https://amigo.geneontology.org/amigo/term/GO:0022625) and is located in the [cytosolic ribosome](https://amigo.geneontology.org/amigo/term/GO:0022626).


The gene length is 5,905 base pairs (16.36% of all genes), the mature length is 588 base pairs, and the primary transcript length is 3,624 base pairs.


The gene RPL17 (NCBI ID: 6139) has been mentioned in [28 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RPL17%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1981 and the middle 50% of publications occurring between 2012 and 2020. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning RPL17, ranked by their scientific influence, include "[Resolvin E1 and protectin D1 activate inflammation-resolution programmes.](https://pubmed.ncbi.nlm.nih.gov/17568749)" (2007) (relative citation ratio: 20.85), "[Intratumoral Activity of the CXCR3 Chemokine System Is Required for the Efficacy of Anti-PD-1 Therapy.](https://pubmed.ncbi.nlm.nih.gov/31097342)" (2019) (relative citation ratio: 14.23), "[A Phase II Basket Trial of Dual Anti-CTLA-4 and Anti-PD-1 Blockade in Rare Tumors (DART SWOG 1609) in Patients with Nonpancreatic Neuroendocrine Tumors.](https://pubmed.ncbi.nlm.nih.gov/31969335)" (2020) (relative citation ratio: 12.5), "[<i>Ex Vivo</i> Profiling of PD-1 Blockade Using Organotypic Tumor Spheroids.](https://pubmed.ncbi.nlm.nih.gov/29101162)" (2018) (relative citation ratio: 11.97), and "[A Chimeric Switch-Receptor Targeting PD1 Augments the Efficacy of Second-Generation CAR T Cells in Advanced Solid Tumors.](https://pubmed.ncbi.nlm.nih.gov/26979791)" (2016) (relative citation ratio: 11.86). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RPL17](https://www.proteinatlas.org/ENSG00000265681-RPL17) is a ribonucleoprotein and ribosomal protein with evidence at the protein level. It is detected in all tissues and is primarily located in the endoplasmic reticulum and cytosol. RPL17 is part of various non-specific expression clusters related to transcription and translation, including Cluster 43 in blood, Cluster 18 in tissues, Cluster 7 in the brain, Cluster 25 in cell lines, and Cluster 49 in single cells.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 7 experiments, [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113) with 5 experiments, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 4 experiments, [TCF7L2](https://www.ncbi.nlm.nih.gov/gene/6934) with 4 experiments, and [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [11091](https://www.ncbi.nlm.nih.gov/gene/11091) and [8861](https://www.ncbi.nlm.nih.gov/gene/8861), each shown to be regulating genes in two experiments. Additionally, [3190](https://www.ncbi.nlm.nih.gov/gene/3190), [6830](https://www.ncbi.nlm.nih.gov/gene/6830), and [9188](https://www.ncbi.nlm.nih.gov/gene/9188) were each found to be regulating genes in one experiment.




The gene is expressed in various tissues, including the pancreas, ovary, and cervix uteri, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Lymphoma Burkitt's (Raji), CD4+ T cells, CD8+ T cells, Leukemia promyelocytic-HL-60, Lymphoma Burkitt's (Daudi), and lymph nodes.


The input data highlights various pathways involved in protein metabolism and gene expression, including 3'-UTR-mediated translational regulation, L13a-mediated translational silencing of ceruloplasmin expression, and GTP hydrolysis and joining of the 60S ribosomal subunit. Key processes such as cap-dependent translation initiation, eukaryotic translation initiation, formation of a pool of free 40S subunits, eukaryotic translation elongation, and eukaryotic translation termination are also noted. Additionally, the data covers peptide chain elongation, viral mRNA translation, and nonsense-mediated decay (NMD) both independent of and enhanced by the exon junction complex (EJC). Pathways related to SRP-dependent cotranslational protein targeting to the membrane and translation are included. Furthermore, the data encompasses influenza infection, the influenza life cycle, influenza viral RNA transcription and replication, and disease.



The analyzed protein sequence has a GRAVY value of -0.901 (7.33rd percentile), indicating a hydrophilic nature. It exhibits a charge of 23.0 at pH 7.0 (95.02nd percentile) and a median structural flexibility of 1.008 (74.85th percentile). The protein's secondary structure is predicted to be 38.04% helix (88.86th percentile), 27.72% sheet (17.36th percentile), and 21.74% turn (5.8th percentile). Additionally, it has an instability index of 60.22 (83.45th percentile), an isoelectric point of 10.18 (97.36th percentile), a length of 184 amino acids (12.15th percentile), and a molecular weight of 21396.79 Da (13.15th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |