---
layout: post
title: TGFBR2
date: 2025-01-17 16:55 CST
description: TGFBR2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7048) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7048  | TGFBR2 | ENSG00000163513 | 3p24.1 |



The gene is involved in various biological processes, including [blood vessel development](https://amigo.geneontology.org/amigo/term/GO:0001568), [branching involved in blood vessel morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0001569), [vasculogenesis](https://amigo.geneontology.org/amigo/term/GO:0001570), [in utero embryonic development](https://amigo.geneontology.org/amigo/term/GO:0001701), and [epithelial to mesenchymal transition](https://amigo.geneontology.org/amigo/term/GO:0001837). It plays a role in [heart looping](https://amigo.geneontology.org/amigo/term/GO:0001947), [positive regulation of mesenchymal cell proliferation](https://amigo.geneontology.org/amigo/term/GO:0002053), and [lens development in camera-type eye](https://amigo.geneontology.org/amigo/term/GO:0002088). Additionally, it is involved in the regulation of tolerance induction to self antigen, B cell tolerance induction, and T cell tolerance induction. The gene is also active in processes such as [outflow tract septum morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0003148), [membranous septum morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0003149), and [outflow tract morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0003151). It enables [transmembrane receptor protein serine/threonine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004675) and [transforming growth factor beta receptor activity](https://amigo.geneontology.org/amigo/term/GO:0005024). The gene is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), and is active in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). It is also involved in [protein phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0006468) and the [apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0006915). The gene plays a role in the [transforming growth factor beta receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007179) and the [Notch signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007219). It is involved in [gastrulation](https://amigo.geneontology.org/amigo/term/GO:0007369), [brain development](https://amigo.geneontology.org/amigo/term/GO:0007420), and [heart development](https://amigo.geneontology.org/amigo/term/GO:0007507). The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524), and [glycosaminoglycan binding](https://amigo.geneontology.org/amigo/term/GO:0005539). It is also located in the [caveola](https://amigo.geneontology.org/amigo/term/GO:0005901) and the [external side of plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009897). The gene is part of the [receptor complex](https://amigo.geneontology.org/amigo/term/GO:0043235) and the [transforming growth factor beta ligand-receptor complex](https://amigo.geneontology.org/amigo/term/GO:0070021). It enables [activin receptor activity](https://amigo.geneontology.org/amigo/term/GO:0017002), [kinase activator activity](https://amigo.geneontology.org/amigo/term/GO:0019209), and [type I transforming growth factor beta receptor binding](https://amigo.geneontology.org/amigo/term/GO:0034713). The gene is involved in the [activation of protein kinase activity](https://amigo.geneontology.org/amigo/term/GO:0032147) and the [activin receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0032924). It plays a role in [embryonic hemopoiesis](https://amigo.geneontology.org/amigo/term/GO:0035162) and [aorta morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0035909). The gene is involved in the [regulation of cell population proliferation](https://amigo.geneontology.org/amigo/term/GO:0042127) and [myeloid dendritic cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0043011). It is located in the [membrane raft](https://amigo.geneontology.org/amigo/term/GO:0045121) and enables [SMAD binding](https://amigo.geneontology.org/amigo/term/GO:0046332) and [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872). The gene is involved in the [positive regulation of angiogenesis](https://amigo.geneontology.org/amigo/term/GO:0045766) and the [response to cholesterol](https://amigo.geneontology.org/amigo/term/GO:0070723). It plays a role in the [cellular response to growth factor stimulus](https://amigo.geneontology.org/amigo/term/GO:0071363) and the [regulation of stem cell proliferation](https://amigo.geneontology.org/amigo/term/GO:0072091). The gene is involved in the [positive regulation of epithelial to mesenchymal transition involved in endocardial cushion formation](https://amigo.geneontology.org/amigo/term/GO:1905007) and the [inferior endocardial cushion morphogenesis](https://amigo.geneontology.org/amigo/term/GO:1905317). It is also involved in the [lens fiber cell apoptotic process](https://amigo.geneontology.org/amigo/term/GO:1990086) and [miRNA transport](https://amigo.geneontology.org/amigo/term/GO:1990428). The gene plays a role in the [positive regulation of reactive oxygen species metabolic process](https://amigo.geneontology.org/amigo/term/GO:2000379) and the [positive regulation of CD4-positive, alpha-beta T cell proliferation](https://amigo.geneontology.org/amigo/term/GO:2000563). It is involved in the [regulation of stem cell differentiation](https://amigo.geneontology.org/amigo/term/GO:2000736).


The gene length is 18,424 base pairs (40.13% of all genes), the mature length is 1,172.5 base pairs, and the primary transcript length is 17,550.5 base pairs.


The gene TGFBR2 (NCBI ID: 7048) has been mentioned in [366 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22TGFBR2%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication mentioning this gene is 1992, and the middle 50% of publications occurred between 2006 and 2018.


The top five publications mentioning TGFBR2, ranked by their scientific influence, include "[A syndrome of altered cardiovascular, craniofacial, neurocognitive and skeletal development caused by mutations in TGFBR1 or TGFBR2.](https://pubmed.ncbi.nlm.nih.gov/15731757)" (2005) (relative citation ratio: 25.73), "[Cloning of a TGF beta type I receptor that forms a heteromeric complex with the TGF beta type II receptor.](https://pubmed.ncbi.nlm.nih.gov/8242743)" (1993) (relative citation ratio: 15.31), "[Heterozygous TGFBR2 mutations in Marfan syndrome.](https://pubmed.ncbi.nlm.nih.gov/15235604)" (2004) (relative citation ratio: 9.32), "[Dual targeting of TGF-β and PD-L1 via a bifunctional anti-PD-L1/TGF-βRII agent: status of preclinical and clinical advances.](https://pubmed.ncbi.nlm.nih.gov/32079617)" (2020) (relative citation ratio: 9.11), and "[Genetic changes in the transforming growth factor beta (TGF-beta) type II receptor gene in human gastric cancer cells: correlation with sensitivity to growth inhibition by TGF-beta.](https://pubmed.ncbi.nlm.nih.gov/8090721)" (1994) (relative citation ratio: 8.71). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[TGFBR2](https://www.proteinatlas.org/ENSG00000163513-TGFBR2) is a kinase and receptor with serine/threonine-protein kinase and transferase molecular functions. It is involved in various diseases, including aortic aneurysm, cancer-related genes, craniosynostosis, hereditary nonpolyposis colorectal cancer, and is a target for FDA-approved drugs. Evidence for this gene is available at the protein level. TGFBR2 is detected in all RNA tissue distributions and is primarily located in the plasma membrane. Its expression is clustered in adipose tissue with mixed function, choroid plexus with mitochondrial function, connective tissue cells involved in ECM organization, and adipocytes and endothelial cells with mixed function.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170), [TAL1](https://www.ncbi.nlm.nih.gov/gene/6886), [MAX](https://www.ncbi.nlm.nih.gov/gene/4149), [RXRA](https://www.ncbi.nlm.nih.gov/gene/6256), and [EZH2](https://www.ncbi.nlm.nih.gov/gene/2146), each shown to be regulating in 4 experiments.



The input data indicates associations with a wide range of disease conditions, including breast cancer, thoracic cancer, and various forms of cancer such as organ system cancer and endocrine gland cancer, notably pancreatic cancer. Additionally, the data highlights respiratory system diseases, including lower respiratory tract disease and lung disease. Cardiovascular system diseases, such as artery disease, vascular disease, and cerebrovascular disease, including carotid artery disease, are also represented. Furthermore, the data encompasses nervous system diseases, particularly central nervous system diseases like neurodegenerative disease, Alzheimer's disease, tauopathy, dementia, and cognitive disorder, as well as diseases of mental health.



The gene is expressed in adipose tissue and breast according to BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles indicate expression in cardiac myocytes, CD4+ T cells, CD8+ T cells, whole blood, adipocytes, and smooth muscle.


The input data highlights various pathways and processes involving TGF-beta receptor signaling, including its downregulation and activation of SMADs. Notably, several pathways are associated with cancer, such as the loss of function of SMAD2/3, SMAD4, TGFBR1, and TGFBR2, as well as mutations in TGFBR2 kinase domain and frameshift mutants due to microsatellite instability (MSI). Additionally, the data includes pathways related to TGF-beta receptor complex signaling, SMAD mutations, and the role of TGF-beta signaling in epithelial to mesenchymal transition (EMT). Broad categories of disease and signal transduction are also mentioned.


The mouse ortholog gene 21813, which corresponds to the human ortholog gene 7048, is associated with several phenotypic abnormalities. These include abnormal heart and kidney morphology, enlarged heart and kidney, small liver, and preweaning lethality with complete penetrance. Notably, these phenotypes are observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.354 (51.73rd percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -15.14 (15.19th percentile). The median structural flexibility is 0.9999 (41.09th percentile), with a secondary structure composition of 31.04% helix (41.4th percentile), 35.45% sheet (62.4th percentile), and 27.87% turn (45.99th percentile). The instability index is 42.18 (29.22nd percentile), and the isoelectric point is 5.60 (20.55th percentile). The protein is 567 amino acids long (65.59th percentile) with a molecular weight of 64567.05 Da (66.69th percentile). For more details on the sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |