---
layout: post
title: NIPSNAP1
date: 2025-01-17 16:55 CST
description: NIPSNAP1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8508) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8508  | NIPSNAP1 | ENSG00000184117 | 22q12.2 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is active in and located in the [mitochondrion](https://amigo.geneontology.org/amigo/term/GO:0005739). It is involved in [sensory perception of pain](https://amigo.geneontology.org/amigo/term/GO:0019233) and is also located in the [synaptic membrane](https://amigo.geneontology.org/amigo/term/GO:0097060).


The gene length is 35,732 base pairs (57.45% of all genes), the mature length is 1,011 base pairs (22.92% of all genes), and the primary transcript length is 32,018 base pairs (64.33% of all genes).


NIPSNAP1 (Gene ID: 8508) has been mentioned in [4 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22NIPSNAP1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning this gene was in 1998, and the middle 50% of publications occurred between 2007 and 2014.


The top publications mentioning NIPSNAP1 include "[NIPSNAP1 and NIPSNAP2 act as 'eat me' signals to allow sustained recruitment of autophagy receptors during mitophagy.](https://pubmed.ncbi.nlm.nih.gov/31251109)" (2019) with a relative citation ratio of 1.43, "[Characterization of the human NIPSNAP1 gene from 22q12: a member of a novel gene family.](https://pubmed.ncbi.nlm.nih.gov/9661659)" (1998) with a relative citation ratio of 0.97, "[Sequence variants in four candidate genes (NIPSNAP1, GBAS, CHCHD1 and METT11D1) in patients with combined oxidative phosphorylation system deficiencies.](https://pubmed.ncbi.nlm.nih.gov/24137763)" (2010) with a relative citation ratio of 0.37, and "[Chromosomal structural variations during progression of a prostate epithelial cell line to a malignant metastatic state inactivate the NF2, NIPSNAP1, UGT2B17, and LPIN2 genes.](https://pubmed.ncbi.nlm.nih.gov/23792589)" (2013) with a relative citation ratio of 0.33. The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[NIPSNAP1](https://www.proteinatlas.org/ENSG00000184117-NIPSNAP1) is a gene with evidence at the protein level. Its RNA is detected in all tissues, with notable expression in the liver and kidney, suggesting a role in metabolism. In the brain, it is expressed in neurons with mixed functions. The gene is also expressed in T-cells, although its function in these cells is unknown. At the single-cell level, NIPSNAP1 is found in proximal tubular cells, indicating a role in amino acid metabolism. The gene's expression in various cancer types is unprognostic, suggesting it may not significantly influence cancer outcomes.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046) and [KDM1A](https://www.ncbi.nlm.nih.gov/gene/23028), each with 5 experiments. Additionally, [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), and [BRD9](https://www.ncbi.nlm.nih.gov/gene/65980) were each found to be regulating in 4 experiments.





The gene is expressed in multiple tissue samples, including the liver, adrenal gland, and kidney, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in various tissues such as Lymphoma (Burkitt's, Raji), Liver, Leukemia (promyelocytic-HL-60), Cerebellum, Kidney, Fetal liver, and Leukemia (lymphoblastic, MOLT-4).




The analyzed protein sequence has a GRAVY value of -0.648 (21.3% percentile), indicating a hydrophilic nature. It exhibits a charge of 9.204 at pH 7.0 (79.94% percentile) and a median structural flexibility of 1.0004 (43.78% percentile). The protein's secondary structure is predicted to be 33.1% helix (60.11% percentile), 34.2% sheet (53.14% percentile), and 27.5% turn (42.41% percentile). The instability index is 43.1 (32.55% percentile), with an isoelectric point of 9.352 (88.14% percentile). The protein has a length of 284 amino acids (26.21% percentile) and a molecular weight of 33309.58 Da (28.29% percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |