---
layout: post
title: VEGFC
date: 2025-01-17 16:55 CST
description: VEGFC description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7424) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7424  | VEGFC | ENSG00000150630 | 4q34.3 |



The gene is involved in various biological processes, including [response to hypoxia](https://amigo.geneontology.org/amigo/term/GO:0001666), [positive regulation of protein phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0001934), [positive regulation of endothelial cell proliferation](https://amigo.geneontology.org/amigo/term/GO:0001938), and [sprouting angiogenesis](https://amigo.geneontology.org/amigo/term/GO:0002040). It enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576) and [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615). Additionally, it is involved in [substrate-dependent cell migration](https://amigo.geneontology.org/amigo/term/GO:0006929), [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), and has [growth factor activity](https://amigo.geneontology.org/amigo/term/GO:0008083). The gene also plays a role in the [positive regulation of cell population proliferation](https://amigo.geneontology.org/amigo/term/GO:0008284) and [response to xenobiotic stimulus](https://amigo.geneontology.org/amigo/term/GO:0009410). Furthermore, it is involved in [animal organ morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0009887), [glial cell proliferation](https://amigo.geneontology.org/amigo/term/GO:0014009), and is located in the [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020). The gene is also involved in the [regulation of vascular endothelial growth factor receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0030947) and is located in the [platelet alpha granule lumen](https://amigo.geneontology.org/amigo/term/GO:0031093). It enables [chemoattractant activity](https://amigo.geneontology.org/amigo/term/GO:0042056) and [vascular endothelial growth factor receptor 3 binding](https://amigo.geneontology.org/amigo/term/GO:0043185). Additionally, the gene is involved in the [positive regulation of blood vessel endothelial cell migration](https://amigo.geneontology.org/amigo/term/GO:0043536), [negative regulation of osteoblast differentiation](https://amigo.geneontology.org/amigo/term/GO:0045668), [positive regulation of angiogenesis](https://amigo.geneontology.org/amigo/term/GO:0045766), and [negative regulation of blood pressure](https://amigo.geneontology.org/amigo/term/GO:0045776). It also plays a role in the [vascular endothelial growth factor receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0048010), [positive regulation of protein secretion](https://amigo.geneontology.org/amigo/term/GO:0050714), [positive chemotaxis](https://amigo.geneontology.org/amigo/term/GO:0050918), and [induction of positive chemotaxis](https://amigo.geneontology.org/amigo/term/GO:0050930). Furthermore, the gene is involved in the [positive regulation of cell division](https://amigo.geneontology.org/amigo/term/GO:0051781), [positive regulation of glial cell proliferation](https://amigo.geneontology.org/amigo/term/GO:0060252), [positive regulation of mast cell chemotaxis](https://amigo.geneontology.org/amigo/term/GO:0060754), [positive regulation of lymphangiogenesis](https://amigo.geneontology.org/amigo/term/GO:1901492), [positive regulation of mesenchymal stem cell proliferation](https://amigo.geneontology.org/amigo/term/GO:1902462), and [cellular response to leukemia inhibitory factor](https://amigo.geneontology.org/amigo/term/GO:1990830).


The gene length is 14,050 base pairs (33.54% of all genes), the mature length is 1,975 base pairs (52.14% of all genes), and the primary transcript length is 11,532 base pairs (37.07% of all genes).


VEGFC (Gene ID: 7424) has been mentioned in [774 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22VEGFC%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning VEGFC was in 1996, and the middle 50% of publications occurred between 2007 and 2016.


The top 5 publications mentioning VEGFC, ranked by their scientific influence, include "[Induction of tumor lymphangiogenesis by VEGF-C promotes breast cancer metastasis.](https://pubmed.ncbi.nlm.nih.gov/11175850)" (2001) (relative citation ratio: 26.77), "[A novel vascular endothelial growth factor, VEGF-C, is a ligand for the Flt4 (VEGFR-3) and KDR (VEGFR-2) receptor tyrosine kinases.](https://pubmed.ncbi.nlm.nih.gov/8617204)" (1996) (relative citation ratio: 25.61), "[VEGF-C-driven lymphatic drainage enables immunosurveillance of brain tumours.](https://pubmed.ncbi.nlm.nih.gov/31942068)" (2020) (relative citation ratio: 16.92), "[Epigenetically upregulated oncoprotein PLCE1 drives esophageal carcinoma angiogenesis and proliferation via activating the PI-PLCε-NF-κB signaling pathway and VEGF-C/ Bcl-2 expression.](https://pubmed.ncbi.nlm.nih.gov/30609930)" (2019) (relative citation ratio: 11.08), and "[Vascular endothelial growth factor C promotes tumor lymphangiogenesis and intralymphatic tumor growth.](https://pubmed.ncbi.nlm.nih.gov/11280723)" (2001) (relative citation ratio: 10.38). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[VEGFC](https://www.proteinatlas.org/ENSG00000150630-VEGFC) is a developmental protein, growth factor, and mitogen with evidence at the protein level. It is implicated in cancer-related genes and is detected in many tissues. VEGFC is expressed in adipose tissue with mixed function (Cluster 7), astrocytes with mixed function (Cluster 19), connective tissue cells involved in ECM organization (Cluster 64), and adipocytes and endothelial cells with mixed function (Cluster 41).


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), [E2F1](https://www.ncbi.nlm.nih.gov/gene/1869), [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169), [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861), and [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025), each shown to be regulating in 4 experiments.




The gene VEGFC has been associated with [serum levels of protein VEGFC](https://pubmed.ncbi.nlm.nih.gov/35078996), which maps closely to the MeSH term [Proteins](https://meshb.nlm.nih.gov/record/ui?ui=D011506) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method. Additionally, it has been linked to [vascular endothelial growth factor C levels (VEGFC.3132.1.1)](https://pubmed.ncbi.nlm.nih.gov/29875488), with the closest MeSH match being [Growth](https://meshb.nlm.nih.gov/record/ui?ui=D006128), also identified through embedding match.


The gene is expressed in a variety of tissues, including skin and thyroid, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Thyroid, Adipocyte, Smooth Muscle, Bronchial Epithelial Cells, Lung, Fetal Thyroid, Adrenal Gland, and Uterus.


The proteins are involved in various pathways, including signaling by VEGF, signal transduction, hemostasis, platelet activation, signaling and aggregation, platelet degranulation, and response to elevated platelet cytosolic Ca2+. Additionally, they participate in VEGF binding to VEGFR, leading to receptor dimerization, and VEGF ligand-receptor interactions.



The analyzed protein sequence has a GRAVY value of -0.496 (35.23rd percentile), indicating a hydrophilic nature. It exhibits a charge of 3.06 at pH 7.0 (61.5th percentile) and a median structural flexibility of 1.002 (49.05th percentile). The protein's secondary structure is predicted to be 29.6% helix (28.37th percentile), 27.4% sheet (16.12th percentile), and 28.2% turn (48.78th percentile). The instability index is 58.1 (79.38th percentile), suggesting potential instability. The isoelectric point is 7.77 (56.87th percentile), with a molecular weight of 46883 Da (48.15th percentile) and a length of 419 amino acids (47.96th percentile). For detailed sequence analysis methods, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |