---
layout: post
title: ABR
date: 2025-01-17 16:55 CST
description: ABR description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/29) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 29  | ABR | ENSG00000159842 | 17p13.3 |



The gene enables [guanyl-nucleotide exchange factor activity](https://amigo.geneontology.org/amigo/term/GO:0005085) and [GTPase activator activity](https://amigo.geneontology.org/amigo/term/GO:0005096), and is involved in [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [axon](https://amigo.geneontology.org/amigo/term/GO:0030424), [dendritic spine](https://amigo.geneontology.org/amigo/term/GO:0043197), [Schaffer collateral - CA1 synapse](https://amigo.geneontology.org/amigo/term/GO:0098685), and [glutamatergic synapse](https://amigo.geneontology.org/amigo/term/GO:0098978). The gene is involved in [small GTPase-mediated signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007264), [modulation of chemical synaptic transmission](https://amigo.geneontology.org/amigo/term/GO:0050804), [regulation of small GTPase mediated signal transduction](https://amigo.geneontology.org/amigo/term/GO:0051056), and [activation of GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0090630).


The gene has a length of 30,909.0 base pairs (53.52 percentile of all genes), a mature length of 1,393.5 base pairs, and a primary transcript length of 25,979.0 base pairs.


The gene ABR (NCBI ID: 29) has been mentioned in [119 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ABR%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1968 and the middle 50% of publications occurring between 1988 and 2013. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning ABR, ranked by their scientific influence, include "[Estimation of auditory brainstem response, ABR, by means of Bayesian inference.](https://pubmed.ncbi.nlm.nih.gov/4023604)" (1985) (relative citation ratio: 7.03), "[Quantifiable hearing and no ABR: paradox or error?](https://pubmed.ncbi.nlm.nih.gov/7429038)" (1980) (relative citation ratio: 5.03), "[Maturational time course for the ABR in preterm and full term infants.](https://pubmed.ncbi.nlm.nih.gov/3372369)" (1988) (relative citation ratio: 4.57), "[Interactions of age, gender, and sensorineural hearing loss on ABR latency.](https://pubmed.ncbi.nlm.nih.gov/3169397)" (1988) (relative citation ratio: 4.41), and "[Predictive value of ABR in infants and children with moderate to profound hearing impairment.](https://pubmed.ncbi.nlm.nih.gov/3653534)" (1987) (relative citation ratio: 4.22). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ABR](https://www.proteinatlas.org/ENSG00000159842-ABR) is a gene with a molecular function in GTPase activation and acts as a guanine-nucleotide releasing factor. Evidence for its function is available at the protein level. The gene is detected in many tissues and is primarily localized in the nucleoplasm and cytosol. It is expressed in various clusters, including Cluster 25 for blood with unknown function, Cluster 33 for tissue related to neuronal signaling in the brain, Cluster 51 for brain neurons with mixed function, Cluster 46 for lymphoma cell lines involved in humoral immune response, and Cluster 36 for late spermatids with unknown function.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SP1](https://www.ncbi.nlm.nih.gov/gene/6667) with 6 experiments, followed by [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113), [JARID2](https://www.ncbi.nlm.nih.gov/gene/3720), [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), and [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), each with 5 experiments.



The input data from GWAS indicates associations with a wide range of disease conditions, including various types of cancer such as leukemia, lymphoma, and immune system cancer. Additionally, respiratory conditions like asthma, bronchial disease, and obstructive lung disease are noted. Cardiovascular issues, including artery disease, vascular disease, and hypertension, are also present. Mental health disorders, such as cognitive disorder, mood disorder, and major depressive disorder, are included. Furthermore, the data highlights musculoskeletal conditions like arthritis, rheumatoid arthritis, and bone disease.



The gene is expressed in various brain regions, including the prefrontal cortex, amygdala, whole brain, fetal brain, temporal lobe, medulla oblongata, occipital lobe, and globus pallidus, as well as in the retina and colon.


The input data highlights several key pathways involving nerve growth factor (NGF) signaling, including cell death signaling via NRAGE, NRIF, and NADE, as well as NRAGE-mediated death signaling through JNK. Additionally, the data covers p75 NTR receptor-mediated signaling, G alpha (12/13) signaling events, and the Rho GTPase cycle. Other notable pathways include signaling by Rho GTPases, GPCR downstream signaling, signaling by GPCR, and general signal transduction processes.



The protein sequence analyzed has a GRAVY value of -0.402 (45.89th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -9.53 (23.2nd percentile). The median structural flexibility is 1.004 (61.49th percentile), with a predicted secondary structure composition of 35.04% helix (74.91st percentile), 36.44% sheet (69.27th percentile), and 27.24% turn (40.21st percentile). The instability index is 47.55 (48.53rd percentile), and the isoelectric point is 6.12 (33.65th percentile). The protein is 859 amino acids long (83.01st percentile) with a molecular weight of 97596.67 Da (83.59th percentile). For more details on the sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |