---
layout: post
title: RPL30
date: 2024-12-08 23:57 CST
description: RPL30 description
tags: [cooccuring-genes]
categories: longest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6156) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6156  | RPL30 | ENSG00000156482 | 8q22.2 |



The gene length is 1,697,919.0 base pairs (99.94% of all genes), the mature length is 3,683.5 base pairs (80.72% of all genes), and the primary transcript length is 583,826.0 base pairs (99.5% of all genes).


The gene RPL30, with NCBI ID 6156, has been featured in [11 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RPL30%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1986, and the middle 50% of publications occurred between 1995 and 2007.


The top five publications mentioning RPL30, ranked by their scientific influence, include "[Guideline on the management of hand eczema ICD-10 Code: L20. L23. L24. L25. L30.](https://pubmed.ncbi.nlm.nih.gov/19522920)" (2009) with a relative citation ratio of 2.82, "[Ribosomal protein L30 is a component of the UGA-selenocysteine recoding machinery in eukaryotes.](https://pubmed.ncbi.nlm.nih.gov/15821744)" (2005) with a relative citation ratio of 2.69, "[Enhanced expression of S8, L12, L23a, L27 and L30 ribosomal protein mRNAs in human hepatocellular carcinoma.](https://pubmed.ncbi.nlm.nih.gov/11724303)" (2001) with a relative citation ratio of 1.09, "[Cutaneous activation of the inhibitory L30 interneurons provides a mechanism for regulating adaptive gain control in the siphon withdrawal reflex of Aplysia.](https://pubmed.ncbi.nlm.nih.gov/7823178)" (1995) with a relative citation ratio of 0.62, and "[A taxonomic review of the genus Microbispora by analysis of ribosomal protein AT-L30.](https://pubmed.ncbi.nlm.nih.gov/8427809)" (1993) with a relative citation ratio of 0.49. The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [ARID2](https://www.ncbi.nlm.nih.gov/gene/196528) with 5 experiments, followed by [GATA2](https://www.ncbi.nlm.nih.gov/gene/2624), [OTX2](https://www.ncbi.nlm.nih.gov/gene/5015), [TRIM28](https://www.ncbi.nlm.nih.gov/gene/10155), and [INTS11](https://www.ncbi.nlm.nih.gov/gene/54973), each with 4 experiments.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [AR](https://www.ncbi.nlm.nih.gov/gene/6872) and [KLF10](https://www.ncbi.nlm.nih.gov/gene/8861). AR was shown to be regulating genes in 2 experiments, while KLF10 was found in 1 experiment.








The analyzed protein sequence has a GRAVY value of -0.295 (58.62 percentile), indicating a hydrophilic nature. It exhibits a charge of 9.64 at pH 7.0 (80.62 percentile) and a median structural flexibility of 1.0003 (43.22 percentile). The protein's secondary structure is predicted to be 39.13% helix (91.77 percentile), 33.91% sheet (51.33 percentile), and 24.35% turn (17.29 percentile). The instability index is 61.60 (85.73 percentile), with an isoelectric point of 9.65 (93.16 percentile). The protein has a length of 115 amino acids (4.08 percentile) and a molecular weight of 12783.92 Da (4.15 percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |