---
layout: post
title: HSP90AB1
date: 2024-12-08 23:57 CST
description: HSP90AB1 description
tags: [cooccuring-genes]
categories: longest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3326) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3326  | HSP90AB1 | ENSG00000096384 | 6p21.1 |



The gene length is 1,380,359.0 base pairs (99.88% of all genes), the mature length is 2,009.5 base pairs, and the primary transcript length is 1,280,893.0 base pairs.


The gene HSP90AB1, with NCBI Gene ID 3326, has been featured in [56 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22HSP90AB1%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1988, and the middle 50% of publications occurred between 2010 and 2019.


The top five publications mentioning HSP90AB1, ranked by their scientific influence, include "[Hsp90 and FKBP51: complex regulators of psychiatric diseases.](https://pubmed.ncbi.nlm.nih.gov/29203717)" (2018) with a relative citation ratio of 3.78, "[Structure-guided design of an Hsp90β N-terminal isoform-selective inhibitor.](https://pubmed.ncbi.nlm.nih.gov/29382832)" (2018) with a relative citation ratio of 3.57, "[Hsp90ab1 stabilizes LRP5 to promote epithelial-mesenchymal transition via activating of AKT and Wnt/β-catenin signaling pathways in gastric cancer progression.](https://pubmed.ncbi.nlm.nih.gov/30305727)" (2019) with a relative citation ratio of 3.3, "[HSP90AB1: Helping the good and the bad.](https://pubmed.ncbi.nlm.nih.gov/26358502)" (2016) with a relative citation ratio of 3.02, and "[MicroRNA-361-Mediated Inhibition of HSP90 Expression and EMT in Cervical Cancer Is Counteracted by Oncogenic lncRNA NEAT1.](https://pubmed.ncbi.nlm.nih.gov/32151082)" (2020) with a relative citation ratio of 2.8. The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [LEO1](https://www.ncbi.nlm.nih.gov/gene/123169), [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599), [KDM2B](https://www.ncbi.nlm.nih.gov/gene/84678), and [KMT2B](https://www.ncbi.nlm.nih.gov/gene/9757), each shown to be regulating in 5 experiments.


The top transcription factors (TFs) identified in terms of the number of Perturb-seq experiments include [8861](https://www.ncbi.nlm.nih.gov/gene/8861), [9968](https://www.ncbi.nlm.nih.gov/gene/9968), and [7776](https://www.ncbi.nlm.nih.gov/gene/7776), each shown to be regulating genes in one experiment.








The analyzed protein sequence has a GRAVY value of -0.678 (19.05th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -39.19 (4.02nd percentile), and the median structural flexibility is 1.012 (87.19th percentile). The protein has a high affinity for helix structures (40.06%, 93.47th percentile), with moderate affinities for sheet (33.15%, 46.31st percentile) and turn (25.41%, 24.51st percentile) structures. The instability index is 42.15 (29.12th percentile), and the isoelectric point is 4.97 (7.15th percentile). The protein is 724 amino acids long (76.76th percentile) with a molecular weight of 83263.21 Da (77.99th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |