---
layout: post
title: PLCB4
date: 2025-01-17 16:55 CST
description: PLCB4 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5332) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5332  | PLCB4 | ENSG00000101333 | 20p12.3-p12.2 |



The gene enables [phosphatidylinositol phospholipase C activity](https://amigo.geneontology.org/amigo/term/GO:0004435) and [phospholipase C activity](https://amigo.geneontology.org/amigo/term/GO:0004629), and is involved in [calcium ion binding](https://amigo.geneontology.org/amigo/term/GO:0005509) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829) and plays a role in the [G protein-coupled receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007186), [lipid catabolic process](https://amigo.geneontology.org/amigo/term/GO:0016042), [phosphatidylinositol metabolic process](https://amigo.geneontology.org/amigo/term/GO:0046488), [phosphatidylinositol-mediated signaling](https://amigo.geneontology.org/amigo/term/GO:0048015), and the [release of sequestered calcium ion into cytosol](https://amigo.geneontology.org/amigo/term/GO:0051209).


The gene length is 42,363 base pairs (61.79% of all genes), the mature length is 1,582 base pairs (40.6% of all genes), and the primary transcript length is 41,414 base pairs (70.85% of all genes).


The gene PLCB4 (NCBI ID: 5332) has been mentioned in [25 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PLCB4%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning this gene was in 1994, and the middle 50% of publications occurred between 2009 and 2021.


The top 5 publications mentioning PLCB4, ranked by their scientific influence, include studies that explore various aspects of the gene's function and associated conditions. The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Notable publications include "[A human homeotic transformation resulting from mutations in PLCB4 and GNAI3 causes auriculocondylar syndrome.](https://pubmed.ncbi.nlm.nih.gov/22560091)" (2012) (relative citation ratio: 1.79), "[Regulation of phospholipase C-beta 4 by ribonucleotides and the alpha subunit of Gq.](https://pubmed.ncbi.nlm.nih.gov/7929227)" (1994) (relative citation ratio: 1.71), "[Auriculocondylar syndrome 2 results from the dominant-negative action of PLCB4 variants.](https://pubmed.ncbi.nlm.nih.gov/35284927)" (2022) (relative citation ratio: 1.38), "[Phospholipase C beta 4 in the medial septum controls cholinergic theta oscillations and anxiety behaviors.](https://pubmed.ncbi.nlm.nih.gov/20007462)" (2009) (relative citation ratio: 1.32), and "[Genetic variants in PLCB4/PLCB1 as susceptibility loci for coronary artery aneurysm formation in Kawasaki disease in Han Chinese in Taiwan.](https://pubmed.ncbi.nlm.nih.gov/26434682)" (2015) (relative citation ratio: 0.97). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PLCB4](https://www.proteinatlas.org/ENSG00000101333-PLCB4) is a gene with hydrolase and transducer molecular functions, known to be involved in disease variants. Evidence for its function is available at the protein level. The gene is detected in many tissues and is particularly expressed in the cerebellum, where it is associated with nervous system development. In the brain, it is involved in non-specific transcription processes. Additionally, PLCB4 is found in dendritic cells, although its function in these cells is unknown. The gene is also expressed in kidney cancer cell lines, where it is associated with membrane components, and in serous glandular cells, where it plays a role in salivary secretion.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613), [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), [MYC](https://www.ncbi.nlm.nih.gov/gene/4609), [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), and [ZBTB7A](https://www.ncbi.nlm.nih.gov/gene/51341), each shown to be regulating in 5 experiments.



The GWAS data indicates associations with a wide range of disease conditions, including infectious diseases such as viral infectious diseases and human immunodeficiency virus infectious disease. Additionally, the data highlights various types of cancer, including leukemia, hematologic cancer, immune system cancer, and organ system cancer. Metabolic disorders, particularly diabetes mellitus and type 2 diabetes mellitus, are also implicated. Cardiovascular conditions such as coronary artery disease, hypertension, and heart disease are noted, along with respiratory diseases like lung disease and lower respiratory tract disease. Neurological conditions, including neurodegenerative diseases like Parkinson's disease and cognitive disorders, are also represented. Furthermore, the data includes kidney disease, urinary system disease, and various inherited metabolic disorders.



The gene is expressed in various tissues, including nerve, salivary gland, and pituitary, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in pineal gland (both day and night), retina, thyroid, thalamus, cerebellum peduncles, and salivary gland.


The proteins are involved in various pathways, including metabolism, G alpha (q) signaling events, the gastrin-CREB signaling pathway via PKC and MAPK, GPCR downstream signaling, signaling by GPCR, signal transduction, G-protein mediated events, PLC beta mediated events, opioid signaling, inositol phosphate metabolism, and the synthesis of IP3 and IP4 in the cytosol.


The mouse ortholog gene 18798, which corresponds to the human ortholog gene 5332, has been associated with several phenotypic abnormalities. These include abnormal gait, abnormal gallbladder morphology, decreased locomotor activity, enlarged gallbladder, increased bone mineral content, increased grip strength, increased vertical activity, and preweaning lethality with incomplete penetrance, all observed in both sexes. Additionally, abnormal uterus morphology and hydrometra were noted, although these were not observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.593 (26th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -5.94 (31.16th percentile), and the median structural flexibility is 1.007 (72.17th percentile). The protein has a notable affinity for helix structures (37.02%, 85.26th percentile), with a lower affinity for sheet (32.34%, 41.55th percentile) and turn structures (25.87%, 28.1th percentile). The instability index is 42.36 (29.94th percentile), and the isoelectric point is 6.47 (42.01st percentile). The protein sequence is 1175 amino acids long (91.24th percentile) with a molecular weight of 134462.1 Da (91.95th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |