---
layout: post
title: B4GAT1
date: 2025-01-17 16:55 CST
description: B4GAT1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/11041) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 11041  | B4GAT1 | ENSG00000174684 | 11q13.2 |



The gene is located in the [Golgi membrane](https://amigo.geneontology.org/amigo/term/GO:0000139) and the [Golgi apparatus](https://amigo.geneontology.org/amigo/term/GO:0005794). It enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [N-acetyllactosaminide beta-1,3-N-acetylglucosaminyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0008532), and [glucuronosyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0015020). The gene is involved in [axon guidance](https://amigo.geneontology.org/amigo/term/GO:0007411), the [keratan sulfate biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0018146), and [protein O-linked mannosylation](https://amigo.geneontology.org/amigo/term/GO:0035269). Additionally, it enables [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872) and is located in the [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062).


The gene length is 57,181 base pairs (69.85th percentile), the mature length is 1,815 base pairs (47.65th percentile), and the primary transcript length is 56,851 base pairs (78.19th percentile).


The gene B4GAT1 (NCBI ID: 11041) has been mentioned in [4 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22B4GAT1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning this gene was in 1993, and the middle 50% of publications occurred between 2008 and 2016.


The gene B4GAT1 has been extensively studied in the context of its role in glycosylation processes. The top publications mentioning B4GAT1 include "[The glucuronyltransferase B4GAT1 is required for initiation of LARGE-mediated α-dystroglycan functional glycosylation.](https://pubmed.ncbi.nlm.nih.gov/25279699)" (2014) (relative citation ratio: 2.4) and "[B4GAT1 is the priming enzyme for the LARGE-dependent functional glycosylation of α-dystroglycan.](https://pubmed.ncbi.nlm.nih.gov/25279697)" (2014) (relative citation ratio: 2.04). Additionally, the study "[Purification and characterization of UDP-GlcNAc:Gal beta 1-4Glc(NAc) beta-1,3-N-acetylglucosaminyltransferase (poly-N-acetyllactosamine extension enzyme) from calf serum.](https://pubmed.ncbi.nlm.nih.gov/8262950)" (1993) (relative citation ratio: 0.63) provides insights into the enzymatic functions of B4GAT1. The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[B4GAT1](https://www.proteinatlas.org/ENSG00000174684-B4GAT1) is a gene encoding a protein with glycosyltransferase and transferase molecular functions. It has been implicated in various diseases, including congenital muscular dystrophy, dystroglycanopathy, and lissencephaly. Evidence for this gene is available at the protein level. Its RNA is detected in all tissues, with notable expression in the brain, particularly in the hypothalamus, where it is involved in neuropeptide signaling. Additionally, it is expressed in liver cancer cell lines and cytotrophoblasts, where it plays a role in mitochondrial processes. The gene is part of the non-specific nuclear processes blood expression cluster and has been found to be unprognostic in several cancer types, including bladder urothelial carcinoma, breast invasive carcinoma, and others.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [MYC](https://www.ncbi.nlm.nih.gov/gene/4609) with 5 experiments, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 4 experiments, [ZBTB7A](https://www.ncbi.nlm.nih.gov/gene/51341) with 4 experiments, [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) with 4 experiments, and [OLIG2](https://www.ncbi.nlm.nih.gov/gene/10215) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.





The gene is expressed in various neural tissues, including the nerve, brain, and pituitary, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles show expression in specific brain regions such as the amygdala, whole brain, hypothalamus, and cerebellum.


The input data includes various pathways and diseases related to carbohydrate metabolism, glycosaminoglycan metabolism, and diseases of glycosylation. Notable conditions include Myoclonic epilepsy of Lafora, Glycogen storage diseases, and several types of Mucopolysaccharidoses (MPS), such as MPS VII - Sly syndrome, MPS VI - Maroteaux-Lamy syndrome, and MPS I - Hurler syndrome. Additionally, the data highlights defects in specific genes like B3GAT3, B4GALT1, and CHST14, which are associated with conditions such as JDSSDHD, B4GALT1-CDG (CDG-2d), and EDS, musculocontractural type, respectively. Other pathways mentioned include keratan sulfate biosynthesis and general metabolism.


The mouse ortholog gene 108902 and its human ortholog 11041 are associated with several phenotypic outcomes. Notably, the mouse ortholog is linked to embryonic lethality prior to organogenesis and embryonic lethality prior to the tooth bud stage, both specific to one sex. Additionally, it is associated with preweaning lethality with complete penetrance, which affects both sexes.


The analyzed protein sequence has a GRAVY value of -0.224 (66.78th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -0.732 (47.61st percentile). The median structural flexibility is 0.994 (17.88th percentile), with a propensity for helix, sheet, and turn structures of 34.7% (72.84th percentile), 35.4% (62.16th percentile), and 25.8% (27.38th percentile), respectively. The instability index is 51.8 (62.85th percentile), suggesting moderate stability. The isoelectric point is 6.77 (47.35th percentile), with a molecular weight of 47118.5 Da (48.49th percentile). The sequence length is 415 amino acids (47.26th percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |