---
layout: post
title: POLR2K
date: 2024-12-08 23:57 CST
description: POLR2K description
tags: [cooccuring-genes]
categories: shortest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5440) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5440  | POLR2K | ENSG00000147669 | 8q22.2 |



The gene length is 8,030 base pairs (21.38% of all genes), with a mature length of 409 base pairs and a primary transcript length of 409 base pairs.


The gene POLR2K, with NCBI ID 5440, has been featured in [3 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22POLR2K%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727). The earliest year of publication is 2000, and the middle 50% of publications occurred between 2003 and 2008.


The top publications mentioning POLR2K include "[BRCA1 interaction with RNA polymerase II reveals a role for hRPB2 and hRPB10alpha in activated transcription.](https://pubmed.ncbi.nlm.nih.gov/10725406)" (2000) (relative citation ratio: 0.76), "[The archaeal RNA polymerase subunit P and the eukaryotic polymerase subunit Rpb12 are interchangeable in vivo and in vitro.](https://pubmed.ncbi.nlm.nih.gov/19183282)" (2009) (relative citation ratio: 0.28), and "[The NS5A protein of hepatitis C virus represses gene expression of hRPB10alpha, a common subunit of host RNA polymerases, through interferon regulatory factor-1 binding site.](https://pubmed.ncbi.nlm.nih.gov/17714821)" (2007) (relative citation ratio: 0.02). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664) and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), each with 5 experiments. Additionally, [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), [MAX](https://www.ncbi.nlm.nih.gov/gene/4149), and [STAT5B](https://www.ncbi.nlm.nih.gov/gene/6777) were each found to be regulating in 4 experiments.











The analyzed protein sequence exhibits a GRAVY value of -1.007 (4.27th percentile), indicating a hydrophilic nature. It has a charge of 4.55 at pH 7.0 (66.46th percentile) and a median structural flexibility of 1.007 (73.26th percentile). The protein's secondary structure is predicted to be 25.86% helix (7.23rd percentile), 27.59% sheet (16.76th percentile), and 20.69% turn (3.35th percentile). The instability index is 61.74 (85.91st percentile), and the isoelectric point is 9.27 (86.13th percentile). The protein has a length of 58 amino acids (0.22nd percentile) and a molecular weight of 7004.14 Da (0.38th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |