---
layout: post
title: PPIC
date: 2025-01-17 16:55 CST
description: PPIC description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5480) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5480  | PPIC | ENSG00000168938 | 5q23.2 |



The gene is involved in [protein peptidyl-prolyl isomerization](https://amigo.geneontology.org/amigo/term/GO:0000413) and [protein folding](https://amigo.geneontology.org/amigo/term/GO:0006457). It enables [peptidyl-prolyl cis-trans isomerase activity](https://amigo.geneontology.org/amigo/term/GO:0003755), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [cyclosporin A binding](https://amigo.geneontology.org/amigo/term/GO:0016018). The gene is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737) and [intracellular membrane-bounded organelle](https://amigo.geneontology.org/amigo/term/GO:0043231), and is located in the [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062).


The gene length is 21,050 base pairs (43.29% of all genes), the mature length is 1,321.5 base pairs, and the primary transcript length is 16,524.5 base pairs.


The gene PPIC (NCBI ID: 5480) has been mentioned in [8 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PPIC%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1993, and the middle 50% of publications range from 1994 to 2017.


The top 5 publications mentioning PPIC, ranked by their scientific influence, include studies that have significantly contributed to the understanding of cyclophilins and their roles in various biological contexts. The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Notable publications include "[Cloning and characterization of cyclophilin C-associated protein: a candidate natural cellular ligand for cyclophilin C.](https://pubmed.ncbi.nlm.nih.gov/8341703)" (1993) (relative citation ratio: 1.85), "[Human cyclophilin C: primary structure, tissue distribution, and determination of binding specificity for cyclosporins.](https://pubmed.ncbi.nlm.nih.gov/8031755)" (1994) (relative citation ratio: 1.54), "[PPIC, EMP3 and CHI3L1 Are Novel Prognostic Markers for High Grade Glioma.](https://pubmed.ncbi.nlm.nih.gov/27801851)" (2016) (relative citation ratio: 1.23), "[Crosstalk between cyclophilins and T lymphocytes in coronary artery disease.](https://pubmed.ncbi.nlm.nih.gov/33582093)" (2021) (relative citation ratio: 1.23), and "[Depletion of cyclophilins B and C leads to dysregulation of endoplasmic reticulum redox homeostasis.](https://pubmed.ncbi.nlm.nih.gov/24990953)" (2014) (relative citation ratio: 1.02). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PPIC](https://www.proteinatlas.org/ENSG00000168938-PPIC) is a gene encoding a protein with isomerase and rotamase molecular functions. Evidence for this protein is established at the protein level. The RNA expression of PPIC is detected in many tissues. The gene is notably expressed in connective tissue, contributing to extracellular matrix (ECM) organization, as indicated by its inclusion in Tissue expression cluster 62. Additionally, PPIC is expressed in the choroid plexus, with a role in mitochondrial function, as seen in Brain expression cluster 42. In cell lines, PPIC is associated with kidney cancer and membrane components, as part of Cell line expression cluster 15. Single-cell expression data also place PPIC in connective tissue cells involved in ECM organization, as per Single cell expression cluster 50.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) and [SUZ12](https://www.ncbi.nlm.nih.gov/gene/23512), each with 6 experiments. Additionally, [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885), [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), and [AR](https://www.ncbi.nlm.nih.gov/gene/367) were each found to be regulating in 5 experiments.


The transcription factor [8243](https://www.ncbi.nlm.nih.gov/gene/8243) was found to be regulating genes in 2 experiments according to Perturb-seq data.




The gene is expressed in various tissues, including blood vessel, thyroid, and colon, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in thyroid, adipocyte, smooth muscle, pancreas, colon, and small intestine, suggesting that the gene is ubiquitously expressed in multiple tissue samples.




The protein sequence analyzed has a GRAVY value of 0.0736 (87.07th percentile), indicating a relatively hydrophobic nature. At pH 7.0, the protein carries a charge of 1.93 (57.29th percentile). The median structural flexibility is 0.998 (30.72nd percentile), with a secondary structure composition of 23.58% helix (3.05th percentile), 41.51% sheet (89.76th percentile), and 33.49% turn (84.05th percentile). The instability index is 6.57 (0.08th percentile), and the isoelectric point is 8.48 (66.27th percentile). The protein has a length of 212 amino acids (16.3rd percentile) and a molecular weight of 22762.98 Da (15.09th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |