---
layout: post
title: SLC25A27
date: 2025-01-17 16:55 CST
description: SLC25A27 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9481) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9481  | SLC25A27 | ENSG00000153291 | 6p12.3  |



The gene is located in the [mitochondrion](https://amigo.geneontology.org/amigo/term/GO:0005739), specifically in the [mitochondrial inner membrane](https://amigo.geneontology.org/amigo/term/GO:0005743) and the [mitochondrial membrane](https://amigo.geneontology.org/amigo/term/GO:0031966). It is also found in the [neuronal cell body](https://amigo.geneontology.org/amigo/term/GO:0043025) and the [apical part of the cell](https://amigo.geneontology.org/amigo/term/GO:0045177). The gene is involved in various processes, including the [positive regulation of cell population proliferation](https://amigo.geneontology.org/amigo/term/GO:0008284), [response to cold](https://amigo.geneontology.org/amigo/term/GO:0009409), [negative regulation of mitochondrial membrane potential](https://amigo.geneontology.org/amigo/term/GO:0010917), [intracellular triglyceride homeostasis](https://amigo.geneontology.org/amigo/term/GO:0035356), [negative regulation of neuron apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0043524), [regulation of glucose import](https://amigo.geneontology.org/amigo/term/GO:0046324), [inner ear development](https://amigo.geneontology.org/amigo/term/GO:0048839), [negative regulation of mitochondrial calcium ion concentration](https://amigo.geneontology.org/amigo/term/GO:0051562), [chloride transmembrane transport](https://amigo.geneontology.org/amigo/term/GO:1902476), and [proton transmembrane transport](https://amigo.geneontology.org/amigo/term/GO:1902600). Additionally, the gene enables [proton transmembrane transporter activity](https://amigo.geneontology.org/amigo/term/GO:0015078), [chloride transmembrane transporter activity](https://amigo.geneontology.org/amigo/term/GO:0015108), [transmembrane transporter activity](https://amigo.geneontology.org/amigo/term/GO:0022857), and [protein homodimerization activity](https://amigo.geneontology.org/amigo/term/GO:0042803).


The gene length is 175,938 base pairs (90.98% of all genes), the mature length is 866 base pairs (16.84% of all genes), and the primary transcript length is 158,729 base pairs (94.17% of all genes).


The gene SLC25A27, with NCBI ID 9481, has been mentioned in [19 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SLC25A27%22) (note: the count in the hyperlink may be different since the count is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication mentioning this gene is 1999, and the middle 50% of publications occurred between 2007 and 2013.


The top 5 publications mentioning SLC25A27, ranked by their scientific influence, include "[UCP4, a novel brain-specific mitochondrial protein that reduces membrane potential in mammalian cells.](https://pubmed.ncbi.nlm.nih.gov/10025957)" (1999) (relative citation ratio: 6.73), "[Human neuronal uncoupling proteins 4 and 5 (UCP4 and UCP5): structural properties, regulation, and physiological role in protection against oxidative stress and mitochondrial dysfunction.](https://pubmed.ncbi.nlm.nih.gov/22950050)" (2012) (relative citation ratio: 2.62), "[Mitochondrial uncoupling protein-4 regulates calcium homeostasis and sensitivity to store depletion-induced apoptosis in neural cells.](https://pubmed.ncbi.nlm.nih.gov/17035241)" (2006) (relative citation ratio: 2.04), "[Homologues of the uncoupling protein from brown adipose tissue (UCP1): UCP2, UCP3, BMCP1 and UCP4.](https://pubmed.ncbi.nlm.nih.gov/11239488)" (2001) (relative citation ratio: 1.81), and "[Mitochondrial uncoupling proteins and phylogenesis--UCP4 as the ancestral uncoupling protein.](https://pubmed.ncbi.nlm.nih.gov/11334880)" (2001) (relative citation ratio: 1.63). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[SLC25A27](https://www.proteinatlas.org/ENSG00000153291-SLC25A27) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in the mitochondria and is associated with neuronal transcription in various tissue expression clusters, including Cluster 64 for general tissue, Cluster 15 for brain tissue, and Cluster 11 for cell line expression. Additionally, it is found in Cluster 76 for single-cell expression, specifically in oligodendrocyte precursors with an unknown function.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [YY1](https://www.ncbi.nlm.nih.gov/gene/7528), [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), and [TP53](https://www.ncbi.nlm.nih.gov/gene/7157), each with 5 experiments. Additionally, [AR](https://www.ncbi.nlm.nih.gov/gene/367) and [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) were found to be regulating in 4 experiments each.



The GWAS data indicates associations with a range of conditions, including coronary artery disease, artery disease, vascular disease, and broader cardiovascular system diseases. Additionally, the data highlights links to general disease categories, diseases of anatomical entities, and mental health disorders. Specifically, substance-related disorders, substance dependence, and alcohol dependence are also implicated.



The gene is expressed in the ovary and pituitary.




The analyzed protein sequence has a GRAVY value of -0.077 (79.65th percentile), indicating a hydrophilic nature. It exhibits a charge of 6.12 at pH 7.0 (71.62nd percentile) and a median structural flexibility of 0.993 (16.81st percentile). The protein's secondary structure is predicted to be 35.0% helix (74.59th percentile), 37.8% sheet (77.29th percentile), and 25.7% turn (26.71st percentile). The instability index is 40.6 (23.96th percentile), with an isoelectric point of 9.15 (83.12th percentile). The protein is 323 amino acids long (33.42nd percentile) and has a molecular weight of 36063.6 Da (33.65th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |