---
layout: post
title: SLC9A2
date: 2025-01-17 16:55 CST
description: SLC9A2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6549) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6549  | SLC9A2 | ENSG00000115616 | 2q12.1 |



The gene is located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886) and [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), specifically in the [apical plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0016324). It is involved in various processes such as [monoatomic ion transport](https://amigo.geneontology.org/amigo/term/GO:0006811), [protein localization](https://amigo.geneontology.org/amigo/term/GO:0008104), [epithelial cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0030855), [regulation of intracellular pH](https://amigo.geneontology.org/amigo/term/GO:0051453), [potassium ion transmembrane transport](https://amigo.geneontology.org/amigo/term/GO:0071805), [sodium ion import across plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0098719), and [proton transmembrane transport](https://amigo.geneontology.org/amigo/term/GO:1902600). The gene enables [sodium:proton antiporter activity](https://amigo.geneontology.org/amigo/term/GO:0015385) and [potassium:proton antiporter activity](https://amigo.geneontology.org/amigo/term/GO:0015386).


The gene length is 75,574 base pairs (76.17th percentile), the mature length is 8,294 base pairs (97.98th percentile), and the primary transcript length is 33,704 base pairs (65.76th percentile).


The gene SLC9A2 (NCBI ID: 6549) has been mentioned in [32 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SLC9A2%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1992 and the middle 50% of publications occurring between 1997 and 2012. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning SLC9A2, ranked by their scientific influence, include "[NHE2 and NHE3 are human and rabbit intestinal brush-border proteins.](https://pubmed.ncbi.nlm.nih.gov/8772498)" (1996) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 6.45, "[Cloning and expression of a rabbit cDNA encoding a serum-activated ethylisopropylamiloride-resistant epithelial Na+/H+ exchanger isoform (NHE-2).](https://pubmed.ncbi.nlm.nih.gov/7685025)" (1993) with an RCR of 5.24, "[Intestinal distribution of human Na+/H+ exchanger isoforms NHE-1, NHE-2, and NHE-3 mRNA.](https://pubmed.ncbi.nlm.nih.gov/8843774)" (1996) with an RCR of 2.35, "[IFN-gamma downregulates expression of Na(+)/H(+) exchangers NHE2 and NHE3 in rat intestine and human Caco-2/bbe cells.](https://pubmed.ncbi.nlm.nih.gov/11287336)" (2001) with an RCR of 2.33, and "[Macula densa Na(+)/H(+) exchange activities mediated by apical NHE2 and basolateral NHE4 isoforms.](https://pubmed.ncbi.nlm.nih.gov/10710550)" (2000) with an RCR of 1.62. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[SLC9A2](https://www.proteinatlas.org/ENSG00000115616-SLC9A2) is a gene with evidence at the protein level. Its RNA is detected in many tissues, with notable expression in the intestine, where it is involved in digestion, and in neurons and synapses, suggesting a role in synaptic function. At the subcellular level, the protein is localized to cell junctions. In single-cell expression analysis, it is found in intestinal goblet cells, indicating a role in mucin production. The gene is also expressed in various cell lines associated with cell proliferation. While SLC9A2 is not prognostic for several cancer types, its expression patterns suggest diverse functional roles in different tissue contexts.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [AR](https://www.ncbi.nlm.nih.gov/gene/367), [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046), [JARID2](https://www.ncbi.nlm.nih.gov/gene/3720), [KDM2B](https://www.ncbi.nlm.nih.gov/gene/84678), and [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227), each shown to be regulating in 5 experiments.



The input data from GWAS indicates associations with a wide range of disease conditions, including respiratory disorders such as asthma, bronchial disease, obstructive lung disease, lower respiratory tract disease, lung disease, and respiratory system disease. Metabolic conditions are also represented, including disease of metabolism, acquired metabolic disease, carbohydrate metabolism disease, glucose metabolism disease, diabetes mellitus, and type 2 diabetes mellitus. Cardiovascular and nervous system diseases are noted, encompassing cardiovascular system disease, congestive heart failure, heart disease, nervous system disease, central nervous system disease, cognitive disorder, disease of mental health, bipolar disorder, mood disorder, substance-related disorder, substance dependence, alcohol dependence, anxiety disorder, obsessive-compulsive disorder, tic disorder, Gilles de la Tourette syndrome, developmental disorder of mental health, specific developmental disorder, and attention deficit hyperactivity disorder. Additionally, the data includes bone inflammation disease, arthritis, bone disease, connective tissue disease, musculoskeletal system disease, personality disorder, colitis, ulcerative colitis, inflammatory bowel disease, intestinal disease, gastrointestinal system disease, and brain disease.



The gene is expressed in various tissues, including the colon and stomach, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in cardiac myocytes, heart, liver, cingulate cortex, bronchial epithelial cells, colon, and testis interstitial cells, suggesting a ubiquitous expression pattern across multiple tissue samples.


The input data highlights several key pathways involved in transmembrane transport processes. These include the transport of inorganic cations/anions and amino acids/oligopeptides, SLC-mediated transmembrane transport, the transmembrane transport of small molecules, and the activity of sodium/proton exchangers.



The analyzed protein sequence has a GRAVY value of 0.1047 (88.1% percentile), indicating a relatively hydrophobic nature. The protein exhibits a charge of 16.80 at pH 7.0 (90.84% percentile) and an isoelectric point of 9.197 (84.35% percentile). The structural flexibility at the median is 0.991 (12.21% percentile), with a length of 812 amino acids (81.23% percentile) and a molecular weight of 91518.68 Da (81.39% percentile). The secondary structure affinities are 31.03% for helix (41.34% percentile), 42.86% for sheet (92.11% percentile), and 26.11% for turn (30.14% percentile). The instability index is 46.13 (43.42% percentile). These values were calculated using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |