---
layout: post
title: GRSF1
date: 2025-01-17 16:55 CST
description: GRSF1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2926) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2926  | GRSF1 | ENSG00000132463 | 4q13.3 |



The gene is involved in various biological processes, including the [mitochondrial RNA catabolic process](https://amigo.geneontology.org/amigo/term/GO:0000957) and its positive regulation, as well as [mRNA polyadenylation](https://amigo.geneontology.org/amigo/term/GO:0006378), [tRNA processing](https://amigo.geneontology.org/amigo/term/GO:0008033), [anterior/posterior pattern specification](https://amigo.geneontology.org/amigo/term/GO:0009952), [morphogenesis of embryonic epithelium](https://amigo.geneontology.org/amigo/term/GO:0016331), and [regulation of RNA splicing](https://amigo.geneontology.org/amigo/term/GO:0043484). It enables functions such as [G-quadruplex RNA binding](https://amigo.geneontology.org/amigo/term/GO:0002151), [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [mRNA binding](https://amigo.geneontology.org/amigo/term/GO:0003729), and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is active in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654) and is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [mitochondrion](https://amigo.geneontology.org/amigo/term/GO:0005739), [mitochondrial matrix](https://amigo.geneontology.org/amigo/term/GO:0005759), [ribonucleoprotein granule](https://amigo.geneontology.org/amigo/term/GO:0035770), and [mitochondrial nucleoid](https://amigo.geneontology.org/amigo/term/GO:0042645). Additionally, it is part of the [ribonucleoprotein complex](https://amigo.geneontology.org/amigo/term/GO:1990904).


The gene length is 11,081 base pairs (27.87% of all genes), the mature length is 1,738 base pairs (45.43% of all genes), and the primary transcript length is 6,922.5 base pairs (24.68% of all genes).


The gene GRSF1 (NCBI ID: 2926) has been mentioned in [23 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22GRSF1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1994 and the middle 50% of publications occurring between 2013 and 2020. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning GRSF1, ranked by their scientific influence, include "[HuR and GRSF1 modulate the nuclear export and mitochondrial localization of the lncRNA RMRP.](https://pubmed.ncbi.nlm.nih.gov/27198227)" (2016) with a relative citation ratio of 5.11, "[The mitochondrial RNA-binding protein GRSF1 localizes to RNA granules and is required for posttranscriptional mitochondrial gene expression.](https://pubmed.ncbi.nlm.nih.gov/23473033)" (2013) with a relative citation ratio of 4.37, "[GRSF1 regulates RNA processing in mitochondrial RNA granules.](https://pubmed.ncbi.nlm.nih.gov/23473034)" (2013) with a relative citation ratio of 4.31, "[GRSF1-mediated MIR-G-1 promotes malignant behavior and nuclear autophagy by directly upregulating TMED5 and LMNB1 in cervical cancer cells.](https://pubmed.ncbi.nlm.nih.gov/30394198)" (2019) with a relative citation ratio of 3.16, and "[miR-346 and miR-138 competitively regulate hTERT in GRSF1- and AGO2-dependent manners, respectively.](https://pubmed.ncbi.nlm.nih.gov/26507454)" (2015) with a relative citation ratio of 2.23. The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[GRSF1](https://www.proteinatlas.org/ENSG00000132463-GRSF1) is an RNA-binding protein with evidence at the protein level. It is detected in all RNA tissue distributions and is primarily localized in the mitochondria. GRSF1 is expressed in various clusters, including Cluster 8 in blood, specifically in plasmacytoid dendritic cells with unknown function, and Cluster 35 in tissues, indicating non-specific mitochondrial involvement. In the brain, it is part of Cluster 21, associated with non-specific metabolism, and in cell lines, it is found in Cluster 22, linked to mRNA splicing and the cell cycle. Additionally, GRSF1 is expressed in single-cell clusters, specifically Cluster 14, which includes Langerhans cells involved in the innate immune response.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), [TAF1](https://www.ncbi.nlm.nih.gov/gene/6872), and [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861), each with 6 experiments. Additionally, [MAX](https://www.ncbi.nlm.nih.gov/gene/4149) and [PGR](https://www.ncbi.nlm.nih.gov/gene/5241) were found to be regulating in 5 experiments each.



The GWAS data indicates associations with various disease conditions, including iris disease, uveal disease, nervous system disease, eye and adnexa disease, eye disease, and sensory system disease.



The gene is expressed in various tissues and cell types, including muscle and adrenal gland. Additionally, it is found in BDCA4+ dendritic cells, promyelocytic leukemia (HL-60), Burkitt's lymphoma (Daudi), amygdala, bronchial epithelial cells, thymus, and lymphoblastic leukemia (MOLT-4).


The input data highlights several pathways related to influenza, including viral mRNA translation, influenza infection, the influenza life cycle, and influenza viral RNA transcription and replication. Additionally, the data mentions disease as a broader category.



The analyzed protein sequence has a GRAVY value of -0.327 (54.81% percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -8.44 (25.43% percentile), and the median structural flexibility is 1.000 (41.63% percentile). The protein has a high affinity for helix structures (35.42%, 77.36% percentile) and a moderate affinity for sheet structures (31.25%, 35.33% percentile). The instability index is 54.86 (71.75% percentile), suggesting potential instability. The isoelectric point is 5.83 (26.16% percentile), with a molecular weight of 53125.73 Da (55.23% percentile). The sequence length is 480 amino acids (55.91% percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |