---
layout: post
title: GNA11
date: 2025-01-17 16:55 CST
description: GNA11 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2767) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2767  | GNA11 | ENSG00000088256 | 19p13.3 |



The gene is involved in [skeletal system development](https://amigo.geneontology.org/amigo/term/GO:0001501) and [action potential](https://amigo.geneontology.org/amigo/term/GO:0001508). It enables [G protein-coupled receptor binding](https://amigo.geneontology.org/amigo/term/GO:0001664) and is located in the [photoreceptor outer segment](https://amigo.geneontology.org/amigo/term/GO:0001750). The gene also enables [GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0003924), [G protein activity](https://amigo.geneontology.org/amigo/term/GO:0003925), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [GTP binding](https://amigo.geneontology.org/amigo/term/GO:0005525). It is active in and located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), as well as located in the [lysosomal membrane](https://amigo.geneontology.org/amigo/term/GO:0005765), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), and [synapse](https://amigo.geneontology.org/amigo/term/GO:0045202). Additionally, it is part of the [heterotrimeric G-protein complex](https://amigo.geneontology.org/amigo/term/GO:0005834) and located in the [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). The gene is involved in various processes such as [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), [adenylate cyclase-modulating G protein-coupled receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007188), [phospholipase C-activating G protein-coupled acetylcholine receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007207), [G protein-coupled acetylcholine receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007213), [heart development](https://amigo.geneontology.org/amigo/term/GO:0007507), [phototransduction, visible light](https://amigo.geneontology.org/amigo/term/GO:0007603), [regulation of blood pressure](https://amigo.geneontology.org/amigo/term/GO:0008217), [entrainment of circadian clock](https://amigo.geneontology.org/amigo/term/GO:0009649), [positive regulation of insulin secretion](https://amigo.geneontology.org/amigo/term/GO:0032024), [regulation of melanocyte differentiation](https://amigo.geneontology.org/amigo/term/GO:0045634), [developmental pigmentation](https://amigo.geneontology.org/amigo/term/GO:0048066), [phospholipase C-activating dopamine receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0060158), [cellular response to pH](https://amigo.geneontology.org/amigo/term/GO:0071467), [endothelin receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0086100), [ion channel modulating, G protein-coupled receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0099105), [cranial skeletal system development](https://amigo.geneontology.org/amigo/term/GO:1904888), and [ligand-gated ion channel signaling pathway](https://amigo.geneontology.org/amigo/term/GO:1990806). The gene enables [enzyme regulator activity](https://amigo.geneontology.org/amigo/term/GO:0030234), [G-protein beta/gamma-subunit complex binding](https://amigo.geneontology.org/amigo/term/GO:0031683), and [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872).


The gene length is 75,831 base pairs (76.27th percentile), the mature length is 3,859 base pairs (82.57th percentile), and the primary transcript length is 75,785 base pairs (83.8th percentile).


The gene GNA11 (NCBI ID: 2767) has been mentioned in [102 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22GNA11%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1992 and the middle 50% of publications occurring between 2013 and 2019. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning GNA11, ranked by their scientific influence, include "[Mutations in GNA11 in uveal melanoma.](https://pubmed.ncbi.nlm.nih.gov/21083380)" (2010) (relative citation ratio: 24.73), "[Mutations affecting G-protein subunit Î±11 in hypercalcemia and hypocalcemia.](https://pubmed.ncbi.nlm.nih.gov/23802516)" (2013) (relative citation ratio: 7.67), "[Combined PKC and MEK inhibition in uveal melanoma with GNAQ and GNA11 mutations.](https://pubmed.ncbi.nlm.nih.gov/24141786)" (2014) (relative citation ratio: 4.52), "[Mosaic Activating Mutations in GNA11 and GNAQ Are Associated with Phakomatosis Pigmentovascularis and Extensive Dermal Melanocytosis.](https://pubmed.ncbi.nlm.nih.gov/26778290)" (2016) (relative citation ratio: 4.46), and "[A somatic GNA11 mutation is associated with extremity capillary malformation and overgrowth.](https://pubmed.ncbi.nlm.nih.gov/28120216)" (2017) (relative citation ratio: 4.05). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[GNA11](https://www.proteinatlas.org/ENSG00000088256-GNA11) is a transducer gene with evidence at the protein level, implicated in cancer-related diseases and disease variants. Its RNA is detected in all tissues, with notable expression in dendritic cells of unknown function (Cluster 2). The gene is also expressed in various tissues and cell types, including those involved in endocytosis (Cluster 80), transcription (Cluster 15 and Cluster 23), and digestion (Cluster 22).


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) and [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613), each with 5 experiments. Additionally, [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227), and [PGR](https://www.ncbi.nlm.nih.gov/gene/5241) were each found to be regulating in 4 experiments.





The gene is expressed in various tissues, including the colon and small intestine according to BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles indicate expression in the retina, cerebellum, cerebellum peduncles, prostate, lung, colon, and small intestine.


The input data highlights various pathways involving proteins, including metabolism, ADP signalling through P2Y purinoceptor 1, signal amplification, thromboxane signalling through TP receptor, and thrombin signalling through proteinase activated receptors (PARs). Additionally, it covers the integration of energy metabolism, regulation of insulin secretion, G alpha (q) signalling events, and the gastrin-CREB signalling pathway via PKC and MAPK. Other notable pathways include GPCR downstream signalling, signalling by GPCR, signal transduction, and the regulation of insulin secretion by acetylcholine and fatty acids bound to GPR40 (FFAR1). The data also encompasses hemostasis and platelet activation, signalling, and aggregation.


The mouse ortholog gene 14672 and its human ortholog gene 2767 exhibit several phenotypic associations in both sexes, including decreased erythrocyte cell number, decreased hematocrit, decreased hemoglobin content, increased circulating calcium level, increased red blood cell distribution width, thrombocytopenia, and vertebral fusion.


The analyzed protein sequence has a GRAVY value of -0.348 (52.45th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -7.04 (28.3rd percentile). The median structural flexibility is 0.998 (32.87th percentile), with a propensity for helix, sheet, and turn structures of 34.0% (67.73rd percentile), 40.1% (86.62nd percentile), and 21.7% (5.78th percentile), respectively. The instability index is 47.3 (47.7th percentile), and the isoelectric point is 5.51 (18.62nd percentile). The protein has a length of 359 amino acids (39.48th percentile) and a molecular weight of 42122.84 Da (42.26th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |