---
layout: post
title: SNRNP70
date: 2025-01-17 16:55 CST
description: SNRNP70 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6625) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6625  | SNRNP70 | ENSG00000104852 | 19q13.33 |



The gene is involved in [mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0000398) and enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [mRNA binding](https://amigo.geneontology.org/amigo/term/GO:0003729), and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), and is part of the [spliceosomal complex](https://amigo.geneontology.org/amigo/term/GO:0005681), [U1 snRNP](https://amigo.geneontology.org/amigo/term/GO:0005685), and [U2-type prespliceosome](https://amigo.geneontology.org/amigo/term/GO:0071004). Additionally, it enables [U1 snRNA binding](https://amigo.geneontology.org/amigo/term/GO:0030619) and is involved in the [regulation of ATP-dependent activity](https://amigo.geneontology.org/amigo/term/GO:0043462), [regulation of RNA splicing](https://amigo.geneontology.org/amigo/term/GO:0043484), [positive regulation of mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0048026), [negative regulation of protein refolding](https://amigo.geneontology.org/amigo/term/GO:0061084), and [negative regulation of chaperone-mediated autophagy](https://amigo.geneontology.org/amigo/term/GO:1904715). It is also located in the [nuclear speck](https://amigo.geneontology.org/amigo/term/GO:0016607).


The gene length is 3,389 base pairs (9.5% of all genes). The mature length and primary transcript length are also 3,389 base pairs.


The gene SNRNP70 (NCBI ID: 6625) has been mentioned in [84 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SNRNP70%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1985 and the middle 50% of publications occurring between 1994 and 2014. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning SNRNP70, ranked by their scientific influence, include "[Functional expression of cloned human splicing factor SF2: homology to RNA-binding proteins, U1 70K, and Drosophila splicing regulators.](https://pubmed.ncbi.nlm.nih.gov/1830244)" (1991) (relative citation ratio: 8.48), "[The neonatal lupus syndrome associated with U1RNP (nRNP) antibodies.](https://pubmed.ncbi.nlm.nih.gov/3494943)" (1987) (relative citation ratio: 5.5), "[U1 snRNP inhibits pre-mRNA polyadenylation through a direct interaction between U1 70K and poly(A) polymerase.](https://pubmed.ncbi.nlm.nih.gov/9659922)" (1998) (relative citation ratio: 3.72), "[Interaction between the RNA binding domains of Ser-Arg splicing factor 1 and U1-70K snRNP protein determines early spliceosome assembly.](https://pubmed.ncbi.nlm.nih.gov/21536904)" (2011) (relative citation ratio: 3.34), and "[Characteristics and Survival of Anti-U1 RNP Antibody-Positive Patients With Connective Tissue Disease-Associated Pulmonary Arterial Hypertension.](https://pubmed.ncbi.nlm.nih.gov/26415038)" (2016) (relative citation ratio: 2.92). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[SNRNP70](https://www.proteinatlas.org/ENSG00000104852-SNRNP70) is a gene with a molecular function in ribonucleoprotein and RNA-binding. It is detected in all tissues and primarily localized in the nucleoplasm. The gene is expressed in various clusters, including Cluster 22 in eosinophils related to transcription, Cluster 66 in non-specific tissues related to transcription, Cluster 4 in the brain related to mRNA splicing and cell cycle, Cluster 35 in the RS4;11 cell line with an unknown function, and Cluster 39 in single cells related to basic cellular processes. Evidence for this gene is available at the protein level.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [JUN](https://www.ncbi.nlm.nih.gov/gene/3725), [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908), [CDX2](https://www.ncbi.nlm.nih.gov/gene/1045), [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), and [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), each shown to be regulating in 5 experiments.



The GWAS data indicates associations with several disease conditions, including leukocyte disease, leukopenia, hematopoietic system disease, immune system disease, agranulocytosis, and neutropenia. These findings highlight the involvement of the immune and hematopoietic systems in the genetic underpinnings of these conditions.



The gene is expressed in the ovary and prostate according to BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles indicate expression in the Prefrontal Cortex, CD4+ T cells, CD8+ T cells, and Skeletal Muscle.


The proteins are involved in several key processes, including gene expression, processing of capped intron-containing pre-mRNA, and mRNA splicing, with a particular focus on the major pathway of mRNA splicing.


The mouse ortholog gene 20637 and its human ortholog gene 6625 are associated with the phenotype of preweaning lethality with complete penetrance, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -1.8398 (0.12 percentile), indicating a high hydrophilicity. It exhibits a charge of 19.70 at pH 7.0 (93.3 percentile) and a median structural flexibility of 1.0262 (98.82 percentile). The protein's secondary structure affinities are 28.38% for helix (19.41 percentile), 16.25% for sheet (0.86 percentile), and 34.55% for turn (87.94 percentile). The instability index is 79.51 (98.14 percentile), with an isoelectric point of 9.94 (96.0 percentile). The protein has a length of 437 amino acids (50.34 percentile) and a molecular weight of 51556.08 Da (53.41 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |