---
layout: post
title: FNTA
date: 2025-01-17 16:55 CST
description: FNTA description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2339) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2339  | FNTA | ENSG00000168522 | 8p11.21 |



The gene contributes to and enables [protein farnesyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0004660), enables [protein geranylgeranyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0004661), contributes to [CAAX-protein geranylgeranyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0004662), and enables [Rab geranylgeranyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0004663). It also enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737). The gene is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829) and is part of the [microtubule associated complex](https://amigo.geneontology.org/amigo/term/GO:0005875) and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). Additionally, it is part of the [CAAX-protein geranylgeranyltransferase complex](https://amigo.geneontology.org/amigo/term/GO:0005953) and [protein farnesyltransferase complex](https://amigo.geneontology.org/amigo/term/GO:0005965). The gene is involved in the [transforming growth factor beta receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007179), enables [microtubule binding](https://amigo.geneontology.org/amigo/term/GO:0008017), and is involved in [protein farnesylation](https://amigo.geneontology.org/amigo/term/GO:0018343) and [protein geranylgeranylation](https://amigo.geneontology.org/amigo/term/GO:0018344). It enables [receptor tyrosine kinase binding](https://amigo.geneontology.org/amigo/term/GO:0030971) and is involved in the [positive regulation of Rac protein signal transduction](https://amigo.geneontology.org/amigo/term/GO:0035022). The gene enables [alpha-tubulin binding](https://amigo.geneontology.org/amigo/term/GO:0043014) and [molecular adaptor activity](https://amigo.geneontology.org/amigo/term/GO:0060090). It is also involved in [skeletal muscle acetylcholine-gated channel clustering](https://amigo.geneontology.org/amigo/term/GO:0071340), [positive regulation of tubulin deacetylation](https://amigo.geneontology.org/amigo/term/GO:0090044), [positive regulation of deacetylase activity](https://amigo.geneontology.org/amigo/term/GO:0090045), and [positive regulation of skeletal muscle acetylcholine-gated channel clustering](https://amigo.geneontology.org/amigo/term/GO:1904395).


The gene has a length of 3829 base pairs (10.65% of all genes), a mature length of 2278 base pairs, and a primary transcript length of 3190 base pairs.


The gene FNTA (NCBI ID: 2339) has been mentioned in [1 publication](https://pubmed.ncbi.nlm.nih.gov/?term=%22FNTA%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1993, and the middle 50% of publications occurred between 1993 and 1993.


The top publication mentioning FNTA is "[cDNA cloning of the two subunits of human CAAX farnesyltransferase and chromosomal mapping of FNTA and FNTB loci and related sequences.](https://pubmed.ncbi.nlm.nih.gov/8276393)" (1993) (relative citation ratio: 0.39). The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, and benchmarking to the median for NIH publications, the value of which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[FNTA](https://www.proteinatlas.org/ENSG00000168522-FNTA) is a gene encoding a protein with prenyltransferase and transferase molecular functions. It is an FDA-approved drug target with evidence at the protein level. The gene is detected in all tissues and is primarily localized in the cytosol. Its expression is associated with basic cellular processes and is particularly notable in oligodendrocytes within the brain. In single-cell expression analysis, it is clustered with cytotrophoblasts, indicating a role in mitochondrial function.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [TBP](https://www.ncbi.nlm.nih.gov/gene/6908) with 7 experiments, [GATAD2B](https://www.ncbi.nlm.nih.gov/gene/57459) with 6 experiments, and [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025), and [RBFOX2](https://www.ncbi.nlm.nih.gov/gene/23543), each with 5 experiments.



The input data indicates associations with various disease conditions, including neurodegenerative diseases such as Alzheimer's disease and tauopathy. Additionally, it encompasses cognitive disorders, dementia, and diseases of mental health, all of which are subcategories of central nervous system diseases and broader nervous system diseases.



The gene is expressed in various tissues and cell types, including nerve and pituitary from the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, it is expressed in Thyroid, CD56+ NK cells, CD4+ T cells, Lymphoma (Burkitt's, Daudi), Spinal cord, Bronchial Epithelial Cells, and Olfactory Bulb according to the GTEx Tissue Gene Expression Profiles.


The input data highlights several pathways involved in visual processes and cellular regulation, including the phototransduction cascade, its inactivation, recovery, and regulation, as well as visual phototransduction and diseases associated with visual transduction. Additionally, the data covers pathways related to apoptosis, such as programmed cell death, apoptotic cleavage of cellular proteins, and the apoptotic execution phase. Other notable pathways include disease and signal transduction.


The mouse ortholog gene 14272, which corresponds to the human ortholog gene 2339, is associated with several phenotypic outcomes. These include embryonic lethality prior to organogenesis, embryonic lethality prior to the tooth bud stage, and preweaning lethality with complete penetrance. Notably, the preweaning lethality phenotype is observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.754 (14.07th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -20.47 (10.8th percentile). The median structural flexibility is 1.005 (65.9th percentile), with a secondary structure composition of 31.4% helix (44.79th percentile), 33.51% sheet (48.59th percentile), and 27.7% turn (44.66th percentile). The instability index is 51.03 (60.53rd percentile), and the isoelectric point is 4.96 (7.03rd percentile). The protein is 379 amino acids long (42.48th percentile) with a molecular weight of 44408.08 Da (45.03rd percentile). For detailed sequence analysis methods, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |