---
layout: post
title: DMD
date: 2025-01-17 16:55 CST
description: DMD description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1756) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1756  | DMD | ENSG00000198947 | Xp21.2-p21.1 |



The gene is involved in the [regulation of heart rate](https://amigo.geneontology.org/amigo/term/GO:0002027) and enables [dystroglycan binding](https://amigo.geneontology.org/amigo/term/GO:0002162) and [actin binding](https://amigo.geneontology.org/amigo/term/GO:0003779). It is a [structural constituent of the cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0005200) and enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829) and [cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0005856), and is involved in [cytoskeleton organization](https://amigo.geneontology.org/amigo/term/GO:0007010). It plays a role in [muscle organ development](https://amigo.geneontology.org/amigo/term/GO:0007517) and [skeletal muscle tissue development](https://amigo.geneontology.org/amigo/term/GO:0007519), and is involved in [protein localization](https://amigo.geneontology.org/amigo/term/GO:0008104). The gene enables [zinc ion binding](https://amigo.geneontology.org/amigo/term/GO:0008270) and is a [structural constituent of muscle](https://amigo.geneontology.org/amigo/term/GO:0008307). It is located on the [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986) and is involved in the [regulation of release of sequestered calcium ion into cytosol by sarcoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0010880), [regulation of cardiac muscle contraction by regulation of the release of sequestered calcium ion](https://amigo.geneontology.org/amigo/term/GO:0010881), and [regulation of skeletal muscle contraction by regulation of release of sequestered calcium ion](https://amigo.geneontology.org/amigo/term/GO:0014809). The gene is part of the [dystrophin-associated glycoprotein complex](https://amigo.geneontology.org/amigo/term/GO:0016010) and [syntrophin complex](https://amigo.geneontology.org/amigo/term/GO:0016013), and colocalizes with the [lateral plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0016328). It enables [myosin binding](https://amigo.geneontology.org/amigo/term/GO:0017022) and [vinculin binding](https://amigo.geneontology.org/amigo/term/GO:0017166), and is located in the [Z disc](https://amigo.geneontology.org/amigo/term/GO:0030018), [cell-substrate junction](https://amigo.geneontology.org/amigo/term/GO:0030055), [filopodium](https://amigo.geneontology.org/amigo/term/GO:0030175), and [filopodium membrane](https://amigo.geneontology.org/amigo/term/GO:0031527). The gene is part of a [protein-containing complex](https://amigo.geneontology.org/amigo/term/GO:0032991) and is involved in the [negative regulation of peptidyl-serine phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0033137) and [maintenance of the blood-brain barrier](https://amigo.geneontology.org/amigo/term/GO:0035633). It is active in and located in the [sarcolemma](https://amigo.geneontology.org/amigo/term/GO:0042383), and is located in the [costamere](https://amigo.geneontology.org/amigo/term/GO:0043034), [neuron projection terminus](https://amigo.geneontology.org/amigo/term/GO:0044306), [membrane raft](https://amigo.geneontology.org/amigo/term/GO:0045121), [synapse](https://amigo.geneontology.org/amigo/term/GO:0045202), and [postsynaptic membrane](https://amigo.geneontology.org/amigo/term/GO:0045211). The gene is involved in [muscle cell cellular homeostasis](https://amigo.geneontology.org/amigo/term/GO:0046716), [neuron development](https://amigo.geneontology.org/amigo/term/GO:0048666), and enables [nitric-oxide synthase binding](https://amigo.geneontology.org/amigo/term/GO:0050998). It is involved in [muscle cell development](https://amigo.geneontology.org/amigo/term/GO:0055001), [cardiac muscle contraction](https://amigo.geneontology.org/amigo/term/GO:0060048), [regulation of ryanodine-sensitive calcium-release channel activity](https://amigo.geneontology.org/amigo/term/GO:0060314), and [protein-containing complex assembly](https://amigo.geneontology.org/amigo/term/GO:0065003). The gene is involved in the [cardiac muscle cell action potential](https://amigo.geneontology.org/amigo/term/GO:0086001), [regulation of muscle system process](https://amigo.geneontology.org/amigo/term/GO:0090257), [synaptic signaling](https://amigo.geneontology.org/amigo/term/GO:0099536), [regulation of voltage-gated calcium channel activity](https://amigo.geneontology.org/amigo/term/GO:1901385), [negative regulation of peptidyl-cysteine S-nitrosylation](https://amigo.geneontology.org/amigo/term/GO:1902083), and [positive regulation of sodium ion transmembrane transporter activity](https://amigo.geneontology.org/amigo/term/GO:2000651).


The gene length is 14,451 base pairs (34.19% of all genes), the mature length is 840 base pairs (15.71% of all genes), and the primary transcript length is 13,734 base pairs (41.78% of all genes).


The gene DMD (NCBI ID: 1756) has been mentioned in [1,636 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22DMD%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1986, and the middle 50% of publications range from 1994 to 2014.


The top five publications mentioning the DMD gene, ranked by their scientific influence, include "[The complete sequence of dystrophin predicts a rod-shaped cytoskeletal protein.](https://pubmed.ncbi.nlm.nih.gov/3282674)" (1988) (relative citation ratio: 34.43), "[Membrane organization of the dystrophin-glycoprotein complex.](https://pubmed.ncbi.nlm.nih.gov/1913804)" (1991) (relative citation ratio: 26.51), "[A role for the dystrophin-glycoprotein complex as a transmembrane linker between laminin and actin.](https://pubmed.ncbi.nlm.nih.gov/8349731)" (1993) (relative citation ratio: 25.94), "[Postnatal genome editing partially restores dystrophin expression in a mouse model of muscular dystrophy.](https://pubmed.ncbi.nlm.nih.gov/26721683)" (2016) (relative citation ratio: 23.13), and "[An explanation for the phenotypic differences between patients bearing partial deletions of the DMD locus.](https://pubmed.ncbi.nlm.nih.gov/3384440)" (1988) (relative citation ratio: 21.99). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[DMD](https://www.proteinatlas.org/ENSG00000198947-DMD) is an actin-binding gene involved in various diseases, including cancer-related genes, cardiomyopathy, and disease variants. It is also a target for FDA-approved drugs. The gene's expression is detected in many tissues, with notable expression in neurons and basophils. In the brain, DMD is expressed in neurons with mixed functions, while in cell lines, it is associated with neuronal visual perception. Single-cell expression data also indicate its role in neuronal signaling. Evidence supporting these functions is available at the protein level.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), [AR](https://www.ncbi.nlm.nih.gov/gene/367), [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046), [PGR](https://www.ncbi.nlm.nih.gov/gene/5241), and [COMMD3-BMI1](https://www.ncbi.nlm.nih.gov/gene/100532731), each shown to be regulating in 3 experiments.



The GWAS data indicates associations with a wide range of disease conditions, including various types of cancer such as breast cancer, thoracic cancer, lung cancer, and organ system cancer. Infectious diseases, particularly those caused by viral agents like human immunodeficiency virus, are also implicated. Metabolic disorders, including diabetes mellitus and type 2 diabetes mellitus, are noted, along with cardiovascular conditions such as coronary artery disease and vascular disease. Neurological and mental health disorders are prominent, encompassing neurodegenerative diseases like Parkinson's disease, mood disorders such as bipolar disorder and major depressive disorder, and substance-related disorders including alcohol dependence. Additionally, the data highlights immune system diseases, hepatobiliary diseases, and various developmental disorders of mental health, including attention deficit hyperactivity disorder.


The genome-wide association studies have identified several traits and diseases associated with specific genetic variations. Notably, the trait [Half-life of apixaban](https://pubmed.ncbi.nlm.nih.gov/36867504) was mapped to the closest MeSH term [Dabigatran](https://meshb.nlm.nih.gov/record/ui?ui=D000069604) using the method [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0). Additionally, both [Influenza A (H1N1) infection](https://pubmed.ncbi.nlm.nih.gov/26379185) and [Mild influenza (H1N1) infection](https://pubmed.ncbi.nlm.nih.gov/26379185) were mapped to the MeSH term [Influenza, Human](https://meshb.nlm.nih.gov/record/ui?ui=D007251) using the same embedding match method.


The gene is expressed in various tissues, including muscle, nerve, and blood vessel, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the salivary gland, colon, superior cervical ganglion, trigeminal ganglion, dorsal root ganglion, olfactory bulb, and uterus.


The proteins are involved in several key biological processes, including muscle contraction, striated muscle contraction, extracellular matrix organization, and non-integrin membrane-ECM interactions.


The mouse ortholog gene 13405 and its human ortholog gene 1756 are associated with several phenotypic abnormalities, including abnormal bone structure, decreased bone mineral content, and decreased total body fat amount. Additionally, these genes are linked to increased levels of circulating alanine transaminase, aspartate transaminase, and total protein, as well as increased lean body mass, red blood cell distribution width, and thrombocytosis. These phenotypes are observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.695 (17.97th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -81.24 (0.8th percentile), and the median structural flexibility is 1.008 (75.69th percentile). The protein has a high affinity for helix structures (38.78%, 90.93rd percentile) and sheet structures (33.60%, 49.14th percentile), with a lower affinity for turn structures (22.90%, 9.53rd percentile). The instability index is 56.31 (75.48th percentile), and the isoelectric point is 5.64 (21.44th percentile). The protein is relatively long with 3685 amino acids (99.51st percentile) and has a molecular weight of 426744.53 Da (99.54th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |