---
layout: post
title: MR1
date: 2025-01-17 16:55 CST
description: MR1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3140) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3140  | MR1 | ENSG00000153029 | 1q25.3 |



The gene is located in the [Golgi membrane](https://amigo.geneontology.org/amigo/term/GO:0000139), [endoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0005783), [endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0005789), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [early endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0031901), and [late endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0031902). It is active in the [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615) and the [external side of plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009897). The gene is involved in various processes including [antigen processing and presentation of peptide antigen via MHC class I](https://amigo.geneontology.org/amigo/term/GO:0002474), [positive regulation of T cell mediated cytotoxicity directed against tumor cell target](https://amigo.geneontology.org/amigo/term/GO:0002854), [immune response](https://amigo.geneontology.org/amigo/term/GO:0006955), [antigen processing and presentation of exogenous antigen](https://amigo.geneontology.org/amigo/term/GO:0019884), [T cell differentiation in thymus](https://amigo.geneontology.org/amigo/term/GO:0033077), [innate immune response](https://amigo.geneontology.org/amigo/term/GO:0045087), [defense response to Gram-negative bacterium](https://amigo.geneontology.org/amigo/term/GO:0050829), and [defense response to Gram-positive bacterium](https://amigo.geneontology.org/amigo/term/GO:0050830). Additionally, it enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [beta-2-microglobulin binding](https://amigo.geneontology.org/amigo/term/GO:0030881), [MHC class I receptor activity](https://amigo.geneontology.org/amigo/term/GO:0032393), and [T cell receptor binding](https://amigo.geneontology.org/amigo/term/GO:0042608). The gene is also part of the [MHC class I protein complex](https://amigo.geneontology.org/amigo/term/GO:0042612).


The gene length is 15,666 base pairs (36.05% of all genes), the mature length is 4,531 base pairs, and the primary transcript length is 15,666 base pairs.


The gene MR1 (NCBI ID: 3140) has been mentioned in [96 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22MR1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning MR1 was in 1993, and the middle 50% of publications occurred between 2013 and 2020.


The top five publications mentioning MR1, ranked by their scientific influence, include "[Selection of evolutionarily conserved mucosal-associated invariant T cells by MR1.](https://pubmed.ncbi.nlm.nih.gov/12634786)" (2003) (relative citation ratio: 13.01), "[Antigen-loaded MR1 tetramers define T cell receptor heterogeneity in mucosal-associated invariant T cells.](https://pubmed.ncbi.nlm.nih.gov/24101382)" (2013) (relative citation ratio: 11.06), "[Activation, exhaustion, and persistent decline of the antimicrobial MR1-restricted MAIT-cell population in chronic HIV-1 infection.](https://pubmed.ncbi.nlm.nih.gov/23243281)" (2013) (relative citation ratio: 7.53), "[Human blood MAIT cell subsets defined using MR1 tetramers.](https://pubmed.ncbi.nlm.nih.gov/29437263)" (2018) (relative citation ratio: 6.31), and "[Biology of CD1- and MR1-restricted T cells.](https://pubmed.ncbi.nlm.nih.gov/24499274)" (2014) (relative citation ratio: 5.67). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[MR1](https://www.proteinatlas.org/ENSG00000153029-MR1) is an FDA-approved drug target with evidence at the protein level. It is detected in all tissues and is primarily located in the plasma membrane. MR1 is associated with vesicular transport in blood, transcription in tissues, immune response in the brain, extracellular matrix organization in cell lines, and innate immune response in single cells.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) and [AR](https://www.ncbi.nlm.nih.gov/gene/367), each with 6 experiments. Additionally, [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861), [PGR](https://www.ncbi.nlm.nih.gov/gene/5241), and [MYC](https://www.ncbi.nlm.nih.gov/gene/4609) were each found to be regulating in 4 experiments.



The GWAS data indicates associations with a range of conditions, including lower respiratory tract disease, lung disease, respiratory system disease, disease of metabolism, acquired metabolic disease, nutrition disease, overnutrition, obesity, cognitive disorder, disease of mental health, female reproductive system disease, reproductive system disease, ovarian disease, and premature ovarian failure.



The gene is expressed in various tissues, including bladder and blood, according to the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles indicate that the gene is ubiquitously expressed in multiple tissue samples, such as cardiac myocytes, heart, bone marrow, appendix, superior cervical ganglion, skeletal muscle, and skin.



The mouse ortholog gene 15064 and its human ortholog gene 3140 are associated with the phenotype of decreased circulating creatinine level, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.303 (57.66th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -8.27 (25.8th percentile), and the median structural flexibility is 0.994 (18.66th percentile). The protein has a propensity for helix, sheet, and turn structures of 29.33% (26th percentile), 39.30% (84.04th percentile), and 25.22% (23.12th percentile), respectively. The instability index is 46.97 (46.52nd percentile), with an isoelectric point of 5.87 (27.11th percentile). The sequence length is 341 amino acids (36.53rd percentile), and the molecular weight is 39365.53 Da (38.59th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |