---
layout: post
title: SELENOP
date: 2025-01-17 16:55 CST
description: SELENOP description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6414) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6414  | SELENOP | ENSG00000250722 | 5p12 |



The gene is involved in several biological processes, including [selenium compound metabolic process](https://amigo.geneontology.org/amigo/term/GO:0001887), [response to oxidative stress](https://amigo.geneontology.org/amigo/term/GO:0006979), [brain development](https://amigo.geneontology.org/amigo/term/GO:0007420), [locomotory behavior](https://amigo.geneontology.org/amigo/term/GO:0007626), [post-embryonic development](https://amigo.geneontology.org/amigo/term/GO:0009791), [sexual reproduction](https://amigo.geneontology.org/amigo/term/GO:0019953), and [regulation of growth](https://amigo.geneontology.org/amigo/term/GO:0040008). It is active in and located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), and is also found in the [platelet dense granule lumen](https://amigo.geneontology.org/amigo/term/GO:0031089) and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). Additionally, the gene enables [selenium binding](https://amigo.geneontology.org/amigo/term/GO:0008430).


The gene length is 32,313 base pairs (54.68% of all genes), the mature length is 3,738 base pairs (81.26% of all genes), and the primary transcript length is 27,601 base pairs (60.41% of all genes).


The gene SELENOP, with NCBI ID 6414, has been mentioned in [195 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SELENOP%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1958, and the middle 50% of publications occurred between 2007 and 2018.


The top 5 publications mentioning SELENOP, ranked by their scientific influence, include "[Prediction of survival odds in COVID-19 by zinc, age and selenoprotein P as composite biomarker.](https://pubmed.ncbi.nlm.nih.gov/33126054)" (2021) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 17.41, "[Selenoprotein P-expression, functions, and roles in mammals.](https://pubmed.ncbi.nlm.nih.gov/19345254)" (2009) with an RCR of 10.74, "[A liver-derived secretory protein, selenoprotein P, causes insulin resistance.](https://pubmed.ncbi.nlm.nih.gov/21035759)" (2010) with an RCR of 10.62, "[Selenium Transport Mechanism via Selenoprotein P-Its Physiological Role and Related Diseases.](https://pubmed.ncbi.nlm.nih.gov/34124127)" (2021) with an RCR of 6.66, and "[Selenoprotein P as an <i>in vivo</i> redox regulator: disorders related to its deficiency and excess.](https://pubmed.ncbi.nlm.nih.gov/32001950)" (2020) with an RCR of 5.92. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[SELENOP](https://www.proteinatlas.org/ENSG00000250722-SELENOP) is a gene with evidence at the protein level. Its RNA is detected in all tissues, with notable expression in the nucleoplasm, Golgi apparatus, and connecting piece. The gene is part of Cluster 12 for blood expression, which is non-specific and related to mitochondrial translation. In tissue expression, it belongs to Cluster 83, associated with liver metabolism. Additionally, SELENOP is part of Cluster 30 for brain expression, linked to white matter myelination, and Cluster 18 for cell line expression, related to liver cancer metabolism. In single-cell expression, it is part of Cluster 22, associated with proximal enterocytes and digestion.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [PGR](https://www.ncbi.nlm.nih.gov/gene/5241) with 5 experiments, [CEBPB](https://www.ncbi.nlm.nih.gov/gene/1051) with 4 experiments, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 4 experiments, [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930) with 4 experiments, and [MED12](https://www.ncbi.nlm.nih.gov/gene/9968) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [ZNF740](https://www.ncbi.nlm.nih.gov/gene/8243) and [ZNF783](https://www.ncbi.nlm.nih.gov/gene/9968). ZNF740 was shown to be regulating genes in 1 experiment, while ZNF783 was found to be regulating genes in 4 experiments.




The gene is expressed in the liver and small intestine according to BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles indicate expression in the spinal cord, small intestine, and fetal liver.




The analyzed protein sequence has a GRAVY value of -0.643 (21.74th percentile), indicating a hydrophilic nature. It exhibits a charge of 4.39 at pH 7.0 (65.94th percentile) and a median structural flexibility of 1.005 (64.32nd percentile). The protein's secondary structure is predicted to be 31.54% helix (46.08th percentile), 29.11% sheet (23.76th percentile), and 29.11% turn (56.84th percentile). The instability index is 57.82 (78.79th percentile), with an isoelectric point of 8.08 (59.66th percentile). The protein has a length of 381 amino acids (42.77th percentile) and a molecular weight of 41672.81 Da (41.63rd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |