---
layout: post
title: HLA-DMA
date: 2024-12-08 23:57 CST
description: HLA-DMA description
tags: [cooccuring-genes]
categories: longest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3108) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3108  | HLA-DMA | ENSG00000204257 | 6p21.32 |



The gene length is 906,136.0 base pairs (99.52% of all genes), the mature length is 3,164.5 base pairs, and the primary transcript length is 906,106.0 base pairs.


The gene HLA-DMA, with NCBI ID 3108, has been featured in [56 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22HLA-DMA%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1977, and the middle 50% of publications occurred between 1997 and 2012.


The top five publications mentioning HLA-DMA, ranked by their scientific influence, include "[An essential role for HLA-DM in antigen presentation by class II major histocompatibility molecules.](https://pubmed.ncbi.nlm.nih.gov/8139689)" (1994) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 6.84, "[Editing of the HLA-DR-peptide repertoire by HLA-DM.](https://pubmed.ncbi.nlm.nih.gov/8947036)" (1996) with an RCR of 4.1, "[A lysosomal targeting signal in the cytoplasmic tail of the beta chain directs HLA-DM to MHC class II compartments.](https://pubmed.ncbi.nlm.nih.gov/7593164)" (1995) with an RCR of 3.05, "[Crystal structure of the HLA-DM-HLA-DR1 complex defines mechanisms for rapid peptide selection.](https://pubmed.ncbi.nlm.nih.gov/23260142)" (2012) with an RCR of 3.0, and "[HLA-DM acts as a molecular chaperone and rescues empty HLA-DR molecules at lysosomal pH.](https://pubmed.ncbi.nlm.nih.gov/9075930)" (1997) with an RCR of 2.59. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861) with 6 experiments, [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170) with 5 experiments, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 5 experiments, [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227) with 5 experiments, and [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.





The gene HLA-DMA has been associated with [Educational attainment](https://pubmed.ncbi.nlm.nih.gov/35361970), which maps closely to the MeSH term [Educational Status](https://meshb.nlm.nih.gov/record/ui?ui=D004522) using the method [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0).









| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |