---
layout: post
title: PLXNC1
date: 2025-01-17 16:55 CST
description: PLXNC1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10154) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10154  | PLXNC1 | ENSG00000136040 | 12q22 |



The gene is part of the [semaphorin receptor complex](https://amigo.geneontology.org/amigo/term/GO:0002116) and enables [signaling receptor binding](https://amigo.geneontology.org/amigo/term/GO:0005102) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886) and the [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), as well as the [cerebellar climbing fiber to Purkinje cell synapse](https://amigo.geneontology.org/amigo/term/GO:0150053). The gene is involved in various processes, including [cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0007155), [negative regulation of cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0007162), [regulation of cell shape](https://amigo.geneontology.org/amigo/term/GO:0008360), [regulation of cell migration](https://amigo.geneontology.org/amigo/term/GO:0030334), [regulation of GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0043087), [positive regulation of axonogenesis](https://amigo.geneontology.org/amigo/term/GO:0050772), and [regulation of synapse pruning](https://amigo.geneontology.org/amigo/term/GO:1905806). Additionally, it enables [semaphorin receptor activity](https://amigo.geneontology.org/amigo/term/GO:0017154).


The gene length is 117,999 base pairs (84.95th percentile of all genes), the mature length is 853 base pairs (16.24th percentile), and the primary transcript length is 24,498.5 base pairs (57.34th percentile).


The gene PLXNC1 (NCBI ID: 10154) has been mentioned in [17 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PLXNC1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1998 and the middle 50% of publications occurring between 2009 and 2020. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning PLXNC1 include "[A poxvirus-encoded semaphorin induces cytokine production from monocytes and binds to a novel cellular semaphorin receptor, VESPR.](https://pubmed.ncbi.nlm.nih.gov/9586637)" (1998) (relative citation ratio: 3.49), "[Plexin C1, a receptor for semaphorin 7a, inactivates cofilin and is a potential tumor suppressor for melanoma progression.](https://pubmed.ncbi.nlm.nih.gov/18987670)" (2009) (relative citation ratio: 1.55), "[Soluble CD100 functions on human monocytes and immature dendritic cells require plexin C1 and plexin B1, respectively.](https://pubmed.ncbi.nlm.nih.gov/15746246)" (2005) (relative citation ratio: 1.39), "[MiR-4500 Regulates PLXNC1 and Inhibits Papillary Thyroid Cancer Progression.](https://pubmed.ncbi.nlm.nih.gov/31317324)" (2019) (relative citation ratio: 1.24), and "[Expression patterns of semaphorin7A and plexinC1 during rat neural development suggest roles in axon guidance and neuronal migration.](https://pubmed.ncbi.nlm.nih.gov/17727705)" (2007) (relative citation ratio: 1.21). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PLXNC1](https://www.proteinatlas.org/ENSG00000136040-PLXNC1) is a gene with evidence at the protein level, detected in many tissues. It is primarily localized in the mitochondria and is associated with various expression clusters, including Cluster 7 in blood, which is involved in the innate immune response, and Cluster 70 in tissues, which is related to the immune response in lymphoid tissue. Additionally, it is part of Cluster 56 in the brain, linked to neuropeptide signaling in the hypothalamus, and Cluster 9 in cell lines, associated with melanin biosynthesis in skin cancer. In single-cell expression, it is part of Cluster 74, involved in neutrophil degranulation.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 5 experiments, [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) with 4 experiments, [SP1](https://www.ncbi.nlm.nih.gov/gene/6667) with 4 experiments, [PPARG](https://www.ncbi.nlm.nih.gov/gene/5468) with 4 experiments, and [EP300](https://www.ncbi.nlm.nih.gov/gene/2033) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with a wide range of disease conditions, including various types of cancer such as breast cancer, thoracic cancer, and organ system cancer. Infectious diseases, particularly those caused by viral agents like human immunodeficiency virus, are also implicated. Metabolic disorders, including diabetes mellitus, specifically type 2 diabetes mellitus, and carbohydrate metabolism diseases, are notable. Cardiovascular conditions such as coronary artery disease, artery disease, and vascular disease are highlighted. Additionally, the data points to nervous system diseases, including neurodegenerative diseases like Parkinson's disease, and mental health disorders such as substance-related disorders and alcohol dependence. Kidney and urinary system diseases, as well as skin diseases, are also associated. Furthermore, the data includes acquired immunodeficiency syndrome and angioedema.


The PLXNC1 gene has been associated with various traits and diseases through genome-wide association studies. Notably, [Plexin-C1 levels](https://pubmed.ncbi.nlm.nih.gov/34648354) have been linked to [Ephrin-A5](https://meshb.nlm.nih.gov/record/ui?ui=D036386) using the BioWordVec method. Additionally, [Plexin-C1 levels (PLXNC1.4564.2.2)](https://pubmed.ncbi.nlm.nih.gov/29875488) have also been mapped to Ephrin-A5. Furthermore, [Serum levels of protein PLXNC1](https://pubmed.ncbi.nlm.nih.gov/35078996) have been associated with [Proteins](https://meshb.nlm.nih.gov/record/ui?ui=D011506). These findings highlight the potential role of PLXNC1 in various biological processes and diseases.


The gene is expressed in various tissues, including the ovary and blood, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in CD34+ cells, CD33+ myeloid cells, BDCA4+ dendritic cells, CD14+ monocytes, whole blood, fetal brain, and testis germ cells.


The proteins are involved in several pathways related to axon guidance and developmental biology, with a notable focus on semaphorin interactions, including other semaphorin interactions.


The mouse ortholog gene 54712 and its human ortholog gene 10154 are associated with several phenotypic abnormalities in both sexes, including abnormal kidney and spleen morphology, enlarged and small kidney, small spleen, and increased circulating HDL cholesterol levels.


The analyzed protein sequence has a GRAVY value of -0.266 (61.91 percentile), indicating a hydrophilic nature. It exhibits a charge of 6.21 at pH 7.0 (71.92 percentile) and a median structural flexibility of 1.001 (46.5 percentile). The protein's secondary structure is predicted to be 31.2% helix (42.91 percentile), 37.1% sheet (73.4 percentile), and 27.9% turn (46.05 percentile). The instability index is 43.91 (35.36 percentile), with an isoelectric point of 7.78 (57.0 percentile). The protein has a length of 1568 amino acids (95.49 percentile) and a molecular weight of 175740.03 Da (95.64 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |