---
layout: post
title: RPL22
date: 2025-01-17 16:55 CST
description: RPL22 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6146) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6146  | RPL22 | ENSG00000116251 | 1p36.31 |



The gene is involved in [cytoplasmic translation](https://amigo.geneontology.org/amigo/term/GO:0002181) and [translation](https://amigo.geneontology.org/amigo/term/GO:0006412), and it enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [structural constituent of ribosome](https://amigo.geneontology.org/amigo/term/GO:0003735), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [heparin binding](https://amigo.geneontology.org/amigo/term/GO:0008201). It is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925), [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062), [presynapse](https://amigo.geneontology.org/amigo/term/GO:0098793), and [glutamatergic synapse](https://amigo.geneontology.org/amigo/term/GO:0098978). Additionally, it is part of the [cytosolic large ribosomal subunit](https://amigo.geneontology.org/amigo/term/GO:0022625) and [ribonucleoprotein complex](https://amigo.geneontology.org/amigo/term/GO:1990904), and it is involved in [alpha-beta T cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0046632) and [translation at presynapse](https://amigo.geneontology.org/amigo/term/GO:0140236).


The gene length is 48,583 base pairs (65.5% of all genes), the mature length is 1,843.5 base pairs, and the primary transcript length is 8,126.5 base pairs.


The gene RPL22 (NCBI ID: 6146) has been mentioned in [37 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RPL22%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1988 and the middle 50% of publications occurring between 1999 and 2017. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning RPL22, ranked by their scientific influence, include "[The 3;21 translocation in myelodysplasia results in a fusion transcript between the AML1 gene and the gene for EAP, a highly conserved protein associated with the Epstein-Barr virus small RNA EBER 1.](https://pubmed.ncbi.nlm.nih.gov/8395054)" (1993) (relative citation ratio: 3.44), "[Inactivation of ribosomal protein L22 promotes transformation by induction of the stemness factor, Lin28B.](https://pubmed.ncbi.nlm.nih.gov/22976955)" (2012) (relative citation ratio: 2.59), "[The Epstein-Barr virus (EBV) small RNA EBER1 binds and relocalizes ribosomal protein L22 in EBV-infected human B lymphocytes.](https://pubmed.ncbi.nlm.nih.gov/8159770)" (1994) (relative citation ratio: 2.51), "[The <i>Staphylococcus aureus</i> Extracellular Adherence Protein Eap Is a DNA Binding Protein Capable of Blocking Neutrophil Extracellular Trap Formation.](https://pubmed.ncbi.nlm.nih.gov/30038902)" (2018) (relative citation ratio: 1.89), and "[Ribosomal Proteins Rpl22 and Rpl22l1 Control Morphogenesis by Regulating Pre-mRNA Splicing.](https://pubmed.ncbi.nlm.nih.gov/28076796)" (2017) (relative citation ratio: 1.89). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RPL22](https://www.proteinatlas.org/ENSG00000116251-RPL22) is a ribosomal protein with molecular functions including heparin-binding, ribonucleoprotein, and RNA-binding. It is involved in cancer-related genes and has evidence at the protein level. The gene is detected in all RNA tissue distributions and is primarily located in the nucleoplasm and nucleoli. RPL22 is part of various non-specific expression clusters related to transcription, translation, and ribosomal functions across different tissues, brain regions, cell lines, and single cells.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [TAL1](https://www.ncbi.nlm.nih.gov/gene/6886) and [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), each with 6 experiments. Additionally, [RARA](https://www.ncbi.nlm.nih.gov/gene/5914), [MAX](https://www.ncbi.nlm.nih.gov/gene/4149), and [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046) were each found to be regulating in 5 experiments.


The transcription factor [GATA1](https://www.ncbi.nlm.nih.gov/gene/11091) was found to be regulating genes in 2 Perturb-seq experiments, according to data from GTRD's database of transcription factor binding sites and the study published in [PubMed](https://pubmed.ncbi.nlm.nih.gov/35688146/).




The gene is expressed in various tissues and cell types, including the ovary, cervix uteri, Lymphoma Burkitt's (Raji), CD34+ cells, CD4+ T cells, CD8+ T cells, CD19+ B cells (neg. sel.), and Leukemia promyelocytic-HL-60 cells. This indicates a ubiquitous expression pattern across multiple tissue samples.


The input data highlights various pathways involved in protein metabolism and gene expression, including 3'-UTR-mediated translational regulation, L13a-mediated translational silencing of ceruloplasmin expression, and GTP hydrolysis and joining of the 60S ribosomal subunit. Key processes such as cap-dependent translation initiation, eukaryotic translation initiation, formation of a pool of free 40S subunits, eukaryotic translation elongation, and eukaryotic translation termination are also noted. Additionally, the data covers peptide chain elongation, viral mRNA translation, and nonsense-mediated decay (NMD) both independent of and enhanced by the exon junction complex (EJC). Pathways related to SRP-dependent cotranslational protein targeting to the membrane and translation are included. Furthermore, the data encompasses influenza infection, the influenza life cycle, influenza viral RNA transcription and replication, and disease.



The analyzed protein sequence has a GRAVY value of -0.755 (14.02% percentile), indicating a hydrophilic nature. It exhibits a charge of 4.60 at pH 7.0 (66.67% percentile) and a median structural flexibility of 1.011 (82.62% percentile). The protein's secondary structure is predicted to be 39.06% helix (91.63% percentile), 35.16% sheet (60.36% percentile), and 24.22% turn (16.29% percentile). The instability index is 49.51 (55.63% percentile), with an isoelectric point of 9.21 (84.88% percentile). The protein has a length of 128 amino acids (5.57% percentile) and a molecular weight of 14786.81 Da (6.14% percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |