---
layout: post
title: TSPAN5
date: 2025-01-17 16:55 CST
description: TSPAN5 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10098) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10098  | TSPAN5 | ENSG00000168785 | 4q23 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [enzyme binding](https://amigo.geneontology.org/amigo/term/GO:0019899). It is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [endoplasmic reticulum lumen](https://amigo.geneontology.org/amigo/term/GO:0005788), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), and [actin cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0015629). The gene is involved in the [positive regulation of Notch signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0045747), [regulation of membrane protein ectodomain proteolysis](https://amigo.geneontology.org/amigo/term/GO:0051043), [protein maturation](https://amigo.geneontology.org/amigo/term/GO:0051604), and [protein localization to plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0072659).


The gene length is 14,152 base pairs (33.64% of all genes), the mature length is 739 base pairs (10.97% of all genes), and the primary transcript length is 7,150 base pairs (25.41% of all genes).


TSPAN5 has been mentioned in [21 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22TSPAN5%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1988, and the middle 50% of publications range from 2006 to 2018.


The top 5 publications mentioning TSPAN5, ranked by their scientific influence as measured by the [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541), include: ["Single-chain magnet (NEt4)[Mn2(5-MeOsalen)2Fe(CN)6] Made of Mn(III)-Fe(III)-Mn(III) trinuclear single-molecule magnet with an S(T) = 9/2 spin ground state."](https://pubmed.ncbi.nlm.nih.gov/15740148) (2005) (relative citation ratio: 6.34), ["TSPAN5, ERICH3 and selective serotonin reuptake inhibitors in major depressive disorder: pharmacometabolomics-informed pharmacogenomics."](https://pubmed.ncbi.nlm.nih.gov/26903268) (2016) (relative citation ratio: 3.82), ["miR-155 increases stemness and decitabine resistance in triple-negative breast cancer cells by inhibiting TSPAN5."](https://pubmed.ncbi.nlm.nih.gov/32096299) (2020) (relative citation ratio: 1.96), ["New Luminescent Lanthanide Tetrakis-Complexes NEt<sub>4</sub> [LnL<sub>4</sub> ] Based on Dimethyl-N-Benzoylamidophosphate."](https://pubmed.ncbi.nlm.nih.gov/35536851) (2022) (relative citation ratio: 1.89), and ["Tspan5 promotes epithelial-mesenchymal transition and tumour metastasis of hepatocellular carcinoma by activating Notch signalling."](https://pubmed.ncbi.nlm.nih.gov/33955149) (2021) (relative citation ratio: 1.72). The citation count is sourced from [iCite](https://icite.od.nih.gov).


[TSPAN5](https://www.proteinatlas.org/ENSG00000168785-TSPAN5) is a gene with evidence at the protein level, detected in many RNA tissue distributions. It is primarily localized in the nucleoplasm and plasma membrane. The gene is expressed in T-cells with unknown function (Cluster 1) and is associated with neuronal transcription (Cluster 64). In the brain, it shows non-specific mixed function expression (Cluster 34), while in cell lines, it is involved in neuronal signal transduction (Cluster 72). Additionally, TSPAN5 is expressed in oligodendrocytes with unknown function (Cluster 51).


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [EZH2](https://www.ncbi.nlm.nih.gov/gene/2146), [MYC](https://www.ncbi.nlm.nih.gov/gene/4609), [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), [E2F1](https://www.ncbi.nlm.nih.gov/gene/1869), and [AR](https://www.ncbi.nlm.nih.gov/gene/367), each shown to be regulating in 5 experiments.



The GWAS data indicates associations with a wide range of disease conditions, including infectious diseases such as viral infectious diseases and human immunodeficiency virus infectious disease. Additionally, the data highlights hematopoietic and immune system diseases, including leukocyte disease, leukopenia, and agranulocytosis. Metabolic disorders are also prominent, with links to diabetes mellitus, particularly type 2 diabetes mellitus, and obesity. Cardiovascular conditions such as coronary artery disease, hypertension, and vascular disease are noted, along with kidney and urinary system diseases. Furthermore, the data includes nervous system diseases, inherited metabolic disorders, lipid metabolism disorders, and musculoskeletal system diseases like rheumatoid arthritis. Eye diseases, specifically myopia, are also associated with the GWAS findings.



The gene is expressed in various tissues, including the ovary and brain, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the pineal gland (both day and night), CD71+ early erythroid cells, whole brain, parietal lobe, cerebellum peduncles, bronchial epithelial cells, and bone marrow.



The mouse ortholog gene 56224 and its human ortholog gene 10098 are associated with hyperactivity, a phenotype observed in both sexes.


The analyzed protein sequence has a GRAVY value of 0.411 (93.79th percentile), indicating a high level of hydrophobicity. The protein exhibits a charge of -9.50 at pH 7.0 (23.27th percentile) and an isoelectric point of 4.59 (2.13th percentile). The median structural flexibility is 0.982 (5.46th percentile), with a length of 268 amino acids (24.15th percentile) and a molecular weight of 30336.75 Da (24.77th percentile). The secondary structure affinities are 23.88% for helix (3.38th percentile), 46.27% for sheet (95.69th percentile), and 27.24% for turn (40.19th percentile). The instability index is 35.08 (10.28th percentile). These values were calculated using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |