---
layout: post
title: PODXL
date: 2024-12-08 23:57 CST
description: PODXL description
tags: [cooccuring-genes]
categories: shortest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5420) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5420  | PODXL | ENSG00000128567 | 7q32.3  |



The gene length is 462 base pairs (0.35 percentile of all genes), with a mature length of 462 base pairs and a primary transcript length of 462 base pairs.


The gene PODXL, with NCBI ID 5420, has been featured in [180 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PODXL%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ). The earliest year of publication is 1984, and the middle 50% of publications occurred between 2010 and 2018.


The top five publications mentioning PODXL, ranked by their scientific influence, include "[Selection against undifferentiated human embryonic stem cells by a cytotoxic antibody recognizing podocalyxin-like protein-1.](https://pubmed.ncbi.nlm.nih.gov/18356574)" (2008) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 4.23, "[Endothelial cell membranes contain podocalyxin--the major sialoprotein of visceral glomerular epithelial cells.](https://pubmed.ncbi.nlm.nih.gov/3511072)" (1986) with an RCR of 4.02, "[Identification of podocalyxin-like protein as a high endothelial venule ligand for L-selectin: parallels to CD34.](https://pubmed.ncbi.nlm.nih.gov/9625756)" (1998) with an RCR of 3.89, "[The role of podocalyxin in health and disease.](https://pubmed.ncbi.nlm.nih.gov/19578008)" (2009) with an RCR of 3.78, and "[Mutant p53s generate pro-invasive niches by influencing exosome podocalyxin levels.](https://pubmed.ncbi.nlm.nih.gov/30498210)" (2018) with an RCR of 3.43. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) and [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), each with 5 experiments. Additionally, [KDM2B](https://www.ncbi.nlm.nih.gov/gene/84678), [GATA2](https://www.ncbi.nlm.nih.gov/gene/2624), and [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599) were each found to be regulating in 4 experiments.











The analyzed protein sequence has a GRAVY value of -0.406 (45.49th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -16.27 (14.22nd percentile), and the median structural flexibility is 1.01 (81.48th percentile). The protein has a propensity for helix, sheet, and turn structures of 29.57% (28.16th percentile), 34.95% (58.68th percentile), and 33.51% (84.12th percentile), respectively. The instability index is 49.70 (56.23rd percentile), with an isoelectric point of 5.28 (13.17th percentile). The protein length is 558 amino acids (64.81st percentile), and the molecular weight is 58634.60 Da (61.23rd percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |