---
layout: post
title: ST6GALNAC2
date: 2025-01-17 16:55 CST
description: ST6GALNAC2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10610) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10610  | ST6GALNAC2 | ENSG00000070731 | 17q25.1 |



The gene is located in the [Golgi membrane](https://amigo.geneontology.org/amigo/term/GO:0000139) and enables [alpha-N-acetylgalactosaminide alpha-2,6-sialyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0001665). It is involved in several processes, including [protein glycosylation](https://amigo.geneontology.org/amigo/term/GO:0006486), [protein O-linked glycosylation](https://amigo.geneontology.org/amigo/term/GO:0006493), [O-glycan processing](https://amigo.geneontology.org/amigo/term/GO:0016266), [viral protein processing](https://amigo.geneontology.org/amigo/term/GO:0019082), and [protein sialylation](https://amigo.geneontology.org/amigo/term/GO:1990743). Additionally, it enables [sialyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0008373).


The gene length is 4,716 base pairs (13.21% of all genes), the mature length is 896 base pairs (18.14% of all genes), and the primary transcript length is 3,424 base pairs (12.62% of all genes).


The gene ST6GALNAC2 (NCBI ID: 10610) has been mentioned in [11 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ST6GALNAC2%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning this gene was in 2000, and the middle 50% of publications occurred between 2009 and 2015.


The top 5 publications mentioning ST6GALNAC2, ranked by their scientific influence, include "[Cytokines alter IgA1 O-glycosylation by dysregulating C1GalT1 and ST6GalNAc-II enzymes.](https://pubmed.ncbi.nlm.nih.gov/24398680)" (2014) with a relative citation ratio of 3.72, "[Role of the human ST6GalNAc-I and ST6GalNAc-II in the synthesis of the cancer-associated sialyl-Tn antigen.](https://pubmed.ncbi.nlm.nih.gov/15466199)" (2004) with a relative citation ratio of 3.48, "[Upregulation of microRNA-135b and microRNA-182 promotes chemoresistance of colorectal cancer by targeting ST6GALNAC2 via PI3K/AKT pathway.](https://pubmed.ncbi.nlm.nih.gov/28767179)" (2017) with a relative citation ratio of 3.08, "[miR-182 and miR-135b Mediate the Tumorigenesis and Invasiveness of Colorectal Cancer Cells via Targeting ST6GALNAC2 and PI3K/AKT Pathway.](https://pubmed.ncbi.nlm.nih.gov/29030743)" (2017) with a relative citation ratio of 2.31, and "[Molecular cloning and functional expression of human ST6GalNAc II. Molecular expression in various human cultured cells.](https://pubmed.ncbi.nlm.nih.gov/10742600)" (2000) with a relative citation ratio of 0.96. The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ST6GALNAC2](https://www.proteinatlas.org/ENSG00000070731-ST6GALNAC2) is a gene encoding a protein with glycosyltransferase and transferase molecular functions, supported by evidence at the protein level. Its RNA is detected in all tissues, with notable expression in neutrophils related to degranulation, ciliated cells involved in cilium organization, choroid plexus in the brain, and squamous epithelial cells associated with keratinization. The gene is also expressed in breast cancer cell lines, although its function in this context is unknown. Prognostic significance in various cancers is unprognostic, indicating no significant correlation with patient outcomes.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227), [TP53](https://www.ncbi.nlm.nih.gov/gene/7157), [KLF5](https://www.ncbi.nlm.nih.gov/gene/688), [KLF4](https://www.ncbi.nlm.nih.gov/gene/9314), and [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908), each shown to be regulating in 5 experiments.



The input data indicates associations with various metabolic disorders, including diseases of metabolism, acquired metabolic diseases, carbohydrate metabolism diseases, glucose metabolism diseases, diabetes mellitus, and specifically type 2 diabetes mellitus.



The gene is expressed in the nerve tissue according to BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles indicate expression in the retina, olfactory bulb, and various regions of the testis, including the interstitial tissue, Leydig cells, seminiferous tubules, and germ cells.


The input data highlights various diseases and processes related to glycosylation. Notably, defects in specific proteins such as ALG2, ALG3, ALG6, ALG8, ALG9, DPAGT1, MAN1B1, MGAT2, MOGS, MPDU1, ALG1, ALG12, ALG11, ALG14, and RFT1 are linked to distinct congenital disorders of glycosylation (CDGs) and other conditions like congenital myasthenic syndrome. Additionally, the data covers broader categories including diseases of glycosylation, N-glycosylation of proteins, post-translational protein modification, biosynthesis of N-glycan precursors, sialic acid metabolism, and O-linked glycosylation processes.



The analyzed protein sequence has a GRAVY value of -0.058 (80.84th percentile), indicating a hydrophilic nature. It exhibits a charge of 11.68 at pH 7.0 (84.57th percentile) and a median structural flexibility of 0.990 (11.71st percentile). The protein's secondary structure is predicted to be 29.41% helix (26.73rd percentile), 39.57% sheet (84.97th percentile), and 30.21% turn (65.34th percentile). The instability index is 31.42 (5.29th percentile), with an isoelectric point of 9.38 (88.74th percentile). The protein has a length of 374 amino acids (41.74th percentile) and a molecular weight of 41938.71 Da (41.96th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |