---
layout: post
title: RPL31
date: 2025-01-17 16:55 CST
description: RPL31 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6160) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6160  | RPL31 | ENSG00000071082 | 2q11.2  |



The gene is involved in [cytoplasmic translation](https://amigo.geneontology.org/amigo/term/GO:0002181) and [translation](https://amigo.geneontology.org/amigo/term/GO:0006412). It enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), serves as a [structural constituent of ribosome](https://amigo.geneontology.org/amigo/term/GO:0003735), and facilitates [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). Additionally, it is part of the [cytosolic large ribosomal subunit](https://amigo.geneontology.org/amigo/term/GO:0022625) and is located in the [cytosolic ribosome](https://amigo.geneontology.org/amigo/term/GO:0022626).


The gene length is 43,871 base pairs (62.85th percentile), the mature length is 868 base pairs (16.94th percentile), and the primary transcript length is 21,091 base pairs (53.05th percentile).


The gene RPL31, with NCBI ID 6160, has been mentioned in [9 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RPL31%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication mentioning this gene is 1989, and the middle 50% of publications occurred between 2011 and 2014.


The top 5 publications mentioning RPL31, ranked by their relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows:

- [MicroRNA-483-5p inhibits osteogenic differentiation of human bone marrow mesenchymal stem cells by targeting the RPL31-mediated RAS/MEK/ERK signaling pathway.](https://pubmed.ncbi.nlm.nih.gov/35248705) (2022) (relative citation ratio: 1.36)
- [Characterization of Zn(II)-responsive ribosomal proteins YkgM and L31 in E. coli.](https://pubmed.ncbi.nlm.nih.gov/22196016) (2012) (relative citation ratio: 1.03)
- [Short hairpin RNA library-based functional screening identified ribosomal protein L31 that modulates prostate cancer cell growth via p53 pathway.](https://pubmed.ncbi.nlm.nih.gov/25285958) (2014) (relative citation ratio: 0.86)
- [Phorbol ester induction of differentiation and apoptosis in the K562 cell line is accompanied by marked decreases in the stability of globin mRNAs and decreases in the steady state level of mRNAs encoding for ribosomal proteins L35, L31, L27, and L21.](https://pubmed.ncbi.nlm.nih.gov/7804323) (1994) (relative citation ratio: 0.45)
- [Expression, purification, and evaluation for anticancer activity of ribosomal protein L31 gene (RPL31) from the giant panda (Ailuropoda melanoleuca).](https://pubmed.ncbi.nlm.nih.gov/22714919) (2012) (relative citation ratio: 0.3)

Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RPL31](https://www.proteinatlas.org/ENSG00000071082-RPL31) is a ribonucleoprotein and ribosomal protein with evidence at the protein level. It is detected in all RNA tissue distributions and is associated with non-specific ribosomal expression clusters across various tissues, including the brain, and in different cell lines. Additionally, RPL31 is part of the non-specific transcription and translation cluster in blood expression.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) and [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170), each with 5 experiments. Additionally, [E2F1](https://www.ncbi.nlm.nih.gov/gene/1869), [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885), and [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613) were each found to be regulating in 4 experiments.


The top transcription factors (TFs) identified in terms of the number of Perturb-seq experiments are [11091](https://www.ncbi.nlm.nih.gov/gene/11091) with 3 experiments and [148979](https://www.ncbi.nlm.nih.gov/gene/148979) with 1 experiment. These TFs were found to regulate gene expression in multiple experiments, highlighting their significant roles in transcriptional regulation.




The gene is expressed in various tissues and cell types, including the ovary, cervix uteri, lymphoma (Burkitt's), CD34+ cells, CD19+ B cells, promyelocytic leukemia (HL-60), appendix, atrioventricular node, and ovary. This indicates a ubiquitous expression pattern across multiple tissue samples.


The input data highlights various pathways involved in protein metabolism and gene expression, including 3'-UTR-mediated translational regulation, L13a-mediated translational silencing of ceruloplasmin expression, and GTP hydrolysis and joining of the 60S ribosomal subunit. Key processes such as cap-dependent translation initiation, eukaryotic translation initiation, formation of a pool of free 40S subunits, eukaryotic translation elongation, and eukaryotic translation termination are also noted. Additionally, the data covers peptide chain elongation, viral mRNA translation, and nonsense-mediated decay (NMD) both independent of and enhanced by the exon junction complex (EJC). Pathways related to SRP-dependent cotranslational protein targeting to the membrane and translation are included. Furthermore, the data encompasses influenza infection, the influenza life cycle, influenza viral RNA transcription and replication, and disease.



The analyzed protein sequence has a GRAVY value of -0.814 (11.02 percentile), indicating a hydrophilic nature. It exhibits a charge of 16.68 at pH 7.0 (90.7 percentile) and a median structural flexibility of 1.006 (68.77 percentile). The protein's secondary structure is predicted to be 32.8% helix (57.68 percentile), 34.4% sheet (54.87 percentile), and 24% turn (15.06 percentile). The instability index is 41.61 (27.29 percentile), with an isoelectric point of 10.54 (98.41 percentile). The protein has a length of 125 amino acids (5.12 percentile) and a molecular weight of 14462.68 Da (5.78 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |