---
layout: post
title: ABI2
date: 2025-01-17 16:55 CST
description: ABI2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10152) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10152  | ABI2 | ENSG00000138443 | 2q33.2 |



The gene is involved in various biological processes, including [neuron migration](https://amigo.geneontology.org/amigo/term/GO:0001764), [cytoskeleton organization](https://amigo.geneontology.org/amigo/term/GO:0007010), [nervous system development](https://amigo.geneontology.org/amigo/term/GO:0007399), [actin polymerization or depolymerization](https://amigo.geneontology.org/amigo/term/GO:0008154), [positive regulation of lamellipodium assembly](https://amigo.geneontology.org/amigo/term/GO:0010592), [cell migration](https://amigo.geneontology.org/amigo/term/GO:0016477), [Rac protein signal transduction](https://amigo.geneontology.org/amigo/term/GO:0016601), [regulation of dendritic spine morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0061001), [lens fiber cell morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0070309), and [positive regulation of Arp2/3 complex-mediated actin nucleation](https://amigo.geneontology.org/amigo/term/GO:2000601). It enables functions such as [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [cytoskeletal anchor activity](https://amigo.geneontology.org/amigo/term/GO:0008093), [SH3 domain binding](https://amigo.geneontology.org/amigo/term/GO:0017124), [kinase binding](https://amigo.geneontology.org/amigo/term/GO:0019900), [ubiquitin protein ligase binding](https://amigo.geneontology.org/amigo/term/GO:0031625), [signaling adaptor activity](https://amigo.geneontology.org/amigo/term/GO:0035591), [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802), and [proline-rich region binding](https://amigo.geneontology.org/amigo/term/GO:0070064). The gene is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0005856), [adherens junction](https://amigo.geneontology.org/amigo/term/GO:0005912), [lamellipodium](https://amigo.geneontology.org/amigo/term/GO:0030027), [filopodium tip](https://amigo.geneontology.org/amigo/term/GO:0032433), [dendritic spine](https://amigo.geneontology.org/amigo/term/GO:0043197), and [actin-based cell projection](https://amigo.geneontology.org/amigo/term/GO:0098858). Additionally, it is part of the [SCAR complex](https://amigo.geneontology.org/amigo/term/GO:0031209) and acts upstream of or within [peptidyl-tyrosine phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0018108).


The gene length is 33,294 base pairs (55.5% of all genes), the mature length is 585 base pairs, and the primary transcript length is 21,345 base pairs.


The gene ABI2 (NCBI ID: 10152) has been mentioned in [15 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ABI2%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1995 and the middle 50% of publications occurring between 2003 and 2017. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning ABI2, ranked by their scientific influence, include studies that explore the gene's role in various biological processes. The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Notable publications include "[Differential abscisic acid regulation of guard cell slow anion channels in Arabidopsis wild-type and abi1 and abi2 mutants.](https://pubmed.ncbi.nlm.nih.gov/9090884)" (1997) (relative citation ratio: 6.95), "[Abscisic Acid Signaling Inhibits Brassinosteroid Signaling through Dampening the Dephosphorylation of BIN2 by ABI1 and ABI2.](https://pubmed.ncbi.nlm.nih.gov/29275167)" (2018) (relative citation ratio: 5.2), "[Arabidopsis abi1-1 and abi2-1 phosphatase mutations reduce abscisic acid-induced cytoplasmic calcium rises in guard cells.](https://pubmed.ncbi.nlm.nih.gov/10488243)" (1999) (relative citation ratio: 4.13), "[Abi-2, a novel SH3-containing protein interacts with the c-Abl tyrosine kinase and modulates c-Abl transforming activity.](https://pubmed.ncbi.nlm.nih.gov/7590236)" (1995) (relative citation ratio: 4.11), and "[ABi<sub>2</sub> (IO<sub>3</sub> )<sub>2</sub> F<sub>5</sub> (A=K, Rb, and Cs): A Combination of Halide and Oxide Anionic Units To Create a Large Second-Harmonic Generation Response with a Wide Bandgap.](https://pubmed.ncbi.nlm.nih.gov/28631860)" (2017) (relative citation ratio: 3.29). The citation count is sourced from [iCite](https://icite.od.nih.gov).


[ABI2](https://www.proteinatlas.org/ENSG00000138443-ABI2) is a gene with evidence at the protein level, implicated in intellectual disability. It is detected in all tissues and is primarily localized in the nucleoplasm, vesicles, and cytosol. ABI2 is expressed in various clusters, including Cluster 8 in plasmacytoid DCs with unknown function, Cluster 64 in neuronal tissues related to transcription, Cluster 33 in brain neurons with mixed function, Cluster 26 in non-specific cell lines associated with mitochondria and RNA binding, and Cluster 9 in endometrial stromal cells related to transcription.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [E2F8](https://www.ncbi.nlm.nih.gov/gene/79733), [TAF1](https://www.ncbi.nlm.nih.gov/gene/6872), [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170), and [IKZF1](https://www.ncbi.nlm.nih.gov/gene/10320), each with 5 experiments. Additionally, [SMC3](https://www.ncbi.nlm.nih.gov/gene/9126) was found to be regulating in 4 experiments.



The GWAS data indicates associations with various disease conditions, including diseases of metabolism, acquired metabolic diseases, carbohydrate metabolism diseases, glucose metabolism diseases, diabetes mellitus, and specifically type 2 diabetes mellitus. Additionally, the data highlights nutrition diseases, overnutrition, obesity, artery diseases, vascular diseases, and cardiovascular system diseases. Other notable associations include arteriosclerosis, arteriosclerotic cardiovascular disease, and atherosclerosis.



The gene is expressed in various neural tissues, including the brain and pituitary, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the Superior Cervical Ganglion, Trigeminal Ganglion, and multiple regions of the testis, including the interstitial tissue, Leydig cells, seminiferous tubules, and germ cells.


The input data highlights several key pathways involved in the immune system, including the innate immune system, signaling by VEGF, and the VEGFA-VEGFR2 pathway. Additionally, it covers Fcgamma receptor (FCGR) dependent phagocytosis, regulation of actin dynamics for phagocytic cup formation, and signal transduction.


The mouse ortholog gene 329165, which corresponds to the human ortholog gene 10152, is associated with several phenotypic abnormalities. These include abnormal cornea morphology and abnormal iris morphology, both observed in both sexes. Additionally, the gene is linked to abnormal iris pigmentation, abnormal seminal vesicle morphology, absent urinary bladder, and small testis, although these phenotypes are not observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.612 (24.21% percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -6.65 (29.32% percentile), and the median structural flexibility is 1.011 (84.89% percentile). The protein has a propensity for helix, sheet, and turn structures of 24.56% (4.3% percentile), 30.41% (30.47% percentile), and 39.38% (96.91% percentile), respectively. The instability index is 74.19 (96.61% percentile), suggesting high instability. The isoelectric point is 5.82 (25.74% percentile), with a molecular weight of 55662.56 Da (58.06% percentile). The sequence length is 513 amino acids (59.84% percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |