---
layout: post
title: GRN
date: 2025-01-17 16:55 CST
description: GRN description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2896) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2896  | GRN | ENSG00000030582 | 17q21.31 |



The gene is involved in various biological processes, including [astrocyte activation involved in immune response](https://amigo.geneontology.org/amigo/term/GO:0002265), [microglial cell activation involved in immune response](https://amigo.geneontology.org/amigo/term/GO:0002282), [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), and [positive regulation of inflammatory response](https://amigo.geneontology.org/amigo/term/GO:0050727). It enables functions such as [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [cytokine activity](https://amigo.geneontology.org/amigo/term/GO:0005125), and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is located in multiple cellular components, including the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615), [lysosome](https://amigo.geneontology.org/amigo/term/GO:0005764), [lysosomal membrane](https://amigo.geneontology.org/amigo/term/GO:0005765), [endosome](https://amigo.geneontology.org/amigo/term/GO:0005768), [late endosome](https://amigo.geneontology.org/amigo/term/GO:0005770), [endoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0005783), [Golgi apparatus](https://amigo.geneontology.org/amigo/term/GO:0005794), [trans-Golgi network](https://amigo.geneontology.org/amigo/term/GO:0005802), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), and [azurophil granule lumen](https://amigo.geneontology.org/amigo/term/GO:0035578). Additionally, it is involved in processes such as [lysosome organization](https://amigo.geneontology.org/amigo/term/GO:0007040), [lysosomal transport](https://amigo.geneontology.org/amigo/term/GO:0007041), and [lysosomal lumen acidification](https://amigo.geneontology.org/amigo/term/GO:0007042). The gene also plays a role in the [positive regulation of endothelial cell migration](https://amigo.geneontology.org/amigo/term/GO:0010595), [positive regulation of cell migration](https://amigo.geneontology.org/amigo/term/GO:0030335), [negative regulation of neuron apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0043524), [positive regulation of neuron apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0043525), [positive regulation of angiogenesis](https://amigo.geneontology.org/amigo/term/GO:0045766), [positive regulation of axon regeneration](https://amigo.geneontology.org/amigo/term/GO:0048680), [positive regulation of epithelial cell proliferation](https://amigo.geneontology.org/amigo/term/GO:0050679), [protein stabilization](https://amigo.geneontology.org/amigo/term/GO:0050821), [protein-folding chaperone binding](https://amigo.geneontology.org/amigo/term/GO:0051087), [negative regulation of respiratory burst involved in inflammatory response](https://amigo.geneontology.org/amigo/term/GO:0060266), [positive regulation of inflammatory response to wounding](https://amigo.geneontology.org/amigo/term/GO:0106016), [positive regulation of defense response to bacterium](https://amigo.geneontology.org/amigo/term/GO:1900426), [negative regulation of neutrophil activation](https://amigo.geneontology.org/amigo/term/GO:1902564), [positive regulation of protein folding](https://amigo.geneontology.org/amigo/term/GO:1903334), [negative regulation of microglial cell activation](https://amigo.geneontology.org/amigo/term/GO:1903979), [positive regulation of aspartic-type peptidase activity](https://amigo.geneontology.org/amigo/term/GO:1905247), and [positive regulation of lysosome organization](https://amigo.geneontology.org/amigo/term/GO:1905673).


The gene has a length of 8,760 base pairs (23.01% of all genes), a mature length of 1,971 base pairs, and a primary transcript length of 8,242 base pairs.


The gene GRN (NCBI ID: 2896) has been mentioned in [872 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22GRN%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication mentioning GRN is 1973, and the middle 50% of publications occurred between 2011 and 2019.


The top five publications mentioning GRN, ranked by their scientific influence, include "[Null mutations in progranulin cause ubiquitin-positive frontotemporal dementia linked to chromosome 17q21.](https://pubmed.ncbi.nlm.nih.gov/16862115)" (2006) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 24.06, "[Progranulin as a therapeutic target in neurodegenerative diseases.](https://pubmed.ncbi.nlm.nih.gov/35039149)" (2022) with an RCR of 12.01, "[Neuroimaging signatures of frontotemporal dementia genetics: C9ORF72, tau, progranulin and sporadics.](https://pubmed.ncbi.nlm.nih.gov/22366795)" (2012) with an RCR of 10.57, "[Mutations in progranulin are a major cause of ubiquitin-positive frontotemporal lobar degeneration.](https://pubmed.ncbi.nlm.nih.gov/16950801)" (2006) with an RCR of 9.55, and "[Macrophage-secreted granulin supports pancreatic cancer metastasis by inducing liver fibrosis.](https://pubmed.ncbi.nlm.nih.gov/27088855)" (2016) with an RCR of 9.47. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[GRN](https://www.proteinatlas.org/ENSG00000030582-GRN) functions as a cytokine and is implicated in neurodegeneration and neuronal ceroid lipofuscinosis, with evidence at the protein level. It is detected in all tissues and is primarily localized in endosomes and lysosomes. GRN is expressed in monocytes and neutrophils, associated with degranulation, and is found in bone marrow, linked to cell proliferation. In the brain, it is expressed in macrophages and microglia, involved in immune response. Additionally, GRN is expressed in breast cancer cell lines, although its function there is unknown. In single-cell analysis, it is found in macrophages, contributing to the innate immune response.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), and [ZNF143](https://www.ncbi.nlm.nih.gov/gene/7702), each with 4 experiments. Additionally, [SMAD3](https://www.ncbi.nlm.nih.gov/gene/4088) and [BCL3](https://www.ncbi.nlm.nih.gov/gene/602) were found to be regulating in 3 experiments each.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [8243](https://www.ncbi.nlm.nih.gov/gene/8243) with 3 experiments, [6829](https://www.ncbi.nlm.nih.gov/gene/6829) and [9968](https://www.ncbi.nlm.nih.gov/gene/9968) each with 2 experiments, and [2623](https://www.ncbi.nlm.nih.gov/gene/2623) and [9126](https://www.ncbi.nlm.nih.gov/gene/9126) each with 1 experiment. These TFs were found to regulate gene expression in multiple Perturb-seq experiments, highlighting their significant roles in transcriptional regulation.




The gene is expressed in various tissues and cell types, including blood and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Lymphoma Burkitt's (Raji), CD33+ Myeloid cells, BDCA4+ Dendritic Cells, CD14+ Monocytes, Lung, Placenta, and Bone marrow.




The protein sequence analyzed has a GRAVY value of -0.231 (66.11th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -5.94 (31.16th percentile). The median structural flexibility is 0.992 (14.06th percentile), with a propensity for turns (31.4%, 73.02nd percentile), followed by sheets (23.4%, 5.58th percentile) and helices (21.1%, 1.52nd percentile). The instability index is 51.49 (61.96th percentile), and the isoelectric point is 6.43 (40.96th percentile). The protein has a length of 593 amino acids (68th percentile) and a molecular weight of 63544.14 Da (65.96th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |