---
layout: post
title: NGF
date: 2025-01-17 16:55 CST
description: NGF description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4803) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4803  | NGF | ENSG00000134259 | 1p13.2 |



The gene enables [nerve growth factor receptor binding](https://amigo.geneontology.org/amigo/term/GO:0005163) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is involved in [transmembrane receptor protein tyrosine kinase signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007169), [peripheral nervous system development](https://amigo.geneontology.org/amigo/term/GO:0007422), and [memory](https://amigo.geneontology.org/amigo/term/GO:0007613). It is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615), [Golgi lumen](https://amigo.geneontology.org/amigo/term/GO:0005796), and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and is active in the [synaptic vesicle](https://amigo.geneontology.org/amigo/term/GO:0008021), [axon](https://amigo.geneontology.org/amigo/term/GO:0030424), and [dendrite](https://amigo.geneontology.org/amigo/term/GO:0030425). Additionally, it enables [growth factor activity](https://amigo.geneontology.org/amigo/term/GO:0008083), [metalloendopeptidase inhibitor activity](https://amigo.geneontology.org/amigo/term/GO:0008191), and [lipid binding](https://amigo.geneontology.org/amigo/term/GO:0008289). The gene is also involved in various regulatory processes such as [negative regulation of cell population proliferation](https://amigo.geneontology.org/amigo/term/GO:0008285), [extrinsic apoptotic signaling pathway via death domain receptors](https://amigo.geneontology.org/amigo/term/GO:0008625), [positive regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0010628), [nerve development](https://amigo.geneontology.org/amigo/term/GO:0021675), [positive regulation of peptidyl-serine phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0033138), [nerve growth factor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0038180), [positive regulation of DNA binding](https://amigo.geneontology.org/amigo/term/GO:0043388), [negative regulation of neuron apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0043524), [regulation of neuron differentiation](https://amigo.geneontology.org/amigo/term/GO:0045664), [positive regulation of neuron differentiation](https://amigo.geneontology.org/amigo/term/GO:0045666), [positive regulation of Ras protein signal transduction](https://amigo.geneontology.org/amigo/term/GO:0046579), [positive regulation of collateral sprouting](https://amigo.geneontology.org/amigo/term/GO:0048672), [neuron projection morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0048812), and [modulation of chemical synaptic transmission](https://amigo.geneontology.org/amigo/term/GO:0050804).


The gene length is 4,758 base pairs (13.31% of all genes), the mature length is 2,264 base pairs, and the primary transcript length is 3,784.5 base pairs (14.01% of all genes).


The gene NGF (NCBI ID: 4803) has been mentioned in [2,293 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22NGF%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1964, and the middle 50% of publications occurred between 1996 and 2014.


The top five publications mentioning NGF, ranked by their scientific influence, include "[Neurotrophin-3: a neurotrophic factor related to NGF and BDNF.](https://pubmed.ncbi.nlm.nih.gov/2321006)" (1990) (relative citation ratio: 28.91), "[Changes of nerve growth factor synthesis in nonneuronal cells in response to sciatic nerve transection.](https://pubmed.ncbi.nlm.nih.gov/3034917)" (1987) (relative citation ratio: 27.14), "[High-affinity NGF binding requires coexpression of the trk proto-oncogene and the low-affinity NGF receptor.](https://pubmed.ncbi.nlm.nih.gov/1850821)" (1991) (relative citation ratio: 25.88), "[Tyrosine phosphorylation and tyrosine kinase activity of the trk proto-oncogene product induced by NGF.](https://pubmed.ncbi.nlm.nih.gov/1706478)" (1991) (relative citation ratio: 21.92), and "[Nerve growth factor signaling, neuroprotection, and neural repair.](https://pubmed.ncbi.nlm.nih.gov/11520933)" (2001) (relative citation ratio: 21.32). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[NGF](https://www.proteinatlas.org/ENSG00000134259-NGF) is a multifunctional gene that acts as a growth factor, metalloenzyme inhibitor, metalloprotease inhibitor, and protease inhibitor. It is implicated in various diseases, including cancer-related genes, neurodegeneration, and neuropathy, with evidence at the protein level. NGF is detected in many tissues and is particularly expressed in smooth muscle tissue, where it is involved in ECM organization. In the brain, it is associated with neurons and synapses, playing a role in synaptic function. Additionally, NGF is expressed in connective tissue cells, contributing to ECM organization.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) and [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227), each with 4 experiments. Additionally, [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), and [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664) were each found to be regulating in 3 experiments.



The GWAS data indicates associations with a range of disease conditions, including metabolic disorders such as obesity and overnutrition, mental health conditions like bipolar disorder, major depressive disorder, schizophrenia, and substance dependence, particularly alcohol dependence. Additionally, the data highlights associations with musculoskeletal system diseases, including rheumatoid arthritis and juvenile rheumatoid arthritis, as well as neurological conditions such as epilepsy syndrome.



The gene is expressed in various tissues, including the ovary and fallopian tube, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Lymphoma (Burkitt's, Raji), Subthalamic Nucleus, Pons, Salivary Gland, Skeletal Muscle, and Ciliary Ganglion.


The input data describes various signaling pathways and processes involving proteins such as NF-kB, p75NTR, TRKA, and NGF. Key events include the activation of NF-kB, which signals survival, and the recruitment of signaling complexes by p75NTR. Additionally, p75NTR signals via NF-kB and regulates axonogenesis, while NRIF signals cell death from the nucleus. Other notable pathways include ARMS- and Frs2-mediated activation, prolonged ERK activation, signaling to p38 via RIT and RIN, and signaling to ERKs and RAS. PI3K/AKT activation and NGF signaling via TRKA from the plasma membrane are also highlighted. Furthermore, TRKA receptors are activated by NGF, leading to signaling to STAT3. The data also mentions axonal growth stimulation, ceramide signaling, and the binding of NGF and proNGF to p75NTR. Additionally, p75NTR negatively regulates the cell cycle via SC1, and NADE modulates death signaling. Cell death signaling via NRAGE, NRIF, and NADE, as well as NRAGE signaling death through JNK, are also noted. Other processes include NGF processing, p75NTR receptor-mediated signaling, signal transduction, PLC-gamma1 signaling, and retrograde neurotrophin signaling.



The analyzed protein sequence has a GRAVY value of -0.363 (50.65th percentile), indicating a hydrophilic nature. It exhibits a charge of 14.25 at pH 7.0 (88.18th percentile) and a median structural flexibility of 0.998 (33.45th percentile). The protein's secondary structure is predicted to be 25.73% helix (6.75th percentile), 34.44% sheet (55.14th percentile), and 26.56% turn (34.15th percentile). The instability index is 46.89 (46.2nd percentile), with an isoelectric point of 9.94 (95.98th percentile). The protein is 241 amino acids long (20.37th percentile) and has a molecular weight of 26958.47 Da (20.52nd percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |