---
layout: post
title: LYVE1
date: 2025-01-17 16:55 CST
description: LYVE1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10894) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10894  | LYVE1 | ENSG00000133800 | 11p15.4 |



The gene is involved in [positive regulation of cellular extravasation](https://amigo.geneontology.org/amigo/term/GO:0002693) and [receptor-mediated endocytosis](https://amigo.geneontology.org/amigo/term/GO:0006898). It enables [transmembrane signaling receptor activity](https://amigo.geneontology.org/amigo/term/GO:0004888), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [hyaluronic acid binding](https://amigo.geneontology.org/amigo/term/GO:0005540). Additionally, it is involved in [cell-matrix adhesion](https://amigo.geneontology.org/amigo/term/GO:0007160), [response to wounding](https://amigo.geneontology.org/amigo/term/GO:0009611), [anatomical structure morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0009653), and [hyaluronan catabolic process](https://amigo.geneontology.org/amigo/term/GO:0030214). The gene is active in and located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), as well as the [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020) and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). It also enables [signaling receptor activity](https://amigo.geneontology.org/amigo/term/GO:0038023) and [cargo receptor activity](https://amigo.geneontology.org/amigo/term/GO:0038024).


The gene length is 1,521 base pairs (3.85th percentile), the mature length is 795 base pairs, and the primary transcript length is 1,521 base pairs (4.51st percentile).


The gene LYVE1 (NCBI ID: 10894) has been mentioned in [116 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22LYVE1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink). The earliest year of publication is 1958, and the middle 50% of publications occurred between 2006 and 2018.


The top 5 publications mentioning LYVE1, ranked by their scientific influence, include "[LYVE-1, a new homologue of the CD44 glycoprotein, is a lymph-specific receptor for hyaluronan.](https://pubmed.ncbi.nlm.nih.gov/10037799)" (1999) (relative citation ratio: 26.4), "[Splicing of a novel androgen receptor exon generates a constitutively active androgen receptor that mediates prostate cancer therapy resistance.](https://pubmed.ncbi.nlm.nih.gov/18593950)" (2008) (relative citation ratio: 13.5), "[LYVE-1 is not restricted to the lymph vessels: expression in normal liver blood sinusoids and down-regulation in human liver cancer and cirrhosis.](https://pubmed.ncbi.nlm.nih.gov/11719431)" (2001) (relative citation ratio: 6.62), "[LYVE-1, the lymphatic system and tumor lymphangiogenesis.](https://pubmed.ncbi.nlm.nih.gov/11377291)" (2001) (relative citation ratio: 5.34), and "[Dendritic cells enter lymph vessels by hyaluronan-mediated docking to the endothelial receptor LYVE-1.](https://pubmed.ncbi.nlm.nih.gov/28504698)" (2017) (relative citation ratio: 4.79). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[LYVE1](https://www.proteinatlas.org/ENSG00000133800-LYVE1) is a receptor with evidence at the protein level. It is detected in many tissues and is particularly expressed in adipose tissue, choroid plexus, adipocytes, endothelial cells, and macrophages, where it plays a role in angiogenesis and innate immune response. The gene is unprognostic for various cancer types, including Bladder Urothelial Carcinoma, Breast Invasive Carcinoma, Colon Adenocarcinoma, Glioblastoma Multiforme, Head and Neck Squamous Cell Carcinoma, Kidney Chromophobe, Kidney Renal Clear Cell Carcinoma, Kidney Renal Papillary Cell Carcinoma, Liver Hepatocellular Carcinoma, Lung Adenocarcinoma, Lung Squamous Cell Carcinoma, Ovary Serous Cystadenocarcinoma, Pancreatic Adenocarcinoma, Rectum Adenocarcinoma, Stomach Adenocarcinoma, and Testicular Germ Cell Tumor.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [PPARG](https://www.ncbi.nlm.nih.gov/gene/5468) and [RELA](https://www.ncbi.nlm.nih.gov/gene/5970), each shown to be regulating in 3 experiments. Additionally, [INO80](https://www.ncbi.nlm.nih.gov/gene/54617), [FOS](https://www.ncbi.nlm.nih.gov/gene/2353), and [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) were each found to be regulating in 2 experiments.



The GWAS data indicates associations with various conditions, including cognitive disorder, disease of mental health, psychotic disorder, and schizophrenia.



The gene is expressed in adipose tissue and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in various tissues, including thyroid, placenta, adrenal gland, fetal liver, ovary, fetal lung, and lymph node.


The input data includes various pathways and diseases related to carbohydrate metabolism, glycosaminoglycan metabolism, and diseases of glycosylation. Notable conditions include Myoclonic epilepsy of Lafora, Glycogen storage diseases, and several types of Mucopolysaccharidoses (MPS), such as MPS VII - Sly syndrome, MPS VI - Maroteaux-Lamy syndrome, and MPS II - Hunter syndrome. Additionally, the data highlights defects in specific genes like B3GAT3, B4GALT1, and CHST14, which are associated with conditions like JDSSDHD, B4GALT1-CDG (CDG-2d), and EDS, musculocontractural type, respectively. Other pathways mentioned include hyaluronan metabolism and uptake, as well as general metabolism and disease processes.



The analyzed protein sequence has a GRAVY value of -0.128 (76.05th percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is 4.445 (66.1st percentile), and the median structural flexibility is 1.002 (52.58th percentile). The protein has a predicted affinity for helix, sheet, and turn structures of 32.61% (56.06th percentile), 38.82% (82.36th percentile), and 26.40% (32.76th percentile), respectively. The instability index is 38.30 (17.62nd percentile), and the isoelectric point is 8.59 (68.62nd percentile). The protein is 322 amino acids long (33.25th percentile) with a molecular weight of 35213.03 Da (31.79th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |