---
layout: post
title: CSRP1
date: 2025-01-17 16:55 CST
description: CSRP1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1465) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1465  | CSRP1 | ENSG00000159176 | 1q32.1 |



The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is involved in [zinc ion binding](https://amigo.geneontology.org/amigo/term/GO:0008270) and [actinin binding](https://amigo.geneontology.org/amigo/term/GO:0042805). It is active in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), and is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). Additionally, it is active in the [Z disc](https://amigo.geneontology.org/amigo/term/GO:0030018) and plays a role as a [structural constituent of muscle](https://amigo.geneontology.org/amigo/term/GO:0008307). The gene is involved in [sarcomere organization](https://amigo.geneontology.org/amigo/term/GO:0045214), [muscle tissue development](https://amigo.geneontology.org/amigo/term/GO:0060537), and [platelet aggregation](https://amigo.geneontology.org/amigo/term/GO:0070527).


The gene length is 17,284 base pairs (38.4% of all genes), the mature length is 3,410.5 base pairs, and the primary transcript length is 11,267.5 base pairs.


The gene CSRP1 (NCBI ID: 1465) has been mentioned in [7 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CSRP1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning this gene was in 1992, and the middle 50% of publications occurred between 1999 and 2012.


The top 5 publications mentioning CSRP1, ranked by their scientific influence, include "[CSRP's Impact on low-income preschoolers' preacademic skills: self-regulation as a mediating mechanism.](https://pubmed.ncbi.nlm.nih.gov/21291447)" (2011) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 12.53, "[CRP1, a LIM domain protein implicated in muscle differentiation, interacts with alpha-actinin.](https://pubmed.ncbi.nlm.nih.gov/9314536)" (1997) with an RCR of 2.15, and "[LIM-domain protein cysteine- and glycine-rich protein 2 (CRP2) is a novel marker of hepatic stellate cells and binding partner of the protein inhibitor of activated STAT1.](https://pubmed.ncbi.nlm.nih.gov/11672422)" (2001) with an RCR of 1.02. The RCR values measure the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CSRP1](https://www.proteinatlas.org/ENSG00000159176-CSRP1) is a gene with evidence at the protein level. Its RNA is detected in all tissues, and it is notably expressed in monocytes and neutrophils involved in degranulation (Cluster 38). In tissue expression, it is associated with smooth muscle tissue and extracellular matrix (ECM) organization (Cluster 41). In the brain, it is linked to astrocytes with mixed functions (Cluster 19). Additionally, CSRP1 is expressed in skin cancer cell lines related to ECM organization (Cluster 69) and in single cells, specifically smooth muscle cells involved in muscle contraction (Cluster 32).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [MYB](https://www.ncbi.nlm.nih.gov/gene/4602) with 6 experiments, [TFAP2C](https://www.ncbi.nlm.nih.gov/gene/7022) with 5 experiments, [AFF4](https://www.ncbi.nlm.nih.gov/gene/27125) with 5 experiments, [SIN3A](https://www.ncbi.nlm.nih.gov/gene/25942) with 5 experiments, and [PGR](https://www.ncbi.nlm.nih.gov/gene/5241) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with a range of disease conditions, including cardiovascular system disease, nervous system disease, central nervous system disease, neurodegenerative disease, amyotrophic lateral sclerosis, motor neuron disease, bone disease, connective tissue disease, musculoskeletal system disease, and bone structure disease.



The gene is expressed in multiple tissue samples, including the prefrontal cortex, appendix, and uterus.




The protein sequence analyzed has a GRAVY value of -0.5264 (32.25th percentile), indicating a hydrophilic nature. It exhibits a charge of 9.89 at pH 7.0 (81.23rd percentile) and a median structural flexibility of 1.007 (72.67th percentile). The protein's secondary structure is predicted to be 31.61% helix (46.77th percentile), 25.39% sheet (9.67th percentile), and 30.57% turn (67.95th percentile). The instability index is 24.91 (1.52nd percentile), with an isoelectric point of 8.90 (76.55th percentile). The protein has a length of 193 amino acids (13.45th percentile) and a molecular weight of 20567.21 Da (12.07th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |