---
layout: post
title: DUSP5
date: 2025-01-17 16:55 CST
description: DUSP5 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1847) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1847  | DUSP5 | ENSG00000138166 | 10q25.2 |



The gene is involved in the [MAPK cascade](https://amigo.geneontology.org/amigo/term/GO:0000165) and [endoderm formation](https://amigo.geneontology.org/amigo/term/GO:0001706). It enables [protein tyrosine phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0004725) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is active in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), and is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654). Additionally, it is involved in [protein dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0006470) and [dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0016311), enabling various phosphatase activities such as [protein tyrosine/serine/threonine phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0008138), [protein tyrosine/threonine phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0008330), [phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0016791), [MAP kinase tyrosine/serine/threonine phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0017017), and [myosin phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0017018). The gene is also involved in [peptidyl-tyrosine dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0035335), [peptidyl-threonine dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0035970), [negative regulation of MAPK cascade](https://amigo.geneontology.org/amigo/term/GO:0043409), and the [ERK1 and ERK2 cascade](https://amigo.geneontology.org/amigo/term/GO:0070371).


The gene length is 8,622 base pairs (22.68% of all genes), the mature length is 1,201 base pairs (29.03% of all genes), and the primary transcript length is 8,622 base pairs (29.64% of all genes).


DUSP5 (Gene ID: 1847) has been mentioned in [65 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22DUSP5%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1969 and the middle 50% of publications occurring between 2013 and 2019. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning DUSP5, ranked by their scientific influence, include "[Regulation of Dual-Specificity Phosphatase (DUSP) Ubiquitination and Protein Stability.](https://pubmed.ncbi.nlm.nih.gov/31151270)" (2019) (relative citation ratio: 5.43), "[Long noncoding RNA CRNDE promotes colorectal cancer cell proliferation via epigenetically silencing DUSP5/CDKN1A expression.](https://pubmed.ncbi.nlm.nih.gov/28796262)" (2017) (relative citation ratio: 5.36), "[SARS-CoV-2 Switches 'on' MAPK and NFÎºB Signaling via the Reduction of Nuclear DUSP1 and DUSP5 Expression.](https://pubmed.ncbi.nlm.nih.gov/33995033)" (2021) (relative citation ratio: 5.2), "[DUSP meet immunology: dual specificity MAPK phosphatases in control of the inflammatory response.](https://pubmed.ncbi.nlm.nih.gov/17114416)" (2006) (relative citation ratio: 4.84), and "[MAP4K Family Kinases and DUSP Family Phosphatases in T-Cell Signaling and Systemic Lupus Erythematosus.](https://pubmed.ncbi.nlm.nih.gov/31766293)" (2019) (relative citation ratio: 3.19). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[DUSP5](https://www.proteinatlas.org/ENSG00000138166-DUSP5) is a gene encoding a protein with hydrolase and protein phosphatase molecular functions. Evidence for this gene is available at the protein level. Its RNA is detected in all tissues, with notable expression in epithelium and extracellular exosomes. In the brain, it is involved in non-specific transcription processes. The gene is also expressed in squamous epithelial cells, where it is associated with keratinization, and in plasma cells, where it plays a role in protein processing. Additionally, it is found in plasmacytoid dendritic cells, although its function in these cells is unknown.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613), [STAT3](https://www.ncbi.nlm.nih.gov/gene/6774), and [TP53](https://www.ncbi.nlm.nih.gov/gene/7157), each with 5 experiments. Additionally, [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) and [NFE2](https://www.ncbi.nlm.nih.gov/gene/4778) were found to be regulating in 4 experiments each.





The gene is expressed in the esophagus and salivary gland according to BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles indicate expression in Lymphoma Burkitt's (Raji), BDCA4+ Dendritic Cells, Cerebellum, Bronchial Epithelial Cells, Placenta, Trachea, and Adrenal Cortex.




The analyzed protein sequence has a GRAVY value of -0.097 (78.26th percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is 7.28 (75.14th percentile), and the median structural flexibility is 0.996 (24.44th percentile). The protein has a high instability index of 61.96 (86.31st percentile) and an isoelectric point of 8.69 (70.92nd percentile). The sequence length is 384 amino acids (43.11th percentile) with a molecular weight of 42046.86 Da (42.14th percentile). The secondary structure affinities are 31.77% for helix (48.39th percentile), 33.59% for sheet (49.11th percentile), and 27.34% for turn (41.25th percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |