---
layout: post
title: MATR3
date: 2025-01-17 16:55 CST
description: MATR3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9782) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9782  | MATR3 | ENSG00000015479 | 5q31.2 |



The gene is involved in [blastocyst formation](https://amigo.geneontology.org/amigo/term/GO:0001825) and the [activation of innate immune response](https://amigo.geneontology.org/amigo/term/GO:0002218). It acts upstream of or within [heart valve development](https://amigo.geneontology.org/amigo/term/GO:0003170) and [ventricular septum development](https://amigo.geneontology.org/amigo/term/GO:0003281). The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [structural molecule activity](https://amigo.geneontology.org/amigo/term/GO:0005198), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [zinc ion binding](https://amigo.geneontology.org/amigo/term/GO:0008270), [miRNA binding](https://amigo.geneontology.org/amigo/term/GO:0035198), and [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802). It is active in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and located in the [nuclear inner membrane](https://amigo.geneontology.org/amigo/term/GO:0005637), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), and [nuclear matrix](https://amigo.geneontology.org/amigo/term/GO:0016363). Additionally, the gene is involved in the [post-transcriptional regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0010608) and the [innate immune response](https://amigo.geneontology.org/amigo/term/GO:0045087).


The gene length is 35,972 base pairs (57.61% of all genes), the mature length is 1,346 base pairs (33.48% of all genes), and the primary transcript length is 28,733 base pairs (61.54% of all genes).


The gene MATR3 (NCBI ID: 9782) has been mentioned in [75 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22MATR3%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1998 and the middle 50% of publications occurring between 2014 and 2020. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning MATR3, ranked by their scientific influence, include "[Mutations in the Matrin 3 gene cause familial amyotrophic lateral sclerosis.](https://pubmed.ncbi.nlm.nih.gov/24686783)" (2014) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 10.58, "[Involvement of Matrin 3 and SFPQ/NONO in the DNA damage response.](https://pubmed.ncbi.nlm.nih.gov/20421735)" (2010) with an RCR of 3.2, "[Nuclear matrix protein Matrin3 regulates alternative splicing and forms overlapping regulatory networks with PTB.](https://pubmed.ncbi.nlm.nih.gov/25599992)" (2015) with an RCR of 2.82, "[Matrin 3 binds and stabilizes mRNA.](https://pubmed.ncbi.nlm.nih.gov/21858232)" (2011) with an RCR of 2.72, and "[Autosomal-dominant distal myopathy associated with a recurrent missense mutation in the gene encoding the nuclear matrix protein, matrin 3.](https://pubmed.ncbi.nlm.nih.gov/19344878)" (2009) with an RCR of 2.55. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[MATR3](https://www.proteinatlas.org/ENSG00000015479-MATR3) is an RNA-binding protein primarily located in the nucleoplasm. It is implicated in various diseases, including amyotrophic lateral sclerosis, and is associated with neurodegeneration. Evidence for its function is supported at the protein level. MATR3 is detected in all RNA tissue distributions and is involved in transcription processes, as indicated by its expression clusters in neuronal tissues, brain, and single cells. Additionally, it is associated with non-specific cell proliferation in blood expression clusters and mRNA splicing and cell cycle in cell line expression clusters.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861), [GATA2](https://www.ncbi.nlm.nih.gov/gene/2624), and [PPARG](https://www.ncbi.nlm.nih.gov/gene/5468), each with 6 experiments. Additionally, [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930) and [KDM5B](https://www.ncbi.nlm.nih.gov/gene/10765) were found to be regulating in 5 experiments each.



The input data indicates associations with various disease conditions, including general disease, disease of anatomical entity, nervous system disease, central nervous system disease, neurodegenerative disease, synucleinopathy, Parkinson's disease, and sensory system disease.



The gene is expressed in various tissues, including the ovary and brain, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the pineal gland (night), prefrontal cortex, amygdala, fetal brain, medulla oblongata, and occipital lobe.




The analyzed protein sequence has a GRAVY value of -0.929 (6.34th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -15.32 (15th percentile), and the median structural flexibility is 1.015 (92.08th percentile). The protein has a high affinity for helix (33.18%, 60.76th percentile) and turn (35.66%, 91st percentile) structures, with a lower affinity for sheet structures (25.86%, 10.88th percentile). The instability index is 52.91 (66.39th percentile), and the isoelectric point is 5.87 (27.11th percentile). The protein is 847 amino acids long (82.57th percentile) with a molecular weight of 94622.14 Da (82.57th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |