---
layout: post
title: PKN2
date: 2025-01-17 16:55 CST
description: PKN2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5586) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5586  | PKN2 | ENSG00000065243 | 1p22.2 |



The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [protein kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004672), [protein serine/threonine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004674), [diacylglycerol-dependent serine/threonine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004697), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524), [kinase activity](https://amigo.geneontology.org/amigo/term/GO:0016301), [small GTPase binding](https://amigo.geneontology.org/amigo/term/GO:0031267), [histone deacetylase binding](https://amigo.geneontology.org/amigo/term/GO:0042826), [cadherin binding](https://amigo.geneontology.org/amigo/term/GO:0045296), and [RNA polymerase binding](https://amigo.geneontology.org/amigo/term/GO:0070063). It is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [centrosome](https://amigo.geneontology.org/amigo/term/GO:0005813), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [nuclear body](https://amigo.geneontology.org/amigo/term/GO:0016604), [lamellipodium](https://amigo.geneontology.org/amigo/term/GO:0030027), [midbody](https://amigo.geneontology.org/amigo/term/GO:0030496), [cleavage furrow](https://amigo.geneontology.org/amigo/term/GO:0032154), [apical junction complex](https://amigo.geneontology.org/amigo/term/GO:0043296), [intermediate filament cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0045111), and [perinuclear region of cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0048471). It is involved in [protein phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0006468), [apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0006915), [cell cycle](https://amigo.geneontology.org/amigo/term/GO:0007049), [cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0007155), [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), [epithelial cell migration](https://amigo.geneontology.org/amigo/term/GO:0010631), [peptidyl-serine phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0018105), [cell projection organization](https://amigo.geneontology.org/amigo/term/GO:0030030), [positive regulation of cytokinesis](https://amigo.geneontology.org/amigo/term/GO:0032467), [intracellular signal transduction](https://amigo.geneontology.org/amigo/term/GO:0035556), [apical junction assembly](https://amigo.geneontology.org/amigo/term/GO:0043297), [positive regulation of viral genome replication](https://amigo.geneontology.org/amigo/term/GO:0045070), [positive regulation of mitotic cell cycle](https://amigo.geneontology.org/amigo/term/GO:0045931), [cell division](https://amigo.geneontology.org/amigo/term/GO:0051301), and [regulation of cell motility](https://amigo.geneontology.org/amigo/term/GO:2000145). Additionally, it is part of a [protein-containing complex](https://amigo.geneontology.org/amigo/term/GO:0032991).


The gene length is 63,509.0 base pairs (72.21% of all genes), the mature length is 3,378.0 base pairs, and the primary transcript length is 61,696.5 base pairs.


The gene PKN2 (NCBI ID: 5586) has been mentioned in [28 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PKN2%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1970 and the middle 50% of publications occurring between 2000 and 2020. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning PKN2, ranked by their scientific influence, include "[PDK1 acquires PDK2 activity in the presence of a synthetic peptide derived from the carboxyl terminus of PRK2.](https://pubmed.ncbi.nlm.nih.gov/10226025)" (1999) with a relative citation ratio of 6.68, "[Discovery of new PKN2 inhibitory chemotypes via QSAR-guided selection of docking-based pharmacophores.](https://pubmed.ncbi.nlm.nih.gov/35507210)" (2023) with a relative citation ratio of 4.03, "[PKN2 in colon cancer cells inhibits M2 phenotype polarization of tumor-associated macrophages via regulating DUSP6-Erk1/2 pathway.](https://pubmed.ncbi.nlm.nih.gov/29368606)" (2018) with a relative citation ratio of 3.29, "[The PRK2 kinase is a potential effector target of both Rho and Rac GTPases and regulates actin cytoskeletal organization.](https://pubmed.ncbi.nlm.nih.gov/9121475)" (1997) with a relative citation ratio of 3.0, and "[Protein kinase N2 mediates flow-induced endothelial NOS activation and vascular tone regulation.](https://pubmed.ncbi.nlm.nih.gov/34499618)" (2021) with a relative citation ratio of 2.77. The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PKN2](https://www.proteinatlas.org/ENSG00000065243-PKN2) is a serine/threonine-protein kinase with transferase activity, functioning as a kinase. Evidence supports its existence at the protein level. The gene is detected in all RNA tissue distributions and is localized in various subcellular compartments, including the nucleoplasm, nuclear bodies, plasma membrane, intermediate filaments, centrosome, and cytosol. PKN2 is associated with blood expression cluster 21, which is specific to neutrophils with unknown function, and tissue expression cluster 40, which is non-specific and related to transcription. Additionally, it is part of brain expression cluster 55, cell line expression cluster 37, and single cell expression cluster 55, all of which are non-specific and related to transcription or mRNA processing.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025) with 7 experiments, followed by [PGR](https://www.ncbi.nlm.nih.gov/gene/5241), [GATA2](https://www.ncbi.nlm.nih.gov/gene/2624), [TAL1](https://www.ncbi.nlm.nih.gov/gene/6886), and [TCF3](https://www.ncbi.nlm.nih.gov/gene/6929), each with 5 experiments.



The input data indicates associations with various disease conditions, including hematopoietic system disease, immune system disease, artery disease, vascular disease, cardiovascular system disease, arteriosclerosis, arteriosclerotic cardiovascular disease, atherosclerosis, hypersensitivity reaction disease, hypersensitivity reaction type II disease, autoimmune disease of the gastrointestinal tract, and celiac disease.


The PKN2 gene has been associated with several traits and diseases through genome-wide association studies. Notably, it has been linked to [Intraocular pressure](https://pubmed.ncbi.nlm.nih.gov/29617998), which maps closely to the MeSH term [Intraocular Pressure](https://meshb.nlm.nih.gov/record/ui?ui=D007429) using a direct canonical name match. Additionally, PKN2 has been associated with [Mean corpuscular hemoglobin](https://pubmed.ncbi.nlm.nih.gov/34594039), which corresponds to the MeSH term [Erythrocyte Indices](https://meshb.nlm.nih.gov/record/ui?ui=D004909) through a direct synonym match. These associations highlight the potential role of PKN2 in both ocular and hematological traits.


The gene is expressed in various immune cell types, including CD34+, CD33+ myeloid cells, CD4+ T cells, and CD8+ T cells. Additionally, it is expressed in CD105+ endothelial cells, CD71+ early erythroid cells, thyroid tissue, fetal lung tissue, and the ovary.



The mouse ortholog gene 109333 corresponds to the human ortholog gene 5586. This gene is associated with the phenotype of decreased circulating unsaturated transferrin level, observed in a sex-specific manner.


The analyzed protein sequence has a GRAVY value of -0.503 (34.6th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -10.95 (21.02nd percentile), and the median structural flexibility is 1.007 (70.83rd percentile). The protein has a propensity for helix, sheet, and turn structures of 32.72% (57.02nd percentile), 34.04% (52.36th percentile), and 28.76% (54th percentile), respectively. The instability index is 49.4 (55.27th percentile), and the isoelectric point is 5.95 (29.19th percentile). The protein sequence is 984 amino acids long (87.2nd percentile) with a molecular weight of 112033.29 Da (87.7th percentile). For more details on sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |