---
layout: post
title: ERBB3
date: 2025-01-17 16:55 CST
description: ERBB3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2065) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2065  | ERBB3 | ENSG00000065361 | 12q13.2 |



The gene is involved in various biological processes, including [endocardial cushion development](https://amigo.geneontology.org/amigo/term/GO:0003197), [negative regulation of cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0007162), [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), and [transmembrane receptor protein tyrosine kinase signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007169). It enables [transmembrane signaling receptor activity](https://amigo.geneontology.org/amigo/term/GO:0004888), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524). The gene is located in the [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615) and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), and is active in the [basal plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009925). Additionally, it is involved in [multicellular organism development](https://amigo.geneontology.org/amigo/term/GO:0007275), [peripheral nervous system development](https://amigo.geneontology.org/amigo/term/GO:0007422), and [heart development](https://amigo.geneontology.org/amigo/term/GO:0007507). The gene also plays a role in the [positive regulation of cell population proliferation](https://amigo.geneontology.org/amigo/term/GO:0008284) and [negative regulation of signal transduction](https://amigo.geneontology.org/amigo/term/GO:0009968). Furthermore, it is involved in [Schwann cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0014037), [Schwann cell development](https://amigo.geneontology.org/amigo/term/GO:0014044), and [phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0016310). The gene is located in the [basolateral plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0016323), [apical plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0016324), and [lateral plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0016328). It enables [growth factor binding](https://amigo.geneontology.org/amigo/term/GO:0019838), [protein tyrosine kinase activator activity](https://amigo.geneontology.org/amigo/term/GO:0030296), and [ubiquitin protein ligase binding](https://amigo.geneontology.org/amigo/term/GO:0031625). The gene is also involved in the [ERBB2-ERBB3 signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0038133) and is part of the [ERBB3:ERBB2 complex](https://amigo.geneontology.org/amigo/term/GO:0038143) and [receptor complex](https://amigo.geneontology.org/amigo/term/GO:0043235). It plays roles in [wound healing](https://amigo.geneontology.org/amigo/term/GO:0042060), [regulation of cell population proliferation](https://amigo.geneontology.org/amigo/term/GO:0042127), and [myelination](https://amigo.geneontology.org/amigo/term/GO:0042552). The gene enables [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0043066) and [ErbB-3 class receptor binding](https://amigo.geneontology.org/amigo/term/GO:0043125), and is involved in the [negative regulation of apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0043066) and [phosphatidylinositol 3-kinase/protein kinase B signal transduction](https://amigo.geneontology.org/amigo/term/GO:0043491). Additionally, it plays roles in the [negative regulation of neuron apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0043524), [positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction](https://amigo.geneontology.org/amigo/term/GO:0051897), and [positive regulation of cardiac muscle tissue development](https://amigo.geneontology.org/amigo/term/GO:0055025). The gene is also involved in the [positive regulation of calcineurin-NFAT signaling cascade](https://amigo.geneontology.org/amigo/term/GO:0070886), [motor neuron apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0097049), and [extrinsic apoptotic signaling pathway in absence of ligand](https://amigo.geneontology.org/amigo/term/GO:0097192). Lastly, it is involved in the [negative regulation of motor neuron apoptotic process](https://amigo.geneontology.org/amigo/term/GO:2000672).


The gene length is 29,529.0 base pairs (52.27 percentile of all genes), the mature length is 1,776.0 base pairs, and the primary transcript length is 29,344.5 base pairs.


The gene ERBB3 (NCBI ID: 2065) has been mentioned in [843 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ERBB3%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning this gene is 1964, and the middle 50% of publications occurred between 2009 and 2018.


The top 5 publications mentioning ERBB3, ranked by their scientific influence, include "[MET amplification leads to gefitinib resistance in lung cancer by activating ERBB3 signaling.](https://pubmed.ncbi.nlm.nih.gov/17463250)" (2007) with a relative citation ratio of 76.46, "[Novel anticancer targets: revisiting ERBB2 and discovering ERBB3.](https://pubmed.ncbi.nlm.nih.gov/19536107)" (2009) with a relative citation ratio of 19.57, "[HER kinase inhibition in patients with HER2- and HER3-mutant cancers.](https://pubmed.ncbi.nlm.nih.gov/29420467)" (2018) with a relative citation ratio of 19.48, "[Efficacy and Safety of Patritumab Deruxtecan (HER3-DXd) in EGFR Inhibitor-Resistant, <i>EGFR</i>-Mutated Non-Small Cell Lung Cancer.](https://pubmed.ncbi.nlm.nih.gov/34548309)" (2022) with a relative citation ratio of 18.26, and "[Isolation and characterization of ERBB3, a third member of the ERBB/epidermal growth factor receptor family: evidence for overexpression in a subset of human mammary tumors.](https://pubmed.ncbi.nlm.nih.gov/2687875)" (1989) with a relative citation ratio of 15.56. The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ERBB3](https://www.proteinatlas.org/ENSG00000065361-ERBB3) is a kinase and receptor with tyrosine-protein kinase and transferase molecular functions. It is implicated in cancer-related genes and disease variants, with evidence at the protein level. ERBB3 is detected in many tissues and is primarily localized to the plasma membrane and actin filaments. It is notably expressed in the stomach, where it is involved in proteolysis (Cluster 48), and in the white matter of the brain, where it plays a role in signal transduction (Cluster 11). Additionally, ERBB3 is expressed in skin cancer cell lines, contributing to extracellular matrix organization (Cluster 69), and in intestinal goblet cells, where it is involved in mucin production (Cluster 16).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 6 experiments, [HDAC2](https://www.ncbi.nlm.nih.gov/gene/3066) with 5 experiments, [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) with 4 experiments, [E2F1](https://www.ncbi.nlm.nih.gov/gene/1869) with 4 experiments, and [TP53](https://www.ncbi.nlm.nih.gov/gene/7157) with 4 experiments. These TFs are notable for their regulatory roles as evidenced by the multiple experiments conducted.



The GWAS data indicates associations with a range of respiratory conditions, including asthma, bronchial disease, obstructive lung disease, lower respiratory tract disease, lung disease, and respiratory system disease. Additionally, the data highlights metabolic disorders such as disease of metabolism, acquired metabolic disease, carbohydrate metabolism disease, glucose metabolism disease, diabetes mellitus, nutrition disease, overnutrition, and obesity. Furthermore, associations with nervous system diseases, including central nervous system disease, neurodegenerative disease, demyelinating disease, and multiple sclerosis, are noted. Specific conditions like type 1 diabetes mellitus and polycystic ovary syndrome are also implicated.


ERBB3 has been associated with several diseases and traits through genome-wide association studies. Notably, it has been linked to [Autoimmune thyroid disease](https://pubmed.ncbi.nlm.nih.gov/32581359), which maps closely to the MeSH term [Autoimmunity](https://meshb.nlm.nih.gov/record/ui?ui=D015551) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method. Additionally, ERBB3 is associated with [Obesity-related traits](https://pubmed.ncbi.nlm.nih.gov/23251661), which aligns with the MeSH term [Obesity](https://meshb.nlm.nih.gov/record/ui?ui=D009765) through embedding match. Furthermore, ERBB3 has been directly linked to [Polycystic ovary syndrome](https://pubmed.ncbi.nlm.nih.gov/30566500), corresponding to the MeSH term [Polycystic Ovary Syndrome](https://meshb.nlm.nih.gov/record/ui?ui=D011085).


The gene is expressed in various tissues, including the medulla oblongata, pancreas, small intestine, kidney, dorsal root ganglion, and olfactory bulb. Additionally, it is expressed in nerve tissue.


The input data highlights various signaling pathways and immune system processes. Key pathways include downregulation of ERBB2:ERBB3 signaling, GRB7 and SHC1 events in ERBB2 signaling, and multiple PI3K/AKT signaling events in cancer and ERBB2/ERBB4 signaling. Additionally, the data covers adaptive and innate immune system processes, as well as signaling by various receptors such as EGFR, ERBB2, ERBB4, FGFR, and KIT. Other notable pathways include DAP12 signaling, downstream signal transduction, and NGF signaling via TRKA. The data also mentions disease and general signal transduction processes.



The analyzed protein sequence has a GRAVY value of -0.387 (47.68th percentile), indicating a hydrophilic nature. It exhibits a charge of -18.85 at pH 7.0 (11.93rd percentile) and a median structural flexibility of 1.0 (41.59th percentile). The protein's secondary structure is predicted to be 28.32% helix (19.0th percentile), 31.52% sheet (36.8th percentile), and 32.41% turn (78.91st percentile). The instability index is 49.61 (55.97th percentile), with an isoelectric point of 6.11 (33.3rd percentile). The protein has a length of 1342 amino acids (93.68th percentile) and a molecular weight of 148096.7 Da (93.59th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |