---
layout: post
title: CELSR2
date: 2025-01-17 16:55 CST
description: CELSR2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1952) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1952  | CELSR2 | ENSG00000143126 | 1p13.3 |



The gene is involved in various biological processes, including [cilium movement](https://amigo.geneontology.org/amigo/term/GO:0003341), [regulation of DNA-templated transcription](https://amigo.geneontology.org/amigo/term/GO:0006355), [homophilic cell adhesion via plasma membrane adhesion molecules](https://amigo.geneontology.org/amigo/term/GO:0007156), and [G protein-coupled receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007186). It enables [G protein-coupled receptor activity](https://amigo.geneontology.org/amigo/term/GO:0004930) and [calcium ion binding](https://amigo.geneontology.org/amigo/term/GO:0005509). The gene is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), and [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020). Additionally, it plays roles in [Wnt signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0016055), [ventricular system development](https://amigo.geneontology.org/amigo/term/GO:0021591), [neural plate anterior/posterior regionalization](https://amigo.geneontology.org/amigo/term/GO:0021999), [regulation of cell-cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0022407), [regulation of protein localization](https://amigo.geneontology.org/amigo/term/GO:0032880), [cerebrospinal fluid secretion](https://amigo.geneontology.org/amigo/term/GO:0033326), [dendrite morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0048813), [Wnt signaling pathway, planar cell polarity pathway](https://amigo.geneontology.org/amigo/term/GO:0060071), [cilium assembly](https://amigo.geneontology.org/amigo/term/GO:0060271), [motor neuron migration](https://amigo.geneontology.org/amigo/term/GO:0097475), and [cell-cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0098609).


The gene length is 11,546 base pairs (28.8% of all genes), the mature length is 2,064 base pairs (54.43% of all genes), and the primary transcript length is 9,408 base pairs (31.7% of all genes).


The gene CELSR2 (NCBI ID: 1952) has been mentioned in [13 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CELSR2%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 2008. The middle 50% of publications occurred between 2017 and 2021. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning CELSR2, ranked by their scientific influence, include studies that explore the gene's association with various conditions. The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Notable publications include "[CELSR2-PSRC1-SORT1 gene expression and association with coronary artery disease and plasma lipid levels in an Asian Indian cohort.](https://pubmed.ncbi.nlm.nih.gov/24674750)" (2014) (relative citation ratio: 1.51), "[The novel genetic variant predisposing to coronary artery disease in the region of the PSRC1 and CELSR2 genes on chromosome 1 associates with serum cholesterol.](https://pubmed.ncbi.nlm.nih.gov/18649068)" (2008) (relative citation ratio: 1.35), and "[Inactivating Celsr2 promotes motor axon fasciculation and regeneration in mouse and human.](https://pubmed.ncbi.nlm.nih.gov/34983065)" (2022) (relative citation ratio: 1.27). Additionally, "[Mutations in <i>KIAA1109</i>, <i>CACNA1C</i>, <i>BSN</i>, <i>AKAP13</i>, <i>CELSR2</i>, and <i>HELZ2</i> Are Associated With the Prognosis in Endometrial Cancer.](https://pubmed.ncbi.nlm.nih.gov/31787999)" (2019) (relative citation ratio: 1.16) and "[CELSR2, encoding a planar cell polarity protein, is a putative gene in Joubert syndrome with cortical heterotopia, microophthalmia, and growth hormone deficiency.](https://pubmed.ncbi.nlm.nih.gov/28052552)" (2017) (relative citation ratio: 0.85) are also significant. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CELSR2](https://www.proteinatlas.org/ENSG00000143126-CELSR2) is a developmental protein that functions as a G-protein coupled receptor and transducer. It is detected in many tissues and is primarily localized in vesicles and the cytosol. The gene is part of Cluster 80 in tissue expression, indicating non-specific involvement in endocytosis. In the brain, it is associated with Cluster 51, which includes neurons with mixed functions. Additionally, CELSR2 is found in Cluster 15 in kidney cancer cell lines, suggesting a role in membrane components, and in Cluster 12 in single-cell expression, indicating involvement in squamous epithelial cells and keratinization.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [SUZ12](https://www.ncbi.nlm.nih.gov/gene/23512) with 6 experiments, [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 5 experiments, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 5 experiments, [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599) with 5 experiments, and [AR](https://www.ncbi.nlm.nih.gov/gene/367) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent detection in multiple experiments.



The input data highlights a range of disease conditions associated with various biological processes and systems. Notably, it includes several types of cancer, such as lung cancer, respiratory system cancer, and lung carcinoma, with a specific mention of non-small cell lung carcinoma. Additionally, the data encompasses metabolic disorders, including diabetes mellitus, particularly type 2 diabetes mellitus, and conditions related to carbohydrate and glucose metabolism. Cardiovascular diseases are also prominent, with mentions of coronary artery disease, artery disease, vascular disease, and atherosclerosis. Furthermore, the data includes musculoskeletal system diseases, such as myopathy and muscular disease, as well as lipid metabolism disorders like cholesterol ester storage disease and cholesterol embolism.


CELSR2 has been associated with several traits and diseases through genome-wide association studies. Notably, it has been linked to [Apolipoprotein B levels](https://pubmed.ncbi.nlm.nih.gov/33462484), which maps closely to the MeSH term [Apolipoproteins](https://meshb.nlm.nih.gov/record/ui?ui=D001053) using the BioWordVec method. Additionally, CELSR2 is associated with [Glutathione S-transferase Mu 4 levels](https://pubmed.ncbi.nlm.nih.gov/34648354), mapped to [Glutathione](https://meshb.nlm.nih.gov/record/ui?ui=D005978), and [Granulins levels](https://pubmed.ncbi.nlm.nih.gov/34814699), mapped to [Granulins](https://meshb.nlm.nih.gov/record/ui?ui=D000077155). Furthermore, it is linked to [LDL cholesterol](https://pubmed.ncbi.nlm.nih.gov/30275531), which directly matches the MeSH term [Cholesterol, LDL](https://meshb.nlm.nih.gov/record/ui?ui=D008078). Other associations include [Low density lipoprotein cholesterol levels](https://pubmed.ncbi.nlm.nih.gov/33462484), mapped to [Lipoproteins](https://meshb.nlm.nih.gov/record/ui?ui=D008074), [Non-HDL cholesterol levels](https://pubmed.ncbi.nlm.nih.gov/34887591), and [Total cholesterol levels](https://pubmed.ncbi.nlm.nih.gov/33462484), both mapped to [Cholesterol](https://meshb.nlm.nih.gov/record/ui?ui=D002784).


The gene is expressed in various brain regions, including the prefrontal cortex, amygdala, whole brain, fetal brain, cingulate cortex, medulla oblongata, and occipital lobe.




The analyzed protein sequence has a GRAVY value of -0.271 (61.36th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -96.82 (0.56th percentile), and the median structural flexibility is 0.9998 (40.74th percentile). The protein has a length of 2923 amino acids (99.12th percentile) and a molecular weight of 317448.86 Da (99.1st percentile). The secondary structure affinities are 26.03% for helix (7.82nd percentile), 34.52% for sheet (55.55th percentile), and 34.28% for turn (87.05th percentile). The instability index is 46.39 (44.41st percentile), and the isoelectric point is 5.14 (10.37th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |