---
layout: post
title: DDT
date: 2025-01-17 16:55 CST
description: DDT description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1652) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1652  | DDT | ENSG00000099977 | 22q11.23 |



The gene enables [dopachrome isomerase activity](https://amigo.geneontology.org/amigo/term/GO:0004167) and [cytokine receptor binding](https://amigo.geneontology.org/amigo/term/GO:0005126), and is active in the [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615). It is also located in the [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615) and [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737). The gene is involved in the [negative regulation of macrophage chemotaxis](https://amigo.geneontology.org/amigo/term/GO:0010760), [positive regulation of tumor necrosis factor production](https://amigo.geneontology.org/amigo/term/GO:0032760), [melanin biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0042438), [positive regulation of inflammatory response](https://amigo.geneontology.org/amigo/term/GO:0050729), and [positive regulation of ERK1 and ERK2 cascade](https://amigo.geneontology.org/amigo/term/GO:0070374). Additionally, it enables [D-dopachrome decarboxylase activity](https://amigo.geneontology.org/amigo/term/GO:0033981) and [phenylpyruvate tautomerase activity](https://amigo.geneontology.org/amigo/term/GO:0050178), and is located in the [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062).


The gene length is 192,371 base pairs (92.02% of all genes), the mature length is 2,176 base pairs (57.06% of all genes), and the primary transcript length is 191,851 base pairs (95.5% of all genes).


The gene DDT (NCBI ID: 1652) has been mentioned in [154 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22DDT%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1945, and the middle 50% of publications range from 1970 to 2014.


The top five publications mentioning DDT, ranked by their scientific influence, include "[Unusual persistence of DDT in some Western USA soils.](https://pubmed.ncbi.nlm.nih.gov/1536998)" (1992) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 10.86, "[The Pine River statement: human health consequences of DDT use.](https://pubmed.ncbi.nlm.nih.gov/19750098)" (2009) with an RCR of 6.75, "[Carcinogenicity of lindane, DDT, and 2,4-dichlorophenoxyacetic acid.](https://pubmed.ncbi.nlm.nih.gov/26111929)" (2015) with an RCR of 6.06, "[Estrogenic activity of o,p'-DDT in the mammalian uterus and avian oviduct.](https://pubmed.ncbi.nlm.nih.gov/5677532)" (1968) with an RCR of 4.59, and "[Cross-sectional study of a community with exceptional exposure to DDT.](https://pubmed.ncbi.nlm.nih.gov/6971946)" (1981) with an RCR of 4.35. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[DDT](https://www.proteinatlas.org/ENSG00000099977-DDT) is a gene encoding a protein with lyase molecular function, supported by evidence at the protein level. The gene is expressed in all tissues, with notable expression in the liver, where it is involved in metabolism. Subcellularly, the protein is localized to the mitochondria. In single-cell analysis, DDT is particularly expressed in hepatocytes, further emphasizing its role in metabolism. The gene's expression in various cancer types is unprognostic, indicating no significant correlation with patient outcomes.


The top transcription factors (TFs) identified from the GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [EGR3](https://www.ncbi.nlm.nih.gov/gene/1960), [TEAD4](https://www.ncbi.nlm.nih.gov/gene/7004), [ZNF580](https://www.ncbi.nlm.nih.gov/gene/51157), [TP53](https://www.ncbi.nlm.nih.gov/gene/7157), and [HDAC1](https://www.ncbi.nlm.nih.gov/gene/3065), each shown to be regulating in one experiment.





The gene is expressed in various tissues, including the liver and kidney, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Lymphoma Burkitts (Raji), Heart, Liver, 721 B Lymphoblasts, Lymphoma Burkitts (Daudi), CD71+ Early Erythroid, and Prostate.




The protein sequence analyzed has a GRAVY value of 0.231 (91.09th percentile), indicating a relatively hydrophobic nature. At pH 7.0, the protein carries a charge of -0.33 (48.96th percentile). The median structural flexibility is 0.997 (27.66th percentile), with a propensity for helix, sheet, and turn structures of 35.6% (78.42nd percentile), 37.3% (74.65th percentile), and 28.0% (46.95th percentile), respectively. The instability index is 35.6 (11.4th percentile), and the isoelectric point is 6.72 (46.6th percentile). The protein has a length of 118 amino acids (4.48th percentile) and a molecular weight of 12711.6 Da (4.07th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |