---
layout: post
title: FCER1G
date: 2025-01-17 16:55 CST
description: FCER1G description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2207) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2207  | FCER1G | ENSG00000158869 | 1q23.3  |



The gene is located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886) and is active on the [external side of the plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009897). It is also found on the [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986) and in the [tertiary granule membrane](https://amigo.geneontology.org/amigo/term/GO:0070821) and [ficolin-1-rich granule membrane](https://amigo.geneontology.org/amigo/term/GO:0101003). The gene is involved in various immune responses, including [neutrophil activation involved in immune response](https://amigo.geneontology.org/amigo/term/GO:0002283), [T cell differentiation involved in immune response](https://amigo.geneontology.org/amigo/term/GO:0002292), [Fc receptor mediated stimulatory signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0002431), [immunoglobulin mediated immune response](https://amigo.geneontology.org/amigo/term/GO:0016064), and [innate immune response](https://amigo.geneontology.org/amigo/term/GO:0045087). It enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [IgE receptor activity](https://amigo.geneontology.org/amigo/term/GO:0019767), [IgE binding](https://amigo.geneontology.org/amigo/term/GO:0019863), [IgG binding](https://amigo.geneontology.org/amigo/term/GO:0019864), [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802), and [protein homodimerization activity](https://amigo.geneontology.org/amigo/term/GO:0042803). Additionally, the gene plays a role in [antigen processing and presentation of exogenous peptide antigen via MHC class II](https://amigo.geneontology.org/amigo/term/GO:0019886) and [MHC class I](https://amigo.geneontology.org/amigo/term/GO:0042590), [neutrophil chemotaxis](https://amigo.geneontology.org/amigo/term/GO:0030593), [receptor internalization](https://amigo.geneontology.org/amigo/term/GO:0031623), [positive regulation of interleukin-4 production](https://amigo.geneontology.org/amigo/term/GO:0032753), [Fc-gamma receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0038094), [Fc-epsilon receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0038095), [interleukin-3-mediated signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0038156), [defense response to bacterium](https://amigo.geneontology.org/amigo/term/GO:0042742), [positive regulation of phagocytosis](https://amigo.geneontology.org/amigo/term/GO:0050766), and [cellular response to low-density lipoprotein particle stimulus](https://amigo.geneontology.org/amigo/term/GO:0071404). The gene is part of the [Fc-epsilon receptor I complex](https://amigo.geneontology.org/amigo/term/GO:0032998) and [Fc-gamma receptor III complex](https://amigo.geneontology.org/amigo/term/GO:0033001).


The gene length is 60,349 base pairs (71.13% of all genes), the mature length is 1,395 base pairs (34.89% of all genes), and the primary transcript length is 40,504 base pairs (70.27% of all genes).


The gene FCER1G (NCBI ID: 2207) has been mentioned in [104 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22FCER1G%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1982 and the middle 50% of publications occurring between 1999 and 2016. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning FCER1G, ranked by their scientific influence, include "[FcRgamma activation regulates inflammation-associated squamous carcinogenesis.](https://pubmed.ncbi.nlm.nih.gov/20138013)" (2010) (relative citation ratio: 8.39), "[The Fc(epsilon)RIbeta subunit functions as an amplifier of Fc(epsilon)RIgamma-mediated cell activation signals.](https://pubmed.ncbi.nlm.nih.gov/8674126)" (1996) (relative citation ratio: 5.44), "[Peripheral blood dendritic cells express Fc epsilon RI as a complex composed of Fc epsilon RI alpha- and Fc epsilon RI gamma-chains and can use this receptor for IgE-mediated allergen presentation.](https://pubmed.ncbi.nlm.nih.gov/8752908)" (1996) (relative citation ratio: 5.22), "[Glycoprotein VI is the collagen receptor in platelets which underlies tyrosine phosphorylation of the Fc receptor gamma-chain.](https://pubmed.ncbi.nlm.nih.gov/9280292)" (1997) (relative citation ratio: 4.74), and "[Plasmacytoid dendritic cell-specific receptor ILT7-Fc epsilonRI gamma inhibits Toll-like receptor-induced interferon production.](https://pubmed.ncbi.nlm.nih.gov/16735691)" (2006) (relative citation ratio: 3.54). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[FCER1G](https://www.proteinatlas.org/ENSG00000158869-FCER1G) is an IgE-binding protein and receptor with evidence at the protein level. It is a known FDA-approved drug target and is detected in all tissues. The gene is highly expressed in lymphoid tissue and bone marrow, where it plays a role in the innate immune response. Additionally, it is expressed in macrophages and microglia in the brain, contributing to the immune response. In cell lines, FCER1G is associated with lymphoma and inflammatory responses, while in single cells, it is found in monocytes involved in the innate immune response. The gene is part of the non-specific membrane trafficking cluster in blood expression.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664) with 4 experiments, [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) with 3 experiments, [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) with 2 experiments, [CPSF3](https://www.ncbi.nlm.nih.gov/gene/51692) with 2 experiments, and [RXRA](https://www.ncbi.nlm.nih.gov/gene/6256) with 2 experiments.


The top transcription factors (TFs) identified in Perturb-seq experiments include [BCL6](https://www.ncbi.nlm.nih.gov/gene/5885) and [KLF12](https://www.ncbi.nlm.nih.gov/gene/8861), each shown to regulate genes in two experiments. Additionally, [ZNF263](https://www.ncbi.nlm.nih.gov/gene/23028), [GATA2](https://www.ncbi.nlm.nih.gov/gene/2623), [ZNF274](https://www.ncbi.nlm.nih.gov/gene/8243), and [ZNF91](https://www.ncbi.nlm.nih.gov/gene/9126) were each found to regulate genes in one experiment. These findings highlight the regulatory roles of these TFs as identified through uniform ChIP-seq data from the GTRD database and Perturb-seq experiments.


The input data indicates associations with various metabolic disorders, including diseases of metabolism, acquired metabolic diseases, carbohydrate metabolism diseases, glucose metabolism diseases, diabetes mellitus, and specifically type 2 diabetes mellitus.


The gene FCER1G has been associated with [Phospholipid levels in very large HDL](https://pubmed.ncbi.nlm.nih.gov/27005778), which maps closely to the MeSH term [Lipids](https://meshb.nlm.nih.gov/record/ui?ui=D008055) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method.


The gene is expressed in multiple tissue samples, including lung, blood, and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in various cell types and tissues such as CD33+ Myeloid cells, BDCA4+ Dendritic cells, CD56+ NK cells, CD14+ Monocytes, Whole Blood, Lung, and Bone marrow.


The input data highlights several pathways involved in the immune system, particularly the innate immune system. Key processes include fc epsilon receptor (fcÎµri) signaling and the gpvi-mediated activation cascade. Additionally, the data emphasizes cell surface interactions at the vascular wall, including platelet adhesion to exposed collagen, hemostasis, and platelet activation, signaling, and aggregation.


The mouse ortholog gene 14127 and its human ortholog gene 2207 are associated with several phenotypes, including decreased mean corpuscular hemoglobin, decreased mean corpuscular volume, and preweaning lethality with incomplete penetrance. These phenotypes are observed in both sexes.


The protein sequence analyzed has a GRAVY value of 0.366 (93.17th percentile), indicating a relatively hydrophobic nature. The charge at pH 7.0 is -0.424 (48.53rd percentile), and the median structural flexibility is 0.986 (8.27th percentile). The protein has a high affinity for helix (0.419, 95.88th percentile) and sheet (0.488, 97.64th percentile) structures, with a lower affinity for turns (0.163, 0.37th percentile). The instability index is 17.105 (0.37th percentile), suggesting a stable protein. The isoelectric point is 6.544 (43.47th percentile), with a molecular weight of 9667.35 Da (1.63rd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |