---
layout: post
title: HLA-DMA
date: 2025-01-17 16:55 CST
description: HLA-DMA description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3108) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3108  | HLA-DMA | ENSG00000204257 | 6p21.32 |



The gene is involved in several biological processes, including the [adaptive immune response](https://amigo.geneontology.org/amigo/term/GO:0002250) and [antigen processing and presentation of exogenous peptide antigen via MHC class II](https://amigo.geneontology.org/amigo/term/GO:0019886). It plays a role in [peptide antigen assembly with MHC class II protein complex](https://amigo.geneontology.org/amigo/term/GO:0002503) and the [positive regulation of immune response](https://amigo.geneontology.org/amigo/term/GO:0050778), as well as the [positive regulation of T cell activation](https://amigo.geneontology.org/amigo/term/GO:0050870). The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [MHC class II protein complex binding](https://amigo.geneontology.org/amigo/term/GO:0023026), and [peptide antigen binding](https://amigo.geneontology.org/amigo/term/GO:0042605). It is part of the [MHC class II protein complex](https://amigo.geneontology.org/amigo/term/GO:0042613) and is active in the [lysosomal membrane](https://amigo.geneontology.org/amigo/term/GO:0005765) and [late endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0031902). Additionally, the gene is located in the [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), and [intracellular membrane-bounded organelle](https://amigo.geneontology.org/amigo/term/GO:0043231).


The gene length is 906,136.0 base pairs (99.52% of all genes), the mature length is 3,164.5 base pairs, and the primary transcript length is 906,106.0 base pairs.


The gene HLA-DMA (NCBI ID: 3108) has been mentioned in [56 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22HLA-DMA%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1977, and the middle 50% of publications occurred between 1997 and 2012.


The top five publications mentioning HLA-DMA, ranked by their scientific influence, include "[An essential role for HLA-DM in antigen presentation by class II major histocompatibility molecules.](https://pubmed.ncbi.nlm.nih.gov/8139689)" (1994) (relative citation ratio: 6.84), "[Editing of the HLA-DR-peptide repertoire by HLA-DM.](https://pubmed.ncbi.nlm.nih.gov/8947036)" (1996) (relative citation ratio: 4.1), "[A lysosomal targeting signal in the cytoplasmic tail of the beta chain directs HLA-DM to MHC class II compartments.](https://pubmed.ncbi.nlm.nih.gov/7593164)" (1995) (relative citation ratio: 3.05), "[Crystal structure of the HLA-DM-HLA-DR1 complex defines mechanisms for rapid peptide selection.](https://pubmed.ncbi.nlm.nih.gov/23260142)" (2012) (relative citation ratio: 3.0), and "[HLA-DM acts as a molecular chaperone and rescues empty HLA-DR molecules at lysosomal pH.](https://pubmed.ncbi.nlm.nih.gov/9075930)" (1997) (relative citation ratio: 2.59). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[HLA-DMA](https://www.proteinatlas.org/ENSG00000204257-HLA-DMA) is a gene with evidence at the protein level, detected in many tissues. It is primarily localized in vesicles and is associated with immune response functions in various tissues and cell types. In the blood, it is expressed in dendritic cells with unknown function, while in tissues, it is predominantly found in the spleen. In the brain, it is expressed in macrophages and microglia, and in cell lines, it is associated with lymphoma and humoral immune response. Single-cell expression data indicate its role in dendritic cells involved in antigen presentation. The gene is unprognostic for multiple cancer types, including bladder urothelial carcinoma, breast invasive carcinoma, and cervical squamous cell carcinoma, among others.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861) with 6 experiments, [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170) with 5 experiments, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 5 experiments, [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227) with 5 experiments, and [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with a range of disease conditions, including infectious diseases such as those caused by viral agents, specifically human immunodeficiency virus infectious disease. Additionally, the data highlights nervous system diseases, particularly central nervous system diseases and neurodegenerative diseases. Musculoskeletal system diseases are also prominent, encompassing bone inflammation diseases like arthritis and rheumatoid arthritis, as well as bone diseases and connective tissue diseases. Furthermore, demyelinating diseases, specifically multiple sclerosis, are noted in the dataset.


The gene HLA-DMA has been associated with [Educational attainment](https://pubmed.ncbi.nlm.nih.gov/35361970), which maps closely to the MeSH term [Educational Status](https://meshb.nlm.nih.gov/record/ui?ui=D004522) using the method [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0).


The gene is expressed in various tissues and cell types, including lung, spleen, Lymphoma Burkitt's (Raji), CD33+ Myeloid cells, BDCA4+ Dendritic cells, CD19+ B cells (neg. sel.), CD14+ Monocytes, Lung, Tonsil, and Lymph node. This indicates a ubiquitous expression pattern across multiple tissue samples.


The proteins are involved in several key immune-related pathways, including the adaptive immune system, the broader immune system, and MHC class II antigen presentation.







| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |