---
layout: post
title: PKM
date: 2025-01-17 16:55 CST
description: PKM description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5315) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5315  | PKM | ENSG00000067225 | 15q23 |



The gene enables [magnesium ion binding](https://amigo.geneontology.org/amigo/term/GO:0000287), [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [mRNA binding](https://amigo.geneontology.org/amigo/term/GO:0003729), [protein tyrosine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004713), and [pyruvate kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004743). It is involved in [glycolytic process](https://amigo.geneontology.org/amigo/term/GO:0006096), [programmed cell death](https://amigo.geneontology.org/amigo/term/GO:0012501), [phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0016310), [cellular response to insulin stimulus](https://amigo.geneontology.org/amigo/term/GO:0032869), [canonical glycolysis](https://amigo.geneontology.org/amigo/term/GO:0061621), and [positive regulation of sprouting angiogenesis](https://amigo.geneontology.org/amigo/term/GO:1903672). The gene is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [mitochondrion](https://amigo.geneontology.org/amigo/term/GO:0005739), [rough endoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0005791), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [cilium](https://amigo.geneontology.org/amigo/term/GO:0005929), [vesicle](https://amigo.geneontology.org/amigo/term/GO:0031982), [secretory granule lumen](https://amigo.geneontology.org/amigo/term/GO:0034774), [collagen-containing extracellular matrix](https://amigo.geneontology.org/amigo/term/GO:0062023), [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062), [extracellular vesicle](https://amigo.geneontology.org/amigo/term/GO:1903561), and [ficolin-1-rich granule lumen](https://amigo.geneontology.org/amigo/term/GO:1904813). Additionally, it enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524), [MHC class II protein complex binding](https://amigo.geneontology.org/amigo/term/GO:0023026), [potassium ion binding](https://amigo.geneontology.org/amigo/term/GO:0030955), and [cadherin binding](https://amigo.geneontology.org/amigo/term/GO:0045296). The gene is also involved in [positive regulation of cytoplasmic translation](https://amigo.geneontology.org/amigo/term/GO:2000767).


The gene length is 25,543 base pairs (48.23% of all genes), the mature length is 585 base pairs (4.73% of all genes), and the primary transcript length is 15,050 base pairs (44.24% of all genes).


The gene PKM (NCBI ID: 5315) has been mentioned in [594 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PKM%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning PKM is 1974, and the middle 50% of publications occurred between 2015 and 2020.


The top 5 publications mentioning PKM, ranked by their scientific influence, include "[Exosome-delivered circRNA promotes glycolysis to induce chemoresistance through the miR-122-PKM2 axis in colorectal cancer.](https://pubmed.ncbi.nlm.nih.gov/31901148)" (2020) (relative citation ratio: 23.87), "[Nuclear PKM2 regulates Î²-catenin transactivation upon EGFR activation.](https://pubmed.ncbi.nlm.nih.gov/22056988)" (2011) (relative citation ratio: 17.91), "[Pyruvate Kinase M2 and Cancer: The Role of PKM2 in Promoting Tumorigenesis.](https://pubmed.ncbi.nlm.nih.gov/32195169)" (2020) (relative citation ratio: 17.78), "[ERK1/2-dependent phosphorylation and nuclear translocation of PKM2 promotes the Warburg effect.](https://pubmed.ncbi.nlm.nih.gov/23178880)" (2012) (relative citation ratio: 15.83), and "[The glycolytic enzyme PKM2 bridges metabolic and inflammatory dysfunction in coronary artery disease.](https://pubmed.ncbi.nlm.nih.gov/26926996)" (2016) (relative citation ratio: 12.6). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PKM](https://www.proteinatlas.org/ENSG00000067225-PKM) is an allosteric enzyme with kinase and transferase molecular functions. It is involved in cancer-related genes and has evidence at the protein level. The gene is detected in all RNA tissue distributions and is localized in vesicles, cytosol, and the principal piece. PKM is part of Cluster 26 in blood expression, which includes monocytes and plasma membrane proteins. In tissue expression, it belongs to Cluster 14, associated with skeletal muscle and muscle contraction. The gene is also found in Cluster 45 for both brain and single cell expression, indicating non-specific mixed functions. Additionally, PKM is part of Cluster 15 in cell line expression, related to kidney cancer and membrane components.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ELF1](https://www.ncbi.nlm.nih.gov/gene/1997) and [STAT3](https://www.ncbi.nlm.nih.gov/gene/6774), each with 6 experiments. Additionally, [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599), [SMAD3](https://www.ncbi.nlm.nih.gov/gene/4088), and [SIN3A](https://www.ncbi.nlm.nih.gov/gene/25942) were each found to be regulating in 5 experiments.


The transcription factor [ATF4](https://www.ncbi.nlm.nih.gov/gene/23028) was found to be regulating genes in 4 Perturb-seq experiments, making it one of the top transcription factors identified in the study.


The GWAS data indicates associations with various disease conditions, including disease of anatomical entity, bone inflammation disease, arthritis, rheumatoid arthritis, bone disease, connective tissue disease, and musculoskeletal system disease.



The gene is expressed in various tissues, including muscle, adrenal gland, and blood, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in pineal gland (both day and night), thalamus, colorectal adenocarcinoma, and skeletal muscle.


The input data highlights several pathways related to carbohydrate metabolism, including general metabolism, glucose metabolism, and glycolysis. Additionally, it includes pathways associated with specific diseases such as myoclonic epilepsy of Lafora, glycogen storage diseases, and general disease processes.


The mouse ortholog gene 18746 and its human ortholog gene 5315 are associated with several phenotypic abnormalities. These include abnormal embryo size, embryonic growth retardation, and preweaning lethality with complete penetrance, all of which affect both sexes.


The analyzed protein sequence has a GRAVY value of -0.128, indicating a hydrophilic nature (76.01% percentile). It exhibits a charge of 2.54 at pH 7.0 (59.46% percentile) and a median structural flexibility of 0.999 (38.73% percentile). The protein's secondary structure is predicted to be 34.27% helix (69.97% percentile), 33.90% sheet (51.24% percentile), and 26.18% turn (30.82% percentile). The instability index is 27.03 (2.2% percentile), with an isoelectric point of 7.96 (58.35% percentile). The protein has a length of 531 amino acids (62.12% percentile) and a molecular weight of 57936.24 Da (60.64% percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |