---
layout: post
title: DUSP4
date: 2025-01-17 16:55 CST
description: DUSP4 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1846) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1846  | DUSP4 | ENSG00000120875 | 8p12 |



The gene is involved in [endoderm formation](https://amigo.geneontology.org/amigo/term/GO:0001706) and [dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0016311), including [peptidyl-tyrosine dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0035335). It enables various molecular functions such as [protein tyrosine phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0004725), [protein tyrosine/threonine phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0008330), [phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0016791), [MAP kinase tyrosine/serine/threonine phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0017017), [myosin phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0017018), [MAP kinase tyrosine phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0033550), and [MAP kinase serine/threonine phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:1990439). Additionally, it is involved in the [negative regulation of MAPK cascade](https://amigo.geneontology.org/amigo/term/GO:0043409) and [negative regulation of ERK1 and ERK2 cascade](https://amigo.geneontology.org/amigo/term/GO:0070373). The gene is active in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), and is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654). It also enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515).


The gene length is 191,476 base pairs (91.97th percentile of all genes), the mature length is 4,193 base pairs (85.34th percentile), and the primary transcript length is 191,476 base pairs (95.47th percentile).


The gene DUSP4 (NCBI ID: 1846) has been mentioned in [88 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22DUSP4%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1977 and the middle 50% of publications occurring between 2013 and 2020. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning DUSP4, ranked by their scientific influence, include "[The mitogen-activated protein kinase phosphatases PAC1, MKP-1, and MKP-2 have unique substrate specificities and reduced activity in vivo toward the ERK2 sevenmaker mutation.](https://pubmed.ncbi.nlm.nih.gov/8626452)" (1996) (relative citation ratio: 7.01), "[Profiling of residual breast cancers after neoadjuvant chemotherapy identifies DUSP4 deficiency as a mechanism of drug resistance.](https://pubmed.ncbi.nlm.nih.gov/22683778)" (2012) (relative citation ratio: 4.7), "[MiR-122-5p inhibits cell migration and invasion in gastric cancer by down-regulating DUSP4.](https://pubmed.ncbi.nlm.nih.gov/29509059)" (2018) (relative citation ratio: 3.84), "[An integrated genomic analysis of lung cancer reveals loss of DUSP4 in EGFR-mutant tumors.](https://pubmed.ncbi.nlm.nih.gov/19525976)" (2009) (relative citation ratio: 3.71), and "[Mutated KRAS results in overexpression of DUSP4, a MAP-kinase phosphatase, and SMYD3, a histone methyltransferase, in rectal carcinomas.](https://pubmed.ncbi.nlm.nih.gov/20725992)" (2010) (relative citation ratio: 3.57). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[DUSP4](https://www.proteinatlas.org/ENSG00000120875-DUSP4) is a gene encoding a protein with hydrolase and protein phosphatase molecular functions. It is implicated in cancer-related processes and has evidence at the protein level. The gene is detected in many tissues, with notable expression in the nucleoplasm. Its expression is associated with various clusters, including T-regs involved in cell cycle regulation in blood, proteolysis in the pancreas, mixed functions in the forebrain, ECM organization in skin cancer cell lines, and lipid metabolism in basal prostatic cells.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), [YY1](https://www.ncbi.nlm.nih.gov/gene/7528), and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), each with 5 experiments. Additionally, [AR](https://www.ncbi.nlm.nih.gov/gene/367) and [REST](https://www.ncbi.nlm.nih.gov/gene/5978) were found to be regulating in 4 experiments each.





The gene is expressed in various tissues, including breast and stomach, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in pineal gland (night), retina, adipocytes, bronchial epithelial cells, lung, placenta, pancreas, and pancreatic islets.


The input data highlights various pathways involved in the immune system, particularly the innate immune system. Key pathways include NGF signaling via TRKA from the plasma membrane, signaling by NGF, and multiple toll-like receptor (TLR) cascades such as TLR4, TLR3, TLR10, TLR5, TLR7/8, and TLR9. These pathways involve MyD88-dependent and MyD88-independent cascades, TRIF-mediated signaling, and TRAF6-mediated induction of proinflammatory cytokines. Additionally, the data covers MAP kinase activation, ERK/MAPK targets, and nuclear events related to kinase and transcription factor activation. Overall, the data emphasizes signal transduction processes within the immune system.


The mouse ortholog gene 319520 and its human ortholog gene 1846 are associated with phenotypes including decreased circulating glucose level and decreased circulating potassium level, observed in both sexes.


The protein sequence analyzed has a GRAVY value of -0.1297 (75.83rd percentile), indicating a relatively hydrophilic nature. At pH 7.0, the protein carries a charge of 0.291 (51.33rd percentile). The median structural flexibility is 0.998 (32.1st percentile), with a notable instability index of 61.32 (85.33rd percentile). The protein's isoelectric point is 7.10 (51.48th percentile), and it has a molecular weight of 42952.7 Da (43.42nd percentile). In terms of secondary structure, the protein has a 30.96% affinity for helix (40.65th percentile), 31.98% for sheet (39.37th percentile), and 30.96% for turn (70.35th percentile). These values were calculated using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |