---
layout: post
title: ACTG1
date: 2025-01-17 16:55 CST
description: ACTG1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/71) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 71  | ACTG1 | ENSG00000184009 | 17q25.3 |



The gene is involved in various biological processes, including [angiogenesis](https://amigo.geneontology.org/amigo/term/GO:0001525), [morphogenesis of a polarized epithelium](https://amigo.geneontology.org/amigo/term/GO:0001738), [axonogenesis](https://amigo.geneontology.org/amigo/term/GO:0007409), and [positive regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0010628). It enables several molecular functions such as [structural constituent of cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0005200), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [profilin binding](https://amigo.geneontology.org/amigo/term/GO:0005522), and [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524). The gene is located in multiple cellular components, including the [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615), [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0005856), [actin filament](https://amigo.geneontology.org/amigo/term/GO:0005884), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [cell-cell junction](https://amigo.geneontology.org/amigo/term/GO:0005911), [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [myofibril](https://amigo.geneontology.org/amigo/term/GO:0030016), [axon](https://amigo.geneontology.org/amigo/term/GO:0030424), [filamentous actin](https://amigo.geneontology.org/amigo/term/GO:0031941), [NuA4 histone acetyltransferase complex](https://amigo.geneontology.org/amigo/term/GO:0035267), [apical junction complex](https://amigo.geneontology.org/amigo/term/GO:0043296), [calyx of Held](https://amigo.geneontology.org/amigo/term/GO:0044305), [synapse](https://amigo.geneontology.org/amigo/term/GO:0045202), [phagocytic vesicle](https://amigo.geneontology.org/amigo/term/GO:0045335), [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062), [blood microparticle](https://amigo.geneontology.org/amigo/term/GO:0072562), [dense body](https://amigo.geneontology.org/amigo/term/GO:0097433), [Schaffer collateral - CA1 synapse](https://amigo.geneontology.org/amigo/term/GO:0098685), and [basal body patch](https://amigo.geneontology.org/amigo/term/GO:0120220). Additionally, it is involved in processes such as [positive regulation of cell migration](https://amigo.geneontology.org/amigo/term/GO:0030335), [maintenance of blood-brain barrier](https://amigo.geneontology.org/amigo/term/GO:0035633), [sarcomere organization](https://amigo.geneontology.org/amigo/term/GO:0045214), [cell motility](https://amigo.geneontology.org/amigo/term/GO:0048870), [regulation of stress fiber assembly](https://amigo.geneontology.org/amigo/term/GO:0051492), [regulation of focal adhesion assembly](https://amigo.geneontology.org/amigo/term/GO:0051893), [platelet aggregation](https://amigo.geneontology.org/amigo/term/GO:0070527), [cellular response to type II interferon](https://amigo.geneontology.org/amigo/term/GO:0071346), [positive regulation of wound healing](https://amigo.geneontology.org/amigo/term/GO:0090303), [postsynaptic actin cytoskeleton organization](https://amigo.geneontology.org/amigo/term/GO:0098974), [tight junction assembly](https://amigo.geneontology.org/amigo/term/GO:0120192), [regulation of transepithelial transport](https://amigo.geneontology.org/amigo/term/GO:0150111), [regulation of synaptic vesicle endocytosis](https://amigo.geneontology.org/amigo/term/GO:1900242), and [protein localization to bicellular tight junction](https://amigo.geneontology.org/amigo/term/GO:1902396).


The gene length is 101,069 base pairs (81.96% of all genes), the mature length is 1,801 base pairs, and the primary transcript length is 36,703 base pairs (67.92% of all genes).


The gene ACTG1 (NCBI ID: 71) has been mentioned in [110 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ACTG1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning this gene is 1992, and the middle 50% of publications occurred between 2001 and 2016.


The top 5 publications mentioning ACTG1, ranked by their scientific influence as measured by the [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541), include: ["Bone mineral density and fractures in antiretroviral-naive persons randomized to receive abacavir-lamivudine or tenofovir disoproxil fumarate-emtricitabine along with efavirenz or atazanavir-ritonavir: Aids Clinical Trials Group A5224s, a substudy of ACTG A5202."](https://pubmed.ncbi.nlm.nih.gov/21606537) (2011) (relative citation ratio: 13.12), ["Pharmacokinetic interactions between protease inhibitors and statins in HIV seronegative volunteers: ACTG Study A5047."](https://pubmed.ncbi.nlm.nih.gov/11873000) (2002) (relative citation ratio: 6.03), ["Standardization of sensitive human immunodeficiency virus coculture procedures and establishment of a multicenter quality assurance program for the AIDS Clinical Trials Group. The NIH/NIAID/DAIDS/ACTG Virology Laboratories."](https://pubmed.ncbi.nlm.nih.gov/1629336) (1992) (relative citation ratio: 5.86), ["Body Composition Changes After Initiation of Raltegravir or Protease Inhibitors: ACTG A5260s."](https://pubmed.ncbi.nlm.nih.gov/26797215) (2016) (relative citation ratio: 5.72), and ["De novo mutations in the actin genes ACTB and ACTG1 cause Baraitser-Winter syndrome."](https://pubmed.ncbi.nlm.nih.gov/22366783) (2012) (relative citation ratio: 4.74). The relative citation ratio measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ACTG1](https://www.proteinatlas.org/ENSG00000184009-ACTG1) is a gene encoding a protein with hydrolase molecular function. It is implicated in various diseases, including deafness, intellectual disability, and non-syndromic deafness. The gene's expression is detected in all RNA tissue distributions. In terms of tissue expression, ACTG1 is associated with squamous epithelium and innate immune response. Its blood expression is linked to vesicular transport, while its brain expression is non-specific with mixed functions. In cell lines, ACTG1 is expressed in breast cancer with an unknown function. Single-cell expression data also indicate a non-specific, mixed function.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599) with 6 experiments, [SMARCB1](https://www.ncbi.nlm.nih.gov/gene/6598) with 5 experiments, [RELA](https://www.ncbi.nlm.nih.gov/gene/5970) with 5 experiments, [GATA3](https://www.ncbi.nlm.nih.gov/gene/2625) with 5 experiments, and [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.


The transcription factor [NFKB1](https://www.ncbi.nlm.nih.gov/gene/2623) was found to be regulating genes in 2 experiments, according to Perturb-seq data from a study published in PubMed (PMID: 35688146). This data was further supported by GTRD's database of transcription factor binding sites derived from uniform ChIP-seq data.


The input data indicates associations with various disease conditions, including infectious diseases such as anthrax, neurodegenerative disorders like Parkinson's disease, and broader categories such as nervous system diseases and central nervous system diseases. Additionally, the data highlights synucleinopathies, which are a subset of neurodegenerative diseases characterized by the abnormal accumulation of alpha-synuclein protein.


The ACTG1 gene has been associated with several traits and diseases. Notably, it has been linked to [Hair color](https://pubmed.ncbi.nlm.nih.gov/30595370), which maps directly to the MeSH term [Hair Color](https://meshb.nlm.nih.gov/record/ui?ui=D006200). Additionally, ACTG1 has been associated with [Lung function (FVC)](https://pubmed.ncbi.nlm.nih.gov/30595370), which was mapped to the MeSH term [Lung](https://meshb.nlm.nih.gov/record/ui?ui=D008168) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method.


The gene is expressed in various tissues, including the ovary and salivary gland, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in cardiac myocytes, smooth muscle, thymus, and testis.


The input data highlights several key biological pathways, including those involved in the immune system, particularly the innate immune system, and processes such as axon guidance and developmental biology. Notably, EPH-Ephrin signaling, specifically EPHB-mediated forward signaling, is mentioned. Additionally, the data includes Fcgamma receptor (FCGR) dependent phagocytosis and the regulation of actin dynamics for phagocytic cup formation.


The mouse ortholog gene 11465 and its human ortholog gene 71 are associated with several phenotypic abnormalities, including abnormal craniofacial morphology, edema, embryonic growth retardation, hemorrhage, microphthalmia, pallor, preweaning lethality with incomplete penetrance, and spina bifida. These phenotypes are observed in both sexes.


The protein sequence analyzed has a GRAVY value of -0.199 (69.26th percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is -11.705 (19.89th percentile), and the median structural flexibility is 0.999 (35.77th percentile). The protein has a propensity for helix, sheet, and turn structures of 32.27% (52.9th percentile), 36% (66.18th percentile), and 26.93% (37.43rd percentile), respectively. The instability index is 37.95 (16.58th percentile), with an isoelectric point of 5.31 (13.61st percentile). The sequence length is 375 amino acids (41.89th percentile), and the molecular weight is 41792.39 Da (41.78th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |