---
layout: post
title: TMEM59
date: 2024-12-08 23:57 CST
description: TMEM59 description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9528) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9528  | TMEM59 | ENSG00000116209 | 1p32.3 |



The gene length is 24,560 base pairs (47.29% of all genes), the mature length is 2,237 base pairs, and the primary transcript length is 24,560 base pairs.


TMEM59 has been studied in 14 publications, with the earliest publication in 2006 and the middle 50% of publications occurring between 2015 and 2020. For more information, see the [14 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22TMEM59%22) on PubMed (note that the count may differ since it is derived from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)).


The top 5 publications mentioning TMEM59, ranked by their relative citation ratio (RCR), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows. The most influential publication is "[TMEM59 defines a novel ATG16L1-binding motif that promotes local activation of LC3.](https://pubmed.ncbi.nlm.nih.gov/23376921)" (2013) with an RCR of 2.4. This is followed by "[TMEM59 interacts with TREM2 and modulates TREM2-dependent microglial activities.](https://pubmed.ncbi.nlm.nih.gov/32826884)" (2020) with an RCR of 1.34, and "[TMEM59 potentiates Wnt signaling by promoting signalosome formation.](https://pubmed.ncbi.nlm.nih.gov/29632210)" (2018) with an RCR of 1.12. Additionally, "[Whole exome sequencing, in silico and functional studies confirm the association of the <i>GJB2</i> mutation p.Cys169Tyr with deafness and suggest a role for the <i>TMEM59</i> gene in the hearing process.](https://pubmed.ncbi.nlm.nih.gov/34354426)" (2021) has an RCR of 0.69, and "[Overexpression of DCF1 inhibits glioma through destruction of mitochondria and activation of apoptosis pathway.](https://pubmed.ncbi.nlm.nih.gov/24424470)" (2014) has an RCR of 0.49. For more information on RCR, please refer to [this article](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541). Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [RXRA](https://www.ncbi.nlm.nih.gov/gene/6256) with 6 experiments, [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664) with 5 experiments, [PPARG](https://www.ncbi.nlm.nih.gov/gene/5468) with 5 experiments, [KDM5B](https://www.ncbi.nlm.nih.gov/gene/10765) with 5 experiments, and [BRD9](https://www.ncbi.nlm.nih.gov/gene/65980) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [6829](https://www.ncbi.nlm.nih.gov/gene/6829) and [9126](https://www.ncbi.nlm.nih.gov/gene/9126), each shown to be regulating genes in 4 experiments. Additionally, [8243](https://www.ncbi.nlm.nih.gov/gene/8243) was found to regulate genes in 1 experiment, and [9968](https://www.ncbi.nlm.nih.gov/gene/9968) in 2 experiments. These findings highlight the significant regulatory roles of these TFs in gene expression.








The analyzed protein sequence has a GRAVY value of -0.133 (75.54th percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is -13.86 (16.79th percentile), and the median structural flexibility is 0.995 (22.21st percentile). The protein has a high instability index of 65.46 (90.72nd percentile) and an isoelectric point of 4.997 (7.72nd percentile). The sequence length is 323 amino acids (33.42nd percentile) with a molecular weight of 36222.98 Da (33.91st percentile). The secondary structure affinities are 35.29% for helix (76.58th percentile), 35.91% for sheet (65.6th percentile), and 26.32% for turn (32.08th percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |