---
layout: post
title: COG5
date: 2025-01-17 16:55 CST
description: COG5 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10466) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10466  | COG5 | ENSG00000164597 | 7q22.3 |



The gene is located in the [Golgi membrane](https://amigo.geneontology.org/amigo/term/GO:0000139), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [Golgi apparatus](https://amigo.geneontology.org/amigo/term/GO:0005794), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), and [trans-Golgi network membrane](https://amigo.geneontology.org/amigo/term/GO:0032588). It is involved in [retrograde transport, vesicle recycling within Golgi](https://amigo.geneontology.org/amigo/term/GO:0000301), [intra-Golgi vesicle-mediated transport](https://amigo.geneontology.org/amigo/term/GO:0006891), [Golgi organization](https://amigo.geneontology.org/amigo/term/GO:0007030), [protein transport](https://amigo.geneontology.org/amigo/term/GO:0015031), and [glycosylation](https://amigo.geneontology.org/amigo/term/GO:0070085). The gene enables [molecular function](https://amigo.geneontology.org/amigo/term/GO:0003674) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is part of the [Golgi transport complex](https://amigo.geneontology.org/amigo/term/GO:0017119) and acts upstream of or within [inter-Golgi cisterna vesicle-mediated transport](https://amigo.geneontology.org/amigo/term/GO:0048219).


The gene length is 140,566 base pairs (87.94th percentile), the mature length is 4,605 base pairs (88.15th percentile), and the primary transcript length is 128,318 base pairs (91.76th percentile).


The gene COG5, with NCBI ID 10466, has been mentioned in [12 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22COG5%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727). The earliest year of publication is 2002, and the middle 50% of publications occurred between 2011 and 2019. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning COG5, ranked by their scientific influence, include studies that delve into the gene's role in congenital disorders of glycosylation and its impact on cellular functions. The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Notable publications include "[Deficiency in COG5 causes a moderate form of congenital disorders of glycosylation.](https://pubmed.ncbi.nlm.nih.gov/19690088)" (2009) (relative citation ratio: 1.87), "[MALDI-MS profiling of serum O-glycosylation and N-glycosylation in COG5-CDG.](https://pubmed.ncbi.nlm.nih.gov/28444691)" (2017) (relative citation ratio: 1.27), and "[Genetic analysis of the subunit organization and function of the conserved oligomeric golgi (COG) complex: studies of COG5- and COG7-deficient mammalian cells.](https://pubmed.ncbi.nlm.nih.gov/16051600)" (2005) (relative citation ratio: 1.19). Additionally, "[COG5-CDG with a Mild Neurohepatic Presentation.](https://pubmed.ncbi.nlm.nih.gov/23430875)" (2012) (relative citation ratio: 0.98) and "[COG5-CDG: expanding the clinical spectrum.](https://pubmed.ncbi.nlm.nih.gov/23228021)" (2012) (relative citation ratio: 0.88) contribute to the understanding of COG5's clinical implications. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[COG5](https://www.proteinatlas.org/ENSG00000164597-COG5) is a gene involved in congenital disorders of glycosylation, with evidence supporting its function at the protein level. The gene is detected in all RNA tissue distributions and is localized in the nucleoplasm and Golgi apparatus. It is expressed in various clusters, including Cluster 5 for blood expression related to cell proliferation, Cluster 51 for tissue expression in the thyroid gland with unknown function, Cluster 4 for brain expression related to mRNA splicing and cell cycle, Cluster 39 for cell line expression related to transcription, and Cluster 8 for single cell expression in neurons related to neuronal signaling.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113), [SUPT5H](https://www.ncbi.nlm.nih.gov/gene/6829), [HNF4A](https://www.ncbi.nlm.nih.gov/gene/3172), and [BRD9](https://www.ncbi.nlm.nih.gov/gene/65980), each shown to be regulating in 5 experiments.



The input data from GWAS indicates associations with a wide range of disease conditions, including various types of cancer such as leukemia, lymphoma, and immune system cancer. Additionally, respiratory conditions like asthma, bronchial disease, and obstructive lung disease are noted. Cardiovascular issues, including artery disease, vascular disease, and arteriosclerosis, are also prominent. Furthermore, the data highlights musculoskeletal conditions such as arthritis, osteoarthritis, and bone inflammation disease. Metabolic disorders, specifically those related to mineral and calcium metabolism, are also implicated.



The gene is expressed in various tissues, including the uterus, cervix uteri, and vagina, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the pineal gland (both day and night), CD34+ cells, CD56+ NK cells, bronchial epithelial cells, colon, and small intestine.



The mouse ortholog gene 238123, which corresponds to the human ortholog gene 10466, is associated with several phenotypic abnormalities. These include abnormal morphologies in the digit, heart, skin, and spleen, as well as embryonic lethality prior to organogenesis and prior to the tooth bud stage. Additionally, the gene is linked to enlarged heart and spleen, and complete penetrance preweaning lethality. Notably, most of these phenotypes are observed in both sexes, except for the embryonic lethality phenotypes.


The analyzed protein sequence has a GRAVY value of -0.161 (73.07th percentile), indicating a relatively hydrophilic nature. The charge at pH 7 is -6.97 (28.52nd percentile), and the median structural flexibility is 0.999 (37.52nd percentile). The protein has a notable affinity for helix (0.333, 61.58th percentile) and sheet (0.349, 58.18th percentile) structures, with a lower affinity for turns (0.276, 43.26th percentile). The instability index is 49.49 (55.57th percentile), and the isoelectric point is 6.15 (34.52nd percentile). The protein is 860 amino acids long (83.06th percentile) with a molecular weight of 94900.2 Da (82.63rd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |