---
layout: post
title: SON
date: 2025-01-17 16:55 CST
description: SON description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6651) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6651  | SON | ENSG00000159140 | 21q22.11 |



The gene acts upstream of [microtubule cytoskeleton organization](https://amigo.geneontology.org/amigo/term/GO:0000226) and [mitotic cytokinesis](https://amigo.geneontology.org/amigo/term/GO:0000281), and is involved in the [negative regulation of apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0043066). It enables [DNA binding](https://amigo.geneontology.org/amigo/term/GO:0003677), [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). Additionally, it is involved in [mRNA processing](https://amigo.geneontology.org/amigo/term/GO:0006397), [RNA splicing](https://amigo.geneontology.org/amigo/term/GO:0008380), and the [regulation of RNA splicing](https://amigo.geneontology.org/amigo/term/GO:0043484), specifically the [regulation of mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0048024). The gene is located in the [nuclear speck](https://amigo.geneontology.org/amigo/term/GO:0016607) and acts upstream of the [regulation of cell cycle](https://amigo.geneontology.org/amigo/term/GO:0051726).


The gene length is 21,388 base pairs (43.7% of all genes), the mature length is 4,686 base pairs, and the primary transcript length is 12,088.5 base pairs.


The gene SON (NCBI ID: 6651) has been mentioned in [4 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SON%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest publication mentioning SON was in 1985, and the middle 50% of publications occurred between 1987 and 2003.


The top publications mentioning the SON gene include "[Transcription repression of human hepatitis B virus genes by negative regulatory element-binding protein/SON.](https://pubmed.ncbi.nlm.nih.gov/11306577)" (2001) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 0.73, "[Regulation of ghrelin signaling by a leptin-induced gene, negative regulatory element-binding protein, in the hypothalamic neurons.](https://pubmed.ncbi.nlm.nih.gov/20876580)" (2010) with an RCR of 0.4, and "[Decoding of the primary structure of the son3 region in human genome: identification of a new protein with unusual structure and homology with DNA-binding proteins.](https://pubmed.ncbi.nlm.nih.gov/3054499)" (1988) with an RCR of 0.21. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[SON](https://www.proteinatlas.org/ENSG00000159140-SON) is a gene with molecular functions in DNA-binding and RNA-binding. It is implicated in disease variants associated with intellectual disability. Evidence for its function is available at the protein level. The gene is detected in all RNA tissue distributions and is primarily localized in nuclear speckles. In terms of expression, SON is found in Cluster 22 of blood expression, specifically in eosinophils related to transcription. It is also part of Cluster 40 in tissue expression, Cluster 50 in brain expression, and Cluster 47 in cell line expression, all related to transcription and nucleic acid binding. Additionally, it is present in Cluster 55 of single cell expression, which is non-specific but related to transcription.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613), [STAT3](https://www.ncbi.nlm.nih.gov/gene/6774), [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), and [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227), each shown to be regulating in 5 experiments.


The transcription factor [65123](https://www.ncbi.nlm.nih.gov/gene/65123) was found to be regulating genes in one Perturb-seq experiment, according to data from GTRD's database of transcription factor binding sites and the study published in [PubMed](https://pubmed.ncbi.nlm.nih.gov/35688146/).


The GWAS data indicates associations with several disease conditions, including cancer, diseases of cellular proliferation, and general disease.



The gene is expressed in various immune cell types, including CD33+ myeloid cells, BDCA4+ dendritic cells, CD56+ NK cells, CD4+ T cells, CD8+ T cells, CD19+ B cells (negatively selected), and CD71+ early erythroid cells. Additionally, it is expressed in the thyroid tissue.




The analyzed protein sequence has a GRAVY value of -0.559 (28.92 percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -44.94 (3.05 percentile), and the median structural flexibility is 1.011 (85.03 percentile). The protein has a high instability index of 81.35 (98.5 percentile) and an isoelectric point of 5.50 (18.09 percentile). The sequence length is 2426 amino acids (98.51 percentile), with a molecular weight of 263827.18 Da (98.44 percentile). The secondary structure affinities are 34.30% for helix (70.15 percentile), 27.58% for sheet (16.71 percentile), and 34.17% for turn (86.68 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |