---
layout: post
title: TJP2
date: 2025-01-17 16:55 CST
description: TJP2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9414) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9414  | TJP2 | ENSG00000119139 | 9q21.11 |



The gene enables [guanylate kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004385) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is involved in [protein domain specific binding](https://amigo.geneontology.org/amigo/term/GO:0019904) and [protein-macromolecule adaptor activity](https://amigo.geneontology.org/amigo/term/GO:0030674). It is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [adherens junction](https://amigo.geneontology.org/amigo/term/GO:0005912), and [bicellular tight junction](https://amigo.geneontology.org/amigo/term/GO:0005923). The gene is active in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886) and [bicellular tight junction](https://amigo.geneontology.org/amigo/term/GO:0005923). It is involved in various processes including [response to organic substance](https://amigo.geneontology.org/amigo/term/GO:0010033), [homotypic cell-cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0034109), [maintenance of blood-brain barrier](https://amigo.geneontology.org/amigo/term/GO:0035633), [cell-cell junction organization](https://amigo.geneontology.org/amigo/term/GO:0045216), [GMP metabolic process](https://amigo.geneontology.org/amigo/term/GO:0046037), [GDP metabolic process](https://amigo.geneontology.org/amigo/term/GO:0046710), [intestinal absorption](https://amigo.geneontology.org/amigo/term/GO:0050892), [establishment of endothelial intestinal barrier](https://amigo.geneontology.org/amigo/term/GO:0090557), [regulation of membrane permeability](https://amigo.geneontology.org/amigo/term/GO:0090559), [cell-cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0098609), [protein localization to cell-cell junction](https://amigo.geneontology.org/amigo/term/GO:0150105), and [positive regulation of blood-brain barrier permeability](https://amigo.geneontology.org/amigo/term/GO:1905605). Additionally, the gene enables [transmembrane transporter binding](https://amigo.geneontology.org/amigo/term/GO:0044325), [cadherin binding](https://amigo.geneontology.org/amigo/term/GO:0045296), [cell adhesion molecule binding](https://amigo.geneontology.org/amigo/term/GO:0050839), and [protein tyrosine kinase binding](https://amigo.geneontology.org/amigo/term/GO:1990782).


The gene length is 5,410 base pairs (15.15% of all genes), the mature length is 899 base pairs, and the primary transcript length is 5,163 base pairs.


The gene TJP2 (NCBI ID: 9414) has been mentioned in [47 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22TJP2%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1995 and the middle 50% of publications occurring between 2004 and 2017. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning TJP2, ranked by their scientific influence, include "[ZO-1 and ZO-2 independently determine where claudins are polymerized in tight-junction strand formation.](https://pubmed.ncbi.nlm.nih.gov/16923393)" (2006) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 12.58, "[Molecular mechanisms underlying the probiotic effects of Escherichia coli Nissle 1917 involve ZO-2 and PKCzeta redistribution resulting in tight junction and epithelial barrier repair.](https://pubmed.ncbi.nlm.nih.gov/17087734)" (2007) with an RCR of 7.3, "[Mutations in TJP2 cause progressive cholestatic liver disease.](https://pubmed.ncbi.nlm.nih.gov/24614073)" (2014) with an RCR of 6.26, "[Complex inheritance of familial hypercholanemia with associated mutations in TJP2 and BAAT.](https://pubmed.ncbi.nlm.nih.gov/12704386)" (2003) with an RCR of 4.18, and "[An expanded role for heterozygous mutations of ABCB4, ABCB11, ATP8B1, ABCC2 and TJP2 in intrahepatic cholestasis of pregnancy.](https://pubmed.ncbi.nlm.nih.gov/28924228)" (2017) with an RCR of 3.98. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[TJP2](https://www.proteinatlas.org/ENSG00000119139-TJP2) is a gene involved in cancer-related processes and intrahepatic cholestasis, with evidence supporting its role at the protein level. The gene is detected in all RNA tissue distributions and is localized in the nucleoplasm, plasma membrane, and cell junctions. Its expression is notable in monocytes related to innate immune response, retina and testis associated with cilium function, oligodendrocytes with mixed functions, and squamous epithelial cells involved in keratinization.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) with 6 experiments, [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 5 experiments, [TP53](https://www.ncbi.nlm.nih.gov/gene/7157) with 5 experiments, [HDAC1](https://www.ncbi.nlm.nih.gov/gene/3065) with 5 experiments, and [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [2118](https://www.ncbi.nlm.nih.gov/gene/2118) and [9126](https://www.ncbi.nlm.nih.gov/gene/9126), each shown to regulate gene expression in one experiment. Additionally, [8243](https://www.ncbi.nlm.nih.gov/gene/8243), [9968](https://www.ncbi.nlm.nih.gov/gene/9968), and [57634](https://www.ncbi.nlm.nih.gov/gene/57634) were found to regulate gene expression in two experiments each.


The GWAS data indicates associations with a range of disease conditions, including various types of cancer such as lung cancer and respiratory system cancer, as well as metabolic disorders like obesity and acquired metabolic disease. Additionally, the data highlights associations with cardiovascular system diseases, including artery disease and hypertension, and urinary system diseases such as kidney disease. Furthermore, there are links to nervous system diseases, inherited metabolic disorders, and eye diseases, including refractive errors. The data also suggests associations with lipid storage diseases, lysosomal storage diseases, and fatty liver disease.


The TJP2 gene has been associated with several ocular traits and conditions. Notably, it has been linked to [Refractive error](https://pubmed.ncbi.nlm.nih.gov/23396134), which maps closely to the MeSH term [Refractive Errors](https://meshb.nlm.nih.gov/record/ui?ui=D012030). Additionally, TJP2 has been implicated in [Spherical equivalent](https://pubmed.ncbi.nlm.nih.gov/29808027), which was mapped to the MeSH term [Phosmet](https://meshb.nlm.nih.gov/record/ui?ui=D010706) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method. Furthermore, the gene is associated with [Spherical equivalent or myopia (age of diagnosis)](https://pubmed.ncbi.nlm.nih.gov/29808027), which maps to the MeSH term [Myopia](https://meshb.nlm.nih.gov/record/ui?ui=D009216).


The gene is expressed in various tissues, including the bladder, retina, thyroid, spinal cord, olfactory bulb, and colorectal adenocarcinoma.


The input data highlights several key biological processes, including apoptosis and programmed cell death, with a focus on the apoptotic cleavage of cellular and cell adhesion proteins during the execution phase. Additionally, the data covers signal transduction, specifically signaling by the Hippo pathway.



The analyzed protein sequence has a GRAVY value of -0.948 (5.72 percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -0.257 (49.29 percentile). The median structural flexibility is 1.015 (91.63 percentile), suggesting high flexibility. The protein's secondary structure is predicted to be 29.1% helix (24.1 percentile), 26.8% sheet (13.77 percentile), and 34.0% turn (86.12 percentile). The instability index is 56.74 (76.52 percentile), and the isoelectric point is 6.96 (49.89 percentile). The protein is 1190 amino acids long (91.52 percentile) with a molecular weight of 133956.41 Da (91.82 percentile). For more details on the sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |