---
layout: post
title: GNAQ
date: 2025-01-17 16:55 CST
description: GNAQ description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2776) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2776  | GNAQ | ENSG00000156052 | 9q21.2 |



The gene is involved in various biological processes, including [action potential](https://amigo.geneontology.org/amigo/term/GO:0001508), [negative regulation of protein kinase activity](https://amigo.geneontology.org/amigo/term/GO:0006469), and [phototransduction, visible light](https://amigo.geneontology.org/amigo/term/GO:0007603). It enables molecular functions such as [G protein-coupled receptor binding](https://amigo.geneontology.org/amigo/term/GO:0001664), [GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0003924), and [GTPase activator activity](https://amigo.geneontology.org/amigo/term/GO:0005096). The gene is located in multiple cellular components, including the [photoreceptor outer segment](https://amigo.geneontology.org/amigo/term/GO:0001750), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [lysosomal membrane](https://amigo.geneontology.org/amigo/term/GO:0005765), and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). Additionally, it is part of the [heterotrimeric G-protein complex](https://amigo.geneontology.org/amigo/term/GO:0005834) and is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737). The gene also plays roles in various signaling pathways, including the [adenylate cyclase-activating G protein-coupled receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007189) and the [glutamate receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007215).


The gene length is 115,841 base pairs (84.51 percentile of all genes), the mature length is 2,570 base pairs, and the primary transcript length is 115,841 base pairs.


The gene GNAQ (NCBI ID: 2776) has been mentioned in [289 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22GNAQ%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning GNAQ is 1992, and the middle 50% of publications occurred between 2009 and 2019.


The top 5 publications mentioning GNAQ, ranked by their scientific influence, include "[A Platform of Synthetic Lethal Gene Interaction Networks Reveals that the GNAQ Uveal Melanoma Oncogene Controls the Hippo Pathway through FAK.](https://pubmed.ncbi.nlm.nih.gov/30773340)" (2019) (relative citation ratio: 6.63), "[Combined PKC and MEK inhibition in uveal melanoma with GNAQ and GNA11 mutations.](https://pubmed.ncbi.nlm.nih.gov/24141786)" (2014) (relative citation ratio: 4.52), "[Mosaic Activating Mutations in GNA11 and GNAQ Are Associated with Phakomatosis Pigmentovascularis and Extensive Dermal Melanocytosis.](https://pubmed.ncbi.nlm.nih.gov/26778290)" (2016) (relative citation ratio: 4.46), "[Endothelial <i>GNAQ</i> p.R183Q Increases ANGPT2 (Angiopoietin-2) and Drives Formation of Enlarged Blood Vessels.](https://pubmed.ncbi.nlm.nih.gov/34670408)" (2022) (relative citation ratio: 4.28), and "[Snapshot of activated G proteins at the membrane: the Galphaq-GRK2-Gbetagamma complex.](https://pubmed.ncbi.nlm.nih.gov/16339447)" (2005) (relative citation ratio: 4.25). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[GNAQ](https://www.proteinatlas.org/ENSG00000156052-GNAQ) is a transducer gene with evidence at the protein level, implicated in cancer-related diseases. It is detected in all tissues and is particularly expressed in the brain and testis, where it plays a role in signal transduction. Subcellularly, GNAQ is localized to nuclear speckles, the plasma membrane, and the cytosol. In blood, it is found in basophils, although its function in this context is unknown. Additionally, GNAQ is expressed in neurons and astrocytes, as well as in rhabdoid cell lines, where it is associated with membrane components.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [MYC](https://www.ncbi.nlm.nih.gov/gene/4609) with 6 experiments, [ZBTB7A](https://www.ncbi.nlm.nih.gov/gene/51341) with 5 experiments, [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170) with 5 experiments, [EP300](https://www.ncbi.nlm.nih.gov/gene/2033) with 5 experiments, and [AR](https://www.ncbi.nlm.nih.gov/gene/367) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent detection in multiple experiments.



The GWAS data indicates associations with various disease conditions, including artery disease, vascular disease, cardiovascular system disease, hypertension, cognitive disorder, disease of mental health, bipolar disorder, and mood disorder.



The gene is expressed in various tissues, including the brain and lung, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the Pons, Superior Cervical Ganglion, Trigeminal Ganglion, Atrioventricular Node, Ciliary Ganglion, Skin, and Dorsal Root Ganglion.


The input data highlights various pathways involving proteins, including metabolism, ADP signalling through P2Y purinoceptor 1, signal amplification, thromboxane signalling through TP receptor, and thrombin signalling through proteinase activated receptors (PARs). Additionally, it covers the integration of energy metabolism, regulation of insulin secretion, G alpha (q) signalling events, and the gastrin-CREB signalling pathway via PKC and MAPK. Other notable pathways include GPCR downstream signalling, signalling by GPCR, signal transduction, and the regulation of insulin secretion by acetylcholine and fatty acids bound to GPR40 (FFAR1). The data also encompasses hemostasis and platelet activation, signalling, and aggregation.



The analyzed protein sequence has a GRAVY value of -0.4401 (41.67th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -7.95 (26.54th percentile). The median structural flexibility is 0.999 (36.68th percentile), with a propensity for secondary structures as follows: 32.31% helix (53.41st percentile), 37.88% sheet (77.89th percentile), and 23.96% turn (14.77th percentile). The instability index is 48.91 (53.53rd percentile), and the isoelectric point is 5.48 (17.79th percentile). The protein is 359 amino acids long (39.48th percentile) with a molecular weight of 42141.60 Da (42.29th percentile). For detailed sequence analysis methods, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |