---
layout: post
title: WWP2
date: 2024-12-08 23:57 CST
description: WWP2 description
tags: [cooccuring-genes]
categories: longest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/11060) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 11060  | WWP2 | ENSG00000198373 | 16q22.1 |



The gene length is 577,062 base pairs (98.78% of all genes), the mature length is 2,830 base pairs, and the primary transcript length is 573,283 base pairs.


The gene WWP2, with NCBI ID 11060, has been the subject of [46 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22WWP2%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 2002 and the middle 50% of publications occurring between 2014 and 2019. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning WWP2, ranked by their scientific influence, include "[The AIP2 E3 ligase acts as a novel negative regulator of ABA signaling by promoting ABI3 degradation.](https://pubmed.ncbi.nlm.nih.gov/15998807)" (2005) with a relative citation ratio of 5.57, "[WWP2 is an E3 ubiquitin ligase for PTEN.](https://pubmed.ncbi.nlm.nih.gov/21532586)" (2011) with a relative citation ratio of 5.37, "[MiR-140 is co-expressed with Wwp2-C transcript and activated by Sox9 to target Sp1 in maintaining the chondrocyte proliferation.](https://pubmed.ncbi.nlm.nih.gov/21872590)" (2011) with a relative citation ratio of 2.78, "[Wwp2 is essential for palatogenesis mediated by the interaction between Sox9 and mediator subunit 25.](https://pubmed.ncbi.nlm.nih.gov/21427722)" (2011) with a relative citation ratio of 2.67, and "[WWP2 ubiquitylates RNA polymerase II for DNA-PK-dependent transcription arrest and repair at DNA breaks.](https://pubmed.ncbi.nlm.nih.gov/31048545)" (2019) with a relative citation ratio of 2.4. The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 5 experiments, [HDAC2](https://www.ncbi.nlm.nih.gov/gene/3066) with 4 experiments, [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046) with 4 experiments, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 4 experiments, and [CREB1](https://www.ncbi.nlm.nih.gov/gene/1385) with 3 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.











The protein sequence analyzed has a GRAVY value of -0.690 (18.21% percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -2.44 (41.44% percentile), and the median structural flexibility is 1.006 (66.84% percentile). The protein has a propensity for helix, sheet, and turn structures of 28.05% (17.27% percentile), 34.60% (56.06% percentile), and 30.69% (68.61% percentile), respectively. The instability index is 50.86 (60.0% percentile), with an isoelectric point of 6.67 (45.76% percentile). The sequence length is 870 amino acids (83.42% percentile), and the molecular weight is 98910.86 Da (83.97% percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |