---
layout: post
title: EBNA1BP2
date: 2025-01-17 16:55 CST
description: EBNA1BP2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10969) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10969  | EBNA1BP2 | ENSG00000117395 | 1p34.2 |



The gene is located in the [chromosome](https://amigo.geneontology.org/amigo/term/GO:0005694) and is active in both the [nucleolus](https://amigo.geneontology.org/amigo/term/GO:0005730) and the [nuclear periphery](https://amigo.geneontology.org/amigo/term/GO:0034399). It enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is involved in [rRNA processing](https://amigo.geneontology.org/amigo/term/GO:0006364) and [ribosomal large subunit biogenesis](https://amigo.geneontology.org/amigo/term/GO:0042273). Additionally, it is part of the [preribosome, large subunit precursor](https://amigo.geneontology.org/amigo/term/GO:0030687).


The gene length is 152,982 base pairs (89.18% of all genes), the mature length is 3,664 base pairs, and the primary transcript length is 77,785 base pairs (84.21% of all genes).


EBNA1BP2 has been mentioned in [8 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22EBNA1BP2%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 2003, and the middle 50% of publications range from 2004 to 2015.


The top 5 publications mentioning EBNA1BP2 include "[EBP2 plays a key role in Epstein-Barr virus mitotic segregation and is regulated by aurora family kinases.](https://pubmed.ncbi.nlm.nih.gov/15923612)" (2005) (relative citation ratio: 1.1), "[EBNA1 partitions Epstein-Barr virus plasmids in yeast cells by attaching to human EBNA1-binding protein 2 on mitotic chromosomes.](https://pubmed.ncbi.nlm.nih.gov/12768013)" (2003) (relative citation ratio: 0.8), "[Mak5 and Ebp2 act together on early pre-60S particles and their reduced functionality bypasses the requirement for the essential pre-60S factor Nsa1.](https://pubmed.ncbi.nlm.nih.gov/24312670)" (2013) (relative citation ratio: 0.74), "[Ebp2p, the yeast homolog of Epstein-Barr virus nuclear antigen 1-binding protein 2, interacts with factors of both the 60 S and the 40 s ribosomal subunit assembly.](https://pubmed.ncbi.nlm.nih.gov/15078877)" (2004) (relative citation ratio: 0.44), and "[Antibody response to a new member of the DBL family (EBP2) after a brief Plasmodium vivax exposure.](https://pubmed.ncbi.nlm.nih.gov/35714097)" (2022) (relative citation ratio: 0.37). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, and benchmarking to the median for NIH publications, the value of which is set at 1.0. Citation counts are from [iCite](https://icite.od.nih.gov).


[EBNA1BP2](https://www.proteinatlas.org/ENSG00000117395-EBNA1BP2) is a gene with evidence at the protein level. Its RNA is detected in all tissues, and the protein is localized to the nucleoli, nucleoli rim, and mitotic chromosome. The gene is expressed in various clusters, including Cluster 13 for blood expression, Cluster 53 for tissue expression, Cluster 23 for brain expression, Cluster 37 for cell line expression, and Cluster 39 for single cell expression, indicating its involvement in basic cellular processes and mixed functions.


The top transcription factors (TFs) identified from the GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [E2F1](https://www.ncbi.nlm.nih.gov/gene/1869) and [CREB1](https://www.ncbi.nlm.nih.gov/gene/1385), each shown to be regulating in 5 experiments. Additionally, [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), [TRIM24](https://www.ncbi.nlm.nih.gov/gene/8805), and [KLF9](https://www.ncbi.nlm.nih.gov/gene/687) were each found to be regulating in 4 experiments.





The gene is expressed in various tissues and cell types, including liver and pituitary. Additionally, it is expressed in 721_B_lymphoblasts, Leukemia_chronicMyelogenousK-562, Leukemia_promyelocytic-HL-60, Lymphoma_burkitts(Daudi), CD105+_Endothelial cells, SmoothMuscle, BronchialEpithelialCells, and Colorectaladenocarcinoma.



The mouse ortholog gene 69072, which corresponds to the human ortholog gene 10969, is associated with several phenotypic abnormalities. These include abnormal uterus morphology and embryonic lethality prior to organogenesis, both observed in females. Additionally, both sexes exhibit enlarged kidney, enlarged spleen, and preweaning lethality with complete penetrance.


The analyzed protein sequence has a GRAVY value of -1.15 (2.19th percentile), indicating a hydrophilic nature. It exhibits a charge of 29.87 at pH 7.0 (97.17th percentile) and a median structural flexibility of 1.02 (97.55th percentile). The protein's secondary structure is predicted to be 39.87% helix (93.18th percentile), 23.53% sheet (5.7th percentile), and 27.12% turn (39.07th percentile). The instability index is 44.76 (38.34th percentile), with an isoelectric point of 10.10 (97.02nd percentile). The protein has a length of 306 amino acids (29.42nd percentile) and a molecular weight of 34851.54 Da (30.81st percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |