---
layout: post
title: TCAF1
date: 2025-01-17 16:55 CST
description: TCAF1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9747) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9747  | TCAF1 | ENSG00000198420 | 7q35  |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [transmembrane transporter binding](https://amigo.geneontology.org/amigo/term/GO:0044325). It is located in and active in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). Additionally, it is involved in the [negative regulation of cell migration](https://amigo.geneontology.org/amigo/term/GO:0030336), [positive regulation of protein targeting to membrane](https://amigo.geneontology.org/amigo/term/GO:0090314), and [positive regulation of anion channel activity](https://amigo.geneontology.org/amigo/term/GO:1901529).


The gene length is 70,782 base pairs (74.78th percentile), the mature length is 7,244 base pairs (96.83rd percentile), and the primary transcript length is 68,925 base pairs (82.13rd percentile).


There are no publications that mention this gene.



[TCAF1](https://www.proteinatlas.org/ENSG00000198420-TCAF1) is a gene with evidence at the protein level. Its RNA is detected in all tissues, and the protein is localized to nuclear bodies, vesicles, and cell junctions. The gene is part of Cluster 33 in blood expression, which is associated with non-specific ATP binding, and Cluster 64 in tissue expression, which is neuronal and related to transcription. In the brain, it is part of Cluster 33, associated with neurons and mixed function. In cell lines, it is part of Cluster 18, related to liver cancer and metabolism, and in single-cell expression, it is part of Cluster 24, associated with cytotrophoblasts and mitochondria.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599), [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), [PGR](https://www.ncbi.nlm.nih.gov/gene/5241), and [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), each with 4 experiments, and [GTF2F1](https://www.ncbi.nlm.nih.gov/gene/2962) with 3 experiments.





The gene is expressed in various brain regions, including the amygdala, caudate nucleus, and fetal brain, as well as in the pineal gland during the night. Additionally, expression is observed in the pituitary and adrenal glands.




The analyzed protein sequence has a GRAVY value of -0.037 (82.14th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -7.97 (26.49th percentile), and the median structural flexibility is 0.995 (21.82nd percentile). The protein has a length of 921 amino acids (85.21st percentile) and a molecular weight of 102124.67 Da (85.05th percentile). The secondary structure affinities are 32.03% for helix (50.58th percentile), 38.87% for sheet (82.57th percentile), and 28.56% for turn (52.2nd percentile). The instability index is 41.69 (27.58th percentile), and the isoelectric point is 6.12 (33.7th percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |