---
layout: post
title: SKIC2
date: 2025-01-17 16:55 CST
description: SKIC2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6499) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6499  | SKIC2 | ENSG00000204351 | 6p21.33 |



The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [RNA helicase activity](https://amigo.geneontology.org/amigo/term/GO:0003724), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524), [ATP hydrolysis activity](https://amigo.geneontology.org/amigo/term/GO:0016887), and [3'-5' RNA helicase activity](https://amigo.geneontology.org/amigo/term/GO:0034458). It is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and is part of the [Ski complex](https://amigo.geneontology.org/amigo/term/GO:0055087). The gene is involved in the [nuclear-transcribed mRNA catabolic process, 3'-5' exonucleolytic nonsense-mediated decay](https://amigo.geneontology.org/amigo/term/GO:0070478) and the [rescue of stalled ribosome](https://amigo.geneontology.org/amigo/term/GO:0072344).


The gene length is 177,690 base pairs (91.13% of all genes), the mature length is 1,822 base pairs (47.83% of all genes), and the primary transcript length is 15,954 base pairs (45.74% of all genes).


The gene SKIC2, with NCBI ID 6499, has been mentioned in [28 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SKIC2%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727). The earliest year of publication mentioning SKIC2 is 1997, and the middle 50% of publications occurred between 2012 and 2018. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning SKIC2, ranked by their scientific influence, include "[The SKIV2L RNA exosome limits activation of the RIG-I-like receptors.](https://pubmed.ncbi.nlm.nih.gov/25064072)" (2014) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 3.78, "[SKIV2L mutations cause syndromic diarrhea, or trichohepatoenteric syndrome.](https://pubmed.ncbi.nlm.nih.gov/22444670)" (2012) with an RCR of 2.48, "[The cryo-EM structure of a ribosome-Ski2-Ski3-Ski8 helicase complex.](https://pubmed.ncbi.nlm.nih.gov/27980209)" (2016) with an RCR of 2.31, "[Ski2-like RNA helicase structures: common themes and complex assemblies.](https://pubmed.ncbi.nlm.nih.gov/22995828)" (2013) with an RCR of 1.52, and "[Tricho-Hepato-Enteric Syndrome mutation update: Mutations spectrum of TTC37 and SKIV2L, clinical analysis and future prospects.](https://pubmed.ncbi.nlm.nih.gov/29527791)" (2018) with an RCR of 1.29. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[SKIC2](https://www.proteinatlas.org/ENSG00000204351-SKIC2) is a gene with molecular functions including helicase, hydrolase, and RNA-binding. It is involved in disease variants and has evidence at the protein level. The gene is detected in many tissues and is primarily localized in the nucleoplasm, nuclear bodies, mitotic spindle, and cytosol. SKIC2 is part of Cluster 43 for tissue expression, indicating non-specific mitochondrial translation, and Cluster 19 for brain expression, suggesting a mixed function in astrocytes. In cell lines, it is part of Cluster 28, associated with non-specific nuclear processes, and in single cells, it is part of Cluster 49, linked to non-specific transcription.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), [TBP](https://www.ncbi.nlm.nih.gov/gene/6908), [HDAC1](https://www.ncbi.nlm.nih.gov/gene/3065), [HDAC2](https://www.ncbi.nlm.nih.gov/gene/3066), and [REST](https://www.ncbi.nlm.nih.gov/gene/5978), each shown to be regulating in 5 experiments.



The GWAS data indicates associations with a wide range of disease conditions, including infectious diseases such as viral infectious diseases and human immunodeficiency virus infectious disease. Additionally, the data highlights various types of cancer, including lung cancer, hematologic cancer, immune system cancer, and nervous system cancer. Metabolic disorders are also prominent, with links to diabetes mellitus, particularly type 2 diabetes mellitus, and carbohydrate metabolism diseases. Cardiovascular conditions, such as coronary artery disease and vascular disease, are also noted. Furthermore, the data includes neurodegenerative diseases, mental health disorders like substance dependence and alcohol dependence, and musculoskeletal conditions such as rheumatoid arthritis. Eye diseases, specifically retinal degeneration and age-related macular degeneration, are also associated, along with lipid storage diseases and lysosomal storage diseases. Other notable conditions include multiple sclerosis, systemic lupus erythematosus, and various forms of lung cancer, including lung adenocarcinoma and non-small cell lung carcinoma. Additionally, hepatitis B and neuroblastoma are mentioned.


SKIC2 has been associated with various traits and diseases through genome-wide association studies. Notably, it has been linked to [ATP synthase subunit beta, mitochondrial levels](https://pubmed.ncbi.nlm.nih.gov/34648354), which maps to the MeSH term [Adenosine Triphosphate](https://meshb.nlm.nih.gov/record/ui?ui=D000255) using the BioWordVec method. Additionally, SKIC2 is associated with [Armadillo repeat-containing protein 5 levels](https://pubmed.ncbi.nlm.nih.gov/34648354), mapped to [Proteins](https://meshb.nlm.nih.gov/record/ui?ui=D011506), and [Complement factor B levels](https://pubmed.ncbi.nlm.nih.gov/36168886), mapped to [Complement System Proteins](https://meshb.nlm.nih.gov/record/ui?ui=D003165). Other associations include [Diastolic blood pressure](https://pubmed.ncbi.nlm.nih.gov/34594039), mapped to [Pressure](https://meshb.nlm.nih.gov/record/ui?ui=D011312), and [Platelet count](https://pubmed.ncbi.nlm.nih.gov/32888494), mapped directly to [Platelet Count](https://meshb.nlm.nih.gov/record/ui?ui=D010976). Furthermore, SKIC2 is linked to seropositivity for various peptides, including [human gammaherpesvirus 4 (Epstein-Barr virus) peptide](https://pubmed.ncbi.nlm.nih.gov/37164013), mapped to [Humans](https://meshb.nlm.nih.gov/record/ui?ui=D006801), and [staphylococcus aureus peptide](https://pubmed.ncbi.nlm.nih.gov/37164013), mapped to [Staphylococcus](https://meshb.nlm.nih.gov/record/ui?ui=D013210). These findings highlight the diverse roles of SKIC2 in various biological processes and diseases.


The gene is expressed in various tissues and cell types, including the pituitary and spleen. Additionally, it is expressed in Lymphoma Burkitt's (Raji), CD56+ NK cells, CD4+ T cells, CD8+ T cells, CD105+ endothelial cells, CD71+ early erythroid cells, and the prostate.


The input data highlights several key processes involved in protein metabolism and folding. These include the broader category of metabolism of proteins, as well as more specific pathways such as the association of TriC/CCT with target proteins during biosynthesis, chaperonin-mediated protein folding, and general protein folding.


The mouse ortholog gene 108077 and its human ortholog gene 6499 are associated with phenotypes including embryonic lethality prior to organogenesis, which is not observed in both sexes, and preweaning lethality with complete penetrance, which is observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.187 (70.5th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -23.71 (8.84th percentile), and the median structural flexibility is 0.999 (37.58th percentile). The protein has a notable affinity for helix (0.351, 75.11th percentile) and sheet (0.356, 63.16th percentile) structures, with a lower affinity for turns (0.274, 41.54th percentile). The instability index is 47.16 (47.19th percentile), and the isoelectric point is 5.72 (23.36th percentile). The protein's length is 1246 amino acids (92.47th percentile), with a molecular weight of 137753.65 Da (92.38th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |