---
layout: post
title: PTPRM
date: 2025-01-17 16:55 CST
description: PTPRM description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5797) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5797  | PTPRM | ENSG00000173482 | 18p11.23 |



The gene is involved in [negative regulation of endothelial cell proliferation](https://amigo.geneontology.org/amigo/term/GO:0001937) and [negative regulation of angiogenesis](https://amigo.geneontology.org/amigo/term/GO:0016525). It enables [protein tyrosine phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0004725) and [transmembrane receptor protein tyrosine phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0005001), and is involved in [protein dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0006470). The gene is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [cell-cell junction](https://amigo.geneontology.org/amigo/term/GO:0005911), [adherens junction](https://amigo.geneontology.org/amigo/term/GO:0005912), [lamellipodium](https://amigo.geneontology.org/amigo/term/GO:0030027), and the [perinuclear region of cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0048471). It is active in the [cell-cell junction](https://amigo.geneontology.org/amigo/term/GO:0005911) and [intracellular non-membrane-bounded organelle](https://amigo.geneontology.org/amigo/term/GO:0043232). Additionally, the gene is involved in [homophilic cell adhesion via plasma membrane adhesion molecules](https://amigo.geneontology.org/amigo/term/GO:0007156), [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), [response to xenobiotic stimulus](https://amigo.geneontology.org/amigo/term/GO:0009410), [negative regulation of endothelial cell migration](https://amigo.geneontology.org/amigo/term/GO:0010596), [retina layer formation](https://amigo.geneontology.org/amigo/term/GO:0010842), [neuron projection development](https://amigo.geneontology.org/amigo/term/GO:0031175), and [retinal ganglion cell axon guidance](https://amigo.geneontology.org/amigo/term/GO:0031290). The gene also enables [phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0016791), [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802), and [cadherin binding](https://amigo.geneontology.org/amigo/term/GO:0045296).


The gene length is 169,991 base pairs (90.6% of all genes), the mature length is 2,392 base pairs, and the primary transcript length is 60,045 base pairs.


The gene PTPRM (NCBI ID: 5797) has been mentioned in [24 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PTPRM%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1993 and the middle 50% of publications occurring between 2000 and 2020. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning PTPRM, ranked by their scientific influence, include "[CircRNA PTPRM Promotes Non-Small Cell Lung Cancer Progression by Modulating the miR-139-5p/SETD5 Axis.](https://pubmed.ncbi.nlm.nih.gov/35491723)" (2022) (relative citation ratio: 1.78), "[Receptor protein-tyrosine phosphatase RPTPmu binds to and dephosphorylates the catenin p120(ctn).](https://pubmed.ncbi.nlm.nih.gov/10753936)" (2000) (relative citation ratio: 1.78), "[DDIAS promotes STAT3 activation by preventing STAT3 recruitment to PTPRM in lung cancer cells.](https://pubmed.ncbi.nlm.nih.gov/31900385)" (2020) (relative citation ratio: 1.72), "[Proteolytic cleavage of protein tyrosine phosphatase mu regulates glioblastoma cell migration.](https://pubmed.ncbi.nlm.nih.gov/19690139)" (2009) (relative citation ratio: 1.37), and "[Increased proteolytic processing of protein tyrosine phosphatase mu in confluent vascular endothelial cells: the role of PC5, a member of the subtilisin family.](https://pubmed.ncbi.nlm.nih.gov/8620001)" (1996) (relative citation ratio: 1.15). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PTPRM](https://www.proteinatlas.org/ENSG00000173482-PTPRM) is a gene encoding a protein with hydrolase, protein phosphatase, and receptor molecular functions. Evidence for this gene is available at the protein level. The RNA expression of PTPRM is detected in all tissues, with notable expression in adipose tissue and sub-cortical regions of the brain. The protein is localized to the plasma membrane. In single-cell expression analysis, PTPRM is associated with neurons and synaptic function. The gene is part of various expression clusters, including Cluster 30 for non-specific DNA binding in blood, Cluster 7 for mixed function in tissue, Cluster 38 for mixed function in the brain, Cluster 15 for kidney cancer membrane components in cell lines, and Cluster 65 for neurons with synaptic function in single cells.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930) and [E2F1](https://www.ncbi.nlm.nih.gov/gene/1869), each with 4 experiments. Additionally, [PGR](https://www.ncbi.nlm.nih.gov/gene/5241), [KDM2B](https://www.ncbi.nlm.nih.gov/gene/84678), and [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) were each found to be regulating in 3 experiments.



The input data from GWAS indicates associations with a wide range of disease conditions, including various types of cancer such as male reproductive organ cancer, prostate cancer, testicular cancer, head and neck cancer, colorectal cancer, lung cancer, leukemia, and immune system cancer. Additionally, metabolic disorders such as diabetes mellitus, type 2 diabetes mellitus, and obesity are highlighted. Cardiovascular diseases, including coronary artery disease and hypertension, are also noted. Neurodegenerative diseases like Alzheimer's disease, dementia, and cognitive disorders are included, along with mental health conditions such as bipolar disorder, schizophrenia, and major depressive disorder. Other significant conditions mentioned are arthritis, gout, retinal disease, chronic kidney failure, and Gaucher's disease.



The gene is expressed in multiple tissue samples, including adipose tissue, breast, thyroid, and lung, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the thyroid, caudate nucleus, hypothalamus, thalamus, adipocytes, fetal thyroid, and fetal lung.



The mouse ortholog gene 19274 and its human ortholog gene 5797 are associated with abnormal heart morphology and enlarged heart phenotypes, with these effects observed in both sexes.


The protein sequence analyzed has a GRAVY value of -0.383 (48.13th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -13.29 (17.49th percentile). The median structural flexibility is 0.999 (38.1st percentile), with a secondary structure composition of 25.69% helix (6.66th percentile), 37.47% sheet (75.62nd percentile), and 29.13% turn (57.01st percentile). The instability index is 38.15 (17.2nd percentile), and the isoelectric point is 6.21 (35.78th percentile). The protein has a length of 1452 amino acids (94.68th percentile) and a molecular weight of 163680.51 Da (94.87th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |