---
layout: post
title: KCNJ9
date: 2025-01-17 16:55 CST
description: KCNJ9 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3765) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3765  | KCNJ9 | ENSG00000162728 | 1q23.2 |



The gene enables [inward rectifier potassium channel activity](https://amigo.geneontology.org/amigo/term/GO:0005242) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886) and is part of the [monoatomic ion channel complex](https://amigo.geneontology.org/amigo/term/GO:0034702). The gene is involved in the [regulation of monoatomic ion transmembrane transport](https://amigo.geneontology.org/amigo/term/GO:0034765), [regulation of presynaptic membrane potential](https://amigo.geneontology.org/amigo/term/GO:0099505), and [potassium ion import across plasma membrane](https://amigo.geneontology.org/amigo/term/GO:1990573). Additionally, it is located in the [presynaptic membrane](https://amigo.geneontology.org/amigo/term/GO:0042734) and the [parallel fiber to Purkinje cell synapse](https://amigo.geneontology.org/amigo/term/GO:0098688).


The gene length is 4,295 base pairs (12.1% of all genes), the mature length is 971.5 base pairs, and the primary transcript length is 3,518 base pairs.


The gene KCNJ9 (NCBI ID: 3765) has been mentioned in [6 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22KCNJ9%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning this gene was in 1995, and the middle 50% of publications occurred between 2000 and 2006.


The top 5 publications mentioning KCNJ9, ranked by their relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows. The most influential publication is "[Functional and biochemical evidence for G-protein-gated inwardly rectifying K+ (GIRK) channels composed of GIRK2 and GIRK3.](https://pubmed.ncbi.nlm.nih.gov/10956667)" (2000) with a relative citation ratio of 1.61. Another notable study is "[Evidence for oligomerization between GABAB receptors and GIRK channels containing the GIRK1 and GIRK3 subunits.](https://pubmed.ncbi.nlm.nih.gov/20846323)" (2010), which has a relative citation ratio of 1.19. Additionally, "[Axonal sorting of Kir3.3 defines a GABA-containing neuron in the CA3 region of rodent hippocampus.](https://pubmed.ncbi.nlm.nih.gov/14664820)" (2003) has a relative citation ratio of 0.46. The publication "[Analysis of linkage disequilibrium between polymorphisms in the KCNJ9 gene with type 2 diabetes mellitus in Pima Indians.](https://pubmed.ncbi.nlm.nih.gov/11350189)" (2001) has a relative citation ratio of 0.39, and "[Assignment of human G-protein-coupled inward rectifier K+ channel homolog GIRK3 gene to chromosome 1q21-q23.](https://pubmed.ncbi.nlm.nih.gov/8575783)" (1995) has a relative citation ratio of 0.27. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[KCNJ9](https://www.proteinatlas.org/ENSG00000162728-KCNJ9) is a gene encoding a protein with a molecular function as an ion channel, specifically a voltage-gated channel. Evidence for this function is supported at the protein level. The gene's RNA is detected in some tissues, with notable expression in the cerebellum, contributing to nervous system development. In the brain, its expression is non-specific and associated with mixed functions. Additionally, KCNJ9 is expressed in neuroblastoma cell lines and neurons, where it plays a role in neuronal signaling. The gene's prognostic significance in various cancers is generally unprognostic, as seen in studies on Breast Invasive Carcinoma and Glioblastoma Multiforme.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) and [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), each with 3 experiments. Additionally, [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), [STAT5A](https://www.ncbi.nlm.nih.gov/gene/6776), and [GATA2](https://www.ncbi.nlm.nih.gov/gene/2624) were each found to be regulating in 2 experiments.



The GWAS data indicates associations with various metabolic and nutritional conditions, including diseases of metabolism, acquired metabolic diseases, nutrition diseases, overnutrition, and obesity.



The gene is ubiquitously expressed in multiple tissue samples, including nerve, brain, and pituitary, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in various brain regions such as the Amygdala, Whole brain, Medulla Oblongata, Occipital Lobe, and Cerebellum, as well as in the Uterus Corpus and Globus Pallidus.


The input data highlights various pathways involving neuronal signaling and ion channel regulation. These include the activation of G protein gated potassium channels, inhibition of voltage-gated Ca2+ channels via Gbeta/gamma subunits, and the activation of GABAB receptors. Additionally, the data mentions inwardly rectifying K+ channels, GABA B receptor activation, GABA receptor activation, and potassium channels. These pathways are integral to the neuronal system, particularly in transmission across chemical synapses and neurotransmitter receptor binding and downstream transmission in the postsynaptic cell.



The analyzed protein sequence has a GRAVY value of -0.064 (80.51 percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is -20.69 (10.68 percentile), and the median structural flexibility is 0.995 (20.9 percentile). The protein has a propensity for helix, sheet, and turn structures of 32.06% (50.88 percentile), 38.42% (80.65 percentile), and 25.70% (26.74 percentile), respectively. The instability index is 41.56 (27.15 percentile), with an isoelectric point of 4.86 (5.36 percentile). The protein is 393 amino acids long (44.33 percentile) and has a molecular weight of 44019.37 Da (44.6 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |