---
layout: post
title: SNRPN
date: 2025-01-17 16:55 CST
description: SNRPN description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6638) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6638  | SNRPN | ENSG00000128739 | 15q11.2 |



The gene is involved in [mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0000398) and [RNA splicing](https://amigo.geneontology.org/amigo/term/GO:0008380), and plays a role in the [response to hormone](https://amigo.geneontology.org/amigo/term/GO:0009725). It enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654) and is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737). It is part of various complexes, including the [spliceosomal complex](https://amigo.geneontology.org/amigo/term/GO:0005681), [U5 snRNP](https://amigo.geneontology.org/amigo/term/GO:0005682), [U1 snRNP](https://amigo.geneontology.org/amigo/term/GO:0005685), [U2 snRNP](https://amigo.geneontology.org/amigo/term/GO:0005686), [U4 snRNP](https://amigo.geneontology.org/amigo/term/GO:0005687), [small nuclear ribonucleoprotein complex](https://amigo.geneontology.org/amigo/term/GO:0030532), [U4/U6 x U5 tri-snRNP complex](https://amigo.geneontology.org/amigo/term/GO:0046540), [U2-type prespliceosome](https://amigo.geneontology.org/amigo/term/GO:0071004), and [catalytic step 2 spliceosome](https://amigo.geneontology.org/amigo/term/GO:0071013).


The gene length is 246,133 base pairs (94.5% of all genes), the mature length is 4,365 base pairs, and the primary transcript length is 123,900 base pairs.


The gene SNRPN (NCBI ID: 6638) has been mentioned in [115 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SNRPN%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1990 and the middle 50% of publications occurring between 1996 and 2012. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning SNRPN, ranked by their scientific influence, include "[Deletions of a differentially methylated CpG island at the SNRPN gene define a putative imprinting control region.](https://pubmed.ncbi.nlm.nih.gov/7987392)" (1994) (relative citation ratio: 6.93), "[Maternal imprinting of the mouse Snrpn gene and conserved linkage homology with the human Prader-Willi syndrome region.](https://pubmed.ncbi.nlm.nih.gov/1303276)" (1992) (relative citation ratio: 4.97), "[The IC-SNURF-SNRPN transcript serves as a host for multiple small nucleolar RNA species and as an antisense RNA for UBE3A.](https://pubmed.ncbi.nlm.nih.gov/11726556)" (2001) (relative citation ratio: 4.53), "[Gene structure, DNA methylation, and imprinted expression of the human SNRPN gene.](https://pubmed.ncbi.nlm.nih.gov/8571960)" (1996) (relative citation ratio: 4.0), and "[Imprint switching on human chromosome 15 may involve alternative transcripts of the SNRPN gene.](https://pubmed.ncbi.nlm.nih.gov/8841186)" (1996) (relative citation ratio: 3.9). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[SNRPN](https://www.proteinatlas.org/ENSG00000128739-SNRPN) is a gene with a molecular function in RNA-binding as a ribonucleoprotein. It is detected in all tissues and is primarily located in the nucleoplasm. The gene is expressed in various clusters, including Cluster 30 for blood, Cluster 23 for tissue, Cluster 1 for brain, Cluster 34 for cell lines, and Cluster 3 for single cells, indicating its involvement in DNA binding, cellular respiration, mixed neuronal function, testis cancer, and lipid metabolism, respectively. Evidence for this gene is available at the protein level.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) and [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599), each with 4 experiments. Additionally, [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169), [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170), and [EP300](https://www.ncbi.nlm.nih.gov/gene/2033) were each found to be regulating in 3 experiments.



The input data from GWAS indicates associations with a wide range of disease conditions, including respiratory disorders such as asthma, bronchial disease, obstructive lung disease, lower respiratory tract disease, lung disease, and respiratory system disease. Additionally, metabolic conditions such as disease of metabolism, acquired metabolic disease, carbohydrate metabolism disease, glucose metabolism disease, diabetes mellitus, type 2 diabetes mellitus, nutrition disease, overnutrition, and obesity are also implicated. Cardiovascular issues, including coronary artery disease, artery disease, vascular disease, and cardiovascular system disease, are noted. Furthermore, the data highlights nervous system diseases, including central nervous system disease, neurodegenerative disease, Alzheimer's disease, tauopathy, dementia, cognitive disorder, and disease of mental health. Chronic obstructive pulmonary disease and cerebrovascular disease are also associated with the input data.



The gene is expressed in various tissues, including the thyroid, brain, and pituitary, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in CD8+ T cells, hypothalamus, trigeminal ganglion, skeletal muscle, atrioventricular node, globus pallidus, and ovary.




The protein sequence analyzed has a GRAVY value of -0.29 (59.13th percentile), indicating a relatively hydrophilic nature. It exhibits a charge of 17.66 at pH 7.0 (91.56th percentile) and a median structural flexibility of 1.007 (71.79th percentile). The protein's secondary structure is predicted to be 26.67% helix (10.29th percentile), 23.75% sheet (5.99th percentile), and 40.83% turn (97.76th percentile). The instability index is 51.40 (61.65th percentile), with an isoelectric point of 11.20 (99.28th percentile). The protein has a length of 240 amino acids (20.22nd percentile) and a molecular weight of 24613.78 Da (17.48th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |