---
layout: post
title: CALM3
date: 2025-01-17 16:55 CST
description: CALM3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/808) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 808  | CALM3 | ENSG00000160014 | 19q13.32 |



The gene is involved in the [G2/M transition of mitotic cell cycle](https://amigo.geneontology.org/amigo/term/GO:0000086) and is located in the [spindle pole](https://amigo.geneontology.org/amigo/term/GO:0000922). It plays a role in the [regulation of heart rate](https://amigo.geneontology.org/amigo/term/GO:0002027) and enables [calcium ion binding](https://amigo.geneontology.org/amigo/term/GO:0005509). Additionally, it is involved in the [detection of calcium ion](https://amigo.geneontology.org/amigo/term/GO:0005513) and enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is located in various cellular components, including the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [centrosome](https://amigo.geneontology.org/amigo/term/GO:0005813), [spindle microtubule](https://amigo.geneontology.org/amigo/term/GO:0005876), and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). It is also part of the [voltage-gated potassium channel complex](https://amigo.geneontology.org/amigo/term/GO:0008076) and enables [adenylate cyclase binding](https://amigo.geneontology.org/amigo/term/GO:0008179). The gene is involved in the [positive regulation of peptidyl-threonine phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0010800) and [negative regulation of peptidyl-threonine phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0010801), and enables [adenylate cyclase activator activity](https://amigo.geneontology.org/amigo/term/GO:0010856). Furthermore, it is involved in the [regulation of release of sequestered calcium ion into cytosol by sarcoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0010880) and [regulation of cardiac muscle contraction by regulation of the release of sequestered calcium ion](https://amigo.geneontology.org/amigo/term/GO:0010881). The gene enables [protein kinase binding](https://amigo.geneontology.org/amigo/term/GO:0019901) and [titin binding](https://amigo.geneontology.org/amigo/term/GO:0031432), and is involved in [substantia nigra development](https://amigo.geneontology.org/amigo/term/GO:0021762). It is located in the [sarcomere](https://amigo.geneontology.org/amigo/term/GO:0030017) and [vesicle](https://amigo.geneontology.org/amigo/term/GO:0031982), and is part of the [protein-containing complex](https://amigo.geneontology.org/amigo/term/GO:0032991) and [calcium channel complex](https://amigo.geneontology.org/amigo/term/GO:0034704). The gene is also located in the [myelin sheath](https://amigo.geneontology.org/amigo/term/GO:0043209) and [sperm midpiece](https://amigo.geneontology.org/amigo/term/GO:0097225). It enables [protein serine/threonine kinase activator activity](https://amigo.geneontology.org/amigo/term/GO:0043539) and [transmembrane transporter binding](https://amigo.geneontology.org/amigo/term/GO:0044325), and is involved in the [regulation of calcium-mediated signaling](https://amigo.geneontology.org/amigo/term/GO:0050848) and [response to calcium ion](https://amigo.geneontology.org/amigo/term/GO:0051592). The gene is involved in the [regulation of cardiac muscle contraction](https://amigo.geneontology.org/amigo/term/GO:0055117) and [negative regulation of ryanodine-sensitive calcium-release channel activity](https://amigo.geneontology.org/amigo/term/GO:0060315), as well as the [positive regulation of ryanodine-sensitive calcium-release channel activity](https://amigo.geneontology.org/amigo/term/GO:0060316). It is involved in the [positive regulation of protein serine/threonine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0071902) and enables [protein phosphatase activator activity](https://amigo.geneontology.org/amigo/term/GO:0072542). The gene is involved in the [regulation of cardiac muscle cell action potential](https://amigo.geneontology.org/amigo/term/GO:0098901) and the [negative regulation of high voltage-gated calcium channel activity](https://amigo.geneontology.org/amigo/term/GO:1901842). It is also involved in the [regulation of cell communication by electrical coupling involved in cardiac conduction](https://amigo.geneontology.org/amigo/term/GO:1901844) and is part of the [catalytic complex](https://amigo.geneontology.org/amigo/term/GO:1902494). Lastly, the gene is involved in the [negative regulation of calcium ion export across plasma membrane](https://amigo.geneontology.org/amigo/term/GO:1905913).


The gene length is 28,244 base pairs (51st percentile of all genes), the mature length is 2,030 base pairs, and the primary transcript length is 25,688 base pairs.


The gene CALM3 (NCBI ID: 808) has been mentioned in [1,002 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CALM3%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1799, and the middle 50% of publications range from 2006 to 2018.


The top 5 publications mentioning CALM3, ranked by their scientific influence, include [Evaluation of delirium in critically ill patients: validation of the Confusion Assessment Method for the Intensive Care Unit (CAM-ICU).](https://pubmed.ncbi.nlm.nih.gov/11445689) (2001) (relative citation ratio: 40.64), [A review of dental CAD/CAM: current status and future perspectives from 20 years of experience.](https://pubmed.ncbi.nlm.nih.gov/19280967) (2009) (relative citation ratio: 22.19), [Advancements in CAD/CAM technology: Options for practical implementation.](https://pubmed.ncbi.nlm.nih.gov/26935333) (2016) (relative citation ratio: 21.91), [Cam impingement causes osteoarthritis of the hip: a nationwide prospective cohort study (CHECK).](https://pubmed.ncbi.nlm.nih.gov/22730371) (2013) (relative citation ratio: 17.01), and [Cam and pincer femoroacetabular impingement: characteristic MR arthrographic findings in 50 patients.](https://pubmed.ncbi.nlm.nih.gov/16857978) (2006) (relative citation ratio: 13.99). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CALM3](https://www.proteinatlas.org/ENSG00000160014-CALM3) is a gene with evidence at the protein level, known for its involvement in Long QT syndrome. The gene's RNA is detected in all tissues, with notable expression in the brain, particularly in neurons involved in synaptic function. Additionally, CALM3 is part of blood expression cluster 25, which has non-specific and unknown functions, and tissue expression cluster 61, associated with brain neuronal signaling. In single-cell expression, it is found in late spermatids with unknown function.


The top transcription factors (TFs) identified from the GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025) with 6 experiments, followed by [KMT2B](https://www.ncbi.nlm.nih.gov/gene/9757), [HNF4A](https://www.ncbi.nlm.nih.gov/gene/3172), [USF1](https://www.ncbi.nlm.nih.gov/gene/7391), and [TCF12](https://www.ncbi.nlm.nih.gov/gene/6938), each with 5 experiments.





The gene is ubiquitously expressed in multiple tissue samples, including various regions of the brain such as the prefrontal cortex, amygdala, whole brain, cingulate cortex, subthalamic nucleus, temporal lobe, and globus pallidus, as well as in blood vessels.




The analyzed protein sequence has a GRAVY value of -0.654 (20.85th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -24.344 (8.45th percentile), and the median structural flexibility is 1.018 (95.2nd percentile). The protein has a high affinity for helix structures (39.6%, 92.64th percentile), with lower affinities for sheet (30.9%, 33.1st percentile) and turn (26.8%, 36.62nd percentile) structures. The instability index is 28.207 (2.78th percentile), suggesting a stable protein. The isoelectric point is 4.087 (0.22nd percentile), with a molecular weight of 16837.42 Da (8.13th percentile) and a length of 149 amino acids (7.97th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |