---
layout: post
title: CETN3
date: 2025-01-17 16:55 CST
description: CETN3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1070) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1070  | CETN3 | ENSG00000153140 | 5q14.3  |



The gene enables [calcium ion binding](https://amigo.geneontology.org/amigo/term/GO:0005509) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is involved in [microtubule binding](https://amigo.geneontology.org/amigo/term/GO:0008017) and [G-protein beta/gamma-subunit complex binding](https://amigo.geneontology.org/amigo/term/GO:0031683). It is located in the [nucleolus](https://amigo.geneontology.org/amigo/term/GO:0005730), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [centrosome](https://amigo.geneontology.org/amigo/term/GO:0005813), [centriole](https://amigo.geneontology.org/amigo/term/GO:0005814), [photoreceptor connecting cilium](https://amigo.geneontology.org/amigo/term/GO:0032391), and [ciliary basal body](https://amigo.geneontology.org/amigo/term/GO:0036064). Additionally, it is part of the [nuclear pore nuclear basket](https://amigo.geneontology.org/amigo/term/GO:0044615) and the [transcription export complex 2](https://amigo.geneontology.org/amigo/term/GO:0070390). The gene is active in the [microtubule organizing center](https://amigo.geneontology.org/amigo/term/GO:0005815) and is involved in the [centrosome cycle](https://amigo.geneontology.org/amigo/term/GO:0007098), [protein transport](https://amigo.geneontology.org/amigo/term/GO:0015031), [mRNA transport](https://amigo.geneontology.org/amigo/term/GO:0051028), and [cell division](https://amigo.geneontology.org/amigo/term/GO:0051301).


The gene length is 3,933 base pairs (10.98% of all genes), the mature length is 1,246.5 base pairs, and the primary transcript length is 3,905 base pairs.


The gene CETN3 (NCBI ID: 1070) has been mentioned in [9 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CETN3%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 2005 and the middle 50% of publications occurring between 2008 and 2017. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning CETN3, ranked by their scientific influence, include "[The Vps13p-Cdc31p complex is directly required for TGN late endosome transport and TGN homotypic fusion.](https://pubmed.ncbi.nlm.nih.gov/28122955)" (2017) (relative citation ratio: 2.22), "[The THP1-SAC3-SUS1-CDC31 complex works in transcription elongation-mRNA export preventing RNA-mediated genome instability.](https://pubmed.ncbi.nlm.nih.gov/18667528)" (2008) (relative citation ratio: 1.8), "[Calcium and magnesium binding to human centrin 3 and interaction with target peptides.](https://pubmed.ncbi.nlm.nih.gov/15654740)" (2005) (relative citation ratio: 0.94), "[Centrin/Cdc31 is a novel regulator of protein degradation.](https://pubmed.ncbi.nlm.nih.gov/18160718)" (2008) (relative citation ratio: 0.53), and "[Centrin 3 is an inhibitor of centrosomal Mps1 and antagonizes centrin 2 function.](https://pubmed.ncbi.nlm.nih.gov/26354417)" (2015) (relative citation ratio: 0.5). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CETN3](https://www.proteinatlas.org/ENSG00000153140-CETN3) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in the centrosome and basal body. The gene is expressed in various clusters, including Cluster 12 for non-specific mitochondrial translation in blood, Cluster 74 for flagellum assembly in spermatids, Cluster 28 for mixed function in oligodendrocytes in the brain, Cluster 22 for non-specific mRNA splicing and cell cycle in cell lines, and Cluster 31 for spermatogenesis in spermatocytes.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908), and [PGR](https://www.ncbi.nlm.nih.gov/gene/5241), each with 5 experiments. Additionally, [STAT1](https://www.ncbi.nlm.nih.gov/gene/6772) and [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) were found to be regulating in 4 experiments each.





The gene is expressed in the ovary and pituitary according to BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles indicate expression in the pineal gland (both day and night), testis, and various testicular substructures including interstitial tissue, Leydig cells, seminiferous tubules, and germ cells.




The analyzed protein sequence has a GRAVY value of -0.735 (15.45th percentile), indicating a hydrophilic nature. It exhibits a charge of -16.255 at pH 7.0 (14.23rd percentile) and a median structural flexibility of 1.017 (93.96th percentile). The protein's secondary structure is predicted to be predominantly helical (43.11%, 97.06th percentile), with notable contributions from sheet (32.34%, 41.53rd percentile) and turn (23.95%, 14.75th percentile) conformations. The instability index is 53.19 (67.13rd percentile), and the isoelectric point is 4.62 (2.27th percentile). The protein has a length of 167 amino acids (10.18th percentile) and a molecular weight of 19549.77 Da (10.98th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |