---
layout: post
title: NAPG
date: 2025-01-17 16:55 CST
description: NAPG description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8774) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8774  | NAPG | ENSG00000134265 | 18p11.22 |



The gene enables [soluble NSF attachment protein activity](https://amigo.geneontology.org/amigo/term/GO:0005483) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is part of the [SNARE complex](https://amigo.geneontology.org/amigo/term/GO:0031201). It is located in the [mitochondrion](https://amigo.geneontology.org/amigo/term/GO:0005739), [lysosomal membrane](https://amigo.geneontology.org/amigo/term/GO:0005765), [Golgi apparatus](https://amigo.geneontology.org/amigo/term/GO:0005794), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). The gene is involved in [intracellular protein transport](https://amigo.geneontology.org/amigo/term/GO:0006886), [intra-Golgi vesicle-mediated transport](https://amigo.geneontology.org/amigo/term/GO:0006891), [protein stabilization](https://amigo.geneontology.org/amigo/term/GO:0050821), [membrane fusion](https://amigo.geneontology.org/amigo/term/GO:0061025), and [protein-containing complex assembly](https://amigo.geneontology.org/amigo/term/GO:0065003). Additionally, it enables [syntaxin binding](https://amigo.geneontology.org/amigo/term/GO:0019905).


The gene length is 3,181 base pairs (8.84% of all genes), the mature length is 1,798.5 base pairs, and the primary transcript length is 2,239.5 base pairs.


The gene NAPG (NCBI ID: 8774) has been mentioned in [10 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22NAPG%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning this gene was in 1992, and the middle 50% of publications occurred between 2002 and 2014.


The top 5 publications mentioning NAPG, ranked by their relative citation ratio (RCR), include studies that explore various roles and implications of the gene. The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. The most influential publication is ["Roles of NapF, NapG and NapH, subunits of the Escherichia coli periplasmic nitrate reductase, in ubiquinol oxidation."](https://pubmed.ncbi.nlm.nih.gov/11967083) (2002) (relative citation ratio: 5.52). Other notable studies include ["Soybean Cyst Nematode Resistance Quantitative Trait Locus <i>cqSCN-006</i> Alters the Expression of a Î³-SNAP Protein."](https://pubmed.ncbi.nlm.nih.gov/34343024) (2021) (relative citation ratio: 0.99), ["Synaptosomal proteins, beta-soluble N-ethylmaleimide-sensitive factor attachment protein (beta-SNAP), gamma-SNAP and synaptotagmin I in brain of patients with Down syndrome and Alzheimer's disease."](https://pubmed.ncbi.nlm.nih.gov/11244216) (2001) (relative citation ratio: 0.94), ["The activity of Golgi transport vesicles depends on the presence of the N-ethylmaleimide-sensitive factor (NSF) and a soluble NSF attachment protein (alpha SNAP) during vesicle formation."](https://pubmed.ncbi.nlm.nih.gov/1522110) (1992) (relative citation ratio: 0.73), and ["NAPG mutation in family members with hereditary hemorrhagic telangiectasia in China."](https://pubmed.ncbi.nlm.nih.gov/34112136) (2021) (relative citation ratio: 0.45). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[NAPG](https://www.proteinatlas.org/ENSG00000134265-NAPG) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in the plasma membrane. The gene is expressed in eosinophils with unknown function (Cluster 45) and is associated with oligodendrocytes involved in myelination (Cluster 9). In the brain, it is found in the hindbrain with mixed function (Cluster 49). The gene is also expressed in non-specific cell lines and single cells with mixed function (Cluster 13 and Cluster 45, respectively).


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [RELA](https://www.ncbi.nlm.nih.gov/gene/5970) and [MYC](https://www.ncbi.nlm.nih.gov/gene/4609), each with 5 experiments. Additionally, [PPARG](https://www.ncbi.nlm.nih.gov/gene/5468), [KDM5B](https://www.ncbi.nlm.nih.gov/gene/10765), and [ZBTB7A](https://www.ncbi.nlm.nih.gov/gene/51341) were each found to be regulating in 4 experiments.





The gene is expressed in various neural tissues, including the nerve, brain, and adrenal gland. Additionally, it is found in several specific cell types and tissues such as 721_B_lymphoblasts, Leukemia_chronicMyelogenousK-562, SuperiorCervicalGanglion, AtrioventricularNode, CiliaryGanglion, Skin, and DorsalRootGanglion.




The protein sequence analyzed has a GRAVY value of -0.559 (28.99th percentile), indicating a hydrophilic nature. The charge at pH 7 is -9.06 (24.15th percentile), and the median structural flexibility is 1.007 (70.52nd percentile). The protein has a high affinity for helix structures (0.446, 98.04th percentile), with lower affinities for sheet (0.282, 19.31st percentile) and turn (0.234, 11.86th percentile) structures. The instability index is 36.97 (14.08th percentile), and the isoelectric point is 5.30 (13.49th percentile). The protein is 312 amino acids long (30.84th percentile) with a molecular weight of 34745.90 Da (30.55th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |