---
layout: post
title: TNFRSF10B
date: 2025-01-17 16:55 CST
description: TNFRSF10B description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8795) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8795  | TNFRSF10B | ENSG00000120889 | 8p21.3 |



The gene is involved in various biological processes, including [defense response to tumor cell](https://amigo.geneontology.org/amigo/term/GO:0002357), [apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0006915), [cell surface receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007166), and [activation of NF-kappaB-inducing kinase activity](https://amigo.geneontology.org/amigo/term/GO:0007250). It is active in and located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886) and the [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986). The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [TRAIL receptor activity](https://amigo.geneontology.org/amigo/term/GO:0036463), and [signaling receptor activity](https://amigo.geneontology.org/amigo/term/GO:0038023). Additionally, it is involved in [response to endoplasmic reticulum stress](https://amigo.geneontology.org/amigo/term/GO:0034976), [TRAIL-activated apoptotic signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0036462), [regulation of apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0042981), [positive regulation of apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0043065), [positive regulation of canonical NF-kappaB signal transduction](https://amigo.geneontology.org/amigo/term/GO:0043123), and [cellular response to mechanical stimulus](https://amigo.geneontology.org/amigo/term/GO:0071260). The gene also enables [TRAIL binding](https://amigo.geneontology.org/amigo/term/GO:0045569) and is involved in the [intrinsic apoptotic signaling pathway in response to endoplasmic reticulum stress](https://amigo.geneontology.org/amigo/term/GO:0070059).


The gene length is 73,285 base pairs (75.55% of all genes), the mature length is 1,896 base pairs, and the primary transcript length is 71,210 base pairs.


The gene TNFRSF10B (NCBI ID: 8795) has been mentioned in [631 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22TNFRSF10B%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1982, and the middle 50% of publications occurred between 2007 and 2017.


The top 5 publications mentioning TNFRSF10B, ranked by their scientific influence, include "[KILLER/DR5 is a DNA damage-inducible p53-regulated death receptor gene.](https://pubmed.ncbi.nlm.nih.gov/9326928)" (1997) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 16.7, "[CHOP is involved in endoplasmic reticulum stress-induced apoptosis by enhancing DR5 expression in human carcinoma cells.](https://pubmed.ncbi.nlm.nih.gov/15322075)" (2004) with an RCR of 12.2, "[Opposing unfolded-protein-response signals converge on death receptor 5 to control apoptosis.](https://pubmed.ncbi.nlm.nih.gov/24994655)" (2014) with an RCR of 12.07, "[TRAIL receptors 1 (DR4) and 2 (DR5) signal FADD-dependent apoptosis and activate NF-kappaB.](https://pubmed.ncbi.nlm.nih.gov/9430228)" (1997) with an RCR of 10.92, and "[Death receptor 5, a new member of the TNFR family, and DR4 induce FADD-dependent apoptosis and activate the NF-kappaB pathway.](https://pubmed.ncbi.nlm.nih.gov/9430227)" (1997) with an RCR of 10.84. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[TNFRSF10B](https://www.proteinatlas.org/ENSG00000120889-TNFRSF10B) is a receptor involved in cancer-related genes, with evidence at the protein level. Its RNA is detected in all tissues, and it is expressed in various clusters, including Cluster 26 in monocytes' plasma membrane proteins, Cluster 11 in liver plasma proteins, Cluster 12 in brain vasculature, Cluster 53 in squamous epithelial cells related to keratinization, and Cluster 29 in monocytes involved in the innate immune response.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313) with 5 experiments, [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599) with 4 experiments, [KDM1A](https://www.ncbi.nlm.nih.gov/gene/23028) with 4 experiments, [MYC](https://www.ncbi.nlm.nih.gov/gene/4609) with 4 experiments, and [SPI1](https://www.ncbi.nlm.nih.gov/gene/6688) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with a range of metabolic disorders, including diseases of metabolism, acquired metabolic diseases, carbohydrate metabolism diseases, and glucose metabolism diseases. Specifically, diabetes mellitus and type 2 diabetes mellitus are highlighted. Additionally, the data points to associations with various cardiovascular conditions such as artery disease, vascular disease, and cardiovascular system disease. Mental health conditions, including cognitive disorders, bipolar disorder, and mood disorders, are also implicated. Furthermore, the data suggests links to hypertension, inherited metabolic disorders, and reproductive system diseases, particularly female reproductive system diseases like ovarian disease and premature ovarian failure. Lastly, metal metabolism disorders are also noted.


The gene TNFRSF10B has been associated with several traits and diseases through genome-wide association studies. Specifically, it has been linked to [Alanine aminotransferase levels](https://pubmed.ncbi.nlm.nih.gov/34226706), which maps to the MeSH term [Transaminases](https://meshb.nlm.nih.gov/record/ui?ui=D000637) using the BioWordVec method. Additionally, it is associated with [Aspartate aminotransferase levels](https://pubmed.ncbi.nlm.nih.gov/34226706), also mapping to [Transaminases](https://meshb.nlm.nih.gov/record/ui?ui=D000637), and [Gamma glutamyl transferase levels](https://pubmed.ncbi.nlm.nih.gov/34226706), which maps to [Transferases](https://meshb.nlm.nih.gov/record/ui?ui=D014166). Furthermore, TNFRSF10B is linked to [TNF-related apoptosis-inducing ligand receptor 2 levels](https://pubmed.ncbi.nlm.nih.gov/36349687), mapping to the MeSH term [Ligands](https://meshb.nlm.nih.gov/record/ui?ui=D008024). These associations were determined using the BioWordVec method.


The gene is expressed in various tissues and cell types, including skin and adrenal gland, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles show expression in cardiac myocytes, Burkitt's lymphoma (Raji), 721 B lymphoblasts, CD33+ myeloid cells, smooth muscle, bronchial epithelial cells, and colorectal adenocarcinoma.


The input data highlights several key pathways and processes related to cell death, including apoptosis, programmed cell death, caspase-8 activation by cleavage, dimerization of procaspase-8, regulation by c-FLIP, death receptor signaling, TRAIL signaling, and the extrinsic pathway. These pathways are integral to the regulation and execution of cellular death mechanisms.



The analyzed protein sequence has a GRAVY value of -0.409 (45.11th percentile), indicating a hydrophilic nature. The charge at pH 7 is -12.7 (18.36th percentile), and the median structural flexibility is 1.008 (74.43rd percentile). The protein has a high instability index of 58.2 (79.51st percentile) and an isoelectric point of 5.39 (15.51st percentile). The sequence length is 440 amino acids (50.68th percentile) with a molecular weight of 47877.8 Da (49.37th percentile). The secondary structure affinities are 32.3% for helix (52.98th percentile), 29.3% for sheet (24.84th percentile), and 30.5% for turn (67.17th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |