---
layout: post
title: ENO2
date: 2024-12-08 23:57 CST
description: ENO2 description
tags: [cooccuring-genes]
categories: shortest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2026) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2026  | ENO2 | ENSG00000111674 | 12p13.31 |



The gene length is 731.0 base pairs (0.91 percentile of all genes), with a mature length of 731.0 base pairs and a primary transcript length of 731.0 base pairs.


The gene ENO2 (NCBI ID: 2026) has been the subject of [1,297 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ENO2%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1978, and the middle 50% of publications occurred between 1992 and 2016.


The top five publications mentioning ENO2, ranked by their scientific influence, include "[Neurone-specific enolase is a molecular marker for peripheral and central neuroendocrine cells.](https://pubmed.ncbi.nlm.nih.gov/31568)" (1978) (relative citation ratio: 30.12), "[Neuron-specific enolase is produced by neuroendocrine tumours.](https://pubmed.ncbi.nlm.nih.gov/6111674)" (1981) (relative citation ratio: 21.25), "[S-100 protein and neuron-specific enolase in cerebrospinal fluid and serum: markers of cell damage in human central nervous system.](https://pubmed.ncbi.nlm.nih.gov/3629651)" (1987) (relative citation ratio: 16.07), "[Immunohistochemical demonstration of neuron-specific enolase in neoplasms of the CNS and other tissues.](https://pubmed.ncbi.nlm.nih.gov/6547318)" (1984) (relative citation ratio: 15.97), and "[Immunohistochemical localization of gamma-enolase in normal human tissues other than nervous and neuroendocrine tissues.](https://pubmed.ncbi.nlm.nih.gov/3974199)" (1985) (relative citation ratio: 13.69). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [RELA](https://www.ncbi.nlm.nih.gov/gene/5970) and [MYC](https://www.ncbi.nlm.nih.gov/gene/4609), each with 6 experiments. Additionally, [EZH2](https://www.ncbi.nlm.nih.gov/gene/2146), [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046), and [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) were each found to be regulating in 5 experiments.











The protein sequence analyzed has a GRAVY value of -0.188 (70.39th percentile), indicating a hydrophilic nature. It exhibits a charge of -17.95 at pH 7.0 (12.66th percentile) and a median structural flexibility of 1.003 (55.58th percentile). The protein's secondary structure is predicted to be 35.25% helix (76.29th percentile), 34.33% sheet (54.32nd percentile), and 29.49% turn (59.99th percentile). The instability index is 35.30 (10.76th percentile), with an isoelectric point of 4.91 (6.15th percentile). The protein has a length of 434 amino acids (49.95th percentile) and a molecular weight of 47268.01 Da (48.65th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |