---
layout: post
title: SPTAN1
date: 2024-12-08 23:57 CST
description: SPTAN1 description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6709) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6709  | SPTAN1 | ENSG00000197694 | 9q34.11 |



The gene length is 15,304 base pairs (35.49% of all genes), the mature length is 2,013 base pairs, and the primary transcript length is 15,271 base pairs.


The gene SPTAN1 (NCBI ID: 6709) has been the subject of [123 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SPTAN1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication dating back to 1984. The middle 50% of these publications were published between 2003 and 2015.


The top five publications mentioning SPTAN1, ranked by their scientific influence, are as follows: ["Caspase-3 is required for alpha-fodrin cleavage but dispensable for cleavage of other death substrates in apoptosis."](https://pubmed.ncbi.nlm.nih.gov/9624143) (1998) (relative citation ratio: 8.17), ["Identification of alpha-fodrin as a candidate autoantigen in primary Sjögren's syndrome."](https://pubmed.ncbi.nlm.nih.gov/9110981) (1997) (relative citation ratio: 7.45), ["αII-spectrin breakdown products (SBDPs): diagnosis and outcome in severe traumatic brain injury patients."](https://pubmed.ncbi.nlm.nih.gov/20408766) (2010) (relative citation ratio: 4.37), ["The calmodulin-binding site in alpha-fodrin is near the calcium-dependent protease-I cleavage site."](https://pubmed.ncbi.nlm.nih.gov/2844821) (1988) (relative citation ratio: 4.33), and ["Serum concentrations of ubiquitin C-terminal hydrolase-L1 and αII-spectrin breakdown product 145 kDa correlate with outcome after pediatric TBI."](https://pubmed.ncbi.nlm.nih.gov/22022780) (2012) (relative citation ratio: 4.24). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[Actin capping, Actin-binding, Calmodulin-binding](https://www.proteinatlas.org/[Ensembl]-[Gene]) is a gene with multiple molecular functions, including actin capping, actin-binding, and calmodulin-binding. It has been implicated in various diseases such as epilepsy, hereditary spastic paraplegia, intellectual disability, neurodegeneration, and neuropathy. Evidence for this gene is available at the protein level. Its RNA is detected in all tissues, and it is localized in vesicles and microtubules. The gene is part of several expression clusters, including Cluster 16 for non-specific cell proliferation in blood, Cluster 28 for neuronal transcription in tissues, Cluster 24 for mixed function in the brain, Cluster 72 for neuronal signal transduction in cell lines, and Cluster 71 for horizontal and bipolar cells in single-cell expression.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [MAX](https://www.ncbi.nlm.nih.gov/gene/4149), [ZBTB7A](https://www.ncbi.nlm.nih.gov/gene/51341), [SP1](https://www.ncbi.nlm.nih.gov/gene/6667), [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), and [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170), each shown to be regulating in 6 experiments.


The top transcription factors (TFs) identified in terms of the number of Perturb-seq experiments are [6829](https://www.ncbi.nlm.nih.gov/gene/6829) with 2 experiments and [23028](https://www.ncbi.nlm.nih.gov/gene/23028) with 3 experiments. These TFs were found to regulate gene expression in multiple experiments, highlighting their significant role in transcriptional regulation.





The mouse ortholog gene 20740 and its human ortholog 6709 are associated with various abnormal phenotypes observed in both sexes. These include abnormal craniofacial morphology, abnormal eye morphology, abnormal forebrain development, abnormal head shape, abnormal kidney morphology, abnormal midbrain development, abnormal neural tube closure, abnormal neural tube morphology, abnormal placenta morphology, abnormal retina vasculature morphology, abnormal visceral yolk sac morphology, abnormal vitreous body morphology, cataract, decreased body length, edema, embryonic growth retardation, enlarged kidney, microcephaly, microphthalmia, persistence of hyaloid vascular system, and preweaning lethality with complete penetrance.


The analyzed protein sequence has a GRAVY value of -0.7903 (12.12th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -106.70 (0.49th percentile), and the median structural flexibility is 1.01 (81.02nd percentile). The protein has a high affinity for helix structures (42.39%, 96.45th percentile), with lower affinities for sheet (29.13%, 23.82nd percentile) and turn (22.13%, 6.87th percentile) structures. The instability index is 42.60 (30.75th percentile), and the isoelectric point is 5.22 (11.85th percentile). The protein is relatively long with 2472 amino acids (98.58th percentile) and has a molecular weight of 284535.93 Da (98.73rd percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |