---
layout: post
title: ABLIM1
date: 2025-01-17 16:55 CST
description: ABLIM1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3983) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3983  | ABLIM1 | ENSG00000099204 | 10q25.3 |



The gene is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737) and is active in the [stress fiber](https://amigo.geneontology.org/amigo/term/GO:0001725) and [actin cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0015629). It enables [actin binding](https://amigo.geneontology.org/amigo/term/GO:0003779), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [actin filament binding](https://amigo.geneontology.org/amigo/term/GO:0051015), and [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872). The gene is involved in various processes including [cytoskeleton organization](https://amigo.geneontology.org/amigo/term/GO:0007010), [visual perception](https://amigo.geneontology.org/amigo/term/GO:0007601), [animal organ morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0009887), [lamellipodium assembly](https://amigo.geneontology.org/amigo/term/GO:0030032), and [cilium assembly](https://amigo.geneontology.org/amigo/term/GO:0060271). Additionally, it is located in the [lamellipodium](https://amigo.geneontology.org/amigo/term/GO:0030027) and acts upstream of or within [lamellipodium assembly](https://amigo.geneontology.org/amigo/term/GO:0030032) and [cilium assembly](https://amigo.geneontology.org/amigo/term/GO:0060271).


The gene has a length of 27,864 base pairs (50.64% of all genes), a mature length of 489 base pairs, and a primary transcript length of 23,633 base pairs.


The gene ABLIM1 (NCBI ID: 3983) has been mentioned in [11 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ABLIM1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink). The earliest publication mentioning this gene was in 1997, and the middle 50% of publications occurred between 2006 and 2018.


The top 5 publications mentioning ABLIM1, ranked by their scientific influence, include studies that explore the gene's role in various biological processes. The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. The most influential publication is "[The netrin receptor UNC-40/DCC stimulates axon attraction and outgrowth through enabled and, in parallel, Rac and UNC-115/AbLIM.](https://pubmed.ncbi.nlm.nih.gov/12526772)" (2003) (relative citation ratio: 2.74). Another notable study is "[Molecular characterization of abLIM, a novel actin-binding and double zinc finger protein.](https://pubmed.ncbi.nlm.nih.gov/9245787)" (1997) (relative citation ratio: 1.87). Additionally, "[LncRNA <i>ZNF667-AS1</i> Promotes <i>ABLIM1</i> Expression by Adsorbing <i>micro</i> <i>RNA-1290</i> to Suppress Nasopharyngeal Carcinoma Cell Progression.](https://pubmed.ncbi.nlm.nih.gov/32606725)" (2020) (relative citation ratio: 1.13) and "[The Arp2/3 complex, UNC-115/abLIM, and UNC-34/Enabled regulate axon guidance and growth cone filopodia formation in Caenorhabditis elegans.](https://pubmed.ncbi.nlm.nih.gov/19799769)" (2009) (relative citation ratio: 0.94) are also significant. Lastly, "[ABLIM1 splicing is abnormal in skeletal muscle of patients with DM1 and regulated by MBNL, CELF and PTBP1.](https://pubmed.ncbi.nlm.nih.gov/25403273)" (2015) (relative citation ratio: 0.56) contributes to the understanding of ABLIM1. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ABLIM1](https://www.proteinatlas.org/ENSG00000099204-ABLIM1) is an actin-binding protein with evidence at the protein level. It is detected in all tissues and is primarily localized in the cytosol, connecting piece, and mid piece. The gene is expressed in various clusters, including Cluster 43 for non-specific transcription and translation in blood, Cluster 12 for heart muscle and cardiac muscle contraction in tissues, Cluster 35 for non-specific DNA binding in the brain, Cluster 15 for kidney cancer membrane components in cell lines, and Cluster 47 for astrocytes with unknown function in single cells.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [HNF4A](https://www.ncbi.nlm.nih.gov/gene/3172) with 6 experiments, [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) with 5 experiments, [AR](https://www.ncbi.nlm.nih.gov/gene/367) with 5 experiments, [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599) with 5 experiments, and [SMARCA2](https://www.ncbi.nlm.nih.gov/gene/6595) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The input data from GWAS indicates associations with a wide range of disease conditions, including various types of cancer such as male reproductive organ cancer, prostate cancer, lung cancer, and urinary system cancer. Additionally, respiratory system diseases like asthma, bronchial disease, and obstructive lung disease are noted. Metabolic and cardiovascular conditions, including obesity, coronary artery disease, and heart disease, are also represented. Neurodegenerative diseases such as Alzheimer's disease, tauopathy, and dementia, along with mental health disorders like bipolar disorder, major depressive disorder, and mood disorder, are included. Furthermore, gastrointestinal system diseases like intestinal disease and colonic disease, as well as urinary system diseases like kidney disease and bladder carcinoma, are part of the dataset.



The gene is expressed in various tissues, including the heart and ovary, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in pineal gland (night), CD4+ T cells, CD8+ T cells, CD19+ B cells (negatively selected), cerebellum, cerebellum peduncles, colorectal adenocarcinoma, and tongue.


The proteins are involved in several pathways, including axon guidance, developmental biology, DCC-mediated attractive signaling, and netrin-1 signaling.


The mouse ortholog gene 226251 and its human ortholog gene 3983 are associated with a decreased thermal nociceptive threshold. This phenotype was observed in a sex-specific manner.


The analyzed protein sequence has a GRAVY value of -0.716 (16.67th percentile), indicating a hydrophilic nature. It exhibits a charge of 23.8 at pH 7.0 (95.33rd percentile) and a median structural flexibility of 1.004 (61.53rd percentile). The protein's secondary structure is predicted to be 26.99% helix (11.69th percentile), 28.53% sheet (20.82nd percentile), and 31.88% turn (76.04th percentile). Additionally, the protein has an instability index of 58.81 (80.78th percentile), an isoelectric point of 8.91 (76.78th percentile), a length of 778 amino acids (79.62nd percentile), and a molecular weight of 87686.63 Da (79.92nd percentile). These calculations were performed using BioPython's sequence analysis tools, which can be explored further [here](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |