---
layout: post
title: AHCYL1
date: 2025-01-17 16:55 CST
description: AHCYL1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10768) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10768  | AHCYL1 | ENSG00000168710 | 1p13.3 |



The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and it does not enable [adenosylhomocysteinase activity](https://amigo.geneontology.org/amigo/term/GO:0004013). It is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0005789), [apical plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0016324), [mitochondria-associated endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0042802), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062), and is active in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). The gene is involved in various processes including [protein export from nucleus](https://amigo.geneontology.org/amigo/term/GO:0006611), [one-carbon metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006730), [apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0006915), [positive regulation of sodium ion transport](https://amigo.geneontology.org/amigo/term/GO:0010765), [regulation of mRNA 3'-end processing](https://amigo.geneontology.org/amigo/term/GO:0031440), [S-adenosylmethionine cycle](https://amigo.geneontology.org/amigo/term/GO:0033353), [angiotensin-activated signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0038166), [epithelial fluid transport](https://amigo.geneontology.org/amigo/term/GO:0042045), [regulation of monoatomic ion transmembrane transporter activity](https://amigo.geneontology.org/amigo/term/GO:0032412), [regulation of monoatomic anion transport](https://amigo.geneontology.org/amigo/term/GO:0044070), [response to calcium ion](https://amigo.geneontology.org/amigo/term/GO:0051592), and [mitochondrion-endoplasmic reticulum membrane tethering](https://amigo.geneontology.org/amigo/term/GO:1990456). Additionally, it enables [enzyme regulator activity](https://amigo.geneontology.org/amigo/term/GO:0030234) and [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802).


The gene length is 55,143 base pairs (69.04% of all genes), the mature length is 6,012.5 base pairs (94.22% of all genes), and the primary transcript length is 28,597 base pairs (61.44% of all genes).


The gene AHCYL1 (NCBI ID: 10768) has been mentioned in [32 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22AHCYL1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 2006 and the middle 50% of publications occurring between 2011 and 2019. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning AHCYL1, ranked by their scientific influence as measured by the [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541), include: ["IRBIT suppresses IP3 receptor activity by competing with IP3 for the common binding site on the IP3 receptor."](https://pubmed.ncbi.nlm.nih.gov/16793548) (2006) (RCR: 2.72), ["AHCYL1 senses SAH to inhibit autophagy through interaction with PIK3C3 in an MTORC1-independent manner."](https://pubmed.ncbi.nlm.nih.gov/33993848) (2022) (RCR: 2.12), ["IRBIT controls apoptosis by interacting with the Bcl-2 homolog, Bcl2l10, and by promoting ER-mitochondria contact."](https://pubmed.ncbi.nlm.nih.gov/27995898) (2016) (RCR: 1.83), ["Convergence of IRBIT, phosphatidylinositol (4,5) bisphosphate, and WNK/SPAK kinases in regulation of the Na+-HCO3- cotransporters family."](https://pubmed.ncbi.nlm.nih.gov/23431199) (2013) (RCR: 1.82), and ["Activation of Na+/H+ exchanger NHE3 by angiotensin II is mediated by inositol 1,4,5-triphosphate (IP3) receptor-binding protein released with IP3 (IRBIT) and Ca2+/calmodulin-dependent protein kinase II."](https://pubmed.ncbi.nlm.nih.gov/20584908) (2010) (RCR: 1.7). The RCR values measure the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[AHCYL1](https://www.proteinatlas.org/ENSG00000168710-AHCYL1) is an RNA-binding protein with evidence at the protein level. It is detected in all RNA tissue distributions and is primarily located in the cytosol. The gene is expressed in various clusters, including Cluster 2 in dendritic cells with unknown function, Cluster 60 in choroid plexus associated with transmembrane transport, Cluster 44 in the brain with mixed function, Cluster 7 in liver cancer related to metabolism, and Cluster 37 in glandular and luminal cells with unknown function.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [TAF1](https://www.ncbi.nlm.nih.gov/gene/6872), [ZBTB48](https://www.ncbi.nlm.nih.gov/gene/3104), [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), [ZBTB7A](https://www.ncbi.nlm.nih.gov/gene/51341), and [GATA2](https://www.ncbi.nlm.nih.gov/gene/2624), each shown to be regulating in 5 experiments.





The gene is expressed in various brain regions, including the prefrontal cortex, caudate nucleus, and hypothalamus, as well as in the spinal cord. Additionally, it is expressed in the pituitary and kidney.



The mouse ortholog gene 229709 and its human ortholog gene 10768 are associated with the phenotype of increased circulating alkaline phosphatase level, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.265 (62.02 percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -1.92 (43.48 percentile). The median structural flexibility is 1.001 (45.13 percentile), with a predicted secondary structure composition of 32.1% helix (51.04 percentile), 35.1% sheet (59.84 percentile), and 26.6% turn (34.57 percentile). The instability index is 44.9 (38.81 percentile), and the isoelectric point is 6.49 (42.3 percentile). The protein is 530 amino acids long (61.91 percentile) with a molecular weight of 58950.8 Da (61.51 percentile). For detailed sequence analysis methods, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |