---
layout: post
title: ATP5PO
date: 2025-01-17 16:55 CST
description: ATP5PO description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/539) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 539  | ATP5PO | ENSG00000241837 | 21q22.11 |



The gene is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [mitochondrion](https://amigo.geneontology.org/amigo/term/GO:0005739), [mitochondrial inner membrane](https://amigo.geneontology.org/amigo/term/GO:0005743), and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). It is part of the [mitochondrial proton-transporting ATP synthase, stator stalk](https://amigo.geneontology.org/amigo/term/GO:0000274), [mitochondrial proton-transporting ATP synthase complex](https://amigo.geneontology.org/amigo/term/GO:0005753), and [proton-transporting ATP synthase complex, catalytic core F(1)](https://amigo.geneontology.org/amigo/term/GO:0045261). The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and contributes to [proton-transporting ATP synthase activity, rotational mechanism](https://amigo.geneontology.org/amigo/term/GO:0046933). It is involved in the [ATP biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006754), [proton motive force-driven ATP synthesis](https://amigo.geneontology.org/amigo/term/GO:0015986), [proton motive force-driven mitochondrial ATP synthesis](https://amigo.geneontology.org/amigo/term/GO:0042776), and [proton transmembrane transport](https://amigo.geneontology.org/amigo/term/GO:1902600).


The gene length is 28,309.0 base pairs (51.08% of all genes), the mature length is 5,909.5 base pairs, and the primary transcript length is 28,262.5 base pairs.


The gene ATP5PO (NCBI ID: 539) has been mentioned in [14 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ATP5PO%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning this gene was in 1972, and the middle 50% of publications occurred between 1986 and 2018.


The top 5 publications mentioning ATP5PO are as follows: ["Deregulation of mitochondrial F1FO-ATP synthase via OSCP in Alzheimer's disease."](https://pubmed.ncbi.nlm.nih.gov/27151236) (2016) (relative citation ratio: 4.57), ["The unique histidine in OSCP subunit of F-ATP synthase mediates inhibition of the permeability transition pore by acidic pH."](https://pubmed.ncbi.nlm.nih.gov/29217657) (2018) (relative citation ratio: 3.94), ["Cyclophilin D deficiency attenuates mitochondrial F1Fo ATP synthase dysfunction via OSCP in Alzheimer's disease."](https://pubmed.ncbi.nlm.nih.gov/30266287) (2019) (relative citation ratio: 2.06), ["The oligomycin axis of mitochondrial ATP synthase: OSCP and the proton channel."](https://pubmed.ncbi.nlm.nih.gov/15254386) (2000) (relative citation ratio: 1.69), and ["Claudin-10 overexpression suppresses human clear cell renal cell carcinoma growth and metastasis by regulating ATP5O and causing mitochondrial dysfunction."](https://pubmed.ncbi.nlm.nih.gov/35414767) (2022) (relative citation ratio: 1.54). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, and benchmarking to the median for NIH publications, the value of which is set at 1.0. Citation counts are from [iCite](https://icite.od.nih.gov).


[ATP5PO](https://www.proteinatlas.org/ENSG00000241837-ATP5PO) is a gene with evidence at the protein level, implicated in primary mitochondrial disease as a disease variant. It is detected in all tissues and is primarily located in the mitochondria. The gene is associated with non-specific mitochondrial translation in blood expression, non-specific mitochondrial processes in tissue expression, non-specific metabolism in brain expression, leukemia-related hemostasis in cell line expression, and basic cellular processes in single-cell expression.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [STAT1](https://www.ncbi.nlm.nih.gov/gene/6772) with 5 experiments, [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 4 experiments, [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930) with 4 experiments, [CEBPA](https://www.ncbi.nlm.nih.gov/gene/1050) with 4 experiments, and [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613) with 4 experiments.




The gene ATP5PO has been associated with [Hemoglobin levels](https://pubmed.ncbi.nlm.nih.gov/32327693), which maps closely to the MeSH term [Hemoglobins](https://meshb.nlm.nih.gov/record/ui?ui=D006454) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method.


The gene is expressed in various tissues, including the heart, muscle, and kidney, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Leukemia (chronic myelogenous K-562 and promyelocytic HL-60), CD105+ endothelial cells, skeletal muscle, and tongue.


The input data highlights several key metabolic processes, including the formation of ATP by chemiosmotic coupling, respiratory electron transport, and the citric acid (TCA) cycle. These processes are integral to cellular metabolism and energy production.


The mouse ortholog gene 28080 and its human ortholog gene 539 are associated with phenotypes including embryonic lethality prior to organogenesis, which is not observed in both sexes, and preweaning lethality with complete penetrance, which is observed in both sexes.


The protein sequence analyzed has a GRAVY value of -0.0197 (83.06th percentile), indicating a hydrophilic nature. It exhibits a charge of 13.59 at pH 7 (87.39th percentile) and a median structural flexibility of 1.001 (47.36th percentile). The protein's secondary structure is predicted to be 37.09% helix (85.55th percentile), 37.56% sheet (76.16th percentile), and 24.41% turn (17.66th percentile). The instability index is 34.33 (8.98th percentile), with an isoelectric point of 9.97 (96.25th percentile). The sequence length is 213 amino acids (16.45th percentile), and the molecular weight is 23277.04 Da (15.68th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |