---
layout: post
title: RBM39
date: 2025-01-17 16:55 CST
description: RBM39 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9584) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9584  | RBM39 | ENSG00000131051 | 20q11.22 |



The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is involved in [RNA processing](https://amigo.geneontology.org/amigo/term/GO:0006396), [mRNA processing](https://amigo.geneontology.org/amigo/term/GO:0006397), and [RNA splicing](https://amigo.geneontology.org/amigo/term/GO:0008380). It is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [microtubule cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0015630), [nuclear speck](https://amigo.geneontology.org/amigo/term/GO:0016607), and [centriolar satellite](https://amigo.geneontology.org/amigo/term/GO:0034451), and is part of a [protein-containing complex](https://amigo.geneontology.org/amigo/term/GO:0032991). Additionally, it is involved in the [regulation of mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0048024), and enables [RS domain binding](https://amigo.geneontology.org/amigo/term/GO:0050733) and [U1 snRNP binding](https://amigo.geneontology.org/amigo/term/GO:1990446).


The gene length is 2,533 base pairs (6.82% of all genes). The mature length is 2,533 base pairs (64.48% of all genes). The primary transcript length is 2,533 base pairs (8.75% of all genes).


The gene RBM39 (NCBI ID: 9584) has been mentioned in [26 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RBM39%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 2008, and the middle 50% of publications occurred between 2016 and 2020.


The top 5 publications mentioning RBM39, ranked by their relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)), include studies that explore the gene's role in cancer therapeutics and molecular interactions. The most influential publication is "[Anticancer sulfonamides target splicing by inducing RBM39 degradation via recruitment to DCAF15.](https://pubmed.ncbi.nlm.nih.gov/28302793)" (2017) (relative citation ratio: 11.67), followed by "[Selective degradation of splicing factor CAPERÎ± by anticancer sulfonamides.](https://pubmed.ncbi.nlm.nih.gov/28437394)" (2017) (relative citation ratio: 7.36). Other notable works include "[Structural basis of indisulam-mediated RBM39 recruitment to DCAF15 E3 ligase complex.](https://pubmed.ncbi.nlm.nih.gov/31819272)" (2020) (relative citation ratio: 6.39), "[Structural complementarity facilitates E7820-mediated degradation of RBM39 by DCAF15.](https://pubmed.ncbi.nlm.nih.gov/31686031)" (2020) (relative citation ratio: 5.91), and "[Structural Basis and Kinetic Pathway of RBM39 Recruitment to DCAF15 by a Sulfonamide Molecular Glue E7820.](https://pubmed.ncbi.nlm.nih.gov/31693911)" (2019) (relative citation ratio: 3.53). The relative citation ratio measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RBM39](https://www.proteinatlas.org/ENSG00000131051-RBM39) is an RNA-binding protein that functions as an activator, with evidence supporting its role at the protein level. It is detected in all tissues and is primarily localized in the nucleoplasm, nuclear speckles, microtubules, and centriolar satellites. RBM39 is involved in various expression clusters, including Cluster 35 in neutrophils for protein binding, Cluster 40 in non-specific tissues for transcription, Cluster 4 in the brain for mRNA splicing and cell cycle, Cluster 22 in non-specific cell lines for mRNA splicing and cell cycle, and Cluster 15 in non-specific single cells for mixed function.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [OTX2](https://www.ncbi.nlm.nih.gov/gene/5015) and [NFYA](https://www.ncbi.nlm.nih.gov/gene/4800), each with 6 experiments. Additionally, [SIN3A](https://www.ncbi.nlm.nih.gov/gene/25942), [CDK8](https://www.ncbi.nlm.nih.gov/gene/1024), and [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) were each found to be regulating in 5 experiments.


The transcription factor [65123](https://www.ncbi.nlm.nih.gov/gene/65123) was found to be regulating genes in one Perturb-seq experiment, according to data from GTRD's database of transcription factor binding sites and the study published in [PubMed](https://pubmed.ncbi.nlm.nih.gov/35688146/).


The input data indicates associations with a variety of disease conditions, including several types of cancer such as leukemia, lymphoma, and immune system cancer. Additionally, respiratory conditions like asthma, bronchial disease, and obstructive lung disease are noted. The data also highlights hematopoietic and immune system diseases, as well as cardiovascular system disease. Furthermore, specific conditions such as blood coagulation disease, inherited blood coagulation disease, and protein C deficiency are included.



The gene is expressed in the ovary and thyroid, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles show expression in CD4+ T cells, CD8+ T cells, and the lymph node.




The analyzed protein sequence has a GRAVY value of -0.649 (21.23rd percentile), indicating a hydrophilic nature. It exhibits a charge of 32.32 at pH 7.0 (97.58th percentile) and a median structural flexibility of 1.008 (74.49th percentile). The protein's secondary structure is predicted to be 32.45% helix (54.63rd percentile), 27.55% sheet (16.6th percentile), and 29.43% turn (59.52nd percentile). The instability index is 64.65 (89.81st percentile), with an isoelectric point of 10.10 (97.05th percentile). The protein has a length of 530 amino acids (61.91st percentile) and a molecular weight of 59378.90 Da (61.94th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |