---
layout: post
title: LYPLA1
date: 2025-01-17 16:55 CST
description: LYPLA1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10434) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10434  | LYPLA1 | ENSG00000120992 | 8q11.23 |



The gene is involved in [protein depalmitoylation](https://amigo.geneontology.org/amigo/term/GO:0002084) and [fatty acid metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006631), as well as [fatty acid transport](https://amigo.geneontology.org/amigo/term/GO:0015908). It enables [phospholipase activity](https://amigo.geneontology.org/amigo/term/GO:0004620), [lysophospholipase activity](https://amigo.geneontology.org/amigo/term/GO:0004622), [palmitoyl-(protein) hydrolase activity](https://amigo.geneontology.org/amigo/term/GO:0008474), [lipase activity](https://amigo.geneontology.org/amigo/term/GO:0016298), and [carboxylic ester hydrolase activity](https://amigo.geneontology.org/amigo/term/GO:0052689), and is involved in [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [endoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0005783), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986), [nuclear membrane](https://amigo.geneontology.org/amigo/term/GO:0031965), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). Additionally, it is involved in the [negative regulation of Golgi to plasma membrane protein transport](https://amigo.geneontology.org/amigo/term/GO:0042997).


The gene length is 31,760.0 base pairs (54.29% of all genes), the mature length is 1,949.5 base pairs, and the primary transcript length is 30,225.0 base pairs.


The gene LYPLA1 (NCBI ID: 10434) has been mentioned in [22 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22LYPLA1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1992 and the middle 50% of publications occurring between 2009 and 2018. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning LYPLA1, ranked by their scientific influence, include "[A functional screen implicates microRNA-138-dependent regulation of the depalmitoylation enzyme APT1 in dendritic spine morphogenesis.](https://pubmed.ncbi.nlm.nih.gov/19465924)" (2009) (relative citation ratio: 7.97), "[Small-molecule inhibition of APT1 affects Ras localization and signaling.](https://pubmed.ncbi.nlm.nih.gov/20418879)" (2010) (relative citation ratio: 6.31), "[Dynamic palmitoylation links cytosol-membrane shuttling of acyl-protein thioesterase-1 and acyl-protein thioesterase-2 with that of proto-oncogene H-ras product and growth-associated protein-43.](https://pubmed.ncbi.nlm.nih.gov/23396970)" (2013) (relative citation ratio: 2.63), "[Molecular Mechanism for Isoform-Selective Inhibition of Acyl Protein Thioesterases 1 and 2 (APT1 and APT2).](https://pubmed.ncbi.nlm.nih.gov/27748579)" (2016) (relative citation ratio: 2.18), and "[Inhibition of cell proliferation and migration in nonâ€‘small cell lung cancer cells through the suppression of LYPLA1.](https://pubmed.ncbi.nlm.nih.gov/30431103)" (2019) (relative citation ratio: 0.98). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[LYPLA1](https://www.proteinatlas.org/ENSG00000120992-LYPLA1) is a gene encoding a protein with hydrolase molecular function, supported by evidence at the protein level. The gene is detected in all tissues and is primarily localized in the nucleoplasm and cytosol. It is part of Cluster 10 for blood expression, which is non-specific and related to membrane trafficking. In tissue expression, it belongs to Cluster 35, which is non-specific and related to mitochondria. For brain expression, it is in Cluster 44, which is non-specific with mixed function. In cell line expression, it is part of Cluster 40, which is non-specific with unknown function. Additionally, it is found in Cluster 11 for single cell expression, specifically in late spermatids with unknown function.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [NFYA](https://www.ncbi.nlm.nih.gov/gene/4800) and [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), each with 6 experiments. Additionally, [YY1](https://www.ncbi.nlm.nih.gov/gene/7528), [AR](https://www.ncbi.nlm.nih.gov/gene/367), and [KDM2B](https://www.ncbi.nlm.nih.gov/gene/84678) were each found to be regulating in 5 experiments.





The gene is expressed in various tissues and cell types, including salivary gland, Lymphoma (Burkitt's, Raji), Leukemia (chronic myelogenous K-562 and promyelocytic HL-60), bronchial epithelial cells, placenta, colon, and small intestine.


The input data highlights several pathways related to metabolism, specifically focusing on the metabolism of nitric oxide. Key processes include the activation and regulation of eNOS, with a particular emphasis on eNOS activation.


The mouse ortholog gene 18777 and its human ortholog gene 10434 are associated with phenotypes such as decreased total retina thickness and preweaning lethality with incomplete penetrance, affecting both sexes.


The analyzed protein sequence has a GRAVY value of 0.066 (86.78th percentile), indicating a slightly hydrophobic nature. The charge at pH 7 is -2.012 (43.12th percentile), and the median structural flexibility is 0.996 (25.1st percentile). The protein has a propensity for helix, sheet, and turn structures of 30.87% (39.71st percentile), 33.91% (51.33rd percentile), and 32.17% (77.67th percentile), respectively. The instability index is 50.82 (59.92nd percentile), suggesting moderate stability. The isoelectric point is 6.29 (37.72nd percentile), with a molecular weight of 24669.32 Da (17.55th percentile). The sequence length is 230 amino acids (19.01st percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |