---
layout: post
title: YWHAB
date: 2025-01-17 16:55 CST
description: YWHAB description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7529) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7529  | YWHAB | ENSG00000166913 | 20q13.12 |



The gene enables [protein kinase inhibitor activity](https://amigo.geneontology.org/amigo/term/GO:0004860) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737). It is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [vacuolar membrane](https://amigo.geneontology.org/amigo/term/GO:0005774), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [melanosome](https://amigo.geneontology.org/amigo/term/GO:0042470), [perinuclear region of cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0048471), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). It is involved in [protein targeting](https://amigo.geneontology.org/amigo/term/GO:0006605), [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), [negative regulation of protein dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0035308), [positive regulation of catalytic activity](https://amigo.geneontology.org/amigo/term/GO:0043085), [negative regulation of G protein-coupled receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0045744), and [cytoplasmic sequestering of protein](https://amigo.geneontology.org/amigo/term/GO:0051220). Additionally, it enables [enzyme binding](https://amigo.geneontology.org/amigo/term/GO:0019899), [protein domain specific binding](https://amigo.geneontology.org/amigo/term/GO:0019904), [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802), [histone deacetylase binding](https://amigo.geneontology.org/amigo/term/GO:0042826), [cadherin binding](https://amigo.geneontology.org/amigo/term/GO:0045296), [phosphoserine residue binding](https://amigo.geneontology.org/amigo/term/GO:0050815), and [phosphoprotein binding](https://amigo.geneontology.org/amigo/term/GO:0051219).


The gene length is 10,884 base pairs (27.42% of all genes), the mature length is 2,754 base pairs, and the primary transcript length is 10,823 base pairs (35.35% of all genes).


The gene YWHAB (NCBI ID: 7529) has been mentioned in [9 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22YWHAB%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1995 and the middle 50% of publications occurring between 2010 and 2020. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning YWHAB, ranked by their relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows:

- [Coat proteins of necroviruses target 14-3-3a to subvert MAPKKKÎ±-mediated antiviral immunity in plants.](https://pubmed.ncbi.nlm.nih.gov/35132090) (2022) (relative citation ratio: 3.68)
- [14-3-3 alpha and delta are the phosphorylated forms of raf-activating 14-3-3 beta and zeta. In vivo stoichiometric phosphorylation in brain at a Ser-Pro-Glu-Lys MOTIF.](https://pubmed.ncbi.nlm.nih.gov/7890696) (1995) (relative citation ratio: 3.15)
- [Genomic deletion of the whole IgH 3' regulatory region (hs3a, hs1,2, hs3b, and hs4) dramatically affects class switch recombination and Ig secretion to all isotypes.](https://pubmed.ncbi.nlm.nih.gov/20538806) (2010) (relative citation ratio: 2.04)
- [Circ SMARCA5 Inhibited Tumor Metastasis by Interacting with SND1 and Downregulating the YWHAB Gene in Cervical Cancer.](https://pubmed.ncbi.nlm.nih.gov/33588586) (2021) (relative citation ratio: 1.75)
- [14-3-3 protein beta/alpha as a urinary biomarker for renal cell carcinoma: proteomic analysis of cyst fluid.](https://pubmed.ncbi.nlm.nih.gov/21553213) (2011) (relative citation ratio: 0.57)

Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[YWHAB](https://www.proteinatlas.org/ENSG00000166913-YWHAB) is a cancer-related gene with evidence at the protein level. It is detected in all RNA tissue distributions and is primarily located in the cytosol. The gene is part of Cluster 42 for blood expression, which is associated with non-specific vesicular transport, and Cluster 61 for tissue expression, which is linked to brain neuronal signaling. In the brain, it is part of Cluster 14, involved in neurons' synaptic function. Additionally, YWHAB is found in Cluster 62 for cell line expression, related to non-specific plasma proteins, and Cluster 45 for single-cell expression, which has mixed functions.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), [HDAC2](https://www.ncbi.nlm.nih.gov/gene/3066), [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), [AR](https://www.ncbi.nlm.nih.gov/gene/367), and [SP1](https://www.ncbi.nlm.nih.gov/gene/6667), each shown to be regulating in 5 experiments.





The gene is expressed in various brain regions, including the amygdala, cingulate cortex, parietal lobe, and occipital lobe.


The input data highlights various biological pathways and processes, including gene expression, cytokine signaling in the immune system, regulation of mRNA stability by proteins that bind AU-rich elements, apoptosis, and programmed cell death. Additionally, it covers signaling pathways such as ARMS-mediated activation, Frs2-mediated activation, prolonged ERK activation events, and signaling to p38 via RIT and RIN. The data also includes GRB2 and SHC1 events in EGFR and ERBB2 signaling, RAF/MAP kinase cascade, SOS-mediated signaling, and SHC-related events. Furthermore, it encompasses signaling by leptin, ERKs, RAS, VEGFR2-mediated cell proliferation, MEK activation, RAF phosphorylation of MEK, and RAF activation. The adaptive, innate, and overall immune systems are also mentioned, along with signaling by various receptors like SCF-KIT, ERBB4, DAP12, PDGF, FGFR, and EGFR in both normal and cancer contexts. The data also includes NGF signaling via TRKA, Fc epsilon receptor (FCERI) signaling, IGF1R signaling cascade, insulin receptor signaling, VEGF signaling, interleukin-2 signaling, and signaling by interleukins. Developmental biology processes such as axon guidance and NCAM signaling for neurite outgrowth are noted, as well as disease-related pathways and signal transduction mechanisms. Additionally, the data covers Rap1 signaling, mRNA destabilization by BRF1 and TTP, activation of BAD and BH3-only proteins, the intrinsic pathway for apoptosis, signaling by Hippo, membrane trafficking, and translocation of GLUT4 to the plasma membrane.


The mouse ortholog gene 54401, which corresponds to the human ortholog gene 7529, is associated with several phenotypic abnormalities. These include abnormal retina morphology, decreased blood urea nitrogen level, decreased circulating HDL cholesterol level, decreased circulating cholesterol level, and decreased total retina thickness. All these phenotypes are observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.729 (15.79th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -14.28 (16.27th percentile), and the median structural flexibility is 1.012 (85.25th percentile). The protein has a high affinity for helix structures (0.419, 95.89th percentile), with a moderate affinity for sheet structures (0.309, 33.27th percentile) and turn structures (0.252, 23.01st percentile). The instability index is 47.26 (47.52nd percentile), and the isoelectric point is 4.76 (3.98th percentile). The protein sequence is 246 amino acids long (21.09th percentile) with a molecular weight of 28082.08 Da (21.97th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |