---
layout: post
title: CEP57
date: 2025-01-17 16:55 CST
description: CEP57 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9702) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9702  | CEP57 | ENSG00000166037 | 11q21  |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [microtubule binding](https://amigo.geneontology.org/amigo/term/GO:0008017), and is involved in [spermatid development](https://amigo.geneontology.org/amigo/term/GO:0007286) and the [fibroblast growth factor receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0008543). It is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [Golgi apparatus](https://amigo.geneontology.org/amigo/term/GO:0005794), [centrosome](https://amigo.geneontology.org/amigo/term/GO:0005813), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [microtubule](https://amigo.geneontology.org/amigo/term/GO:0005874), and [centriolar satellite](https://amigo.geneontology.org/amigo/term/GO:0034451). Additionally, it enables [fibroblast growth factor binding](https://amigo.geneontology.org/amigo/term/GO:0017134), [protein homodimerization activity](https://amigo.geneontology.org/amigo/term/GO:0042803), and [gamma-tubulin binding](https://amigo.geneontology.org/amigo/term/GO:0043015), and is involved in [protein homooligomerization](https://amigo.geneontology.org/amigo/term/GO:0051260).


The gene length is 100,175 base pairs (81.77% of all genes), the mature length is 5,745.5 base pairs (93.33% of all genes), and the primary transcript length is 98,228.5 base pairs (87.97% of all genes).


CEP57 has been mentioned in [17 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CEP57%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 2004, and the middle 50% of publications occurred between 2013 and 2020.


The top 5 publications mentioning CEP57 include "[The Cep57-pericentrin module organizes PCM expansion and centriole engagement.](https://pubmed.ncbi.nlm.nih.gov/30804344)" (2019) (relative citation ratio: 1.99), "[Mutations in CEP57 cause mosaic variegated aneuploidy syndrome.](https://pubmed.ncbi.nlm.nih.gov/21552266)" (2011) (relative citation ratio: 1.9), "[Requirement of the Cep57-Cep63 Interaction for Proper Cep152 Recruitment and Centriole Duplication.](https://pubmed.ncbi.nlm.nih.gov/32152252)" (2020) (relative citation ratio: 1.39), "[Double homozygosity in CEP57 and DYNC2H1 genes detected by WES: Composite or expanded phenotype?](https://pubmed.ncbi.nlm.nih.gov/31943948)" (2020) (relative citation ratio: 0.63), and "[Cep57, a NEDD1-binding pericentriolar material component, is essential for spindle pole integrity.](https://pubmed.ncbi.nlm.nih.gov/22508265)" (2012) (relative citation ratio: 0.62). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CEP57](https://www.proteinatlas.org/ENSG00000166037-CEP57) is a gene with evidence at the protein level. Its RNA is detected in all tissues, and the protein is localized in the centriolar satellite, centrosome, and cytosol. The gene is expressed in various clusters, including Cluster 5 for blood (non-specific - cell proliferation), Cluster 16 for tissue (non-specific - transcription), Cluster 31 for brain (non-specific - transcription), Cluster 24 for cell lines (non-specific - unknown function), and Cluster 11 for single cells (late spermatids - unknown function).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SUPT5H](https://www.ncbi.nlm.nih.gov/gene/6829) with 6 experiments, followed by [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025), [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169), and [SIN3A](https://www.ncbi.nlm.nih.gov/gene/25942), each with 5 experiments.



The GWAS data indicates associations with several disease conditions, including artery disease, vascular disease, cardiovascular system disease, hypertension, and broader categories such as disease and disease of anatomical entity.



The gene is expressed in various immune cell types, including B lymphoblasts, CD34+ cells, CD4+ T cells, CD8+ T cells, and CD105+ endothelial cells. Additionally, it is expressed in CD71+ early erythroid cells and leukemia lymphoblastic cells (MOLT-4). The gene is also found to be expressed in the ovary.


The proteins are involved in various pathways related to the cell cycle, including mitotic cell cycle, centrosome maturation, and G2/M transition. Additionally, they participate in the anchoring of the basal body to the plasma membrane, recruitment of mitotic centrosome proteins and complexes, and the regulation of PLK1 activity at the G2/M transition. These proteins also play roles in the assembly of the primary cilium and organelle biogenesis and maintenance.



The analyzed protein sequence has a GRAVY value of -0.902, indicating a hydrophilic nature. It exhibits a charge of 17.59 at pH 7.0 and a median structural flexibility of 1.017. The protein's secondary structure is predicted to be predominantly helical (42%), with 26.8% in sheets and 25.4% in turns. The instability index is 56.89, and the isoelectric point is 9.35. The protein has a length of 500 amino acids and a molecular weight of 57088.34 Da. These values place the protein in the 7.31st percentile for GRAVY, 91.51st for charge at pH 7.0, 93.8th for flexibility, 96.01st for helix affinity, 76.82nd for instability index, 88.02nd for isoelectric point, 58.21st for length, and 59.65th for molecular weight compared to all genes. For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |