---
layout: post
title: HMGA1
date: 2024-12-08 23:57 CST
description: HMGA1 description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3159) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3159  | HMGA1 | ENSG00000137309 | 6p21.31 |



The gene length is 4,242 base pairs (11.92% of all genes), the mature length is 1,995.5 base pairs, and the primary transcript length is 3,072 base pairs (11.14% of all genes).


The gene HMGA1 (NCBI ID: 3159) has been the subject of [362 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22HMGA1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication dating back to 1988. The middle 50% of these publications were published between 2005 and 2018. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning HMGA1, ranked by their scientific influence, include "[The high mobility group protein HMG I(Y) is required for NF-kappa B-dependent virus induction of the human IFN-beta gene.](https://pubmed.ncbi.nlm.nih.gov/1330326)" (1992) (relative citation ratio: 10.84), "[LINC00460/DHX9/IGF2BP2 complex promotes colorectal cancer proliferation and metastasis by mediating HMGA1 mRNA stability depending on m6A modification.](https://pubmed.ncbi.nlm.nih.gov/33526059)" (2021) (relative citation ratio: 10.6), "[In Pulmonary Arterial Hypertension, Reduced BMPR2 Promotes Endothelial-to-Mesenchymal Transition via HMGA1 and Its Target Slug.](https://pubmed.ncbi.nlm.nih.gov/27045138)" (2016) (relative citation ratio: 6.5), "[HIV-1 cDNA integration: requirement of HMG I(Y) protein for function of preintegration complexes in vitro.](https://pubmed.ncbi.nlm.nih.gov/9038339)" (1997) (relative citation ratio: 5.39), and "[The solution structure of an HMG-I(Y)-DNA complex defines a new architectural minor groove binding motif.](https://pubmed.ncbi.nlm.nih.gov/9253416)" (1997) (relative citation ratio: 5.07). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599), [CEBPA](https://www.ncbi.nlm.nih.gov/gene/1050), [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908), [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169), and [AR](https://www.ncbi.nlm.nih.gov/gene/367), each shown to be regulating in 5 experiments.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [CTCF](https://www.ncbi.nlm.nih.gov/gene/6829) with 4 experiments, [ZNF274](https://www.ncbi.nlm.nih.gov/gene/8243) with 1 experiment, [ZNF263](https://www.ncbi.nlm.nih.gov/gene/8861) with 1 experiment, [ZNF143](https://www.ncbi.nlm.nih.gov/gene/9968) with 1 experiment, and [ZNF783](https://www.ncbi.nlm.nih.gov/gene/30827) with 2 experiments. These TFs were found to regulate gene expression in multiple Perturb-seq experiments, highlighting their significant roles in transcriptional regulation.


The HMGA1 gene has been associated with several traits and diseases through genome-wide association studies. Notably, it has been linked to [Body mass index](https://pubmed.ncbi.nlm.nih.gov/31669095), which maps directly to the MeSH term [Body Mass Index](https://meshb.nlm.nih.gov/record/ui?ui=D015992). Additionally, HMGA1 has been associated with [Hemoglobin levels](https://pubmed.ncbi.nlm.nih.gov/32327693), which was mapped to the MeSH term [Hemoglobins](https://meshb.nlm.nih.gov/record/ui?ui=D006454) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method.


The mouse ortholog gene 15361 and its human ortholog gene 3159 are associated with phenotypes including preweaning lethality with incomplete penetrance and unresponsiveness to tactile stimuli, observed in both sexes.


The protein sequence analyzed has a GRAVY value of -1.8065 (0.16 percentile), indicating a high hydrophilicity. It exhibits a charge of 11.52 at pH 7.0 (84.3 percentile) and a median structural flexibility of 1.0401 (99.86 percentile). The protein's secondary structure affinities are 35.51% for helix (77.99 percentile), 14.02% for sheet (0.57 percentile), and 37.38% for turn (94.36 percentile). The instability index is 82.34 (98.61 percentile), with an isoelectric point of 10.31 (97.87 percentile). The sequence length is 107 amino acids (3.35 percentile), and the molecular weight is 11675.88 Da (3.2 percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |