---
layout: post
title: SLU7
date: 2025-01-17 16:55 CST
description: SLU7 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10569) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10569  | SLU7 | ENSG00000164609 | 5q33.3 |



The gene is involved in various biological processes, including [RNA splicing, via transesterification reactions](https://amigo.geneontology.org/amigo/term/GO:0000375), [alternative mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0000380), [mRNA 3'-splice site recognition](https://amigo.geneontology.org/amigo/term/GO:0000389), [mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0000398), and [RNA splicing](https://amigo.geneontology.org/amigo/term/GO:0008380). It enables [second spliceosomal transesterification activity](https://amigo.geneontology.org/amigo/term/GO:0000386), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [zinc ion binding](https://amigo.geneontology.org/amigo/term/GO:0008270), and [pre-mRNA 3'-splice site binding](https://amigo.geneontology.org/amigo/term/GO:0030628). The gene is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [nuclear speck](https://amigo.geneontology.org/amigo/term/GO:0016607), and [intracellular membrane-bounded organelle](https://amigo.geneontology.org/amigo/term/GO:0043231). It is part of the [spliceosomal complex](https://amigo.geneontology.org/amigo/term/GO:0005681), [small nuclear ribonucleoprotein complex](https://amigo.geneontology.org/amigo/term/GO:0030532), and [catalytic step 2 spliceosome](https://amigo.geneontology.org/amigo/term/GO:0071013). Additionally, the gene is involved in [intracellular protein transport](https://amigo.geneontology.org/amigo/term/GO:0006886) and [cellular response to heat](https://amigo.geneontology.org/amigo/term/GO:0034605).


The gene length is 7,115 base pairs (19.27% of all genes), the mature length is 1,510.5 base pairs, and the primary transcript length is 3,993 base pairs.


SLU7 has been mentioned in [130 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SLU7%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1991, and the middle 50% of publications occurred between 2011 and 2020.


The top five publications mentioning SLU7, ranked by their scientific influence, include "[Aberrant RNA splicing in cancer; expression changes and driver mutations of splicing factor genes.](https://pubmed.ncbi.nlm.nih.gov/26300000)" (2016) (relative citation ratio: 11.07), "[Somatic Mutational Landscape of Splicing Factor Genes and Their Functional Consequences across 33 Cancer Types.](https://pubmed.ncbi.nlm.nih.gov/29617667)" (2018) (relative citation ratio: 9.65), "[Splicing factor derived circular RNA circUHRF1 accelerates oral squamous cell carcinoma tumorigenesis via feedback loop.](https://pubmed.ncbi.nlm.nih.gov/31570856)" (2020) (relative citation ratio: 8.35), "[Molecular basis of UG-rich RNA recognition by the human splicing factor TDP-43.](https://pubmed.ncbi.nlm.nih.gov/24240615)" (2013) (relative citation ratio: 6.62), and "[The Augmented R-Loop Is a Unifying Mechanism for Myelodysplastic Syndromes Induced by High-Risk Splicing Factor Mutations.](https://pubmed.ncbi.nlm.nih.gov/29395063)" (2018) (relative citation ratio: 5.85). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[SLU7](https://www.proteinatlas.org/ENSG00000164609-SLU7) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in the nucleoplasm and nuclear speckles. The gene is expressed in various clusters, including Cluster 35 in neutrophils related to protein binding, Cluster 40 in non-specific tissues related to transcription, Cluster 34 in the brain with mixed functions, Cluster 22 in cell lines related to mRNA splicing and cell cycle, and Cluster 55 in single cells related to transcription.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) with 6 experiments, [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025) with 5 experiments, and [MYC](https://www.ncbi.nlm.nih.gov/gene/4609), [MAX](https://www.ncbi.nlm.nih.gov/gene/4149), and [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861) each with 4 experiments.





The gene is expressed in the brain.



The mouse ortholog gene 193116 and its human ortholog gene 10569 are associated with the phenotype of preweaning lethality with complete penetrance, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -1.24 (1.6 percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -1.87 (43.63 percentile), and the median structural flexibility is 1.02 (95.07 percentile). The protein has a high affinity for helix structures (39.25%, 92.03 percentile) and a moderate affinity for turn structures (26.28%, 31.66 percentile), with a lower affinity for sheet structures (25.26%, 9.31 percentile). The instability index is 50.46 (58.85 percentile), and the isoelectric point is 6.69 (46.06 percentile). The protein sequence is 586 amino acids long (67.37 percentile) with a molecular weight of 68385.88 Da (69.5 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |