---
layout: post
title: GDA
date: 2025-01-17 16:55 CST
description: GDA description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9615) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9615  | GDA | ENSG00000119125 | 9q21.13 |



The gene is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829) and is active in this cellular component. It is involved in various metabolic processes, including the [allantoin metabolic process](https://amigo.geneontology.org/amigo/term/GO:0000255), [nucleobase-containing compound metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006139), [guanine catabolic process](https://amigo.geneontology.org/amigo/term/GO:0006147), [deoxyguanosine catabolic process](https://amigo.geneontology.org/amigo/term/GO:0006161), [amide catabolic process](https://amigo.geneontology.org/amigo/term/GO:0043605), [GMP catabolic process](https://amigo.geneontology.org/amigo/term/GO:0046038), [dGMP catabolic process](https://amigo.geneontology.org/amigo/term/GO:0046055), and [guanine metabolic process](https://amigo.geneontology.org/amigo/term/GO:0046098). Additionally, it plays a role in [nervous system development](https://amigo.geneontology.org/amigo/term/GO:0007399). The gene enables molecular functions such as [zinc ion binding](https://amigo.geneontology.org/amigo/term/GO:0008270) and [guanine deaminase activity](https://amigo.geneontology.org/amigo/term/GO:0008892).


The gene length is 43,784 base pairs (62.78% of all genes), the mature length is 3,850 base pairs (82.4% of all genes), and the primary transcript length is 23,132 base pairs (55.7% of all genes).


The gene GDA (NCBI ID: 9615) has been mentioned in [157 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22GDA%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1950, and the middle 50% of publications occurred between 1972 and 1994.


The top five publications mentioning GDA, ranked by their scientific influence, include "[In vivo lability of glucose-6-phosphate dehydrogenase in GdA- and GdMediterranean deficiency.](https://pubmed.ncbi.nlm.nih.gov/5641629)" (1968) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 6.29, "[Comparison of GdA and GdB activities in Nigerians. A study of the variation of the G6PD activity.](https://pubmed.ncbi.nlm.nih.gov/835573)" (1977) with an RCR of 2.45, "[Guanine Deaminase Stimulates Ultraviolet-induced Keratinocyte Senescence in Seborrhoeic Keratosis via Guanine Metabolites.](https://pubmed.ncbi.nlm.nih.gov/32215662)" (2020) with an RCR of 2.02, "[Guanine Deaminase in Human Epidermal Keratinocytes Contributes to Skin Pigmentation.](https://pubmed.ncbi.nlm.nih.gov/32517074)" (2020) with an RCR of 1.64, and "[RhoA regulates dendrite branching in hippocampal neurons by decreasing cypin protein levels.](https://pubmed.ncbi.nlm.nih.gov/17670984)" (2007) with an RCR of 1.59. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[GDA](https://www.proteinatlas.org/ENSG00000119125-GDA) is a hydrolase with evidence at the protein level. It is detected in many tissues and is primarily located in the nucleoplasm. The gene is highly expressed in the small intestine, specifically in proximal enterocytes involved in digestion. In the brain, it is expressed in neurons with mixed functions. Additionally, GDA is expressed in kidney cancer cell lines, particularly in membrane components. The gene is unprognostic for various cancers, including bladder urothelial carcinoma, breast invasive carcinoma, cervical squamous cell carcinoma, and others.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [AR](https://www.ncbi.nlm.nih.gov/gene/367), [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), and [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), each with 4 experiments. Additionally, [HNF4A](https://www.ncbi.nlm.nih.gov/gene/3172) and [KLF5](https://www.ncbi.nlm.nih.gov/gene/688) were found to be regulating in 3 experiments each.



The input data indicates associations with a range of disease conditions, including cardiovascular system disease, kidney disease, urinary system disease, and various neurological and mental health disorders. Specifically, neurodegenerative diseases such as Alzheimer's disease and tauopathy are highlighted, along with cognitive disorders and dementia. Mental health conditions include bipolar disorder, mood disorders, substance-related disorders, and developmental disorders such as attention deficit hyperactivity disorder. Additionally, substance abuse and alcohol dependence are noted.


The genome-wide association study on [Scarlet fever](https://pubmed.ncbi.nlm.nih.gov/28928442) has been mapped to the closest existing MeSH term, [Scarlet Fever](https://meshb.nlm.nih.gov/record/ui?ui=D012541), using a method called [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0).


The gene is expressed in multiple tissue samples, including the liver, brain, small intestine, and kidney, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the salivary gland, atrioventricular node, ciliary ganglion, skin, globus pallidus, and ovary.


The input data highlights pathways involved in metabolism, specifically focusing on the metabolism of nucleotides. Within this broader category, the data emphasizes purine metabolism, with a particular focus on purine catabolism.


The mouse ortholog gene 14544 and its human ortholog 9615 are associated with phenotypic changes in T-helper cells. Specifically, there is a decrease in T-helper cell number, observed in one sex, and an increase in effector memory T-helper cell number, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.158 (73.26th percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is -15.19 (15.12th percentile), and the median structural flexibility is 0.997 (29.57th percentile). The protein has a length of 454 amino acids (52.43rd percentile) and a molecular weight of 51002.42 Da (52.81st percentile). The secondary structure affinities are 32.16% for helix (51.78th percentile), 38.77% for sheet (82.15th percentile), and 27.53% for turn (43.06th percentile). The instability index is 39.73 (21.41st percentile), and the isoelectric point is 5.44 (16.7th percentile). These values were calculated using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |