---
layout: post
title: TOB1
date: 2025-01-17 16:55 CST
description: TOB1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10140) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10140  | TOB1 | ENSG00000141232 | 17q21.33 |



The gene enables [transcription corepressor activity](https://amigo.geneontology.org/amigo/term/GO:0003714) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is active in both the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737). It is involved in the [negative regulation of cell population proliferation](https://amigo.geneontology.org/amigo/term/GO:0008285), [regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0010468), and [negative regulation of translation](https://amigo.geneontology.org/amigo/term/GO:0017148). Additionally, it is part of the [CCR4-NOT complex](https://amigo.geneontology.org/amigo/term/GO:0030014) and plays roles in the [negative regulation of BMP signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0030514), [negative regulation of osteoblast differentiation](https://amigo.geneontology.org/amigo/term/GO:0045668), and [negative regulation of DNA-templated transcription](https://amigo.geneontology.org/amigo/term/GO:0045892). The gene also enables [receptor tyrosine kinase binding](https://amigo.geneontology.org/amigo/term/GO:0030971) and [SMAD binding](https://amigo.geneontology.org/amigo/term/GO:0046332). Furthermore, it is involved in the [negative regulation of nuclear-transcribed mRNA poly(A) tail shortening](https://amigo.geneontology.org/amigo/term/GO:0060212), [positive regulation of nuclear-transcribed mRNA poly(A) tail shortening](https://amigo.geneontology.org/amigo/term/GO:0060213), [regulation of SMAD protein signal transduction](https://amigo.geneontology.org/amigo/term/GO:0060390), and [positive regulation of nuclear-transcribed mRNA catabolic process, deadenylation-dependent decay](https://amigo.geneontology.org/amigo/term/GO:1900153).


The gene length is 6,964 base pairs (18.88% of all genes), the mature length is 1,467.5 base pairs, and the primary transcript length is 6,918 base pairs.


The gene TOB1 (NCBI ID: 10140) has been mentioned in [84 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22TOB1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning TOB1 is 1994, and the middle 50% of publications occurred between 2007 and 2018.


The top five publications mentioning TOB1, ranked by their scientific influence, include "[The mammalian anti-proliferative BTG/Tob protein family.](https://pubmed.ncbi.nlm.nih.gov/19746446)" (2010) (relative citation ratio: 4.68), "[MicroRNA-25 promotes gastric cancer migration, invasion and proliferation by directly targeting transducer of ERBB2, 1 and correlates with poor survival.](https://pubmed.ncbi.nlm.nih.gov/25043310)" (2015) (relative citation ratio: 3.76), "[In search of a function for the TIS21/PC3/BTG1/TOB family.](https://pubmed.ncbi.nlm.nih.gov/11377414)" (2001) (relative citation ratio: 3.36), "[Tob is a negative regulator of activation that is expressed in anergic and quiescent T cells.](https://pubmed.ncbi.nlm.nih.gov/11694881)" (2001) (relative citation ratio: 3.05), and "[Exosomal miR-552-5p promotes tumorigenesis and disease progression via the PTEN/TOB1 axis in gastric cancer.](https://pubmed.ncbi.nlm.nih.gov/35154457)" (2022) (relative citation ratio: 2.27). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[TOB1](https://www.proteinatlas.org/ENSG00000141232-TOB1) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in the nucleoplasm and vesicles. The gene is expressed in various clusters, including Cluster 42 for blood expression related to non-specific vesicular transport, Cluster 68 for tissue expression related to non-specific protein processing, Cluster 39 for brain expression in astrocytes with mixed function, Cluster 27 for cell line expression in breast cancer with unknown function, and Cluster 78 for single cell expression in NK-cells and T-cells related to the immune system and transcription.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [ZNF639](https://www.ncbi.nlm.nih.gov/gene/51193), [NFYA](https://www.ncbi.nlm.nih.gov/gene/4800), [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313), [MAX](https://www.ncbi.nlm.nih.gov/gene/4149), and [KLF5](https://www.ncbi.nlm.nih.gov/gene/688), each shown to be regulating in 5 experiments.





The gene is expressed in various tissues, including the bladder and salivary gland, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the prostate, colon, small intestine, adrenal cortex, and adrenal gland.




The analyzed protein sequence has a GRAVY value of -0.573 (27.7th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -1.14 (46.18th percentile), and the median structural flexibility is 1.013 (88.05th percentile). The protein has a predicted secondary structure with 26.7% helix (10.25th percentile), 29.9% sheet (27.49th percentile), and 38.3% turn (95.6th percentile) affinities. The instability index is 60.24 (83.48th percentile), suggesting potential instability. The isoelectric point is 6.45 (41.51st percentile), with a molecular weight of 38154.48 Da (36.69th percentile) and a length of 345 amino acids (37.13th percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |