---
layout: post
title: ADD3
date: 2025-01-17 16:55 CST
description: ADD3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/120) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 120  | ADD3 | ENSG00000148700 | 10q25.1-q25.2 |



The gene is located in the [condensed nuclear chromosome](https://amigo.geneontology.org/amigo/term/GO:0000794), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [brush border](https://amigo.geneontology.org/amigo/term/GO:0005903), [cell-cell junction](https://amigo.geneontology.org/amigo/term/GO:0005911), [cell cortex](https://amigo.geneontology.org/amigo/term/GO:0005938), and [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020). It is active in the [cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0005856), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), and [postsynaptic density](https://amigo.geneontology.org/amigo/term/GO:0014069). The gene enables [protein kinase C binding](https://amigo.geneontology.org/amigo/term/GO:0005080), [structural constituent of cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0005200), [calmodulin binding](https://amigo.geneontology.org/amigo/term/GO:0005516), and [actin filament binding](https://amigo.geneontology.org/amigo/term/GO:0051015). It is involved in [cytoskeleton organization](https://amigo.geneontology.org/amigo/term/GO:0007010), [response to xenobiotic stimulus](https://amigo.geneontology.org/amigo/term/GO:0009410), [positive regulation of vasoconstriction](https://amigo.geneontology.org/amigo/term/GO:0045907), [barbed-end actin filament capping](https://amigo.geneontology.org/amigo/term/GO:0051016), and [positive regulation of cytoskeleton organization](https://amigo.geneontology.org/amigo/term/GO:0051495).


The gene length is 22,257.0 base pairs (44.69% of all genes), the mature length is 1,997.0 base pairs, and the primary transcript length is 22,257.0 base pairs.


The gene ADD3 (NCBI ID: 120) has been mentioned in [15 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ADD3%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning ADD3 was in 1999, and the middle 50% of publications occurred between 2008 and 2020.


The top 5 publications mentioning ADD3, ranked by their scientific influence, include "[MiR-145 functions as a tumor-suppressive RNA by targeting Sox9 and adducin 3 in human glioma cells.](https://pubmed.ncbi.nlm.nih.gov/23814265)" (2013) (relative citation ratio: 3.18), "[Common genetic variants regulating ADD3 gene expression alter biliary atresia risk.](https://pubmed.ncbi.nlm.nih.gov/23872602)" (2013) (relative citation ratio: 2.12), "[QKI-5 regulates the alternative splicing of cytoskeletal gene ADD3 in lung cancer.](https://pubmed.ncbi.nlm.nih.gov/33196842)" (2021) (relative citation ratio: 1.86), "[Replication of a GWAS signal in a Caucasian population implicates ADD3 in susceptibility to biliary atresia.](https://pubmed.ncbi.nlm.nih.gov/24104524)" (2014) (relative citation ratio: 1.64), and "[Role of long non-coding RNA-adducin 3 antisense RNA1 in liver fibrosis of biliary atresia.](https://pubmed.ncbi.nlm.nih.gov/35246014)" (2022) (relative citation ratio: 1.48). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ADD3](https://www.proteinatlas.org/ENSG00000148700-ADD3) is a gene with actin-binding and calmodulin-binding molecular functions. It is involved in disease variants and has evidence at the protein level. The gene is detected in all tissues and is primarily located in the plasma membrane. ADD3 is expressed in various clusters, including Cluster 42 for non-specific vesicular transport in blood, Cluster 9 for oligodendrocytes involved in myelination in tissues, Cluster 39 for astrocytes with mixed functions in the brain, Cluster 44 for connective tissue cells involved in ECM organization in cell lines, and Cluster 53 for fibroblasts involved in ECM organization in single cells.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [MAX](https://www.ncbi.nlm.nih.gov/gene/4149) with 6 experiments, [YY1](https://www.ncbi.nlm.nih.gov/gene/7528) with 5 experiments, [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861) with 5 experiments, [TAF1](https://www.ncbi.nlm.nih.gov/gene/6872) with 5 experiments, and [GATA2](https://www.ncbi.nlm.nih.gov/gene/2624) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.


The top transcription factors (TFs) identified in terms of the number of Perturb-seq experiments are [8861](https://www.ncbi.nlm.nih.gov/gene/8861) and [9968](https://www.ncbi.nlm.nih.gov/gene/9968), with 2 and 3 experiments respectively. These TFs were found to regulate gene expression in multiple experiments, highlighting their significant role in transcriptional regulation.


The GWAS data indicates associations with a range of disease conditions, including various metabolic disorders such as diabetes mellitus and type 2 diabetes mellitus, as well as cardiovascular system disease. Additionally, the data highlights associations with kidney disease, including chronic kidney failure and end-stage renal failure, and nervous system disease. Eye and adnexa diseases, particularly retinal degeneration and age-related macular degeneration, are also implicated. Furthermore, the data shows links to hepatobiliary disease, including biliary tract disease and biliary atresia, as well as gastrointestinal system disease and cholestasis.



The gene is ubiquitously expressed in multiple tissue samples, including nerve, thyroid, and brain, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in various tissues and cell types such as Prefrontal Cortex, CD56+ NK Cells, CD4+ T cells, CD8+ T cells, CD19+ B Cells (neg. sel.), Amygdala, and Olfactory Bulb.




The analyzed protein sequence has a GRAVY value of -0.567 (28.23rd percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -10.85 (21.2nd percentile). The median structural flexibility is 1.009 (78.82nd percentile), with a propensity for helix, sheet, and turn structures of 31.87% (49.23rd percentile), 32.58% (42.88th percentile), and 30.45% (67.16th percentile), respectively. The instability index is 56.78 (76.6th percentile), and the isoelectric point is 5.92 (28.46th percentile). The protein is 706 amino acids long (75.64th percentile) with a molecular weight of 79153.71 Da (75.77th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |