---
layout: post
title: SRP68
date: 2025-01-17 16:55 CST
description: SRP68 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6730) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6730  | SRP68 | ENSG00000167881 | 17q25.1 |



The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and [signal recognition particle binding](https://amigo.geneontology.org/amigo/term/GO:0005047), and is involved in [SRP-dependent cotranslational protein targeting to membrane](https://amigo.geneontology.org/amigo/term/GO:0006614) and [response to xenobiotic stimulus](https://amigo.geneontology.org/amigo/term/GO:0009410). It is located in the [nucleolus](https://amigo.geneontology.org/amigo/term/GO:0005730), [endoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0005783), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [ribosome](https://amigo.geneontology.org/amigo/term/GO:0005840), and [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925). Additionally, it is part of the [signal recognition particle, endoplasmic reticulum targeting](https://amigo.geneontology.org/amigo/term/GO:0005786) and [signal recognition particle](https://amigo.geneontology.org/amigo/term/GO:0048500). The gene also enables [7S RNA binding](https://amigo.geneontology.org/amigo/term/GO:0008312), [protein domain specific binding](https://amigo.geneontology.org/amigo/term/GO:0019904), and [endoplasmic reticulum signal peptide binding](https://amigo.geneontology.org/amigo/term/GO:0030942), and contributes to [ribosome binding](https://amigo.geneontology.org/amigo/term/GO:0043022).


The gene length is 8,703 base pairs (22.85th percentile), the mature length is 1,566 base pairs (40.16th percentile), and the primary transcript length is 3,616 base pairs (13.34th percentile).


The gene SRP68, with NCBI ID 6730, has been mentioned in [6 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SRP68%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727). The earliest year of publication is 2006, and the middle 50% of publications occurred between 2009 and 2016. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning SRP68, ranked by their relative citation ratio (RCR), include studies that explore the gene's role in protein translocation and its association with congenital neutropenia. The RCR values measure the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. The most influential publication is "[SRP RNA remodeling by SRP68 explains its role in protein translocation.](https://pubmed.ncbi.nlm.nih.gov/24700861)" (2014) with an RCR of 0.91. Other notable works include "[Identification of biallelic germline variants of SRP68 in a sporadic case with severe congenital neutropenia.](https://pubmed.ncbi.nlm.nih.gov/32273475)" (2021) with an RCR of 0.5, and "[Human apo-SRP72 and SRP68/72 complex structures reveal the molecular basis of protein translocation.](https://pubmed.ncbi.nlm.nih.gov/28369529)" (2017) with an RCR of 0.46. Additionally, "[A novel histone H4 arginine 3 methylation-sensitive histone H4 binding activity and transcriptional regulatory function for signal recognition particle subunits SRP68 and SRP72.](https://pubmed.ncbi.nlm.nih.gov/23048028)" (2012) has an RCR of 0.36, and "[Protein SRP68 of human signal recognition particle: identification of the RNA and SRP72 binding domains.](https://pubmed.ncbi.nlm.nih.gov/16672232)" (2006) has an RCR of 0.26. The citation counts are sourced from [iCite](https://icite.od.nih.gov).


[SRP68](https://www.proteinatlas.org/ENSG00000167881-SRP68) is a gene with a molecular function in ribonucleoprotein and RNA-binding. Evidence for its function is available at the protein level. The gene is detected in all tissues and is primarily located in the cytosol. It is expressed in eosinophils with unknown function (Cluster 45) and is involved in non-specific transcription (Cluster 79). In the brain, it is associated with mRNA splicing and the cell cycle (Cluster 4). Additionally, SRP68 is expressed in breast cancer cell lines (Cluster 27) and syncytiotrophoblasts during pregnancy (Cluster 27).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [HDAC1](https://www.ncbi.nlm.nih.gov/gene/3065) with 6 experiments, [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) with 5 experiments, [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046) with 5 experiments, [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 5 experiments, and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The input data indicates associations with various disease conditions, including metabolic disorders such as carbohydrate metabolism disease, glucose metabolism disease, diabetes mellitus, and specifically type 2 diabetes mellitus. Additionally, the data highlights cognitive disorders and diseases of mental health.



The gene is expressed in various tissues, including muscle, skin, and pituitary, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in cardiac myocytes, cerebellum, smooth muscle, and leukemia lymphoblastic cells (MOLT-4).


The input data highlights several key biological processes, including the metabolism of proteins, gene expression, SRP-dependent cotranslational protein targeting to the membrane, and translation. These processes are integral to cellular function and protein synthesis.



The analyzed protein sequence has a GRAVY value of -0.553 (29.73rd percentile), indicating a hydrophilic nature. It exhibits a charge of 8.405 at pH 7.0 (77.92nd percentile) and a median structural flexibility of 1.007 (71.39th percentile). The protein's secondary structure is predicted to be 38.44% helix (89.94th percentile), 32.06% sheet (39.8th percentile), and 25.04% turn (21.88th percentile). The instability index is 49.62 (56th percentile), with an isoelectric point of 8.75 (72.59th percentile). The protein has a length of 627 amino acids (70.56th percentile) and a molecular weight of 70728.77 Da (71.02nd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |