---
layout: post
title: SRP72
date: 2025-01-17 16:55 CST
description: SRP72 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6731) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6731  | SRP72 | ENSG00000174780 | 4q12  |



The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [signal recognition particle binding](https://amigo.geneontology.org/amigo/term/GO:0005047), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [7S RNA binding](https://amigo.geneontology.org/amigo/term/GO:0008312). It is located in the [endoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0005783) and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and is part of the [signal recognition particle, endoplasmic reticulum targeting](https://amigo.geneontology.org/amigo/term/GO:0005786) and [signal recognition particle](https://amigo.geneontology.org/amigo/term/GO:0048500). The gene is involved in [SRP-dependent cotranslational protein targeting to membrane](https://amigo.geneontology.org/amigo/term/GO:0006614) and contributes to [ribosome binding](https://amigo.geneontology.org/amigo/term/GO:0043022). Additionally, it enables [TPR domain binding](https://amigo.geneontology.org/amigo/term/GO:0030911).


The gene length is 14,977 base pairs (35.06% of all genes), the mature length is 3,065 base pairs, and the primary transcript length is 14,966 base pairs.


The gene SRP72, with NCBI ID 6731, has been mentioned in [13 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SRP72%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727). The earliest year of publication is 1993, and the middle 50% of publications occurred between 2007 and 2012. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning SRP72, ranked by their relative citation ratio (RCR), include studies that have significantly contributed to the understanding of this gene. The RCR values measure the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. The most influential publication is "[Exome sequencing identifies autosomal-dominant SRP72 mutations associated with familial aplasia and myelodysplasia.](https://pubmed.ncbi.nlm.nih.gov/22541560)" (2012) (relative citation ratio: 1.87). Another notable study is "[Comparative study on the expression of stress-response protein (srp) 72, srp 27, alpha B-crystallin and ubiquitin in brain tumours. An immunohistochemical investigation.](https://pubmed.ncbi.nlm.nih.gov/8278027)" (1993) (relative citation ratio: 0.94). Additionally, "[Structures of human SRP72 complexes provide insights into SRP RNA remodeling and ribosome interaction.](https://pubmed.ncbi.nlm.nih.gov/27899666)" (2017) (relative citation ratio: 0.76) and "[Inhibitors of MAPK pathway ERK1/2 or p38 prevent the IL-1{beta}-induced up-regulation of SRP72 autoantigen in Jurkat cells.](https://pubmed.ncbi.nlm.nih.gov/20729213)" (2010) (relative citation ratio: 0.51) have also made significant contributions. Lastly, "[Human apo-SRP72 and SRP68/72 complex structures reveal the molecular basis of protein translocation.](https://pubmed.ncbi.nlm.nih.gov/28369529)" (2017) (relative citation ratio: 0.46) is another important study in this area. The citation counts are sourced from [iCite](https://icite.od.nih.gov).


[SRP72](https://www.proteinatlas.org/ENSG00000174780-SRP72) is a gene with a molecular function related to ribonucleoprotein and is involved in disease variants. Evidence for its function is available at the protein level. The gene's RNA is detected in all tissues, and it is associated with various expression clusters, including Cluster 12 for non-specific mitochondrial translation in blood, Cluster 40 for non-specific transcription in tissues, Cluster 50 for non-specific nucleic acid binding in the brain, Cluster 22 for non-specific mRNA splicing and cell cycle in cell lines, and Cluster 77 for plasma cells involved in protein processing in single cells.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046), [CHD1](https://www.ncbi.nlm.nih.gov/gene/1105), and [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), each with 5 experiments. Additionally, [TP53](https://www.ncbi.nlm.nih.gov/gene/7157) was found to be regulating in 4 experiments.



The input data indicates associations with various disease conditions, including obstructive lung disease, lower respiratory tract disease, lung disease, respiratory system disease, coronary artery disease, artery disease, vascular disease, cardiovascular system disease, and chronic obstructive pulmonary disease.



The gene is expressed in various immune cell types, including CD34+ cells, BDCA4+ dendritic cells, and CD19+ B cells (neg. sel.). Additionally, it is found in leukemia cell lines such as chronic myelogenous leukemia K-562 and lymphoblastic leukemia MOLT-4. Expression is also detected in CD105+ endothelial cells and bronchial epithelial cells.


The input data highlights several key biological processes, including the metabolism of proteins, gene expression, SRP-dependent cotranslational protein targeting to the membrane, and translation. These processes are integral to cellular function and protein synthesis.


The mouse ortholog gene 66661 and its human ortholog gene 6731 are associated with the phenotype of preweaning lethality with complete penetrance, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.642 (21.78th percentile), indicating a hydrophilic nature. It exhibits a charge of 24.69 at pH 7.0 (95.63rd percentile) and a median structural flexibility of 1.012 (86.21st percentile). The protein's secondary structure is predicted to be 38.3% helix (89.59th percentile), 30.25% sheet (29.71st percentile), and 26.68% turn (35.23rd percentile). The instability index is 45.83 (42.3rd percentile), with an isoelectric point of 9.31 (87.3rd percentile). The protein has a length of 671 amino acids (73.56th percentile) and a molecular weight of 74605.31 Da (73.36th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |