---
layout: post
title: VAMP4
date: 2025-01-17 16:55 CST
description: VAMP4 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8674) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8674  | VAMP4 | ENSG00000117533 | 1q24.3 |



The gene is located in the [Golgi membrane](https://amigo.geneontology.org/amigo/term/GO:0000139), [lysosome](https://amigo.geneontology.org/amigo/term/GO:0005764), [endosome](https://amigo.geneontology.org/amigo/term/GO:0005768), [Golgi apparatus](https://amigo.geneontology.org/amigo/term/GO:0005794), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986), [transport vesicle](https://amigo.geneontology.org/amigo/term/GO:0030133), [clathrin-coated endocytic vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030669), and [trans-Golgi network membrane](https://amigo.geneontology.org/amigo/term/GO:0032588). It is part of the [SNARE complex](https://amigo.geneontology.org/amigo/term/GO:0031201) and is active in the [trans-Golgi network](https://amigo.geneontology.org/amigo/term/GO:0005802). The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is involved in various processes including [synaptic vesicle to endosome fusion](https://amigo.geneontology.org/amigo/term/GO:0016189), [endocytic recycling](https://amigo.geneontology.org/amigo/term/GO:0032456), [SNARE complex assembly](https://amigo.geneontology.org/amigo/term/GO:0035493), [Golgi ribbon formation](https://amigo.geneontology.org/amigo/term/GO:0090161), and [regulation of synaptic vesicle endocytosis](https://amigo.geneontology.org/amigo/term/GO:1900242).


The gene length is 7,945 base pairs (21.14% of all genes), the mature length is 876 base pairs, and the primary transcript length is 5,956 base pairs.


VAMP4 (Gene ID: 8674) has been mentioned in [13 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22VAMP4%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning VAMP4 was in 2001, and the middle 50% of publications occurred between 2007 and 2021.


The top 5 publications mentioning VAMP4, ranked by their relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)), include "[Analysis of DNM3 and VAMP4 as genetic modifiers of LRRK2 Parkinson's disease.](https://pubmed.ncbi.nlm.nih.gov/32873436)" (2021) (relative citation ratio: 1.92), "[Control of synaptic vesicle release probability via VAMP4 targeting to endolysosomes.](https://pubmed.ncbi.nlm.nih.gov/33931449)" (2021) (relative citation ratio: 1.43), "[The Di-leucine motif of vesicle-associated membrane protein 4 is required for its localization and AP-1 binding.](https://pubmed.ncbi.nlm.nih.gov/11598115)" (2001) (relative citation ratio: 1.1), "[The vSNAREs VAMP2 and VAMP4 control recycling and intracellular sorting of post-synaptic receptors in neuronal dendrites.](https://pubmed.ncbi.nlm.nih.gov/34496238)" (2021) (relative citation ratio: 1.07), and "[VAMP4 cycles from the cell surface to the trans-Golgi network via sorting and recycling endosomes.](https://pubmed.ncbi.nlm.nih.gov/17327277)" (2007) (relative citation ratio: 1.04). The relative citation ratio measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[VAMP4](https://www.proteinatlas.org/ENSG00000117533-VAMP4) is a gene with evidence at the protein level. Its RNA is detected in all tissues, and it is primarily localized in the Golgi apparatus. The gene is expressed in various clusters, including Cluster 5 for blood expression related to cell proliferation, Cluster 65 for tissue expression associated with cerebellum and nervous system development, Cluster 19 for brain expression in astrocytes with mixed function, Cluster 45 for cell line expression related to nuclear processes, and Cluster 55 for single cell expression related to transcription.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), [AR](https://www.ncbi.nlm.nih.gov/gene/367), and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), each with 5 experiments. Additionally, [ZNF341](https://www.ncbi.nlm.nih.gov/gene/84905) and [BRD3](https://www.ncbi.nlm.nih.gov/gene/8019) were found to be regulating in 4 experiments each.



The GWAS data indicates associations with a range of disease conditions, including hematopoietic system disease, immune system disease, cardiovascular system disease, cognitive disorder, disease of mental health, bipolar disorder, mood disorder, psychotic disorder, schizophrenia, and heart disease.



The gene is expressed in various tissues, including nerve, blood vessel, and pituitary, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in pineal gland (night), superior cervical ganglion, trigeminal ganglion, skeletal muscle, atrioventricular node, and ciliary ganglion.



The mouse ortholog gene 53330 and its human ortholog 8674 are associated with various abnormal morphologies, including those of the duodenum, heart, kidney, lung, lymph node, skeletal muscle, spleen, and thymus, as well as decreased exploration in new environments. These phenotypes are observed in both sexes. Additionally, the gene is linked to enlargement of the heart, kidney, lung, lymph nodes, spleen, and thymus, also observed in both sexes. Notably, abnormal and enlarged seminal vesicle morphology is not observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.589 (26.27th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -0.302 (49.08th percentile). The median structural flexibility is 1.013 (88.51st percentile), suggesting high flexibility. The protein has a notable affinity for helix and sheet structures, with fractions of 0.326 (56.21st percentile) and 0.326 (43.13th percentile), respectively, and a lower affinity for turn structures at 0.284 (50.67th percentile). The instability index is 50.82 (59.9th percentile), and the isoelectric point is 6.75 (47.13th percentile). The protein sequence is 141 amino acids long (7.01st percentile) with a molecular weight of 16396.63 Da (7.62nd percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |