---
layout: post
title: MAT2A
date: 2025-01-17 16:55 CST
description: MAT2A description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4144) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4144  | MAT2A | ENSG00000168906 | 2p11.2 |



The gene enables [methionine adenosyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0004478) and is involved in [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524). It is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829) and is active in the same. The gene plays a role in the [S-adenosylmethionine biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006556) and the [one-carbon metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006730). Additionally, it is involved in [protein hexamerization](https://amigo.geneontology.org/amigo/term/GO:0034214), [protein heterooligomerization](https://amigo.geneontology.org/amigo/term/GO:0051291), and the [cellular response to leukemia inhibitory factor](https://amigo.geneontology.org/amigo/term/GO:1990830). The gene also enables [small molecule binding](https://amigo.geneontology.org/amigo/term/GO:0036094), [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802), and [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872). Furthermore, it is part of the [methionine adenosyltransferase complex](https://amigo.geneontology.org/amigo/term/GO:0048269).


The gene length is 3,710 base pairs (10.3% of all genes), the mature length is 1,198.5 base pairs, and the primary transcript length is 3,706.5 base pairs.


The gene MAT2A (NCBI ID: 4144) has been mentioned in [67 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22MAT2A%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1989 and the middle 50% of publications occurring between 2009 and 2021. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning MAT2A, ranked by their scientific influence, include "[MAT2A Inhibition Blocks the Growth of MTAP-Deleted Cancer Cells by Reducing PRMT5-Dependent mRNA Splicing and Inducing DNA Damage.](https://pubmed.ncbi.nlm.nih.gov/33450196)" (2021) with a relative citation ratio of 7.06, "[MTAP Deletions in Cancer Create Vulnerability to Targeting of the MAT2A/PRMT5/RIOK1 Axis.](https://pubmed.ncbi.nlm.nih.gov/27068473)" (2016) with a relative citation ratio of 6.97, "[A conserved ubiquitin ligase of the nuclear envelope/endoplasmic reticulum that functions in both ER-associated and Matalpha2 repressor degradation.](https://pubmed.ncbi.nlm.nih.gov/11641273)" (2001) with a relative citation ratio of 5.98, "[The Genetics of Thoracic Aortic Aneurysms and Dissection: A Clinical Perspective.](https://pubmed.ncbi.nlm.nih.gov/31991693)" (2020) with a relative citation ratio of 5.8, and "[Oxidative Stress Aggravates Apoptosis of Nucleus Pulposus Cells through m<sup>6</sup>A Modification of MAT2A Pre-mRNA by METTL16.](https://pubmed.ncbi.nlm.nih.gov/35069973)" (2022) with a relative citation ratio of 3.78. The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[MAT2A](https://www.proteinatlas.org/ENSG00000168906-MAT2A) is a transferase with evidence at the protein level. It is detected in all RNA tissue distributions and is primarily localized in the nucleoplasm and cytosol. The gene is part of Cluster 33 in blood expression, which is non-specific and related to ATP binding. In tissue expression, it belongs to Cluster 15, which is specific to the pancreas and involved in proteolysis. In brain expression, it is part of Cluster 28, associated with oligodendrocytes and mixed functions. Additionally, it is found in Cluster 56 for cell line expression, which is non-specific and related to sensory perception, and in Cluster 80 for single cell expression, which is non-specific and related to transcription.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [HIF1A](https://www.ncbi.nlm.nih.gov/gene/3091) and [CDK8](https://www.ncbi.nlm.nih.gov/gene/1024), each with 6 experiments. Additionally, [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599), [MAF](https://www.ncbi.nlm.nih.gov/gene/4094), and [STAT1](https://www.ncbi.nlm.nih.gov/gene/6772) were each found to be regulating in 5 experiments.





The gene is expressed in the pancreas and thyroid according to the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles indicate expression in the pineal gland (both day and night), 721 B lymphoblasts, CD34+ cells, CD105+ endothelial cells, pancreas, and adrenal cortex.


The input data highlights various pathways and disorders related to biological oxidations, phase II conjugation, and methylation. Notably, defective AHCY is linked to Hypermethioninemia with S-adenosylhomocysteine hydrolase deficiency (HMAHCHD), while defective UGT1A1 and UGT1A4 cause hyperbilirubinemia. Additionally, defects in GCLC, GGT1, and GSS are associated with Hemolytic anemia due to gamma-glutamylcysteine synthetase deficiency (HAGGSD), Glutathionuria (GLUTH), and Glutathione synthetase deficiency (GSS deficiency), respectively. Other disorders include Methionine adenosyltransferase deficiency (MATD) due to defective MAT1A, 5-oxoprolinase deficiency (OPLAHD) due to defective OPLAH, Schneckenbecken dysplasia (SCHBCKD) due to defective SLC35D1, and Thiopurine S-methyltransferase deficiency (TPMT deficiency) due to defective TPMT. These conditions fall under metabolic disorders of biological oxidation enzymes and are part of broader metabolic processes.


The mouse ortholog gene 232087 and its human ortholog gene 4144 are associated with several phenotypic traits, including decreased prepulse inhibition, increased startle reflex, and preweaning lethality with complete penetrance. These traits are observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.2802531645569626 (60.09th percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is -6.366870930148764 (29.89th percentile), and the median structural flexibility is 0.9988869047619048 (36.28th percentile). The protein has a length of 395 amino acids (44.6th percentile) and a molecular weight of 43660.149 Da (44.17th percentile). The secondary structure affinities are 30.89% for helix (39.86th percentile), 35.70% for sheet (64.23rd percentile), and 27.34% for turn (41.22nd percentile). The instability index is 38.41367088607593 (17.92nd percentile), and the isoelectric point is 6.024671745300293 (31.22nd percentile). These values were calculated using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |