---
layout: post
title: RPS23
date: 2024-12-08 23:57 CST
description: RPS23 description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6228) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6228  | RPS23 | ENSG00000186468 | 5q14.2 |



The gene length is 5,261 base pairs (14.73% of all genes), the mature length is 2,780 base pairs, and the primary transcript length is 4,139 base pairs (15.45% of all genes).


The gene RPS23, with NCBI ID 6228, has been featured in [4 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RPS23%22) (count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 2015, and the middle 50% of publications occurred between 2016 and 2020.


The top publications mentioning RPS23 include "[Identification and functional characterization of ribosomal protein S23 as a new member of antimicrobial protein.](https://pubmed.ncbi.nlm.nih.gov/32423862)" (2020) (relative citation ratio: 1.27), "[In vitro characterization of S-23 metabolites produced by human liver microsomes, and subsequent application to urine after a controlled oral administration.](https://pubmed.ncbi.nlm.nih.gov/35182830)" (2022) (relative citation ratio: 1.2), "[Dysregulated COL3A1 and RPL8, RPS16, and RPS23 in Disc Degeneration Revealed by Bioinformatics Methods.](https://pubmed.ncbi.nlm.nih.gov/25893343)" (2015) (relative citation ratio: 0.52), and "[Prolyl dihydroxylation of unassembled uS12/Rps23 regulates fungal hypoxic adaptation.](https://pubmed.ncbi.nlm.nih.gov/29083304)" (2017) (relative citation ratio: 0.25). The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [E2F8](https://www.ncbi.nlm.nih.gov/gene/79733), [TAF1](https://www.ncbi.nlm.nih.gov/gene/6872), [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170), [YY1](https://www.ncbi.nlm.nih.gov/gene/7528), and [RXRA](https://www.ncbi.nlm.nih.gov/gene/6256), each shown to be regulating in 5 experiments.


Based on the data from GTRD's database of transcription factor (TF) binding sites and filtered for Perturb-seq data, several TFs were identified as significant regulators in multiple experiments. Notably, [TBP](https://www.ncbi.nlm.nih.gov/gene/6872) was found to regulate genes in five experiments, while [AR](https://www.ncbi.nlm.nih.gov/gene/367), [CEBPB](https://www.ncbi.nlm.nih.gov/gene/1054), [GATA2](https://www.ncbi.nlm.nih.gov/gene/2623), [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908), and [STAT1](https://www.ncbi.nlm.nih.gov/gene/6772) were each shown to regulate genes in two experiments. Additionally, [BCL6](https://www.ncbi.nlm.nih.gov/gene/608), [FOXP3](https://www.ncbi.nlm.nih.gov/gene/5094), and [IRF1](https://www.ncbi.nlm.nih.gov/gene/3802) were found to regulate genes in one experiment each. These findings highlight the regulatory roles of these TFs in gene expression as evidenced by Perturb-seq experiments.








The protein sequence analyzed has a GRAVY value of -0.519 (33.02 percentile), indicating a hydrophilic nature. It exhibits a charge of 20.91 at pH 7.0 (93.99 percentile) and a median structural flexibility of 1.003 (56.95 percentile). The protein's secondary structure is predicted to be 35.66% helix (78.89 percentile), 29.37% sheet (25.1 percentile), and 25.87% turn (28.11 percentile). The instability index is 22.29 (0.89 percentile), with an isoelectric point of 10.50 (98.32 percentile). The protein has a length of 143 amino acids (7.26 percentile) and a molecular weight of 15807.36 Da (7.1 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |