---
layout: post
title: POLR2I
date: 2024-12-08 23:57 CST
description: POLR2I description
tags: [cooccuring-genes]
categories: shortest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5438) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5438  | POLR2I | ENSG00000105258 | 19q13.12 |



The gene length is 389.0 base pairs (0.26 percentile of all genes), with a mature length of 389.0 base pairs and a primary transcript length of 389.0 base pairs.


The gene POLR2I, with NCBI ID 5438, has been the subject of [7 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22POLR2I%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 2006, and the middle 50% of publications occurred between 2009 and 2021.


The top 5 publications mentioning POLR2I, ranked by their scientific influence, include "[The RNA polymerase II subunit RPB-9 recruits the integrator complex to terminate Caenorhabditis elegans piRNA transcription.](https://pubmed.ncbi.nlm.nih.gov/33533030)" (2021) (relative citation ratio: 1.08), "[Fidelity of RNA polymerase II transcription: Role of Rpb9 [corrected] in error detection and proofreading.](https://pubmed.ncbi.nlm.nih.gov/24099331)" (2013) (relative citation ratio: 0.58), "[Evidence that the transcription elongation function of Rpb9 is involved in transcription-coupled DNA repair in Saccharomyces cerevisiae.](https://pubmed.ncbi.nlm.nih.gov/17030604)" (2006) (relative citation ratio: 0.38), "[Comprehensive Bioinformatics Analysis Identifies <i>POLR2I</i> as a Key Gene in the Pathogenesis of Hypertensive Nephropathy.](https://pubmed.ncbi.nlm.nih.gov/34422001)" (2021) (relative citation ratio: 0.34), and "[Absence of the Rpb9 subunit of RNA polymerase II reduces the chronological life span in fission yeast.](https://pubmed.ncbi.nlm.nih.gov/35618649)" (2022) (relative citation ratio: 0.3). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [RELA](https://www.ncbi.nlm.nih.gov/gene/5970) and [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), each with 6 experiments. Additionally, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025), and [STAT3](https://www.ncbi.nlm.nih.gov/gene/6774) were each found to be regulating in 5 experiments.











The analyzed protein sequence has a GRAVY value of -0.734 (15.53% percentile), indicating a hydrophilic nature. The charge at pH 7 is -9.03 (24.22% percentile), and the median structural flexibility is 0.998 (31.76% percentile). The protein has a propensity for helix, sheet, and turn structures of 27.2% (12.84% percentile), 29.6% (26.26% percentile), and 24.8% (20.14% percentile), respectively. The instability index is 51.12 (60.81% percentile), with an isoelectric point of 5.04 (8.49% percentile). The protein is 125 amino acids long (5.12% percentile) and has a molecular weight of 14523.08 Da (5.87% percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |