---
layout: post
title: CAMK2B
date: 2025-01-17 16:55 CST
description: CAMK2B description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/816) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 816  | CAMK2B | ENSG00000058404 | 7p13  |



The gene enables [actin binding](https://amigo.geneontology.org/amigo/term/GO:0003779), [calmodulin-dependent protein kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004683), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [calmodulin binding](https://amigo.geneontology.org/amigo/term/GO:0005516), and [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524). It is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [centrosome](https://amigo.geneontology.org/amigo/term/GO:0005813), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [endocytic vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030666), [sarcoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0033017), and [synapse](https://amigo.geneontology.org/amigo/term/GO:0045202). It is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737) and [neuron projection](https://amigo.geneontology.org/amigo/term/GO:0043005), and is part of the [calcium- and calmodulin-dependent protein kinase complex](https://amigo.geneontology.org/amigo/term/GO:0005954). The gene is involved in various processes including [protein phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0006468), [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), [nervous system development](https://amigo.geneontology.org/amigo/term/GO:0007399), [positive regulation of neuron projection development](https://amigo.geneontology.org/amigo/term/GO:0010976), [regulation of skeletal muscle adaptation](https://amigo.geneontology.org/amigo/term/GO:0014733), [peptidyl-serine phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0018105), [cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0030154), [protein autophosphorylation](https://amigo.geneontology.org/amigo/term/GO:0046777), [regulation of long-term neuronal synaptic plasticity](https://amigo.geneontology.org/amigo/term/GO:0048169), [regulation of synapse structural plasticity](https://amigo.geneontology.org/amigo/term/GO:0051823), [regulation of calcium ion transport](https://amigo.geneontology.org/amigo/term/GO:0051924), [regulation of dendritic spine development](https://amigo.geneontology.org/amigo/term/GO:0060998), [positive regulation of dendritic spine morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0061003), [positive regulation of synapse maturation](https://amigo.geneontology.org/amigo/term/GO:0090129), and [regulation of neuron migration](https://amigo.geneontology.org/amigo/term/GO:2001222). Additionally, it enables [calcium-dependent protein serine/threonine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0009931), [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802), [protein homodimerization activity](https://amigo.geneontology.org/amigo/term/GO:0042803), and [protein serine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0106310).


The gene length is 10,675 base pairs (27.03% of all genes), the mature length is 2,918 base pairs, and the primary transcript length is 10,505 base pairs (34.58% of all genes).


CAMK2B (Gene NCBI ID: 816) has been mentioned in [24 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CAMK2B%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1989, and the middle 50% of publications range from 2002 to 2020.


The top five publications mentioning CAMK2B, ranked by their scientific influence, include studies that highlight the gene's role in various biological processes. The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Notable publications include "[De Novo Mutations in Protein Kinase Genes CAMK2A and CAMK2B Cause Intellectual Disability.](https://pubmed.ncbi.nlm.nih.gov/29100089)" (2017) (relative citation ratio: 4.25), "[CAMK2/CaMKII activates MLKL in short-term starvation to facilitate autophagic flux.](https://pubmed.ncbi.nlm.nih.gov/34282994)" (2022) (relative citation ratio: 3.5), "[Mechanistic study of TRPM2-Ca(2+)-CAMK2-BECN1 signaling in oxidative stress-induced autophagy inhibition.](https://pubmed.ncbi.nlm.nih.gov/27245989)" (2016) (relative citation ratio: 2.57), "[<i>De novo</i> variants in <i>CAMK2A</i> and <i>CAMK2B</i> cause neurodevelopmental disorders.](https://pubmed.ncbi.nlm.nih.gov/29560374)" (2018) (relative citation ratio: 2.34), and "[Comparison of the CAM2 and NAL-NL2 hearing aid fitting methods.](https://pubmed.ncbi.nlm.nih.gov/22878351)" (2013) (relative citation ratio: 2.29). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CAMK2B](https://www.proteinatlas.org/ENSG00000058404-CAMK2B) is a multifunctional protein with roles in actin-binding, calmodulin-binding, and serine/threonine-protein kinase activity. It is implicated in intellectual disability and has evidence at the protein level. The gene is detected in many tissues and is localized to various subcellular regions, including the plasma membrane, cell junctions, and primary cilium. Its expression is notable in neurons and ciliated cells, with specific clusters in non-specific transcription, mixed function neurons, ion channel activity, and visual perception cells. While CAMK2B is unprognostic in several cancer types, its diverse molecular functions and tissue distribution highlight its importance in biological processes.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), [MYC](https://www.ncbi.nlm.nih.gov/gene/4609), and [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), each shown to be regulating in 5 experiments.



The GWAS data indicates associations with a range of metabolic and cardiovascular conditions, including diseases of metabolism, acquired metabolic diseases, carbohydrate metabolism diseases, glucose metabolism diseases, and nutrition diseases, particularly overnutrition. Additionally, there are links to artery disease, vascular disease, and cardiovascular system diseases. The data also highlights associations with hypertension, inherited metabolic disorders, lipid metabolism disorders, and various musculoskeletal conditions such as bone inflammation diseases, arthritis, bone diseases, connective tissue diseases, and osteoarthritis. Furthermore, metabolic syndrome X is also implicated.


CAMK2B has been associated with several diseases and traits through genome-wide association studies. Notably, it has been linked to [Lung function (FVC)](https://pubmed.ncbi.nlm.nih.gov/30595370), with the closest MeSH match being [Lung](https://meshb.nlm.nih.gov/record/ui?ui=D008168), identified using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method. Additionally, CAMK2B is associated with [Medication use (drugs used in diabetes)](https://pubmed.ncbi.nlm.nih.gov/34594039), mapped to [Therapeutics](https://meshb.nlm.nih.gov/record/ui?ui=D013812) through embedding match. Furthermore, it is implicated in [Metabolic syndrome](https://pubmed.ncbi.nlm.nih.gov/22399527), directly matching the MeSH term [Metabolic Syndrome](https://meshb.nlm.nih.gov/record/ui?ui=D024821), and [Type 2 diabetes](https://pubmed.ncbi.nlm.nih.gov/34594039), which directly matches the synonym for [Diabetes Mellitus, Type 2](https://meshb.nlm.nih.gov/record/ui?ui=D003924).


The gene is expressed in various tissues, including muscle, brain, and pituitary, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in specific brain regions such as the Amygdala, Whole brain, Subthalamic Nucleus, Medulla Oblongata, Pons, and Cerebellum Peduncles.


The input data highlights various pathways from Reactome, including cytokine signaling in the immune system, interferon signaling, interferon gamma signaling, and cellular responses to stress. Additionally, it covers immune system processes, HSF1-dependent transactivation, and cellular response to heat stress. Neuronal system pathways are also prominent, encompassing the activation of NMDA receptors upon glutamate binding, post NMDA receptor activation events, CREB phosphorylation through the activation of Ras and CaMKII, and Ras activation upon Ca2+ influx through NMDA receptors. Furthermore, the data includes the unblocking of NMDA receptors, glutamate binding and activation, transmission across chemical synapses, neurotransmitter receptor binding, and downstream transmission in the postsynaptic cell. Lastly, it covers glutamate binding, activation of AMPA receptors, synaptic plasticity, and trafficking of AMPA receptors.


The mouse ortholog gene 12323 and its human ortholog gene 816.0 are associated with the phenotype of decreased prepulse inhibition, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.3495 (52.29th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -0.7235 (47.65th percentile). The median structural flexibility is 1.0045 (61.76th percentile), with a secondary structure composition of 30.93% helix (40.36th percentile), 32.58% sheet (42.92nd percentile), and 30.33% turn (66.22nd percentile). The instability index is 52.69 (65.76th percentile), and the isoelectric point is 6.87 (48.85th percentile). The protein sequence is 666 amino acids long (73.23rd percentile) with a molecular weight of 72676.91 Da (72.2nd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |