---
layout: post
title: ITGAE
date: 2025-01-17 16:55 CST
description: ITGAE description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3682) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3682  | ITGAE | ENSG00000083457 | 17p13.2 |



The gene enables [integrin binding](https://amigo.geneontology.org/amigo/term/GO:0005178) and [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872). It is located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886) and is part of the [integrin complex](https://amigo.geneontology.org/amigo/term/GO:0008305), specifically on the [external side of the plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009897). The gene is involved in several processes, including [cell-matrix adhesion](https://amigo.geneontology.org/amigo/term/GO:0007160), the [integrin-mediated signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007229), [cell adhesion mediated by integrin](https://amigo.geneontology.org/amigo/term/GO:0033627), and [cell-cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0098609).


The gene has a length of 5,412 base pairs (15.16% of all genes), a mature length of 2,492 base pairs, and a primary transcript length of 5,394 base pairs.


The gene ITGAE (NCBI ID: 3682) has been mentioned in [295 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ITGAE%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1990 and the middle 50% of publications occurring between 2011 and 2019. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning ITGAE, ranked by their scientific influence, include "[A functionally specialized population of mucosal CD103+ DCs induces Foxp3+ regulatory T cells via a TGF-beta and retinoic acid-dependent mechanism.](https://pubmed.ncbi.nlm.nih.gov/17620361)" (2007) (relative citation ratio: 43.22), "[Expansion and Activation of CD103(+) Dendritic Cell Progenitors at the Tumor Site Enhances Tumor Responses to Therapeutic PD-L1 and BRAF Inhibition.](https://pubmed.ncbi.nlm.nih.gov/27096321)" (2016) (relative citation ratio: 23.5), "[Critical Role for CD103(+)/CD141(+) Dendritic Cells Bearing CCR7 for Tumor Antigen Trafficking and Priming of T Cell Immunity in Melanoma.](https://pubmed.ncbi.nlm.nih.gov/27424807)" (2016) (relative citation ratio: 19.14), "[Co-expression of CD39 and CD103 identifies tumor-reactive CD8 T cells in human solid tumors.](https://pubmed.ncbi.nlm.nih.gov/30006565)" (2018) (relative citation ratio: 17.61), and "[Goblet cells deliver luminal antigen to CD103+ dendritic cells in the small intestine.](https://pubmed.ncbi.nlm.nih.gov/22422267)" (2012) (relative citation ratio: 16.88). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ITGAE](https://www.proteinatlas.org/ENSG00000083457-ITGAE) is an integrin receptor with evidence at the protein level. Its RNA is detected in some tissues, with notable expression in T-cells of unknown function (Cluster 27) and small intestine absorption (Cluster 25). In the brain, it is expressed in neurons with mixed function (Cluster 33), and in cell lines, it is found in lymphoma associated with humoral immune response (Cluster 46). Single-cell expression data also places it in T-cells related to T-cell receptor function (Cluster 7). The gene's role in cancer prognostics is generally unprognostic across various studies, including those for bladder, breast, cervical, colon, glioblastoma, head and neck, liver, lung, ovarian, pancreatic, stomach, testicular, and uterine cancers.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [KLF5](https://www.ncbi.nlm.nih.gov/gene/688) and [MYC](https://www.ncbi.nlm.nih.gov/gene/4609), each with 5 experiments. Additionally, [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169), [RBFOX2](https://www.ncbi.nlm.nih.gov/gene/23543), and [USF2](https://www.ncbi.nlm.nih.gov/gene/7392) were each found to be regulating in 4 experiments.



The GWAS data indicates associations with various disease conditions, including kidney disease, urinary system disease, and nervous system disease. Notably, there are links to mental health disorders, specifically developmental disorders of mental health, including attention deficit hyperactivity disorder. Additionally, the data highlights associations with eye and adnexa diseases, particularly retinal diseases such as retinal degeneration, age-related macular degeneration, degeneration of the macula and posterior pole, and macular degeneration.



The gene is expressed in multiple tissue samples, including lung, spleen, and small intestine. Additionally, it is expressed in various cell types such as Lymphoma Burkitt's (Raji), CD34+ cells, BDCA4+ dendritic cells, CD19+ B cells (neg. sel.), and CD14+ monocytes.


The proteins are involved in extracellular matrix organization and integrin cell surface interactions.



The analyzed protein sequence has a GRAVY value of -0.1502 (73.95th percentile), indicating a relatively hydrophilic nature. The charge at pH 7 is -32.57 (5.47th percentile), and the median structural flexibility is 0.998 (31.26th percentile). The protein has a notable affinity for helix (0.311, 42.37th percentile) and sheet (0.363, 68.36th percentile) structures, with a lower affinity for turns (0.282, 49.48th percentile). The instability index is 43.18 (32.87th percentile), and the isoelectric point is 5.48 (17.76th percentile). The protein is 1179 amino acids long (91.29th percentile) with a molecular weight of 130157.23 Da (91.17th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |