---
layout: post
title: CDC42
date: 2025-01-17 16:55 CST
description: CDC42 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/998) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 998  | CDC42 | ENSG00000070831 | 1p36.12 |



The gene is located in the [Golgi membrane](https://amigo.geneontology.org/amigo/term/GO:0000139), [storage vacuole](https://amigo.geneontology.org/amigo/term/GO:0000322), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0005789), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [cell-cell junction](https://amigo.geneontology.org/amigo/term/GO:0005911), [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [secretory granule](https://amigo.geneontology.org/amigo/term/GO:0030141), [filopodium](https://amigo.geneontology.org/amigo/term/GO:0030175), [midbody](https://amigo.geneontology.org/amigo/term/GO:0030496), [leading edge membrane](https://amigo.geneontology.org/amigo/term/GO:0031256), [cytoplasmic ribonucleoprotein granule](https://amigo.geneontology.org/amigo/term/GO:0036464), [neuron projection](https://amigo.geneontology.org/amigo/term/GO:0043005), [neuronal cell body](https://amigo.geneontology.org/amigo/term/GO:0043025), [dendritic spine](https://amigo.geneontology.org/amigo/term/GO:0043197), [apical part of cell](https://amigo.geneontology.org/amigo/term/GO:0045177), [phagocytic vesicle](https://amigo.geneontology.org/amigo/term/GO:0045335), [spindle midzone](https://amigo.geneontology.org/amigo/term/GO:0051233), [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062), [mitotic spindle](https://amigo.geneontology.org/amigo/term/GO:0072686), [Schaffer collateral - CA1 synapse](https://amigo.geneontology.org/amigo/term/GO:0098685), and [glutamatergic synapse](https://amigo.geneontology.org/amigo/term/GO:0098978). It is involved in various biological processes such as [sprouting angiogenesis](https://amigo.geneontology.org/amigo/term/GO:0002040), [cardiac conduction system development](https://amigo.geneontology.org/amigo/term/GO:0003161), [cardiac neural crest cell migration involved in outflow tract morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0003253), [endocytosis](https://amigo.geneontology.org/amigo/term/GO:0006897), [phagocytosis, engulfment](https://amigo.geneontology.org/amigo/term/GO:0006911), [actin filament organization](https://amigo.geneontology.org/amigo/term/GO:0007015), [Golgi organization](https://amigo.geneontology.org/amigo/term/GO:0007030), [regulation of mitotic nuclear division](https://amigo.geneontology.org/amigo/term/GO:0007088), [nuclear migration](https://amigo.geneontology.org/amigo/term/GO:0007097), [establishment or maintenance of cell polarity](https://amigo.geneontology.org/amigo/term/GO:0007163), [integrin-mediated signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007229), [protein localization](https://amigo.geneontology.org/amigo/term/GO:0008104), [regulation of lamellipodium assembly](https://amigo.geneontology.org/amigo/term/GO:0010591), [positive regulation of lamellipodium assembly](https://amigo.geneontology.org/amigo/term/GO:0010592), [substantia nigra development](https://amigo.geneontology.org/amigo/term/GO:0021762), [actin cytoskeleton organization](https://amigo.geneontology.org/amigo/term/GO:0030036), [macrophage differentiation](https://amigo.geneontology.org/amigo/term/GO:0030225), [positive regulation of cell growth](https://amigo.geneontology.org/amigo/term/GO:0030307), [positive regulation of cell migration](https://amigo.geneontology.org/amigo/term/GO:0030335), [positive regulation of pseudopodium assembly](https://amigo.geneontology.org/amigo/term/GO:0031274), [negative regulation of protein-containing complex assembly](https://amigo.geneontology.org/amigo/term/GO:0031333), [positive regulation of cytokinesis](https://amigo.geneontology.org/amigo/term/GO:0032467), [Cdc42 protein signal transduction](https://amigo.geneontology.org/amigo/term/GO:0032488), [regulation of actin cytoskeleton organization](https://amigo.geneontology.org/amigo/term/GO:0032956), [cell junction assembly](https://amigo.geneontology.org/amigo/term/GO:0034329), [adherens junction organization](https://amigo.geneontology.org/amigo/term/GO:0034332), [embryonic heart tube development](https://amigo.geneontology.org/amigo/term/GO:0035050), [dendritic cell migration](https://amigo.geneontology.org/amigo/term/GO:0036336), [neuropilin signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0038189), [positive regulation of neuron apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0043525), [modulation by host of viral process](https://amigo.geneontology.org/amigo/term/GO:0044788), [establishment of epithelial cell apical/basal polarity](https://amigo.geneontology.org/amigo/term/GO:0045198), [positive regulation of DNA replication](https://amigo.geneontology.org/amigo/term/GO:0045740), [positive regulation of JNK cascade](https://amigo.geneontology.org/amigo/term/GO:0046330), [filopodium assembly](https://amigo.geneontology.org/amigo/term/GO:0046847), [positive regulation of pinocytosis](https://amigo.geneontology.org/amigo/term/GO:0048549), [neuron fate determination](https://amigo.geneontology.org/amigo/term/GO:0048664), [regulation of filopodium assembly](https://amigo.geneontology.org/amigo/term/GO:0051489), [positive regulation of filopodium assembly](https://amigo.geneontology.org/amigo/term/GO:0051491), [regulation of stress fiber assembly](https://amigo.geneontology.org/amigo/term/GO:0051492), [positive regulation of stress fiber assembly](https://amigo.geneontology.org/amigo/term/GO:0051496), [establishment of Golgi localization](https://amigo.geneontology.org/amigo/term/GO:0051683), [positive regulation of synapse structural plasticity](https://amigo.geneontology.org/amigo/term/GO:0051835), [positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction](https://amigo.geneontology.org/amigo/term/GO:0051897), [regulation of attachment of spindle microtubules to kinetochore](https://amigo.geneontology.org/amigo/term/GO:0051988), [heart contraction](https://amigo.geneontology.org/amigo/term/GO:0060047), [Wnt signaling pathway, planar cell polarity pathway](https://amigo.geneontology.org/amigo/term/GO:0060071), [positive regulation of epithelial cell proliferation involved in lung morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0060501), [submandibular salivary gland formation](https://amigo.geneontology.org/amigo/term/GO:0060661), [dendritic spine morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0060997), [cellular response to type II interferon](https://amigo.geneontology.org/amigo/term/GO:0071346), [organelle transport along microtubule](https://amigo.geneontology.org/amigo/term/GO:0072384), [endothelin receptor signaling pathway involved in heart process](https://amigo.geneontology.org/amigo/term/GO:0086101), [actin filament branching](https://amigo.geneontology.org/amigo/term/GO:0090135), [positive regulation of intracellular protein transport](https://amigo.geneontology.org/amigo/term/GO:0090316), [regulation of modification of postsynaptic structure](https://amigo.geneontology.org/amigo/term/GO:0099159), [regulation of postsynapse organization](https://amigo.geneontology.org/amigo/term/GO:0099175), and [positive regulation of substrate adhesion-dependent cell spreading](https://amigo.geneontology.org/amigo/term/GO:1900026). Additionally, it enables various molecular functions such as [GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0003924), [G protein activity](https://amigo.geneontology.org/amigo/term/GO:0003925), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [GTP binding](https://amigo.geneontology.org/amigo/term/GO:0005525), [protein kinase binding](https://amigo.geneontology.org/amigo/term/GO:0019901), [GTP-dependent protein binding](https://amigo.geneontology.org/amigo/term/GO:0030742), [mitogen-activated protein kinase kinase kinase binding](https://amigo.geneontology.org/amigo/term/GO:0031435), [thioesterase binding](https://amigo.geneontology.org/amigo/term/GO:0031996), [GBD domain binding](https://amigo.geneontology.org/amigo/term/GO:0032427), [apolipoprotein A-I receptor binding](https://amigo.geneontology.org/amigo/term/GO:0034191), [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802), and [ubiquitin protein ligase activity](https://amigo.geneontology.org/amigo/term/GO:0061630). It is also part of the [Golgi transport complex](https://amigo.geneontology.org/amigo/term/GO:0017119) and [protein-containing complex](https://amigo.geneontology.org/amigo/term/GO:0032991).


The gene length is 8,262 base pairs (21.86% of all genes), the mature length is 988 base pairs (22.08% of all genes), and the primary transcript length is 7,100 base pairs (25.25% of all genes).


The gene CDC42 (NCBI ID: 998) has been mentioned in [1,131 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CDC42%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1990, and the middle 50% of publications occurred between 2004 and 2017.


The top 5 publications mentioning CDC42, ranked by their scientific influence, include "[Selective activation of the JNK signaling cascade and c-Jun transcriptional activity by the small GTPases Rac and Cdc42Hs.](https://pubmed.ncbi.nlm.nih.gov/7600582)" (1995) (relative citation ratio: 27.36), "[A brain serine/threonine protein kinase activated by Cdc42 and Rac1.](https://pubmed.ncbi.nlm.nih.gov/8107774)" (1994) (relative citation ratio: 25.07), "[The Rho family GTPases RhoA, Rac1, and CDC42Hs regulate transcriptional activation by SRF.](https://pubmed.ncbi.nlm.nih.gov/7600583)" (1995) (relative citation ratio: 22.38), "[An essential role for Rho, Rac, and Cdc42 GTPases in cell cycle progression through G1.](https://pubmed.ncbi.nlm.nih.gov/7652575)" (1995) (relative citation ratio: 19.85), and "[The Ras-related protein Cdc42Hs and bradykinin promote formation of peripheral actin microspikes and filopodia in Swiss 3T3 fibroblasts.](https://pubmed.ncbi.nlm.nih.gov/7891688)" (1995) (relative citation ratio: 16.63). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CDC42](https://www.proteinatlas.org/ENSG00000070831-CDC42) is a hydrolase involved in intellectual disability, with evidence at the protein level. It is detected in all RNA tissues and is localized to microtubules and the cytokinetic bridge. CDC42 is expressed in neutrophils with unknown function (Cluster 28) and is associated with cell proliferation in bone marrow (Cluster 77). In the brain, it is found in oligodendrocytes with mixed function (Cluster 28). Additionally, CDC42 is involved in mRNA processing in non-specific cell lines (Cluster 37) and has mixed functions in non-specific single cells (Cluster 45).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227) with 6 experiments, [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 5 experiments, [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613) with 5 experiments, [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930) with 5 experiments, and [BRD9](https://www.ncbi.nlm.nih.gov/gene/65980) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.





The gene is expressed in various immune cell types, including CD33+ myeloid cells and CD56+ NK cells, as well as in multiple brain regions such as the whole brain, fetal brain, hypothalamus, and occipital lobe. Additionally, it is expressed in blood samples.


The input data highlights various biological pathways, including those involved in immune system regulation such as the adaptive and innate immune systems, and signaling pathways like those mediated by EGFR, VEGF, and Robo receptors. Additionally, the data covers processes related to myogenesis, axon guidance, and developmental biology. Pathways involved in cellular signaling, such as EPH-Ephrin signaling, semaphorin interactions, and netrin-1 signaling, are also represented. Furthermore, the data includes pathways related to hemostasis, platelet activation, and GPCR signaling, as well as those involved in disease and signal transduction.



The analyzed protein sequence has a GRAVY value of -0.157 (73.33rd percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is -1.36 (45.17th percentile), and the median structural flexibility is 1.004 (59.18th percentile). The protein has a predicted secondary structure with 32.46% helix (54.76th percentile), 40.84% sheet (88.4th percentile), and 26.18% turn (30.82nd percentile) affinities. The instability index is 44.06 (35.91st percentile), and the isoelectric point is 6.16 (34.61st percentile). The protein is 191 amino acids long (13.19th percentile) with a molecular weight of 21258.36 Da (13.01st percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |