---
layout: post
title: UGP2
date: 2025-01-17 16:55 CST
description: UGP2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7360) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7360  | UGP2 | ENSG00000169764 | 2p15 |



The gene enables [UTP:glucose-1-phosphate uridylyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0003983), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [glucose binding](https://amigo.geneontology.org/amigo/term/GO:0005536), [pyrimidine ribonucleotide binding](https://amigo.geneontology.org/amigo/term/GO:0032557), [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802), and [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872). It is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). The gene is involved in the [glycogen metabolic process](https://amigo.geneontology.org/amigo/term/GO:0005977), [glycogen biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0005978), [UDP-glucose metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006011), [brain development](https://amigo.geneontology.org/amigo/term/GO:0007420), and [glucose 1-phosphate metabolic process](https://amigo.geneontology.org/amigo/term/GO:0019255).


The gene length is 11,482 base pairs (28.67% of all genes), the mature length is 1,648 base pairs (42.54% of all genes), and the primary transcript length is 6,568 base pairs (23.77% of all genes).


The gene UGP2, with NCBI ID 7360, has been mentioned in [29 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22UGP2%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727). The earliest year of publication mentioning UGP2 is 1963, and the middle 50% of publications range from 1967 to 2016.


The top 5 publications mentioning UGP2, ranked by their relative citation ratio (RCR), include "[STUDIES ON UDPG: ALPHA-1,4-GLUCAN ALPHA-4-GLUCOSYLTRANSFERASE. VII. CONVERSION OF THE ENZYME FROM GLUCOSE-6-PHOSPHATE-DEPENDENT TO INDEPENDENT FORM IN LIVER.](https://pubmed.ncbi.nlm.nih.gov/14235348)" (1964) (relative citation ratio: 5.64), "[Loss of UGP2 in brain leads to a severe epileptic encephalopathy, emphasizing that bi-allelic isoform-specific start-loss mutations of essential genes can cause genetic diseases.](https://pubmed.ncbi.nlm.nih.gov/31820119)" (2020) (relative citation ratio: 2.17), "[Determination of UDPG and UDPGA in tissues.](https://pubmed.ncbi.nlm.nih.gov/6083398)" (1967) (relative citation ratio: 1.75), "[Identification of UGP2 as a progression marker that promotes cell growth and motility in human glioma.](https://pubmed.ncbi.nlm.nih.gov/30816613)" (2019) (relative citation ratio: 0.77), and "[Expression of UGP2 and CFL1 expression levels in benign and malignant pancreatic lesions and their clinicopathological significance.](https://pubmed.ncbi.nlm.nih.gov/29347944)" (2018) (relative citation ratio: 0.74). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[UGP2](https://www.proteinatlas.org/ENSG00000169764-UGP2) is a gene encoding a protein with nucleotidyltransferase and transferase molecular functions. It has been implicated in epilepsy as a disease variant. Evidence for this gene is available at the protein level. The gene's RNA is detected in all tissues, with notable expression in the nucleoplasm, centrosome, and mitochondria. Its expression is associated with various clusters, including Cluster 42 for vesicular transport in blood, Cluster 19 for basic cellular processes in tissues, Cluster 52 for synaptic function in the brain, Cluster 64 for ECM organization in connective tissue cells, and Cluster 17 for lactation in breast glandular cells.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) with 6 experiments, [JUN](https://www.ncbi.nlm.nih.gov/gene/3725) with 5 experiments, [E2F1](https://www.ncbi.nlm.nih.gov/gene/1869) with 5 experiments, [EP300](https://www.ncbi.nlm.nih.gov/gene/2033) with 5 experiments, and [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The input data indicates associations with various disease conditions, including mental health disorders, substance-related disorders, substance dependence, and specifically, alcohol dependence.



The gene is expressed in various tissues, including muscle and liver, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the Subthalamic Nucleus, Temporal Lobe, Skeletal Muscle, Tongue, and Globus Pallidus.


The input data highlights various pathways and disorders related to metabolism, particularly focusing on carbohydrate metabolism and associated diseases. Key pathways include metabolism of carbohydrates, glycogen synthesis, glucose metabolism, and biological oxidations. Notable disorders linked to these pathways are myoclonic epilepsy of Lafora, glycogen storage diseases, and several defects causing conditions such as hypermethioninemia, hyperbilirubinemia, hemolytic anemia, and glutathione synthetase deficiency. Additionally, the data covers phase II conjugation processes like glucuronidation and the formation of the active cofactor, UDP-glucuronate.


The mouse ortholog gene 216558 and its human ortholog gene 7360 are associated with phenotypes including hemorrhage and preweaning lethality with complete penetrance, affecting both sexes.


The analyzed protein sequence has a GRAVY value of -0.293 (58.77th percentile), indicating a hydrophilic nature. It exhibits a charge of 2.496 at pH 7 (59.27th percentile) and a median structural flexibility of 1.005 (61.95th percentile). The protein's secondary structure is predicted to be 31.3% helix (43.93rd percentile), 38.6% sheet (81.3rd percentile), and 30.5% turn (67.51st percentile). The instability index is 33.63 (7.99th percentile), with an isoelectric point of 8.15 (60.54th percentile). The protein has a length of 508 amino acids (59.26th percentile) and a molecular weight of 56939.55 Da (59.52nd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |