---
layout: post
title: B3GNT2
date: 2025-01-17 16:55 CST
description: B3GNT2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10678) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10678  | B3GNT2 | ENSG00000170340 | 2p15 |



The gene is located in the [Golgi membrane](https://amigo.geneontology.org/amigo/term/GO:0000139) and is active in this cellular component. It enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is involved in various biological processes, including [protein O-linked glycosylation](https://amigo.geneontology.org/amigo/term/GO:0006493), [axon guidance](https://amigo.geneontology.org/amigo/term/GO:0007411), and [sensory perception of smell](https://amigo.geneontology.org/amigo/term/GO:0007608). Additionally, it plays a role in [O-glycan processing](https://amigo.geneontology.org/amigo/term/GO:0016266), [keratan sulfate biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0018146), and [poly-N-acetyllactosamine biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0030311). The gene also enables several molecular functions, such as [beta-galactosyl-N-acetylglucosaminylgalactosylglucosyl-ceramide beta-1,3-acetylglucosaminyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0008457), [UDP-galactose:beta-N-acetylglucosamine beta-1,3-galactosyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0008499), and [N-acetyllactosaminide beta-1,3-N-acetylglucosaminyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0008532). Furthermore, it is involved in the [cellular response to leukemia inhibitory factor](https://amigo.geneontology.org/amigo/term/GO:1990830).


The gene length is 29,333 base pairs (52.12% of all genes), the mature length is 2,232.5 base pairs (58.31% of all genes), and the primary transcript length is 24,394 base pairs (57.21% of all genes).


The gene B3GNT2, with NCBI ID 10678, has been mentioned in [131 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22B3GNT2%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink). The earliest year of publication mentioning this gene is 1990, and the middle 50% of publications occurred between 2008 and 2016.


The top 5 publications mentioning B3GNT2, ranked by their scientific influence, include "[The structural basis for agonist and partial agonist action on a β(1)-adrenergic receptor.](https://pubmed.ncbi.nlm.nih.gov/21228877)" (2011) (relative citation ratio: 14.76), "[Integrin alpha 2 beta 1 is upregulated in fibroblasts and highly aggressive melanoma cells in three-dimensional collagen lattices and mediates the reorganization of collagen I fibrils.](https://pubmed.ncbi.nlm.nih.gov/1955483)" (1991) (relative citation ratio: 9.54), "[Transforming growth factor-beta 1 modulates beta 1 and beta 5 integrin receptors and induces the de novo expression of the alpha v beta 6 heterodimer in normal human keratinocytes: implications for wound healing.](https://pubmed.ncbi.nlm.nih.gov/7537276)" (1995) (relative citation ratio: 6.65), "[MT1-MMP-dependent control of skeletal stem cell commitment via a β1-integrin/YAP/TAZ signaling axis.](https://pubmed.ncbi.nlm.nih.gov/23685250)" (2013) (relative citation ratio: 5.65), and "[Beta-1 and Not Beta-3 Adrenergic Receptors May Be the Primary Regulator of Human Brown Adipocyte Metabolism.](https://pubmed.ncbi.nlm.nih.gov/31867674)" (2020) (relative citation ratio: 4.88). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[B3GNT2](https://www.proteinatlas.org/ENSG00000170340-B3GNT2) is a glycosyltransferase and transferase with evidence at the protein level. It is detected in all tissues and is primarily localized in vesicles. The gene is part of Cluster 3 in blood expression, indicating involvement in nuclear processes, and Cluster 77 in tissue expression, suggesting a role in cell proliferation within bone marrow. In the brain, it is associated with Cluster 33, which includes neurons with mixed functions. Additionally, B3GNT2 is part of Cluster 55 in cell line expression, linked to immune response in lymphoma, and Cluster 48 in single-cell expression, related to signal transduction in smooth muscle cells.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [EP300](https://www.ncbi.nlm.nih.gov/gene/2033) with 6 experiments, [RARA](https://www.ncbi.nlm.nih.gov/gene/5914) with 5 experiments, [AR](https://www.ncbi.nlm.nih.gov/gene/367) with 4 experiments, [INTS11](https://www.ncbi.nlm.nih.gov/gene/54973) with 4 experiments, and [KLF5](https://www.ncbi.nlm.nih.gov/gene/688) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The input data indicates associations with various disease conditions, including mental health disorders, substance-related disorders, substance dependence, and specifically, alcohol dependence.



The gene is expressed in various tissues and cell types, including the salivary gland and blood, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the retina, B lymphoblasts, CD34+ cells, CD4+ T cells, CD8+ T cells, CD71+ early erythroid cells, and leukemia lymphoblastic cells (MOLT-4).


The input data includes various pathways and diseases related to carbohydrate metabolism, glycosaminoglycan metabolism, and glycosylation processes. Notable conditions include Myoclonic epilepsy of Lafora, Glycogen storage diseases, and several types of Mucopolysaccharidoses (MPS), such as MPS VII - Sly syndrome, MPS VI - Maroteaux-Lamy syndrome, and MPS I - Hurler syndrome. Additionally, the data highlights defects in specific genes like B3GAT3, B4GALT1, and CHST14, which are associated with conditions like JDSSDHD, B4GALT1-CDG (CDG-2d), and EDS, musculocontractural type, respectively. The data also covers broader categories such as diseases associated with glycosaminoglycan metabolism, diseases of glycosylation, and post-translational protein modification.



The analyzed protein sequence has a GRAVY value of -0.379 (48.73rd percentile), indicating a hydrophilic nature. It exhibits a charge of 6.69 at pH 7.0 (73.31st percentile) and a median structural flexibility of 0.997 (26.58th percentile). The protein's secondary structure is predicted to be 30.23% helix (34.01st percentile), 38.29% sheet (80.05th percentile), and 28.97% turn (55.67th percentile). The instability index is 40.36 (23.33rd percentile), with an isoelectric point of 8.75 (72.35th percentile). The protein has a length of 397 amino acids (44.91st percentile) and a molecular weight of 46021.45 Da (46.94th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |