---
layout: post
title: ADK
date: 2025-01-17 16:55 CST
description: ADK description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/132) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 132  | ADK | ENSG00000156110 | 10q22.2|10q11-q24  |



The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [adenosine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004001), [deoxyadenosine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004136), and [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524). It is active in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). The gene is involved in various processes including [purine nucleobase metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006144), [purine ribonucleoside salvage](https://amigo.geneontology.org/amigo/term/GO:0006166), [dATP biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006175), [ribonucleoside monophosphate biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0009156), [phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0016310), [GMP salvage](https://amigo.geneontology.org/amigo/term/GO:0032263), [AMP salvage](https://amigo.geneontology.org/amigo/term/GO:0044209), and [dAMP salvage](https://amigo.geneontology.org/amigo/term/GO:0106383). Additionally, it enables [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872).


The gene length is 11,679 base pairs (29.11% of all genes), the mature length is 2,900 base pairs, and the primary transcript length is 11,677 base pairs.


The gene ADK (NCBI ID: 132) has been mentioned in [152 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ADK%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1967 and the middle 50% of publications occurring between 1992 and 2015. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning ADK, ranked by their scientific influence, include "[Adenosine kinase from rabbit liver. II. Substrate and inhibitor specificity.](https://pubmed.ncbi.nlm.nih.gov/218934)" (1979) (relative citation ratio: 7.58), "[Adenosine kinase: exploitation for therapeutic gain.](https://pubmed.ncbi.nlm.nih.gov/23592612)" (2013) (relative citation ratio: 6.92), "[S-adenosylhomocysteine toxicity in normal and adenosine kinase-deficient lymphoblasts of human origin.](https://pubmed.ncbi.nlm.nih.gov/221926)" (1979) (relative citation ratio: 5.73), "[Adenosine-Metabolizing Enzymes, Adenosine Kinase and Adenosine Deaminase, in Cancer.](https://pubmed.ncbi.nlm.nih.gov/35327609)" (2022) (relative citation ratio: 4.92), and "[Large-scale allosteric conformational transitions of adenylate kinase appear to involve a population-shift mechanism.](https://pubmed.ncbi.nlm.nih.gov/18000050)" (2007) (relative citation ratio: 4.51). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ADK](https://www.proteinatlas.org/ENSG00000156110-ADK) is a gene encoding a protein with kinase and transferase molecular functions. It is involved in disease variants and has evidence at the protein level. The gene is detected in all tissues, with notable expression in the nucleoplasm, plasma membrane, and cytosol. ADK is part of several expression clusters, including B-cells - Immunoglobulins in blood, Liver & Kidney - Metabolism in tissues, Neurons - Synaptic function in the brain, Non-specific - Mitochondria in cell lines, and Neurons - Neuronal signaling in single cells.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [SMAD2](https://www.ncbi.nlm.nih.gov/gene/4087), [CREB1](https://www.ncbi.nlm.nih.gov/gene/1385), [PGR](https://www.ncbi.nlm.nih.gov/gene/5241), [MYB](https://www.ncbi.nlm.nih.gov/gene/4602), and [STAT3](https://www.ncbi.nlm.nih.gov/gene/6774), each shown to be regulating in 5 experiments.



The GWAS data indicates associations with several disease conditions, including coronary artery disease, artery disease, vascular disease, cardiovascular system disease, disease of anatomical entity, gastrointestinal system disease, mouth disease, and tooth disease.



The gene is expressed in multiple tissue samples, including liver and skin, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in various tissues and cell types such as Cardiac Myocytes, Liver, 721_B_lymphoblasts, CD19+ B Cells (neg. sel.), Leukemia promyelocytic-HL-60, Smooth Muscle, and Bronchial Epithelial Cells.


The input data highlights pathways involved in metabolism, specifically focusing on the metabolism of nucleotides. Within this broader category, the data emphasizes purine metabolism, with a particular focus on purine salvage.


The mouse ortholog gene 11534 and its human ortholog gene 132 are associated with several phenotypic abnormalities, including abnormal morphologies of the lung, skin, spleen, and thymus, as well as enlargement of the lung and thymus. Notably, these phenotypes are observed in both sexes. Additionally, the gene is linked to complete penetrance preweaning lethality.


The analyzed protein sequence has a GRAVY value of -0.318 (55.89th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -3.64 (37.59th percentile), and the median structural flexibility is 1.003 (56.8th percentile). The protein has a high affinity for helix structures (37.85%, 88.19th percentile) and sheet structures (33.70%, 49.92nd percentile), with a lower affinity for turn structures (23.48%, 12.37th percentile). The instability index is 39.47 (20.68th percentile), and the isoelectric point is 6.24 (36.52nd percentile). The protein sequence is 362 amino acids long (40th percentile) with a molecular weight of 40544.96 Da (40.18th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |