---
layout: post
title: HSPA1A
date: 2025-01-17 16:55 CST
description: HSPA1A description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3303) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3303  | HSPA1A | ENSG00000204389 | 6p21.33 |



The gene is involved in [negative regulation of transcription by RNA polymerase II](https://amigo.geneontology.org/amigo/term/GO:0000122) and enables [virus receptor activity](https://amigo.geneontology.org/amigo/term/GO:0001618), [G protein-coupled receptor binding](https://amigo.geneontology.org/amigo/term/GO:0001664), [transcription corepressor activity](https://amigo.geneontology.org/amigo/term/GO:0003714), [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [signaling receptor binding](https://amigo.geneontology.org/amigo/term/GO:0005102), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524). It is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615), [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [mitochondrion](https://amigo.geneontology.org/amigo/term/GO:0005739), [endoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0005783), [centrosome](https://amigo.geneontology.org/amigo/term/GO:0005813), [centriole](https://amigo.geneontology.org/amigo/term/GO:0005814), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925), [inclusion body](https://amigo.geneontology.org/amigo/term/GO:0016234), [aggresome](https://amigo.geneontology.org/amigo/term/GO:0016235), [nuclear speck](https://amigo.geneontology.org/amigo/term/GO:0016607), [vesicle](https://amigo.geneontology.org/amigo/term/GO:0031982), [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062), [blood microparticle](https://amigo.geneontology.org/amigo/term/GO:0072562), and [ficolin-1-rich granule lumen](https://amigo.geneontology.org/amigo/term/GO:1904813). It is also involved in various biological processes such as [mRNA catabolic process](https://amigo.geneontology.org/amigo/term/GO:0006402), [response to unfolded protein](https://amigo.geneontology.org/amigo/term/GO:0006986), [lysosomal transport](https://amigo.geneontology.org/amigo/term/GO:0007041), [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), and many others, highlighting its multifaceted roles in cellular functions.


The gene length is 17,698 base pairs (39.01% of all genes), the mature length is 2,367 base pairs (61.22% of all genes), and the primary transcript length is 17,361 base pairs (47.84% of all genes).


The gene HSPA1A (NCBI ID: 3303) has been mentioned in [400 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22HSPA1A%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1988, and the middle 50% of publications occurred between 2002 and 2015.


The top 5 publications mentioning HSPA1A, ranked by their scientific influence, include "[Membrane-associated Hsp72 from tumor-derived exosomes mediates STAT3-dependent immunosuppressive function of mouse and human myeloid-derived suppressor cells.](https://pubmed.ncbi.nlm.nih.gov/20093776)" (2010) (relative citation ratio: 17.63), "[HSP72 protects against obesity-induced insulin resistance.](https://pubmed.ncbi.nlm.nih.gov/18223156)" (2008) (relative citation ratio: 9.38), "[A stress-inducible 72-kDa heat-shock protein (HSP72) is expressed on the surface of human tumor cells, but not on normal cells.](https://pubmed.ncbi.nlm.nih.gov/7705958)" (1995) (relative citation ratio: 8.3), "[LNCAROD is stabilized by m6A methylation and promotes cancer progression via forming a ternary complex with HSPA1A and YBX1 in head and neck squamous cell carcinoma.](https://pubmed.ncbi.nlm.nih.gov/32216017)" (2020) (relative citation ratio: 7.42), and "[Dual targeting of HSC70 and HSP72 inhibits HSP90 function and induces tumor-specific apoptosis.](https://pubmed.ncbi.nlm.nih.gov/18772114)" (2008) (relative citation ratio: 5.52). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[HSPA1A](https://www.proteinatlas.org/ENSG00000204389-HSPA1A) is a gene with multiple molecular functions, including chaperone, host cell receptor for virus entry, and receptor. It is detected in many tissues and is localized in various subcellular locations such as vesicles, perinuclear theca, calyx, flagellar centriole, and annulus. The gene is part of Cluster 4 in blood expression, which is associated with neutrophil degranulation. In tissue expression, it belongs to Cluster 43, which is involved in mitochondrial translation. Additionally, HSPA1A is part of Cluster 55 in brain expression and Cluster 66 in cell line expression, both of which are non-specific but related to transcription and mitochondria, respectively. In single-cell expression, it is part of Cluster 80, which is non-specific but related to transcription.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), [PGR](https://www.ncbi.nlm.nih.gov/gene/5241), and [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227), each with 5 experiments. Additionally, [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930) and [GATAD2B](https://www.ncbi.nlm.nih.gov/gene/57459) were found to be regulating in 4 experiments each.


The transcription factor [SREBF1](https://www.ncbi.nlm.nih.gov/gene/6426) was found to be regulating genes in one Perturb-seq experiment, according to data from GTRD's database of transcription factor binding sites and the study published in [PubMed](https://pubmed.ncbi.nlm.nih.gov/35688146/).




The gene is expressed in multiple tissue samples, including the prostate, atrioventricular node, ciliary ganglion, and olfactory bulb.


The input data highlights various biological processes and pathways, including gene expression, influenza infection and life cycle, export of viral ribonucleoproteins from the nucleus, cellular responses to stress, and the regulation of mRNA stability by proteins that bind AU-rich elements. Additionally, it covers the attenuation phase, HSF1-dependent transactivation and activation, cellular response to heat stress, regulation of HSF1-mediated heat shock response, and the presence of viral RNP complexes in the host cell nucleus. The data also includes information related to disease.



The analyzed protein sequence has a GRAVY value of -0.383 (48.11th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -10.82 (21.26th percentile). The median structural flexibility is 1.007 (72.44th percentile), with a predicted secondary structure composition of 33.23% helix (61.34th percentile), 34.32% sheet (54.23rd percentile), and 29.33% turn (58.6th percentile). The instability index is 32.17 (5.97th percentile), and the isoelectric point is 5.48 (17.75th percentile). The protein is 641 amino acids long (71.57th percentile) with a molecular weight of 70051.37 Da (70.56th percentile). For more details on the sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |