---
layout: post
title: HLA-F
date: 2025-01-17 16:55 CST
description: HLA-F description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3134) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3134  | HLA-F | ENSG00000204642 | 6p22.1 |



The gene is located in the [Golgi membrane](https://amigo.geneontology.org/amigo/term/GO:0000139), [lysosomal membrane](https://amigo.geneontology.org/amigo/term/GO:0005765), [endoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0005783), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [ER to Golgi transport vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0012507), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [phagocytic vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030670), [early endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0031901), [recycling endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0055038), and [lumenal side of endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0098553). It is active in the [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615) and the [external side of plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009897). The gene is involved in the [positive regulation of T cell mediated cytotoxicity](https://amigo.geneontology.org/amigo/term/GO:0001916), [antigen processing and presentation of endogenous peptide antigen via MHC class Ib](https://amigo.geneontology.org/amigo/term/GO:0002476), [antigen processing and presentation of exogenous peptide antigen via MHC class Ib](https://amigo.geneontology.org/amigo/term/GO:0002477), [antigen processing and presentation of endogenous peptide antigen via MHC class I via ER pathway, TAP-independent](https://amigo.geneontology.org/amigo/term/GO:0002486), [negative regulation of T cell cytokine production](https://amigo.geneontology.org/amigo/term/GO:0002725), [negative regulation of natural killer cell cytokine production](https://amigo.geneontology.org/amigo/term/GO:0002728), [positive regulation of natural killer cell cytokine production](https://amigo.geneontology.org/amigo/term/GO:0002729), [immune response](https://amigo.geneontology.org/amigo/term/GO:0006955), [negative regulation of natural killer cell degranulation](https://amigo.geneontology.org/amigo/term/GO:0043322), [positive regulation of natural killer cell degranulation](https://amigo.geneontology.org/amigo/term/GO:0043323), and [negative regulation of natural killer cell mediated cytotoxicity](https://amigo.geneontology.org/amigo/term/GO:0045953). The gene enables [signaling receptor binding](https://amigo.geneontology.org/amigo/term/GO:0005102), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [beta-2-microglobulin binding](https://amigo.geneontology.org/amigo/term/GO:0030881), [peptide antigen binding](https://amigo.geneontology.org/amigo/term/GO:0042605), [TAP1 binding](https://amigo.geneontology.org/amigo/term/GO:0046978), [TAP2 binding](https://amigo.geneontology.org/amigo/term/GO:0046979), and [14-3-3 protein binding](https://amigo.geneontology.org/amigo/term/GO:0071889). Additionally, it is part of the [MHC class Ib protein complex](https://amigo.geneontology.org/amigo/term/GO:0032398) and the [MHC class I protein complex](https://amigo.geneontology.org/amigo/term/GO:0042612).


The gene length is 115,973 base pairs (84.54% of all genes), the mature length is 3,741 base pairs, and the primary transcript length is 115,585 base pairs.


HLA-F (Gene ID: 3134) has been mentioned in [58 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22HLA-F%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest publication mentioning HLA-F was in 1990, with the middle 50% of publications occurring between 2008 and 2019.


The top 5 publications in citations mentioning HLA-F are as follows. The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, and benchmarking to the median for NIH publications, the value of which is set at 1.0. The most cited publications include:

- [Open conformers of HLA-F are high-affinity ligands of the activating NK-cell receptor KIR3DS1.](https://pubmed.ncbi.nlm.nih.gov/27455421) (2016) (relative citation ratio: 6.07)
- [Differential expression of HLA-E, HLA-F, and HLA-G transcripts in human tissue.](https://pubmed.ncbi.nlm.nih.gov/2249951) (1990) (relative citation ratio: 4.56)
- [Definitive class I human leukocyte antigen expression in gestational placentation: HLA-F, HLA-E, HLA-C, and HLA-G in extravillous trophoblast invasion on placentation, pregnancy, and parturition.](https://pubmed.ncbi.nlm.nih.gov/28185362) (2017) (relative citation ratio: 4.44)
- [HLA-F and MHC-I Open Conformers Bind Natural Killer Cell Ig-Like Receptor KIR3DS1.](https://pubmed.ncbi.nlm.nih.gov/27649529) (2016) (relative citation ratio: 3.55)
- [Endometrial HLA-F expression is influenced by genotypes and correlates differently with immune cell infiltration in IVF and recurrent implantation failure patients.](https://pubmed.ncbi.nlm.nih.gov/35689445) (2022) (relative citation ratio: 3.49)

Citation counts are from [iCite](https://icite.od.nih.gov).


[HLA-F](https://www.proteinatlas.org/ENSG00000204642-HLA-F) is a gene with evidence at the protein level. Its RNA is detected in many tissues, with notable expression in the spleen, where it is involved in immune response (Cluster 47). In the brain, it is associated with non-specific ribosomal functions (Cluster 7). Additionally, HLA-F is expressed in cell lines related to antiviral immune response (Cluster 31) and in single cells, specifically in Langerhans cells involved in innate immune response (Cluster 14).


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), [SPI1](https://www.ncbi.nlm.nih.gov/gene/6688), [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), and [ZBTB7A](https://www.ncbi.nlm.nih.gov/gene/51341), each shown to be regulating in 5 experiments.



The input data indicates associations with a range of disease conditions, including gastrointestinal system cancer, cancer, disease of cellular proliferation, organ system cancer, vascular disease, cardiovascular system disease, disease, disease of anatomical entity, nervous system disease, central nervous system disease, neurodegenerative disease, demyelinating disease, multiple sclerosis, Beh√ßet's disease, vasculitis, pharynx cancer, and nasopharynx carcinoma.


The HLA-F gene has been associated with [cognitive performance](https://pubmed.ncbi.nlm.nih.gov/30038396), which maps closely to the MeSH term [Cognition](https://meshb.nlm.nih.gov/record/ui?ui=D003071) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method.


The gene is expressed in various tissues, including blood, spleen, and small intestine, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Lymphoma Burkitt's (Raji), CD56+ NK cells, CD4+ T cells, CD8+ T cells, CD14+ monocytes, whole blood, and lymph nodes.


The input data highlights several key pathways involved in the immune system, including antigen presentation and processing, cytokine signaling, and interferon signaling. Specific pathways mentioned are 'Antigen Presentation: Folding, assembly and peptide loading of class I MHC', 'Cytokine Signaling in Immune system', 'Interferon Signaling', 'Interferon gamma signaling', 'Interferon alpha/beta signaling', 'Antigen processing-Cross presentation', 'ER-Phagosome pathway', 'Class I MHC mediated antigen processing & presentation', 'Adaptive Immune System', 'Immune System', 'Immunoregulatory interactions between a Lymphoid and a non-Lymphoid cell', and 'Endosomal/Vacuolar pathway'. These pathways collectively contribute to the body's defense mechanisms and immune responses.



The analyzed protein sequence has a GRAVY value of -0.535 (31.46th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -11.59 (20.04th percentile). The median structural flexibility is 0.997 (29.7th percentile), with a predicted secondary structure composition of 31.21% helix (43.2nd percentile), 34.68% sheet (56.64th percentile), and 25.43% turn (24.7th percentile). The instability index is 41.94 (28.44th percentile), and the isoelectric point is 5.37 (15.05th percentile). The protein has a length of 346 amino acids (37.32nd percentile) and a molecular weight of 39061.2 Da (38.1st percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |