---
layout: post
title: FCGR3B
date: 2025-01-17 16:55 CST
description: FCGR3B description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2215) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2215  | FCGR3B | ENSG00000162747 | 1q23.3 |



The gene is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [secretory granule membrane](https://amigo.geneontology.org/amigo/term/GO:0030667), [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062), and [side of membrane](https://amigo.geneontology.org/amigo/term/GO:0098552). It is involved in [antibody-dependent cellular cytotoxicity](https://amigo.geneontology.org/amigo/term/GO:0001788), [immune response](https://amigo.geneontology.org/amigo/term/GO:0006955), and [cell surface receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007166). The gene is active in the [external side of plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009897) and enables [IgG receptor activity](https://amigo.geneontology.org/amigo/term/GO:0019770), [IgG binding](https://amigo.geneontology.org/amigo/term/GO:0019864), and [GPI anchor binding](https://amigo.geneontology.org/amigo/term/GO:0034235).


The gene length is 4,199 base pairs (11.77% of all genes), the mature length is 2,044 base pairs, and the primary transcript length is 3,955 base pairs.


The gene FCGR3B (NCBI ID: 2215) has been mentioned in [131 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22FCGR3B%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1992 and the middle 50% of publications occurring between 2001 and 2014. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning FCGR3B, ranked by their scientific influence as measured by the [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541), include studies such as "[Persistence of SARS CoV-2 S1 Protein in CD16+ Monocytes in Post-Acute Sequelae of COVID-19 (PASC) up to 15 Months Post-Infection.](https://pubmed.ncbi.nlm.nih.gov/35082777)" (2022) with an RCR of 21.99, "[Persistent Oxidative Stress and Inflammasome Activation in CD14<sup>high</sup>CD16<sup>-</sup> Monocytes From COVID-19 Patients.](https://pubmed.ncbi.nlm.nih.gov/35095880)" (2022) with an RCR of 6.88, "[FCGR3B copy number variation is associated with susceptibility to systemic, but not organ-specific, autoimmunity.](https://pubmed.ncbi.nlm.nih.gov/17529978)" (2007) with an RCR of 6.75, "[Immobilized immune complexes induce neutrophil extracellular trap release by human neutrophil granulocytes via FcÎ³RIIIB and Mac-1.](https://pubmed.ncbi.nlm.nih.gov/25024378)" (2014) with an RCR of 5.79, and "[Glycoengineered CD20 antibody obinutuzumab activates neutrophils and mediates phagocytosis through CD16B more efficiently than rituximab.](https://pubmed.ncbi.nlm.nih.gov/24106207)" (2013) with an RCR of 4.81. The RCR values measure the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[FCGR3B](https://www.proteinatlas.org/ENSG00000162747-FCGR3B) is an IgG-binding protein and receptor with evidence at the protein level. It is a known FDA-approved drug target and is detected in many tissues. The gene is highly expressed in neutrophils, particularly in degranulation processes, and is associated with immune response in lymphoid tissue. Additionally, it is expressed in the vasculature of the brain and in myeloid leukemia cell lines, where it plays a role in the innate immune response.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), [SPI1](https://www.ncbi.nlm.nih.gov/gene/6688), [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), and [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), each with 2 experiments. Additionally, [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) was found to be regulating in 1 experiment.




FCGR3B has been associated with several hematological traits through genome-wide association studies. Specifically, it has been linked to [Basophil count](https://pubmed.ncbi.nlm.nih.gov/32888493) and [Basophil percentage of white cells](https://pubmed.ncbi.nlm.nih.gov/32888494), both of which map closely to the MeSH term [Basophils](https://meshb.nlm.nih.gov/record/ui?ui=D001491) using the BioWordVec method. Additionally, FCGR3B is associated with [Monocyte count](https://pubmed.ncbi.nlm.nih.gov/32888494), which maps to the MeSH term [Monocytes](https://meshb.nlm.nih.gov/record/ui?ui=D009000) through embedding match.


The gene is expressed in various tissues, including lung, blood, and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in CD56+ NK cells, CD14+ monocytes, whole blood, medulla oblongata, placenta, bone marrow, and fetal lung.




The analyzed protein sequence has a GRAVY value of -0.091 (78.75th percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is -2.83 (40.27th percentile), and the median structural flexibility is 0.994 (19.03rd percentile). The protein has a propensity for helix, turn, and sheet structures of 27.9% (16.4th percentile), 29.2% (57.46th percentile), and 42.1% (90.82nd percentile), respectively. The instability index is 40.08 (22.47th percentile), with an isoelectric point of 6.22 (36.18th percentile). The sequence length is 233 amino acids (19.26th percentile), and the molecular weight is 26242.56 Da (19.71st percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |