---
layout: post
title: EPHX2
date: 2025-01-17 16:55 CST
description: EPHX2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2053) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2053  | EPHX2 | ENSG00000120915 | 8p21.2-p21.1 |



The gene enables [magnesium ion binding](https://amigo.geneontology.org/amigo/term/GO:0000287) and [epoxide hydrolase activity](https://amigo.geneontology.org/amigo/term/GO:0004301). It is located in the [peroxisome](https://amigo.geneontology.org/amigo/term/GO:0005777), specifically in the [peroxisomal matrix](https://amigo.geneontology.org/amigo/term/GO:0005782), and is also found in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). Additionally, it is involved in the [response to toxic substance](https://amigo.geneontology.org/amigo/term/GO:0009636), [positive regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0010628), [dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0016311), [cholesterol homeostasis](https://amigo.geneontology.org/amigo/term/GO:0042632), [stilbene catabolic process](https://amigo.geneontology.org/amigo/term/GO:0046272), [phospholipid dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0046839), [regulation of cholesterol metabolic process](https://amigo.geneontology.org/amigo/term/GO:0090181), and [epoxide metabolic process](https://amigo.geneontology.org/amigo/term/GO:0097176). The gene also enables [toxic substance binding](https://amigo.geneontology.org/amigo/term/GO:0015643), [phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0016791), [10-hydroxy-9-(phosphonooxy)octadecanoate phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0033885), [lipid phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0042577), [protein homodimerization activity](https://amigo.geneontology.org/amigo/term/GO:0042803), and [lysophosphatidic acid phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0052642). Furthermore, it is located in the [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062).


The gene length is 12,873 base pairs (31.35% of all genes), the mature length is 502 base pairs, and the primary transcript length is 787 base pairs.


The gene EPHX2 (NCBI ID: 2053) has been mentioned in [318 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22EPHX2%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1974 and the middle 50% of publications occurring between 2008 and 2019. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning EPHX2, ranked by their scientific influence, include "[Impact of soluble epoxide hydrolase and epoxyeicosanoids on human health.](https://pubmed.ncbi.nlm.nih.gov/23020295)" (2013) (relative citation ratio: 13.77), "[Epoxides and soluble epoxide hydrolase in cardiovascular physiology.](https://pubmed.ncbi.nlm.nih.gov/22298653)" (2012) (relative citation ratio: 9.28), "[Soluble epoxide hydrolase is a therapeutic target for acute inflammation.](https://pubmed.ncbi.nlm.nih.gov/15994227)" (2005) (relative citation ratio: 8.38), "[Maternal glyphosate exposure causes autism-like behaviors in offspring through increased expression of soluble epoxide hydrolase.](https://pubmed.ncbi.nlm.nih.gov/32398374)" (2020) (relative citation ratio: 8.01), and "[A rapid assay for epoxide hydratase activity with benzo (a)pyrene 4,5-(K-region)-oxide as substrate.](https://pubmed.ncbi.nlm.nih.gov/962086)" (1976) (relative citation ratio: 7.37). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[EPHX2](https://www.proteinatlas.org/ENSG00000120915-EPHX2) is a multifunctional enzyme with hydrolase activity, supported by evidence at the protein level. It is primarily located in the cytosol and is detected in all tissues. The gene is highly expressed in the liver and intestine, where it plays a role in lipid metabolism. In the brain, it is specifically expressed in the choroid plexus. Additionally, EPHX2 is expressed in stomach and colon cancer cell lines, as well as in proximal enterocytes, suggesting a role in digestion.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [PGR](https://www.ncbi.nlm.nih.gov/gene/5241) and [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), each with 6 experiments. Additionally, [EP300](https://www.ncbi.nlm.nih.gov/gene/2033) and [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930) were found in 5 experiments each, while [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170) was identified in 4 experiments.



The input data from GWAS indicates associations with a range of disease conditions, including cognitive disorder, disease of mental health, mood disorder, substance-related disorder, mental depression, anxiety disorder, obsessive-compulsive disorder, tic disorder, Gilles de la Tourette syndrome, developmental disorder of mental health, specific developmental disorder, attention deficit hyperactivity disorder, substance abuse, and antidepressant type abuse.


EPHX2 has been associated with several traits and diseases through genome-wide association studies. Notably, it has been linked to [Protein quantitative trait loci (liver)](https://pubmed.ncbi.nlm.nih.gov/32778093), which maps closely to the MeSH term [Genes](https://meshb.nlm.nih.gov/record/ui?ui=D005796) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method. Additionally, EPHX2 is associated with [RBC levels of 12(13)-EpOME](https://pubmed.ncbi.nlm.nih.gov/36395887), corresponding to the MeSH term [Erythrocytes](https://meshb.nlm.nih.gov/record/ui?ui=D004912), and [RBC levels of Undecanoyl carnitine](https://pubmed.ncbi.nlm.nih.gov/36395887), which maps to the MeSH term [Carnitine](https://meshb.nlm.nih.gov/record/ui?ui=D002331).


The gene is expressed in multiple tissue samples, including the liver, adrenal gland, and kidney, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the pineal gland, liver, chronic myelogenous leukemia (K-562), prostate, small intestine, kidney, and thymus.


The input data highlights several pathways involved in metabolism, specifically focusing on the metabolism of lipids and lipoproteins. Notably, it includes the arachidonic acid metabolism pathway, with a particular emphasis on the synthesis of epoxy (EET) and dihydroxyeicosatrienoic acids (DHET).



The analyzed protein sequence has a GRAVY value of -0.135 (75.28th percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is -6.66 (29.29th percentile), and the median structural flexibility is 0.998 (32.84th percentile). The protein has a notable affinity for helix (36.6%, 83.46th percentile) and sheet (35.7%, 64.07th percentile) structures, with a lower affinity for turns (26.3%, 31.9th percentile). The instability index is 44.7 (37.99th percentile), and the isoelectric point is 5.91 (28.37th percentile). The protein is 555 amino acids long (64.56th percentile) with a molecular weight of 62615.08 Da (65.21st percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |