---
layout: post
title: NUP58
date: 2025-01-17 16:55 CST
description: NUP58 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9818) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9818  | NUP58 | ENSG00000139496 | 13q12.13 |



The gene is located in the [nuclear envelope](https://amigo.geneontology.org/amigo/term/GO:0005635) and is part of the [nuclear pore](https://amigo.geneontology.org/amigo/term/GO:0005643). It is also found in the [nuclear membrane](https://amigo.geneontology.org/amigo/term/GO:0031965). The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [nuclear localization sequence binding](https://amigo.geneontology.org/amigo/term/GO:0008139), [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802), and [protein-containing complex binding](https://amigo.geneontology.org/amigo/term/GO:0044877). Additionally, it is involved in [nucleocytoplasmic transport](https://amigo.geneontology.org/amigo/term/GO:0006913), [protein transport](https://amigo.geneontology.org/amigo/term/GO:0015031), [regulation of protein import into nucleus](https://amigo.geneontology.org/amigo/term/GO:0042306), and [mRNA transport](https://amigo.geneontology.org/amigo/term/GO:0051028). The gene also serves as a [structural constituent of nuclear pore](https://amigo.geneontology.org/amigo/term/GO:0017056).


The gene length is 49,451 base pairs (65.98th percentile), the mature length is 2,259 base pairs (58.95th percentile), and the primary transcript length is 49,209 base pairs (74.84th percentile).


NUP58 has been mentioned in [5 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22NUP58%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 2007, and the middle 50% of publications occurred between 2015 and 2019.


The top 5 publications mentioning NUP58, ranked by their scientific influence, include "[Crystal structure of the metazoan Nup62•Nup58•Nup54 nucleoporin complex.](https://pubmed.ncbi.nlm.nih.gov/26292704)" (2015) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 2.28, "[Structure of Nup58/45 suggests flexible nuclear pore diameter by intermolecular sliding.](https://pubmed.ncbi.nlm.nih.gov/17379812)" (2007) with an RCR of 1.55, "[Ordered Regions of Channel Nucleoporins Nup62, Nup54, and Nup58 Form Dynamic Complexes in Solution.](https://pubmed.ncbi.nlm.nih.gov/26025361)" (2015) with an RCR of 0.36, "[NUP58 facilitates metastasis and epithelial-mesenchymal transition of lung adenocarcinoma via the GSK-3β/Snail signaling pathway.](https://pubmed.ncbi.nlm.nih.gov/30787996)" (2019) with an RCR of 0.27, and "[Nucleoporin Nup58 localizes to centrosomes and mid-bodies during mitosis.](https://pubmed.ncbi.nlm.nih.gov/31388347)" (2019) with an RCR of 0.24. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[NUP58](https://www.proteinatlas.org/ENSG00000139496-NUP58) is a gene with evidence at the protein level. Its RNA is detected in all tissues, and it is notably expressed in monocytes and neutrophils, playing a role in degranulation. The gene is also involved in non-specific transcription processes across various tissue types, including the brain. In single-cell expression analysis, NUP58 is associated with monocytes and their role in the innate immune response. However, it is unprognostic for multiple cancer types, including bladder urothelial carcinoma, breast invasive carcinoma, and cervical squamous cell carcinoma.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [GATA4](https://www.ncbi.nlm.nih.gov/gene/2626), [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), [AR](https://www.ncbi.nlm.nih.gov/gene/367), [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), and [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), each shown to be regulating in 4 experiments.



The GWAS data indicates associations with various disease conditions, including bone inflammation disease, arthritis, bone disease, connective tissue disease, musculoskeletal system disease, and osteoarthritis.



The gene is expressed in multiple tissue samples, including the pituitary and lung, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the Prefrontal Cortex, CD105+ Endothelial cells, Amygdala, Fetal brain, Pancreas, Ovary, and Leukemia lymphoblastic (MOLT-4) cells.


The input data highlights various biological pathways and processes, including the metabolism of carbohydrates, glycogen storage diseases, and gene expression. It also covers transmembrane transport mechanisms, such as SLC-mediated transport and the transport of small molecules. Additionally, the data includes pathways related to influenza infection, its life cycle, and antiviral mechanisms involving IFN-stimulated genes and ISG15. Several pathways related to HIV infection and its life cycle are also noted, including interactions with host cellular proteins and the nuclear export of viral RNA. Furthermore, the data encompasses glucose and hexose transport, the regulation of glucokinase, and various stages of the cell cycle, including mitotic prophase and nuclear envelope breakdown. Immune system processes, such as cytokine signaling and interferon signaling, are also represented, along with regulatory RNA pathways and the processing of pre-mRNA.



The analyzed protein sequence has a GRAVY value of -0.195 (69.76th percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is 8.86 (79.04th percentile), and the median structural flexibility is 1.006 (69.73rd percentile). The protein has a predicted affinity for helix, sheet, and turn structures of 29.05% (23.9th percentile), 35.39% (61.94th percentile), and 36.06% (91.91st percentile), respectively. The instability index is 36.25 (12.51st percentile), and the isoelectric point is 9.36 (88.4th percentile). The protein is 599 amino acids long (68.51st percentile) with a molecular weight of 60896.13 Da (63.6th percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |