---
layout: post
title: CS
date: 2025-01-17 16:55 CST
description: CS description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1431) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1431  | CS | ENSG00000062485 | 12q13.3 |



The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and [citrate (Si)-synthase activity](https://amigo.geneontology.org/amigo/term/GO:0004108). It is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [mitochondrion](https://amigo.geneontology.org/amigo/term/GO:0005739), and [mitochondrial matrix](https://amigo.geneontology.org/amigo/term/GO:0005759), and is active in the mitochondrial matrix. Additionally, it is involved in the [carbohydrate metabolic process](https://amigo.geneontology.org/amigo/term/GO:0005975), [tricarboxylic acid cycle](https://amigo.geneontology.org/amigo/term/GO:0006099), and [citrate metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006101). The gene is also located in the [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062).


The gene length is 262,196 base pairs (94.9% of all genes), the mature length is 3,702 base pairs (80.95% of all genes), and the primary transcript length is 169,193 base pairs (94.66% of all genes).


The gene CS (NCBI ID: 1431) has been mentioned in [1,094 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CS%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1953, and the middle 50% of publications range from 1989 to 2020.


The top 5 publications mentioning CS, ranked by their relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows:

1. [Cigarette smoke exposure and alveolar macrophages: mechanisms for lung disease.](https://pubmed.ncbi.nlm.nih.gov/33986144) (2022) (relative citation ratio: 24.14)
2. [Measures of adult shoulder function: Disabilities of the Arm, Shoulder, and Hand Questionnaire (DASH) and its short version (QuickDASH), Shoulder Pain and Disability Index (SPADI), American Shoulder and Elve Surgeons (ASES) Society standardized shoulder assessment form, Constant (Murley) Score (CS), Simple Shoulder Test (SST), Oxford Shoulder Score (OSS), Shoulder Disability Questionnaire (SDQ), and Western Ontario Shoulder Instability Index (WOSI).](https://pubmed.ncbi.nlm.nih.gov/22588743) (2011) (relative citation ratio: 19.0)
3. [Chitosan nanoparticles as a promising tool in nanomedicine with particular emphasis on oncological treatment.](https://pubmed.ncbi.nlm.nih.gov/34167552) (2021) (relative citation ratio: 16.22)
4. [Systematic analysis of domain motions in proteins from conformational change: new results on citrate synthase and T4 lysozyme.](https://pubmed.ncbi.nlm.nih.gov/9489922) (1998) (relative citation ratio: 13.56)
5. [Synthesis of Chitosan-Based Gold Nanoparticles: Antimicrobial and Wound-Healing Activities.](https://pubmed.ncbi.nlm.nih.gov/35683965) (2022) (relative citation ratio: 12.11)

Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CS](https://www.proteinatlas.org/ENSG00000062485-CS) is a transferase with evidence at the protein level. It is detected in all RNA tissues and is primarily localized in the mitochondria. The gene is expressed in various clusters, including Cluster 29 in blood, Cluster 35 in tissues, Cluster 52 in the brain, Cluster 38 in cell lines, and Cluster 68 in single cells, indicating its involvement in muscle contraction and synaptic function.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [E2F1](https://www.ncbi.nlm.nih.gov/gene/1869) and [AR](https://www.ncbi.nlm.nih.gov/gene/367), each with 6 experiments. Additionally, [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), [YY1](https://www.ncbi.nlm.nih.gov/gene/7528), and [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) were each found to be regulating in 5 experiments.



The input data indicates associations with a range of disease conditions, including various types of cancer such as male reproductive organ cancer, reproductive organ cancer, prostate cancer, and organ system cancer. Additionally, the data highlights metabolic disorders, specifically diabetes mellitus and type 2 diabetes mellitus, as well as diseases related to cellular proliferation and carbohydrate metabolism. Furthermore, nervous system diseases, particularly those affecting the eye and adnexa, including corneal disease, are also noted.



The gene is ubiquitously expressed in multiple tissue samples, including the heart, muscle, and adrenal gland, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the heart, 721_B_lymphoblasts, Leukemia_promyelocytic-HL-60, Lymphoma_burkitts(Daudi), cerebellum, adipocyte, and skeletal muscle.


The input data highlights several key pathways involved in cellular metabolism and energy production. These include the metabolism of proteins, general metabolism, the citric acid cycle (TCA cycle), pyruvate metabolism and the TCA cycle, the TCA cycle and respiratory electron transport, and mitochondrial protein import. These pathways are essential for the breakdown and synthesis of proteins, the generation of energy through the TCA cycle, and the import of proteins into mitochondria for proper cellular function.



The analyzed protein sequence has a GRAVY value of -0.165 (72.7th percentile), indicating a hydrophilic nature. It exhibits a charge of 3.814 at pH 7.0 (63.99th percentile) and a median structural flexibility of 0.996 (23.73rd percentile). The protein's secondary structure is predicted to be 35.19% helix (75.92nd percentile), 37.12% sheet (73.84th percentile), and 28.11% turn (48.34th percentile). The instability index is 22.40 (0.93rd percentile), with an isoelectric point of 8.45 (65.62nd percentile). The protein has a length of 466 amino acids (53.93rd percentile) and a molecular weight of 51711.83 Da (53.58th percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |