---
layout: post
title: ATP6V1A
date: 2025-01-17 16:55 CST
description: ATP6V1A description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/523) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 523  | ATP6V1A | ENSG00000114573 | 3q13.31 |



The gene is located in the [Golgi membrane](https://amigo.geneontology.org/amigo/term/GO:0000139), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [lysosomal membrane](https://amigo.geneontology.org/amigo/term/GO:0005765), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [microvillus](https://amigo.geneontology.org/amigo/term/GO:0005902), [endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0010008), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [apical plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0016324), [secretory granule](https://amigo.geneontology.org/amigo/term/GO:0030141), [clathrin-coated vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030665), [intracellular membrane-bounded organelle](https://amigo.geneontology.org/amigo/term/GO:0043231), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). It is part of the [vacuolar proton-transporting V-type ATPase, V1 domain](https://amigo.geneontology.org/amigo/term/GO:0000221), [proton-transporting two-sector ATPase complex](https://amigo.geneontology.org/amigo/term/GO:0016469), [proton-transporting V-type ATPase complex](https://amigo.geneontology.org/amigo/term/GO:0033176), and [transmembrane transporter complex](https://amigo.geneontology.org/amigo/term/GO:1902495). The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524), [ATP hydrolysis activity](https://amigo.geneontology.org/amigo/term/GO:0016887), [proton-transporting ATP synthase activity, rotational mechanism](https://amigo.geneontology.org/amigo/term/GO:0043231), and [proton-transporting ATPase activity, rotational mechanism](https://amigo.geneontology.org/amigo/term/GO:0046961). It is involved in [intracellular iron ion homeostasis](https://amigo.geneontology.org/amigo/term/GO:0006879), [vacuolar acidification](https://amigo.geneontology.org/amigo/term/GO:0007035), [lysosomal lumen acidification](https://amigo.geneontology.org/amigo/term/GO:0007042), [proton motive force-driven ATP synthesis](https://amigo.geneontology.org/amigo/term/GO:0015986), [regulation of macroautophagy](https://amigo.geneontology.org/amigo/term/GO:0016241), [cellular response to increased oxygen levels](https://amigo.geneontology.org/amigo/term/GO:0036295), [endosomal lumen acidification](https://amigo.geneontology.org/amigo/term/GO:0048388), [intracellular pH reduction](https://amigo.geneontology.org/amigo/term/GO:0051452), [Golgi lumen acidification](https://amigo.geneontology.org/amigo/term/GO:0061795), [synaptic vesicle lumen acidification](https://amigo.geneontology.org/amigo/term/GO:0097401), and [proton transmembrane transport](https://amigo.geneontology.org/amigo/term/GO:1902600).


The gene length is 51,407 base pairs (67.2% of all genes), the mature length is 642 base pairs, and the primary transcript length is 44,583 base pairs.


The gene ATP6V1A (NCBI ID: 523) has been mentioned in [12 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ATP6V1A%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink). The earliest publication mentioning this gene was in 1995, and the middle 50% of publications occurred between 2016 and 2021.


The top 5 publications mentioning ATP6V1A, ranked by their scientific influence, include studies that highlight the gene's role in various diseases and biological processes. The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Notable publications include "[Mutations in ATP6V1E1 or ATP6V1A Cause Autosomal-Recessive Cutis Laxa.](https://pubmed.ncbi.nlm.nih.gov/28065471)" (2017) (relative citation ratio: 2.91), "[De novo mutations of the ATP6V1A gene cause developmental encephalopathy with epilepsy.](https://pubmed.ncbi.nlm.nih.gov/29668857)" (2018) (relative citation ratio: 2.48), "[Phenotypic and genetic spectrum of ATP6V1A encephalopathy: a disorder of lysosomal homeostasis.](https://pubmed.ncbi.nlm.nih.gov/35675510)" (2022) (relative citation ratio: 2.18), "[Downregulation of ATP6V1A Involved in Alzheimer's Disease via Synaptic Vesicle Cycle, Phagosome, and Oxidative Phosphorylation.](https://pubmed.ncbi.nlm.nih.gov/33981384)" (2021) (relative citation ratio: 1.83), and "[AMP-activated protein kinase regulates the vacuolar H+-ATPase via direct phosphorylation of the A subunit (ATP6V1A) in the kidney.](https://pubmed.ncbi.nlm.nih.gov/23863464)" (2013) (relative citation ratio: 1.37). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ATP6V1A](https://www.proteinatlas.org/ENSG00000114573-ATP6V1A) is a gene encoding a protein with translocase molecular function. It is implicated in epilepsy as a disease variant, with evidence at the protein level. The gene is detected in all RNA tissue distributions and is localized in the nucleoplasm, vesicles, and cytosol. It is expressed in monocytes and neutrophils related to degranulation (Cluster 38) and is involved in neuronal signaling in the brain (Cluster 61). Additionally, it is expressed in neurons with mixed functions in the brain (Cluster 24) and is involved in protein processing in non-specific cell lines (Cluster 63). The gene is also detected in late spermatids with an unknown function (Cluster 11).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [TBP](https://www.ncbi.nlm.nih.gov/gene/6908), [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), [RELA](https://www.ncbi.nlm.nih.gov/gene/5970), and [JUN](https://www.ncbi.nlm.nih.gov/gene/3725), each with 5 experiments. Additionally, [STAT1](https://www.ncbi.nlm.nih.gov/gene/6772) was found to be regulating in 4 experiments.



The GWAS data indicates associations with several disease conditions, including artery disease, vascular disease, cardiovascular system disease, hypertension, and broader categories such as disease and disease of anatomical entity.



The gene is expressed in multiple tissue samples, including the brain, adrenal gland, and kidney, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the pineal gland (both day and night), CD33+ myeloid cells, amygdala, subthalamic nucleus, occipital lobe, and kidney.


The input data highlights several biological pathways, including insulin receptor recycling, transferrin endocytosis and recycling, iron uptake and transport, latent infection of Homo sapiens with Mycobacterium tuberculosis, phagosomal maturation (early endosomal stage), transmembrane transport of small molecules, signaling by insulin receptor, disease, and signal transduction.


The mouse ortholog gene 11964 and its human ortholog gene 523 are associated with the phenotype of preweaning lethality with complete penetrance, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.186 (70.63rd percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is -13.68 (17.08th percentile), and the median structural flexibility is 1.000 (41.89th percentile). The protein has a predicted secondary structure with 31.12% helix (42.26th percentile), 36.47% sheet (69.47th percentile), and 29.82% turn (62.49th percentile) affinities. The instability index is 34.38 (9.06th percentile), and the isoelectric point is 5.35 (14.68th percentile). The protein is 617 amino acids long (69.76th percentile) with a molecular weight of 68303.30 Da (69.45th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |