---
layout: post
title: ITK
date: 2025-01-17 16:55 CST
description: ITK description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3702) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3702  | ITK | ENSG00000113263 | 5q33.3 |



The gene is involved in various biological processes, including [positive regulation of cytokine production](https://amigo.geneontology.org/amigo/term/GO:0001819), [NK T cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0001865), and [adaptive immune response](https://amigo.geneontology.org/amigo/term/GO:0002250). It enables [non-membrane spanning protein tyrosine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004715), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524). The gene is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and [cell-cell junction](https://amigo.geneontology.org/amigo/term/GO:0005911), and is active in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). Additionally, it is involved in [protein phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0006468), [cellular defense response](https://amigo.geneontology.org/amigo/term/GO:0006968), [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), [activation of phospholipase C activity](https://amigo.geneontology.org/amigo/term/GO:0007202), [phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0016310), [intracellular signal transduction](https://amigo.geneontology.org/amigo/term/GO:0035556), [T cell activation](https://amigo.geneontology.org/amigo/term/GO:0042110), [gamma-delta T cell activation](https://amigo.geneontology.org/amigo/term/GO:0046629), [T cell receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0050852), and [B cell receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0050853). The gene also enables [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872).


The gene length is 117,107.0 base pairs (84.81% of all genes), the mature length is 3,268.5 base pairs, and the primary transcript length is 116,715.5 base pairs.


The gene ITK (NCBI ID: 3702) has been mentioned in [395 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ITK%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1976, and the middle 50% of publications occurred between 2010 and 2020.


The top 5 publications mentioning ITK, ranked by their scientific influence, include "[Distinct EMT programs control normal mammary stem cells and tumour-initiating cells.](https://pubmed.ncbi.nlm.nih.gov/26331542)" (2015) (relative citation ratio: 16.07), "[Repair Schwann cell update: Adaptive reprogramming, EMT, and stemness in regenerating nerves.](https://pubmed.ncbi.nlm.nih.gov/30632639)" (2019) (relative citation ratio: 12.22), "[A switch in the expression of embryonic EMT-inducers drives the development of malignant melanoma.](https://pubmed.ncbi.nlm.nih.gov/24075834)" (2013) (relative citation ratio: 10.33), "[The effect of 2D and 3D cell cultures on treatment response, EMT profile and stem cell features in head and neck cancer.](https://pubmed.ncbi.nlm.nih.gov/30651721)" (2019) (relative citation ratio: 8.55), and "[Autophagy induction impairs migration and invasion by reversing EMT in glioblastoma cells.](https://pubmed.ncbi.nlm.nih.gov/26022108)" (2015) (relative citation ratio: 7.88). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ITK](https://www.proteinatlas.org/ENSG00000113263-ITK) is a kinase and transferase with tyrosine-protein kinase activity, implicated in cancer-related genes and disease variants. Evidence supports its function at the protein level. ITK is detected in some RNA tissue distributions and is primarily localized in the cytosol. It is highly expressed in T-cells, particularly those involved in the T-cell receptor pathway, and is also found in the thymus, where it plays a role in the adaptive immune response. In cell lines, ITK is clustered with leukemia-related T-cell receptor genes. Single-cell expression analysis reveals its involvement in NK-cells and T-cells, impacting the immune system and transcription. While ITK is unprognostic in several cancer types, including breast invasive carcinoma, kidney renal clear cell carcinoma, liver hepatocellular carcinoma, lung adenocarcinoma, lung squamous cell carcinoma, pancreatic adenocarcinoma, and testicular germ cell tumor, its role in these contexts warrants further investigation.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ZNF143](https://www.ncbi.nlm.nih.gov/gene/7702) with 6 experiments, [TP53](https://www.ncbi.nlm.nih.gov/gene/7157) and [RELA](https://www.ncbi.nlm.nih.gov/gene/5970) each with 5 experiments, and [KLF5](https://www.ncbi.nlm.nih.gov/gene/688) and [ELF1](https://www.ncbi.nlm.nih.gov/gene/1997) each with 4 experiments.



The GWAS data indicates associations with a range of disease conditions, including immune system disease, various mental health disorders such as substance-related disorder and personality disorder, and musculoskeletal system diseases like arthritis and rheumatoid arthritis. Additionally, the data highlights associations with bone inflammation disease, connective tissue disease, and integumentary system diseases, including alopecia and alopecia areata.



The gene is expressed in various tissues and cell types, including blood, spleen, and small intestine, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in CD56+ NK cells, CD4+ T cells, CD8+ T cells, whole blood, thymus, leukemia lymphoblastic cells (MOLT-4), and lymph nodes.


The proteins are involved in various immune system pathways, including the adaptive and innate immune systems. Specifically, they participate in fc epsilon receptor (fcεri) signaling, fcεri-mediated ca+2 mobilization, tcr signaling, and the generation of second messenger molecules.


The mouse ortholog gene 16428 and its human ortholog gene 3702 are associated with the phenotype of increased startle reflex, observed in both sexes.


The protein sequence analyzed has a GRAVY value of -0.467 (38.59th percentile), indicating a hydrophilic nature. It exhibits a charge of 1.841 at pH 7.0 (56.96th percentile) and a median structural flexibility of 0.999 (38.94th percentile). The secondary structure affinities are 31.29% for helix (43.85th percentile), 36.77% for sheet (71.57th percentile), and 26.29% for turn (31.79th percentile). The instability index is 44.73 (38.23rd percentile), with an isoelectric point of 7.51 (54.08th percentile). The protein has a length of 620 amino acids (69.99th percentile) and a molecular weight of 71830.12 Da (71.75th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |