---
layout: post
title: ZFP36L1
date: 2025-01-17 16:55 CST
description: ZFP36L1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/677) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 677  | ZFP36L1 | ENSG00000185650 | 14q24.1 |



The gene is involved in various biological processes, including the [MAPK cascade](https://amigo.geneontology.org/amigo/term/GO:0000165), [nuclear-transcribed mRNA catabolic process, deadenylation-dependent decay](https://amigo.geneontology.org/amigo/term/GO:0000288), [vasculogenesis](https://amigo.geneontology.org/amigo/term/GO:0001570), and [proepicardium development](https://amigo.geneontology.org/amigo/term/GO:0003342). It enables functions such as [DNA binding](https://amigo.geneontology.org/amigo/term/GO:0003677), [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [mRNA binding](https://amigo.geneontology.org/amigo/term/GO:0003729), and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and [P-body](https://amigo.geneontology.org/amigo/term/GO:0000932). Additionally, it is involved in processes like [mRNA processing](https://amigo.geneontology.org/amigo/term/GO:0006397), [regulation of translation](https://amigo.geneontology.org/amigo/term/GO:0006417), [apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0006915), and [heart development](https://amigo.geneontology.org/amigo/term/GO:0007507). The gene also plays a role in the [response to wounding](https://amigo.geneontology.org/amigo/term/GO:0009611), [regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0010468), and various other cellular responses and regulatory processes.


The gene length is 12,233 base pairs (30.03% of all genes), the mature length is 1,753 base pairs, and the primary transcript length is 12,194.5 base pairs.


The gene ZFP36L1 (NCBI ID: 677) has been mentioned in [47 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ZFP36L1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1994 and the middle 50% of publications occurring between 2009 and 2019. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning ZFP36L1, ranked by their scientific influence, include studies that explore the gene's role in various biological processes. The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Notable publications include "[RNA-binding proteins ZFP36L1 and ZFP36L2 promote cell quiescence.](https://pubmed.ncbi.nlm.nih.gov/27102483)" (2016) (relative citation ratio: 3.06), "[Deletion of the RNA-binding proteins ZFP36L1 and ZFP36L2 leads to perturbed thymic development and T lymphoblastic leukemia.](https://pubmed.ncbi.nlm.nih.gov/20622884)" (2010) (relative citation ratio: 2.84), "[RNA-binding protein ZFP36L1 regulates osteoarthritis by modulating members of the heat shock protein 70 family.](https://pubmed.ncbi.nlm.nih.gov/30622281)" (2019) (relative citation ratio: 2.53), "[Chorioallantoic fusion defects and embryonic lethality resulting from disruption of Zfp36L1, a gene encoding a CCCH tandem zinc finger protein of the Tristetraprolin family.](https://pubmed.ncbi.nlm.nih.gov/15226444)" (2004) (relative citation ratio: 2.36), and "[ZFP36L1 and ZFP36L2 inhibit cell proliferation in a cyclin D-dependent and p53-independent manner.](https://pubmed.ncbi.nlm.nih.gov/29426877)" (2018) (relative citation ratio: 1.88). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ZFP36L1](https://www.proteinatlas.org/ENSG00000185650-ZFP36L1) is a developmental protein with DNA-binding and RNA-binding capabilities, classified as a ribonucleoprotein. It is involved in cancer-related genes and has evidence at the protein level. The gene is detected in all RNA tissue distributions and is associated with various expression clusters, including Cluster 20 for non-specific DNA binding in blood, Cluster 62 for connective tissue and ECM organization, Cluster 39 for astrocytes with mixed function in the brain, Cluster 46 for lymphoma and humoral immune response in cell lines, and Cluster 12 for squamous epithelial cells involved in keratinization in single-cell expression.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), [POU5F1](https://www.ncbi.nlm.nih.gov/gene/5460), [TP53](https://www.ncbi.nlm.nih.gov/gene/7157), [TEAD4](https://www.ncbi.nlm.nih.gov/gene/7004), and [STAT3](https://www.ncbi.nlm.nih.gov/gene/6774), each shown to be regulating in 5 experiments.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [2623](https://www.ncbi.nlm.nih.gov/gene/2623) and [8861](https://www.ncbi.nlm.nih.gov/gene/8861). These TFs were shown to be regulating genes in 2 and 1 Perturb-seq experiments, respectively.


The GWAS data indicates associations with various disease conditions, including immune system disease, disease of metabolism, acquired metabolic disease, carbohydrate metabolism disease, glucose metabolism disease, diabetes mellitus, type 1 diabetes mellitus, hypersensitivity reaction disease, hypersensitivity reaction type II disease, autoimmune disease of the gastrointestinal tract, and celiac disease.



The gene is expressed in various tissues, including the ovary and cervix uteri, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, data from the GTEx Tissue Gene Expression Profiles show expression in whole blood, colorectal adenocarcinoma, ovary, fetal lung, and uterus.


The input data highlights several pathways involving gene expression and mRNA regulation. Specifically, it includes the pathway for the regulation of mRNA stability by proteins that bind AU-rich elements. Additionally, it mentions the role of Butyrate Response Factor 1 (BRF1) in destabilizing mRNA.



The analyzed protein sequence has a GRAVY value of -0.481 (37th percentile), indicating a hydrophilic nature. It exhibits a charge of 2.43 at pH 7.0 (59.06th percentile) and a median structural flexibility of 1.007 (71.22nd percentile). The protein's secondary structure is predicted to be 26.63% helix (10.11th percentile), 28.11% sheet (18.93rd percentile), and 40.24% turn (97.5th percentile). The instability index is 68.10 (93.26th percentile), with an isoelectric point of 8.12 (60.13th percentile). The protein has a length of 338 amino acids (36.06th percentile) and a molecular weight of 36313.14 Da (34.04th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |