---
layout: post
title: RPS3A
date: 2025-01-17 16:55 CST
description: RPS3A description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6189) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6189  | RPS3A | ENSG00000145425 | 4q31.3 |



The gene is involved in [cytoplasmic translation](https://amigo.geneontology.org/amigo/term/GO:0002181) and [translation](https://amigo.geneontology.org/amigo/term/GO:0006412), and plays a role in [translational initiation](https://amigo.geneontology.org/amigo/term/GO:0006413). It enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and acts as a [structural constituent of ribosome](https://amigo.geneontology.org/amigo/term/GO:0003735). Additionally, it is involved in [cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0030154), [ribosomal small subunit biogenesis](https://amigo.geneontology.org/amigo/term/GO:0042274), and [negative regulation of apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0043066). The gene is located in various cellular components, including the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [nucleolus](https://amigo.geneontology.org/amigo/term/GO:0005730), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [endoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0005783), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925), [cytosolic ribosome](https://amigo.geneontology.org/amigo/term/GO:0022626), [synapse](https://amigo.geneontology.org/amigo/term/GO:0045202), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). It is also part of the [cytosolic small ribosomal subunit](https://amigo.geneontology.org/amigo/term/GO:0022627), [small-subunit processome](https://amigo.geneontology.org/amigo/term/GO:0032040), and [ribonucleoprotein complex](https://amigo.geneontology.org/amigo/term/GO:1990904). Furthermore, it enables [mRNA 5'-UTR binding](https://amigo.geneontology.org/amigo/term/GO:0048027).


The gene length is 43,798 base pairs (62.8% of all genes), the mature length is 1,622 base pairs, and the primary transcript length is 8,163.5 base pairs.


The gene RPS3A, with NCBI ID 6189, has been mentioned in [18 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RPS3A%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727). The earliest year of publication is 1992, and the middle 50% of publications occurred between 1998 and 2007. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning RPS3A, ranked by their relative citation ratio (RCR), include studies that highlight the gene's role in various biological processes. The RCR values measure the scientific influence of each paper by field- and time-adjusting the citations it has received, and benchmarking to the median for NIH publications, the value of which is set at 1.0. The most influential publication is "[Fte-1, a v-fos transformation effector gene, encodes the mammalian homologue of a yeast gene involved in protein import into mitochondria.](https://pubmed.ncbi.nlm.nih.gov/1549582)" (1992) (relative citation ratio: 1.17). Other notable studies include "[High RPS3A expression correlates with low tumor immune cell infiltration and unfavorable prognosis in hepatocellular carcinoma patients.](https://pubmed.ncbi.nlm.nih.gov/33042616)" (2020) (relative citation ratio: 1.16) and "[RPS3a over-expressed in HBV-associated hepatocellular carcinoma enhances the HBx-induced NF-ÎºB signaling via its novel chaperoning function.](https://pubmed.ncbi.nlm.nih.gov/21857917)" (2011) (relative citation ratio: 1.05). Additionally, "[Dual role of the Leishmania major ribosomal protein S3a homologue in regulation of T- and B-cell activation.](https://pubmed.ncbi.nlm.nih.gov/11598026)" (2001) (relative citation ratio: 0.92) and "[Human ribosomal protein S3a: cloning of the cDNA and primary structure of the protein.](https://pubmed.ncbi.nlm.nih.gov/1398113)" (1992) (relative citation ratio: 0.59) are also significant. The citation count is sourced from [iCite](https://icite.od.nih.gov).


[RPS3A](https://www.proteinatlas.org/ENSG00000145425-RPS3A) is a ribonucleoprotein and ribosomal protein with evidence at the protein level. It is detected in all tissues and is primarily located in the nucleoli, endoplasmic reticulum, and cytosol. The gene is part of various non-specific expression clusters related to transcription and ribosomal functions across different tissue types, including blood, brain, and cell lines.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) with 6 experiments, [E2F1](https://www.ncbi.nlm.nih.gov/gene/1869) with 5 experiments, [INTS11](https://www.ncbi.nlm.nih.gov/gene/54973) with 5 experiments, [SIN3A](https://www.ncbi.nlm.nih.gov/gene/25942) with 5 experiments, and [PHF8](https://www.ncbi.nlm.nih.gov/gene/23133) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.


Based on the data from GTRD's database of transcription factor (TF) binding sites and filtered for Perturb-seq data, several TFs were identified as regulators in multiple experiments. Notably, [CTCF](https://www.ncbi.nlm.nih.gov/gene/11091) was found to regulate gene expression in three experiments, while [GATA2](https://www.ncbi.nlm.nih.gov/gene/2623) was identified in two experiments. Additionally, [FOXA1](https://www.ncbi.nlm.nih.gov/gene/2475), [TCF7L2](https://www.ncbi.nlm.nih.gov/gene/7469), and [ZNF281](https://www.ncbi.nlm.nih.gov/gene/84148) were each found to regulate gene expression in one experiment.




The gene is expressed in various tissues and cell types, including the ovary, cervix uteri, CD8+ T cells, leukemia promyelocytic cells (HL-60), Burkitt's lymphoma cells (Daudi), uterus corpus, and lymphoblastic leukemia cells (MOLT-4).


The input data highlights various pathways involved in protein metabolism and gene expression, including '3\'-UTR-mediated translational regulation', 'L13a-mediated translational silencing of Ceruloplasmin expression', and 'GTP hydrolysis and joining of the 60S ribosomal subunit'. Key processes such as 'Cap-dependent Translation Initiation', 'Eukaryotic Translation Initiation', 'Eukaryotic Translation Elongation', and 'Eukaryotic Translation Termination' are also noted. Additionally, the data covers 'Viral mRNA Translation', 'Nonsense Mediated Decay (NMD)' both independent of and enhanced by the Exon Junction Complex (EJC), and 'SRP-dependent cotranslational protein targeting to membrane'. Influenza-related pathways, including 'Influenza Infection', 'Influenza Life Cycle', and 'Influenza Viral RNA Transcription and Replication', are also included. The data further details specific steps in translation initiation, such as 'Activation of the mRNA upon binding of the cap-binding complex and eIFs, and subsequent binding to 43S', 'Translation initiation complex formation', 'Ribosomal scanning and start codon recognition', and 'Formation of the ternary complex, and subsequently, the 43S complex'. Lastly, the data mentions 'Disease'.



The analyzed protein sequence has a GRAVY value of -0.618 (23.69th percentile), indicating a hydrophilic nature. It exhibits a charge of 19.91 at pH 7.0 (93.42nd percentile) and a median structural flexibility of 1.012 (85.89th percentile). The protein's secondary structure is predicted to be 36.0% helix (80.62nd percentile), 33.0% sheet (45.18th percentile), and 24.6% turn (18.86th percentile). The instability index is 27.84 (2.57th percentile), suggesting a stable protein. The isoelectric point is 9.75 (94.4th percentile), with a molecular weight of 29944.59 Da (24.26th percentile) and a length of 264 amino acids (23.65th percentile). For detailed sequence analysis methods, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |