---
layout: post
title: TGOLN2
date: 2025-01-17 16:55 CST
description: TGOLN2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10618) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10618  | TGOLN2 | ENSG00000152291 | 2p11.2 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [endoplasmic reticulum lumen](https://amigo.geneontology.org/amigo/term/GO:0005788), [Golgi apparatus](https://amigo.geneontology.org/amigo/term/GO:0005794), [trans-Golgi network](https://amigo.geneontology.org/amigo/term/GO:0005802), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [transport vesicle](https://amigo.geneontology.org/amigo/term/GO:0030133), and [clathrin-coated endocytic vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030669). It is active in the [endosome](https://amigo.geneontology.org/amigo/term/GO:0005768), [trans-Golgi network](https://amigo.geneontology.org/amigo/term/GO:0005802), and [trans-Golgi network transport vesicle](https://amigo.geneontology.org/amigo/term/GO:0030140).


The gene length is 28,628 base pairs (51.4% of all genes), the mature length is 1,383 base pairs, and the primary transcript length is 28,605 base pairs.


The gene TGOLN2 (NCBI ID: 10618) has been mentioned in [25 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22TGOLN2%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink). The earliest year of publication is 1990, and the middle 50% of publications occurred between 1993 and 1999.


The top 5 publications mentioning TGOLN2, ranked by their scientific influence, include "[Identification, sequencing and expression of an integral membrane protein of the trans-Golgi network (TGN38).](https://pubmed.ncbi.nlm.nih.gov/2204342)" (1990) (relative citation ratio: 6.13), "[Localization of TGN38 to the trans-Golgi network: involvement of a cytoplasmic tyrosine-containing sequence.](https://pubmed.ncbi.nlm.nih.gov/8436587)" (1993) (relative citation ratio: 4.14), "[Mechanism of acidification of the trans-Golgi network (TGN). In situ measurements of pH using retrieval of TGN38 and furin from the cell surface.](https://pubmed.ncbi.nlm.nih.gov/9442042)" (1998) (relative citation ratio: 3.25), "[GLUT4 recycles via a trans-Golgi network (TGN) subdomain enriched in Syntaxins 6 and 16 but not TGN38: involvement of an acidic targeting motif.](https://pubmed.ncbi.nlm.nih.gov/12631717)" (2003) (relative citation ratio: 2.92), and "[The membrane on the surface of hepatitis E virus particles is derived from the intracellular membrane and contains trans-Golgi network protein 2.](https://pubmed.ncbi.nlm.nih.gov/24221250)" (2014) (relative citation ratio: 2.31). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[TGOLN2](https://www.proteinatlas.org/ENSG00000152291-TGOLN2) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in the nucleoplasm and Golgi apparatus. The gene is expressed in various clusters, including Cluster 40 in monocytes related to innate immune response, Cluster 55 in kidney and intestine associated with metabolism, Cluster 28 in oligodendrocytes with mixed function, Cluster 62 in non-specific plasma proteins, and Cluster 58 in macrophages involved in innate immune response.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [HDAC2](https://www.ncbi.nlm.nih.gov/gene/3066), [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169), [INTS11](https://www.ncbi.nlm.nih.gov/gene/54973), [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), and [CHD1](https://www.ncbi.nlm.nih.gov/gene/1105), each shown to be regulating in 5 experiments.



The input data indicates associations with various disease conditions, including metabolic disorders such as diabetes mellitus and type 2 diabetes mellitus, as well as neurological conditions like amyotrophic lateral sclerosis and other neurodegenerative diseases. Additionally, the data encompasses broader categories such as diseases of metabolism, acquired metabolic diseases, carbohydrate metabolism diseases, and glucose metabolism diseases. Furthermore, it includes general references to diseases, diseases of anatomical entities, nervous system diseases, and central nervous system diseases.



The gene is expressed in various tissues and cell types, including blood vessels and the small intestine, as indicated by BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles show expression in CD33+ myeloid cells, CD8+ T cells, the prostate, and the placenta.


The proteins are involved in various pathways related to vesicle dynamics and trafficking, including clathrin-derived vesicle budding, trans-Golgi network vesicle budding, Golgi-associated vesicle biogenesis, and membrane trafficking.



The analyzed protein sequence has a GRAVY value of -1.2164759725400447 (1.73 percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -10.704767388138698 (21.45 percentile), and the median structural flexibility is 1.0394107142857143 (99.86 percentile). The protein has a high affinity for helix structures (34.096109839816935, 68.6 percentile) and turns (41.647597254004576, 98.23 percentile), with a lower affinity for sheet structures (19.221967963386727, 1.7 percentile). The instability index is 54.87826086956528 (71.81 percentile), and the isoelectric point is 5.456509208679199 (17.17 percentile). The protein sequence is 437 amino acids long (50.34 percentile) with a molecular weight of 45879.00270000004 Da (46.8 percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |