---
layout: post
title: AKAP7
date: 2025-01-17 16:55 CST
description: AKAP7 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9465) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9465  | AKAP7 | ENSG00000118507 | 6q23.2 |



The gene enables [nucleotide binding](https://amigo.geneontology.org/amigo/term/GO:0000166) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), including [protein kinase binding](https://amigo.geneontology.org/amigo/term/GO:0019901) and [protein kinase A regulatory subunit binding](https://amigo.geneontology.org/amigo/term/GO:0034237). It is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and is part of a [protein-containing complex](https://amigo.geneontology.org/amigo/term/GO:0032991). The gene is involved in various biological processes, including the [regulation of protein kinase A signaling](https://amigo.geneontology.org/amigo/term/GO:0010738) and the [modulation of chemical synaptic transmission](https://amigo.geneontology.org/amigo/term/GO:0050804). Additionally, it is located in the [hippocampal mossy fiber to CA3 synapse](https://amigo.geneontology.org/amigo/term/GO:0098686).


The gene length is 23,531 base pairs (46.15th percentile), the mature length is 2,255.5 base pairs (58.89th percentile), and the primary transcript length is 21,506 base pairs (53.61st percentile).


AKAP7 has been mentioned in [10 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22AKAP7%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 2002, and the middle 50% of publications occurred between 2006 and 2016.


The top 5 publications mentioning AKAP7, ranked by their scientific influence, include studies that explore the gene's role in various biological processes. The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Notable publications include "[A novel leucine zipper targets AKAP15 and cyclic AMP-dependent protein kinase to the C terminus of the skeletal muscle Ca2+ channel and modulates its function.](https://pubmed.ncbi.nlm.nih.gov/11733497)" (2002) (relative citation ratio: 1.72), "[Molecular mechanism of convergent regulation of brain Na(+) channels by protein kinase C and protein kinase A anchored to AKAP-15.](https://pubmed.ncbi.nlm.nih.gov/12359152)" (2002) (relative citation ratio: 1.48), "[Peripheral blood AKAP7 expression as an early marker for lymphocyte-mediated post-stroke blood brain barrier disruption.](https://pubmed.ncbi.nlm.nih.gov/28446746)" (2017) (relative citation ratio: 1.1), "[Murine AKAP7 has a 2',5'-phosphodiesterase domain that can complement an inactive murine coronavirus ns2 gene.](https://pubmed.ncbi.nlm.nih.gov/24987090)" (2014) (relative citation ratio: 1.07), and "[AKAP18 contains a phosphoesterase domain that binds AMP.](https://pubmed.ncbi.nlm.nih.gov/18082768)" (2008) (relative citation ratio: 1.06). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[AKAP7](https://www.proteinatlas.org/ENSG00000118507-AKAP7) is a gene with evidence at the protein level. Its RNA is detected in all tissues, and the protein is localized to vesicles. AKAP7 is expressed in T-cells of unknown function within the blood, and it is associated with proteolysis in the stomach. In the brain, it is linked to nucleosome function in neurons. Additionally, AKAP7 is expressed in bone cancer cell lines, where it is involved in neuronal signaling, and in oligodendrocyte precursors, although its function in these cells is unknown.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [MYC](https://www.ncbi.nlm.nih.gov/gene/4609) and [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613), each with 6 experiments. Additionally, [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025), [SP1](https://www.ncbi.nlm.nih.gov/gene/6667), and [EP300](https://www.ncbi.nlm.nih.gov/gene/2033) were each found to be regulating in 5 experiments.



The input data from GWAS indicates associations with a wide range of disease conditions, including various types of cancer such as lung cancer, leukemia, and multiple myeloma. Additionally, metabolic disorders like type 2 diabetes mellitus and cardiovascular diseases such as hypertension and heart disease are also implicated. Neurological conditions, including neurodegenerative diseases like amyotrophic lateral sclerosis and cognitive disorders such as bipolar disorder, are also represented in the data.



The gene is expressed in various tissues, including the pancreas and adrenal gland, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in CD105+ endothelial cells, CD71+ early erythroid cells, fetal brain, pancreatic islet, adrenal cortex, adrenal gland, superior cervical ganglion, and ciliary ganglion.




The analyzed protein sequence has a GRAVY value of -0.857 (9.16th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -4.71 (34.48th percentile). The median structural flexibility is 1.019 (95.88th percentile), suggesting high flexibility. The protein has a propensity for helical structures (35.41%, 77.32nd percentile), with lower affinities for sheet (26.86%, 13.97th percentile) and turn (33.89%, 85.56th percentile) conformations. The instability index is 39.58 (21st percentile), and the isoelectric point is 5.37 (15.18th percentile). The protein is 226 amino acids long (18.41st percentile) with a molecular weight of 25490.92 Da (18.74th percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |