---
layout: post
title: APOC1
date: 2025-01-17 16:55 CST
description: APOC1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/341) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 341  | APOC1 | ENSG00000130208 | 19q13.32 |



The gene enables [phospholipase inhibitor activity](https://amigo.geneontology.org/amigo/term/GO:0004859) and [fatty acid binding](https://amigo.geneontology.org/amigo/term/GO:0005504), and is involved in [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576) and the [endoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0005783). The gene plays a role in various metabolic processes, including [lipid metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006629), [triglyceride metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006641), and [cholesterol metabolic process](https://amigo.geneontology.org/amigo/term/GO:0008203). It is also involved in the negative regulation of several processes, such as [phosphatidylcholine catabolic process](https://amigo.geneontology.org/amigo/term/GO:0010900), [very-low-density lipoprotein particle clearance](https://amigo.geneontology.org/amigo/term/GO:0010916), and [cholesterol transport](https://amigo.geneontology.org/amigo/term/GO:0032375). Additionally, the gene is part of the [very-low-density lipoprotein particle](https://amigo.geneontology.org/amigo/term/GO:0034361) and the [high-density lipoprotein particle](https://amigo.geneontology.org/amigo/term/GO:0034364), and it participates in processes like [cholesterol efflux](https://amigo.geneontology.org/amigo/term/GO:0033344), [phospholipid efflux](https://amigo.geneontology.org/amigo/term/GO:0033700), and [lipoprotein metabolic process](https://amigo.geneontology.org/amigo/term/GO:0042157). The gene also enables [phosphatidylcholine binding](https://amigo.geneontology.org/amigo/term/GO:0031210), [lipase inhibitor activity](https://amigo.geneontology.org/amigo/term/GO:0055102), and [phosphatidylcholine-sterol O-acyltransferase activator activity](https://amigo.geneontology.org/amigo/term/GO:0060228).


The gene length is 40,595 base pairs (60.66% of all genes), the mature length is 3,282 base pairs, and the primary transcript length is 39,063 base pairs.


The gene APOC1 (NCBI ID: 341) has been mentioned in [146 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22APOC1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning APOC1 is 1976, and the middle 50% of publications occurred between 1999 and 2013.


The top 5 publications mentioning APOC1, ranked by their scientific influence, include "[Role of ApoCs in lipoprotein metabolism: functional differences between ApoC1, ApoC2, and ApoC3.](https://pubmed.ncbi.nlm.nih.gov/10073946)" (1999) (relative citation ratio: 9.19), "[Apolipoprotein C-I modulates the interaction of apolipoprotein E with beta-migrating very low density lipoproteins (beta-VLDL) and inhibits binding of beta-VLDL to low density lipoprotein receptor-related protein.](https://pubmed.ncbi.nlm.nih.gov/2266137)" (1990) (relative citation ratio: 5.52), "[Apolipoprotein C1: Its Pleiotropic Effects in Lipid Metabolism and Beyond.](https://pubmed.ncbi.nlm.nih.gov/31779116)" (2019) (relative citation ratio: 4.65), "[Evidence for cross-genotype neutralization of hepatitis C virus pseudo-particles and enhancement of infectivity by apolipoprotein C1.](https://pubmed.ncbi.nlm.nih.gov/15767578)" (2005) (relative citation ratio: 4.42), and "[ApoC1 promotes the metastasis of clear cell renal cell carcinoma via activation of STAT3.](https://pubmed.ncbi.nlm.nih.gov/32826950)" (2020) (relative citation ratio: 3.3). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[APOC1](https://www.proteinatlas.org/ENSG00000130208-APOC1) is a cancer-related gene with evidence at the protein level. Its RNA is detected in all tissues, and it is highly expressed in the liver, where it plays a role in metabolism. In the blood, it is associated with dendritic cells, although its function in this context is unknown. Additionally, APOC1 is expressed in hepatocytes and is involved in metabolism at the single-cell level. In cell lines, it is associated with leukemia and hemostasis.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 5 experiments, [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170) with 4 experiments, [HEY1](https://www.ncbi.nlm.nih.gov/gene/23462) with 4 experiments, [NR2F6](https://www.ncbi.nlm.nih.gov/gene/2063) with 4 experiments, and [GATA2](https://www.ncbi.nlm.nih.gov/gene/2624) with 3 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.


The transcription factors [STAT1](https://www.ncbi.nlm.nih.gov/gene/6872) and [ZNF274](https://www.ncbi.nlm.nih.gov/gene/55929) were found to be regulating genes in one Perturb-seq experiment each, according to data from GTRD's database of transcription factor binding sites and the study published in [PubMed](https://pubmed.ncbi.nlm.nih.gov/35688146/).


The input data from GWAS indicates associations with a wide range of disease conditions, including various types of cancer such as gastrointestinal system cancer, lung cancer, and esophageal cancer. Infectious diseases, particularly those caused by viral agents like human immunodeficiency virus, are also highlighted. Metabolic disorders, including type 2 diabetes mellitus and obesity, are prominently featured. Cardiovascular diseases, such as coronary artery disease and hypertension, are noted, along with neurodegenerative conditions like Alzheimer's disease and dementia. Additionally, the data includes eye diseases, specifically age-related macular degeneration, and musculoskeletal system diseases, including myopathy and muscular disease.



The gene is expressed in multiple tissue samples, including the liver, adrenal gland, and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the liver, adipocyte, lung, adrenal cortex, adrenal gland, fetal liver, and lymph node.




The protein sequence analyzed has a GRAVY value of -0.119, indicating a hydrophilic nature (76.65th percentile). It exhibits a charge of 0.518 at pH 7.0 (51.96th percentile) and a median structural flexibility of 1.013 (87.87th percentile). The protein's secondary structure is predicted to be 43.37% helix (97.3rd percentile), 37.35% sheet (75.05th percentile), and 25.3% turn (23.69th percentile). The instability index is 30.78 (4.7th percentile), with an isoelectric point of 8.01 (58.89th percentile). The protein has a length of 83 amino acids (1.39th percentile) and a molecular weight of 9331.81 Da (1.45th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |