---
layout: post
title: RLBP1
date: 2024-12-08 23:57 CST
description: RLBP1 description
tags: [cooccuring-genes]
categories: longest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6017) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6017  | RLBP1 | ENSG00000140522 | 15q26.1 |



The gene length is 466,056 base pairs (98.08% of all genes), the mature length is 4,116.5 base pairs, and the primary transcript length is 456,569 base pairs (99.11% of all genes).


The gene RLBP1 (NCBI ID: 6017) has been the subject of [38 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RLBP1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication dating back to 1990. The middle 50% of these publications were published between 2003 and 2013.


The top five publications mentioning RLBP1, ranked by their scientific influence, are as follows. The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. The most influential publication is "[Mutation of the gene encoding cellular retinaldehyde-binding protein in autosomal recessive retinitis pigmentosa.](https://pubmed.ncbi.nlm.nih.gov/9326942)" (1997) with a relative citation ratio of 6.01. This is followed by "[Visual cycle impairment in cellular retinaldehyde binding protein (CRALBP) knockout mice results in delayed dark adaptation.](https://pubmed.ncbi.nlm.nih.gov/11301032)" (2001) with a relative citation ratio of 3.95. The publication "[CRALBP supports the mammalian retinal visual cycle and cone vision.](https://pubmed.ncbi.nlm.nih.gov/25607845)" (2015) has a relative citation ratio of 3.61. Additionally, "[Biochemical properties of purified human retinol dehydrogenase 12 (RDH12): catalytic efficiency toward retinoids and C9 aldehydes and effects of cellular retinol-binding protein type I (CRBPI) and cellular retinaldehyde-binding protein (CRALBP) on the oxidation and reduction of retinoids.](https://pubmed.ncbi.nlm.nih.gov/15865448)" (2005) has a relative citation ratio of 2.04. Lastly, "[Newfoundland rod-cone dystrophy, an early-onset retinal dystrophy, is caused by splice-junction mutations in RLBP1.](https://pubmed.ncbi.nlm.nih.gov/11868161)" (2002) has a relative citation ratio of 1.43. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


The gene [None](https://www.proteinatlas.org/None-None) is implicated as a disease variant with evidence at the protein level. It is detected in some RNA tissues and is localized in the nucleoplasm, centrosome, and cytosol. The gene is expressed in various clusters, including Cluster 46 in basophils related to proteolysis, Cluster 76 in the retina involved in visual perception, Cluster 23 in the choroid plexus with mixed function, Cluster 9 in skin cancer related to melanin biosynthesis, and Cluster 10 in Muller glia cells involved in visual perception. The gene is unprognostic for Glioblastoma Multiforme (TCGA) and Skin Cutaneous Melanoma (TCGA).


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 5 experiments, [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) and [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) each with 3 experiments, and [ZEB1](https://www.ncbi.nlm.nih.gov/gene/6935) and [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227) each with 2 experiments.











The analyzed protein sequence has a GRAVY value of -0.406 (45.38th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -14.01 (16.62nd percentile). The median structural flexibility is 1.002 (50.03rd percentile), with a predicted secondary structure composition of 34.70% helix (72.86th percentile), 35.96% sheet (65.94th percentile), and 22.71% turn (8.81st percentile). The instability index is 55.53 (73.53rd percentile), suggesting potential instability. The isoelectric point is 4.98 (7.41st percentile), with a molecular weight of 36473.84 Da (34.26th percentile) and a length of 317 amino acids (32.33rd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |