---
layout: post
title: MT1G
date: 2025-01-17 16:55 CST
description: MT1G description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4495) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4495  | MT1G | ENSG00000125144 | 16q13  |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [zinc ion binding](https://amigo.geneontology.org/amigo/term/GO:0008270), and is involved in [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872). It is active and located in both the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737). The gene plays roles in various biological processes, including [intracellular zinc ion homeostasis](https://amigo.geneontology.org/amigo/term/GO:0006882), [detoxification of copper ion](https://amigo.geneontology.org/amigo/term/GO:0010273), [monocyte differentiation](https://amigo.geneontology.org/amigo/term/GO:0030224), [cellular response to vascular endothelial growth factor stimulus](https://amigo.geneontology.org/amigo/term/GO:0035924), [monocyte activation](https://amigo.geneontology.org/amigo/term/GO:0042117), [negative regulation of growth](https://amigo.geneontology.org/amigo/term/GO:0045926), [cellular response to cadmium ion](https://amigo.geneontology.org/amigo/term/GO:0071276), [cellular response to copper ion](https://amigo.geneontology.org/amigo/term/GO:0071280), and [cellular response to zinc ion](https://amigo.geneontology.org/amigo/term/GO:0071294).


The gene length is 1,033,041.0 base pairs (99.66% of all genes), the mature length is 1,506.0 base pairs, and the primary transcript length is 418,576.0 base pairs.


The gene MT1G (NCBI ID: 4495) has been mentioned in [25 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22MT1G%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1991 and the middle 50% of publications occurring between 2006 and 2020. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning MT1G, ranked by their scientific influence, include "[Metallothionein-1G facilitates sorafenib resistance through inhibition of ferroptosis.](https://pubmed.ncbi.nlm.nih.gov/27015352)" (2016) (relative citation ratio: 14.5), "[Epigenetic regulation of the DNMT1/MT1G/KLF4/CA9 axis synergises the anticancer effects of sorafenib in hepatocellular carcinoma.](https://pubmed.ncbi.nlm.nih.gov/35550167)" (2022) (relative citation ratio: 2.22), "[MT1M and MT1G promoter methylation as biomarkers for hepatocellular carcinoma.](https://pubmed.ncbi.nlm.nih.gov/24782625)" (2014) (relative citation ratio: 1.83), "[Naringenin inhibits pro‑inflammatory cytokine production in macrophages through inducing MT1G to suppress the activation of NF‑κB.](https://pubmed.ncbi.nlm.nih.gov/34252709)" (2021) (relative citation ratio: 1.81), and "[Hypermethylation, but not LOH, is associated with the low expression of MT1G and CRABP1 in papillary thyroid carcinoma.](https://pubmed.ncbi.nlm.nih.gov/12640681)" (2003) (relative citation ratio: 1.79). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[MT1G](https://www.proteinatlas.org/ENSG00000125144-MT1G) is a cancer-related gene with evidence at the protein level. Its RNA is detected in many tissues, with notable expression in the liver and kidney, where it is involved in metabolism. In the brain, it is expressed in astrocytes with mixed functions. Additionally, MT1G is expressed in T-cells related to the T-cell receptor and in myeloid leukemia cell lines involved in the innate immune response. At the single-cell level, it is found in proximal tubular cells, contributing to amino acid metabolism.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) and [SUZ12](https://www.ncbi.nlm.nih.gov/gene/23512), each with 4 experiments. Additionally, [JARID2](https://www.ncbi.nlm.nih.gov/gene/3720), [MYB](https://www.ncbi.nlm.nih.gov/gene/4602), and [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170) were each found to be regulating in 3 experiments.





The gene is expressed in multiple tissue samples, including the liver, kidney, thyroid, pancreas, fetal thyroid, fetal liver, skeletal muscle, and fetal lung.




The protein sequence analyzed has a GRAVY value of 0.276 (91.8% percentile), indicating a relatively hydrophobic nature. It exhibits a charge of 4.301 at pH 7.0 (65.68% percentile) and a median structural flexibility of 0.989 (10.93% percentile). The protein's secondary structure affinities are 30.65% for helix (37.56% percentile), 8.06% for sheet (0.26% percentile), and 27.42% for turn (41.92% percentile). The instability index is 72.10 (95.77% percentile), suggesting high instability. The isoelectric point is 8.38 (64.27% percentile), with a molecular weight of 6141.38 Da (0.18% percentile). The sequence length is 62 amino acids (0.34% percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |