---
layout: post
title: DNAJC7
date: 2024-12-08 23:57 CST
description: DNAJC7 description
tags: [cooccuring-genes]
categories: shortest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7266) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7266  | DNAJC7 | ENSG00000168259 | 17q21.2 |



The gene length is 731.0 base pairs (0.91 percentile of all genes), with a mature length of 731.0 base pairs and a primary transcript length of 731.0 base pairs.


The gene DNAJC7, with NCBI ID 7266, has been the subject of [12 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22DNAJC7%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 2001 and the middle 50% of publications occurring between 2012 and 2021. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning DNAJC7, ranked by their scientific influence, include "[Exome sequencing in amyotrophic lateral sclerosis implicates a novel gene, DNAJC7, encoding a heat-shock protein.](https://pubmed.ncbi.nlm.nih.gov/31768050)" (2019) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 3.93, "[DnaJC7 in Amyotrophic Lateral Sclerosis.](https://pubmed.ncbi.nlm.nih.gov/35456894)" (2022) with an RCR of 2.09, "[Cofactor Tpr2 combines two TPR domains and a J domain to regulate the Hsp70/Hsp90 chaperone system.](https://pubmed.ncbi.nlm.nih.gov/12853476)" (2003) with an RCR of 1.83, "[DnaJC7 binds natively folded structural elements in tau to inhibit amyloid formation.](https://pubmed.ncbi.nlm.nih.gov/34504072)" (2021) with an RCR of 1.32, and "[Mutation screening of the DNAJC7 gene in Japanese patients with sporadic amyotrophic lateral sclerosis.](https://pubmed.ncbi.nlm.nih.gov/35039179)" (2022) with an RCR of 1.09. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [TBP](https://www.ncbi.nlm.nih.gov/gene/6908), [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885), [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170), [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046), and [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), each shown to be regulating in 5 experiments.








The mouse ortholog gene 56354 and its human ortholog gene 7266 are associated with the phenotype of decreased mean corpuscular hemoglobin, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.7218623481781368 (16.26th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -1.7650753720467378 (43.99th percentile), and the median structural flexibility is 1.0084166666666667 (76.41st percentile). The protein has a high affinity for helix structures (0.423076923076923, 96.32nd percentile), with lower affinities for sheet (0.2773279352226721, 17.43rd percentile) and turn structures (0.23279352226720645, 11.23rd percentile). The instability index is 40.94696356275303 (25.1st percentile), and the isoelectric point is 6.557537269592285 (43.74th percentile). The protein has a length of 494 amino acids (57.52nd percentile) and a molecular weight of 56440.23910000001 Da (58.92nd percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |