---
layout: post
title: TLE1
date: 2025-01-17 16:55 CST
description: TLE1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7088) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7088  | TLE1 | ENSG00000196781 | 9q21.32 |



The gene enables [transcription corepressor activity](https://amigo.geneontology.org/amigo/term/GO:0003714) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is part of the [transcription regulator complex](https://amigo.geneontology.org/amigo/term/GO:0005667) and the [beta-catenin-TCF complex](https://amigo.geneontology.org/amigo/term/GO:1990907). It is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). The gene is involved in various processes, including [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), [animal organ morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0009887), [positive regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0010628), [Wnt signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0016055), [negative regulation of Wnt signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0030178), [negative regulation of canonical NF-kappaB signal transduction](https://amigo.geneontology.org/amigo/term/GO:0043124), [negative regulation of DNA-templated transcription](https://amigo.geneontology.org/amigo/term/GO:0045892), [negative regulation of canonical Wnt signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0090090), and [negative regulation of anoikis](https://amigo.geneontology.org/amigo/term/GO:2000811). Additionally, it enables [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802) and [DNA-binding transcription factor binding](https://amigo.geneontology.org/amigo/term/GO:0140297).


The gene length is 78,577 base pairs (76.99% of all genes), the mature length is 2,586 base pairs (65.59% of all genes), and the primary transcript length is 77,775 base pairs (84.2% of all genes).


The gene TLE1 (NCBI ID: 7088) has been mentioned in [61 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22TLE1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1996 and the middle 50% of publications occurring between 2010 and 2019. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning TLE1, ranked by their scientific influence, include "[TLE1 as a diagnostic immunohistochemical marker for synovial sarcoma emerging from gene expression profiling studies.](https://pubmed.ncbi.nlm.nih.gov/17255769)" (2007) (relative citation ratio: 5.82), "[Immunohistochemical staining for TLE1 distinguishes synovial sarcoma from histologic mimics.](https://pubmed.ncbi.nlm.nih.gov/21571956)" (2011) (relative citation ratio: 5.15), "[TLE1 expression is not specific for synovial sarcoma: a whole section study of 163 soft tissue and bone neoplasms.](https://pubmed.ncbi.nlm.nih.gov/19363472)" (2009) (relative citation ratio: 4.99), "[Activating the PARP-1 sensor component of the groucho/ TLE1 corepressor complex mediates a CaMKinase IIdelta-dependent neurogenic gene activation pathway.](https://pubmed.ncbi.nlm.nih.gov/15607978)" (2004) (relative citation ratio: 4.01), and "[Prospective evaluation of TLE1 as a diagnostic immunohistochemical marker in synovial sarcoma.](https://pubmed.ncbi.nlm.nih.gov/19809272)" (2009) (relative citation ratio: 3.15). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[TLE1](https://www.proteinatlas.org/ENSG00000196781-TLE1) is a repressor protein with evidence at the protein level. It is detected in all tissues and primarily localized in the nucleoplasm. TLE1 is expressed in various clusters, including Cluster 22 in eosinophils related to transcription, Cluster 83 in the liver associated with metabolism, Cluster 51 in neurons with mixed function, Cluster 15 in kidney cancer related to membrane components, and Cluster 37 in glandular and luminal cells with unknown function.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) with 6 experiments, [SS18](https://www.ncbi.nlm.nih.gov/gene/6760) with 5 experiments, [SMC3](https://www.ncbi.nlm.nih.gov/gene/9126) with 5 experiments, [PGR](https://www.ncbi.nlm.nih.gov/gene/5241) with 5 experiments, and [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with a range of disease conditions, including lower respiratory tract disease, lung disease, respiratory system disease, disease of metabolism, acquired metabolic disease, carbohydrate metabolism disease, glucose metabolism disease, diabetes mellitus, type 2 diabetes mellitus, nutrition disease, overnutrition, obesity, artery disease, vascular disease, cardiovascular system disease, hypertension, kidney disease, urinary system disease, arteriosclerosis, cerebrovascular disease, and carotid artery disease.



The gene is expressed in various tissues, including nerve, uterus, and cervix uteri, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in pineal gland (both day and night), heart, CD56+ NK cells, CD19+ B cells (negatively selected), lung, and uterus.


The input data highlights various pathways involving NOTCH1 and WNT signaling in cancer, including the formation of the beta-catenin:TCF transactivating complex and the regulation of transcription by the NOTCH1 intracellular domain. Mutations in genes such as FBXW7, NOTCH1, AMER1, APC, AXIN, CTNNB1, GSK3beta, TCF7L2, and LRP5 are implicated in these pathways, leading to the destabilization of the destruction complex, enhanced WNT signaling, and altered protein interactions. Additionally, the data includes pathways related to signal transduction and disease, as well as the effects of specific inhibitors like XAV939 on tankyrase and AXIN stabilization.


The mouse ortholog gene 21885 and its human ortholog gene 7088 are associated with the phenotype of decreased total retina thickness, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.442 (41.38th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -1.793 (43.88th percentile), and the median structural flexibility is 1.006 (68.46th percentile). The protein has a propensity for helix, sheet, and turn structures of 28.05% (17.32nd percentile), 29.35% (24.98th percentile), and 35.84% (91.43rd percentile), respectively. The instability index is 48.37 (51.57th percentile), with an isoelectric point of 6.79 (47.7th percentile). The sequence length is 770 amino acids (79.2nd percentile), and the molecular weight is 83199.88 Da (77.94th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |