---
layout: post
title: HGS
date: 2025-01-17 16:55 CST
description: HGS description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9146) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9146  | HGS | ENSG00000185359 | 17q25.3 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [protein domain specific binding](https://amigo.geneontology.org/amigo/term/GO:0019904), and is located in various cellular components including the [lysosome](https://amigo.geneontology.org/amigo/term/GO:0005764), [endosome](https://amigo.geneontology.org/amigo/term/GO:0005768), [early endosome](https://amigo.geneontology.org/amigo/term/GO:0005769), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [early endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0031901), [multivesicular body membrane](https://amigo.geneontology.org/amigo/term/GO:0032585), [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062), and [phagocytic vesicle lumen](https://amigo.geneontology.org/amigo/term/GO:0097013). It is also part of the [ESCRT-0 complex](https://amigo.geneontology.org/amigo/term/GO:0033565). The gene is involved in numerous biological processes such as [protein targeting to lysosome](https://amigo.geneontology.org/amigo/term/GO:0006622), [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), [negative regulation of cell population proliferation](https://amigo.geneontology.org/amigo/term/GO:0008285), [membrane invagination](https://amigo.geneontology.org/amigo/term/GO:0010324), [positive regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0010628), [endosomal transport](https://amigo.geneontology.org/amigo/term/GO:0016197), [macroautophagy](https://amigo.geneontology.org/amigo/term/GO:0016236), [receptor internalization](https://amigo.geneontology.org/amigo/term/GO:0031623), [endocytic recycling](https://amigo.geneontology.org/amigo/term/GO:0032456), [multivesicular body assembly](https://amigo.geneontology.org/amigo/term/GO:0036258), [regulation of protein catabolic process](https://amigo.geneontology.org/amigo/term/GO:0042176), [protein transport to vacuole involved in ubiquitin-dependent protein catabolic process via the multivesicular body sorting pathway](https://amigo.geneontology.org/amigo/term/GO:0043328), [regulation of MAP kinase activity](https://amigo.geneontology.org/amigo/term/GO:0043405), [negative regulation of receptor signaling pathway via JAK-STAT](https://amigo.geneontology.org/amigo/term/GO:0046426), [protein localization to membrane](https://amigo.geneontology.org/amigo/term/GO:0072657), [membrane fission](https://amigo.geneontology.org/amigo/term/GO:0090148), and [positive regulation of exosomal secretion](https://amigo.geneontology.org/amigo/term/GO:1903543). Additionally, the gene enables [phosphatidylinositol binding](https://amigo.geneontology.org/amigo/term/GO:0035091), [ubiquitin binding](https://amigo.geneontology.org/amigo/term/GO:0043130), [ubiquitin-like protein ligase binding](https://amigo.geneontology.org/amigo/term/GO:0044389), and [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872).


The gene length is 64,767 base pairs (72.81 percentile of all genes), the mature length is 3,000 base pairs (72.5 percentile), and the primary transcript length is 58,535 base pairs (78.82 percentile).


The gene HGS (NCBI ID: 9146) has been mentioned in [70 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22HGS%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1989, and the middle 50% of publications occurred between 2003 and 2015.


The top 5 publications mentioning HGS, ranked by their scientific influence, include "[Hrs sorts ubiquitinated proteins into clathrin-coated microdomains of early endosomes.](https://pubmed.ncbi.nlm.nih.gov/11988743)" (2002) (relative citation ratio: 8.92), "[Hrs regulates multivesicular body formation via ESCRT recruitment to endosomes.](https://pubmed.ncbi.nlm.nih.gov/12900395)" (2003) (relative citation ratio: 5.85), "[HGS-ETR1, a fully human TRAIL-receptor 1 monoclonal antibody, induces cell death in multiple tumour types in vitro and in vivo.](https://pubmed.ncbi.nlm.nih.gov/15846298)" (2005) (relative citation ratio: 5.06), "[Hrs recruits clathrin to early endosomes.](https://pubmed.ncbi.nlm.nih.gov/11532964)" (2001) (relative citation ratio: 4.9), and "[Hrs- and CD63-dependent competing mechanisms make different sized endosomal intraluminal vesicles.](https://pubmed.ncbi.nlm.nih.gov/24279430)" (2014) (relative citation ratio: 4.5). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[HGS](https://www.proteinatlas.org/ENSG00000185359-HGS) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in endosomes, lysosomes, and the cytosol. The gene is expressed in various clusters, including Cluster 28 in neutrophils with unknown function, Cluster 66 in non-specific tissues related to transcription, Cluster 32 in the cerebral cortex with mixed function, Cluster 27 in breast cancer cell lines with unknown function, and Cluster 63 in late spermatids involved in spermatogenesis.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [KLF5](https://www.ncbi.nlm.nih.gov/gene/688) and [E2F8](https://www.ncbi.nlm.nih.gov/gene/79733), each with 6 experiments. Additionally, [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613), [TCF12](https://www.ncbi.nlm.nih.gov/gene/6938), and [KMT2B](https://www.ncbi.nlm.nih.gov/gene/9757) were each found to be regulating in 5 experiments.




The genome-wide association study identified a significant association with [Lung function (FVC)](https://pubmed.ncbi.nlm.nih.gov/34226706), which was mapped to the closest existing MeSH term [Lung](https://meshb.nlm.nih.gov/record/ui?ui=D008168) using the method [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0).


The gene is expressed in various tissues, including the prostate, pituitary, and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Lymphoma (Burkitt's, Raji), CD33+ Myeloid cells, CD8+ T cells, and Colorectal adenocarcinoma.


The input data highlights several pathways involving the epidermal growth factor receptor (EGFR), including its downregulation, signaling in various cancer contexts, and its role in disease, signal transduction, and membrane trafficking. Additionally, the endosomal sorting complex required for transport (ESCRT) is mentioned, indicating its involvement in these processes.


The mouse ortholog gene 15239 and its human ortholog gene 9146 are associated with the phenotype of preweaning lethality with complete penetrance, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.697 (17.77th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -10.24 (22.09th percentile). The median structural flexibility is 1.011 (84.02nd percentile), with a predicted secondary structure composition of 34.75% helix (73.15th percentile), 25.87% sheet (10.93rd percentile), and 28.83% turn (54.56th percentile). The instability index is 69.59 (94.37th percentile), suggesting high instability. The isoelectric point is 5.84 (26.35th percentile), with a molecular weight of 86191.07 Da (79.24th percentile). The sequence length is 777 amino acids (79.55th percentile). For detailed information on these calculations, refer to [BioPython's sequence analysis](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |