---
layout: post
title: HLA-DPA1
date: 2025-01-17 16:55 CST
description: HLA-DPA1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3113) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3113  | HLA-DPA1 | ENSG00000231389 | 6p21.32 |



The gene is located in the [Golgi membrane](https://amigo.geneontology.org/amigo/term/GO:0000139), [lysosomal membrane](https://amigo.geneontology.org/amigo/term/GO:0005765), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986), [ER to Golgi transport vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0012507), [transport vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030658), [endocytic vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030666), [clathrin-coated endocytic vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030669), [trans-Golgi network membrane](https://amigo.geneontology.org/amigo/term/GO:0032588), [intracellular membrane-bounded organelle](https://amigo.geneontology.org/amigo/term/GO:0043231), and the [lumenal side of endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0098553). It is involved in the [adaptive immune response](https://amigo.geneontology.org/amigo/term/GO:0002250), [peptide antigen assembly with MHC class II protein complex](https://amigo.geneontology.org/amigo/term/GO:0002503), [immune response](https://amigo.geneontology.org/amigo/term/GO:0006955), [antigen processing and presentation of exogenous peptide antigen via MHC class II](https://amigo.geneontology.org/amigo/term/GO:0019886), [positive regulation of type II interferon production](https://amigo.geneontology.org/amigo/term/GO:0032729), [positive regulation of T cell proliferation](https://amigo.geneontology.org/amigo/term/GO:0042102), [positive regulation of immune response](https://amigo.geneontology.org/amigo/term/GO:0050778), [positive regulation of T cell activation](https://amigo.geneontology.org/amigo/term/GO:0050870), and [cellular response to type II interferon](https://amigo.geneontology.org/amigo/term/GO:0071346). Additionally, it enables [MHC class II protein complex binding](https://amigo.geneontology.org/amigo/term/GO:0023026), [MHC class II receptor activity](https://amigo.geneontology.org/amigo/term/GO:0032395), and [peptide antigen binding](https://amigo.geneontology.org/amigo/term/GO:0042605). The gene is also a part of the [MHC class II protein complex](https://amigo.geneontology.org/amigo/term/GO:0042613) and is active in the [lysosomal membrane](https://amigo.geneontology.org/amigo/term/GO:0005765) and [late endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0031902).


The gene length is 413,574 base pairs (97.56% of all genes), the mature length is 1,053 base pairs (24.39% of all genes), and the primary transcript length is 166,140 base pairs (94.5% of all genes).


The gene HLA-DPA1 (NCBI ID: 3113) has been mentioned in [176 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22HLA-DPA1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning this gene was in 1985, and the middle 50% of publications occurred between 2010 and 2020.


The top five publications mentioning HLA-DPA1, ranked by their scientific influence, include "[Five amino acids in three HLA proteins explain most of the association between MHC and seropositive rheumatoid arthritis.](https://pubmed.ncbi.nlm.nih.gov/22286218)" (2012) (relative citation ratio: 17.37), "[The IMGT/HLA database.](https://pubmed.ncbi.nlm.nih.gov/23080122)" (2013) (relative citation ratio: 14.43), "[Peptide binding predictions for HLA DR, DP and DQ molecules.](https://pubmed.ncbi.nlm.nih.gov/21092157)" (2010) (relative citation ratio: 11.04), "[A genomic perspective on HLA evolution.](https://pubmed.ncbi.nlm.nih.gov/28687858)" (2018) (relative citation ratio: 4.86), and "[A high-resolution HLA reference panel capturing global population diversity enables multi-ancestry fine-mapping in HIV host response.](https://pubmed.ncbi.nlm.nih.gov/34611364)" (2021) (relative citation ratio: 4.34). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[HLA-DPA1](https://www.proteinatlas.org/ENSG00000231389-HLA-DPA1) is a gene with evidence at the protein level. Its RNA is detected in all tissues, and the protein is localized in vesicles. The gene is expressed in dendritic cells with unknown function (Cluster 2) and is associated with immune response in lymphoid tissue (Cluster 70). In single-cell expression analysis, it is found in dendritic cells involved in antigen presentation (Cluster 28). The gene's expression in various cancer types is generally unprognostic, including bladder urothelial carcinoma, breast invasive carcinoma, and others.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), [AR](https://www.ncbi.nlm.nih.gov/gene/367), and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), each with 4 experiments. Additionally, [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) and [PPARG](https://www.ncbi.nlm.nih.gov/gene/5468) were found to be regulating in 3 experiments each.



The input data indicates associations with a wide range of disease conditions, including various types of cancer such as reproductive organ cancer, gastrointestinal system cancer, and organ system cancer. Infectious diseases, particularly viral infectious diseases like hepatitis B, are also represented. Respiratory conditions such as asthma, bronchial disease, obstructive lung disease, and lower respiratory tract disease are noted. Metabolic disorders, including diabetes mellitus and type 1 diabetes mellitus, are included, as well as immune system diseases like rheumatoid arthritis, scleroderma, and primary immunodeficiency disease. Additionally, the data encompasses cardiovascular system diseases, liver diseases such as hepatocellular carcinoma and liver cancer, and specific conditions like Wegener's granulomatosis and vasculitis.



The gene is expressed in multiple tissue samples, including lung, blood, and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in various cell types and tissues such as 721_B_lymphoblasts, CD33+_Myeloid, BDCA4+_DentriticCells, CD19+_BCells(neg. sel.), CD14+_Monocytes, Lung, and small intestine.


The input data highlights various pathways involved in immune system function, including cytokine signaling in the immune system, interferon signaling, and interferon gamma signaling. Additionally, it covers adaptive immune system processes, costimulation by the CD28 family, and downstream TCR signaling. Other notable pathways include TCR signaling, generation of second messenger molecules, PD-1 signaling, phosphorylation of CD3 and TCR zeta chains, translocation of ZAP-70 to the immunological synapse, and MHC class II antigen presentation.



The analyzed protein sequence has a GRAVY value of -0.029 (82.57th percentile), indicating a slightly hydrophilic nature. The charge at pH 7 is -14.52 (15.92nd percentile), and the median structural flexibility is 0.992 (14.57th percentile). The protein has a propensity for helix, sheet, and turn structures of 31.92% (49.63rd percentile), 41.54% (89.83rd percentile), and 24.62% (18.8th percentile), respectively. The instability index is 41.66 (27.51st percentile), with an isoelectric point of 5.07 (9.05th percentile). The protein is 260 amino acids long (23.02nd percentile) and has a molecular weight of 29380.22 Da (23.62nd percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |