---
layout: post
title: AP4S1
date: 2025-01-17 16:55 CST
description: AP4S1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/11154) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 11154  | AP4S1 | ENSG00000100478 | 14q12 |



The gene is located in the [trans-Golgi network](https://amigo.geneontology.org/amigo/term/GO:0005802) and the [trans-Golgi network membrane](https://amigo.geneontology.org/amigo/term/GO:0032588), as well as the [endosome lumen](https://amigo.geneontology.org/amigo/term/GO:0031904) and [intracellular membrane-bounded organelle](https://amigo.geneontology.org/amigo/term/GO:0043231). It is involved in [protein targeting](https://amigo.geneontology.org/amigo/term/GO:0006605), [protein localization](https://amigo.geneontology.org/amigo/term/GO:0008104), [protein transport](https://amigo.geneontology.org/amigo/term/GO:0015031), and [vesicle-mediated transport](https://amigo.geneontology.org/amigo/term/GO:0016192). Additionally, it is part of the [AP-4 adaptor complex](https://amigo.geneontology.org/amigo/term/GO:0030124) and is active in the [intracellular membrane-bounded organelle](https://amigo.geneontology.org/amigo/term/GO:0043231).


The gene length is 53,823 base pairs (68.4% of all genes), the mature length is 640 base pairs (7.02% of all genes), and the primary transcript length is 30,837 base pairs (63.3% of all genes).


AP4S1 has been mentioned in [5 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22AP4S1%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 2015, and the middle 50% of publications occurred between 2017 and 2020.


The top 5 publications mentioning AP4S1 include studies that highlight the gene's role in various neurological conditions. The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Notable publications include "[Recessive loss-of-function mutations in AP4S1 cause mild fever-sensitive seizures, developmental delay and spastic paraplegia through loss of AP-4 complex assembly.](https://pubmed.ncbi.nlm.nih.gov/25552650)" (2015) (relative citation ratio: 1.22), "[Loss of ap4s1 in zebrafish leads to neurodevelopmental defects resembling spastic paraplegia 52.](https://pubmed.ncbi.nlm.nih.gov/32216065)" (2020) (relative citation ratio: 1.18), and "[A homozygous splice variant in AP4S1 mimicking neurodegeneration with brain iron accumulation.](https://pubmed.ncbi.nlm.nih.gov/28150420)" (2017) (relative citation ratio: 0.6). These studies provide insights into the genetic basis of neurological disorders associated with AP4S1. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[AP4S1](https://www.proteinatlas.org/ENSG00000100478-AP4S1) is implicated in hereditary spastic paraplegia and neurodegeneration, with evidence at the protein level. The gene is detected in all RNA tissue distributions and is primarily localized in vesicles. It is part of Cluster 3 in blood expression, which is associated with non-specific nuclear processes, and Cluster 5 in tissue expression, which is linked to brain and testis signal transduction. In the brain, it belongs to Cluster 34, indicating non-specific mixed function, and in cell lines, it is part of Cluster 30, associated with gene expression regulation. Additionally, AP4S1 is found in Cluster 31 in single-cell expression, which is related to spermatocytes and spermatogenesis.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113), [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885), [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169), and [AR](https://www.ncbi.nlm.nih.gov/gene/367), each shown to be regulating in 5 experiments.



The GWAS data indicates associations with various disease conditions, including general diseases, diseases of anatomical entities, nervous system diseases, and sensory system diseases.



The gene is expressed in various tissues, including the brain and pituitary, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in cardiac myocytes, heart, superior cervical ganglion, trigeminal ganglion, skeletal muscle, atrioventricular node, uterus corpus, and tongue.




The analyzed protein sequence has a GRAVY value of 0.0417 (85.82nd percentile), indicating a slightly hydrophobic nature. The charge at pH 7.0 is -6.24 (30.3rd percentile), and the median structural flexibility is 0.9884 (10.12nd percentile). The protein has a propensity for helix, sheet, and turn structures of 34.72% (72.97th percentile), 43.75% (93.17th percentile), and 20.83% (3.59th percentile), respectively. The instability index is 42.56 (30.58th percentile), with an isoelectric point of 5.08 (9.27th percentile). The protein is 144 amino acids long (7.32nd percentile) and has a molecular weight of 17004.57 Da (8.32nd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |