---
layout: post
title: CELF1
date: 2025-01-17 16:55 CST
description: CELF1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10658) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10658  | CELF1 | ENSG00000149187 | 11p11.2 |



The gene is involved in the [regulation of alternative mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0000381) and [mRNA splice site recognition](https://amigo.geneontology.org/amigo/term/GO:0006376), playing a crucial role in [mRNA processing](https://amigo.geneontology.org/amigo/term/GO:0006397). It enables [mRNA regulatory element binding translation repressor activity](https://amigo.geneontology.org/amigo/term/GO:0000900), [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [mRNA binding](https://amigo.geneontology.org/amigo/term/GO:0003729), and [mRNA 3'-UTR binding](https://amigo.geneontology.org/amigo/term/GO:0003730). Additionally, it is active in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), and is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytoplasmic stress granule](https://amigo.geneontology.org/amigo/term/GO:0010494), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), and [perinucleolar compartment](https://amigo.geneontology.org/amigo/term/GO:0097356). The gene is also part of the [ribonucleoprotein complex](https://amigo.geneontology.org/amigo/term/GO:1990904) and acts upstream of or within [positive regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0010628), [negative regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0010629), and [regulation of inflammatory response](https://amigo.geneontology.org/amigo/term/GO:0050727). Furthermore, it is involved in [germ cell development](https://amigo.geneontology.org/amigo/term/GO:0007281), [embryo development ending in birth or egg hatching](https://amigo.geneontology.org/amigo/term/GO:0009792), [post-transcriptional gene silencing](https://amigo.geneontology.org/amigo/term/GO:0016441), [negative regulation of translation](https://amigo.geneontology.org/amigo/term/GO:0017148), [regulatory ncRNA-mediated post-transcriptional gene silencing](https://amigo.geneontology.org/amigo/term/GO:0035194), [regulation of RNA splicing](https://amigo.geneontology.org/amigo/term/GO:0043484), and [mRNA destabilization](https://amigo.geneontology.org/amigo/term/GO:0061157).


The gene length is 19,499 base pairs (41.47th percentile), the mature length is 10,885 base pairs, and the primary transcript length is 19,499 base pairs (50.99th percentile).


The gene CELF1 (NCBI ID: 10658) has been mentioned in [96 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CELF1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1998 and the middle 50% of publications occurring between 2009 and 2017. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning CELF1, ranked by their scientific influence, include "[Increased steady-state levels of CUGBP1 in myotonic dystrophy 1 are due to PKC-mediated hyperphosphorylation.](https://pubmed.ncbi.nlm.nih.gov/17936705)" (2007) (relative citation ratio: 6.44), "[The importance of CELF control: molecular and biological roles of the CUG-BP, Elav-like family of RNA-binding proteins.](https://pubmed.ncbi.nlm.nih.gov/22180311)" (2012) (relative citation ratio: 3.92), "[Long noncoding RNA TUG1 is downregulated in non-small cell lung cancer and can regulate CELF1 on binding to PRC2.](https://pubmed.ncbi.nlm.nih.gov/27485439)" (2016) (relative citation ratio: 3.74), "[CUGBP1 and MBNL1 preferentially bind to 3' UTRs and facilitate mRNA decay.](https://pubmed.ncbi.nlm.nih.gov/22355723)" (2012) (relative citation ratio: 3.25), and "[Conserved GU-rich elements mediate mRNA decay by binding to CUG-binding protein 1.](https://pubmed.ncbi.nlm.nih.gov/18243120)" (2008) (relative citation ratio: 3.25). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CELF1](https://www.proteinatlas.org/ENSG00000149187-CELF1) is an RNA-binding activator with evidence at the protein level. It is detected in all tissues and primarily localizes to the nucleoplasm. CELF1 is involved in basic cellular processes and transcription, as indicated by its tissue and brain expression clusters. In blood, it is associated with the innate immune response. However, its expression in various cancer types, including bladder, breast, and lung cancers, is unprognostic.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [YY1](https://www.ncbi.nlm.nih.gov/gene/7528), [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025), [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885), [ZFX](https://www.ncbi.nlm.nih.gov/gene/7543), and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), each shown to be regulating in 5 experiments.



The input data indicates associations with various disease conditions, including immune system disease, nervous system disease, central nervous system disease, neurodegenerative disease, Alzheimer's disease, tauopathy, dementia, cognitive disorder, disease of mental health, B cell deficiency, primary immunodeficiency disease, dysgammaglobulinemia, selective immunoglobulin deficiency disease, and selective IgA deficiency disease.



The gene is expressed in various tissues, including lung and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in CD4+ T cells, CD8+ T cells, CD71+ early erythroid cells, fetal brain, cerebellum, and cerebellum peduncles.




The analyzed protein sequence has a GRAVY value of -0.237 (65.42 percentile), indicating a hydrophilic nature. The charge at pH 7 is 5.07 (68.32 percentile), and the median structural flexibility is 1.0005 (44.01 percentile). The protein has a high affinity for helix (33.54%, 63.98 percentile) and turn (32.10%, 77.25 percentile) structures, with a lower affinity for sheet structures (30.04%, 28.62 percentile). The instability index is 42.97 (32.08 percentile), and the isoelectric point is 8.70 (71.23 percentile). The protein is 486 amino acids long (56.58 percentile) with a molecular weight of 52062.60 Da (53.99 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |