---
layout: post
title: GSPT1
date: 2025-01-17 16:55 CST
description: GSPT1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2935) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2935  | GSPT1 | ENSG00000103342 | 16p13.13 |



The gene is involved in the [G1/S transition of mitotic cell cycle](https://amigo.geneontology.org/amigo/term/GO:0000082) and the [nuclear-transcribed mRNA catabolic process, nonsense-mediated decay](https://amigo.geneontology.org/amigo/term/GO:0000184). It enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [translation release factor activity](https://amigo.geneontology.org/amigo/term/GO:0003747), [GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0003924), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [GTP binding](https://amigo.geneontology.org/amigo/term/GO:0005525). The gene is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737) and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and is involved in [translation](https://amigo.geneontology.org/amigo/term/GO:0006412), [translational termination](https://amigo.geneontology.org/amigo/term/GO:0006415), and the [regulation of translational termination](https://amigo.geneontology.org/amigo/term/GO:0006449). It acts upstream of or within [protein methylation](https://amigo.geneontology.org/amigo/term/GO:0006479) and is part of the [translation release factor complex](https://amigo.geneontology.org/amigo/term/GO:0018444). The gene is active in the [cytosolic ribosome](https://amigo.geneontology.org/amigo/term/GO:0022626).


The gene length is 194,120 base pairs (92.11% of all genes), the mature length is 6,445.5 base pairs, and the primary transcript length is 193,498 base pairs.


The gene GSPT1 (NCBI ID: 2935) has been mentioned in [24 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22GSPT1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1992 and the middle 50% of publications occurring between 2007 and 2020. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning GSPT1, ranked by their scientific influence, include "[A novel cereblon modulator recruits GSPT1 to the CRL4(CRBN) ubiquitin ligase.](https://pubmed.ncbi.nlm.nih.gov/27338790)" (2016) (relative citation ratio: 10.15), "[CC-90009: A Cereblon E3 Ligase Modulating Drug That Promotes Selective Degradation of GSPT1 for the Treatment of Acute Myeloid Leukemia.](https://pubmed.ncbi.nlm.nih.gov/33591756)" (2021) (relative citation ratio: 4.28), "[Translation Termination Factor GSPT1 Is a Phenotypically Relevant Off-Target of Heterobifunctional Phthalimide Degraders.](https://pubmed.ncbi.nlm.nih.gov/29356495)" (2018) (relative citation ratio: 3.74), "[Merging PROTAC and molecular glue for degrading BTK and GSPT1 proteins concurrently.](https://pubmed.ncbi.nlm.nih.gov/34417569)" (2021) (relative citation ratio: 2.71), and "[miRNA-144 suppresses proliferation and migration of colorectal cancer cells through GSPT1.](https://pubmed.ncbi.nlm.nih.gov/26349975)" (2015) (relative citation ratio: 2.5). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[GSPT1](https://www.proteinatlas.org/ENSG00000103342-GSPT1) is a hydrolase with evidence at the protein level. It is detected in all tissues and is primarily localized in vesicles and the cytosol. The gene is part of Cluster 12 in blood expression, which is associated with non-specific mitochondrial translation. In tissue expression, it belongs to Cluster 19, linked to basic cellular processes. In the brain, it is part of Cluster 28, associated with oligodendrocytes and mixed functions. In cell line expression, it is found in Cluster 54, related to non-specific transcription, and in single-cell expression, it is part of Cluster 78, associated with NK-cells, T-cells, and the immune system and transcription.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2B](https://www.ncbi.nlm.nih.gov/gene/9757) with 6 experiments, [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) with 5 experiments, [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861) with 5 experiments, [OLIG2](https://www.ncbi.nlm.nih.gov/gene/10215) with 5 experiments, and [PGR](https://www.ncbi.nlm.nih.gov/gene/5241) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with various metabolic disorders, including diseases of metabolism, acquired metabolic diseases, carbohydrate metabolism diseases, glucose metabolism diseases, and specifically, diabetes mellitus.



The gene is expressed in various tissues and cell types, including muscle and adrenal gland. Additionally, it is expressed in 721_B_lymphoblasts, Leukemia_chronicMyelogenousK-562, Leukemia_promyelocytic-HL-60, Lymphoma_burkitts(Daudi), CD105+_Endothelial, CD71+_EarlyErythroid, BronchialEpithelialCells, and Bonemarrow.



The mouse ortholog gene 14852 and its human ortholog gene 2935 exhibit complete penetrance preweaning lethality in both sexes.


The analyzed protein sequence has a GRAVY value of -0.43146292585170287 (42.52nd percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -16.253667824426415 (14.24th percentile), and the median structural flexibility is 1.006452380952381 (69.66th percentile). The protein has a high affinity for helix structures (77.75th percentile) and a moderate affinity for sheet structures (47.11th percentile), with a turn affinity of 0.2785571142284569 (45.93rd percentile). The instability index is 40.60120240480961 (23.96th percentile), and the isoelectric point is 5.4466760635375975 (16.93rd percentile). The protein sequence is 499 amino acids long (58.09th percentile) with a molecular weight of 55755.28879999998 Da (58.19th percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |