---
layout: post
title: FCMR
date: 2025-01-17 16:55 CST
description: FCMR description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9214) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9214  | FCMR | ENSG00000162894 | 1q32.1 |



The gene enables [polymeric immunoglobulin binding](https://amigo.geneontology.org/amigo/term/GO:0001790), [IgM binding](https://amigo.geneontology.org/amigo/term/GO:0001791), and [high-affinity IgM receptor activity](https://amigo.geneontology.org/amigo/term/GO:0002172). It is involved in [immunoglobulin transcytosis in epithelial cells](https://amigo.geneontology.org/amigo/term/GO:0002414), [humoral immune response mediated by circulating immunoglobulin](https://amigo.geneontology.org/amigo/term/GO:0002455), [cellular defense response](https://amigo.geneontology.org/amigo/term/GO:0006968), [Fc receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0038093), [negative regulation of apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0043066), [regulation of B cell receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0050855), and [Fc receptor-mediated immune complex endocytosis](https://amigo.geneontology.org/amigo/term/GO:0160006). The gene is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [lysosomal membrane](https://amigo.geneontology.org/amigo/term/GO:0005765), [early endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0031901), [trans-Golgi network membrane](https://amigo.geneontology.org/amigo/term/GO:0032588), and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), where it is also active. Additionally, it enables [transmembrane signaling receptor activity](https://amigo.geneontology.org/amigo/term/GO:0004888) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515).


The gene length is 35,022 base pairs (56.91% of all genes), the mature length is 1,377.5 base pairs, and the primary transcript length is 10,554 base pairs (34.69% of all genes).


The gene FCMR (NCBI ID: 9214) has been mentioned in [45 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22FCMR%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning FCMR was in 1980, and the middle 50% of publications occurred between 2009 and 2016.


The top 5 publications mentioning FCMR, ranked by their scientific influence, include "[Identity of the elusive IgM Fc receptor (FcmuR) in humans.](https://pubmed.ncbi.nlm.nih.gov/19858324)" (2009) (relative citation ratio: 3.14), "[Emerging Functions of Natural IgM and Its Fc Receptor FCMR in Immune Homeostasis.](https://pubmed.ncbi.nlm.nih.gov/27014278)" (2016) (relative citation ratio: 2.28), "[Mouse IgM Fc receptor, FCMR, promotes B cell development and modulates antigen-driven immune responses.](https://pubmed.ncbi.nlm.nih.gov/23267023)" (2013) (relative citation ratio: 1.7), "[TOSO, the Fcmicro receptor, is highly expressed on chronic lymphocytic leukemia B cells, internalizes upon IgM binding, shuttles to the lysosome, and is downregulated in response to TLR activation.](https://pubmed.ncbi.nlm.nih.gov/21908732)" (2011) (relative citation ratio: 1.41), and "[Chimeric antigen receptor T cells targeting Fc Î¼ receptor selectively eliminate CLL cells while sparing healthy B cells.](https://pubmed.ncbi.nlm.nih.gov/27535994)" (2016) (relative citation ratio: 1.38). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[FCMR](https://www.proteinatlas.org/ENSG00000162894-FCMR) is a gene with evidence at the protein level, detected in many RNA tissues. It is primarily localized in the nucleoplasm, Golgi apparatus, plasma membrane, and centrosome. The gene is expressed in various clusters, including Cluster 43 for non-specific transcription and translation in blood, Cluster 36 for cytokine signaling in lymphoid tissue, Cluster 10 for nucleic acid binding in the cerebellum, Cluster 46 for humoral immune response in lymphoma cell lines, and Cluster 73 for B-cell function in single cells.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [EZH2](https://www.ncbi.nlm.nih.gov/gene/2146), [AR](https://www.ncbi.nlm.nih.gov/gene/367), [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), [SUZ12](https://www.ncbi.nlm.nih.gov/gene/23512), and [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), each shown to be regulating in 4 experiments.



The input data indicates associations with various metabolic disorders, including diseases of metabolism, acquired metabolic diseases, carbohydrate metabolism diseases, glucose metabolism diseases, diabetes mellitus, and specifically type 2 diabetes mellitus.



The gene is expressed in various tissues and cell types, including blood, spleen, and small intestine, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in CD56+ NK cells, CD4+ T cells, CD8+ T cells, CD19+ B cells (neg. sel.), tonsil, thymus, and lymph node.



The mouse ortholog gene 69169 and its human ortholog gene 9214 are associated with the phenotype of enlarged adrenal glands, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.403 (45.77th percentile), indicating a hydrophilic nature. It exhibits a charge of 18.53 at pH 7.0 (92.38th percentile) and a median structural flexibility of 1.001 (45.78th percentile). The protein's secondary structure is predicted to be 29.49% helix (27.4th percentile), 33.33% sheet (47.43rd percentile), and 30.00% turn (63.98th percentile). The instability index is 56.66 (76.31st percentile), with an isoelectric point of 9.78 (94.67th percentile). The protein has a length of 390 amino acids (43.91st percentile) and a molecular weight of 43145.99 Da (43.66th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |