---
layout: post
title: NRCAM
date: 2025-01-17 16:55 CST
description: NRCAM description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4897) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4897  | NRCAM | ENSG00000091129 | 7q31.1 |



The gene is involved in various biological processes, including [angiogenesis](https://amigo.geneontology.org/amigo/term/GO:0001525), [neuron migration](https://amigo.geneontology.org/amigo/term/GO:0001764), [axonogenesis](https://amigo.geneontology.org/amigo/term/GO:0007409), [axon guidance](https://amigo.geneontology.org/amigo/term/GO:0007411), [axonal fasciculation](https://amigo.geneontology.org/amigo/term/GO:0007413), [synapse assembly](https://amigo.geneontology.org/amigo/term/GO:0007416), [central nervous system development](https://amigo.geneontology.org/amigo/term/GO:0007417), [brain development](https://amigo.geneontology.org/amigo/term/GO:0007420), [protein localization](https://amigo.geneontology.org/amigo/term/GO:0008104), [regulation of neuron projection development](https://amigo.geneontology.org/amigo/term/GO:0010975), [neuronal action potential propagation](https://amigo.geneontology.org/amigo/term/GO:0019227), [regulation of axon extension](https://amigo.geneontology.org/amigo/term/GO:0030516), [retinal ganglion cell axon guidance](https://amigo.geneontology.org/amigo/term/GO:0031290), [heterotypic cell-cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0034113), [clustering of voltage-gated sodium channels](https://amigo.geneontology.org/amigo/term/GO:0045162), [positive regulation of neuron differentiation](https://amigo.geneontology.org/amigo/term/GO:0045666), [cell-cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0098609), and [regulation of postsynapse organization](https://amigo.geneontology.org/amigo/term/GO:0099175). It enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [ankyrin binding](https://amigo.geneontology.org/amigo/term/GO:0030506), [protein binding involved in heterotypic cell-cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0086080), and [cell-cell adhesion mediator activity](https://amigo.geneontology.org/amigo/term/GO:0098632). The gene is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [external side of plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009897), [neuron projection](https://amigo.geneontology.org/amigo/term/GO:0043005), [axon initial segment](https://amigo.geneontology.org/amigo/term/GO:0043194), [postsynaptic density membrane](https://amigo.geneontology.org/amigo/term/GO:0098839), and [glutamatergic synapse](https://amigo.geneontology.org/amigo/term/GO:0098978). It is active in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886) and [axon](https://amigo.geneontology.org/amigo/term/GO:0030424).


The gene has a length of 10,970 base pairs (27.67% of all genes), a mature length of 868 base pairs, and a primary transcript length of 2,445 base pairs.


The gene NRCAM (NCBI ID: 4897) has been mentioned in [47 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22NRCAM%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1992 and the middle 50% of publications occurring between 2001 and 2014. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning NRCAM, ranked by their scientific influence, include "[A glial signal consisting of gliomedin and NrCAM clusters axonal Na+ channels during the formation of nodes of Ranvier.](https://pubmed.ncbi.nlm.nih.gov/20188654)" (2010) (relative citation ratio: 3.68), "[Identification of endothelial cell genes expressed in an in vitro model of angiogenesis: induction of ESM-1, (beta)ig-h3, and NrCAM.](https://pubmed.ncbi.nlm.nih.gov/11866539)" (2002) (relative citation ratio: 2.58), "[Nr-CAM is a target gene of the beta-catenin/LEF-1 pathway in melanoma and colon cancer and its expression enhances motility and confers tumorigenesis.](https://pubmed.ncbi.nlm.nih.gov/12183361)" (2002) (relative citation ratio: 2.4), "[The role of NrCAM in neural development and disorders--beyond a simple glue in the brain.](https://pubmed.ncbi.nlm.nih.gov/22182708)" (2012) (relative citation ratio: 2.37), and "[Induction of axonal growth by heterophilic interactions between the cell surface recognition proteins F11 and Nr-CAM/Bravo.](https://pubmed.ncbi.nlm.nih.gov/8274278)" (1993) (relative citation ratio: 2.11). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[NRCAM](https://www.proteinatlas.org/ENSG00000091129-NRCAM) is a gene with evidence at the protein level, implicated in intellectual disability. It is detected in many tissues and is localized to various subcellular regions including the nucleoplasm, vesicles, plasma membrane, primary cilium, and cytosol. NRCAM is associated with neuronal and synaptic functions, as indicated by its tissue expression cluster, and is particularly active in astrocyte-neuron interactions in the brain. It is also involved in signal transduction in neuronal cell lines and neuronal signaling in single cells.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SUZ12](https://www.ncbi.nlm.nih.gov/gene/23512) with 5 experiments, [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861) with 4 experiments, [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227) with 4 experiments, [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046) with 4 experiments, and [JUN](https://www.ncbi.nlm.nih.gov/gene/3725) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The input data from GWAS indicates associations with a wide range of disease conditions, including leukocyte disease, leukopenia, hematopoietic system disease, immune system disease, and various metabolic disorders such as diabetes mellitus and type 2 diabetes mellitus. Additionally, the data highlights associations with cardiovascular system diseases, including arterial and vascular diseases, as well as hypertension. Neurological conditions such as Alzheimer's disease, tauopathy, dementia, and cognitive disorders are also implicated. Furthermore, the data includes associations with mental health disorders like schizophrenia, and various bone and musculoskeletal diseases, including osteoporosis. Immunodeficiency conditions, such as B cell deficiency and primary immunodeficiency disease, are also noted.



The gene is expressed in various brain regions, including the prefrontal cortex, amygdala, thalamus, cingulate cortex, subthalamic nucleus, parietal lobe, and occipital lobe. Additionally, it is expressed in the pituitary and adrenal gland.


The proteins are involved in several pathways related to neural development and cellular interactions, including axon guidance, developmental biology, interaction between L1 and ankyrins, L1CAM interactions, neurofascin interactions, and NrCAM interactions.


The mouse ortholog gene 319504, which corresponds to the human ortholog gene 4897, is associated with convulsive seizures in a sex-specific manner and increased fasting circulating glucose levels in both sexes.


The analyzed protein sequence has a GRAVY value of -0.3796 (48.62 percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -28.27 (6.62 percentile), and the median structural flexibility is 1.0047 (62.67 percentile). The protein has a length of 1304 amino acids (93.2 percentile) and a molecular weight of 143887.96 Da (93.13 percentile). The secondary structure affinities are 28.60% for helix (20.86 percentile), 35.74% for sheet (64.47 percentile), and 33.21% for turn (82.89 percentile). The instability index is 39.35 (20.36 percentile), and the isoelectric point is 5.45 (16.94 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |