---
layout: post
title: ZNF222
date: 2024-12-08 23:57 CST
description: ZNF222 description
tags: [cooccuring-genes]
categories: longest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7673) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7673  | ZNF222 | ENSG00000159885 | 19q13.31 |



The gene length is 448,316 base pairs (97.93% of all genes), the mature length is 10,953 base pairs, and the primary transcript length is 448,078.5 base pairs (99.07% of all genes).





The top 5 publications mentioning ZNF222, ranked by their scientific influence as measured by the [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541), are as follows:

1. [Title of the first publication](https://pubmed.ncbi.nlm.nih.gov/[PubMed ID]) (Year) ...... (Relative Citation Ratio)
2. [Title of the second publication](https://pubmed.ncbi.nlm.nih.gov/[PubMed ID]) (Year) ...... (Relative Citation Ratio)
3. [Title of the third publication](https://pubmed.ncbi.nlm.nih.gov/[PubMed ID]) (Year) ...... (Relative Citation Ratio)
4. [Title of the fourth publication](https://pubmed.ncbi.nlm.nih.gov/[PubMed ID]) (Year) ...... (Relative Citation Ratio)
5. [Title of the fifth publication](https://pubmed.ncbi.nlm.nih.gov/[PubMed ID]) (Year) ...... (Relative Citation Ratio)

The Relative Citation Ratio (RCR) values measure the scientific influence of each paper by field- and time-adjusting the citations it has received, and benchmarking to the median for NIH publications, the value of which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [PPARG](https://www.ncbi.nlm.nih.gov/gene/5468) with 5 experiments, [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046) with 4 experiments, [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) with 4 experiments, [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) with 4 experiments, and [SUPT5H](https://www.ncbi.nlm.nih.gov/gene/6829) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent detection in multiple experiments.











The protein sequence analyzed has a GRAVY value of -0.795 (11.93rd percentile), indicating a hydrophilic nature. It exhibits a charge of 22.69 at pH 7.0 (94.81st percentile) and a median structural flexibility of 1.004 (60.32nd percentile). The secondary structure affinities are 30.16% for helix (33.35th percentile), 28.16% for sheet (19.1st percentile), and 24.61% for turn (18.75th percentile). The instability index is 38.21 (17.4th percentile), with an isoelectric point of 8.96 (78.23rd percentile). The protein has a length of 451 amino acids (52nd percentile) and a molecular weight of 52483.54 Da (54.47th percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |