---
layout: post
title: HNRNPR
date: 2025-01-17 16:55 CST
description: HNRNPR description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10236) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10236  | HNRNPR | ENSG00000125944 | 1p36.12 |



The gene is involved in [mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0000398) and [mRNA processing](https://amigo.geneontology.org/amigo/term/GO:0006397). It enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [mRNA binding](https://amigo.geneontology.org/amigo/term/GO:0003729), and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is active in and located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), but not in the [nucleolus](https://amigo.geneontology.org/amigo/term/GO:0005730). Additionally, it is located in the [endoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0005783) and is part of the [spliceosomal complex](https://amigo.geneontology.org/amigo/term/GO:0005681), [catalytic step 2 spliceosome](https://amigo.geneontology.org/amigo/term/GO:0071013), and [ribonucleoprotein complex](https://amigo.geneontology.org/amigo/term/GO:1990904).


The gene length is 11,040 base pairs (27.78% of all genes), the mature length is 839 base pairs (15.66% of all genes), and the primary transcript length is 5,091 base pairs (19.0% of all genes).


The gene HNRNPR (NCBI ID: 10236) has been mentioned in [10 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22HNRNPR%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning this gene was in 2005, and the middle 50% of publications occurred between 2013 and 2018.


The top 5 publications mentioning HNRNPR, ranked by their scientific influence, include "[HnRNPR-CCNB1/CENPF axis contributes to gastric cancer proliferation and metastasis.](https://pubmed.ncbi.nlm.nih.gov/31527303)" (2019) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 3.58, "[Presynaptic localization of Smn and hnRNP R in axon terminals of embryonic and postnatal mouse motoneurons.](https://pubmed.ncbi.nlm.nih.gov/25338097)" (2014) with an RCR of 1.36, "[HNRNPR Regulates the Expression of Classical and Nonclassical MHC Class I Proteins.](https://pubmed.ncbi.nlm.nih.gov/27194785)" (2016) with an RCR of 1.32, "[hnRNP R and its main interactor, the noncoding RNA 7SK, coregulate the axonal transcriptome of motoneurons.](https://pubmed.ncbi.nlm.nih.gov/29507242)" (2018) with an RCR of 1.02, and "[Loss of full-length hnRNP R isoform impairs DNA damage response in motoneurons by inhibiting Yb1 recruitment to chromatin.](https://pubmed.ncbi.nlm.nih.gov/34850154)" (2021) with an RCR of 0.52. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[HNRNPR](https://www.proteinatlas.org/ENSG00000125944-HNRNPR) is an RNA-binding ribonucleoprotein with a molecular function in RNA-binding. It is implicated in intellectual disability as a disease variant. Evidence for its function is available at the protein level. The gene is detected in all RNA tissue distributions and is primarily localized in the nucleoplasm. It is part of various non-specific expression clusters, including Cluster 33 for blood (ATP binding), Cluster 16 for tissue (transcription), Cluster 55 for brain (transcription), Cluster 37 for cell line (mRNA processing), and Cluster 39 for single cell (basic cellular processes).


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313), [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025), [KLF5](https://www.ncbi.nlm.nih.gov/gene/688), and [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), each shown to be regulating in 6 experiments.





The gene is expressed in various reproductive tissues, including the ovary, uterus, fallopian tube, and cervix uteri. Additionally, it is expressed in several immune and blood cell types, such as B lymphoblasts, CD34+ cells, CD8+ T cells, and specific leukemia cell lines, including promyelocytic HL-60 and lymphoblastic MOLT-4. The gene is also expressed in CD105+ endothelial cells.


The proteins are involved in several key processes, including gene expression, processing of capped intron-containing pre-mRNA, and mRNA splicing, with a particular focus on the major pathway of mRNA splicing.


The mouse ortholog gene 74326 and its human ortholog 10236 are associated with several phenotypic traits in both sexes, including decreased mean corpuscular hemoglobin concentration, increased hematocrit, increased leukocyte cell number, increased lymphocyte cell number, and increased mean corpuscular volume.


The protein sequence analyzed has a GRAVY value of -0.933 (6.16th percentile), indicating a hydrophilic nature. It exhibits a charge of 3.34 at pH 7.0 (62.31st percentile) and a median structural flexibility of 1.014 (89.97th percentile). The protein's secondary structure is predicted to be 28.6% helix (20.81st percentile), 28.4% sheet (20.38th percentile), and 33.2% turn (82.76th percentile). The instability index is 38.67 (18.6th percentile), with an isoelectric point of 8.23 (61.61st percentile). The protein has a length of 633 amino acids (71.02nd percentile) and a molecular weight of 70942.2 Da (71.18th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |