---
layout: post
title: ORC5
date: 2025-01-17 16:55 CST
description: ORC5 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5001) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5001  | ORC5 | ENSG00000164815 | 7q22.1-q22.2 |



The gene enables [nucleotide binding](https://amigo.geneontology.org/amigo/term/GO:0000166) and [DNA replication origin binding](https://amigo.geneontology.org/amigo/term/GO:0003688), and is involved in [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524). It is located in the [chromosome, telomeric region](https://amigo.geneontology.org/amigo/term/GO:0000781), [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). The gene is part of [chromatin](https://amigo.geneontology.org/amigo/term/GO:0000785), the [origin recognition complex](https://amigo.geneontology.org/amigo/term/GO:0000808), and the [nuclear origin of replication recognition complex](https://amigo.geneontology.org/amigo/term/GO:0005664). It plays a role in [DNA replication](https://amigo.geneontology.org/amigo/term/GO:0006260), [DNA replication initiation](https://amigo.geneontology.org/amigo/term/GO:0006270), and the [regulation of DNA replication](https://amigo.geneontology.org/amigo/term/GO:0006275).


The gene length is 20,109 base pairs (42.2% of all genes), the mature length is 918 base pairs, and the primary transcript length is 14,780 base pairs.


The gene ORC5 (NCBI ID: 5001) has been mentioned in [5 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ORC5%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1998 and the middle 50% of publications occurring between 2016 and 2020. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning ORC5, ranked by their relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows. The most influential publication is "[ORC5L, a new member of the human origin recognition complex, is deleted in uterine leiomyomas and malignant myeloid diseases.](https://pubmed.ncbi.nlm.nih.gov/9765232)" (1998) (relative citation ratio: 1.03). Other notable publications include "[A human cancer cell line initiates DNA replication normally in the absence of ORC5 and ORC2 proteins.](https://pubmed.ncbi.nlm.nih.gov/32989049)" (2020) (relative citation ratio: 0.68), "[A Novel Rrm3 Function in Restricting DNA Replication via an Orc5-Binding Domain Is Genetically Separable from Rrm3 Function as an ATPase/Helicase in Facilitating Fork Progression.](https://pubmed.ncbi.nlm.nih.gov/27923055)" (2016) (relative citation ratio: 0.44), "[PSMC2, ORC5 and KRTDAP are specific biomarkers for HPV-negative head and neck squamous cell carcinoma.](https://pubmed.ncbi.nlm.nih.gov/33732365)" (2021) (relative citation ratio: 0.41), and "[Orc5 induces large-scale chromatin decondensation in a GCN5-dependent manner.](https://pubmed.ncbi.nlm.nih.gov/26644179)" (2016) (relative citation ratio: 0.26). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ORC5](https://www.proteinatlas.org/ENSG00000164815-ORC5) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in the nucleoplasm and cytosol. ORC5 is part of Cluster 3 in blood expression, indicating involvement in nuclear processes, and Cluster 78 in tissue expression, suggesting a role in testis-related transcription. In the brain, it belongs to Cluster 31, associated with non-specific transcription processes. Additionally, ORC5 is part of Cluster 39 in cell line expression and Cluster 44 in single-cell expression, which is linked to oocytes with an unknown function.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [SMAD2](https://www.ncbi.nlm.nih.gov/gene/4087) with 5 experiments, [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599) with 4 experiments, [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 4 experiments, [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227) with 4 experiments, and [IKZF1](https://www.ncbi.nlm.nih.gov/gene/10320) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The input data indicates associations with various disease conditions, including neurodegenerative diseases such as amyotrophic lateral sclerosis and motor neuron disease. Additionally, there are links to mental health disorders, specifically substance-related disorders like substance dependence and alcohol dependence. Furthermore, the data highlights integumentary system diseases, particularly skin diseases and angioedema.



The gene is expressed in various tissues, including nerve, skin, and prostate. Additionally, it is expressed in specific cell types and tissues such as chronic myelogenous leukemia (K-562), Burkitt's lymphoma (Daudi), appendix, superior cervical ganglion, trigeminal ganglion, atrioventricular node, and ciliary ganglion.


The input data highlights various pathways involved in DNA replication and cell cycle regulation. These include the assembly and activation of the pre-replicative complex, association of licensing factors with the pre-replicative complex, and the regulation of DNA replication. Additionally, the data covers processes such as the removal of Orc1 from chromatin, switching of origins to a post-replicative state, and the removal of licensing factors from origins. The pathways also encompass broader cell cycle phases and checkpoints, including the M/G1 transition, S phase, G1/S transition, and G2/M checkpoints. Furthermore, the data includes the activation of ATR in response to replication stress and E2F-mediated regulation of DNA replication.



The protein sequence analyzed has a GRAVY value of -0.132 (75.63rd percentile), indicating a relatively hydrophilic nature. It exhibits a charge of 1.942 at pH 7.0 (57.34th percentile) and a median structural flexibility of 0.992 (14.82nd percentile). The protein's secondary structure is predicted to be 32.41% helix (54.3rd percentile), 43.91% sheet (93.38th percentile), and 23.68% turn (13.31st percentile). The instability index is 37.09 (14.41st percentile), with an isoelectric point of 7.63 (55.55th percentile). The protein has a length of 435 amino acids (50.07th percentile) and a molecular weight of 50282.46 Da (52nd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |