---
layout: post
title: EIF3A
date: 2024-12-08 23:57 CST
description: EIF3A description
tags: [cooccuring-genes]
categories: longest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8661) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8661  | EIF3A | ENSG00000107581 | 10q26.11 |



The gene length is 832,793 base pairs (99.39% of all genes), the mature length is 3,326 base pairs, and the primary transcript length is 576,951 base pairs (99.47% of all genes).


EIF3A has been studied in 166 publications, with the earliest publication dating back to 1973 and the middle 50% of publications occurring between 2001 and 2016. For more information, see the [166 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22EIF3A%22) (note that the count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)).


The top five publications mentioning EIF3A, ranked by their scientific influence, include "[A subcomplex of the proteasome regulatory particle required for ubiquitin-conjugate degradation and related to the COP9-signalosome and eIF3.](https://pubmed.ncbi.nlm.nih.gov/9741626)" (1998) (relative citation ratio: 12.27), "[eIF3 targets cell-proliferation messenger RNAs for translational activation or repression.](https://pubmed.ncbi.nlm.nih.gov/25849773)" (2015) (relative citation ratio: 7.95), "[eIF3: a versatile scaffold for translation initiation complexes.](https://pubmed.ncbi.nlm.nih.gov/16920360)" (2006) (relative citation ratio: 5.38), "[Molecular architecture of the 40S⋅eIF1⋅eIF3 translation initiation complex.](https://pubmed.ncbi.nlm.nih.gov/25171412)" (2014) (relative citation ratio: 4.67), and "[Hormonal modulation of HER-2/neu protooncogene messenger ribonucleic acid and p185 protein expression in human breast cancer cell lines.](https://pubmed.ncbi.nlm.nih.gov/1972345)" (1990) (relative citation ratio: 4.1). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [JUN](https://www.ncbi.nlm.nih.gov/gene/3725) with 6 experiments, [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908) and [HIF1A](https://www.ncbi.nlm.nih.gov/gene/3091) each with 5 experiments, and [SIN3A](https://www.ncbi.nlm.nih.gov/gene/25942) and [CLOCK](https://www.ncbi.nlm.nih.gov/gene/9575) each with 4 experiments.











The analyzed protein sequence has a GRAVY value of -1.489 (0.64 percentile), indicating a hydrophilic nature. The charge at pH 7 is -7.015 (28.4 percentile), and the median structural flexibility is 1.021 (97.31 percentile). The protein has a high instability index of 62.858 (87.61 percentile) and an isoelectric point of 6.383 (39.94 percentile). The sequence length is 1382 amino acids (94.05 percentile) with a molecular weight of 166567.41 Da (95.12 percentile). The secondary structure affinities are 34.73% for helix (73.02 percentile), 23.66% for sheet (5.87 percentile), and 26.77% for turn (36.07 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |