---
layout: post
title: PDK1
date: 2025-01-17 16:55 CST
description: PDK1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5163) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5163  | PDK1 | ENSG00000152256 | 2q31.1 |



The gene enables [protein kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004672) and [pyruvate dehydrogenase (acetyl-transferring) kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004740). It is involved in [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524). The gene is active in the [mitochondrion](https://amigo.geneontology.org/amigo/term/GO:0005739) and located in the [mitochondrial matrix](https://amigo.geneontology.org/amigo/term/GO:0005759), as part of the [mitochondrial pyruvate dehydrogenase complex](https://amigo.geneontology.org/amigo/term/GO:0005967). It plays roles in various processes, including the [glucose metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006006), [protein phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0006468), [cell population proliferation](https://amigo.geneontology.org/amigo/term/GO:0008283), the [intrinsic apoptotic signaling pathway in response to oxidative stress](https://amigo.geneontology.org/amigo/term/GO:0008631), [regulation of acetyl-CoA biosynthetic process from pyruvate](https://amigo.geneontology.org/amigo/term/GO:0010510), [regulation of glucose metabolic process](https://amigo.geneontology.org/amigo/term/GO:0010906), [phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0016310), and the [hypoxia-inducible factor-1alpha signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0097411).


The gene length is 24,176 base pairs (46.88th percentile), the mature length is 6,987 base pairs (96.44th percentile), and the primary transcript length is 24,145 base pairs (56.94th percentile).


The gene PDK1 (NCBI ID: 5163) has been mentioned in [392 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PDK1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning PDK1 is 1992, and the middle 50% of publications occurred between 2009 and 2019.


The top five publications mentioning PDK1, ranked by their scientific influence, include "[The PI3K-PDK1 connection: more than just a road to PKB.](https://pubmed.ncbi.nlm.nih.gov/10698680)" (2000) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 20.96, "[Protein kinase C isotypes controlled by phosphoinositide 3-kinase through the protein kinase PDK1.](https://pubmed.ncbi.nlm.nih.gov/9748166)" (1998) with an RCR of 17.87, "[Phosphorylation and activation of p70s6k by PDK1.](https://pubmed.ncbi.nlm.nih.gov/9445476)" (1998) with an RCR of 12.94, "[PDK1-Dependent Metabolic Reprogramming Dictates Metastatic Potential in Breast Cancer.](https://pubmed.ncbi.nlm.nih.gov/26365179)" (2015) with an RCR of 11.75, and "[3-Phosphoinositide-dependent protein kinase-1 (PDK1): structural and functional homology with the Drosophila DSTPK61 kinase.](https://pubmed.ncbi.nlm.nih.gov/9368760)" (1997) with an RCR of 10.88. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PDK1](https://www.proteinatlas.org/ENSG00000152256-PDK1) is a kinase and transferase with evidence at the protein level. It is an FDA-approved drug target and is detected in all tissues. The gene is expressed in various clusters, including Cluster 22 in eosinophils, Cluster 16 in non-specific tissues, Cluster 2 in neurons, Cluster 51 in myeloma cell lines, and Cluster 77 in plasma cells.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), [RELA](https://www.ncbi.nlm.nih.gov/gene/5970), [GABPA](https://www.ncbi.nlm.nih.gov/gene/2551), [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), and [TRIM28](https://www.ncbi.nlm.nih.gov/gene/10155), each shown to be regulating in 5 experiments.



The GWAS data indicates associations with a range of disease conditions, including coronary artery disease, artery disease, vascular disease, cardiovascular system disease, nervous system disease, central nervous system disease, neurodegenerative disease, Alzheimer's disease, tauopathy, dementia, cognitive disorder, disease of mental health, and cerebrovascular disease.



The gene is expressed in various tissues and cell types, including ovary and blood, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in 721_B_lymphoblasts, CD4+ T cells, CD8+ T cells, Leukemia promyelocytic-HL-60, Smooth Muscle, Bronchial Epithelial Cells, and Leukemia lymphoblastic (MOLT-4).


The input data highlights several key pathways involved in cellular processes, including metabolism, pyruvate metabolism, and the citric acid (TCA) cycle. Specific pathways mentioned are pyruvate metabolism, regulation of the pyruvate dehydrogenase (PDH) complex, and the TCA cycle along with respiratory electron transport. Additionally, signaling pathways such as those involving retinoic acid and general signal transduction are also noted.


The mouse gene with NCBI ID 228026 and its human ortholog with NCBI ID 5163 are associated with abnormal testis morphology and small testis phenotypes in male mice.


The analyzed protein sequence has a GRAVY value of -0.334 (53.95th percentile), indicating a hydrophilic nature. It exhibits a charge of 6.47 at pH 7.0 (72.54th percentile) and a median structural flexibility of 0.998 (31.72nd percentile). The protein's secondary structure is predicted to be 29.13% helix (24.51st percentile), 35.09% sheet (59.83rd percentile), and 31.42% turn (73.3rd percentile). The instability index is 51.35 (61.54th percentile), with an isoelectric point of 8.92 (77th percentile). The protein has a length of 436 amino acids (50.2nd percentile) and a molecular weight of 49243.63 Da (50.82nd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |