---
layout: post
title: MYO1E
date: 2024-12-08 23:57 CST
description: MYO1E description
tags: [cooccuring-genes]
categories: longest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4643) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4643  | MYO1E | ENSG00000157483 | 15q22.2 |



The gene length is 588,016 base pairs (98.85th percentile), the mature length is 4,371 base pairs (86.8th percentile), and the primary transcript length is 586,287 base pairs (99.51st percentile).


The gene MYO1E, with NCBI ID 4643, has been the subject of [18 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22MYO1E%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 2002 and the middle 50% of publications occurring between 2011 and 2015. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning MYO1E, ranked by their scientific influence, include "[MYO1E mutations and childhood familial focal segmental glomerulosclerosis.](https://pubmed.ncbi.nlm.nih.gov/21756023)" (2011) (relative citation ratio: 4.53), "[Myosin 1E coordinates actin assembly and cargo trafficking during clathrin-mediated endocytosis.](https://pubmed.ncbi.nlm.nih.gov/22675027)" (2012) (relative citation ratio: 1.78), "[Exome sequencing identified MYO1E and NEIL1 as candidate genes for human autosomal recessive steroid-resistant nephrotic syndrome.](https://pubmed.ncbi.nlm.nih.gov/21697813)" (2011) (relative citation ratio: 1.26), "[Coinheritance of COL4A5 and MYO1E mutations accentuate the severity of kidney disease.](https://pubmed.ncbi.nlm.nih.gov/25739341)" (2015) (relative citation ratio: 1.22), and "[The kinetic mechanism of Myo1e (human myosin-IC).](https://pubmed.ncbi.nlm.nih.gov/11940582)" (2002) (relative citation ratio: 0.94). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2B](https://www.ncbi.nlm.nih.gov/gene/9757) with 7 experiments, [MAX](https://www.ncbi.nlm.nih.gov/gene/4149) with 6 experiments, [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861) with 5 experiments, [SMC3](https://www.ncbi.nlm.nih.gov/gene/9126) with 5 experiments, and [RELA](https://www.ncbi.nlm.nih.gov/gene/5970) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.











The analyzed protein sequence has a GRAVY value of -0.5714801444043316 (27.91% percentile), indicating a hydrophilic nature. The charge at pH 7.0 is 20.721944486015758 (93.92% percentile), and the median structural flexibility is 1.0042857142857142 (60.92% percentile). The protein has a length of 1108 amino acids (90.06% percentile) and a molecular weight of 127060.34900000053 Da (90.63% percentile). The secondary structure affinities are 0.3068592057761733 for helix (37.98% percentile), 0.3429602888086642 for sheet (54.01% percentile), and 0.2833935018050542 for turn (50.35% percentile). The instability index is 47.42148916967535 (48.04% percentile), and the isoelectric point is 9.01314754486084 (79.58% percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |