---
layout: post
title: G3BP2
date: 2025-01-17 16:55 CST
description: G3BP2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9908) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9908  | G3BP2 | ENSG00000138757 | 4q21.1 |



The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [mRNA binding](https://amigo.geneontology.org/amigo/term/GO:0003729), and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737) and is active in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). The gene is involved in various processes, including [cytoplasmic sequestering of NF-kappaB](https://amigo.geneontology.org/amigo/term/GO:0007253), [Ras protein signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007265), [stress granule assembly](https://amigo.geneontology.org/amigo/term/GO:0034063), [innate immune response](https://amigo.geneontology.org/amigo/term/GO:0045087), [mRNA transport](https://amigo.geneontology.org/amigo/term/GO:0051028), [protein homooligomerization](https://amigo.geneontology.org/amigo/term/GO:0051260), and [positive regulation of stress granule assembly](https://amigo.geneontology.org/amigo/term/GO:0062029). Additionally, it enables [signaling receptor complex adaptor activity](https://amigo.geneontology.org/amigo/term/GO:0030159) and [molecular condensate scaffold activity](https://amigo.geneontology.org/amigo/term/GO:0140693). The gene is also part of the [ribonucleoprotein complex](https://amigo.geneontology.org/amigo/term/GO:1990904).


The gene length is 24,635 base pairs (47.37th percentile), the mature length is 2,393 base pairs (61.82nd percentile), and the primary transcript length is 22,016.5 base pairs (54.27th percentile).


The gene G3BP2 (NCBI ID: 9908) has been mentioned in [21 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22G3BP2%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 2010 and the middle 50% of publications occurring between 2015 and 2021. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning G3BP2, ranked by their scientific influence, include "[Matrix stiffness drives epithelial-mesenchymal transition and tumour metastasis through a TWIST1-G3BP2 mechanotransduction pathway.](https://pubmed.ncbi.nlm.nih.gov/25893917)" (2015) with a relative citation ratio of 19.69, "[G3BP1, G3BP2 and CAPRIN1 are required for translation of interferon stimulated mRNAs and are targeted by a dengue virus non-coding RNA.](https://pubmed.ncbi.nlm.nih.gov/24992036)" (2014) with a relative citation ratio of 5.7, "[Both G3BP1 and G3BP2 contribute to stress granule formation.](https://pubmed.ncbi.nlm.nih.gov/23279204)" (2013) with a relative citation ratio of 5.12, "[Invasion-related circular RNA circFNDC3B inhibits bladder cancer progression through the miR-1178-3p/G3BP2/SRC/FAK axis.](https://pubmed.ncbi.nlm.nih.gov/30458784)" (2018) with a relative citation ratio of 5.08, and "[Stress granule components G3BP1 and G3BP2 play a proviral role early in Chikungunya virus replication.](https://pubmed.ncbi.nlm.nih.gov/25653451)" (2015) with a relative citation ratio of 4.03. The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[G3BP2](https://www.proteinatlas.org/ENSG00000138757-G3BP2) is an RNA-binding protein with evidence at the protein level. It is detected in all tissues and primarily localized in the cytosol. The gene is expressed in various clusters, including Cluster 5 for blood expression related to cell proliferation, Cluster 35 for tissue expression related to mitochondria, Cluster 14 for brain expression related to neurons and synaptic function, Cluster 22 for cell line expression related to mRNA splicing and cell cycle, and Cluster 78 for single cell expression related to NK-cells, T-cells, the immune system, and transcription.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025) and [TP53](https://www.ncbi.nlm.nih.gov/gene/7157), each with 6 experiments. Additionally, [TCF12](https://www.ncbi.nlm.nih.gov/gene/6938), [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227), and [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) were each found to be regulating in 5 experiments.





The gene is expressed in various tissues, including muscle and brain, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in 721_B_lymphoblasts, CD8+ T cells, Occipital Lobe, Bronchial Epithelial Cells, Pancreatic Islet, and colon.



The mouse ortholog gene 23881 and its human ortholog gene 9908 exhibit complete penetrance preweaning lethality in both sexes.


The analyzed protein sequence has a GRAVY value of -0.9761410788381734 (4.94th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -16.220884587952497 (14.26th percentile), and the median structural flexibility is 1.0170714285714284 (94.39th percentile). The protein has a helix affinity of 0.2800829875518672 (17.02nd percentile), a sheet affinity of 0.2572614107883817 (10.53rd percentile), and a turn affinity of 0.35684647302904565 (91.06th percentile). The instability index is 56.56248962655626 (76.04th percentile), with an isoelectric point of 5.406036186218262 (15.98th percentile). The protein has a length of 482 amino acids (56.14th percentile) and a molecular weight of 54120.48500000007 Da (56.21st percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |