---
layout: post
title: PTBP3
date: 2025-01-17 16:55 CST
description: PTBP3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9991) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9991  | PTBP3 | ENSG00000119314 | 9q32  |



The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and [mRNA binding](https://amigo.geneontology.org/amigo/term/GO:0003729), is active in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), and is involved in various processes including [mRNA processing](https://amigo.geneontology.org/amigo/term/GO:0006397), [RNA splicing](https://amigo.geneontology.org/amigo/term/GO:0008380), [anatomical structure morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0009653), [negative regulation of RNA splicing](https://amigo.geneontology.org/amigo/term/GO:0033119), [erythrocyte maturation](https://amigo.geneontology.org/amigo/term/GO:0043249), [regulation of RNA splicing](https://amigo.geneontology.org/amigo/term/GO:0043484), and [regulation of cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0045595).


The gene length is 2,994 base pairs (8.15% of all genes), the mature length is 1,091 base pairs (25.61% of all genes), and the primary transcript length is 1,874 base pairs (5.96% of all genes).


The gene PTBP3 (NCBI ID: 9991) has been mentioned in [24 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PTBP3%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning this gene was in 2009, and the middle 50% of publications occurred between 2015 and 2020.


The top 5 publications mentioning PTBP3, ranked by their scientific influence, include "[LncRNA BCRT1 promotes breast cancer progression by targeting miR-1303/PTBP3 axis.](https://pubmed.ncbi.nlm.nih.gov/32384893)" (2020) (relative citation ratio: 19.65), "[PTBP3-Mediated Regulation of ZEB1 mRNA Stability Promotes Epithelial-Mesenchymal Transition in Breast Cancer.](https://pubmed.ncbi.nlm.nih.gov/29187406)" (2018) (relative citation ratio: 2.62), "[PTBP3 promotes malignancy and hypoxia-induced chemoresistance in pancreatic cancer cells by ATG12 up-regulation.](https://pubmed.ncbi.nlm.nih.gov/31989778)" (2020) (relative citation ratio: 2.17), "[MicroRNA-210 regulates cell proliferation and apoptosis by targeting regulator of differentiation 1 in glioblastoma cells.](https://pubmed.ncbi.nlm.nih.gov/26443314)" (2015) (relative citation ratio: 1.69), and "[PTBP3 modulates P53 expression and promotes colorectal cancer cell proliferation by maintaining UBE4A mRNA stability.](https://pubmed.ncbi.nlm.nih.gov/35136024)" (2022) (relative citation ratio: 1.53). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PTBP3](https://www.proteinatlas.org/ENSG00000119314-PTBP3) is an RNA-binding repressor with evidence at the protein level. It is detected in all tissues and primarily localized in the nucleoplasm. PTBP3 is expressed in neutrophils, lymphoid tissue, bone marrow, and neurons, where it is involved in cell proliferation, protein binding, and mixed functions. In single-cell expression analysis, it is associated with neutrophil degranulation.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [SMAD3](https://www.ncbi.nlm.nih.gov/gene/4088), [KDM2B](https://www.ncbi.nlm.nih.gov/gene/84678), [MAX](https://www.ncbi.nlm.nih.gov/gene/4149), [JUN](https://www.ncbi.nlm.nih.gov/gene/3725), and [GATA2](https://www.ncbi.nlm.nih.gov/gene/2624), each shown to be regulating in 6 experiments.



The GWAS data indicates associations with a range of conditions, including leukocyte disease, leukopenia, hematopoietic system disease, immune system disease, agranulocytosis, neutropenia, bone inflammation disease, arthritis, rheumatoid arthritis, bone disease, connective tissue disease, musculoskeletal system disease, and osteoarthritis.



The gene is expressed in various immune cell types, including CD33+ myeloid cells, BDCA4+ dendritic cells, and CD56+ NK cells. Additionally, it is expressed in several neural tissues such as the superior cervical ganglion, atrioventricular node, and ciliary ganglion, as well as in the testis interstitial tissue.




The analyzed protein sequence has a GRAVY value of -0.0998 (78.09th percentile), indicating a hydrophilic nature. It exhibits a charge of 9.79 at pH 7.0 (81.01st percentile) and a median structural flexibility of 0.9996 (39.53rd percentile). The protein's secondary structure is predicted to be 30.80% helix (38.99th percentile), 35.51% sheet (62.84th percentile), and 32.61% turn (79.91st percentile). The instability index is 38.88 (19.11th percentile), with an isoelectric point of 9.13 (82.62nd percentile). The protein has a length of 552 amino acids (64.29th percentile) and a molecular weight of 59688.83 Da (62.34th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |