---
layout: post
title: ELAVL1
date: 2025-01-17 16:55 CST
description: ELAVL1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1994) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1994  | ELAVL1 | ENSG00000066044 | 19p13.2 |



The gene is located in the [P-body](https://amigo.geneontology.org/amigo/term/GO:0000932), [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [endoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0005783), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [cytoplasmic stress granule](https://amigo.geneontology.org/amigo/term/GO:0010494), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [sarcoplasm](https://amigo.geneontology.org/amigo/term/GO:0016528), [cytoplasmic vesicle](https://amigo.geneontology.org/amigo/term/GO:0031410), [postsynapse](https://amigo.geneontology.org/amigo/term/GO:0098794), and [glutamatergic synapse](https://amigo.geneontology.org/amigo/term/GO:0098978). It enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [double-stranded RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003725), [mRNA binding](https://amigo.geneontology.org/amigo/term/GO:0003729), [mRNA 3'-UTR binding](https://amigo.geneontology.org/amigo/term/GO:0003730), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [protein kinase binding](https://amigo.geneontology.org/amigo/term/GO:0019901), [miRNA binding](https://amigo.geneontology.org/amigo/term/GO:0035198), [mRNA 3'-UTR AU-rich region binding](https://amigo.geneontology.org/amigo/term/GO:0035925), [protein homodimerization activity](https://amigo.geneontology.org/amigo/term/GO:0042803), and [lncRNA binding](https://amigo.geneontology.org/amigo/term/GO:0106222). It is involved in [protein import into nucleus](https://amigo.geneontology.org/amigo/term/GO:0006606), [post-transcriptional gene silencing](https://amigo.geneontology.org/amigo/term/GO:0016441), [positive regulation of superoxide anion generation](https://amigo.geneontology.org/amigo/term/GO:0032930), [positive regulation of translation](https://amigo.geneontology.org/amigo/term/GO:0045727), [mRNA stabilization](https://amigo.geneontology.org/amigo/term/GO:0048255), [protein homooligomerization](https://amigo.geneontology.org/amigo/term/GO:0051260), [negative regulation of miRNA-mediated gene silencing](https://amigo.geneontology.org/amigo/term/GO:0060965), [mRNA destabilization](https://amigo.geneontology.org/amigo/term/GO:0061157), [3'-UTR-mediated mRNA stabilization](https://amigo.geneontology.org/amigo/term/GO:0070935), and [regulation of stem cell population maintenance](https://amigo.geneontology.org/amigo/term/GO:2000036). Additionally, it is part of the [ribonucleoprotein complex](https://amigo.geneontology.org/amigo/term/GO:1990904).


The gene length is 28,866 base pairs (51.68% of all genes), the mature length is 726.5 base pairs (10.37% of all genes), and the primary transcript length is 24,168 base pairs (56.98% of all genes).


ELAVL1 (Gene ID: 1994) has been mentioned in [690 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ELAVL1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1991. The middle 50% of publications occurred between 2010 and 2019. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning ELAVL1, ranked by their scientific influence, include "[Identification of HuR target circular RNAs uncovers suppression of PABPN1 translation by CircPABPN1.](https://pubmed.ncbi.nlm.nih.gov/28080204)" (2017) (relative citation ratio: 22.56), "[WTAP facilitates progression of hepatocellular carcinoma via m6A-HuR-dependent epigenetic silencing of ETS1.](https://pubmed.ncbi.nlm.nih.gov/31438961)" (2019) (relative citation ratio: 17.23), "[Exosomal circSHKBP1 promotes gastric cancer progression via regulating the miR-582-3p/HUR/VEGF axis and suppressing HSP90 degradation.](https://pubmed.ncbi.nlm.nih.gov/32600329)" (2020) (relative citation ratio: 15.32), "[HuR and mRNA stability.](https://pubmed.ncbi.nlm.nih.gov/11289308)" (2001) (relative citation ratio: 14.89), and "[Hypoxia-induced HIF-1Î±/lncRNA-PMAN inhibits ferroptosis by promoting the cytoplasmic translocation of ELAVL1 in peritoneal dissemination from gastric cancer.](https://pubmed.ncbi.nlm.nih.gov/35447413)" (2022) (relative citation ratio: 13.24). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ELAVL1](https://www.proteinatlas.org/ENSG00000066044-ELAVL1) is an RNA-binding protein with evidence at the protein level. It is detected in all tissues and is primarily localized in the nucleoplasm, nucleoli, and cytosol. ELAVL1 is expressed in various clusters, including Cluster 45 in eosinophils with unknown function, Cluster 4 in testis related to basic cellular processes, Cluster 14 in neurons associated with synaptic function, Cluster 23 in non-specific cell lines involved in transcription, and Cluster 52 in non-specific single cells related to cell proliferation.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [STAT5B](https://www.ncbi.nlm.nih.gov/gene/6777), [BCL6](https://www.ncbi.nlm.nih.gov/gene/604), [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169), and [HIF1A](https://www.ncbi.nlm.nih.gov/gene/3091), each shown to be regulating in 5 experiments.





The gene is expressed in various tissues and cell types, including uterus, 721_B_lymphoblasts, CD34+ cells, Lymphoma burkitts (Daudi), CD105+ endothelial cells, Leukemia lymphoblastic (MOLT-4), and testis.


The input data highlights several key processes involving gene expression. Specifically, it includes the regulation of mRNA stability by proteins that bind AU-rich elements and the role of HuR in stabilizing mRNA.


The mouse ortholog gene 15568 and its human ortholog gene 1994 are associated with several phenotypic abnormalities observed in both sexes, including abnormal embryo size, abnormal limb morphology, anophthalmia, edema, embryonic growth retardation, impaired pupillary reflex, and complete penetrance preweaning lethality.


The analyzed protein sequence has a GRAVY value of -0.356 (51.47th percentile), indicating a hydrophilic nature. It exhibits a charge of 7.2 at pH 7.0 (74.97th percentile) and a median structural flexibility of 1.001 (46.42nd percentile). The protein's secondary structure is predicted to be 27.0% helix (11.7th percentile), 33.4% sheet (48.18th percentile), and 33.1% turn (82.49th percentile). The instability index is 36.0 (12.09th percentile), with an isoelectric point of 9.23 (85.33rd percentile). The protein has a length of 326 amino acids (34.01st percentile) and a molecular weight of 36091.46 Da (33.7th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |