---
layout: post
title: MAL
date: 2025-01-17 16:55 CST
description: MAL description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4118) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4118  | MAL | ENSG00000172005 | 2q11.1 |



The gene is involved in various biological processes, including [membrane raft polarization](https://amigo.geneontology.org/amigo/term/GO:0001766), [protein localization to the paranode region of axon](https://amigo.geneontology.org/amigo/term/GO:0002175), [apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0006915), [central nervous system development](https://amigo.geneontology.org/amigo/term/GO:0007417), [central nervous system myelination](https://amigo.geneontology.org/amigo/term/GO:0022010), [cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0030154), and [myelination](https://amigo.geneontology.org/amigo/term/GO:0042552). It enables functions such as [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [lipid binding](https://amigo.geneontology.org/amigo/term/GO:0008289), [peptidase activator activity involved in apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0016505), and serves as a [structural constituent of myelin sheath](https://amigo.geneontology.org/amigo/term/GO:0019911). The gene is located in several cellular components, including the [endoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0005783), [apical plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0016324), [Schmidt-Lanterman incisure](https://amigo.geneontology.org/amigo/term/GO:0043220), [plasma membrane raft](https://amigo.geneontology.org/amigo/term/GO:0044853), [membrane raft](https://amigo.geneontology.org/amigo/term/GO:0045121), and the [hinge region between urothelial plaques of apical plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0120003). Additionally, it is involved in [apical protein localization](https://amigo.geneontology.org/amigo/term/GO:0045176), [protein insertion into plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0098737), and [positive regulation of extrinsic apoptotic signaling pathway via death domain receptors](https://amigo.geneontology.org/amigo/term/GO:1902043).


The gene length is 2,225 base pairs (5.99% of all genes), the mature length is 959 base pairs, and the primary transcript length is 2,209 base pairs.


The gene MAL (NCBI ID: 4118) has been mentioned in [217 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22MAL%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1821 and the middle 50% of publications occurring between 2006 and 2017. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning MAL, ranked by their scientific influence, include "[Pathogen-specific burdens of community diarrhoea in developing countries: a multisite birth cohort study (MAL-ED).](https://pubmed.ncbi.nlm.nih.gov/26202075)" (2015) (relative citation ratio: 28.99), "[Mal (MyD88-adapter-like) is required for Toll-like receptor-4 signal transduction.](https://pubmed.ncbi.nlm.nih.gov/11544529)" (2001) (relative citation ratio: 16.97), "[Use of quantitative molecular diagnostic methods to investigate the effect of enteropathogen infections on linear growth in children in low-resource settings: longitudinal analysis of results from the MAL-ED cohort study.](https://pubmed.ncbi.nlm.nih.gov/30287125)" (2018) (relative citation ratio: 13.68), "[Suppressor of cytokine signaling 1 negatively regulates Toll-like receptor signaling by mediating Mal degradation.](https://pubmed.ncbi.nlm.nih.gov/16415872)" (2006) (relative citation ratio: 8.14), and "[Assessment of environmental enteropathy in the MAL-ED cohort study: theoretical and analytic framework.](https://pubmed.ncbi.nlm.nih.gov/25305293)" (2014) (relative citation ratio: 5.11). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[MAL](https://www.proteinatlas.org/ENSG00000172005-MAL) is a gene with evidence at the protein level, detected in many tissues. It is primarily localized in the Golgi apparatus, centrosome, and basal body. The gene is expressed in various clusters, including T-cells involved in T-cell receptor signaling, squamous epithelium related to innate immune response, white matter associated with signal transduction, kidney cancer membrane components, and suprabasal keratinocytes involved in cornification.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 6 experiments, [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227) with 3 experiments, [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 3 experiments, [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) with 3 experiments, and [EZH2](https://www.ncbi.nlm.nih.gov/gene/2146) with 3 experiments.



The GWAS data indicates associations with various metabolic and nutritional conditions, including diseases of metabolism, acquired metabolic diseases, nutrition diseases, overnutrition, and obesity.



The gene is expressed in the esophagus and vagina according to BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles indicate expression in the kidney, tongue, olfactory bulb, thymus, and leukemia lymphoblastic (MOLT-4) cells.




The protein sequence analyzed has a GRAVY value of 1.0124 (99.91st percentile), indicating high hydrophobicity. It exhibits a charge of -4.08 at pH 7.0 (36.32nd percentile) and a median structural flexibility of 0.968 (0.09th percentile). The protein's secondary structure affinities are 28.76% for helix (21.83rd percentile), 54.90% for sheet (99.87th percentile), and 20.92% for turn (3.73rd percentile). The instability index is 37.66 (15.78th percentile), with an isoelectric point of 5.55 (19.52nd percentile). The sequence length is 153 amino acids (8.43rd percentile), and the molecular weight is 16713.35 Da (8.02nd percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |