---
layout: post
title: CLDN5
date: 2025-01-17 16:55 CST
description: CLDN5 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7122) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7122  | CLDN5 | ENSG00000184113 | 22q11.21 |



The gene is involved in various biological processes, including [outflow tract morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0003151), [cell-cell junction assembly](https://amigo.geneontology.org/amigo/term/GO:0007043), [cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0007155), and [transforming growth factor beta receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007179). It plays a role in [learning](https://amigo.geneontology.org/amigo/term/GO:0007612), [positive regulation of cell population proliferation](https://amigo.geneontology.org/amigo/term/GO:0008284), and both [positive](https://amigo.geneontology.org/amigo/term/GO:0010628) and [negative regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0010629). The gene is located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [cell-cell junction](https://amigo.geneontology.org/amigo/term/GO:0005911), [bicellular tight junction](https://amigo.geneontology.org/amigo/term/GO:0005923), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [apicolateral plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0016327), [lateral plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0016328), [cell junction](https://amigo.geneontology.org/amigo/term/GO:0030054), [paranode region of axon](https://amigo.geneontology.org/amigo/term/GO:0033270), [tight junction](https://amigo.geneontology.org/amigo/term/GO:0070160), and [Schmidt-Lanterman incisure](https://amigo.geneontology.org/amigo/term/GO:0043220). It enables [structural molecule activity](https://amigo.geneontology.org/amigo/term/GO:0005198) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is active in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886) and [bicellular tight junction](https://amigo.geneontology.org/amigo/term/GO:0005923). Additionally, the gene colocalizes with the [cortical actin cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0030864) and is involved in processes such as [calcium-independent cell-cell adhesion via plasma membrane cell-adhesion molecules](https://amigo.geneontology.org/amigo/term/GO:0016338), [negative regulation of angiogenesis](https://amigo.geneontology.org/amigo/term/GO:0016525), [negative regulation of cell migration](https://amigo.geneontology.org/amigo/term/GO:0030336), [positive regulation of protein binding](https://amigo.geneontology.org/amigo/term/GO:0032092), [maintenance of blood-brain barrier](https://amigo.geneontology.org/amigo/term/GO:0035633), [myelination](https://amigo.geneontology.org/amigo/term/GO:0042552), [negative regulation of vascular permeability](https://amigo.geneontology.org/amigo/term/GO:0043116), [response to ethanol](https://amigo.geneontology.org/amigo/term/GO:0045471), [roof of mouth development](https://amigo.geneontology.org/amigo/term/GO:0060021), [face morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0060325), [bicellular tight junction assembly](https://amigo.geneontology.org/amigo/term/GO:0070830), [tight junction assembly](https://amigo.geneontology.org/amigo/term/GO:0120192), [positive regulation of establishment of endothelial barrier](https://amigo.geneontology.org/amigo/term/GO:1903142), [positive regulation of bicellular tight junction assembly](https://amigo.geneontology.org/amigo/term/GO:1903348), and [establishment of blood-retinal barrier](https://amigo.geneontology.org/amigo/term/GO:1990963).


The gene length is 36,602.0 base pairs (58.09% of all genes), the mature length is 885.5 base pairs, and the primary transcript length is 27,713.5 base pairs.


The gene CLDN5 (NCBI ID: 7122) has been mentioned in [133 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CLDN5%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning this gene is 2003, and the middle 50% of publications occurred between 2011 and 2020.


The top 5 publications mentioning CLDN5, ranked by their scientific influence, are as follows. The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. The most influential publication is "[Endothelial adherens junctions control tight junctions by VE-cadherin-mediated upregulation of claudin-5.](https://pubmed.ncbi.nlm.nih.gov/18604199)" (2008) with a relative citation ratio of 10.58. This is followed by "[Dose-dependent expression of claudin-5 is a modifying factor in schizophrenia.](https://pubmed.ncbi.nlm.nih.gov/28993710)" (2018) with a relative citation ratio of 6.09. The publication "[Focusing on claudin-5: A promising candidate in the regulation of BBB to treat ischemic stroke.](https://pubmed.ncbi.nlm.nih.gov/29217457)" (2018) has a relative citation ratio of 5.0. Additionally, "[Phosphorylation of claudin-5 and occludin by rho kinase in brain endothelial cells.](https://pubmed.ncbi.nlm.nih.gov/18187566)" (2008) has a relative citation ratio of 4.26, and "[Brain DNA Methylation Patterns in CLDN5 Associated With Cognitive Decline.](https://pubmed.ncbi.nlm.nih.gov/33838873)" (2022) has a relative citation ratio of 3.99. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CLDN5](https://www.proteinatlas.org/ENSG00000184113-CLDN5) is a gene with evidence at the protein level. Its RNA is detected in many tissues, and it is expressed in various clusters, including Cluster 4 in neutrophils related to degranulation, Cluster 34 in non-specific mixed function tissues, Cluster 42 in the choroid plexus associated with mitochondria, Cluster 12 in bone cancer cell lines related to neuronal signaling, and Cluster 54 in lymphatic endothelial cells involved in lymph vessel development.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 6 experiments, [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) with 4 experiments, [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 3 experiments, [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170) with 3 experiments, and [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113) with 3 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.





The gene is expressed in multiple tissue samples, including adipose tissue, breast, and lung, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Thyroid, Lung, Appendix, Skin, Olfactory Bulb, and Fetal lung.


The proteins are involved in various pathways related to cell junction organization, cell-cell junction organization, cell-cell communication, and tight junction interactions.



The analyzed protein sequence has a GRAVY value of 0.756 (98.42 percentile), indicating a high degree of hydrophobicity. The protein exhibits a charge of 2.56 at pH 7.0 (59.53 percentile) and a median structural flexibility of 0.977 (3.11 percentile). The secondary structure affinities are notable, with 34.4% of residues tending to be in helices (70.82 percentile), 44.95% in sheets (94.48 percentile), and 22.48% in turns (7.98 percentile). The instability index is 43.94 (35.46 percentile), and the isoelectric point is 8.25 (61.94 percentile). The protein has a length of 218 amino acids (17.19 percentile) and a molecular weight of 23147.21 Da (15.53 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |