---
layout: post
title: RPS26
date: 2025-01-17 16:55 CST
description: RPS26 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6231) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6231  | RPS26 | ENSG00000197728 | 12q13.2 |



The gene is involved in [cytoplasmic translation](https://amigo.geneontology.org/amigo/term/GO:0002181) and [translation](https://amigo.geneontology.org/amigo/term/GO:0006412), and plays a role in the [negative regulation of RNA splicing](https://amigo.geneontology.org/amigo/term/GO:0033119). It enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [mRNA binding](https://amigo.geneontology.org/amigo/term/GO:0003729), and acts as a [structural constituent of ribosome](https://amigo.geneontology.org/amigo/term/GO:0003735) with [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [cadherin binding](https://amigo.geneontology.org/amigo/term/GO:0045296) capabilities. The gene is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [ribosome](https://amigo.geneontology.org/amigo/term/GO:0005840), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [synapse](https://amigo.geneontology.org/amigo/term/GO:0045202), [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062), and the [cytoplasmic side of rough endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0098556). Additionally, it is part of the [small ribosomal subunit](https://amigo.geneontology.org/amigo/term/GO:0015935) and the [cytosolic small ribosomal subunit](https://amigo.geneontology.org/amigo/term/GO:0022627).


The gene length is 87,016 base pairs (79.19% of all genes), the mature length is 1,880 base pairs (49.57% of all genes), and the primary transcript length is 77,601 base pairs (84.17% of all genes).


The gene RPS26 (NCBI ID: 6231) has been mentioned in [16 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RPS26%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning this gene was in 1995, and the middle 50% of publications occurred between 2003 and 2016.


The top 5 publications mentioning RPS26, ranked by their scientific influence, include "[Ribosomal protein genes RPS10 and RPS26 are commonly mutated in Diamond-Blackfan anemia.](https://pubmed.ncbi.nlm.nih.gov/20116044)" (2010) (relative citation ratio: 3.65), "[The chaperone Tsr2 regulates Rps26 release and reincorporation from mature ribosomes to enable a reversible, ribosome-mediated response to stress.](https://pubmed.ncbi.nlm.nih.gov/35213229)" (2022) (relative citation ratio: 2.73), "[The ribosomal protein S26 regulates p53 activity in response to DNA damage.](https://pubmed.ncbi.nlm.nih.gov/23728348)" (2014) (relative citation ratio: 2.25), "[Loss of oocyte Rps26 in mice arrests oocyte growth and causes premature ovarian failure.](https://pubmed.ncbi.nlm.nih.gov/30451825)" (2018) (relative citation ratio: 1.28), and "[Statistical independence of the colocalized association signals for type 1 diabetes and RPS26 gene expression on chromosome 12q13.](https://pubmed.ncbi.nlm.nih.gov/19039033)" (2009) (relative citation ratio: 1.14). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RPS26](https://www.proteinatlas.org/ENSG00000197728-RPS26) is a ribonucleoprotein and ribosomal protein involved in Diamond-Blackfan anemia, with evidence at the protein level. It is detected in all tissues and is primarily located in the nucleoli, endoplasmic reticulum, and cytosol. The gene is part of Cluster 43 in blood expression, Cluster 18 in tissue expression, Cluster 13 in brain expression, Cluster 25 in cell line expression, and Cluster 49 in single cell expression, all related to non-specific transcription and translation functions.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [SMC3](https://www.ncbi.nlm.nih.gov/gene/9126), [GATA2](https://www.ncbi.nlm.nih.gov/gene/2624), [PGR](https://www.ncbi.nlm.nih.gov/gene/5241), [STAT1](https://www.ncbi.nlm.nih.gov/gene/6772), and [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170), each shown to be regulating in 5 experiments.



The input data indicates associations with various metabolic disorders, including diseases of metabolism, acquired metabolic diseases, carbohydrate metabolism diseases, and glucose metabolism diseases. Notably, there is a specific focus on diabetes mellitus, particularly type 1 diabetes mellitus.



The gene is expressed in various tissues, including the adrenal gland, colon, and small intestine, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles show expression in CD19+ B cells (negatively selected), colon, adrenal gland, and thymus.


The input data highlights various pathways involved in protein metabolism and gene expression, including '3\'-UTR-mediated translational regulation', 'L13a-mediated translational silencing of Ceruloplasmin expression', and 'GTP hydrolysis and joining of the 60S ribosomal subunit'. Key processes such as 'Cap-dependent Translation Initiation', 'Eukaryotic Translation Initiation', 'Eukaryotic Translation Elongation', and 'Eukaryotic Translation Termination' are also noted. Additionally, the data covers 'Viral mRNA Translation', 'Nonsense Mediated Decay (NMD)' both independent of and enhanced by the Exon Junction Complex (EJC), and 'SRP-dependent cotranslational protein targeting to membrane'. Influenza-related pathways, including 'Influenza Infection', 'Influenza Life Cycle', and 'Influenza Viral RNA Transcription and Replication', are also included. The data further details specific steps in translation initiation, such as 'Activation of the mRNA upon binding of the cap-binding complex and eIFs, and subsequent binding to 43S', 'Translation initiation complex formation', 'Ribosomal scanning and start codon recognition', and 'Formation of the ternary complex, and subsequently, the 43S complex'. Lastly, the data mentions 'Disease'.



The analyzed protein sequence has a GRAVY value of -0.66 (20.45th percentile), indicating a hydrophilic nature. It exhibits a charge of 21.71 at pH 7 (94.33rd percentile) and a median structural flexibility of 1.002 (49.02nd percentile). The protein's secondary structure is predicted to be 29.57% helix (28.11th percentile), 25.22% sheet (9.2nd percentile), and 26.09% turn (29.97th percentile). The instability index is 47.4 (47.97th percentile), with an isoelectric point of 11.01 (99.03rd percentile). The protein has a length of 115 amino acids (4.08th percentile) and a molecular weight of 13015.28 Da (4.4th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |