---
layout: post
title: RAC2
date: 2025-01-17 16:55 CST
description: RAC2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5880) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5880  | RAC2 | ENSG00000128340 | 22q13.1 |



The gene enables [GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0003924), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [GTP binding](https://amigo.geneontology.org/amigo/term/GO:0005525). It is located in the [nuclear envelope](https://amigo.geneontology.org/amigo/term/GO:0005635), [mitochondrial outer membrane](https://amigo.geneontology.org/amigo/term/GO:0005741), [endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0005789), and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). The gene colocalizes with [actin filament](https://amigo.geneontology.org/amigo/term/GO:0005884) and is active in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). Additionally, it is involved in various processes such as [chemotaxis](https://amigo.geneontology.org/amigo/term/GO:0006935), [actin filament organization](https://amigo.geneontology.org/amigo/term/GO:0007015), [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), and [G protein-coupled receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007186). The gene also plays a role in the [regulation of hydrogen peroxide metabolic process](https://amigo.geneontology.org/amigo/term/GO:0010310), [positive regulation of lamellipodium assembly](https://amigo.geneontology.org/amigo/term/GO:0010592), and [regulation of cell-substrate adhesion](https://amigo.geneontology.org/amigo/term/GO:0010810). Furthermore, it enables [protein kinase regulator activity](https://amigo.geneontology.org/amigo/term/GO:0019887) and is part of the [NADPH oxidase complex](https://amigo.geneontology.org/amigo/term/GO:0043020). The gene is involved in the regulation of various cellular processes, including [cell migration](https://amigo.geneontology.org/amigo/term/GO:0030334), [T cell proliferation](https://amigo.geneontology.org/amigo/term/GO:0042129), [erythrocyte enucleation](https://amigo.geneontology.org/amigo/term/GO:0043131), and [mast cell degranulation](https://amigo.geneontology.org/amigo/term/GO:0043304). It also plays a role in [bone resorption](https://amigo.geneontology.org/amigo/term/GO:0045453), [respiratory burst](https://amigo.geneontology.org/amigo/term/GO:0045730), and the regulation of [mast cell chemotaxis](https://amigo.geneontology.org/amigo/term/GO:0060753). The gene is located in the [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062) and is involved in [mast cell proliferation](https://amigo.geneontology.org/amigo/term/GO:0070662), [lymphocyte aggregation](https://amigo.geneontology.org/amigo/term/GO:0071593), and the [positive regulation of neutrophil chemotaxis](https://amigo.geneontology.org/amigo/term/GO:0090023). Lastly, it is involved in the [regulation of neutrophil migration](https://amigo.geneontology.org/amigo/term/GO:1902622) and the [positive regulation of protein targeting to mitochondrion](https://amigo.geneontology.org/amigo/term/GO:1903955).


The gene length is 43,773 base pairs (62.76% of all genes), the mature length is 1,872 base pairs (49.31% of all genes), and the primary transcript length is 16,167 base pairs (46.05% of all genes).


The gene RAC2 (NCBI ID: 5880) has been mentioned in [150 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RAC2%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning RAC2 is 1990, and the middle 50% of publications occurred between 2001 and 2016.


The top 5 publications mentioning RAC2, ranked by their scientific influence, include "[Regulation of phagocyte oxygen radical production by the GTP-binding protein Rac 2.](https://pubmed.ncbi.nlm.nih.gov/1660188)" (1991) (relative citation ratio: 12.85), "[Hematopoietic cell regulation by Rac1 and Rac2 guanosine triphosphatases.](https://pubmed.ncbi.nlm.nih.gov/14564009)" (2003) (relative citation ratio: 6.34), "[Human neutrophil immunodeficiency syndrome is associated with an inhibitory Rac2 mutation.](https://pubmed.ncbi.nlm.nih.gov/10758162)" (2000) (relative citation ratio: 5.77), "[Cdc42, Rac1, and Rac2 display distinct patterns of activation during phagocytosis.](https://pubmed.ncbi.nlm.nih.gov/15169870)" (2004) (relative citation ratio: 4.79), and "[Dominant negative mutation of the hematopoietic-specific Rho GTPase, Rac2, is associated with a human phagocyte immunodeficiency.](https://pubmed.ncbi.nlm.nih.gov/10961859)" (2000) (relative citation ratio: 4.41). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RAC2](https://www.proteinatlas.org/ENSG00000128340-RAC2) is a gene with evidence at the protein level and is implicated in disease variants. Its RNA is detected in many tissues, with notable expression in neutrophils (Cluster 35) and lymphoid tissue (Cluster 70), where it is involved in immune response. Additionally, RAC2 is expressed in the vasculature of the brain (Cluster 12) and in HMC-1 cell lines (Cluster 49), contributing to the innate immune response. It is also found in NK-cells (Cluster 19), playing a role in the adaptive immune response.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), [CDK8](https://www.ncbi.nlm.nih.gov/gene/1024), [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861), and [CEBPA](https://www.ncbi.nlm.nih.gov/gene/1050), each shown to be regulating in 4 experiments.


The top transcription factors (TFs) identified in terms of the number of Perturb-seq experiments are [2623](https://www.ncbi.nlm.nih.gov/gene/2623) and [8861](https://www.ncbi.nlm.nih.gov/gene/8861), each shown to be regulating genes in one experiment.


The input data indicates associations with a range of disease conditions, including various types of cancer such as organ system cancer, endocrine gland cancer, and pancreatic cancer. Additionally, the data highlights immune system diseases, including primary immunodeficiency disease, b cell deficiency, and selective immunoglobulin deficiency disease. Cardiovascular and vascular diseases are also represented, encompassing artery disease, cerebrovascular disease, and cardiovascular system disease. Furthermore, the data includes endocrine system diseases, specifically thyroid gland disease, autoimmune disease of the endocrine system, Graves' disease, and hyperthyroidism. Lastly, hypersensitivity reactions, particularly type II hypersensitivity reactions, are noted.



The gene is expressed in various tissues and cell types, including blood, spleen, and small intestine, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles show expression in Lymphoma Burkitt's (Raji), BDCA4+ dendritic cells, CD56+ NK cells, CD4+ T cells, CD8+ T cells, whole blood, bone marrow, and thymus.


The proteins are involved in various pathways, including the PCP/CE pathway, beta-catenin independent WNT signaling, and signaling by Wnt. Additionally, they participate in the GPVI-mediated activation cascade, axon guidance, and developmental biology processes. Semaphorin signaling is also prominent, with involvement in Sema4D signaling, Sema4D-induced cell migration and growth-cone collapse, and semaphorin interactions. The proteins are further implicated in G alpha (12/13) signaling events, the Rho GTPase cycle, and signaling by Rho GTPases. They also play roles in GPCR downstream signaling, signaling by GPCR, and broader signal transduction processes. Moreover, the proteins are associated with hemostasis, as well as platelet activation, signaling, and aggregation.


The mouse ortholog gene 19354 and its human ortholog gene 5880 are associated with abnormal liver morphology, a phenotype observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.121 (76.53rd percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is 0.623 (52.37th percentile), and the median structural flexibility is 0.999 (34.76th percentile). The protein has a propensity for helix, sheet, and turn structures of 0.313 (43.54th percentile), 0.396 (85.02nd percentile), and 0.266 (34.19th percentile), respectively. The instability index is 39.008 (19.46th percentile), and the isoelectric point is 7.522 (54.14th percentile). The protein sequence is 192 amino acids long (13.31st percentile) with a molecular weight of 21428.56 Da (13.22nd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |