---
layout: post
title: LMAN1
date: 2025-01-17 16:55 CST
description: LMAN1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3998) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3998  | LMAN1 | ENSG00000074695 | 18q21.32 |



The gene is located in the [Golgi membrane](https://amigo.geneontology.org/amigo/term/GO:0000139), [endoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0005783), [endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0005789), [endoplasmic reticulum-Golgi intermediate compartment](https://amigo.geneontology.org/amigo/term/GO:0005793), [ER to Golgi transport vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0012507), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [sarcomere](https://amigo.geneontology.org/amigo/term/GO:0030017), [endoplasmic reticulum-Golgi intermediate compartment membrane](https://amigo.geneontology.org/amigo/term/GO:0033116), [collagen-containing extracellular matrix](https://amigo.geneontology.org/amigo/term/GO:0062023), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). It is active in the [Golgi membrane](https://amigo.geneontology.org/amigo/term/GO:0000139), [endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0005789), [endoplasmic reticulum-Golgi intermediate compartment](https://amigo.geneontology.org/amigo/term/GO:0005793), and [COPII-coated ER to Golgi transport vesicle](https://amigo.geneontology.org/amigo/term/GO:0030134). The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [mannose binding](https://amigo.geneontology.org/amigo/term/GO:0005537), [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872), and [unfolded protein binding](https://amigo.geneontology.org/amigo/term/GO:0051082). It is involved in [protein folding](https://amigo.geneontology.org/amigo/term/GO:0006457), [endoplasmic reticulum to Golgi vesicle-mediated transport](https://amigo.geneontology.org/amigo/term/GO:0006888), [endoplasmic reticulum organization](https://amigo.geneontology.org/amigo/term/GO:0007029), [Golgi organization](https://amigo.geneontology.org/amigo/term/GO:0007030), [blood coagulation](https://amigo.geneontology.org/amigo/term/GO:0007596), [positive regulation of organelle organization](https://amigo.geneontology.org/amigo/term/GO:0010638), and [negative regulation of protein targeting to mitochondrion](https://amigo.geneontology.org/amigo/term/GO:1903215). The gene is not involved in [protein exit from endoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0032527) and [early endosome to Golgi transport](https://amigo.geneontology.org/amigo/term/GO:0034498).


The gene has a length of 36,732 base pairs (58.21% of all genes), a mature length of 1,850 base pairs, and a primary transcript length of 36,330 base pairs.


The gene LMAN1 (NCBI ID: 3998) has been mentioned in [72 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22LMAN1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1986 and the middle 50% of publications occurring between 1998 and 2010. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning LMAN1, ranked by their scientific influence, include "[Mutations in the ER-Golgi intermediate compartment protein ERGIC-53 cause combined deficiency of coagulation factors V and VIII.](https://pubmed.ncbi.nlm.nih.gov/9546392)" (1998) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 6.04, "[The lectin ERGIC-53 is a cargo transport receptor for glycoproteins.](https://pubmed.ncbi.nlm.nih.gov/10559958)" (1999) with an RCR of 4.45, "[ERGIC-53 and traffic in the secretory pathway.](https://pubmed.ncbi.nlm.nih.gov/10652252)" (2000) with an RCR of 4.37, "[Fine specificity of the human immune response to the major neutralization epitopes expressed on cytomegalovirus gp58/116 (gB), as determined with human monoclonal antibodies.](https://pubmed.ncbi.nlm.nih.gov/7678304)" (1993) with an RCR of 2.81, and "[The recycling pathway of protein ERGIC-53 and dynamics of the ER-Golgi intermediate compartment.](https://pubmed.ncbi.nlm.nih.gov/9788882)" (1998) with an RCR of 2.79. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[LMAN1](https://www.proteinatlas.org/ENSG00000074695-LMAN1) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in the endoplasmic reticulum and vesicles. The gene is expressed in various clusters, including Cluster 5 for blood expression related to cell proliferation, Cluster 1 for tissue expression related to liver and kidney metabolism, Cluster 55 for brain expression related to transcription, Cluster 13 for cell line expression with mixed function, and Cluster 77 for single cell expression in plasma cells involved in protein processing.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [STAT5B](https://www.ncbi.nlm.nih.gov/gene/6777), [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908), [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113), and [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), each shown to be regulating in 4 experiments.



The GWAS data indicates associations with various disease conditions, including general diseases, diseases of anatomical entities, gastrointestinal system diseases, gallbladder diseases, and specifically cholecystolithiasis.


The gene LMAN1 has been associated with [Cardiac troponin-I levels](https://pubmed.ncbi.nlm.nih.gov/31014085), which maps closely to the MeSH term [Troponin I](https://meshb.nlm.nih.gov/record/ui?ui=D019210) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method.


The gene is expressed in the liver and pituitary according to BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles indicate expression in the medulla oblongata, smooth muscle, appendix, superior cervical ganglion, skeletal muscle, dorsal root ganglion, and globus pallidus.


The input data highlights various diseases and processes related to glycosylation, particularly asparagine N-linked glycosylation. Notably, defects in specific proteins such as ALG2, ALG3, ALG6, ALG8, ALG9, DPAGT1, MAN1B1, MGAT2, MOGS, MPDU1, ALG1, ALG12, ALG11, ALG14, and RFT1 are linked to distinct congenital disorders of glycosylation (CDGs) and other conditions like congenital myasthenic syndrome. These defects underscore the importance of proper protein modification and transport processes in maintaining cellular health. Additionally, the data encompasses broader categories such as diseases associated with N-glycosylation of proteins, post-translational protein modification, metabolism of proteins, and transport to the Golgi and subsequent modification, all of which are integral to understanding the pathogenesis of these diseases.



The analyzed protein sequence has a GRAVY value of -0.542 (30.81 percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -4.97 (33.84 percentile), and the median structural flexibility is 1.002 (49.45 percentile). The protein has a predicted secondary structure with 28.04% helix (17.24 percentile), 32.75% sheet (43.86 percentile), and 29.02% turn (56.05 percentile) affinities. The instability index is 39.27 (20.21 percentile), and the isoelectric point is approximately 6.30 (38.12 percentile). The protein sequence is 510 amino acids long (59.52 percentile) with a molecular weight of 57548.25 Da (60.19 percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |