---
layout: post
title: HLA-C
date: 2025-01-17 16:55 CST
description: HLA-C description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3107) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3107  | HLA-C | ENSG00000204525 | 6p21.33 |



The gene is located in the [Golgi membrane](https://amigo.geneontology.org/amigo/term/GO:0000139), [endoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0005783), [Golgi apparatus](https://amigo.geneontology.org/amigo/term/GO:0005794), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986), [ER to Golgi transport vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0012507), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [secretory granule membrane](https://amigo.geneontology.org/amigo/term/GO:0030667), [phagocytic vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030670), [early endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0031901), [recycling endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0055038), [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062), and [lumenal side of endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0098553). It is involved in the [positive regulation of T cell mediated cytotoxicity](https://amigo.geneontology.org/amigo/term/GO:0001916), [adaptive immune response](https://amigo.geneontology.org/amigo/term/GO:0002250), [antigen processing and presentation of endogenous peptide antigen via MHC class Ib](https://amigo.geneontology.org/amigo/term/GO:0002476), [antigen processing and presentation of endogenous peptide antigen via MHC class I via ER pathway, TAP-independent](https://amigo.geneontology.org/amigo/term/GO:0002486), and [immune response](https://amigo.geneontology.org/amigo/term/GO:0006955). The gene enables [signaling receptor binding](https://amigo.geneontology.org/amigo/term/GO:0005102), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [peptide antigen binding](https://amigo.geneontology.org/amigo/term/GO:0042605), and [TAP binding](https://amigo.geneontology.org/amigo/term/GO:0046977). It is active in the [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615) and [external side of plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009897), and is part of the [MHC class I protein complex](https://amigo.geneontology.org/amigo/term/GO:0042612). Additionally, it is involved in the [innate immune response](https://amigo.geneontology.org/amigo/term/GO:0045087).


The gene length is 11,173 base pairs (28.04% of all genes), the mature length is 3,795 base pairs, and the primary transcript length is 11,173 base pairs.


The gene HLA-C (NCBI ID: 3107) has been mentioned in [1,859 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22HLA-C%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication mentioning this gene is 1971, and the middle 50% of publications occurred between 1996 and 2014.


The top 5 publications mentioning HLA-C, ranked by their scientific influence, include "[MHC-restricted cytotoxic T cells: studies on the biological role of polymorphic major transplantation antigens determining T-cell restriction-specificity, function, and responsiveness.](https://pubmed.ncbi.nlm.nih.gov/92183)" (1979) (relative citation ratio: 36.38), "[MHC ligands and peptide motifs: first listing.](https://pubmed.ncbi.nlm.nih.gov/7890324)" (1995) (relative citation ratio: 31.25), "[Three myosin heavy chain isoforms in type 2 skeletal muscle fibres.](https://pubmed.ncbi.nlm.nih.gov/2547831)" (1989) (relative citation ratio: 25.08), "[The ORF8 protein of SARS-CoV-2 mediates immune evasion through down-regulating MHC-Î™.](https://pubmed.ncbi.nlm.nih.gov/34021074)" (2021) (relative citation ratio: 24.38), and "[A molecular basis for familial hypertrophic cardiomyopathy: a beta cardiac myosin heavy chain gene missense mutation.](https://pubmed.ncbi.nlm.nih.gov/1975517)" (1990) (relative citation ratio: 23.36). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[HLA-C](https://www.proteinatlas.org/ENSG00000204525-HLA-C) is a gene with evidence at the protein level. Its RNA is detected in many tissues, and it is notably expressed in monocytes and neutrophils involved in degranulation (Cluster 38). In tissue expression, it is associated with the spleen and immune response (Cluster 47). Additionally, HLA-C is expressed in immune cells within the brain (Cluster 8) and myeloma cell lines (Cluster 51). In single-cell expression, it is found in Langerhans cells involved in the innate immune response (Cluster 14).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 5 experiments, followed by [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), [PGR](https://www.ncbi.nlm.nih.gov/gene/5241), and [AR](https://www.ncbi.nlm.nih.gov/gene/367), each with 4 experiments.


The transcription factor [9968](https://www.ncbi.nlm.nih.gov/gene/9968) was found to be regulating genes in one Perturb-seq experiment, according to data from GTRD's database of transcription factor binding sites and the study published in [PubMed](https://pubmed.ncbi.nlm.nih.gov/35688146/).


The GWAS data indicates associations with a wide range of disease conditions, including infectious diseases such as viral and bacterial infections, notably human immunodeficiency virus infectious disease, hepatitis B, and leprosy. Respiratory conditions are also prominent, encompassing obstructive lung disease, lower respiratory tract disease, and chronic obstructive pulmonary disease. Metabolic disorders, particularly diabetes mellitus and type 1 diabetes mellitus, are linked, along with cardiovascular issues like artery disease, vascular disease, and hypertension. Additionally, the data highlights immune system diseases, inflammatory conditions such as rheumatoid arthritis and psoriasis, and gastrointestinal disorders including inflammatory bowel disease and Crohn's disease.


HLA-C has been associated with various diseases and traits through genome-wide association studies. Notably, it has been linked to [A body shape index](https://pubmed.ncbi.nlm.nih.gov/34021172) (closest MeSH match: [Index](https://meshb.nlm.nih.gov/record/ui?ui=D020481)), [Educational attainment](https://pubmed.ncbi.nlm.nih.gov/35361970) (closest MeSH match: [Educational Status](https://meshb.nlm.nih.gov/record/ui?ui=D004522)), and [Gastric cancer](https://pubmed.ncbi.nlm.nih.gov/34594039) (closest MeSH match: [Stomach Neoplasms](https://meshb.nlm.nih.gov/record/ui?ui=D013274)). Additionally, associations have been found with [Lymphocyte count](https://pubmed.ncbi.nlm.nih.gov/34594039) (closest MeSH match: [Lymphocyte Count](https://meshb.nlm.nih.gov/record/ui?ui=D018655)), [Mean corpuscular volume](https://pubmed.ncbi.nlm.nih.gov/32888493) (closest MeSH match: [Erythrocyte Indices](https://meshb.nlm.nih.gov/record/ui?ui=D004909)), and [Methazolamide-induced Stevens-Johnson syndrome / toxic epidermal necrolysis](https://pubmed.ncbi.nlm.nih.gov/35122707) (closest MeSH match: [Dermatitis, Exfoliative](https://meshb.nlm.nih.gov/record/ui?ui=D003873)). Furthermore, HLA-C has been implicated in [Resistance to COVID-19 infection (Exposed negative vs positive)](https://pubmed.ncbi.nlm.nih.gov/36662838) (closest MeSH match: [Infections](https://meshb.nlm.nih.gov/record/ui?ui=D007239)) and [White blood cell count](https://pubmed.ncbi.nlm.nih.gov/31217584) (closest MeSH match: [Leukocyte Count](https://meshb.nlm.nih.gov/record/ui?ui=D007958)). These associations were mapped using a method called [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0).


The gene is expressed in multiple tissue samples, including lung, blood, and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles show expression in CD56+ NK cells, whole blood, and lung.


The input data highlights various pathways involved in the immune system, including antigen presentation and processing, cytokine signaling, interferon signaling, and interactions between lymphoid and non-lymphoid cells. Specific pathways mentioned include the folding, assembly, and peptide loading of class I MHC, interferon gamma and alpha/beta signaling, ER-phagosome pathway, and endosomal/vacuolar pathway. Additionally, the data covers broader categories such as the adaptive and innate immune systems, as well as DAP12 interactions.



The analyzed protein sequence has a GRAVY value of -0.5352 (31.43rd percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -6.82 (28.93rd percentile), and the median structural flexibility is 1.0023 (52.36th percentile). The protein has an affinity for helix and sheet structures at 31.42% each (45.01st and 36.23rd percentiles, respectively), and a turn structure affinity of 27.32% (41.02nd percentile). The instability index is 41.62 (27.32nd percentile), with an isoelectric point of 5.71 (23.01st percentile). The protein length is 366 amino acids (40.59th percentile), and the molecular weight is 40648.09 Da (40.35th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |