---
layout: post
title: SERTAD2
date: 2025-01-17 16:55 CST
description: SERTAD2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9792) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9792  | SERTAD2 | ENSG00000179833 | 2p14  |



The gene enables [transcription coactivator activity](https://amigo.geneontology.org/amigo/term/GO:0003713) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is active in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634). It is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). The gene acts upstream of or within [negative regulation of cell growth](https://amigo.geneontology.org/amigo/term/GO:0030308) and is involved in [positive regulation of DNA-templated transcription](https://amigo.geneontology.org/amigo/term/GO:0045893). Additionally, it enables [transcription regulator activity](https://amigo.geneontology.org/amigo/term/GO:0140110).


The gene length is 19,977 base pairs (42.06% of all genes), the mature length is 1,516 base pairs, and the primary transcript length is 19,975 base pairs.


The gene SERTAD2 (NCBI ID: 9792) has been mentioned in [7 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SERTAD2%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 2008, and the middle 50% of publications occurred between 2011 and 2017.


The top 5 publications mentioning SERTAD2, ranked by their relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows:

- [Ablation of TRIP-Br2, a regulator of fat lipolysis, thermogenesis and oxidative metabolism, prevents diet-induced obesity and insulin resistance.](https://pubmed.ncbi.nlm.nih.gov/23291629) (2013) (relative citation ratio: 1.64)
- [miR-128-3p regulates 3T3-L1 adipogenesis and lipolysis by targeting Pparg and Sertad2.](https://pubmed.ncbi.nlm.nih.gov/29654510) (2018) (relative citation ratio: 1.19)
- [The obesity-induced transcriptional regulator TRIP-Br2 mediates visceral fat endoplasmic reticulum stress-induced inflammation.](https://pubmed.ncbi.nlm.nih.gov/27109496) (2016) (relative citation ratio: 1.18)
- [REV1 promotes lung tumorigenesis by activating the Rad18/SERTAD2 axis.](https://pubmed.ncbi.nlm.nih.gov/35115490) (2022) (relative citation ratio: 0.89)
- [TRIP-Br2 promotes oncogenesis in nude mice and is frequently overexpressed in multiple human tumors.](https://pubmed.ncbi.nlm.nih.gov/19152710) (2009) (relative citation ratio: 0.4)

Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[SERTAD2](https://www.proteinatlas.org/ENSG00000179833-SERTAD2) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in the nucleoplasm and cytosol. The gene is expressed in various tissue clusters, including non-specific transcription clusters and white matter signal transduction clusters in the brain. Additionally, it is expressed in rhabdoid cell lines and squamous epithelial cells, with no specific prognostic significance in various cancer types.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SP1](https://www.ncbi.nlm.nih.gov/gene/6667), [TAL1](https://www.ncbi.nlm.nih.gov/gene/6886), and [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861), each with 6 experiments. Additionally, [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170) and [EP300](https://www.ncbi.nlm.nih.gov/gene/2033) were found to be regulating in 5 experiments each.



The GWAS data indicates associations with various disease conditions, including cardiovascular system disease, general disease, and diseases of anatomical entities.



The gene is expressed in various tissues, including the vagina and skin. Additionally, it is expressed in several immune cell types such as CD34+, CD33+ myeloid cells, BDCA4+ dendritic cells, CD56+ NK cells, CD19+ B cells (neg. sel.), and CD71+ early erythroid cells, as well as in the thymus.




The analyzed protein sequence has a GRAVY value of -0.44968152866242 (40.44th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -24.054959085145587 (8.62nd percentile), and the median structural flexibility is 1.012202380952381 (86.59th percentile). The protein has a high instability index of 65.4904458598726 (90.76th percentile) and an isoelectric point of 4.416810417175294 (1.19th percentile). The sequence length is 314 amino acids (31.55th percentile) with a molecular weight of 33896.47650000004 Da (29.09th percentile). The secondary structure affinities are 0.32484076433121023 for helix (54.95th percentile), 0.32165605095541405 for sheet (40.48th percentile), and 0.3662420382165605 for turn (92.99th percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |