---
layout: post
title: PNPLA6
date: 2025-01-17 16:55 CST
description: PNPLA6 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10908) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10908  | PNPLA6 | ENSG00000032444 | 19p13.2 |



The gene enables [lysophospholipase activity](https://amigo.geneontology.org/amigo/term/GO:0004622) and [phosphatidyl phospholipase B activity](https://amigo.geneontology.org/amigo/term/GO:0102545). It is located in the [endoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0005783), [endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0005789), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020). Additionally, it is involved in the [phosphatidylcholine metabolic process](https://amigo.geneontology.org/amigo/term/GO:0046470) and the [glycerophospholipid catabolic process](https://amigo.geneontology.org/amigo/term/GO:0046475).


The gene length is 38,981 base pairs (59.69% of all genes), the mature length is 1,724.5 base pairs (44.99% of all genes), and the primary transcript length is 28,058 base pairs (60.83% of all genes).


The gene PNPLA6 (NCBI ID: 10908) has been mentioned in [105 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PNPLA6%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink). The earliest year of publication is 1980, and the middle 50% of publications range from 1994 to 2015.


The top five publications mentioning PNPLA6, ranked by their scientific influence, include "[Organophosphates and delayed neuropathy--is NTE alive and well?](https://pubmed.ncbi.nlm.nih.gov/2180130)" (1990) (relative citation ratio: 6.05), "[PNPLA6 mutations cause Boucher-Neuhauser and Gordon Holmes syndromes as part of a broad neurodegenerative spectrum.](https://pubmed.ncbi.nlm.nih.gov/24355708)" (2014) (relative citation ratio: 5.12), "[Neuropathy target esterase gene mutations cause motor neuron disease.](https://pubmed.ncbi.nlm.nih.gov/18313024)" (2008) (relative citation ratio: 3.88), "[Neuropathy target esterase and its yeast homologue degrade phosphatidylcholine to glycerophosphocholine in living cells.](https://pubmed.ncbi.nlm.nih.gov/15044461)" (2004) (relative citation ratio: 3.62), and "[Human neuropathy target esterase catalyzes hydrolysis of membrane lipids.](https://pubmed.ncbi.nlm.nih.gov/11927584)" (2002) (relative citation ratio: 2.84). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PNPLA6](https://www.proteinatlas.org/ENSG00000032444-PNPLA6) is a hydrolase with evidence at the protein level, implicated in various diseases including dwarfism, hereditary spastic paraplegia, hypogonadotropic hypogonadism, intellectual disability, neurodegeneration, and retinitis pigmentosa. The gene is detected in all RNA tissue distributions and is localized in the endoplasmic reticulum and cytosol. It is expressed in various clusters, including Cluster 25 for blood, Cluster 34 for tissue, Cluster 32 for brain, Cluster 23 for cell lines, and Cluster 58 for single cells, with specific functions such as mixed function in the cerebral cortex and innate immune response in macrophages.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908) with 6 experiments, [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861) with 5 experiments, [PPARG](https://www.ncbi.nlm.nih.gov/gene/5468) with 5 experiments, [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 5 experiments, and [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with various disease conditions, including hematopoietic system disease, immune system disease, bone disease, connective tissue disease, musculoskeletal system disease, bone remodeling disease, bone resorption disease, agammaglobulinemia, common variable immunodeficiency, blood protein disease, B cell deficiency, and primary immunodeficiency disease.



The gene is expressed in multiple tissue samples, including the pituitary, lung, and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the Prefrontal Cortex, CD14+ Monocytes, CD71+ Early Erythroid cells, Amygdala, Lung, and Testis.




The analyzed protein sequence has a GRAVY value of -0.205 (68.62nd percentile), indicating a hydrophilic nature. It exhibits a charge of 4.91 at pH 7.0 (67.76th percentile) and a median structural flexibility of 1.0003 (43.11th percentile). The protein's secondary structure is predicted to be 30.47% helix (36.11th percentile), 34.76% sheet (57.33rd percentile), and 29.53% turn (60.26th percentile). The instability index is 49.25 (54.68th percentile), with an isoelectric point of 7.89 (57.79th percentile). The protein has a length of 1375 amino acids (94th percentile) and a molecular weight of 150952.16 Da (93.86th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |