---
layout: post
title: RHOC
date: 2024-12-08 23:57 CST
description: RHOC description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/389) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 389  | RHOC | ENSG00000155366 | 1p13.2 |



The gene length is 16,901 base pairs (37.87% of all genes), the mature length is 681 base pairs (8.52% of all genes), and the primary transcript length is 5,641 base pairs (20.66% of all genes).


The gene RHOC, with NCBI ID 389, has been featured in [183 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RHOC%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1970, and the middle 50% of publications occurred between 2008 and 2016.


The top five publications mentioning RHOC, ranked by their scientific influence, include "[Genomic analysis of metastasis reveals an essential role for RhoC.](https://pubmed.ncbi.nlm.nih.gov/10952316)" (2000) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 21.03, "[The mammalian G protein rhoC is ADP-ribosylated by Clostridium botulinum exoenzyme C3 and affects actin microfilaments in Vero cells.](https://pubmed.ncbi.nlm.nih.gov/2501082)" (1989) with an RCR of 9.26, "[Why three Rho proteins? RhoA, RhoB, RhoC, and cell motility.](https://pubmed.ncbi.nlm.nih.gov/15501444)" (2004) with an RCR of 6.83, "[MicroRNA-10b promotes nucleus pulposus cell proliferation through RhoC-Akt pathway by targeting HOXD10 in intervetebral disc degeneration.](https://pubmed.ncbi.nlm.nih.gov/24376640)" (2013) with an RCR of 6.49, and "[MicroRNA-10b is overexpressed in malignant glioma and associated with tumor invasive factors, uPAR and RhoC.](https://pubmed.ncbi.nlm.nih.gov/19536818)" (2009) with an RCR of 5.61. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313) with 6 experiments, [SS18](https://www.ncbi.nlm.nih.gov/gene/6760) with 5 experiments, [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 5 experiments, [JUND](https://www.ncbi.nlm.nih.gov/gene/3727) with 5 experiments, and [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [STAT1](https://www.ncbi.nlm.nih.gov/gene/6829) and [IRF1](https://www.ncbi.nlm.nih.gov/gene/9126), each shown to regulate genes in three experiments. Additionally, [NFE2L2](https://www.ncbi.nlm.nih.gov/gene/2623), [MAFK](https://www.ncbi.nlm.nih.gov/gene/8243), [STAT2](https://www.ncbi.nlm.nih.gov/gene/6830), [CEBPB](https://www.ncbi.nlm.nih.gov/gene/8861), and [BCL6](https://www.ncbi.nlm.nih.gov/gene/9968) were found to regulate genes in two experiments each.


The genome-wide association study has identified a link between RHOC and [Cortical surface area (visual PC2)](https://pubmed.ncbi.nlm.nih.gov/32198502).





The analyzed protein sequence has a GRAVY value of -0.4212 (43.64th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -1.3414 (45.25th percentile), and the median structural flexibility is 1.0084 (76.28th percentile). The protein has a high instability index of 61.09 (84.96th percentile) and an isoelectric point of 6.20 (35.62nd percentile). The sequence length is 193 amino acids (13.45th percentile) with a molecular weight of 22006.12 Da (14.11th percentile). The secondary structure affinities are 0.3316 for helix (60.61st percentile), 0.3368 for sheet (49.73rd percentile), and 0.2591 for turn (28.33rd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |