---
layout: post
title: ASAH1
date: 2024-12-08 23:57 CST
description: ASAH1 description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/427) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 427  | ASAH1 | ENSG00000104763 | 8p22 |



The gene length is 53,931 base pairs (68.49% of all genes), the mature length is 1,342 base pairs, and the primary transcript length is 19,364 base pairs (50.81% of all genes).


The gene ASAH1 (NCBI ID: 427) has been the subject of [150 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ASAH1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication dating back to 1980. The middle 50% of these publications were published between 2008 and 2019. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning ASAH1, ranked by their scientific influence, include "[Molecular characterization of N-acylethanolamine-hydrolyzing acid amidase, a novel member of the choloylglycine hydrolase family with structural and functional similarity to acid ceramidase.](https://pubmed.ncbi.nlm.nih.gov/15655246)" (2005) (relative citation ratio: 6.19), "[Lysosomal glycosphingolipid catabolism by acid ceramidase: formation of glycosphingoid bases during deficiency of glycosidases.](https://pubmed.ncbi.nlm.nih.gov/26898341)" (2016) (relative citation ratio: 4.39), "[Molecular cloning and characterization of a full-length complementary DNA encoding human acid ceramidase. Identification Of the first molecular lesion causing Farber disease.](https://pubmed.ncbi.nlm.nih.gov/8955159)" (1996) (relative citation ratio: 4.25), "[Acid ceramidase deficiency: Farber disease and SMA-PME.](https://pubmed.ncbi.nlm.nih.gov/30029679)" (2018) (relative citation ratio: 4.18), and "[Purification, characterization, and biosynthesis of human acid ceramidase.](https://pubmed.ncbi.nlm.nih.gov/7744740)" (1995) (relative citation ratio: 4.05). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [CREB1](https://www.ncbi.nlm.nih.gov/gene/1385) with 6 experiments, [YY1](https://www.ncbi.nlm.nih.gov/gene/7528) with 5 experiments, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 5 experiments, [CDK8](https://www.ncbi.nlm.nih.gov/gene/1024) with 5 experiments, and [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [5885](https://www.ncbi.nlm.nih.gov/gene/5885) with 5 experiments, [9126](https://www.ncbi.nlm.nih.gov/gene/9126) with 4 experiments, and [6829](https://www.ncbi.nlm.nih.gov/gene/6829), [2623](https://www.ncbi.nlm.nih.gov/gene/2623), [2118](https://www.ncbi.nlm.nih.gov/gene/2118), [8243](https://www.ncbi.nlm.nih.gov/gene/8243), and [9968](https://www.ncbi.nlm.nih.gov/gene/9968) each with 2 experiments. Additionally, [6830](https://www.ncbi.nlm.nih.gov/gene/6830) was found to be regulating in 1 experiment. These TFs were shown to significantly impact gene expression in the Perturb-seq experiments.


The genome-wide association study has identified a link between the ASAH1 gene and [Metabolite peak levels (QI3139)](https://pubmed.ncbi.nlm.nih.gov/35995766).





The analyzed protein sequence has a GRAVY value of -0.1111 (77.25th percentile), indicating a relatively hydrophilic nature. At pH 7.0, the protein carries a charge of 0.8973 (53.54th percentile). The median structural flexibility is 0.9960 (24.01st percentile), suggesting a fairly rigid structure. The protein has a length of 395 amino acids (44.6th percentile) and a molecular weight of 44659.12 Da (45.3rd percentile). The secondary structure affinities are 29.62% for helix (28.64th percentile), 42.53% for sheet (91.65th percentile), and 28.61% for turn (52.71st percentile). The instability index is 38.48 (18.08th percentile), and the isoelectric point is 7.52 (54.15th percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |