---
layout: post
title: RPL3
date: 2025-01-17 16:55 CST
description: RPL3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6122) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6122  | RPL3 | ENSG00000100316 | 22q13.1 |



The gene is involved in [cytoplasmic translation](https://amigo.geneontology.org/amigo/term/GO:0002181) and [translation](https://amigo.geneontology.org/amigo/term/GO:0006412). It enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), acts as a [structural constituent of ribosome](https://amigo.geneontology.org/amigo/term/GO:0003735), and enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleolus](https://amigo.geneontology.org/amigo/term/GO:0005730), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925). It is part of the [cytosolic large ribosomal subunit](https://amigo.geneontology.org/amigo/term/GO:0022625) and [protein-containing complex](https://amigo.geneontology.org/amigo/term/GO:0032991). Additionally, the gene is located in the [cytosolic ribosome](https://amigo.geneontology.org/amigo/term/GO:0022626) and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). It is also involved in the [cellular response to interleukin-4](https://amigo.geneontology.org/amigo/term/GO:0071353).


The gene length is 67,556 base pairs (73.83rd percentile), the mature length is 558 base pairs (3.06th percentile), and the primary transcript length is 45,586 base pairs (73.2nd percentile).


The gene RPL3 (NCBI ID: 6122) has been mentioned in [23 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RPL3%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 2000 and the middle 50% of publications occurring between 2008 and 2017. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning RPL3, ranked by their scientific influence as measured by the [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541), include "[The ASC-1 Complex Disassembles Collided Ribosomes.](https://pubmed.ncbi.nlm.nih.gov/32579943)" (2020) with an RCR of 4.64, "[Asc1, Hel2, and Slh1 couple translation arrest to nascent chain degradation.](https://pubmed.ncbi.nlm.nih.gov/28223409)" (2017) with an RCR of 2.81, "[rpL3 promotes the apoptosis of p53 mutated lung cancer cells by down-regulating CBS and NFκB upon 5-FU treatment.](https://pubmed.ncbi.nlm.nih.gov/27924828)" (2016) with an RCR of 2.67, "[Human METTL18 is a histidine-specific methyltransferase that targets RPL3 and affects ribosome biogenesis and function.](https://pubmed.ncbi.nlm.nih.gov/33693809)" (2021) with an RCR of 2.59, and "[5-FU targets rpL3 to induce mitochondrial apoptosis via cystathionine-β-synthase in colon cancer cells lacking p53.](https://pubmed.ncbi.nlm.nih.gov/27385096)" (2016) with an RCR of 2.58. The RCR values measure the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RPL3](https://www.proteinatlas.org/ENSG00000100316-RPL3) is a ribonucleoprotein and ribosomal protein with evidence at the protein level. It is involved in FDA approved drug targets and is detected in all tissues. The protein is localized in the nucleoli and cytosol. RPL3 is part of non-specific clusters related to transcription and translation in blood, and ribosome-related clusters in various tissues, including the brain and cell lines.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930) with 7 experiments, [OTX2](https://www.ncbi.nlm.nih.gov/gene/5015) with 6 experiments, [STAT1](https://www.ncbi.nlm.nih.gov/gene/6772) with 5 experiments, [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) with 5 experiments, and [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [TCF3](https://www.ncbi.nlm.nih.gov/gene/7541), [ARNT](https://www.ncbi.nlm.nih.gov/gene/2475), [HNF4A](https://www.ncbi.nlm.nih.gov/gene/11091), and [BRCA1](https://www.ncbi.nlm.nih.gov/gene/6830). These TFs were shown to regulate gene expression in 1, 1, 3, and 1 Perturb-seq experiments, respectively.


The GWAS data indicates associations with various disease conditions, including immune system disease, hypersensitivity reaction disease (specifically type II), hepatobiliary disease, liver disease, liver cirrhosis, autoimmune disease of the urogenital tract, primary biliary cirrhosis, and gastrointestinal system disease.



The gene is expressed in the ovary. Additionally, it is expressed in various tissues including the pineal gland during both day and night, as well as in CD34+ cells and CD19+ B cells.


The input data highlights various pathways involved in protein metabolism and gene expression, including 3'-UTR-mediated translational regulation, L13a-mediated translational silencing of ceruloplasmin expression, and GTP hydrolysis and joining of the 60S ribosomal subunit. Key processes such as cap-dependent translation initiation, eukaryotic translation initiation, formation of a pool of free 40S subunits, eukaryotic translation elongation, and eukaryotic translation termination are also noted. Additionally, the data covers peptide chain elongation, viral mRNA translation, and nonsense-mediated decay (NMD) both independent of and enhanced by the exon junction complex (EJC). Pathways related to SRP-dependent cotranslational protein targeting to the membrane and translation are included. Furthermore, the data encompasses influenza infection, the influenza life cycle, influenza viral RNA transcription and replication, and disease.



The analyzed protein sequence has a GRAVY value of -0.647 (21.43rd percentile), indicating a hydrophilic nature. It exhibits a charge of 47.86 at pH 7.0 (99.14th percentile) and a median structural flexibility of 1.005 (65.71st percentile). The protein's secondary structure is predicted to be 33.25% helix (61.51st percentile), 33.75% sheet (50.2nd percentile), and 23.33% turn (11.47th percentile). The instability index is 30.14 (4.1st percentile), with an isoelectric point of 10.19 (97.43rd percentile). The protein has a length of 403 amino acids (45.75th percentile) and a molecular weight of 46108.34 Da (47.07th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |