---
layout: post
title: EIF3M
date: 2025-01-17 16:55 CST
description: EIF3M description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10480) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10480  | EIF3M | ENSG00000149100 | 11p13 |



The gene is involved in the [formation of cytoplasmic translation initiation complex](https://amigo.geneontology.org/amigo/term/GO:0001732) and [cytoplasmic translational initiation](https://amigo.geneontology.org/amigo/term/GO:0002183), contributing to [translation initiation factor activity](https://amigo.geneontology.org/amigo/term/GO:0003743). It enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [translation initiation factor binding](https://amigo.geneontology.org/amigo/term/GO:0031369). The gene is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829) and is part of the [eukaryotic translation initiation factor 3 complex](https://amigo.geneontology.org/amigo/term/GO:0005852), specifically the [eukaryotic translation initiation factor 3 complex, eIF3m](https://amigo.geneontology.org/amigo/term/GO:0071541). Additionally, it is part of the [eukaryotic 43S preinitiation complex](https://amigo.geneontology.org/amigo/term/GO:0016282) and the [eukaryotic 48S preinitiation complex](https://amigo.geneontology.org/amigo/term/GO:0033290), playing a role in [translational initiation](https://amigo.geneontology.org/amigo/term/GO:0006413).


The gene length is 81,626 base pairs (77.76% of all genes), the mature length is 3,889 base pairs (82.79% of all genes), and the primary transcript length is 37,250 base pairs (68.22% of all genes).


EIF3M (Gene ID: 10480) has been mentioned in [7 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22EIF3M%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest publication mentioning EIF3M was in 2005, and the middle 50% of publications occurred between 2008 and 2020.


The top 5 publications mentioning EIF3M, ranked by their relative citation ratio (RCR), include studies that explore the gene's role in various biological processes. The RCR values measure the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. The most influential publication is "[Long non-coding RNA small nucleolar RNA host gene 12 (SNHG12) promotes tumorigenesis and metastasis by targeting miR-199a/b-5p in hepatocellular carcinoma.](https://pubmed.ncbi.nlm.nih.gov/28073380)" (2017) with an RCR of 5.76. Other notable studies include "[PCI proteins eIF3e and eIF3m define distinct translation initiation factor 3 complexes.](https://pubmed.ncbi.nlm.nih.gov/15904532)" (2005) with an RCR of 1.96, "[EIF3m promotes the malignant phenotype of lung adenocarcinoma by the up-regulation of oncogene CAPRIN1.](https://pubmed.ncbi.nlm.nih.gov/33791168)" (2021) with an RCR of 1.03, "[eIF3m expression influences the regulation of tumorigenesis-related genes in human colon cancer.](https://pubmed.ncbi.nlm.nih.gov/20838379)" (2011) with an RCR of 0.96, and "[In Vivo RNAi-Mediated eIF3m Knockdown Affects Ribosome Biogenesis and Transcription but Has Limited Impact on mRNA-Specific Translation.](https://pubmed.ncbi.nlm.nih.gov/31855834)" (2020) with an RCR of 0.89. The citation counts are sourced from [iCite](https://icite.od.nih.gov).


[EIF3M](https://www.proteinatlas.org/ENSG00000149100-EIF3M) is an initiation factor with evidence at the protein level. It is detected in all RNA tissue distributions and is primarily located in the cytosol. The gene is part of various non-specific expression clusters, including Cluster 13 in blood, Cluster 18 in tissue, Cluster 7 in the brain, Cluster 30 in cell lines, and Cluster 15 in single cells. These clusters are associated with mitochondria, ribosomes, gene expression regulation, and mixed functions.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169), [TAF1](https://www.ncbi.nlm.nih.gov/gene/6872), [GATA2](https://www.ncbi.nlm.nih.gov/gene/2624), [TP53](https://www.ncbi.nlm.nih.gov/gene/7157), and [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908), each shown to be regulating in 5 experiments.





The gene is expressed in the ovary and adrenal gland, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in various hematological malignancies, including Burkitt's lymphoma (Raji and Daudi cell lines), chronic myelogenous leukemia (K-562 cell line), and promyelocytic leukemia (HL-60 cell line).




The analyzed protein sequence has a GRAVY value of -0.184 (70.83rd percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is -10.73 (21.42nd percentile), and the median structural flexibility is 0.999 (37.32nd percentile). The protein has a notable affinity for helix (0.385, 90.11th percentile) and sheet (0.377, 76.95th percentile) structures, with a lower affinity for turns (0.235, 12.6th percentile). The instability index is 34.55 (9.31st percentile), and the isoelectric point is 5.41 (16.01st percentile). The protein is 374 amino acids long (41.74th percentile) with a molecular weight of 42502.41 Da (42.84th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |