---
layout: post
title: PAIP1
date: 2025-01-17 16:55 CST
description: PAIP1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10605) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10605  | PAIP1 | ENSG00000172239 | 5p12 |



The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737). It is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829) and is involved in [translational initiation](https://amigo.geneontology.org/amigo/term/GO:0006413) and its [regulation](https://amigo.geneontology.org/amigo/term/GO:0006446). The gene enables [translation activator activity](https://amigo.geneontology.org/amigo/term/GO:0008494) and acts upstream of positive effect on [viral translation](https://amigo.geneontology.org/amigo/term/GO:0019081). It is involved in the [positive regulation by host of viral process](https://amigo.geneontology.org/amigo/term/GO:0044794), [mRNA stabilization](https://amigo.geneontology.org/amigo/term/GO:0048255), and [CRD-mediated mRNA stabilization](https://amigo.geneontology.org/amigo/term/GO:0070934). Additionally, it is part of the [mCRD-mediated mRNA stability complex](https://amigo.geneontology.org/amigo/term/GO:0106002) and is involved in the [negative regulation of nuclear-transcribed mRNA catabolic process, deadenylation-dependent decay](https://amigo.geneontology.org/amigo/term/GO:1900152) and the [positive regulation of cytoplasmic translation](https://amigo.geneontology.org/amigo/term/GO:2000767).


The gene length is 55,396.0 base pairs (69.12% of all genes), the mature length is 562.5 base pairs (3.29% of all genes), and the primary transcript length is 41,338.0 base pairs (70.81% of all genes).


The gene PAIP1 (NCBI ID: 10605) has been mentioned in [17 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PAIP1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 2002 and the middle 50% of publications occurring between 2014 and 2020. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning PAIP1, ranked by their scientific influence, include "[The SARS-unique domain (SUD) of SARS-CoV and SARS-CoV-2 interacts with human Paip1 to enhance viral RNA translation.](https://pubmed.ncbi.nlm.nih.gov/33876849)" (2021) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 2.03, "[Paip1 interacts with poly(A) binding protein through two independent binding motifs.](https://pubmed.ncbi.nlm.nih.gov/11997512)" (2002) with an RCR of 1.9, "[Poly(A)-binding protein-interacting protein 1 binds to eukaryotic translation initiation factor 3 to stimulate translation.](https://pubmed.ncbi.nlm.nih.gov/18725400)" (2008) with an RCR of 1.86, "[Polyadenylate-binding protein-interacting proteins PAIP1 and PAIP2 affect translation termination.](https://pubmed.ncbi.nlm.nih.gov/30992367)" (2019) with an RCR of 0.82, and "[Control of Paip1-eukayrotic translation initiation factor 3 interaction by amino acids through S6 kinase.](https://pubmed.ncbi.nlm.nih.gov/24396066)" (2014) with an RCR of 0.76. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PAIP1](https://www.proteinatlas.org/ENSG00000172239-PAIP1) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in the plasma membrane and cytosol. The gene is expressed in various clusters, including Cluster 8 in blood (Plasmacytoid DCs - Unknown function), Cluster 19 in tissue (Non-specific - Basic cellular processes), Cluster 31 in the brain (Non-specific - Transcription), Cluster 29 in cell lines (Non-specific - Unknown function), and Cluster 44 in single cells (Oocytes - Unknown function).


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [RELA](https://www.ncbi.nlm.nih.gov/gene/5970), [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313), [ZFX](https://www.ncbi.nlm.nih.gov/gene/7543), [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169), and [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908), each shown to be regulating in 5 experiments.





The gene is expressed in various tissues, including muscle and adipose tissue, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the thyroid, CD105+ endothelial cells, amygdala, fetal brain, colon, and small intestine.


The input data indicates that the proteins are involved in several key processes related to gene regulation and mRNA degradation. These include gene expression, deadenylation of mRNA, and deadenylation-dependent mRNA decay.


The mouse ortholog gene 218693 and its human ortholog gene 10605 are associated with enlarged lymph nodes in both sexes and male infertility specifically in males.


The analyzed protein sequence has a GRAVY value of -0.690 (18.21% percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -26.93 (7.22% percentile), and the median structural flexibility is 1.011 (83.76% percentile). The protein has a high instability index of 53.99 (69.29% percentile) and an isoelectric point of 4.71 (3.27% percentile). The sequence length is 479 amino acids (55.78% percentile) with a molecular weight of 53524.02 Da (55.62% percentile). The secondary structure affinities are 0.315 for helix (45.99% percentile), 0.301 for sheet (28.75% percentile), and 0.330 for turn (81.87% percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |