---
layout: post
title: GNB5
date: 2025-01-17 16:55 CST
description: GNB5 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10681) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10681  | GNB5 | ENSG00000069966 | 15q21.2 |



The gene enables [GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0003924) and contributes to [GTPase activator activity](https://amigo.geneontology.org/amigo/term/GO:0005096). It is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), specifically in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). It is part of the [heterotrimeric G-protein complex](https://amigo.geneontology.org/amigo/term/GO:0005834) and the [GTPase activator complex](https://amigo.geneontology.org/amigo/term/GO:1902773). The gene is involved in various processes, including [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), [G protein-coupled receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007186), [dopamine receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007212), [light adaptation](https://amigo.geneontology.org/amigo/term/GO:0036367), [positive regulation of GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0043547), and [negative regulation of voltage-gated calcium channel activity](https://amigo.geneontology.org/amigo/term/GO:1901386). Additionally, it enables [signaling receptor complex adaptor activity](https://amigo.geneontology.org/amigo/term/GO:0030159), [G-protein gamma-subunit binding](https://amigo.geneontology.org/amigo/term/GO:0031682), and [protein-folding chaperone binding](https://amigo.geneontology.org/amigo/term/GO:0051087). The gene is also located in the [presynapse](https://amigo.geneontology.org/amigo/term/GO:0098793) and is involved in [dark adaptation](https://amigo.geneontology.org/amigo/term/GO:1990603).


The gene length is 102,593.0 base pairs (82.24% of all genes), the mature length is 5,446.0 base pairs (92.2% of all genes), and the primary transcript length is 61,213.5 base pairs (79.74% of all genes).


The gene GNB5 (NCBI ID: 10681) has been mentioned in [47 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22GNB5%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1986 and the middle 50% of publications occurring between 2006 and 2018. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning GNB5, ranked by their relative citation ratio (RCR), include "[A G protein gamma subunit-like domain shared between RGS11 and other RGS proteins specifies binding to Gbeta5 subunits.](https://pubmed.ncbi.nlm.nih.gov/9789084)" (1998) (RCR: 4.17), "[Cryo-EM structure of human GPR158 receptor coupled to the RGS7-Gβ5 signaling complex.](https://pubmed.ncbi.nlm.nih.gov/34793198)" (2022) (RCR: 3.59), "[Regulators of G protein signaling 6 and 7. Purification of complexes with gbeta5 and assessment of their effects on g protein-mediated signaling pathways.](https://pubmed.ncbi.nlm.nih.gov/10521509)" (1999) (RCR: 2.25), "[RGS6/Gβ5 complex accelerates IKACh gating kinetics in atrial myocytes and modulates parasympathetic regulation of heart rate.](https://pubmed.ncbi.nlm.nih.gov/20884879)" (2010) (RCR: 1.98), and "[Crystal structure of the multifunctional Gbeta5-RGS9 complex.](https://pubmed.ncbi.nlm.nih.gov/18204463)" (2008) (RCR: 1.79). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[GNB5](https://www.proteinatlas.org/ENSG00000069966-GNB5) functions as a transducer and is implicated in intellectual disability as a disease variant. Evidence supports its presence at the protein level. The gene is detected in all tissues, with notable expression in nuclear speckles, centrosomes, rods, and rings. It is part of Cluster 43 in blood expression, Cluster 28 in tissue expression, Cluster 52 in brain expression, Cluster 58 in cell line expression, and Cluster 25 in single-cell expression, indicating its involvement in transcription, translation, and synaptic function.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169), [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), and [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113), each with 5 experiments. Additionally, [AR](https://www.ncbi.nlm.nih.gov/gene/367) and [SP1](https://www.ncbi.nlm.nih.gov/gene/6667) were found to be regulating in 4 experiments each.





The gene is expressed in various tissues, including the ovary and uterus, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the pineal gland (both day and night), CD19+ B cells (negatively selected), amygdala, whole brain, cerebellum, and cerebellum peduncles.


The input data includes various pathways from Reactome, such as those involved in phototransduction, including its inactivation, recovery, and regulation, as well as diseases associated with visual transduction. Metabolic processes and transmembrane transport of small molecules are also represented. Signaling pathways are prominent, with Wnt signaling, G-protein signaling events (including G alpha, G beta:gamma, and G alpha (12/13) signaling), and ADP signaling through P2Y purinoceptors. Additionally, the data covers thrombin and thromboxane signaling, glutamate receptor activation, prostacyclin signaling, and the regulation of insulin secretion by glucagon-like peptide-1 (GLP1) and adrenaline/noradrenaline. Other notable pathways include aquaporin-mediated transport, vasopressin regulation of renal water homeostasis, and various GPCR signaling events. The data also encompasses pathways related to hemostasis, platelet homeostasis, and neuronal systems, including transmission across chemical synapses and neurotransmitter receptor binding.


The mouse gene with NCBI ID 14697 has a human ortholog with NCBI ID 10681. This gene is associated with the phenotype of preweaning lethality with incomplete penetrance, observed in both sexes.


The protein sequence analyzed has a GRAVY value of -0.231 (66.09th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -5.674 (31.95th percentile). The median structural flexibility is 0.999 (36.26th percentile), with a fraction of amino acids in helix, sheet, and turn structures being 0.306 (37.42nd percentile), 0.329 (44.84th percentile), and 0.294 (59.0th percentile), respectively. The instability index is 38.624 (18.49th percentile), and the isoelectric point is 6.016 (31.01st percentile). The protein has a length of 395 amino acids (44.6th percentile) and a molecular weight of 43565.874 Da (44.06th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |