---
layout: post
title: KIF2A
date: 2025-01-17 16:55 CST
description: KIF2A description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3796) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3796  | KIF2A | ENSG00000068796 | 5q12.1 |



The gene is involved in [microtubule cytoskeleton organization](https://amigo.geneontology.org/amigo/term/GO:0000226) and colocalizes with the [spindle pole](https://amigo.geneontology.org/amigo/term/GO:0000922). It enables [cytoskeletal motor activity](https://amigo.geneontology.org/amigo/term/GO:0003774), [microtubule motor activity](https://amigo.geneontology.org/amigo/term/GO:0003777), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524). The gene is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [nucleolus](https://amigo.geneontology.org/amigo/term/GO:0005730), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [centrosome](https://amigo.geneontology.org/amigo/term/GO:0005813), [centriole](https://amigo.geneontology.org/amigo/term/GO:0005814), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), and [nuclear body](https://amigo.geneontology.org/amigo/term/GO:0016604). It is active in the [centrosome](https://amigo.geneontology.org/amigo/term/GO:0005813) and [spindle](https://amigo.geneontology.org/amigo/term/GO:0005819), and is part of the [kinesin complex](https://amigo.geneontology.org/amigo/term/GO:0005871) and [centriolar subdistal appendage](https://amigo.geneontology.org/amigo/term/GO:0120103). The gene colocalizes with the [microtubule](https://amigo.geneontology.org/amigo/term/GO:0005874) and [spindle microtubule](https://amigo.geneontology.org/amigo/term/GO:0005876). It is involved in [microtubule-based movement](https://amigo.geneontology.org/amigo/term/GO:0007018), [microtubule depolymerization](https://amigo.geneontology.org/amigo/term/GO:0007019), [mitotic spindle organization](https://amigo.geneontology.org/amigo/term/GO:0007052), [nervous system development](https://amigo.geneontology.org/amigo/term/GO:0007399), [cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0030154), [regulation of cell migration](https://amigo.geneontology.org/amigo/term/GO:0030334), [cell division](https://amigo.geneontology.org/amigo/term/GO:0051301), and [mitotic spindle assembly](https://amigo.geneontology.org/amigo/term/GO:0090307). Additionally, it enables [microtubule binding](https://amigo.geneontology.org/amigo/term/GO:0008017) and [ATP hydrolysis activity](https://amigo.geneontology.org/amigo/term/GO:0016887).


The gene length is 2,359 base pairs (6.32% of all genes), the mature length is 847.5 base pairs, and the primary transcript length is 2,345.5 base pairs.


KIF2A (Gene ID: 3796) has been mentioned in [102 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22KIF2A%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1997 and the middle 50% of publications occurring between 2013 and 2019. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning KIF2A, ranked by their relative citation ratio (RCR), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows: [Mutations in TUBG1, DYNC1H1, KIF5C and KIF2A cause malformations of cortical development and microcephaly.](https://pubmed.ncbi.nlm.nih.gov/23603762) (2013) (RCR: 8.54), [Modeling microcephaly with cerebral organoids reveals a WDR62-CEP170-KIF2A pathway promoting cilium disassembly in neural progenitors.](https://pubmed.ncbi.nlm.nih.gov/31197141) (2019) (RCR: 4.84), [Ciliary entry of the kinesin-2 motor KIF17 is regulated by importin-beta2 and RanGTP.](https://pubmed.ncbi.nlm.nih.gov/20526328) (2010) (RCR: 4.49), [Ciliary protein trafficking mediated by IFT and BBSome complexes with the aid of kinesin-2 and dynein-2 motors.](https://pubmed.ncbi.nlm.nih.gov/29272450) (2018) (RCR: 3.56), and [The kinesin-13 proteins Kif2a, Kif2b, and Kif2c/MCAK have distinct roles during mitosis in human cells.](https://pubmed.ncbi.nlm.nih.gov/17538014) (2007) (RCR: 3.3). The citation count is sourced from [iCite](https://icite.od.nih.gov).


[KIF2A](https://www.proteinatlas.org/ENSG00000068796-KIF2A) is a developmental and motor protein with evidence at the protein level. It is implicated in cancer-related genes and is a disease variant associated with lissencephaly. The gene is detected in all RNA tissue distributions and is localized in various subcellular locations, including the nucleoplasm, nucleoli, mitotic spindle, centrosome, basal body, cytosol, mid piece, principal piece, and annulus. KIF2A is expressed in T-cells with unknown function (Cluster 1) and is involved in transcription in non-specific tissues (Cluster 16). In the brain, it is found in oligodendrocytes with mixed function (Cluster 28), and in cell lines, it is involved in mRNA splicing and cell cycle (Cluster 22). Additionally, KIF2A is expressed in platelets, playing a role in hemostasis (Cluster 57).


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [TAL1](https://www.ncbi.nlm.nih.gov/gene/6886), [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046), [SP1](https://www.ncbi.nlm.nih.gov/gene/6667), [CDK8](https://www.ncbi.nlm.nih.gov/gene/1024), and [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025), each shown to be regulating in 5 experiments.



The GWAS data indicates associations with various conditions, including cognitive disorder, disease of mental health, psychotic disorder, and schizophrenia.



The gene is expressed in various tissues, including the brain, pituitary, and blood, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in 721_B_lymphoblasts, CD8+_Tcells, Leukemia_chronicMyelogenousK-562, Amygdala, Fetalbrain, and TestisIntersitial.


The proteins are involved in various cellular processes, including the cell cycle, particularly the mitotic phase and its sub-phases such as anaphase, metaphase, and prometaphase. They also play roles in the separation and resolution of sister chromatids. Additionally, these proteins participate in the adaptive immune system, specifically in MHC class II antigen presentation, and are involved in hemostasis and the development of megakaryocytes and platelet production. Furthermore, they are associated with kinesins, which are crucial for intracellular transport and cell division.


The mouse ortholog gene 16563 and its human ortholog gene 3796 are associated with several phenotypes, including abnormal placenta morphology, embryonic growth retardation, and preweaning lethality with complete penetrance. These phenotypes are observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.647 (21.42 percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -4.412 (35.19 percentile), and the median structural flexibility is 1.009 (79.45 percentile). The protein has a predicted secondary structure with 32.58% helix (55.8 percentile), 31.16% sheet (34.86 percentile), and 27.62% turn (43.89 percentile) affinities. The instability index is 47.84 (49.52 percentile), and the isoelectric point is 6.28 (37.51 percentile). The protein is 706 amino acids long (75.64 percentile) with a molecular weight of 79953.63 Da (76.23 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |