---
layout: post
title: UBE2M
date: 2025-01-17 16:55 CST
description: UBE2M description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9040) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9040  | UBE2M | ENSG00000130725 | 19q13.43 |



The gene enables [ubiquitin-protein transferase activity](https://amigo.geneontology.org/amigo/term/GO:0004842) and [NEDD8 transferase activity](https://amigo.geneontology.org/amigo/term/GO:0019788), and is involved in [protein modification process](https://amigo.geneontology.org/amigo/term/GO:0036211), [post-translational protein modification](https://amigo.geneontology.org/amigo/term/GO:0043687), and [protein neddylation](https://amigo.geneontology.org/amigo/term/GO:0045116). It is active in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654) and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). Additionally, it enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524), and [NEDD8 conjugating enzyme activity](https://amigo.geneontology.org/amigo/term/GO:0061654). The gene also plays a role in the [positive regulation of neuron apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0043525).


The gene length is 222,350 base pairs (93.55% of all genes), the mature length is 2,855 base pairs, and the primary transcript length is 220,423 base pairs.


The gene UBE2M, with NCBI ID 9040, has been mentioned in [25 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22UBE2M%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727). The earliest year of publication is 2000, and the middle 50% of publications occurred between 2018 and 2020.


The top 5 publications mentioning UBE2M, ranked by their relative citation ratio (RCR), include "[Structural basis for recruitment of Ubc12 by an E2 binding domain in NEDD8's E1.](https://pubmed.ncbi.nlm.nih.gov/15694336)" (2005) (RCR: 2.44), "[Development of Highly Potent, Selective, and Cellular Active Triazolo[1,5- a]pyrimidine-Based Inhibitors Targeting the DCN1-UBC12 Protein-Protein Interaction.](https://pubmed.ncbi.nlm.nih.gov/30803229)" (2019) (RCR: 2.22), "[A potent small-molecule inhibitor of the DCN1-UBC12 interaction that selectively blocks cullin 3 neddylation.](https://pubmed.ncbi.nlm.nih.gov/29074978)" (2017) (RCR: 2.16), "[UBE2M Is a Stress-Inducible Dual E2 for Neddylation and Ubiquitylation that Promotes Targeted Degradation of UBE2F.](https://pubmed.ncbi.nlm.nih.gov/29932898)" (2018) (RCR: 2.03), and "[Validation of NEDD8-conjugating enzyme UBC12 as a new therapeutic target in lung cancer.](https://pubmed.ncbi.nlm.nih.gov/31208947)" (2019) (RCR: 1.57). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[UBE2M](https://www.proteinatlas.org/ENSG00000130725-UBE2M) is a transferase with evidence at the protein level. It is detected in all tissues and is primarily localized in the nucleoplasm, nucleoli, nuclear bodies, and cytosol. UBE2M is involved in basic cellular processes and is expressed in various clusters related to innate immune response, transcription, and DNA binding. Notably, it is also associated with neurons and synaptic function in the brain. The gene's expression in cancer prognostics is generally unprognostic across multiple cancer types, including bladder urothelial carcinoma, breast invasive carcinoma, and others.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313) with 6 experiments, [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930) and [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) each with 5 experiments, and [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908) and [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613) each with 4 experiments.





The gene is expressed in various tissues, including the brain, pituitary, and adrenal gland, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the Prefrontal Cortex, Heart, Whole Brain, Temporal Lobe, and Colorectal Adenocarcinoma.


The input data highlights various pathways involving TGF-beta receptor signaling and its components, particularly focusing on cancer-related mutations and loss of function. Key pathways include 'TGF-beta receptor signaling activates SMADs', 'Loss of Function of SMAD2/3 in Cancer', 'TGFBR2 Kinase Domain Mutants in Cancer', and 'Signaling by TGF-beta Receptor Complex in Cancer'. Additionally, the data covers immune system processes such as 'Antigen processing: Ubiquitination & Proteasome degradation', 'Class I MHC mediated antigen processing & presentation', and broader categories like 'Adaptive Immune System', 'Immune System', and 'Disease'. Signal transduction pathways are also noted.


The mouse ortholog gene 22192 and its human ortholog gene 9040 are associated with phenotypes including embryonic lethality prior to organogenesis, which is not observed in both sexes, and preweaning lethality with complete penetrance, which is observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.517 (33.21% percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of 0.651 (52.49% percentile). The median structural flexibility is 1.008 (74.03% percentile), with a propensity for helix, sheet, and turn structures of 0.328 (57.57% percentile), 0.339 (51.12% percentile), and 0.311 (71.52% percentile), respectively. The instability index is 40.48 (23.62% percentile), and the isoelectric point is 7.57 (54.83% percentile). The protein has a length of 183 amino acids (11.98% percentile) and a molecular weight of 20899.73 Da (12.55% percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |