---
layout: post
title: PAPSS1
date: 2025-01-17 16:55 CST
description: PAPSS1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9061) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9061  | PAPSS1 | ENSG00000138801 | 4q25 |



The gene is involved in [sulfate assimilation](https://amigo.geneontology.org/amigo/term/GO:0000103) and [skeletal system development](https://amigo.geneontology.org/amigo/term/GO:0001501). It enables [adenylylsulfate kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004020), [sulfate adenylyltransferase (ATP) activity](https://amigo.geneontology.org/amigo/term/GO:0004781), [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524), [nucleotidyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0016779), and [protein homodimerization activity](https://amigo.geneontology.org/amigo/term/GO:0042803). Additionally, it is involved in [phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0016310) and the [3'-phosphoadenosine 5'-phosphosulfate biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0050428). The gene is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829).


The gene length is 97,324.0 base pairs (81.26% of all genes), the mature length is 592.5 base pairs (5.17% of all genes), and the primary transcript length is 51,216.5 base pairs (75.87% of all genes).


The gene PAPSS1 (NCBI ID: 9061) has been mentioned in [4 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PAPSS1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 2000 and the middle 50% of publications ranging from 2000 to 2012. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top publications mentioning PAPSS1 include "[Nuclear localization of PAPS synthetase 1: a sulfate activation pathway in the nucleus of eukaryotic cells.](https://pubmed.ncbi.nlm.nih.gov/10657990)" (2000) (relative citation ratio: 0.98), "[Evidence for association with hepatocellular carcinoma at the PAPSS1 locus on chromosome 4q25 in a family-based study.](https://pubmed.ncbi.nlm.nih.gov/19337310)" (2009) (relative citation ratio: 0.4), "[Transcriptional regulation of human 3'-phosphoadenosine 5'-phosphosulfate synthase 1.](https://pubmed.ncbi.nlm.nih.gov/11396968)" (2001) (relative citation ratio: 0.13), and "[Role of aspartic acid residues D87 and D89 in APS kinase domain of human 3'-phosphoadenosine 5'-phosphosulfate synthase 1 and 2b: A commonality with phosphatases/kinases.](https://pubmed.ncbi.nlm.nih.gov/34712849)" (2021) (relative citation ratio: 0.13). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PAPSS1](https://www.proteinatlas.org/ENSG00000138801-PAPSS1) is a multifunctional enzyme with kinase, nucleotidyltransferase, and transferase activities. It is detected in all tissues and is primarily localized in the nucleoplasm, nucleoli rim, and mitotic chromosome. PAPSS1 is expressed in various cell types, including basophils, oligodendrocytes, and glandular cells, with notable expression in white matter related to myelination. The gene's expression is also observed in rhabdoid cell lines. Evidence for PAPSS1 is established at the protein level.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169), [RUNX1T1](https://www.ncbi.nlm.nih.gov/gene/862), and [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227), each with 6 experiments. Additionally, [RELA](https://www.ncbi.nlm.nih.gov/gene/5970) and [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613) were found to be regulating in 5 experiments each.


The transcription factor [GATA1](https://www.ncbi.nlm.nih.gov/gene/9968) was found to be regulating genes in one Perturb-seq experiment, according to data from GTRD's database of transcription factor binding sites and the study published in [PubMed](https://pubmed.ncbi.nlm.nih.gov/35688146/).


The GWAS data indicates associations with various disease conditions, including metabolic disorders such as diabetes mellitus and type 2 diabetes mellitus, as well as cardiovascular diseases like coronary artery disease, artery disease, vascular disease, and cerebrovascular disease. Additionally, the data highlights associations with musculoskeletal system diseases and muscular diseases.



The gene is expressed in various tissues, including the salivary gland, brain, and stomach, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the fetal brain, hypothalamus, and spinal cord.


The input data includes various pathways and diseases related to carbohydrate metabolism, glycosaminoglycan metabolism, and biological oxidations. Notable conditions include Myoclonic epilepsy of Lafora, several types of Mucopolysaccharidoses (MPS) such as Sly syndrome, Maroteaux-Lamy syndrome, and Morquio syndrome, as well as defects in genes like B3GAT3, B4GALT1, and CHST14 leading to specific disorders. Additionally, the data covers diseases associated with glycosylation, phase II conjugation, and metabolic disorders of biological oxidation enzymes, highlighting defects in genes like AHCY, UGT1A1, and GCLC.


The mouse ortholog gene 23971 and its human ortholog gene 9061 are associated with the phenotype of preweaning lethality with complete penetrance, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.413 (44.61% percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -4.78 (34.33% percentile), and the median structural flexibility is 0.998 (34.4% percentile). The protein has a propensity for helix and sheet structures, with affinities of 0.340 (67.67% percentile) and 0.348 (57.4% percentile), respectively, and a turn affinity of 0.252 (22.71% percentile). The instability index is 37.45 (15.23% percentile), and the isoelectric point is 6.40 (40.23% percentile). The protein is 624 amino acids long (70.35% percentile) with a molecular weight of 70832.36 Da (71.09% percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |