---
layout: post
title: NUP88
date: 2025-01-17 16:55 CST
description: NUP88 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4927) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4927  | NUP88 | ENSG00000108559 | 17p13.2 |



The gene is involved in [ribosomal large subunit export from nucleus](https://amigo.geneontology.org/amigo/term/GO:0000055) and [ribosomal small subunit export from nucleus](https://amigo.geneontology.org/amigo/term/GO:0000056). It enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is involved in [mRNA export from nucleus](https://amigo.geneontology.org/amigo/term/GO:0006406), [protein import into nucleus](https://amigo.geneontology.org/amigo/term/GO:0006606), and [nucleocytoplasmic transport](https://amigo.geneontology.org/amigo/term/GO:0006913). The gene is located in the [nuclear envelope](https://amigo.geneontology.org/amigo/term/GO:0005635), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and is part of the [nuclear pore](https://amigo.geneontology.org/amigo/term/GO:0005643). Additionally, it enables the [structural constituent of nuclear pore](https://amigo.geneontology.org/amigo/term/GO:0017056).


The gene length is 41,825 base pairs (61.48% of all genes), the mature length is 616 base pairs (6.16% of all genes), and the primary transcript length is 15,912 base pairs (45.66% of all genes).


The gene NUP88 (NCBI ID: 4927) has been mentioned in [33 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22NUP88%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1997 and the middle 50% of publications occurring between 2006 and 2012. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning NUP88, ranked by their scientific influence, include studies that explore the gene's role in nuclear pore complexes and its association with cancer. The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Notable publications include "[The human homologue of yeast CRM1 is in a dynamic subcomplex with CAN/Nup214 and a novel nuclear pore component Nup88.](https://pubmed.ncbi.nlm.nih.gov/9049309)" (1997) (relative citation ratio: 6.25), "[Nup358/RanBP2 attaches to the nuclear pore complex via association with Nup88 and Nup214/CAN and plays a supporting role in CRM1-mediated nuclear protein export.](https://pubmed.ncbi.nlm.nih.gov/14993277)" (2004) (relative citation ratio: 2.12), and "[Nup88 mRNA overexpression is associated with high aggressiveness of breast cancer.](https://pubmed.ncbi.nlm.nih.gov/14999780)" (2004) (relative citation ratio: 1.3). Additionally, "[Nup214-Nup88 nucleoporin subcomplex is required for CRM1-mediated 60 S preribosomal nuclear export.](https://pubmed.ncbi.nlm.nih.gov/16675447)" (2006) (relative citation ratio: 1.09) and "[The nuclear pore complex protein Nup88 is overexpressed in tumor cells.](https://pubmed.ncbi.nlm.nih.gov/10554006)" (1999) (relative citation ratio: 1.08) are also significant. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[NUP88](https://www.proteinatlas.org/ENSG00000108559-NUP88) is a gene with evidence at the protein level and is implicated in disease variants. Its RNA is detected in all tissues, with notable expression in the nucleoplasm. The gene is particularly expressed in B-cells within the blood, testis for basic cellular processes, cerebellum for nucleic acid binding, and spermatocytes and spermatids for spermatid development. However, it is unprognostic across various cancer types, including Bladder Urothelial Carcinoma, Breast Invasive Carcinoma, and others.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2B](https://www.ncbi.nlm.nih.gov/gene/9757) with 6 experiments, [STAT5B](https://www.ncbi.nlm.nih.gov/gene/6777) with 5 experiments, [SP1](https://www.ncbi.nlm.nih.gov/gene/6667) with 5 experiments, [KLF5](https://www.ncbi.nlm.nih.gov/gene/688) with 5 experiments, and [CREB1](https://www.ncbi.nlm.nih.gov/gene/1385) with 5 experiments.



The GWAS data indicates associations with various metabolic and nutritional conditions, including diseases of metabolism, acquired metabolic diseases, nutrition diseases, overnutrition, and obesity.



The gene is expressed in muscle and thyroid tissues according to the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles indicate expression in CD19+ B cells (neg. sel.), Burkitt's lymphoma (Daudi), CD71+ early erythroid cells, and various testis tissues including interstitial, Leydig cells, seminiferous tubules, and germ cells.


The input data highlights various biological pathways and processes, including the metabolism of carbohydrates, glycogen storage diseases, and gene expression. It also covers transmembrane transport mechanisms, such as SLC-mediated transport and the transport of small molecules. Additionally, the data includes pathways related to influenza infection, its life cycle, and antiviral mechanisms involving IFN-stimulated genes and ISG15. Several pathways related to HIV infection and its life cycle are also noted, including interactions with host cellular proteins and the nuclear export of viral RNA. Furthermore, the data encompasses glucose and hexose transport, the regulation of glucokinase, and various stages of the cell cycle, including mitotic prophase and nuclear envelope breakdown. Immune system processes, such as cytokine signaling and interferon signaling, are also represented, along with regulatory RNA pathways and the processing of pre-mRNA.



The analyzed protein sequence has a GRAVY value of -0.288 (59.3% percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -23.52 (8.93% percentile), and the median structural flexibility is 1.002 (51.24% percentile). The protein has a high affinity for helix and sheet structures, with values of 0.356 (78.7% percentile) and 0.356 (63.72% percentile) respectively, and a lower affinity for turn structures at 0.271 (39.08% percentile). The instability index is 58.50 (80.11% percentile), suggesting potential instability. The isoelectric point is 5.47 (17.45% percentile), with a length of 741 amino acids (77.74% percentile) and a molecular weight of 83540.95 Da (78.12% percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |