---
layout: post
title: CHAD
date: 2025-01-17 16:55 CST
description: CHAD description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1101) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1101  | CHAD | ENSG00000136457 | 17q21.33 |



The gene is active in the [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615) and [extracellular matrix](https://amigo.geneontology.org/amigo/term/GO:0031012). It is involved in [bone development](https://amigo.geneontology.org/amigo/term/GO:0060348) and plays a role in the [negative regulation of bone trabecula formation](https://amigo.geneontology.org/amigo/term/GO:1900155).


The gene length is 31,803.0 base pairs (54.32% of all genes), the mature length is 3,096.5 base pairs (73.8% of all genes), and the primary transcript length is 21,988.0 base pairs (54.23% of all genes).


The gene CHAD, with NCBI ID 1101, has been mentioned in [41 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CHAD%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication mentioning CHAD is 1997, and the middle 50% of publications occurred between 2013 and 2017.


The top 5 publications mentioning CHAD, ranked by their scientific influence, include "[Effect of a serogroup A meningococcal conjugate vaccine (PsA-TT) on serogroup A meningococcal meningitis and carriage in Chad: a community study [corrected].](https://pubmed.ncbi.nlm.nih.gov/24035220)" (2014) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 8.36, "[Can vaccination coverage be improved by reducing missed opportunities for vaccination? Findings from assessments in Chad and Malawi using the new WHO methodology.](https://pubmed.ncbi.nlm.nih.gov/30677072)" (2019) with an RCR of 2.85, "[Human and animal vaccination delivery to remote nomadic families, Chad.](https://pubmed.ncbi.nlm.nih.gov/17552089)" (2007) with an RCR of 2.55, "[Rapid Contraceptive Uptake and Changing Method Mix With High Use of Long-Acting Reversible Contraceptives in Crisis-Affected Populations in Chad and the Democratic Republic of the Congo.](https://pubmed.ncbi.nlm.nih.gov/27540125)" (2016) with an RCR of 2.09, and "[Integrin alpha2beta1 is a receptor for the cartilage matrix protein chondroadherin.](https://pubmed.ncbi.nlm.nih.gov/9281592)" (1997) with an RCR of 1.87. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CHAD](https://www.proteinatlas.org/ENSG00000136457-CHAD) is a gene with evidence at the protein level. Its RNA is detected in many tissues and is particularly expressed in B-cells within the blood, aligning with Cluster 17 for immunoglobulins. In terms of tissue expression, CHAD is associated with Cluster 68, indicating non-specific protein processing. Its brain expression falls under Cluster 15, related to transcription, and in cell lines, it is found in Cluster 27, associated with breast cancer but with an unknown function. At the single-cell level, CHAD is expressed in intestinal goblet cells, contributing to mucin production. The gene is unprognostic for several cancer types, including Breast Invasive Carcinoma, Cervical Squamous Cell Carcinoma, Kidney Renal Clear Cell Carcinoma, Liver Hepatocellular Carcinoma, Lung Adenocarcinoma, Pancreatic Adenocarcinoma, Rectum Adenocarcinoma, Skin Cutaneous Melanoma, Stomach Adenocarcinoma, and Testicular Germ Cell Tumor.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 5 experiments, [EZH2](https://www.ncbi.nlm.nih.gov/gene/2146) with 4 experiments, [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) with 3 experiments, [SUZ12](https://www.ncbi.nlm.nih.gov/gene/23512) with 3 experiments, and [CAT](https://www.ncbi.nlm.nih.gov/gene/847) with 3 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.





The gene is expressed in a variety of tissues, including the liver, heart, whole blood, cerebellum peduncles, trachea, and adrenal gland. Additionally, it is expressed in Burkitt's lymphoma (Raji) cells.




The analyzed protein sequence has a GRAVY value of -0.2398 (65.06th percentile), indicating a hydrophilic nature. It exhibits a charge of 15.57 at pH 7.0 (89.56th percentile) and a median structural flexibility of 0.995 (21.21st percentile). The protein's secondary structure is predicted to be 37.05% helix (85.36th percentile), 35.93% sheet (65.77th percentile), and 27.86% turn (45.91st percentile). The instability index is 49.16 (54.41st percentile), with an isoelectric point of 9.49 (90.91st percentile). The protein has a length of 359 amino acids (39.48th percentile) and a molecular weight of 40475.34 Da (40.06th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |