---
layout: post
title: AIF1
date: 2024-12-08 23:57 CST
description: AIF1 description
tags: [cooccuring-genes]
categories: longest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/199) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 199  | AIF1 | ENSG00000204472 | 6p21.33 |



The gene has a length of 566,190 base pairs (98.72% of all genes), a mature length of 8,308 base pairs, and a primary transcript length of 565,620 base pairs (99.43% of all genes).


The gene AIF1 has an NCBI Gene ID of 199. It has been featured in [136 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22AIF1%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1996, and the middle 50% of publications occurred between 2007 and 2019.


The top five publications mentioning AIF1, ranked by their scientific influence, include "[Sequential activation of microglia and astrocyte cytokine expression precedes increased Iba-1 or GFAP immunoreactivity following systemic immune challenge.](https://pubmed.ncbi.nlm.nih.gov/26470014)" (2016) (relative citation ratio: 15.41), "[Metal Sensing by the IRT1 Transporter-Receptor Orchestrates Its Own Degradation and Plant Metal Nutrition.](https://pubmed.ncbi.nlm.nih.gov/29547723)" (2018) (relative citation ratio: 9.47), "[Co-expression patterns of microglia markers Iba1, TMEM119 and P2RY12 in Alzheimer's disease.](https://pubmed.ncbi.nlm.nih.gov/35247551)" (2022) (relative citation ratio: 8.07), "[Staining of HLA-DR, Iba1 and CD68 in human microglia reveals partially overlapping expression depending on cellular morphology and pathology.](https://pubmed.ncbi.nlm.nih.gov/28601280)" (2017) (relative citation ratio: 6.86), and "[Involvement of Iba1 in membrane ruffling and phagocytosis of macrophages/microglia.](https://pubmed.ncbi.nlm.nih.gov/10934045)" (2000) (relative citation ratio: 6.01). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 4 experiments, followed by [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861), [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313), and [KDM1A](https://www.ncbi.nlm.nih.gov/gene/23028), each with 3 experiments.


Based on the data from GTRD's database of transcription factor (TF) binding sites and filtered for Perturb-seq data, the top transcription factors in terms of the number of experiments showing their regulatory role are as follows: [23028](https://www.ncbi.nlm.nih.gov/gene/23028) with 3 experiments, [2623](https://www.ncbi.nlm.nih.gov/gene/2623) with 2 experiments, and both [4066](https://www.ncbi.nlm.nih.gov/gene/4066) and [3190](https://www.ncbi.nlm.nih.gov/gene/3190) with 1 experiment each. These findings highlight the significant regulatory roles of these transcription factors in gene expression.


AIF1 has been associated with variations in [Mean platelet volume](https://pubmed.ncbi.nlm.nih.gov/32888494), which maps closely to the MeSH term [Mean Platelet Volume](https://meshb.nlm.nih.gov/record/ui?ui=D063847). Additionally, it has been linked to [Platelet count](https://pubmed.ncbi.nlm.nih.gov/32888494), corresponding to the MeSH term [Platelet Count](https://meshb.nlm.nih.gov/record/ui?ui=D010976). These associations were identified using a method called [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0).


The mouse ortholog gene 11629 and its human ortholog gene 199 are associated with abnormal optic disk morphology and decreased blood urea nitrogen levels, with these phenotypes observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.699 (17.67th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -1.38 (45.15th percentile), and the median structural flexibility is 1.022 (97.62nd percentile). The protein has a high affinity for helix structures (0.463, 98.9th percentile), with lower affinities for sheet (0.286, 21.03rd percentile) and turn (0.299, 63.41st percentile) structures. The instability index is 52.45 (64.99th percentile), and the isoelectric point is 5.97 (29.83rd percentile). The protein is 147 amino acids long (7.7th percentile) with a molecular weight of 16703.13 Da (8.01st percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |