---
layout: post
title: CDYL
date: 2025-01-17 16:55 CST
description: CDYL description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9425) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9425  | CDYL | ENSG00000153046 | 6p25.1 |



The gene enables [chromatin binding](https://amigo.geneontology.org/amigo/term/GO:0003682) and [transcription corepressor activity](https://amigo.geneontology.org/amigo/term/GO:0003714), and is involved in [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [chromosome](https://amigo.geneontology.org/amigo/term/GO:0005694), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), and [nuclear speck](https://amigo.geneontology.org/amigo/term/GO:0016607). The gene is involved in processes such as [spermatogenesis](https://amigo.geneontology.org/amigo/term/GO:0007283), [spermatid development](https://amigo.geneontology.org/amigo/term/GO:0007286), [negative regulation of DNA-templated transcription](https://amigo.geneontology.org/amigo/term/GO:0045892), [random inactivation of X chromosome](https://amigo.geneontology.org/amigo/term/GO:0060816), and [negative regulation of peptidyl-lysine crotonylation](https://amigo.geneontology.org/amigo/term/GO:0120094). Additionally, it enables [protein-macromolecule adaptor activity](https://amigo.geneontology.org/amigo/term/GO:0030674), [methylated histone binding](https://amigo.geneontology.org/amigo/term/GO:0035064), and [crotonyl-CoA hydratase activity](https://amigo.geneontology.org/amigo/term/GO:0120092).


The gene length is 4,703 base pairs (13.17% of all genes), the mature length is 1,204 base pairs (29.15% of all genes), and the primary transcript length is 3,738 base pairs (13.8% of all genes).


The gene CDYL (NCBI ID: 9425) has been mentioned in [18 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CDYL%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 2008 and the middle 50% of publications occurring between 2017 and 2020. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning CDYL, ranked by their scientific influence, include studies that explore the gene's role in various biological processes. The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Notable publications include "[Circular RNA Cdyl promotes abdominal aortic aneurysm formation by inducing M1 macrophage polarization and M1-type inflammation.](https://pubmed.ncbi.nlm.nih.gov/34547461)" (2022) (relative citation ratio: 8.3), "[Autophagy-associated circRNA circCDYL augments autophagy and promotes breast cancer progression.](https://pubmed.ncbi.nlm.nih.gov/32213200)" (2020) (relative citation ratio: 8.29), "[Chromodomain Protein CDYL Acts as a Crotonyl-CoA Hydratase to Regulate Histone Crotonylation and Spermatogenesis.](https://pubmed.ncbi.nlm.nih.gov/28803779)" (2017) (relative citation ratio: 4.58), "[Global crotonylome reveals CDYL-regulated RPA1 crotonylation in homologous recombination-mediated DNA repair.](https://pubmed.ncbi.nlm.nih.gov/32201722)" (2020) (relative citation ratio: 3.44), and "[Circular RNA circ-CDYL sponges miR-1180 to elevate yes-associated protein in multiple myeloma.](https://pubmed.ncbi.nlm.nih.gov/32321304)" (2020) (relative citation ratio: 2.14). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CDYL](https://www.proteinatlas.org/ENSG00000153046-CDYL) is a gene that functions as a lyase and repressor, with evidence supporting its role at the protein level. It is detected in all RNA tissue distributions and is primarily localized to nuclear speckles. CDYL is part of blood expression cluster 30, which is involved in non-specific DNA binding, and tissue expression cluster 19, which is associated with basic cellular processes. In the brain, it is part of expression cluster 39, which is linked to astrocytes with mixed functions. Additionally, CDYL is found in cell line expression cluster 28, which is involved in nuclear processes, and single cell expression cluster 28, which is associated with dendritic cells and antigen presentation.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) with 5 experiments, [E2F1](https://www.ncbi.nlm.nih.gov/gene/1869) with 4 experiments, [TAL1](https://www.ncbi.nlm.nih.gov/gene/6886) with 4 experiments, [KLF5](https://www.ncbi.nlm.nih.gov/gene/688) with 4 experiments, and [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with a range of respiratory conditions, including asthma, bronchial disease, obstructive lung disease, lower respiratory tract disease, lung disease, and respiratory system disease. Additionally, the data highlights metabolic disorders such as disease of metabolism, acquired metabolic disease, carbohydrate metabolism disease, glucose metabolism disease, diabetes mellitus, and type 2 diabetes mellitus. Cardiovascular issues are also implicated, with associations found for artery disease, vascular disease, cardiovascular system disease, and hypertension. Furthermore, the data includes chronic obstructive pulmonary disease, physical disorder, and orofacial cleft.



The gene is expressed in a variety of tissues and cell types, including 721_B_lymphoblasts, CD34+ cells, BDCA4+ dendritic cells, CD56+ NK cells, CD105+ endothelial cells, CD71+ early erythroid cells, and placenta.




The analyzed protein sequence has a GRAVY value of -0.565 (28.43rd percentile), indicating a hydrophilic nature. It exhibits a charge of 22.1 at pH 7.0 (94.55th percentile) and a median structural flexibility of 1.008 (76.1st percentile). The protein's secondary structure is predicted to be 31.6% helix (46.76th percentile), 30.9% sheet (33.52nd percentile), and 30.1% turn (64.57th percentile). The instability index is 46.1 (43.35th percentile), with an isoelectric point of 9.5 (91.06th percentile). The protein has a length of 598 amino acids (68.44th percentile) and a molecular weight of 66480.9 Da (68.18th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |