---
layout: post
title: FLI1
date: 2024-12-08 23:57 CST
description: FLI1 description
tags: [cooccuring-genes]
categories: shortest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2313) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2313  | FLI1 | ENSG00000151702 | 11q24.3 |



The gene length is 686.0 base pairs (0.81 percentile of all genes), the mature length is 660.0 base pairs, and the primary transcript length is 660.0 base pairs.


The gene FLI1 (NCBI ID: 2313) has been the subject of [437 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22FLI1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication dating back to 1990. The middle 50% of these publications were published between 2003 and 2017. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning FLI1, ranked by their scientific influence, include "[Immunohistochemical expression of endothelial markers CD31, CD34, von Willebrand factor, and Fli-1 in normal human tissues.](https://pubmed.ncbi.nlm.nih.gov/16234507)" (2006) with a relative citation ratio of 13.19, "[A novel FLI1 exonic circular RNA promotes metastasis in breast cancer by coordinately regulating TET1 and DNMT1.](https://pubmed.ncbi.nlm.nih.gov/30537986)" (2018) with a relative citation ratio of 10.58, "[EWS-FLI1 fusion transcript structure is an independent determinant of prognosis in Ewing's sarcoma.](https://pubmed.ncbi.nlm.nih.gov/9552022)" (1998) with a relative citation ratio of 8.03, "[Erythroleukemia induction by Friend murine leukemia virus: insertional activation of a new member of the ets gene family, Fli-1, closely linked to c-ets-1.](https://pubmed.ncbi.nlm.nih.gov/2044959)" (1991) with a relative citation ratio of 7.26, and "[A small molecule blocking oncogenic protein EWS-FLI1 interaction with RNA helicase A inhibits growth of Ewing's sarcoma.](https://pubmed.ncbi.nlm.nih.gov/19584866)" (2009) with a relative citation ratio of 6.52. The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[EMPTY]


The top transcription factors (TFs) identified from the GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313), [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046), [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), and [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), each with 5 experiments. Additionally, [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) was found to be regulating in 4 experiments.











The analyzed protein sequence has a GRAVY value of -0.794 (11.98th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -2.19 (42.49th percentile), and the median structural flexibility is 1.008 (74.26th percentile). The protein has a propensity for turns (37.39%, 94.38th percentile), with lower affinities for helices (26.11%, 8.05th percentile) and sheets (30.09%, 28.83rd percentile). The instability index is 49.94 (57.07th percentile), and the isoelectric point is 6.57 (44.01st percentile). The protein is 452 amino acids long (52.14th percentile) with a molecular weight of 50981.25 Da (52.8th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |