---
layout: post
title: CMKLR1
date: 2024-12-08 23:57 CST
description: CMKLR1 description
tags: [cooccuring-genes]
categories: longest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1240) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1240  | CMKLR1 | ENSG00000174600 | 12q23.3 |



The gene length is 547,109 base pairs (98.62% of all genes), the mature length is 6,818.5 base pairs, and the primary transcript length is 547,099 base pairs.


The gene CMKLR1, with NCBI ID 1240, has been the subject of [89 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CMKLR1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1996, and the middle 50% of publications occurred between 2014 and 2020.


The top five publications mentioning CMKLR1, ranked by their scientific influence, include "[Resolvin E1 selectively interacts with leukotriene B4 receptor BLT1 and ChemR23 to regulate inflammation.](https://pubmed.ncbi.nlm.nih.gov/17339491)" (2007) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 10.68, "[International Union of Basic and Clinical Pharmacology CIII: Chemerin Receptors CMKLR1 (Chemerin<sub>1</sub>) and GPR1 (Chemerin<sub>2</sub>) Nomenclature, Pharmacology, and Function.](https://pubmed.ncbi.nlm.nih.gov/29279348)" (2018) with an RCR of 5.77, "[Mouse ChemR23 is expressed in dendritic cell subsets and macrophages, and mediates an anti-inflammatory activity of chemerin in a lung disease model.](https://pubmed.ncbi.nlm.nih.gov/19841182)" (2009) with an RCR of 4.89, "[Signaling Properties of Chemerin Receptors CMKLR1, GPR1 and CCRL2.](https://pubmed.ncbi.nlm.nih.gov/27716822)" (2016) with an RCR of 4.67, and "[Chemerin/chemR23 axis in inflammation onset and resolution.](https://pubmed.ncbi.nlm.nih.gov/25548799)" (2015) with an RCR of 4.67. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885) with 5 experiments, [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613) with 3 experiments, [SUZ12](https://www.ncbi.nlm.nih.gov/gene/23512) with 3 experiments, [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) with 3 experiments, and [SPI1](https://www.ncbi.nlm.nih.gov/gene/6688) with 3 experiments.











The protein sequence analyzed has a GRAVY value of 0.485 (94.89th percentile), indicating a high level of hydrophobicity. It exhibits a charge of 6.45 at pH 7.0 (72.51st percentile) and a median structural flexibility of 0.979 (3.84th percentile). The protein's secondary structure is predicted to be 28.42% helix (19.67th percentile), 46.65% sheet (96.01st percentile), and 25.47% turn (24.95th percentile). The instability index is 37.29 (14.81st percentile), with an isoelectric point of 8.71 (71.38th percentile). The protein has a length of 373 amino acids (41.57th percentile) and a molecular weight of 42321.52 Da (42.6th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |