---
layout: post
title: LRP4
date: 2025-01-17 16:55 CST
description: LRP4 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4038) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4038  | LRP4 | ENSG00000134569 | 11p11.2 |



The gene is involved in various biological processes, including [kidney development](https://amigo.geneontology.org/amigo/term/GO:0001822), [hair follicle development](https://amigo.geneontology.org/amigo/term/GO:0001942), [endocytosis](https://amigo.geneontology.org/amigo/term/GO:0006897), and [Wnt signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0016055). It enables functions such as [calcium ion binding](https://amigo.geneontology.org/amigo/term/GO:0005509), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [coreceptor activity](https://amigo.geneontology.org/amigo/term/GO:0015026). The gene is located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886) and is active in the [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986), [postsynaptic density](https://amigo.geneontology.org/amigo/term/GO:0014069), and [dendrite](https://amigo.geneontology.org/amigo/term/GO:0030425). Additionally, it is involved in processes like [dorsal/ventral pattern formation](https://amigo.geneontology.org/amigo/term/GO:0009953), [proximal/distal pattern formation](https://amigo.geneontology.org/amigo/term/GO:0009954), and [negative regulation of ossification](https://amigo.geneontology.org/amigo/term/GO:0030279). The gene also plays a role in [neuromuscular junction](https://amigo.geneontology.org/amigo/term/GO:0031594) and [neuronal cell body](https://amigo.geneontology.org/amigo/term/GO:0043025) functions, as well as [plasma membrane raft](https://amigo.geneontology.org/amigo/term/GO:0044853) and [synaptic membrane](https://amigo.geneontology.org/amigo/term/GO:0097060) localization.


The gene length is 4,223 base pairs (11.84% of all genes), the mature length is 1,741 base pairs, and the primary transcript length is 4,217 base pairs.


The gene LRP4 (NCBI ID: 4038) has been mentioned in [102 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22LRP4%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 2002 and the middle 50% of publications occurring between 2013 and 2020. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning LRP4, ranked by their scientific influence, include "[Lrp4 is a receptor for Agrin and forms a complex with MuSK.](https://pubmed.ncbi.nlm.nih.gov/18848351)" (2008) (relative citation ratio: 10.09), "[Autoantibodies to low-density lipoprotein receptor-related protein 4 in myasthenia gravis.](https://pubmed.ncbi.nlm.nih.gov/21387385)" (2011) (relative citation ratio: 9.38), "[LRP4 serves as a coreceptor of agrin.](https://pubmed.ncbi.nlm.nih.gov/18957220)" (2008) (relative citation ratio: 8.18), "[A comprehensive analysis of the epidemiology and clinical characteristics of anti-LRP4 in myasthenia gravis.](https://pubmed.ncbi.nlm.nih.gov/24373505)" (2014) (relative citation ratio: 7.3), and "[Autoantibodies to lipoprotein-related protein 4 in patients with double-seronegative myasthenia gravis.](https://pubmed.ncbi.nlm.nih.gov/22158716)" (2012) (relative citation ratio: 7.01). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[LRP4](https://www.proteinatlas.org/ENSG00000134569-LRP4) is a developmental protein and receptor with evidence at the protein level. It is implicated in congenital myasthenic syndrome and is detected in many tissues. The protein is localized in the nucleoplasm and mitochondria. LRP4 is expressed in various clusters, including Cluster 80 for non-specific endocytosis, Cluster 19 for astrocytes with mixed function, Cluster 57 for stomach and colon cancer related to digestion, and Cluster 76 for oligodendrocyte precursors with unknown function.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169), [EZH2](https://www.ncbi.nlm.nih.gov/gene/2146), [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), and [PGR](https://www.ncbi.nlm.nih.gov/gene/5241), each with 4 experiments. Additionally, [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) was found to be regulating in 3 experiments.



The GWAS data indicates associations with a wide range of disease conditions, including hematopoietic system disease, immune system disease, artery disease, vascular disease, cardiovascular system disease, cognitive disorder, disease of mental health, bipolar disorder, mood disorder, bone disease, connective tissue disease, musculoskeletal system disease, bone remodeling disease, bone resorption disease, agammaglobulinemia, common variable immunodeficiency, blood protein disease, B cell deficiency, primary immunodeficiency disease, cerebrovascular disease, and blood coagulation disease.


LRP4 has been associated with [Heel bone mineral density](https://pubmed.ncbi.nlm.nih.gov/30598549), which was mapped to the closest existing MeSH term [Minerals](https://meshb.nlm.nih.gov/record/ui?ui=D008903) using a method called [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0). Additionally, LRP4 has been linked to [Vertex-wise cortical thickness](https://pubmed.ncbi.nlm.nih.gov/34910505).


The gene is ubiquitously expressed in multiple tissue samples, including nerve, skin, and brain, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in various brain regions such as the Prefrontal Cortex, Amygdala, Whole brain, Caudate nucleus, Hypothalamus, and Cingulate Cortex, as well as in the Pancreas.


The proteins are involved in extracellular matrix organization and ECM proteoglycans.



The analyzed protein sequence has a GRAVY value of -0.4943 (35.46th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -79.36 (0.85th percentile), and the median structural flexibility is 0.9999 (41.06th percentile). The protein has a propensity for helix, sheet, and turn structures of 24.51% (4.24th percentile), 31.92% (38.99th percentile), and 34.33% (87.24th percentile), respectively. The instability index is 41.60 (27.27th percentile), with an isoelectric point of 5.06 (8.89th percentile). The protein is 1905 amino acids long (97.19th percentile) and has a molecular weight of 212042.73 Da (97.21st percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |