---
layout: post
title: ITPRID2
date: 2025-01-17 16:55 CST
description: ITPRID2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6744) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6744  | ITPRID2 | ENSG00000138434 | 2q31.3  |



The gene enables [signaling receptor binding](https://amigo.geneontology.org/amigo/term/GO:0005102) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is involved in [actin filament binding](https://amigo.geneontology.org/amigo/term/GO:0051015). It is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886).


The gene length is 444,327 base pairs (97.89% of all genes), the mature length is 4,529 base pairs, and the primary transcript length is 444,057 base pairs.


The gene ITPRID2 (NCBI ID: 6744) has been mentioned in [20 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ITPRID2%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1985, and the middle 50% of publications range from 2007 to 2021.


The top 5 publications mentioning ITPRID2, ranked by their relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows:

1. [CS1-specific chimeric antigen receptor (CAR)-engineered natural killer cells enhance in vitro and in vivo antitumor activity against human multiple myeloma.](https://pubmed.ncbi.nlm.nih.gov/24067492) (2014) (relative citation ratio: 9.25)
2. [The minimal essential sequence for a major cell type-specific adhesion site (CS1) within the alternatively spliced type III connecting segment domain of fibronectin is leucine-aspartic acid-valine.](https://pubmed.ncbi.nlm.nih.gov/1869542) (1991) (relative citation ratio: 7.13)
3. [KRAP tethers IP<sub>3</sub> receptors to actin and licenses them to evoke cytosolic Ca<sup>2+</sup> signals.](https://pubmed.ncbi.nlm.nih.gov/34301929) (2021) (relative citation ratio: 2.41)
4. [Antibodies derived from an enterotoxigenic Escherichia coli (ETEC) adhesin tip MEFA (multiepitope fusion antigen) against adherence of nine ETEC adhesins: CFA/I, CS1, CS2, CS3, CS4, CS5, CS6, CS21 and EtpA.](https://pubmed.ncbi.nlm.nih.gov/27228947) (2016) (relative citation ratio: 1.71)
5. [Co-administered Tag-Less Toxoid Fusion 3xSTa<sub>N12S</sub>-mnLT<sub>R192G/L211A</sub> and CFA/I/II/IV MEFA (Multiepitope Fusion Antigen) Induce Neutralizing Antibodies to 7 Adhesins (CFA/I, CS1-CS6) and Both Enterotoxins (LT, STa) of Enterotoxigenic <i>Escherichia coli</i> (ETEC).](https://pubmed.ncbi.nlm.nih.gov/29922268) (2018) (relative citation ratio: 1.43)

Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ITPRID2](https://www.proteinatlas.org/ENSG00000138434-ITPRID2) is an actin-binding protein with evidence at the protein level. It is detected in all tissues and is primarily localized in the nucleoplasm, plasma membrane, and cytosol. The gene is part of various expression clusters, including Cluster 10 for non-specific membrane trafficking in blood, Cluster 48 for proteolysis in the stomach, Cluster 19 for mixed function in brain astrocytes, Cluster 53 for keratinization in squamous epithelial cells, and Cluster 5 for absorption in enterocytes.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908) and [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313), each with 6 experiments, followed by [YY1](https://www.ncbi.nlm.nih.gov/gene/7528), also with 6 experiments. Additionally, [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227) and [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885) were found to be regulating in 5 experiments each.



The GWAS data indicates associations with various disease conditions, including general diseases, diseases of anatomical entities, musculoskeletal system diseases, and muscular diseases.



The gene is expressed in the pancreas and nerve tissues according to the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles indicate expression in the salivary gland, superior cervical ganglion, trigeminal ganglion, atrioventricular node, ciliary ganglion, skin, and dorsal root ganglion.




The analyzed protein sequence has a GRAVY value of -0.655 (20.76th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -51.84 (2.28th percentile), and the median structural flexibility is 1.012 (86.35th percentile). The protein has a high instability index of 67.70 (92.99th percentile) and an isoelectric point of 5.11 (9.75th percentile). The sequence length is 1259 amino acids (92.69th percentile) with a molecular weight of 138384.38 Da (92.47th percentile). The secondary structure affinities are 33.76% for helix (65.92nd percentile), 27.56% for sheet (16.65th percentile), and 32.88% for turn (81.34th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |