---
layout: post
title: RAB2A
date: 2025-01-17 16:55 CST
description: RAB2A description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5862) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5862  | RAB2A | ENSG00000104388 | 8q12.1-q12.2 |



The gene is located in the [Golgi membrane](https://amigo.geneontology.org/amigo/term/GO:0000139), [acrosomal vesicle](https://amigo.geneontology.org/amigo/term/GO:0001669), [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [lysosomal membrane](https://amigo.geneontology.org/amigo/term/GO:0005765), [endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0005789), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [endoplasmic reticulum-Golgi intermediate compartment membrane](https://amigo.geneontology.org/amigo/term/GO:0033116), [melanosome](https://amigo.geneontology.org/amigo/term/GO:0042470), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). It is active in the [Golgi apparatus](https://amigo.geneontology.org/amigo/term/GO:0005794) and enables [GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0003924), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [GTP binding](https://amigo.geneontology.org/amigo/term/GO:0005525), and [GDP binding](https://amigo.geneontology.org/amigo/term/GO:0019003). The gene is involved in [Golgi organization](https://amigo.geneontology.org/amigo/term/GO:0007030), [protein transport](https://amigo.geneontology.org/amigo/term/GO:0015031), and [vesicle-mediated transport](https://amigo.geneontology.org/amigo/term/GO:0016192), but is not involved in [endoplasmic reticulum to Golgi vesicle-mediated transport](https://amigo.geneontology.org/amigo/term/GO:0006888).


The gene length is 9,233 base pairs (24.01% of all genes), the mature length is 2,194 base pairs (57.43% of all genes), and the primary transcript length is 9,002 base pairs (30.67% of all genes).


The gene RAB2A, with NCBI ID 5862, has been mentioned in [50 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RAB2A%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ). The earliest year of publication mentioning this gene is 1992, and the middle 50% of publications occurred between 2003 and 2017.


The top 5 publications mentioning RAB2A, ranked by their scientific influence, include "[Rab2 promotes autophagic and endocytic lysosomal degradation.](https://pubmed.ncbi.nlm.nih.gov/28483915)" (2017) (relative citation ratio: 3.35), "[A GRASP55-rab2 effector complex linking Golgi structure to membrane traffic.](https://pubmed.ncbi.nlm.nih.gov/11739401)" (2001) (relative citation ratio: 2.87), "[Genetic screen in Drosophila muscle identifies autophagy-mediated T-tubule remodeling and a Rab2 role in autophagy.](https://pubmed.ncbi.nlm.nih.gov/28063257)" (2017) (relative citation ratio: 2.79), "[Identification of a Brucella spp. secreted effector specifically interacting with human small GTPase Rab2.](https://pubmed.ncbi.nlm.nih.gov/21501366)" (2011) (relative citation ratio: 2.56), and "[RAB2A controls MT1-MMP endocytic and E-cadherin polarized Golgi trafficking to promote invasive breast cancer programs.](https://pubmed.ncbi.nlm.nih.gov/27255086)" (2016) (relative citation ratio: 2.16). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RAB2A](https://www.proteinatlas.org/ENSG00000104388-RAB2A) is a gene that functions as a hydrolase, with evidence supporting its activity at the protein level. Its RNA is detected in all tissues, and it is notably expressed in monocytes and neutrophils, suggesting a role in the innate immune response. The gene is also involved in basic cellular processes and synaptic function in neurons. Additionally, it is expressed in plasma cells, indicating a role in protein processing.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [CREB1](https://www.ncbi.nlm.nih.gov/gene/1385), [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169), and [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), each with 6 experiments. Additionally, [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664) and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) were found to be regulating in 5 experiments each.



The input data indicates associations with various disease conditions, including several types of cancer such as leukemia, lymphoma, and immune system cancer. Additionally, respiratory conditions like asthma, bronchial disease, and obstructive lung disease are noted. Other significant associations include hematopoietic system disease, immune system disease, and cardiovascular system disease.



The gene is expressed in a variety of tissues, including adipose tissue, thyroid, amygdala, placenta, colon, and small intestine.


The proteins are involved in various cellular processes, including mitotic prophase, the cell cycle, and the M phase. Additionally, they participate in the reorganization of the Golgi cisternae pericentriolar stack.


The mouse ortholog gene 59021 and its human ortholog gene 5862 are associated with several phenotypic abnormalities. These include abnormal embryo size, abnormal skin morphology, and preweaning lethality with complete penetrance, all of which are observed in both sexes.


The protein sequence analyzed has a GRAVY value of -0.354 (51.79th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -2.97 (39.84th percentile). The median structural flexibility is 1.003 (54.81st percentile). The protein's secondary structure is predicted to be 31.13% helix (42.4th percentile), 34.91% sheet (58.35th percentile), and 25.47% turn (24.99th percentile). The instability index is 33.93 (8.44th percentile), and the isoelectric point is 6.08 (32.62nd percentile). The protein is 212 amino acids long (16.3rd percentile) with a molecular weight of 23545.28 Da (16.1st percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |