---
layout: post
title: MBNL2
date: 2025-01-17 16:55 CST
description: MBNL2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10150) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10150  | MBNL2 | ENSG00000139793 | 13q32.1 |



The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872), and is active in [sequence-specific double-stranded DNA binding](https://amigo.geneontology.org/amigo/term/GO:1990837). It is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654) and [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), where it is involved in [mRNA processing](https://amigo.geneontology.org/amigo/term/GO:0006397), [RNA splicing](https://amigo.geneontology.org/amigo/term/GO:0008380), and the [regulation of RNA splicing](https://amigo.geneontology.org/amigo/term/GO:0043484).


The gene length is 18,511 base pairs (40.22 percentile of all genes), the mature length is 6,754 base pairs (95.95 percentile of all genes), and the primary transcript length is 18,462 base pairs (49.53 percentile of all genes).


The gene MBNL2 (NCBI ID: 10150) has been mentioned in [12 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22MBNL2%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 2002, and the middle 50% of publications occurred between 2012 and 2020.


The top 5 publications mentioning MBNL2, ranked by their [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows. The most influential publication is "[Three proteins, MBNL, MBLL and MBXL, co-localize in vivo with nuclear foci of expanded-repeat transcripts in DM1 and DM2 cells.](https://pubmed.ncbi.nlm.nih.gov/11929853)" (2002) with a relative citation ratio of 5.78. Other notable publications include "[MBNL2 Regulates DNA Damage Response via Stabilizing p21.](https://pubmed.ncbi.nlm.nih.gov/33466733)" (2021) with a relative citation ratio of 1.06, "[Muscleblind-like 2 controls the hypoxia response of cancer cells.](https://pubmed.ncbi.nlm.nih.gov/32127384)" (2020) with a relative citation ratio of 1.0, "[Tau exon 2 responsive elements deregulated in myotonic dystrophy type I are proximal to exon 2 and synergistically regulated by MBNL1 and MBNL2.](https://pubmed.ncbi.nlm.nih.gov/24440524)" (2014) with a relative citation ratio of 0.76, and "[Protective effects of mirtazapine in mice lacking the Mbnl2 gene in forebrain glutamatergic neurons: Relevance for myotonic dystrophy 1.](https://pubmed.ncbi.nlm.nih.gov/32171677)" (2020) with a relative citation ratio of 0.72. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[MBNL2](https://www.proteinatlas.org/ENSG00000139793-MBNL2) is an RNA-binding protein with evidence at the protein level. It is detected in all RNA tissue distributions and is primarily located in the nucleoplasm. The gene is expressed in various clusters, including Cluster 5 for blood expression, which is associated with non-specific cell proliferation, and Cluster 69 for tissue expression, which is linked to liver oxidoreductase activity. In the brain, it is part of Cluster 28, associated with oligodendrocytes and mixed function. Additionally, MBNL2 is found in Cluster 19 for cell line expression and Cluster 51 for single-cell expression, both of which have unknown functions.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [RELA](https://www.ncbi.nlm.nih.gov/gene/5970) with 4 experiments, [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861) with 3 experiments, [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908) with 3 experiments, [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 3 experiments, and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 3 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with a wide range of disease conditions, including infectious diseases such as viral infectious diseases and human immunodeficiency virus infectious disease. Additionally, the data highlights associations with hematopoietic and immune system diseases, as well as metabolic disorders, particularly those related to carbohydrate and glucose metabolism, including type 2 diabetes mellitus. Cardiovascular conditions such as artery disease, vascular disease, and hypertension are also implicated. Furthermore, the data shows links to various mental health disorders, including cognitive disorders, mood disorders, substance-related disorders, psychotic disorders, and specific conditions like schizophrenia and major depressive disorder. Anemia, particularly congenital hemolytic anemia and hemoglobinopathy, is also noted. Lastly, the data includes associations with integumentary system diseases, specifically hair diseases like alopecia and skin diseases such as psoriasis.



The gene is expressed in various neural tissues, including the nerve, caudate nucleus, cingulate cortex, medulla oblongata, parietal lobe, and occipital lobe. Additionally, it is expressed in the adrenal gland.




The analyzed protein sequence has a GRAVY value of -0.208 (68.32nd percentile), indicating a hydrophilic nature. It exhibits a charge of 9.4 at pH 7.0 (80.19th percentile) and a median structural flexibility of 0.996 (24.93rd percentile). The protein's secondary structure is predicted to be 29.76% helix (29.87th percentile), 31.9% sheet (38.92nd percentile), and 26% turn (29.13th percentile). The instability index is 44.95 (39.01st percentile), with an isoelectric point of 8.77 (72.88th percentile). The protein has a length of 373 amino acids (41.57th percentile) and a molecular weight of 40517.24 Da (40.13th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |