---
layout: post
title: CALM1
date: 2025-01-17 16:55 CST
description: CALM1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/801) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 801  | CALM1 | ENSG00000198668 | 14q32.11 |



The gene is involved in various biological processes, including the [G2/M transition of mitotic cell cycle](https://amigo.geneontology.org/amigo/term/GO:0000086) and the [regulation of heart rate](https://amigo.geneontology.org/amigo/term/GO:0002027). It is located in multiple cellular components such as the [spindle pole](https://amigo.geneontology.org/amigo/term/GO:0000922), [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [centrosome](https://amigo.geneontology.org/amigo/term/GO:0005813), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [spindle microtubule](https://amigo.geneontology.org/amigo/term/GO:0005876), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [sarcomere](https://amigo.geneontology.org/amigo/term/GO:0030017), [vesicle](https://amigo.geneontology.org/amigo/term/GO:0031982), [myelin sheath](https://amigo.geneontology.org/amigo/term/GO:0043209), and [sperm midpiece](https://amigo.geneontology.org/amigo/term/GO:0097225). The gene enables functions such as [calcium ion binding](https://amigo.geneontology.org/amigo/term/GO:0005509), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [adenylate cyclase activator activity](https://amigo.geneontology.org/amigo/term/GO:0010856), [calcium channel inhibitor activity](https://amigo.geneontology.org/amigo/term/GO:0019855), [protein kinase binding](https://amigo.geneontology.org/amigo/term/GO:0019901), [titin binding](https://amigo.geneontology.org/amigo/term/GO:0031432), [protein serine/threonine kinase activator activity](https://amigo.geneontology.org/amigo/term/GO:0043539), [transmembrane transporter binding](https://amigo.geneontology.org/amigo/term/GO:0044325), and [calcium-dependent protein binding](https://amigo.geneontology.org/amigo/term/GO:0048306). Additionally, it is part of the [voltage-gated potassium channel complex](https://amigo.geneontology.org/amigo/term/GO:0008076), [protein-containing complex](https://amigo.geneontology.org/amigo/term/GO:0032991), [calcium channel complex](https://amigo.geneontology.org/amigo/term/GO:0034704), and [catalytic complex](https://amigo.geneontology.org/amigo/term/GO:1902494). The gene is also involved in various regulatory processes, including the [G protein-coupled receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007186), [positive regulation of peptidyl-threonine phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0010800), [negative regulation of peptidyl-threonine phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0010801), [regulation of release of sequestered calcium ion into cytosol by sarcoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0010880), [regulation of cardiac muscle contraction by regulation of the release of sequestered calcium ion](https://amigo.geneontology.org/amigo/term/GO:0010881), [autophagosome membrane docking](https://amigo.geneontology.org/amigo/term/GO:0016240), [substantia nigra development](https://amigo.geneontology.org/amigo/term/GO:0021762), [positive regulation of protein autophosphorylation](https://amigo.geneontology.org/amigo/term/GO:0031954), [regulation of cytokinesis](https://amigo.geneontology.org/amigo/term/GO:0032465), [positive regulation of phosphoprotein phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0032516), [positive regulation of protein dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0035307), [cellular response to interferon-beta](https://amigo.geneontology.org/amigo/term/GO:0035458), [positive regulation of receptor signaling pathway via JAK-STAT](https://amigo.geneontology.org/amigo/term/GO:0046427), [regulation of calcium-mediated signaling](https://amigo.geneontology.org/amigo/term/GO:0050848), [positive regulation of cyclic-nucleotide phosphodiesterase activity](https://amigo.geneontology.org/amigo/term/GO:0051343), [response to calcium ion](https://amigo.geneontology.org/amigo/term/GO:0051592), [regulation of cardiac muscle contraction](https://amigo.geneontology.org/amigo/term/GO:0055117), [regulation of ryanodine-sensitive calcium-release channel activity](https://amigo.geneontology.org/amigo/term/GO:0060314), [negative regulation of ryanodine-sensitive calcium-release channel activity](https://amigo.geneontology.org/amigo/term/GO:0060315), [positive regulation of ryanodine-sensitive calcium-release channel activity](https://amigo.geneontology.org/amigo/term/GO:0060316), [cellular response to type II interferon](https://amigo.geneontology.org/amigo/term/GO:0071346), [positive regulation of protein serine/threonine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0071902), [protein phosphatase activator activity](https://amigo.geneontology.org/amigo/term/GO:0072542), [organelle localization by membrane tethering](https://amigo.geneontology.org/amigo/term/GO:0140056), [negative regulation of high voltage-gated calcium channel activity](https://amigo.geneontology.org/amigo/term/GO:1901842), [regulation of cell communication by electrical coupling involved in cardiac conduction](https://amigo.geneontology.org/amigo/term/GO:1901844), [negative regulation of calcium ion export across plasma membrane](https://amigo.geneontology.org/amigo/term/GO:1905913), and [mitochondrion-endoplasmic reticulum membrane tethering](https://amigo.geneontology.org/amigo/term/GO:1990456).


The gene length is 151,604 base pairs (89.04% of all genes), the mature length is 2,571 base pairs (65.24% of all genes), and the primary transcript length is 147,903 base pairs (93.4% of all genes).


The gene CALM1 (NCBI ID: 801) has been mentioned in [3,172 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CALM1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning this gene is 1979, and the middle 50% of publications occurred between 1987 and 2008.


The top five publications mentioning CALM1, ranked by their scientific influence, include [Fluorescent indicators for Ca2+ based on green fluorescent proteins and calmodulin.](https://pubmed.ncbi.nlm.nih.gov/9278050) (1997) (relative citation ratio: 44.31), [Calmodulin.](https://pubmed.ncbi.nlm.nih.gov/6250447) (1980) (relative citation ratio: 35.45), [Calmodulin--an intracellular calcium receptor.](https://pubmed.ncbi.nlm.nih.gov/6990273) (1980) (relative citation ratio: 30.62), [Structure of calmodulin refined at 2.2 A resolution.](https://pubmed.ncbi.nlm.nih.gov/3145979) (1988) (relative citation ratio: 25.55), and [Calmodulin.](https://pubmed.ncbi.nlm.nih.gov/6762067) (1982) (relative citation ratio: 25.11). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CALM1](https://www.proteinatlas.org/ENSG00000198668-CALM1) is a gene with evidence at the protein level, implicated in Long QT syndrome and is an FDA-approved drug target. Its RNA is detected in all tissues, with notable expression in the brain, particularly in neurons involved in synaptic function. The protein is primarily located in the cytosol. In terms of expression clusters, CALM1 is associated with cell proliferation in blood, synaptic signal transduction in tissues, and synaptic function in the brain. Additionally, it is involved in gene expression regulation in cell lines and immune system functions in single cells, specifically in NK-cells and T-cells.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 6 experiments, [GATAD2A](https://www.ncbi.nlm.nih.gov/gene/54815) with 5 experiments, [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170) with 5 experiments, [YY1](https://www.ncbi.nlm.nih.gov/gene/7528) with 5 experiments, and [EP300](https://www.ncbi.nlm.nih.gov/gene/2033) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent detection in multiple experiments.


The transcription factor [8861](https://www.ncbi.nlm.nih.gov/gene/8861) was found to be regulating genes in 4 Perturb-seq experiments, making it one of the top transcription factors identified in the study.


The GWAS data indicates associations with various disease conditions, including artery disease, vascular disease, cardiovascular system disease, cerebrovascular disease, and general disease of anatomical entities.



The gene is expressed in various tissues, including muscle and brain, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in multiple brain regions such as the Cingulate Cortex, Subthalamic Nucleus, Parietal Lobe, Occipital Lobe, Pons, and Globus Pallidus.


The input data includes a diverse set of biological pathways and processes, such as the metabolism of carbohydrates, glycogen storage diseases, and various signaling pathways. Notably, it covers phototransduction cascades, mitochondrial biogenesis, and immune system functions. Additionally, it includes signaling by Wnt, FGFR, EGFR, and VEGF, as well as processes related to calcium signaling, glucose metabolism, and neurotransmitter receptor binding. The data also encompasses muscle contraction, hemostasis, platelet activation, and membrane trafficking.



The analyzed protein sequence has a GRAVY value of -0.654 (20.85th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -24.344 (8.45th percentile), and the median structural flexibility is 1.018 (95.2nd percentile). The protein has a high affinity for helix structures (39.6%, 92.64th percentile), with lower affinities for sheet (30.9%, 33.1st percentile) and turn (26.8%, 36.62nd percentile) structures. The instability index is 28.207 (2.78th percentile), suggesting a stable protein. The isoelectric point is 4.087 (0.22nd percentile), with a molecular weight of 16837.42 Da (8.13th percentile) and a length of 149 amino acids (7.97th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |