---
layout: post
title: CLIP1
date: 2025-01-17 16:55 CST
description: CLIP1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6249) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6249  | CLIP1 | ENSG00000130779 | 12q24.31 |



The gene is involved in the [mitotic cell cycle](https://amigo.geneontology.org/amigo/term/GO:0000278) and [microtubule bundle formation](https://amigo.geneontology.org/amigo/term/GO:0001578), and plays a role in the [positive regulation of microtubule polymerization](https://amigo.geneontology.org/amigo/term/GO:0031116) and [cytoplasmic microtubule organization](https://amigo.geneontology.org/amigo/term/GO:0031122). It is located in the [kinetochore](https://amigo.geneontology.org/amigo/term/GO:0000776), [ruffle](https://amigo.geneontology.org/amigo/term/GO:0001726), [centrosome](https://amigo.geneontology.org/amigo/term/GO:0005813), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [microtubule](https://amigo.geneontology.org/amigo/term/GO:0005874), [intermediate filament](https://amigo.geneontology.org/amigo/term/GO:0005882), [microtubule cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0015630), and [cytoplasmic vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030659). It is active in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [cell cortex](https://amigo.geneontology.org/amigo/term/GO:0005938). The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [microtubule binding](https://amigo.geneontology.org/amigo/term/GO:0008017), [zinc ion binding](https://amigo.geneontology.org/amigo/term/GO:0008270), [tubulin binding](https://amigo.geneontology.org/amigo/term/GO:0015631), and [microtubule plus-end binding](https://amigo.geneontology.org/amigo/term/GO:0051010). Additionally, it colocalizes with the [microtubule plus-end](https://amigo.geneontology.org/amigo/term/GO:0035371) and [macropinosome](https://amigo.geneontology.org/amigo/term/GO:0044354).


The gene length is 46,466 base pairs (64.44% of all genes), the mature length is 890 base pairs (17.92% of all genes), and the primary transcript length is 23,593.5 base pairs (56.35% of all genes).


The gene CLIP1 (NCBI ID: 6249) has been mentioned in [630 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CLIP1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning CLIP1 is 1945, and the middle 50% of publications occurred between 2013 and 2020.


The top 5 publications mentioning CLIP1, ranked by their scientific influence, include "[Argonaute HITS-CLIP decodes microRNA-mRNA interaction maps.](https://pubmed.ncbi.nlm.nih.gov/19536157)" (2009) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 27.8, "[Percutaneous mitral repair with the MitraClip system: safety and midterm durability in the initial EVEREST (Endovascular Valve Edge-to-Edge REpair Study) cohort.](https://pubmed.ncbi.nlm.nih.gov/19679246)" (2009) with an RCR of 18.87, "[The structure of an intermediate in class II MHC maturation: CLIP bound to HLA-DR3.](https://pubmed.ncbi.nlm.nih.gov/7477400)" (1995) with an RCR of 9.93, "[Rac1 and Cdc42 capture microtubules through IQGAP1 and CLIP-170.](https://pubmed.ncbi.nlm.nih.gov/12110184)" (2002) with an RCR of 7.68, and "[Acute outcomes of MitraClip therapy for mitral regurgitation in high-surgical-risk patients: emphasis on adverse valve morphology and severe left ventricular dysfunction.](https://pubmed.ncbi.nlm.nih.gov/20219746)" (2010) with an RCR of 7.55. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CLIP1](https://www.proteinatlas.org/ENSG00000130779-CLIP1) is a cancer-related gene with evidence at the protein level. Its RNA is detected in all tissues, and the protein is localized to microtubule ends, cytosol, mid piece, and principal piece. CLIP1 is expressed in neutrophils with unknown function (Cluster 28) and is involved in basic cellular processes across various tissues (Cluster 19). In the brain, it is associated with neurons and nucleosome functions (Cluster 2), while in cell lines, it is linked to non-specific transcription processes (Cluster 38). Additionally, CLIP1 is expressed in syncytiotrophoblasts related to pregnancy (Cluster 27).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [MYC](https://www.ncbi.nlm.nih.gov/gene/4609) with 5 experiments, [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908) with 4 experiments, [JUN](https://www.ncbi.nlm.nih.gov/gene/3725) with 4 experiments, [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) with 4 experiments, and [BRD9](https://www.ncbi.nlm.nih.gov/gene/65980) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with a range of conditions, including lower respiratory tract disease, lung disease, respiratory system disease, coronary artery disease, artery disease, vascular disease, cardiovascular system disease, heart disease, and broader categories such as disease and disease of anatomical entity.


The genome-wide association study has identified a significant association between CLIP1 and [High density lipoprotein cholesterol levels](https://pubmed.ncbi.nlm.nih.gov/33462484), with the closest MeSH match being [Lipoproteins](https://meshb.nlm.nih.gov/record/ui?ui=D008074). This association was determined using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method.


The gene is expressed in a variety of tissues, including the thyroid, amygdala, placenta, appendix, superior cervical ganglion, trigeminal ganglion, skeletal muscle, and uterus. Additionally, it is expressed in muscle tissue according to BioGPS Human Cell Type and Tissue Gene Expression Profiles.


The proteins are involved in various stages of the cell cycle, particularly focusing on mitotic processes. These include the broader mitotic phase, as well as specific stages such as mitotic anaphase, mitotic metaphase and anaphase, and mitotic prometaphase. Key processes within these stages include the separation of sister chromatids and the resolution of sister chromatid cohesion.



The analyzed protein sequence has a GRAVY value of -0.869 (8.58th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -48.88 (2.53rd percentile), and the median structural flexibility is 1.02 (96.87th percentile). The protein has a high affinity for helix structures (45.83%, 98.68th percentile), with lower affinities for sheet (27.82%, 17.72nd percentile) and turn (23.44%, 12.11th percentile) structures. The instability index is 46.90 (46.23rd percentile), and the isoelectric point is 5.29 (13.25th percentile). The protein is 1438 amino acids long (94.55th percentile) with a molecular weight of 162244.44 Da (94.79th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |