---
layout: post
title: NGFR
date: 2025-01-17 16:55 CST
description: NGFR description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4804) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4804  | NGFR | ENSG00000064300 | 17q21.33 |



The gene enables [amyloid-beta binding](https://amigo.geneontology.org/amigo/term/GO:0001540), [transmembrane signaling receptor activity](https://amigo.geneontology.org/amigo/term/GO:0004888), [death receptor activity](https://amigo.geneontology.org/amigo/term/GO:0005035), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [calmodulin binding](https://amigo.geneontology.org/amigo/term/GO:0005516), [coreceptor activity](https://amigo.geneontology.org/amigo/term/GO:0015026), [small GTPase binding](https://amigo.geneontology.org/amigo/term/GO:0031267), [ubiquitin protein ligase binding](https://amigo.geneontology.org/amigo/term/GO:0031625), [neurotrophin binding](https://amigo.geneontology.org/amigo/term/GO:0043121), and [nerve growth factor binding](https://amigo.geneontology.org/amigo/term/GO:0048406). It is involved in [intracellular glucose homeostasis](https://amigo.geneontology.org/amigo/term/GO:0001678), [intracellular protein transport](https://amigo.geneontology.org/amigo/term/GO:0006886), [activation of cysteine-type endopeptidase activity involved in apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0006919), [Rho protein signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007266), [axon guidance](https://amigo.geneontology.org/amigo/term/GO:0007411), [central nervous system development](https://amigo.geneontology.org/amigo/term/GO:0007417), [fibroblast growth factor receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0008543), [nerve development](https://amigo.geneontology.org/amigo/term/GO:0021675), [negative regulation of cell migration](https://amigo.geneontology.org/amigo/term/GO:0030336), [hair follicle morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0031069), [circadian regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0032922), [dorsal aorta development](https://amigo.geneontology.org/amigo/term/GO:0035907), [negative regulation of fibroblast growth factor receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0040037), [odontogenesis of dentin-containing tooth](https://amigo.geneontology.org/amigo/term/GO:0042475), [positive regulation of odontogenesis of dentin-containing tooth](https://amigo.geneontology.org/amigo/term/GO:0042488), [glucose homeostasis](https://amigo.geneontology.org/amigo/term/GO:0042593), [fibroblast proliferation](https://amigo.geneontology.org/amigo/term/GO:0048144), [positive regulation of fibroblast proliferation](https://amigo.geneontology.org/amigo/term/GO:0048146), [neuron apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0051402), [negative regulation of hair follicle development](https://amigo.geneontology.org/amigo/term/GO:0051799), [presynaptic modulation of chemical synaptic transmission](https://amigo.geneontology.org/amigo/term/GO:0099171), [positive regulation of protein localization to nucleus](https://amigo.geneontology.org/amigo/term/GO:1900182), [positive regulation of miRNA transcription](https://amigo.geneontology.org/amigo/term/GO:1902895), [negative regulation of blood vessel endothelial cell proliferation involved in sprouting angiogenesis](https://amigo.geneontology.org/amigo/term/GO:1903588), and [positive regulation of apoptotic signaling pathway](https://amigo.geneontology.org/amigo/term/GO:2001235). It is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [endosome](https://amigo.geneontology.org/amigo/term/GO:0005768), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [cell-cell junction](https://amigo.geneontology.org/amigo/term/GO:0005911), [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986), [postsynaptic density](https://amigo.geneontology.org/amigo/term/GO:0014069), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [growth cone](https://amigo.geneontology.org/amigo/term/GO:0030426), [neuromuscular junction](https://amigo.geneontology.org/amigo/term/GO:0031594), [dendritic spine](https://amigo.geneontology.org/amigo/term/GO:0043197), [perikaryon](https://amigo.geneontology.org/amigo/term/GO:0043204), and [presynapse](https://amigo.geneontology.org/amigo/term/GO:0098793). It is active in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886) and [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986). Additionally, it is involved in the [detection of temperature stimulus](https://amigo.geneontology.org/amigo/term/GO:0016048).


The gene length is 269,549 base pairs (95.04% of all genes), the mature length is 6,351 base pairs (95.09% of all genes), and the primary transcript length is 121,351 base pairs (91.12% of all genes).


The gene NGFR (NCBI ID: 4804) has been mentioned in [621 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22NGFR%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1973, and the middle 50% of publications occurred between 1996 and 2016.


The top five publications mentioning NGFR, ranked by their scientific influence, include "[High-affinity NGF binding requires coexpression of the trk proto-oncogene and the low-affinity NGF receptor.](https://pubmed.ncbi.nlm.nih.gov/1850821)" (1991) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 25.88, "[Expression and structure of the human NGF receptor.](https://pubmed.ncbi.nlm.nih.gov/3022937)" (1986) with an RCR of 18.97, "[ProBDNF induces neuronal apoptosis via activation of a receptor complex of p75NTR and sortilin.](https://pubmed.ncbi.nlm.nih.gov/15930396)" (2005) with an RCR of 16.61, "[Induction of apoptosis by the low-affinity NGF receptor.](https://pubmed.ncbi.nlm.nih.gov/8332899)" (1993) with an RCR of 16.26, and "[Binding of brain-derived neurotrophic factor to the nerve growth factor receptor.](https://pubmed.ncbi.nlm.nih.gov/2157470)" (1990) with an RCR of 14.77. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[NGFR](https://www.proteinatlas.org/ENSG00000064300-NGFR) is a developmental protein and receptor with evidence at the protein level. It is implicated in cancer-related genes and is detected in many tissues. The protein is localized in the nucleoplasm and plasma membrane. NGFR is expressed in various clusters, including Cluster 8 in plasmacytoid DCs with unknown function, Cluster 30 in heart muscle related to heart development, Cluster 6 in the brain associated with immune response, Cluster 69 in skin cancer related to ECM organization, and Cluster 10 in Muller glia cells involved in visual perception.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 6 experiments, [EZH2](https://www.ncbi.nlm.nih.gov/gene/2146) with 5 experiments, [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908) with 4 experiments, [AR](https://www.ncbi.nlm.nih.gov/gene/367) with 4 experiments, and [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with a range of disease conditions, including infectious diseases such as those caused by viral agents, specifically human immunodeficiency virus infectious disease. Additionally, there are associations with various types of cancer, including leukemia, lymphoma, and other hematologic and immune system cancers. Respiratory conditions such as asthma, bronchial disease, obstructive lung disease, and other lower respiratory tract diseases are also implicated. Furthermore, the data highlights associations with hematopoietic system diseases, immune system diseases, and cardiovascular system diseases.



The gene is expressed in various tissues, including nerve and thyroid, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the cerebellum peduncles, colorectal adenocarcinoma, superior cervical ganglion, atrioventricular node, ciliary ganglion, dorsal root ganglion, and olfactory bulb.


The input data highlights various pathways involving p75NTR and NF-kB. p75NTR recruits signaling complexes and signals via NF-kB, which is activated to promote cell survival. Additionally, p75NTR regulates axonogenesis and negatively influences the cell cycle through SC1. It also participates in cell death signaling via NRAGE, NRIF, and NADE, with NRAGE specifically signaling death through JNK. Other notable pathways include ceramide signaling, axonal growth stimulation and inhibition, and the binding of NGF and proNGF to p75NTR. Overall, these pathways underscore the multifaceted roles of p75NTR in cellular processes.


The mouse ortholog gene 18053 and its human ortholog gene 4804 are associated with several phenotypic abnormalities, including abnormal digit morphology, abnormal gait, limb grasping, preweaning lethality with incomplete penetrance, and tremors. These phenotypes are observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.278 (60.44th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -26.98 (7.19th percentile), and the median structural flexibility is 1.003 (56.79th percentile). The protein has a propensity for helix, sheet, and turn structures of 29.51% (27.62nd percentile), 29.74% (26.97th percentile), and 31.38% (73.12th percentile), respectively. The instability index is 58.70 (80.5th percentile), with an isoelectric point of 4.59 (2.1st percentile). The protein is 427 amino acids long (49.15th percentile) and has a molecular weight of 45183.01 Da (45.95th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |