---
layout: post
title: VLDLR
date: 2025-01-17 16:55 CST
description: VLDLR description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7436) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7436  | VLDLR | ENSG00000147852 | 9p24.2 |



The gene enables [low-density lipoprotein particle receptor activity](https://amigo.geneontology.org/amigo/term/GO:0005041), [calcium ion binding](https://amigo.geneontology.org/amigo/term/GO:0005509), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [very-low-density lipoprotein particle receptor activity](https://amigo.geneontology.org/amigo/term/GO:0030229), [apolipoprotein binding](https://amigo.geneontology.org/amigo/term/GO:0034185), [very-low-density lipoprotein particle binding](https://amigo.geneontology.org/amigo/term/GO:0034189), [cargo receptor activity](https://amigo.geneontology.org/amigo/term/GO:0038024), [reelin receptor activity](https://amigo.geneontology.org/amigo/term/GO:0038025), and [calcium-dependent protein binding](https://amigo.geneontology.org/amigo/term/GO:0048306). It is located in the [lysosomal membrane](https://amigo.geneontology.org/amigo/term/GO:0005765), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [clathrin-coated pit](https://amigo.geneontology.org/amigo/term/GO:0005905), and [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020). The gene is involved in [lipid transport](https://amigo.geneontology.org/amigo/term/GO:0006869), [receptor-mediated endocytosis](https://amigo.geneontology.org/amigo/term/GO:0006898), [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), [nervous system development](https://amigo.geneontology.org/amigo/term/GO:0007399), [memory](https://amigo.geneontology.org/amigo/term/GO:0007613), [cholesterol metabolic process](https://amigo.geneontology.org/amigo/term/GO:0008203), [ventral spinal cord development](https://amigo.geneontology.org/amigo/term/GO:0021517), [glycoprotein transport](https://amigo.geneontology.org/amigo/term/GO:0034436), [very-low-density lipoprotein particle clearance](https://amigo.geneontology.org/amigo/term/GO:0034447), [reelin-mediated signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0038026), [dendrite morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0048813), and [positive regulation of dendrite development](https://amigo.geneontology.org/amigo/term/GO:1900006). Additionally, it is part of the [very-low-density lipoprotein particle](https://amigo.geneontology.org/amigo/term/GO:0034361) and [receptor complex](https://amigo.geneontology.org/amigo/term/GO:0043235).


The gene length is 373,179 base pairs (97.06% of all genes), the mature length is 3,130.5 base pairs, and the primary transcript length is 99,582 base pairs (88.16% of all genes).


The gene VLDLR (NCBI ID: 7436) has been mentioned in [136 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22VLDLR%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1991, and the middle 50% of publications range from 1999 to 2015.


The top 5 publications mentioning VLDLR, ranked by their scientific influence, include "[VLDLR and ApoER2 are receptors for multiple alphaviruses.](https://pubmed.ncbi.nlm.nih.gov/34929721)" (2022) (relative citation ratio: 8.08), "[The proprotein convertase PCSK9 induces the degradation of low density lipoprotein receptor (LDLR) and its closest family members VLDLR and ApoER2.](https://pubmed.ncbi.nlm.nih.gov/18039658)" (2008) (relative citation ratio: 7.87), "[Structure, chromosome location, and expression of the human very low density lipoprotein receptor gene.](https://pubmed.ncbi.nlm.nih.gov/8294473)" (1994) (relative citation ratio: 5.81), "[Enhancement of the binding of triglyceride-rich lipoproteins to the very low density lipoprotein receptor by apolipoprotein E and lipoprotein lipase.](https://pubmed.ncbi.nlm.nih.gov/7797576)" (1995) (relative citation ratio: 4.93), and "[The very low density lipoprotein receptor mediates the cellular catabolism of lipoprotein lipase and urokinase-plasminogen activator inhibitor type I complexes.](https://pubmed.ncbi.nlm.nih.gov/7592875)" (1995) (relative citation ratio: 3.49). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[VLDLR](https://www.proteinatlas.org/ENSG00000147852-VLDLR) is a receptor involved in intellectual disability, with evidence at the protein level. It is detected in many tissues and is localized in vesicles, the cytokinetic bridge, and centriolar satellites. The gene is expressed in various clusters, including Cluster 18 for non-specific ribosome function, Cluster 30 for white matter myelination in the brain, Cluster 15 for kidney cancer membrane components in cell lines, and Cluster 68 for cardiomyocytes involved in muscle contraction in single cells.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [E2F1](https://www.ncbi.nlm.nih.gov/gene/1869), [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), and [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861), each with 5 experiments. Additionally, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) and [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) were found to be regulating in 4 experiments each.



The input data from GWAS indicates associations with a range of disease conditions, including various types of cancer such as lung cancer, lung adenocarcinoma, lung carcinoma, and non-small cell lung carcinoma. Additionally, the data highlights respiratory system diseases, including lower respiratory tract disease and lung disease. Metabolic disorders are also prominent, with links to obesity, overnutrition, and acquired metabolic disease. Cardiovascular conditions are well-represented, including coronary artery disease, artery disease, vascular disease, and atherosclerosis. Furthermore, the data includes nervous system diseases, such as epilepsy syndrome and brain disease, as well as inherited metabolic disorders like lipid metabolism disorder.



The gene is expressed in various tissues, including the heart and ovary, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the thyroid, fetal brain, smooth muscle, bronchial epithelial cells, pancreatic islet, ovary, and uterus.



The mouse ortholog gene 22359 and its human ortholog gene 7436 are associated with abnormal spleen morphology, a phenotype observed in both sexes.


The protein sequence analyzed has a GRAVY value of -0.418 (44.03rd percentile), indicating a hydrophilic nature. It exhibits a charge of -55.5 at pH 7.0 (1.93rd percentile) and a median structural flexibility of 1.001 (46.06th percentile). The protein's secondary structure is predicted to be 25.4% helix (6.03rd percentile), 29.8% sheet (27.14th percentile), and 34.1% turn (86.56th percentile). The instability index is 36.8 (13.84th percentile), with an isoelectric point of 4.62 (2.33rd percentile). The protein has a length of 873 amino acids (83.55th percentile) and a molecular weight of 96097.6 Da (82.99th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |