---
layout: post
title: INPP4B
date: 2025-01-17 16:55 CST
description: INPP4B description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8821) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8821  | INPP4B | ENSG00000109452 | 4q31.21 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), specifically located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). It is involved in the [phosphatidylinositol biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006661) and [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165). The gene enables [phosphatidylinositol-3,4-bisphosphate 4-phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0016316), [inositol-1,3,4-trisphosphate 4-phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0017161), and [inositol-3,4-bisphosphate 4-phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0052828). Additionally, it is involved in the [phosphatidylinositol-3-phosphate biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0036092), [inositol phosphate metabolic process](https://amigo.geneontology.org/amigo/term/GO:0043647), and [phosphatidylinositol dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0046856).


The gene length is 942.0 base pairs (1.65% of all genes). The mature length is 942.0 base pairs, and the primary transcript length is 942.0 base pairs.


The gene INPP4B (NCBI ID: 8821) has been mentioned in [68 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22INPP4B%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 2011 and the middle 50% of publications occurring between 2015 and 2019. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning INPP4B, ranked by their scientific influence, include "[SGK3 mediates INPP4B-dependent PI3K signaling in breast cancer.](https://pubmed.ncbi.nlm.nih.gov/25458846)" (2014) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 3.25, "[Exosomes derived from human bone marrow mesenchymal stem cells transfer miR-222-3p to suppress acute myeloid leukemia cell proliferation by targeting IRF2/INPP4B.](https://pubmed.ncbi.nlm.nih.gov/31968218)" (2020) with an RCR of 3.15, "[In Vivo Role of INPP4B in Tumor and Metastasis Suppression through Regulation of PI3K-AKT Signaling at Endosomes.](https://pubmed.ncbi.nlm.nih.gov/25883022)" (2015) with an RCR of 2.28, "[INPP4B: the new kid on the PI3K block.](https://pubmed.ncbi.nlm.nih.gov/21487159)" (2011) with an RCR of 2.22, and "[Mir-765 promotes cell proliferation by downregulating INPP4B expression in human hepatocellular carcinoma.](https://pubmed.ncbi.nlm.nih.gov/27062697)" (2016) with an RCR of 2.14. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[INPP4B](https://www.proteinatlas.org/ENSG00000109452-INPP4B) is a hydrolase with evidence at the protein level. It is detected in all tissues and is primarily localized to the centrosome. The gene is expressed in various clusters, including T-cells with unknown function in blood, spermatids involved in flagellum assembly in tissues, sub-cortical regions with mixed function in the brain, squamous epithelial cells involved in keratinization in cell lines, and neurons involved in neuronal signaling in single cells.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861), [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), and [ZNF143](https://www.ncbi.nlm.nih.gov/gene/7702), each shown to be regulating in 5 experiments.



The input data from GWAS indicates associations with a wide range of disease conditions, including various types of cancer such as lung cancer and respiratory system cancer, as well as hematopoietic and immune system diseases. Metabolic disorders are also prominent, with specific mentions of diabetes mellitus, particularly type 2 diabetes mellitus, and carbohydrate metabolism diseases. Additionally, the data highlights kidney and urinary system diseases, nervous system disorders including cognitive and mood disorders like bipolar disorder, and substance-related disorders such as alcohol dependence. Eye diseases, specifically retinal diseases like diabetic retinopathy, and inherited metabolic disorders such as lipid storage diseases and lysosomal storage diseases, are also noted. Furthermore, the data includes conditions related to the immune system, such as agammaglobulinemia and common variable immunodeficiency, as well as fatty liver disease.



The gene is expressed in breast tissue according to the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles indicate expression in 721_B_lymphoblasts, CD4+ T cells, CD8+ T cells, bronchial epithelial cells, skeletal muscle, and testis interstitial cells.


The input data highlights various pathways involved in metabolism, particularly focusing on lipid and lipoprotein metabolism. Key processes include phospholipid metabolism and inositol phosphate metabolism. Specifically, the synthesis of IP2, IP, and Ins in the cytosol, as well as PI metabolism, are noted. Additionally, the synthesis of PIPs at both the plasma membrane and the early endosome membrane are significant components of these metabolic pathways.



The analyzed protein sequence has a GRAVY value of -0.3703 (49.78th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -16.25 (14.24th percentile). The median structural flexibility is 1.002 (48.81st percentile), with a predicted secondary structure composition of 33.87% helix (66.83rd percentile), 35.82% sheet (65.06th percentile), and 27.38% turn (41.65th percentile). The instability index is 49.75 (56.41st percentile), and the isoelectric point is 5.87 (27.3rd percentile). The protein is 924 amino acids long (85.31st percentile) with a molecular weight of 104736.77 Da (85.84th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |