---
layout: post
title: TGFBR3
date: 2025-01-17 16:55 CST
description: TGFBR3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7049) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7049  | TGFBR3 | ENSG00000069702 | 1p22.1 |



The gene is involved in various biological processes, including [response to hypoxia](https://amigo.geneontology.org/amigo/term/GO:0001666), [epithelial to mesenchymal transition](https://amigo.geneontology.org/amigo/term/GO:0001837), [liver development](https://amigo.geneontology.org/amigo/term/GO:0001889), [heart morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0003007), and [muscular septum morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0003150). It enables functions such as [transmembrane signaling receptor activity](https://amigo.geneontology.org/amigo/term/GO:0004888), [transforming growth factor beta receptor activity](https://amigo.geneontology.org/amigo/term/GO:0005024), and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is located in the [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), and the [external side of the plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009897). Additionally, it is involved in the [transforming growth factor beta receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007179) and contributes to [transforming growth factor beta binding](https://amigo.geneontology.org/amigo/term/GO:0050431). The gene also plays a role in processes like [cell migration](https://amigo.geneontology.org/amigo/term/GO:0016477), [positive regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0010628), and [animal organ regeneration](https://amigo.geneontology.org/amigo/term/GO:0031100).


The gene length is 362,890 base pairs (96.87th percentile), the mature length is 4,595 base pairs (88.07th percentile), and the primary transcript length is 359,450 base pairs (98.49th percentile).


The gene TGFBR3 (NCBI ID: 7049) has been mentioned in [95 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22TGFBR3%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning this gene is 1989, and the middle 50% of publications occurred between 2005 and 2019.


The top five publications mentioning TGFBR3, ranked by their scientific influence, include "[Betaglycan presents ligand to the TGF beta signaling receptor.](https://pubmed.ncbi.nlm.nih.gov/8391934)" (1993) (relative citation ratio: 16.97), "[Structure and expression of the membrane proteoglycan betaglycan, a component of the TGF-beta receptor system.](https://pubmed.ncbi.nlm.nih.gov/1657406)" (1991) (relative citation ratio: 13.53), "[Macrophage-derived exosomal microRNA-501-3p promotes progression of pancreatic ductal adenocarcinoma through the TGFBR3-mediated TGF-Î² signaling pathway.](https://pubmed.ncbi.nlm.nih.gov/31307515)" (2019) (relative citation ratio: 9.9), "[Betaglycan binds inhibin and can mediate functional antagonism of activin signalling.](https://pubmed.ncbi.nlm.nih.gov/10746731)" (2000) (relative citation ratio: 9.0), and "[Membrane-anchored and soluble forms of betaglycan, a polymorphic proteoglycan that binds transforming growth factor-beta.](https://pubmed.ncbi.nlm.nih.gov/2592419)" (1989) (relative citation ratio: 7.51). The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[TGFBR3](https://www.proteinatlas.org/ENSG00000069702-TGFBR3) is a receptor involved in cancer-related genes, with evidence at the protein level. Its RNA is detected in all tissues, and it is primarily located in the cytosol. The gene is expressed in various clusters, including NK-cells with unknown function in blood, connective tissue for ECM organization in tissue, astrocytes with mixed function in the brain, connective tissue cells for ECM organization in cell lines, and fibroblasts for ECM organization in single cells.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [HIF1A](https://www.ncbi.nlm.nih.gov/gene/3091), [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025), [PHF8](https://www.ncbi.nlm.nih.gov/gene/23133), [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), and [AR](https://www.ncbi.nlm.nih.gov/gene/367), each shown to be regulating in 4 experiments.



The input data from GWAS indicates associations with a wide range of disease conditions, including various types of cancer such as male reproductive organ cancer, testicular cancer, lung cancer, and organ system cancer. Additionally, the data highlights metabolic disorders like type 2 diabetes mellitus, glucose metabolism disease, and acquired metabolic disease. Cardiovascular conditions such as coronary artery disease, hypertension, and congestive heart failure are also represented. Neurological and mental health disorders are prominent, including Alzheimer's disease, bipolar disorder, and substance dependence. Furthermore, the data includes immune system diseases, musculoskeletal system diseases like rheumatoid arthritis, and autoimmune conditions such as celiac disease. Genetic and chromosomal diseases, including familial atrial fibrillation and gonadal dysgenesis, are also noted.



The gene is expressed in adipose tissue and ovary according to BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles indicate expression in CD56+ NK cells, adipocytes, placenta, pancreatic islet, adrenal cortex, ovary, and uterus.



The mouse ortholog gene 21814 and its human ortholog 7049 are associated with several abnormal phenotypes observed in both sexes, including abnormal blood vessel morphology, abnormal kidney morphology, abnormal placenta morphology and vasculature, edema, embryonic growth retardation, enlarged kidney, hemorrhage, microphthalmia, pale liver, and complete penetrance preweaning lethality.


The analyzed protein sequence has a GRAVY value of -0.20329 (68.82nd percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -24.342 (8.46th percentile), and the median structural flexibility is 1.002 (51.54th percentile). The protein has a propensity for helix, sheet, and turn structures of 28.79% (22.03rd percentile), 34.90% (58.3rd percentile), and 32.90% (81.46th percentile), respectively. The instability index is 47.741 (49.2nd percentile), with an isoelectric point of 5.452 (17.02nd percentile). The protein is 851 amino acids long (82.67th percentile) and has a molecular weight of 93498.29 Da (82.13th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |