---
layout: post
title: ELMO1
date: 2025-01-17 16:55 CST
description: ELMO1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9844) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9844  | ELMO1 | ENSG00000155849 | 7p14.2-p14.1 |



The gene contributes to [guanyl-nucleotide exchange factor activity](https://amigo.geneontology.org/amigo/term/GO:0005085) and enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [SH3 domain binding](https://amigo.geneontology.org/amigo/term/GO:0017124). It is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), and [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), and is active in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). It is involved in various processes including [phagocytosis, engulfment](https://amigo.geneontology.org/amigo/term/GO:0006911), [apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0006915), [actin filament organization](https://amigo.geneontology.org/amigo/term/GO:0007015), [Rac protein signal transduction](https://amigo.geneontology.org/amigo/term/GO:0016601), [actin cytoskeleton organization](https://amigo.geneontology.org/amigo/term/GO:0030036), and [cell motility](https://amigo.geneontology.org/amigo/term/GO:0048870). Additionally, it is part of the [guanyl-nucleotide exchange factor complex](https://amigo.geneontology.org/amigo/term/GO:0032045).


The gene length is 196,586 base pairs (92.27% of all genes), the mature length is 5,811 base pairs (93.56% of all genes), and the primary transcript length is 160,593 base pairs (94.28% of all genes).


ELMO1 has been mentioned in [62 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ELMO1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest mention in 2001. The middle 50% of publications occurred between 2012 and 2019. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning ELMO1, ranked by their scientific influence, include "[CED-12/ELMO, a novel member of the CrkII/Dock180/Rac pathway, is required for phagocytosis and cell migration.](https://pubmed.ncbi.nlm.nih.gov/11595183)" (2001) (relative citation ratio: 7.11), "[TNF-Î±-mediated m<sup>6</sup>A modification of ELMO1 triggers directional migration of mesenchymal stem cell in ankylosing spondylitis.](https://pubmed.ncbi.nlm.nih.gov/34508078)" (2021) (relative citation ratio: 4.68), "[Dock180 and ELMO1 proteins cooperate to promote evolutionarily conserved Rac-dependent cell migration.](https://pubmed.ncbi.nlm.nih.gov/14638695)" (2004) (relative citation ratio: 3.07), "[Genetic variations in the gene encoding ELMO1 are associated with susceptibility to diabetic nephropathy.](https://pubmed.ncbi.nlm.nih.gov/15793258)" (2005) (relative citation ratio: 2.96), and "[Unexpected requirement for ELMO1 in clearance of apoptotic germ cells in vivo.](https://pubmed.ncbi.nlm.nih.gov/20844538)" (2010) (relative citation ratio: 2.89). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ELMO1](https://www.proteinatlas.org/ENSG00000155849-ELMO1) is a gene with evidence at the protein level. Its RNA is detected in all tissues, and it is notably expressed in oligodendrocytes, contributing to myelination. In the brain, it is involved in signal transduction within white matter. Additionally, ELMO1 is expressed in leukemia cell lines and is associated with the innate immune response in blood. However, it is unprognostic for various cancer types, including bladder urothelial carcinoma, breast invasive carcinoma, and others.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313), [MYB](https://www.ncbi.nlm.nih.gov/gene/4602), [EZH2](https://www.ncbi.nlm.nih.gov/gene/2146), and [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113), each with 5 experiments. Additionally, [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) was found to be regulating in 4 experiments.



The input data from GWAS indicates associations with a wide range of disease conditions, including various types of cancer such as integumentary system cancer, skin cancer, skin melanoma, breast cancer, thoracic cancer, male reproductive organ cancer, prostate cancer, testicular cancer, head and neck cancer, colorectal cancer, intestinal cancer, large intestine cancer, gastrointestinal system cancer, myeloid leukemia, acute myeloid leukemia, colon cancer, lung cancer, respiratory system cancer, leukemia, hematologic cancer, and immune system cancer. Additionally, the data includes associations with infectious diseases like hand, foot and mouth disease, viral infectious disease, and human immunodeficiency virus infectious disease. Metabolic disorders such as diabetes mellitus, type 2 diabetes mellitus, obesity, and cardiovascular diseases like coronary artery disease, artery disease, vascular disease, and hypertension are also represented. Neurological conditions include Alzheimer's disease, tauopathy, dementia, cognitive disorder, bipolar disorder, mood disorder, anxiety disorder, obsessive-compulsive disorder, tic disorder, Gilles de la Tourette syndrome, attention deficit hyperactivity disorder, and multiple sclerosis. Other notable conditions include rheumatoid arthritis, kidney failure, chronic kidney failure, end stage renal failure, narcolepsy, celiac disease, liver cirrhosis, primary biliary cirrhosis, long QT syndrome, myocardial infarction, pneumonia, viral pneumonia, psoriasis, acquired immunodeficiency syndrome, and vein disease.



The gene is ubiquitously expressed in multiple tissue samples, including nerve, thyroid, brain, and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in various brain regions such as the whole brain, thalamus, cingulate cortex, subthalamic nucleus, medulla oblongata, and cerebellum.


The proteins are involved in various pathways related to HIV infection, including host interactions of HIV factors, immune system processes, and innate immune responses. They also participate in signaling pathways such as those mediated by VEGF and the VEGFA-VEGFR2 pathway. Additionally, these proteins play roles in Fcgamma receptor (FCGR) dependent phagocytosis and the regulation of actin dynamics for phagocytic cup formation. Furthermore, they are implicated in disease processes, signal transduction, and the specific roles of Nef in HIV-1 replication and disease pathogenesis.


The mouse ortholog gene 140580 and its human ortholog 9844 are associated with phenotypes including increased grip strength and increased leukocyte cell number, with these effects observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.383 (48.14th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -11.84 (19.66th percentile), and the median structural flexibility is 1.002 (52.5th percentile). The protein has a high affinity for helix (0.354, 76.95th percentile) and sheet (0.347, 56.5th percentile) structures, with a lower affinity for turns (0.256, 25.82nd percentile). The instability index is 48.004 (50.24th percentile), and the isoelectric point is 5.89 (27.59th percentile). The protein is 727 amino acids long (76.95th percentile) with a molecular weight of 83828.53 Da (78.29th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |