---
layout: post
title: GYG1
date: 2025-01-17 16:55 CST
description: GYG1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2992) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2992  | GYG1 | ENSG00000163754 | 3q24 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [secretory granule lumen](https://amigo.geneontology.org/amigo/term/GO:0034774), [lysosomal lumen](https://amigo.geneontology.org/amigo/term/GO:0043202), and [ficolin-1-rich granule lumen](https://amigo.geneontology.org/amigo/term/GO:1904813). It is involved in the [glycogen biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0005978) and enables [glycogenin glucosyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0008466), [glycosyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0016757), [manganese ion binding](https://amigo.geneontology.org/amigo/term/GO:0030145), [protein homodimerization activity](https://amigo.geneontology.org/amigo/term/GO:0042803), and [UDP-alpha-D-glucose:glucosyl-glycogenin alpha-D-glucosyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0102751).


The gene length is 37,866 base pairs (58.92% of all genes), the mature length is 2,329 base pairs (60.33% of all genes), and the primary transcript length is 36,467 base pairs (67.79% of all genes).


The gene GYG1 (NCBI ID: 2992) has been mentioned in [24 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22GYG1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1999, and the middle 50% of publications occurred between 2013 and 2018.


The top 5 publications mentioning GYG1, ranked by their scientific influence, include "[A new muscle glycogen storage disease associated with glycogenin-1 deficiency.](https://pubmed.ncbi.nlm.nih.gov/25272951)" (2014) (relative citation ratio: 2.65), "[Glycogenin-1 deficiency and inactivated priming of glycogen synthesis.](https://pubmed.ncbi.nlm.nih.gov/20357282)" (2010) (relative citation ratio: 2.35), "[Cardiomyopathy as presenting sign of glycogenin-1 deficiency-report of three cases and review of the literature.](https://pubmed.ncbi.nlm.nih.gov/27718144)" (2017) (relative citation ratio: 1.49), "[Muscle pathology and whole-body MRI in a polyglucosan myopathy associated with a novel glycogenin-1 mutation.](https://pubmed.ncbi.nlm.nih.gov/26255073)" (2015) (relative citation ratio: 1.19), and "[The structural mechanism of human glycogen synthesis by the GYS1-GYG1 complex.](https://pubmed.ncbi.nlm.nih.gov/35793618)" (2022) (relative citation ratio: 1.0). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[GYG1](https://www.proteinatlas.org/ENSG00000163754-GYG1) is a transferase involved in glycogen storage disease, with evidence supporting its function at the protein level. The gene is detected in all tissues and is primarily localized in vesicles. Its expression is notable in monocytes and neutrophils, contributing to the innate immune response. Additionally, GYG1 is involved in basic cellular processes and neurotransmitter signaling in the brain. It is also expressed in lymphoma cell lines, suggesting a role in the inflammatory response, and in spermatocytes and spermatids, indicating involvement in spermatid development.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [AR](https://www.ncbi.nlm.nih.gov/gene/367), [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169), [TAL1](https://www.ncbi.nlm.nih.gov/gene/6886), [JUN](https://www.ncbi.nlm.nih.gov/gene/3725), and [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), each shown to be regulating in 4 experiments.



The GWAS data indicates associations with various disease conditions, including metabolic disorders such as diabetes mellitus and type 2 diabetes mellitus, as well as mental health conditions like cognitive disorder, bipolar disorder, and mood disorder.


The gene GYG1 has been associated with the trait [Height](https://pubmed.ncbi.nlm.nih.gov/34226706) through genome-wide association studies.


The gene is expressed in various tissues, including muscle and blood, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in bone marrow, skeletal muscle, and multiple testis-related tissues, such as interstitial, Leydig cells, seminiferous tubules, and germ cells.


The input data highlights several pathways related to carbohydrate metabolism, including glycogen synthesis, glycogen breakdown (glycogenolysis), and glucose metabolism. Additionally, it covers diseases associated with glycogen storage and myoclonic epilepsy of Lafora.


The mouse ortholog gene 27357 and its human ortholog gene 2992 are associated with several phenotypes, including preweaning lethality with complete penetrance, unresponsiveness to tactile stimuli, and vertebral fusion. These phenotypes are observed in both sexes.


The protein sequence analyzed has a GRAVY value of -0.11028571428571429 (77.32 percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is -10.734275874496358 (21.39 percentile), and the median structural flexibility is 0.9941190476190476 (18.42 percentile). The protein has a helix affinity of 0.2914285714285714 (24.59 percentile), a sheet affinity of 0.41142857142857137 (89.0 percentile), and a turn affinity of 0.2914285714285714 (57.08 percentile). The instability index is 36.76057142857141 (13.68 percentile), with an isoelectric point of 5.267974281311035 (12.82 percentile). The protein has a length of 350 amino acids (38.03 percentile) and a molecular weight of 39383.30810000006 Da (38.61 percentile). These analyses were conducted using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |