---
layout: post
title: FURIN
date: 2024-12-08 23:57 CST
description: FURIN description
tags: [cooccuring-genes]
categories: shortest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5045) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5045  | FURIN | ENSG00000140564 | 15q26.1 |



The gene length is 736.0 base pairs (0.93% of all genes), the mature length is 635.0 base pairs, and the primary transcript length is 728.0 base pairs.


The gene FURIN, with NCBI ID 5045, has been featured in [847 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22FURIN%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1964, and the middle 50% of publications occurred between 2002 and 2019.


The top 5 publications mentioning FURIN, ranked by their scientific influence, include "[The spike glycoprotein of the new coronavirus 2019-nCoV contains a furin-like cleavage site absent in CoV of the same clade.](https://pubmed.ncbi.nlm.nih.gov/32057769)" (2020) (relative citation ratio: 87.91), "[Loss of furin cleavage site attenuates SARS-CoV-2 pathogenesis.](https://pubmed.ncbi.nlm.nih.gov/33494095)" (2021) (relative citation ratio: 44.3), "[The furin cleavage site in the SARS-CoV-2 spike protein is required for transmission in ferrets.](https://pubmed.ncbi.nlm.nih.gov/33907312)" (2021) (relative citation ratio: 39.69), "[TMPRSS2 and furin are both essential for proteolytic activation of SARS-CoV-2 in human airway cells.](https://pubmed.ncbi.nlm.nih.gov/32703818)" (2020) (relative citation ratio: 38.51), and "[DunedinPACE, a DNA methylation biomarker of the pace of aging.](https://pubmed.ncbi.nlm.nih.gov/35029144)" (2022) (relative citation ratio: 33.59). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613), [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), and [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), each with 5 experiments. Additionally, [MBL2](https://www.ncbi.nlm.nih.gov/gene/4153) and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) were found to be regulating in 4 experiments each.








The mouse ortholog gene 18550 and its human ortholog gene 5045 are associated with several phenotypes. Both sexes exhibit abnormal vibrissa morphology and preweaning lethality with complete penetrance. Additionally, brachydactyly and embryonic lethality prior to the tooth bud stage are observed, although these phenotypes do not affect both sexes equally.


The analyzed protein sequence has a GRAVY value of -0.390 (47.36th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -12.33 (18.91st percentile). The median structural flexibility is 0.999 (37.3rd percentile), with a secondary structure composition of 26.57% helix (9.87th percentile), 33.25% sheet (47.02nd percentile), and 31.49% turn (73.76th percentile). The instability index is 43.28 (33.23rd percentile), and the isoelectric point is 6.01 (30.84th percentile). The protein has a length of 794 amino acids (80.43rd percentile) and a molecular weight of 86677.01 Da (79.46th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |