---
layout: post
title: PSAP
date: 2024-12-08 23:57 CST
description: PSAP description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5660) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5660  | PSAP | ENSG00000197746 | 10q22.1 |



The gene length is 25,905 base pairs (48.63% of all genes), the mature length is 1,237.5 base pairs, and the primary transcript length is 24,570 base pairs.


The gene PSAP has an NCBI Gene ID of 5660. It has been featured in [134 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PSAP%22) (note: the count in the hyperlink may be different since the count is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1984, and the middle 50% of publications occurred between 1997 and 2017.


The top five publications mentioning PSAP, ranked by their scientific influence, include "[Coding of two sphingolipid activator proteins (SAP-1 and SAP-2) by same genetic locus.](https://pubmed.ncbi.nlm.nih.gov/2842863)" (1988) (relative citation ratio: 6.86), "[Disruption of each of the secreted aspartyl proteinase genes SAP1, SAP2, and SAP3 of Candida albicans attenuates virulence.](https://pubmed.ncbi.nlm.nih.gov/9284116)" (1997) (relative citation ratio: 5.85), "[Identification of prosaposin as a neurotrophic factor.](https://pubmed.ncbi.nlm.nih.gov/7937812)" (1994) (relative citation ratio: 4.58), "[GPR37 and GPR37L1 are receptors for the neuroprotective and glioprotective factors prosaptide and prosaposin.](https://pubmed.ncbi.nlm.nih.gov/23690594)" (2013) (relative citation ratio: 4.4), and "[Variants in saposin D domain of prosaposin gene linked to Parkinson's disease.](https://pubmed.ncbi.nlm.nih.gov/32201884)" (2020) (relative citation ratio: 3.97). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


The gene [Gene](https://www.proteinatlas.org/[Ensembl]-[Gene]) is implicated in various diseases, including Gangliosidosis, Gaucher disease, Leukodystrophy, Metachromatic leukodystrophy, Neurodegeneration, Parkinson disease, and Parkinsonism. Evidence for its involvement is supported at the protein level. The gene's RNA is detected in all tissues, with subcellular localization in vesicles. It is expressed in monocytes and neutrophils, involved in degranulation, and is part of non-specific mitochondrial and metabolic clusters in various tissues and cell lines. Additionally, it plays a role in the innate immune response in monocytes.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [CEBPA](https://www.ncbi.nlm.nih.gov/gene/1050), [CREBBP](https://www.ncbi.nlm.nih.gov/gene/1387), [E2F8](https://www.ncbi.nlm.nih.gov/gene/79733), [AR](https://www.ncbi.nlm.nih.gov/gene/367), and [OTX2](https://www.ncbi.nlm.nih.gov/gene/5015), each shown to be regulating in 3 experiments.


Based on the data from GTRD's database of transcription factor (TF) binding sites and filtered for Perturb-seq data, several transcription factors were identified as regulators in multiple experiments. Notably, [CTCF](https://www.ncbi.nlm.nih.gov/gene/6829) and [NANOG](https://www.ncbi.nlm.nih.gov/gene/2623) were each found to regulate gene expression in two experiments. Additionally, [TCF7L2](https://www.ncbi.nlm.nih.gov/gene/7023), [ZNF274](https://www.ncbi.nlm.nih.gov/gene/8243), [KLF12](https://www.ncbi.nlm.nih.gov/gene/8861), [RORA](https://www.ncbi.nlm.nih.gov/gene/9968), and [NR2F2](https://www.ncbi.nlm.nih.gov/gene/9126) were each identified in one experiment.


The genome-wide association studies have identified associations with [Granulins levels](https://pubmed.ncbi.nlm.nih.gov/34648354) and [Prosaposin levels](https://pubmed.ncbi.nlm.nih.gov/34648354). These traits have been mapped to their closest MeSH terms using [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0), with Granulins levels corresponding to [Granulins](https://meshb.nlm.nih.gov/record/ui?ui=D000077155) and Prosaposin levels corresponding to [Saposins](https://meshb.nlm.nih.gov/record/ui?ui=D049231).





The protein sequence analyzed has a GRAVY value of -0.064 (80.51 percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is -20.76 (10.63 percentile), and the median structural flexibility is 0.998 (33.02 percentile). The protein has a high affinity for helix structures (37.40%, 86.68 percentile) and sheet structures (34.16%, 53.18 percentile), with a lower affinity for turn structures (25%, 21.6 percentile). The instability index is 47.52 (48.39 percentile), and the isoelectric point is 5.06 (8.92 percentile). The protein has a length of 524 amino acids (61.19 percentile) and a molecular weight of 58112.11 Da (60.79 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |