---
layout: post
title: FTL
date: 2025-01-17 16:55 CST
description: FTL description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2512) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2512  | FTL | ENSG00000087086 | 19q13.33 |



The gene enables [iron ion binding](https://amigo.geneontology.org/amigo/term/GO:0005506) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), including [ferrous iron binding](https://amigo.geneontology.org/amigo/term/GO:0008198) and [ferric iron binding](https://amigo.geneontology.org/amigo/term/GO:0008199). It is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [azurophil granule lumen](https://amigo.geneontology.org/amigo/term/GO:0035578), [autolysosome](https://amigo.geneontology.org/amigo/term/GO:0044754), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). The gene is involved in [iron ion transport](https://amigo.geneontology.org/amigo/term/GO:0006826), [intracellular iron ion homeostasis](https://amigo.geneontology.org/amigo/term/GO:0006879), and [intracellular sequestering of iron ion](https://amigo.geneontology.org/amigo/term/GO:0006880). Additionally, it is part of the [intracellular ferritin complex](https://amigo.geneontology.org/amigo/term/GO:0008043) and enables [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802).


The gene length is 28,873 base pairs (51.7% of all genes), the mature length is 2,065 base pairs, and the primary transcript length is 28,873 base pairs.


The gene FTL (NCBI ID: 2512) has been mentioned in [66 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22FTL%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning FTL was in 1985, and the middle 50% of publications occurred between 2005 and 2018.


The top 5 publications mentioning FTL, ranked by their scientific influence, include "[Mutation in the gene encoding ferritin light polypeptide causes dominant adult-onset basal ganglia disease.](https://pubmed.ncbi.nlm.nih.gov/11438811)" (2001) (relative citation ratio: 8.35), "[The NSUN5-FTH1/FTL pathway mediates ferroptosis in bone marrow-derived mesenchymal stem cells.](https://pubmed.ncbi.nlm.nih.gov/35249107)" (2022) (relative citation ratio: 3.82), "[Interplay of Ferritin Accumulation and Ferroportin Loss in Ageing Brain: Implication for Protein Aggregation in Down Syndrome Dementia, Alzheimer's, and Parkinson's Diseases.](https://pubmed.ncbi.nlm.nih.gov/35162984)" (2022) (relative citation ratio: 3.4), "[Ferritin Light Chain Confers Protection Against Sepsis-Induced Inflammation and Organ Injury.](https://pubmed.ncbi.nlm.nih.gov/30804939)" (2019) (relative citation ratio: 3.33), and "[Hereditary hyperferritinemia-cataract syndrome: relationship between phenotypes and specific mutations in the iron-responsive element of ferritin light-chain mRNA.](https://pubmed.ncbi.nlm.nih.gov/9226182)" (1997) (relative citation ratio: 3.07). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[FTL](https://www.proteinatlas.org/ENSG00000087086-FTL) is a gene involved in various diseases, including cancer, cataract, and neurodegeneration. It is also a target for FDA-approved drugs. The gene's expression is detected in all RNA tissues, with notable expression in the liver and plasma proteins. At the subcellular level, FTL is located in the cytosol. In single-cell expression analysis, it is associated with macrophages and the innate immune response. Evidence for the gene's function is established at the protein level.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [CEBPB](https://www.ncbi.nlm.nih.gov/gene/1051), [STAT3](https://www.ncbi.nlm.nih.gov/gene/6774), [HNF4A](https://www.ncbi.nlm.nih.gov/gene/3172), [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861), and [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599), each shown to be regulating in 4 experiments.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, filtered for Perturb-seq data, include [BRCA1](https://www.ncbi.nlm.nih.gov/gene/6829), [MTA3](https://www.ncbi.nlm.nih.gov/gene/863), [ZNF384](https://www.ncbi.nlm.nih.gov/gene/7936), [ZNF281](https://www.ncbi.nlm.nih.gov/gene/6872), and [ZNF263](https://www.ncbi.nlm.nih.gov/gene/8208). Among these, ZNF281 was shown to be regulating genes in the highest number of Perturb-seq experiments, with a total of 3 experiments. BRCA1, MTA3, ZNF384, and ZNF263 were each found to be regulating genes in 1 experiment. Additionally, [ZNF274](https://www.ncbi.nlm.nih.gov/gene/9968), [ZNF559](https://www.ncbi.nlm.nih.gov/gene/57634), and [ZNF596](https://www.ncbi.nlm.nih.gov/gene/55929) were each found to be regulating genes in 2 experiments.




The gene is expressed in various tissues, including blood and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the liver, adipocytes, lung, kidney, and fetal liver.


The proteins are involved in various pathways related to membrane trafficking and transport processes. These include scavenging by class A receptors, iron uptake and transport, transmembrane transport of small molecules, binding and uptake of ligands by scavenger receptors, clathrin-derived vesicle budding, trans-Golgi network vesicle budding, Golgi-associated vesicle biogenesis, and general membrane trafficking.


The mouse ortholog gene 14325 and its human ortholog gene 2512 are associated with several phenotypes. Notably, they are linked to abnormal circulating iron levels in a sex-specific manner, as well as abnormal tooth morphology, short tibia, and thrombocytosis, which are observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.52 (32.93rd percentile), indicating a hydrophilic nature. The charge at pH 7 is -6.86 (28.83rd percentile), and the median structural flexibility is 1.001 (48.37th percentile). The protein has a high affinity for helix structures (0.43, 96.84th percentile), with moderate affinities for sheet (0.34, 50.01st percentile) and turn (0.25, 18.54th percentile) structures. The instability index is 24.62 (1.44th percentile), and the isoelectric point is 5.51 (18.56th percentile). The protein is 175 amino acids long (11th percentile) with a molecular weight of 20019.44 Da (11.45th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |