---
layout: post
title: PLPP2
date: 2024-12-08 23:57 CST
description: PLPP2 description
tags: [cooccuring-genes]
categories: longest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8612) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8612  | PLPP2 | ENSG00000141934 | 19p13.3 |



The gene length is 2,304,997.0 base pairs (99.99% of all genes), the mature length is 4,010.0 base pairs (83.89% of all genes), and the primary transcript length is 503,058.5 base pairs (99.25% of all genes).


The gene PLPP2, with NCBI ID 8612, has been the subject of [3 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PLPP2%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1969, and the middle 50% of publications occurred between 1989 and 2014.


The top publications mentioning PLPP2 include "[Genomics screen in transformed stem cells reveals RNASEH2A, PPAP2C, and ADARB1 as putative anticancer drug targets.](https://pubmed.ncbi.nlm.nih.gov/19139135)" (2009) (relative citation ratio: 1.3), "[Circular RNA ZNF609 Promoted Hepatocellular Carcinoma Progression by Upregulating PAP2C Expression via Sponging miR-342-3p.](https://pubmed.ncbi.nlm.nih.gov/32801783)" (2020) (relative citation ratio: 0.92), and "[Serological and electron microscopic characterization of a new group of blue-green algal viruses (LPP-2).](https://pubmed.ncbi.nlm.nih.gov/4982613)" (1969) (relative citation ratio: 0.65). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [AR](https://www.ncbi.nlm.nih.gov/gene/367) and [SP1](https://www.ncbi.nlm.nih.gov/gene/6667), each with 4 experiments. Additionally, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), [DNMT3B](https://www.ncbi.nlm.nih.gov/gene/1789), and [E2F6](https://www.ncbi.nlm.nih.gov/gene/1876) were each found to be regulating in 3 experiments.











The analyzed protein sequence has a GRAVY value of 0.228 (91.04th percentile), indicating a relatively hydrophobic nature. The charge at pH 7.0 is 5.12 (68.49th percentile), and the median structural flexibility is 0.98 (4.47th percentile). The protein has a high affinity for sheet structures (0.448, 94.32nd percentile), with moderate affinities for helix (0.295, 27.67th percentile) and turn (0.243, 16.91st percentile) structures. The instability index is 36.35 (12.76th percentile), and the isoelectric point is 8.65 (70th percentile). The protein is 288 amino acids long (26.66th percentile) with a molecular weight of 32573.4 Da (27.33rd percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |