---
layout: post
title: CD59
date: 2025-01-17 16:55 CST
description: CD59 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/966) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 966  | CD59 | ENSG00000085063 | 11p13 |



The gene is located in the [Golgi membrane](https://amigo.geneontology.org/amigo/term/GO:0000139), [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615), [endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0005789), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925), [external side of plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009897), [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986), [ER to Golgi transport vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0012507), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [transport vesicle](https://amigo.geneontology.org/amigo/term/GO:0030133), [vesicle](https://amigo.geneontology.org/amigo/term/GO:0031982), [endoplasmic reticulum-Golgi intermediate compartment membrane](https://amigo.geneontology.org/amigo/term/GO:0033116), [specific granule membrane](https://amigo.geneontology.org/amigo/term/GO:0035579), [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062), and [tertiary granule membrane](https://amigo.geneontology.org/amigo/term/GO:0070821). It enables [complement binding](https://amigo.geneontology.org/amigo/term/GO:0001848) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is involved in the [negative regulation of activation of membrane attack complex](https://amigo.geneontology.org/amigo/term/GO:0001971), [cell surface receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007166), and [blood coagulation](https://amigo.geneontology.org/amigo/term/GO:0007596). Additionally, it acts upstream of or within the [regulation of complement activation](https://amigo.geneontology.org/amigo/term/GO:0030449) and [regulation of complement-dependent cytotoxicity](https://amigo.geneontology.org/amigo/term/GO:1903659).


The gene length is 76,252.0 base pairs (76.36% of all genes), the mature length is 721.5 base pairs, and the primary transcript length is 39,080.5 base pairs.


The gene CD59 (NCBI ID: 966) has been mentioned in [204 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CD59%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning CD59 is 1971, and the middle 50% of publications occurred between 1995 and 2014.


The top five publications mentioning CD59, ranked by their scientific influence, include studies that delve into the gene's role in complement inhibition and its implications in various biological processes. The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. The most influential publication is "[Human protectin (CD59), an 18,000-20,000 MW complement lysis restricting factor, inhibits C5b-8 catalysed insertion of C9 into lipid bilayers.](https://pubmed.ncbi.nlm.nih.gov/1698710)" (1990) (relative citation ratio: 12.17). Another significant study is "[The complement-inhibitory activity of CD59 resides in its capacity to block incorporation of C9 into membrane C5b-9.](https://pubmed.ncbi.nlm.nih.gov/1691760)" (1990) (relative citation ratio: 8.02). Additionally, "[Inhibition of homologous complement by CD59 is mediated by a species-selective recognition conferred through binding to C8 within C5b-8 or C9 within C5b-9.](https://pubmed.ncbi.nlm.nih.gov/1706395)" (1991) (relative citation ratio: 5.12) provides further insights into CD59's mechanisms. More recent work includes "[Protectin D1n-3 DPA and resolvin D5n-3 DPA are effectors of intestinal protection.](https://pubmed.ncbi.nlm.nih.gov/28356517)" (2017) (relative citation ratio: 5.04) and "[Role of virion-associated glycosylphosphatidylinositol-linked proteins CD55 and CD59 in complement resistance of cell line-derived and primary isolates of HIV-1.](https://pubmed.ncbi.nlm.nih.gov/7543140)" (1995) (relative citation ratio: 4.55). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CD59](https://www.proteinatlas.org/ENSG00000085063-CD59) is a gene involved in cancer-related processes and hereditary hemolytic anemia, with evidence supporting its role at the protein level. The gene's RNA is detected in all tissues, and its protein is localized to the Golgi apparatus and vesicles. CD59 is associated with various expression clusters, including Cluster 42 for non-specific vesicular transport in blood, Cluster 60 for transmembrane transport in choroid plexus tissue, Cluster 44 for mixed function in the brain, Cluster 64 for ECM organization in connective tissue cells, and Cluster 45 for mixed function in single cells.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [AR](https://www.ncbi.nlm.nih.gov/gene/367), [E2F1](https://www.ncbi.nlm.nih.gov/gene/1869), [HIRA](https://www.ncbi.nlm.nih.gov/gene/7290), [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046), and [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), each shown to be regulating in 5 experiments.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [9968](https://www.ncbi.nlm.nih.gov/gene/9968), which was shown to be regulating genes in 2 experiments. Additionally, [2118](https://www.ncbi.nlm.nih.gov/gene/2118), [8243](https://www.ncbi.nlm.nih.gov/gene/8243), and [57634](https://www.ncbi.nlm.nih.gov/gene/57634) were each found to be regulating genes in 1 experiment.


The GWAS data indicates associations with a range of conditions, including coronary artery disease, artery disease, vascular disease, and broader cardiovascular system diseases. Additionally, the data highlights links to kidney disease and urinary system disease. Furthermore, cognitive disorders, diseases of mental health, psychotic disorders, and specifically schizophrenia are also implicated.



The gene is expressed in various tissues, including nerve, salivary gland, and lung, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles show expression in cardiac myocytes, smooth muscle, placenta, salivary gland, dorsal root ganglion, and olfactory bulb.


The input data highlights several pathways related to the immune system, including the innate immune system, regulation of the complement cascade, and the complement cascade itself.



The protein sequence analyzed has a GRAVY value of 0.0664 (86.8th percentile), indicating a relatively hydrophobic nature. The charge at pH 7.0 is -2.25 (42.24th percentile), and the median structural flexibility is 0.9864 (8.57th percentile). The protein has a propensity for helix, turn, and sheet structures of 31.25% (43.54th percentile), 27.34% (41.25th percentile), and 39.06% (83.25th percentile), respectively. The instability index is 28.4 (2.91st percentile), with an isoelectric point of 6.02 (31.23rd percentile). The sequence length is 128 amino acids (5.57th percentile), and the molecular weight is 14177.16 Da (5.46th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |