---
layout: post
title: ICAM2
date: 2025-01-17 16:55 CST
description: ICAM2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3384) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3384  | ICAM2 | ENSG00000108622 | 17q23.3 |



The gene is located in the [uropod](https://amigo.geneontology.org/amigo/term/GO:0001931), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [microvillus](https://amigo.geneontology.org/amigo/term/GO:0005902), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), and [cleavage furrow](https://amigo.geneontology.org/amigo/term/GO:0032154). It enables [integrin binding](https://amigo.geneontology.org/amigo/term/GO:0005178) and is involved in [cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0007155) and [cell-cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0098609).


The gene has a length of 12,654 base pairs (30.9% of all genes), a mature length of 526 base pairs, and a primary transcript length of 6,084 base pairs.


The gene ICAM2 (NCBI ID: 3384) has been mentioned in [74 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ICAM2%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1989 and the middle 50% of publications occurring between 1996 and 2006. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning ICAM2, ranked by their scientific influence, include "[Characterization of ICAM-2 and evidence for a third counter-receptor for LFA-1.](https://pubmed.ncbi.nlm.nih.gov/1676048)" (1991) (relative citation ratio: 11.21), "[DC-SIGN-ICAM-2 interaction mediates dendritic cell trafficking.](https://pubmed.ncbi.nlm.nih.gov/11017109)" (2000) (relative citation ratio: 6.87), "[Association of ezrin with intercellular adhesion molecule-1 and -2 (ICAM-1 and ICAM-2). Regulation by phosphatidylinositol 4, 5-bisphosphate.](https://pubmed.ncbi.nlm.nih.gov/9705328)" (1998) (relative citation ratio: 4.96), "[Expression patterns of leukocyte adhesion ligand molecules on human liver endothelia. Lack of ELAM-1 and CD62 inducibility on sinusoidal endothelia and distinct distribution of VCAM-1, ICAM-1, ICAM-2, and LFA-3.](https://pubmed.ncbi.nlm.nih.gov/8434643)" (1993) (relative citation ratio: 4.59), and "[ICAM-2 redistributed by ezrin as a target for killer cells.](https://pubmed.ncbi.nlm.nih.gov/8717043)" (1996) (relative citation ratio: 3.62). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ICAM2](https://www.proteinatlas.org/ENSG00000108622-ICAM2) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in the plasma membrane and cytosol. The gene is expressed in various clusters, including Cluster 5 for non-specific cell proliferation in blood, Cluster 70 for immune response in lymphoid tissue, Cluster 47 for vasculature in the brain, Cluster 48 for angiogenesis in adipocytes and endothelial cells, and Cluster 41 for mixed function in single cells.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025), [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), and [RUNX1T1](https://www.ncbi.nlm.nih.gov/gene/862), each with 5 experiments. Additionally, [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) and [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930) were found to be regulating in 4 experiments each.





The gene is expressed in adipose tissue, lung, and spleen according to the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles indicate expression in Lymphoma (Burkitt's, Raji), heart, CD56+ NK cells, CD4+ T cells, CD8+ T cells, CD14+ monocytes, and lung.


The proteins are involved in several key pathways, including the adaptive immune system, the broader immune system, immunoregulatory interactions between lymphoid and non-lymphoid cells, extracellular matrix organization, and integrin cell surface interactions.



The protein sequence analyzed has a GRAVY value of -0.0156 (83.26th percentile), indicating a slightly hydrophilic nature. The charge at pH 7.0 is 0.234 (51.16th percentile), and the median structural flexibility is 0.994 (18.19th percentile). The protein has an instability index of 50.4 (58.64th percentile) and an isoelectric point of 7.095 (51.41st percentile). The sequence length is 275 amino acids (25.1st percentile) with a molecular weight of 30653.97 Da (25.15th percentile). The secondary structure affinities are 28.4% for helix (19.33rd percentile), 40.0% for sheet (86.16th percentile), and 25.8% for turn (27.67th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |