---
layout: post
title: TRAF1
date: 2025-01-17 16:55 CST
description: TRAF1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7185) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7185  | TRAF1 | ENSG00000056558 | 9q33.2 |



The gene enables [tumor necrosis factor receptor binding](https://amigo.geneontology.org/amigo/term/GO:0005164) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is involved in [apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0006915) and [tumor necrosis factor-mediated signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0033209). It is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737) and the [cytoplasmic side of plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009898), and is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). Additionally, it enables [zinc ion binding](https://amigo.geneontology.org/amigo/term/GO:0008270), [ubiquitin protein ligase binding](https://amigo.geneontology.org/amigo/term/GO:0031625), [thioesterase binding](https://amigo.geneontology.org/amigo/term/GO:0031996), and [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802). The gene is also involved in the [regulation of canonical NF-kappaB signal transduction](https://amigo.geneontology.org/amigo/term/GO:0043122), [positive regulation of NF-kappaB transcription factor activity](https://amigo.geneontology.org/amigo/term/GO:0051092), [protein-containing complex assembly](https://amigo.geneontology.org/amigo/term/GO:0065003), [protein K63-linked ubiquitination](https://amigo.geneontology.org/amigo/term/GO:0070534), and [regulation of extrinsic apoptotic signaling pathway](https://amigo.geneontology.org/amigo/term/GO:2001236). Furthermore, it is part of the [plasma membrane signaling receptor complex](https://amigo.geneontology.org/amigo/term/GO:0098802).


The gene length is 4,145 base pairs (11.63% of all genes), the mature length is 2,065 base pairs, and the primary transcript length is 3,035 base pairs.


The gene TRAF1 (NCBI ID: 7185) has been mentioned in [123 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22TRAF1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1996, and the middle 50% of publications occurred between 2007 and 2016.


The top five publications mentioning TRAF1, ranked by their scientific influence, include "[NF-kappaB antiapoptosis: induction of TRAF1 and TRAF2 and c-IAP1 and c-IAP2 to suppress caspase-8 activation.](https://pubmed.ncbi.nlm.nih.gov/9733516)" (1998) (relative citation ratio: 49.68), "[TRAF1-C5 as a risk locus for rheumatoid arthritis--a genomewide study.](https://pubmed.ncbi.nlm.nih.gov/17804836)" (2007) (relative citation ratio: 13.49), "[Association of TRAF1, TRAF2, and TRAF3 with an Epstein-Barr virus LMP1 domain important for B-lymphocyte transformation: role in NF-kappaB activation.](https://pubmed.ncbi.nlm.nih.gov/8943365)" (1996) (relative citation ratio: 8.28), "[N<sup>6</sup>-methyladenosine-modified TRAF1 promotes sunitinib resistance by regulating apoptosis and angiogenesis in a METTL14-dependent manner in renal cell carcinoma.](https://pubmed.ncbi.nlm.nih.gov/35538475)" (2022) (relative citation ratio: 5.22), and "[A candidate gene approach identifies the TRAF1/C5 region as a risk factor for rheumatoid arthritis.](https://pubmed.ncbi.nlm.nih.gov/17880261)" (2007) (relative citation ratio: 4.54). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[TRAF1](https://www.proteinatlas.org/ENSG00000056558-TRAF1) is a cancer-related gene with evidence at the protein level. It is detected in all RNA tissue distributions and is primarily located in the nucleoplasm. TRAF1 is expressed in various clusters, including Cluster 9 in blood, which is associated with T-regs and cell cycle regulation, Cluster 36 in tissue, which is linked to lymphoid tissue and cytokine signaling, Cluster 11 in the brain, which is involved in white matter and signal transduction, Cluster 55 in cell lines, which is related to lymphoma and immune response, and Cluster 14 in single cells, which is associated with Langerhans cells and innate immune response.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599) with 5 experiments, [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) with 5 experiments, [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) with 5 experiments, [YY1](https://www.ncbi.nlm.nih.gov/gene/7528) with 4 experiments, and [MYC](https://www.ncbi.nlm.nih.gov/gene/4609) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with a wide range of disease conditions, including infectious diseases such as those caused by viral agents and human immunodeficiency virus, as well as immune system disorders. Cardiovascular issues are prominent, encompassing artery disease, vascular disease, and hypertension. Additionally, the data highlights kidney and urinary system diseases, along with various nervous system conditions, including neurodegenerative diseases and multiple sclerosis. Musculoskeletal system diseases, such as arthritis and rheumatoid arthritis, are also noted. Furthermore, the data includes autoimmune diseases like celiac disease and specific heart conditions such as familial atrial fibrillation.


The TRAF1 gene has been associated with variations in [Cystatin C levels](https://pubmed.ncbi.nlm.nih.gov/33462484), which maps closely to the MeSH term [Cystatins](https://meshb.nlm.nih.gov/record/ui?ui=D015891) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method. Additionally, TRAF1 is linked to [Non-albumin protein levels](https://pubmed.ncbi.nlm.nih.gov/33462484), with the closest MeSH match being [Proteins](https://meshb.nlm.nih.gov/record/ui?ui=D011506), also identified through embedding matching.


The gene is expressed in various blood-related tissues and cells, including CD4+ T cells, CD8+ T cells, and specific lymphoblasts such as 721 B lymphoblasts and Burkitt's lymphoma (Raji). Additionally, the gene shows expression in the pineal gland during both day and night cycles, as well as in promyelocytic leukemia (HL-60) cells.




The analyzed protein sequence has a GRAVY value of -0.3322 (54.14th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -7.71 (27.03rd percentile), and the median structural flexibility is 1.0023 (52.08th percentile). The protein has a high affinity for helix structures (35.10%, 75.28th percentile), with moderate affinities for sheet (31.01%, 33.98th percentile) and turn (27.88%, 46.2nd percentile) structures. The instability index is 51.14 (60.86th percentile), and the isoelectric point is 5.77 (24.71st percentile). The protein is 416 amino acids long (47.4th percentile) with a molecular weight of 46163.09 Da (47.17th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |