---
layout: post
title: MID2
date: 2025-01-17 16:55 CST
description: MID2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/11043) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 11043  | MID2 | ENSG00000080561 | Xq22.3  |



The gene enables [transcription coactivator activity](https://amigo.geneontology.org/amigo/term/GO:0003713) and [microtubule binding](https://amigo.geneontology.org/amigo/term/GO:0008017), and is involved in [positive regulation of autophagy](https://amigo.geneontology.org/amigo/term/GO:0010508) and [protein ubiquitination](https://amigo.geneontology.org/amigo/term/GO:0016567). It is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737) and colocalizes with the [microtubule](https://amigo.geneontology.org/amigo/term/GO:0005874). Additionally, it enables [zinc ion binding](https://amigo.geneontology.org/amigo/term/GO:0008270), [transferase activity](https://amigo.geneontology.org/amigo/term/GO:0016740), [enzyme binding](https://amigo.geneontology.org/amigo/term/GO:0019899), and [protein homodimerization activity](https://amigo.geneontology.org/amigo/term/GO:0042803). The gene is also involved in [negative regulation of viral transcription](https://amigo.geneontology.org/amigo/term/GO:0032897), [protein localization to microtubule](https://amigo.geneontology.org/amigo/term/GO:0035372), [positive regulation of canonical NF-kappaB signal transduction](https://amigo.geneontology.org/amigo/term/GO:0043123), [suppression of viral release by host](https://amigo.geneontology.org/amigo/term/GO:0044790), [innate immune response](https://amigo.geneontology.org/amigo/term/GO:0045087), [positive regulation of DNA-templated transcription](https://amigo.geneontology.org/amigo/term/GO:0045893), [negative regulation of viral entry into host cell](https://amigo.geneontology.org/amigo/term/GO:0046597), [positive regulation of DNA-binding transcription factor activity](https://amigo.geneontology.org/amigo/term/GO:0051091), and [positive regulation of NF-kappaB transcription factor activity](https://amigo.geneontology.org/amigo/term/GO:0051092). Furthermore, it enables [phosphoprotein binding](https://amigo.geneontology.org/amigo/term/GO:0051219) and is located in the [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062).


The gene length is 33,992 base pairs (56.12% of all genes), the mature length is 2,496 base pairs (63.85% of all genes), and the primary transcript length is 33,883 base pairs (65.91% of all genes).


MID2 has been mentioned in [15 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22MID2%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1984, and the middle 50% of publications range from 2006 to 2019.


The top five publications mentioning MID2, ranked by their relative citation ratio (RCR), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows. The most influential publication is "[An anillin homologue, Mid2p, acts during fission yeast cytokinesis to organize the septin ring and promote cell separation.](https://pubmed.ncbi.nlm.nih.gov/12668659)" (2003) with a relative citation ratio of 1.94. Another significant study is "[MID1 and MID2 homo- and heterodimerise to tether the rapamycin-sensitive PP2A regulatory subunit, alpha 4, to microtubules: implications for the clinical variability of X-linked Opitz GBBB syndrome and other developmental disorders.](https://pubmed.ncbi.nlm.nih.gov/11806752)" (2002), which has a relative citation ratio of 1.88. Additionally, "[The E3 ligase TRIM1 ubiquitinates LRRK2 and controls its localization, degradation, and toxicity.](https://pubmed.ncbi.nlm.nih.gov/35266954)" (2022) has a relative citation ratio of 1.75. The publication "[MID1 and MID2 are required for Xenopus neural tube closure through the regulation of microtubule organization.](https://pubmed.ncbi.nlm.nih.gov/20534674)" (2010) has a relative citation ratio of 1.24. Lastly, "[MID2, a homologue of the Opitz syndrome gene MID1: similarities in subcellular localization and differences in expression during development.](https://pubmed.ncbi.nlm.nih.gov/10400986)" (1999) has a relative citation ratio of 0.89. The citation counts are sourced from [iCite](https://icite.od.nih.gov).


[MID2](https://www.proteinatlas.org/ENSG00000080561-MID2) is a transferase with evidence at the protein level. It is implicated in intellectual disability as a disease variant. The gene is detected in many tissues and is expressed in T-cells of unknown function. In the brain, it is associated with neuropeptide signaling in the hypothalamus. Additionally, MID2 is involved in mRNA processing in non-specific cell lines and lymph vessel development in lymphatic endothelial cells.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) with 6 experiments, followed by [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025), [MYC](https://www.ncbi.nlm.nih.gov/gene/4609), and [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), each with 5 experiments.



The GWAS data indicates associations with several disease conditions, including cancer, diseases of cellular proliferation, and general disease.



The gene is expressed in various tissues, including cardiac myocytes, superior cervical ganglion, trigeminal ganglion, skeletal muscle, dorsal root ganglion, uterus corpus, and vagina.



The mouse ortholog gene 23947 and its human ortholog 11043 are associated with several phenotypic abnormalities. Both sexes exhibit abnormal skin morphology, including scaly skin, as well as abnormal spleen morphology, with a specific presentation of an enlarged spleen.


The analyzed protein sequence has a GRAVY value of -0.345 (52.79th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of 1.294 (54.91st percentile). The median structural flexibility is 0.999 (36.14th percentile), with a propensity for helix, sheet, and turn structures of 30.34% (34.94th percentile), 33.47% (48.35th percentile), and 28.16% (48.8th percentile), respectively. The instability index is 47.99 (50.18th percentile), and the isoelectric point is 7.24 (52.64th percentile). The protein is 735 amino acids long (77.36th percentile) with a molecular weight of 83209.41 Da (77.96th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |