---
layout: post
title: RAB33A
date: 2025-01-17 16:55 CST
description: RAB33A description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9363) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9363  | RAB33A | ENSG00000134594 | Xq26.1  |



The gene is located in the [Golgi membrane](https://amigo.geneontology.org/amigo/term/GO:0000139) and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). It is active in the [endosome](https://amigo.geneontology.org/amigo/term/GO:0005768) and [Golgi apparatus](https://amigo.geneontology.org/amigo/term/GO:0005794). The gene enables [GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0003924), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [GTP binding](https://amigo.geneontology.org/amigo/term/GO:0005525). Additionally, it is involved in [antigen processing and presentation](https://amigo.geneontology.org/amigo/term/GO:0019882) and [Rab protein signal transduction](https://amigo.geneontology.org/amigo/term/GO:0032482).


The gene length is 30,775 base pairs (53.41% of all genes), the mature length is 1,296 base pairs (31.83% of all genes), and the primary transcript length is 26,130 base pairs (58.95% of all genes).


RAB33A has been mentioned in [2 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RAB33A%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 2006, and the middle 50% of publications occurred between 2009 and 2015.


The gene RAB33A has been studied in several publications, with notable works including "[Rab33A: characterization, expression, and suppression by epigenetic modification.](https://pubmed.ncbi.nlm.nih.gov/16810302)" (2006) (relative citation ratio: 0.4) and "[Rab33a and Rab33ba mediate the outgrowth of forebrain commissural axons in the zebrafish brain.](https://pubmed.ncbi.nlm.nih.gov/30755680)" (2019) (relative citation ratio: 0.37). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RAB33A](https://www.proteinatlas.org/ENSG00000134594-RAB33A) is a gene with evidence at the protein level, detected in many RNA tissue distributions. It is primarily localized in vesicles and is associated with various expression clusters, including Cluster 3 for non-specific nuclear processes in blood, Cluster 9 for oligodendrocytes and myelination in tissues, Cluster 28 for mixed function in the brain, Cluster 71 for lymphoma and inflammatory response in cell lines, and Cluster 28 for dendritic cells and antigen presentation in single cells. The gene is unprognostic for several cancer types, including Bladder Urothelial Carcinoma, Breast Invasive Carcinoma, Cervical Squamous Cell Carcinoma, Glioblastoma Multiforme, Head and Neck Squamous Cell Carcinoma, Kidney Renal Clear Cell Carcinoma, Lung Adenocarcinoma, Lung Squamous Cell Carcinoma, Ovary Serous Cystadenocarcinoma, Pancreatic Adenocarcinoma, Skin Cutaneous Melanoma, Stomach Adenocarcinoma, Testicular Germ Cell Tumor, and Uterine Corpus Endometrial Carcinoma.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 5 experiments, [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) with 5 experiments, [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 5 experiments, [KDM2B](https://www.ncbi.nlm.nih.gov/gene/84678) with 4 experiments, and [SP1](https://www.ncbi.nlm.nih.gov/gene/6667) with 3 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.





The gene is expressed in various tissues, including the brain and blood. Additionally, it is expressed in specific cell types and tissues such as Lymphoma Burkitt's (Raji), 721 B lymphoblasts, Amygdala, Whole brain, Hypothalamus, Cerebellum, and Leukemia lymphoblastic (MOLT-4).




The protein sequence analyzed has a GRAVY value of -0.2274 (66.42 percentile), indicating a hydrophilic nature. It exhibits a charge of 1.89 at pH 7.0 (57.13 percentile) and a median structural flexibility of 1.003 (53.38 percentile). The secondary structure affinities are 32.91% for helix (58.59 percentile), 35.02% for sheet (59.34 percentile), and 24.47% for turn (18.02 percentile). The instability index is 38.65 (18.53 percentile), with an isoelectric point of 8.07 (59.52 percentile). The protein has a length of 237 amino acids (19.8 percentile) and a molecular weight of 26592.39 Da (20.14 percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |