---
layout: post
title: ARHGAP1
date: 2025-01-17 16:55 CST
description: ARHGAP1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/392) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 392  | ARHGAP1 | ENSG00000175220 | 11p11.2 |



The gene enables [GTPase activator activity](https://amigo.geneontology.org/amigo/term/GO:0005096) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737). It is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0010008), [perinuclear region of cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0048471), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062), and colocalizes with the [sorting endosome](https://amigo.geneontology.org/amigo/term/GO:0097443). The gene is involved in [small GTPase-mediated signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007264), [Rho protein signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007266), [endosomal transport](https://amigo.geneontology.org/amigo/term/GO:0016197), [transferrin transport](https://amigo.geneontology.org/amigo/term/GO:0033572), and [negative regulation of endocytic recycling](https://amigo.geneontology.org/amigo/term/GO:2001136). Additionally, it enables [SH3 domain binding](https://amigo.geneontology.org/amigo/term/GO:0017124), [small GTPase binding](https://amigo.geneontology.org/amigo/term/GO:0031267), and [cadherin binding](https://amigo.geneontology.org/amigo/term/GO:0045296), and is involved in the [regulation of small GTPase mediated signal transduction](https://amigo.geneontology.org/amigo/term/GO:0051056).


The gene length is 755,290 base pairs (99.27th percentile), the mature length is 2,927 base pairs (71.42nd percentile), and the primary transcript length is 157,100 base pairs (94.08th percentile).


ARHGAP1 (Gene ID: 392) has been mentioned in [115 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ARHGAP1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1991 and the middle 50% of publications occurring between 2003 and 2016. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning ARHGAP1, ranked by their scientific influence, include "[Cancer-secreted hsa-miR-940 induces an osteoblastic phenotype in the bone metastatic microenvironment via targeting ARHGAP1 and FAM134A.](https://pubmed.ncbi.nlm.nih.gov/29440427)" (2018) (relative citation ratio: 8.48), "[Current knowledge of the large RhoGAP family of proteins.](https://pubmed.ncbi.nlm.nih.gov/17222083)" (2007) (relative citation ratio: 6.54), "[Central spindle assembly and cytokinesis require a kinesin-like protein/RhoGAP complex with microtubule bundling activity.](https://pubmed.ncbi.nlm.nih.gov/11782313)" (2002) (relative citation ratio: 6.25), "[Oligophrenin-1 encodes a rhoGAP protein involved in X-linked mental retardation.](https://pubmed.ncbi.nlm.nih.gov/9582072)" (1998) (relative citation ratio: 5.88), and "[The RhoGAP activity of the Yersinia pseudotuberculosis cytotoxin YopE is required for antiphagocytic function and virulence.](https://pubmed.ncbi.nlm.nih.gov/10931345)" (2000) (relative citation ratio: 4.2). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ARHGAP1](https://www.proteinatlas.org/ENSG00000175220-ARHGAP1) is a gene with a molecular function in GTPase activation. Evidence for its function is supported at the protein level. The gene is detected in all RNA tissue distributions and is localized in vesicles, primary cilium, and cytosol. ARHGAP1 is expressed in various clusters, including Cluster 37 in monocytes and neutrophils related to innate immune response, Cluster 41 in smooth muscle tissue involved in ECM organization, Cluster 11 in white matter for signal transduction, Cluster 24 in non-specific cell lines with unknown function, and Cluster 48 in smooth muscle cells for signal transduction.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [TCF12](https://www.ncbi.nlm.nih.gov/gene/6938) and [TFAP2C](https://www.ncbi.nlm.nih.gov/gene/7022), each with 5 experiments. Additionally, [SIN3A](https://www.ncbi.nlm.nih.gov/gene/25942), [SMAD2](https://www.ncbi.nlm.nih.gov/gene/4087), and [CDK8](https://www.ncbi.nlm.nih.gov/gene/1024) were each found to be regulating in 4 experiments.




ARHGAP1 has been associated with [serum levels of protein ARHGAP1](https://pubmed.ncbi.nlm.nih.gov/35078996), which maps closely to the MeSH term [Proteins](https://meshb.nlm.nih.gov/record/ui?ui=D011506) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method.


The gene is expressed in various tissues, including blood vessel and bladder, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles show expression in the heart, liver, adipocyte, prostate, and uterus.


The proteins are involved in the Rho GTPase cycle, signaling by Rho GTPases, and signal transduction pathways.



The analyzed protein sequence has a GRAVY value of -0.367 (50.27th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -9.15 (23.92nd percentile), and the median structural flexibility is 1.0004 (43.57th percentile). The protein has a propensity for helix, sheet, and turn structures of 32.8% (57.7th percentile), 39.64% (85.16th percentile), and 28.47% (51.53rd percentile), respectively. The instability index is 42.55 (30.55th percentile), with an isoelectric point of 5.85 (26.53rd percentile). The protein length is 439 amino acids (50.56th percentile), and the molecular weight is 50435.1 Da (52.19th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |