---
layout: post
title: NNMT
date: 2025-01-17 16:55 CST
description: NNMT description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4837) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4837  | NNMT | ENSG00000166741 | 11q23.2 |



The gene is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829) and is active in this cellular component. It is involved in various biological processes, including the [nicotinamide metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006769), [response to xenobiotic stimulus](https://amigo.geneontology.org/amigo/term/GO:0009410), [response to organonitrogen compound](https://amigo.geneontology.org/amigo/term/GO:0010243), [animal organ regeneration](https://amigo.geneontology.org/amigo/term/GO:0031100), [methylation](https://amigo.geneontology.org/amigo/term/GO:0032259), [NAD biosynthesis via nicotinamide riboside salvage pathway](https://amigo.geneontology.org/amigo/term/GO:0034356), [positive regulation of gluconeogenesis](https://amigo.geneontology.org/amigo/term/GO:0045722), and [positive regulation of protein deacetylation](https://amigo.geneontology.org/amigo/term/GO:0090312). Additionally, the gene enables [nicotinamide N-methyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0008112) and [pyridine N-methyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0030760).


The gene has a length of 34,524 base pairs (56.55% of all genes), a mature length of 784 base pairs (13.13% of all genes), and a primary transcript length of 7,854 base pairs (27.45% of all genes).


The gene NNMT (NCBI ID: 4837) has been mentioned in [142 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22NNMT%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1990 and the middle 50% of publications occurring between 2013 and 2021. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning NNMT, ranked by their scientific influence, include "[Proteomics reveals NNMT as a master metabolic regulator of cancer-associated fibroblasts.](https://pubmed.ncbi.nlm.nih.gov/31043742)" (2019) with a relative citation ratio of 12.04, "[NNMT promotes epigenetic remodeling in cancer by creating a metabolic methylation sink.](https://pubmed.ncbi.nlm.nih.gov/23455543)" (2013) with a relative citation ratio of 7.64, "[Nicotinamide N-Methyltransferase: More Than a Vitamin B3 Clearance Enzyme.](https://pubmed.ncbi.nlm.nih.gov/28291578)" (2017) with a relative citation ratio of 6.18, "[Nicotinamide N-methyltransferase in endothelium protects against oxidant stress-induced endothelial injury.](https://pubmed.ncbi.nlm.nih.gov/34153425)" (2021) with a relative citation ratio of 5.55, and "[Nicotinamide N-methyltransferase: At the crossroads between cellular metabolism and epigenetic regulation.](https://pubmed.ncbi.nlm.nih.gov/33453420)" (2021) with a relative citation ratio of 4.82. The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[NNMT](https://www.proteinatlas.org/ENSG00000166741-NNMT) is a gene encoding a protein with methyltransferase and transferase molecular functions. It is an FDA-approved drug target with evidence at the protein level. The gene is detected in many tissues and is primarily localized in the Golgi apparatus and cytosol. NNMT is highly expressed in the liver, where it plays a role in metabolism, and is also found in the vasculature of the brain. In single-cell expression analysis, it is notably expressed in hepatocytes involved in metabolism. Additionally, NNMT is expressed in kidney cancer cell lines, particularly in membrane components.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [CDK8](https://www.ncbi.nlm.nih.gov/gene/1024), [MED12](https://www.ncbi.nlm.nih.gov/gene/9968), [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313), [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), and [RELA](https://www.ncbi.nlm.nih.gov/gene/5970), each shown to be regulating in 4 experiments.





The gene is expressed in multiple tissue samples, including liver, cardiac myocytes, adipocytes, smooth muscle, trigeminal ganglion, skeletal muscle, atrioventricular node, and ciliary ganglion.


The input data highlights various pathways and disorders related to biological oxidations, phase II conjugation, and methylation. Notably, defective AHCY is linked to Hypermethioninemia with S-adenosylhomocysteine hydrolase deficiency (HMAHCHD), while defective UGT1A1 and UGT1A4 cause hyperbilirubinemia. Additionally, defects in GCLC, GGT1, and GSS are associated with Hemolytic anemia due to gamma-glutamylcysteine synthetase deficiency (HAGGSD), Glutathionuria (GLUTH), and Glutathione synthetase deficiency (GSS deficiency), respectively. Other disorders include Methionine adenosyltransferase deficiency (MATD) due to defective MAT1A, 5-oxoprolinase deficiency (OPLAHD) due to defective OPLAH, Schneckenbecken dysplasia (SCHBCKD) due to defective SLC35D1, and Thiopurine S-methyltransferase deficiency (TPMT deficiency) due to defective TPMT. These conditions fall under metabolic disorders of biological oxidation enzymes and are part of broader metabolic processes.


The mouse ortholog gene 18113 and its human ortholog gene 4837 are associated with several phenotypic abnormalities. These include abnormal ovary morphology and enlarged ovary, specific to one sex, as well as abnormal pancreas morphology, abnormal skin morphology, and enlarged spleen, which are observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.179 (71.35th percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is -4.28 (35.59th percentile), and the median structural flexibility is 1.001 (45.07th percentile). The protein has a notable affinity for helix (36.36%, 82.41st percentile) and sheet (37.12%, 73.82nd percentile) structures, with a lower affinity for turns (28.41%, 51.09th percentile). The instability index is 42.72 (31.18th percentile), and the isoelectric point is 5.56 (19.72nd percentile). The protein has a length of 264 amino acids (23.65th percentile) and a molecular weight of 29573.68 Da (23.87th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |