---
layout: post
title: GNAS
date: 2024-12-08 23:57 CST
description: GNAS description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2778) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2778  | GNAS | ENSG00000087460 | 20q13.32 |



The gene length is 102,332.0 base pairs (82.2% of all genes), the mature length is 2,067.5 base pairs, and the primary transcript length is 82,209.5 base pairs.


The gene GNAS (NCBI ID: 2778) has been the subject of [767 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22GNAS%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1966, and the middle 50% of publications occurred between 2000 and 2015.


The top five publications mentioning GNAS, ranked by their scientific influence, include "[Recurrent GNAS mutations define an unexpected pathway for pancreatic cyst development.](https://pubmed.ncbi.nlm.nih.gov/21775669)" (2011) (relative citation ratio: 15.37), "[Crystal structure of the catalytic domains of adenylyl cyclase in a complex with Gsalpha.GTPgammaS.](https://pubmed.ncbi.nlm.nih.gov/9417641)" (1997) (relative citation ratio: 12.47), "[Development and characterization of novel erythropoiesis stimulating protein (NESP).](https://pubmed.ncbi.nlm.nih.gov/11308268)" (2001) (relative citation ratio: 9.11), "[Isolation and characterization of the human Gs alpha gene.](https://pubmed.ncbi.nlm.nih.gov/3127824)" (1988) (relative citation ratio: 8.83), and "[Whole-exome sequencing uncovers frequent GNAS mutations in intraductal papillary mucinous neoplasms of the pancreas.](https://pubmed.ncbi.nlm.nih.gov/22355676)" (2011) (relative citation ratio: 8.22). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908), [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861), [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599), [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), and [MYC](https://www.ncbi.nlm.nih.gov/gene/4609), each shown to be regulating in 6 experiments.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [CTCF](https://www.ncbi.nlm.nih.gov/gene/6829) with 3 experiments, [ZNF274](https://www.ncbi.nlm.nih.gov/gene/8243) with 2 experiments, and [ZNF668](https://www.ncbi.nlm.nih.gov/gene/9968) with 2 experiments. Additionally, [COL1A1](https://www.ncbi.nlm.nih.gov/gene/2118) was found to be regulating in 1 experiment. These findings highlight the regulatory roles of these TFs in gene expression changes as observed in Perturb-seq experiments.


The GNAS gene has been associated with several traits and diseases through genome-wide association studies. Notably, it has been linked to [Creatinine levels](https://pubmed.ncbi.nlm.nih.gov/29124443), with the closest MeSH match being [Creatinine](https://meshb.nlm.nih.gov/record/ui?ui=D003404), and [Estimated glomerular filtration rate](https://pubmed.ncbi.nlm.nih.gov/29124443), mapped to [Filtration](https://meshb.nlm.nih.gov/record/ui?ui=D005374) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method. Additionally, GNAS is associated with [Hyperuricemia](https://pubmed.ncbi.nlm.nih.gov/29124443), directly matching the canonical name, and [Serum uric acid levels](https://pubmed.ncbi.nlm.nih.gov/29124443), mapped to [Serum](https://meshb.nlm.nih.gov/record/ui?ui=D044967). Furthermore, it is linked to [Thyroid stimulating hormone levels](https://pubmed.ncbi.nlm.nih.gov/37872160), with the closest MeSH match being [Hormones](https://meshb.nlm.nih.gov/record/ui?ui=D006728).





The analyzed protein sequence has a GRAVY value of -0.671 (19.63rd percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -9.311 (23.59th percentile), and the median structural flexibility is 1.013 (88.18th percentile). The protein has a high instability index of 75.096 (96.92nd percentile) and an isoelectric point of 5.478 (17.7th percentile). The sequence length is 510 amino acids (59.52nd percentile) with a molecular weight of 56805.6 Da (59.32nd percentile). The secondary structure affinities are 31.51% for helix (45.82nd percentile), 23.68% for sheet (5.89th percentile), and 33.42% for turn (83.83rd percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |