---
layout: post
title: ATP6V1G2
date: 2025-01-17 16:55 CST
description: ATP6V1G2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/534) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 534  | ATP6V1G2 | ENSG00000213760 | 6p21.33 |



The gene is part of the [vacuolar proton-transporting V-type ATPase, V1 domain](https://amigo.geneontology.org/amigo/term/GO:0000221) and enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [clathrin-coated vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030665), [synaptic vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030672), and [melanosome](https://amigo.geneontology.org/amigo/term/GO:0042470). The gene is involved in the [regulation of macroautophagy](https://amigo.geneontology.org/amigo/term/GO:0016241) and [synaptic vesicle lumen acidification](https://amigo.geneontology.org/amigo/term/GO:0097401). Additionally, it enables [ATP hydrolysis activity](https://amigo.geneontology.org/amigo/term/GO:0016887) and [proton-transporting ATPase activity, rotational mechanism](https://amigo.geneontology.org/amigo/term/GO:0046961).


The gene length is 418,048 base pairs (97.6% of all genes), the mature length is 7,793 base pairs (97.48% of all genes), and the primary transcript length is 299,187 base pairs (97.89% of all genes).


The gene ATP6V1G2 (NCBI ID: 534) has been mentioned in [1 publication](https://pubmed.ncbi.nlm.nih.gov/?term=%22ATP6V1G2%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 2005, and the middle 50% of publications range from 2005 to 2005.


The top publication mentioning ATP6V1G2 is "[The haplotype block, NFKBIL1-ATP6V1G2-BAT1-MICB-MICA, within the class III-class I boundary region of the human major histocompatibility complex may control susceptibility to hepatitis C virus-associated dilated cardiomyopathy.](https://pubmed.ncbi.nlm.nih.gov/16101831)" (2005) (relative citation ratio: 0.46). The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, and benchmarking to the median for NIH publications, the value of which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ATP6V1G2](https://www.proteinatlas.org/ENSG00000213760-ATP6V1G2) is a gene with evidence at the protein level. Its RNA is detected in many tissues and is particularly expressed in the cerebellum, contributing to nervous system development. In the brain, it is predominantly found in neurons and is associated with nucleosome function. The gene is also expressed in neuroblastoma cell lines, where it plays a role in neuronal signaling. Additionally, it is found in bipolar cells, contributing to visual perception. The gene is unprognostic in several cancer types, including Breast Invasive Carcinoma, Glioblastoma Multiforme, Ovary Serous Cystadenocarcinoma, Prostate Adenocarcinoma, and Uterine Corpus Endometrial Carcinoma.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [PPARG](https://www.ncbi.nlm.nih.gov/gene/5468) and [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169), each with 5 experiments. Additionally, [CHD8](https://www.ncbi.nlm.nih.gov/gene/57680) was found in 4 experiments, while [NIPBL](https://www.ncbi.nlm.nih.gov/gene/25836) and [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113) were each identified in 3 experiments.



The GWAS data indicates associations with a range of disease conditions, including immune system disease, vascular disease, cardiovascular system disease, cognitive disorder, disease of mental health, bone inflammation disease, arthritis, rheumatoid arthritis, bone disease, connective tissue disease, musculoskeletal system disease, hypersensitivity reaction disease, Beh√ßet's disease, and vasculitis.



The gene is ubiquitously expressed in multiple tissue samples, including the whole brain, cingulate cortex, subthalamic nucleus, temporal lobe, occipital lobe, cerebellum, and cerebellum peduncles.


The input data highlights several biological pathways, including insulin receptor recycling, transferrin endocytosis and recycling, iron uptake and transport, latent infection of Homo sapiens with Mycobacterium tuberculosis, phagosomal maturation (early endosomal stage), transmembrane transport of small molecules, signaling by insulin receptor, disease, and signal transduction.



The protein sequence analyzed has a GRAVY value of -1.141 (2.31% percentile), indicating a hydrophilic nature. It exhibits a charge of 7.692 at pH 7 (76.08% percentile) and a median structural flexibility of 1.017 (93.9% percentile). The protein's secondary structure is predicted to be predominantly helical, with 38.98% of amino acids in helix form (91.45% percentile), 17.8% in sheet form (1.2% percentile), and 16.95% in turn form (0.56% percentile). The instability index is 65.75 (91.05% percentile), suggesting a relatively stable protein. The isoelectric point is 10.26 (97.61% percentile), with a molecular weight of 13604.21 Da (4.89% percentile). The sequence length is 118 amino acids (4.48% percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |