---
layout: post
title: CHKA
date: 2024-12-08 23:57 CST
description: CHKA description
tags: [cooccuring-genes]
categories: longest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1119) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1119  | CHKA | ENSG00000110721 | 11q13.2 |



The gene length is 429,239 base pairs (97.74% of all genes), the mature length is 4,711.5 base pairs, and the primary transcript length is 428,786.5 base pairs.


The gene CHKA, with NCBI ID 1119, has been the subject of [118 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CHKA%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1977, and the middle 50% of publications occurred between 2004 and 2016.


The top five publications mentioning CHKA, ranked by their scientific influence, include "[Axin-mediated CKI phosphorylation of beta-catenin at Ser 45: a molecular switch for the Wnt pathway.](https://pubmed.ncbi.nlm.nih.gov/12000790)" (2002) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 9.13, "[Advances in covalent kinase inhibitors.](https://pubmed.ncbi.nlm.nih.gov/32227030)" (2020) with an RCR of 7.74, "[EK (DLEK, DSEK, DMEK): New Frontier in Cornea Surgery.](https://pubmed.ncbi.nlm.nih.gov/28697678)" (2017) with an RCR of 6.69, "[Developmental pattern of small intestinal enterokinase and disaccharidase activities in the human fetus.](https://pubmed.ncbi.nlm.nih.gov/558125)" (1977) with an RCR of 6.41, and "[c-Ki-ras point mutations in ductectatic-type mucinous cystic neoplasms of the pancreas.](https://pubmed.ncbi.nlm.nih.gov/1955373)" (1991) with an RCR of 3.24. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613), [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908), [HDAC1](https://www.ncbi.nlm.nih.gov/gene/3065), and [NFYA](https://www.ncbi.nlm.nih.gov/gene/4800), each with 6 experiments, followed by [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861) with 5 experiments.








The mouse ortholog gene 12660 and its human ortholog gene 1119 are associated with the phenotype of preweaning lethality with complete penetrance, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.442 (41.43rd percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -3.79 (37.13rd percentile). The median structural flexibility is 1.004 (61.69th percentile), with a predicted secondary structure composition of 37.2% helix (86th percentile), 32.8% sheet (44.22nd percentile), and 28.4% turn (51.32nd percentile). The instability index is 63.82 (88.84th percentile), suggesting high instability. The isoelectric point is 6.16 (34.61st percentile), with a molecular weight of 52248.43 Da (54.19th percentile). The sequence length is 457 amino acids (52.79th percentile). These calculations were performed using BioPython's sequence analysis tools, which can be referenced [here](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |