---
layout: post
title: HMGN2
date: 2025-01-17 16:55 CST
description: HMGN2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3151) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3151  | HMGN2 | ENSG00000198830 | 1p36.11 |



The gene is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), and is also found in the [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615). It is part of [chromatin](https://amigo.geneontology.org/amigo/term/GO:0000785) and is active in the nucleus. The gene enables [chromatin binding](https://amigo.geneontology.org/amigo/term/GO:0003682), [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [nucleosomal DNA binding](https://amigo.geneontology.org/amigo/term/GO:0031492). It is involved in [chromatin organization](https://amigo.geneontology.org/amigo/term/GO:0006325), [killing of cells of another organism](https://amigo.geneontology.org/amigo/term/GO:0031640), and the [antimicrobial humoral immune response mediated by antimicrobial peptide](https://amigo.geneontology.org/amigo/term/GO:0061844).


The gene length is 360,795 base pairs (96.82% of all genes), the mature length is 8,213 base pairs (97.89% of all genes), and the primary transcript length is 100,699 base pairs (88.29% of all genes).


The gene HMGN2 (NCBI ID: 3151) has been mentioned in [72 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22HMGN2%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1980, and the middle 50% of publications range from 1988 to 2012.


The top five publications mentioning HMGN2, ranked by their scientific influence, include "[Nucleosome cores have two specific binding sites for nonhistone chromosomal proteins HMG 14 and HMG 17.](https://pubmed.ncbi.nlm.nih.gov/7433974)" (1980) (relative citation ratio: 3.34), "[Autoantibodies to nucleosomal proteins: antibodies to HMG-17 in autoimmune diseases.](https://pubmed.ncbi.nlm.nih.gov/6460317)" (1982) (relative citation ratio: 2.22), "[Chromosomal protein HMG-17. Complete human cDNA sequence and evidence for a multigene family.](https://pubmed.ncbi.nlm.nih.gov/3754870)" (1986) (relative citation ratio: 1.59), "[Specific acetylation of chromosomal protein HMG-17 by PCAF alters its interaction with nucleosomes.](https://pubmed.ncbi.nlm.nih.gov/10207070)" (1999) (relative citation ratio: 1.2), and "[HDAC6 Deacetylates HMGN2 to Regulate Stat5a Activity and Breast Cancer Growth.](https://pubmed.ncbi.nlm.nih.gov/27358110)" (2016) (relative citation ratio: 1.11). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[HMGN2](https://www.proteinatlas.org/ENSG00000198830-HMGN2) is a DNA-binding protein with evidence at the protein level. It is detected in all RNA tissue distributions and is primarily localized in the nucleoplasm and nucleoli. The gene is expressed in various tissues, with notable expression in bone marrow related to cell proliferation, and is involved in transcription processes in the brain. In single-cell expression clusters, it is associated with cell proliferation. However, HMGN2 is unprognostic across multiple cancer types, including bladder urothelial carcinoma, breast invasive carcinoma, and others, indicating no significant prognostic value in these contexts.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [CEBPA](https://www.ncbi.nlm.nih.gov/gene/1050), [MYC](https://www.ncbi.nlm.nih.gov/gene/4609), [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908), and [NEUROD1](https://www.ncbi.nlm.nih.gov/gene/4760), each with 6 experiments. Additionally, [AR](https://www.ncbi.nlm.nih.gov/gene/367) was found to be regulating in 5 experiments.





The gene is expressed in various tissues, including the ovary and pituitary, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Leukemia promyelocytic (HL-60), CD105+ Endothelial cells, CD71+ Early Erythroid cells, bone marrow, thymus, and Leukemia lymphoblastic (MOLT-4).




The protein sequence analyzed has a GRAVY value of -1.72 (0.19 percentile), indicating a hydrophilic nature. It exhibits a charge of 11.51 at pH 7 (84.27 percentile) and a median structural flexibility of 1.05 (99.97 percentile). The protein has a high affinity for helix structures (52.22%, 99.81 percentile) and turns (36.67%, 93.05 percentile), with a lower affinity for sheet structures (4.44%, 0.09 percentile). The instability index is 35.05 (10.21 percentile), and the isoelectric point is 9.997 (96.41 percentile). The sequence length is 90 amino acids (1.85 percentile), with a molecular weight of 9392.55 Da (1.48 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |