---
layout: post
title: TSN
date: 2024-12-08 23:57 CST
description: TSN description
tags: [cooccuring-genes]
categories: longest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7247) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7247  | TSN | ENSG00000211460 | 2q14.3 |



The gene length is 1,111,986.0 base pairs (99.69% of all genes), the mature length is 2,163.0 base pairs, and the primary transcript length is 1,107,645.0 base pairs (99.93% of all genes).


The gene TSN, with NCBI ID 7247, has been featured in [69 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22TSN%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1993, and the middle 50% of publications occurred between 2000 and 2016.


The top five publications mentioning the TSN gene, ranked by their scientific influence, include "[Dendritic trafficking of BDNF mRNA is mediated by translin and blocked by the G196A (Val66Met) mutation.](https://pubmed.ncbi.nlm.nih.gov/19805324)" (2009) with a relative citation ratio of 3.98, "[A novel gene, Translin, encodes a recombination hotspot binding protein associated with chromosomal translocations.](https://pubmed.ncbi.nlm.nih.gov/7663511)" (1995) with a relative citation ratio of 3.56, "[Structure of C3PO and mechanism of human RISC activation.](https://pubmed.ncbi.nlm.nih.gov/21552258)" (2011) with a relative citation ratio of 2.35, "[translin Is Required for Metabolic Regulation of Sleep.](https://pubmed.ncbi.nlm.nih.gov/27020744)" (2016) with a relative citation ratio of 1.94, and "[Testis/brain RNA-binding protein attaches translationally repressed and transported mRNAs to microtubules.](https://pubmed.ncbi.nlm.nih.gov/7568171)" (1995) with a relative citation ratio of 1.8. The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [KDM5B](https://www.ncbi.nlm.nih.gov/gene/10765) with 6 experiments, [KMT2B](https://www.ncbi.nlm.nih.gov/gene/9757) with 4 experiments, [EP300](https://www.ncbi.nlm.nih.gov/gene/2033) with 4 experiments, [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 4 experiments, and [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885) with 4 experiments.











The analyzed protein sequence has a GRAVY value of -0.19035087719298238 (70.19th percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is -3.8543969995304757 (36.89th percentile), and the median structural flexibility is 0.9964166666666666 (25.8th percentile). The protein has a helix affinity of 0.35526315789473684 (78.06th percentile), a sheet affinity of 0.4210526315789474 (90.91st percentile), and a turn affinity of 0.20175438596491224 (2.69th percentile). The instability index is 47.75438596491229 (49.25th percentile), with an isoelectric point of 6.013588142395019 (30.89th percentile). The protein has a length of 228 amino acids (18.71st percentile) and a molecular weight of 26182.62460000003 Da (19.64th percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |