---
layout: post
title: PTBP1
date: 2025-01-17 16:55 CST
description: PTBP1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5725) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5725  | PTBP1 | ENSG00000011304 | 19p13.3 |



The gene is involved in the [regulation of alternative mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0000381) and [mRNA processing](https://amigo.geneontology.org/amigo/term/GO:0006397). It enables [DNA binding](https://amigo.geneontology.org/amigo/term/GO:0003677), [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [mRNA binding](https://amigo.geneontology.org/amigo/term/GO:0003729), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [poly-pyrimidine tract binding](https://amigo.geneontology.org/amigo/term/GO:0008187). The gene is active in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [nucleolus](https://amigo.geneontology.org/amigo/term/GO:0005730), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). Additionally, it is involved in [RNA splicing](https://amigo.geneontology.org/amigo/term/GO:0008380), [neurogenesis](https://amigo.geneontology.org/amigo/term/GO:0022008), [negative regulation of RNA splicing](https://amigo.geneontology.org/amigo/term/GO:0033119), [positive regulation of protein dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0035307), [regulation of RNA splicing](https://amigo.geneontology.org/amigo/term/GO:0043484), [regulation of cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0045595), [negative regulation of neuron differentiation](https://amigo.geneontology.org/amigo/term/GO:0045665), [positive regulation of transcription by RNA polymerase II](https://amigo.geneontology.org/amigo/term/GO:0045944), [negative regulation of mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0048025), [negative regulation of muscle cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0051148), [positive regulation of calcineurin-NFAT signaling cascade](https://amigo.geneontology.org/amigo/term/GO:0070886), and [IRES-dependent viral translational initiation](https://amigo.geneontology.org/amigo/term/GO:0075522).


The gene length is 108,692 base pairs (83.45% of all genes), the mature length is 4,044 base pairs (84.13% of all genes), and the primary transcript length is 108,395 base pairs (89.6% of all genes).


The gene PTBP1 (NCBI ID: 5725) has been mentioned in [346 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PTBP1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning PTBP1 is 1956, and the middle 50% of publications occurred between 2006 and 2019.


The top 5 publications mentioning PTBP1, ranked by their scientific influence, include "[Robust transcriptome-wide discovery of RNA-binding protein binding sites with enhanced CLIP (eCLIP).](https://pubmed.ncbi.nlm.nih.gov/27018577)" (2016) (relative citation ratio: 23.63), "[Direct conversion of fibroblasts to neurons by reprogramming PTB-regulated microRNA circuits.](https://pubmed.ncbi.nlm.nih.gov/23313552)" (2013) (relative citation ratio: 10.2), "[Circular RNA circRHOBTB3 represses metastasis by regulating the HuR-mediated mRNA stability of PTBP1 in colorectal cancer.](https://pubmed.ncbi.nlm.nih.gov/34158864)" (2021) (relative citation ratio: 7.45), "[Identification of MicroRNA-124 as a Major Regulator of Enhanced Endothelial Cell Glycolysis in Pulmonary Arterial Hypertension via PTBP1 (Polypyrimidine Tract Binding Protein) and Pyruvate Kinase M2.](https://pubmed.ncbi.nlm.nih.gov/28971999)" (2017) (relative citation ratio: 6.85), and "[Genome-wide analysis of PTB-RNA interactions reveals a strategy used by the general splicing repressor to modulate exon inclusion or skipping.](https://pubmed.ncbi.nlm.nih.gov/20064465)" (2009) (relative citation ratio: 6.27). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PTBP1](https://www.proteinatlas.org/ENSG00000011304-PTBP1) is an RNA-binding protein that functions as both an activator and repressor, with evidence supporting its role at the protein level. It is detected in all tissues and primarily localizes to the nucleoplasm. PTBP1 is associated with specific expression clusters in various contexts, including Cluster 12 for blood expression related to mitochondrial translation, Cluster 47 for tissue expression related to immune response in the spleen, Cluster 42 for brain expression related to mitochondria in the choroid plexus, and Cluster 23 for cell line expression related to transcription. Additionally, it is part of Cluster 80 for single-cell expression related to transcription.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) and [CDK8](https://www.ncbi.nlm.nih.gov/gene/1024), each with 6 experiments. Additionally, [RARA](https://www.ncbi.nlm.nih.gov/gene/5914), [HDAC1](https://www.ncbi.nlm.nih.gov/gene/3065), and [INTS11](https://www.ncbi.nlm.nih.gov/gene/54973) were each found to be regulating in 5 experiments.




The gene PTBP1 has been associated with [Neutrophil forward scatter](https://pubmed.ncbi.nlm.nih.gov/37596262), which maps closely to the MeSH term [Neutrophils](https://meshb.nlm.nih.gov/record/ui?ui=D009504) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method.


The gene is expressed in the spleen, as well as in various blood cell types, including B lymphoblasts, CD34+ cells, and leukemia cell lines such as chronic myelogenous leukemia (K-562) and lymphoblastic leukemia (MOLT-4).


The proteins are involved in several key processes, including gene expression, processing of capped intron-containing pre-mRNA, and mRNA splicing, with a particular focus on the major pathway of mRNA splicing.



The protein sequence analyzed has a GRAVY value of -0.116, indicating a hydrophilic nature (76.83rd percentile). It exhibits a charge of 9.81 at pH 7.0 (81.05th percentile) and a median structural flexibility of 1.001 (44.31st percentile). The protein's secondary structure is predicted to be 31.24% helix (43.43rd percentile), 33.03% sheet (45.7th percentile), and 33.39% turn (83.69th percentile). The instability index is 37.88 (16.41st percentile), with an isoelectric point of 9.21 (84.74th percentile). The sequence length is 557 amino acids (64.73rd percentile), and the molecular weight is 59632.34 Da (62.25th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |