---
layout: post
title: LAIR1
date: 2025-01-17 16:55 CST
description: LAIR1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3903) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3903  | LAIR1 | ENSG00000167613 | 19q13.42 |



The gene is involved in the [adaptive immune response](https://amigo.geneontology.org/amigo/term/GO:0002250) and enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is active in and located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), as well as located in the [specific granule membrane](https://amigo.geneontology.org/amigo/term/GO:0035579) and the [tertiary granule membrane](https://amigo.geneontology.org/amigo/term/GO:0070821).


The gene length is 13,586 base pairs (32.62% of all genes), the mature length is 2,738.5 base pairs, and the primary transcript length is 13,581.5 base pairs.


LAIR1 (Gene ID: 3903) has been mentioned in [66 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22LAIR1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest mention in 1997. The middle 50% of these publications were published between 2008 and 2019. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning LAIR1, ranked by their scientific influence, include "[Collagen promotes anti-PD-1/PD-L1 resistance in cancer through LAIR1-dependent CD8<sup>+</sup> T cell exhaustion.](https://pubmed.ncbi.nlm.nih.gov/32908154)" (2020) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 11.58, "[Remodeling the tumor microenvironment via blockade of LAIR-1 and TGF-Î² signaling enables PD-L1-mediated tumor eradication.](https://pubmed.ncbi.nlm.nih.gov/35230974)" (2022) with an RCR of 7.07, "[LAIR-1, a novel inhibitory receptor expressed on human mononuclear leukocytes.](https://pubmed.ncbi.nlm.nih.gov/9285412)" (1997) with an RCR of 5.19, "[A LAIR1 insertion generates broadly reactive antibodies against malaria variant antigens.](https://pubmed.ncbi.nlm.nih.gov/26700814)" (2016) with an RCR of 3.51, and "[C1q limits dendritic cell differentiation and activation by engaging LAIR-1.](https://pubmed.ncbi.nlm.nih.gov/23093673)" (2012) with an RCR of 3.17. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[LAIR1](https://www.proteinatlas.org/ENSG00000167613-LAIR1) is a receptor with evidence at the protein level, detected in many tissues. It is primarily located in the plasma membrane and is associated with immune response functions, particularly in the spleen and macrophages. LAIR1 is also expressed in myeloid leukemia cell lines and is part of the non-specific immune response cluster in the brain. Its expression in blood is categorized as non-specific with unknown function.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) with 5 experiments, followed by [CEBPA](https://www.ncbi.nlm.nih.gov/gene/1050), [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313), [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), and [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), each with 4 experiments.





The gene is expressed in multiple tissue samples, including lung, blood, and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in various cell types such as CD34+, CD33+ Myeloid, BDCA4+ Dendritic Cells, CD56+ NK Cells, CD19+ B Cells (neg. sel.), bone marrow, and thymus.




The analyzed protein sequence has a GRAVY value of -0.515 (33.46th percentile), indicating a hydrophilic nature. The charge at pH 7 is -8.71 (24.93rd percentile), and the median structural flexibility is 1.012 (86.93rd percentile). The protein has a propensity for helix, sheet, and turn structures of 29.62% (28.59th percentile), 31.71% (37.86th percentile), and 31.36% (72.97th percentile), respectively. The instability index is 53.16 (67.09th percentile), with an isoelectric point of 5.50 (18.23rd percentile). The protein is 287 amino acids long (26.49th percentile) and has a molecular weight of 31468.77 Da (26.07th percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |