---
layout: post
title: CPNE3
date: 2025-01-17 16:55 CST
description: CPNE3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8895) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8895  | CPNE3 | ENSG00000085719 | 8q21.3 |



The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [protein serine/threonine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004674), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [calcium-dependent phospholipid binding](https://amigo.geneontology.org/amigo/term/GO:0005544). It is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [nucleolus](https://amigo.geneontology.org/amigo/term/GO:0005730), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [mitochondrion](https://amigo.geneontology.org/amigo/term/GO:0005739), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925), [cell junction](https://amigo.geneontology.org/amigo/term/GO:0030054), [azurophil granule membrane](https://amigo.geneontology.org/amigo/term/GO:0035577), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). It is involved in the [positive regulation of cell migration](https://amigo.geneontology.org/amigo/term/GO:0030335), [ERBB2 signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0038128), [cellular response to calcium ion](https://amigo.geneontology.org/amigo/term/GO:0071277), and [cellular response to growth factor stimulus](https://amigo.geneontology.org/amigo/term/GO:0071363). Additionally, it enables [receptor tyrosine kinase binding](https://amigo.geneontology.org/amigo/term/GO:0030971), [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872), and [calcium-dependent protein binding](https://amigo.geneontology.org/amigo/term/GO:0048306).


The gene length is 23,235 base pairs (45.81% of all genes), the mature length is 2,375.5 base pairs, and the primary transcript length is 15,634 base pairs (45.15% of all genes).


The gene CPNE3 (NCBI ID: 8895) has been mentioned in [12 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CPNE3%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning this gene was in 2000, and the middle 50% of publications occurred between 2015 and 2019.


The top 5 publications mentioning CPNE3, ranked by their scientific influence, include "[Circulating exosomal CPNE3 as a diagnostic and prognostic biomarker for colorectal cancer.](https://pubmed.ncbi.nlm.nih.gov/30078189)" (2019) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 4.55, "[Quantitative proteomic analysis identifies CPNE3 as a novel metastasis-promoting gene in NSCLC.](https://pubmed.ncbi.nlm.nih.gov/23713811)" (2013) with an RCR of 1.21, "[Copine-III interacts with ErbB2 and promotes tumor cell migration.](https://pubmed.ncbi.nlm.nih.gov/20010870)" (2010) with an RCR of 0.95, "[Low CPNE3 expression is associated with risk of acute myocardial infarction: A feasible genetic marker of acute myocardial infarction in patients with stable coronary artery disease.](https://pubmed.ncbi.nlm.nih.gov/29297177)" (2019) with an RCR of 0.8, and "[Characterization of human copine III as a phosphoprotein with associated kinase activity.](https://pubmed.ncbi.nlm.nih.gov/11041869)" (2000) with an RCR of 0.57. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CPNE3](https://www.proteinatlas.org/ENSG00000085719-CPNE3) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in the nucleoplasm and cytosol. In terms of expression clusters, CPNE3 is found in Cluster 2 for blood expression, specifically in dendritic cells with unknown function, and in Cluster 56 for tissue expression with non-specific function. In the brain, it is part of Cluster 55, associated with transcription. Additionally, CPNE3 is involved in Cluster 40 for cell line expression and Cluster 28 for single cell expression, both with non-specific functions and antigen presentation, respectively.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227) with 7 experiments, [AR](https://www.ncbi.nlm.nih.gov/gene/367) with 6 experiments, [PGR](https://www.ncbi.nlm.nih.gov/gene/5241) with 5 experiments, and both [RELA](https://www.ncbi.nlm.nih.gov/gene/5970) and [TBP](https://www.ncbi.nlm.nih.gov/gene/6908) with 4 experiments each. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent detection in CHIP-seq studies.



The GWAS data indicates associations with a range of conditions, including lower respiratory tract disease, lung disease, respiratory system disease, nervous system disease, central nervous system disease, neurodegenerative disease, Alzheimer's disease, tauopathy, dementia, cognitive disorder, disease of mental health, bipolar disorder, mood disorder, synucleinopathy, and Parkinson's disease.



The gene is expressed in various tissues and cell types, including BDCA4+ dendritic cells, bronchial epithelial cells, pancreatic islets, colon, small intestine, and bone marrow.




The protein sequence analyzed has a GRAVY value of -0.252 (63.65th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -7.064 (28.24th percentile). The median structural flexibility is 1.001 (47.91st percentile), with a propensity for amino acids to be in helix, sheet, and turn conformations at 27.37% (13.75th percentile), 37.06% (73.44th percentile), and 30.35% (66.36th percentile), respectively. The instability index is 37.80 (16.17th percentile), and the isoelectric point is 5.598 (20.55th percentile). The protein has a length of 537 amino acids (62.75th percentile) and a molecular weight of 60130 Da (62.89th percentile). These calculations were performed using BioPython's sequence analysis tools, which can be found [here](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |