---
layout: post
title: HNRNPAB
date: 2025-01-17 16:55 CST
description: HNRNPAB description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3182) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3182  | HNRNPAB | ENSG00000197451 | 5q35.3 |



The gene is involved in [epithelial to mesenchymal transition](https://amigo.geneontology.org/amigo/term/GO:0001837) and [regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0010468). It enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [mRNA binding](https://amigo.geneontology.org/amigo/term/GO:0003729), and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), and is also found in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737). Additionally, it is part of the [mRNA editing complex](https://amigo.geneontology.org/amigo/term/GO:0045293), [RNA polymerase II transcription regulator complex](https://amigo.geneontology.org/amigo/term/GO:0090575), and [ribonucleoprotein complex](https://amigo.geneontology.org/amigo/term/GO:1990904). The gene is involved in [mRNA modification](https://amigo.geneontology.org/amigo/term/GO:0016556), [positive regulation of DNA-templated transcription](https://amigo.geneontology.org/amigo/term/GO:0045893), [positive regulation of DNA demethylation](https://amigo.geneontology.org/amigo/term/GO:1901537), and [negative regulation of nuclear-transcribed mRNA catabolic process, nonsense-mediated decay](https://amigo.geneontology.org/amigo/term/GO:2000623).


The gene length is 189,227 base pairs (91.82% of all genes), the mature length is 1,220 base pairs, and the primary transcript length is 88,578.5 base pairs.


The gene HNRNPAB (NCBI ID: 3182) has been mentioned in [19 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22HNRNPAB%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1994 and the middle 50% of publications occurring between 2005 and 2020. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning HNRNPAB, ranked by their scientific influence, include "[Function of conserved domains of hnRNP A1 and other hnRNP A/B proteins.](https://pubmed.ncbi.nlm.nih.gov/7957114)" (1994) (relative citation ratio: 5.28), "[hnRNP A/B proteins are required for inhibition of HIV-1 pre-mRNA splicing.](https://pubmed.ncbi.nlm.nih.gov/10406810)" (1999) (relative citation ratio: 3.72), "[Tumor suppressor lnc-CTSLP4 inhibits EMT and metastasis of gastric cancer by attenuating HNRNPAB-dependent Snail transcription.](https://pubmed.ncbi.nlm.nih.gov/33717650)" (2021) (relative citation ratio: 2.44), "[HNRNPAB induces epithelial-mesenchymal transition and promotes metastasis of hepatocellular carcinoma by transcriptionally activating SNAIL.](https://pubmed.ncbi.nlm.nih.gov/24638979)" (2014) (relative citation ratio: 2.26), and "[HNRNPAB-regulated lncRNA-ELF209 inhibits the malignancy of hepatocellular carcinoma.](https://pubmed.ncbi.nlm.nih.gov/31090062)" (2020) (relative citation ratio: 1.82). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[HNRNPAB](https://www.proteinatlas.org/ENSG00000197451-HNRNPAB) is an RNA-binding protein with evidence at the protein level. It is detected in all tissues and primarily localized in the nucleoplasm. The gene is expressed in various clusters, including Cluster 22 in eosinophils related to transcription, Cluster 25 in the small intestine associated with absorption, Cluster 56 in the hypothalamus involved in neuropeptide signaling, Cluster 22 in non-specific cell lines related to mRNA splicing and cell cycle, and Cluster 39 in non-specific single cells involved in basic cellular processes.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [HDAC1](https://www.ncbi.nlm.nih.gov/gene/3065) with 6 experiments, [EP300](https://www.ncbi.nlm.nih.gov/gene/2033) with 5 experiments, [SP1](https://www.ncbi.nlm.nih.gov/gene/6667) with 5 experiments, [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613) with 5 experiments, and [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) with 5 experiments. These TFs are notable for their regulatory roles as evidenced by the high number of experiments in which they were found to be active.



The GWAS data indicates associations with various disease conditions, including general diseases, diseases of anatomical entities, endocrine system diseases, thyroid gland diseases, and specifically hypothyroidism.



The gene is expressed in various tissues and cell types, including colon, 721_B_lymphoblasts, CD34+ cells, Lymphoma burkitts (Daudi), CD105+ endothelial cells, CD71+ early erythroid cells, and colorectal adenocarcinoma. This indicates a ubiquitous expression pattern across multiple tissue samples.



The mouse ortholog gene 15384 and its human ortholog gene 3182 are associated with phenotypes including abnormal embryo size and preweaning lethality with incomplete penetrance, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -1.0096 (4.21 percentile), indicating a hydrophilic nature. It exhibits a charge of 1.87 at pH 7.0 (57.09 percentile) and a median structural flexibility of 1.02 (96.58 percentile). The protein's secondary structure is predicted to be 27.71% helix (15.41 percentile), 25.60% sheet (10.17 percentile), and 37.65% turn (94.8 percentile). The instability index is 39.04 (19.56 percentile), with an isoelectric point of 8.21 (61.4 percentile). The protein has a length of 332 amino acids (35.08 percentile) and a molecular weight of 36224.43 Da (33.92 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |