---
layout: post
title: PDK3
date: 2025-01-17 16:55 CST
description: PDK3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5165) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5165  | PDK3 | ENSG00000067992 | Xp22.11 |



The gene enables [protein kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004672), [protein serine/threonine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004674), and [pyruvate dehydrogenase (acetyl-transferring) kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004740). It is involved in [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524). The gene is located in the [nucleolus](https://amigo.geneontology.org/amigo/term/GO:0005730) and [mitochondrion](https://amigo.geneontology.org/amigo/term/GO:0005739), specifically in the [mitochondrial matrix](https://amigo.geneontology.org/amigo/term/GO:0005759). It is active in the [mitochondrion](https://amigo.geneontology.org/amigo/term/GO:0005739) and plays roles in various processes, including [protein phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0006468), [regulation of acetyl-CoA biosynthetic process from pyruvate](https://amigo.geneontology.org/amigo/term/GO:0010510), [regulation of glucose metabolic process](https://amigo.geneontology.org/amigo/term/GO:0010906), [peptidyl-serine phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0018105), [peroxisome proliferator activated receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0035357), [cellular response to fatty acid](https://amigo.geneontology.org/amigo/term/GO:0071398), [hypoxia-inducible factor-1alpha signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0097411), and [regulation of reactive oxygen species metabolic process](https://amigo.geneontology.org/amigo/term/GO:2000377).


The gene length is 64,402 base pairs (72.68% of all genes), the mature length is 2,091 base pairs (55.07% of all genes), and the primary transcript length is 45,848 base pairs (73.31% of all genes).


The gene PDK3 (NCBI ID: 5165) has been mentioned in [16 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PDK3%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 2005 and the middle 50% of publications occurring between 2011 and 2020. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning PDK3, ranked by their scientific influence, include "[Discovery of Hordenine as a Potential Inhibitor of Pyruvate Dehydrogenase Kinase 3: Implication in Lung Cancer Therapy.](https://pubmed.ncbi.nlm.nih.gov/32422877)" (2020) (relative citation ratio: 5.04), "[Investigation of inhibitory potential of quercetin to the pyruvate dehydrogenase kinase 3: Towards implications in anticancer therapy.](https://pubmed.ncbi.nlm.nih.gov/31233792)" (2019) (relative citation ratio: 4.56), "[Inactivation of the HIF-1Î±/PDK3 signaling axis drives melanoma toward mitochondrial oxidative metabolism and potentiates the therapeutic activity of pro-oxidants.](https://pubmed.ncbi.nlm.nih.gov/22865452)" (2012) (relative citation ratio: 2.79), "[Overexpression of pyruvate dehydrogenase kinase 3 increases drug resistance and early recurrence in colon cancer.](https://pubmed.ncbi.nlm.nih.gov/21763680)" (2011) (relative citation ratio: 2.31), and "[Molecular interaction studies on ellagic acid for its anticancer potential targeting pyruvate dehydrogenase kinase 3.](https://pubmed.ncbi.nlm.nih.gov/35514501)" (2019) (relative citation ratio: 2.15). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PDK3](https://www.proteinatlas.org/ENSG00000067992-PDK3) is a kinase and transferase with evidence at the protein level. It is implicated in several diseases, including Charcot-Marie-Tooth disease, neurodegeneration, and neuropathy. The gene is detected in all tissues and is localized to the nucleoli and mitochondria. In terms of expression clusters, PDK3 is associated with Cluster 35 in neutrophils for blood expression, Cluster 53 for non-specific tissue expression, Cluster 14 in neurons for brain expression, Cluster 46 in lymphoma for cell line expression, and Cluster 32 in smooth muscle cells for single-cell expression.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025) with 6 experiments, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) and [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) each with 5 experiments, and [SMAD2](https://www.ncbi.nlm.nih.gov/gene/4087) and [RELA](https://www.ncbi.nlm.nih.gov/gene/5970) each with 4 experiments.



The input data indicates associations with various disease conditions, including neurodegenerative diseases such as Alzheimer's disease and tauopathy. Additionally, it encompasses cognitive disorders, dementia, and diseases of mental health, all of which are subcategories of central nervous system diseases and broader nervous system diseases.



The gene is expressed in various tissues, including blood vessels, fallopian tubes, and blood, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles show expression in cardiac myocytes, B lymphoblasts, CD33+ myeloid cells, whole blood, the atrioventricular node, and testis.


The input data highlights several key pathways involved in cellular processes, including metabolism, pyruvate metabolism, and the citric acid (TCA) cycle. Specific pathways mentioned are pyruvate metabolism, regulation of the pyruvate dehydrogenase (PDH) complex, and the TCA cycle along with respiratory electron transport. Additionally, signaling pathways such as those involving retinoic acid and general signal transduction are also noted.



The analyzed protein sequence has a GRAVY value of -0.390 (47.31% percentile), indicating a hydrophilic nature. It exhibits a charge of 3.13 at pH 7.0 (61.73% percentile) and a median structural flexibility of 0.999 (38.94% percentile). The protein's secondary structure is predicted to be 32.51% helix (55.18% percentile), 37.93% sheet (78.16% percentile), and 28.57% turn (52.36% percentile). The instability index is 47.49 (48.31% percentile), with an isoelectric point of 8.46 (65.88% percentile). The protein has a length of 406 amino acids (46.11% percentile) and a molecular weight of 46938.34 Da (48.23% percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |