---
layout: post
title: GLE1
date: 2025-01-17 16:55 CST
description: GLE1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2733) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2733  | GLE1 | ENSG00000119392 | 9q34.11 |



The gene enables [inositol hexakisphosphate binding](https://amigo.geneontology.org/amigo/term/GO:0000822), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [phospholipid binding](https://amigo.geneontology.org/amigo/term/GO:0005543), and [translation initiation factor binding](https://amigo.geneontology.org/amigo/term/GO:0031369). It is located in the [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615), [nuclear envelope](https://amigo.geneontology.org/amigo/term/GO:0005635), [nucleolus](https://amigo.geneontology.org/amigo/term/GO:0005730), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [centrosome](https://amigo.geneontology.org/amigo/term/GO:0005813), [centriole](https://amigo.geneontology.org/amigo/term/GO:0005814), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [nuclear membrane](https://amigo.geneontology.org/amigo/term/GO:0031965), and [ciliary basal body](https://amigo.geneontology.org/amigo/term/GO:0036064). It is also part of the [nuclear pore](https://amigo.geneontology.org/amigo/term/GO:0005643) and [nuclear pore cytoplasmic filaments](https://amigo.geneontology.org/amigo/term/GO:0044614). The gene is involved in [mRNA export from nucleus](https://amigo.geneontology.org/amigo/term/GO:0006406), [nucleocytoplasmic transport](https://amigo.geneontology.org/amigo/term/GO:0006913), [protein transport](https://amigo.geneontology.org/amigo/term/GO:0015031), and [poly(A)+ mRNA export from nucleus](https://amigo.geneontology.org/amigo/term/GO:0016973).


The gene length is 42,444 base pairs (61.83% of all genes), the mature length is 1,530 base pairs (38.91% of all genes), and the primary transcript length is 17,710 base pairs (48.44% of all genes).


The gene GLE1 (NCBI ID: 2733) has been mentioned in [42 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22GLE1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1999 and the middle 50% of publications occurring between 2011 and 2019. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning GLE1, ranked by their scientific influence, include "[Ca2+ signalling between single L-type Ca2+ channels and ryanodine receptors in heart cells.](https://pubmed.ncbi.nlm.nih.gov/11279498)" (2001) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 6.38, "[Activation of the DExD/H-box protein Dbp5 by the nuclear-pore protein Gle1 and its coactivator InsP6 is required for mRNA export.](https://pubmed.ncbi.nlm.nih.gov/16783364)" (2006) with an RCR of 3.45, "[Deleterious mutations in the essential mRNA metabolism factor, hGle1, in amyotrophic lateral sclerosis.](https://pubmed.ncbi.nlm.nih.gov/25343993)" (2015) with an RCR of 3.15, "[Mutations in mRNA export mediator GLE1 result in a fetal motoneuron disease.](https://pubmed.ncbi.nlm.nih.gov/18204449)" (2008) with an RCR of 2.61, and "[The mRNA export factor Gle1 and inositol hexakisphosphate regulate distinct stages of translation.](https://pubmed.ncbi.nlm.nih.gov/18724935)" (2008) with an RCR of 2.13. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[GLE1](https://www.proteinatlas.org/ENSG00000119392-GLE1) is a gene with evidence at the protein level and is implicated in disease variants. It is detected in all RNA tissue distributions and is localized in the nuclear membrane, nucleoli, and cytosol. The gene is expressed in monocytes and neutrophils, playing a role in the innate immune response, and is also expressed in the testis, involved in basic cellular processes. In the brain, it is non-specifically expressed and associated with transcription. Additionally, GLE1 is expressed in spermatocytes, contributing to spermatogenesis.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [NFATC1](https://www.ncbi.nlm.nih.gov/gene/4772), [SPI1](https://www.ncbi.nlm.nih.gov/gene/6688), [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), and [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313), each with 4 experiments. Additionally, [TRIM28](https://www.ncbi.nlm.nih.gov/gene/10155) was found to be regulating in 3 experiments.



The input data indicates associations with various metabolic disorders, including diseases of metabolism, acquired metabolic diseases, carbohydrate metabolism diseases, glucose metabolism diseases, diabetes mellitus, and specifically type 2 diabetes mellitus.



The gene is expressed in various tissues, including the pituitary, adrenal gland, and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in 721_B_lymphoblasts, BDCA4+ Dentritic Cells, and multiple regions of the testis, including the interstitial area, Leydig cells, seminiferous tubules, and germ cells.



The mouse ortholog gene 74412 and its human ortholog gene 2733 are associated with phenotypes such as increased heart weight and preweaning lethality with complete penetrance, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.614 (24.05th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of 0.293 (51.35th percentile). The median structural flexibility is 1.008 (75.51st percentile), with a notable instability index of 60.16 (83.31st percentile). The protein's isoelectric point is 7.06 (51.08th percentile), and it has a molecular weight of 79835.23 Da (76.15th percentile). In terms of secondary structure, the protein exhibits a preference for helix (37.97%, 88.58th percentile), sheet (29.66%, 26.53rd percentile), and turn (24.21%, 16.25th percentile) conformations. These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |