---
layout: post
title: ITPKB
date: 2025-01-17 16:55 CST
description: ITPKB description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3707) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3707  | ITPKB | ENSG00000143772 | 1q42.12 |



The gene is involved in various biological processes, including the [MAPK cascade](https://amigo.geneontology.org/amigo/term/GO:0000165), [regulation of protein phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0001932), and [myeloid cell homeostasis](https://amigo.geneontology.org/amigo/term/GO:0002262). It enables functions such as [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [calmodulin binding](https://amigo.geneontology.org/amigo/term/GO:0005516), and [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524). The gene is active in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), and is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [endoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0005783), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0005856), and [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020). Additionally, it is involved in [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), [cell surface receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007166), [phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0016310), and various other processes related to inositol phosphate metabolism and immune cell regulation. The gene also enables [inositol-1,4,5-trisphosphate 3-kinase activity](https://amigo.geneontology.org/amigo/term/GO:0008440) and [inositol tetrakisphosphate kinase activity](https://amigo.geneontology.org/amigo/term/GO:0051765).


The gene length is 28,463 base pairs (51.25th percentile), the mature length is 1,491 base pairs, and the primary transcript length is 17,730 base pairs (48.5th percentile).


The gene ITPKB (NCBI ID: 3707) has been mentioned in [15 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ITPKB%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 2000 and the middle 50% of publications occurring between 2011 and 2020. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning ITPKB, ranked by their relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows: [miR-132 loss de-represses ITPKB and aggravates amyloid and TAU pathology in Alzheimer's brain.](https://pubmed.ncbi.nlm.nih.gov/27485122) (2016) (relative citation ratio: 4.03), [The Parkinson's disease-associated gene ITPKB protects against Î±-synuclein aggregation by regulating ER-to-mitochondria calcium release.](https://pubmed.ncbi.nlm.nih.gov/33443159) (2021) (relative citation ratio: 2.6), [ROS-regulated phosphorylation of ITPKB by CAMK2G drives cisplatin resistance in ovarian cancer.](https://pubmed.ncbi.nlm.nih.gov/35039634) (2022) (relative citation ratio: 2.4), [Inositol trisphosphate 3-kinase B is increased in human Alzheimer brain and exacerbates mouse Alzheimer pathology.](https://pubmed.ncbi.nlm.nih.gov/24401760) (2014) (relative citation ratio: 1.73), and [Crystal structure of the catalytic core of inositol 1,4,5-trisphosphate 3-kinase.](https://pubmed.ncbi.nlm.nih.gov/15350215) (2004) (relative citation ratio: 1.04). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ITPKB](https://www.proteinatlas.org/ENSG00000143772-ITPKB) is a gene with molecular functions including calmodulin-binding, kinase, and transferase activities. It is involved in cancer-related processes and has evidence at the protein level. The gene is detected in many tissues and is primarily located in the nucleoplasm. Its expression is associated with various clusters, including non-specific nuclear processes in blood, transmembrane transport in choroid plexus tissue, mixed function in brain astrocytes, and melanin biosynthesis in skin cancer cell lines. Additionally, it is expressed in astrocytes with an unknown function in single-cell analysis.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [PPARG](https://www.ncbi.nlm.nih.gov/gene/5468) and [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046), each with 5 experiments. Additionally, [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), and [SUZ12](https://www.ncbi.nlm.nih.gov/gene/23512) were each found to be regulating in 4 experiments.



The GWAS data indicates associations with a range of disease conditions, including various metabolic disorders such as diabetes mellitus, type 2 diabetes mellitus, and obesity. Additionally, there are links to cardiovascular diseases, including hypertension and artery disease. The data also highlights associations with nervous system diseases, including cognitive disorders and substance-related disorders like alcohol dependence. Furthermore, eye diseases such as retinal degeneration and glaucoma, as well as orofacial clefts and prion diseases, are also implicated.


The gene ITPKB has been associated with several traits through genome-wide association studies. Notably, it has been linked to [Calcium levels](https://pubmed.ncbi.nlm.nih.gov/33887147), which maps closely to the MeSH term [Calcium](https://meshb.nlm.nih.gov/record/ui?ui=D002118) using the BioWordVec method. Additionally, ITPKB has been associated with [Refractive error](https://pubmed.ncbi.nlm.nih.gov/32231278), which directly matches the MeSH term [Refractive Errors](https://meshb.nlm.nih.gov/record/ui?ui=D012030). These associations highlight the potential role of ITPKB in regulating calcium homeostasis and visual acuity.


The gene is expressed in the uterus, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, according to the GTEx Tissue Gene Expression Profiles, the gene is expressed in various tissues including the Prefrontal Cortex, retina, CD4+ T cells, CD8+ T cells, Hypothalamus, Spinal cord, Thalamus, Olfactory Bulb, and Thymus.


The input data highlights key pathways involved in metabolism, specifically focusing on inositol phosphate metabolism. Notably, it includes the synthesis of IP3 and IP4 in the cytosol.



The analyzed protein sequence has a GRAVY value of -0.685 (18.65th percentile), indicating a hydrophilic nature. It exhibits a charge of 9.495 at pH 7.0 (80.34th percentile) and a median structural flexibility of 1.014 (90.31st percentile). The protein's secondary structure is predicted to be 29.81% helix (30.29th percentile), 25.58% sheet (10.13th percentile), and 37.10% turn (93.88th percentile). The instability index is 64.60 (89.77th percentile), with an isoelectric point of 8.70 (71.25th percentile). The protein has a length of 946 amino acids (86.02nd percentile) and a molecular weight of 102375.03 Da (85.14th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |