---
layout: post
title: MAD1L1
date: 2025-01-17 16:55 CST
description: MAD1L1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8379) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8379  | MAD1L1 | ENSG00000002822 | 7p22.3 |



The gene is located in the [kinetochore](https://amigo.geneontology.org/amigo/term/GO:0000776), [nuclear envelope](https://amigo.geneontology.org/amigo/term/GO:0005635), [centrosome](https://amigo.geneontology.org/amigo/term/GO:0005813), [spindle](https://amigo.geneontology.org/amigo/term/GO:0005819), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and [mitotic spindle](https://amigo.geneontology.org/amigo/term/GO:0072686). It is involved in [mitotic spindle assembly checkpoint signaling](https://amigo.geneontology.org/amigo/term/GO:0007094), [negative regulation of T cell proliferation](https://amigo.geneontology.org/amigo/term/GO:0042130), [thymus development](https://amigo.geneontology.org/amigo/term/GO:0048538), [cell division](https://amigo.geneontology.org/amigo/term/GO:0051301), [attachment of mitotic spindle microtubules to kinetochore](https://amigo.geneontology.org/amigo/term/GO:0051315), [regulation of metaphase plate congression](https://amigo.geneontology.org/amigo/term/GO:0090235), and [positive regulation of mitotic cell cycle spindle assembly checkpoint](https://amigo.geneontology.org/amigo/term/GO:0090267). The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802), and [kinetochore binding](https://amigo.geneontology.org/amigo/term/GO:0043515). Additionally, it is part of the [nuclear pore nuclear basket](https://amigo.geneontology.org/amigo/term/GO:0044615), [MAD1 complex](https://amigo.geneontology.org/amigo/term/GO:1990706), and [mitotic spindle assembly checkpoint MAD1-MAD2 complex](https://amigo.geneontology.org/amigo/term/GO:1990728).


The gene length is 7,619 base pairs (20.4% of all genes). The mature length and primary transcript length are also 7,619 base pairs.


The gene MAD1L1 (NCBI ID: 8379) has been mentioned in [50 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22MAD1L1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1998 and the middle 50% of publications occurring between 2010 and 2018. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning MAD1L1, ranked by their scientific influence, include "[Human T cell leukemia virus type 1 oncoprotein Tax targets the human mitotic checkpoint protein MAD1.](https://pubmed.ncbi.nlm.nih.gov/9546394)" (1998) (relative citation ratio: 7.28), "[Sustained Mps1 activity is required in mitosis to recruit O-Mad2 to the Mad1-C-Mad2 core complex.](https://pubmed.ncbi.nlm.nih.gov/20624899)" (2010) (relative citation ratio: 5.1), "[Mad1 kinetochore recruitment by Mps1-mediated phosphorylation of Bub1 signals the spindle checkpoint.](https://pubmed.ncbi.nlm.nih.gov/24402315)" (2014) (relative citation ratio: 3.74), "[Nuclear pores protect genome integrity by assembling a premitotic and Mad1-dependent anaphase inhibitor.](https://pubmed.ncbi.nlm.nih.gov/24581499)" (2014) (relative citation ratio: 2.91), and "[Constitutive Mad1 targeting to kinetochores uncouples checkpoint signalling from chromosome biorientation.](https://pubmed.ncbi.nlm.nih.gov/21394085)" (2011) (relative citation ratio: 2.73). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[MAD1L1](https://www.proteinatlas.org/ENSG00000002822-MAD1L1) is a gene with evidence at the protein level and is implicated in disease variants. It is detected in all RNA tissue distributions and is primarily localized in the nucleoplasm and nuclear membrane. The gene is expressed in various clusters, including Cluster 9 in blood, which is associated with T-regs and cell cycle regulation, and Cluster 70 in tissue, which is linked to lymphoid tissue and immune response. Additionally, it is expressed in Cluster 11 in the brain, related to white matter and signal transduction, and Cluster 7 in single cells, associated with T-cells and T-cell receptors.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 6 experiments, [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046) with 5 experiments, [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599) with 5 experiments, [SMAD2](https://www.ncbi.nlm.nih.gov/gene/4087) with 5 experiments, and [USF2](https://www.ncbi.nlm.nih.gov/gene/7392) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The input data from GWAS indicates associations with a wide range of disease conditions, including various types of cancer such as breast cancer, thoracic cancer, male reproductive organ cancer, prostate cancer, testicular cancer, head and neck cancer, colorectal cancer, intestinal cancer, large intestine cancer, gastrointestinal system cancer, myeloid leukemia, acute myeloid leukemia, colon cancer, lung cancer, respiratory system cancer, leukemia, hematologic cancer, immune system cancer, and testicular germ cell cancer. Additionally, the data includes infectious diseases like hand, foot and mouth disease, viral infectious diseases, and human immunodeficiency virus infectious disease. Metabolic disorders such as diabetes mellitus, type 1 diabetes mellitus, and iron metabolism disease are also represented. Cardiovascular conditions include coronary artery disease, artery disease, vascular disease, cardiovascular system disease, and myocardial infarction. Neurological and mental health conditions include Alzheimer's disease, tauopathy, dementia, cognitive disorder, bipolar disorder, mood disorder, substance-related disorder, substance dependence, alcohol dependence, psychotic disorder, schizophrenia, endogenous depression, major depressive disorder, mental depression, attention deficit hyperactivity disorder, autism spectrum disorder, and pervasive developmental disorder. Other conditions include hypertension, kidney disease, urinary system disease, nervous system disease, central nervous system disease, neurodegenerative disease, lymphatic system disease, lymph node disease, lymphadenitis, Kawasaki disease, interstitial lung disease, and diarrhea.



The gene is expressed in various tissues and cell types, including blood and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, according to the GTEx Tissue Gene Expression Profiles, the gene is expressed in the liver, CD56+ NK cells, CD4+ T cells, CD8+ T cells, CD19+ B cells (neg. sel.), CD14+ monocytes, and lung.


The input data highlights various proteins involved in critical cell cycle processes, including cell cycle checkpoints, mitotic phases such as anaphase and metaphase, and specific events like the separation of sister chromatids and resolution of their cohesion. Additionally, it covers the mitotic spindle checkpoint and the amplification of signals from unattached kinetochores via a MAD2 inhibitory signal.



The analyzed protein sequence has a GRAVY value of -0.874 (8.37th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -13.15 (17.69th percentile). The median structural flexibility is 1.014 (89.53rd percentile), suggesting high flexibility. The protein has a strong tendency to form helices (44.99%, 98.29th percentile) and a moderate tendency to form sheets (27.44%, 16.09th percentile) and turns (19.36%, 1.84th percentile). The instability index is 57.75 (78.69th percentile), indicating potential instability. The isoelectric point is 5.72 (23.51st percentile), with a molecular weight of 83065.93 Da (77.85th percentile). The sequence length is 718 amino acids (76.44th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |