---
layout: post
title: SET
date: 2024-12-08 23:57 CST
description: SET description
tags: [cooccuring-genes]
categories: longest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6418) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6418  | SET | ENSG00000119335 | 9q34.11 |



The gene length is 753,821 base pairs (99.26% of all genes), the mature length is 6,515 base pairs, and the primary transcript length is 750,980.5 base pairs (99.74% of all genes).


The gene SET (NCBI ID: 6418) has been the subject of [49 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SET%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1994, and the middle 50% of publications occurred between 2006 and 2014.


The top five publications mentioning the SET gene, ranked by their scientific influence, include "[Sphingosine analogue drug FTY720 targets I2PP2A/SET and mediates lung tumour suppression via activation of PP2A-RIPK1-dependent necroptosis.](https://pubmed.ncbi.nlm.nih.gov/23180565)" (2013) (relative citation ratio: 6.06), "[Targeting SET/I(2)PP2A oncoprotein functions as a multi-pathway strategy for cancer therapy.](https://pubmed.ncbi.nlm.nih.gov/21297667)" (2011) (relative citation ratio: 3.39), "[Mechanism of inhibition of PP2A activity and abnormal hyperphosphorylation of tau by I2(PP2A)/SET.](https://pubmed.ncbi.nlm.nih.gov/21806989)" (2011) (relative citation ratio: 2.3), "[Relationship between the structure of SET/TAF-Ibeta/INHAT and its histone chaperone activity.](https://pubmed.ncbi.nlm.nih.gov/17360516)" (2007) (relative citation ratio: 2.26), and "[SET protein (TAF1beta, I2PP2A) is involved in neuronal apoptosis induced by an amyloid precursor protein cytoplasmic subdomain.](https://pubmed.ncbi.nlm.nih.gov/16162853)" (2005) (relative citation ratio: 2.05). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SUMO2](https://www.ncbi.nlm.nih.gov/gene/6613) with 6 experiments, followed by [TFAP2C](https://www.ncbi.nlm.nih.gov/gene/7022), [SMC3](https://www.ncbi.nlm.nih.gov/gene/9126), [E2F1](https://www.ncbi.nlm.nih.gov/gene/1869), and [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599), each with 5 experiments.











The analyzed protein sequence has a GRAVY value of -1.33 (1.11 percentile), indicating a hydrophilic nature. It exhibits a charge of -52.82 at pH 7 (2.18 percentile) and a median structural flexibility of 1.03 (99.17 percentile). The protein's secondary structure is predicted to be 37.59% helix (87.35 percentile), 25.17% sheet (9.1 percentile), and 33.1% turn (82.4 percentile). The instability index is 68.24 (93.35 percentile), with an isoelectric point of 4.23 (0.53 percentile). The protein has a length of 290 amino acids (26.99 percentile) and a molecular weight of 33488.45 Da (28.52 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |