---
layout: post
title: LSS
date: 2025-01-17 16:55 CST
description: LSS description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4047) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4047  | LSS | ENSG00000160285 | 21q22.3 |



The gene enables [lanosterol synthase activity](https://amigo.geneontology.org/amigo/term/GO:0000250) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is located in the [endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0005789) and [lipid droplet](https://amigo.geneontology.org/amigo/term/GO:0005811), and is active in the [lipid droplet](https://amigo.geneontology.org/amigo/term/GO:0005811). The gene is involved in the [steroid biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006694), [cholesterol biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006695), [triterpenoid biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0016104), and [regulation of protein stability](https://amigo.geneontology.org/amigo/term/GO:0031647). Additionally, it is located in the [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020).


The gene length is 1,200 base pairs (2.95% of all genes), the mature length is 844 base pairs, and the primary transcript length is 1,200 base pairs.


The gene LSS (NCBI ID: 4047) has been mentioned in [32 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22LSS%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning LSS was in 1977, and the middle 50% of publications occurred between 2004 and 2019.


The top 5 publications mentioning LSS, ranked by their scientific influence, include "[Stickler Syndrome (SS): Laser Prophylaxis for Retinal Detachment (Modified Ora Secunda Cerclage, OSC/SS).](https://pubmed.ncbi.nlm.nih.gov/33447008)" (2021) (relative citation ratio: 3.98), "[Target identification reveals lanosterol synthase as a vulnerability in glioma.](https://pubmed.ncbi.nlm.nih.gov/30923116)" (2019) (relative citation ratio: 1.78), "[Biallelic Variants in Lanosterol Synthase (LSS) Cause Palmoplantar Keratoderma-Congenital Alopecia Syndrome Type 2.](https://pubmed.ncbi.nlm.nih.gov/35413293)" (2022) (relative citation ratio: 1.64), "[Enhanced synthesis of the oxysterol 24(S),25-epoxycholesterol in macrophages by inhibitors of 2,3-oxidosqualene:lanosterol cyclase: a novel mechanism for the attenuation of foam cell formation.](https://pubmed.ncbi.nlm.nih.gov/14512442)" (2003) (relative citation ratio: 1.58), and "[Molecular cloning of the human gene encoding lanosterol synthase from a liver cDNA library.](https://pubmed.ncbi.nlm.nih.gov/7639730)" (1995) (relative citation ratio: 1.53). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[LSS](https://www.proteinatlas.org/ENSG00000160285-LSS) is an isomerase with evidence at the protein level. It is implicated in various diseases, including cataracts, disease variants, hypotrichosis, and intellectual disability. The gene is detected in all RNA tissue distributions and is associated with specific expression clusters in blood (Cluster 12: Non-specific - Mitochondrial translation), tissue (Cluster 43: Non-specific - Mitochondrial translation), brain (Cluster 49: Hindbrain - Mixed function), cell lines (Cluster 47: Leukemia - Hemostasis), and single cells (Cluster 62: Hepatocytes - Metabolism).


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), [KDM1A](https://www.ncbi.nlm.nih.gov/gene/23028), [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908), and [USF2](https://www.ncbi.nlm.nih.gov/gene/7392), each with 5 experiments. Additionally, [HNF4A](https://www.ncbi.nlm.nih.gov/gene/3172) was found to be regulating in 4 experiments.



The input data indicates associations with a variety of disease conditions, including several types of cancer such as leukemia, lymphoma, and immune system cancer. Additionally, respiratory conditions like asthma, bronchial disease, and obstructive lung disease are noted. The data also encompasses hematopoietic system diseases, immune system diseases, and cardiovascular system diseases. Furthermore, musculoskeletal system diseases, including bone disease and connective tissue disease, are mentioned.



The gene is expressed in various tissues, including the ovary, pituitary, and adrenal gland, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Lymphoma Burkitt's (Raji), 721 B lymphoblasts, adipocytes, adrenal cortex, and fetal liver.


The input data highlights several key biological processes, including the activation of gene expression by SREBF (SREBP), the regulation of cholesterol biosynthesis by SREBP (SREBF), cholesterol biosynthesis, and broader metabolic processes such as the metabolism of lipids and lipoproteins.


The mouse ortholog gene 16987 and its human ortholog 4047 are associated with several phenotypes, including embryonic lethality prior to organogenesis, embryonic lethality prior to tooth bud stage, and preweaning lethality with complete penetrance. Notably, the preweaning lethality phenotype is observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.319 (55.82nd percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -9.41 (23.37th percentile), and the median structural flexibility is 0.994 (18.24th percentile). The protein has a length of 732 amino acids (77.22nd percentile) and a molecular weight of 83307.8 Da (78.02nd percentile). The secondary structure affinities are 30.6% for helix (37.17th percentile), 36.1% for sheet (66.76th percentile), and 26.2% for turn (31.22nd percentile). The instability index is 48.47 (51.91st percentile), and the isoelectric point is 6.16 (34.66th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |