---
layout: post
title: DCC
date: 2024-12-08 23:57 CST
description: DCC description
tags: [cooccuring-genes]
categories: longest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1630) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1630  | DCC | ENSG00000187323 | 18q21.2 |



The gene length is 500,047 base pairs (98.36% of all genes), the mature length is 7,313 base pairs, and the primary transcript length is 490,356 base pairs (99.2% of all genes).


The gene DCC (NCBI ID: 1630) has been the subject of [297 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22DCC%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1952, and the middle 50% of publications occurred between 1997 and 2015.


The top five publications mentioning the DCC gene, ranked by their scientific influence, include "[Phenotype of mice lacking functional Deleted in colorectal cancer (Dcc) gene.](https://pubmed.ncbi.nlm.nih.gov/9126737)" (1997) with a relative citation ratio of 11.35, "[The DCC protein and prognosis in colorectal cancer.](https://pubmed.ncbi.nlm.nih.gov/8929264)" (1996) with a relative citation ratio of 5.58, "[Frequent loss of heterozygosity at the DCC locus in gastric cancer.](https://pubmed.ncbi.nlm.nih.gov/1591722)" (1992) with a relative citation ratio of 4.94, "[Netrin-1 acts as a survival factor via its receptors UNC5H and DCC.](https://pubmed.ncbi.nlm.nih.gov/11387206)" (2001) with a relative citation ratio of 4.6, and "[The DCC gene: structural analysis and mutations in colorectal carcinomas.](https://pubmed.ncbi.nlm.nih.gov/8188295)" (1994) with a relative citation ratio of 4.24. The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


The gene [None](https://www.proteinatlas.org/None-None) functions as a developmental protein and receptor, with evidence at the protein level. It is implicated in various diseases, including cancer-related genes, disease variants, intellectual disability, and acts as a tumor suppressor. The gene is detected in some RNA tissue distributions and is localized in the Golgi apparatus, centriolar satellite, centrosome, and basal body. It is expressed in specific tissue clusters related to testis spermatogenesis, neurons with mixed function, myeloma immune response, and oligodendrocyte precursor cells. The gene is unprognostic for breast invasive carcinoma, glioblastoma multiforme, and lung squamous cell carcinoma.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170), [HIF1A](https://www.ncbi.nlm.nih.gov/gene/3091), [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), and [OTX2](https://www.ncbi.nlm.nih.gov/gene/5015), each with 3 experiments. Additionally, [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) was found to be regulating in 2 experiments.








The mouse ortholog gene 13176 and its human ortholog gene 1630 are associated with the phenotype of preweaning lethality with complete penetrance, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.332 (54.22 percentile), indicating a hydrophilic nature. The charge at pH 7 is -8.03 (26.33 percentile), and the median structural flexibility is 1.005 (62.95 percentile). The protein has a propensity for helix, sheet, and turn structures of 26.26% (8.67 percentile), 35.52% (62.96 percentile), and 34.14% (86.57 percentile), respectively. The instability index is 48.54 (52.16 percentile), with an isoelectric point of 6.32 (38.54 percentile). The protein is 1447 amino acids long (94.64 percentile) and has a molecular weight of 158454.64 Da (94.45 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |