---
layout: post
title: SYNGR1
date: 2025-01-17 16:55 CST
description: SYNGR1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9145) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9145  | SYNGR1 | ENSG00000100321 | 22q13.1 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [azurophil granule membrane](https://amigo.geneontology.org/amigo/term/GO:0035577), and [melanosome](https://amigo.geneontology.org/amigo/term/GO:0042470). It is active in the [synaptic vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030672) and [neuromuscular junction](https://amigo.geneontology.org/amigo/term/GO:0031594). The gene is involved in various processes including [protein targeting](https://amigo.geneontology.org/amigo/term/GO:0006605), [regulated exocytosis](https://amigo.geneontology.org/amigo/term/GO:0045055), [regulation of long-term neuronal synaptic plasticity](https://amigo.geneontology.org/amigo/term/GO:0048169), [regulation of short-term neuronal synaptic plasticity](https://amigo.geneontology.org/amigo/term/GO:0048172), [synaptic vesicle membrane organization](https://amigo.geneontology.org/amigo/term/GO:0048499), and [cellular response to leukemia inhibitory factor](https://amigo.geneontology.org/amigo/term/GO:1990830).


The gene length is 117,668 base pairs (84.88% of all genes), the mature length is 4,339.5 base pairs, and the primary transcript length is 59,714.5 base pairs.


SYNGR1 (gene NCBI ID: 9145) has been mentioned in [6 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SYNGR1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 2004, and the middle 50% of publications occurred between 2005 and 2009.


The top 5 publications mentioning SYNGR1 include studies that explore its association with various disorders. The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Notable publications include [SYNGR1 is associated with schizophrenia and bipolar disorder in southern India.](https://pubmed.ncbi.nlm.nih.gov/16215643) (2005) (relative citation ratio: 0.48), [A nonsense mutation in the synaptogyrin 1 gene in a family with schizophrenia.](https://pubmed.ncbi.nlm.nih.gov/14732601) (2004) (relative citation ratio: 0.37), [Evaluation of the association of <i>UBASH3A</i> and <i>SYNGR1</i> with rheumatoid arthritis and disease activity and severity in Han Chinese.](https://pubmed.ncbi.nlm.nih.gov/29262569) (2017) (relative citation ratio: 0.28), [Association study and mutational screening of SYNGR1 as a candidate susceptibility gene for schizophrenia.](https://pubmed.ncbi.nlm.nih.gov/19641478) (2009) (relative citation ratio: 0.17), and [Identification of rare mutations of synaptogyrin 1 gene in patients with schizophrenia.](https://pubmed.ncbi.nlm.nih.gov/17049558) (2007) (relative citation ratio: 0.16). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[SYNGR1](https://www.proteinatlas.org/ENSG00000100321-SYNGR1) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in the cytosol and is expressed in various clusters, including Cluster 28 for neuronal transcription, Cluster 32 for cerebral cortex mixed function, and Cluster 66 for non-specific mitochondria in cell lines. Additionally, it is found in Cluster 1 for erythroid cells involved in oxygen transport.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599), [ZBTB7A](https://www.ncbi.nlm.nih.gov/gene/51341), [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), [AR](https://www.ncbi.nlm.nih.gov/gene/367), and [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613), each shown to be regulating in 4 experiments.



The input data indicates associations with a wide range of disease conditions, including various types of cancer such as hematologic and immune system cancers, as well as nervous system diseases like neurodegenerative disorders and Alzheimer's disease. Additionally, the data highlights conditions related to mental health, such as cognitive disorders and substance dependence, specifically alcohol dependence. Other notable associations include orofacial clefts and hypersensitivity reaction diseases, particularly type II hypersensitivity reactions.


The gene SYNGR1 has been associated with [Intelligence](https://pubmed.ncbi.nlm.nih.gov/29942086), which maps closely to the MeSH term [Intelligence](https://meshb.nlm.nih.gov/record/ui?ui=D007360) using the method [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0).


The gene is expressed in various tissues, including the ovary and brain, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the Prefrontal Cortex, CD34+ cells, CD105+ Endothelial cells, Amygdala, Cerebellum, and Cerebellum Peduncles.



The mouse ortholog gene 20972 and its human ortholog 9145 are associated with several morphological abnormalities. Both sexes exhibit abnormal morphologies in the lymph nodes, skeletal muscle, spleen, and thymus, as well as enlargement of the lymph nodes, spleen, and thymus. Additionally, abnormal and enlarged uterus morphology is observed, although these phenotypes are not present in both sexes.


The analyzed protein sequence has a GRAVY value of 0.0167 (84.75th percentile), indicating a slight hydrophobicity. The charge at pH 7.0 is -8.44 (25.43rd percentile), and the median structural flexibility is 0.994 (17.02nd percentile). The protein has a propensity for secondary structures with 25.75% helix (6.81st percentile), 37.77% sheet (77.28th percentile), and 32.62% turn (79.93rd percentile) affinities. The instability index is 34.34 (8.98th percentile), and the isoelectric point is 4.50 (1.51st percentile). The protein is 233 amino acids long (19.26th percentile) with a molecular weight of 25455.29 Da (18.7th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |