---
layout: post
title: KLRD1
date: 2025-01-17 16:55 CST
description: KLRD1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3824) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3824  | KLRD1 | ENSG00000134539 | 12p13.2 |



The gene is involved in various biological processes, including [negative regulation of T cell mediated cytotoxicity](https://amigo.geneontology.org/amigo/term/GO:0001915), [stimulatory C-type lectin receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0002223), [natural killer cell mediated immunity](https://amigo.geneontology.org/amigo/term/GO:0002228), [adaptive immune response](https://amigo.geneontology.org/amigo/term/GO:0002250), [cell surface receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007166), [regulation of natural killer cell activation](https://amigo.geneontology.org/amigo/term/GO:0032814), [negative regulation of natural killer cell mediated cytotoxicity](https://amigo.geneontology.org/amigo/term/GO:0045953), and [positive regulation of natural killer cell mediated cytotoxicity](https://amigo.geneontology.org/amigo/term/GO:0045954). It enables functions such as [transmembrane signaling receptor activity](https://amigo.geneontology.org/amigo/term/GO:0004888), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [MHC class I protein complex binding](https://amigo.geneontology.org/amigo/term/GO:0023024), [MHC class Ib protein binding, via antigen binding groove](https://amigo.geneontology.org/amigo/term/GO:0023030), [carbohydrate binding](https://amigo.geneontology.org/amigo/term/GO:0030246), and [protein antigen binding](https://amigo.geneontology.org/amigo/term/GO:1990405). The gene is located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886) and is active on the [external side of plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009897). Additionally, it is part of the [receptor complex](https://amigo.geneontology.org/amigo/term/GO:0043235).


The gene length is 15,253 base pairs (35.41% of all genes), the mature length is 2,635 base pairs (66.52% of all genes), and the primary transcript length is 13,332.5 base pairs (40.96% of all genes).


The gene KLRD1 (NCBI ID: 3824) has been mentioned in [66 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22KLRD1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1983 and the middle 50% of publications occurring between 2003 and 2013. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning KLRD1, ranked by their scientific influence, include "[Imprint of human cytomegalovirus infection on the NK cell receptor repertoire.](https://pubmed.ncbi.nlm.nih.gov/15304389)" (2004) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 11.32, "[Recognition of human histocompatibility leukocyte antigen (HLA)-E complexed with HLA class I signal sequence-derived peptides by CD94/NKG2 confers protection from natural killer cell-mediated lysis.](https://pubmed.ncbi.nlm.nih.gov/9480992)" (1998) with an RCR of 10.27, "[The CD94 and NKG2-A C-type lectins covalently assemble to form a natural killer cell inhibitory receptor for HLA class I molecules.](https://pubmed.ncbi.nlm.nih.gov/9045931)" (1997) with an RCR of 4.19, "[NK cell receptor imbalance and NK cell dysfunction in HBV infection and hepatocellular carcinoma.](https://pubmed.ncbi.nlm.nih.gov/25308752)" (2015) with an RCR of 3.55, and "[NKG2A complexed with CD94 defines a novel inhibitory natural killer cell receptor.](https://pubmed.ncbi.nlm.nih.gov/9034158)" (1997) with an RCR of 3.52. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[KLRD1](https://www.proteinatlas.org/ENSG00000134539-KLRD1) is a receptor with evidence at the protein level. Its RNA is detected in many tissues, and it is highly expressed in NK-cells, lymphoid tissue, and bone marrow, where it is involved in innate immune response. In the brain, it is associated with non-specific receptor signaling, while in bone cancer cell lines, it is linked to neuronal signaling. Single-cell expression data also indicate its role in the adaptive immune response within NK-cells.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) with 4 experiments, [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) with 3 experiments, [IRF4](https://www.ncbi.nlm.nih.gov/gene/3662) with 3 experiments, [STAT5B](https://www.ncbi.nlm.nih.gov/gene/6777) with 3 experiments, and [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 3 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.





The gene is expressed in various tissues, including the pituitary, blood, and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in CD56+ NK cells, CD4+ T cells, CD8+ T cells, whole blood, the superior cervical ganglion, the atrioventricular node, and skin.


The input data highlights several pathways involved in the immune system, including the adaptive and innate immune systems. Specifically, it mentions DAP12 interactions and signaling, as well as immunoregulatory interactions between lymphoid and non-lymphoid cells.



The analyzed protein sequence has a GRAVY value of -0.289 (59.26th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -4.401 (35.25th percentile), and the median structural flexibility is 1.000 (42.51st percentile). The protein has a propensity for helix and turn structures, with affinities of 0.296 (28.54th percentile) and 0.296 (60.9th percentile) respectively, and a higher affinity for sheet structures at 0.374 (75.41st percentile). The instability index is 54.584 (70.96th percentile), and the isoelectric point is 5.186 (11.24th percentile). The protein is 179 amino acids long (11.49th percentile) with a molecular weight of 20513.04 Da (12.02nd percentile). These calculations were performed using BioPython's sequence analysis tools, which can be found [here](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |