---
layout: post
title: DGAT1
date: 2025-01-17 16:55 CST
description: DGAT1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8694) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8694  | DGAT1 | ENSG00000185000 | 8q24.3 |



The gene enables [2-acylglycerol O-acyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0003846) and [diacylglycerol O-acyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0004144), and is involved in [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is located in the [endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0005789), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), and [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020). The gene is also involved in various processes including the [monoacylglycerol biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006640), [triglyceride metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006641), [triglyceride biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0019432), [lipid storage](https://amigo.geneontology.org/amigo/term/GO:0019915), [very-low-density lipoprotein particle assembly](https://amigo.geneontology.org/amigo/term/GO:0034379), [long-chain fatty-acyl-CoA metabolic process](https://amigo.geneontology.org/amigo/term/GO:0035336), [retinol metabolic process](https://amigo.geneontology.org/amigo/term/GO:0042572), [diacylglycerol metabolic process](https://amigo.geneontology.org/amigo/term/GO:0046339), and [fatty acid homeostasis](https://amigo.geneontology.org/amigo/term/GO:0055089). Additionally, it enables [acyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0016746), [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802), and [retinol O-fatty-acyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0050252).


The gene length is 39,509.0 base pairs (60.06% of all genes), the mature length is 3,005.5 base pairs (72.59% of all genes), and the primary transcript length is 27,495.5 base pairs (60.33% of all genes).


The gene DGAT1 (NCBI ID: 8694) has been mentioned in [118 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22DGAT1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1993 and the middle 50% of publications occurring between 2011 and 2019. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning DGAT1, ranked by their scientific influence, include "[Thematic review series: glycerolipids. DGAT enzymes and triacylglycerol biosynthesis.](https://pubmed.ncbi.nlm.nih.gov/18757836)" (2008) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 17.28, "[DGAT1-Dependent Lipid Droplet Biogenesis Protects Mitochondrial Function during Starvation-Induced Autophagy.](https://pubmed.ncbi.nlm.nih.gov/28697336)" (2017) with an RCR of 11.93, "[Targeting DGAT1 Ameliorates Glioblastoma by Increasing Fat Catabolism and Oxidative Stress.](https://pubmed.ncbi.nlm.nih.gov/32559414)" (2020) with an RCR of 8.49, "[Triglyceride Synthesis by DGAT1 Protects Adipocytes from Lipid-Induced ER Stress during Lipolysis.](https://pubmed.ncbi.nlm.nih.gov/28768178)" (2017) with an RCR of 7.67, and "[Acyl-CoA:diacylglycerol acyltransferase: molecular biology, biochemistry and biotechnology.](https://pubmed.ncbi.nlm.nih.gov/22705711)" (2012) with an RCR of 7.13. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[DGAT1](https://www.proteinatlas.org/ENSG00000185000-DGAT1) is an acyltransferase and transferase with evidence at the protein level. It is involved in disease variants and is detected in many tissues. The gene is primarily localized in the nucleoli, nucleoli rim, and endoplasmic reticulum. Its expression is notable in the liver and intestine, where it plays a role in lipid metabolism. Additionally, DGAT1 is expressed in proximal enterocytes, contributing to digestion.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313) with 7 experiments, [KMT2B](https://www.ncbi.nlm.nih.gov/gene/9757) with 6 experiments, [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908) with 5 experiments, [E2F8](https://www.ncbi.nlm.nih.gov/gene/79733) with 5 experiments, and [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent detection in multiple experiments.




The DGAT1 gene has been associated with [Height](https://pubmed.ncbi.nlm.nih.gov/30595370) in genome-wide association studies.


The gene is expressed in a variety of tissues, including the adrenal gland, colon, and small intestine, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the liver, CD33+ myeloid cells, CD14+ monocytes, colon, small intestine, adrenal gland, and testis.


The input data highlights several pathways involved in lipid metabolism, including triglyceride biosynthesis, fatty acid, triacylglycerol, and ketone body metabolism, as well as the broader categories of metabolism and metabolism of lipids and lipoproteins. Additionally, it covers glycerophospholipid biosynthesis, phospholipid metabolism, and acyl chain remodeling of DAG and TAG.



The analyzed protein sequence has a GRAVY value of 0.156 (89.45th percentile), indicating a relatively hydrophobic nature. The charge at pH 7.0 is 14.5 (88.47th percentile), and the median structural flexibility is 0.983 (5.99th percentile). The protein has a high affinity for helix (0.322, 51.93rd percentile) and sheet (0.398, 85.52nd percentile) structures, with a moderate affinity for turns (0.258, 27.68th percentile). The instability index is 43.53 (34.04th percentile), and the isoelectric point is 9.41 (89.29th percentile). The protein is 488 amino acids long (56.78th percentile) with a molecular weight of 55277.66 Da (57.66th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |