---
layout: post
title: CDK13
date: 2025-01-17 16:55 CST
description: CDK13 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8621) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8621  | CDK13 | ENSG00000065883 | 7p14.1 |



The gene is involved in [alternative mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0000380) and is part of the [cyclin K-CDK13 complex](https://amigo.geneontology.org/amigo/term/GO:0002945). It enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [protein kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004672), [cyclin-dependent protein serine/threonine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004693), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524). The gene is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615), [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [Golgi apparatus](https://amigo.geneontology.org/amigo/term/GO:0005794), and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). It is involved in [protein phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0006468), [regulation of signal transduction](https://amigo.geneontology.org/amigo/term/GO:0009966), [phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0016310), [hemopoiesis](https://amigo.geneontology.org/amigo/term/GO:0030097), [positive regulation of transcription elongation by RNA polymerase II](https://amigo.geneontology.org/amigo/term/GO:0032968), [positive regulation of transcription by RNA polymerase II](https://amigo.geneontology.org/amigo/term/GO:0045944), [regulation of cell cycle](https://amigo.geneontology.org/amigo/term/GO:0051726), and [negative regulation of stem cell differentiation](https://amigo.geneontology.org/amigo/term/GO:2000737). Additionally, it enables [RNA polymerase II CTD heptapeptide repeat kinase activity](https://amigo.geneontology.org/amigo/term/GO:0008353), [protein kinase binding](https://amigo.geneontology.org/amigo/term/GO:0019901), [cyclin binding](https://amigo.geneontology.org/amigo/term/GO:0030332), and [protein serine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0106310). The gene is also part of the [cyclin/CDK positive transcription elongation factor complex](https://amigo.geneontology.org/amigo/term/GO:0008024) and the [nuclear cyclin-dependent protein kinase holoenzyme complex](https://amigo.geneontology.org/amigo/term/GO:0019908). It is active in the [nuclear speck](https://amigo.geneontology.org/amigo/term/GO:0016607) and located in the [ficolin-1-rich granule lumen](https://amigo.geneontology.org/amigo/term/GO:1904813).


The gene length is 185,423 base pairs (91.6% of all genes), the mature length is 8,105 base pairs (97.79% of all genes), and the primary transcript length is 183,085 base pairs (95.2% of all genes).


The gene CDK13, with NCBI ID 8621, has been mentioned in [26 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CDK13%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication mentioning this gene is 1998, and the middle 50% of publications occurred between 2015 and 2021.


The top 5 publications mentioning CDK13, ranked by their relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows:

- [Therapeutic Targeting of CDK12/CDK13 in Triple-Negative Breast Cancer.](https://pubmed.ncbi.nlm.nih.gov/31668947) (2019) (relative citation ratio: 6.56)
- [CDK13 cooperates with CDK12 to control global RNA polymerase II processivity.](https://pubmed.ncbi.nlm.nih.gov/32917631) (2020) (relative citation ratio: 3.71)
- [Characterization of human cyclin-dependent kinase 12 (CDK12) and CDK13 complexes in C-terminal domain phosphorylation, gene transcription, and RNA processing.](https://pubmed.ncbi.nlm.nih.gov/25561469) (2015) (relative citation ratio: 3.55)
- [CDK13 upregulation-induced formation of the positive feedback loop among circCDK13, miR-212-5p/miR-449a and E2F5 contributes to prostate carcinogenesis.](https://pubmed.ncbi.nlm.nih.gov/33390186) (2021) (relative citation ratio: 3.14)
- [Structural and Functional Analysis of the Cdk13/Cyclin K Complex.](https://pubmed.ncbi.nlm.nih.gov/26748711) (2016) (relative citation ratio: 2.95)

Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CDK13](https://www.proteinatlas.org/ENSG00000065883-CDK13) is a serine/threonine-protein kinase with transferase activity, primarily localized in nuclear speckles, the Golgi apparatus, and the cytosol. It is involved in intellectual disability and has been detected at the protein level. CDK13 is expressed in all tissues and is associated with non-specific DNA binding and transcription clusters in various tissue types, including the brain and single cells.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313) with 7 experiments, [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613) and [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861) each with 6 experiments, and [SP1](https://www.ncbi.nlm.nih.gov/gene/6667) and [AR](https://www.ncbi.nlm.nih.gov/gene/367) each with 5 experiments.



The input data indicates associations with various disease conditions, including amyotrophic lateral sclerosis, motor neuron disease, and sensory system disease. These conditions fall under the broader categories of neurodegenerative disease, central nervous system disease, and nervous system disease, highlighting a focus on diseases of the anatomical entity.



The gene is expressed in various tissues, including the ovary and thyroid, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in cardiac myocytes, CD4+ T cells, CD8+ T cells, CD19+ B cells (neg. sel.), whole blood, lymphoma (Burkitt's, Daudi), and tonsil.




The analyzed protein sequence has a GRAVY value of -0.8271825396825429 (10.47th percentile), indicating a hydrophilic nature. It exhibits a charge of 65.61490677913173 at pH 7.0 (99.72nd percentile) and a median structural flexibility of 1.0168214285714285 (94.09th percentile). The protein's secondary structure affinities are 31.35% for helix (44.38th percentile), 25.26% for sheet (9.32nd percentile), and 36.44% for turn (92.66th percentile). The instability index is 72.55265211640251 (95.94th percentile), with an isoelectric point of 9.706182670593261 (93.83rd percentile). The protein has a length of 1512 amino acids (95.17th percentile) and a molecular weight of 164921.22680000047 Da (95.02nd percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |