---
layout: post
title: FCGRT
date: 2025-01-17 16:55 CST
description: FCGRT description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2217) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2217  | FCGRT | ENSG00000104870 | 19q13.33 |



The gene is involved in [IgG immunoglobulin transcytosis in epithelial cells mediated by FcRn immunoglobulin receptor](https://amigo.geneontology.org/amigo/term/GO:0002416) and [immune response](https://amigo.geneontology.org/amigo/term/GO:0006955). It enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [IgG binding](https://amigo.geneontology.org/amigo/term/GO:0019864), and [beta-2-microglobulin binding](https://amigo.geneontology.org/amigo/term/GO:0030881), but does not enable [peptide antigen binding](https://amigo.geneontology.org/amigo/term/GO:0042605). The gene is active in the [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615) and the [external side of plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009897), and is located in the [endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0010008).


The gene length is 52,727.0 base pairs (67.86% of all genes), the mature length is 3,612.5 base pairs, and the primary transcript length is 40,449.5 base pairs.


The gene FCGRT (NCBI ID: 2217) has been mentioned in [540 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22FCGRT%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1960 and the middle 50% of publications occurring between 1994 and 2017. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning FCGRT, ranked by their scientific influence, include "[High resolution mapping of the binding site on human IgG1 for Fc gamma RI, Fc gamma RII, Fc gamma RIII, and FcRn and design of IgG1 variants with improved binding to the Fc gamma R.](https://pubmed.ncbi.nlm.nih.gov/11096108)" (2001) (relative citation ratio: 16.55), "[Treatment of pemphigus vulgaris and foliaceus with efgartigimod, a neonatal Fc receptor inhibitor: a phase II multicentre, open-label feasibility trial.](https://pubmed.ncbi.nlm.nih.gov/34608631)" (2022) (relative citation ratio: 9.31), "[Randomized phase 2 study of FcRn antagonist efgartigimod in generalized myasthenia gravis.](https://pubmed.ncbi.nlm.nih.gov/31118245)" (2019) (relative citation ratio: 7.92), "[Phase 2 study of efgartigimod, a novel FcRn antagonist, in adult patients with primary immune thrombocytopenia.](https://pubmed.ncbi.nlm.nih.gov/31821591)" (2020) (relative citation ratio: 7.56), and "[Enhanced neonatal Fc receptor function improves protection against primate SHIV infection.](https://pubmed.ncbi.nlm.nih.gov/25119033)" (2014) (relative citation ratio: 7.51). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[FCGRT](https://www.proteinatlas.org/ENSG00000104870-FCGRT) is an IgG-binding protein and receptor with evidence at the protein level. It is a known FDA-approved drug target and is detected in all tissues. The gene is highly expressed in monocytes and neutrophils, particularly in the context of degranulation (Cluster 38). It is also associated with liver plasma proteins (Cluster 11), immune cells involved in immune response (Cluster 8), bone cancer with neuronal signaling (Cluster 12), and macrophages involved in innate immune response (Cluster 58).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SP1](https://www.ncbi.nlm.nih.gov/gene/6667) with 6 experiments, [HDAC2](https://www.ncbi.nlm.nih.gov/gene/3066) and [ZBTB7A](https://www.ncbi.nlm.nih.gov/gene/51341) each with 5 experiments, and [PGR](https://www.ncbi.nlm.nih.gov/gene/5241) and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) each with 4 experiments.


The top transcription factors (TFs) identified from the GTRD's database of TF binding sites, filtered for Perturb-seq data, include [2623](https://www.ncbi.nlm.nih.gov/gene/2623), [2118](https://www.ncbi.nlm.nih.gov/gene/2118), [8243](https://www.ncbi.nlm.nih.gov/gene/8243), and [9968](https://www.ncbi.nlm.nih.gov/gene/9968). Among these, [9968](https://www.ncbi.nlm.nih.gov/gene/9968) was shown to be regulating genes in the highest number of Perturb-seq experiments, with a total of 2 experiments. The other TFs, [2623](https://www.ncbi.nlm.nih.gov/gene/2623), [2118](https://www.ncbi.nlm.nih.gov/gene/2118), and [8243](https://www.ncbi.nlm.nih.gov/gene/8243), were each found to be regulating genes in 1 experiment.


The GWAS data indicates associations with various conditions, including cognitive disorders and diseases of mental health.


Genome-wide association studies have linked FCGRT to several traits and diseases. Notably, FCGRT has been associated with the [aspartate aminotransferase to alanine aminotransferase ratio](https://pubmed.ncbi.nlm.nih.gov/33462484), which was mapped to [Aspartic Acid](https://meshb.nlm.nih.gov/record/ui?ui=D001224) using the BioWordVec method. Additionally, FCGRT is linked to [calcium levels](https://pubmed.ncbi.nlm.nih.gov/33462484), mapped to [Calcium](https://meshb.nlm.nih.gov/record/ui?ui=D002118), and [hematocrit](https://pubmed.ncbi.nlm.nih.gov/32888494), directly matched to [Hematocrit](https://meshb.nlm.nih.gov/record/ui?ui=D006400). Other associations include [hemoglobin](https://pubmed.ncbi.nlm.nih.gov/32888494) and [hemoglobin levels](https://pubmed.ncbi.nlm.nih.gov/34226706), both mapped to [Hemoglobins](https://meshb.nlm.nih.gov/record/ui?ui=D006454). FCGRT is also associated with [non-albumin protein levels](https://pubmed.ncbi.nlm.nih.gov/33462484), mapped to [Proteins](https://meshb.nlm.nih.gov/record/ui?ui=D011506), and [red blood cell count](https://pubmed.ncbi.nlm.nih.gov/32888494), directly matched to [Erythrocyte Count](https://meshb.nlm.nih.gov/record/ui?ui=D004906). Furthermore, FCGRT is linked to [serum albumin levels](https://pubmed.ncbi.nlm.nih.gov/33462484), mapped to [Albumins](https://meshb.nlm.nih.gov/record/ui?ui=D000418), [serum alkaline phosphatase levels](https://pubmed.ncbi.nlm.nih.gov/34226706), mapped to [Phosphoric Monoester Hydrolases](https://meshb.nlm.nih.gov/record/ui?ui=D010744), and [serum total protein levels](https://pubmed.ncbi.nlm.nih.gov/33462484), mapped to [Serum](https://meshb.nlm.nih.gov/record/ui?ui=D044967).


The gene is expressed in the spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, according to the GTEx Tissue Gene Expression Profiles, the gene is ubiquitously expressed in multiple tissue samples, including the heart, liver, CD33+ myeloid cells, BDCA4+ dendritic cells, CD14+ monocytes, whole blood, and lung.




The protein sequence analyzed has a GRAVY value of -0.139 (74.8th percentile), indicating a relatively hydrophilic nature. At pH 7.0, the protein carries a charge of -3.81 (37.07th percentile). The median structural flexibility is 1.000 (43.29th percentile), with a predicted secondary structure composition of 34.0% helix (67.66th percentile), 35.3% sheet (61.6th percentile), and 32.3% turn (78.38th percentile). The instability index is 39.34 (20.34th percentile), and the isoelectric point is 6.09 (32.78th percentile). The protein has a length of 365 amino acids (40.45th percentile) and a molecular weight of 39742.86 Da (39.13th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |