---
layout: post
title: ZFYVE9
date: 2025-01-17 16:55 CST
description: ZFYVE9 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9372) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9372  | ZFYVE9 | ENSG00000157077 | 1p32.3  |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [1-phosphatidylinositol binding](https://amigo.geneontology.org/amigo/term/GO:0005545), and is involved in [endocytosis](https://amigo.geneontology.org/amigo/term/GO:0006897) and the [transforming growth factor beta receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007179). It is located in the [early endosome](https://amigo.geneontology.org/amigo/term/GO:0005769) and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and is active in the [early endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0031901). Additionally, it is part of a [protein-containing complex](https://amigo.geneontology.org/amigo/term/GO:0032991) and located in an [intracellular membrane-bounded organelle](https://amigo.geneontology.org/amigo/term/GO:0043231). The gene also enables [protein domain specific binding](https://amigo.geneontology.org/amigo/term/GO:0019904) and [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872), and is involved in [endosomal transport](https://amigo.geneontology.org/amigo/term/GO:0016197).


The gene length is 15,558 base pairs (35.92% of all genes), the mature length is 617 base pairs (6.21% of all genes), and the primary transcript length is 11,569 base pairs (37.13% of all genes).


The gene ZFYVE9 (NCBI ID: 9372) has been mentioned in [138 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ZFYVE9%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication mentioning this gene is 1971, and the middle 50% of publications occurred between 2001 and 2016.


The top 5 publications mentioning ZFYVE9, ranked by their scientific influence, include "[SARA, a FYVE domain protein that recruits Smad2 to the TGFbeta receptor.](https://pubmed.ncbi.nlm.nih.gov/9865696)" (1998) (relative citation ratio: 13.13), "[Catheter ablation vs. antiarrhythmic drug treatment of persistent atrial fibrillation: a multicentre, randomized, controlled trial (SARA study).](https://pubmed.ncbi.nlm.nih.gov/24135832)" (2014) (relative citation ratio: 9.42), "[Molecular characterization of a novel serine protease involved in activation of the complement system by mannose-binding protein.](https://pubmed.ncbi.nlm.nih.gov/8018603)" (1994) (relative citation ratio: 3.84), "[Characterization of human HtrA2, a novel serine protease involved in the mammalian cellular stress response.](https://pubmed.ncbi.nlm.nih.gov/10971580)" (2000) (relative citation ratio: 3.79), and "[Structural basis of Smad2 recognition by the Smad anchor for receptor activation.](https://pubmed.ncbi.nlm.nih.gov/10615055)" (2000) (relative citation ratio: 3.41). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ZFYVE9](https://www.proteinatlas.org/ENSG00000157077-ZFYVE9) is a gene that functions as a receptor, with evidence supporting its role at the protein level. Its RNA is detected in all tissues, and the protein is localized in vesicles and the cytosol. The gene is expressed in various clusters, including Cluster 24 for blood expression, Cluster 3 for tissue expression, Cluster 23 for brain expression, Cluster 37 for cell line expression, and Cluster 8 for single cell expression, which is associated with neuronal signaling. Notably, ZFYVE9 does not show significant prognostic value in various cancer types, including Bladder Urothelial Carcinoma, Breast Invasive Carcinoma, and others, as indicated by the unprognostic status in the provided data.


The top transcription factors (TFs) identified from the GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [CNOT3](https://www.ncbi.nlm.nih.gov/gene/4849), [TRIM28](https://www.ncbi.nlm.nih.gov/gene/10155), [BCL6](https://www.ncbi.nlm.nih.gov/gene/604), [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046), and [POU5F1](https://www.ncbi.nlm.nih.gov/gene/5460), each shown to be regulating in 4 experiments.



The GWAS data indicates associations with various disease conditions, including hematopoietic system disease, immune system disease, disease of metabolism, acquired metabolic disease, carbohydrate metabolism disease, glucose metabolism disease, diabetes mellitus, type 2 diabetes mellitus, cardiovascular system disease, heart disease, and general disease conditions.



The gene is expressed in various tissues, including the brain, cardiac myocytes, thyroid, cingulate cortex, temporal lobe, and medulla oblongata. Additionally, it is expressed in Burkitt's lymphoma (Raji) cells.


The input data highlights various pathways and processes related to TGF-beta receptor signaling and its dysregulation in cancer. Key pathways include the downregulation of TGF-beta receptor signaling, activation of SMADs by TGF-beta receptor signaling, and multiple loss-of-function mutations in SMAD2/3, SMAD4, TGFBR1, and TGFBR2. Additionally, the data covers TGFBR2 kinase domain mutants, TGFBR2 MSI frameshift mutants, and various mutations in SMAD2/3 and SMAD4 domains. The pathways also encompass signaling by the TGF-beta receptor complex, both in general and specifically in cancer, as well as broader categories of disease and signal transduction.



The analyzed protein sequence has a GRAVY value of -0.4054 (45.52nd percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -65.89 (1.35th percentile), and the median structural flexibility is 1.0069 (71.36th percentile). The protein has a propensity for helix, sheet, and turn structures of 30.39% (35.39th percentile), 31.09% (34.39th percentile), and 34.67% (88.27th percentile), respectively. The instability index is 46.15 (43.5th percentile), and the isoelectric point is 4.89 (5.79th percentile). The sequence length is 1425 amino acids (94.39th percentile), with a molecular weight of 156401.21 Da (94.33rd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |