---
layout: post
title: EIF4A1
date: 2025-01-17 16:55 CST
description: EIF4A1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1973) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1973  | EIF4A1 | ENSG00000161960 | 17p13.1 |



The gene enables [RNA cap binding](https://amigo.geneontology.org/amigo/term/GO:0000339), [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [RNA helicase activity](https://amigo.geneontology.org/amigo/term/GO:0003724), [double-stranded RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003725), [mRNA binding](https://amigo.geneontology.org/amigo/term/GO:0003729), [translation initiation factor activity](https://amigo.geneontology.org/amigo/term/GO:0003743), [helicase activity](https://amigo.geneontology.org/amigo/term/GO:0004386), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524), and [translation factor activity, RNA binding](https://amigo.geneontology.org/amigo/term/GO:0008135). It is involved in [cytoplasmic translational initiation](https://amigo.geneontology.org/amigo/term/GO:0002183) and [translational initiation](https://amigo.geneontology.org/amigo/term/GO:0006413). The gene is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062), but not in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634). It is part of the [eukaryotic translation initiation factor 4F complex](https://amigo.geneontology.org/amigo/term/GO:0016281) and enables [ATP hydrolysis activity](https://amigo.geneontology.org/amigo/term/GO:0016887).


The gene length is 12,490 base pairs (30.55th percentile), the mature length is 1,218 base pairs, and the primary transcript length is 12,446 base pairs.


The gene EIF4A1 (NCBI ID: 1973) has been mentioned in [117 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22EIF4A1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count), with the earliest publication in 1989. The middle 50% of publications occurred between 2011 and 2020.


The top five publications mentioning EIF4A1, ranked by their scientific influence, include "[RNA G-quadruplexes cause eIF4A-dependent oncogene translation in cancer.](https://pubmed.ncbi.nlm.nih.gov/25079319)" (2014) with a relative citation ratio of 11.2, "[Broad-spectrum antiviral activity of the eIF4A inhibitor silvestrol against corona- and picornaviruses.](https://pubmed.ncbi.nlm.nih.gov/29258862)" (2018) with a relative citation ratio of 6.71, "[Dominant negative mutants of mammalian translation initiation factor eIF-4A define a critical role for eIF-4F in cap-dependent and cap-independent initiation of translation.](https://pubmed.ncbi.nlm.nih.gov/8131750)" (1994) with a relative citation ratio of 6.2, "[Rocaglates convert DEAD-box protein eIF4A into a sequence-selective translational repressor.](https://pubmed.ncbi.nlm.nih.gov/27309803)" (2016) with a relative citation ratio of 5.73, and "[Functional characterization of IRESes by an inhibitor of the RNA helicase eIF4A.](https://pubmed.ncbi.nlm.nih.gov/16532013)" (2006) with a relative citation ratio of 5.1. The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[EIF4A1](https://www.proteinatlas.org/ENSG00000161960-EIF4A1) is a multifunctional protein with roles as a helicase, hydrolase, initiation factor, and RNA-binding protein. It is primarily located in the cytosol and is detected in all tissues. EIF4A1 is expressed in various clusters, including Cluster 26 in monocytes, Cluster 75 in epithelium, Cluster 21 in the brain, Cluster 42 in cell lines, and Cluster 49 in single cells. The protein's expression is associated with plasma membrane proteins, extracellular exosomes, metabolism, and transcription, although its specific function in some cell line clusters remains unknown. Evidence for EIF4A1 is established at the protein level.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [KDM1A](https://www.ncbi.nlm.nih.gov/gene/23028) and [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), each with 6 experiments. Additionally, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025), and [STAT1](https://www.ncbi.nlm.nih.gov/gene/6772) were each found to be regulating in 5 experiments.





The gene is expressed in various tissues, including the fallopian tube and skin. Additionally, it is expressed in several immune cell types such as CD34+, CD33+ myeloid cells, BDCA4+ dendritic cells, CD4+ T cells, CD8+ T cells, and promyelocytic leukemia HL-60 cells.


The input data highlights various pathways involving proteins, with a significant focus on translation and immune system processes. Key pathways include metabolism of proteins, gene expression, and several stages of translation such as cap-dependent translation initiation, eukaryotic translation initiation, and translation initiation complex formation. Additionally, the data covers antiviral mechanisms, including those mediated by ISG15 and interferon signaling, as well as broader immune system signaling pathways. The regulation of mRNA is also prominent, with pathways like 3'-UTR-mediated translational regulation, deadenylation of mRNA, and deadenylation-dependent mRNA decay being noted.



The analyzed protein sequence has a GRAVY value of -0.2613 (62.55th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -9.02 (24.24th percentile), and the median structural flexibility is 0.9984 (34.06th percentile). The protein has a length of 406 amino acids (46.11th percentile) and a molecular weight of 46153.42 Da (47.14th percentile). The secondary structure affinities are 31.28% for helix (43.74th percentile), 37.44% for sheet (75.45th percentile), and 25.62% for turn (26.05th percentile). The instability index is 49.47 (55.47th percentile), and the isoelectric point is 5.32 (14.02nd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |