---
layout: post
title: C1QB
date: 2025-01-17 16:55 CST
description: C1QB description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/713) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 713  | C1QB | ENSG00000173369 | 1p36.12 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576). It is part of the [collagen trimer](https://amigo.geneontology.org/amigo/term/GO:0005581) and the [complement component C1 complex](https://amigo.geneontology.org/amigo/term/GO:0005602), specifically the [complement component C1q complex](https://amigo.geneontology.org/amigo/term/GO:0062167). The gene is involved in [complement activation](https://amigo.geneontology.org/amigo/term/GO:0006956), particularly the [complement activation, classical pathway](https://amigo.geneontology.org/amigo/term/GO:0006958), and plays a role in the [innate immune response](https://amigo.geneontology.org/amigo/term/GO:0045087) and [synapse pruning](https://amigo.geneontology.org/amigo/term/GO:0098883). Additionally, it is located in the [synapse](https://amigo.geneontology.org/amigo/term/GO:0045202), [collagen-containing extracellular matrix](https://amigo.geneontology.org/amigo/term/GO:0062023), [blood microparticle](https://amigo.geneontology.org/amigo/term/GO:0072562), and [postsynapse](https://amigo.geneontology.org/amigo/term/GO:0098794).


The gene length is 38,095 base pairs (59.11% of all genes), the mature length is 1,672 base pairs, and the primary transcript length is 10,652 base pairs (34.97% of all genes).


The gene C1QB (NCBI ID: 713) has been mentioned in [11 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22C1QB%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning this gene was in 1988, and the middle 50% of publications occurred between 2011 and 2020.


The top 5 publications mentioning C1QB, ranked by their scientific influence as measured by the [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541), include studies that highlight the gene's role in various diseases. The most influential study, "[Complement C1q (C1qA, C1qB, and C1qC) May Be a Potential Prognostic Factor and an Index of Tumor Microenvironment Remodeling in Osteosarcoma.](https://pubmed.ncbi.nlm.nih.gov/34079754)" (2021), has an RCR of 4.37. Another notable study, "[Integrated genomic and transcriptomic analysis reveals unique characteristics of hepatic metastases and pro-metastatic role of complement C1q in pancreatic ductal adenocarcinoma.](https://pubmed.ncbi.nlm.nih.gov/33397441)" (2021), has an RCR of 1.78. Additionally, the study "[Identification of <i>TYROBP</i> and <i>C1QB</i> as Two Novel Key Genes With Prognostic Value in Gastric Cancer by Network Analysis.](https://pubmed.ncbi.nlm.nih.gov/33014868)" (2020) has an RCR of 1.62. Older studies such as "[A homozygous point mutation results in a stop codon in the C1q B-chain of a C1q-deficient individual.](https://pubmed.ncbi.nlm.nih.gov/2894352)" (1988) and "[C1qB and clusterin mRNA increase in association with neurodegeneration in sporadic amyotrophic lateral sclerosis.](https://pubmed.ncbi.nlm.nih.gov/10471215)" (1999) have RCRs of 1.15 and 1.07, respectively. The citation counts are derived from [iCite](https://icite.od.nih.gov).


[C1QB](https://www.proteinatlas.org/ENSG00000173369-C1QB) is a gene with evidence at the transcript level, detected in all RNA tissue distributions. It is predominantly expressed in monocytes, contributing to the innate immune response (Cluster 40). In tissue expression, it is clustered in lymphoid tissue, also associated with immune response (Cluster 70). In the brain, C1QB is expressed in macrophages and microglia, again linked to immune response (Cluster 20). Additionally, it is found in myeloma cell lines (Cluster 51) and single-cell expression clusters of macrophages involved in innate immune response (Cluster 58).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [RELA](https://www.ncbi.nlm.nih.gov/gene/5970) with 4 experiments, followed by [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), [STAT3](https://www.ncbi.nlm.nih.gov/gene/6774), and [SPI1](https://www.ncbi.nlm.nih.gov/gene/6688), each with 3 experiments.



The GWAS data indicates associations with various disease conditions, including cancer, diseases of cellular proliferation, immune system diseases, connective tissue diseases, musculoskeletal system diseases, collagen diseases, rheumatic diseases, scleroderma, systemic scleroderma, hypersensitivity reaction diseases, and hypersensitivity reaction type II diseases.


The gene C1QB has been associated with [Sphingomyelin (d38:2) levels](https://pubmed.ncbi.nlm.nih.gov/36193934), which maps closely to the MeSH term [Sphingomyelins](https://meshb.nlm.nih.gov/record/ui?ui=D013109) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method.


The gene is expressed in various tissues, including the lung and spleen, according to the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles indicate that the gene is ubiquitously expressed in multiple tissue samples, such as the heart, liver, CD105+ endothelial cells, lung, placenta, ovary, and lymph node.


The input data highlights several pathways within the immune system, particularly focusing on the innate immune system. Key processes include classical antibody-mediated complement activation, the creation of C4 and C2 activators, initial triggering of complement, and the overall complement cascade.



The protein sequence analyzed has a GRAVY value of -0.257 (63.09th percentile), indicating a hydrophilic nature. It exhibits a charge of 3.74 at pH 7.0 (63.66th percentile) and a median structural flexibility of 1.007 (71.36th percentile). The protein's secondary structure is predicted to be 27.67% helix (15.2nd percentile), 31.62% sheet (37.32nd percentile), and 39.13% turn (96.65th percentile). The instability index is 17.07 (0.36th percentile), with an isoelectric point of 8.83 (74.67th percentile). The protein has a length of 253 amino acids (22.06th percentile) and a molecular weight of 26721.47 Da (20.29th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |