---
layout: post
title: CHN2
date: 2024-12-08 23:57 CST
description: CHN2 description
tags: [cooccuring-genes]
categories: longest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1124) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1124  | CHN2 | ENSG00000106069 | 7p14.3 |



The gene length is 829,729 base pairs (99.38% of all genes), the mature length is 5,167.5 base pairs, and the primary transcript length is 725,468 base pairs (99.7% of all genes).


The gene CHN2, with NCBI ID 1124, has been the subject of [42 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CHN2%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication dating back to 1959. The middle 50% of these publications were published between 2001 and 2019. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning CHN2, ranked by their scientific influence as measured by the [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541), include "[The separated enantiomers of 2'-deoxy-3'-thiacytidine (BCH 189) both inhibit human immunodeficiency virus replication in vitro.](https://pubmed.ncbi.nlm.nih.gov/1590690)" (1992) with an RCR of 6.35, "[Rapid interaction of cathepsin L by Z-Phe-PheCHN12 and Z-Phe-AlaCHN2.](https://pubmed.ncbi.nlm.nih.gov/7306089)" (1981) with an RCR of 3.64, "[BCH, an inhibitor of system L amino acid transporters, induces apoptosis in cancer cells.](https://pubmed.ncbi.nlm.nih.gov/18520037)" (2008) with an RCR of 2.54, "[Catalytic Asymmetric Homologation of 4-Substituted Cyclohexanones with CF<sub>3</sub> CHN<sub>2</sub> : Enantioselective Synthesis of Î±-Trifluoromethyl Cycloheptanones.](https://pubmed.ncbi.nlm.nih.gov/34939732)" (2022) with an RCR of 1.28, and "[The crystal structure of human dipeptidyl peptidase I (cathepsin C) in complex with the inhibitor Gly-Phe-CHN2.](https://pubmed.ncbi.nlm.nih.gov/17020538)" (2007) with an RCR of 1.27. The RCR values measure the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) and [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227), each with 5 experiments. Additionally, [HDAC2](https://www.ncbi.nlm.nih.gov/gene/3066), [ZBTB7A](https://www.ncbi.nlm.nih.gov/gene/51341), and [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313) were each found to be regulating in 4 experiments.











The protein sequence analyzed has a GRAVY value of -0.463 (39.0th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of 0.214 (51.1th percentile). The median structural flexibility is 0.999 (36.9th percentile), with a predicted secondary structure composition of 31.2% helix (43.04th percentile), 36.3% sheet (68.5th percentile), and 25.2% turn (23.09th percentile). The instability index is 36.5 (13.01th percentile), and the isoelectric point is 7.045 (50.9th percentile). The protein has a length of 468 amino acids (54.21th percentile) and a molecular weight of 53922.96 Da (56.01th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |