---
layout: post
title: CSTF3
date: 2025-01-17 16:55 CST
description: CSTF3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1479) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1479  | CSTF3 | ENSG00000176102 | 11p13 |



The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and [mRNA binding](https://amigo.geneontology.org/amigo/term/GO:0003729), and is involved in [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is active in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654). Additionally, it is part of the [mRNA cleavage stimulating factor complex](https://amigo.geneontology.org/amigo/term/GO:0005848) and involved in [RNA 3'-end processing](https://amigo.geneontology.org/amigo/term/GO:0031123) and the [co-transcriptional mRNA 3'-end processing, cleavage and polyadenylation pathway](https://amigo.geneontology.org/amigo/term/GO:0180010).


The gene length is 22,808 base pairs (45.21% of all genes), the mature length is 2,825 base pairs (69.81% of all genes), and the primary transcript length is 22,426 base pairs (54.79% of all genes).


The gene CSTF3 (NCBI ID: 1479) has been mentioned in [6 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CSTF3%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1999 and the middle 50% of publications occurring between 2003 and 2008. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning CSTF3, ranked by their scientific influence, include "[The tumor suppressor Cdc73 functionally associates with CPSF and CstF 3' mRNA processing factors.](https://pubmed.ncbi.nlm.nih.gov/19136632)" (2009) (relative citation ratio: 1.85), "[Interactions of CstF-64, CstF-77, and symplekin: implications on localisation and function.](https://pubmed.ncbi.nlm.nih.gov/21119002)" (2011) (relative citation ratio: 1.01), "[The structure of the CstF-77 homodimer provides insights into CstF assembly.](https://pubmed.ncbi.nlm.nih.gov/17584787)" (2007) (relative citation ratio: 0.69), "[An intronic polyadenylation site in human and mouse CstF-77 genes suggests an evolutionarily conserved regulatory mechanism.](https://pubmed.ncbi.nlm.nih.gov/16316725)" (2006) (relative citation ratio: 0.54), and "[Chimeric human CstF-77/Drosophila Suppressor of forked proteins rescue suppressor of forked mutant lethality and mRNA 3' end processing in Drosophila.](https://pubmed.ncbi.nlm.nih.gov/12149458)" (2002) (relative citation ratio: 0.23). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CSTF3](https://www.proteinatlas.org/ENSG00000176102-CSTF3) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily located in the nucleoplasm and is associated with various expression clusters, including Cluster 12 for blood expression related to mitochondrial translation, Cluster 50 for tissue expression in the epididymis involved in male reproductive secretion, Cluster 31 for brain expression related to transcription, Cluster 30 for cell line expression related to gene expression regulation, and Cluster 39 for single cell expression related to basic cellular processes.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [KDM1A](https://www.ncbi.nlm.nih.gov/gene/23028), [HDAC1](https://www.ncbi.nlm.nih.gov/gene/3065), [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), and [GATA3](https://www.ncbi.nlm.nih.gov/gene/2625), each with 5 experiments. Additionally, [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599) was found to be regulating in 4 experiments.



The GWAS data indicates associations with various disease conditions, including general diseases, diseases of anatomical entities, nervous system diseases, and sensory system diseases.



The gene is expressed in various tissues and cell types, including pituitary, 721_B_lymphoblasts, Leukemia chronic myelogenous K-562 cells, bronchial epithelial cells, appendix, superior cervical ganglion, and trigeminal ganglion.


The input data highlights various processes involved in gene expression and RNA processing. These include RNA polymerase II transcription, transcription termination, and post-elongation processing of both intron-containing and intronless pre-mRNAs. Specific processes mentioned are mRNA 3'-end processing, mRNA splicing, and the major pathway of mRNA splicing. Additionally, the data covers the cleavage of growing transcripts in the termination region and the processing of capped intronless and intron-containing pre-mRNAs.


The mouse ortholog gene 228410, which corresponds to the human ortholog gene 1479, is associated with several abnormal phenotypes affecting the eye. These include abnormal lens morphology, abnormal retina blood vessel morphology, abnormal retina morphology, abnormal retina vasculature morphology, cataract, and persistence of the hyaloid vascular system. All these phenotypes are observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.481 (36.96th percentile), indicating a hydrophilic nature. It exhibits a charge of 4.5 at pH 7.0 (66.3rd percentile) and a median structural flexibility of 1.004 (58.31st percentile). The protein's secondary structure is predicted to be 37.24% helix (86.15th percentile), 34.73% sheet (56.95th percentile), and 23.71% turn (13.44th percentile). The instability index is 41.51 (26.97th percentile), with an isoelectric point of 8.26 (62.16th percentile). The protein has a length of 717 amino acids (76.37th percentile) and a molecular weight of 82920.56 Da (77.75th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |