---
layout: post
title: TAPBP
date: 2024-12-08 23:57 CST
description: TAPBP description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6892) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6892  | TAPBP | ENSG00000231925 | 6p21.32 |



The gene length is 134,248 base pairs (87.17% of all genes), the mature length is 5,875 base pairs (93.75% of all genes), and the primary transcript length is 100,836 base pairs (88.32% of all genes).


The gene TAPBP, with NCBI ID 6892, has been the subject of [153 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22TAPBP%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication dating back to 1966. The middle 50% of these publications were published between 2002 and 2013. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning TAPBP, ranked by their scientific influence, include "[Roles for calreticulin and a novel glycoprotein, tapasin, in the interaction of MHC class I molecules with TAP.](https://pubmed.ncbi.nlm.nih.gov/8769474)" (1996) (relative citation ratio: 10.9), "[A critical role for tapasin in the assembly and function of multimeric MHC class I-TAP complexes.](https://pubmed.ncbi.nlm.nih.gov/9271576)" (1997) (relative citation ratio: 7.4), "[Insights into MHC class I peptide loading from the structure of the tapasin-ERp57 thiol oxidoreductase heterodimer.](https://pubmed.ncbi.nlm.nih.gov/19119025)" (2009) (relative citation ratio: 4.64), "[Optimization of the MHC class I peptide cargo is dependent on tapasin.](https://pubmed.ncbi.nlm.nih.gov/11970875)" (2002) (relative citation ratio: 4.62), and "[TAP- and tapasin-dependent HLA-E surface expression correlates with the binding of an MHC class I leader peptide.](https://pubmed.ncbi.nlm.nih.gov/9427624)" (1998) (relative citation ratio: 4.1). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [MYC](https://www.ncbi.nlm.nih.gov/gene/4609), [AR](https://www.ncbi.nlm.nih.gov/gene/367), and [INTS11](https://www.ncbi.nlm.nih.gov/gene/54973), each with 5 experiments. Additionally, [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) and [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908) were found to be regulating in 4 experiments each.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [CTCF](https://www.ncbi.nlm.nih.gov/gene/6829) with 3 experiments, [ZNF274](https://www.ncbi.nlm.nih.gov/gene/23028) with 2 experiments, [ZNF143](https://www.ncbi.nlm.nih.gov/gene/8243) with 2 experiments, and [ZNF384](https://www.ncbi.nlm.nih.gov/gene/9968) with 2 experiments. These TFs were found to regulate gene expression in multiple Perturb-seq experiments, highlighting their significant roles in transcriptional regulation.








The analyzed protein sequence has a GRAVY value of -0.0246 (82.83rd percentile), indicating a slightly hydrophilic nature. The charge at pH 7.0 is -2.36 (41.77th percentile), and the median structural flexibility is 0.996 (23.19th percentile). The protein has a notable affinity for helix (0.366, 83.58th percentile) and sheet (0.330, 45.71st percentile) structures, with a turn affinity of 0.319 (76.31st percentile). The instability index is 56.31 (75.48th percentile), suggesting moderate stability. The isoelectric point is 6.50 (42.61st percentile), and the sequence length is 448 amino acids (51.65th percentile), with a molecular weight of 47570.10 Da (49.02nd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |