---
layout: post
title: FUT6
date: 2025-01-17 16:55 CST
description: FUT6 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2528) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2528  | FUT6 | ENSG00000156413 | 19p13.3 |



The gene is located in the [Golgi membrane](https://amigo.geneontology.org/amigo/term/GO:0000139), [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [Golgi apparatus](https://amigo.geneontology.org/amigo/term/GO:0005794), [Golgi cisterna membrane](https://amigo.geneontology.org/amigo/term/GO:0032580), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). It is involved in various processes including [protein glycosylation](https://amigo.geneontology.org/amigo/term/GO:0006486), [protein N-linked glycosylation](https://amigo.geneontology.org/amigo/term/GO:0006487), [protein O-linked glycosylation](https://amigo.geneontology.org/amigo/term/GO:0006493), [ceramide metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006672), [glycosphingolipid biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006688), [oocyte axis specification](https://amigo.geneontology.org/amigo/term/GO:0007309), [oligosaccharide biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0009312), [N-glycan fucosylation](https://amigo.geneontology.org/amigo/term/GO:0036071), and [L-fucose catabolic process](https://amigo.geneontology.org/amigo/term/GO:0042355). Additionally, it enables [fucosyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0008417), [4-galactosyl-N-acetylglucosaminide 3-alpha-L-fucosyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0017083), and [alpha-(1->3)-fucosyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0046920). It is also part of the [methylosome](https://amigo.geneontology.org/amigo/term/GO:0034709).


The gene length is 14,964 base pairs (35.04% of all genes), the mature length is 3,237.5 base pairs (75.71% of all genes), and the primary transcript length is 10,840 base pairs (35.42% of all genes).


The gene FUT6 (NCBI ID: 2528) has been mentioned in [28 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22FUT6%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1993, and the middle 50% of publications range from 1996 to 2016.


The top 5 publications mentioning FUT6, ranked by their relative citation ratio (RCR), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows. The most influential publication is "[miR-125a-3p/FUT5-FUT6 axis mediates colorectal cancer cell proliferation, migration, invasion and pathological angiogenesis via PI3K-Akt pathway.](https://pubmed.ncbi.nlm.nih.gov/28771224)" (2017) with an RCR of 3.87. Following this, "[HOTAIR/miR-326/FUT6 axis facilitates colorectal cancer progression through regulating fucosylation of CD44 via PI3K/AKT/mTOR pathway.](https://pubmed.ncbi.nlm.nih.gov/30742932)" (2019) has an RCR of 3.11. Additionally, "[Molecular basis for plasma alpha(1,3)-fucosyltransferase gene deficiency (FUT6).](https://pubmed.ncbi.nlm.nih.gov/8175676)" (1994) has an RCR of 1.89. "[MicroRNA-106b targets FUT6 to promote cell migration, invasion, and proliferation in human breast cancer.](https://pubmed.ncbi.nlm.nih.gov/27519168)" (2016) follows with an RCR of 1.61. Lastly, "[Relative positions of two clusters of human alpha-L-fucosyltransferases in 19q (FUT1-FUT2) and 19p (FUT6-FUT3-FUT5) within the microsatellite genetic map of chromosome 19.](https://pubmed.ncbi.nlm.nih.gov/7656588)" (1995) has an RCR of 1.58. The citation count is sourced from [iCite](https://icite.od.nih.gov).


[FUT6](https://www.proteinatlas.org/ENSG00000156413-FUT6) is a gene encoding a protein with glycosyltransferase and transferase molecular functions, supported by evidence at the protein level. The gene product is localized to the Golgi apparatus and is detected in many tissues. FUT6 is expressed in epithelium and extracellular exosomes (Cluster 75), with non-specific mixed function in the brain (Cluster 48). In cell lines, it is associated with stomach and colon cancer, specifically in digestion (Cluster 57). Single-cell expression analysis places FUT6 in proximal enterocytes involved in digestion (Cluster 22).


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861), and [SPI1](https://www.ncbi.nlm.nih.gov/gene/6688), each shown to be regulating in 3 experiments. Additionally, [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) and [PGR](https://www.ncbi.nlm.nih.gov/gene/5241) were found to be regulating in 2 experiments each.



The GWAS data indicates associations with a wide range of disease conditions, including various types of cancer such as gastrointestinal system cancer, liver cancer, pancreatic cancer, and esophageal cancer. Additionally, the data highlights links to immune system diseases, metabolic disorders, and specific conditions like diabetes mellitus and vitamin B12 deficiency. The findings also encompass cardiovascular system diseases and inherited metabolic disorders, suggesting a broad impact on multiple organ systems and cellular processes.


FUT6 has been associated with several traits and diseases through genome-wide association studies. Notably, it has been linked to [Aspartate aminotransferase levels](https://pubmed.ncbi.nlm.nih.gov/34594039), which maps closely to the MeSH term [Transaminases](https://meshb.nlm.nih.gov/record/ui?ui=D000637) using the BioWordVec method. Additionally, FUT6 is associated with [Lactoperoxidase levels](https://pubmed.ncbi.nlm.nih.gov/34814699), corresponding to the MeSH term [Lactoperoxidase](https://meshb.nlm.nih.gov/record/ui?ui=D007784) through embedding match.


The gene is expressed in the salivary gland and kidney according to BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles indicate expression in the liver, trachea, colon, kidney, superior cervical ganglion, skeletal muscle, and uterus corpus.




The analyzed protein sequence has a GRAVY value of -0.425 (43.22 percentile), indicating a hydrophilic nature. It exhibits a charge of 7.32 at pH 7.0 (75.22 percentile) and a median structural flexibility of 0.997 (30.0 percentile). The protein's secondary structure is predicted to be 27.6% helix (14.81 percentile), 36.5% sheet (69.66 percentile), and 29.0% turn (55.69 percentile). The instability index is 50.92 (60.2 percentile), with an isoelectric point of 8.92 (77.1 percentile). The protein has a length of 359 amino acids (39.48 percentile) and a molecular weight of 41859.33 Da (41.86 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |