---
layout: post
title: PTP4A2
date: 2025-01-17 16:55 CST
description: PTP4A2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8073) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8073  | PTP4A2 | ENSG00000184007 | 1p35.2 |



The gene enables [protein tyrosine phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0004725) and [prenylated protein tyrosine phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0004727), and is involved in [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is active in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), and is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [early endosome](https://amigo.geneontology.org/amigo/term/GO:0005769), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). Additionally, it is involved in [dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0016311).


The gene length is 687,786 base pairs (99.11% of all genes), the mature length is 5,506 base pairs (92.42% of all genes), and the primary transcript length is 192,019 base pairs (95.51% of all genes).


The gene PTP4A2, with NCBI ID 8073, has been mentioned in [30 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PTP4A2%22) (note: the count in the hyperlink may be different since the count is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication mentioning this gene is 1969, and the middle 50% of publications occurred between 2001 and 2017.


The top 5 publications mentioning PTP4A2, ranked by their relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows: [The protein tyrosine phosphatase PRL-2 interacts with the magnesium transporter CNNM3 to promote oncogenesis.](https://pubmed.ncbi.nlm.nih.gov/24632616) (2015) (relative citation ratio: 2.84), [MiR-29c mediates epithelial-to-mesenchymal transition in human colorectal carcinoma metastasis via PTP4A and GNA13 regulation of Î²-catenin signaling.](https://pubmed.ncbi.nlm.nih.gov/25193986) (2014) (relative citation ratio: 2.8), [Gas--liquid chromatographic retention indices of 296 non-drug substances on SE-30 or OV-1 likely to be encountered in toxicological analyses.](https://pubmed.ncbi.nlm.nih.gov/7380921) (1980) (relative citation ratio: 2.71), [Long non-coding RNA USP30-AS1 aggravates the malignant progression of cervical cancer by sequestering microRNA-299-3p and thereby overexpressing PTP4A1.](https://pubmed.ncbi.nlm.nih.gov/33986866) (2021) (relative citation ratio: 2.7), and [Expression and prognostic impact of the protein tyrosine phosphatases PRL-1, PRL-2, and PRL-3 in breast cancer.](https://pubmed.ncbi.nlm.nih.gov/16832410) (2006) (relative citation ratio: 1.76). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PTP4A2](https://www.proteinatlas.org/ENSG00000184007-PTP4A2) is a gene encoding a protein with hydrolase and protein phosphatase molecular functions. Evidence for this gene is available at the protein level. Its RNA is detected in all tissues, with notable expression in oligodendrocytes involved in myelination, white matter related to signal transduction, leukemia T-cell receptor, and monocytes involved in innate immune response. The gene is also associated with non-specific vesicular transport in blood.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930) with 6 experiments, followed by [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), [JUN](https://www.ncbi.nlm.nih.gov/gene/3725), [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861), and [KLF5](https://www.ncbi.nlm.nih.gov/gene/688), each with 5 experiments.


The transcription factor [8861](https://www.ncbi.nlm.nih.gov/gene/8861) was found to be regulating genes in 3 Perturb-seq experiments, making it one of the top transcription factors identified in the study.


The GWAS data indicates associations with various disease conditions, including cardiovascular system disease, disease of mental health, substance-related disorders such as substance dependence and alcohol dependence, heart disease, and substance abuse, including alcohol abuse.



The gene is expressed in various tissues, including muscle, adipose tissue, bladder, and brain, according to the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles indicate expression in CD56+ NK cells, whole blood, thymus, and leukemia lymphoblastic cells (MOLT-4).



The mouse ortholog gene 19244 and its human ortholog 8073 are associated with several phenotypic abnormalities. Both sexes exhibit abnormal skin and stomach morphology, decreased body length, mean corpuscular volume, and prepulse inhibition, as well as enlarged lymph nodes, increased grip strength, short tibia, and thick skin. Additionally, males exhibit small testis.


The analyzed protein sequence has a GRAVY value of -0.3497 (52.25th percentile), indicating a hydrophilic nature. It exhibits a charge of 3.80 at pH 7.0 (63.95th percentile) and a median structural flexibility of 0.9995 (39.1st percentile). The protein's secondary structure is predicted to be 32.93% helix (58.85th percentile), 35.93% sheet (65.72nd percentile), and 23.95% turn (14.75th percentile). The instability index is 42.56 (30.58th percentile), with an isoelectric point of 8.67 (70.34th percentile). The protein has a length of 167 amino acids (10.18th percentile) and a molecular weight of 19126.99 Da (10.55th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |