---
layout: post
title: IL15RA
date: 2025-01-17 16:55 CST
description: IL15RA description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3601) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3601  | IL15RA | ENSG00000134470 | 10p15.1 |



The gene is located in the [Golgi membrane](https://amigo.geneontology.org/amigo/term/GO:0000139), [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [endosome](https://amigo.geneontology.org/amigo/term/GO:0005768), [endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0005789), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986), [cytoplasmic vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030659), and [nuclear membrane](https://amigo.geneontology.org/amigo/term/GO:0031965). It is involved in [natural killer cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0001779), [receptor signaling pathway via JAK-STAT](https://amigo.geneontology.org/amigo/term/GO:0007259), [negative regulation of neuron projection development](https://amigo.geneontology.org/amigo/term/GO:0010977), [response to nutrient levels](https://amigo.geneontology.org/amigo/term/GO:0031667), [positive regulation of natural killer cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0032825), [interleukin-15-mediated signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0035723), and [positive regulation of phagocytosis](https://amigo.geneontology.org/amigo/term/GO:0050766). The gene enables [cytokine receptor activity](https://amigo.geneontology.org/amigo/term/GO:0004896), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [protein kinase binding](https://amigo.geneontology.org/amigo/term/GO:0019901), and [interleukin-15 receptor activity](https://amigo.geneontology.org/amigo/term/GO:0042010).


The gene length is 8,051 base pairs (21.47% of all genes), the mature length is 1,128 base pairs, and the primary transcript length is 7,721 base pairs.


The gene IL15RA, with NCBI ID 3601, has been mentioned in [81 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22IL15RA%22) (note: the count in the hyperlink may be different since the count is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication mentioning this gene is 1998, and the middle 50% of publications occurred between 2008 and 2018.


The top 5 publications mentioning IL15RA, ranked by their scientific influence, include "[IL-15Ralpha recycles and presents IL-15 In trans to neighboring cells.](https://pubmed.ncbi.nlm.nih.gov/12433361)" (2002) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 11.04, "[Exercise-induced engagement of the IL-15/IL-15Rα axis promotes anti-tumor immunity in pancreatic cancer.](https://pubmed.ncbi.nlm.nih.gov/35660135)" (2022) with an RCR of 9.76, "[An Oncolytic Virus Expressing IL15/IL15Rα Combined with Off-the-Shelf EGFR-CAR NK Cells Targets Glioblastoma.](https://pubmed.ncbi.nlm.nih.gov/34006525)" (2021) with an RCR of 7.22, "[Converting IL-15 to a superagonist by binding to soluble IL-15R{alpha}.](https://pubmed.ncbi.nlm.nih.gov/16757567)" (2006) with an RCR of 5.68, and "[IL-15Ralpha chaperones IL-15 to stable dendritic cell membrane complexes that activate NK cells via trans presentation.](https://pubmed.ncbi.nlm.nih.gov/18458113)" (2008) with an RCR of 4.2. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[IL15RA](https://www.proteinatlas.org/ENSG00000134470-IL15RA) is a receptor with evidence at the protein level. It is detected in many tissues and is localized in vesicles, the plasma membrane, and the cytosol. The gene is part of Cluster 42 in blood expression, which is associated with non-specific vesicular transport, and Cluster 57 in tissue expression, which is linked to lung function. In the brain, it is part of Cluster 47, associated with endothelial cells and vasculature. In cell lines, it is part of Cluster 31, related to the antiviral immune response, and in single cells, it is part of Cluster 53, involved in ECM organization.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313) with 7 experiments, [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025) with 6 experiments, [MAX](https://www.ncbi.nlm.nih.gov/gene/4149) with 6 experiments, [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227) with 6 experiments, and [EP300](https://www.ncbi.nlm.nih.gov/gene/2033) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.




The IL15RA gene has been associated with various diseases and traits through genome-wide association studies. Notably, it has been linked to [Interleukin-15 receptor subunit alpha levels](https://pubmed.ncbi.nlm.nih.gov/37563310), which maps closely to the MeSH term [Interleukin-15](https://meshb.nlm.nih.gov/record/ui?ui=D019409) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method. Additionally, IL15RA has been implicated in [Neuroblastoma (pediatric)](https://pubmed.ncbi.nlm.nih.gov/32599975), which aligns with the MeSH term [Neuroblastoma](https://meshb.nlm.nih.gov/record/ui?ui=D009447) through embedding match.


The gene is expressed in various tissues, including the cervix uteri and lung, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Lymphoma (Burkitt's), 721 B lymphoblasts, Leukemia (chronic myelogenous K-562), CD105+ endothelial cells, CD71+ early erythroid cells, colorectal adenocarcinoma, lung, and uterus corpus.




The analyzed protein sequence has a GRAVY value of -0.267 (61.85th percentile), indicating a hydrophilic nature. It exhibits a charge of 4.23 at pH 7.0 (65.44th percentile) and a median structural flexibility of 1.004 (61.6th percentile). The protein's secondary structure is predicted to be 29.96% helix (31.66th percentile), 32.58% sheet (42.93rd percentile), and 33.71% turn (84.85th percentile). The instability index is 52.06 (63.79th percentile), with an isoelectric point of 8.64 (69.78th percentile). The protein has a length of 267 amino acids (24.02nd percentile) and a molecular weight of 28232.7 Da (22.15th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |