---
layout: post
title: EFS
date: 2025-01-17 16:55 CST
description: EFS description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10278) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10278  | EFS | ENSG00000100842 | 14q11.2 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [SH3 domain binding](https://amigo.geneontology.org/amigo/term/GO:0017124), and is involved in [cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0007155), [transmembrane receptor protein tyrosine kinase signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007169), [cell migration](https://amigo.geneontology.org/amigo/term/GO:0016477), and [intracellular signal transduction](https://amigo.geneontology.org/amigo/term/GO:0035556). It is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737) and [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925), and colocalizes with the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). Additionally, it enables [protein domain specific binding](https://amigo.geneontology.org/amigo/term/GO:0019904).


The gene length is 8,852.0 base pairs (23.23% of all genes), the mature length is 1,049.5 base pairs, and the primary transcript length is 7,233.5 base pairs.


The gene EFS (NCBI ID: 10278) has been mentioned in [285 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22EFS%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1890, and the middle 50% of publications range from 1992 to 2017.


The top 5 publications mentioning EFS, ranked by their scientific influence, include "[Immunity to dengue virus: a tale of original antigenic sin and tropical cytokine storms.](https://pubmed.ncbi.nlm.nih.gov/21760609)" (2011) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 14.21, "[Original antigenic sin and apoptosis in the pathogenesis of dengue hemorrhagic fever.](https://pubmed.ncbi.nlm.nih.gov/12808447)" (2003) with an RCR of 11.3, "[On sin versus sickness. A theory of perceived responsibility and social motivation.](https://pubmed.ncbi.nlm.nih.gov/8214914)" (1993) with an RCR of 7.57, "[From Original Antigenic Sin to the Universal Influenza Virus Vaccine.](https://pubmed.ncbi.nlm.nih.gov/28867526)" (2018) with an RCR of 6.8, and "[Disquisitions of Original Antigenic Sin. I. Evidence in man.](https://pubmed.ncbi.nlm.nih.gov/5922742)" (1966) with an RCR of 5.69. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[EFS](https://www.proteinatlas.org/ENSG00000100842-EFS) is a gene with evidence at the protein level, detected in many tissues. It is primarily localized in the nucleoplasm and cytosol. The gene is expressed in various tissue clusters, including Cluster 34 (Non-specific - Mixed function), Cluster 11 (White matter - Signal transduction) in the brain, Cluster 12 (Bone cancer - Neuronal signaling) in cell lines, and Cluster 17 (Breast glandular cells - Lactation) in single cells.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [PGR](https://www.ncbi.nlm.nih.gov/gene/5241) with 5 experiments, [TFAP2C](https://www.ncbi.nlm.nih.gov/gene/7022) with 4 experiments, [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) with 4 experiments, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 4 experiments, and [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664) with 3 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.




The genome-wide association study on [Vertex-wise sulcal depth](https://pubmed.ncbi.nlm.nih.gov/34910505) has been mapped to the closest existing MeSH term, [Map](https://meshb.nlm.nih.gov/record/ui?ui=D019532), using a method called [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0).


The gene is expressed in various tissues, including the retina, hypothalamus, spinal cord, prostate, placenta, and olfactory bulb.




The analyzed protein sequence has a GRAVY value of -0.400178 (46.09th percentile), indicating a hydrophilic nature. The charge at pH 7 is -20.52 (10.77th percentile), and the median structural flexibility is 1.0093 (79.27th percentile). The protein has a high instability index of 69.69 (94.44th percentile) and an isoelectric point of 4.98 (7.33rd percentile). The sequence length is 561 amino acids (65.11th percentile) with a molecular weight of 58814.66 Da (61.38th percentile). The secondary structure affinities are 32.44% for helix (54.55th percentile), 26.02% for sheet (11.34th percentile), and 38.50% for turn (95.92nd percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |