---
layout: post
title: FERMT2
date: 2025-01-17 16:55 CST
description: FERMT2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10979) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10979  | FERMT2 | ENSG00000073712 | 14q22.1 |



The gene is located in various cellular components, including the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [adherens junction](https://amigo.geneontology.org/amigo/term/GO:0005912), [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925), [cell cortex](https://amigo.geneontology.org/amigo/term/GO:0005938), [cytoplasmic side of plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009898), [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986), [cell junction](https://amigo.geneontology.org/amigo/term/GO:0030054), [lamellipodium membrane](https://amigo.geneontology.org/amigo/term/GO:0031258), [I band](https://amigo.geneontology.org/amigo/term/GO:0031674), and [stress fiber](https://amigo.geneontology.org/amigo/term/GO:0001725). It enables various molecular functions, such as [actin binding](https://amigo.geneontology.org/amigo/term/GO:0003779), [integrin binding](https://amigo.geneontology.org/amigo/term/GO:0005178), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [phosphatidylinositol-3,4,5-trisphosphate binding](https://amigo.geneontology.org/amigo/term/GO:0005547), [protein kinase binding](https://amigo.geneontology.org/amigo/term/GO:0019901), [type I transforming growth factor beta receptor binding](https://amigo.geneontology.org/amigo/term/GO:0034713), [SMAD binding](https://amigo.geneontology.org/amigo/term/GO:0046332), and [actin filament binding](https://amigo.geneontology.org/amigo/term/GO:0051015). The gene is involved in numerous biological processes, including [cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0007155), [cell-matrix adhesion](https://amigo.geneontology.org/amigo/term/GO:0007160), [transforming growth factor beta receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007179), [integrin-mediated signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007229), [regulation of cell shape](https://amigo.geneontology.org/amigo/term/GO:0008360), [positive regulation of epithelial to mesenchymal transition](https://amigo.geneontology.org/amigo/term/GO:0010718), [Wnt signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0016055), [regulation of cell morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0022604), [positive regulation of cell migration](https://amigo.geneontology.org/amigo/term/GO:0030335), [integrin activation](https://amigo.geneontology.org/amigo/term/GO:0033622), [positive regulation of integrin activation](https://amigo.geneontology.org/amigo/term/GO:0033625), [adherens junction maintenance](https://amigo.geneontology.org/amigo/term/GO:0034334), [substrate adhesion-dependent cell spreading](https://amigo.geneontology.org/amigo/term/GO:0034446), [positive regulation of myosin light chain kinase activity](https://amigo.geneontology.org/amigo/term/GO:0035505), [negative regulation of vascular permeability](https://amigo.geneontology.org/amigo/term/GO:0043116), [positive regulation of GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0043547), [negative regulation of fat cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0045599), [positive regulation of osteoblast differentiation](https://amigo.geneontology.org/amigo/term/GO:0045669), [focal adhesion assembly](https://amigo.geneontology.org/amigo/term/GO:0048041), [positive regulation of stress fiber assembly](https://amigo.geneontology.org/amigo/term/GO:0051496), [positive regulation of focal adhesion assembly](https://amigo.geneontology.org/amigo/term/GO:0051894), [positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction](https://amigo.geneontology.org/amigo/term/GO:0051897), [limb development](https://amigo.geneontology.org/amigo/term/GO:0060173), [positive regulation of ERK1 and ERK2 cascade](https://amigo.geneontology.org/amigo/term/GO:0070374), [protein localization to membrane](https://amigo.geneontology.org/amigo/term/GO:0072657), [positive regulation of substrate adhesion-dependent cell spreading](https://amigo.geneontology.org/amigo/term/GO:1900026), [positive regulation of protein localization to nucleus](https://amigo.geneontology.org/amigo/term/GO:1900182), [protein localization to cell junction](https://amigo.geneontology.org/amigo/term/GO:1902414), [positive regulation of mesenchymal stem cell proliferation](https://amigo.geneontology.org/amigo/term/GO:1902462), and [positive regulation of wound healing, spreading of epidermal cells](https://amigo.geneontology.org/amigo/term/GO:1903691).


The gene length is 174,216 base pairs (90.85th percentile), the mature length is 4,678 base pairs (88.61st percentile), and the primary transcript length is 89,379.5 base pairs (86.71st percentile).


The gene FERMT2 (NCBI ID: 10979) has been mentioned in [126 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22FERMT2%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 2000, and the middle 50% of publications occurred between 2014 and 2020.


The top five publications mentioning FERMT2, ranked by their scientific influence, include "[Kindlin-2 inhibits Nlrp3 inflammasome activation in nucleus pulposus to maintain homeostasis of the intervertebral disc.](https://pubmed.ncbi.nlm.nih.gov/35013104)" (2022) (relative citation ratio: 9.84), "[Kindlin-2 cooperates with talin to activate integrins and induces cell spreading by directly binding paxillin.](https://pubmed.ncbi.nlm.nih.gov/26821125)" (2016) (relative citation ratio: 6.28), "[Kindlin-2 (Mig-2): a co-activator of beta3 integrins.](https://pubmed.ncbi.nlm.nih.gov/18458155)" (2008) (relative citation ratio: 5.35), "[Migfilin and Mig-2 link focal adhesions to filamin and the actin cytoskeleton and function in cell shape modulation.](https://pubmed.ncbi.nlm.nih.gov/12679033)" (2003) (relative citation ratio: 5.08), and "[Kindlin-2 haploinsufficiency protects against fatty liver by targeting Foxo1 in mice.](https://pubmed.ncbi.nlm.nih.gov/35197460)" (2022) (relative citation ratio: 3.47). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[FERMT2](https://www.proteinatlas.org/ENSG00000073712-FERMT2) is a gene with evidence at the protein level, detected in many RNA tissue distributions. It is primarily localized in the nucleoplasm, focal adhesion sites, and cytosol. The gene is expressed in various clusters, including Cluster 1 of T-cells with unknown function in blood, Cluster 62 of connective tissue involved in ECM organization, Cluster 13 of oligodendrocytes with mixed function in the brain, Cluster 64 of connective tissue cells involved in ECM organization in cell lines, and Cluster 32 of smooth muscle cells involved in muscle contraction in single cells.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930) and [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613), each with 6 experiments. Additionally, [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025), [E2F1](https://www.ncbi.nlm.nih.gov/gene/1869), and [SMARCB1](https://www.ncbi.nlm.nih.gov/gene/6598) were each found to be regulating in 5 experiments.



The input data indicates associations with a range of disease conditions, including various types of cancer such as male reproductive organ cancer, reproductive organ cancer, prostate cancer, and organ system cancer. Additionally, the data highlights metabolic disorders like disease of metabolism, acquired metabolic disease, nutrition disease, overnutrition, and obesity. Cardiovascular conditions such as artery disease, vascular disease, cardiovascular system disease, and hypertension are also noted. Furthermore, the data includes nervous system diseases, specifically central nervous system disease, neurodegenerative disease, Alzheimer's disease, tauopathy, dementia, cognitive disorder, and disease of mental health.



The gene is expressed in various tissues and cell types, including cardiac myocytes, adipocytes, smooth muscle, colon, olfactory bulb, fetal lung, and uterus. Additionally, it is expressed in blood vessels.


The proteins are involved in various pathways related to cell junction organization, cell-cell communication, and cell-extracellular matrix interactions.



The protein sequence analyzed has a GRAVY value of -0.395 (46.71 percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -5.20 (33.17 percentile). The median structural flexibility is 1.003 (56.6 percentile), with a predicted secondary structure affinity of 36.18% for helix (81.62 percentile), 36.62% for sheet (70.53 percentile), and 26.18% for turn (30.81 percentile). The instability index is 41.74 (27.83 percentile), and the isoelectric point is 6.26 (37.02 percentile). The protein has a length of 680 amino acids (74.12 percentile) and a molecular weight of 77859.64 Da (75.12 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |