---
layout: post
title: CTNNA1
date: 2025-01-17 16:55 CST
description: CTNNA1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1495) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1495  | CTNNA1 | ENSG00000044115 | 5q31.2 |



The gene is involved in [ovarian follicle development](https://amigo.geneontology.org/amigo/term/GO:0001541) and is located in various cellular components, including the [acrosomal vesicle](https://amigo.geneontology.org/amigo/term/GO:0001669), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [cell-cell junction](https://amigo.geneontology.org/amigo/term/GO:0005911), [adherens junction](https://amigo.geneontology.org/amigo/term/GO:0005912), [zonula adherens](https://amigo.geneontology.org/amigo/term/GO:0005915), [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925), [intercalated disc](https://amigo.geneontology.org/amigo/term/GO:0014704), [actin cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0015629), [lamellipodium](https://amigo.geneontology.org/amigo/term/GO:0030027), [cell junction](https://amigo.geneontology.org/amigo/term/GO:0030054), and [intracellular membrane-bounded organelle](https://amigo.geneontology.org/amigo/term/GO:0043231). It enables functions such as [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [structural molecule activity](https://amigo.geneontology.org/amigo/term/GO:0005198), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [beta-catenin binding](https://amigo.geneontology.org/amigo/term/GO:0008013), [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802), [gamma-catenin binding](https://amigo.geneontology.org/amigo/term/GO:0045295), [cadherin binding](https://amigo.geneontology.org/amigo/term/GO:0045296), and [actin filament binding](https://amigo.geneontology.org/amigo/term/GO:0051015). Additionally, it is involved in processes like [cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0007155), [establishment or maintenance of cell polarity](https://amigo.geneontology.org/amigo/term/GO:0007163), [smoothened signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007224), [integrin-mediated signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007229), [neuroblast proliferation](https://amigo.geneontology.org/amigo/term/GO:0007405), [negative regulation of neuroblast proliferation](https://amigo.geneontology.org/amigo/term/GO:0007406), [protein localization](https://amigo.geneontology.org/amigo/term/GO:0008104), [male gonad development](https://amigo.geneontology.org/amigo/term/GO:0008584), [gap junction assembly](https://amigo.geneontology.org/amigo/term/GO:0016264), [cell migration](https://amigo.geneontology.org/amigo/term/GO:0016477), [axon regeneration](https://amigo.geneontology.org/amigo/term/GO:0031103), [odontogenesis of dentin-containing tooth](https://amigo.geneontology.org/amigo/term/GO:0042475), [apical junction assembly](https://amigo.geneontology.org/amigo/term/GO:0043297), [response to estrogen](https://amigo.geneontology.org/amigo/term/GO:0043627), [positive regulation of smoothened signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0045880), [cellular response to indole-3-methanol](https://amigo.geneontology.org/amigo/term/GO:0071681), [epithelial cell-cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0090136), [extrinsic apoptotic signaling pathway in absence of ligand](https://amigo.geneontology.org/amigo/term/GO:0097192), [negative regulation of protein localization to nucleus](https://amigo.geneontology.org/amigo/term/GO:1900181), [negative regulation of cell motility](https://amigo.geneontology.org/amigo/term/GO:2000146), [negative regulation of integrin-mediated signaling pathway](https://amigo.geneontology.org/amigo/term/GO:2001045), [negative regulation of extrinsic apoptotic signaling pathway in absence of ligand](https://amigo.geneontology.org/amigo/term/GO:2001240), and [positive regulation of extrinsic apoptotic signaling pathway in absence of ligand](https://amigo.geneontology.org/amigo/term/GO:2001241). Furthermore, it is part of the [catenin complex](https://amigo.geneontology.org/amigo/term/GO:0016342) and [flotillin complex](https://amigo.geneontology.org/amigo/term/GO:0016600).


The gene length is 26,731 base pairs (49.52% of all genes), the mature length is 1,509 base pairs, and the primary transcript length is 26,714 base pairs.


The gene CTNNA1 (NCBI ID: 1495) has been mentioned in [43 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CTNNA1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1994 and the middle 50% of publications occurring between 2008 and 2016. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning CTNNA1, ranked by their scientific influence, include "[αE-catenin inhibits a Src-YAP1 oncogenic module that couples tyrosine kinases and the effector of Hippo signaling pathway.](https://pubmed.ncbi.nlm.nih.gov/27013234)" (2016) (relative citation ratio: 4.28), "[alphaE-catenin controls cerebral cortical size by regulating the hedgehog signaling pathway.](https://pubmed.ncbi.nlm.nih.gov/16543460)" (2006) (relative citation ratio: 3.49), "[Chromosome 5q deletion and epigenetic suppression of the gene encoding alpha-catenin (CTNNA1) in myeloid cell transformation.](https://pubmed.ncbi.nlm.nih.gov/17159988)" (2007) (relative citation ratio: 2.7), "[αE-catenin is an autoinhibited molecule that coactivates vinculin.](https://pubmed.ncbi.nlm.nih.gov/22586082)" (2012) (relative citation ratio: 2.65), and "[AlphaE-catenin regulates actin dynamics independently of cadherin-mediated cell-cell adhesion.](https://pubmed.ncbi.nlm.nih.gov/20404114)" (2010) (relative citation ratio: 2.46). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CTNNA1](https://www.proteinatlas.org/ENSG00000044115-CTNNA1) is a gene with evidence at the protein level. Its RNA is detected in all tissues, and the protein is localized in various subcellular locations including the Golgi apparatus, vesicles, plasma membrane, cell junctions, and different parts of the spermatozoon. The gene is expressed in specific clusters related to monocytes and neutrophils degranulation in blood, squamous epithelium keratinization in tissues, signal transduction in the brain's white matter, and mRNA splicing and cell cycle in cell lines. Single-cell expression analysis shows a mixed function cluster.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930) and [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), each with 6 experiments. Additionally, [POU5F1](https://www.ncbi.nlm.nih.gov/gene/5460), [RELA](https://www.ncbi.nlm.nih.gov/gene/5970), and [E2F1](https://www.ncbi.nlm.nih.gov/gene/1869) were each found to be regulating in 5 experiments.



The GWAS data indicates associations with various disease conditions, including cardiovascular system disease, general disease, and diseases of anatomical entities.



The gene is expressed in nerve tissue according to BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles indicate expression in Cardiac Myocytes, Colorectal Adenocarcinoma, and Dorsal Root Ganglion.


The proteins are involved in various pathways, including signaling by VEGF, the VEGFA-VEGFR2 pathway, and VEGFR2-mediated vascular permeability. Additionally, they play roles in CDO in myogenesis, myogenesis itself, and broader developmental biology processes. These proteins are also crucial for signal transduction, adherens junctions interactions, cell junction organization, cell-cell junction organization, and overall cell-cell communication.



The analyzed protein sequence has a GRAVY value of -0.368 (50.13th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -8.40 (25.51st percentile), and the median structural flexibility is 1.006 (66.84th percentile). The protein has a high affinity for helix structures (0.402, 93.72nd percentile), with lower affinities for sheet (0.302, 29.62nd percentile) and turn (0.251, 21.96th percentile) structures. The instability index is 40.72 (24.23rd percentile), and the isoelectric point is 5.95 (29.25th percentile). The protein is 906 amino acids long (84.63rd percentile) with a molecular weight of 100070.16 Da (84.29th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |