---
layout: post
title: ATP6V0B
date: 2025-01-17 16:55 CST
description: ATP6V0B description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/533) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 533  | ATP6V0B | ENSG00000117410 | 1p34.1 |



The gene is located in the [Golgi membrane](https://amigo.geneontology.org/amigo/term/GO:0000139), [lysosomal membrane](https://amigo.geneontology.org/amigo/term/GO:0005765), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0010008), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [clathrin-coated vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030665), and [phagocytic vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030670). It is part of the [vacuolar proton-transporting V-type ATPase, V0 domain](https://amigo.geneontology.org/amigo/term/GO:0000220), [proton-transporting V-type ATPase complex](https://amigo.geneontology.org/amigo/term/GO:0033176), and [transmembrane transporter complex](https://amigo.geneontology.org/amigo/term/GO:1902495). The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [proton-transporting ATPase activity, rotational mechanism](https://amigo.geneontology.org/amigo/term/GO:0046961). It is involved in [vacuolar acidification](https://amigo.geneontology.org/amigo/term/GO:0007035), [lysosomal lumen acidification](https://amigo.geneontology.org/amigo/term/GO:0007042), [regulation of macroautophagy](https://amigo.geneontology.org/amigo/term/GO:0016241), [endosomal lumen acidification](https://amigo.geneontology.org/amigo/term/GO:0048388), [intracellular pH reduction](https://amigo.geneontology.org/amigo/term/GO:0051452), [Golgi lumen acidification](https://amigo.geneontology.org/amigo/term/GO:0061795), and [proton transmembrane transport](https://amigo.geneontology.org/amigo/term/GO:1902600).


The gene length is 60,469 base pairs (71.19% of all genes), the mature length is 2,744 base pairs, and the primary transcript length is 57,148 base pairs.


The gene ATP6V0B, with NCBI ID 533, has been mentioned in [2 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ATP6V0B%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1998, and the middle 50% of publications occurred between 1998 and 2000.


The top publications mentioning ATP6V0B include "[Identification and characterization of the gene encoding a second proteolipid subunit of human vacuolar H(+)-ATPase (ATP6F).](https://pubmed.ncbi.nlm.nih.gov/9653649)" (1998) (relative citation ratio: 0.27) and "[Mouse Atp6f, the gene encoding the 23-kDa proteolipid of vacuolar proton translocating ATPase.](https://pubmed.ncbi.nlm.nih.gov/11675001)" (2001) (relative citation ratio: 0.12). The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are from [iCite](https://icite.od.nih.gov).


[ATP6V0B](https://www.proteinatlas.org/ENSG00000117410-ATP6V0B) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily located in the cytosol and is associated with various tissue expression clusters, including choroid plexus for transmembrane transport. The gene is also involved in non-specific innate immune response in blood, mixed function in the brain, mRNA processing in cell lines, and mixed function in single cells.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170) with 6 experiments, [TAL1](https://www.ncbi.nlm.nih.gov/gene/6886) with 5 experiments, [KDM5B](https://www.ncbi.nlm.nih.gov/gene/10765) with 5 experiments, [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 5 experiments, and [JUN](https://www.ncbi.nlm.nih.gov/gene/3725) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.


The top transcription factors (TFs) identified from the GTRD's database of TF binding sites, filtered for Perturb-seq data, include [STAT1](https://www.ncbi.nlm.nih.gov/gene/6830) and [TBP](https://www.ncbi.nlm.nih.gov/gene/9968). STAT1 was shown to be regulating genes in 2 experiments, while TBP was found to be regulating genes in 1 experiment.




The gene is expressed in multiple tissue samples, including the pituitary, blood, and kidney, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the pineal gland (night), heart, liver, CD33+ myeloid cells, whole blood, whole brain, and lung.


The input data highlights several biological pathways, including insulin receptor recycling, transferrin endocytosis and recycling, iron uptake and transport, latent infection of Homo sapiens with Mycobacterium tuberculosis, phagosomal maturation (early endosomal stage), transmembrane transport of small molecules, signaling by insulin receptor, disease, and signal transduction.


The mouse ortholog gene 114143 and its human ortholog gene 533 are associated with several phenotypes, including decreased bone mineral content, increased total body fat amount, and preweaning lethality with complete penetrance. These phenotypes are observed in both sexes.


The protein sequence analyzed has a GRAVY value of 1.0048780487804876 (99.9th percentile), indicating high hydrophobicity. At pH 7.0, the protein carries a charge of 0.6331966561668221 (52.4th percentile). The median structural flexibility is 0.9783988095238095 (3.62nd percentile), with a helix affinity of 0.2829268292682927 (18.85th percentile), sheet affinity of 0.4585365853658537 (95.36th percentile), and turn affinity of 0.2829268292682927 (49.99th percentile). The instability index is 19.95219512195121 (0.61st percentile), and the isoelectric point is 7.613264656066895 (55.34th percentile). The protein has a length of 205 amino acids (15.23rd percentile) and a molecular weight of 21406.07289999998 Da (13.17th percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |