---
layout: post
title: ITM2B
date: 2025-01-17 16:55 CST
description: ITM2B description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9445) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9445  | ITM2B | ENSG00000136156 | 13q14.2 |



The gene is located in the [Golgi membrane](https://amigo.geneontology.org/amigo/term/GO:0000139), [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615), [Golgi apparatus](https://amigo.geneontology.org/amigo/term/GO:0005794), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0010008), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [Golgi-associated vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030660), [organelle membrane](https://amigo.geneontology.org/amigo/term/GO:0031090), [intracellular membrane-bounded organelle](https://amigo.geneontology.org/amigo/term/GO:0043231), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). It enables [amyloid-beta binding](https://amigo.geneontology.org/amigo/term/GO:0001540), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524). The gene is active in the [Golgi apparatus](https://amigo.geneontology.org/amigo/term/GO:0005794) and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). Additionally, it is involved in [nervous system development](https://amigo.geneontology.org/amigo/term/GO:0007399) and the [negative regulation of amyloid precursor protein biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0042985).


The gene length is 38,286.0 base pairs (59.26% of all genes), the mature length is 2,465.5 base pairs, and the primary transcript length is 38,165.0 base pairs.


The gene ITM2B (NCBI ID: 9445) has been mentioned in [87 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ITM2B%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning this gene is 1954, and the middle 50% of publications occurred between 2004 and 2018.


The top 5 publications mentioning ITM2B, ranked by their scientific influence, include "[A stop-codon mutation in the BRI gene associated with familial British dementia.](https://pubmed.ncbi.nlm.nih.gov/10391242)" (1999) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 7.41, "[A decamer duplication in the 3' region of the BRI gene originates an amyloid peptide that is associated with dementia in a Danish kindred.](https://pubmed.ncbi.nlm.nih.gov/10781099)" (2000) with an RCR of 4.69, "[Do BRI policy and institutional quality influence economic growth and environmental quality? An empirical analysis from South Asian countries affiliated with the Belt and Road Initiative.](https://pubmed.ncbi.nlm.nih.gov/34490560)" (2022) with an RCR of 3.55, "[Augmentation of Bri2 molecular chaperone activity against amyloid-Î² reduces neurotoxicity in mouse hippocampus in vitro.](https://pubmed.ncbi.nlm.nih.gov/31959875)" (2020) with an RCR of 2.65, and "[The familial dementia BRI2 gene binds the Alzheimer gene amyloid-beta precursor protein and inhibits amyloid-beta production.](https://pubmed.ncbi.nlm.nih.gov/15983050)" (2005) with an RCR of 2.5. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ITM2B](https://www.proteinatlas.org/ENSG00000136156-ITM2B) is a gene implicated in various diseases, including amyloidosis, deafness, and neurodegeneration. Evidence for its involvement is supported at the protein level. The gene is detected in all RNA tissue distributions and is localized in the Golgi apparatus and vesicles. ITM2B is expressed in neutrophils with unknown function (Cluster 21) and is associated with transmembrane transport in the kidney (Cluster 52). In the brain, it is involved in metabolism (Cluster 21), and in kidney cancer cell lines, it is associated with membrane components (Cluster 15). Additionally, ITM2B is involved in transcription in non-specific single cells (Cluster 80).


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), [CREB1](https://www.ncbi.nlm.nih.gov/gene/1385), [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861), and [KMT2B](https://www.ncbi.nlm.nih.gov/gene/9757), each shown to be regulating in 4 experiments.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [23028](https://www.ncbi.nlm.nih.gov/gene/23028) and [4849](https://www.ncbi.nlm.nih.gov/gene/4849), each shown to be regulating genes in two experiments. Additionally, [2623](https://www.ncbi.nlm.nih.gov/gene/2623), [2308](https://www.ncbi.nlm.nih.gov/gene/2308), [8243](https://www.ncbi.nlm.nih.gov/gene/8243), [8861](https://www.ncbi.nlm.nih.gov/gene/8861), and [9126](https://www.ncbi.nlm.nih.gov/gene/9126) were each found to be regulating genes in one experiment.


The input data indicates associations with various metabolic disorders, including diseases of metabolism, acquired metabolic diseases, carbohydrate metabolism diseases, glucose metabolism diseases, diabetes mellitus, and specifically type 2 diabetes mellitus.



The gene is expressed in multiple tissue samples, including the thyroid, whole blood, amygdala, spinal cord, cerebellum, and kidney. Additionally, it is expressed in blood according to the BioGPS Human Cell Type and Tissue Gene Expression Profiles.


The input data focuses on amyloids and their association with disease, as indicated by the target categories.



The analyzed protein sequence has a GRAVY value of -0.0665 (80.34th percentile), indicating a relatively hydrophilic nature. The charge at pH 7 is -12.02 (19.41st percentile), and the median structural flexibility is 0.9955 (22.34th percentile). The protein has a propensity for helix, sheet, and turn structures of 34.21% (69.45th percentile), 38.72% (81.92nd percentile), and 24.44% (17.83rd percentile), respectively. The instability index is 47.95 (50.02nd percentile), with an isoelectric point of 4.999 (7.77th percentile). The protein is 266 amino acids long (23.9th percentile) and has a molecular weight of 30337.69 Da (24.77th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |