---
layout: post
title: TSNAX
date: 2025-01-17 16:55 CST
description: TSNAX description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7257) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7257  | TSNAX | ENSG00000116918 | 1q42.2 |



The gene enables [DNA binding](https://amigo.geneontology.org/amigo/term/GO:0003677), [single-stranded DNA binding](https://amigo.geneontology.org/amigo/term/GO:0003697), [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [RNA endonuclease activity](https://amigo.geneontology.org/amigo/term/GO:0004521), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [A2A adenosine receptor binding](https://amigo.geneontology.org/amigo/term/GO:0031687), [sequence-specific DNA binding](https://amigo.geneontology.org/amigo/term/GO:0043565), [protein-containing complex binding](https://amigo.geneontology.org/amigo/term/GO:0044877), and [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872). It is active in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), and is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [Golgi apparatus](https://amigo.geneontology.org/amigo/term/GO:0005794), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and the [perinuclear region of cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0048471). The gene is involved in [spermatogenesis](https://amigo.geneontology.org/amigo/term/GO:0007283), [cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0030154), [siRNA processing](https://amigo.geneontology.org/amigo/term/GO:0030422), and [regulatory ncRNA-mediated post-transcriptional gene silencing](https://amigo.geneontology.org/amigo/term/GO:0035194). Additionally, it is part of the [endoribonuclease complex](https://amigo.geneontology.org/amigo/term/GO:1902555).


The gene length is 18,332 base pairs (39.98% of all genes), the mature length is 3,159 base pairs, and the primary transcript length is 17,792.5 base pairs.


The gene TSNAX (NCBI ID: 7257) has been mentioned in [36 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22TSNAX%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1993 and the middle 50% of publications occurring between 1997 and 2012. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning TSNAX, ranked by their relative citation ratio (RCR), include "[Association of DISC1/TRAX haplotypes with schizophrenia, reduced prefrontal gray matter, and impaired short- and long-term memory.](https://pubmed.ncbi.nlm.nih.gov/16275808)" (2005) (relative citation ratio: 5.88), "[Tranexamic acid administration to pediatric trauma patients in a combat setting: the pediatric trauma and tranexamic acid study (PED-TRAX).](https://pubmed.ncbi.nlm.nih.gov/25423534)" (2014) (relative citation ratio: 4.31), "[Association between the TRAX/DISC locus and both bipolar disorder and schizophrenia in the Scottish population.](https://pubmed.ncbi.nlm.nih.gov/15838535)" (2005) (relative citation ratio: 2.93), "[Isolation and characterization of a cDNA encoding a Translin-like protein, TRAX.](https://pubmed.ncbi.nlm.nih.gov/9013868)" (1997) (relative citation ratio: 1.54), and "[Evaluation of the FACScount, TRAx CD4 and Dynabeads methods for CD4 lymphocyte determination.](https://pubmed.ncbi.nlm.nih.gov/8814325)" (1996) (relative citation ratio: 1.51). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[TSNAX](https://www.proteinatlas.org/ENSG00000116918-TSNAX) is a developmental protein with DNA-binding molecular function, supported by evidence at the protein level. It is detected in all tissues and is primarily localized in the nucleoplasm, Golgi apparatus, and cytosol. TSNAX is associated with various expression clusters, including Cluster 42 for blood (non-specific - vesicular transport), Cluster 35 for tissue (non-specific - mitochondria), Cluster 21 for brain (non-specific - metabolism), Cluster 45 for cell lines (non-specific - nuclear processes), and Cluster 30 for single cells (granulocytes - unknown function).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [KDM5B](https://www.ncbi.nlm.nih.gov/gene/10765) with 5 experiments, [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025) with 5 experiments, [RELA](https://www.ncbi.nlm.nih.gov/gene/5970) with 5 experiments, [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 4 experiments, and [E2F8](https://www.ncbi.nlm.nih.gov/gene/79733) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent detection in multiple experiments.



The GWAS data indicates associations with various disease conditions, including immune system disease, general disease, disease of anatomical entity, and hypersensitivity reaction disease.



The gene is expressed in adipose tissue and breast according to the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles indicate expression in pineal gland (both day and night), CD33+ myeloid cells, CD56+ NK cells, CD14+ monocytes, CD105+ endothelial cells, and the amygdala.


The input data highlights several key biological processes, including gene expression, regulatory rna pathways, and small interfering rna (sirna) biogenesis. These pathways are integral to the regulation and function of genetic information within cells.


The mouse ortholog gene 53424 and its human ortholog 7257 are associated with several phenotypic abnormalities in both sexes, including abnormal coat/hair pigmentation, abnormal eye morphology, abnormal pancreas morphology, abnormal skin morphology, decreased body length, decreased grip strength, and decreased heart weight.


The analyzed protein sequence has a GRAVY value of -0.558 (29.04th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -3.764 (37.24th percentile), and the median structural flexibility is 1.004 (59.02nd percentile). The protein has a propensity for helix, sheet, and turn structures of 30.69% (38.01st percentile), 34.48% (55.34th percentile), and 27.93% (46.58th percentile), respectively. The instability index is 41.81 (28.05th percentile), and the isoelectric point is 6.10 (33.17th percentile). The protein is 290 amino acids long (26.99th percentile) with a molecular weight of 33112.09 Da (27.97th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |