---
layout: post
title: UBE2H
date: 2024-12-08 23:57 CST
description: UBE2H description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7328) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7328  | UBE2H | ENSG00000186591 | 7q32.2 |



The gene length is 21,241.0 base pairs (43.49% of all genes), the mature length is 1,168.5 base pairs, and the primary transcript length is 20,718.0 base pairs.


UBE2H has been studied in 8 publications, with the earliest publication in 1990 and the middle 50% of publications occurring between 1994 and 2011. For more information, see the [8 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22UBE2H%22) on PubMed (note that the count may differ since it is derived from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)).


The top 5 publications mentioning UBE2H, ranked by their scientific influence, include "[TNF-alpha increases ubiquitin-conjugating activity in skeletal muscle by up-regulating UbcH2/E220k.](https://pubmed.ncbi.nlm.nih.gov/12773487)" (2003) with a relative citation ratio of 3.97, "[Predictive Potential of Circulating Ube2h mRNA as an E2 Ubiquitin-Conjugating Enzyme for Diagnosis or Treatment of Alzheimer's Disease.](https://pubmed.ncbi.nlm.nih.gov/32403399)" (2020) with a relative citation ratio of 0.95, "[A human ubiquitin-conjugating enzyme homologous to yeast UBC8.](https://pubmed.ncbi.nlm.nih.gov/8132613)" (1994) with a relative citation ratio of 0.85, "[MET-UBE2H Fusion as a Novel Mechanism of Acquired EGFR Resistance in Lung Adenocarcinoma.](https://pubmed.ncbi.nlm.nih.gov/30244854)" (2018) with a relative citation ratio of 0.81, and "[Mutation screening and association study of the UBE2H gene on chromosome 7q32 in autistic disorder.](https://pubmed.ncbi.nlm.nih.gov/14639049)" (2003) with a relative citation ratio of 0.38. The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313), [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), and [MYC](https://www.ncbi.nlm.nih.gov/gene/4609), each with 5 experiments. Additionally, [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) was found to be regulating in 4 experiments.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, filtered for Perturb-seq data, include [CTCF](https://www.ncbi.nlm.nih.gov/gene/6829), [NRF1](https://www.ncbi.nlm.nih.gov/gene/2623), and [REST](https://www.ncbi.nlm.nih.gov/gene/9126), each shown to regulate gene expression in three experiments. Additionally, [ZNF263](https://www.ncbi.nlm.nih.gov/gene/8243) was found to regulate gene expression in two experiments, while [ZNF281](https://www.ncbi.nlm.nih.gov/gene/8861) and [ZNF783](https://www.ncbi.nlm.nih.gov/gene/9968) were each identified in one experiment.








The analyzed protein sequence has a GRAVY value of -0.549 (29.99th percentile), indicating a hydrophilic nature. It exhibits a charge of -14.2 at pH 7.0 (16.35th percentile) and a median structural flexibility of 1.009 (78.45th percentile). The protein's secondary structure is predicted to be 32.8% helix (57.57th percentile), 33.9% sheet (51.12th percentile), and 32.8% turn (80.88th percentile). The instability index is 63.1 (87.92nd percentile), suggesting potential instability. The isoelectric point is 4.55 (1.85th percentile), with a molecular weight of 20654.9 Da (12.2nd percentile). The sequence length is 183 amino acids (11.98th percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |