---
layout: post
title: RPL28
date: 2025-01-17 16:55 CST
description: RPL28 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6158) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6158  | RPL28 | ENSG00000108107 | 19q13.42 |



The gene is involved in [cytoplasmic translation](https://amigo.geneontology.org/amigo/term/GO:0002181) and [translation](https://amigo.geneontology.org/amigo/term/GO:0006412). It enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), acts as a [structural constituent of ribosome](https://amigo.geneontology.org/amigo/term/GO:0003735), and facilitates [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [dendrite](https://amigo.geneontology.org/amigo/term/GO:0030425), [cytoplasmic ribonucleoprotein granule](https://amigo.geneontology.org/amigo/term/GO:0036464), [cell body](https://amigo.geneontology.org/amigo/term/GO:0044297), [synapse](https://amigo.geneontology.org/amigo/term/GO:0045202), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). Additionally, it is part of the [cytosolic large ribosomal subunit](https://amigo.geneontology.org/amigo/term/GO:0022625) and located in the [cytosolic ribosome](https://amigo.geneontology.org/amigo/term/GO:0022626).


The gene length is 57,361 base pairs (69.9% of all genes), the mature length is 6,116 base pairs, and the primary transcript length is 57,361 base pairs.


The gene RPL28, with NCBI ID 6158, has been mentioned in [4 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RPL28%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1989, and the middle 50% of publications range from 2005 to 2017.


The top publications mentioning RPL28 include "[Identified facilitator neurons L29 and L28 are excited by cutaneous stimuli used in dishabituation, sensitization, and classical conditioning of Aplysia.](https://pubmed.ncbi.nlm.nih.gov/2593000)" (1989) (relative citation ratio: 1.34) and "[The Ribosomal Protein L28 Gene Induces Sorafenib Resistance in Hepatocellular Carcinoma.](https://pubmed.ncbi.nlm.nih.gov/34307151)" (2021) (relative citation ratio: 0.23). The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RPL28](https://www.proteinatlas.org/ENSG00000108107-RPL28) is a ribonucleoprotein and ribosomal protein with evidence at the protein level. It is detected in all tissues and is primarily located in the endoplasmic reticulum and cytosol. The gene is part of non-specific translation and ribosome clusters across various tissue types, including blood, brain, and cell lines. Notably, RPL28 does not show significant prognostic value in multiple cancer types, including bladder urothelial carcinoma, breast invasive carcinoma, and others.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [AR](https://www.ncbi.nlm.nih.gov/gene/367), [ELF1](https://www.ncbi.nlm.nih.gov/gene/1997), [E2F1](https://www.ncbi.nlm.nih.gov/gene/1869), [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861), and [CREB1](https://www.ncbi.nlm.nih.gov/gene/1385), each shown to be regulating in 5 experiments.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, filtered for Perturb-seq data, include [6908](https://www.ncbi.nlm.nih.gov/gene/6908), [11091](https://www.ncbi.nlm.nih.gov/gene/11091), [6749](https://www.ncbi.nlm.nih.gov/gene/6749), [84148](https://www.ncbi.nlm.nih.gov/gene/84148), and [51755](https://www.ncbi.nlm.nih.gov/gene/51755). These TFs were found to regulate gene expression in 1, 2, 1, 2, and 1 Perturb-seq experiments, respectively.




The gene is expressed in various tissues, including the ovary and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Lymphoma (Burkitt's), CD34+ cells, CD19+ B cells (negatively selected), Leukemia (promyelocytic-HL-60), smooth muscle, and colorectal adenocarcinoma.


The input data highlights various pathways involved in protein metabolism and gene expression, including 3'-UTR-mediated translational regulation, L13a-mediated translational silencing of ceruloplasmin expression, and GTP hydrolysis and joining of the 60S ribosomal subunit. Key processes such as cap-dependent translation initiation, eukaryotic translation initiation, formation of a pool of free 40S subunits, eukaryotic translation elongation, and eukaryotic translation termination are also noted. Additionally, the data covers peptide chain elongation, viral mRNA translation, and nonsense-mediated decay (NMD) both independent of and enhanced by the exon junction complex (EJC). Pathways related to SRP-dependent cotranslational protein targeting to the membrane and translation are included. Furthermore, the data encompasses influenza infection, the influenza life cycle, influenza viral RNA transcription and replication, and disease.



The analyzed protein sequence has a GRAVY value of -0.744 (14.86th percentile), indicating a hydrophilic nature. It exhibits a charge of 29.74 at pH 7.0 (97.13th percentile) and a median structural flexibility of 1.006 (66.92nd percentile). The protein's secondary structure is predicted to be 28.47% helix (20th percentile), 30.66% sheet (31.8th percentile), and 25.55% turn (25.48th percentile). The instability index is 65.53 (90.82nd percentile), suggesting high instability. The isoelectric point is approximately 12.0 (99.95th percentile), with a molecular weight of 15747.33 Da (7.05th percentile). The sequence length is 137 amino acids (6.61st percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |