---
layout: post
title: CBLB
date: 2025-01-17 16:55 CST
description: CBLB description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/868) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 868  | CBLB | ENSG00000114423 | 3q13.11 |



The gene enables [phosphotyrosine residue binding](https://amigo.geneontology.org/amigo/term/GO:0001784) and [calcium ion binding](https://amigo.geneontology.org/amigo/term/GO:0005509), and is involved in [positive regulation of T cell anergy](https://amigo.geneontology.org/amigo/term/GO:0002669) and [T cell anergy](https://amigo.geneontology.org/amigo/term/GO:0002870). It is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654) and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and is active in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886) and [membrane raft](https://amigo.geneontology.org/amigo/term/GO:0045121). Additionally, it is involved in various processes such as [NLS-bearing protein import into nucleus](https://amigo.geneontology.org/amigo/term/GO:0006607), [immune response](https://amigo.geneontology.org/amigo/term/GO:0006955), [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), and [negative regulation of epidermal growth factor-activated receptor activity](https://amigo.geneontology.org/amigo/term/GO:0007175). The gene also enables [zinc ion binding](https://amigo.geneontology.org/amigo/term/GO:0008270), [SH3 domain binding](https://amigo.geneontology.org/amigo/term/GO:0017124), and [receptor tyrosine kinase binding](https://amigo.geneontology.org/amigo/term/GO:0030971), and is involved in [protein ubiquitination](https://amigo.geneontology.org/amigo/term/GO:0016567), [protein catabolic process](https://amigo.geneontology.org/amigo/term/GO:0030163), and [positive regulation of protein ubiquitination](https://amigo.geneontology.org/amigo/term/GO:0031398). Furthermore, it plays roles in [intracellular signal transduction](https://amigo.geneontology.org/amigo/term/GO:0035556), [CD4-positive, alpha-beta T cell proliferation](https://amigo.geneontology.org/amigo/term/GO:0035739), [T cell receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0050852), and [negative regulation of T cell receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0050860). The gene also enables [ubiquitin protein ligase activity](https://amigo.geneontology.org/amigo/term/GO:0061630) and is involved in [negative regulation of CD4-positive, alpha-beta T cell proliferation](https://amigo.geneontology.org/amigo/term/GO:2000562) and [regulation of platelet-derived growth factor receptor-alpha signaling pathway](https://amigo.geneontology.org/amigo/term/GO:2000583).


The gene length is 3,972 base pairs (11.12% of all genes), the mature length is 1,281 base pairs, and the primary transcript length is 3,855.5 base pairs.


The gene CBLB (NCBI ID: 868) has been mentioned in [165 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CBLB%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1981 and the middle 50% of publications occurring between 2008 and 2016. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning CBLB, ranked by their scientific influence, include "[The E3 ligase Cbl-b and TAM receptors regulate cancer metastasis via natural killer cells.](https://pubmed.ncbi.nlm.nih.gov/24553136)" (2014) with a relative citation ratio of 8.41, "[miR-27b-3p inhibits proliferation and potentially reverses multi-chemoresistance by targeting CBLB/GRB2 in breast cancer cells.](https://pubmed.ncbi.nlm.nih.gov/29416005)" (2018) with a relative citation ratio of 5.05, "[Sequential ubiquitination of NLRP3 by RNF125 and Cbl-b limits inflammasome activation and endotoxemia.](https://pubmed.ncbi.nlm.nih.gov/31999304)" (2020) with a relative citation ratio of 4.72, "[c-Cbl and Cbl-b regulate T cell responsiveness by promoting ligand-induced TCR down-modulation.](https://pubmed.ncbi.nlm.nih.gov/12415267)" (2002) with a relative citation ratio of 4.07, and "[Proteolysis-independent regulation of PI3K by Cbl-b-mediated ubiquitination in T cells.](https://pubmed.ncbi.nlm.nih.gov/11526404)" (2001) with a relative citation ratio of 3.25. The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CBLB](https://www.proteinatlas.org/ENSG00000114423-CBLB) is a transferase involved in cancer-related genes and disease variants, with evidence at the protein level. It is detected in all RNA tissue distributions and is primarily located in the nucleoplasm and cytosol. CBLB is expressed in NK-cells with unknown function in blood, adipose tissue with mixed function in general tissue, cerebellum with nucleic acid binding in the brain, connective tissue cells with ECM organization in cell lines, and NK-cells and T-cells related to the immune system and transcription in single cells.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025), [SP2](https://www.ncbi.nlm.nih.gov/gene/6668), and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), each shown to be regulating in 5 experiments.



The input data highlights a diverse range of disease conditions associated with GWAS studies, including various forms of cancer such as leukemia, lymphoma, and immune system cancer. Additionally, respiratory diseases like asthma, bronchial disease, and obstructive lung disease are noted. Neurodegenerative conditions, including Alzheimer's disease, tauopathy, and dementia, are also prominent. Furthermore, the data encompasses autoimmune diseases like celiac disease and multiple sclerosis, as well as cardiovascular and gastrointestinal system diseases.


The genome-wide association study has identified a link between the CBLB gene and the [Number of clonal hematopoiesis mutations](https://pubmed.ncbi.nlm.nih.gov/34815255), with the closest MeSH match being [Mutation](https://meshb.nlm.nih.gov/record/ui?ui=D009154), determined using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method.


The gene is expressed in various tissues, including adipose tissue, breast, and nerve, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in 721_B_lymphoblasts, BDCA4+ Dentritic Cells, CD56+ NK Cells, CD4+ T cells, CD8+ T cells, CD19+ B Cells (neg. sel.), Whole Blood, and Uterus.


The proteins are involved in several immune-related pathways, including antigen processing via ubiquitination and proteasome degradation, class I MHC-mediated antigen processing and presentation, and signaling by the B cell receptor (BCR), which leads to the generation of second messengers. These pathways are integral parts of the adaptive immune system and the broader immune system.



The protein sequence analyzed has a GRAVY value of -0.6234 (23.24th percentile), indicating a hydrophilic nature. It exhibits a charge of 6.82 at pH 7.0 (73.76th percentile) and a median structural flexibility of 1.0092 (78.79th percentile). The protein's secondary structure is predicted to be 26.68% helix (10.31st percentile), 28.72% sheet (21.8th percentile), and 37.68% turn (94.85th percentile). The instability index is 57.46 (78.1st percentile), with an isoelectric point of 8.15 (60.41st percentile). The sequence length is 982 amino acids (87.16th percentile), and the molecular weight is 109448.49 Da (87.07th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |