---
layout: post
title: PCCA
date: 2025-01-17 16:55 CST
description: PCCA description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5095) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5095  | PCCA | ENSG00000175198 | 13q32.3 |



The gene enables [propionyl-CoA carboxylase activity](https://amigo.geneontology.org/amigo/term/GO:0004658), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524), [biotin binding](https://amigo.geneontology.org/amigo/term/GO:0009374), [enzyme binding](https://amigo.geneontology.org/amigo/term/GO:0019899), and [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872). It is located in the [mitochondrion](https://amigo.geneontology.org/amigo/term/GO:0005739), specifically in the [mitochondrial matrix](https://amigo.geneontology.org/amigo/term/GO:0005759), and is also found in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). The gene is involved in the [fatty acid metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006631), [branched-chain amino acid metabolic process](https://amigo.geneontology.org/amigo/term/GO:0009081), and [short-chain fatty acid catabolic process](https://amigo.geneontology.org/amigo/term/GO:0019626). Additionally, it is part of the [catalytic complex](https://amigo.geneontology.org/amigo/term/GO:1902494).


The gene length is 90,959.0 base pairs (80.1% of all genes), the mature length is 4,605.0 base pairs, and the primary transcript length is 90,655.0 base pairs.


The gene PCCA (NCBI ID: 5095) has been mentioned in [23 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PCCA%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1986 and the middle 50% of publications occurring between 2001 and 2018. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning PCCA, ranked by their relative citation ratio (RCR), include "[Isolation of cDNA clones coding for the alpha and beta chains of human propionyl-CoA carboxylase: chromosomal assignments and DNA polymorphisms associated with PCCA and PCCB genes.](https://pubmed.ncbi.nlm.nih.gov/3460076)" (1986) (RCR: 3.48), "[Overview of mutations in the PCCA and PCCB genes causing propionic acidemia.](https://pubmed.ncbi.nlm.nih.gov/10502773)" (1999) (RCR: 1.62), "[Differentially expressed genes <i>PCCA</i>, <i>ECHS1</i>, and <i>HADH</i> are potential prognostic biomarkers for gastric cancer.](https://pubmed.ncbi.nlm.nih.gov/33881965)" (2021) (RCR: 1.15), "[Functional characterization of PCCA mutations causing propionic acidemia.](https://pubmed.ncbi.nlm.nih.gov/12385775)" (2002) (RCR: 0.71), and "[Mutation spectrum of the PCCA and PCCB genes in Japanese patients with propionic acidemia.](https://pubmed.ncbi.nlm.nih.gov/15059621)" (2004) (RCR: 0.71). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PCCA](https://www.proteinatlas.org/ENSG00000175198-PCCA) is a gene encoding a protein with ligase molecular function, primarily located in the mitochondria. It is involved in disease variants and has evidence at the protein level. The gene's RNA is detected in all tissues, with specific expression patterns observed in various clusters, including Cluster 33 for blood (non-specific - ATP binding), Cluster 53 for tissue (non-specific - Glycosylation), Cluster 25 for brain (non-specific - Transcription), Cluster 57 for cell lines (Stomach & Colon cancer - Digestion), and Cluster 37 for single cells (Glandular & Luminal cells - Unknown function).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613) with 5 experiments, [NELFA](https://www.ncbi.nlm.nih.gov/gene/7469) with 4 experiments, [SUMO2](https://www.ncbi.nlm.nih.gov/gene/6613) with 4 experiments, [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599) with 4 experiments, and [RELA](https://www.ncbi.nlm.nih.gov/gene/5970) with 4 experiments.



The input data from GWAS indicates associations with a wide range of disease conditions, including various types of cancer such as lung cancer and respiratory system cancer, as well as hematopoietic and immune system diseases. Metabolic disorders are also prominent, with specific mentions of diabetes mellitus, particularly type 2 diabetes mellitus, and carbohydrate metabolism diseases. Cardiovascular conditions are represented by coronary artery disease, artery disease, and vascular disease. Additionally, the data includes nervous system diseases, encompassing cognitive disorders, mood disorders, and specific conditions like schizophrenia and major depressive disorder. Other notable associations include eye diseases, anemia, and hemoglobinopathies.


The genome-wide association studies have identified associations between genetic variants and specific traits. Notably, the trait [Ethylmalonate levels](https://pubmed.ncbi.nlm.nih.gov/36357675) has been mapped to the closest existing MeSH term [Phenylethylmalonamide](https://meshb.nlm.nih.gov/record/ui?ui=D010657) using a method called [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0). Additionally, the trait [Propionylcarnitine levels](https://pubmed.ncbi.nlm.nih.gov/33414548) has been mapped to the closest existing MeSH term [Acetylcarnitine](https://meshb.nlm.nih.gov/record/ui?ui=D000108) using the same method.


The gene is expressed in multiple tissue samples, including the liver, adrenal gland, and kidney, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the thyroid, prostate, pancreas, colon, kidney, and trigeminal ganglion.


The input data highlights various pathways and defects related to metabolism, including fatty acid, triacylglycerol, and ketone body metabolism, as well as the metabolism of lipids, lipoproteins, vitamins, and cofactors. Specific defects mentioned include those in biotin and cobalamin (B12) metabolism, which are linked to conditions such as hereditary megaloblastic anemia, methylmalonic aciduria, and homocystinuria. Additionally, the data covers propionyl-CoA catabolism and mitochondrial fatty acid beta-oxidation.



The analyzed protein sequence has a GRAVY value of -0.145 (74.35th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of 0.905 (53.57th percentile). The median structural flexibility is 0.999 (38.57th percentile), suggesting high flexibility. The protein's secondary structure is predicted to be 32.83% helix (57.91st percentile), 34.62% sheet (56.19th percentile), and 27.61% turn (43.78th percentile). The instability index is 36.47 (13.03rd percentile), and the isoelectric point is 7.24 (52.68th percentile). The protein is 728 amino acids long (77.02nd percentile) with a molecular weight of 80058.03 Da (76.3rd percentile). For more details on the sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |