---
layout: post
title: CD63
date: 2024-12-08 23:57 CST
description: CD63 description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/967) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 967  | CD63 | ENSG00000135404 | 12q13.2 |



The gene length is 96,133 base pairs (81.06% of all genes), the mature length is 2,014 base pairs (53.09% of all genes), and the primary transcript length is 93,665 base pairs (87.37% of all genes).


The gene CD63, with NCBI ID 967, has been featured in [78 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CD63%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1986, and the middle 50% of publications occurred between 1999 and 2019.


The top 5 publications mentioning CD63, ranked by their scientific influence, include "[Specificities of exosome versus small ectosome secretion revealed by live intracellular tracking of CD63 and CD9.](https://pubmed.ncbi.nlm.nih.gov/34282141)" (2021) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 32.77, "[High levels of exosomes expressing CD63 and caveolin-1 in plasma of melanoma patients.](https://pubmed.ncbi.nlm.nih.gov/19381331)" (2009) with an RCR of 16.32, "[The tetraspanin CD63 regulates ESCRT-independent and -dependent endosomal sorting during melanogenesis.](https://pubmed.ncbi.nlm.nih.gov/21962903)" (2011) with an RCR of 15.85, "[CD9, CD63, CD81, and CD82 are components of a surface tetraspan network connected to HLA-DR and VLA integrins.](https://pubmed.ncbi.nlm.nih.gov/8921952)" (1996) with an RCR of 6.41, and "[CD63 is regulated by iron via the IRE-IRP system and is important for ferritin secretion by extracellular vesicles.](https://pubmed.ncbi.nlm.nih.gov/34265052)" (2021) with an RCR of 5.29. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), [ARID2](https://www.ncbi.nlm.nih.gov/gene/196528), [NIPBL](https://www.ncbi.nlm.nih.gov/gene/25836), [TP53](https://www.ncbi.nlm.nih.gov/gene/7157), and [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), each shown to be regulating in 5 experiments.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [CTCF](https://www.ncbi.nlm.nih.gov/gene/6829) (2 experiments), [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885) (4 experiments), [COL1A2](https://www.ncbi.nlm.nih.gov/gene/2118) (1 experiment), [HNF4A](https://www.ncbi.nlm.nih.gov/gene/11091) (1 experiment), [KLF4](https://www.ncbi.nlm.nih.gov/gene/8861) (2 experiments), [ZNF143](https://www.ncbi.nlm.nih.gov/gene/9968) (2 experiments), [ZNF384](https://www.ncbi.nlm.nih.gov/gene/30827) (2 experiments), and [REST](https://www.ncbi.nlm.nih.gov/gene/9126) (2 experiments). These TFs were found to regulate gene expression in multiple Perturb-seq experiments, highlighting their significant roles in transcriptional regulation.








The analyzed protein sequence exhibits a GRAVY value of 0.7672 (98.54th percentile), indicating a high degree of hydrophobicity. The protein has a charge of 2.54 at pH 7.0 (59.48th percentile) and a median structural flexibility of 0.98 (4.38th percentile). The secondary structure affinities are notable, with 34.03% of residues tending to be in a helix (68.13th percentile), 40.76% in a sheet (88.27th percentile), and 22.27% in a turn (7.31st percentile). The instability index is 25.07 (1.57th percentile), and the isoelectric point is 8.14 (60.36th percentile). The protein has a length of 238 amino acids (19.93rd percentile) and a molecular weight of 25636.37 Da (18.89th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |