---
layout: post
title: GADD45G
date: 2025-01-17 16:55 CST
description: GADD45G description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10912) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10912  | GADD45G | ENSG00000130222 | 9q22.2  |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is active in and located in both the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737). It is involved in various biological processes, including the [apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0006915), [cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0030154), [positive regulation of apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0043065), [positive regulation of JNK cascade](https://amigo.geneontology.org/amigo/term/GO:0046330), [regulation of cell cycle](https://amigo.geneontology.org/amigo/term/GO:0051726), [positive regulation of cold-induced thermogenesis](https://amigo.geneontology.org/amigo/term/GO:0120162), and [positive regulation of p38MAPK cascade](https://amigo.geneontology.org/amigo/term/GO:1900745).


The gene length is 16,321 base pairs (37.03% of all genes), the mature length is 4,815 base pairs (89.37% of all genes), and the primary transcript length is 15,452 base pairs (44.83% of all genes).


The gene GADD45G (NCBI ID: 10912) has been mentioned in [53 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22GADD45G%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning this gene is 1999, and the middle 50% of publications occurred between 2008 and 2015.


The top 5 publications mentioning GADD45G, ranked by their scientific influence, include "[GADD45b and GADD45g are cdc2/cyclinB1 kinase inhibitors with a role in S and G2/M cell cycle checkpoints induced by genotoxic stress.](https://pubmed.ncbi.nlm.nih.gov/12124778)" (2002) (relative citation ratio: 4.59), "[The stress-responsive gene GADD45G is a functional tumor suppressor, with its response to environmental stresses frequently disrupted epigenetically in multiple tumors.](https://pubmed.ncbi.nlm.nih.gov/16166418)" (2005) (relative citation ratio: 3.42), "[Gadd45γ and Map3k4 interactions regulate mouse testis determination via p38 MAPK-mediated control of Sry expression.](https://pubmed.ncbi.nlm.nih.gov/23102580)" (2012) (relative citation ratio: 2.84), "[Loss of expression of GADD45 gamma, a growth inhibitory gene, in human pituitary adenomas: implications for tumorigenesis.](https://pubmed.ncbi.nlm.nih.gov/11889197)" (2002) (relative citation ratio: 2.33), and "[SIRT1 activation enhances HDAC inhibition-mediated upregulation of GADD45G by repressing the binding of NF-κB/STAT3 complex to its promoter in malignant lymphoid cells.](https://pubmed.ncbi.nlm.nih.gov/23681230)" (2013) (relative citation ratio: 1.44). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[GADD45G](https://www.proteinatlas.org/ENSG00000130222-GADD45G) is a developmental protein with evidence at the protein level, implicated in cancer-related genes. It is detected in all tissues and is notably expressed in eosinophils involved in protein ubiquitination (Cluster 36), liver metabolism (Cluster 83), non-specific brain transcription (Cluster 25), ciliated cells with ion channel activity (Cluster 10), and syncytiotrophoblasts related to pregnancy (Cluster 27).


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 5 experiments, [SPI1](https://www.ncbi.nlm.nih.gov/gene/6688) with 4 experiments, [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 4 experiments, [JUN](https://www.ncbi.nlm.nih.gov/gene/3725) with 4 experiments, and [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.





The gene is expressed in the liver and pituitary according to BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles indicate expression in the pineal gland (both day and night), CD71+ early erythroid cells, prostate, placenta, pancreatic islet, and uterus corpus.




The analyzed protein sequence has a GRAVY value of -0.036 (82.18th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -18.24 (12.36th percentile), and the median structural flexibility is 1.002 (48.86th percentile). The protein has a propensity for helical structures (35.22%, 76.07th percentile), with a lower tendency for sheet (33.33%, 47.43rd percentile) and turn (28.30%, 50.06th percentile) conformations. The instability index is 32.91 (6.97th percentile), and the isoelectric point is 4.26 (0.61st percentile). The protein is 159 amino acids long (9.15th percentile) with a molecular weight of 17121.06 Da (8.43rd percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |