---
layout: post
title: GATM
date: 2025-01-17 16:55 CST
description: GATM description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2628) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2628  | GATM | ENSG00000171766 | 15q21.1 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [amidinotransferase activity](https://amigo.geneontology.org/amigo/term/GO:0015067), specifically [glycine amidinotransferase activity](https://amigo.geneontology.org/amigo/term/GO:0015068). It is located in the [mitochondrion](https://amigo.geneontology.org/amigo/term/GO:0005739), particularly in the [mitochondrial inner membrane](https://amigo.geneontology.org/amigo/term/GO:0005743) and [mitochondrial intermembrane space](https://amigo.geneontology.org/amigo/term/GO:0005758), as well as in the [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). The gene is involved in the [creatine metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006600) and [creatine biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006601), acting upstream of or within the latter. Additionally, it plays roles in [learning or memory](https://amigo.geneontology.org/amigo/term/GO:0007611), [muscle atrophy](https://amigo.geneontology.org/amigo/term/GO:0014889), and the [positive regulation of cold-induced thermogenesis](https://amigo.geneontology.org/amigo/term/GO:0120162).


The gene length is 43,218 base pairs (62.35% of all genes), the mature length is 3,461.5 base pairs (78.36% of all genes), and the primary transcript length is 39,911 base pairs (69.89% of all genes).


The gene GATM (NCBI ID: 2628) has been mentioned in [70 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22GATM%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1956 and the middle 50% of publications ranging from 1968 to 2010. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning GATM, ranked by their scientific influence, include "[A statin-dependent QTL for GATM expression is associated with statin-induced myopathy.](https://pubmed.ncbi.nlm.nih.gov/23995691)" (2013) (relative citation ratio: 4.58), "[Arginine:glycine amidinotransferase deficiency: the third inborn error of creatine metabolism in humans.](https://pubmed.ncbi.nlm.nih.gov/11555793)" (2001) (relative citation ratio: 4.04), "[Creatine synthesis and transport during rat embryogenesis: spatiotemporal expression of AGAT, GAMT and CT1.](https://pubmed.ncbi.nlm.nih.gov/15918910)" (2005) (relative citation ratio: 2.69), "[AGAT, GAMT and SLC6A8 distribution in the central nervous system, in relation to creatine deficiency syndromes: a review.](https://pubmed.ncbi.nlm.nih.gov/18392746)" (2008) (relative citation ratio: 2.09), and "[Myocardial expression of the arginine:glycine amidinotransferase gene is elevated in heart failure and normalized after recovery: potential implications for local creatine synthesis.](https://pubmed.ncbi.nlm.nih.gov/16820567)" (2006) (relative citation ratio: 1.84). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[GATM](https://www.proteinatlas.org/ENSG00000171766-GATM) is a transferase with evidence at the protein level, implicated in disease variants. It is detected in all tissues and primarily localized in mitochondria. GATM is expressed in B-cells related to immunoglobulins (Cluster 23) and shows non-specific protein processing in tissues (Cluster 68). In the brain, it is associated with oligodendrocytes (Cluster 28), while in cell lines, it is found in rhabdoid cells with unknown function (Cluster 5). Single-cell expression data indicates its presence in proximal tubular cells involved in amino acid metabolism (Cluster 56).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) and [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), each with 6 experiments. Additionally, [TRIM28](https://www.ncbi.nlm.nih.gov/gene/10155), [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227), and [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885) were each found to be regulating in 5 experiments.




The GATM gene has been associated with variations in [Creatine levels](https://pubmed.ncbi.nlm.nih.gov/36635386), which maps closely to the MeSH term [Creatine](https://meshb.nlm.nih.gov/record/ui?ui=D003401) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method. Additionally, it has been linked to [Creatinine levels](https://pubmed.ncbi.nlm.nih.gov/34226706), with the closest MeSH match being [Creatinine](https://meshb.nlm.nih.gov/record/ui?ui=D003404), also identified through embedding matching.


The gene is ubiquitously expressed in multiple tissue samples, including the pancreas, liver, and kidney, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the liver, spinal cord, pancreas, kidney, fetal liver, dorsal root ganglion, and olfactory bulb.


The input data highlights several key pathways involved in metabolism, including creatine metabolism and the metabolism of amino acids and their derivatives.


The mouse ortholog gene 67092, which corresponds to the human ortholog gene 2628, has been associated with various phenotypes. These include abnormal QT variability, abnormal sinus arrhythmia, cardiovascular system phenotype, decreased circulating alkaline phosphatase level, decreased food intake, decreased grip strength, decreased heart rate, decreased locomotor activity, decreased red blood cell distribution width, female infertility, increased circulating amylase level, increased heart rate variability, male infertility, prolonged QRS complex duration, prolonged QT interval, and prolonged RR interval. Notably, some of these phenotypes, such as decreased circulating alkaline phosphatase level, decreased grip strength, decreased heart rate, decreased locomotor activity, decreased red blood cell distribution width, increased circulating amylase level, prolonged QRS complex duration, prolonged QT interval, and prolonged RR interval, were observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.424 (43.39th percentile), indicating a hydrophilic nature. It exhibits a charge of 3.52 at pH 7.0 (62.86th percentile) and a median structural flexibility of 0.999 (36.21st percentile). The protein's secondary structure is predicted to be 27.42% helix (14.01st percentile), 35.70% sheet (64.24th percentile), and 27.90% turn (46.3rd percentile). The instability index is 53.39 (67.72nd percentile), with an isoelectric point of 8.26 (62.18th percentile). The protein has a length of 423 amino acids (48.57th percentile) and a molecular weight of 48454.89 Da (49.99th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |