---
layout: post
title: C1QC
date: 2025-01-17 16:55 CST
description: C1QC description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/714) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 714  | C1QC | ENSG00000159189 | 1p36.12 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615), [synapse](https://amigo.geneontology.org/amigo/term/GO:0045202), [collagen-containing extracellular matrix](https://amigo.geneontology.org/amigo/term/GO:0062023), [blood microparticle](https://amigo.geneontology.org/amigo/term/GO:0072562), and [postsynapse](https://amigo.geneontology.org/amigo/term/GO:0098794). It is part of the [collagen trimer](https://amigo.geneontology.org/amigo/term/GO:0005581), [complement component C1 complex](https://amigo.geneontology.org/amigo/term/GO:0005602), and [complement component C1q complex](https://amigo.geneontology.org/amigo/term/GO:0062167). The gene is involved in the [immune response](https://amigo.geneontology.org/amigo/term/GO:0006955), [complement activation, classical pathway](https://amigo.geneontology.org/amigo/term/GO:0006958), [negative regulation of granulocyte differentiation](https://amigo.geneontology.org/amigo/term/GO:0030853), [innate immune response](https://amigo.geneontology.org/amigo/term/GO:0045087), [negative regulation of macrophage differentiation](https://amigo.geneontology.org/amigo/term/GO:0045650), and [synapse pruning](https://amigo.geneontology.org/amigo/term/GO:0098883).


The gene length is 135,461.0 base pairs (87.34% of all genes), the mature length is 5,554.5 base pairs, and the primary transcript length is 130,029.0 base pairs.


The gene C1QC (NCBI ID: 714) has been mentioned in [4 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22C1QC%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning this gene was in 2014, with the middle 50% of publications occurring between 2017 and 2021.


The top publications mentioning C1QC, ranked by their scientific influence, include "[Complement C1q (C1qA, C1qB, and C1qC) May Be a Potential Prognostic Factor and an Index of Tumor Microenvironment Remodeling in Osteosarcoma.](https://pubmed.ncbi.nlm.nih.gov/34079754)" (2021) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 4.37, "[Multi-Omics Analysis Showed the Clinical Value of Gene Signatures of C1QC<sup>+</sup> and SPP1<sup>+</sup> TAMs in Cervical Cancer.](https://pubmed.ncbi.nlm.nih.gov/34295336)" (2021) with an RCR of 1.55, "[<i>C1QA</i> and <i>C1QC</i> modify age-at-onset in familial amyloid polyneuropathy patients.](https://pubmed.ncbi.nlm.nih.gov/31019999)" (2019) with an RCR of 0.67, and "[Rothmund-Thomson syndrome and glomerulonephritis in a homozygous C1q-deficient patient due to a Gly164Ser C1qC mutation.](https://pubmed.ncbi.nlm.nih.gov/24157463)" (2014) with an RCR of 0.15. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[C1QC](https://www.proteinatlas.org/ENSG00000159189-C1QC) is a gene with evidence at the protein level and is implicated in disease variants. Its RNA is detected in all tissues, with notable expression in lymphoid tissue, macrophages, and microglia, suggesting a significant role in immune response. The gene is particularly expressed in monocytes and myeloid leukemia cell lines, further emphasizing its involvement in innate immune processes.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [RELA](https://www.ncbi.nlm.nih.gov/gene/5970) and [SPI1](https://www.ncbi.nlm.nih.gov/gene/6688), each shown to be regulating in 2 experiments. Additionally, [EP300](https://www.ncbi.nlm.nih.gov/gene/2033) was also found to be regulating in 2 experiments. Furthermore, [IRF1](https://www.ncbi.nlm.nih.gov/gene/3659) and [MYB](https://www.ncbi.nlm.nih.gov/gene/4602) were each identified in 1 experiment.





The gene is expressed in various tissues, including the lung and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the heart, CD105+ endothelial cells, lung, placenta, small intestine, thymus, and lymph nodes, suggesting a ubiquitous expression pattern across multiple tissue samples.


The input data highlights several pathways within the immune system, particularly focusing on the innate immune system. Key processes include classical antibody-mediated complement activation, the creation of C4 and C2 activators, initial triggering of complement, and the overall complement cascade.


The mouse ortholog gene 12262 and its human ortholog gene 714 are associated with several phenotypes, including abnormal autopod morphology, abnormal startle reflex, and increased circulating aspartate transaminase level. These phenotypes are observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.186 (70.61 percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is 2.995 (61.31 percentile), and the median structural flexibility is 1.005 (61.86 percentile). The protein has a predicted affinity for helix, sheet, and turn structures of 24.49%, 34.29%, and 41.22%, respectively. The instability index is 29.07 (3.34 percentile), suggesting a stable protein. The isoelectric point is 8.61 (69.15 percentile), with a molecular weight of 25773.35 Da (19.07 percentile). The sequence length is 245 amino acids (20.94 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |