---
layout: post
title: CD7
date: 2025-01-17 16:55 CST
description: CD7 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/924) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 924  | CD7 | ENSG00000173762 | 17q25.3 |



The gene is located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886) and [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020). It is involved in the [adaptive immune response](https://amigo.geneontology.org/amigo/term/GO:0002250), [immune response](https://amigo.geneontology.org/amigo/term/GO:0006955), [transmembrane receptor protein tyrosine kinase signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007169), and [T cell activation](https://amigo.geneontology.org/amigo/term/GO:0042110). The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [signaling receptor activity](https://amigo.geneontology.org/amigo/term/GO:0038023).


The gene length is 13,165 base pairs (31.91 percentile of all genes), the mature length is 1,723 base pairs, and the primary transcript length is 13,134 base pairs.


The gene CD7 (NCBI ID: 924) has been mentioned in [355 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CD7%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning CD7 is 1951, and the middle 50% of publications occurred between 1993 and 2006.


The top 5 publications mentioning CD7, ranked by their scientific influence, include "[Donor-Derived CD7 Chimeric Antigen Receptor T Cells for T-Cell Acute Lymphoblastic Leukemia: First-in-Human, Phase I Trial.](https://pubmed.ncbi.nlm.nih.gov/34324392)" (2021) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 10.96, "[Naturally selected CD7 CAR-T therapy without genetic manipulations for T-ALL/LBL: first-in-human phase 1 clinical trial.](https://pubmed.ncbi.nlm.nih.gov/35500125)" (2022) with an RCR of 10.16, "[CD4+ CD7- CD28- T cells are expanded in rheumatoid arthritis and are characterized by autoreactivity.](https://pubmed.ncbi.nlm.nih.gov/8621791)" (1996) with an RCR of 8.47, "[CD29 and CD7 mediate galectin-3-induced type II T-cell apoptosis.](https://pubmed.ncbi.nlm.nih.gov/14678989)" (2003) with an RCR of 5.52, and "[Leu-8 and Leu-9 antigen phenotypes: immunologic criteria for the distinction of mycosis fungoides from cutaneous inflammation.](https://pubmed.ncbi.nlm.nih.gov/3088069)" (1986) with an RCR of 5.47. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CD7](https://www.proteinatlas.org/ENSG00000173762-CD7) is a receptor with evidence at the protein level. It is detected in many tissues and is associated with various expression clusters, including Cluster 1: T-cells - Unknown function in blood, Cluster 70: Lymphoid tissue - Immune response in tissues, Cluster 44: Non-specific - Mixed function in the brain, Cluster 59: Leukemia - T-cell receptor in cell lines, and Cluster 19: NK-cells - Adaptive immune response in single cells.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) with 6 experiments, [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) each with 4 experiments, [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313) with 3 experiments, and [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) with 2 experiments.





The gene is ubiquitously expressed in multiple tissue samples, including blood, spleen, and small intestine, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in various immune-related tissues and cells, such as CD56+ NK cells, CD4+ T cells, CD8+ T cells, whole blood, tonsil, thymus, and lymph node.




The analyzed protein sequence has a GRAVY value of -0.022 (82.89th percentile), indicating a slightly hydrophilic nature. The charge at pH 7.0 is -0.211 (49.54th percentile), and the median structural flexibility is 0.995 (20.63rd percentile). The protein has a propensity for helix, sheet, and turn structures of 27.92% (16.5th percentile), 35% (59.19th percentile), and 32.5% (79.39th percentile), respectively. The instability index is 54.97 (72.13rd percentile), suggesting moderate stability. The isoelectric point is 6.87 (48.84th percentile), with a molecular weight of 25408.72 Da (18.61st percentile) and a length of 240 amino acids (20.22nd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |