---
layout: post
title: FGFR1
date: 2025-01-17 16:55 CST
description: FGFR1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2260) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2260  | FGFR1 | ENSG00000077782 | 8p11.23 |



The gene is involved in various biological processes, including [negative regulation of transcription by RNA polymerase II](https://amigo.geneontology.org/amigo/term/GO:0000122), [MAPK cascade](https://amigo.geneontology.org/amigo/term/GO:0000165), [skeletal system development](https://amigo.geneontology.org/amigo/term/GO:0001501), [angiogenesis](https://amigo.geneontology.org/amigo/term/GO:0001525), and [ureteric bud development](https://amigo.geneontology.org/amigo/term/GO:0001657). It enables functions such as [protein tyrosine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004713) and [fibroblast growth factor receptor activity](https://amigo.geneontology.org/amigo/term/GO:0005007). The gene is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). Additionally, it is involved in processes like [protein phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0006468), [transmembrane receptor protein tyrosine kinase signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007169), and [multicellular organism development](https://amigo.geneontology.org/amigo/term/GO:0007275). The gene also plays a role in [sensory perception of sound](https://amigo.geneontology.org/amigo/term/GO:0007605) and enables [heparin binding](https://amigo.geneontology.org/amigo/term/GO:0008201). It is involved in the [positive regulation of cell population proliferation](https://amigo.geneontology.org/amigo/term/GO:0008284) and the [fibroblast growth factor receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0008543). Furthermore, the gene is involved in [mesenchymal cell proliferation](https://amigo.geneontology.org/amigo/term/GO:0010463), [gene expression](https://amigo.geneontology.org/amigo/term/GO:0010467), and the [positive regulation of phospholipase activity](https://amigo.geneontology.org/amigo/term/GO:0010518). It is located in the [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020) and is involved in [cell migration](https://amigo.geneontology.org/amigo/term/GO:0016477). The gene enables [fibroblast growth factor binding](https://amigo.geneontology.org/amigo/term/GO:0017134) and is involved in [peptidyl-tyrosine phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0018108). It plays a role in [ventricular zone neuroblast division](https://amigo.geneontology.org/amigo/term/GO:0021847) and [cell projection assembly](https://amigo.geneontology.org/amigo/term/GO:0030031). The gene is involved in [embryonic limb morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0030326) and [midbrain development](https://amigo.geneontology.org/amigo/term/GO:0030901). It is located in the [cytoplasmic vesicle](https://amigo.geneontology.org/amigo/term/GO:0031410) and is involved in the [positive regulation of kinase activity](https://amigo.geneontology.org/amigo/term/GO:0033674). The gene enables [SH2 domain binding](https://amigo.geneontology.org/amigo/term/GO:0042169) and [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802). It is involved in [chordate embryonic development](https://amigo.geneontology.org/amigo/term/GO:0043009) and is part of the [receptor complex](https://amigo.geneontology.org/amigo/term/GO:0043235). The gene is involved in the [positive regulation of MAP kinase activity](https://amigo.geneontology.org/amigo/term/GO:0043406) and the [positive regulation of MAPK cascade](https://amigo.geneontology.org/amigo/term/GO:0043410). It plays a role in the [positive regulation of blood vessel endothelial cell migration](https://amigo.geneontology.org/amigo/term/GO:0043536) and the [regulation of cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0045595). The gene is involved in the [positive regulation of neuron differentiation](https://amigo.geneontology.org/amigo/term/GO:0045666) and [protein autophosphorylation](https://amigo.geneontology.org/amigo/term/GO:0046777). It is involved in [phosphatidylinositol-mediated signaling](https://amigo.geneontology.org/amigo/term/GO:0048015) and [paraxial mesoderm development](https://amigo.geneontology.org/amigo/term/GO:0048339). The gene plays a role in the [regulation of lateral mesodermal cell fate specification](https://amigo.geneontology.org/amigo/term/GO:0048378) and [cell maturation](https://amigo.geneontology.org/amigo/term/GO:0048469). It is involved in [skeletal system morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0048705) and [stem cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0048863). The gene is involved in the [positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction](https://amigo.geneontology.org/amigo/term/GO:0051897) and [calcium ion homeostasis](https://amigo.geneontology.org/amigo/term/GO:0055074). It plays a role in [cardiac muscle cell proliferation](https://amigo.geneontology.org/amigo/term/GO:0060038) and the [positive regulation of cardiac muscle cell proliferation](https://amigo.geneontology.org/amigo/term/GO:0060045). The gene is involved in [auditory receptor cell development](https://amigo.geneontology.org/amigo/term/GO:0060117) and the [branching involved in salivary gland morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0060445). It plays a role in the [regulation of branching involved in salivary gland morphogenesis by mesenchymal-epithelial signaling](https://amigo.geneontology.org/amigo/term/GO:0060665) and the [vitamin D3 metabolic process](https://amigo.geneontology.org/amigo/term/GO:0070640). The gene is involved in the [diphosphate metabolic process](https://amigo.geneontology.org/amigo/term/GO:0071344) and [cementum mineralization](https://amigo.geneontology.org/amigo/term/GO:0071529). It plays a role in [stem cell proliferation](https://amigo.geneontology.org/amigo/term/GO:0072089) and the [positive regulation of MAPKKK cascade by fibroblast growth factor receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0090080). The gene is involved in the [negative regulation of fibroblast growth factor production](https://amigo.geneontology.org/amigo/term/GO:0090272) and enables [receptor-receptor interaction](https://amigo.geneontology.org/amigo/term/GO:0090722). It is involved in the [positive regulation of mitotic cell cycle DNA replication](https://amigo.geneontology.org/amigo/term/GO:1903465) and the [response to sodium phosphate](https://amigo.geneontology.org/amigo/term/GO:1904383). The gene plays a role in the [positive regulation of vascular endothelial cell proliferation](https://amigo.geneontology.org/amigo/term/GO:1905564) and the [positive regulation of endothelial cell chemotaxis to fibroblast growth factor](https://amigo.geneontology.org/amigo/term/GO:2000546). It is involved in the [positive regulation of stem cell proliferation](https://amigo.geneontology.org/amigo/term/GO:2000648) and the [positive regulation of parathyroid hormone secretion](https://amigo.geneontology.org/amigo/term/GO:2000830). Finally, the gene is involved in the [regulation of extrinsic apoptotic signaling pathway in absence of ligand](https://amigo.geneontology.org/amigo/term/GO:2001239).


The gene has a length of 25,608 base pairs (48.3% of all genes), a mature length of 3,319 base pairs, and a primary transcript length of 25,608 base pairs.


The gene FGFR1 (NCBI ID: 2260) has been mentioned in [850 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22FGFR1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning this gene is 1963, and the middle 50% of publications occurred between 2009 and 2019.


The top five publications mentioning FGFR1, ranked by their scientific influence, include "[FGFR1 amplification drives endocrine therapy resistance and is a therapeutic target in breast cancer.](https://pubmed.ncbi.nlm.nih.gov/20179196)" (2010) (relative citation ratio: 12.19), "[Upregulated lncRNA-UCA1 contributes to progression of hepatocellular carcinoma through inhibition of miR-216b and activation of FGFR1/ERK signaling pathway.](https://pubmed.ncbi.nlm.nih.gov/25760077)" (2015) (relative citation ratio: 11.29), "[Loss-of-function mutations in FGFR1 cause autosomal dominant Kallmann syndrome.](https://pubmed.ncbi.nlm.nih.gov/12627230)" (2003) (relative citation ratio: 10.68), "[Human fibroblast growth factor receptor 1 is a co-receptor for infection by adeno-associated virus 2.](https://pubmed.ncbi.nlm.nih.gov/9883842)" (1999) (relative citation ratio: 10.66), and "[A common mutation in the fibroblast growth factor receptor 1 gene in Pfeiffer syndrome.](https://pubmed.ncbi.nlm.nih.gov/7874169)" (1994) (relative citation ratio: 10.59). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[FGFR1](https://www.proteinatlas.org/ENSG00000077782-FGFR1) is a multifunctional gene with roles as a heparin-binding protein, kinase, receptor, transferase, and tyrosine-protein kinase. It is implicated in various diseases, including cancer, craniosynostosis, dwarfism, holoprosencephaly, hypogonadotropic hypogonadism, intellectual disability, and Kallmann syndrome. Evidence supports its function at the protein level. The gene is detected in all RNA tissue distributions and is localized in microtubules, cytokinetic bridge, mitotic spindle, primary cilium, centriolar satellite, basal body, and cytosol. Its expression is clustered in smooth muscle tissue for ECM organization and is found in fibroblasts for ECM organization in single-cell expression clusters.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) and [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), each with 6 experiments. Additionally, [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313), [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885), and [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) were each found to be regulating in 5 experiments.



The GWAS data indicates associations with several disease conditions, including coronary artery disease, artery disease, vascular disease, and cardiovascular system disease. Additionally, broader categories such as disease and disease of anatomical entity are also implicated.



The gene is expressed in various tissues, including the ovary, cerebellum, appendix, trigeminal ganglion, and uterus.


The input data highlights various pathways involving FGFR signaling, including FGFR ligand binding and activation, FGFR1 ligand binding and activation, and signaling by FGFR mutants. Additionally, it covers immune system pathways such as the adaptive and innate immune systems, as well as PI3K/AKT signaling in cancer and related cascades. The data also includes pathways related to EGFR signaling in cancer, insulin receptor signaling, and developmental biology processes like axon guidance and NCAM signaling for neurite outgrowth. Furthermore, it encompasses signal transduction events mediated by various receptors and ligands, including DAP12, LAT2, and NGF signaling via TRKA.


The mouse ortholog gene 14182 and its human ortholog gene 2260 are associated with several phenotypic abnormalities. Both sexes exhibit abnormal embryo size and embryonic growth retardation. Additionally, there is embryonic lethality prior to the tooth bud stage, although this is not observed in both sexes. Furthermore, complete penetrance preweaning lethality is noted in both sexes.


The analyzed protein sequence has a GRAVY value of -0.367 (50.27th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -14.64 (15.81st percentile), and the median structural flexibility is 1.002 (52.93rd percentile). The protein has a helix affinity of 0.315 (45.83rd percentile), a sheet affinity of 0.341 (52.47th percentile), and a turn affinity of 0.315 (73.88th percentile). The instability index is 52.26 (64.45th percentile), with an isoelectric point of 5.82 (25.86th percentile). The sequence length is 822 amino acids (81.57th percentile), and the molecular weight is 91866.73 Da (81.51st percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |