---
layout: post
title: EFTUD2
date: 2025-01-17 16:55 CST
description: EFTUD2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9343) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9343  | EFTUD2 | ENSG00000108883 | 17q21.31 |



The gene is involved in [mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0000398) and [response to cocaine](https://amigo.geneontology.org/amigo/term/GO:0042220), as well as [cellular response to xenobiotic stimulus](https://amigo.geneontology.org/amigo/term/GO:0071466). It enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0003924), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [GTP binding](https://amigo.geneontology.org/amigo/term/GO:0005525), and [U5 snRNA binding](https://amigo.geneontology.org/amigo/term/GO:0030623). The gene is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [Cajal body](https://amigo.geneontology.org/amigo/term/GO:0015030), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [nuclear speck](https://amigo.geneontology.org/amigo/term/GO:0016607), and [intracellular membrane-bounded organelle](https://amigo.geneontology.org/amigo/term/GO:0043231). Additionally, it is part of the [U4/U6 x U5 tri-snRNP complex](https://amigo.geneontology.org/amigo/term/GO:0046540), [U2-type precatalytic spliceosome](https://amigo.geneontology.org/amigo/term/GO:0071005), [U2-type catalytic step 2 spliceosome](https://amigo.geneontology.org/amigo/term/GO:0071007), and [catalytic step 2 spliceosome](https://amigo.geneontology.org/amigo/term/GO:0071013).


The gene length is 1,637 base pairs (4.27% of all genes). The mature length is 1,637 base pairs, and the primary transcript length is 1,637 base pairs.


The gene EFTUD2 (NCBI ID: 9343) has been mentioned in [35 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22EFTUD2%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 2006 and the middle 50% of publications occurring between 2014 and 2020. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning EFTUD2 include "[Haploinsufficiency of a spliceosomal GTPase encoded by EFTUD2 causes mandibulofacial dysostosis with microcephaly.](https://pubmed.ncbi.nlm.nih.gov/22305528)" (2012) (relative citation ratio: 3.6), "[The EF-G-like GTPase Snu114p regulates spliceosome dynamics mediated by Brr2p, a DExD/H box ATPase.](https://pubmed.ncbi.nlm.nih.gov/16885028)" (2006) (relative citation ratio: 2.36), "[EFTUD2 haploinsufficiency leads to syndromic oesophageal atresia.](https://pubmed.ncbi.nlm.nih.gov/23188108)" (2012) (relative citation ratio: 2.14), "[Mutation in Eftud2 causes craniofacial defects in mice via mis-splicing of Mdm2 and increased P53.](https://pubmed.ncbi.nlm.nih.gov/33601405)" (2021) (relative citation ratio: 1.94), and "[Spliceosomal protein eftud2 mutation leads to p53-dependent apoptosis in zebrafish neural progenitors.](https://pubmed.ncbi.nlm.nih.gov/27899647)" (2017) (relative citation ratio: 1.79). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[EFTUD2](https://www.proteinatlas.org/ENSG00000108883-EFTUD2) is a gene with evidence at the protein level, implicated in intellectual disability. It is detected in all RNA tissue distributions and is primarily localized in the nucleoplasm, vesicles, and cytosol. The gene is expressed in various clusters, including Cluster 29 for blood, Cluster 19 for tissue, Cluster 14 for brain, Cluster 27 for cell lines, and Cluster 39 for single cells, suggesting its involvement in basic cellular processes and synaptic function in neurons.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [PPARG](https://www.ncbi.nlm.nih.gov/gene/5468), [MAX](https://www.ncbi.nlm.nih.gov/gene/4149), [REST](https://www.ncbi.nlm.nih.gov/gene/5978), [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), and [AR](https://www.ncbi.nlm.nih.gov/gene/367), each shown to be regulating in 5 experiments.



The GWAS data indicates associations with various metabolic and mental health conditions, including diseases of metabolism, acquired metabolic diseases, carbohydrate metabolism diseases, glucose metabolism diseases, diabetes mellitus, and specifically type 2 diabetes mellitus. Additionally, the data highlights associations with dementia, cognitive disorders, and diseases of mental health.



The gene is expressed in various tissues, including muscle, skin, adrenal gland, and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the kidney, bone marrow, superior cervical ganglion, trigeminal ganglion, ciliary ganglion, dorsal root ganglion, and ovary.


The input data highlights several key biological processes, including gene expression and various stages of mRNA processing. Specifically, it covers the processing of capped intron-containing pre-mRNA, as well as both major and minor pathways of mRNA splicing.


The mouse ortholog gene 20624, which corresponds to the human ortholog gene 9343, is associated with several phenotypic traits. These include decreased and increased heart left ventricle size, increased circulating HDL cholesterol level, increased circulating cholesterol level, increased circulating serum albumin level, and preweaning lethality with complete penetrance. All these phenotypes are observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.245 (64.36th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -54.28 (2.05th percentile). The median structural flexibility is 0.99995 (41.38th percentile), with a propensity for helix, sheet, and turn structures of 30.14% (33.27th percentile), 39.40% (84.43rd percentile), and 28.19% (49.02nd percentile), respectively. The instability index is 38.24 (17.49th percentile), and the isoelectric point is 4.84 (5.12th percentile). The protein is 972 amino acids long (86.83rd percentile) with a molecular weight of 109434.48 Da (87.06th percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |