---
layout: post
title: POLR2A
date: 2024-12-08 23:57 CST
description: POLR2A description
tags: [cooccuring-genes]
categories: shortest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5430) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5430  | POLR2A | ENSG00000181222 | 17p13.1 |



The gene length is 496.0 base pairs (0.47% of all genes), with a mature length of 496.0 base pairs and a primary transcript length of 496.0 base pairs.


The gene POLR2A, with NCBI ID 5430, has been the subject of [34 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22POLR2A%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727). The earliest year of publication is 1995, and the middle 50% of publications occurred between 2013 and 2020. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning POLR2A, ranked by their scientific influence, include "[Phylogenetic analyses of RPB1 and RPB2 support a middle Cretaceous origin for a clade comprising all agriculturally and medically important fusaria](https://pubmed.ncbi.nlm.nih.gov/23357352)" (2013) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 11.62, "[Recurrent somatic mutations in POLR2A define a distinct subset of meningiomas](https://pubmed.ncbi.nlm.nih.gov/27548314)" (2016) with an RCR of 8.2, "[Evasion of the innate immune response: the Old World alphavirus nsP2 protein induces rapid degradation of Rpb1, a catalytic subunit of RNA polymerase II](https://pubmed.ncbi.nlm.nih.gov/22514352)" (2012) with an RCR of 4.2, "[Precise targeting of POLR2A as a therapeutic strategy for human triple negative breast cancer](https://pubmed.ncbi.nlm.nih.gov/30804480)" (2019) with an RCR of 4.13, and "[Live-cell analysis of endogenous GFP-RPB1 uncovers rapid turnover of initiating and promoter-paused RNA Polymerase II](https://pubmed.ncbi.nlm.nih.gov/29632207)" (2018) with an RCR of 3.84. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [NCOR1](https://www.ncbi.nlm.nih.gov/gene/9611) with 5 experiments, [TBX21](https://www.ncbi.nlm.nih.gov/gene/30009) with 4 experiments, [GATA3](https://www.ncbi.nlm.nih.gov/gene/2625) with 4 experiments, [STAT3](https://www.ncbi.nlm.nih.gov/gene/6774) with 4 experiments, and [NEUROD1](https://www.ncbi.nlm.nih.gov/gene/4760) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [TCF7L2](https://www.ncbi.nlm.nih.gov/gene/6872), which was shown to be regulating genes in 4 experiments. Additionally, [TCF7](https://www.ncbi.nlm.nih.gov/gene/7023) and [TCF7L1](https://www.ncbi.nlm.nih.gov/gene/8607) were each found to be regulating genes in 1 experiment.








The analyzed protein sequence has a GRAVY value of -0.534 (31.61% percentile), indicating a hydrophilic nature. The charge at pH 7.0 is 0.16 (50.98% percentile), and the median structural flexibility is 1.01 (76.6% percentile). The protein's secondary structure tends to be in helix, sheet, and turn conformations at 25.79% (6.95% percentile), 33.05% (45.77% percentile), and 36.8% (93.29% percentile), respectively. The instability index is 63.18 (88.09% percentile), suggesting high instability. The isoelectric point is 7.02 (50.54% percentile), with a length of 1970 amino acids (97.5% percentile) and a molecular weight of 217173.66 Da (97.38% percentile). For more details on these calculations, refer to [BioPython's sequence analysis](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |