---
layout: post
title: RABGGTA
date: 2025-01-17 16:55 CST
description: RABGGTA description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5875) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5875  | RABGGTA | ENSG00000100949 | 14q12 |



The gene contributes to and enables [Rab geranylgeranyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0004663) and enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [zinc ion binding](https://amigo.geneontology.org/amigo/term/GO:0008270). It is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829) and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), and is part of the [Rab-protein geranylgeranyltransferase complex](https://amigo.geneontology.org/amigo/term/GO:0005968). The gene is involved in [visual perception](https://amigo.geneontology.org/amigo/term/GO:0007601), [protein geranylgeranylation](https://amigo.geneontology.org/amigo/term/GO:0018344), and the [protein modification process](https://amigo.geneontology.org/amigo/term/GO:0036211). Additionally, it enables [small GTPase binding](https://amigo.geneontology.org/amigo/term/GO:0031267).


The gene length is 494.0 base pairs (0.45 percentile of all genes), with a mature length of 494.0 base pairs and a primary transcript length of 494.0 base pairs.


There are no publications that mention this gene.



[RABGGTA](https://www.proteinatlas.org/ENSG00000100949-RABGGTA) is a gene with prenyltransferase and transferase molecular functions, supported by evidence at the protein level. Its RNA is detected in all tissues, with notable expression in squamous epithelium related to keratinization. In blood, it is predominantly expressed in monocytes and neutrophils, suggesting a role in innate immune response. The gene is also expressed in respiratory epithelial cells, although its function in this context is unknown.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [KLF4](https://www.ncbi.nlm.nih.gov/gene/9314), [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170), [TBP](https://www.ncbi.nlm.nih.gov/gene/6908), [SP1](https://www.ncbi.nlm.nih.gov/gene/6667), and [HIF1A](https://www.ncbi.nlm.nih.gov/gene/3091). Each of these TFs, except HIF1A, was shown to be regulating in 5 experiments, while HIF1A was found in 4 experiments.





The gene is expressed in various tissues, including the esophagus, salivary gland, pituitary, and spleen. Additionally, it is expressed in the heart, liver, B lymphoblasts, NK cells, B cells, monocytes, whole brain, and bronchial epithelial cells.




The analyzed protein sequence has a GRAVY value of -0.249 (63.97th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -16.24 (14.25th percentile). The median structural flexibility is 0.996 (23.78th percentile), with a significant portion of amino acids predicted to be in helix (38.27%, 89.51st percentile) and sheet (37.74%, 77.15th percentile) structures. The instability index is 58.28 (79.7th percentile), suggesting potential instability. The isoelectric point is 5.45 (17.12th percentile), and the molecular weight is 65071.03 Da (67.12th percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |