---
layout: post
title: YBX1
date: 2024-12-08 23:57 CST
description: YBX1 description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4904) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4904  | YBX1 | ENSG00000065978 | 1p34.2 |



The gene length is 401,178.0 base pairs (97.43% of all genes), the mature length is 2,643.5 base pairs, and the primary transcript length is 171,737.5 base pairs.


The gene YBX1, with NCBI ID 4904, has been the subject of [587 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22YBX1%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1979, and the middle 50% of publications occurred between 2009 and 2019.


The top 5 publications mentioning YBX1, ranked by their scientific influence, include "[Circular RNA ACTN4 promotes intrahepatic cholangiocarcinoma progression by recruiting YBX1 to initiate FZD7 transcription.](https://pubmed.ncbi.nlm.nih.gov/34509526)" (2022) with a relative citation ratio of 18.56, "[Endogenous tRNA-Derived Fragments Suppress Breast Cancer Progression via YBX1 Displacement.](https://pubmed.ncbi.nlm.nih.gov/25957686)" (2015) with a relative citation ratio of 16.39, "[LNCAROD is stabilized by m6A methylation and promotes cancer progression via forming a ternary complex with HSPA1A and YBX1 in head and neck squamous cell carcinoma.](https://pubmed.ncbi.nlm.nih.gov/32216017)" (2020) with a relative citation ratio of 7.42, "[Translational activation of snail1 and other developmentally regulated transcription factors by YB-1 promotes an epithelial-mesenchymal transition.](https://pubmed.ncbi.nlm.nih.gov/19411069)" (2009) with a relative citation ratio of 7.26, and "[The pleiotropic functions of the Y-box-binding protein, YB-1.](https://pubmed.ncbi.nlm.nih.gov/12815724)" (2003) with a relative citation ratio of 7.18. The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046) with 6 experiments, [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664) with 5 experiments, [EP300](https://www.ncbi.nlm.nih.gov/gene/2033) with 5 experiments, [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908) with 5 experiments, and [TAL1](https://www.ncbi.nlm.nih.gov/gene/6886) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [CTCF](https://www.ncbi.nlm.nih.gov/gene/6829) and [BRCA1](https://www.ncbi.nlm.nih.gov/gene/6872), each shown to regulate genes in three experiments. Additionally, [NFE2L2](https://www.ncbi.nlm.nih.gov/gene/4849), [GATA1](https://www.ncbi.nlm.nih.gov/gene/2623), [LRRK2](https://www.ncbi.nlm.nih.gov/gene/2118), [KLF1](https://www.ncbi.nlm.nih.gov/gene/6426), [BCL6](https://www.ncbi.nlm.nih.gov/gene/8861), and [TCF7L2](https://www.ncbi.nlm.nih.gov/gene/9968) were each found to regulate genes in two experiments. Furthermore, [ZNF274](https://www.ncbi.nlm.nih.gov/gene/8243), [ZNF273](https://www.ncbi.nlm.nih.gov/gene/6830), and [KLF12](https://www.ncbi.nlm.nih.gov/gene/9126) were each identified in one experiment.








The analyzed protein sequence has a GRAVY value of -1.4842592592592594 (0.65 percentile), indicating a hydrophilic nature. It exhibits a charge of 14.738034681244585 at pH 7 (88.79 percentile) and a median structural flexibility of 1.021654761904762 (97.62 percentile). The protein's secondary structure affinities are 24.69% for helix (4.5 percentile), 18.52% for sheet (1.39 percentile), and 37.96% for turn (95.27 percentile). The instability index is 65.90617283950617 (91.24 percentile), with an isoelectric point of 9.874896430969237 (95.52 percentile). The protein has a length of 324 amino acids (33.62 percentile) and a molecular weight of 35923.7455 Da (33.42 percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |