---
layout: post
title: STMN1
date: 2024-12-08 23:57 CST
description: STMN1 description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3925) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3925  | STMN1 | ENSG00000117632 | 1p36.11 |



The gene length is 29,441 base pairs (52.19% of all genes), the mature length is 950 base pairs (20.47% of all genes), and the primary transcript length is 27,983 base pairs (60.77% of all genes).


The gene STMN1 (NCBI ID: 3925) has been featured in [588 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22STMN1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1927, and the middle 50% of publications occurred between 2006 and 2017.


The top 5 publications mentioning STMN1, ranked by their scientific influence, include "[Lag signs in the diagnosis of rotator cuff rupture.](https://pubmed.ncbi.nlm.nih.gov/8872929)" (1996) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 9.88, "[MicroRNA-223 is commonly repressed in hepatocellular carcinoma and potentiates expression of Stathmin1.](https://pubmed.ncbi.nlm.nih.gov/18555017)" (2008) with an RCR of 8.13, "[Chronotype, Social Jet Lag, and Cardiometabolic Risk Factors in Early Adolescence.](https://pubmed.ncbi.nlm.nih.gov/31524936)" (2019) with an RCR of 7.41, "[Lag Phase Is a Dynamic, Organized, Adaptive, and Evolvable Period That Prepares Bacteria for Cell Division.](https://pubmed.ncbi.nlm.nih.gov/30642990)" (2019) with an RCR of 7.41, and "[Managing Travel Fatigue and Jet Lag in Athletes: A Review and Consensus Statement.](https://pubmed.ncbi.nlm.nih.gov/34263388)" (2021) with an RCR of 6.5. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), [JUN](https://www.ncbi.nlm.nih.gov/gene/3725), [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), [SUPT5H](https://www.ncbi.nlm.nih.gov/gene/6829), and [MAX](https://www.ncbi.nlm.nih.gov/gene/4149), each shown to be regulating in 5 experiments.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, filtered for Perturb-seq data, include [6829](https://www.ncbi.nlm.nih.gov/gene/6829) with 5 experiments, [23028](https://www.ncbi.nlm.nih.gov/gene/23028) with 2 experiments, and [8243](https://www.ncbi.nlm.nih.gov/gene/8243) with 1 experiment. These TFs were found to regulate gene expression in multiple Perturb-seq experiments, highlighting their significant roles in transcriptional regulation.








The analyzed protein sequence has a GRAVY value of -1.311 (1.19 percentile), indicating a hydrophilic nature. The charge at pH 7 is -4.08 (36.31 percentile), and the median structural flexibility is 1.036 (99.72 percentile). The protein has a high affinity for helix structures (57.05%, 99.94 percentile) and a moderate affinity for turn structures (20.13%, 2.62 percentile), with a lower affinity for sheet structures (18.79%, 1.52 percentile). The instability index is 70.87 (95.14 percentile), suggesting the protein may be unstable. The isoelectric point is 5.76 (24.44 percentile), with a molecular weight of 17302.31 Da (8.64 percentile). The sequence length is 149 amino acids (7.97 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |