---
layout: post
title: LCK
date: 2025-01-17 16:55 CST
description: LCK description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3932) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3932  | LCK | ENSG00000182866 | 1p35.2 |



The gene is located in the [pericentriolar material](https://amigo.geneontology.org/amigo/term/GO:0000242), [immunological synapse](https://amigo.geneontology.org/amigo/term/GO:0001772), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [membrane raft](https://amigo.geneontology.org/amigo/term/GO:0045121), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). It enables [phosphotyrosine residue binding](https://amigo.geneontology.org/amigo/term/GO:0001784), [protein tyrosine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004713), [non-membrane spanning protein tyrosine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004715), [protein serine/threonine phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0004722), [signaling receptor binding](https://amigo.geneontology.org/amigo/term/GO:0005102), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524), [phospholipase activator activity](https://amigo.geneontology.org/amigo/term/GO:0016004), [protein kinase binding](https://amigo.geneontology.org/amigo/term/GO:0019901), [protein phosphatase binding](https://amigo.geneontology.org/amigo/term/GO:0019903), [SH2 domain binding](https://amigo.geneontology.org/amigo/term/GO:0042169), [T cell receptor binding](https://amigo.geneontology.org/amigo/term/GO:0042608), [CD4 receptor binding](https://amigo.geneontology.org/amigo/term/GO:0042609), [CD8 receptor binding](https://amigo.geneontology.org/amigo/term/GO:0042610), [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802), [phospholipase binding](https://amigo.geneontology.org/amigo/term/GO:0043274), [phosphatidylinositol 3-kinase binding](https://amigo.geneontology.org/amigo/term/GO:0043548), and [ATPase binding](https://amigo.geneontology.org/amigo/term/GO:0051117). It is involved in [protein phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0006468), [intracellular zinc ion homeostasis](https://amigo.geneontology.org/amigo/term/GO:0006882), [activation of cysteine-type endopeptidase activity involved in apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0006919), [transmembrane receptor protein tyrosine kinase signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007169), [response to xenobiotic stimulus](https://amigo.geneontology.org/amigo/term/GO:0009410), [peptidyl-tyrosine phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0018108), [hemopoiesis](https://amigo.geneontology.org/amigo/term/GO:0030097), [platelet activation](https://amigo.geneontology.org/amigo/term/GO:0030168), [T cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0030217), [T cell costimulation](https://amigo.geneontology.org/amigo/term/GO:0031295), [positive regulation of heterotypic cell-cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0034116), [intracellular signal transduction](https://amigo.geneontology.org/amigo/term/GO:0035556), [peptidyl-tyrosine autophosphorylation](https://amigo.geneontology.org/amigo/term/GO:0038083), [Fc-gamma receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0038094), [innate immune response](https://amigo.geneontology.org/amigo/term/GO:0045087), [T cell receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0050852), [B cell receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0050853), [positive regulation of T cell receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0050862), [positive regulation of T cell activation](https://amigo.geneontology.org/amigo/term/GO:0050870), [leukocyte migration](https://amigo.geneontology.org/amigo/term/GO:0050900), [release of sequestered calcium ion into cytosol](https://amigo.geneontology.org/amigo/term/GO:0051209), [regulation of lymphocyte activation](https://amigo.geneontology.org/amigo/term/GO:0051249), [positive regulation of leukocyte cell-cell adhesion](https://amigo.geneontology.org/amigo/term/GO:1903039), and [positive regulation of intrinsic apoptotic signaling pathway](https://amigo.geneontology.org/amigo/term/GO:2001244).


The gene length is 49,056 base pairs (65.76% of all genes), the mature length is 811 base pairs (14.29% of all genes), and the primary transcript length is 25,648 base pairs (58.47% of all genes).


The gene LCK (NCBI ID: 3932) has been mentioned in [710 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22LCK%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning LCK is 1986, and the middle 50% of publications occurred between 1995 and 2008.


The top 5 publications mentioning LCK, ranked by their scientific influence, include "[Discovery of a novel, potent, and Src family-selective tyrosine kinase inhibitor. Study of Lck- and FynT-dependent T cell activation.](https://pubmed.ncbi.nlm.nih.gov/8557675)" (1996) with a relative citation ratio of 36.52, "[Signal transduction through the CD4 receptor involves the activation of the internal membrane tyrosine-protein kinase p56lck.](https://pubmed.ncbi.nlm.nih.gov/2784195)" (1989) with a relative citation ratio of 13.14, "[Interaction of the IL-2 receptor with the src-family kinase p56lck: identification of novel intermolecular association.](https://pubmed.ncbi.nlm.nih.gov/2047859)" (1991) with a relative citation ratio of 11.96, "[Function of the Src-family kinases, Lck and Fyn, in T-cell development and activation.](https://pubmed.ncbi.nlm.nih.gov/15489916)" (2004) with a relative citation ratio of 8.6, and "[Loss of T-cell receptor zeta chain and p56lck in T-cells infiltrating human renal cell carcinoma.](https://pubmed.ncbi.nlm.nih.gov/8242613)" (1993) with a relative citation ratio of 8.29. The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[LCK](https://www.proteinatlas.org/ENSG00000182866-LCK) is a gene encoding a protein with kinase, transferase, and tyrosine-protein kinase molecular functions. It is involved in various cancer-related processes, serving as a proto-oncogene and a target for FDA-approved drugs. Evidence for its function is established at the protein level. LCK is detected in many tissues and is localized to the Golgi apparatus. Its expression is notably high in T-cells, thymus, immune cells, and leukemia cell lines, suggesting a significant role in adaptive immune response and T-cell receptor signaling.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 5 experiments, [NFYA](https://www.ncbi.nlm.nih.gov/gene/4800) with 3 experiments, [KDM2B](https://www.ncbi.nlm.nih.gov/gene/84678) with 3 experiments, [TP53](https://www.ncbi.nlm.nih.gov/gene/7157) with 3 experiments, and [SP1](https://www.ncbi.nlm.nih.gov/gene/6667) with 3 experiments.





The gene is ubiquitously expressed in multiple tissue samples, including blood, spleen, and small intestine, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in various immune-related tissues and cells, such as 721_B_lymphoblasts, CD56+ NK cells, CD4+ T cells, CD8+ T cells, thymus, leukemia lymphoblastic (MOLT-4) cells, and lymph nodes.


The input data highlights various pathways involving immune system processes, including cytokine signaling, adaptive and innate immune responses, and interactions with HIV. Additionally, it emphasizes signaling pathways such as PI3K/AKT, ERBB2, ERBB4, EGFR, and FGFR, which are implicated in cancer and other diseases. The data also covers signaling by interleukins, TCR, BCR, and other receptors, as well as platelet activation and hemostasis. Furthermore, it includes pathways related to the role of Nef in HIV-1 replication and disease pathogenesis.



The analyzed protein sequence has a GRAVY value of -0.4696 (38.24th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -17.43 (13.09th percentile), and the median structural flexibility is 1.002 (49.78th percentile). The protein has a predicted secondary structure with 30.45% helix (35.93rd percentile), 35.56% sheet (63.26th percentile), and 29.27% turn (58.17th percentile) affinities. The instability index is 41.93 (28.42nd percentile), and the isoelectric point is 5.23 (12.21st percentile). The protein is 509 amino acids long (59.4th percentile) with a molecular weight of 57999.92 Da (60.69th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |