---
layout: post
title: FLNA
date: 2025-01-17 16:55 CST
description: FLNA description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2316) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2316  | FLNA | ENSG00000196924 | Xq28 |



The gene is involved in various biological processes, including [angiogenesis](https://amigo.geneontology.org/amigo/term/GO:0001525), [epithelial to mesenchymal transition](https://amigo.geneontology.org/amigo/term/GO:0001837), [blood vessel remodeling](https://amigo.geneontology.org/amigo/term/GO:0001974), and [heart morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0003007). It enables molecular functions such as [G protein-coupled receptor binding](https://amigo.geneontology.org/amigo/term/GO:0001664), [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), and [protein kinase C binding](https://amigo.geneontology.org/amigo/term/GO:0005080). The gene is located in multiple cellular components, including the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleolus](https://amigo.geneontology.org/amigo/term/GO:0005730), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), and [trans-Golgi network](https://amigo.geneontology.org/amigo/term/GO:0005802). Additionally, it is involved in processes like [adenylate cyclase-inhibiting dopamine receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007195), [blood coagulation, intrinsic pathway](https://amigo.geneontology.org/amigo/term/GO:0007597), and [positive regulation of platelet activation](https://amigo.geneontology.org/amigo/term/GO:0010572). The gene also plays roles in [negative regulation of neuron projection development](https://amigo.geneontology.org/amigo/term/GO:0010977) and enables [potassium channel regulator activity](https://amigo.geneontology.org/amigo/term/GO:0015459). It is located in the [actin cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0015629) and [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), and is involved in [negative regulation of transcription by RNA polymerase I](https://amigo.geneontology.org/amigo/term/GO:0016479) and [formation of radial glial scaffolds](https://amigo.geneontology.org/amigo/term/GO:0021943). The gene is also part of the [Myb complex](https://amigo.geneontology.org/amigo/term/GO:0031523) and enables [Fc-gamma receptor I complex binding](https://amigo.geneontology.org/amigo/term/GO:0034988). It is involved in [megakaryocyte development](https://amigo.geneontology.org/amigo/term/GO:0035855) and [negative regulation of protein catabolic process](https://amigo.geneontology.org/amigo/term/GO:0042177), among other processes.


The gene length is 980.0 base pairs (2.12% of all genes). The mature length is 980.0 base pairs, and the primary transcript length is 980.0 base pairs.


The gene FLNA (NCBI ID: 2316) has been mentioned in [382 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22FLNA%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1974 and the middle 50% of publications occurring between 2009 and 2019. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning FLNA, ranked by their scientific influence, include "[Mutations in filamin 1 prevent migration of cerebral cortical neurons in human periventricular heterotopia.](https://pubmed.ncbi.nlm.nih.gov/9883725)" (1998) (relative citation ratio: 11.61), "[A role for tissue factor in cell adhesion and migration mediated by interaction with actin-binding protein 280.](https://pubmed.ncbi.nlm.nih.gov/9490735)" (1998) (relative citation ratio: 5.43), "[FilGAP, a Rho- and ROCK-regulated GAP for Rac binds filamin A to control actin remodelling.](https://pubmed.ncbi.nlm.nih.gov/16862148)" (2006) (relative citation ratio: 5.35), "[Periventricular heterotopia: phenotypic heterogeneity and correlation with Filamin A mutations.](https://pubmed.ncbi.nlm.nih.gov/16684786)" (2006) (relative citation ratio: 5.2), and "[The ER Stress Sensor PERK Coordinates ER-Plasma Membrane Contact Site Formation through Interaction with Filamin-A and F-Actin Remodeling.](https://pubmed.ncbi.nlm.nih.gov/28238652)" (2017) (relative citation ratio: 5.06). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[FLNA](https://www.proteinatlas.org/ENSG00000196924-FLNA) is an actin-binding protein involved in deafness and disease variants, with evidence at the protein level. It is detected in all RNA tissues and is primarily localized in the plasma membrane, actin filaments, and cytosol. FLNA is part of Cluster 33 in blood expression, which is non-specific and related to ATP binding. In tissue expression, it belongs to Cluster 41, associated with smooth muscle tissue and ECM organization. In brain expression, it is part of Cluster 12, which is non-specific and related to vasculature. Additionally, FLNA is found in Cluster 36 in cell line expression, which is non-specific and related to mRNA processing, and in Cluster 32 in single-cell expression, associated with smooth muscle cells and muscle contraction.


The top transcription factors (TFs) identified from the GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599), [MYC](https://www.ncbi.nlm.nih.gov/gene/4609), [KDM2B](https://www.ncbi.nlm.nih.gov/gene/84678), [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885), and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), each shown to be regulating in 5 experiments.





The gene is expressed in various tissues, including colorectal adenocarcinoma, prostate, lung, appendix, ovary, and uterus. Additionally, it is expressed in blood vessels.


The proteins are involved in various pathways, including cell junction organization, cell-cell communication, cell-extracellular matrix interactions, GP1b-IX-V activation signaling, hemostasis, platelet activation, signaling and aggregation, platelet degranulation, and response to elevated platelet cytosolic Ca2+.



The analyzed protein sequence has a GRAVY value of -0.317 (56.12th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -48.94 (2.51st percentile), and the median structural flexibility is 1.007 (72.41st percentile). The protein has a propensity for helix, sheet, and turn structures of 25.65% (6.57th percentile), 33.43% (48.17th percentile), and 34.87% (88.92nd percentile), respectively. The instability index is 36.77 (13.7th percentile), with an isoelectric point of 5.70 (22.98th percentile). The sequence length is 2647 amino acids (98.86th percentile), and the molecular weight is 280735.63 Da (98.69th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |