---
layout: post
title: RAB13
date: 2025-01-17 16:55 CST
description: RAB13 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5872) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5872  | RAB13 | ENSG00000143545 | 1q21.3 |



The gene enables [GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0003924), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [GTP binding](https://amigo.geneontology.org/amigo/term/GO:0005525). It is active in the [endosome](https://amigo.geneontology.org/amigo/term/GO:0005768), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), and [synaptic vesicle](https://amigo.geneontology.org/amigo/term/GO:0008021), and is located in the [trans-Golgi network](https://amigo.geneontology.org/amigo/term/GO:0005802), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [bicellular tight junction](https://amigo.geneontology.org/amigo/term/GO:0005923), [lateral plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0016328), [lamellipodium](https://amigo.geneontology.org/amigo/term/GO:0030027), [cell junction](https://amigo.geneontology.org/amigo/term/GO:0030054), [endocytic vesicle](https://amigo.geneontology.org/amigo/term/GO:0030139), [cytoplasmic vesicle](https://amigo.geneontology.org/amigo/term/GO:0031410), [insulin-responsive compartment](https://amigo.geneontology.org/amigo/term/GO:0032593), [neuron projection](https://amigo.geneontology.org/amigo/term/GO:0043005), [recycling endosome](https://amigo.geneontology.org/amigo/term/GO:0055037), [recycling endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0055038), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). The gene is involved in various processes including [vesicle docking involved in exocytosis](https://amigo.geneontology.org/amigo/term/GO:0006904), [protein secretion](https://amigo.geneontology.org/amigo/term/GO:0009306), [protein kinase A signaling](https://amigo.geneontology.org/amigo/term/GO:0010737), [endosomal transport](https://amigo.geneontology.org/amigo/term/GO:0016197), [regulation of exocytosis](https://amigo.geneontology.org/amigo/term/GO:0017157), [cortical actin cytoskeleton organization](https://amigo.geneontology.org/amigo/term/GO:0030866), [neuron projection development](https://amigo.geneontology.org/amigo/term/GO:0031175), [endocytic recycling](https://amigo.geneontology.org/amigo/term/GO:0032456), [cellular response to insulin stimulus](https://amigo.geneontology.org/amigo/term/GO:0032869), [endothelial cell chemotaxis](https://amigo.geneontology.org/amigo/term/GO:0035767), [trans-Golgi network to recycling endosome transport](https://amigo.geneontology.org/amigo/term/GO:0044795), [Golgi vesicle fusion to target membrane](https://amigo.geneontology.org/amigo/term/GO:0048210), [bicellular tight junction assembly](https://amigo.geneontology.org/amigo/term/GO:0070830), [protein localization to plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0072659), [establishment of Sertoli cell barrier](https://amigo.geneontology.org/amigo/term/GO:0097368), and [protein localization to cell leading edge](https://amigo.geneontology.org/amigo/term/GO:1902463).


The gene length is 178,177 base pairs (91.16% of all genes), the mature length is 802 base pairs (13.96% of all genes), and the primary transcript length is 79,358 base pairs (84.58% of all genes).


RAB13, with NCBI Gene ID 5872, has been mentioned in [28 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RAB13%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning RAB13 was in 1998, and the middle 50% of publications occurred between 2011 and 2019.


The top 5 publications mentioning RAB13, ranked by their scientific influence, include studies that explore the gene's role in various cellular processes. The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Notable publications include "[Rab35 promotes the recruitment of Rab8, Rab13 and Rab36 to recycling endosomes through MICAL-L1 during neurite outgrowth.](https://pubmed.ncbi.nlm.nih.gov/25086062)" (2014) (relative citation ratio: 1.91), "[DENND2B activates Rab13 at the leading edge of migrating cells and promotes metastatic behavior.](https://pubmed.ncbi.nlm.nih.gov/25713415)" (2015) (relative citation ratio: 1.85), "[The interaction of JRAB/MICAL-L2 with Rab8 and Rab13 coordinates the assembly of tight junctions and adherens junctions.](https://pubmed.ncbi.nlm.nih.gov/18094055)" (2008) (relative citation ratio: 1.77), "[Rab13 regulates sEV secretion in mutant KRAS colorectal cancer cells.](https://pubmed.ncbi.nlm.nih.gov/32978434)" (2020) (relative citation ratio: 1.65), and "[A complex of Rab13 with MICAL-L2 and Î±-actinin-4 is essential for insulin-dependent GLUT4 exocytosis.](https://pubmed.ncbi.nlm.nih.gov/26538022)" (2016) (relative citation ratio: 1.65). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RAB13](https://www.proteinatlas.org/ENSG00000143545-RAB13) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in the plasma membrane and cytosol. The gene is expressed in dendritic cells with unknown function (Cluster 2) and is notably expressed in the thyroid gland (Cluster 51), also with unknown function. In the brain, RAB13 shows non-specific expression with mixed function (Cluster 44), while in cell lines, it is associated with nuclear processes (Cluster 45). Single-cell expression data indicates its involvement in macrophages, contributing to the innate immune response (Cluster 58).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [AR](https://www.ncbi.nlm.nih.gov/gene/367) with 6 experiments, [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908) with 5 experiments, [SMC1A](https://www.ncbi.nlm.nih.gov/gene/8243) with 5 experiments, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 5 experiments, and [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.





The gene is expressed in various tissues, including the prostate and adrenal gland, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in adipocytes, smooth muscle, and the uterus corpus.


The input data highlights two key pathways: membrane trafficking and the translocation of GLUT4 to the plasma membrane. These processes are integral to cellular function and glucose homeostasis, respectively.



The analyzed protein sequence has a GRAVY value of -0.481 (36.93rd percentile), indicating a hydrophilic nature. It exhibits a charge of 7.66 at pH 7.0 (75.98th percentile) and a median structural flexibility of 1.01 (73.06th percentile). The protein's secondary structure is predicted to be 34.48% helix (71.45th percentile), 34.00% sheet (51.9th percentile), and 28.57% turn (52.36th percentile). The instability index is 22.39 (0.92nd percentile), with an isoelectric point of 9.27 (86.21st percentile). The protein has a length of 203 amino acids (14.94th percentile) and a molecular weight of 22773.86 Da (15.1st percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |