---
layout: post
title: CDH4
date: 2025-01-17 16:55 CST
description: CDH4 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1002) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1002  | CDH4 | ENSG00000179242 | 20q13.33 |



The gene is located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886) and is active in the [adherens junction](https://amigo.geneontology.org/amigo/term/GO:0005912). It is involved in various processes including [cell morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0000902), [cell-cell junction assembly](https://amigo.geneontology.org/amigo/term/GO:0007043), [cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0007155), [homophilic cell adhesion via plasma membrane adhesion molecules](https://amigo.geneontology.org/amigo/term/GO:0007156), [heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules](https://amigo.geneontology.org/amigo/term/GO:0007157), [multicellular organism development](https://amigo.geneontology.org/amigo/term/GO:0007275), [axon guidance](https://amigo.geneontology.org/amigo/term/GO:0007411), [calcium-dependent cell-cell adhesion via plasma membrane cell adhesion molecules](https://amigo.geneontology.org/amigo/term/GO:0016339), [adherens junction organization](https://amigo.geneontology.org/amigo/term/GO:0034332), [cell-cell adhesion mediated by cadherin](https://amigo.geneontology.org/amigo/term/GO:0044331), [positive regulation of axon extension](https://amigo.geneontology.org/amigo/term/GO:0045773), and [axon extension](https://amigo.geneontology.org/amigo/term/GO:0048675). The gene enables [calcium ion binding](https://amigo.geneontology.org/amigo/term/GO:0005509) and [cadherin binding](https://amigo.geneontology.org/amigo/term/GO:0045296), and is part of the [catenin complex](https://amigo.geneontology.org/amigo/term/GO:0016342).


The gene has a length of 83,455 base pairs (78.3% of all genes), a mature length of 1,376 base pairs, and a primary transcript length of 80,889 base pairs.


The gene CDH4 (NCBI ID: 1002) has been mentioned in [24 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CDH4%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning CDH4 is 1991, and the middle 50% of publications occurred between 2004 and 2019.


The top 5 publications mentioning CDH4, ranked by their scientific influence, include "[Diversity of the cadherin family: evidence for eight new cadherins in nervous tissue.](https://pubmed.ncbi.nlm.nih.gov/2059658)" (1991) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 7.54, "[Frequent aberrant methylation of the CDH4 gene promoter in human colorectal and gastric cancer.](https://pubmed.ncbi.nlm.nih.gov/15548679)" (2004) with an RCR of 1.62, "[R-cadherin:beta-catenin complex and its association with vascular smooth muscle cell proliferation.](https://pubmed.ncbi.nlm.nih.gov/15117735)" (2004) with an RCR of 1.34, "[Characterization of cadherin-4 and cadherin-5 reveals new aspects of cadherins.](https://pubmed.ncbi.nlm.nih.gov/7962210)" (1994) with an RCR of 1.33, and "[CDH4 is a novel determinant of osteosarcoma tumorigenesis and metastasis.](https://pubmed.ncbi.nlm.nih.gov/29610525)" (2018) with an RCR of 1.32. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CDH4](https://www.proteinatlas.org/ENSG00000179242-CDH4) is a gene with evidence at the protein level, detected in some RNA tissue distributions. It is primarily located in the plasma membrane and is notably expressed in the brain, specifically in neuronal signaling pathways within the hypothalamus. CDH4 is also associated with neuronal signal transduction in cell lines and is found in astrocytes, although its function in these cells is unknown. The gene is unprognostic for several cancer types, including Breast Invasive Carcinoma, Glioblastoma Multiforme, Kidney Renal Clear Cell Carcinoma, Kidney Renal Papillary Cell Carcinoma, and Thyroid Carcinoma.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 4 experiments, [E2F1](https://www.ncbi.nlm.nih.gov/gene/1869) with 3 experiments, and [SOX2](https://www.ncbi.nlm.nih.gov/gene/6657), [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170), and [HDAC2](https://www.ncbi.nlm.nih.gov/gene/3066), each with 2 experiments.



The input data from GWAS indicates associations with a wide range of disease conditions, including various types of cancer such as male reproductive organ cancer, testicular cancer, head and neck cancer, colorectal cancer, lung cancer, and leukemia. Additionally, the data highlights associations with metabolic disorders like type 2 diabetes mellitus and obesity, as well as cardiovascular diseases such as coronary artery disease and atherosclerosis. Neurodegenerative diseases, including Alzheimer's disease and amyotrophic lateral sclerosis, are also implicated. Furthermore, the data shows links to mental health conditions like bipolar disorder, schizophrenia, and major depressive disorder. Other notable associations include liver disease, eye disease, and musculoskeletal system diseases.


The gene CDH4 has been associated with [Obesity-related traits](https://pubmed.ncbi.nlm.nih.gov/23251661), which maps closely to the MeSH term [Obesity](https://meshb.nlm.nih.gov/record/ui?ui=D009765) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method.


The gene is expressed in various tissues, including the ovary, brain, and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the Prefrontal Cortex, Cardiac Myocytes, Lymphoma (Burkitt's, Raji), Amygdala, Fetal Brain, Subthalamic Nucleus, and Superior Cervical Ganglion.


The input data highlights several biological processes and pathways, including cdo in myogenesis, myogenesis, developmental biology, adherens junctions interactions, cell junction organization, cell-cell junction organization, and cell-cell communication. These pathways are integral to various aspects of cellular development and communication.


The mouse ortholog gene 12561 and its human ortholog gene 1002.0 are associated with decreased vertical activity and increased circulating sodium level, with these phenotypes observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.323 (55.3% percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -46.19 (2.84% percentile), and the median structural flexibility is 1.003 (54.16% percentile). The protein has a propensity for helix, sheet, and turn structures of 26.75% (10.59% percentile), 35.48% (62.59% percentile), and 34.17% (86.68% percentile), respectively. The instability index is 35.86 (11.8% percentile), with an isoelectric point of 4.65 (2.55% percentile). The sequence length is 916 amino acids (85.01% percentile), and the molecular weight is 100279.4 Da (84.35% percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |