---
layout: post
title: OAS2
date: 2025-01-17 16:55 CST
description: OAS2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4939) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4939  | OAS2 | ENSG00000111335 | 12q24.13 |



The gene enables [2'-5'-oligoadenylate synthetase activity](https://amigo.geneontology.org/amigo/term/GO:0001730) and [double-stranded RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003725), and is involved in [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524). It is active in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654) and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829) and [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020). The gene is involved in various processes including [nucleobase-containing compound metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006139), [RNA catabolic process](https://amigo.geneontology.org/amigo/term/GO:0006401), [response to virus](https://amigo.geneontology.org/amigo/term/GO:0009615), [positive regulation of interferon-beta production](https://amigo.geneontology.org/amigo/term/GO:0032728), [positive regulation of tumor necrosis factor production](https://amigo.geneontology.org/amigo/term/GO:0032760), [defense response to bacterium](https://amigo.geneontology.org/amigo/term/GO:0042742), [negative regulation of viral genome replication](https://amigo.geneontology.org/amigo/term/GO:0045071), [defense response to virus](https://amigo.geneontology.org/amigo/term/GO:0051607), [type I interferon-mediated signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0060337), [regulation of ribonuclease activity](https://amigo.geneontology.org/amigo/term/GO:0060700), [interleukin-27-mediated signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0070106), and [regulation of lactation](https://amigo.geneontology.org/amigo/term/GO:1903487). Additionally, it enables [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872) and is located in the [intracellular membrane-bounded organelle](https://amigo.geneontology.org/amigo/term/GO:0043231) and the [perinuclear region of cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0048471).


The gene length is 314,939 base pairs (96.13% of all genes), the mature length is 4,003 base pairs (83.82% of all genes), and the primary transcript length is 293,217.5 base pairs (97.8% of all genes).


The gene OAS2 (NCBI ID: 4939) has been mentioned in [13 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22OAS2%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 2009 and the middle 50% of publications occurring between 2016 and 2020. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning OAS2, ranked by their relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)), include studies that explore the gene's role in various diseases. The RCR measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Notable publications include "[Prognostic characterization of OAS1/OAS2/OAS3/OASL in breast cancer.](https://pubmed.ncbi.nlm.nih.gov/32560641)" (2020) (relative citation ratio: 2.8), "[OAS1, OAS2, and OAS3 Contribute to Epidermal Keratinocyte Proliferation by Regulating Cell Cycle and Augmenting IFN-1‒Induced Jak1‒Signal Transducer and Activator of Transcription 1 Phosphorylation in Psoriasis.](https://pubmed.ncbi.nlm.nih.gov/35305973)" (2022) (relative citation ratio: 2.0), and "[MALAT1 is involved in type I IFNs-mediated systemic lupus erythematosus by up-regulating OAS2, OAS3, and OASL.](https://pubmed.ncbi.nlm.nih.gov/32321151)" (2020) (relative citation ratio: 1.84). Additionally, "[Quantitative Proteomic Profile of Psoriatic Epidermis Identifies OAS2 as a Novel Biomarker for Disease Activity.](https://pubmed.ncbi.nlm.nih.gov/32849499)" (2020) (relative citation ratio: 1.41) and "[Opposite functions of GSN and OAS2 on colorectal cancer metastasis, mediating perineural and lymphovascular invasion, respectively.](https://pubmed.ncbi.nlm.nih.gov/30148861)" (2018) (relative citation ratio: 1.04) are also significant. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[OAS2](https://www.proteinatlas.org/ENSG00000111335-OAS2) is a gene with molecular functions including nucleotidyltransferase, RNA-binding, and transferase activities. Evidence for its function is established at the protein level. The gene is detected in all RNA tissue distributions and is primarily localized in the centrosome. Its expression is notable in various clusters, including Cluster 33 for blood (non-specific - ATP binding), Cluster 70 for tissue (lymphoid tissue - immune response), Cluster 43 for brain (hindbrain - mixed function), Cluster 31 for cell lines (non-specific - antiviral immune response), and Cluster 58 for single cells (macrophages - innate immune response).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 5 experiments, [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046) with 4 experiments, [MYC](https://www.ncbi.nlm.nih.gov/gene/4609) with 4 experiments, [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313) with 4 experiments, and [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with various disease conditions, including metabolic disorders such as diabetes mellitus, specifically type 2 diabetes mellitus, and carbohydrate and glucose metabolism diseases. Additionally, there are associations with cardiovascular conditions, including coronary artery disease, artery disease, vascular disease, and hypertension. Mental health disorders, particularly substance-related disorders like substance dependence, alcohol dependence, and alcohol abuse, are also implicated.


The gene OAS2 has been associated with [Adult asthma](https://pubmed.ncbi.nlm.nih.gov/27611488), which maps closely to the MeSH term [Asthma](https://meshb.nlm.nih.gov/record/ui?ui=D001249) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method.


The gene is ubiquitously expressed in multiple tissue samples, including lung, blood, and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in various immune cell types such as B lymphoblasts, myeloid cells, dendritic cells, NK cells, and monocytes, as well as in lymphoma (Burkitt's), appendix, and tonsil.


The proteins are involved in various immune-related pathways, including cytokine signaling in the immune system, interferon signaling, interferon gamma signaling, interferon alpha/beta signaling, and the broader immune system.



The analyzed protein sequence has a GRAVY value of -0.404 (45.61% percentile), indicating a hydrophilic nature. It exhibits a charge of 8.39 at pH 7.0 (77.9% percentile) and a median structural flexibility of 1.004 (57.99% percentile). The protein's secondary structure is predicted to be 31.57% helix (46.51% percentile), 37.97% sheet (78.34% percentile), and 28.09% turn (48.15% percentile). The instability index is 42.58 (30.68% percentile), with an isoelectric point of 8.55 (67.62% percentile). The protein has a length of 719 amino acids (76.51% percentile) and a molecular weight of 82429.6 Da (77.47% percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |