---
layout: post
title: PEX16
date: 2025-01-17 16:55 CST
description: PEX16 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9409) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9409  | PEX16 | ENSG00000121680 | 11p11.2 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is located in the [peroxisome](https://amigo.geneontology.org/amigo/term/GO:0005777), [peroxisomal membrane](https://amigo.geneontology.org/amigo/term/GO:0005778), [endoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0005783), [endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0005789), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020). It is involved in [protein targeting to peroxisome](https://amigo.geneontology.org/amigo/term/GO:0006625), [peroxisome organization](https://amigo.geneontology.org/amigo/term/GO:0007031), [peroxisome membrane biogenesis](https://amigo.geneontology.org/amigo/term/GO:0016557), [protein import into peroxisome matrix](https://amigo.geneontology.org/amigo/term/GO:0016558), [protein to membrane docking](https://amigo.geneontology.org/amigo/term/GO:0022615), [ER-dependent peroxisome organization](https://amigo.geneontology.org/amigo/term/GO:0032581), [protein import into peroxisome membrane](https://amigo.geneontology.org/amigo/term/GO:0045046), and [ER-dependent peroxisome localization](https://amigo.geneontology.org/amigo/term/GO:0106101).


The gene length is 5,617 base pairs (15.59% of all genes), the mature length is 1,878.5 base pairs, and the primary transcript length is 5,601 base pairs (20.53% of all genes).


The gene PEX16 (NCBI ID: 9409) has been mentioned in [19 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PEX16%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink). The earliest year of publication is 1998, and the middle 50% of publications occurred between 2009 and 2018.


The top five publications mentioning PEX16, ranked by their scientific influence, include studies that elucidate the gene's role in peroxisome biogenesis and related disorders. The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Notable works include "[The origin and maintenance of mammalian peroxisomes involves a de novo PEX16-dependent pathway from the ER.](https://pubmed.ncbi.nlm.nih.gov/16717127)" (2006) (relative citation ratio: 4.71), "[Mutation in PEX16 is causal in the peroxisome-deficient Zellweger syndrome of complementation group D.](https://pubmed.ncbi.nlm.nih.gov/9837814)" (1998) (relative citation ratio: 2.8), "[The peroxisomal membrane protein import receptor Pex3p is directly transported to peroxisomes by a novel Pex19p- and Pex16p-dependent pathway.](https://pubmed.ncbi.nlm.nih.gov/19114594)" (2008) (relative citation ratio: 2.47), "[Identification of an unusual variant peroxisome biogenesis disorder caused by mutations in the PEX16 gene.](https://pubmed.ncbi.nlm.nih.gov/20647552)" (2010) (relative citation ratio: 1.77), and "[PEX16 contributes to peroxisome maintenance by constantly trafficking PEX3 via the ER.](https://pubmed.ncbi.nlm.nih.gov/25002403)" (2014) (relative citation ratio: 1.39). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PEX16](https://www.proteinatlas.org/ENSG00000121680-PEX16) is a gene with evidence at the protein level, implicated in peroxisome biogenesis disorder and Zellweger syndrome. Its RNA is detected in all tissues, with notable expression in T-cells, mitochondrial translation clusters, and enterocytes involved in absorption. The gene's expression is also observed in various brain, tissue, and cell line clusters, although its molecular function and subcellular location remain undetermined.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SMC3](https://www.ncbi.nlm.nih.gov/gene/9126) with 6 experiments, [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170) with 5 experiments, [SUPT5H](https://www.ncbi.nlm.nih.gov/gene/6829) with 5 experiments, [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930) with 5 experiments, and [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with a range of conditions, including lower respiratory tract disease, lung disease, respiratory system disease, nervous system disease, refractive error, myopia, eye and adnexa disease, eye disease, and sensory system disease.



The gene is expressed in the pituitary and spleen according to BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles indicate expression in the heart, liver, CD56+ NK cells, CD4+ T cells, CD8+ T cells, CD19+ B cells (neg. sel.), CD14+ monocytes, and the whole brain.



The mouse ortholog gene 18633 and its human ortholog gene 9409 are associated with the phenotype of preweaning lethality with complete penetrance, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.1404761904761908 (74.66th percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is 12.405336407874806 (85.71st percentile), and the median structural flexibility is 0.9912380952380954 (13.06th percentile). The protein has a high isoelectric point of 9.847690773010253 (95.22nd percentile) and an instability index of 52.14589285714285 (64.04th percentile). The secondary structure affinities are 0.3601190476190476 for helix (80.79th percentile), 0.41964285714285715 for sheet (90.61st percentile), and 0.22916666666666666 for turn (9.59th percentile). The protein has a length of 336 amino acids (35.75th percentile) and a molecular weight of 38628.46290000001 Da (37.38th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |