---
layout: post
title: CDKN2C
date: 2025-01-17 16:55 CST
description: CDKN2C description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1031) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1031  | CDKN2C | ENSG00000123080 | 1p32.3 |



The gene is involved in the [regulation of cyclin-dependent protein serine/threonine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0000079) and enables [cyclin-dependent protein serine/threonine kinase inhibitor activity](https://amigo.geneontology.org/amigo/term/GO:0004861). It is active in and located in both the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), as well as located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). The gene plays a role in the [cell cycle](https://amigo.geneontology.org/amigo/term/GO:0007049) and is involved in the [negative regulation of cell population proliferation](https://amigo.geneontology.org/amigo/term/GO:0008285), [negative regulation of cell growth](https://amigo.geneontology.org/amigo/term/GO:0030308), [negative regulation of phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0042326), [oligodendrocyte differentiation](https://amigo.geneontology.org/amigo/term/GO:0048709), and [stem cell proliferation](https://amigo.geneontology.org/amigo/term/GO:0072089). Additionally, it is involved in the [regulation of G1/S transition of the mitotic cell cycle](https://amigo.geneontology.org/amigo/term/GO:2000045) and the [negative regulation of stem cell proliferation](https://amigo.geneontology.org/amigo/term/GO:2000647). The gene also enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [protein kinase binding](https://amigo.geneontology.org/amigo/term/GO:0019901).


The gene length is 3,707 base pairs (10.29% of all genes), the mature length is 702 base pairs, and the primary transcript length is 3,456 base pairs.


The gene CDKN2C (NCBI ID: 1031) has been mentioned in [117 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CDKN2C%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count), with the earliest publication in 1994 and the middle 50% of publications occurring between 1998 and 2012.


The top five publications mentioning CDKN2C, ranked by their scientific influence, include "[Growth suppression by p18, a p16INK4/MTS1- and p14INK4B/MTS2-related CDK6 inhibitor, correlates with wild-type pRb function.](https://pubmed.ncbi.nlm.nih.gov/8001816)" (1994) (relative citation ratio: 14.51), "[Review of alterations of the cyclin-dependent kinase inhibitor INK4 family genes p15, p16, p18 and p19 in human leukemia-lymphoma cells.](https://pubmed.ncbi.nlm.nih.gov/9639410)" (1998) (relative citation ratio: 6.4), "[Mutations in the p16INK4/MTS1/CDKN2, p15INK4B/MTS2, and p18 genes in primary and metastatic lung cancer.](https://pubmed.ncbi.nlm.nih.gov/7882351)" (1995) (relative citation ratio: 5.42), "[The role of germline AIP, MEN1, PRKAR1A, CDKN1B and CDKN2C mutations in causing pituitary adenomas in a large cohort of children, adolescents, and patients with genetic syndromes.](https://pubmed.ncbi.nlm.nih.gov/20507346)" (2010) (relative citation ratio: 3.82), and "[Analysis of a family of cyclin-dependent kinase inhibitors: p15/MTS2/INK4B, p16/MTS1/INK4A, and p18 genes in acute lymphoblastic leukemia of childhood.](https://pubmed.ncbi.nlm.nih.gov/7606004)" (1995) (relative citation ratio: 3.32). The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CDKN2C](https://www.proteinatlas.org/ENSG00000123080-CDKN2C) is a cancer-related gene with evidence at the protein level. It is detected in all RNA tissue distributions and is primarily localized in the nucleoplasm and cytosol. The gene is part of Cluster 3 in blood expression, which is associated with non-specific nuclear processes, and Cluster 16 in tissue expression, which is linked to non-specific transcription. In the brain, it is part of Cluster 11, associated with white matter and signal transduction. Additionally, CDKN2C is found in Cluster 10 in cell line expression, related to ciliated cells and ion channel activity, and Cluster 6 in single-cell expression, associated with ovarian stromal cells and transcription.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD9](https://www.ncbi.nlm.nih.gov/gene/65980) with 6 experiments, followed by [FOS](https://www.ncbi.nlm.nih.gov/gene/2353), [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908), [TAF1](https://www.ncbi.nlm.nih.gov/gene/6872), and [KMT2B](https://www.ncbi.nlm.nih.gov/gene/9757), each with 5 experiments.





The gene is expressed in various tissues, including adipose tissue and breast, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Leukemia promyelocytic (HL-60), Lymphoma burkitts (Daudi), CD71+ Early Erythroid, Adipocyte, Bone marrow, Thymus, and Leukemia lymphoblastic (MOLT-4).


The proteins are involved in various pathways related to cellular senescence, including cellular responses to stress, oxidative stress-induced senescence, and the senescence-associated secretory phenotype (SASP). Additionally, they participate in cell cycle regulation, specifically in G1 phase processes such as cyclin D-associated events and mitotic G1-G1/S phases, as well as broader cell cycle and mitotic cell cycle pathways.



The protein sequence analyzed has a GRAVY value of -0.296 (58.41 percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -2.95 (39.89 percentile). The median structural flexibility is 0.9998 (40.74 percentile), with a propensity for helix, sheet, and turn structures of 38.10% (89.02 percentile), 31.55% (36.97 percentile), and 29.17% (57.27 percentile), respectively. The instability index is 28.65 (3.09 percentile), and the isoelectric point is 6.05 (31.76 percentile). The protein has a length of 168 amino acids (10.27 percentile) and a molecular weight of 18127.17 Da (9.63 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |