---
layout: post
title: EXOSC10
date: 2025-01-17 16:55 CST
description: EXOSC10 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5394) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5394  | EXOSC10 | ENSG00000171824 | 1p36.22 |



The gene enables [nucleotide binding](https://amigo.geneontology.org/amigo/term/GO:0000166) and [3'-5'-RNA exonuclease activity](https://amigo.geneontology.org/amigo/term/GO:0000175), and is part of the [nuclear exosome (RNase complex)](https://amigo.geneontology.org/amigo/term/GO:0000176), [cytoplasmic exosome (RNase complex)](https://amigo.geneontology.org/amigo/term/GO:0000177), and [exosome (RNase complex)](https://amigo.geneontology.org/amigo/term/GO:0000178). It is involved in various processes such as [nuclear-transcribed mRNA catabolic process, nonsense-mediated decay](https://amigo.geneontology.org/amigo/term/GO:0000184), [maturation of 5.8S rRNA](https://amigo.geneontology.org/amigo/term/GO:0000460), and [exonucleolytic trimming to generate mature 3'-end of 5.8S rRNA](https://amigo.geneontology.org/amigo/term/GO:0000467). The gene is also part of [euchromatin](https://amigo.geneontology.org/amigo/term/GO:0000791) and is involved in [nuclear-transcribed mRNA catabolic process](https://amigo.geneontology.org/amigo/term/GO:0000956). It enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [single-stranded RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003727), [RNA exonuclease activity](https://amigo.geneontology.org/amigo/term/GO:0004532), and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [nucleolus](https://amigo.geneontology.org/amigo/term/GO:0005730), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and is active in the [nucleolus](https://amigo.geneontology.org/amigo/term/GO:0005730). Additionally, it is involved in [DNA repair](https://amigo.geneontology.org/amigo/term/GO:0006281), [rRNA processing](https://amigo.geneontology.org/amigo/term/GO:0006364), [RNA processing](https://amigo.geneontology.org/amigo/term/GO:0006396), and [RNA catabolic process](https://amigo.geneontology.org/amigo/term/GO:0006401). The gene is also part of the [small-subunit processome](https://amigo.geneontology.org/amigo/term/GO:0032040) and is involved in [negative regulation of telomere maintenance via telomerase](https://amigo.geneontology.org/amigo/term/GO:0032211) and [ribosomal small subunit biogenesis](https://amigo.geneontology.org/amigo/term/GO:0042274). It enables [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872) and [telomerase RNA binding](https://amigo.geneontology.org/amigo/term/GO:0070034), and is involved in various catabolic processes such as [nuclear mRNA surveillance](https://amigo.geneontology.org/amigo/term/GO:0071028), [CUT catabolic process](https://amigo.geneontology.org/amigo/term/GO:0071034), and [nuclear polyadenylation-dependent rRNA catabolic process](https://amigo.geneontology.org/amigo/term/GO:0071035). The gene is also part of the [nucleolar exosome (RNase complex)](https://amigo.geneontology.org/amigo/term/GO:0101019) and is involved in [regulation of telomerase RNA localization to Cajal body](https://amigo.geneontology.org/amigo/term/GO:1904872) and [positive regulation of mRNA cis splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:1905746).


The gene length is 43,811.0 base pairs (62.8% of all genes), the mature length is 2,971.5 base pairs, and the primary transcript length is 42,725.0 base pairs.


The gene EXOSC10, with NCBI ID 5394, has been mentioned in [90 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22EXOSC10%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning this gene was in 1985, and the middle 50% of publications occurred between 2005 and 2017.


The top 5 publications mentioning EXOSC10, ranked by their scientific influence, include "[The yeast exosome and human PM-Scl are related complexes of 3' --> 5' exonucleases.](https://pubmed.ncbi.nlm.nih.gov/10465791)" (1999) (relative citation ratio: 6.04), "[Serum autoantibody to the nucleolar antigen PM-Scl. Clinical and immunogenetic associations.](https://pubmed.ncbi.nlm.nih.gov/1418007)" (1992) (relative citation ratio: 5.11), "[The clinical phenotype of systemic sclerosis patients with anti-PM/Scl antibodies: results from the EUSTAR cohort.](https://pubmed.ncbi.nlm.nih.gov/33580257)" (2021) (relative citation ratio: 4.73), "[Muscular and extramuscular clinical features of patients with anti-PM/Scl autoantibodies.](https://pubmed.ncbi.nlm.nih.gov/29728522)" (2018) (relative citation ratio: 4.39), and "[Rrp6p, the yeast homologue of the human PM-Scl 100-kDa autoantigen, is essential for efficient 5.8 S rRNA 3' end formation.](https://pubmed.ncbi.nlm.nih.gov/9582370)" (1998) (relative citation ratio: 4.0). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[EXOSC10](https://www.proteinatlas.org/ENSG00000171824-EXOSC10) is a gene with molecular functions including exonuclease, hydrolase, nuclease, and RNA-binding. Evidence for its function is available at the protein level. The gene is detected in all tissues and is primarily localized in the nucleoplasm, nucleoli, nucleoli rim, and cytosol. It is associated with various expression clusters, including Cluster 12 for blood (non-specific - mitochondrial translation), Cluster 16 for tissue (non-specific - transcription), Cluster 4 for brain (non-specific - mRNA splicing & cell cycle), Cluster 37 for cell line (non-specific - mRNA processing), and Cluster 49 for single cell (non-specific - transcription).


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), [AR](https://www.ncbi.nlm.nih.gov/gene/367), [JUN](https://www.ncbi.nlm.nih.gov/gene/3725), and [MYC](https://www.ncbi.nlm.nih.gov/gene/4609), each shown to be regulating in 4 experiments.





The gene is expressed in various tissues, including nerve, ovary, and pituitary, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in CD34+ cells, CD56+ NK cells, CD4+ T cells, CD8+ T cells, promyelocytic leukemia (HL-60), Burkitt's lymphoma (Daudi), and CD105+ endothelial cells.




The analyzed protein sequence has a GRAVY value of -0.606 (24.77th percentile), indicating a hydrophilic nature. It exhibits a charge of 11.15 at pH 7.0 (83.75th percentile) and a median structural flexibility of 1.007 (70.69th percentile). The protein's secondary structure is predicted to be 34.46% helix (71.26th percentile), 31.53% sheet (36.84th percentile), and 27.34% turn (41.28th percentile). The instability index is 46.91 (46.26th percentile), with an isoelectric point of 8.68 (70.55th percentile). The protein has a length of 885 amino acids (83.9th percentile) and a molecular weight of 100830 Da (84.54th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |