---
layout: post
title: HNRNPD
date: 2025-01-17 16:55 CST
description: HNRNPD description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3184) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3184  | HNRNPD | ENSG00000138668 | 4q21.22 |



The gene is involved in [liver development](https://amigo.geneontology.org/amigo/term/GO:0001889) and plays a role in the [regulation of DNA-templated transcription](https://amigo.geneontology.org/amigo/term/GO:0006355), [RNA processing](https://amigo.geneontology.org/amigo/term/GO:0006396), and [RNA catabolic process](https://amigo.geneontology.org/amigo/term/GO:0006401). It enables [minor groove of adenine-thymine-rich DNA binding](https://amigo.geneontology.org/amigo/term/GO:0003680), [chromatin binding](https://amigo.geneontology.org/amigo/term/GO:0003682), [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and is active in the nucleoplasm. Additionally, it is involved in the [regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0010468), [negative regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0010629), and various other processes such as [cerebellum development](https://amigo.geneontology.org/amigo/term/GO:0021549) and [regulation of telomere maintenance](https://amigo.geneontology.org/amigo/term/GO:0032204). The gene also enables [mRNA 3'-UTR AU-rich region binding](https://amigo.geneontology.org/amigo/term/GO:0035925) and [telomeric DNA binding](https://amigo.geneontology.org/amigo/term/GO:0042162), and is involved in the [regulation of circadian rhythm](https://amigo.geneontology.org/amigo/term/GO:0042752) and [positive regulation of translation](https://amigo.geneontology.org/amigo/term/GO:0045727). Furthermore, it is located in the [postsynaptic density](https://amigo.geneontology.org/amigo/term/GO:0014069) and [glutamatergic synapse](https://amigo.geneontology.org/amigo/term/GO:0098978), and is part of the [mCRD-mediated mRNA stability complex](https://amigo.geneontology.org/amigo/term/GO:0106002) and [ribonucleoprotein complex](https://amigo.geneontology.org/amigo/term/GO:1990904). The gene is also involved in various cellular responses and regulations, including [response to calcium ion](https://amigo.geneontology.org/amigo/term/GO:0051592), [response to electrical stimulus](https://amigo.geneontology.org/amigo/term/GO:0051602), and [positive regulation of cytoplasmic translation](https://amigo.geneontology.org/amigo/term/GO:2000767).


The gene length is 29,225 base pairs (51.97th percentile of all genes), the mature length is 6,745 base pairs, and the primary transcript length is 29,225 base pairs.


The gene HNRNPD (NCBI ID: 3184) has been mentioned in [148 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22HNRNPD%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1993 and the middle 50% of publications occurring between 2006 and 2016. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning HNRNPD, ranked by their scientific influence, include studies that have significantly contributed to the understanding of the gene's function and interactions. The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Notable publications include "[Purification, characterization, and cDNA cloning of an AU-rich element RNA-binding protein, AUF1.](https://pubmed.ncbi.nlm.nih.gov/8246982)" (1993) (relative citation ratio: 9.36), "[Concurrent versus individual binding of HuR and AUF1 to common labile target mRNAs.](https://pubmed.ncbi.nlm.nih.gov/15257295)" (2004) (relative citation ratio: 6.73), "[AUF1 protects against ferroptosis to alleviate sepsis-induced acute lung injury by regulating NRF2 and ATF3.](https://pubmed.ncbi.nlm.nih.gov/35391558)" (2022) (relative citation ratio: 5.94), "[AUF1 ligand circPCNX reduces cell proliferation by competing with p21 mRNA to increase p21 production.](https://pubmed.ncbi.nlm.nih.gov/33444453)" (2021) (relative citation ratio: 4.33), and "[AUF1 binding affinity to A+U-rich elements correlates with rapid mRNA degradation.](https://pubmed.ncbi.nlm.nih.gov/8647811)" (1996) (relative citation ratio: 4.3). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[HNRNPD](https://www.proteinatlas.org/ENSG00000138668-HNRNPD) is a gene with molecular functions including DNA-binding, ribonucleoprotein, and RNA-binding. It is detected in all RNA tissue distributions and is primarily located in the nucleoplasm. The gene is part of various expression clusters, including Cluster 30 for blood expression, Cluster 16 for tissue and brain expression, Cluster 22 for cell line expression, and Cluster 39 for single cell expression, all of which are non-specific but related to basic cellular processes such as DNA binding, transcription, mRNA splicing, and the cell cycle. Evidence for this gene is available at the protein level.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113), [TCF7L1](https://www.ncbi.nlm.nih.gov/gene/83439), [TP53](https://www.ncbi.nlm.nih.gov/gene/7157), and [STAT1](https://www.ncbi.nlm.nih.gov/gene/6772), each shown to be regulating in 5 experiments.





The gene is expressed in various immune cell types, including CD34+, CD4+ T cells, and CD8+ T cells, as well as in CD105+ endothelial cells. Additionally, expression is observed in the small intestine and leukemia lymphoblastic cells (MOLT-4). The gene is also expressed in the ovary.


The input data highlights several key processes related to gene expression and mRNA regulation. These include the processing of capped intron-containing pre-mRNA, mRNA splicing, and the major pathway of mRNA splicing. Additionally, the data mentions the role of AUF1 (hnRNP D0) in destabilizing mRNA and the regulation of mRNA stability by proteins that bind AU-rich elements.


The mouse ortholog gene 11991 and its human ortholog gene 3184 are associated with phenotypes including decreased leukocyte cell number and decreased lymphocyte cell number, with these effects observed in both sexes.


The protein sequence analyzed has a GRAVY value of -0.924 (6.5th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of 0.955 (53.8th percentile). The median structural flexibility is 1.016 (93.03rd percentile), suggesting high flexibility. The protein has a length of 355 amino acids (38.89th percentile) and a molecular weight of 38433.71 Da (37.08th percentile). The secondary structure affinities are 30.42% for helix (35.72nd percentile), 25.07% for sheet (8.88th percentile), and 35.77% for turn (91.27th percentile). The instability index is 38.99 (19.41st percentile), and the isoelectric point is 7.61 (55.32nd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |