---
layout: post
title: FEZ2
date: 2025-01-17 16:55 CST
description: FEZ2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9637) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9637  | FEZ2 | ENSG00000171055 | 2p22.2 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737) and [axon](https://amigo.geneontology.org/amigo/term/GO:0030424). It is involved in several biological processes, including [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), [nervous system development](https://amigo.geneontology.org/amigo/term/GO:0007399), [axon guidance](https://amigo.geneontology.org/amigo/term/GO:0007411), and [negative regulation of autophagosome assembly](https://amigo.geneontology.org/amigo/term/GO:1902902).


The gene length is 61,537 base pairs (71.57% of all genes), the mature length is 2,211 base pairs (57.78% of all genes), and the primary transcript length is 57,958 base pairs (78.56% of all genes).


The gene FEZ2 (NCBI ID: 9637) has been mentioned in [1 publication](https://pubmed.ncbi.nlm.nih.gov/?term=%22FEZ2%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 2011, and the middle 50% of publications range from 2011 to 2011.


The top publication mentioning FEZ2 is "[FEZ2 has acquired additional protein interaction partners relative to FEZ1: functional and evolutionary implications.](https://pubmed.ncbi.nlm.nih.gov/21408165)" (2011) (relative citation ratio: 0.3). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, and benchmarking to the median for NIH publications, the value of which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[FEZ2](https://www.proteinatlas.org/ENSG00000171055-FEZ2) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is localized in the nucleoli, nucleoli rim, Golgi apparatus, and cytosol. The gene is expressed in various clusters, including Cluster 29 for blood expression, Cluster 54 for tissue expression, Cluster 30 for brain expression, Cluster 72 for cell line expression, and Cluster 58 for single cell expression, which is associated with macrophages and innate immune response.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 5 experiments, [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046) with 4 experiments, [AR](https://www.ncbi.nlm.nih.gov/gene/367) with 4 experiments, [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885) with 4 experiments, and [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The input data indicates associations with various disease conditions, including diseases of metabolism, acquired metabolic diseases, nutrition diseases, overnutrition, obesity, cardiovascular system diseases, inherited metabolic disorders, and lipid metabolism disorders.



The gene is expressed in various tissues, including muscle and nerve, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, data from the GTEx Tissue Gene Expression Profiles show expression in the Prefrontal Cortex, BDCA4+ Dendritic Cells, Amygdala, Skeletal Muscle, and Olfactory Bulb.




The analyzed protein sequence has a GRAVY value of -0.685 (18.58th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -31.99 (5.61st percentile), and the median structural flexibility is 1.017 (93.83rd percentile). The protein has a high instability index of 64.39 (89.51st percentile) and an isoelectric point of 4.52 (1.68th percentile). The sequence length is 353 amino acids (38.55th percentile) with a molecular weight of 39665.82 Da (39.02nd percentile). The secondary structure affinities are 38.24% for helix (89.45th percentile), 30.88% for sheet (33.14th percentile), and 30.31% for turn (66.02nd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |