---
layout: post
title: PTPRJ
date: 2025-01-17 16:55 CST
description: PTPRJ description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5795) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5795  | PTPRJ | ENSG00000149177 | 11p11.2 |



The gene is involved in various biological processes, including [angiogenesis](https://amigo.geneontology.org/amigo/term/GO:0001525), [protein dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0006470), and [negative regulation of cell population proliferation](https://amigo.geneontology.org/amigo/term/GO:0008285). It is located in the [immunological synapse](https://amigo.geneontology.org/amigo/term/GO:0001772), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [cell-cell junction](https://amigo.geneontology.org/amigo/term/GO:0005911), [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986), [ruffle membrane](https://amigo.geneontology.org/amigo/term/GO:0032587), [specific granule membrane](https://amigo.geneontology.org/amigo/term/GO:0035579), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). The gene enables several molecular functions, such as [protein tyrosine phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0004725), [platelet-derived growth factor receptor binding](https://amigo.geneontology.org/amigo/term/GO:0005161), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [beta-catenin binding](https://amigo.geneontology.org/amigo/term/GO:0008013), [phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0016791), [protein kinase binding](https://amigo.geneontology.org/amigo/term/GO:0019901), [gamma-catenin binding](https://amigo.geneontology.org/amigo/term/GO:0045295), [cadherin binding](https://amigo.geneontology.org/amigo/term/GO:0045296), and [delta-catenin binding](https://amigo.geneontology.org/amigo/term/GO:0070097). Additionally, it plays roles in various regulatory processes, including [negative regulation of platelet-derived growth factor receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0010642), [cytokine-mediated signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0019221), [regulation of cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0030155), [B cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0030183), [negative regulation of cell growth](https://amigo.geneontology.org/amigo/term/GO:0030308), [negative regulation of cell migration](https://amigo.geneontology.org/amigo/term/GO:0030336), [positive regulation of tumor necrosis factor production](https://amigo.geneontology.org/amigo/term/GO:0032760), [peptidyl-tyrosine dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0035335), [calcium-mediated signaling using intracellular calcium source](https://amigo.geneontology.org/amigo/term/GO:0035584), [negative regulation of epidermal growth factor receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0042059), [negative regulation of vascular permeability](https://amigo.geneontology.org/amigo/term/GO:0043116), [negative regulation of MAP kinase activity](https://amigo.geneontology.org/amigo/term/GO:0043407), [positive regulation of MAPK cascade](https://amigo.geneontology.org/amigo/term/GO:0043410), [positive regulation of cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0045785), [platelet-derived growth factor receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0048008), [positive regulation of peptidyl-tyrosine phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0050731), [positive regulation of phagocytosis](https://amigo.geneontology.org/amigo/term/GO:0050766), [T cell receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0050852), [negative regulation of T cell receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0050860), [positive chemotaxis](https://amigo.geneontology.org/amigo/term/GO:0050918), [positive regulation of focal adhesion assembly](https://amigo.geneontology.org/amigo/term/GO:0051894), [positive regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction](https://amigo.geneontology.org/amigo/term/GO:0051897), [negative regulation of phosphatidylinositol 3-kinase/protein kinase B signal transduction](https://amigo.geneontology.org/amigo/term/GO:0051898), [contact inhibition](https://amigo.geneontology.org/amigo/term/GO:0060242), and [positive regulation of Fc receptor mediated stimulatory signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0060369).


The gene length is 121,303 base pairs (85.37th percentile), the mature length is 2,803 base pairs (69.56th percentile), and the primary transcript length is 106,815 base pairs (89.37th percentile).


The gene PTPRJ (NCBI ID: 5795) has been mentioned in [87 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PTPRJ%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1994 and the middle 50% of publications occurring between 2003 and 2015. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning PTPRJ, ranked by their scientific influence, include "[Contact inhibition of VEGF-induced proliferation requires vascular endothelial cadherin, beta-catenin, and the phosphatase DEP-1/CD148.](https://pubmed.ncbi.nlm.nih.gov/12771128)" (2003) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 5.21, "[Expression of DEP-1, a receptor-like protein-tyrosine-phosphatase, is enhanced with increasing cell density.](https://pubmed.ncbi.nlm.nih.gov/7937872)" (1994) with an RCR of 3.8, "[CD45, CD148, and Lyp/Pep: critical phosphatases regulating Src family kinase signaling networks in immune cells.](https://pubmed.ncbi.nlm.nih.gov/19290935)" (2009) with an RCR of 2.62, "[Hepatocyte growth factor receptor tyrosine kinase met is a substrate of the receptor protein-tyrosine phosphatase DEP-1.](https://pubmed.ncbi.nlm.nih.gov/12475979)" (2003) with an RCR of 2.41, and "[The protein tyrosine phosphatase DEP-1 is induced during differentiation and inhibits growth of breast cancer cells.](https://pubmed.ncbi.nlm.nih.gov/8797598)" (1996) with an RCR of 2.27. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PTPRJ](https://www.proteinatlas.org/ENSG00000149177-PTPRJ) is a gene encoding a protein with hydrolase and protein phosphatase molecular functions. Evidence for this gene is available at the protein level. The RNA expression of PTPRJ is detected in many tissues, with notable expression in lymphoid tissue and bone marrow, where it is involved in the innate immune response. Subcellularly, the protein is localized to the nucleoplasm, nucleoli, nuclear bodies, and cell junctions. In blood, PTPRJ is expressed in monocytes and neutrophils, playing a role in degranulation. Additionally, it is expressed in neurons, contributing to neuronal signaling.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 6 experiments, [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885) with 5 experiments, [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227) with 5 experiments, [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) with 5 experiments, and [KMT2B](https://www.ncbi.nlm.nih.gov/gene/9757) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent detection in multiple experiments.



The input data from GWAS indicates associations with a wide range of disease conditions, including various types of cancer such as breast cancer, thoracic cancer, male reproductive organ cancer, prostate cancer, testicular cancer, head and neck cancer, colorectal cancer, intestinal cancer, large intestine cancer, gastrointestinal system cancer, myeloid leukemia, acute myeloid leukemia, colon cancer, lung cancer, respiratory system cancer, leukemia, hematologic cancer, and immune system cancer. Additionally, the data includes associations with infectious diseases like diarrhea, hand, foot and mouth disease, and viral infectious diseases. Cardiovascular conditions such as coronary artery disease, artery disease, vascular disease, and cardiovascular system disease are also represented. Furthermore, the data encompasses mental health disorders including cognitive disorder, bipolar disorder, mood disorder, substance-related disorder, substance dependence, alcohol dependence, and nicotine dependence. Metabolic disorders such as inherited metabolic disorder, lipid metabolism disorder, and arteriosclerosis are noted, along with developmental disorders of mental health like attention deficit hyperactivity disorder. Other conditions include kidney disease, urinary system disease, carcinoma, squamous cell carcinoma, cell type cancer, and mouth disease.


The gene PTPRJ has been associated with [thyroid stimulating hormone levels](https://pubmed.ncbi.nlm.nih.gov/37872160), which maps closely to the MeSH term [Hormones](https://meshb.nlm.nih.gov/record/ui?ui=D006728) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method.


The gene is expressed in various tissues and cell types, including blood and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, data from the GTEx Tissue Gene Expression Profiles show expression in cardiac myocytes, Burkitt's lymphoma (Raji), heart, whole blood, superior cervical ganglion, and the atrioventricular node.




The analyzed protein sequence has a GRAVY value of -0.317 (56.09th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -30.27 (6.11th percentile), and the median structural flexibility is 1.003 (57.52nd percentile). The protein has a length of 1337 amino acids (93.6th percentile) and a molecular weight of 145939.7 Da (93.35th percentile). The secondary structure affinities are 24.23% for helix (3.77th percentile), 38.37% for sheet (80.41st percentile), and 33.21% for turn (82.91st percentile). The instability index is 41.22 (26.08th percentile), and the isoelectric point is 5.38 (15.31st percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |