---
layout: post
title: STK10
date: 2025-01-17 16:55 CST
description: STK10 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6793) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6793  | STK10 | ENSG00000072786 | 5q35.1 |



The gene enables [protein serine/threonine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004674) and [protein serine kinase activity](https://amigo.geneontology.org/amigo/term/GO:0106310), and is involved in [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524), [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802), and [protein homodimerization activity](https://amigo.geneontology.org/amigo/term/GO:0042803). It is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737) and located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [specific granule membrane](https://amigo.geneontology.org/amigo/term/GO:0035579), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). The gene is involved in several biological processes, including [protein phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0006468), [protein autophosphorylation](https://amigo.geneontology.org/amigo/term/GO:0046777), [cell cycle](https://amigo.geneontology.org/amigo/term/GO:0007049), [lymphocyte aggregation](https://amigo.geneontology.org/amigo/term/GO:0071593), and [regulation of lymphocyte migration](https://amigo.geneontology.org/amigo/term/GO:2000401).


The gene length is 385,444 base pairs (97.22% of all genes), the mature length is 3,169 base pairs (74.86% of all genes), and the primary transcript length is 95,806 base pairs (87.64% of all genes).


The gene STK10 (NCBI ID: 6793) has been mentioned in [85 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22STK10%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1962 and the middle 50% of publications occurring between 2007 and 2018. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning STK10, ranked by their scientific influence, include "[Accuracy of intravitreal injection volume for aflibercept pre-filled syringe and BD Luer-Lok one-milliliter syringe.](https://pubmed.ncbi.nlm.nih.gov/35382900)" (2022) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 4.11, "[Diagnostic Accuracy of APRI, AAR, FIB-4, FI, King, Lok, Forns, and FibroIndex Scores in Predicting the Presence of Esophageal Varices in Liver Cirrhosis: A Systematic Review and Meta-Analysis.](https://pubmed.ncbi.nlm.nih.gov/26496312)" (2015) with an RCR of 3.63, "[Hem-o-Lok clip migration into renal pelvis and stone formation as a long-term complication following laparoscopic pyelolithotomy: a case report and literature review.](https://pubmed.ncbi.nlm.nih.gov/35440078)" (2022) with an RCR of 2.52, "[LOK is a major ERM kinase in resting lymphocytes and regulates cytoskeletal rearrangement through ERM phosphorylation.](https://pubmed.ncbi.nlm.nih.gov/19255442)" (2009) with an RCR of 2.26, and "[Influence of Laser-Lok surface on immediate functional loading of implants in single-tooth replacement: a 2-year prospective clinical study.](https://pubmed.ncbi.nlm.nih.gov/24396842)" (2014) with an RCR of 2.19. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[STK10](https://www.proteinatlas.org/ENSG00000072786-STK10) is a serine/threonine-protein kinase with transferase activity, functioning as a kinase. Evidence supports its existence at the protein level. The gene is detected in all RNA tissue distributions and is primarily localized in the nucleoplasm, nuclear bodies, and plasma membrane. STK10 is notably expressed in monocytes and neutrophils, contributing to the innate immune response, and is also found in lymphoid tissue and bone marrow. In the brain, its expression is non-specific and related to transcription. Additionally, STK10 is expressed in myeloid leukemia cell lines and NK-cells, playing roles in both innate and adaptive immune responses.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) and [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025), each with 6 experiments. Additionally, [NFYA](https://www.ncbi.nlm.nih.gov/gene/4800), [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613), and [MYB](https://www.ncbi.nlm.nih.gov/gene/4602) were each found to be regulating in 5 experiments.



The GWAS data indicates associations with a wide range of disease conditions, including various types of cancer such as lung cancer and respiratory system cancer, metabolic disorders like type 2 diabetes mellitus and carbohydrate metabolism disease, cardiovascular diseases including coronary artery disease and hypertension, kidney disease, and nervous system disorders such as schizophrenia and migraine. Additionally, the data highlights associations with cognitive disorders and psychotic disorders, as well as inherited metabolic disorders like lipid metabolism disorder and arteriosclerosis.



The gene is expressed in multiple tissue samples, including lung, blood, and spleen. Additionally, it is expressed in various immune cell types such as CD33+ myeloid cells, BDCA4+ dendritic cells, CD56+ NK cells, CD4+ T cells, CD8+ T cells, and CD14+ monocytes, as well as in whole blood.




The analyzed protein sequence has a GRAVY value of -0.949 (5.69th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -4.34 (35.43rd percentile), and the median structural flexibility is 1.016 (93.48th percentile). The protein has a high affinity for helix structures (0.414, 95.41st percentile), with lower affinities for sheet (0.268, 13.56th percentile) and turn (0.237, 13.19th percentile) structures. The instability index is 59.64 (82.42nd percentile), suggesting potential instability. The isoelectric point is 6.52 (43.03rd percentile), with a length of 968 amino acids (86.74th percentile) and a molecular weight of 112134 Da (87.73rd percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |