---
layout: post
title: RASA4
date: 2025-01-17 16:55 CST
description: RASA4 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10156) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10156  | RASA4 | ENSG00000105808 | 7q22.1 |



The gene enables [GTPase activator activity](https://amigo.geneontology.org/amigo/term/GO:0005096) and [phospholipid binding](https://amigo.geneontology.org/amigo/term/GO:0005543), and is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). It colocalizes with the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886) and is involved in the [negative regulation of GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0034260), [intracellular signal transduction](https://amigo.geneontology.org/amigo/term/GO:0035556), [negative regulation of Ras protein signal transduction](https://amigo.geneontology.org/amigo/term/GO:0046580), and [cellular response to calcium ion](https://amigo.geneontology.org/amigo/term/GO:0071277). Additionally, it enables [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872).


The gene length is 123,107.0 base pairs (85.69% of all genes), the mature length is 655.0 base pairs (7.56% of all genes), and the primary transcript length is 79,752.0 base pairs (84.68% of all genes).


The gene RASA4, with NCBI ID 10156, has been mentioned in [122 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RASA4%22) (note: the count in the hyperlink may be different since the count is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication mentioning this gene is 1964, and the middle 50% of publications occurred between 2001 and 2014.


The top 5 publications mentioning RASA4, ranked by their scientific influence, include "[Digital remote monitoring plus usual care versus usual care in patients treated with oral anticancer agents: the randomized phase 3 CAPRI trial.](https://pubmed.ncbi.nlm.nih.gov/35469070)" (2022) (relative citation ratio: 5.97), "[Prediction of protein assemblies, the next frontier: The CASP14-CAPRI experiment.](https://pubmed.ncbi.nlm.nih.gov/34453465)" (2021) (relative citation ratio: 5.95), "[Modeling protein-protein, protein-peptide, and protein-oligosaccharide complexes: CAPRI 7th edition.](https://pubmed.ncbi.nlm.nih.gov/31886916)" (2020) (relative citation ratio: 5.26), "[Achieving reliability and high accuracy in automated protein docking: ClusPro, PIPER, SDU, and stability analysis in CAPRI rounds 13-19.](https://pubmed.ncbi.nlm.nih.gov/20818657)" (2010) (relative citation ratio: 4.9), and "[The challenge of modeling protein assemblies: the CASP12-CAPRI experiment.](https://pubmed.ncbi.nlm.nih.gov/29127686)" (2018) (relative citation ratio: 3.8). The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RASA4](https://www.proteinatlas.org/ENSG00000105808-RASA4) is a gene involved in GTPase activation, with evidence supporting its function at the protein level. The RNA expression of RASA4 is detected in all tissues, and the protein is localized to vesicles and cell junctions. In terms of expression clusters, RASA4 is associated with dendritic cells of unknown function in blood, skeletal muscle involved in striated muscle contraction, astrocytes involved in astrocyte-neuron interactions in the brain, connective tissue cells involved in ECM organization in cell lines, and non-specific transcription in single cells.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599), [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), [ETS2](https://www.ncbi.nlm.nih.gov/gene/2114), [EGR1](https://www.ncbi.nlm.nih.gov/gene/1958), and [SMARCB1](https://www.ncbi.nlm.nih.gov/gene/6598), each shown to be regulating in 2 experiments.





The gene is expressed in various immune cell types, including CD34+, CD33+ myeloid cells, CD4+ T cells, CD8+ T cells, CD19+ B cells, and CD14+ monocytes, as well as in lung tissue. Additionally, it is expressed in muscle tissue.




The analyzed protein sequence has a GRAVY value of -0.359 (51.11th percentile), indicating a hydrophilic nature. It exhibits a charge of 4.4 at pH 7.0 (65.96th percentile) and a median structural flexibility of 1.002 (50.6th percentile). The protein's secondary structure is predicted to be 34% helix (67.81st percentile), 35.12% sheet (59.98th percentile), and 26.15% turn (30.55th percentile). The instability index is 47.04 (46.78th percentile), with an isoelectric point of 8.01 (58.9th percentile). The protein has a length of 803 amino acids (80.9th percentile) and a molecular weight of 90456.52 Da (81.03rd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |