---
layout: post
title: RAB4A
date: 2025-01-17 16:55 CST
description: RAB4A description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5867) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5867  | RAB4A | ENSG00000168118 | 1q42.13 |



The gene enables [GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0003924) and [G protein activity](https://amigo.geneontology.org/amigo/term/GO:0003925), and is involved in [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [GTP binding](https://amigo.geneontology.org/amigo/term/GO:0005525). It is also involved in various processes such as [protein transport](https://amigo.geneontology.org/amigo/term/GO:0015031), [vesicle-mediated transport](https://amigo.geneontology.org/amigo/term/GO:0016192), [antigen processing and presentation](https://amigo.geneontology.org/amigo/term/GO:0019882), and [regulation of endocytosis](https://amigo.geneontology.org/amigo/term/GO:0030100). The gene is located in the [cytoplasmic vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030659), [early endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0031901), [vesicle](https://amigo.geneontology.org/amigo/term/GO:0031982), [perinuclear region of cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0048471), [recycling endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0055038), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). Additionally, it is active in the [insulin-responsive compartment](https://amigo.geneontology.org/amigo/term/GO:0032593) and [postsynaptic recycling endosome](https://amigo.geneontology.org/amigo/term/GO:0098837).


The gene length is 80,186 base pairs (77.42% of all genes), the mature length is 1,789.5 base pairs, and the primary transcript length is 76,223 base pairs (83.88% of all genes).


RAB4A (NCBI Gene ID: 5867) has been mentioned in [79 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RAB4A%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1988 and the middle 50% of publications occurring between 2001 and 2013. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning RAB4A, ranked by their scientific influence, include "[The small GTP-binding protein rab4 controls an early sorting event on the endocytic pathway.](https://pubmed.ncbi.nlm.nih.gov/1516131)" (1992) (relative citation ratio: 10.25), "[Rab4 and cellubrevin define different early endosome populations on the pathway of transferrin receptor recycling.](https://pubmed.ncbi.nlm.nih.gov/8790369)" (1996) (relative citation ratio: 5.05), "[The small GTP-binding protein rab4 is associated with early endosomes.](https://pubmed.ncbi.nlm.nih.gov/1906178)" (1991) (relative citation ratio: 4.76), "[Distinct Rab-binding domains mediate the interaction of Rabaptin-5 with GTP-bound Rab4 and Rab5.](https://pubmed.ncbi.nlm.nih.gov/9524117)" (1998) (relative citation ratio: 2.95), and "[From sorting endosomes to exocytosis: association of Rab4 and Rab11 GTPases with the Fc receptor, FcRn, during recycling.](https://pubmed.ncbi.nlm.nih.gov/15689494)" (2005) (relative citation ratio: 2.59). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RAB4A](https://www.proteinatlas.org/ENSG00000168118-RAB4A) is a gene encoding a protein with hydrolase molecular function, supported by evidence at the protein level. Its RNA is detected in all tissues, and it is expressed in various clusters including Cluster 45 in blood (Eosinophils - Unknown function), Cluster 68 in tissues (Non-specific - Protein processing), Cluster 38 in the brain (Sub-cortical - Mixed function), Cluster 45 in cell lines (Non-specific - Nuclear processes), and Cluster 75 in single cells (Serous glandular cells - Salivary secretion).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046) and [PGR](https://www.ncbi.nlm.nih.gov/gene/5241), each with 5 experiments. Additionally, [MAX](https://www.ncbi.nlm.nih.gov/gene/4149), [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113), and [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930) were each found to be regulating in 4 experiments.



The GWAS data indicates associations with several disease conditions, including artery disease, vascular disease, cardiovascular system disease, hypertension, and broader categories such as disease and disease of anatomical entity.



The gene is expressed in various tissues, including the esophagus, salivary gland, and stomach, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the thyroid, amygdala, whole brain, bronchial epithelial cells, prostate, trachea, colon, and small intestine, suggesting that the gene is ubiquitously expressed in multiple tissue samples.


The input data highlights two key pathways: membrane trafficking and the translocation of GLUT4 to the plasma membrane. These processes are integral to cellular function and glucose homeostasis, respectively.



The analyzed protein sequence has a GRAVY value of -0.315 (56.32 percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -2.33 (41.9 percentile). The median structural flexibility is 1.002 (51.37 percentile), with a predicted secondary structure composition of 33.94% helix (67.37 percentile), 34.86% sheet (58.01 percentile), and 25.69% turn (26.66 percentile). The instability index is 43.87 (35.25 percentile), and the isoelectric point is 5.82 (25.82 percentile). The protein has a length of 218 amino acids (17.19 percentile) and a molecular weight of 24389.37 Da (17.19 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |