---
layout: post
title: DGKA
date: 2025-01-17 16:55 CST
description: DGKA description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1606) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1606  | DGKA | ENSG00000065357 | 12q13.2 |



The gene enables [ATP-dependent diacylglycerol kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004143), [calcium ion binding](https://amigo.geneontology.org/amigo/term/GO:0005509), [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524), [phospholipid binding](https://amigo.geneontology.org/amigo/term/GO:0005543), [lipid binding](https://amigo.geneontology.org/amigo/term/GO:0008289), and [kinase activity](https://amigo.geneontology.org/amigo/term/GO:0016301). It is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), and [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020). The gene is involved in the [phosphatidic acid biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006654), [protein kinase C-activating G protein-coupled receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007205), [platelet activation](https://amigo.geneontology.org/amigo/term/GO:0030168), [intracellular signal transduction](https://amigo.geneontology.org/amigo/term/GO:0035556), [diacylglycerol metabolic process](https://amigo.geneontology.org/amigo/term/GO:0046339), [glycerolipid metabolic process](https://amigo.geneontology.org/amigo/term/GO:0046486), and [lipid phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0046834). Additionally, it enables [alkylglycerol kinase activity](https://amigo.geneontology.org/amigo/term/GO:0047649).


The gene length is 2,648 base pairs (7.26% of all genes). The mature length is 2,648 base pairs (66.75% of all genes). The primary transcript length is 2,648 base pairs (9.3% of all genes).


The gene DGKA, with NCBI ID 1606, has been mentioned in [73 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22DGKA%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727). The earliest year of publication is 1989, and the middle 50% of publications occurred between 2007 and 2019. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning DGKA, ranked by their scientific influence, include "[T cell anergy is reversed by active Ras and is regulated by diacylglycerol kinase-alpha.](https://pubmed.ncbi.nlm.nih.gov/17028589)" (2006) (relative citation ratio: 3.54), "[Diacylglycerol kinase α regulates the formation and polarisation of mature multivesicular bodies involved in the secretion of Fas ligand-containing exosomes in T lymphocytes.](https://pubmed.ncbi.nlm.nih.gov/21252909)" (2011) (relative citation ratio: 2.58), "[Diacylglycerol kinase α controls RCP-dependent integrin trafficking to promote invasive migration.](https://pubmed.ncbi.nlm.nih.gov/22270919)" (2012) (relative citation ratio: 2.53), "[Diacylglycerol kinase α is a critical signaling node and novel therapeutic target in glioblastoma and other cancers.](https://pubmed.ncbi.nlm.nih.gov/23558954)" (2013) (relative citation ratio: 2.28), and "[Role of diacylglycerol kinase alpha in the attenuation of receptor signaling.](https://pubmed.ncbi.nlm.nih.gov/11285286)" (2001) (relative citation ratio: 2.0). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[DGKA](https://www.proteinatlas.org/ENSG00000065357-DGKA) is a gene encoding a protein with kinase and transferase molecular functions. Evidence for this protein is established at the protein level. The gene is detected in all RNA tissue distributions and is primarily localized in the nucleoplasm, vesicles, and mitochondria. DGKA is notably expressed in T-cells, as indicated by its blood expression cluster, and is associated with cell proliferation in lymphoid tissue and bone marrow. Additionally, it is expressed in leukemia cell lines and T-cells in single-cell expression clusters.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [TAF1](https://www.ncbi.nlm.nih.gov/gene/6872) and [MYC](https://www.ncbi.nlm.nih.gov/gene/4609), each with 6 experiments. Additionally, [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908), [KLF5](https://www.ncbi.nlm.nih.gov/gene/688), and [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664) were each found to be regulating in 5 experiments.





The gene is expressed in various tissues and cell types, including the spleen and small intestine, as indicated by BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles show expression in BDCA4+ dendritic cells, CD4+ T cells, CD8+ T cells, bronchial epithelial cells, skeletal muscle, tonsil, thymus, and lymph node, suggesting a ubiquitous expression pattern across multiple tissue samples.


The proteins are involved in various signaling pathways, including G alpha (q) signaling events, the gastrin-CREB signaling pathway via PKC and MAPK, GPCR downstream signaling, and signaling by GPCR. Additionally, they participate in signal transduction, the effects of PIP2 hydrolysis, hemostasis, and platelet activation, signaling, and aggregation.



The protein sequence analyzed has a GRAVY value of -0.264 (62.16th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -5.98 (31.08th percentile). The median structural flexibility is 0.9999 (41.31st percentile), with a secondary structure composition of 30.61% helix (37.29th percentile), 35.92% sheet (65.67th percentile), and 29.93% turn (63.41st percentile). The instability index is 42.39 (30.04th percentile), and the isoelectric point is 6.28 (37.67th percentile). The protein has a length of 735 amino acids (77.36th percentile) and a molecular weight of 82629.33 Da (77.58th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |