---
layout: post
title: LDLR
date: 2025-01-17 16:55 CST
description: LDLR description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3949) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3949  | LDLR | ENSG00000130164 | 19p13.2 |



The gene is involved in various molecular functions, including [amyloid-beta binding](https://amigo.geneontology.org/amigo/term/GO:0001540), [virus receptor activity](https://amigo.geneontology.org/amigo/term/GO:0001618), [protease binding](https://amigo.geneontology.org/amigo/term/GO:0002020), and [low-density lipoprotein particle receptor activity](https://amigo.geneontology.org/amigo/term/GO:0005041). It also enables [calcium ion binding](https://amigo.geneontology.org/amigo/term/GO:0005509) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is located in several cellular components, including the [lysosome](https://amigo.geneontology.org/amigo/term/GO:0005764), [early endosome](https://amigo.geneontology.org/amigo/term/GO:0005769), [late endosome](https://amigo.geneontology.org/amigo/term/GO:0005770), [Golgi apparatus](https://amigo.geneontology.org/amigo/term/GO:0005794), and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). It is active in the [early endosome](https://amigo.geneontology.org/amigo/term/GO:0005769) and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). The gene is involved in numerous biological processes such as [negative regulation of receptor recycling](https://amigo.geneontology.org/amigo/term/GO:0001920), [lipid metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006629), [endocytosis](https://amigo.geneontology.org/amigo/term/GO:0006897), [receptor-mediated endocytosis](https://amigo.geneontology.org/amigo/term/GO:0006898), and [phagocytosis](https://amigo.geneontology.org/amigo/term/GO:0006909). Additionally, it plays roles in [long-term memory](https://amigo.geneontology.org/amigo/term/GO:0007616), [cholesterol metabolic process](https://amigo.geneontology.org/amigo/term/GO:0008203), and [positive regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0010628). The gene is also part of the [low-density lipoprotein particle](https://amigo.geneontology.org/amigo/term/GO:0034362) and the [PCSK9-LDLR complex](https://amigo.geneontology.org/amigo/term/GO:1990666).


The gene length is 174,474 base pairs (90.88% of all genes), the mature length is 5,018 base pairs (90.42% of all genes), and the primary transcript length is 96,903 base pairs (87.8% of all genes).


The gene LDLR (NCBI ID: 3949) has been mentioned in [3,126 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22LDLR%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication mentioning this gene is 1948, and the middle 50% of publications occurred between 1995 and 2017.


The top 5 publications mentioning LDLR, ranked by their scientific influence, are as follows: "[Receptor-mediated endocytosis: concepts emerging from the LDL receptor system.](https://pubmed.ncbi.nlm.nih.gov/2881559)" (1985) (relative citation ratio: 34.25), "[The human LDL receptor: a cysteine-rich protein with multiple Alu sequences in its mRNA.](https://pubmed.ncbi.nlm.nih.gov/6091915)" (1984) (relative citation ratio: 29.77), "[Diagnostic Yield and Clinical Utility of Sequencing Familial Hypercholesterolemia Genes in Patients With Severe Hypercholesterolemia.](https://pubmed.ncbi.nlm.nih.gov/27050191)" (2016) (relative citation ratio: 24.28), "[Surface location and high affinity for calcium of a 500-kd liver membrane protein closely related to the LDL-receptor suggest a physiological role as lipoprotein receptor.](https://pubmed.ncbi.nlm.nih.gov/3266596)" (1988) (relative citation ratio: 21.6), and "[Association of Triglyceride-Lowering LPL Variants and LDL-C-Lowering LDLR Variants With Risk of Coronary Heart Disease.](https://pubmed.ncbi.nlm.nih.gov/30694319)" (2019) (relative citation ratio: 21.21). The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[LDLR](https://www.proteinatlas.org/ENSG00000130164-LDLR) functions as a host cell receptor for virus entry and a receptor, with evidence at the protein level. It is involved in disease variants and is detected in all RNA tissue distributions. The protein is localized in vesicles and is expressed in various clusters, including Cluster 41 in basophils with unknown function, Cluster 68 in non-specific tissues related to protein processing, Cluster 49 in the hindbrain with mixed function, Cluster 53 in squamous epithelial cells related to keratinization, and Cluster 72 in alveolar cells associated with lung function.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), [E2F1](https://www.ncbi.nlm.nih.gov/gene/1869), [SMAD3](https://www.ncbi.nlm.nih.gov/gene/4088), [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), and [LEO1](https://www.ncbi.nlm.nih.gov/gene/123169), each shown to be regulating in 5 experiments.



The input data from GWAS indicates associations with a wide range of disease conditions, including various types of cancer such as gastrointestinal system cancer, lung cancer, esophageal cancer, and stomach cancer. Additionally, metabolic disorders are prominently featured, encompassing diabetes mellitus, type 2 diabetes mellitus, obesity, and lipid metabolism disorders. Cardiovascular diseases, including coronary artery disease, arteriosclerosis, and atherosclerosis, are also implicated. Furthermore, the data highlights musculoskeletal system diseases, such as myopathy and muscular disease, as well as gastrointestinal system diseases like stomach disease and gastric outlet obstruction.


The LDLR gene has been associated with various traits and diseases through genome-wide association studies. Notably, it has been linked to [Apolipoprotein B levels](https://pubmed.ncbi.nlm.nih.gov/33462484), which maps to the MeSH term [Apolipoproteins](https://meshb.nlm.nih.gov/record/ui?ui=D001053) using the BioWordVec method. Additionally, LDLR is associated with [Cholesteryl esters in medium HDL](https://pubmed.ncbi.nlm.nih.gov/36764567), mapped to the MeSH term [Lipids](https://meshb.nlm.nih.gov/record/ui?ui=D008055), and [Coronary artery disease](https://pubmed.ncbi.nlm.nih.gov/33020668), which directly matches the MeSH term [Coronary Artery Disease](https://meshb.nlm.nih.gov/record/ui?ui=D003324). Other associations include [Direct low density lipoprotein cholesterol levels](https://pubmed.ncbi.nlm.nih.gov/34226706), mapped to [Lipoproteins](https://meshb.nlm.nih.gov/record/ui?ui=D008074), and [Free cholesterol levels in IDL](https://pubmed.ncbi.nlm.nih.gov/32150548), mapped to [Cholesterol](https://meshb.nlm.nih.gov/record/ui?ui=D002784). Furthermore, LDLR is linked to [LDL cholesterol](https://pubmed.ncbi.nlm.nih.gov/19060911), which directly matches the MeSH term [Cholesterol, LDL](https://meshb.nlm.nih.gov/record/ui?ui=D008078), and [LDL cholesterol levels](https://pubmed.ncbi.nlm.nih.gov/26690388), mapped to [Cholesterol](https://meshb.nlm.nih.gov/record/ui?ui=D002784). The gene is also associated with [Low density lipoprotein cholesterol levels](https://pubmed.ncbi.nlm.nih.gov/31217584), mapped to [Lipoproteins](https://meshb.nlm.nih.gov/record/ui?ui=D008074), and [Serum metabolite levels](https://pubmed.ncbi.nlm.nih.gov/31636271), mapped to [Serum](https://meshb.nlm.nih.gov/record/ui?ui=D044967). Lastly, LDLR is linked to [Total cholesterol levels](https://pubmed.ncbi.nlm.nih.gov/29403010), mapped to [Cholesterol](https://meshb.nlm.nih.gov/record/ui?ui=D002784).


The gene is expressed in various tissues, including the salivary gland and adrenal gland, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in leukemia (chronic myelogenous K-562), adipocytes, smooth muscle, bronchial epithelial cells, colorectal adenocarcinoma, adrenal cortex, and adrenal gland.


The input data highlights several pathways associated with visual transduction, including visual phototransduction and retinoid metabolism and transport. Additionally, it covers various lipid-related processes such as chylomicron-mediated lipid transport, lipid digestion, mobilization, and transport, lipoprotein metabolism, and LDL-mediated lipid transport. The data also encompasses broader categories like metabolism, specifically metabolism of lipids and lipoproteins, as well as disease and signal transduction.


The mouse ortholog gene 16835 and its human ortholog gene 3949 are associated with several phenotypic effects. Notably, there is a decrease in circulating unsaturated transferrin levels in one sex, while both sexes exhibit increased circulating levels of HDL cholesterol, total cholesterol, and iron.


The protein sequence analyzed has a GRAVY value of -0.389 (47.46th percentile), indicating a hydrophilic nature. It exhibits a charge of -40.3 at pH 7.0 (3.81st percentile) and a median structural flexibility of 0.9997 (40.21st percentile). The protein's secondary structure is predicted to be 23.72% helix (3.18th percentile), 31.74% sheet (38.04th percentile), and 33.37% turn (83.66th percentile). The instability index is 36.08 (12.25th percentile), with an isoelectric point of 4.86 (5.42nd percentile). The protein has a length of 860 amino acids (83.06th percentile) and a molecular weight of 95375.16 Da (82.79th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |