---
layout: post
title: NUTF2
date: 2025-01-17 16:55 CST
description: NUTF2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10204) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10204  | NUTF2 | ENSG00000102898 | 16q22.1 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is located in the [nuclear inner membrane](https://amigo.geneontology.org/amigo/term/GO:0005637), [nuclear outer membrane](https://amigo.geneontology.org/amigo/term/GO:0005640), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). It is involved in [protein import into nucleus](https://amigo.geneontology.org/amigo/term/GO:0006606) and [protein export from nucleus](https://amigo.geneontology.org/amigo/term/GO:0006611). Additionally, it enables [structural constituent of nuclear pore](https://amigo.geneontology.org/amigo/term/GO:0017056) and [small GTPase binding](https://amigo.geneontology.org/amigo/term/GO:0031267). The gene is also located in the [nuclear membrane](https://amigo.geneontology.org/amigo/term/GO:0031965) and is part of the [nuclear pore central transport channel](https://amigo.geneontology.org/amigo/term/GO:0044613). Furthermore, it is involved in [mRNA transport](https://amigo.geneontology.org/amigo/term/GO:0051028) and enables [nuclear import signal receptor activity](https://amigo.geneontology.org/amigo/term/GO:0061608). The gene is located in the [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062) and is involved in [protein localization to nuclear pore](https://amigo.geneontology.org/amigo/term/GO:0090204).


The gene length is 106,544 base pairs (83.08% of all genes), the mature length is 4,058 base pairs, and the primary transcript length is 106,454.5 base pairs.


The gene NUTF2 (NCBI ID: 10204) has been mentioned in [81 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22NUTF2%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1980 and the middle 50% of publications occurring between 2006 and 2016. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning NUTF2, ranked by their scientific influence, include "[Identification of NTF2, a cytosolic factor for nuclear import that interacts with nuclear pore complex protein p62.](https://pubmed.ncbi.nlm.nih.gov/7744965)" (1995) with a relative citation ratio of 5.95, "[NTF2 mediates nuclear import of Ran.](https://pubmed.ncbi.nlm.nih.gov/9822603)" (1998) with a relative citation ratio of 5.77, "[SARS-CoV-2 Nucleocapsid Protein Targets a Conserved Surface Groove of the NTF2-like Domain of G3BP1.](https://pubmed.ncbi.nlm.nih.gov/35240128)" (2022) with a relative citation ratio of 3.61, "[High-accuracy vapor pressure data of the extended [C(n)C1im][Ntf2] ionic liquid series: trend changes and structural shifts.](https://pubmed.ncbi.nlm.nih.gov/21815684)" (2011) with a relative citation ratio of 3.45, and "[Structural basis for the recognition of a nucleoporin FG repeat by the NTF2-like domain of the TAP/p15 mRNA nuclear export factor.](https://pubmed.ncbi.nlm.nih.gov/11583626)" (2001) with a relative citation ratio of 2.94. The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[NUTF2](https://www.proteinatlas.org/ENSG00000102898-NUTF2) is a gene with evidence at the protein level. Its RNA is detected in all tissues, and it is associated with various expression clusters, including Cluster 13 for blood expression, Cluster 35 for tissue expression, Cluster 27 for brain expression, Cluster 68 for cell line expression, and Cluster 15 for single cell expression. These clusters suggest non-specific roles related to mitochondria, mixed function, and cell cycle regulation.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [SIN3A](https://www.ncbi.nlm.nih.gov/gene/25942), [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), [HIF1A](https://www.ncbi.nlm.nih.gov/gene/3091), [AFF1](https://www.ncbi.nlm.nih.gov/gene/4299), and [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313), each shown to be regulating in 5 experiments.



The GWAS data indicates associations with a range of disease conditions, including hematopoietic system disease, immune system disease, and various metabolic disorders such as diabetes mellitus, particularly type 2 diabetes mellitus. Additionally, the data highlights links to obesity, coronary artery disease, and other cardiovascular system diseases. Inherited metabolic disorders, including lipid metabolism disorders and specific conditions like atherosclerosis and cholesterol ester storage disease, are also implicated.



The gene is expressed in various tissues, including blood vessel, uterus, skin, and brain, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the heart, 721_B_lymphoblasts, Leukemia_promyelocytic-HL-60, BronchialEpithelialCells, and Leukemialymphoblastic (MOLT-4).



The mouse ortholog gene 68051, which corresponds to the human ortholog gene 10204, is associated with several phenotypes including abnormal lens morphology, cataract, and preweaning lethality with complete penetrance. These phenotypes are observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.201 (69.04th percentile), indicating a hydrophilic nature. The charge at pH 7 is -6.08 (30.81st percentile), and the median structural flexibility is 0.993 (15.24th percentile). The protein has a propensity for helix, sheet, and turn structures of 26.0% (7.6th percentile), 35.4% (62.25th percentile), and 28.3% (50.42nd percentile), respectively. The instability index is 31.2 (5.01st percentile), and the isoelectric point is 5.10 (9.59th percentile). The protein is 127 amino acids long (5.42nd percentile) with a molecular weight of 14478.3 Da (5.81st percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |