---
layout: post
title: NME3
date: 2025-01-17 16:55 CST
description: NME3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4832) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4832  | NME3 | ENSG00000103024 | 16p13.3 |



The gene enables [nucleoside diphosphate kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004550), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524). It is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [mitochondrial outer membrane](https://amigo.geneontology.org/amigo/term/GO:0005741), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and [ciliary basal body](https://amigo.geneontology.org/amigo/term/GO:0036064). The gene is involved in various processes including [GTP biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006183), [UTP biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006228), [CTP biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006241), [DNA repair](https://amigo.geneontology.org/amigo/term/GO:0006281), [apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0006915), [mitochondrial fusion](https://amigo.geneontology.org/amigo/term/GO:0008053), [nucleoside triphosphate biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0009142), and [phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0016310). Additionally, it enables [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872).


The gene length is 100,564 base pairs (81.88% of all genes), the mature length is 3,259 base pairs, and the primary transcript length is 100,078 base pairs.


The gene NME3 (NCBI ID: 4832) has been mentioned in [17 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22NME3%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1995, and the middle 50% of publications range from 2001 to 2018.


The top 5 publications mentioning NME3, ranked by their scientific influence as measured by the [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541), include studies that have significantly contributed to the understanding of this gene's functions. The most influential publication is "[Overexpression of DR-nm23, a protein encoded by a member of the nm23 gene family, inhibits granulocyte differentiation and induces apoptosis in 32Dc13 myeloid cells.](https://pubmed.ncbi.nlm.nih.gov/7638209)" (1995) with an RCR of 3.26. Another notable study is "[The Lewis Base Stabilized Parent Arsanylborane H2AsBH2â‹…NMe3.](https://pubmed.ncbi.nlm.nih.gov/23907931)" (2013), which has an RCR of 1.26. Additionally, "[Two separate functions of NME3 critical for cell survival underlie a neurodegenerative disorder.](https://pubmed.ncbi.nlm.nih.gov/30587587)" (2019) has an RCR of 1.24, and "[Gene structure, promoter activity, and chromosomal location of the DR-nm23 gene, a related member of the nm23 gene family.](https://pubmed.ncbi.nlm.nih.gov/9067290)" (1997) has an RCR of 1.01. Lastly, "[Mammalian Homologue NME3 of DYNAMO1 Regulates Peroxisome Division.](https://pubmed.ncbi.nlm.nih.gov/33126676)" (2020) has an RCR of 0.76. The citation counts are sourced from [iCite](https://icite.od.nih.gov).


[NME3](https://www.proteinatlas.org/ENSG00000103024-NME3) is a gene with molecular functions as a kinase and transferase. Evidence for its function is available at the protein level. The gene's RNA is detected in all tissues, and it is associated with various expression clusters, including Cluster 12 for blood expression (non-specific - mitochondrial translation), Cluster 66 for tissue expression (non-specific - transcription), Cluster 27 for brain expression (non-specific - mixed function), Cluster 54 for cell line expression (non-specific - transcription), and Cluster 33 for single cell expression (non-specific - DNA binding).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [MYC](https://www.ncbi.nlm.nih.gov/gene/4609), [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), and [MAX](https://www.ncbi.nlm.nih.gov/gene/4149), each with 5 experiments. Additionally, [YY1](https://www.ncbi.nlm.nih.gov/gene/7528) and [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599) were found to be regulating in 4 experiments each.





The gene is expressed in various tissues, including the prostate, pituitary, and adrenal gland, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the pineal gland (both day and night), heart, CD4+ T cells, CD8+ T cells, prostate, and lung.




The protein sequence analyzed has a GRAVY value of 0.0237 (85.08th percentile), indicating a slightly hydrophobic nature. At pH 7.0, the protein carries a charge of 0.852 (53.35th percentile). The median structural flexibility is 0.995 (19.65th percentile), suggesting a relatively rigid structure. The protein has a propensity for helix and sheet structures, with affinities of 0.355 (77.93rd percentile) and 0.379 (77.84th percentile), respectively, and a lower affinity for turns at 0.219 (6.17th percentile). The instability index is 50.78 (59.8th percentile), and the isoelectric point is 7.71 (56.36th percentile). The protein is 169 amino acids long (10.36th percentile) with a molecular weight of 19014.82 Da (10.45th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |