---
layout: post
title: SOCS3
date: 2025-01-17 16:55 CST
description: SOCS3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9021) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9021  | SOCS3 | ENSG00000184557 | 17q25.3 |



The gene enables [phosphotyrosine residue binding](https://amigo.geneontology.org/amigo/term/GO:0001784), [protein kinase inhibitor activity](https://amigo.geneontology.org/amigo/term/GO:0004860), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [protein tyrosine kinase inhibitor activity](https://amigo.geneontology.org/amigo/term/GO:0030292), and [miRNA binding](https://amigo.geneontology.org/amigo/term/GO:0035198). It is active in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829) and on the [cytoplasmic side of the plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009898), and is part of the [phosphatidylinositol 3-kinase complex](https://amigo.geneontology.org/amigo/term/GO:0005942). The gene is involved in various processes, including [protein ubiquitination](https://amigo.geneontology.org/amigo/term/GO:0016567), [intracellular signal transduction](https://amigo.geneontology.org/amigo/term/GO:0035556), [negative regulation of tyrosine phosphorylation of STAT protein](https://amigo.geneontology.org/amigo/term/GO:0042532), [negative regulation of apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0043066), [positive regulation of cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0045597), [negative regulation of receptor signaling pathway via JAK-STAT](https://amigo.geneontology.org/amigo/term/GO:0046426), [negative regulation of insulin receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0046627), [phosphatidylinositol phosphate biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0046854), [negative regulation of inflammatory response](https://amigo.geneontology.org/amigo/term/GO:0050728), [branching involved in labyrinthine layer morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0060670), [placenta blood vessel development](https://amigo.geneontology.org/amigo/term/GO:0060674), [T-helper 17 cell lineage commitment](https://amigo.geneontology.org/amigo/term/GO:0072540), [cellular response to interleukin-17](https://amigo.geneontology.org/amigo/term/GO:0097398), and [cellular response to leukemia inhibitory factor](https://amigo.geneontology.org/amigo/term/GO:1990830). Additionally, it enables [1-phosphatidylinositol-3-kinase regulator activity](https://amigo.geneontology.org/amigo/term/GO:0046935).


The gene length is 9,486 base pairs (24.68% of all genes), the mature length is 1,075 base pairs (25.12% of all genes), and the primary transcript length is 8,243 base pairs (28.59% of all genes).


The gene SOCS3, with NCBI ID 9021, has been mentioned in [759 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SOCS3%22) (note: the count in the hyperlink may be different since the count is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication mentioning SOCS3 is 1963, and the middle 50% of publications occurred between 2008 and 2018.


The top five publications mentioning SOCS3, ranked by their scientific influence, include "[Identification of SOCS-3 as a potential mediator of central leptin resistance.](https://pubmed.ncbi.nlm.nih.gov/9660946)" (1998) (relative citation ratio: 17.18), "[Stat3/Socs3 activation by IL-6 transsignaling promotes progression of pancreatic intraepithelial neoplasia and development of pancreatic cancer.](https://pubmed.ncbi.nlm.nih.gov/21481788)" (2011) (relative citation ratio: 15.35), "[IL-6 induces an anti-inflammatory response in the absence of SOCS3 in macrophages.](https://pubmed.ncbi.nlm.nih.gov/12754507)" (2003) (relative citation ratio: 11.21), "[The role of SOCS-3 in leptin signaling and leptin resistance.](https://pubmed.ncbi.nlm.nih.gov/10514492)" (1999) (relative citation ratio: 10.45), and "[Suppressor of cytokine signaling 1 (SOCS-1) and SOCS-3 cause insulin resistance through inhibition of tyrosine phosphorylation of insulin receptor substrate proteins by discrete mechanisms.](https://pubmed.ncbi.nlm.nih.gov/15169905)" (2004) (relative citation ratio: 10.25). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[SOCS3](https://www.proteinatlas.org/ENSG00000184557-SOCS3) is a signal transduction inhibitor with evidence at the protein level. It is detected in all RNA tissue distributions and is primarily located in the plasma membrane and cytosol. SOCS3 is expressed in neutrophils with unknown function (Cluster 21) and is associated with adipose tissue with mixed function (Cluster 7). In the brain, it is found in non-specific vasculature (Cluster 12), and in cell lines, it is linked to lymphoma with an inflammatory response (Cluster 71). Additionally, SOCS3 is expressed in adipocytes and endothelial cells with mixed function (Cluster 41).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [TCF12](https://www.ncbi.nlm.nih.gov/gene/6938) with 6 experiments, [JUN](https://www.ncbi.nlm.nih.gov/gene/3725) with 5 experiments, [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) with 4 experiments, [AR](https://www.ncbi.nlm.nih.gov/gene/367) with 4 experiments, and [ELF1](https://www.ncbi.nlm.nih.gov/gene/1997) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.





The gene is expressed in various tissues, including adipose tissue, fallopian tube, and lung, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the Pons, Superior Cervical Ganglion, Trigeminal Ganglion, Skeletal Muscle, and Ciliary Ganglion.


The proteins are involved in various immune-related pathways, including cytokine signaling in the immune system, interferon signaling (gamma, alpha/beta), regulation of IFNA and IFNG signaling, and antigen processing via ubiquitination, proteasome degradation, and class I MHC presentation. Additionally, they participate in signaling by leptin, interleukin-6, interleukins in general, and growth hormone receptor. These pathways are integral to the adaptive immune system and overall immune system function, as well as broader signal transduction processes.



The protein sequence analyzed has a GRAVY value of -0.426 (43.18th percentile), indicating a hydrophilic nature. It exhibits a charge of 4.02 at pH 7.0 (64.8th percentile) and a median structural flexibility of 1.007 (72.17th percentile). The protein's secondary structure is predicted to be 26.7% helix (10.25th percentile), 33.3% sheet (47.43rd percentile), and 35.6% turn (90.75th percentile). The instability index is 66.9 (92.26th percentile), suggesting high instability. The isoelectric point is 8.97 (78.55th percentile), with a molecular weight of 24770 Da (17.68th percentile) and a length of 225 amino acids (18.27th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |