---
layout: post
title: MT2A
date: 2025-01-17 16:55 CST
description: MT2A description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4502) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4502  | MT2A | ENSG00000125148 | 16q13  |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [zinc ion binding](https://amigo.geneontology.org/amigo/term/GO:0008270), and is involved in [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872). It is active and located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), and is also located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). The gene plays roles in various processes, including [intracellular copper ion homeostasis](https://amigo.geneontology.org/amigo/term/GO:0006878), [intracellular zinc ion homeostasis](https://amigo.geneontology.org/amigo/term/GO:0006882), [detoxification of copper ion](https://amigo.geneontology.org/amigo/term/GO:0010273), [cellular response to interleukin-3](https://amigo.geneontology.org/amigo/term/GO:0036016), [cellular response to erythropoietin](https://amigo.geneontology.org/amigo/term/GO:0036018), [negative regulation of growth](https://amigo.geneontology.org/amigo/term/GO:0045926), [cellular response to cadmium ion](https://amigo.geneontology.org/amigo/term/GO:0071276), and [cellular response to copper ion](https://amigo.geneontology.org/amigo/term/GO:0071280), as well as [cellular response to zinc ion](https://amigo.geneontology.org/amigo/term/GO:0071294).


The gene length is 32,808 base pairs (55.06% of all genes), the mature length is 481 base pairs (1.41% of all genes), and the primary transcript length is 20,017 base pairs (51.69% of all genes).


The gene MT2A (NCBI ID: 4502) has been mentioned in [217 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22MT2A%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1978 and the middle 50% of publications occurring between 1999 and 2015. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning MT2A, ranked by their scientific influence, include "[MT1 and MT2 Melatonin Receptors: A Therapeutic Perspective.](https://pubmed.ncbi.nlm.nih.gov/26514204)" (2016) (relative citation ratio: 16.22), "[Monoclonal antibodies (MT1, MT2, MB1, MB2, MB3) reactive with leukocyte subsets in paraffin-embedded tissue sections.](https://pubmed.ncbi.nlm.nih.gov/3296769)" (1987) (relative citation ratio: 11.16), "[Systematic application of high-resolution, phase-sensitive two-dimensional 1H-NMR techniques for the identification of the amino-acid-proton spin systems in proteins. Rabbit metallothionein-2.](https://pubmed.ncbi.nlm.nih.gov/2992961)" (1985) (relative citation ratio: 6.26), "[Anatomical and cellular localization of melatonin MT1 and MT2 receptors in the adult rat brain.](https://pubmed.ncbi.nlm.nih.gov/25726952)" (2015) (relative citation ratio: 5.59), and "[New selective ligands of human cloned melatonin MT1 and MT2 receptors.](https://pubmed.ncbi.nlm.nih.gov/12764576)" (2003) (relative citation ratio: 4.77). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[MT2A](https://www.proteinatlas.org/ENSG00000125148-MT2A) is a gene with evidence at the protein level. Its RNA is detected in all tissues, and it is notably expressed in the liver, where it is involved in metabolism. In the brain, it is expressed in astrocytes with mixed functions. Additionally, MT2A is expressed in proximal tubular cells, where it plays a role in amino acid metabolism. In blood, it is associated with T-regs and cell cycle regulation. However, MT2A is unprognostic for various cancer types, including bladder urothelial carcinoma, breast invasive carcinoma, and others.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) with 6 experiments, [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930) with 5 experiments, [PGR](https://www.ncbi.nlm.nih.gov/gene/5241) with 5 experiments, [STAT1](https://www.ncbi.nlm.nih.gov/gene/6772) with 5 experiments, and [SMAD3](https://www.ncbi.nlm.nih.gov/gene/4088) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.





The gene is expressed in the liver and skin, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in bronchial epithelial cells and the adrenal cortex.


The proteins are involved in several key pathways, including cytokine signaling in the immune system, interferon signaling, interferon gamma signaling, and the broader immune system.



The protein sequence analyzed has a GRAVY value of 0.131 (88.82nd percentile), indicating a relatively hydrophobic nature. At pH 7.0, the protein carries a charge of 3.301 (62.19th percentile). The median structural flexibility is 0.993 (16.64th percentile), with a tendency for amino acids to be in a helix (27.87%, 16.21st percentile), sheet (6.56%, 0.19th percentile), or turn (31.15%, 71.51st percentile) conformation. The instability index is 68.57 (93.68th percentile), suggesting high instability. The isoelectric point is 8.233 (61.72nd percentile), and the protein has a molecular weight of 6042.20 Da (0.13th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |