---
layout: post
title: APOC3
date: 2025-01-17 16:55 CST
description: APOC3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/345) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 345  | APOC3 | ENSG00000110245 | 11q23.3 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [phospholipid binding](https://amigo.geneontology.org/amigo/term/GO:0005543), and is involved in [triglyceride metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006641) and [G protein-coupled receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007186). It is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615), and [early endosome](https://amigo.geneontology.org/amigo/term/GO:0005769). Additionally, it plays a role in the negative regulation of various processes, including [triglyceride catabolic process](https://amigo.geneontology.org/amigo/term/GO:0010897), [very-low-density lipoprotein particle remodeling](https://amigo.geneontology.org/amigo/term/GO:0010903), [very-low-density lipoprotein particle clearance](https://amigo.geneontology.org/amigo/term/GO:0010916), [high-density lipoprotein particle clearance](https://amigo.geneontology.org/amigo/term/GO:0010987), and [low-density lipoprotein particle clearance](https://amigo.geneontology.org/amigo/term/GO:0010989). The gene also enables [cholesterol binding](https://amigo.geneontology.org/amigo/term/GO:0015485) and [enzyme regulator activity](https://amigo.geneontology.org/amigo/term/GO:0030234), and is involved in [cholesterol efflux](https://amigo.geneontology.org/amigo/term/GO:0033344) and [phospholipid efflux](https://amigo.geneontology.org/amigo/term/GO:0033700). It is part of the [very-low-density lipoprotein particle](https://amigo.geneontology.org/amigo/term/GO:0034361), [intermediate-density lipoprotein particle](https://amigo.geneontology.org/amigo/term/GO:0034363), and [spherical high-density lipoprotein particle](https://amigo.geneontology.org/amigo/term/GO:0034366). Furthermore, the gene is involved in [high-density lipoprotein particle remodeling](https://amigo.geneontology.org/amigo/term/GO:0034375), [very-low-density lipoprotein particle assembly](https://amigo.geneontology.org/amigo/term/GO:0034379), and [chylomicron remnant clearance](https://amigo.geneontology.org/amigo/term/GO:0034382). It also plays a role in [lipoprotein metabolic process](https://amigo.geneontology.org/amigo/term/GO:0042157) and [cholesterol homeostasis](https://amigo.geneontology.org/amigo/term/GO:0042632), and is part of the [chylomicron](https://amigo.geneontology.org/amigo/term/GO:0042627). The gene is involved in [reverse cholesterol transport](https://amigo.geneontology.org/amigo/term/GO:0043691) and the negative regulation of [fatty acid biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0045717), [lipid metabolic process](https://amigo.geneontology.org/amigo/term/GO:0045833), [receptor-mediated endocytosis](https://amigo.geneontology.org/amigo/term/GO:0048261), [lipid catabolic process](https://amigo.geneontology.org/amigo/term/GO:0050995), and [lipoprotein lipase activity](https://amigo.geneontology.org/amigo/term/GO:0051005). It enables [lipase inhibitor activity](https://amigo.geneontology.org/amigo/term/GO:0055102) and [high-density lipoprotein particle receptor binding](https://amigo.geneontology.org/amigo/term/GO:0070653), and is involved in [triglyceride homeostasis](https://amigo.geneontology.org/amigo/term/GO:0070328). The gene is located in the [collagen-containing extracellular matrix](https://amigo.geneontology.org/amigo/term/GO:0060621) and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062).


The gene length is 117,250 base pairs (84.83rd percentile), the mature length is 5,498 base pairs (92.39th percentile), and the primary transcript length is 110,980 base pairs (89.92nd percentile).


The gene APOC3 (NCBI ID: 345) has been mentioned in [559 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22APOC3%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning APOC3 is 1974, and the middle 50% of publications occurred between 2001 and 2016.


The top five publications mentioning APOC3, ranked by their scientific influence, include "[Loss-of-function mutations in APOC3, triglycerides, and coronary disease.](https://pubmed.ncbi.nlm.nih.gov/24941081)" (2014) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 24.47, "[Loss-of-function mutations in APOC3 and risk of ischemic vascular disease.](https://pubmed.ncbi.nlm.nih.gov/24941082)" (2014) with an RCR of 21.16, "[Antisense Inhibition of Apolipoprotein C-III in Patients with Hypertriglyceridemia.](https://pubmed.ncbi.nlm.nih.gov/26222559)" (2015) with an RCR of 13.29, "[Hypertriglyceridemia as a result of human apo CIII gene expression in transgenic mice.](https://pubmed.ncbi.nlm.nih.gov/2167514)" (1990) with an RCR of 11.87, and "[Apolipoprotein C-III reduction in subjects with moderate hypertriglyceridaemia and at high cardiovascular risk.](https://pubmed.ncbi.nlm.nih.gov/35025993)" (2022) with an RCR of 10.99. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[APOC3](https://www.proteinatlas.org/ENSG00000110245-APOC3) is a gene involved in cancer-related processes and is an FDA-approved drug target. It has evidence at the protein level and is detected in many tissues, with notable expression in the liver, specifically in hepatocytes involved in metabolism. The gene is also expressed in liver cancer cell lines. Subcellularly, APOC3 is localized to the nucleoplasm, nucleoli, and cell junctions. While it is unprognostic for several cancer types, including breast invasive carcinoma, kidney renal clear cell carcinoma, liver hepatocellular carcinoma, pancreatic adenocarcinoma, rectum adenocarcinoma, stomach adenocarcinoma, and testicular germ cell tumor, its role in cancer biology remains significant.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [GTF2F1](https://www.ncbi.nlm.nih.gov/gene/2962), [RARA](https://www.ncbi.nlm.nih.gov/gene/5914), [HNF4A](https://www.ncbi.nlm.nih.gov/gene/3172), and [RELA](https://www.ncbi.nlm.nih.gov/gene/5970), each with 3 experiments. Additionally, [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) was found to be regulating in 2 experiments.



The input data indicates associations with various disease conditions, including diseases of metabolism, acquired metabolic diseases, carbohydrate metabolism diseases, glucose metabolism diseases, diabetes mellitus, type 2 diabetes mellitus, nutrition diseases, overnutrition, obesity, cardiovascular system diseases, inherited metabolic disorders, and lipid metabolism disorders.


APOC3 has been associated with various traits and diseases through genome-wide association studies. Notably, it has been linked to [Apolipoprotein A1 levels](https://pubmed.ncbi.nlm.nih.gov/33462484) and [Apolipoprotein B levels](https://pubmed.ncbi.nlm.nih.gov/34226706), both mapped to [Apolipoproteins](https://meshb.nlm.nih.gov/record/ui?ui=D001053) using the BioWordVec method. Additionally, associations include [Creatinine levels](https://pubmed.ncbi.nlm.nih.gov/34226706), mapped to [Creatinine](https://meshb.nlm.nih.gov/record/ui?ui=D003404), and [Cystatin C levels](https://pubmed.ncbi.nlm.nih.gov/34226706), mapped to [Cystatins](https://meshb.nlm.nih.gov/record/ui?ui=D015891). Other notable associations include [Diglycerides (36:3) levels](https://pubmed.ncbi.nlm.nih.gov/35393526), mapped to [Diglycerides](https://meshb.nlm.nih.gov/record/ui?ui=D004075), and [Direct low density lipoprotein cholesterol levels](https://pubmed.ncbi.nlm.nih.gov/34226706), mapped to [Lipoproteins](https://meshb.nlm.nih.gov/record/ui?ui=D008074). Furthermore, APOC3 is associated with [HDL cholesterol](https://pubmed.ncbi.nlm.nih.gov/28270201), mapped to [Cholesterol, HDL](https://meshb.nlm.nih.gov/record/ui?ui=D008076), and various other lipid traits and cellular indices. These findings highlight the multifaceted role of APOC3 in lipid metabolism and cellular processes.


The gene is expressed in multiple tissue samples, including the liver, blood, and small intestine, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the liver, prostate, small intestine, fetal liver, skeletal muscle, uterus corpus, and fetal lung.


The input data highlights several pathways associated with visual transduction, including visual phototransduction and retinoid metabolism and transport. Additionally, it covers various lipid-related processes such as chylomicron-mediated lipid transport, lipid digestion, mobilization, and transport, lipoprotein metabolism, and HDL-mediated lipid transport. The data also encompasses broader categories like metabolism, specifically metabolism of lipids and lipoproteins, as well as disease and signal transduction.



The protein sequence analyzed has a GRAVY value of -0.086 (79.11th percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is -2.40 (41.65th percentile), and the median structural flexibility is 1.004 (58.7th percentile). The protein has a high affinity for helix structures (40.4%, 94.04th percentile), with lower affinities for sheet (34.3%, 54.44th percentile) and turn (25.3%, 23.35th percentile) structures. The instability index is 29.21 (3.4th percentile), suggesting a stable protein. The isoelectric point is 5.23 (12.11th percentile), with a molecular weight of 10852.18 Da (2.44th percentile). The sequence length is 99 amino acids (2.65th percentile). For more details on the sequence analysis, refer to the [BioPython ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |