---
layout: post
title: GSTP1
date: 2024-12-08 23:57 CST
description: GSTP1 description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2950) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2950  | GSTP1 | ENSG00000084207 | 11q13.2 |



The gene length is 6,315 base pairs (17.47% of all genes), the mature length is 1,691.5 base pairs, and the primary transcript length is 5,797 base pairs (21.22% of all genes).


The gene GSTP1, with NCBI ID 2950, has been the subject of [951 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22GSTP1%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1983, and the middle 50% of publications occurred between 2006 and 2016.


The top five publications mentioning GSTP1, ranked by their scientific influence, are as follows: "[Human glutathione S-transferase P1 polymorphisms: relationship to lung tissue enzyme activity and population frequency distribution.](https://pubmed.ncbi.nlm.nih.gov/9498276)" (1998) (relative citation ratio: 14.39), "[Single GST-P positive liver cells--putative initiated hepatocytes.](https://pubmed.ncbi.nlm.nih.gov/3815743)" (1987) (relative citation ratio: 6.35), "[Association between glutathione S-transferase P1, T1, and M1 genetic polymorphism and survival of patients with metastatic colorectal cancer.](https://pubmed.ncbi.nlm.nih.gov/12072547)" (2002) (relative citation ratio: 6.29), "[Inactivation of glutathione S-transferase P1 gene by promoter hypermethylation in human neoplasia.](https://pubmed.ncbi.nlm.nih.gov/9788592)" (1998) (relative citation ratio: 5.99), and "[The human glutathione S-transferases: studies on the tissue distribution and genetic variation of the GST1, GST2 and GST3 isozymes.](https://pubmed.ncbi.nlm.nih.gov/6712152)" (1984) (relative citation ratio: 5.21). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [MAX](https://www.ncbi.nlm.nih.gov/gene/4149), [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313), [ARID2](https://www.ncbi.nlm.nih.gov/gene/196528), [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), and [YY1](https://www.ncbi.nlm.nih.gov/gene/7528), each shown to be regulating in 5 experiments.


The top transcription factors (TFs) identified from the GTRD's database of TF binding sites, filtered for Perturb-seq data, include [TCF7L2](https://www.ncbi.nlm.nih.gov/gene/4149) with 5 experiments, [ZNF384](https://www.ncbi.nlm.nih.gov/gene/57634) with 2 experiments, and [FOXA1](https://www.ncbi.nlm.nih.gov/gene/5929) with 2 experiments. Additionally, [ZNF740](https://www.ncbi.nlm.nih.gov/gene/7023), [ZNF787](https://www.ncbi.nlm.nih.gov/gene/7090), [ZNF786](https://www.ncbi.nlm.nih.gov/gene/7541), and [ZNF771](https://www.ncbi.nlm.nih.gov/gene/9968) each showed regulation in 1 experiment.


The GSTP1 gene has been associated with variations in [Glutathione S-transferase P levels](https://pubmed.ncbi.nlm.nih.gov/29875488) in studies, with the closest MeSH match being [Glutathione](https://meshb.nlm.nih.gov/record/ui?ui=D005978). Additionally, GSTP1 has been linked to [Glutathione S-transferase P level in Chronic kidney disease with hypertension and no diabetes](https://pubmed.ncbi.nlm.nih.gov/35870639). These associations highlight the gene's role in glutathione metabolism and its potential implications in chronic kidney disease.





The protein sequence analyzed has a GRAVY value of -0.1214 (76.5th percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is -3.35 (38.44th percentile), and the median structural flexibility is 0.993 (15.27th percentile). The protein has a propensity for helix, sheet, and turn structures of 34.29% (70.08th percentile), 39.52% (84.82nd percentile), and 28.57% (52.36th percentile), respectively. The instability index is 32.60 (6.51st percentile), with an isoelectric point of 5.43 (16.6th percentile). The sequence length is 210 amino acids (16th percentile), and the molecular weight is 23355.55 Da (15.81st percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |