---
layout: post
title: SCD
date: 2024-12-08 23:57 CST
description: SCD description
tags: [cooccuring-genes]
categories: longest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6319) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6319  | SCD | ENSG00000099194 | 10q24.31 |



The gene length is 448,961 base pairs (97.94th percentile of all genes), the mature length is 1,681 base pairs, and the primary transcript length is 448,109 base pairs.


The gene SCD (NCBI ID: 6319) has been the subject of [392 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SCD%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1969, and the middle 50% of publications occurred between 2010 and 2019.


The top 5 publications mentioning SCD, ranked by their scientific influence, include "[FBW7-NRA41-SCD1 axis synchronously regulates apoptosis and ferroptosis in pancreatic cancer cells.](https://pubmed.ncbi.nlm.nih.gov/33271455)" (2021) (relative citation ratio: 11.93), "[HCAR1/MCT1 Regulates Tumor Ferroptosis through the Lactate-Mediated AMPK-SCD1 Activity and Its Therapeutic Implications.](https://pubmed.ncbi.nlm.nih.gov/33296645)" (2020) (relative citation ratio: 9.75), "[Berberine attenuates nonalcoholic hepatic steatosis through the AMPK-SREBP-1c-SCD1 pathway.](https://pubmed.ncbi.nlm.nih.gov/31226399)" (2019) (relative citation ratio: 8.3), "[Lipidomic Analysis of Î±-Synuclein Neurotoxicity Identifies Stearoyl CoA Desaturase as a Target for Parkinson Treatment.](https://pubmed.ncbi.nlm.nih.gov/30527540)" (2019) (relative citation ratio: 8.29), and "[Tumor resistance to ferroptosis driven by Stearoyl-CoA Desaturase-1 (SCD1) in cancer cells and Fatty Acid Biding Protein-4 (FABP4) in tumor microenvironment promote tumor recurrence.](https://pubmed.ncbi.nlm.nih.gov/34030117)" (2021) (relative citation ratio: 8.26). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [AFF4](https://www.ncbi.nlm.nih.gov/gene/27125), [BRD9](https://www.ncbi.nlm.nih.gov/gene/65980), and [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908), each with 5 experiments. Additionally, [OLIG2](https://www.ncbi.nlm.nih.gov/gene/10215) and [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227) were found to be regulating in 4 experiments each.











The analyzed protein sequence has a GRAVY value of -0.191 (70.18th percentile), indicating a relatively hydrophilic nature. The charge at pH 7 is 7.882 (76.6th percentile), and the median structural flexibility is 0.987 (8.87th percentile). The protein has a notable affinity for sheet structures (0.423, 91.36th percentile), with moderate affinities for helix (0.292, 25.33rd percentile) and turn (0.265, 33.36th percentile) structures. The instability index is 44.836 (38.58th percentile), and the isoelectric point is 9.066 (80.98th percentile). The protein is 359 amino acids long (39.48th percentile) with a molecular weight of 41522.15 Da (41.48th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |