---
layout: post
title: POLE
date: 2024-12-08 23:57 CST
description: POLE description
tags: [cooccuring-genes]
categories: shortest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5426) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5426  | POLE | ENSG00000177084 | 12q24.33 |



The gene length is 503 base pairs (0.5% of all genes), with a mature length of 503 base pairs and a primary transcript length of 503 base pairs.


The gene POLE, with NCBI ID 5426, has been featured in [7 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22POLE%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1977, and the middle 50% of publications span from 1977 to 2004.


The top 5 publications mentioning POLE, ranked by their scientific influence, include "[A novel POLE mutation associated with cancers of colon, pancreas, ovaries and small intestine.](https://pubmed.ncbi.nlm.nih.gov/25860647)" (2015) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 1.79, "[Molecular cloning of the cDNA for the catalytic subunit of human DNA polymerase epsilon.](https://pubmed.ncbi.nlm.nih.gov/8486689)" (1993) with an RCR of 1.21, and "[A patient with polymerase E1 deficiency (POLE1): clinical features and overlap with DNA breakage/instability syndromes.](https://pubmed.ncbi.nlm.nih.gov/25948378)" (2015) with an RCR of 0.73. The RCR values measure the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [YY1](https://www.ncbi.nlm.nih.gov/gene/7528) with 6 experiments, [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025) with 6 experiments, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 6 experiments, [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599) with 5 experiments, and [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.











The analyzed protein sequence has a GRAVY value of -0.351 (52.07th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -32.19 (5.53rd percentile), and the median structural flexibility is 0.999 (38.9th percentile). The protein has a propensity for helix and sheet structures, with affinities of 0.335 (63.4th percentile) and 0.355 (62.57th percentile), respectively, and a turn affinity of 0.254 (24.52nd percentile). The instability index is 54.26 (70.06th percentile), and the isoelectric point is 5.98 (30.16th percentile). The protein is 2286 amino acids long (98.28th percentile) with a molecular weight of 261514.87 Da (98.4th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |