---
layout: post
title: LTK
date: 2024-12-08 23:57 CST
description: LTK description
tags: [cooccuring-genes]
categories: longest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4058) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4058  | LTK | ENSG00000062524 | 15q15.1 |



The gene length is 1,743,270 base pairs (99.94% of all genes), the mature length is 5,618 base pairs (92.88% of all genes), and the primary transcript length is 610,079 base pairs (99.54% of all genes).


The gene LTK, with NCBI ID 4058, has been the subject of [23 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22LTK%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1990, and the middle 50% of publications occurred between 1993 and 2010.


The top five publications mentioning LTK, ranked by their scientific influence, are as follows. The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. The most influential publication is "[Signaling from the small GTP-binding proteins Rac1 and Cdc42 to the c-Jun N-terminal kinase/stress-activated protein kinase pathway. A role for mixed lineage kinase 3/protein-tyrosine kinase 1, a novel member of the mixed lineage kinase family.](https://pubmed.ncbi.nlm.nih.gov/8910292)" (1996) (relative citation ratio: 5.57). Another notable study is "[Hyperphosphorylation of a novel 80 kDa protein-tyrosine kinase similar to Ltk in a human Ki-1 lymphoma cell line, AMS3.](https://pubmed.ncbi.nlm.nih.gov/8183550)" (1994) (relative citation ratio: 4.07). Additionally, "[The CLIP1-LTK fusion is an oncogenic driver in non-small-cell lung cancer.](https://pubmed.ncbi.nlm.nih.gov/34819663)" (2021) (relative citation ratio: 2.52) and "[Correction of the class H defect in glycosylphosphatidylinositol anchor biosynthesis in Ltk- cells by a human cDNA clone.](https://pubmed.ncbi.nlm.nih.gov/8407896)" (1993) (relative citation ratio: 1.69) have also made significant contributions. Lastly, "[LTK fusions: A new target emerges in non-small cell lung cancer.](https://pubmed.ncbi.nlm.nih.gov/35016028)" (2022) (relative citation ratio: 1.03) is another important study in the field. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[Kinase, Receptor, Transferase, Tyrosine-protein kinase](https://www.proteinatlas.org/[Ensembl]-[Gene]) is implicated in systemic lupus erythematosus, with evidence at the protein level. This kinase is detected in many tissues and is localized to vesicles. It is expressed in various clusters, including Cluster 8 in plasmacytoid DCs, Cluster 25 in the small intestine, Cluster 53 in neurons, Cluster 35 in RS4;11 cell lines, and Cluster 5 in enterocytes. The gene's expression is associated with absorption functions in the small intestine and enterocytes.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [TP63](https://www.ncbi.nlm.nih.gov/gene/8626) and [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), each with 4 experiments. Additionally, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), [KLF4](https://www.ncbi.nlm.nih.gov/gene/9314), and [SUPT5H](https://www.ncbi.nlm.nih.gov/gene/6829) were each found to be regulating in 3 experiments.





The genome-wide association study has identified a link between the trait of [Spelling](https://pubmed.ncbi.nlm.nih.gov/35998220) and the closest MeSH term [Comprehension](https://meshb.nlm.nih.gov/record/ui?ui=D032882), mapped using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method.





The analyzed protein sequence has a GRAVY value of -0.185 (70.74th percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is -8.12 (26.12th percentile), and the median structural flexibility is 1.002 (49.83rd percentile). The protein has a predicted affinity for helix, sheet, and turn structures of 30.21% (33.85th percentile), 30.44% (30.68th percentile), and 35.65% (90.98th percentile), respectively. The instability index is 60.75 (84.41st percentile), suggesting a relatively stable protein. The isoelectric point is 6.07 (32.33rd percentile), with a molecular weight of 91680.26 Da (81.42nd percentile) and a length of 864 amino acids (83.22nd percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |