---
layout: post
title: TLN1
date: 2025-01-17 16:55 CST
description: TLN1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7094) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7094  | TLN1 | ENSG00000137076 | 9p13.3 |



The gene is located in the [ruffle](https://amigo.geneontology.org/amigo/term/GO:0001726), [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0005856), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [adherens junction](https://amigo.geneontology.org/amigo/term/GO:0005912), [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925), [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986), [ruffle membrane](https://amigo.geneontology.org/amigo/term/GO:0032587), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). It enables [phosphatidylserine binding](https://amigo.geneontology.org/amigo/term/GO:0001786), [integrin binding](https://amigo.geneontology.org/amigo/term/GO:0005178), [structural constituent of cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0005200), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [vinculin binding](https://amigo.geneontology.org/amigo/term/GO:0017166), [LIM domain binding](https://amigo.geneontology.org/amigo/term/GO:0030274), [phosphatidylinositol binding](https://amigo.geneontology.org/amigo/term/GO:0035091), [cadherin binding](https://amigo.geneontology.org/amigo/term/GO:0045296), and [actin filament binding](https://amigo.geneontology.org/amigo/term/GO:0051015). The gene is involved in [cell-cell junction assembly](https://amigo.geneontology.org/amigo/term/GO:0007043), [cell-substrate junction assembly](https://amigo.geneontology.org/amigo/term/GO:0007044), [integrin-mediated signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007229), [cortical actin cytoskeleton organization](https://amigo.geneontology.org/amigo/term/GO:0030866), [integrin activation](https://amigo.geneontology.org/amigo/term/GO:0033622), [regulation of focal adhesion assembly](https://amigo.geneontology.org/amigo/term/GO:0051893), [platelet aggregation](https://amigo.geneontology.org/amigo/term/GO:0070527), and [cell-cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0098609).


The gene length is 62,886 base pairs (71.97% of all genes), the mature length is 2,128 base pairs (56.01% of all genes), and the primary transcript length is 50,801.5 base pairs (75.67% of all genes).


The gene TLN1 (NCBI ID: 7094) has been mentioned in [75 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22TLN1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 2003 and the middle 50% of publications occurring between 2012 and 2020. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning TLN1, ranked by their scientific influence, include studies that explore the gene's role in various biological processes. The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. The most influential publication is "[Distinct roles for talin-1 and kindlin-3 in LFA-1 extension and affinity regulation.](https://pubmed.ncbi.nlm.nih.gov/22431571)" (2012) (relative citation ratio: 4.42), followed by "[Loss of talin1 in platelets abrogates integrin activation, platelet aggregation, and thrombus formation in vitro and in vivo.](https://pubmed.ncbi.nlm.nih.gov/18086864)" (2007) (relative citation ratio: 3.86). Other notable studies include "[Talin1 is critical for force-dependent reinforcement of initial integrin-cytoskeleton bonds but not tyrosine kinase activation.](https://pubmed.ncbi.nlm.nih.gov/14581461)" (2003) (relative citation ratio: 3.72), "[Talin1 promotes tumor invasion and metastasis via focal adhesion signaling and anoikis resistance.](https://pubmed.ncbi.nlm.nih.gov/20160039)" (2010) (relative citation ratio: 3.71), and "[Podocyte-associated talin1 is critical for glomerular filtration barrier maintenance.](https://pubmed.ncbi.nlm.nih.gov/24531545)" (2014) (relative citation ratio: 3.46). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[TLN1](https://www.proteinatlas.org/ENSG00000137076-TLN1) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in the plasma membrane, focal adhesion sites, centriolar satellite, and cytosol. The gene is expressed in various tissue clusters, including smooth muscle tissue involved in ECM organization, macrophages and microglia related to immune response, squamous epithelial cells associated with keratinization, and smooth muscle cells involved in muscle contraction. Notably, TLN1 is unprognostic for multiple cancer types, including Bladder Urothelial Carcinoma, Breast Invasive Carcinoma, and Kidney Renal Clear Cell Carcinoma.


The top transcription factors (TFs) identified in terms of the number of CHIP-seq experiments include [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169), [TFAP2A](https://www.ncbi.nlm.nih.gov/gene/7020), [ZBTB7A](https://www.ncbi.nlm.nih.gov/gene/51341), [TP53](https://www.ncbi.nlm.nih.gov/gene/7157), and [TAF1](https://www.ncbi.nlm.nih.gov/gene/6872), each shown to be regulating in 5 experiments.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [2623](https://www.ncbi.nlm.nih.gov/gene/2623), [8861](https://www.ncbi.nlm.nih.gov/gene/8861), and [9968](https://www.ncbi.nlm.nih.gov/gene/9968). These TFs were shown to regulate gene expression in 1, 2, and 3 Perturb-seq experiments, respectively.


The GWAS data indicates associations with a range of respiratory conditions, including asthma, bronchial disease, obstructive lung disease, lower respiratory tract disease, lung disease, and respiratory system disease. Additionally, the data highlights links to metabolic disorders such as disease of metabolism, acquired metabolic disease, nutrition disease, overnutrition, and obesity. These findings suggest a broad spectrum of health implications related to both respiratory and metabolic functions.


The TLN1 gene has been associated with several hematological traits through genome-wide association studies. Specifically, it has been linked to [Monocyte count](https://pubmed.ncbi.nlm.nih.gov/32888493), which maps closely to the MeSH term [Monocytes](https://meshb.nlm.nih.gov/record/ui?ui=D009000) using the BioWordVec method. Additionally, TLN1 is associated with [Platelet distribution width](https://pubmed.ncbi.nlm.nih.gov/32888494), corresponding to the MeSH term [Blood Platelets](https://meshb.nlm.nih.gov/record/ui?ui=D001792), and [White blood cell count](https://pubmed.ncbi.nlm.nih.gov/34594039), which directly matches the MeSH term [Leukocyte Count](https://meshb.nlm.nih.gov/record/ui?ui=D007958). These associations highlight the role of TLN1 in various aspects of blood cell biology.


The gene is expressed in various tissues, including blood vessel and bladder, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in cardiac myocytes, Burkitt's lymphoma (Raji), retina, CD14+ monocytes, whole blood, smooth muscle, and uterus.


The proteins are involved in various pathways, including metabolism of proteins, axon guidance, and developmental biology. They also participate in semaphorin interactions and signal transduction, with specific roles in SEMA3A-Plexin repulsion signaling by inhibiting integrin adhesion. Additionally, they are implicated in muscle contraction, smooth muscle contraction, and several integrin signaling pathways, such as GRB2:SOS and p130Cas linkage to MAPK signaling for integrins, as well as integrin alphaIIb beta3 signaling. Furthermore, these proteins play roles in platelet aggregation, hemostasis, and platelet activation, signaling, and aggregation, including platelet degranulation and response to elevated platelet cytosolic Ca2+. They are also involved in the unfolded protein response (UPR), with IRE1alpha and XBP1(S) activating chaperones and chaperone genes, respectively.



The protein sequence analyzed has a GRAVY value of -0.24037780401416606 (65th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -31.934779194441262 (5.63rd percentile), and the median structural flexibility is 1.0050416666666666 (64.14th percentile). The protein has a high affinity for helix structures (0.41282959464777647, 95.15th percentile), with lower affinities for sheet (0.29201101928374656, 24.22nd percentile) and turn structures (0.24793388429752067, 20.1st percentile). The instability index is 37.14856749311325 (14.56th percentile), and the isoelectric point is 5.774807548522949 (24.73rd percentile). The protein has a length of 2541 amino acids (98.69th percentile) and a molecular weight of 269764.0274000008 Da (98.54th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |