---
layout: post
title: NME1
date: 2025-01-17 16:55 CST
description: NME1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4830) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4830  | NME1 | ENSG00000239672 | 17q21.33 |



The gene enables [magnesium ion binding](https://amigo.geneontology.org/amigo/term/GO:0000287), [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [DNA nuclease activity](https://amigo.geneontology.org/amigo/term/GO:0004536), [nucleoside diphosphate kinase activity](https://amigo.geneontology.org/amigo/term/GO:0004550), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524), [GTP binding](https://amigo.geneontology.org/amigo/term/GO:0005525), and [3'-5' exonuclease activity](https://amigo.geneontology.org/amigo/term/GO:0008408). It is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [ruffle membrane](https://amigo.geneontology.org/amigo/term/GO:0032587), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). The gene is involved in various processes including [GTP biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006183), [UTP biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006228), [CTP biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006241), [DNA metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006259), [endocytosis](https://amigo.geneontology.org/amigo/term/GO:0006897), [nervous system development](https://amigo.geneontology.org/amigo/term/GO:0007399), [lactation](https://amigo.geneontology.org/amigo/term/GO:0007595), [negative regulation of cell population proliferation](https://amigo.geneontology.org/amigo/term/GO:0008285), [phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0016310), [cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0030154), [regulation of apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0042981), [positive regulation of DNA binding](https://amigo.geneontology.org/amigo/term/GO:0043388), and [positive regulation of epithelial cell proliferation](https://amigo.geneontology.org/amigo/term/GO:0050679).


The gene length is 94,244 base pairs (80.72% of all genes), the mature length is 4,374.5 base pairs (86.83% of all genes), and the primary transcript length is 71,094 base pairs (82.76% of all genes).


The gene NME1 (NCBI ID: 4830) has been mentioned in [887 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22NME1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1980 and the middle 50% of publications occurring between 1997 and 2009. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning NME1, ranked by their scientific influence, include "[Reduced tumor incidence, metastatic potential, and cytokine responsiveness of nm23-transfected melanoma cells.](https://pubmed.ncbi.nlm.nih.gov/2013093)" (1991) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 12.87, "[Association of low nm23 RNA levels in human primary infiltrating ductal breast carcinomas with lymph node involvement and other histopathological indicators of high metastatic potential.](https://pubmed.ncbi.nlm.nih.gov/2475243)" (1989) with an RCR of 11.3, "[Identification of a second human nm23 gene, nm23-H2.](https://pubmed.ncbi.nlm.nih.gov/1988104)" (1991) with an RCR of 9.45, "[Expression of the antimetastatic gene nm23 in human breast cancer: an association with good prognosis.](https://pubmed.ncbi.nlm.nih.gov/1994057)" (1991) with an RCR of 8.9, and "[Transfection of human nm23-H1 into the human MDA-MB-435 breast carcinoma cell line: effects on tumor metastatic potential, colonization and enzymatic activity.](https://pubmed.ncbi.nlm.nih.gov/8395676)" (1993) with an RCR of 6.25. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[NME1](https://www.proteinatlas.org/ENSG00000239672-NME1) is a kinase and transferase involved in cancer-related processes, with evidence supporting its function at the protein level. The gene is detected in all RNA tissue distributions and is primarily localized in the cytosol. Its expression is notable in various clusters, including Cluster 13 for non-specific mitochondrial function in blood, Cluster 17 for neuronal and synaptic function in tissues, Cluster 14 for neurons and synaptic function in the brain, Cluster 27 for breast cancer in cell lines, and Cluster 66 for photoreceptor cells involved in visual perception in single cells.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861), [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170), [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), [YY1](https://www.ncbi.nlm.nih.gov/gene/7528), and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), each shown to be regulating in 5 experiments.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [2623](https://www.ncbi.nlm.nih.gov/gene/2623), which was shown to be regulating genes in 2 experiments, followed by [8861](https://www.ncbi.nlm.nih.gov/gene/8861) and [9968](https://www.ncbi.nlm.nih.gov/gene/9968), each regulating genes in 1 experiment.


The GWAS data indicates associations with several disease conditions, including cardiovascular system disease, general disease, disease of anatomical entity, and heart disease.



The gene is expressed in various tissues and cell types, including the pineal gland during both day and night, B lymphoblasts (721_B), chronic myelogenous leukemia cells (K-562), promyelocytic leukemia cells (HL-60), and Burkitt's lymphoma cells (Daudi).


The input data highlights key pathways involved in metabolism, specifically focusing on the synthesis and interconversion of nucleotide di- and triphosphates, as well as the broader metabolism of nucleotides.



The analyzed protein sequence has a GRAVY value of -0.194 (69.82nd percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is -3.14 (39.12nd percentile), and the median structural flexibility is 0.997 (29.13rd percentile). The protein has a propensity for helix, sheet, and turn structures of 30.92% (40.25th percentile), 36.18% (67.57th percentile), and 26.97% (37.71st percentile), respectively. The instability index is 30.85 (4.77th percentile), and the isoelectric point is 5.83 (26.13rd percentile). The protein is 152 amino acids long (8.27th percentile) with a molecular weight of 17148.53 Da (8.46th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |