---
layout: post
title: HLA-B
date: 2025-01-17 16:55 CST
description: HLA-B description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3106) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3106  | HLA-B | ENSG00000234745 | 6p21.33 |



The gene is located in the [Golgi membrane](https://amigo.geneontology.org/amigo/term/GO:0000139), [endoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0005783), [Golgi apparatus](https://amigo.geneontology.org/amigo/term/GO:0005794), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986), [ER to Golgi transport vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0012507), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [secretory granule membrane](https://amigo.geneontology.org/amigo/term/GO:0030667), [phagocytic vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030670), [early endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0031901), [recycling endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0055038), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). It is involved in the [positive regulation of T cell mediated cytotoxicity](https://amigo.geneontology.org/amigo/term/GO:0001916), [adaptive immune response](https://amigo.geneontology.org/amigo/term/GO:0002250), [antigen processing and presentation of endogenous peptide antigen via MHC class Ib](https://amigo.geneontology.org/amigo/term/GO:0002476), [antigen processing and presentation of endogenous peptide antigen via MHC class I via ER pathway, TAP-independent](https://amigo.geneontology.org/amigo/term/GO:0002486), [regulation of T cell anergy](https://amigo.geneontology.org/amigo/term/GO:0002667), [defense response](https://amigo.geneontology.org/amigo/term/GO:0006952), [immune response](https://amigo.geneontology.org/amigo/term/GO:0006955), [detection of bacterium](https://amigo.geneontology.org/amigo/term/GO:0016045), [regulation of interleukin-12 production](https://amigo.geneontology.org/amigo/term/GO:0032655), [regulation of interleukin-6 production](https://amigo.geneontology.org/amigo/term/GO:0032675), [protection from natural killer cell mediated cytotoxicity](https://amigo.geneontology.org/amigo/term/GO:0042270), [innate immune response](https://amigo.geneontology.org/amigo/term/GO:0045087), and [regulation of dendritic cell differentiation](https://amigo.geneontology.org/amigo/term/GO:2001198). The gene enables [signaling receptor binding](https://amigo.geneontology.org/amigo/term/GO:0005102), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [peptide antigen binding](https://amigo.geneontology.org/amigo/term/GO:0042605), and [protein-folding chaperone binding](https://amigo.geneontology.org/amigo/term/GO:0051087). It is active in the [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615) and the [external side of plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009897). Additionally, it is part of the [MHC class I protein complex](https://amigo.geneontology.org/amigo/term/GO:0042612) and does not enable [TAP binding](https://amigo.geneontology.org/amigo/term/GO:0046977).


The gene length is 5,341 base pairs (14.95th percentile), the mature length is 2,833.5 base pairs, and the primary transcript length is 5,271.5 base pairs (19.54th percentile).


The gene HLA-B (NCBI ID: 3106) has been mentioned in [3,184 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22HLA-B%22) (note: the count in the hyperlink may be different since the count is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication mentioning this gene is 1975, and the middle 50% of publications occurred between 1995 and 2015.


The top five publications mentioning HLA-B, ranked by their scientific influence, include "[HLA-B*5701 screening for hypersensitivity to abacavir.](https://pubmed.ncbi.nlm.nih.gov/18256392)" (2008) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 30.86, "[HLA-B*5801 allele as a genetic marker for severe cutaneous adverse reactions caused by allopurinol.](https://pubmed.ncbi.nlm.nih.gov/15743917)" (2005) with an RCR of 22.18, "[The germfree state prevents development of gut and joint inflammatory disease in HLA-B27 transgenic rats.](https://pubmed.ncbi.nlm.nih.gov/7964509)" (1994) with an RCR of 20.24, "[Prevalence of spondylarthropathies in HLA-B27 positive and negative blood donors.](https://pubmed.ncbi.nlm.nih.gov/9433870)" (1998) with an RCR of 20.14, and "[Association between presence of HLA-B*5701, HLA-DR7, and HLA-DQ3 and hypersensitivity to HIV-1 reverse-transcriptase inhibitor abacavir.](https://pubmed.ncbi.nlm.nih.gov/11888582)" (2002) with an RCR of 20.11. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[HLA-B](https://www.proteinatlas.org/ENSG00000234745-HLA-B) is a cancer-related gene with evidence at the protein level. It is detected in many tissues and is associated with various immune response clusters, including Cluster 7 for non-specific innate immune response in blood, Cluster 47 for spleen immune response, Cluster 43 for hindbrain mixed function, Cluster 55 for lymphoma immune response in cell lines, and Cluster 14 for Langerhans cells innate immune response in single cells.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [EP300](https://www.ncbi.nlm.nih.gov/gene/2033) with 6 experiments, [PGR](https://www.ncbi.nlm.nih.gov/gene/5241) and [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861) each with 5 experiments, and [CDK8](https://www.ncbi.nlm.nih.gov/gene/1024) and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) each with 4 experiments.



The GWAS data indicates associations with a wide range of disease conditions, including various types of cancer such as reproductive organ cancer, organ system cancer, and specifically female reproductive organ cancer and cervical cancer. Infectious diseases, particularly those caused by viral agents like human immunodeficiency virus, are also implicated. Metabolic disorders, including diabetes mellitus, type 1 diabetes mellitus, and acquired metabolic diseases related to carbohydrate and glucose metabolism, are notable. Cardiovascular diseases, such as coronary artery disease, artery disease, and vascular disease, are prominent, along with heart disease and congenital heart disease. Additionally, the data highlights immune system diseases, including rheumatoid arthritis, Behcet's disease, and vasculitis, as well as kidney disease, urinary system disease, and various musculoskeletal conditions like bone inflammation disease and connective tissue disease. Endocrine system diseases, specifically thyroid gland disease and hypothyroidism, are also associated with the GWAS findings.


Genome-wide association studies have linked HLA-B to various diseases and traits, including [A body shape index](https://pubmed.ncbi.nlm.nih.gov/34021172) (closest MeSH match: [Index](https://meshb.nlm.nih.gov/record/ui?ui=D020481)), [Angiopoietin-1 receptor, soluble levels](https://pubmed.ncbi.nlm.nih.gov/34648354) (closest MeSH match: [Angiopoietin-1](https://meshb.nlm.nih.gov/record/ui?ui=D042683)), and [Anti-thyroid drug induced agranulocytosis](https://pubmed.ncbi.nlm.nih.gov/27157822) (closest MeSH match: [Agranulocytosis](https://meshb.nlm.nih.gov/record/ui?ui=D000380)). Additionally, associations were found with [Autism spectrum disorder or schizophrenia](https://pubmed.ncbi.nlm.nih.gov/28540026) (closest MeSH match: [Autistic Disorder](https://meshb.nlm.nih.gov/record/ui?ui=D001321)), [Basal cell carcinoma](https://pubmed.ncbi.nlm.nih.gov/27539887) (closest MeSH match: [Carcinoma, Basal Cell](https://meshb.nlm.nih.gov/record/ui?ui=D002280)), and [Basophil count](https://pubmed.ncbi.nlm.nih.gov/34469753) (closest MeSH match: [Basophils](https://meshb.nlm.nih.gov/record/ui?ui=D001491)). Other notable traits include various cholesterol and lipid levels, such as [Cholesterol levels in small HDL](https://pubmed.ncbi.nlm.nih.gov/35213538) (closest MeSH match: [Cholesterol](https://meshb.nlm.nih.gov/record/ui?ui=D002784)) and [Concentration of HDL particles](https://pubmed.ncbi.nlm.nih.gov/35213538) (closest MeSH match: [Lipoproteins](https://meshb.nlm.nih.gov/record/ui?ui=D008074)). These associations were mapped using a method called [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0).


The gene is expressed in multiple tissue samples, including lung, blood, and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in various immune cell types such as CD33+ Myeloid cells, BDCA4+ Dendritic cells, CD56+ NK cells, and CD14+ Monocytes, as well as in Whole Blood and Lung.


The input data highlights various pathways involved in the immune system, including antigen presentation and processing, cytokine signaling, interferon signaling, and interactions between lymphoid and non-lymphoid cells. Specific pathways mentioned include the folding, assembly, and peptide loading of class I MHC, interferon gamma and alpha/beta signaling, ER-phagosome pathway, and endosomal/vacuolar pathway. Additionally, the data covers broader categories such as the adaptive and innate immune systems, as well as DAP12 interactions.



The analyzed protein sequence has a GRAVY value of -0.5599 (28.86th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -8.70 (24.94th percentile). The median structural flexibility is 1.0026 (53.6th percentile), with a predicted secondary structure composition of 29.83% helix (30.51st percentile), 32.87% sheet (44.59th percentile), and 27.07% turn (38.65th percentile). The instability index is 43.56 (34.16th percentile), and the isoelectric point is 5.57 (19.96th percentile). The protein has a length of 362 amino acids (40th percentile) and a molecular weight of 40459.60 Da (40.02nd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |