---
layout: post
title: PEX1
date: 2025-01-17 16:55 CST
description: PEX1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5189) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5189  | PEX1 | ENSG00000127980 | 7q21.2 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524), and is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [peroxisome](https://amigo.geneontology.org/amigo/term/GO:0005777), and [peroxisomal membrane](https://amigo.geneontology.org/amigo/term/GO:0005778). It is active in the [peroxisomal membrane](https://amigo.geneontology.org/amigo/term/GO:0005778) and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). The gene is involved in [protein targeting to peroxisome](https://amigo.geneontology.org/amigo/term/GO:0006625), [peroxisome organization](https://amigo.geneontology.org/amigo/term/GO:0007031), [protein import into peroxisome matrix](https://amigo.geneontology.org/amigo/term/GO:0016558), and [protein import into peroxisome matrix, receptor recycling](https://amigo.geneontology.org/amigo/term/GO:0016562). Additionally, it enables [ATP hydrolysis activity](https://amigo.geneontology.org/amigo/term/GO:0016887) and [protein-containing complex binding](https://amigo.geneontology.org/amigo/term/GO:0044877), and is involved in [protein unfolding](https://amigo.geneontology.org/amigo/term/GO:0043335). The gene acts upstream of or within [microtubule-based peroxisome localization](https://amigo.geneontology.org/amigo/term/GO:0060152) and is located in the [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). Furthermore, it enables [ubiquitin-dependent protein binding](https://amigo.geneontology.org/amigo/term/GO:0140036) and [protein transporter activity](https://amigo.geneontology.org/amigo/term/GO:0140318).


The gene length is 8,150 base pairs (21.66% of all genes), the mature length is 1,557 base pairs, and the primary transcript length is 3,836 base pairs.


The gene PEX1 (NCBI ID: 5189) has been mentioned in [51 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PEX1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1995 and the middle 50% of publications occurring between 2002 and 2017. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning PEX1, ranked by their scientific influence, include "[Mutations in PEX1 are the most common cause of peroxisome biogenesis disorders.](https://pubmed.ncbi.nlm.nih.gov/9398847)" (1997) (relative citation ratio: 3.54), "[Human PEX1 is mutated in complementation group 1 of the peroxisome biogenesis disorders.](https://pubmed.ncbi.nlm.nih.gov/9398848)" (1997) (relative citation ratio: 2.42), "[Unique double-ring structure of the peroxisomal Pex1/Pex6 ATPase complex revealed by cryo-electron microscopy.](https://pubmed.ncbi.nlm.nih.gov/26170309)" (2015) (relative citation ratio: 2.06), "[Disruption of a PEX1-PEX6 interaction is the most common cause of the neurologic disorders Zellweger syndrome, neonatal adrenoleukodystrophy, and infantile Refsum disease.](https://pubmed.ncbi.nlm.nih.gov/9671729)" (1998) (relative citation ratio: 1.95), and "[Human PEX1 cloned by functional complementation on a CHO cell mutant is responsible for peroxisome-deficient Zellweger syndrome of complementation group I.](https://pubmed.ncbi.nlm.nih.gov/9539740)" (1998) (relative citation ratio: 1.92). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PEX1](https://www.proteinatlas.org/ENSG00000127980-PEX1) is a gene encoding a protein with hydrolase molecular function, primarily located in peroxisomes. It is involved in several diseases, including Amelogenesis imperfecta, Deafness, Peroxisome biogenesis disorder, and Zellweger syndrome. The gene's expression is detected in all RNA tissue distributions and is notably expressed in the kidney and intestine for tissue-specific expression, and in oligodendrocytes within the brain. Evidence for this gene's function is supported at the protein level.


The top transcription factors (TFs) identified from the GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), [STAT3](https://www.ncbi.nlm.nih.gov/gene/6774), [SIN3A](https://www.ncbi.nlm.nih.gov/gene/25942), [MYOD1](https://www.ncbi.nlm.nih.gov/gene/4654), and [SOX4](https://www.ncbi.nlm.nih.gov/gene/6659), each shown to be regulating in 5 experiments.



The input data indicates associations with various disease conditions, including reproductive organ cancer, cancer, disease of cellular proliferation, organ system cancer, disease, disease of mental health, substance-related disorder, substance dependence, alcohol dependence, uterine cancer, endometrial cancer, and female reproductive organ cancer.



The gene is expressed in the thyroid and pituitary according to BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles indicate expression in CD4+ T cells, CD8+ T cells, the superior cervical ganglion, trigeminal ganglion, skeletal muscle, atrioventricular node, and ovary.



The mouse ortholog gene 71382 and its human ortholog gene 5189 are associated with several phenotypes, including abnormal liver morphology, decreased heart rate, and dilated heart left ventricle, all of which are observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.277 (60.54th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -19.49 (11.47th percentile), and the median structural flexibility is 1.001 (46.76th percentile). The protein has a propensity for helix, sheet, and turn structures of 32.42% (54.4th percentile), 35.15% (60.3th percentile), and 29.54% (60.37th percentile), respectively. The instability index is 47.70 (49.04th percentile), with an isoelectric point of 5.91 (28.35th percentile). The sequence length is 1283 amino acids (92.95th percentile), and the molecular weight is approximately 142865.31 Da (93.03rd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |