---
layout: post
title: SELL
date: 2025-01-17 16:55 CST
description: SELL description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6402) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6402  | SELL | ENSG00000188404 | 1q24.2 |



The gene enables [protease binding](https://amigo.geneontology.org/amigo/term/GO:0002020), [calcium ion binding](https://amigo.geneontology.org/amigo/term/GO:0005509), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [heparin binding](https://amigo.geneontology.org/amigo/term/GO:0008201), [carbohydrate binding](https://amigo.geneontology.org/amigo/term/GO:0030246), [sialic acid binding](https://amigo.geneontology.org/amigo/term/GO:0033691), [glycosphingolipid binding](https://amigo.geneontology.org/amigo/term/GO:0043208), and [oligosaccharide binding](https://amigo.geneontology.org/amigo/term/GO:0070492). It is active in the [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615) and the [external side of the plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009897), and is located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886) and [secretory granule membrane](https://amigo.geneontology.org/amigo/term/GO:0030667). The gene is involved in [cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0007155), [heterophilic cell-cell adhesion via plasma membrane cell adhesion molecules](https://amigo.geneontology.org/amigo/term/GO:0007157), [leukocyte cell-cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0007159), [calcium-dependent cell-cell adhesion via plasma membrane cell adhesion molecules](https://amigo.geneontology.org/amigo/term/GO:0016339), [response to cytokine](https://amigo.geneontology.org/amigo/term/GO:0034097), and [leukocyte tethering or rolling](https://amigo.geneontology.org/amigo/term/GO:0050901).


The gene length is 55,370 base pairs (69.1% of all genes), the mature length is 2,754.5 base pairs, and the primary transcript length is 35,928 base pairs.


The gene SELL (NCBI ID: 6402) has been mentioned in [871 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SELL%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning SELL is 1965, and the middle 50% of publications occurred between 1996 and 2008.


The top 5 publications mentioning SELL, ranked by their scientific influence, include "[Two-step model of leukocyte-endothelial cell interaction in inflammation: distinct roles for LECAM-1 and the leukocyte beta 2 integrins in vivo.](https://pubmed.ncbi.nlm.nih.gov/1715568)" (1991) (relative citation ratio: 23.52), "[An endothelial ligand for L-selectin is a novel mucin-like molecule.](https://pubmed.ncbi.nlm.nih.gov/1376638)" (1992) (relative citation ratio: 14.35), "[The neutrophil selectin LECAM-1 presents carbohydrate ligands to the vascular selectins ELAM-1 and GMP-140.](https://pubmed.ncbi.nlm.nih.gov/1716182)" (1991) (relative citation ratio: 13.68), "[L-selectin: A Major Regulator of Leukocyte Adhesion, Migration and Signaling.](https://pubmed.ncbi.nlm.nih.gov/31139190)" (2019) (relative citation ratio: 12.05), and "[Ligands for L-selectin: homing, inflammation, and beyond.](https://pubmed.ncbi.nlm.nih.gov/15032576)" (2004) (relative citation ratio: 10.56). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[SELL](https://www.proteinatlas.org/ENSG00000188404-SELL) is a cancer-related gene with evidence at the protein level. It is detected in many tissues and is primarily localized in the cytosol. SELL is associated with neutrophils in blood expression, lymphoid tissue in tissue expression, oligodendrocytes in brain expression, and lymphoma in cell line expression. In single-cell expression, it is linked to neutrophils involved in degranulation.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046) and [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), each with 5 experiments. Additionally, [PGR](https://www.ncbi.nlm.nih.gov/gene/5241), [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599), and [CREB1](https://www.ncbi.nlm.nih.gov/gene/1385) were each found to be regulating in 4 experiments.



The GWAS data indicates associations with various disease conditions, including metabolic disorders such as diabetes mellitus and type 2 diabetes mellitus, as well as neurological conditions like neurodegenerative diseases, cognitive disorders, and specific mental health issues such as bipolar disorder and mood disorders. Additionally, the data highlights substance-related disorders, including alcohol dependence, and motor neuron diseases like amyotrophic lateral sclerosis.



The gene is expressed in multiple tissue samples, including lung, blood, and spleen. Additionally, it is expressed in various immune cell types such as CD33+ myeloid cells, BDCA4+ dendritic cells, CD4+ T cells, CD8+ T cells, CD19+ B cells (neg. sel.), and CD14+ monocytes, as well as in whole blood.


The proteins are involved in several key pathways, including the adaptive immune system, immune system, immunoregulatory interactions between a lymphoid and a non-lymphoid cell, cell surface interactions at the vascular wall, and hemostasis.



The analyzed protein sequence has a GRAVY value of -0.3704 (49.76th percentile), indicating a hydrophilic nature. The charge at pH 7 is -3.10 (39.33rd percentile), and the median structural flexibility is 0.998 (31.11th percentile). The protein has a length of 372 amino acids (41.39th percentile) and a molecular weight of 42186.69 Da (42.36th percentile). The secondary structure affinities are 26.88% for helix (11.16th percentile), 33.87% for sheet (51.04th percentile), and 30.38% for turn (66.55th percentile). The instability index is 43.59 (34.25th percentile), and the isoelectric point is 6.20 (35.51st percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |