---
layout: post
title: ELOB
date: 2024-12-08 23:57 CST
description: ELOB description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6923) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6923  | ELOB | ENSG00000103363 | 16p13.3 |



The gene length is 50,261.0 base pairs (66.48% of all genes), the mature length is 1,444.0 base pairs, and the primary transcript length is 47,104.0 base pairs.


ELOB has been studied in 23 publications, with the earliest publication in 1974 and the middle 50% of publications occurring between 1997 and 2015. For more information, see the [23 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ELOB%22) on PubMed (note that the count may differ since it is derived from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)).


The top 5 publications mentioning ELOB, ranked by their scientific influence, include "[Structure of the VHL-ElonginC-ElonginB complex: implications for VHL tumor suppressor function.](https://pubmed.ncbi.nlm.nih.gov/10205047)" (1999) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 11.19, "[Binding of the von Hippel-Lindau tumor suppressor protein to Elongin B and C.](https://pubmed.ncbi.nlm.nih.gov/7660130)" (1995) with an RCR of 10.27, "[Ubiquitination of APOBEC3G by an HIV-1 Vif-Cullin5-Elongin B-Elongin C complex is essential for Vif function.](https://pubmed.ncbi.nlm.nih.gov/15781449)" (2005) with an RCR of 2.44, "[The pVHL-associated SCF ubiquitin ligase complex: molecular genetic analysis of elongin B and C, Rbx1 and HIF-1alpha in renal cell carcinoma.](https://pubmed.ncbi.nlm.nih.gov/11526493)" (2001) with an RCR of 2.39, and "[Structure of the SOCS4-ElonginB/C complex reveals a distinct SOCS box interface and the molecular basis for SOCS-dependent EGFR degradation.](https://pubmed.ncbi.nlm.nih.gov/17997974)" (2007) with an RCR of 1.86. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 6 experiments, [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599) with 5 experiments, [SPI1](https://www.ncbi.nlm.nih.gov/gene/6688) with 5 experiments, [TAF1](https://www.ncbi.nlm.nih.gov/gene/6872) with 5 experiments, and [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.


The top transcription factors (TFs) identified in terms of the number of Perturb-seq experiments are [6829](https://www.ncbi.nlm.nih.gov/gene/6829) with 3 experiments and [11091](https://www.ncbi.nlm.nih.gov/gene/11091) with 2 experiments. These TFs were found to regulate gene expression in multiple experiments, highlighting their significant role in transcriptional regulation.








The analyzed protein sequence has a GRAVY value of -0.497 (35.2% percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -6.403 (29.81% percentile), and the median structural flexibility is 1.014 (89.56% percentile). The protein has a high instability index of 63.034 (87.86% percentile) and an isoelectric point of 4.730 (3.44% percentile). The sequence length is 118 amino acids (4.48% percentile) with a molecular weight of 13132.67 Da (4.49% percentile). The secondary structure affinities are 32.203% for both helix and sheet (52.24% and 40.74% percentiles, respectively), and 29.661% for turns (61.3% percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |