---
layout: post
title: SCAMP1
date: 2025-01-17 16:55 CST
description: SCAMP1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9522) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9522  | SCAMP1 | ENSG00000085365 | 5q14.1 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [protein domain specific binding](https://amigo.geneontology.org/amigo/term/GO:0019904). It is located in the [trans-Golgi network](https://amigo.geneontology.org/amigo/term/GO:0005802), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [clathrin-coated vesicle](https://amigo.geneontology.org/amigo/term/GO:0030136), [synaptic vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030672), [specific granule membrane](https://amigo.geneontology.org/amigo/term/GO:0035579), [zymogen granule membrane](https://amigo.geneontology.org/amigo/term/GO:0042589), and [recycling endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0055038). It is active in the [trans-Golgi network membrane](https://amigo.geneontology.org/amigo/term/GO:0032588) and [recycling endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0055038). The gene is involved in [exocytosis](https://amigo.geneontology.org/amigo/term/GO:0006887), [post-Golgi vesicle-mediated transport](https://amigo.geneontology.org/amigo/term/GO:0006892), [endocytosis](https://amigo.geneontology.org/amigo/term/GO:0006897), [protein transport](https://amigo.geneontology.org/amigo/term/GO:0015031), and [establishment of localization in cell](https://amigo.geneontology.org/amigo/term/GO:0051649).


The gene length is 31,047 base pairs (53.66% of all genes), the mature length is 1,040 base pairs (23.97% of all genes), and the primary transcript length is 21,812 base pairs (54.02% of all genes).


The gene SCAMP1 (NCBI ID: 9522) has been mentioned in [8 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SCAMP1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 2013 and the middle 50% of publications occurring between 2018 and 2020. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning SCAMP1 include studies that explore its role in various cancers. The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Notable publications include "[Knockdown of LncRNA SCAMP1 suppressed malignant biological behaviours of glioma cells via modulating miR-499a-5p/LMX1A/NLRC5 pathway.](https://pubmed.ncbi.nlm.nih.gov/31207033)" (2019) (relative citation ratio: 2.11), "[LncRNA SCAMP1 regulates ZEB1/JUN and autophagy to promote pediatric renal cell carcinoma under oxidative stress via miR-429.](https://pubmed.ncbi.nlm.nih.gov/31550675)" (2019) (relative citation ratio: 1.11), and "[MTSS1 and SCAMP1 cooperate to prevent invasion in breast cancer.](https://pubmed.ncbi.nlm.nih.gov/29497041)" (2018) (relative citation ratio: 1.0). Additionally, "[Long Noncoding RNA SCAMP1 Targets miR-137/CXCL12 Axis to Boost Cell Invasion and Angiogenesis in Ovarian Cancer.](https://pubmed.ncbi.nlm.nih.gov/32401536)" (2020) (relative citation ratio: 0.91) and "[Silencing SCAMP1-TV2 Inhibited the Malignant Biological Behaviors of Breast Cancer Cells by Interaction With PUM2 to Facilitate INSM1 mRNA Degradation.](https://pubmed.ncbi.nlm.nih.gov/32670859)" (2020) (relative citation ratio: 0.83) are also significant. The citation count is from [iCite](https://icite.od.nih.gov).


[SCAMP1](https://www.proteinatlas.org/ENSG00000085365-SCAMP1) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in the nucleoplasm, vesicles, and cell junctions. The gene is notably expressed in monocytes and neutrophils, playing a role in the innate immune response. Additionally, it is involved in neuronal signaling in the brain and is particularly active in neurons, contributing to mixed functions. In cell lines, SCAMP1 is associated with neuronal visual perception, and in single cells, it is found in photoreceptor cells, also linked to visual perception.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [TP53](https://www.ncbi.nlm.nih.gov/gene/7157), [AR](https://www.ncbi.nlm.nih.gov/gene/367), [KMT2B](https://www.ncbi.nlm.nih.gov/gene/9757), and [SUPT5H](https://www.ncbi.nlm.nih.gov/gene/6829), each with 5 experiments. Additionally, [SMC1A](https://www.ncbi.nlm.nih.gov/gene/8243) was found to be regulating in 4 experiments.



The input data highlights a diverse range of disease conditions associated with GWAS studies, including various forms of cancer such as leukemia, lymphoma, and immune system cancers. Additionally, respiratory diseases like asthma, bronchial disease, and obstructive lung disease are noted. Metabolic disorders, particularly diabetes mellitus and type 2 diabetes mellitus, are also prominent. Furthermore, the data includes mental health conditions such as cognitive disorders, mood disorders, and substance-related disorders, including mental depression and substance abuse.



The gene is expressed in various neural tissues, including the brain, pituitary, pineal gland (both day and night), amygdala, fetal brain, and hypothalamus. Additionally, expression is noted in the atrioventricular node, indicating a role in cardiac conduction.




The analyzed protein sequence has a GRAVY value of -0.0494 (81.41st percentile), indicating a hydrophilic nature. The charge at pH 7.0 is 0.0721 (50.65th percentile), and the median structural flexibility is 0.9968 (27.48th percentile). The protein has a propensity for helix and sheet structures, with affinities of 0.2870 (21.48th percentile) and 0.3698 (72.85th percentile), respectively. The instability index is 36.61 (13.31st percentile), and the isoelectric point is 7.03 (50.77th percentile). The protein sequence is 338 amino acids long (36.06th percentile) with a molecular weight of 37920 Da (36.43rd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |