---
layout: post
title: SF3B2
date: 2025-01-17 16:55 CST
description: SF3B2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10992) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10992  | SF3B2 | ENSG00000087365 | 11q13.1 |



The gene is involved in [mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0000398) and [mRNA processing](https://amigo.geneontology.org/amigo/term/GO:0006397), and it plays a role in [RNA splicing](https://amigo.geneontology.org/amigo/term/GO:0008380) and [U2-type prespliceosome assembly](https://amigo.geneontology.org/amigo/term/GO:1903241). It enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), and it is part of the [spliceosomal complex](https://amigo.geneontology.org/amigo/term/GO:0005681), [U2-type spliceosomal complex](https://amigo.geneontology.org/amigo/term/GO:0005684), [U2 snRNP](https://amigo.geneontology.org/amigo/term/GO:0005686), [U12-type spliceosomal complex](https://amigo.geneontology.org/amigo/term/GO:0005689), [U2-type precatalytic spliceosome](https://amigo.geneontology.org/amigo/term/GO:0071005), [precatalytic spliceosome](https://amigo.geneontology.org/amigo/term/GO:0071011), and [catalytic step 2 spliceosome](https://amigo.geneontology.org/amigo/term/GO:0071013). Additionally, it is located in the [nuclear speck](https://amigo.geneontology.org/amigo/term/GO:0016607).


The gene length is 3,355 base pairs (9.39% of all genes), the mature length is 1,758.5 base pairs, and the primary transcript length is 1,826 base pairs (5.75% of all genes).


SF3B2 has been mentioned in [12 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SF3B2%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1994, and the middle 50% of publications range from 2004 to 2019.


The top 5 publications mentioning SF3B2, ranked by their scientific influence, include "[Haploinsufficiency of SF3B2 causes craniofacial microsomia](https://pubmed.ncbi.nlm.nih.gov/34344887)" (2021) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 4.73, "[SF3B2-Mediated RNA Splicing Drives Human Prostate Cancer Progression](https://pubmed.ncbi.nlm.nih.gov/31431456)" (2019) with an RCR of 2.06, "[The prespliceosome components SAP 49 and SAP 145 interact in a complex implicated in tethering U2 snRNP to the branch site](https://pubmed.ncbi.nlm.nih.gov/7958871)" (1994) with an RCR of 1.87, "[Herpes simplex virus IE63 (ICP27) protein interacts with spliceosome-associated protein 145 and inhibits splicing prior to the first catalytic step](https://pubmed.ncbi.nlm.nih.gov/11287586)" (2001) with an RCR of 1.5, and "[Anionic Se-Substitution toward High-Performance CuS<sub>1-</sub> <sub>x</sub> Se<sub>x</sub> Nanosheet Cathode for Rechargeable Magnesium Batteries](https://pubmed.ncbi.nlm.nih.gov/31460703)" (2019) with an RCR of 1.02. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[SF3B2](https://www.proteinatlas.org/ENSG00000087365-SF3B2) is a gene with evidence at the protein level, known for its disease variant involvement. It is detected in all RNA tissues and is primarily localized in nuclear speckles. The gene is expressed in eosinophils, involved in protein ubiquitination, and shows specific expression patterns in the pituitary gland related to hormone signaling, white matter for signal transduction, and various cell lines and single cells with mixed functions.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 6 experiments, [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908) with 6 experiments, [BRD9](https://www.ncbi.nlm.nih.gov/gene/65980) with 6 experiments, [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) with 5 experiments, and [RUNX1T1](https://www.ncbi.nlm.nih.gov/gene/862) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.





The gene is expressed in the ovary and pituitary, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in leukemia lymphoblastic (MOLT-4) cells, testis, testis interstitial tissue, and testis seminiferous tubule.


The input data highlights several key biological processes, including gene expression and various stages of mRNA processing. Specifically, it covers the processing of capped intron-containing pre-mRNA, as well as both major and minor pathways of mRNA splicing.



The analyzed protein sequence has a GRAVY value of -0.987 (4.7th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -20.86 (10.58th percentile), and the median structural flexibility is 1.021 (97.38th percentile). The protein has a high affinity for helix structures (38.55%, 90.23rd percentile) and turns (31.28%, 72.41st percentile), with a lower affinity for sheet structures (24.36%, 7.32nd percentile). The instability index is 65.56 (90.86th percentile), suggesting high instability. The isoelectric point is 5.52 (18.79th percentile), and the sequence length is 895 amino acids (84.24th percentile), with a molecular weight of 100226.37 Da (84.34th percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |