---
layout: post
title: SLC16A3
date: 2025-01-17 16:55 CST
description: SLC16A3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9123) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9123  | SLC16A3 | ENSG00000141526 | 17q25.3 |



The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is involved in [monocarboxylic acid transport](https://amigo.geneontology.org/amigo/term/GO:0015718), [protein catabolic process](https://amigo.geneontology.org/amigo/term/GO:0030163), [lactate transmembrane transport](https://amigo.geneontology.org/amigo/term/GO:0035873), and [pyruvate catabolic process](https://amigo.geneontology.org/amigo/term/GO:0042867). It is located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [basolateral plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0016323), [apical plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0016324), [lateral plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0016328), and [nuclear membrane](https://amigo.geneontology.org/amigo/term/GO:0031965). Additionally, it is active in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [basolateral plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0016323), and [synapse](https://amigo.geneontology.org/amigo/term/GO:0045202). The gene also enables [lactate:proton symporter activity](https://amigo.geneontology.org/amigo/term/GO:0015650) and [pyruvate transmembrane transporter activity](https://amigo.geneontology.org/amigo/term/GO:0050833), and is involved in [pyruvate transmembrane transport](https://amigo.geneontology.org/amigo/term/GO:1901475).


The gene length is 1,965 base pairs (5.2% of all genes), the mature length is 757 base pairs (11.82% of all genes), and the primary transcript length is 1,271 base pairs (3.45% of all genes).


The gene SLC16A3 (NCBI ID: 9123) has been mentioned in [130 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SLC16A3%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning this gene was in 1999, and the middle 50% of publications occurred between 2014 and 2019.


The top 5 publications mentioning SLC16A3, ranked by their scientific influence, include "[Monocarboxylate transporter 4 inhibition potentiates hepatocellular carcinoma immunotherapy through enhancing T cell infiltration and immune attack.](https://pubmed.ncbi.nlm.nih.gov/35043976)" (2023) with a relative citation ratio of 10.43, "[CD147 is tightly associated with lactate transporters MCT1 and MCT4 and facilitates their cell surface expression.](https://pubmed.ncbi.nlm.nih.gov/10921872)" (2000) with a relative citation ratio of 9.91, "[CD147 subunit of lactate/H+ symporters MCT1 and hypoxia-inducible MCT4 is critical for energetics and growth of glycolytic tumors.](https://pubmed.ncbi.nlm.nih.gov/21930917)" (2011) with a relative citation ratio of 8.87, "[Dual Inhibition of the Lactate Transporters MCT1 and MCT4 Is Synthetic Lethal with Metformin due to NAD+ Depletion in Cancer Cells.](https://pubmed.ncbi.nlm.nih.gov/30540938)" (2018) with a relative citation ratio of 8.59, and "[Evidence for a stromal-epithelial 'lactate shuttle' in human tumors: MCT4 is a marker of oxidative stress in cancer-associated fibroblasts.](https://pubmed.ncbi.nlm.nih.gov/21558814)" (2011) with a relative citation ratio of 8.47. The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[SLC16A3](https://www.proteinatlas.org/ENSG00000141526-SLC16A3) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is localized to the nuclear membrane and plasma membrane. The gene is expressed in various clusters, including Cluster 21 in neutrophils with unknown function, Cluster 19 in non-specific tissues involved in basic cellular processes, Cluster 20 in brain macrophages and microglia related to immune response, Cluster 15 in kidney cancer cell lines associated with membrane components, and Cluster 4 in extravillous trophoblasts with unknown function.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [RARA](https://www.ncbi.nlm.nih.gov/gene/5914), [MYC](https://www.ncbi.nlm.nih.gov/gene/4609), [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046), [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), and [RELA](https://www.ncbi.nlm.nih.gov/gene/5970), each shown to be regulating in 5 experiments.





The gene is expressed in various tissues, including muscle and blood. Additionally, it is expressed in specific cell types and tissues such as cardiac myocytes, retina, CD33+ myeloid cells, CD14+ monocytes, whole blood, smooth muscle, bronchial epithelial cells, and colorectal adenocarcinoma.


The proteins are involved in various metabolic and transport processes, including pyruvate metabolism, the citric acid (TCA) cycle, and respiratory electron transport. They also participate in SLC-mediated transmembrane transport, particularly of glucose, sugars, bile salts, organic acids, metal ions, and amine compounds. Additionally, these proteins play roles in basigin interactions, cell surface interactions at the vascular wall, and hemostasis.



The analyzed protein sequence has a GRAVY value of 0.662 (97.15th percentile), indicating a high level of hydrophobicity. The charge at pH 7.0 is 2.861 (60.84th percentile), and the median structural flexibility is 0.983 (6.12th percentile). The protein has a notable affinity for sheet structures (0.426, 91.72nd percentile) and a moderate affinity for helix (0.325, 54.85th percentile) and turn (0.288, 54.5th percentile) structures. The instability index is 30.332 (4.28th percentile), suggesting a relatively stable protein. The isoelectric point is 8.227 (61.63rd percentile), with a molecular weight of 49468.804 Da (51.03rd percentile) and a length of 465 amino acids (53.78th percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |