---
layout: post
title: CD164
date: 2024-12-08 23:57 CST
description: CD164 description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8763) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8763  | CD164 | ENSG00000135535 | 6q21  |



The gene length is 5,436 base pairs (15.21% of all genes), the mature length is 3,561 base pairs, and the primary transcript length is 4,537 base pairs (16.9% of all genes).


The gene CD164, with NCBI ID 8763, has been the subject of [31 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CD164%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1998, and the middle 50% of publications occurred between 2005 and 2017.


The top 5 publications mentioning CD164, ranked by their scientific influence, include "[Identification of CD13, CD107a, and CD164 as novel basophil-activation markers and dissection of two response patterns in time kinetics of IgE-dependent upregulation.](https://pubmed.ncbi.nlm.nih.gov/15916720)" (2005) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 2.76, "[CD164 is a host factor for lymphocytic choriomeningitis virus entry.](https://pubmed.ncbi.nlm.nih.gov/35235462)" (2022) with an RCR of 1.8, "[Update on CD164 and LMX1A genes to strengthen their causative role in autosomal dominant hearing loss.](https://pubmed.ncbi.nlm.nih.gov/35254497)" (2022) with an RCR of 1.5, "[MicroRNA-124 suppresses tumor cell proliferation and invasion by targeting CD164 signaling pathway in non-small cell lung cancer.](https://pubmed.ncbi.nlm.nih.gov/27376157)" (2016) with an RCR of 1.19, and "[Functionally defined CD164 epitopes are expressed on CD34(+) cells throughout ontogeny but display distinct distribution patterns in adult hematopoietic and nonhematopoietic tissues.](https://pubmed.ncbi.nlm.nih.gov/10807777)" (2000) with an RCR of 1.09. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313) and [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025), each with 6 experiments. Additionally, [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), and [FOXM1](https://www.ncbi.nlm.nih.gov/gene/2305) were each found to be regulating in 5 experiments.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, filtered for Perturb-seq data, include [STAT1](https://www.ncbi.nlm.nih.gov/gene/6829) (2 experiments), [ZNF274](https://www.ncbi.nlm.nih.gov/gene/8243) (3 experiments), [ZNF76](https://www.ncbi.nlm.nih.gov/gene/8861) (1 experiment), and [ZNF148](https://www.ncbi.nlm.nih.gov/gene/9968) (4 experiments). These TFs were found to regulate gene expression in multiple Perturb-seq experiments, highlighting their significant roles in transcriptional regulation.








The protein sequence analyzed has a GRAVY value of 0.042 (85.81st percentile), indicating a slightly hydrophobic nature. It exhibits a charge of 2.66 at pH 7.0 (60th percentile) and a median structural flexibility of 1.000 (41.63rd percentile). The protein's secondary structure is predicted to be 19.8% helix (1.16th percentile), 43.15% sheet (92.51st percentile), and 31.47% turn (73.67th percentile). The instability index is 36.99 (14.15th percentile), with an isoelectric point of 8.30 (62.91st percentile). The protein has a length of 197 amino acids (14.02nd percentile) and a molecular weight of 20916.49 Da (12.59th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |