---
layout: post
title: TIAM1
date: 2025-01-17 16:55 CST
description: TIAM1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7074) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7074  | TIAM1 | ENSG00000156299 | 21q22.11 |



The gene is involved in various biological processes, including [cardiac muscle hypertrophy](https://amigo.geneontology.org/amigo/term/GO:0003300), [cell-matrix adhesion](https://amigo.geneontology.org/amigo/term/GO:0007160), and [small GTPase-mediated signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007264). It enables several molecular functions such as [guanyl-nucleotide exchange factor activity](https://amigo.geneontology.org/amigo/term/GO:0005085), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [lipid binding](https://amigo.geneontology.org/amigo/term/GO:0008289). The gene is active in and located in multiple cellular components, including the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [microtubule](https://amigo.geneontology.org/amigo/term/GO:0005874), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), and [cell-cell junction](https://amigo.geneontology.org/amigo/term/GO:0005911). Additionally, it plays roles in the regulation of [epithelial to mesenchymal transition](https://amigo.geneontology.org/amigo/term/GO:0010717), [cell migration](https://amigo.geneontology.org/amigo/term/GO:0016477), and [Rac protein signal transduction](https://amigo.geneontology.org/amigo/term/GO:0016601). The gene is also involved in processes related to neuronal function, such as [response to cocaine](https://amigo.geneontology.org/amigo/term/GO:0042220) and [positive regulation of dendritic spine morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0061003), and is located in neuronal structures like the [neuronal cell body](https://amigo.geneontology.org/amigo/term/GO:0043025) and [dendritic spine](https://amigo.geneontology.org/amigo/term/GO:0043197).


The gene length is 10,517 base pairs (26.74% of all genes), the mature length is 606.5 base pairs, and the primary transcript length is 5,365 base pairs.


The gene TIAM1 (NCBI ID: 7074) has been mentioned in [197 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22TIAM1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning TIAM1 is 1995, and the middle 50% of publications occurred between 2007 and 2016.


The top 5 publications mentioning TIAM1, ranked by their scientific influence, include "[A role for Rac in Tiam1-induced membrane ruffling and invasion.](https://pubmed.ncbi.nlm.nih.gov/7753201)" (1995) (relative citation ratio: 9.21), "[Par-3 controls tight junction assembly through the Rac exchange factor Tiam1.](https://pubmed.ncbi.nlm.nih.gov/15723052)" (2005) (relative citation ratio: 6.32), "[miR-21 and miR-31 converge on TIAM1 to regulate migration and invasion of colon carcinoma cells.](https://pubmed.ncbi.nlm.nih.gov/20826792)" (2010) (relative citation ratio: 5.41), "[The CircRNA-ACAP2/Hsa-miR-21-5p/ Tiam1 Regulatory Feedback Circuit Affects the Proliferation, Migration, and Invasion of Colon Cancer SW480 Cells.](https://pubmed.ncbi.nlm.nih.gov/30212824)" (2018) (relative citation ratio: 5.23), and "[The Rac1-GEF Tiam1 couples the NMDA receptor to the activity-dependent development of dendritic arbors and spines.](https://pubmed.ncbi.nlm.nih.gov/15721239)" (2005) (relative citation ratio: 5.17). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[TIAM1](https://www.proteinatlas.org/ENSG00000156299-TIAM1) is a guanine-nucleotide releasing factor implicated in epilepsy and intellectual disability, with evidence at the protein level. It is detected in many tissues and localizes to the nucleoplasm, nuclear membrane, cell junctions, cytokinetic bridge, and cytosol. TIAM1 is associated with T-regs cell cycle regulation in blood, nervous system development in cerebellum tissue, DNA binding in the brain, ECM organization in skin cancer cell lines, and synaptic function in neurons at the single-cell level.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599), [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597), [MITF](https://www.ncbi.nlm.nih.gov/gene/4286), and [EP300](https://www.ncbi.nlm.nih.gov/gene/2033). Each of these TFs, except EP300, was shown to be regulating in 5 experiments, while EP300 was found in 4 experiments.



The input data highlights a wide range of disease conditions associated with various biological systems and processes. These include several types of cancer, such as leukemia, hematologic cancer, immune system cancer, and organ system cancer. Additionally, the data encompasses respiratory diseases, including lower respiratory tract disease, lung disease, and respiratory system disease. Metabolic disorders are also prominent, with conditions like diabetes mellitus, type 2 diabetes mellitus, and obesity being noted. Cardiovascular and urinary system diseases, such as hypertension, kidney disease, and urinary system disease, are also included. Neurological conditions, including neurodegenerative diseases, cognitive disorders, and mental health diseases like bipolar disorder and major depressive disorder, are part of the dataset. Furthermore, the data covers specific conditions such as acute lymphocytic leukemia, amyotrophic lateral sclerosis, multiple sclerosis, and various immunodeficiency diseases.



The gene is expressed in various brain regions, including the amygdala, thalamus, cingulate cortex, cerebellum, and cerebellum peduncles.


The proteins are involved in various pathways, including signalling by NGF, axon guidance, developmental biology, EPH-Ephrin signaling, EPH-ephrin mediated repulsion of cells, EPHB-mediated forward signaling, cell death signalling via NRAGE, NRIF, and NADE, NRAGE signals death through JNK, p75 NTR receptor-mediated signalling, G alpha (12/13) signalling events, Rho GTPase cycle, signaling by Rho GTPases, GPCR downstream signaling, signaling by GPCR, and signal transduction.


The mouse ortholog gene 21844 and its human ortholog gene 7074 are associated with the phenotype of decreased exploration in a new environment, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.604 (24.93rd percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -18.21 (12.38th percentile), and the median structural flexibility is 1.01 (80.04th percentile). The protein has a high affinity for helix (33.82%, 66.38th percentile) and sheet (30.99%, 33.87th percentile) structures, with a moderate affinity for turns (29.92%, 63.3rd percentile). The instability index is 51.30 (61.43rd percentile), and the isoelectric point is 6.17 (34.8th percentile). The protein is 1591 amino acids long (95.65th percentile) with a molecular weight of 177506.08 Da (95.72nd percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |