---
layout: post
title: RNASE1
date: 2025-01-17 16:55 CST
description: RNASE1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6035) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6035  | RNASE1 | ENSG00000129538 | 14q11.2 |



The gene enables [nucleic acid binding](https://amigo.geneontology.org/amigo/term/GO:0003676), [ribonuclease A activity](https://amigo.geneontology.org/amigo/term/GO:0004522), and [RNA nuclease activity](https://amigo.geneontology.org/amigo/term/GO:0004540). It also enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [lyase activity](https://amigo.geneontology.org/amigo/term/GO:0016829). The gene is involved in [defense response to Gram-positive bacterium](https://amigo.geneontology.org/amigo/term/GO:0050830) but is not involved in [defense response to virus](https://amigo.geneontology.org/amigo/term/GO:0051607). Additionally, the gene is located in the [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062).


The gene length is 28,934 base pairs (51.74% of all genes), the mature length is 4,542 base pairs (87.83% of all genes), and the primary transcript length is 28,575 base pairs (61.4% of all genes).


The gene RNASE1 (NCBI ID: 6035) has been mentioned in [270 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RNASE1%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1965, and the middle 50% of publications occurred between 1997 and 2012.


The top five publications mentioning RNASE1, ranked by their scientific influence, include "[A salt bridge stabilizes the helix formed by isolated C-peptide of RNase A.](https://pubmed.ncbi.nlm.nih.gov/6283528)" (1982) (relative citation ratio: 6.5), "[Acid catalysis of the formation of the slow-folding species of RNase A: evidence that the reaction is proline isomerization.](https://pubmed.ncbi.nlm.nih.gov/283390)" (1978) (relative citation ratio: 6.42), "[Mechanism of the stabilization of ribonuclease A by sorbitol: preferential hydration is greater for the denatured then for the native protein.](https://pubmed.ncbi.nlm.nih.gov/9007993)" (1997) (relative citation ratio: 5.48), "[The structures of RNase A complexed with 3'-CMP and d(CpA): active site conformation and conserved water molecules.](https://pubmed.ncbi.nlm.nih.gov/7756988)" (1994) (relative citation ratio: 4.48), and "[Role of the N terminus in RNase A homologues: differences in catalytic activity, ribonuclease inhibitor interaction and cytotoxicity.](https://pubmed.ncbi.nlm.nih.gov/8632481)" (1996) (relative citation ratio: 4.42). The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RNASE1](https://www.proteinatlas.org/ENSG00000129538-RNASE1) is a gene encoding a protein with molecular functions as an endonuclease, hydrolase, lyase, and nuclease. Evidence supports its expression at the protein level. The RNA transcript is detected in all tissues, with notable expression in the nucleoplasm and vesicles. In terms of expression clusters, RNASE1 is associated with dendritic cells of unknown function in blood, proteolysis in the pancreas, signal transduction in brain white matter, ECM organization in skin cancer cell lines, and innate immune response in macrophages at the single-cell level.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [EP300](https://www.ncbi.nlm.nih.gov/gene/2033) with 4 experiments, [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170) with 3 experiments, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 3 experiments, [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113) with 3 experiments, and [OLIG2](https://www.ncbi.nlm.nih.gov/gene/10215) with 3 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.





The gene is expressed in various tissues, including the pancreas, adipose tissue, and lung, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the retina, pancreas, testis, and specific testicular cell types such as Leydig cells, seminiferous tubules, and germ cells.



The mouse ortholog gene 19752 and its human ortholog gene 6035 are associated with abnormal skin morphology, a phenotype observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.454 (39.92 percentile), indicating a hydrophilic nature. It exhibits a charge of 6.87 at pH 7.0 (73.95 percentile) and a median structural flexibility of 0.999 (35.82 percentile). The protein's secondary structure is predicted to be 25.64% helix (6.53 percentile), 31.41% sheet (36.19 percentile), and 30.77% turn (69.1 percentile). The instability index is 58.43 (79.95 percentile), with an isoelectric point of 9.10 (81.81 percentile). The protein has a length of 156 amino acids (8.81 percentile) and a molecular weight of 17644.08 Da (9.01 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |