---
layout: post
title: ITGA2
date: 2025-01-17 16:55 CST
description: ITGA2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3673) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3673  | ITGA2 | ENSG00000164171 | 5q11.2 |



The gene enables [amyloid-beta binding](https://amigo.geneontology.org/amigo/term/GO:0001540) and [virus receptor activity](https://amigo.geneontology.org/amigo/term/GO:0001618), and is involved in [response to hypoxia](https://amigo.geneontology.org/amigo/term/GO:0001666) and [positive regulation of leukocyte migration](https://amigo.geneontology.org/amigo/term/GO:0002687). It enables [integrin binding](https://amigo.geneontology.org/amigo/term/GO:0005178), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [collagen binding](https://amigo.geneontology.org/amigo/term/GO:0005518). The gene is located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886) and [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925), and is involved in [substrate-dependent cell migration](https://amigo.geneontology.org/amigo/term/GO:0006929) and [hypotonic response](https://amigo.geneontology.org/amigo/term/GO:0006971). It plays a role in [cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0007155) and [cell-matrix adhesion](https://amigo.geneontology.org/amigo/term/GO:0007160), and is involved in the [integrin-mediated signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007229). The gene is also involved in [female pregnancy](https://amigo.geneontology.org/amigo/term/GO:0007565) and [blood coagulation](https://amigo.geneontology.org/amigo/term/GO:0007596), and is part of the [integrin complex](https://amigo.geneontology.org/amigo/term/GO:0008305). It is active in the [external side of the plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009897) and located in the [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986). Additionally, the gene is involved in [positive regulation of epithelial cell migration](https://amigo.geneontology.org/amigo/term/GO:0010634) and [response to amine](https://amigo.geneontology.org/amigo/term/GO:0014075), among other processes.


The gene length is 1,673 base pairs (4.36% of all genes), the mature length is 483.5 base pairs, and the primary transcript length is 1,155 base pairs.


The gene ITGA2 (NCBI ID: 3673) has been mentioned in [241 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ITGA2%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1978 and the middle 50% of publications occurring between 1999 and 2017. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning ITGA2, ranked by their scientific influence, include "[Coexpression of CD49b and LAG-3 identifies human and mouse T regulatory type 1 cells.](https://pubmed.ncbi.nlm.nih.gov/23624599)" (2013) (relative citation ratio: 15.75), "[The membrane glycoprotein Ia-IIa (VLA-2) complex mediates the Mg++-dependent adhesion of platelets to collagen.](https://pubmed.ncbi.nlm.nih.gov/2715183)" (1989) (relative citation ratio: 10.33), "[The human integrin VLA-2 is a collagen receptor on some cells and a collagen/laminin receptor on others.](https://pubmed.ncbi.nlm.nih.gov/2557634)" (1989) (relative citation ratio: 8.85), "[Integrin alpha 2 beta 1 (VLA-2) mediates reorganization and contraction of collagen matrices by human cells.](https://pubmed.ncbi.nlm.nih.gov/1913826)" (1991) (relative citation ratio: 8.8), and "[Characterization of the class III collagen receptor, a phosphorylated, transmembrane glycoprotein expressed in nucleated human cells.](https://pubmed.ncbi.nlm.nih.gov/2831221)" (1988) (relative citation ratio: 8.25). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[ITGA2](https://www.proteinatlas.org/ENSG00000164171-ITGA2) is a gene that functions as a host cell receptor for virus entry, integrin, and receptor. It is involved in cancer-related processes and has evidence at the protein level. The gene is detected in many tissues and is particularly expressed in squamous epithelium with keratinization (Cluster 27), oligodendrocytes with mixed function (Cluster 28), and squamous epithelial cells with keratinization (Cluster 53). Additionally, it is found in non-specific cells with mixed function (Cluster 45).


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [RUNX1T1](https://www.ncbi.nlm.nih.gov/gene/862), [SP1](https://www.ncbi.nlm.nih.gov/gene/6667), [INTS11](https://www.ncbi.nlm.nih.gov/gene/54973), [MAX](https://www.ncbi.nlm.nih.gov/gene/4149), and [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), each shown to be regulating in 5 experiments.



The input data indicates associations with a range of disease conditions, including various types of cancer such as leukemia, lymphoma, and immune system cancer. Additionally, respiratory conditions like asthma, bronchial disease, and obstructive lung disease are noted. The data also encompasses hematopoietic and immune system diseases, as well as cardiovascular system disease. Furthermore, cognitive disorders and diseases of mental health are included in the dataset.


The genome-wide association study has identified a link between ITGA2 and [Platelet distribution width](https://pubmed.ncbi.nlm.nih.gov/32888494), which has been mapped to the closest MeSH term [Blood Platelets](https://meshb.nlm.nih.gov/record/ui?ui=D001792) using the method [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0).


The gene is expressed in various tissues, including the salivary gland and lung, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in cardiac myocytes, the temporal lobe, bronchial epithelial cells, the salivary gland, the adrenal cortex, the appendix, and the tongue.


The proteins are involved in various pathways, including axon guidance, developmental biology, L1CAM interactions, extracellular matrix organization, ECM proteoglycans, integrin cell surface interactions, laminin interactions, non-integrin membrane-ECM interactions, syndecan interactions, CHL1 interactions, platelet adhesion to exposed collagen, and hemostasis.



The analyzed protein sequence has a GRAVY value of -0.0996 (78.11th percentile), indicating a hydrophilic nature. The charge at pH 7 is -30.095 (6.14th percentile), and the median structural flexibility is 0.999 (36.26th percentile). The protein has a propensity for helix, sheet, and turn structures of 26.84% (11th percentile), 38.61% (81.44th percentile), and 31.50% (73.85th percentile), respectively. The instability index is 34.07 (8.65th percentile), with an isoelectric point of 5.10 (9.69th percentile). The protein is 1181 amino acids long (91.33rd percentile) and has a molecular weight of 129294.85 Da (91.02nd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |