---
layout: post
title: DGKE
date: 2024-12-08 23:57 CST
description: DGKE description
tags: [cooccuring-genes]
categories: longest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8526) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8526  | DGKE | ENSG00000153933 | 17q22 |



The gene length is 451,408 base pairs (97.97th percentile of all genes), the mature length is 4,635 base pairs, and the primary transcript length is 449,253 base pairs (99.08th percentile of all genes).


The gene DGKE (NCBI ID: 8526) has been the subject of [41 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22DGKE%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1997, and the middle 50% of publications occurred between 2009 and 2017.


The top five publications mentioning DGKE, ranked by their scientific influence, include "[Recessive mutations in DGKE cause atypical hemolytic-uremic syndrome.](https://pubmed.ncbi.nlm.nih.gov/23542698)" (2013) with a relative citation ratio of 11.18, "[Plant PA signaling via diacylglycerol kinase.](https://pubmed.ncbi.nlm.nih.gov/19394438)" (2009) with a relative citation ratio of 4.35, "[DGKE variants cause a glomerular microangiopathy that mimics membranoproliferative GN.](https://pubmed.ncbi.nlm.nih.gov/23274426)" (2013) with a relative citation ratio of 3.79, "[New Era of Diacylglycerol Kinase, Phosphatidic Acid and Phosphatidic Acid-Binding Protein.](https://pubmed.ncbi.nlm.nih.gov/32947951)" (2020) with a relative citation ratio of 2.94, and "[Loss of DGKÎµ induces endothelial cell activation and death independently of complement activation.](https://pubmed.ncbi.nlm.nih.gov/25498910)" (2015) with a relative citation ratio of 2.28. The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


The gene [Kinase, Transferase](https://www.proteinatlas.org/[Ensembl]-[Gene]) is involved in disease variants associated with Hemolytic uremic syndrome. Evidence for its function is available at the protein level. The gene is detected in all RNA tissue distributions and is localized in the nucleoplasm and cytosol. It is part of Cluster 33 for blood expression, which is non-specific and related to ATP binding. In tissue expression, it belongs to Cluster 28, which is neuronal and related to transcription. The gene is also part of Cluster 5 for brain expression, which is non-specific and has mixed functions. In cell line expression, it is part of Cluster 51, associated with myeloma and immunoglobulins. Additionally, it is part of Cluster 25 for single cell expression, related to bipolar cells and visual perception.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613), [E2F1](https://www.ncbi.nlm.nih.gov/gene/1869), and [JUN](https://www.ncbi.nlm.nih.gov/gene/3725), each with 5 experiments. Additionally, [KDM5B](https://www.ncbi.nlm.nih.gov/gene/10765) and [AR](https://www.ncbi.nlm.nih.gov/gene/367) were found to be regulating in 4 experiments each.








The mouse ortholog gene 56077 and its human ortholog gene 8526 are associated with abnormal sternum morphology, a phenotype observed in both sexes. Additionally, the mouse ortholog gene 56077 and its human ortholog gene 8526 are linked to abnormal testis morphology, although this phenotype is not observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.226 (66.59th percentile), indicating a hydrophilic nature. It exhibits a charge of 4.51 at pH 7.0 (66.35th percentile) and a median structural flexibility of 0.996 (23.88th percentile). The protein's secondary structure is predicted to be 30.51% helix (36.43rd percentile), 35.80% sheet (64.93rd percentile), and 27.34% turn (41.15th percentile). The instability index is 35.15 (10.43rd percentile), with an isoelectric point of 8.02 (58.95th percentile). The protein has a length of 567 amino acids (65.59th percentile) and a molecular weight of 63926.42 Da (66.24th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |