---
layout: post
title: METTL1
date: 2025-01-17 16:55 CST
description: METTL1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4234) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4234  | METTL1 | ENSG00000037897 | 12q14.1 |



The gene enables [tRNA binding](https://amigo.geneontology.org/amigo/term/GO:0000049) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is involved in [tRNA modification](https://amigo.geneontology.org/amigo/term/GO:0006400), [tRNA methylation](https://amigo.geneontology.org/amigo/term/GO:0030488), [RNA (guanine-N7)-methylation](https://amigo.geneontology.org/amigo/term/GO:0036265), and [tRNA stabilization](https://amigo.geneontology.org/amigo/term/GO:0036416). It is part of the [tRNA methyltransferase complex](https://amigo.geneontology.org/amigo/term/GO:0043527) and the [tRNA (m7G46) methyltransferase complex](https://amigo.geneontology.org/amigo/term/GO:0106143). The gene is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [nucleolus](https://amigo.geneontology.org/amigo/term/GO:0005730), and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). Additionally, it enables [tRNA (guanine(46)-N7)-methyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0008176) and [internal mRNA (guanine-N7-)-methyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0160090).


The gene length is 25,902 base pairs (48.62% of all genes), the mature length is 5,589 base pairs (92.77% of all genes), and the primary transcript length is 21,118.5 base pairs (53.11% of all genes).


METTL1 (Gene ID: 4234) has been mentioned in [21 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22METTL1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 2005 and the middle 50% of publications occurring between 2019 and 2022. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning METTL1, ranked by their scientific influence, include "[METTL1-mediated m[7]G modification of Arg-TCT tRNA drives oncogenic transformation.](https://pubmed.ncbi.nlm.nih.gov/34352207)" (2021) (relative citation ratio: 12.25), "[METTL1 Promotes let-7 MicroRNA Processing via m7G Methylation.](https://pubmed.ncbi.nlm.nih.gov/31031083)" (2019) (relative citation ratio: 11.88), "[Aberrant translation regulated by METTL1/WDR4-mediated tRNA N7-methylguanosine modification drives head and neck squamous cell carcinoma progression.](https://pubmed.ncbi.nlm.nih.gov/35179319)" (2022) (relative citation ratio: 11.12), "[METTL1/WDR4-mediated m<sup>7</sup>G tRNA modifications and m<sup>7</sup>G codon usage promote mRNA translation and lung cancer progression.](https://pubmed.ncbi.nlm.nih.gov/34371184)" (2021) (relative citation ratio: 10.35), and "[Mettl1/Wdr4-Mediated m<sup>7</sup>G tRNA Methylome Is Required for Normal mRNA Translation and Embryonic Stem Cell Self-Renewal and Differentiation.](https://pubmed.ncbi.nlm.nih.gov/29983320)" (2018) (relative citation ratio: 9.6). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[METTL1](https://www.proteinatlas.org/ENSG00000037897-METTL1) is a gene encoding a protein with multiple molecular functions, including methyltransferase, RNA-binding, transferase, and tRNA-binding activities. The protein is localized in the nucleoplasm and is detected in all RNA tissue distributions. It is part of various expression clusters, including Cluster 30 for blood, Cluster 68 for tissue, Cluster 21 for brain, Cluster 38 for cell lines, and Cluster 39 for single cells, all of which are non-specific and related to basic cellular processes such as DNA binding, protein processing, metabolism, and transcription. The gene's expression has been found to be unprognostic in several cancer types, including Kidney Renal Clear Cell Carcinoma and Liver Hepatocellular Carcinoma, among others.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113), [PGR](https://www.ncbi.nlm.nih.gov/gene/5241), and [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908), each with 6 experiments. Additionally, [YY1](https://www.ncbi.nlm.nih.gov/gene/7528) and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) were found to be regulating in 5 experiments each.



The input data indicates a hierarchical classification of disease conditions, starting from a general 'disease' category and narrowing down to specific types. These include 'disease of anatomical entity', 'nervous system disease', 'central nervous system disease', 'neurodegenerative disease', 'demyelinating disease', and ultimately 'multiple sclerosis'. This structured data suggests a focus on multiple sclerosis, a specific type of demyelinating neurodegenerative disease affecting the central nervous system.



The gene is expressed in various tissues, including the thyroid, adrenal gland, and kidney, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles show expression in 721_B_lymphoblasts, CD34+ cells, CD105+ endothelial cells, colorectal adenocarcinoma, pituitary gland, and dorsal root ganglion.




The analyzed protein sequence has a GRAVY value of -0.443478260869566 (41.23rd percentile), indicating a hydrophilic nature. The charge at pH 7.0 is 0.47362395545938796 (51.77th percentile), and the median structural flexibility is 0.9974761904761904 (30.18th percentile). The protein has a propensity for helix, sheet, and turn structures of 0.32608695652173914 (56.06th percentile), 0.35144927536231885 (60.23rd percentile), and 0.2644927536231884 (33.23rd percentile), respectively. The instability index is 52.595326086956504 (65.48th percentile), with an isoelectric point of 7.185778617858888 (52.22nd percentile). The protein has a length of 276 amino acids (25.24th percentile) and a molecular weight of 31470.631700000038 Da (26.08th percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |