---
layout: post
title: CNTN1
date: 2025-01-17 16:55 CST
description: CNTN1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1272) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1272  | CNTN1 | ENSG00000018236 | 12q12 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [carbohydrate binding](https://amigo.geneontology.org/amigo/term/GO:0030246), and is active in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886) and [axon](https://amigo.geneontology.org/amigo/term/GO:0030424). It is involved in various biological processes including [cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0007155), [Notch signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007219), [axon guidance](https://amigo.geneontology.org/amigo/term/GO:0007411), [brain development](https://amigo.geneontology.org/amigo/term/GO:0007420), [locomotory behavior](https://amigo.geneontology.org/amigo/term/GO:0007626), [gene expression](https://amigo.geneontology.org/amigo/term/GO:0010467), [positive regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0010628), [positive regulation of sodium ion transport](https://amigo.geneontology.org/amigo/term/GO:0010765), [positive regulation of neuron projection development](https://amigo.geneontology.org/amigo/term/GO:0010976), [cerebellum development](https://amigo.geneontology.org/amigo/term/GO:0021549), and [central nervous system myelin formation](https://amigo.geneontology.org/amigo/term/GO:0032289). Additionally, it is located in the [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [presynaptic membrane](https://amigo.geneontology.org/amigo/term/GO:0042734), [postsynaptic membrane](https://amigo.geneontology.org/amigo/term/GO:0045211), [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062), and [side of membrane](https://amigo.geneontology.org/amigo/term/GO:0098552). The gene also enables [cell-cell adhesion mediator activity](https://amigo.geneontology.org/amigo/term/GO:0098632) and is involved in [cell-cell adhesion](https://amigo.geneontology.org/amigo/term/GO:0098609).


The gene length is 41,087 base pairs (61st percentile of all genes), with a mature length of 7,191 base pairs and a primary transcript length of 41,087 base pairs.


The gene CNTN1 (NCBI ID: 1272) has been mentioned in [63 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CNTN1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1986 and the middle 50% of publications occurring between 2015 and 2021. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning CNTN1, ranked by their scientific influence, include "[Antibodies to contactin-1 in chronic inflammatory demyelinating polyneuropathy.](https://pubmed.ncbi.nlm.nih.gov/23280477)" (2013) (relative citation ratio: 8.22), "[Antibodies to neurofascin, contactin-1, and contactin-associated protein 1 in CIDP: Clinical relevance of IgG isotype.](https://pubmed.ncbi.nlm.nih.gov/31753915)" (2019) (relative citation ratio: 8.06), "[Paranodal dissection in chronic inflammatory demyelinating polyneuropathy with anti-neurofascin-155 and anti-contactin-1 antibodies.](https://pubmed.ncbi.nlm.nih.gov/28073817)" (2017) (relative citation ratio: 6.92), "[Antibodies to the Caspr1/contactin-1 complex in chronic inflammatory demyelinating polyradiculoneuropathy.](https://pubmed.ncbi.nlm.nih.gov/33880507)" (2021) (relative citation ratio: 6.27), and "[Destruction of paranodal architecture in inflammatory neuropathy with anti-contactin-1 autoantibodies.](https://pubmed.ncbi.nlm.nih.gov/25694474)" (2015) (relative citation ratio: 5.26). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CNTN1](https://www.proteinatlas.org/ENSG00000018236-CNTN1) is a gene with evidence at the protein level. Its RNA is detected in many tissues, and it is expressed in various clusters related to neuronal function and signaling, including Cluster 22 in eosinophils, Cluster 64 in neuronal tissues, Cluster 14 in brain neurons, Cluster 16 in thyroid cancer cell lines, and Cluster 34 in single-cell neurons. The gene's expression in different cancer types has been found to be unprognostic, indicating no significant correlation with patient outcomes.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664) with 5 experiments, [CHD7](https://www.ncbi.nlm.nih.gov/gene/55636) with 3 experiments, [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613) with 3 experiments, [PGR](https://www.ncbi.nlm.nih.gov/gene/5241) with 3 experiments, and [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170) with 3 experiments.



The input data from GWAS indicates associations with a wide range of disease conditions, including various types of cancer such as hematologic and immune system cancers, metabolic disorders like type 2 diabetes mellitus and carbohydrate metabolism disease, cardiovascular diseases including coronary artery disease and hypertension, nervous system diseases such as neurodegenerative disorders and cognitive disorders, and mental health conditions like bipolar disorder and anxiety disorder. Additionally, the data highlights associations with kidney disease, urinary system disease, and specific developmental disorders like attention deficit hyperactivity disorder. Other notable conditions include retinal diseases, hypersensitivity reactions, and myocardial infarction.



The gene is expressed in various tissues, including the ovary, bladder, and brain, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the pineal gland (both day and night), fetal brain, occipital lobe, cerebellum, cerebellum peduncles, and pancreatic islet.


The input data highlights various pathways involving NOTCH signaling, including the transmission of signals to the nucleus by activated NOTCH1 and NOTCH2, as well as the roles of NOTCH1 and NOTCH2 in cancer and developmental biology. Additionally, the data covers axon guidance, L1CAM interactions, neurofascin interactions, and broader categories such as disease and signal transduction.



The analyzed protein sequence has a GRAVY value of -0.307 (57.17th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -17.84 (12.76th percentile). The median structural flexibility is 1.001 (44.66th percentile), with a secondary structure composition of 27.70% helix (15.35th percentile), 36.74% sheet (71.33rd percentile), and 31.93% turn (76.35th percentile). The instability index is 35.70 (11.53rd percentile), and the isoelectric point is 5.62 (20.96th percentile). The protein is 1018 amino acids long (87.97th percentile) with a molecular weight of 113319.03 Da (88.01st percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |