---
layout: post
title: SRSF5
date: 2025-01-17 16:55 CST
description: SRSF5 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6430) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6430  | SRSF5 | ENSG00000100650 | 14q24.1 |



The gene is involved in [mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0000398) and [mRNA processing](https://amigo.geneontology.org/amigo/term/GO:0006397), specifically in [mRNA splice site recognition](https://amigo.geneontology.org/amigo/term/GO:0006376). It enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [nucleolus](https://amigo.geneontology.org/amigo/term/GO:0005730), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and is active in the [nuclear speck](https://amigo.geneontology.org/amigo/term/GO:0016607).


The gene length is 151,157 base pairs (89th percentile of all genes), the mature length is 4,529 base pairs, and the primary transcript length is 92,497 base pairs.


SRSF5 (Gene ID: 6430) has been mentioned in [34 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SRSF5%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest mention in 1997. The middle 50% of these publications were published between 2006 and 2017.


The top 5 publications mentioning SRSF5, ranked by their scientific influence, include "[Assessment of cognition using surveys and neuropsychological assessment: the Health and Retirement Study and the Aging, Demographics, and Memory Study.](https://pubmed.ncbi.nlm.nih.gov/21743047)" (2011) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 19.15, "[The Health and Retirement Study Harmonized Cognitive Assessment Protocol Project: Study Design and Methods.](https://pubmed.ncbi.nlm.nih.gov/31563909)" (2020) with an RCR of 8.88, "[Hrs regulates multivesicular body formation via ESCRT recruitment to endosomes.](https://pubmed.ncbi.nlm.nih.gov/12900395)" (2003) with an RCR of 5.85, "[Epstein-Barr Virus LMP1 Promotes Syntenin-1- and Hrs-Induced Extracellular Vesicle Formation for Its Own Secretion To Increase Cell Proliferation and Migration.](https://pubmed.ncbi.nlm.nih.gov/32546618)" (2020) with an RCR of 2.75, and "[Sequence-specific RNA binding by an SR protein requires RS domain phosphorylation: creation of an SRp40-specific splicing enhancer.](https://pubmed.ncbi.nlm.nih.gov/9037021)" (1997) with an RCR of 2.66. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[SRSF5](https://www.proteinatlas.org/ENSG00000100650-SRSF5) is an RNA-binding protein with evidence at the protein level. It is detected in all tissues and is primarily localized in the nucleoplasm, nucleoli, and cytosol. The gene is part of various non-specific expression clusters, including Cluster 33 for blood, Cluster 40 for tissue, Cluster 30 for cell lines, and Cluster 55 for single cells, all related to general cellular processes such as ATP binding, transcription, and gene expression regulation. Additionally, it is specifically expressed in the choroid plexus of the brain, associated with mitochondrial functions (Cluster 42).


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), [SUPT5H](https://www.ncbi.nlm.nih.gov/gene/6829), [AR](https://www.ncbi.nlm.nih.gov/gene/367), [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), and [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046). Each of these TFs was shown to be regulating genes in 5 experiments.





The gene is expressed in various tissues, including the ovary, thyroid, and pituitary. Additionally, it is expressed in several immune cell types such as CD34+ cells, CD33+ myeloid cells, BDCA4+ dendritic cells, CD56+ NK cells, CD4+ T cells, CD8+ T cells, and CD19+ B cells (negatively selected). Expression is also noted in the lymph node.


The input data highlights various pathways involved in gene expression and RNA processing. These include RNA polymerase II transcription, post-elongation processing of the transcript, mRNA 3'-end processing, and mRNA splicing. Additionally, the data covers the transport of mature transcripts to the cytoplasm and the cleavage of growing transcripts in the termination region. These pathways collectively contribute to the regulation and maturation of mRNA, ensuring proper gene expression.



The analyzed protein sequence has a GRAVY value of -1.307352941176472 (1.2 percentile), indicating a hydrophilic nature. It exhibits a charge of 39.78524728431111 at pH 7.0 (98.63 percentile) and a median structural flexibility of 1.0195119047619048 (96.52 percentile). The protein's secondary structure affinities are 22.06% for helix (1.89 percentile), 20.22% for sheet (2.29 percentile), and 37.87% for turn (95.14 percentile). Additionally, it has an instability index of 93.36544117647072 (99.42 percentile), an isoelectric point of 11.585307121276855 (99.69 percentile), a length of 272 amino acids (24.72 percentile), and a molecular weight of 31263.45160000006 Da (25.82 percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |