---
layout: post
title: AARS1
date: 2025-01-17 16:55 CST
description: AARS1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/16) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 16  | AARS1 | ENSG00000090861 | 16q22.1 |



The gene enables [tRNA binding](https://amigo.geneontology.org/amigo/term/GO:0000049) and [aminoacyl-tRNA editing activity](https://amigo.geneontology.org/amigo/term/GO:0002161), and is involved in [Ser-tRNA(Ala) hydrolase activity](https://amigo.geneontology.org/amigo/term/GO:0002196) and [alanine-tRNA ligase activity](https://amigo.geneontology.org/amigo/term/GO:0004813). It also enables [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524) and [zinc ion binding](https://amigo.geneontology.org/amigo/term/GO:0008033), and is active in [amino acid binding](https://amigo.geneontology.org/amigo/term/GO:0016597). The gene is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [mitochondrion](https://amigo.geneontology.org/amigo/term/GO:0005739), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). It is involved in various processes including [tRNA modification](https://amigo.geneontology.org/amigo/term/GO:0006400), [tRNA aminoacylation for protein translation](https://amigo.geneontology.org/amigo/term/GO:0006418), [alanyl-tRNA aminoacylation](https://amigo.geneontology.org/amigo/term/GO:0006419), [tRNA processing](https://amigo.geneontology.org/amigo/term/GO:0008033), [cerebellar Purkinje cell layer development](https://amigo.geneontology.org/amigo/term/GO:0021680), [negative regulation of neuron apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0043524), [neuromuscular process controlling balance](https://amigo.geneontology.org/amigo/term/GO:0050885), [neuron apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0051402), [aminoacyl-tRNA metabolism involved in translational fidelity](https://amigo.geneontology.org/amigo/term/GO:0106074), and [regulation of cytoplasmic translational fidelity](https://amigo.geneontology.org/amigo/term/GO:0140018).


The gene length is 174,633 base pairs (90.89% of all genes), the mature length is 5,578 base pairs (92.71% of all genes), and the primary transcript length is 140,143 base pairs (92.84% of all genes).


The gene AARS1 (NCBI ID: 16) has been mentioned in [45 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22AARS1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1958 and the middle 50% of publications occurring between 2007 and 2019. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning AARS1, ranked by their scientific influence, include "[Autoantibodies against alanyl-tRNA synthetase and tRNAAla coexist and are associated with myositis.](https://pubmed.ncbi.nlm.nih.gov/3701255)" (1986) (relative citation ratio: 5.14), "[Exome sequencing identifies mitochondrial alanyl-tRNA synthetase mutations in infantile mitochondrial cardiomyopathy.](https://pubmed.ncbi.nlm.nih.gov/21549344)" (2011) (relative citation ratio: 4.56), "[A major determinant for binding and aminoacylation of tRNA(Ala) in cytoplasmic Alanyl-tRNA synthetase is mutated in dominant axonal Charcot-Marie-Tooth disease.](https://pubmed.ncbi.nlm.nih.gov/20045102)" (2010) (relative citation ratio: 3.92), "[Synthetic and editing mechanisms of aminoacyl-tRNA synthetases.](https://pubmed.ncbi.nlm.nih.gov/23852030)" (2014) (relative citation ratio: 3.26), and "[Clinical manifestations in patients with antibody to PL-12 antigen (alanyl-tRNA synthetase).](https://pubmed.ncbi.nlm.nih.gov/2178410)" (1990) (relative citation ratio: 3.14). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[AARS1](https://www.proteinatlas.org/ENSG00000090861-AARS1) is a gene with multiple molecular functions, including aminoacyl-tRNA synthetase, ligase, RNA-binding, and tRNA-binding. It is implicated in various diseases such as Charcot-Marie-Tooth disease, epilepsy, neurodegeneration, and neuropathy. Evidence for its function is available at the protein level. The gene is detected in all RNA tissue distributions and is primarily located in the cytosol. Its expression clusters include Cluster 33 for blood (non-specific - ATP binding), Cluster 32 for tissue (endocrine & neural - unknown function), Cluster 45 for brain (non-specific - mixed function), Cluster 13 for cell line (non-specific - mixed function), and Cluster 61 for single cell (plasma cells - immunoglobulins & histones).


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169), [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), [JUN](https://www.ncbi.nlm.nih.gov/gene/3725), and [PPARG](https://www.ncbi.nlm.nih.gov/gene/5468), each with 4 experiments. Additionally, [EP300](https://www.ncbi.nlm.nih.gov/gene/2033) was found to be regulating in 3 experiments.



The GWAS data indicates associations with various disease conditions, including cancer, diseases of cellular proliferation, metabolism, and nutrition. Specifically, the data highlights acquired metabolic diseases, overnutrition, and obesity. Additionally, there are associations with general diseases of anatomical entities, particularly kidney disease and urinary system diseases.


The gene AARS1 has been associated with [Body mass index](https://pubmed.ncbi.nlm.nih.gov/31669095), which maps closely to the MeSH term [Body Mass Index](https://meshb.nlm.nih.gov/record/ui?ui=D015992) using the method [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0).


The gene is expressed in various tissues, including the thyroid and brain. Additionally, it is expressed in specific cell types associated with Lymphoma (Burkitt's, Raji) and Leukemia (Chronic Myelogenous, K-562).


The input data indicates involvement in several biological processes, including gene expression, cytosolic tRNA aminoacylation, and tRNA aminoacylation.


The mouse ortholog gene 234734 and its human ortholog gene 16.0 are associated with the phenotype of preweaning lethality with complete penetrance, observed in both sexes.


The protein sequence analyzed has a GRAVY value of -0.3042 (57.49th percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of -23.97 (8.67th percentile). The median structural flexibility is 1.0040 (59.72nd percentile), with a predicted secondary structure composition of 34.71% helix (72.89th percentile), 34.30% sheet (54.02nd percentile), and 28.00% turn (47.23rd percentile). The instability index is 36.47 (13.0th percentile), and the isoelectric point is 5.34 (14.33rd percentile). The protein has a length of 968 amino acids (86.74th percentile) and a molecular weight of 106809.20 Da (86.46th percentile). For more details on the sequence analysis, refer to the [BioPython ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |