---
layout: post
title: HTT
date: 2025-01-17 16:55 CST
description: HTT description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3064) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3064  | HTT | ENSG00000197386 | 4p16.3 |



The gene is involved in various biological processes, including [establishment of mitotic spindle orientation](https://amigo.geneontology.org/amigo/term/GO:0000132), [apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0006915), and [Golgi organization](https://amigo.geneontology.org/amigo/term/GO:0007030). It enables several molecular functions such as [p53 binding](https://amigo.geneontology.org/amigo/term/GO:0002039), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and [kinase binding](https://amigo.geneontology.org/amigo/term/GO:0019900). The gene is located in multiple cellular components, including the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [early endosome](https://amigo.geneontology.org/amigo/term/GO:0005769), and [Golgi apparatus](https://amigo.geneontology.org/amigo/term/GO:0005794). Additionally, it is active in the [presynaptic cytosol](https://amigo.geneontology.org/amigo/term/GO:0099523) and [postsynaptic cytosol](https://amigo.geneontology.org/amigo/term/GO:0099524), and is part of a [protein-containing complex](https://amigo.geneontology.org/amigo/term/GO:0032991). The gene also plays roles in the regulation of various processes, such as [positive regulation of apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0043065) and [positive regulation of mitophagy](https://amigo.geneontology.org/amigo/term/GO:1901526).


The gene length is 1906.0 base pairs (4.99% of all genes). The mature length is 1906.0 base pairs, and the primary transcript length is 1906.0 base pairs.


The gene HTT (NCBI ID: 3064) has been mentioned in [993 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22HTT%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning HTT is 1993, and the middle 50% of publications occurred between 2008 and 2018.


The top five publications mentioning HTT, ranked by their scientific influence, include "[p62/SQSTM1 forms protein aggregates degraded by autophagy and has a protective effect on huntingtin-induced cell death.](https://pubmed.ncbi.nlm.nih.gov/16286508)" (2005) (relative citation ratio: 50.7), "[Aggregation of huntingtin in neuronal intranuclear inclusions and dystrophic neurites in brain.](https://pubmed.ncbi.nlm.nih.gov/9302293)" (1997) (relative citation ratio: 44.73), "[Inclusion body formation reduces levels of mutant huntingtin and the risk of neuronal death.](https://pubmed.ncbi.nlm.nih.gov/15483602)" (2004) (relative citation ratio: 29.47), "[Huntingtin acts in the nucleus to induce apoptosis but death does not correlate with the formation of intranuclear inclusions.](https://pubmed.ncbi.nlm.nih.gov/9778247)" (1998) (relative citation ratio: 24.38), and "[Targeting Huntingtin Expression in Patients with Huntington's Disease.](https://pubmed.ncbi.nlm.nih.gov/31059641)" (2019) (relative citation ratio: 21.84). The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[HTT](https://www.proteinatlas.org/ENSG00000197386-HTT) is a gene with evidence at the protein level, implicated in various diseases including intellectual disability and neurodegeneration. It is detected in all RNA tissues and is primarily localized in the nucleoplasm and cytosol. The gene is expressed in various clusters, including Cluster 33 for blood expression, Cluster 54 for tissue expression, Cluster 32 for brain expression, Cluster 22 for cell line expression, and Cluster 66 for single cell expression, which is specific to photoreceptor cells involved in visual perception.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [RELA](https://www.ncbi.nlm.nih.gov/gene/5970) with 6 experiments, [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) and [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861) each with 5 experiments, and [ETS1](https://www.ncbi.nlm.nih.gov/gene/2113) and [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908) each with 4 experiments.



The input data from GWAS indicates associations with a wide range of disease conditions, including various types of cancer such as male reproductive organ cancer, prostate cancer, and organ system cancer. Additionally, metabolic disorders are prominently featured, encompassing diabetes mellitus, type 2 diabetes mellitus, and carbohydrate metabolism disease. Cardiovascular conditions, such as coronary artery disease and vascular disease, are also noted. Furthermore, the data includes nervous system diseases, specifically central nervous system disease and brain disease, as well as inherited metabolic disorders like lipid storage disease and lysosomal storage disease.


The HTT gene has been associated with various traits and diseases through genome-wide association studies. Notably, it has been linked to [Apolipoprotein B levels](https://pubmed.ncbi.nlm.nih.gov/33462484), which maps closely to the MeSH term [Apolipoproteins](https://meshb.nlm.nih.gov/record/ui?ui=D001053) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method. Additionally, HTT is associated with [Educational attainment (MTAG)](https://pubmed.ncbi.nlm.nih.gov/30038396) and [Educational attainment (years of education)](https://pubmed.ncbi.nlm.nih.gov/30038396), both mapping to the MeSH term [Education](https://meshb.nlm.nih.gov/record/ui?ui=D004493). Other associations include [Hepatocyte growth factor levels](https://pubmed.ncbi.nlm.nih.gov/27989323), mapping to [Growth](https://meshb.nlm.nih.gov/record/ui?ui=D006128), and [Household income (MTAG)](https://pubmed.ncbi.nlm.nih.gov/31844048), mapping to [Income](https://meshb.nlm.nih.gov/record/ui?ui=D007182). Furthermore, HTT is linked to [Mean platelet volume](https://pubmed.ncbi.nlm.nih.gov/32888494), directly matching the MeSH term [Mean Platelet Volume](https://meshb.nlm.nih.gov/record/ui?ui=D063847), and [Noncognitive aspects of educational attainment](https://pubmed.ncbi.nlm.nih.gov/33414549), mapping to [Education](https://meshb.nlm.nih.gov/record/ui?ui=D004493). Lastly, associations include [Serum sulfate level](https://pubmed.ncbi.nlm.nih.gov/27412988), mapping to [Serum](https://meshb.nlm.nih.gov/record/ui?ui=D044967), and [Triglyceride levels](https://pubmed.ncbi.nlm.nih.gov/33462484), mapping to [Triglycerides](https://meshb.nlm.nih.gov/record/ui?ui=D014280).


The gene is expressed in the pancreas and spleen according to the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles indicate expression in the Prefrontal Cortex, Lymphoma (Burkitt's Raji), 721 B lymphoblasts, CD34+ cells, CD56+ NK cells, CD8+ T cells, CD71+ early erythroid cells, and the Superior Cervical Ganglion.



The mouse ortholog gene 15194 and its human ortholog gene 3064 are associated with phenotypes including embryonic lethality prior to organogenesis, which is not observed in both sexes, and preweaning lethality with complete penetrance, which is observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.066 (80.35th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -56.90 (1.82nd percentile), and the median structural flexibility is 0.996 (24.69th percentile). The protein has a high affinity for helix (0.333, 62nd percentile) and sheet (0.369, 72.11th percentile) structures, with a lower affinity for turns (0.274, 42.05th percentile). The instability index is 56.17 (75.03rd percentile), and the isoelectric point is 5.81 (25.52nd percentile). The protein is relatively long with 3142 amino acids (99.29th percentile) and has a molecular weight of 347599.30 Da (99.29th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |