---
layout: post
title: RCAN3
date: 2025-01-17 16:55 CST
description: RCAN3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/11123) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 11123  | RCAN3 | ENSG00000117602 | 1p36.11 |



The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is active in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737). It is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829) and enables [calcium-dependent protein serine/threonine phosphatase regulator activity](https://amigo.geneontology.org/amigo/term/GO:0008597). The gene is involved in [anatomical structure morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0009653) and [calcium-mediated signaling](https://amigo.geneontology.org/amigo/term/GO:0019722). Additionally, it enables [phosphatase binding](https://amigo.geneontology.org/amigo/term/GO:0019902) and [troponin I binding](https://amigo.geneontology.org/amigo/term/GO:0031013).


The gene length is 18,351 base pairs (40th percentile of all genes), with a mature length of 2,242 base pairs and a primary transcript length of 18,351 base pairs.


The gene RCAN3, with NCBI ID 11123, has been mentioned in [6 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RCAN3%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest publication mentioning this gene was in 2007, and the middle 50% of publications occurred between 2010 and 2018.


The top 5 publications mentioning RCAN3 include "[RCAN3, a novel calcineurin inhibitor that down-regulates NFAT-dependent cytokine gene expression.](https://pubmed.ncbi.nlm.nih.gov/17270291)" (2007) (relative citation ratio: 0.64), "[A novel role for an RCAN3-derived peptide as a tumor suppressor in breast cancer.](https://pubmed.ncbi.nlm.nih.gov/25916653)" (2015) (relative citation ratio: 0.44), "[Complexity of bidirectional transcription and alternative splicing at human RCAN3 locus.](https://pubmed.ncbi.nlm.nih.gov/21961037)" (2011) (relative citation ratio: 0.3), "[Human RCAN3 gene expression and cell growth in endothelial cells.](https://pubmed.ncbi.nlm.nih.gov/21042787)" (2010) (relative citation ratio: 0.16), and "[Functional implications of miR-145/RCAN3 axis in the progression of cervical cancer.](https://pubmed.ncbi.nlm.nih.gov/32345470)" (2020) (relative citation ratio: 0.0). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RCAN3](https://www.proteinatlas.org/ENSG00000117602-RCAN3) is a gene with evidence at the protein level, detected in many RNA tissue distributions. It is primarily localized in the nucleoli fibrillar center and nuclear speckles. The gene is expressed in various clusters, including Cluster 31 in T-cells related to T-cell receptor activity, Cluster 16 in non-specific tissues related to transcription, Cluster 25 in the brain related to transcription, Cluster 10 in ciliated cells related to ion channel activity, and Cluster 78 in NK-cells and T-cells related to the immune system and transcription.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313), [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169), [BCL3](https://www.ncbi.nlm.nih.gov/gene/602), and [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), each shown to be regulating in 5 experiments.



The GWAS data indicates associations with a range of conditions, including cognitive disorder, bipolar disorder, mood disorder, developmental disorder of mental health, specific developmental disorder, communication disorder, language disorder, speech disorder, learning disability, and reading disorder.



The gene is expressed in the salivary gland and prostate according to BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles indicate that the gene is ubiquitously expressed in multiple tissue samples, including cardiac myocytes, heart, liver, CD4+ T cells, prostate, trigeminal ganglion, and atrioventricular node.




The analyzed protein sequence has a GRAVY value of -0.639 (22nd percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -21.12 (10.4th percentile), and the median structural flexibility is 1.01 (80.91st percentile). The protein has a high instability index of 62.33 (86.93rd percentile) and an isoelectric point of 4.54 (1.78th percentile). The sequence length is 241 amino acids (20.37th percentile) with a molecular weight of 27491.45 Da (21.14th percentile). The secondary structure affinities are 36.10% for helix (81.24th percentile), 30.71% for sheet (32.08th percentile), and 27.39% for turn (41.66th percentile). For more details on the sequence analysis, refer to the [BioPython ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |