---
layout: post
title: STYX
date: 2025-01-17 16:55 CST
description: STYX description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6815) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6815  | STYX | ENSG00000198252 | 14q22.1 |



The gene enables [pseudophosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0001691) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is involved in [protein tyrosine/serine/threonine phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0008138) and [F-box domain binding](https://amigo.geneontology.org/amigo/term/GO:1990444). It is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). The gene is active in the nucleoplasm and cytoplasm. Additionally, it is involved in various processes including [dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0016311), [negative regulation of protein binding](https://amigo.geneontology.org/amigo/term/GO:0032091), [MAPK export from nucleus](https://amigo.geneontology.org/amigo/term/GO:0045204), [negative regulation of SCF-dependent proteasomal ubiquitin-dependent catabolic process](https://amigo.geneontology.org/amigo/term/GO:0062026), and [regulation of ERK1 and ERK2 cascade](https://amigo.geneontology.org/amigo/term/GO:0070372).


The gene length is 27,341 base pairs (50.11 percentile of all genes), the mature length is 690 base pairs, and the primary transcript length is 3,760 base pairs.


The gene STYX (NCBI ID: 6815) has been mentioned in [20 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22STYX%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1969, and the middle 50% of publications range from 2002 to 2018.


The top 5 publications mentioning STYX, ranked by their relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows. The most influential publication is "[Gathering STYX: phosphatase-like form predicts functions for unique protein-interaction domains.](https://pubmed.ncbi.nlm.nih.gov/9757831)" (1998) (relative citation ratio: 1.82). Other notable works include "[Discovery of Sclerotinia sclerotiorum Hypovirulence-Associated Virus-1 in Urban River Sediments of Heathcote and Styx Rivers in Christchurch City, New Zealand.](https://pubmed.ncbi.nlm.nih.gov/23929472)" (2013) (relative citation ratio: 1.46), "[The pseudophosphatase STYX targets the F-box of FBXW7 and inhibits SCFFBXW7 function.](https://pubmed.ncbi.nlm.nih.gov/28007894)" (2017) (relative citation ratio: 0.79), "[The pseudophosphatase MK-STYX physically and genetically interacts with the mitochondrial phosphatase PTPMT1.](https://pubmed.ncbi.nlm.nih.gov/24709986)" (2014) (relative citation ratio: 0.69), and "[Cyanthiwigin AC and AD, two novel diterpene skeletons from the Jamaican sponge Myrmekioderma styx.](https://pubmed.ncbi.nlm.nih.gov/14627387)" (2003) (relative citation ratio: 0.66). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[STYX](https://www.proteinatlas.org/ENSG00000198252-STYX) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily located in the nucleoplasm and is associated with various expression clusters, including Cluster 43 for blood (non-specific - transcription & translation), Cluster 83 for tissue (liver - metabolism), Cluster 13 for brain (oligodendrocytes - mixed function), Cluster 30 for cell line (non-specific - gene expression regulation), and Cluster 70 for single cell (spermatocytes & spermatids - spermatid development).


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930), [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313), [TAL1](https://www.ncbi.nlm.nih.gov/gene/6886), [STAT1](https://www.ncbi.nlm.nih.gov/gene/6772), and [PGR](https://www.ncbi.nlm.nih.gov/gene/5241), each shown to be regulating in 4 experiments.



The GWAS data indicates associations with a range of conditions, including cognitive disorder, anxiety disorder, obsessive-compulsive disorder, tic disorder, Gilles de la Tourette syndrome, developmental disorder of mental health, specific developmental disorder, and attention deficit hyperactivity disorder.



The gene is expressed in the liver and adipose tissue according to BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, GTEx Tissue Gene Expression Profiles indicate expression in the Globus Pallidus and Ovary.




The analyzed protein sequence has a GRAVY value of -0.197 (69.43rd percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is -3.97 (36.63rd percentile), and the median structural flexibility is 0.996 (23.59th percentile). The protein has a propensity for helix, sheet, and turn structures of 32.74% (57.1st percentile), 35.87% (65.4th percentile), and 24.22% (16.26th percentile), respectively. The instability index is 49.57 (55.81st percentile), with an isoelectric point of 5.89 (27.66th percentile). The protein is 223 amino acids long (17.96th percentile) and has a molecular weight of 25492.12 Da (18.75th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |