---
layout: post
title: HS3ST4
date: 2024-12-08 23:57 CST
description: HS3ST4 description
tags: [cooccuring-genes]
categories: longest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9951) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9951  | HS3ST4 | ENSG00000182601 | 16p12.1 |



The gene length is 886,904 base pairs (99.47th percentile), the mature length is 1,999.5 base pairs (52.72nd percentile), and the primary transcript length is 839,132 base pairs (99.81st percentile).





The top 5 publications mentioning HS3ST4, ranked by their scientific influence as measured by the [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541), are as follows:

1. [Title of the first publication](https://pubmed.ncbi.nlm.nih.gov/[PubMed ID]) (Year) ...... (Relative Citation Ratio)
2. [Title of the second publication](https://pubmed.ncbi.nlm.nih.gov/[PubMed ID]) (Year) ...... (Relative Citation Ratio)
3. [Title of the third publication](https://pubmed.ncbi.nlm.nih.gov/[PubMed ID]) (Year) ...... (Relative Citation Ratio)
4. [Title of the fourth publication](https://pubmed.ncbi.nlm.nih.gov/[PubMed ID]) (Year) ...... (Relative Citation Ratio)
5. [Title of the fifth publication](https://pubmed.ncbi.nlm.nih.gov/[PubMed ID]) (Year) ...... (Relative Citation Ratio)

The Relative Citation Ratio (RCR) values measure the scientific influence of each paper by field- and time-adjusting the citations it has received, and benchmarking to the median for NIH publications, the value of which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


The gene [None](https://www.proteinatlas.org/None-None) functions as a transferase and has evidence at the protein level. Its RNA is detected in some tissues, with notable expression in the brain, specifically in neuronal signaling clusters. The gene is also expressed in neurons with mixed functions and in bone cancer cell lines with neuronal signaling properties. In terms of cancer prognostics, the gene is unprognostic for Breast Invasive Carcinoma (validation), Ovary Serous Cystadenocarcinoma, Prostate Adenocarcinoma, and Testicular Germ Cell Tumor.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [EZH2](https://www.ncbi.nlm.nih.gov/gene/2146) and [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), each with 4 experiments, followed by [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 4 experiments, [SMARCC1](https://www.ncbi.nlm.nih.gov/gene/6599) with 3 experiments, and [MYC](https://www.ncbi.nlm.nih.gov/gene/4609) with 3 experiments.











The analyzed protein sequence has a GRAVY value of -0.436 (42.12th percentile), indicating a hydrophilic nature. It exhibits a charge of 5.22 at pH 7.0 (68.8th percentile) and a median structural flexibility of 1.007 (70.72nd percentile). The protein's secondary structure is predicted to be 32.24% helix (52.54th percentile), 30.92% sheet (33.44th percentile), and 34.43% turn (87.56th percentile). The instability index is 54.92 (71.95th percentile), with an isoelectric point of 8.82 (74.32nd percentile). The protein has a length of 456 amino acids (52.68th percentile) and a molecular weight of 49798.52 Da (51.47th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |