---
layout: post
title: CDC42EP3
date: 2025-01-17 16:55 CST
description: CDC42EP3 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10602) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10602  | CDC42EP3 | ENSG00000163171 | 2p22.2 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [cytoskeletal regulatory protein binding](https://amigo.geneontology.org/amigo/term/GO:0005519), and is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0005856), and [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). It is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [endomembrane system](https://amigo.geneontology.org/amigo/term/GO:0012505), and [actin cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0015629). The gene is involved in [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), specifically [Rho protein signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007266), and plays a role in the [regulation of cell shape](https://amigo.geneontology.org/amigo/term/GO:0008360). Additionally, it is involved in the [positive regulation of actin filament polymerization](https://amigo.geneontology.org/amigo/term/GO:0030838) and [positive regulation of pseudopodium assembly](https://amigo.geneontology.org/amigo/term/GO:0031274). The gene also enables [small GTPase binding](https://amigo.geneontology.org/amigo/term/GO:0031267).


The gene length is 21,151 base pairs (43.4% of all genes), the mature length is 2,934 base pairs (71.52% of all genes), and the primary transcript length is 17,834 base pairs (48.65% of all genes).


The gene CDC42EP3 (NCBI ID: 10602) has been mentioned in [9 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CDC42EP3%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning this gene was in 2006, and the middle 50% of publications occurred between 2017 and 2021.


The top 5 publications mentioning CDC42EP3 include "[Cdc42 and its BORG2 and BORG3 effectors control the subcellular localization of septins between actin stress fibers and microtubules.](https://pubmed.ncbi.nlm.nih.gov/34329591)" (2021) (relative citation ratio: 1.82), "[CDC42EP3 promotes glioma progression via regulation of CCND1.](https://pubmed.ncbi.nlm.nih.gov/35365622)" (2022) (relative citation ratio: 1.74), "[Cdc42 regulates Cdc42EP3 function in cancer-associated fibroblasts.](https://pubmed.ncbi.nlm.nih.gov/27248291)" (2017) (relative citation ratio: 0.9), "[Deduction of CDC42EP3 suppress development and progression of osteosarcoma.](https://pubmed.ncbi.nlm.nih.gov/34998812)" (2022) (relative citation ratio: 0.85), and "[CDC42EP3 promotes colorectal cancer through regulating cell proliferation, cell apoptosis and cell migration.](https://pubmed.ncbi.nlm.nih.gov/33726765)" (2021) (relative citation ratio: 0.61). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CDC42EP3](https://www.proteinatlas.org/ENSG00000163171-CDC42EP3) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in the plasma membrane and actin filaments. The gene is expressed in various clusters, including Cluster 28 in neutrophils with unknown function, Cluster 41 in smooth muscle tissue involved in ECM organization, Cluster 23 in the choroid plexus with mixed function, Cluster 53 in squamous epithelial cells related to keratinization, and Cluster 41 in adipocytes and endothelial cells with mixed function.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [RUNX1T1](https://www.ncbi.nlm.nih.gov/gene/862) with 6 experiments, [SP1](https://www.ncbi.nlm.nih.gov/gene/6667) with 5 experiments, [BRD2](https://www.ncbi.nlm.nih.gov/gene/6046) with 5 experiments, [TAL1](https://www.ncbi.nlm.nih.gov/gene/6886) with 5 experiments, and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The input data indicates associations with various disease conditions, including neurodegenerative diseases such as Alzheimer's disease and tauopathy. Additionally, it encompasses cognitive disorders, dementia, and diseases of mental health, all of which are subcategories of central nervous system diseases and broader nervous system diseases.



The gene is expressed in various tissues, including the uterus and skin, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in 721_B_lymphoblasts, CD33+_Myeloid cells, CD56+_NK cells, CD14+_Monocytes, Whole Blood, Smooth Muscle, and the Uterus.




The analyzed protein sequence has a GRAVY value of -0.552 (29.82% percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -7.78 (26.9% percentile), and the median structural flexibility is 1.011 (82.97% percentile). The protein has a notable affinity for helix (32.68%, 56.67% percentile) and turn (38.98%, 96.47% percentile) structures, with a lower affinity for sheet structures (29.53%, 25.99% percentile). The instability index is 52.51 (65.18% percentile), and the isoelectric point is 5.49 (17.88% percentile). The protein sequence is 254 amino acids long (22.21% percentile) with a molecular weight of 27677.75 Da (21.43% percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |