---
layout: post
title: HLA-DRA
date: 2025-01-17 16:55 CST
description: HLA-DRA description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/3122) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 3122  | HLA-DRA | ENSG00000204287 | 6p21.32 |



The gene is located in the [Golgi membrane](https://amigo.geneontology.org/amigo/term/GO:0000139), [immunological synapse](https://amigo.geneontology.org/amigo/term/GO:0001772), [lysosome](https://amigo.geneontology.org/amigo/term/GO:0005764), [lysosomal membrane](https://amigo.geneontology.org/amigo/term/GO:0005765), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986), [ER to Golgi transport vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0012507), [transport vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030658), [endocytic vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030666), [clathrin-coated endocytic vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030669), [early endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0031901), [late endosome membrane](https://amigo.geneontology.org/amigo/term/GO:0031902), [trans-Golgi network membrane](https://amigo.geneontology.org/amigo/term/GO:0032831), [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062), [lumenal side of endoplasmic reticulum membrane](https://amigo.geneontology.org/amigo/term/GO:0098553), and [autolysosome membrane](https://amigo.geneontology.org/amigo/term/GO:0120281). It is involved in the [positive regulation of T cell mediated cytotoxicity](https://amigo.geneontology.org/amigo/term/GO:0001916), [adaptive immune response](https://amigo.geneontology.org/amigo/term/GO:0002250), [myeloid dendritic cell antigen processing and presentation](https://amigo.geneontology.org/amigo/term/GO:0002469), [antigen processing and presentation of endogenous peptide antigen via MHC class II](https://amigo.geneontology.org/amigo/term/GO:0002491), [peptide antigen assembly with MHC class II protein complex](https://amigo.geneontology.org/amigo/term/GO:0002503), [antigen processing and presentation of peptide or polysaccharide antigen via MHC class II](https://amigo.geneontology.org/amigo/term/GO:0002504), [immune response](https://amigo.geneontology.org/amigo/term/GO:0006955), [antigen processing and presentation of exogenous peptide antigen via MHC class II](https://amigo.geneontology.org/amigo/term/GO:0019886), [positive regulation of CD4-positive, CD25-positive, alpha-beta regulatory T cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0032395), [positive regulation of memory T cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0043382), [regulation of T-helper cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0045622), [positive regulation of immune response](https://amigo.geneontology.org/amigo/term/GO:0050778), [positive regulation of T cell activation](https://amigo.geneontology.org/amigo/term/GO:0050870), [cognition](https://amigo.geneontology.org/amigo/term/GO:0050890), and [positive regulation of CD4-positive, alpha-beta T cell activation](https://amigo.geneontology.org/amigo/term/GO:2000516). Additionally, it enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [MHC class II protein complex binding](https://amigo.geneontology.org/amigo/term/GO:0023026), [polysaccharide binding](https://amigo.geneontology.org/amigo/term/GO:0030247), [MHC class II receptor activity](https://amigo.geneontology.org/amigo/term/GO:0032395), [peptide antigen binding](https://amigo.geneontology.org/amigo/term/GO:0042605), and [T cell receptor binding](https://amigo.geneontology.org/amigo/term/GO:0042608). It is also part of the [MHC class II protein complex](https://amigo.geneontology.org/amigo/term/GO:0042613).


The gene length is 133,470 base pairs (87.02% of all genes), the mature length is 3,558 base pairs, and the primary transcript length is 129,002 base pairs.


The gene HLA-DRA (NCBI ID: 3122) has been mentioned in [141 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22HLA-DRA%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1982, and the middle 50% of publications range from 1990 to 2013.


The top five publications mentioning HLA-DRA, ranked by their scientific influence, include "[Complete sequence of an HLA-dR beta chain deduced from a cDNA clone and identification of multiple non-allelic DR beta chain genes.](https://pubmed.ncbi.nlm.nih.gov/11894954)" (1983) (relative citation ratio: 6.96), "[Sequence polymorphism of HLA DR beta 1 alleles relating to T-cell-recognized determinants.](https://pubmed.ncbi.nlm.nih.gov/3875800)" (1985) (relative citation ratio: 4.97), "[A new member of the leucine zipper class of proteins that binds to the HLA DR alpha promoter.](https://pubmed.ncbi.nlm.nih.gov/2321018)" (1990) (relative citation ratio: 4.67), "[Allelic polymorphism and complexity of the genes for HLA-DR beta-chains--direct analysis by DNA-DNA hybridization.](https://pubmed.ncbi.nlm.nih.gov/6292729)" (1982) (relative citation ratio: 3.46), and "[HLA class II sequence polymorphisms and susceptibility to rheumatoid arthritis in Greeks. The HLA-DR beta shared-epitope hypothesis accounts for the disease in only a minority of Greek patients.](https://pubmed.ncbi.nlm.nih.gov/1352449)" (1992) (relative citation ratio: 3.41). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[HLA-DRA](https://www.proteinatlas.org/ENSG00000204287-HLA-DRA) is a gene with evidence at the protein level. Its RNA is detected in many tissues, and it is particularly expressed in dendritic cells of unknown function (Cluster 2) within the blood. In tissues, it is associated with lymphoid tissue involved in immune response (Cluster 70). In the brain, it is found in the hindbrain with mixed function (Cluster 43). Additionally, it is expressed in lymphoma cell lines related to adaptive immune response (Cluster 21) and in single cells, specifically dendritic cells involved in antigen presentation (Cluster 28).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 4 experiments, followed by [CEBPB](https://www.ncbi.nlm.nih.gov/gene/1051), [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169), [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861), and [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), each with 3 experiments.



The input data from GWAS indicates associations with a wide range of disease conditions, including infectious diseases such as viral and bacterial infections, various types of cancer such as hematologic, immune system, and organ system cancers, respiratory diseases including asthma and obstructive lung disease, metabolic disorders like type 2 diabetes mellitus and obesity, cardiovascular diseases such as coronary artery disease, neurodegenerative diseases including Alzheimer's and Parkinson's disease, musculoskeletal disorders like rheumatoid arthritis, sleep disorders such as narcolepsy, autoimmune diseases like systemic lupus erythematosus, gastrointestinal diseases including Crohn's disease, reproductive system diseases such as male infertility, and specific conditions like leprosy, sarcoidosis, and various forms of hepatitis.


HLA-DRA has been associated with several diseases and traits through genome-wide association studies. Notably, it has been linked to [A body shape index](https://pubmed.ncbi.nlm.nih.gov/34021172), which maps closely to the MeSH term [Index](https://meshb.nlm.nih.gov/record/ui?ui=D020481) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method. Additionally, HLA-DRA is associated with [Immunoglobulin A vasculitis](https://pubmed.ncbi.nlm.nih.gov/33591409), corresponding to the MeSH term [Immunoglobulins](https://meshb.nlm.nih.gov/record/ui?ui=D007136), and [Non-obstructive azoospermia](https://pubmed.ncbi.nlm.nih.gov/22541561), which maps to the MeSH term [Azoospermia](https://meshb.nlm.nih.gov/record/ui?ui=D053713). Furthermore, it has been linked to [Seropositivity for streptococcus pneumoniae peptide (agilent_241401)](https://pubmed.ncbi.nlm.nih.gov/37164013), with the closest MeSH match being [Streptococcus](https://meshb.nlm.nih.gov/record/ui?ui=D013291).


The gene is expressed in multiple tissue samples, including lung, blood, and spleen. Additionally, it is expressed in various cell types such as BDCA4+ dendritic cells, CD19+ B cells (neg. sel.), and CD14+ monocytes, as well as in tonsil tissue.


The input data highlights various pathways involved in immune system function, including cytokine signaling in the immune system, interferon signaling, and interferon gamma signaling. Additionally, it covers adaptive immune system processes, costimulation by the CD28 family, and downstream TCR signaling. Other notable pathways include TCR signaling, generation of second messenger molecules, PD-1 signaling, phosphorylation of CD3 and TCR zeta chains, translocation of ZAP-70 to the immunological synapse, and MHC class II antigen presentation.



The analyzed protein sequence has a GRAVY value of 0.056 (86.34th percentile), indicating a slightly hydrophobic nature. The charge at pH 7.0 is -12.94 (18.01st percentile), and the median structural flexibility is 0.997 (27.72nd percentile). The protein has a propensity for helix, sheet, and turn structures of 30.71% (38.13rd percentile), 42.52% (91.61st percentile), and 26.38% (32.56th percentile), respectively. The instability index is 44.51 (37.5th percentile), with an isoelectric point of 4.91 (6.1st percentile). The protein is 254 amino acids long (22.21st percentile) and has a molecular weight of 28620.6 Da (22.68th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |