---
layout: post
title: EIF2D
date: 2024-12-08 23:57 CST
description: EIF2D description
tags: [cooccuring-genes]
categories: longest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1939) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1939  | EIF2D | ENSG00000143486 | 1q32.1 |



The gene length is 1,118,922.0 base pairs (99.72% of all genes), the mature length is 4,655.0 base pairs (88.48% of all genes), and the primary transcript length is 676,324.0 base pairs (99.61% of all genes).


EIF2D has been studied in 7 publications, with the earliest publication in 2017 and the middle 50% of publications occurring between 2018 and 2020. For more information, see the [7 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22EIF2D%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)).


The top 5 publications mentioning EIF2D, ranked by their relative citation ratio (RCR), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows. The most influential publication is "[Translational induction of ATF4 during integrated stress response requires noncanonical initiation factors eIF2D and DENR.](https://pubmed.ncbi.nlm.nih.gov/32938929)" (2020) with an RCR of 2.25. Following this, "[A C. elegans model of C9orf72-associated ALS/FTD uncovers a conserved role for eIF2D in RAN translation.](https://pubmed.ncbi.nlm.nih.gov/34654821)" (2021) has an RCR of 2.02. Additionally, "[Tma64/eIF2D, Tma20/MCT-1, and Tma22/DENR Recycle Post-termination 40S Subunits InÂ Vivo.](https://pubmed.ncbi.nlm.nih.gov/30146315)" (2018) has an RCR of 1.78. "[New Pancreatic Cancer Biomarkers eIF1, eIF2D, eIF3C and eIF6 Play a Major Role in Translational Control in Ductal Adenocarcinoma.](https://pubmed.ncbi.nlm.nih.gov/32487605)" (2020) has an RCR of 1.07. Lastly, "[Translatome and transcriptome analysis of TMA20 (MCT-1) and TMA64 (eIF2D) knockout yeast strains.](https://pubmed.ncbi.nlm.nih.gov/30815525)" (2019) has an RCR of 0.49. For more information on RCR, visit [here](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541). Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified in terms of the number of CHIP-seq experiments include [GATA1](https://www.ncbi.nlm.nih.gov/gene/2623), [GTF3C2](https://www.ncbi.nlm.nih.gov/gene/2976), [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908), [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664), and [COMMD3-BMI1](https://www.ncbi.nlm.nih.gov/gene/100532731), each with 4 experiments demonstrating their regulatory roles.








The mouse ortholog gene 16865 and its human ortholog gene 1939 are associated with several phenotypic abnormalities. These include abnormal lymph node morphology, abnormal spleen morphology, enlarged lymph nodes, and enlarged spleen, with these phenotypes observed in both sexes.


The protein sequence analyzed has a GRAVY value of -0.343 (53.01 percentile), indicating a hydrophilic nature. At pH 7.0, the protein carries a charge of 1.69 (56.32 percentile). The median structural flexibility is 1.004 (57.95 percentile). The protein has a notable affinity for helix and sheet structures, with values of 0.348 (73.22 percentile) and 0.348 (57.29 percentile), respectively, and a lower affinity for turn structures at 0.283 (49.6 percentile). The instability index is 41.37 (26.61 percentile), and the isoelectric point is 7.56 (54.67 percentile). The protein sequence is 584 amino acids long (67.17 percentile) with a molecular weight of 64705.71 Da (66.79 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |