---
layout: post
title: U2AF1
date: 2025-01-17 16:55 CST
description: U2AF1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7307) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7307  | U2AF1 | ENSG00000160201 | 21q22.3 |



The gene is involved in [mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0000398) and [mRNA processing](https://amigo.geneontology.org/amigo/term/GO:0006397), and plays a role in [RNA splicing](https://amigo.geneontology.org/amigo/term/GO:0008380). It enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [pre-mRNA 3'-splice site binding](https://amigo.geneontology.org/amigo/term/GO:0030628), [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872), and [RS domain binding](https://amigo.geneontology.org/amigo/term/GO:0050733). The gene is located in the [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [Cajal body](https://amigo.geneontology.org/amigo/term/GO:0015030), and [nuclear speck](https://amigo.geneontology.org/amigo/term/GO:0016607). It is part of the [spliceosomal complex](https://amigo.geneontology.org/amigo/term/GO:0005681), [catalytic step 2 spliceosome](https://amigo.geneontology.org/amigo/term/GO:0071013), and [U2AF complex](https://amigo.geneontology.org/amigo/term/GO:0089701).


The gene length is 79,125 base pairs (77.12% of all genes), the mature length is 2,700 base pairs, and the primary transcript length is 44,875 base pairs.


The gene U2AF1 (NCBI ID: 7307) has been mentioned in [92 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22U2AF1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1996 and the middle 50% of publications occurring between 2013 and 2020. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top five publications mentioning U2AF1, ranked by their scientific influence, include "[Recurrent mutations in the U2AF1 splicing factor in myelodysplastic syndromes.](https://pubmed.ncbi.nlm.nih.gov/22158538)" (2011) (relative citation ratio: 9.12), "[Frequency and prognostic impact of mutations in SRSF2, U2AF1, and ZRSR2 in patients with myelodysplastic syndromes.](https://pubmed.ncbi.nlm.nih.gov/22389253)" (2012) (relative citation ratio: 8.57), "[Splice site m<sup>6</sup>A methylation prevents binding of U2AF35 to inhibit RNA splicing.](https://pubmed.ncbi.nlm.nih.gov/33930289)" (2021) (relative citation ratio: 6.53), "[U2AF1 mutations alter splice site recognition in hematological malignancies.](https://pubmed.ncbi.nlm.nih.gov/25267526)" (2015) (relative citation ratio: 5.87), and "[U2AF1 mutations induce oncogenic IRAK4 isoforms and activate innate immune pathways in myeloid malignancies.](https://pubmed.ncbi.nlm.nih.gov/31011167)" (2019) (relative citation ratio: 5.73). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[U2AF1](https://www.proteinatlas.org/ENSG00000160201-U2AF1) is a gene with RNA-binding molecular function, primarily located in the nucleoplasm. It is involved in cancer-related processes and has been identified as a disease variant. Evidence supporting its function is available at the protein level. The gene is detected in all RNA tissue distributions and is associated with non-specific transcription clusters in various tissue types, including brain and single cells. Additionally, it is linked to a leukemia-hemostasis cluster in cell line expression and a non-specific membrane trafficking cluster in blood expression.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [RBPJ](https://www.ncbi.nlm.nih.gov/gene/3516) with 2 experiments, [SP5](https://www.ncbi.nlm.nih.gov/gene/389058) with 1 experiment, [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613) with 1 experiment, [RUNX1T1](https://www.ncbi.nlm.nih.gov/gene/862) with 1 experiment, and [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861) with 1 experiment.



The GWAS data indicates associations with various metabolic and cardiovascular conditions, including diseases of metabolism, acquired metabolic diseases, carbohydrate metabolism diseases, glucose metabolism diseases, diabetes mellitus, and type 2 diabetes mellitus. Additionally, associations were found with artery disease, vascular disease, cardiovascular system disease, and hypertension.



The gene is expressed in various tissues and cell types, including CD34+ cells, CD105+ endothelial cells, colorectal adenocarcinoma, thymus, and leukemia lymphoblastic (MOLT-4) cells. Additionally, it is expressed in the pituitary gland.


The input data highlights various pathways involved in gene expression and RNA processing. These include RNA polymerase II transcription, post-elongation processing of the transcript, mRNA 3'-end processing, and mRNA splicing. Additionally, the data covers the transport of mature transcripts to the cytoplasm and the cleavage of growing transcripts in the termination region. These pathways collectively contribute to the regulation and maturation of mRNA, ensuring proper gene expression.



The analyzed protein sequence has a GRAVY value of -1.064 (3.23rd percentile), indicating a hydrophilic nature. It exhibits a charge of 8.09 at pH 7 (77.21st percentile) and a median structural flexibility of 1.009 (78.25th percentile). The protein's secondary structure is predicted to be 26.25% helix (8.63rd percentile), 24.17% sheet (6.81st percentile), and 30.42% turn (66.86th percentile). The instability index is 81.78 (98.52nd percentile), suggesting high instability. The isoelectric point is 9.09 (81.63rd percentile), with a molecular weight of 27871.77 Da (21.67th percentile) and a length of 240 amino acids (20.22nd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |