---
layout: post
title: CELF2
date: 2025-01-17 16:55 CST
description: CELF2 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10659) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10659  | CELF2 | ENSG00000048740 | 10p14 |



The gene is involved in the [regulation of alternative mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0000381) and [mRNA splice site recognition](https://amigo.geneontology.org/amigo/term/GO:0006376), playing a crucial role in [RNA processing](https://amigo.geneontology.org/amigo/term/GO:0006396). It enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [mRNA 3'-UTR binding](https://amigo.geneontology.org/amigo/term/GO:0003730), and [pre-mRNA binding](https://amigo.geneontology.org/amigo/term/GO:0036002). The gene is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), and [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), and is part of the [ribonucleoprotein complex](https://amigo.geneontology.org/amigo/term/GO:1990904). Additionally, it is located in the [intracellular membrane-bounded organelle](https://amigo.geneontology.org/amigo/term/GO:0043231) and the [Flemming body](https://amigo.geneontology.org/amigo/term/GO:0090543). The gene is also involved in the [regulation of heart contraction](https://amigo.geneontology.org/amigo/term/GO:0008016).


The gene length is 47,849 base pairs (65.13% of all genes), the mature length is 1,306 base pairs (32.2% of all genes), and the primary transcript length is 22,909 base pairs (55.36% of all genes).


The gene CELF2 (NCBI ID: 10659) has been mentioned in [49 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CELF2%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1999 and the middle 50% of publications occurring between 2010 and 2020. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning CELF2, ranked by their scientific influence, include "[Genome-wide association of familial late-onset Alzheimer's disease replicates BIN1 and CLU and nominates CUGBP2 in interaction with APOE.](https://pubmed.ncbi.nlm.nih.gov/21379329)" (2011) (relative citation ratio: 5.16), "[LncRNA CRNDE facilitates epigenetic suppression of CELF2 and LATS2 to promote proliferation, migration and chemoresistance in hepatocellular carcinoma.](https://pubmed.ncbi.nlm.nih.gov/32826865)" (2020) (relative citation ratio: 3.46), "[Coupled mRNA stabilization and translational silencing of cyclooxygenase-2 by a novel RNA binding protein, CUGBP2.](https://pubmed.ncbi.nlm.nih.gov/12535526)" (2003) (relative citation ratio: 3.24), "[Viral delivery of miR-196a ameliorates the SBMA phenotype via the silencing of CELF2.](https://pubmed.ncbi.nlm.nih.gov/22660636)" (2012) (relative citation ratio: 3.1), and "[miR-615-3p promotes proliferation and migration and inhibits apoptosis through its potential target CELF2 in gastric cancer.](https://pubmed.ncbi.nlm.nih.gov/29501762)" (2018) (relative citation ratio: 2.62). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CELF2](https://www.proteinatlas.org/ENSG00000048740-CELF2) is a repressor and RNA-binding protein with evidence at the protein level. It is implicated in epilepsy and intellectual disability. The gene is detected in all RNA tissue distributions and is localized in the nucleoplasm, vesicles, and midbody ring. CELF2 is expressed in various clusters, including Cluster 35 in neutrophils for blood expression, Cluster 16 for non-specific tissue expression, Cluster 51 in neurons for brain expression, Cluster 49 in HMC-1 for cell line expression, and Cluster 34 in neurons for single cell expression.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [SPI1](https://www.ncbi.nlm.nih.gov/gene/6688), [MYC](https://www.ncbi.nlm.nih.gov/gene/4609), [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), [KDM2B](https://www.ncbi.nlm.nih.gov/gene/84678), and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), each shown to be regulating in 5 experiments.



The input data from GWAS indicates associations with a wide range of disease conditions, including infectious diseases such as viral infectious diseases and human immunodeficiency virus infectious disease. Metabolic disorders are also prominent, with links to diabetes mellitus, particularly type 2 diabetes mellitus, and various metabolic diseases. Cardiovascular conditions are well-represented, including coronary artery disease, hypertension, and congestive heart failure. Neurological and mental health disorders are also notable, encompassing Alzheimer's disease, bipolar disorder, and attention deficit hyperactivity disorder. Additionally, the data highlights associations with musculoskeletal conditions like rheumatoid arthritis, liver diseases such as fatty liver disease, and gastrointestinal disorders including gallbladder disease.



The gene is expressed in various tissues, including nerve and blood, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the Prefrontal Cortex, CD34+ cells, CD33+ Myeloid cells, CD56+ NK cells, CD4+ T cells, CD8+ T cells, Whole Blood, and Fetal brain, suggesting a ubiquitous expression pattern across multiple tissue samples.



The mouse gene with NCBI ID 14007 and its human ortholog with NCBI ID 10659 are associated with the phenotype of preweaning lethality with incomplete penetrance, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.242 (64.79th percentile), indicating a hydrophilic nature. It exhibits a charge of 7.97 at pH 7.0 (76.88th percentile) and a median structural flexibility of 1.001 (45.52nd percentile). The protein's secondary structure is predicted to be 34.84% helix (73.73rd percentile), 29.92% sheet (27.95th percentile), and 30.31% turn (66.05th percentile). The instability index is 44.18 (36.34th percentile), with an isoelectric point of 8.98 (78.62nd percentile). The protein has a length of 508 amino acids (59.26th percentile) and a molecular weight of 54284.18 Da (56.44th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |