---
layout: post
title: STOM
date: 2025-01-17 16:55 CST
description: STOM description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2040) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2040  | STOM | ENSG00000148175 | 9q33.2 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802), and is involved in [protein homodimerization activity](https://amigo.geneontology.org/amigo/term/GO:0042803) and [RNA polymerase binding](https://amigo.geneontology.org/amigo/term/GO:0070063). It is located in various cellular components, including the [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615), [mitochondrion](https://amigo.geneontology.org/amigo/term/GO:0005739), [endoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0005783), [cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0005856), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [vesicle](https://amigo.geneontology.org/amigo/term/GO:0031982), [azurophil granule membrane](https://amigo.geneontology.org/amigo/term/GO:0035577), [specific granule membrane](https://amigo.geneontology.org/amigo/term/GO:0035579), [melanosome](https://amigo.geneontology.org/amigo/term/GO:0042470), [membrane raft](https://amigo.geneontology.org/amigo/term/GO:0045121), [perinuclear region of cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0048471), [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062), [tertiary granule membrane](https://amigo.geneontology.org/amigo/term/GO:0070821), and [blood microparticle](https://amigo.geneontology.org/amigo/term/GO:0072562). Additionally, it is involved in the regulation of [monoatomic ion transmembrane transport](https://amigo.geneontology.org/amigo/term/GO:0034765), [positive regulation by host of viral genome replication](https://amigo.geneontology.org/amigo/term/GO:0044829), [positive regulation of viral process](https://amigo.geneontology.org/amigo/term/GO:0048524), and [positive regulation of protein targeting to membrane](https://amigo.geneontology.org/amigo/term/GO:0090314).


The gene length is 38,766 base pairs (59.53% of all genes), the mature length is 2,978.5 base pairs (72.18% of all genes), and the primary transcript length is 37,658 base pairs (68.41% of all genes).


The gene STOM (NCBI ID: 2040) has been mentioned in [60 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22STOM%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning STOM was in 1970, and the middle 50% of publications occurred between 2001 and 2016.


The top 5 publications mentioning STOM, ranked by their scientific influence, include "[Stomatin, flotillin-1, and flotillin-2 are major integral proteins of erythrocyte lipid rafts.](https://pubmed.ncbi.nlm.nih.gov/11159550)" (2001) (relative citation ratio: 5.36), "[Association of stomatin with lipid bodies.](https://pubmed.ncbi.nlm.nih.gov/15024010)" (2004) (relative citation ratio: 4.0), "[Ca(++)-dependent vesicle release from erythrocytes involves stomatin-specific lipid rafts, synexin (annexin VII), and sorcin.](https://pubmed.ncbi.nlm.nih.gov/11895795)" (2002) (relative citation ratio: 3.64), "[Stomatin interacts with GLUT1/SLC2A1, band 3/SLC4A1, and aquaporin-1 in human erythrocyte membrane domains.](https://pubmed.ncbi.nlm.nih.gov/23219802)" (2013) (relative citation ratio: 2.02), and "[Overexpression of stomatin depresses GLUT-1 glucose transporter activity.](https://pubmed.ncbi.nlm.nih.gov/11287341)" (2001) (relative citation ratio: 1.83). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[STOM](https://www.proteinatlas.org/ENSG00000148175-STOM) is a gene with evidence at the protein level. Its RNA is detected in all tissues, and the protein is localized in vesicles, the plasma membrane, and the cytosol. The gene is expressed in various clusters, including Cluster 33 for non-specific ATP binding in blood, Cluster 11 for liver plasma proteins in tissues, Cluster 39 for astrocytes with mixed function in the brain, Cluster 64 for connective tissue cells involved in ECM organization in cell lines, and Cluster 41 for adipocytes and endothelial cells with mixed function in single cells.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [MAX](https://www.ncbi.nlm.nih.gov/gene/4149) and [CDK8](https://www.ncbi.nlm.nih.gov/gene/1024), each with 5 experiments. Additionally, [JUN](https://www.ncbi.nlm.nih.gov/gene/3725), [GATA2](https://www.ncbi.nlm.nih.gov/gene/2624), and [YY1](https://www.ncbi.nlm.nih.gov/gene/7528) were each found to be regulating in 4 experiments.



The GWAS data indicates associations with various disease conditions, including diseases of metabolism, acquired metabolic diseases, cognitive disorders, diseases of mental health, iron metabolism diseases, and mineral metabolism diseases.



The gene is expressed in various tissues, including adipose tissue, breast, and lung, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in CD71+ early erythroid cells, adipocytes, placenta, bone marrow, fetal liver, and fetal lung.




The protein sequence analyzed has a GRAVY value of 0.043 (85.87th percentile), indicating a slightly hydrophobic nature. At pH 7.0, the protein carries a charge of 0.864 (53.4th percentile). The median structural flexibility is 0.998 (32.8th percentile), suggesting high flexibility. The protein has an affinity for helix and sheet structures, with 34.4% (70.6th percentile) and 38.2% (79.55th percentile) of amino acids tending to form these structures, respectively, while 24.3% (16.92th percentile) tend to be in turns. The instability index is 41.0 (25.32nd percentile), and the isoelectric point is 7.71 (56.36th percentile). The protein is 288 amino acids long (26.66th percentile) with a molecular weight of 31730.42 Da (26.43rd percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |