---
layout: post
title: WNT9A
date: 2025-01-17 16:55 CST
description: WNT9A description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7483) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7483  | WNT9A | ENSG00000143816 | 1q42.13 |



The gene enables [frizzled binding](https://amigo.geneontology.org/amigo/term/GO:0005109) and [cytokine activity](https://amigo.geneontology.org/amigo/term/GO:0005125), and is involved in [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576) and is active in the [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615). The gene plays a role in the [negative regulation of cell population proliferation](https://amigo.geneontology.org/amigo/term/GO:0008285), [neuron differentiation](https://amigo.geneontology.org/amigo/term/GO:0030182), [negative regulation of chondrocyte differentiation](https://amigo.geneontology.org/amigo/term/GO:0032331), and [cell fate commitment](https://amigo.geneontology.org/amigo/term/GO:0045165). Additionally, it enables [receptor ligand activity](https://amigo.geneontology.org/amigo/term/GO:0048018) and is involved in the [canonical Wnt signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0060070), [iris morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0061072), [cornea development in camera-type eye](https://amigo.geneontology.org/amigo/term/GO:0061303), [cellular response to retinoic acid](https://amigo.geneontology.org/amigo/term/GO:0071300), and [embryonic skeletal joint development](https://amigo.geneontology.org/amigo/term/GO:0072498).


The gene length is 39,845 base pairs (60.22% of all genes), the mature length is 2,629 base pairs, and the primary transcript length is 39,003 base pairs.


The gene WNT9A (NCBI ID: 7483) has been mentioned in [20 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22WNT9A%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning this gene is 1997, and the middle 50% of publications occurred between 2009 and 2018.


The top 5 publications mentioning WNT9A include "[Wnt9a Promotes Renal Fibrosis by Accelerating Cellular Senescence in Tubular Epithelial Cells.](https://pubmed.ncbi.nlm.nih.gov/29440280)" (2018) (relative citation ratio: 6.85), "[Genome-wide association of phenotypes based on clustering patterns of hand osteoarthritis identify <i>WNT9A</i> as novel osteoarthritis gene.](https://pubmed.ncbi.nlm.nih.gov/33055079)" (2021) (relative citation ratio: 2.38), "[Molecular cloning and characterization of WNT3A and WNT14 clustered in human chromosome 1q42 region.](https://pubmed.ncbi.nlm.nih.gov/11414706)" (2001) (relative citation ratio: 1.86), "[EGFR is required for Wnt9a-Fzd9b signalling specificity in haematopoietic stem cells.](https://pubmed.ncbi.nlm.nih.gov/31110287)" (2019) (relative citation ratio: 1.43), and "[Distinct requirements of wls, wnt9a, wnt5b and gpc4 in regulating chondrocyte maturation and timing of endochondral ossification.](https://pubmed.ncbi.nlm.nih.gov/27908786)" (2017) (relative citation ratio: 1.31). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, and benchmarking to the median for NIH publications, the value of which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[WNT9A](https://www.proteinatlas.org/ENSG00000143816-WNT9A) is a developmental protein with evidence at the protein level. It is detected in many tissues and is associated with basic cellular processes in tissue expression cluster 19. In the brain, it is part of cluster 44, which has mixed functions. WNT9A is also expressed in squamous epithelial cells involved in keratinization and in ciliated cells involved in cilium assembly. The gene is unprognostic for various cancers, including Bladder Urothelial Carcinoma, Breast Invasive Carcinoma, and others, indicating no significant association with prognosis in these contexts.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [MYC](https://www.ncbi.nlm.nih.gov/gene/4609), [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227), [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), [AR](https://www.ncbi.nlm.nih.gov/gene/367), and [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), each shown to be regulating in 5 experiments.



The GWAS data indicates associations with various disease conditions, including diseases of metabolism, acquired metabolic diseases, nutrition diseases, overnutrition, obesity, general diseases, diseases of anatomical entities, nervous system diseases, and sensory system diseases.



The gene is expressed in various tissues, including muscle and ovary, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the Globus Pallidus, Ovary, and Thymus.


The input data highlights various pathways and processes involving WNT signaling, including enhanced WNT signaling and proliferation in RNF mutants, TCF-dependent signaling in response to WNT, and the stabilization of AXIN by XAV939 through tankyrase inhibition. Additionally, misspliced LRP5 mutants exhibit enhanced beta-catenin-dependent signaling, and WNT ligand biogenesis and trafficking are noted. The data also covers signaling by WNT in cancer, the abrogation of WNT ligand secretion by the PORCN inhibitor LGK974, and the negative regulation of TCF-dependent signaling by WNT ligand antagonists. Other pathways mentioned include GPCR ligand binding, signaling by GPCR, signal transduction, and metabolic disorders of biological oxidation enzymes. Furthermore, the data includes defective ACTH causing obesity and pro-opiomelanocortin deficiency (POMCD), as well as Class B/2 (Secretin family receptors).



The analyzed protein sequence has a GRAVY value of -0.384 (48.01 percentile), indicating a hydrophilic nature. It exhibits a charge of 17.19 at pH 7.0 (91.21 percentile) and a median structural flexibility of 0.998 (34.29 percentile). The protein's secondary structure is predicted to be 33.42% helix (63.08 percentile), 29.59% sheet (26.23 percentile), and 24.66% turn (19.16 percentile). The instability index is 51.36 (61.56 percentile), with an isoelectric point of 9.08 (81.32 percentile). The protein has a length of 365 amino acids (40.45 percentile) and a molecular weight of 40319.79 Da (39.86 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |