---
layout: post
title: RNGTT
date: 2025-01-17 16:55 CST
description: RNGTT description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/8732) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 8732  | RNGTT | ENSG00000111880 | 6q15 |



The gene enables [mRNA guanylyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0004484), [polynucleotide 5'-phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0004651), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [ATP binding](https://amigo.geneontology.org/amigo/term/GO:0005524), [GTP binding](https://amigo.geneontology.org/amigo/term/GO:0005525), [protein tyrosine/serine/threonine phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0008138), [RNA guanylyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0008192), [inorganic triphosphate phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0050355), and [mRNA 5'-phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0140818). It is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654). The gene is involved in [7-methylguanosine mRNA capping](https://amigo.geneontology.org/amigo/term/GO:0006370), [RNA processing](https://amigo.geneontology.org/amigo/term/GO:0006396), and [dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0016311).


The gene length is 7,628 base pairs (20.44% of all genes), the mature length is 2,253.5 base pairs, and the primary transcript length is 7,254 base pairs (25.71% of all genes).


The gene RNGTT, with NCBI ID 8732, has been mentioned in [8 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RNGTT%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning this gene was in 1992, and the middle 50% of publications occurred between 1999 and 2014.


The top 5 publications mentioning RNGTT, ranked by their relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)), include "[Characterization of CPT-11 hydrolysis by human liver carboxylesterase isoforms hCE-1 and hCE-2.](https://pubmed.ncbi.nlm.nih.gov/10728672)" (2000) (relative citation ratio: 7.51), "[Interleukin-6 alters the cellular responsiveness to clopidogrel, irinotecan, and oseltamivir by suppressing the expression of carboxylesterases HCE1 and HCE2.](https://pubmed.ncbi.nlm.nih.gov/17537833)" (2007) (relative citation ratio: 1.68), "[Stable hc/e vortices in a gauge theory of superconductivity in strongly correlated systems.](https://pubmed.ncbi.nlm.nih.gov/10000190)" (1992) (relative citation ratio: 1.0), "[How an mRNA capping enzyme reads distinct RNA polymerase II and Spt5 CTD phosphorylation codes.](https://pubmed.ncbi.nlm.nih.gov/24939935)" (2014) (relative citation ratio: 0.76), and "[Long-term exposure to a 40-GHz electromagnetic field does not affect genotoxicity or heat shock protein expression in HCE-T or SRA01/04 cells.](https://pubmed.ncbi.nlm.nih.gov/31197350)" (2019) (relative citation ratio: 0.36). The relative citation ratio measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RNGTT](https://www.proteinatlas.org/ENSG00000111880-RNGTT) is a multifunctional enzyme with hydrolase, nucleotidyltransferase, protein phosphatase, and transferase activities. It is detected in all tissues and primarily localized in the nucleoplasm. RNGTT is associated with transcription processes in various expression clusters, including blood (Cluster 24), lymphoid tissue (Cluster 36), brain (Cluster 16), lymphoma cell lines (Cluster 46), and B-cells (Cluster 73). The gene's expression patterns suggest involvement in transcription and immune response functions. Evidence supporting these findings is established at the protein level.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 6 experiments, [BRD9](https://www.ncbi.nlm.nih.gov/gene/65980) with 5 experiments, [E2F8](https://www.ncbi.nlm.nih.gov/gene/79733) with 5 experiments, [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885) with 5 experiments, and [KMT2B](https://www.ncbi.nlm.nih.gov/gene/9757) with 5 experiments.



The input data indicates associations with various disease conditions, including neurodegenerative diseases such as amyotrophic lateral sclerosis and motor neuron disease. Additionally, the data highlights mental health disorders and specific sleep disorders like narcolepsy. These conditions are categorized under broader terms such as nervous system diseases and central nervous system diseases.



The gene is expressed in various tissues, including blood and spleen, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Lymphoma Burkitts (Raji), CD19+ B Cells (neg. sel.), Appendix, Superior Cervical Ganglion, and Skeletal Muscle.


The input data highlights various biological processes and pathways, including gene expression, RNA Pol II CTD phosphorylation and interaction with CE, mRNA capping, transcription of the HIV genome, RNA polymerase II transcription, HIV infection, HIV life cycle, late phase of HIV life cycle, transcription, and disease. These pathways involve a range of molecular activities, from general transcription processes to specific mechanisms related to HIV infection and its life cycle.



The analyzed protein sequence has a GRAVY value of -0.592 (26.05th percentile), indicating a hydrophilic nature. It exhibits a charge of 6.49 at pH 7.0 (72.6th percentile) and a median structural flexibility of 1.005 (61.85th percentile). The protein's secondary structure is predicted to be 30.49% helix (36.22nd percentile), 32.99% sheet (45.49th percentile), and 29.98% turn (63.81st percentile). The instability index is 44.08 (35.98th percentile), with an isoelectric point of 8.43 (65.29th percentile). The protein has a length of 597 amino acids (68.37th percentile) and a molecular weight of 68555.97 Da (69.63rd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |