---
layout: post
title: MT1E
date: 2025-01-17 16:55 CST
description: MT1E description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/4493) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 4493  | MT1E | ENSG00000169715 | 16q13  |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and [zinc ion binding](https://amigo.geneontology.org/amigo/term/GO:0008270), and is active in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737). It is involved in various processes including [intracellular zinc ion homeostasis](https://amigo.geneontology.org/amigo/term/GO:0006882), [detoxification of copper ion](https://amigo.geneontology.org/amigo/term/GO:0010273), [negative regulation of growth](https://amigo.geneontology.org/amigo/term/GO:0045926), [cellular response to cadmium ion](https://amigo.geneontology.org/amigo/term/GO:0071276), [cellular response to copper ion](https://amigo.geneontology.org/amigo/term/GO:0071280), and [cellular response to zinc ion](https://amigo.geneontology.org/amigo/term/GO:0071294). Additionally, it enables [metal ion binding](https://amigo.geneontology.org/amigo/term/GO:0046872).


The gene length is 2,085 base pairs (5.51% of all genes). The mature length and primary transcript length are also 2,085 base pairs.


The gene MT1E (NCBI ID: 4493) has been mentioned in [25 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22MT1E%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1981 and the middle 50% of publications occurring between 2000 and 2012. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning MT1E, ranked by their scientific influence, include studies such as "[Role of metallothionein 1E in the migration and invasion of human glioma cell lines.](https://pubmed.ncbi.nlm.nih.gov/22843066)" (2012) (relative citation ratio: 1.21), "[Differential expression of the MT-1E gene in estrogen-receptor-positive and -negative human breast cancer cell lines.](https://pubmed.ncbi.nlm.nih.gov/9422519)" (1998) (relative citation ratio: 1.14), "[A novel Mtd splice isoform is responsible for trophoblast cell death in pre-eclampsia.](https://pubmed.ncbi.nlm.nih.gov/15775999)" (2005) (relative citation ratio: 1.09), "[Using hESCs to Probe the Interaction of the Diabetes-Associated Genes CDKAL1 and MT1E.](https://pubmed.ncbi.nlm.nih.gov/28538172)" (2017) (relative citation ratio: 1.02), and "[Metallothionein 1E mRNA is highly expressed in oestrogen receptor-negative human invasive ductal breast cancer.](https://pubmed.ncbi.nlm.nih.gov/10917545)" (2000) (relative citation ratio: 1.0). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[MT1E](https://www.proteinatlas.org/ENSG00000169715-MT1E) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in the nucleoplasm and is associated with various tissue expression clusters, including Cluster 69 in the liver with oxidoreductase activity, Cluster 19 in the brain related to astrocytes with mixed function, and Cluster 72 in neuronal cell lines involved in signal transduction. Additionally, MT1E is found in Cluster 56 of proximal tubular cells, involved in amino acid metabolism, and in Cluster 9 of T-regs, regulating the cell cycle in blood expression.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [EZH2](https://www.ncbi.nlm.nih.gov/gene/2146) with 6 experiments, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 5 experiments, [ERG](https://www.ncbi.nlm.nih.gov/gene/2078) with 5 experiments, [SUPT5H](https://www.ncbi.nlm.nih.gov/gene/6829) with 5 experiments, and [AR](https://www.ncbi.nlm.nih.gov/gene/367) with 5 experiments.





The gene is expressed in multiple tissue samples, including the liver, bronchial epithelial cells, adrenal gland, kidney, and fetal liver.




The protein sequence analyzed has a GRAVY value of 0.136, indicating a slightly hydrophobic nature (88.95th percentile). It exhibits a charge of 4.30 at pH 7.0 (65.68th percentile) and a median structural flexibility of 0.993 (16.49th percentile). The protein's secondary structure is predicted to be 27.87% helix (16.21st percentile), 8.20% sheet (0.28th percentile), and 29.51% turn (60.09th percentile). The instability index is 69.36 (94.21st percentile), suggesting high instability. The isoelectric point is 8.38 (64.27th percentile), with a molecular weight of 6014.19 Da (0.11th percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |