---
layout: post
title: COX7C
date: 2025-01-17 16:55 CST
description: COX7C description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1350) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1350  | COX7C | ENSG00000127184 | 5q14.3 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is located in the [mitochondrion](https://amigo.geneontology.org/amigo/term/GO:0005739), specifically within the [mitochondrial inner membrane](https://amigo.geneontology.org/amigo/term/GO:0005743) and is part of the [mitochondrial respiratory chain complex IV](https://amigo.geneontology.org/amigo/term/GO:0005751). It is involved in the [generation of precursor metabolites and energy](https://amigo.geneontology.org/amigo/term/GO:0006091), [mitochondrial electron transport, cytochrome c to oxygen](https://amigo.geneontology.org/amigo/term/GO:0006123), and [cellular respiration](https://amigo.geneontology.org/amigo/term/GO:0045333). Additionally, it is located in the [mitochondrial membrane](https://amigo.geneontology.org/amigo/term/GO:0031966).


The gene length is 7,172 base pairs (19.42% of all genes), the mature length is 2,839 base pairs, and the primary transcript length is 4,372.5 base pairs (16.28% of all genes).


The gene COX7C (NCBI ID: 1350) has been mentioned in [3 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22COX7C%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1998 and the middle 50% of publications ranging from 2009 to 2021. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top publications mentioning COX7C include "<i>UBE2D1</i> and <i>COX7C</i> as Potential Biomarkers of Diabetes-Related Sepsis." ([title](https://pubmed.ncbi.nlm.nih.gov/35445133)) (2022) (relative citation ratio: 1.66), "The expression level of COX7C associates with venous thromboembolism in colon cancer patients." ([title](https://pubmed.ncbi.nlm.nih.gov/32653968)) (2020) (relative citation ratio: 0.49), and "Assignment of the human genes coding for cytochrome c oxidase subunits Va (COX5A), VIc (COX6C) and VIIc (COX7C) to chromosome bands 15q25, 8q22–>q23 and 5q14 and of three pseudogenes (COX5AP1, COX6CP1, COX7CP1) to 14q22, 16p12 and 13q14–>q21 by FISH and radiation hybrid mapping." ([title](https://pubmed.ncbi.nlm.nih.gov/10072584)) (1998) (relative citation ratio: 0.4). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[COX7C](https://www.proteinatlas.org/ENSG00000127184-COX7C) is a gene with evidence at the protein level. Its RNA is detected in all tissues, and it is associated with various expression clusters, including Cluster 12 for blood expression related to mitochondrial translation, Cluster 23 for tissue expression related to cellular respiration, Cluster 21 for brain expression related to metabolism, Cluster 22 for cell line expression related to mRNA splicing and cell cycle, and Cluster 39 for single cell expression related to basic cellular processes.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [PGR](https://www.ncbi.nlm.nih.gov/gene/5241) with 5 experiments, [SP1](https://www.ncbi.nlm.nih.gov/gene/6667) with 4 experiments, [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) with 4 experiments, [TBX5](https://www.ncbi.nlm.nih.gov/gene/6910) with 4 experiments, and [CTCF](https://www.ncbi.nlm.nih.gov/gene/10664) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.





The gene is expressed in multiple tissue samples, including the heart and kidney, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the heart, kidney, CD19+ B cells (neg. sel.), promyelocytic leukemia (HL-60), Burkitt's lymphoma (Daudi), and the thalamus.


The proteins are involved in various metabolic processes, including respiratory electron transport, ATP synthesis by chemiosmotic coupling, heat production by uncoupling proteins, the citric acid (TCA) cycle, and transmembrane transport of small molecules. Additionally, some proteins are associated with orphan transporters.



The analyzed protein sequence has a GRAVY value of -0.078 (79.62nd percentile), indicating a relatively hydrophilic nature. It exhibits a charge of 5.67 at pH 7.0 (70.1st percentile) and a median structural flexibility of 0.986 (8.26th percentile). The protein's secondary structure affinities are 31.75% for helix (48.12nd percentile), 41.27% for sheet (89.29th percentile), and 23.81% for turn (13.97th percentile). The instability index is 74.45 (96.7th percentile), suggesting high instability, and the isoelectric point is 10.29 (97.7th percentile). The protein has a length of 63 amino acids (0.37th percentile) and a molecular weight of 7245.41 Da (0.43rd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |