---
layout: post
title: POLR2D
date: 2024-12-08 23:57 CST
description: POLR2D description
tags: [cooccuring-genes]
categories: shortest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5433) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5433  | POLR2D | ENSG00000144231 | 2q14.3  |



The gene length is 511.0 base pairs (0.52% of all genes). The mature length is 511.0 base pairs, and the primary transcript length is 511.0 base pairs.


The gene POLR2D, with NCBI ID 5433, has been mentioned in [1 publication](https://pubmed.ncbi.nlm.nih.gov/?term=%22POLR2D%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1998, and the middle 50% of publications occurred between 1998 and 1998.


The top publication mentioning POLR2D is "[Analysis of the interaction of the novel RNA polymerase II (pol II) subunit hsRPB4 with its partner hsRPB7 and with pol II.](https://pubmed.ncbi.nlm.nih.gov/9528765)" (1998) (relative citation ratio: 0.78). The Relative Citation Ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, and benchmarking to the median for NIH publications, the value of which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170) with 6 experiments, [FOXA1](https://www.ncbi.nlm.nih.gov/gene/3169) with 5 experiments, [RELA](https://www.ncbi.nlm.nih.gov/gene/5970) with 5 experiments, [CREB1](https://www.ncbi.nlm.nih.gov/gene/1385) with 5 experiments, and [KMT2B](https://www.ncbi.nlm.nih.gov/gene/9757) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent detection in CHIP-seq studies.











The analyzed protein sequence has a GRAVY value of -0.631 (22.61% percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -11.202 (20.63% percentile), and the median structural flexibility is 1.013 (87.89% percentile). The protein has a high affinity for helix structures (0.444, 97.97% percentile), with a lower affinity for sheet (0.310, 33.85% percentile) and turn structures (0.218, 5.99% percentile). The instability index is 59.646 (82.42% percentile), suggesting high instability. The isoelectric point is 4.755 (3.78% percentile), and the sequence length is 142 amino acids (7.15% percentile), with a molecular weight of 16311.09 Da (7.5% percentile). For more details on the sequence analysis, refer to [BioPython's sequence analysis documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |