---
layout: post
title: RPS9
date: 2024-12-08 23:57 CST
description: RPS9 description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/6203) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 6203  | RPS9 | ENSG00000170889 | 19q13.42 |



The gene length is 141,479 base pairs (87.99% of all genes), the mature length is 4,206 base pairs (85.46% of all genes), and the primary transcript length is 68,371 base pairs (81.98% of all genes).


The gene RPS9 (NCBI ID: 6203) has been featured in [8 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RPS9%22) (count may differ as it is from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 2008, and the middle 50% of publications occurred between 2011 and 2019.


The top 5 publications mentioning RPS9, ranked by their relative citation ratio (RCR), which measures the scientific influence of each paper by field- and time-adjusting the citations it has received and benchmarking to the median for NIH publications, are as follows. The most influential publication is "[LncRNA BRCAT54 inhibits the tumorigenesis of non-small cell lung cancer by binding to RPS9 to transcriptionally regulate JAK-STAT and calcium pathway genes.](https://pubmed.ncbi.nlm.nih.gov/32459848)" (2021) with a relative citation ratio of 3.29. This is followed by "[Identification of Susceptibility Loci in IL6, RPS9/LILRB3, and an Intergenic Locus on Chromosome 21q22 in Takayasu Arteritis in a Genome-Wide Association Study.](https://pubmed.ncbi.nlm.nih.gov/25604533)" (2015) with a relative citation ratio of 2.65. Additionally, "[Ribosomal protein S9 is a novel B23/NPM-binding protein required for normal cell proliferation.](https://pubmed.ncbi.nlm.nih.gov/18420587)" (2008) has a relative citation ratio of 1.88, and "[Silencing of ribosomal protein S9 elicits a multitude of cellular responses inhibiting the growth of cancer cells subsequent to p53 activation.](https://pubmed.ncbi.nlm.nih.gov/20221446)" (2010) has a relative citation ratio of 1.43. Lastly, "[Down-regulation of RPS9 Inhibits Osteosarcoma Cell Growth through Inactivation of MAPK Signaling Pathway.](https://pubmed.ncbi.nlm.nih.gov/28928861)" (2017) has a relative citation ratio of 0.96. The citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [YY1](https://www.ncbi.nlm.nih.gov/gene/7528), [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613), [SMC3](https://www.ncbi.nlm.nih.gov/gene/9126), and [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170), each with 6 experiments. Additionally, [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313) was found to be regulating in 5 experiments.


The top transcription factors (TFs) identified from the GTRD's database of TF binding sites, filtered for Perturb-seq data, include [AR](https://www.ncbi.nlm.nih.gov/gene/6829) (3 experiments), [PIM1](https://www.ncbi.nlm.nih.gov/gene/6872) (4 experiments), [FOS](https://www.ncbi.nlm.nih.gov/gene/9128) (1 experiment), [ZNF263](https://www.ncbi.nlm.nih.gov/gene/25799) (1 experiment), and [FOXA1](https://www.ncbi.nlm.nih.gov/gene/2958) (1 experiment). These TFs were found to regulate gene expression in multiple Perturb-seq experiments, highlighting their significant roles in transcriptional regulation.








The analyzed protein sequence has a GRAVY value of -0.644 (21.61% percentile), indicating a hydrophilic nature. It exhibits a charge of 21.87 at pH 7.0 (94.41% percentile) and a median structural flexibility of 1.003 (57.46% percentile). The protein's secondary structure is predicted to be 34.54% helix (71.8% percentile), 34.02% sheet (52.16% percentile), and 23.2% turn (10.77% percentile). The instability index is 60.69 (84.25% percentile), with an isoelectric point of 10.66 (98.6% percentile). The protein has a length of 194 amino acids (13.59% percentile) and a molecular weight of 22591.14 Da (14.94% percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |