---
layout: post
title: VIM
date: 2025-01-17 16:55 CST
description: VIM description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7431) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7431  | VIM | ENSG00000026025 | 10p13 |



The gene enables [double-stranded RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003725) and [structural constituent of cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0005200), and is involved in [positive regulation of gene expression](https://amigo.geneontology.org/amigo/term/GO:0010628) and [negative regulation of neuron projection development](https://amigo.geneontology.org/amigo/term/GO:0010977). It is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [intermediate filament](https://amigo.geneontology.org/amigo/term/GO:0005882), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), and [axon](https://amigo.geneontology.org/amigo/term/GO:0030424), and is located in the [peroxisome](https://amigo.geneontology.org/amigo/term/GO:0005777), [microtubule organizing center](https://amigo.geneontology.org/amigo/term/GO:0005815), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0005856), [focal adhesion](https://amigo.geneontology.org/amigo/term/GO:0005925), [nuclear matrix](https://amigo.geneontology.org/amigo/term/GO:0016363), [cell leading edge](https://amigo.geneontology.org/amigo/term/GO:0031252), [intermediate filament cytoskeleton](https://amigo.geneontology.org/amigo/term/GO:0045111), [phagocytic vesicle](https://amigo.geneontology.org/amigo/term/GO:0045335), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). Additionally, it enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [protein domain specific binding](https://amigo.geneontology.org/amigo/term/GO:0019904), [identical protein binding](https://amigo.geneontology.org/amigo/term/GO:0042802), [molecular adaptor activity](https://amigo.geneontology.org/amigo/term/GO:0060090), and [scaffold protein binding](https://amigo.geneontology.org/amigo/term/GO:0097110), and is involved in various processes such as [astrocyte development](https://amigo.geneontology.org/amigo/term/GO:0014002), [neuron projection development](https://amigo.geneontology.org/amigo/term/GO:0031175), [positive regulation of collagen biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0032967), [regulation of mRNA stability](https://amigo.geneontology.org/amigo/term/GO:0043488), [intermediate filament organization](https://amigo.geneontology.org/amigo/term/GO:0045109), [Bergmann glial cell differentiation](https://amigo.geneontology.org/amigo/term/GO:0060020), [lens fiber cell development](https://amigo.geneontology.org/amigo/term/GO:0070307), [cellular response to lipopolysaccharide](https://amigo.geneontology.org/amigo/term/GO:0071222), [cellular response to muramyl dipeptide](https://amigo.geneontology.org/amigo/term/GO:0071225), and [cellular response to type II interferon](https://amigo.geneontology.org/amigo/term/GO:0071346).


The gene has a length of 43,726 base pairs (62.71 percentile of all genes), a mature length of 1,446.5 base pairs, and a primary transcript length of 41,186.5 base pairs.


The gene VIM (NCBI ID: 7431) has been mentioned in [1,778 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22VIM%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication mentioning VIM is 1950, and the middle 50% of publications occurred between 1994 and 2017.


The top 5 publications mentioning VIM, ranked by their scientific influence, include "[Widespread occurrence of intermediate-sized filaments of the vimentin-type in cultured cells from diverse vertebrates.](https://pubmed.ncbi.nlm.nih.gov/114401)" (1979) (relative citation ratio: 25.36), "[Dissemination of Carbapenemases (KPC, NDM, OXA-48, IMP, and VIM) Among Carbapenem-Resistant *Enterobacteriaceae* Isolated From Adult and Children Patients in China.](https://pubmed.ncbi.nlm.nih.gov/32719751)" (2020) (relative citation ratio: 17.2), "[Vimentin induces changes in cell shape, motility, and adhesion during the epithelial to mesenchymal transition.](https://pubmed.ncbi.nlm.nih.gov/20097873)" (2010) (relative citation ratio: 16.26), "[Induction of osteoclastogenesis and bone loss by human autoantibodies against citrullinated vimentin.](https://pubmed.ncbi.nlm.nih.gov/22505457)" (2012) (relative citation ratio: 15.65), and "[Monoclonal antibodies specific for vimentin.](https://pubmed.ncbi.nlm.nih.gov/6428888)" (1984) (relative citation ratio: 14.03). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[VIM](https://www.proteinatlas.org/ENSG00000026025-VIM) is a gene involved in cataract disease variants, with evidence at the protein level. Its RNA is detected in all tissues, and the protein is localized to intermediate filaments. VIM is expressed in monocytes and is associated with plasma membrane proteins in the blood. In tissues, it is involved in the extracellular matrix organization of connective tissue. Additionally, VIM is expressed in the vasculature of the brain and is involved in signal transduction in neuronal cell lines. It also plays a role in DNA binding in single cells.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861), [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313), and [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170), each with 6 experiments. Additionally, [RNF2](https://www.ncbi.nlm.nih.gov/gene/6045) and [INTS11](https://www.ncbi.nlm.nih.gov/gene/54973) were found to be regulating in 5 experiments each.


The top transcription factors (TFs) identified in Perturb-seq experiments include [MYC](https://www.ncbi.nlm.nih.gov/gene/23028) with 4 experiments, [BCL6](https://www.ncbi.nlm.nih.gov/gene/6872) and [TCF3](https://www.ncbi.nlm.nih.gov/gene/7936) each with 3 experiments, and [HNF4A](https://www.ncbi.nlm.nih.gov/gene/3191), [ZNF274](https://www.ncbi.nlm.nih.gov/gene/7469), and [ZNF783](https://www.ncbi.nlm.nih.gov/gene/8328) each with 1 experiment. These TFs were found to regulate gene expression in multiple experiments, highlighting their significant roles in transcriptional regulation.


The GWAS data indicates associations with various disease conditions, including immune system disease, hypersensitivity reaction disease, hypersensitivity reaction type II disease, autoimmune disease of the gastrointestinal tract, and celiac disease.



The gene is expressed in various tissues, including nerve, blood vessel, and ovary, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Cardiac Myocytes, Adipocytes, Superior Cervical Ganglion, Trigeminal Ganglion, Atrioventricular Node, Ciliary Ganglion, and Dorsal Root Ganglion.


The input data highlights several biological processes, including apoptosis and its related mechanisms such as programmed cell death, apoptotic cleavage of cellular proteins, apoptotic execution phase, and caspase-mediated cleavage of cytoskeletal proteins. Additionally, the data includes processes related to muscle contraction, specifically striated muscle contraction.


The mouse ortholog gene 22352 and its human ortholog gene 7431 are associated with the phenotype of increased blood uric acid level. This phenotype was observed to be sex-specific.


The analyzed protein sequence has a GRAVY value of -0.823 (10.61% percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -18.83 (11.95% percentile), and the median structural flexibility is 1.011 (83.08% percentile). The protein has a high affinity for helix structures (37.55%, 87.21% percentile) and a moderate affinity for sheet structures (31.12%, 34.59% percentile) and turn structures (25.32%, 23.85% percentile). The instability index is 58.08 (79.33% percentile), suggesting potential instability. The isoelectric point is 5.06 (8.9% percentile), with a length of 466 amino acids (53.93% percentile) and a molecular weight of 53651.04 Da (55.73% percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |