---
layout: post
title: PPM1B
date: 2025-01-17 16:55 CST
description: PPM1B description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5495) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5495  | PPM1B | ENSG00000138032 | 2p21 |



The gene enables [magnesium ion binding](https://amigo.geneontology.org/amigo/term/GO:0000287) and [protein serine/threonine phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0004722), and is involved in [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is active in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634) and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and is located in the [nucleolus](https://amigo.geneontology.org/amigo/term/GO:0005730), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), and [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020). The gene is involved in various processes including [protein dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0006470), [N-terminal protein myristoylation](https://amigo.geneontology.org/amigo/term/GO:0006499), [negative regulation of interferon-beta production](https://amigo.geneontology.org/amigo/term/GO:0032688), [peptidyl-threonine dephosphorylation](https://amigo.geneontology.org/amigo/term/GO:0035970), [negative regulation of canonical NF-kappaB signal transduction](https://amigo.geneontology.org/amigo/term/GO:0043124), [negative regulation of defense response to virus](https://amigo.geneontology.org/amigo/term/GO:0050687), [positive regulation of canonical Wnt signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0090263), and [negative regulation of non-canonical NF-kappaB signal transduction](https://amigo.geneontology.org/amigo/term/GO:1901223). Additionally, it enables [myosin phosphatase activity](https://amigo.geneontology.org/amigo/term/GO:0017018) and [manganese ion binding](https://amigo.geneontology.org/amigo/term/GO:0030145).


The gene length is 115,248.0 base pairs (84.42% of all genes), the mature length is 2,924.0 base pairs, and the primary transcript length is 92,632.5 base pairs.


The gene PPM1B (NCBI ID: 5495) has been mentioned in [22 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PPM1B%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 2009 and the middle 50% of publications occurring between 2013 and 2020. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning PPM1B, ranked by their scientific influence, include "[Ppm1b negatively regulates necroptosis through dephosphorylating Rip3.](https://pubmed.ncbi.nlm.nih.gov/25751141)" (2015) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 3.32, "[Berberine Improves Inflammatory Responses of Diabetes Mellitus in Zucker Diabetic Fatty Rats and Insulin-Resistant HepG2 Cells through the PPM1B Pathway.](https://pubmed.ncbi.nlm.nih.gov/32908938)" (2020) with an RCR of 1.81, "[PPM1A and PPM1B act as IKKbeta phosphatases to terminate TNFalpha-induced IKKbeta-NF-kappaB activation.](https://pubmed.ncbi.nlm.nih.gov/18930133)" (2009) with an RCR of 1.79, "[TRIM59 promotes tumor growth in hepatocellular carcinoma and regulates the cell cycle by degradation of protein phosphatase 1B.](https://pubmed.ncbi.nlm.nih.gov/31875525)" (2020) with an RCR of 1.56, and "[PPM1B negatively regulates antiviral response via dephosphorylating TBK1.](https://pubmed.ncbi.nlm.nih.gov/22750291)" (2012) with an RCR of 1.24. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PPM1B](https://www.proteinatlas.org/ENSG00000138032-PPM1B) is a protein-coding gene with hydrolase and protein phosphatase molecular functions. Evidence for this gene is available at the protein level. Its RNA is detected in all tissues, with subcellular localization in the nucleoli and cytosol. In terms of expression clusters, PPM1B is found in Cluster 21 for blood (Neutrophils - Unknown function), Cluster 3 for tissue (Non-specific - Unknown function), Cluster 13 for brain (Oligodendrocytes - Mixed function), Cluster 15 for cell line (Kidney cancer - Membrane components), and Cluster 78 for single cell (NK-cells & T-cells - Immune system & Transcription).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [FOXA2](https://www.ncbi.nlm.nih.gov/gene/3170) with 5 experiments, [OTX2](https://www.ncbi.nlm.nih.gov/gene/5015) with 4 experiments, [AR](https://www.ncbi.nlm.nih.gov/gene/367) with 4 experiments, [AHR](https://www.ncbi.nlm.nih.gov/gene/196) with 4 experiments, and [PGR](https://www.ncbi.nlm.nih.gov/gene/5241) with 4 experiments.



The input data indicates associations with various disease conditions, including neurodegenerative diseases such as Alzheimer's disease and tauopathy. Additionally, it highlights cognitive disorders, dementia, and diseases of mental health, specifically substance-related disorders like substance dependence and alcohol dependence. These conditions are interconnected, affecting the central nervous system and overall mental health.



The gene is expressed in various tissues, including muscle, blood vessel, and brain, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in pineal gland (both day and night), prefrontal cortex, retina, whole blood, amygdala, prostate, and colon, suggesting a ubiquitous expression pattern across multiple tissue samples.


The proteins are involved in several key pathways related to immune response, including the antiviral mechanism by IFN-stimulated genes, ISG15 antiviral mechanism, cytokine signaling in the immune system, interferon signaling, and the broader immune system.



The analyzed protein sequence has a GRAVY value of -0.545 (30.48th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -28.15 (6.68th percentile), and the median structural flexibility is 1.007 (73.28th percentile). The protein has a predicted secondary structure with 33.40% helix (62.94th percentile), 27.77% sheet (17.54th percentile), and 33.40% turn (83.72nd percentile) affinities. The instability index is 40.64 (24.04th percentile), with an isoelectric point of 4.95 (6.91st percentile). The protein is 479 amino acids long (55.78th percentile) and has a molecular weight of 52642.03 Da (54.66th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |