---
layout: post
title: EIF4B
date: 2025-01-17 16:55 CST
description: EIF4B description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/1975) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 1975  | EIF4B | ENSG00000063046 | 12q13.13 |



The gene enables [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723) and [translation initiation factor activity](https://amigo.geneontology.org/amigo/term/GO:0003743), and is involved in [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). It is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829) and is involved in [translational initiation](https://amigo.geneontology.org/amigo/term/GO:0006413) and [regulation of translational initiation](https://amigo.geneontology.org/amigo/term/GO:0006446). Additionally, it is part of the [eukaryotic translation initiation factor 4F complex](https://amigo.geneontology.org/amigo/term/GO:0016281).


The gene length is 13,060 base pairs (31.72 percentile of all genes), the mature length is 775 base pairs (12.67 percentile of all genes), and the primary transcript length is 13,028 base pairs (40.32 percentile of all genes).


EIF4B (Gene ID: 1975) has been mentioned in [63 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22EIF4B%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest mention in 1978. The middle 50% of publications occurred between 1998 and 2015. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning EIF4B, ranked by their scientific influence, include "[The mTOR/PI3K and MAPK pathways converge on eIF4B to control its phosphorylation and activity.](https://pubmed.ncbi.nlm.nih.gov/16763566)" (2006) (relative citation ratio: 7.4), "[Phosphorylation of eucaryotic translation initiation factor 4B Ser422 is modulated by S6 kinases.](https://pubmed.ncbi.nlm.nih.gov/15071500)" (2004) (relative citation ratio: 5.97), "[The mTORC1/S6K1 pathway regulates glutamine metabolism through the eIF4B-dependent control of c-Myc translation.](https://pubmed.ncbi.nlm.nih.gov/25220053)" (2014) (relative citation ratio: 5.32), "[Modulation of the helicase activity of eIF4A by eIF4B, eIF4H, and eIF4F.](https://pubmed.ncbi.nlm.nih.gov/11418588)" (2001) (relative citation ratio: 4.1), and "[Translation initiation factors eIF-iso4G and eIF-4B interact with the poly(A)-binding protein and increase its RNA binding activity.](https://pubmed.ncbi.nlm.nih.gov/9195926)" (1997) (relative citation ratio: 3.93). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[EIF4B](https://www.proteinatlas.org/ENSG00000063046-EIF4B) is an initiation factor with RNA-binding molecular function, supported by evidence at the protein level. It is detected in all tissues and is localized in the endoplasmic reticulum, plasma membrane, and cytosol. EIF4B is involved in basic cellular processes and is associated with ribosome function across various tissue and cell types. Its expression in blood is linked to ATP binding, but it is not significantly prognostic in multiple cancer types, including bladder urothelial carcinoma, breast invasive carcinoma, and others.


The top transcription factors (TFs) identified from GTRD's database of TF binding sites, based on the number of CHIP-seq experiments, include [JUN](https://www.ncbi.nlm.nih.gov/gene/3725) with 5 experiments, [BRD9](https://www.ncbi.nlm.nih.gov/gene/65980) with 5 experiments, [STAT1](https://www.ncbi.nlm.nih.gov/gene/6772) with 5 experiments, [PPARG](https://www.ncbi.nlm.nih.gov/gene/5468) with 4 experiments, and [MBD4](https://www.ncbi.nlm.nih.gov/gene/8930) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.



The GWAS data indicates associations with various conditions, including cognitive disorders and diseases of mental health.



The gene is expressed in various tissues, including the ovary and cervix uteri, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Leukemia (chronic myelogenous K-562 and promyelocytic HL-60), Lymphoma (Burkitt's Daudi), smooth muscle, pituitary, and ovary.


The input data highlights various biological processes and pathways, including metabolism of proteins, gene expression, and multiple aspects of translation such as cap-dependent translation initiation, eukaryotic translation initiation, and ribosomal scanning. Additionally, it covers signaling pathways like the IGF1R signaling cascade, insulin receptor signaling cascade, and PI3K cascade. The data also includes mTOR signaling, S6K1 signaling, and mTORC1-mediated signaling, as well as processes related to mRNA decay, such as deadenylation of mRNA and deadenylation-dependent mRNA decay.


The mouse ortholog gene 75705 and its human ortholog gene 1975 are associated with several phenotypic abnormalities, including abnormal locomotor behavior, edema, embryonic growth retardation, hemorrhage, and preweaning lethality with complete penetrance. These phenotypes are observed in both sexes.


The analyzed protein sequence has a GRAVY value of -1.61 (0.39 percentile), indicating a hydrophilic nature. The charge at pH 7 is -8.99 (24.29 percentile), and the median structural flexibility is 1.03 (98.89 percentile). The protein has a helix affinity of 0.25 (5.88 percentile), a sheet affinity of 0.20 (1.89 percentile), and a turn affinity of 0.43 (98.63 percentile). The instability index is 60.88 (84.62 percentile), with an isoelectric point of 5.55 (19.44 percentile). The sequence length is 611 amino acids (69.38 percentile), and the molecular weight is 69150.37 Da (70.04 percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |