---
layout: post
title: RAP1GAP
date: 2025-01-17 16:55 CST
description: RAP1GAP description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5909) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5909  | RAP1GAP | ENSG00000076864 | 1p36.12 |



The gene is located in the [Golgi membrane](https://amigo.geneontology.org/amigo/term/GO:0000139), [early endosome](https://amigo.geneontology.org/amigo/term/GO:0005769), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [axon](https://amigo.geneontology.org/amigo/term/GO:0030424), [dendrite](https://amigo.geneontology.org/amigo/term/GO:0030425), and [neuronal cell body](https://amigo.geneontology.org/amigo/term/GO:0043025). It is active in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737) and [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020). The gene is involved in various processes, including [adaptive immune response](https://amigo.geneontology.org/amigo/term/GO:0002250), [signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007165), [regulation of GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0043087), [positive regulation of GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0043547), [negative regulation of neuron differentiation](https://amigo.geneontology.org/amigo/term/GO:0045665), [regulation of small GTPase mediated signal transduction](https://amigo.geneontology.org/amigo/term/GO:0051056), [negative regulation of microvillus assembly](https://amigo.geneontology.org/amigo/term/GO:1903697), [negative regulation of thyroid gland epithelial cell proliferation](https://amigo.geneontology.org/amigo/term/GO:1904442), and [cellular response to glial cell derived neurotrophic factor](https://amigo.geneontology.org/amigo/term/GO:1990792). Additionally, the gene enables [GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0003924), [GTPase activator activity](https://amigo.geneontology.org/amigo/term/GO:0005096), [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [small GTPase binding](https://amigo.geneontology.org/amigo/term/GO:0031267), and [protein homodimerization activity](https://amigo.geneontology.org/amigo/term/GO:0042803).


The gene length is 40,756 base pairs (60.83% of all genes), the mature length is 1,317.5 base pairs, and the primary transcript length is 29,664 base pairs.


The gene RAP1GAP, with NCBI ID 5909, has been mentioned in [44 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RAP1GAP%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727). The earliest year of publication is 1992, and the middle 50% of publications occurred between 2007 and 2015. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning RAP1GAP, ranked by their scientific influence, include studies that have significantly contributed to the understanding of this gene. The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Notable publications include "[Identification of tuberin, the tuberous sclerosis-2 product. Tuberin possesses specific Rap1GAP activity.](https://pubmed.ncbi.nlm.nih.gov/7608212)" (1995) (relative citation ratio: 6.21), "[Regulation of dendritic spine morphology by SPAR, a PSD-95-associated RapGAP.](https://pubmed.ncbi.nlm.nih.gov/11502259)" (2001) (relative citation ratio: 5.01), "[The tumor suppressor gene rap1GAP is silenced by miR-101-mediated EZH2 overexpression in invasive squamous cell carcinoma.](https://pubmed.ncbi.nlm.nih.gov/21532618)" (2011) (relative citation ratio: 1.89), "[The GTPase-activating protein Rap1GAP uses a catalytic asparagine.](https://pubmed.ncbi.nlm.nih.gov/15141215)" (2004) (relative citation ratio: 1.88), and "[The Rap-RapGAP complex: GTP hydrolysis without catalytic glutamine and arginine residues.](https://pubmed.ncbi.nlm.nih.gov/18309292)" (2008) (relative citation ratio: 1.64). Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RAP1GAP](https://www.proteinatlas.org/ENSG00000076864-RAP1GAP) is a gene involved in GTPase activation, with evidence supporting its function at the protein level. It is primarily located in the cytosol and is detected in many tissues. The gene is notably expressed in the brain, specifically in neuronal signaling clusters, and is also found in neurons with mixed functions. Additionally, RAP1GAP is expressed in liver cancer cell lines associated with metabolism and in intestinal goblet cells involved in mucin production.


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), [ERG](https://www.ncbi.nlm.nih.gov/gene/2078), [YY1](https://www.ncbi.nlm.nih.gov/gene/7528), [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297), and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), each shown to be regulating in 5 experiments.



The input data from GWAS indicates associations with a wide range of disease conditions, including various types of cancer such as breast cancer, thoracic cancer, and lung disease. Additionally, metabolic disorders such as type 2 diabetes mellitus, obesity, and carbohydrate metabolism disease are highlighted. Cardiovascular conditions, including coronary artery disease and hypertension, are also noted. Furthermore, mental health disorders such as bipolar disorder and cognitive disorder are included in the data.


RAP1GAP has been associated with several traits and diseases through genome-wide association studies. Notably, it has been linked to [Educational attainment](https://pubmed.ncbi.nlm.nih.gov/35361970), which maps closely to the MeSH term [Educational Status](https://meshb.nlm.nih.gov/record/ui?ui=D004522) via a direct synonym match. Additionally, RAP1GAP is associated with [Phosphorus levels](https://pubmed.ncbi.nlm.nih.gov/29403010), corresponding to the MeSH term [Phosphorus](https://meshb.nlm.nih.gov/record/ui?ui=D010758) through an embedding match using [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0). Furthermore, it is linked to [Serum alkaline phosphatase levels](https://pubmed.ncbi.nlm.nih.gov/33462484), which maps to the MeSH term [Phosphoric Monoester Hydrolases](https://meshb.nlm.nih.gov/record/ui?ui=D010744), and [Serum phosphate levels](https://pubmed.ncbi.nlm.nih.gov/33462484), corresponding to the MeSH term [Phosphates](https://meshb.nlm.nih.gov/record/ui?ui=D010710), both through embedding matches.


The gene is expressed in the thyroid, as indicated by both BioGPS and GTEx profiles. Additionally, GTEx data shows expression in various brain regions, including the whole brain, caudate nucleus, subthalamic nucleus, and medulla oblongata, as well as in fetal thyroid tissue.


The proteins are involved in several key pathways, including the adaptive immune system, the broader immune system, and rap1 signaling.


The mouse ortholog gene 110351 and its human ortholog gene 5909 are associated with the phenotype of increased eosinophil cell number, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.5174962292609354 (33.14th percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -17.793935318872315 (12.78th percentile), and the median structural flexibility is 1.009267857142857 (79.12th percentile). The protein has a high instability index of 58.37512820512838 (79.87th percentile) and an isoelectric point of 5.55211238861084 (19.52nd percentile). The sequence length is 663 amino acids (73.01st percentile) with a molecular weight of 73360.49930000035 Da (72.63rd percentile). The secondary structure affinities are 0.2956259426847662 for helix (28.08th percentile), 0.3197586726998492 for sheet (39.33rd percentile), and 0.3288084464555053 for turn (81.33rd percentile). For more details on sequence analysis, refer to [BioPython's ProtParam documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |