---
layout: post
title: ALDH5A1
date: 2024-12-08 23:57 CST
description: ALDH5A1 description
tags: [cooccuring-genes]
categories: longest
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7915) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7915  | ALDH5A1 | ENSG00000112294 | 6p22.3  |



The gene length is 453,373 base pairs (97.99% of all genes), the mature length is 5,169 base pairs, and the primary transcript length is 449,408 base pairs.


The gene ALDH5A1, with NCBI ID 7915, has been the subject of [39 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22ALDH5A1%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1988, and the middle 50% of publications occurred between 2010 and 2017.


The top five publications mentioning ALDH5A1, ranked by their scientific influence, include "[Understanding the Molecular Mechanisms of Succinic Semialdehyde Dehydrogenase Deficiency (SSADHD): Towards the Development of SSADH-Targeted Medicine.](https://pubmed.ncbi.nlm.nih.gov/35269750)" (2022) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 2.77, "[Disorders of GABA metabolism: SSADH and GABA-transaminase deficiencies.](https://pubmed.ncbi.nlm.nih.gov/25485164)" (2014) with an RCR of 2.08, "[Succinic semialdehyde dehydrogenase: biochemical-molecular-clinical disease mechanisms, redox regulation, and functional significance.](https://pubmed.ncbi.nlm.nih.gov/20973619)" (2011) with an RCR of 1.87, "[Redox-switch modulation of human SSADH by dynamic catalytic loop.](https://pubmed.ncbi.nlm.nih.gov/19300440)" (2009) with an RCR of 1.68, and "[Human brain "high Km" aldehyde dehydrogenase: purification, characterization, and identification as NAD+ -dependent succinic semialdehyde dehydrogenase.](https://pubmed.ncbi.nlm.nih.gov/3190233)" (1988) with an RCR of 1.61. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) and [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476), each with 5 experiments. Additionally, [CDK9](https://www.ncbi.nlm.nih.gov/gene/1025) was found in 4 experiments, while [TBP](https://www.ncbi.nlm.nih.gov/gene/6908) and [EBF1](https://www.ncbi.nlm.nih.gov/gene/1879) were each identified in 3 experiments.











The analyzed protein sequence has a GRAVY value of 0.010467289719626191 (84.43rd percentile), indicating a slight hydrophobicity. The charge at pH 7.0 is 7.037518520984591 (74.53rd percentile), and the median structural flexibility is 0.9977023809523811 (31.13rd percentile). The protein has a helix affinity of 0.34579439252336447 (72.09th percentile), a sheet affinity of 0.33644859813084105 (49.51st percentile), and a turn affinity of 0.2691588785046729 (37.24th percentile). The instability index is 39.01235514018689 (19.48th percentile), and the isoelectric point is 8.621437644958498 (69.28th percentile). The protein has a length of 535 amino acids (62.57th percentile) and a molecular weight of 57214.10820000011 Da (59.79th percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |