---
layout: post
title: CA4
date: 2025-01-17 16:55 CST
description: CA4 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/762) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 762  | CA4 | ENSG00000167434 | 17q23.1 |



The gene enables [carbonate dehydratase activity](https://amigo.geneontology.org/amigo/term/GO:0004089) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is involved in [one-carbon metabolic process](https://amigo.geneontology.org/amigo/term/GO:0006730) and [bicarbonate transport](https://amigo.geneontology.org/amigo/term/GO:0015701). It is located in various cellular components, including the [rough endoplasmic reticulum](https://amigo.geneontology.org/amigo/term/GO:0005791), [endoplasmic reticulum-Golgi intermediate compartment](https://amigo.geneontology.org/amigo/term/GO:0005793), [Golgi apparatus](https://amigo.geneontology.org/amigo/term/GO:0005794), [trans-Golgi network](https://amigo.geneontology.org/amigo/term/GO:0005802), [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886), [external side of plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009897), [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986), [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), [apical plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0016324), [transport vesicle membrane](https://amigo.geneontology.org/amigo/term/GO:0030658), [secretory granule membrane](https://amigo.geneontology.org/amigo/term/GO:0030667), [brush border membrane](https://amigo.geneontology.org/amigo/term/GO:0031526), [perinuclear region of cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0048471), and [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062). Additionally, it enables [zinc ion binding](https://amigo.geneontology.org/amigo/term/GO:0008270).


The gene length is 61,390 base pairs (71.54% of all genes), the mature length is 10,195 base pairs, and the primary transcript length is 61,370.5 base pairs.


The gene CA4 (NCBI ID: 762) has been mentioned in [111 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22CA4%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count), with the earliest publication in 1959 and the middle 50% of publications occurring between 2000 and 2017.


The top 5 publications mentioning CA4, ranked by their scientific influence, include "[Selective reinnervation of hippocampal area CA1 and the fascia dentata after destruction of CA3-CA4 afferents with kainic acid.](https://pubmed.ncbi.nlm.nih.gov/7350980)" (1980) with a [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) of 11.37, "[Carbonic anhydrase IV from human lung. Purification, characterization, and comparison with membrane carbonic anhydrase from human kidney.](https://pubmed.ncbi.nlm.nih.gov/2111324)" (1990) with an RCR of 5.57, "[Herd immunity in adults against influenza-related illnesses with use of the trivalent-live attenuated influenza vaccine (CAIV-T) in children.](https://pubmed.ncbi.nlm.nih.gov/15694506)" (2005) with an RCR of 5.38, "[Phase I trial of the antivascular agent combretastatin A4 phosphate on a 5-day schedule to patients with cancer: magnetic resonance imaging evidence for altered tumor blood flow.](https://pubmed.ncbi.nlm.nih.gov/14645433)" (2003) with an RCR of 5.36, and "[Potent, orally active heterocycle-based combretastatin A-4 analogues: synthesis, structure-activity relationship, pharmacokinetics, and in vivo antitumor activity evaluation.](https://pubmed.ncbi.nlm.nih.gov/11931625)" (2002) with an RCR of 5.2. The Relative Citation Ratio (RCR) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[CA4](https://www.proteinatlas.org/ENSG00000167434-CA4) is a lyase involved in disease variants and is an FDA-approved drug target, particularly relevant to Retinitis pigmentosa. Evidence for this gene is available at the protein level. It is detected in many tissues and is localized in vesicles and the cytosol. CA4 is expressed in neutrophils with unknown function (Cluster 21) and is associated with endocytosis in non-specific tissues (Cluster 80). In the brain, it is involved in DNA binding (Cluster 35), and in cell lines, it is associated with neuronal visual perception (Cluster 1). Single-cell expression data indicates its role in enterocytes for absorption (Cluster 5).


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [TP53](https://www.ncbi.nlm.nih.gov/gene/7157), [SUZ12](https://www.ncbi.nlm.nih.gov/gene/23512), and [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), each with 3 experiments. Additionally, [ZBTB17](https://www.ncbi.nlm.nih.gov/gene/7709) and [FLI1](https://www.ncbi.nlm.nih.gov/gene/2313) were found to be regulating in 2 experiments each.





The gene is ubiquitously expressed in multiple tissue samples, including lung, colon, heart, retina, thyroid, kidney, and fetal lung.


The input data highlights several key processes involving erythrocytes and metabolism. Erythrocytes play a crucial role in gas exchange, taking up carbon dioxide and releasing oxygen, as well as the reverse process of taking up oxygen and releasing carbon dioxide. Additionally, the data mentions the O2/CO2 exchange in erythrocytes and the reversible hydration of carbon dioxide, all of which are integral to metabolic functions.


The mouse ortholog gene 12351 and its human ortholog gene 762 are associated with the phenotype of preweaning lethality with incomplete penetrance, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.344 (52.92nd percentile), indicating a hydrophilic nature. The charge at pH 7.0 is 1.37 (55.14th percentile), and the median structural flexibility is 1.003 (56.8th percentile). The protein has a notable affinity for helix structures (37.5%, 87.04th percentile) and sheet structures (33.7%, 49.56th percentile), with a lower affinity for turn structures (26.6%, 34.55th percentile). The instability index is 44.3 (36.79th percentile), and the isoelectric point is 7.68 (56.13th percentile). The protein has a length of 312 amino acids (30.84th percentile) and a molecular weight of 35031.93 Da (31.21st percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |