---
layout: post
title: RBM8A
date: 2025-01-17 16:55 CST
description: RBM8A description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9939) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9939  | RBM8A | ENSG00000265241 | 1q21.1 |



The gene is involved in various biological processes, including [nuclear-transcribed mRNA catabolic process, nonsense-mediated decay](https://amigo.geneontology.org/amigo/term/GO:0000184), [regulation of alternative mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0000381), and [mRNA splicing, via spliceosome](https://amigo.geneontology.org/amigo/term/GO:0000398). It enables functions such as [RNA binding](https://amigo.geneontology.org/amigo/term/GO:0003723), [mRNA binding](https://amigo.geneontology.org/amigo/term/GO:0003729), and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515). The gene is located in the [nucleus](https://amigo.geneontology.org/amigo/term/GO:0005634), [nucleoplasm](https://amigo.geneontology.org/amigo/term/GO:0005654), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), and [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). Additionally, it is involved in [mRNA export from nucleus](https://amigo.geneontology.org/amigo/term/GO:0006406), [regulation of translation](https://amigo.geneontology.org/amigo/term/GO:0006417), and [RNA splicing](https://amigo.geneontology.org/amigo/term/GO:0008380). The gene is also part of the [exon-exon junction complex](https://amigo.geneontology.org/amigo/term/GO:0035145), [U2-type catalytic step 1 spliceosome](https://amigo.geneontology.org/amigo/term/GO:0071006), and [catalytic step 2 spliceosome](https://amigo.geneontology.org/amigo/term/GO:0071013). Furthermore, it is involved in the [regulation of mRNA processing](https://amigo.geneontology.org/amigo/term/GO:0050684) and the [regulation of nuclear-transcribed mRNA catabolic process, nonsense-mediated decay](https://amigo.geneontology.org/amigo/term/GO:2000622).


The gene length is 5,844 base pairs (16.18% of all genes), the mature length is 1,301.5 base pairs, and the primary transcript length is 4,470.5 base pairs.


The gene RBM8A has been mentioned in [361 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RBM8A%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1875 and the middle 50% of publications occurring between 1995 and 2011. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning RBM8A, ranked by their scientific influence, include "[A novel CDK9-associated C-type cyclin interacts directly with HIV-1 Tat and mediates its high-affinity, loop-specific binding to TAR RNA.](https://pubmed.ncbi.nlm.nih.gov/9491887)" (1998) (relative citation ratio: 18.33), "[Conformation of the TAR RNA-arginine complex by NMR spectroscopy.](https://pubmed.ncbi.nlm.nih.gov/1621097)" (1992) (relative citation ratio: 11.26), "[Cloning and characterization of a novel cellular protein, TDP-43, that binds to human immunodeficiency virus type 1 TAR DNA sequence motifs.](https://pubmed.ncbi.nlm.nih.gov/7745706)" (1995) (relative citation ratio: 10.84), "[Compound inheritance of a low-frequency regulatory SNP and a rare null mutation in exon-junction complex subunit RBM8A causes TAR syndrome.](https://pubmed.ncbi.nlm.nih.gov/22366785)" (2012) (relative citation ratio: 7.48), and "[Overexpression of TAR sequences renders cells resistant to human immunodeficiency virus replication.](https://pubmed.ncbi.nlm.nih.gov/2225067)" (1990) (relative citation ratio: 7.43). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RBM8A](https://www.proteinatlas.org/ENSG00000265241-RBM8A) is an RNA-binding protein with evidence at the protein level. It is detected in all tissues and is primarily localized to nuclear speckles. The gene is expressed in various clusters, including Cluster 29 in blood, Cluster 19 in tissues, Cluster 7 in the brain, and Cluster 45 in cell lines, indicating its involvement in basic cellular processes, ribosomal functions, and nuclear processes. Additionally, single-cell expression data suggest its role in NK-cells and T-cells, highlighting its potential involvement in the immune system and transcription.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [E2F8](https://www.ncbi.nlm.nih.gov/gene/79733) with 6 experiments, [SMARCA4](https://www.ncbi.nlm.nih.gov/gene/6597) with 5 experiments, [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099) with 5 experiments, [NR3C1](https://www.ncbi.nlm.nih.gov/gene/2908) with 5 experiments, and [AR](https://www.ncbi.nlm.nih.gov/gene/367) with 5 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent detection in multiple experiments.





The gene is expressed in muscle and skin tissues according to the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles indicate expression in pineal gland (both day and night), CD34+ cells, BDCA4+ dendritic cells, CD56+ NK cells, CD8+ T cells, amygdala, and fetal brain.


The input data highlights various pathways involved in gene expression and mRNA processing. These include nonsense-mediated decay (NMD), both generally and as enhanced by the exon junction complex (EJC), transport of mature transcripts to the cytoplasm, RNA polymerase II transcription and its termination, post-elongation processing of transcripts, mRNA 3'-end processing, and mRNA splicing, particularly the major pathway. Additionally, the data covers the cleavage of growing transcripts in the termination region and the processing of capped intron-containing pre-mRNA.



The protein sequence analyzed has a GRAVY value of -1.1534482758620694 (2.14% percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -6.014895500891843 (30.96% percentile), and the median structural flexibility is 1.0198928571428572 (96.8% percentile). The protein has a high instability index of 68.74482758620688 (93.81% percentile) and an isoelectric point of 5.500332069396971 (18.22% percentile). The sequence length is 174 amino acids (10.9% percentile) with a molecular weight of 19888.75979999999 (11.31% percentile). The secondary structure affinities are 33.33% for helix (62.61% percentile), 24.14% for sheet (6.76% percentile), and 31.03% for turn (70.79% percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |