---
layout: post
title: PCMT1
date: 2025-01-17 16:55 CST
description: PCMT1 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/5110) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 5110  | PCMT1 | ENSG00000120265 | 6q25.1 |



The gene enables [protein-L-isoaspartate (D-aspartate) O-methyltransferase activity](https://amigo.geneontology.org/amigo/term/GO:0004719) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is involved in [protein methylation](https://amigo.geneontology.org/amigo/term/GO:0006479) and [protein repair](https://amigo.geneontology.org/amigo/term/GO:0030091). It is located in the [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829), [extracellular exosome](https://amigo.geneontology.org/amigo/term/GO:0070062), and [extracellular vesicle](https://amigo.geneontology.org/amigo/term/GO:1903561), and enables [cadherin binding](https://amigo.geneontology.org/amigo/term/GO:0045296).


The gene length is 116,872.0 base pairs (84.74 percentile of all genes), the mature length is 2,899.5 base pairs, and the primary transcript length is 13,570.5 base pairs.


The gene PCMT1 (NCBI ID: 5110) has been mentioned in [43 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22PCMT1%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1992 and the middle 50% of publications occurring between 2010 and 2018. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning PCMT1, ranked by their scientific influence, include "[Genome-wide CRISPR/Cas9 library screen identifies PCMT1 as a critical driver of ovarian cancer metastasis.](https://pubmed.ncbi.nlm.nih.gov/35033172)" (2022) (relative citation ratio: 3.19), "[Differences in Î±-Crystallin isomerization reveal the activity of protein isoaspartyl methyltransferase (PIMT) in the nucleus and cortex of human lenses.](https://pubmed.ncbi.nlm.nih.gov/29571628)" (2018) (relative citation ratio: 2.09), "[Protein isoaspartate methyltransferase-mediated 18O-labeling of isoaspartic acid for mass spectrometry analysis.](https://pubmed.ncbi.nlm.nih.gov/22132761)" (2012) (relative citation ratio: 1.64), "[Damaged proteins bearing L-isoaspartyl residues and aging: a dynamic equilibrium between generation of isomerized forms and repair by PIMT.](https://pubmed.ncbi.nlm.nih.gov/21204776)" (2011) (relative citation ratio: 1.32), and "[Protein L-isoaspartyl methyltransferase regulates p53 activity.](https://pubmed.ncbi.nlm.nih.gov/22735455)" (2012) (relative citation ratio: 1.24). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[PCMT1](https://www.proteinatlas.org/ENSG00000120265-PCMT1) is a gene encoding a protein with methyltransferase and transferase molecular functions, supported by evidence at the protein level. The gene is expressed in all tissues, with its RNA detected across various tissue types. The protein is primarily localized in the cytosol. In terms of expression clusters, PCMT1 is associated with Cluster 29 in blood, indicating non-specific and unknown function, and Cluster 23 in tissues, suggesting involvement in cellular respiration. Notably, in the brain, it is part of Cluster 14, which is linked to neurons and synaptic function. In cell lines, it belongs to Cluster 28, associated with nuclear processes, and in single cells, it is part of Cluster 21, related to early spermatids and spermatogenesis.


The top transcription factors (TFs) identified in terms of the number of CHIP-seq experiments include [CTCFL](https://www.ncbi.nlm.nih.gov/gene/140690), [TRIM28](https://www.ncbi.nlm.nih.gov/gene/10155), [INTS11](https://www.ncbi.nlm.nih.gov/gene/54973), [EP300](https://www.ncbi.nlm.nih.gov/gene/2033), and [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613), each shown to be regulating in 5 experiments.





The gene is expressed in various tissues, including the heart and muscle, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show that the gene is ubiquitously expressed in multiple brain regions, such as the amygdala, whole brain, temporal lobe, parietal lobe, occipital lobe, and globus pallidus.




The analyzed protein sequence has a GRAVY value of -0.167 (72.53rd percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is -0.812 (47.34th percentile), and the median structural flexibility is 1.001 (46.59th percentile). The protein has a predicted affinity for helix, sheet, and turn structures of 33.04% (59.63rd percentile), 32.16% (40.41st percentile), and 31.28% (72.34th percentile), respectively. The instability index is 50.76 (59.78th percentile), and the isoelectric point is 6.70 (46.28th percentile). The protein is 227 amino acids long (18.56th percentile) with a molecular weight of 24636.11 Da (17.5th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |