---
layout: post
title: FCER1A
date: 2025-01-17 16:55 CST
description: FCER1A description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/2205) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 2205  | FCER1A | ENSG00000179639 | 1q23.2 |



The gene enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515) and is located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). It is involved in the [cell surface receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007166) and is active on the [external side of the plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0009897). Additionally, it is located on the [cell surface](https://amigo.geneontology.org/amigo/term/GO:0009986) and plays a role in the [immunoglobulin mediated immune response](https://amigo.geneontology.org/amigo/term/GO:0016064) and [type I hypersensitivity](https://amigo.geneontology.org/amigo/term/GO:0016068). The gene enables [high-affinity IgE receptor activity](https://amigo.geneontology.org/amigo/term/GO:0019768) and [IgE binding](https://amigo.geneontology.org/amigo/term/GO:0019863). It is also involved in the [Fc-epsilon receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0038095), [type 2 immune response](https://amigo.geneontology.org/amigo/term/GO:0042092), [mast cell degranulation](https://amigo.geneontology.org/amigo/term/GO:0043303), and [eosinophil degranulation](https://amigo.geneontology.org/amigo/term/GO:0043308).


The gene length is 26,068 base pairs (48.81% of all genes), the mature length is 1,764.5 base pairs, and the primary transcript length is 24,851.5 base pairs.


The gene FCER1A (NCBI ID: 2205) has been mentioned in [314 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22FCER1A%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1989 and the middle 50% of publications occurring between 1998 and 2012. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning FCER1A, ranked by their scientific influence, include "[The high-affinity IgE receptor (Fc epsilon RI): from physiology to pathology.](https://pubmed.ncbi.nlm.nih.gov/10358778)" (1999) (relative citation ratio: 16.01), "[Down-regulation of Fc(epsilon)RI expression on human basophils during in vivo treatment of atopic patients with anti-IgE antibody.](https://pubmed.ncbi.nlm.nih.gov/9013989)" (1997) (relative citation ratio: 13.6), "[Human epidermal Langerhans cells express the high affinity receptor for immunoglobulin E (Fc epsilon RI).](https://pubmed.ncbi.nlm.nih.gov/1533242)" (1992) (relative citation ratio: 9.43), "[IgE enhances mouse mast cell Fc(epsilon)RI expression in vitro and in vivo: evidence for a novel amplification mechanism in IgE-dependent reactions.](https://pubmed.ncbi.nlm.nih.gov/9034145)" (1997) (relative citation ratio: 7.97), and "[Omalizumab-induced reductions in mast cell Fce psilon RI expression and function.](https://pubmed.ncbi.nlm.nih.gov/15356552)" (2004) (relative citation ratio: 7.96). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[FCER1A](https://www.proteinatlas.org/ENSG00000179639-FCER1A) is an IgE-binding protein and receptor with evidence at the protein level. It is a known FDA-approved drug target and is detected in many tissues. The gene is highly expressed in basophils, where it is involved in proteolysis, and in the skin, where it plays a role in cornification. Additionally, FCER1A is expressed in leukemia cell lines and is involved in hemostasis. In single-cell expression analysis, it is found in dendritic cells, contributing to antigen presentation.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 3 experiments, followed by [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885), [GATA2](https://www.ncbi.nlm.nih.gov/gene/2624), [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861), and [NCOR1](https://www.ncbi.nlm.nih.gov/gene/9611), each with 2 experiments.



The input data indicates associations with a wide range of disease conditions, including hematopoietic system disease, immune system disease, disease of metabolism, acquired metabolic disease, nutrition disease, overnutrition, obesity, coronary artery disease, artery disease, vascular disease, cardiovascular system disease, nervous system disease, central nervous system disease, neurodegenerative disease, synucleinopathy, Parkinson's disease, refractive error, myopia, eye and adnexa disease, eye disease, sensory system disease, myocardial infarction, B cell deficiency, primary immunodeficiency disease, dysgammaglobulinemia, selective immunoglobulin deficiency disease, and selective IgE deficiency disease.



The gene is expressed in the skin, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, according to the GTEx Tissue Gene Expression Profiles, the gene is expressed in various tissues and cell types, including the heart, CD34+ cells, CD33+ myeloid cells, BDCA4+ dendritic cells, CD14+ monocytes, whole blood, and skeletal muscle.


The input data highlights several key pathways within the immune system, specifically focusing on the innate immune system and the signaling pathway of the Fc epsilon receptor (FCERI).



The analyzed protein sequence has a GRAVY value of -0.256 (63.23rd percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -2.08 (42.87th percentile), and the median structural flexibility is 0.996 (22.4th percentile). The protein has a propensity for helix, turn, and sheet structures of 29.96% (31.64th percentile), 28.02% (47.43rd percentile), and 41.63% (90.06th percentile), respectively. The instability index is 33.66 (8.07th percentile), with an isoelectric point of 6.23 (36.29th percentile). The protein is 257 amino acids long (22.6th percentile) and has a molecular weight of 29595.48 Da (23.88th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |