---
layout: post
title: LAMTOR5
date: 2024-12-08 23:57 CST
description: LAMTOR5 description
tags: [cooccuring-genes]
categories: replogle-least
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/10542) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 10542  | LAMTOR5 | ENSG00000134248 | 1p13.3 |



The gene length is 58,407 base pairs (70.29% of all genes), the mature length is 1,256 base pairs (30.74% of all genes), and the primary transcript length is 37,811.5 base pairs (68.52% of all genes).


The gene LAMTOR5, with NCBI ID 10542, has been the subject of [74 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22LAMTOR5%22) (note: count may differ as it is sourced from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727)). The earliest year of publication is 1994, and the middle 50% of publications occurred between 2013 and 2019.


The top five publications mentioning LAMTOR5, ranked by their scientific influence, include "[HBXIP functions as a cofactor of survivin in apoptosis suppression.](https://pubmed.ncbi.nlm.nih.gov/12773388)" (2003) with a relative citation ratio of 6.5, "[HBXIP drives metabolic reprogramming in hepatocellular carcinoma cells via METTL3-mediated m6A modification of HIF-1Î±.](https://pubmed.ncbi.nlm.nih.gov/33305825)" (2021) with a relative citation ratio of 4.69, "[HBXIP and LSD1 Scaffolded by lncRNA Hotair Mediate Transcriptional Activation by c-Myc.](https://pubmed.ncbi.nlm.nih.gov/26719542)" (2016) with a relative citation ratio of 3.32, "[Oncoprotein HBXIP Modulates Abnormal Lipid Metabolism and Growth of Breast Cancer Cells by Activating the LXRs/SREBP-1c/FAS Signaling Cascade.](https://pubmed.ncbi.nlm.nih.gov/26980761)" (2016) with a relative citation ratio of 2.45, and "[HBXIP promotes gastric cancer <i>via</i> METTL3-mediated MYC mRNA m6A modification.](https://pubmed.ncbi.nlm.nih.gov/33048840)" (2020) with a relative citation ratio of 2.41. The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).





The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [KMT2A](https://www.ncbi.nlm.nih.gov/gene/4297) with 5 experiments, [RELA](https://www.ncbi.nlm.nih.gov/gene/5970) with 4 experiments, [NELFA](https://www.ncbi.nlm.nih.gov/gene/7469) with 4 experiments, [SP5](https://www.ncbi.nlm.nih.gov/gene/389058) with 4 experiments, and [PPARG](https://www.ncbi.nlm.nih.gov/gene/5468) with 4 experiments. These TFs have been shown to play significant roles in gene regulation, as evidenced by their frequent appearance in CHIP-seq data.


The top transcription factors (TFs) identified from the GTRD's database of TF binding sites, filtered for Perturb-seq data, include [6829](https://www.ncbi.nlm.nih.gov/gene/6829) and [9968](https://www.ncbi.nlm.nih.gov/gene/9968). These TFs were shown to be regulating genes in 2 and 1 Perturb-seq experiments, respectively.





The mouse ortholog gene 68576, which corresponds to the human ortholog gene 10542, is associated with several phenotypes. These include abnormal skin coloration and preweaning lethality with complete penetrance, both observed in both sexes. Additionally, the gene is linked to embryonic lethality prior to organogenesis, although this phenotype is not observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.022 (82.92nd percentile), indicating a slightly hydrophilic nature. The charge at pH 7.0 is -7.16 (28.05th percentile), and the median structural flexibility is 1.0002 (42.38th percentile). The protein has a propensity for helix, sheet, and turn structures of 31.87% (49.21st percentile), 32.97% (45.27th percentile), and 30.77% (69.1st percentile), respectively. The instability index is 33.59 (7.95th percentile), and the isoelectric point is 4.69 (2.96th percentile). The protein sequence is 91 amino acids long (1.93rd percentile) with a molecular weight of 9613.83 Da (1.58th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).



| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |