---
layout: post
title: UROS
date: 2025-01-17 16:55 CST
description: UROS description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/7390) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 7390  | UROS | ENSG00000188690 | 10q26.2 |



The gene enables [uroporphyrinogen-III synthase activity](https://amigo.geneontology.org/amigo/term/GO:0004852) and [folic acid binding](https://amigo.geneontology.org/amigo/term/GO:0005542). It is located in the [mitochondrion](https://amigo.geneontology.org/amigo/term/GO:0005739) and is active in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829). The gene is involved in various biosynthetic processes, including [uroporphyrinogen III biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006780), [protoporphyrinogen IX biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006782), [heme biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006783), [heme A biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006784), and [heme B biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006785). Additionally, it plays a role in the [heme O biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0048034) and is involved in cellular responses to various stimuli, including [response to platinum ion](https://amigo.geneontology.org/amigo/term/GO:0070541), [cellular response to arsenic-containing substance](https://amigo.geneontology.org/amigo/term/GO:0071243), and [cellular response to amine stimulus](https://amigo.geneontology.org/amigo/term/GO:0071418).


The gene length is 17,940 base pairs (39.42% of all genes), the mature length is 3,979 base pairs, and the primary transcript length is 17,464 base pairs.


The gene UROS (NCBI ID: 7390) has been mentioned in [51 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22UROS%22) according to [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), with the earliest publication in 1967 and the middle 50% of publications occurring between 1996 and 2011. Note that the publication count in the hyperlink may be different since the count is from Pubtator.


The top 5 publications mentioning UROS, ranked by their scientific influence, include "[Human uroporphyrinogen III synthase: molecular cloning, nucleotide sequence, and expression of a full-length cDNA.](https://pubmed.ncbi.nlm.nih.gov/3174619)" (1988) (relative citation ratio: 2.32), "[Congenital erythropoietic porphyria: identification and expression of exonic mutations in the uroporphyrinogen III synthase gene.](https://pubmed.ncbi.nlm.nih.gov/1737856)" (1992) (relative citation ratio: 2.16), "[The purification and properties of uroporphyrinogen I synthases and uroporphyrinogen III cosynthase. Interactions between the enzymes.](https://pubmed.ncbi.nlm.nih.gov/1056172)" (1975) (relative citation ratio: 1.97), "[Point mutations in the uroporphyrinogen III synthase gene in congenital erythropoietic porphyria (GÃ¼nther's disease).](https://pubmed.ncbi.nlm.nih.gov/2331520)" (1990) (relative citation ratio: 1.96), and "[Congenital erythropoietic porphyria: identification and expression of 10 mutations in the uroporphyrinogen III synthase gene.](https://pubmed.ncbi.nlm.nih.gov/7860775)" (1995) (relative citation ratio: 1.87). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[UROS](https://www.proteinatlas.org/ENSG00000188690-UROS) is a gene encoding a protein with lyase molecular function, and it is implicated in disease variants. Evidence for its function is available at the protein level. The gene's RNA is detected in all tissues, with the protein localized in the nucleoplasm and cytosol. UROS is expressed in various clusters, including Cluster 33 for blood (non-specific - ATP binding), Cluster 54 for tissue (non-specific - basic cellular processes), Cluster 33 for brain (neurons - mixed function), Cluster 52 for cell lines (non-specific - mitochondria), and Cluster 1 for single cells (erythroid cells - oxygen transport).


The top transcription factors (TFs) identified from the GTRD database, based on the number of CHIP-seq experiments, include [AR](https://www.ncbi.nlm.nih.gov/gene/367), [OTX2](https://www.ncbi.nlm.nih.gov/gene/5015), [RELA](https://www.ncbi.nlm.nih.gov/gene/5970), [MAX](https://www.ncbi.nlm.nih.gov/gene/4149), and [BRCA1](https://www.ncbi.nlm.nih.gov/gene/672), each shown to be regulating in 5 experiments.


The transcription factor [57634](https://www.ncbi.nlm.nih.gov/gene/57634) was found to be regulating genes in 2 experiments, according to Perturb-seq data from a study published in PubMed (PMID: 35688146). This data was derived from GTRD's database of transcription factor binding sites from uniform CHIP-seq data.


The input data indicates associations with various disease conditions, including several types of cancer such as leukemia, lymphoma, and immune system cancer. Additionally, respiratory conditions like asthma, bronchial disease, and obstructive lung disease are noted. Other significant associations include hematopoietic system disease, immune system disease, and cardiovascular system disease.



The gene is expressed in various tissues, including the heart, brain, and adrenal gland, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in the liver, CD105+ endothelial cells, CD71+ early erythroid cells, whole brain, caudate nucleus, and fetal liver.


The proteins are involved in several key pathways, including heme biosynthesis, metabolism of porphyrins, and general metabolism.


The mouse ortholog gene 22276 and its human ortholog gene 7390 are associated with the phenotype of preweaning lethality with incomplete penetrance, observed in both sexes.


The analyzed protein sequence has a GRAVY value of -0.117 (76.8th percentile), indicating a relatively hydrophilic nature. The charge at pH 7.0 is -6.19 (30.46th percentile), and the median structural flexibility is 1.001 (48.1st percentile). The protein has a notable affinity for helix (36.6%, 83.57th percentile) and sheet (35.1%, 59.84th percentile) structures, with a lower affinity for turns (29.1%, 56.41st percentile). The instability index is 40.4 (23.38th percentile), and the isoelectric point is 5.35 (14.67th percentile). The protein has a length of 265 amino acids (23.78th percentile) and a molecular weight of 28627.36 Da (22.7th percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | yes    |