---
layout: post
title: RAPGEF4
date: 2025-01-17 16:55 CST
description: RAPGEF4 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/11069) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 11069  | RAPGEF4 | ENSG00000091428 | 2q31.1 |



The gene is involved in various biological processes, including the [adaptive immune response](https://amigo.geneontology.org/amigo/term/GO:0002250), [G protein-coupled receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007186), and [Ras protein signal transduction](https://amigo.geneontology.org/amigo/term/GO:0007265). It enables [guanyl-nucleotide exchange factor activity](https://amigo.geneontology.org/amigo/term/GO:0005085) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and is active in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886). Additionally, it is located in the [cytosol](https://amigo.geneontology.org/amigo/term/GO:0005829) and [membrane](https://amigo.geneontology.org/amigo/term/GO:0016020), and is involved in processes such as [calcium-ion regulated exocytosis](https://amigo.geneontology.org/amigo/term/GO:0017156), [regulation of exocytosis](https://amigo.geneontology.org/amigo/term/GO:0017157), and [insulin secretion](https://amigo.geneontology.org/amigo/term/GO:0030073). The gene also enables [cAMP binding](https://amigo.geneontology.org/amigo/term/GO:0030552), [protein-macromolecule adaptor activity](https://amigo.geneontology.org/amigo/term/GO:0030674), and [small GTPase binding](https://amigo.geneontology.org/amigo/term/GO:0031267). Furthermore, it is involved in the [positive regulation of insulin secretion](https://amigo.geneontology.org/amigo/term/GO:0032024) and [positive regulation of GTPase activity](https://amigo.geneontology.org/amigo/term/GO:0043547). The gene is also located in the [hippocampal mossy fiber to CA3 synapse](https://amigo.geneontology.org/amigo/term/GO:0098686) and is involved in the [regulation of synaptic vesicle cycle](https://amigo.geneontology.org/amigo/term/GO:0098693).


The gene length is 96,943 base pairs (81.19% of all genes), the mature length is 2,184 base pairs (57.23% of all genes), and the primary transcript length is 61,242 base pairs (79.76% of all genes).


The gene RAPGEF4 (NCBI ID: 11069) has been mentioned in [52 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22RAPGEF4%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 2000, and the middle 50% of publications occurred between 2009 and 2019.


The top five publications mentioning RAPGEF4, ranked by their scientific influence, include "[cAMP-GEFII is a direct target of cAMP in regulated exocytosis.](https://pubmed.ncbi.nlm.nih.gov/11056535)" (2000) with a relative citation ratio of 7.04, "[Epac: A new cAMP-binding protein in support of glucagon-like peptide-1 receptor-mediated signal transduction in the pancreatic beta-cell.](https://pubmed.ncbi.nlm.nih.gov/14693691)" (2004) with a relative citation ratio of 6.08, "[Structure of the cyclic-AMP-responsive exchange factor Epac2 in its auto-inhibited state.](https://pubmed.ncbi.nlm.nih.gov/16452984)" (2006) with a relative citation ratio of 3.17, "[Piccolo, a Ca2+ sensor in pancreatic beta-cells. Involvement of cAMP-GEFII.Rim2. Piccolo complex in cAMP-dependent exocytosis.](https://pubmed.ncbi.nlm.nih.gov/12401793)" (2002) with a relative citation ratio of 3.0, and "[Structure of Epac2 in complex with a cyclic AMP analogue and RAP1B.](https://pubmed.ncbi.nlm.nih.gov/18660803)" (2008) with a relative citation ratio of 2.98. The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[RAPGEF4](https://www.proteinatlas.org/ENSG00000091428-RAPGEF4) is a guanine-nucleotide releasing factor with evidence at the protein level. It is detected in many tissues and is primarily localized to the plasma membrane, focal adhesion sites, basal body, cytosol, and mid piece. The gene is highly expressed in brain tissues, specifically in neuronal signaling clusters, and is also found in sub-cortical regions with mixed functions. In cell lines, it is associated with lymphoma and inflammatory response clusters. At the single-cell level, RAPGEF4 is expressed in neurons involved in neuronal signaling.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [BRD4](https://www.ncbi.nlm.nih.gov/gene/23476) with 5 experiments, [YY1](https://www.ncbi.nlm.nih.gov/gene/7528) with 4 experiments, [ZBTB7A](https://www.ncbi.nlm.nih.gov/gene/51341) with 4 experiments, [AR](https://www.ncbi.nlm.nih.gov/gene/367) with 4 experiments, and [RUNX1](https://www.ncbi.nlm.nih.gov/gene/861) with 4 experiments.



The input data indicates associations with a wide range of disease conditions, including immune system disease, disease of metabolism, acquired metabolic disease, carbohydrate metabolism disease, glucose metabolism disease, diabetes mellitus, type 2 diabetes mellitus, artery disease, vascular disease, cardiovascular system disease, hypertension, nervous system disease, central nervous system disease, neurodegenerative disease, Alzheimer's disease, tauopathy, dementia, cognitive disorder, disease of mental health, bipolar disorder, mood disorder, amyotrophic lateral sclerosis, motor neuron disease, neuropathy, autoimmune disease of the nervous system, neuromuscular disease, myasthenia gravis, neuromuscular junction disease, hypersensitivity reaction disease, and hypersensitivity reaction type II disease.


The gene RAPGEF4 has been associated with [GLP-1 levels in response to oral glucose tolerance test (fasting)](https://pubmed.ncbi.nlm.nih.gov/29093273), which maps closely to the MeSH term [Glucose](https://meshb.nlm.nih.gov/record/ui?ui=D005947) using the [BioWordVec](https://www.nature.com/articles/s41597-019-0055-0) method.


The gene is ubiquitously expressed in multiple tissue samples, including the brain, adrenal gland, and lung, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in various brain regions such as the Prefrontal Cortex, Amygdala, Cingulate Cortex, Subthalamic Nucleus, Temporal Lobe, Occipital Lobe, and Globus Pallidus.


The proteins are involved in various pathways, including metabolism, adaptive immune system, immune system, glp1 regulation of insulin secretion, integration of energy metabolism, regulation of insulin secretion, signal transduction, rap1 signaling, integrin alphaIIb beta3 signaling, platelet aggregation, hemostasis, and platelet activation, signaling, and aggregation.



The analyzed protein sequence has a GRAVY value of -0.334 (53.91 percentile), indicating a hydrophilic nature. The charge at pH 7.0 is -8.44 (25.41 percentile), and the median structural flexibility is 0.999 (35.55 percentile). The protein has a propensity for helix and sheet structures, with affinities of 0.339 (67.24 percentile) and 0.368 (71.68 percentile), respectively, and a turn affinity of 0.242 (16.37 percentile). The instability index is 42.57 (30.65 percentile), and the isoelectric point is 6.37 (39.76 percentile). The protein is 1011 amino acids long (87.79 percentile) with a molecular weight of 115520.27 Da (88.47 percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |