---
layout: post
title: SLC7A6
date: 2025-01-17 16:55 CST
description: SLC7A6 description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/9057) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 9057  | SLC7A6 | ENSG00000103064 | 16q22.1 |



The gene is involved in [amino acid transmembrane transport](https://amigo.geneontology.org/amigo/term/GO:0003333) and [nitric oxide biosynthetic process](https://amigo.geneontology.org/amigo/term/GO:0006809). It enables [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), [amino acid transmembrane transporter activity](https://amigo.geneontology.org/amigo/term/GO:0015171), [basic amino acid transmembrane transporter activity](https://amigo.geneontology.org/amigo/term/GO:0015174), [L-amino acid transmembrane transporter activity](https://amigo.geneontology.org/amigo/term/GO:0015179), [arginine binding](https://amigo.geneontology.org/amigo/term/GO:0034618), [L-arginine transmembrane transporter activity](https://amigo.geneontology.org/amigo/term/GO:0061459), and [L-lysine:L-arginine antiporter activity](https://amigo.geneontology.org/amigo/term/GO:0106439). Additionally, it is involved in [neutral amino acid transport](https://amigo.geneontology.org/amigo/term/GO:0015804), [leucine transport](https://amigo.geneontology.org/amigo/term/GO:0015820), [ornithine transport](https://amigo.geneontology.org/amigo/term/GO:0015822), [glycine betaine transport](https://amigo.geneontology.org/amigo/term/GO:0031460), [L-lysine transmembrane transport](https://amigo.geneontology.org/amigo/term/GO:1903401), and [L-arginine transmembrane transport](https://amigo.geneontology.org/amigo/term/GO:1903826). The gene is located in the [plasma membrane](https://amigo.geneontology.org/amigo/term/GO:0005886).


The gene length is 185,916.0 base pairs (91.62% of all genes), the mature length is 1,306.5 base pairs, and the primary transcript length is 162,825.0 base pairs.


The gene SLC7A6 (NCBI ID: 9057) has been mentioned in [9 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22SLC7A6%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest publication mentioning this gene was in 1999, and the middle 50% of publications occurred between 2005 and 2020.


The top 5 publications mentioning SLC7A6, ranked by their scientific influence, include "[y+LAT1 and y+LAT2 contribution to arginine uptake in different human cell models: Implications in the pathophysiology of Lysinuric Protein Intolerance.](https://pubmed.ncbi.nlm.nih.gov/31705628)" (2020) (relative citation ratio: 1.75), "[CircSLC7A6 promotes the progression of Wilms' tumor via microRNA-107/ ABL proto-oncogene 2 axis.](https://pubmed.ncbi.nlm.nih.gov/34787058)" (2022) (relative citation ratio: 1.53), "[STAT5A modulates CDYL2/SLC7A6 pathway to inhibit the proliferation and invasion of hepatocellular carcinoma by targeting to mTORC1.](https://pubmed.ncbi.nlm.nih.gov/35314791)" (2022) (relative citation ratio: 1.24), "[Circular RNA circ-SLC7A6 acts as a tumor suppressor in non-small cell lung cancer through abundantly sponging miR-21.](https://pubmed.ncbi.nlm.nih.gov/32794418)" (2020) (relative citation ratio: 0.94), and "[Association of 4F2hc with light chains LAT1, LAT2 or y+LAT2 requires different domains.](https://pubmed.ncbi.nlm.nih.gov/11311135)" (2001) (relative citation ratio: 0.83). The relative citation ratio ([RCR](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541)) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[SLC7A6](https://www.proteinatlas.org/ENSG00000103064-SLC7A6) is a gene with evidence at the protein level, detected in all RNA tissue distributions. It is primarily localized in vesicles and the plasma membrane. The gene is expressed in various clusters, including Cluster 30 for blood expression, Cluster 19 for tissue expression, Cluster 23 for brain expression, Cluster 68 for cell line expression, and Cluster 78 for single cell expression, which is associated with NK-cells and T-cells in the immune system and transcription.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [TBP](https://www.ncbi.nlm.nih.gov/gene/6908) and [MYCN](https://www.ncbi.nlm.nih.gov/gene/4613), each with 6 experiments. Additionally, [KDM5B](https://www.ncbi.nlm.nih.gov/gene/10765), [GATA2](https://www.ncbi.nlm.nih.gov/gene/2624), and [JUN](https://www.ncbi.nlm.nih.gov/gene/3725) were each found to be regulating in 5 experiments.





The gene is expressed in various tissues and cell types, including muscle, adipose tissue, and nerve, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in BDCA4+ dendritic cells, CD4+ T cells, CD8+ T cells, Burkitt's lymphoma (Daudi), adipocytes, thymus, and lymph nodes.


The proteins are involved in various transport processes, including amino acid and oligopeptide transport via SLC transporters, amino acid transport across the plasma membrane, and SLC-mediated transmembrane transport. Additionally, they participate in the transport of inorganic cations/anions and amino acids/oligopeptides, as well as the transmembrane transport of small molecules. These proteins also play roles in basigin interactions, cell surface interactions at the vascular wall, and hemostasis.



The analyzed protein sequence has a GRAVY value of 0.572 (96.1% percentile), indicating a high level of hydrophobicity. The protein exhibits a charge of -6.055 at pH 7.0 (30.86% percentile) and a median structural flexibility of 0.979 (4.19% percentile). The secondary structure affinities are 28.16% for helix (17.93% percentile), 46.99% for sheet (96.4% percentile), and 26.41% for turn (32.83% percentile). The instability index is 40.24 (22.97% percentile), with an isoelectric point of 5.63 (21.23% percentile). The protein has a length of 515 amino acids (60.07% percentile) and a molecular weight of 56826.84 Da (59.34% percentile). These calculations were performed using [BioPython's sequence analysis tools](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |