---
layout: post
title: BDNF
date: 2025-01-17 16:55 CST
description: BDNF description
tags: [cooccuring-genes]
categories: unknown-mechanism
pretty_table: true
citation: true
related_posts: true
---



| [NCBI ID](https://www.ncbi.nlm.nih.gov/gene/627) | NCBI symbol | ENSEMBL ID | Chromosome location |
| :-------- | :------- | :-------- | :------- |
| 627  | BDNF | ENSG00000176697 | 11p14.1 |



The gene enables [nerve growth factor receptor binding](https://amigo.geneontology.org/amigo/term/GO:0005163) and [protein binding](https://amigo.geneontology.org/amigo/term/GO:0005515), and exhibits [growth factor activity](https://amigo.geneontology.org/amigo/term/GO:0008083). It is located in the [extracellular region](https://amigo.geneontology.org/amigo/term/GO:0005576), [cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0005737), [endoplasmic reticulum lumen](https://amigo.geneontology.org/amigo/term/GO:0005788), and the [perinuclear region of cytoplasm](https://amigo.geneontology.org/amigo/term/GO:0048471). The gene is active in the [extracellular space](https://amigo.geneontology.org/amigo/term/GO:0005615), [synaptic vesicle](https://amigo.geneontology.org/amigo/term/GO:0008021), [axon](https://amigo.geneontology.org/amigo/term/GO:0030424), and [dendrite](https://amigo.geneontology.org/amigo/term/GO:0030425). It is involved in various processes including the [transmembrane receptor protein tyrosine kinase signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0007169), [nervous system development](https://amigo.geneontology.org/amigo/term/GO:0007399), [axon guidance](https://amigo.geneontology.org/amigo/term/GO:0007411), [synapse assembly](https://amigo.geneontology.org/amigo/term/GO:0007416), [peripheral nervous system development](https://amigo.geneontology.org/amigo/term/GO:0007422), [memory](https://amigo.geneontology.org/amigo/term/GO:0007613), [negative regulation of myotube differentiation](https://amigo.geneontology.org/amigo/term/GO:0010832), [positive regulation of neuron projection development](https://amigo.geneontology.org/amigo/term/GO:0010976), [nerve development](https://amigo.geneontology.org/amigo/term/GO:0021675), [brain-derived neurotrophic factor receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0031547), [positive regulation of brain-derived neurotrophic factor receptor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0031550), [positive regulation of peptidyl-serine phosphorylation](https://amigo.geneontology.org/amigo/term/GO:0033138), [nerve growth factor signaling pathway](https://amigo.geneontology.org/amigo/term/GO:0038180), [negative regulation of neuron apoptotic process](https://amigo.geneontology.org/amigo/term/GO:0043524), [regulation of neuron differentiation](https://amigo.geneontology.org/amigo/term/GO:0045664), [collateral sprouting](https://amigo.geneontology.org/amigo/term/GO:0048668), [positive regulation of collateral sprouting](https://amigo.geneontology.org/amigo/term/GO:0048672), [neuron projection morphogenesis](https://amigo.geneontology.org/amigo/term/GO:0048812), [modulation of chemical synaptic transmission](https://amigo.geneontology.org/amigo/term/GO:0050804), [positive regulation of synapse assembly](https://amigo.geneontology.org/amigo/term/GO:0051965), [positive regulation of receptor binding](https://amigo.geneontology.org/amigo/term/GO:1900122), [regulation of protein localization to cell surface](https://amigo.geneontology.org/amigo/term/GO:2000008), and [negative regulation of apoptotic signaling pathway](https://amigo.geneontology.org/amigo/term/GO:2001234).


The gene length is 8,583 base pairs (22.61% of all genes), the mature length is 825.5 base pairs, and the primary transcript length is 4,894 base pairs.


The gene BDNF (NCBI ID: 627) has been mentioned in [6,001 publications](https://pubmed.ncbi.nlm.nih.gov/?term=%22BDNF%22) (count from [Pubtator](https://academic.oup.com/nar/article/47/W1/W587/5494727), may differ from the hyperlink count). The earliest year of publication is 1986, and the middle 50% of publications occurred between 2009 and 2018.


The top five publications mentioning BDNF, ranked by their scientific influence, include "[The ALSFRS-R: a revised ALS functional rating scale that incorporates assessments of respiratory function. BDNF ALS Study Group (Phase III).](https://pubmed.ncbi.nlm.nih.gov/10540002)" (1999) (relative citation ratio: 61.65), "[Neurotrophin regulation of neural circuit development and function.](https://pubmed.ncbi.nlm.nih.gov/23254191)" (2013) (relative citation ratio: 44.66), "[Brain-Derived Neurotrophic Factor: A Key Molecule for Memory in the Healthy and the Pathological Brain.](https://pubmed.ncbi.nlm.nih.gov/31440144)" (2019) (relative citation ratio: 44.45), "[BDNF: A Key Factor with Multipotent Impact on Brain Signaling and Synaptic Plasticity.](https://pubmed.ncbi.nlm.nih.gov/28623429)" (2018) (relative citation ratio: 40.34), and "[Neurotrophin-regulated signalling pathways.](https://pubmed.ncbi.nlm.nih.gov/16939974)" (2006) (relative citation ratio: 35.82). The [Relative Citation Ratio (RCR)](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002541) measures the scientific influence of each paper by field- and time-adjusting the citations it has received, benchmarking to the median for NIH publications, which is set at 1.0. Citation counts are sourced from [iCite](https://icite.od.nih.gov).


[BDNF](https://www.proteinatlas.org/ENSG00000176697-BDNF) is a growth factor involved in cancer-related genes and disease variants, with evidence at the protein level. Its RNA is detected in some tissues, and it is notably expressed in heart muscle and development (Cluster 30), forebrain with mixed function (Cluster 29), connective tissue cells involved in ECM organization (Cluster 64), and non-specific cells with mixed function (Cluster 15). While BDNF is unprognostic for several cancer types, including Breast Invasive Carcinoma, Glioblastoma Multiforme, Kidney Renal Clear Cell Carcinoma, Kidney Renal Papillary Cell Carcinoma, Lung Squamous Cell Carcinoma, Skin Cutaneous Melanoma, and Thyroid Carcinoma, its role in these diseases warrants further investigation.


The top transcription factors (TFs) identified from the GTRD database of TF binding sites, based on the number of CHIP-seq experiments, include [RAD21](https://www.ncbi.nlm.nih.gov/gene/5885) with 6 experiments, followed by [ESR1](https://www.ncbi.nlm.nih.gov/gene/2099), [E2F1](https://www.ncbi.nlm.nih.gov/gene/1869), [TP53](https://www.ncbi.nlm.nih.gov/gene/7157), and [TRPS1](https://www.ncbi.nlm.nih.gov/gene/7227), each with 5 experiments.



The GWAS data indicates associations with a range of disease conditions, including infectious diseases such as those caused by viral agents and specifically human immunodeficiency virus. Additionally, there are links to various types of cancer, particularly lung and respiratory system cancers. Metabolic disorders, including obesity and acquired metabolic diseases, are also implicated. Furthermore, the data highlights associations with mental health conditions, including substance-related disorders like nicotine dependence, as well as developmental disorders and eating disorders.



The gene is expressed in various tissues, including blood vessel, prostate, and brain, as indicated by the BioGPS Human Cell Type and Tissue Gene Expression Profiles. Additionally, the GTEx Tissue Gene Expression Profiles show expression in Cardiac Myocytes, Amygdala, Hypothalamus, Cerebellum, Cerebellum Peduncles, Smooth Muscle, and Globus Pallidus.




The analyzed protein sequence has a GRAVY value of -0.456 (39.66th percentile), indicating a hydrophilic nature. It exhibits a charge of 6.81 at pH 7.0 (73.71st percentile) and a median structural flexibility of 1.003 (55.9th percentile). The protein's secondary structure is predicted to be 31.58% helix (46.57th percentile), 34.41% sheet (54.96th percentile), and 27.53% turn (43.02nd percentile). The instability index is 43.34 (33.4th percentile), with an isoelectric point of 9.01 (79.63rd percentile). The protein has a length of 247 amino acids (21.24th percentile) and a molecular weight of 27817.65 Da (21.63rd percentile). For more details on the sequence analysis, refer to the [BioPython documentation](https://biopython.org/docs/1.75/api/Bio.SeqUtils.ProtParam.html).





| Checks    | Answer |
| :-------- | :------- |
| Sentiment  | neutral   |
| Ethical concerns | no     |
| Coherence    | no    |